{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Loading EDAM into an RDFlib graph"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "from rdflib import ConjunctiveGraph"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Here we initialize the graph. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "kg = ConjunctiveGraph()\n",
    "\n",
    "def print_size():\n",
    "    print(f\"The knowledge graph has {len(kg)} triples\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Here we load the EDAM ontology into the graph. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The knowledge graph has 36884 triples\n"
     ]
    }
   ],
   "source": [
    "kg.load('http://edamontology.org/EDAM.owl', format='xml')\n",
    "print_size()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<Graph identifier=N9aaf1d9771d44cc4bcfd59fde9d6ce18 (<class 'rdflib.graph.ConjunctiveGraph'>)>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "kg.serialize(\"edam.json\", format=\"json-ld\")\n",
    "kg.serialize(\"edam.ttl\", format=\"turtle\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "36884\n"
     ]
    }
   ],
   "source": [
    "# a single function to load EDAM and get the graph object as a result\n",
    "def load_EDAM():\n",
    "    g = ConjunctiveGraph()\n",
    "    g.load('http://edamontology.org/EDAM.owl', format='xml')\n",
    "    return g\n",
    "\n",
    "G = load_EDAM()\n",
    "print(len(G))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Listing the 100 first triples  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(http://edamontology.org/topic_0659, http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym, Non-coding RNA)\n",
      "(http://edamontology.org/operation_2222, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://www.w3.org/2002/07/owl#DeprecatedClass)\n",
      "(http://edamontology.org/operation_0316, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/operation_0478, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Model the structure of a protein in complex with a small molecule or another macromolecule.)\n",
      "(http://edamontology.org/data_1235, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#edam)\n",
      "(http://edamontology.org/operation_2405, http://www.w3.org/2000/01/rdf-schema#label, Protein interaction data processing)\n",
      "(http://edamontology.org/data_1384, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#data)\n",
      "(http://edamontology.org/data_2128, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Informal name for a genetic code, typically an organism name.)\n",
      "(http://edamontology.org/data_1251, http://www.geneontology.org/formats/oboInOwl#consider, http://edamontology.org/data_1249)\n",
      "(http://edamontology.org/format_3312, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Format for the Genetic Data Environment (GDE).)\n",
      "(http://edamontology.org/topic_2640, http://www.geneontology.org/formats/oboInOwl#hasDefinition, The study of cancer, for example, genes and proteins implicated in cancer.)\n",
      "(http://edamontology.org/operation_2514, http://www.w3.org/2000/01/rdf-schema#label, Sequence generation (protein))\n",
      "(http://edamontology.org/topic_3043, http://www.w3.org/2000/01/rdf-schema#label, Protein sequences)\n",
      "(http://edamontology.org/topic_0176, http://edamontology.org/created_in, beta12orEarlier)\n",
      "(http://edamontology.org/format_3980, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#edam)\n",
      "(http://edamontology.org/topic_0749, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/operation_3904, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Identify or predict intrinsically disordered regions in proteins.)\n",
      "(http://edamontology.org/format_3747, http://www.geneontology.org/formats/oboInOwl#hasDefinition, A format for storage, exchange, and processing of protein identifications created from ms/ms-derived peptide sequence data.)\n",
      "(http://edamontology.org/format_3160, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#edam)\n",
      "(http://edamontology.org/data_1116, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Identifier of an entry from the ELMdb database of protein functional sites.)\n",
      "(http://edamontology.org/data_3154, http://www.geneontology.org/formats/oboInOwl#consider, http://edamontology.org/data_0878)\n",
      "(Nf826c16134ba4aac92e32df4a1b08020, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Restriction)\n",
      "(http://edamontology.org/data_2689, http://www.geneontology.org/formats/oboInOwl#consider, http://edamontology.org/data_2610)\n",
      "(Nfdaa6b9d828e4622bb8116ac3ca92d9f, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Restriction)\n",
      "(http://edamontology.org/data_3145, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://www.w3.org/2002/07/owl#DeprecatedClass)\n",
      "(http://edamontology.org/operation_3203, http://edamontology.org/created_in, 1.1)\n",
      "(http://edamontology.org/data_1414, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#obsolete)\n",
      "(http://edamontology.org/operation_0277, http://www.geneontology.org/formats/oboInOwl#consider, http://edamontology.org/operation_3928)\n",
      "(Nf94d34041daf46b4b66edeed6ea68eed, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Restriction)\n",
      "(http://edamontology.org/operation_3218, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#operations)\n",
      "(http://edamontology.org/topic_0110, http://www.geneontology.org/formats/oboInOwl#hasDefinition, The transcription of DNA into mRNA.)\n",
      "(http://edamontology.org/topic_3304, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/topic_3344)\n",
      "(http://edamontology.org/topic_3407, http://edamontology.org/isdebtag, true)\n",
      "(Nf412bb2b187f41a1b98165d15d333929, http://www.w3.org/2002/07/owl#someValuesFrom, http://edamontology.org/data_0881)\n",
      "(http://edamontology.org/data_0932, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/operation_3744, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#operations)\n",
      "(N46e8acb16cb94b04bf05b5ef9d17723f, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Restriction)\n",
      "(http://edamontology.org/data_2218, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_2381, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#obsolete)\n",
      "(http://edamontology.org/data_2395, http://www.w3.org/2002/07/owl#deprecated, true)\n",
      "(http://edamontology.org/format_3879, http://edamontology.org/notRecommendedForAnnotation, true)\n",
      "(http://edamontology.org/format_3508, http://www.w3.org/2000/01/rdf-schema#label, PDF)\n",
      "(http://edamontology.org/operation_0480, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#edam)\n",
      "(http://edamontology.org/topic_0160, http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym, Functional sites)\n",
      "(http://edamontology.org/operation_3186, http://www.geneontology.org/formats/oboInOwl#hasDefinition, The mapping of methylation sites in a DNA (genome) sequence.  Typically, the mapping of high-throughput bisulfite reads to the reference genome.)\n",
      "(http://edamontology.org/data_1068, http://www.w3.org/2000/01/rdf-schema#label, Phylogenetic tree ID)\n",
      "(http://edamontology.org/format_1516, http://www.w3.org/2002/07/owl#deprecated, true)\n",
      "(http://edamontology.org/format_2195, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Format used for ontologies.)\n",
      "(http://edamontology.org/format_1582, http://www.geneontology.org/formats/oboInOwl#hasDefinition, A report format for the kinetics of enzyme-catalysed reaction(s) in a format generated by EMBOSS findkm. This includes Michaelis Menten plot, Hanes Woolf plot, Michaelis Menten constant (Km) and maximum velocity (Vmax).)\n",
      "(N8759aaf8e9564d5ba5e73ea09e35c054, http://www.w3.org/2002/07/owl#someValuesFrom, http://edamontology.org/data_2968)\n",
      "(http://edamontology.org/data_0850, http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym, Alignment reference)\n",
      "(http://edamontology.org/format_2182, http://www.w3.org/2000/01/rdf-schema#comment, This concept may be used for non-standard FASTQ short read-like formats.)\n",
      "(http://edamontology.org/format_1577, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://www.w3.org/2002/07/owl#DeprecatedClass)\n",
      "(http://edamontology.org/data_1545, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_2157, http://www.geneontology.org/formats/oboInOwl#consider, http://edamontology.org/data_1268)\n",
      "(http://edamontology.org/data_2587, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Unique identifier of a blot from a Northern Blot.)\n",
      "(http://edamontology.org/data_2784, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/operation_1814, http://www.geneontology.org/formats/oboInOwl#hasDbXref, WHATIF:DownloadPDB)\n",
      "(http://edamontology.org/format_3878, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#edam)\n",
      "(http://edamontology.org/topic_3422, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#topics)\n",
      "(http://edamontology.org/format_3484, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/format_3326)\n",
      "(N42bbe33b31654cb1a11636a169f3d0e7, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Restriction)\n",
      "(http://edamontology.org/data_2285, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_1744, http://edamontology.org/oldParent, http://edamontology.org/data_1743)\n",
      "(http://edamontology.org/data_3238, http://edamontology.org/regex, CL_[0-9]{7})\n",
      "(http://edamontology.org/data_1587, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#obsolete)\n",
      "(N92b3fdbad40c4de98a6f1281be532781, http://www.w3.org/2002/07/owl#someValuesFrom, http://edamontology.org/data_1597)\n",
      "(http://edamontology.org/topic_2259, http://edamontology.org/isdebtag, true)\n",
      "(http://edamontology.org/data_0853, http://www.geneontology.org/formats/oboInOwl#hasDefinition, The strand of a DNA sequence (forward or reverse).)\n",
      "(http://edamontology.org/data_1260, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#data)\n",
      "(http://edamontology.org/topic_0623, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#events)\n",
      "(http://edamontology.org/format_2095, http://www.w3.org/2000/01/rdf-schema#label, unpure)\n",
      "(http://edamontology.org/operation_2485, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#edam)\n",
      "(http://edamontology.org/data_2797, http://www.w3.org/2000/01/rdf-schema#label, Protein ID (LGICdb))\n",
      "(http://edamontology.org/data_3807, http://edamontology.org/created_in, 1.19)\n",
      "(http://edamontology.org/format_1214, http://www.w3.org/2000/01/rdf-schema#label, unambiguous pure dna)\n",
      "(http://edamontology.org/operation_2491, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://www.w3.org/2002/07/owl#DeprecatedClass)\n",
      "(http://edamontology.org/data_2157, http://www.w3.org/2000/01/rdf-schema#label, Word composition)\n",
      "(http://edamontology.org/data_2925, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#obsolete)\n",
      "(http://edamontology.org/data_2317, http://www.w3.org/2000/01/rdf-schema#label, Cell line name (exact))\n",
      "(http://edamontology.org/operation_0289, http://www.w3.org/2000/01/rdf-schema#subClassOf, N9b5b6324d5b946199506e391e4cf4f16)\n",
      "(http://edamontology.org/topic_3050, http://www.geneontology.org/formats/oboInOwl#hasDbXref, VT 1.5.5 Biodiversity conservation)\n",
      "(http://edamontology.org/data_2134, http://www.w3.org/2002/07/owl#deprecated, true)\n",
      "(http://edamontology.org/operation_0267, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#operations)\n",
      "(http://edamontology.org/data_0906, http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym, Protein non-covalent interactions report)\n",
      "(http://edamontology.org/operation_1835, http://www.w3.org/2000/01/rdf-schema#label, Residue contact calculation (residue-negative ion))\n",
      "(http://edamontology.org/operation_3798, http://www.w3.org/2000/01/rdf-schema#comment, Binning methods use one or a combination of compositional features or sequence similarity.)\n",
      "(http://edamontology.org/data_1140, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_1301, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://www.w3.org/2002/07/owl#DeprecatedClass)\n",
      "(http://edamontology.org/operation_0420, http://www.geneontology.org/formats/oboInOwl#hasExactSynonym, Protein-nucleic acid binding prediction)\n",
      "(http://edamontology.org/data_1064, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#identifiers)\n",
      "(http://edamontology.org/data_1139, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/data_2910)\n",
      "(http://edamontology.org/has_output, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#ObjectProperty)\n",
      "(http://edamontology.org/operation_3198, http://www.geneontology.org/formats/oboInOwl#hasExactSynonym, Oligonucleotide alignment)\n",
      "(http://edamontology.org/data_1170, http://edamontology.org/created_in, beta12orEarlier)\n",
      "(http://edamontology.org/operation_2457, http://edamontology.org/created_in, beta12orEarlier)\n",
      "(http://edamontology.org/data_2091, http://www.geneontology.org/formats/oboInOwl#inSubset, http://purl.obolibrary.org/obo/edam#edam)\n",
      "(http://edamontology.org/data_3732, http://edamontology.org/created_in, 1.15)\n",
      "(http://edamontology.org/topic_0176, http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym, Protein dynamics)\n",
      "(http://edamontology.org/format_1915, http://www.w3.org/2000/01/rdf-schema#seeAlso, \"http://purl.org/dc/elements/1.1/format\")\n"
     ]
    }
   ],
   "source": [
    "i = 0\n",
    "\n",
    "for subject,predicate,obj in kg:\n",
    "    print(f'({subject}, {predicate}, {obj})')\n",
    "    i+=1\n",
    "    \n",
    "    if i > 99:\n",
    "        break\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "from rdflib.namespace import RDF, RDFS, OWL \n",
    "\n",
    "i = 0\n",
    "\n",
    "\n",
    "for s in kg.triples((None, RDF.type, OWL.Class)):\n",
    "    for label in kg.triples((s, RDFS.label, None)):\n",
    "        print(kg.value(s, RDFS.label))\n",
    "        i +=1\n",
    "    \n",
    "    if i > 99:\n",
    "        break\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Evaluating a basic SPARQL query\n",
    "Aim: printing the label and the URI of the direct subclasses of the generic \"EDAM operation\" concept."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Annotation is identified in EDAM with concept http://edamontology.org/operation_0226\n",
      "Indexing is identified in EDAM with concept http://edamontology.org/operation_0227\n",
      "Visualisation is identified in EDAM with concept http://edamontology.org/operation_0337\n",
      "Data handling is identified in EDAM with concept http://edamontology.org/operation_2409\n",
      "Prediction and recognition is identified in EDAM with concept http://edamontology.org/operation_2423\n",
      "Comparison is identified in EDAM with concept http://edamontology.org/operation_2424\n",
      "Optimisation and refinement is identified in EDAM with concept http://edamontology.org/operation_2425\n",
      "Modelling and simulation is identified in EDAM with concept http://edamontology.org/operation_2426\n",
      "Validation is identified in EDAM with concept http://edamontology.org/operation_2428\n",
      "Mapping is identified in EDAM with concept http://edamontology.org/operation_2429\n",
      "Design is identified in EDAM with concept http://edamontology.org/operation_2430\n",
      "Alignment is identified in EDAM with concept http://edamontology.org/operation_2928\n",
      "Analysis is identified in EDAM with concept http://edamontology.org/operation_2945\n",
      "Classification is identified in EDAM with concept http://edamontology.org/operation_2990\n",
      "Generation is identified in EDAM with concept http://edamontology.org/operation_3429\n",
      "Clustering is identified in EDAM with concept http://edamontology.org/operation_3432\n",
      "Conversion is identified in EDAM with concept http://edamontology.org/operation_3434\n",
      "Calculation is identified in EDAM with concept http://edamontology.org/operation_3438\n",
      "Correlation is identified in EDAM with concept http://edamontology.org/operation_3465\n",
      "Service management is identified in EDAM with concept http://edamontology.org/operation_3760\n",
      "Quantification is identified in EDAM with concept http://edamontology.org/operation_3799\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "SELECT ?x ?label WHERE {\n",
    "    ?x rdfs:subClassOf <http://edamontology.org/operation_0004> . \n",
    "    ?x rdfs:label ?label .\n",
    "}\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"{r['label']} is identified in EDAM with concept {r['x']}\") "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Evaluating another SPARQL query\n",
    "Aim: printing the label and the URI of all direct and indirect subclasses of *Alignment* operations (Class <http://edamontology.org/operation_2928>)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sequence alignment is identified is a kind of alignment operation\n",
      "Structure-based sequence alignment is identified is a kind of alignment operation\n",
      "Sequence profile alignment is identified is a kind of alignment operation\n",
      "Pairwise sequence alignment is identified is a kind of alignment operation\n",
      "Multiple sequence alignment is identified is a kind of alignment operation\n",
      "Local alignment is identified is a kind of alignment operation\n",
      "Global alignment is identified is a kind of alignment operation\n",
      "Tree-based sequence alignment is identified is a kind of alignment operation\n",
      "Genome alignment is identified is a kind of alignment operation\n",
      "Structure alignment is identified is a kind of alignment operation\n",
      "Pairwise structure alignment is identified is a kind of alignment operation\n",
      "Multiple structure alignment is identified is a kind of alignment operation\n",
      "Local structure alignment is identified is a kind of alignment operation\n",
      "Global structure alignment is identified is a kind of alignment operation\n",
      "Fold recognition is identified is a kind of alignment operation\n",
      "Protein threading is identified is a kind of alignment operation\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "SELECT ?x ?label WHERE {\n",
    "    ?x rdfs:subClassOf+ <http://edamontology.org/operation_2928> . \n",
    "    ?x rdfs:label ?label .\n",
    "}\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"{r['label']} is identified is a kind of alignment operation\") "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Sort EDAM operations "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Fold recognition : http://edamontology.org/operation_0303\n",
      "Genome alignment : http://edamontology.org/operation_3182\n",
      "Global alignment : http://edamontology.org/operation_0496\n",
      "Global structure alignment : http://edamontology.org/operation_0510\n",
      "Local alignment : http://edamontology.org/operation_0495\n",
      "Local structure alignment : http://edamontology.org/operation_0509\n",
      "Multiple sequence alignment : http://edamontology.org/operation_0492\n",
      "Multiple structure alignment : http://edamontology.org/operation_0504\n",
      "Pairwise sequence alignment : http://edamontology.org/operation_0491\n",
      "Pairwise structure alignment : http://edamontology.org/operation_0503\n",
      "Protein threading : http://edamontology.org/operation_0302\n",
      "Sequence alignment : http://edamontology.org/operation_0292\n",
      "Sequence profile alignment : http://edamontology.org/operation_0300\n",
      "Structure alignment : http://edamontology.org/operation_0295\n",
      "Structure-based sequence alignment : http://edamontology.org/operation_0294\n",
      "Tree-based sequence alignment : http://edamontology.org/operation_0499\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "SELECT ?x ?label WHERE {\n",
    "    ?x rdfs:subClassOf+ <http://edamontology.org/operation_2928> . \n",
    "    ?x rdfs:label ?label .\n",
    "}\n",
    "ORDER BY ASC(str(?label))\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"{r['label']} : {r['x']}\") "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Operation : http://edamontology.org/operation_0004\n",
      "Query and retrieval : http://edamontology.org/operation_0224\n",
      "Annotation : http://edamontology.org/operation_0226\n",
      "Indexing : http://edamontology.org/operation_0227\n",
      "Sequence generation : http://edamontology.org/operation_0230\n",
      "Sequence editing : http://edamontology.org/operation_0231\n",
      "Sequence merging : http://edamontology.org/operation_0232\n",
      "Sequence conversion : http://edamontology.org/operation_0233\n",
      "Sequence complexity calculation : http://edamontology.org/operation_0234\n",
      "Sequence ambiguity calculation : http://edamontology.org/operation_0235\n",
      "Sequence composition calculation : http://edamontology.org/operation_0236\n",
      "Repeat sequence analysis : http://edamontology.org/operation_0237\n",
      "Sequence motif discovery : http://edamontology.org/operation_0238\n",
      "Sequence motif recognition : http://edamontology.org/operation_0239\n",
      "Sequence motif comparison : http://edamontology.org/operation_0240\n",
      "Simulation analysis : http://edamontology.org/operation_0244\n",
      "Structural motif discovery : http://edamontology.org/operation_0245\n",
      "Protein domain recognition : http://edamontology.org/operation_0246\n",
      "Protein architecture analysis : http://edamontology.org/operation_0247\n",
      "Residue interaction calculation : http://edamontology.org/operation_0248\n",
      "Protein geometry calculation : http://edamontology.org/operation_0249\n",
      "Protein property calculation : http://edamontology.org/operation_0250\n",
      "Peptide immunogenicity prediction : http://edamontology.org/operation_0252\n",
      "Sequence feature detection : http://edamontology.org/operation_0253\n",
      "Sequence feature comparison : http://edamontology.org/operation_0256\n",
      "Sequence alignment analysis : http://edamontology.org/operation_0258\n",
      "Sequence alignment comparison : http://edamontology.org/operation_0259\n",
      "Sequence alignment conversion : http://edamontology.org/operation_0260\n",
      "Nucleic acid property calculation : http://edamontology.org/operation_0262\n",
      "Alternative splicing prediction : http://edamontology.org/operation_0264\n",
      "Frameshift detection : http://edamontology.org/operation_0265\n",
      "Vector sequence detection : http://edamontology.org/operation_0266\n",
      "Protein secondary structure prediction : http://edamontology.org/operation_0267\n",
      "Protein super-secondary structure prediction : http://edamontology.org/operation_0268\n",
      "Transmembrane protein prediction : http://edamontology.org/operation_0269\n",
      "Transmembrane protein analysis : http://edamontology.org/operation_0270\n",
      "Residue contact prediction : http://edamontology.org/operation_0272\n",
      "Protein interaction network analysis : http://edamontology.org/operation_0276\n",
      "RNA secondary structure prediction : http://edamontology.org/operation_0278\n",
      "Nucleic acid folding analysis : http://edamontology.org/operation_0279\n",
      "Genetic mapping : http://edamontology.org/operation_0282\n",
      "Linkage analysis : http://edamontology.org/operation_0283\n",
      "Codon usage table generation : http://edamontology.org/operation_0284\n",
      "Codon usage table comparison : http://edamontology.org/operation_0285\n",
      "Codon usage analysis : http://edamontology.org/operation_0286\n",
      "Base position variability plotting : http://edamontology.org/operation_0287\n",
      "Sequence word comparison : http://edamontology.org/operation_0288\n",
      "Sequence distance matrix generation : http://edamontology.org/operation_0289\n",
      "Sequence redundancy removal : http://edamontology.org/operation_0290\n",
      "Sequence clustering : http://edamontology.org/operation_0291\n",
      "Sequence alignment : http://edamontology.org/operation_0292\n",
      "Structure-based sequence alignment : http://edamontology.org/operation_0294\n",
      "Structure alignment : http://edamontology.org/operation_0295\n",
      "Sequence profile generation : http://edamontology.org/operation_0296\n",
      "3D profile generation : http://edamontology.org/operation_0297\n",
      "Sequence profile alignment : http://edamontology.org/operation_0300\n",
      "Protein threading : http://edamontology.org/operation_0302\n",
      "Fold recognition : http://edamontology.org/operation_0303\n",
      "Literature search : http://edamontology.org/operation_0305\n",
      "Text mining : http://edamontology.org/operation_0306\n",
      "Virtual PCR : http://edamontology.org/operation_0307\n",
      "PCR primer design : http://edamontology.org/operation_0308\n",
      "Microarray probe design : http://edamontology.org/operation_0309\n",
      "Sequence assembly : http://edamontology.org/operation_0310\n",
      "Expression profile clustering : http://edamontology.org/operation_0313\n",
      "Gene expression profiling : http://edamontology.org/operation_0314\n",
      "Expression profile comparison : http://edamontology.org/operation_0315\n",
      "Protein secondary structure assignment : http://edamontology.org/operation_0319\n",
      "Protein structure assignment : http://edamontology.org/operation_0320\n",
      "Protein structure validation : http://edamontology.org/operation_0321\n",
      "Molecular model refinement : http://edamontology.org/operation_0322\n",
      "Phylogenetic inference : http://edamontology.org/operation_0323\n",
      "Phylogenetic analysis : http://edamontology.org/operation_0324\n",
      "Phylogenetic tree comparison : http://edamontology.org/operation_0325\n",
      "Phylogenetic tree editing : http://edamontology.org/operation_0326\n",
      "Phylogenetic footprinting : http://edamontology.org/operation_0327\n",
      "Variant effect prediction : http://edamontology.org/operation_0331\n",
      "Enzyme kinetics calculation : http://edamontology.org/operation_0334\n",
      "Formatting : http://edamontology.org/operation_0335\n",
      "Format validation : http://edamontology.org/operation_0336\n",
      "Visualisation : http://edamontology.org/operation_0337\n",
      "Sequence database search : http://edamontology.org/operation_0338\n",
      "Structure database search : http://edamontology.org/operation_0339\n",
      "Sequence similarity search : http://edamontology.org/operation_0346\n",
      "Sequence database search (by property) : http://edamontology.org/operation_0349\n",
      "Structural similarity search : http://edamontology.org/operation_0360\n",
      "Sequence annotation : http://edamontology.org/operation_0361\n",
      "Genome annotation : http://edamontology.org/operation_0362\n",
      "Reverse complement : http://edamontology.org/operation_0363\n",
      "Random sequence generation : http://edamontology.org/operation_0364\n",
      "Restriction digest : http://edamontology.org/operation_0365\n",
      "Protein sequence cleavage : http://edamontology.org/operation_0366\n",
      "Sequence mutation and randomisation : http://edamontology.org/operation_0367\n",
      "Sequence masking : http://edamontology.org/operation_0368\n",
      "Sequence cutting : http://edamontology.org/operation_0369\n",
      "Restriction site creation : http://edamontology.org/operation_0370\n",
      "DNA translation : http://edamontology.org/operation_0371\n",
      "DNA transcription : http://edamontology.org/operation_0372\n",
      "Repeat sequence detection : http://edamontology.org/operation_0379\n",
      "Repeat sequence organisation analysis : http://edamontology.org/operation_0380\n",
      "Accessible surface calculation : http://edamontology.org/operation_0384\n",
      "Protein dipole moment calculation : http://edamontology.org/operation_0386\n",
      "Molecular surface calculation : http://edamontology.org/operation_0387\n",
      "Protein-nucleic acid interaction analysis : http://edamontology.org/operation_0389\n",
      "Protein peeling : http://edamontology.org/operation_0390\n",
      "Protein distance matrix calculation : http://edamontology.org/operation_0391\n",
      "Contact map calculation : http://edamontology.org/operation_0392\n",
      "Residue cluster calculation : http://edamontology.org/operation_0393\n",
      "Hydrogen bond calculation : http://edamontology.org/operation_0394\n",
      "Ramachandran plot calculation : http://edamontology.org/operation_0396\n",
      "Protein molecular weight calculation : http://edamontology.org/operation_0398\n",
      "Protein extinction coefficient calculation : http://edamontology.org/operation_0399\n",
      "Protein pKa calculation : http://edamontology.org/operation_0400\n",
      "Protein titration curve plotting : http://edamontology.org/operation_0402\n",
      "Protein isoelectric point calculation : http://edamontology.org/operation_0403\n",
      "Protein hydrogen exchange rate calculation : http://edamontology.org/operation_0404\n",
      "Protein hydrophobic region calculation : http://edamontology.org/operation_0405\n",
      "Protein aliphatic index calculation : http://edamontology.org/operation_0406\n",
      "Protein hydrophobic moment plotting : http://edamontology.org/operation_0407\n",
      "Protein globularity prediction : http://edamontology.org/operation_0408\n",
      "Protein solubility prediction : http://edamontology.org/operation_0409\n",
      "Protein crystallizability prediction : http://edamontology.org/operation_0410\n",
      "Nucleic acid feature detection : http://edamontology.org/operation_0415\n",
      "Epitope mapping : http://edamontology.org/operation_0416\n",
      "PTM site prediction : http://edamontology.org/operation_0417\n",
      "Protein signal peptide detection : http://edamontology.org/operation_0418\n",
      "Nucleic acids-binding site prediction : http://edamontology.org/operation_0420\n",
      "Protein cleavage site prediction : http://edamontology.org/operation_0422\n",
      "Transposon prediction : http://edamontology.org/operation_0427\n",
      "PolyA signal detection : http://edamontology.org/operation_0428\n",
      "Quadruplex formation site detection : http://edamontology.org/operation_0429\n",
      "CpG island and isochore detection : http://edamontology.org/operation_0430\n",
      "Restriction site recognition : http://edamontology.org/operation_0431\n",
      "Nucleosome position prediction : http://edamontology.org/operation_0432\n",
      "Splice site prediction : http://edamontology.org/operation_0433\n",
      "Operon prediction : http://edamontology.org/operation_0435\n",
      "Coding region prediction : http://edamontology.org/operation_0436\n",
      "SECIS element prediction : http://edamontology.org/operation_0437\n",
      "Transcriptional regulatory element prediction : http://edamontology.org/operation_0438\n",
      "Translation initiation site prediction : http://edamontology.org/operation_0439\n",
      "Promoter prediction : http://edamontology.org/operation_0440\n",
      "cis-regulatory element prediction : http://edamontology.org/operation_0441\n",
      "trans-regulatory element prediction : http://edamontology.org/operation_0443\n",
      "S/MAR prediction : http://edamontology.org/operation_0444\n",
      "Transcription factor binding site prediction : http://edamontology.org/operation_0445\n",
      "Exonic splicing enhancer prediction : http://edamontology.org/operation_0446\n",
      "Sequence alignment validation : http://edamontology.org/operation_0447\n",
      "Sequence alignment analysis (conservation) : http://edamontology.org/operation_0448\n",
      "Sequence alignment analysis (site correlation) : http://edamontology.org/operation_0449\n",
      "Chimera detection : http://edamontology.org/operation_0450\n",
      "Recombination detection : http://edamontology.org/operation_0451\n",
      "Indel detection : http://edamontology.org/operation_0452\n",
      "Nucleic acid thermodynamic property calculation : http://edamontology.org/operation_0455\n",
      "Nucleic acid melting profile plotting : http://edamontology.org/operation_0456\n",
      "Nucleic acid stitch profile plotting : http://edamontology.org/operation_0457\n",
      "Nucleic acid melting curve plotting : http://edamontology.org/operation_0458\n",
      "Nucleic acid probability profile plotting : http://edamontology.org/operation_0459\n",
      "Nucleic acid temperature profile plotting : http://edamontology.org/operation_0460\n",
      "Nucleic acid curvature calculation : http://edamontology.org/operation_0461\n",
      "miRNA target prediction : http://edamontology.org/operation_0463\n",
      "tRNA gene prediction : http://edamontology.org/operation_0464\n",
      "siRNA binding specificity prediction : http://edamontology.org/operation_0465\n",
      "Protein secondary structure prediction (helices) : http://edamontology.org/operation_0468\n",
      "Protein secondary structure prediction (turns) : http://edamontology.org/operation_0469\n",
      "Protein secondary structure prediction (coils) : http://edamontology.org/operation_0470\n",
      "Disulfide bond prediction : http://edamontology.org/operation_0471\n",
      "Protein structure prediction : http://edamontology.org/operation_0474\n",
      "Nucleic acid structure prediction : http://edamontology.org/operation_0475\n",
      "Ab initio structure prediction : http://edamontology.org/operation_0476\n",
      "Protein modelling : http://edamontology.org/operation_0477\n",
      "Molecular docking : http://edamontology.org/operation_0478\n",
      "Backbone modelling : http://edamontology.org/operation_0479\n",
      "Side chain modelling : http://edamontology.org/operation_0480\n",
      "Loop modelling : http://edamontology.org/operation_0481\n",
      "Protein-ligand docking : http://edamontology.org/operation_0482\n",
      "RNA inverse folding : http://edamontology.org/operation_0483\n",
      "SNP detection : http://edamontology.org/operation_0484\n",
      "Radiation Hybrid Mapping : http://edamontology.org/operation_0485\n",
      "Haplotype mapping : http://edamontology.org/operation_0487\n",
      "Linkage disequilibrium calculation : http://edamontology.org/operation_0488\n",
      "Genetic code prediction : http://edamontology.org/operation_0489\n",
      "Dot plot plotting : http://edamontology.org/operation_0490\n",
      "Pairwise sequence alignment : http://edamontology.org/operation_0491\n",
      "Multiple sequence alignment : http://edamontology.org/operation_0492\n",
      "Local alignment : http://edamontology.org/operation_0495\n",
      "Global alignment : http://edamontology.org/operation_0496\n",
      "Tree-based sequence alignment : http://edamontology.org/operation_0499\n",
      "RNA secondary structure alignment : http://edamontology.org/operation_0502\n",
      "Pairwise structure alignment : http://edamontology.org/operation_0503\n",
      "Multiple structure alignment : http://edamontology.org/operation_0504\n",
      "Local structure alignment : http://edamontology.org/operation_0509\n",
      "Global structure alignment : http://edamontology.org/operation_0510\n",
      "Mapping assembly : http://edamontology.org/operation_0523\n",
      "De-novo assembly : http://edamontology.org/operation_0524\n",
      "Genome assembly : http://edamontology.org/operation_0525\n",
      "EST assembly : http://edamontology.org/operation_0526\n",
      "Sequence tag mapping : http://edamontology.org/operation_0527\n",
      "Heat map generation : http://edamontology.org/operation_0531\n",
      "Expression profile pathway mapping : http://edamontology.org/operation_0533\n",
      "Phylogenetic inference (data centric) : http://edamontology.org/operation_0538\n",
      "Phylogenetic inference (method centric) : http://edamontology.org/operation_0539\n",
      "Phylogenetic inference (from molecular sequences) : http://edamontology.org/operation_0540\n",
      "Phylogenetic inference (from continuous quantitative characters) : http://edamontology.org/operation_0541\n",
      "Phylogenetic inference (from gene frequencies) : http://edamontology.org/operation_0542\n",
      "Phylogenetic inference (from polymorphism data) : http://edamontology.org/operation_0543\n",
      "Species tree construction : http://edamontology.org/operation_0544\n",
      "Phylogenetic inference (parsimony methods) : http://edamontology.org/operation_0545\n",
      "Phylogenetic inference (minimum distance methods) : http://edamontology.org/operation_0546\n",
      "Phylogenetic inference (maximum likelihood and Bayesian methods) : http://edamontology.org/operation_0547\n",
      "Phylogenetic inference (quartet methods) : http://edamontology.org/operation_0548\n",
      "Phylogenetic inference (AI methods) : http://edamontology.org/operation_0549\n",
      "DNA substitution modelling : http://edamontology.org/operation_0550\n",
      "Phylogenetic tree topology analysis : http://edamontology.org/operation_0551\n",
      "Phylogenetic tree bootstrapping : http://edamontology.org/operation_0552\n",
      "Gene tree construction : http://edamontology.org/operation_0553\n",
      "Allele frequency distribution analysis : http://edamontology.org/operation_0554\n",
      "Consensus tree construction : http://edamontology.org/operation_0555\n",
      "Phylogenetic sub/super tree construction : http://edamontology.org/operation_0556\n",
      "Phylogenetic tree distances calculation : http://edamontology.org/operation_0557\n",
      "Phylogenetic tree annotation : http://edamontology.org/operation_0558\n",
      "DNA vaccine design : http://edamontology.org/operation_0560\n",
      "Sequence visualisation : http://edamontology.org/operation_0564\n",
      "Sequence cluster visualisation : http://edamontology.org/operation_0566\n",
      "Phylogenetic tree visualisation : http://edamontology.org/operation_0567\n",
      "Structure visualisation : http://edamontology.org/operation_0570\n",
      "Expression data visualisation : http://edamontology.org/operation_0571\n",
      "Map drawing : http://edamontology.org/operation_0573\n",
      "Restriction map drawing : http://edamontology.org/operation_0575\n",
      "Plasmid map drawing : http://edamontology.org/operation_0578\n",
      "Operon drawing : http://edamontology.org/operation_0579\n",
      "Protein function prediction : http://edamontology.org/operation_1777\n",
      "Protein function comparison : http://edamontology.org/operation_1778\n",
      "Gene regulatory network analysis : http://edamontology.org/operation_1781\n",
      "Parsing : http://edamontology.org/operation_1812\n",
      "Surface rendering : http://edamontology.org/operation_1816\n",
      "Cysteine bridge detection : http://edamontology.org/operation_1829\n",
      "Free cysteine detection : http://edamontology.org/operation_1830\n",
      "Metal-bound cysteine detection : http://edamontology.org/operation_1831\n",
      "Protein-metal contact calculation : http://edamontology.org/operation_1834\n",
      "Residue bump detection : http://edamontology.org/operation_1836\n",
      "Salt bridge calculation : http://edamontology.org/operation_1839\n",
      "Residue packing validation : http://edamontology.org/operation_1843\n",
      "Protein geometry validation : http://edamontology.org/operation_1844\n",
      "Protein cysteine and disulfide bond assignment : http://edamontology.org/operation_1850\n",
      "siRNA duplex prediction : http://edamontology.org/operation_2008\n",
      "Sequence alignment refinement : http://edamontology.org/operation_2089\n",
      "Sequence file editing : http://edamontology.org/operation_2121\n",
      "Representative sequence identification : http://edamontology.org/operation_2233\n",
      "Statistical calculation : http://edamontology.org/operation_2238\n",
      "3D-1D scoring matrix generation : http://edamontology.org/operation_2239\n",
      "Transmembrane protein visualisation : http://edamontology.org/operation_2241\n",
      "Nucleic acid density plotting : http://edamontology.org/operation_2284\n",
      "Sequence analysis : http://edamontology.org/operation_2403\n",
      "Sequence motif analysis : http://edamontology.org/operation_2404\n",
      "Protein structure analysis : http://edamontology.org/operation_2406\n",
      "Data handling : http://edamontology.org/operation_2409\n",
      "Protein folding analysis : http://edamontology.org/operation_2415\n",
      "Protein secondary structure analysis : http://edamontology.org/operation_2416\n",
      "Primer and probe design : http://edamontology.org/operation_2419\n",
      "Database search : http://edamontology.org/operation_2421\n",
      "Data retrieval : http://edamontology.org/operation_2422\n",
      "Prediction and recognition : http://edamontology.org/operation_2423\n",
      "Comparison : http://edamontology.org/operation_2424\n",
      "Optimisation and refinement : http://edamontology.org/operation_2425\n",
      "Modelling and simulation : http://edamontology.org/operation_2426\n",
      "Validation : http://edamontology.org/operation_2428\n",
      "Mapping : http://edamontology.org/operation_2429\n",
      "Design : http://edamontology.org/operation_2430\n",
      "Gene-set enrichment analysis : http://edamontology.org/operation_2436\n",
      "Gene regulatory network prediction : http://edamontology.org/operation_2437\n",
      "RNA secondary structure analysis : http://edamontology.org/operation_2439\n",
      "RNA structure prediction : http://edamontology.org/operation_2441\n",
      "DNA structure prediction : http://edamontology.org/operation_2442\n",
      "Sequence comparison : http://edamontology.org/operation_2451\n",
      "Gene prediction : http://edamontology.org/operation_2454\n",
      "Protein-protein binding site prediction : http://edamontology.org/operation_2464\n",
      "Protein architecture comparison : http://edamontology.org/operation_2474\n",
      "Protein architecture recognition : http://edamontology.org/operation_2475\n",
      "Molecular dynamics : http://edamontology.org/operation_2476\n",
      "Nucleic acid sequence analysis : http://edamontology.org/operation_2478\n",
      "Protein sequence analysis : http://edamontology.org/operation_2479\n",
      "Structure analysis : http://edamontology.org/operation_2480\n",
      "Nucleic acid structure analysis : http://edamontology.org/operation_2481\n",
      "Structure comparison : http://edamontology.org/operation_2483\n",
      "Helical wheel drawing : http://edamontology.org/operation_2485\n",
      "Topology diagram drawing : http://edamontology.org/operation_2486\n",
      "Protein structure comparison : http://edamontology.org/operation_2487\n",
      "Protein secondary structure comparison : http://edamontology.org/operation_2488\n",
      "Subcellular localisation prediction : http://edamontology.org/operation_2489\n",
      "Protein interaction prediction : http://edamontology.org/operation_2492\n",
      "Expression analysis : http://edamontology.org/operation_2495\n",
      "Splicing analysis : http://edamontology.org/operation_2499\n",
      "DNA back-translation : http://edamontology.org/operation_2510\n",
      "Nucleic acid structure comparison : http://edamontology.org/operation_2518\n",
      "DNA mapping : http://edamontology.org/operation_2520\n",
      "Protein hydropathy calculation : http://edamontology.org/operation_2574\n",
      "Binding site prediction : http://edamontology.org/operation_2575\n",
      "Structure clustering : http://edamontology.org/operation_2844\n",
      "Sequence tagged site (STS) mapping : http://edamontology.org/operation_2871\n",
      "Alignment : http://edamontology.org/operation_2928\n",
      "Protein fragment weight comparison : http://edamontology.org/operation_2929\n",
      "Protein property comparison : http://edamontology.org/operation_2930\n",
      "Clustering profile plotting : http://edamontology.org/operation_2935\n",
      "Proximity map plotting : http://edamontology.org/operation_2937\n",
      "Dendrogram visualisation : http://edamontology.org/operation_2938\n",
      "Principal component visualisation : http://edamontology.org/operation_2939\n",
      "Scatter plot plotting : http://edamontology.org/operation_2940\n",
      "Treemap visualisation : http://edamontology.org/operation_2942\n",
      "Box-Whisker plot plotting : http://edamontology.org/operation_2943\n",
      "Physical mapping : http://edamontology.org/operation_2944\n",
      "Analysis : http://edamontology.org/operation_2945\n",
      "Protein-protein interaction analysis : http://edamontology.org/operation_2949\n",
      "Residue distance calculation : http://edamontology.org/operation_2950\n",
      "Codon usage bias calculation : http://edamontology.org/operation_2962\n",
      "Codon usage fraction calculation : http://edamontology.org/operation_2964\n",
      "Classification : http://edamontology.org/operation_2990\n",
      "Sequence classification : http://edamontology.org/operation_2995\n",
      "Structure classification : http://edamontology.org/operation_2996\n",
      "Protein comparison : http://edamontology.org/operation_2997\n",
      "Nucleic acid comparison : http://edamontology.org/operation_2998\n",
      "Structure editing : http://edamontology.org/operation_3080\n",
      "Sequence alignment editing : http://edamontology.org/operation_3081\n",
      "Protein feature detection : http://edamontology.org/operation_3092\n",
      "Protein interaction network prediction : http://edamontology.org/operation_3094\n",
      "Nucleic acid design : http://edamontology.org/operation_3095\n",
      "Editing : http://edamontology.org/operation_3096\n",
      "Sequence assembly validation : http://edamontology.org/operation_3180\n",
      "Genome alignment : http://edamontology.org/operation_3182\n",
      "Localised reassembly : http://edamontology.org/operation_3183\n",
      "Sequence assembly visualisation : http://edamontology.org/operation_3184\n",
      "Base-calling : http://edamontology.org/operation_3185\n",
      "Bisulfite mapping : http://edamontology.org/operation_3186\n",
      "Sequence contamination filtering : http://edamontology.org/operation_3187\n",
      "Sequence trimming : http://edamontology.org/operation_3192\n",
      "Genome feature comparison : http://edamontology.org/operation_3194\n",
      "Sequencing error detection : http://edamontology.org/operation_3195\n",
      "Genotyping : http://edamontology.org/operation_3196\n",
      "Genetic variation analysis : http://edamontology.org/operation_3197\n",
      "Read mapping : http://edamontology.org/operation_3198\n",
      "Split read mapping : http://edamontology.org/operation_3199\n",
      "DNA barcoding : http://edamontology.org/operation_3200\n",
      "Chromatogram visualisation : http://edamontology.org/operation_3203\n",
      "Methylation analysis : http://edamontology.org/operation_3204\n",
      "Whole genome methylation analysis : http://edamontology.org/operation_3206\n",
      "Gene methylation analysis : http://edamontology.org/operation_3207\n",
      "Genome visualisation : http://edamontology.org/operation_3208\n",
      "Genome comparison : http://edamontology.org/operation_3209\n",
      "Genome indexing : http://edamontology.org/operation_3211\n",
      "Spectral analysis : http://edamontology.org/operation_3214\n",
      "Peak detection : http://edamontology.org/operation_3215\n",
      "Scaffolding : http://edamontology.org/operation_3216\n",
      "Scaffold gap completion : http://edamontology.org/operation_3217\n",
      "Sequencing quality control : http://edamontology.org/operation_3218\n",
      "Read pre-processing : http://edamontology.org/operation_3219\n",
      "Species frequency estimation : http://edamontology.org/operation_3221\n",
      "Peak calling : http://edamontology.org/operation_3222\n",
      "Differential gene expression profiling : http://edamontology.org/operation_3223\n",
      "Variant classification : http://edamontology.org/operation_3225\n",
      "Variant prioritisation : http://edamontology.org/operation_3226\n",
      "Variant calling : http://edamontology.org/operation_3227\n",
      "Structural variation detection : http://edamontology.org/operation_3228\n",
      "Exome assembly : http://edamontology.org/operation_3229\n",
      "Read depth analysis : http://edamontology.org/operation_3230\n",
      "Gene expression QTL analysis : http://edamontology.org/operation_3232\n",
      "Copy number estimation : http://edamontology.org/operation_3233\n",
      "Primer removal : http://edamontology.org/operation_3237\n",
      "Transcriptome assembly : http://edamontology.org/operation_3258\n",
      "Sequence coordinate conversion : http://edamontology.org/operation_3267\n",
      "Document similarity calculation : http://edamontology.org/operation_3278\n",
      "Document clustering : http://edamontology.org/operation_3279\n",
      "Named-entity and concept recognition : http://edamontology.org/operation_3280\n",
      "ID mapping : http://edamontology.org/operation_3282\n",
      "Anonymisation : http://edamontology.org/operation_3283\n",
      "Sequence checksum generation : http://edamontology.org/operation_3348\n",
      "Bibliography generation : http://edamontology.org/operation_3349\n",
      "Protein quaternary structure prediction : http://edamontology.org/operation_3350\n",
      "Molecular surface analysis : http://edamontology.org/operation_3351\n",
      "Ontology comparison : http://edamontology.org/operation_3352\n",
      "Format detection : http://edamontology.org/operation_3357\n",
      "Splitting : http://edamontology.org/operation_3359\n",
      "Generation : http://edamontology.org/operation_3429\n",
      "Deposition : http://edamontology.org/operation_3431\n",
      "Clustering : http://edamontology.org/operation_3432\n",
      "Conversion : http://edamontology.org/operation_3434\n",
      "Standardisation and normalisation : http://edamontology.org/operation_3435\n",
      "Aggregation : http://edamontology.org/operation_3436\n",
      "Article comparison : http://edamontology.org/operation_3437\n",
      "Calculation : http://edamontology.org/operation_3438\n",
      "Image analysis : http://edamontology.org/operation_3443\n",
      "Diffraction data analysis : http://edamontology.org/operation_3445\n",
      "Cell migration analysis : http://edamontology.org/operation_3446\n",
      "Diffraction data reduction : http://edamontology.org/operation_3447\n",
      "Neurite measurement : http://edamontology.org/operation_3450\n",
      "Diffraction data integration : http://edamontology.org/operation_3453\n",
      "Phasing : http://edamontology.org/operation_3454\n",
      "Molecular replacement : http://edamontology.org/operation_3455\n",
      "Rigid body refinement : http://edamontology.org/operation_3456\n",
      "Single particle analysis : http://edamontology.org/operation_3457\n",
      "Single particle alignment and classification : http://edamontology.org/operation_3458\n",
      "Functional clustering : http://edamontology.org/operation_3459\n",
      "Taxonomic classification : http://edamontology.org/operation_3460\n",
      "Virulence prediction : http://edamontology.org/operation_3461\n",
      "Expression correlation analysis : http://edamontology.org/operation_3463\n",
      "Correlation : http://edamontology.org/operation_3465\n",
      "RNA structure covariance model generation : http://edamontology.org/operation_3469\n",
      "Phylogenetic reconstruction : http://edamontology.org/operation_3478\n",
      "Probabilistic data generation : http://edamontology.org/operation_3480\n",
      "Probabilistic sequence generation : http://edamontology.org/operation_3481\n",
      "Enrichment analysis : http://edamontology.org/operation_3501\n",
      "Chemical similarity enrichment : http://edamontology.org/operation_3502\n",
      "Incident curve plotting : http://edamontology.org/operation_3503\n",
      "Variant pattern analysis : http://edamontology.org/operation_3504\n",
      "Microscope image visualisation : http://edamontology.org/operation_3552\n",
      "Image annotation : http://edamontology.org/operation_3553\n",
      "Imputation : http://edamontology.org/operation_3557\n",
      "Ontology visualisation : http://edamontology.org/operation_3559\n",
      "Maximum occurence analysis : http://edamontology.org/operation_3560\n",
      "Database comparison : http://edamontology.org/operation_3561\n",
      "RNA-seq read count analysis : http://edamontology.org/operation_3563\n",
      "Chemical redundancy removal : http://edamontology.org/operation_3564\n",
      "RNA-seq time series data analysis : http://edamontology.org/operation_3565\n",
      "Simulated gene expression data generation : http://edamontology.org/operation_3566\n",
      "Relation extraction : http://edamontology.org/operation_3625\n",
      "Mass spectra calibration : http://edamontology.org/operation_3627\n",
      "Chromatographic alignment : http://edamontology.org/operation_3628\n",
      "Deisotoping : http://edamontology.org/operation_3629\n",
      "Protein quantification : http://edamontology.org/operation_3630\n",
      "Peptide identification : http://edamontology.org/operation_3631\n",
      "Isotopic distributions calculation : http://edamontology.org/operation_3632\n",
      "Retention time prediction : http://edamontology.org/operation_3633\n",
      "Label-free quantification : http://edamontology.org/operation_3634\n",
      "Labeled quantification : http://edamontology.org/operation_3635\n",
      "MRM/SRM : http://edamontology.org/operation_3636\n",
      "Spectral counting : http://edamontology.org/operation_3637\n",
      "SILAC : http://edamontology.org/operation_3638\n",
      "iTRAQ : http://edamontology.org/operation_3639\n",
      "18O labeling : http://edamontology.org/operation_3640\n",
      "TMT-tag : http://edamontology.org/operation_3641\n",
      "Dimethyl : http://edamontology.org/operation_3642\n",
      "Tag-based peptide identification : http://edamontology.org/operation_3643\n",
      "de Novo sequencing : http://edamontology.org/operation_3644\n",
      "PTM identification : http://edamontology.org/operation_3645\n",
      "Peptide database search : http://edamontology.org/operation_3646\n",
      "Blind peptide database search : http://edamontology.org/operation_3647\n",
      "Target-Decoy : http://edamontology.org/operation_3649\n",
      "Statistical inference : http://edamontology.org/operation_3658\n",
      "Regression analysis : http://edamontology.org/operation_3659\n",
      "Metabolic network modelling : http://edamontology.org/operation_3660\n",
      "SNP annotation : http://edamontology.org/operation_3661\n",
      "Ab-initio gene prediction : http://edamontology.org/operation_3662\n",
      "Homology-based gene prediction : http://edamontology.org/operation_3663\n",
      "Statistical modelling : http://edamontology.org/operation_3664\n",
      "Molecular surface comparison : http://edamontology.org/operation_3666\n",
      "Gene functional annotation : http://edamontology.org/operation_3672\n",
      "Variant filtering : http://edamontology.org/operation_3675\n",
      "Differential binding analysis : http://edamontology.org/operation_3677\n",
      "RNA-Seq analysis : http://edamontology.org/operation_3680\n",
      "Mass spectrum visualisation : http://edamontology.org/operation_3694\n",
      "Filtering : http://edamontology.org/operation_3695\n",
      "Reference identification : http://edamontology.org/operation_3703\n",
      "Ion counting : http://edamontology.org/operation_3704\n",
      "Isotope-coded protein label : http://edamontology.org/operation_3705\n",
      "Metabolic labeling : http://edamontology.org/operation_3715\n",
      "Cross-assembly : http://edamontology.org/operation_3730\n",
      "Sample comparison : http://edamontology.org/operation_3731\n",
      "Differential protein expression profiling : http://edamontology.org/operation_3741\n",
      "Multiple sample visualisation : http://edamontology.org/operation_3744\n",
      "Ancestral reconstruction : http://edamontology.org/operation_3745\n",
      "PTM localisation : http://edamontology.org/operation_3755\n",
      "Service management : http://edamontology.org/operation_3760\n",
      "Service discovery : http://edamontology.org/operation_3761\n",
      "Service composition : http://edamontology.org/operation_3762\n",
      "Service invocation : http://edamontology.org/operation_3763\n",
      "Weighted correlation network analysis : http://edamontology.org/operation_3766\n",
      "Protein identification : http://edamontology.org/operation_3767\n",
      "Text annotation : http://edamontology.org/operation_3778\n",
      "Collapsing methods : http://edamontology.org/operation_3791\n",
      "miRNA expression analysis : http://edamontology.org/operation_3792\n",
      "Read summarisation : http://edamontology.org/operation_3793\n",
      "In vitro selection : http://edamontology.org/operation_3795\n",
      "Rarefaction : http://edamontology.org/operation_3797\n",
      "Read binning : http://edamontology.org/operation_3798\n",
      "Quantification : http://edamontology.org/operation_3799\n",
      "RNA-Seq quantification : http://edamontology.org/operation_3800\n",
      "Spectral library search : http://edamontology.org/operation_3801\n",
      "Sorting : http://edamontology.org/operation_3802\n",
      "Natural product identification : http://edamontology.org/operation_3803\n",
      "DMR identification : http://edamontology.org/operation_3809\n",
      "Multilocus sequence typing : http://edamontology.org/operation_3840\n",
      "Spectrum calculation : http://edamontology.org/operation_3860\n",
      "Trajectory visualization : http://edamontology.org/operation_3890\n",
      "Essential dynamics : http://edamontology.org/operation_3891\n",
      "Forcefield parameterisation : http://edamontology.org/operation_3893\n",
      "Active site prediction : http://edamontology.org/operation_3896\n",
      "Ligand-binding site prediction : http://edamontology.org/operation_3897\n",
      "Metal-binding site prediction : http://edamontology.org/operation_3898\n",
      "Protein-protein docking : http://edamontology.org/operation_3899\n",
      "DNA-binding protein prediction : http://edamontology.org/operation_3900\n",
      "RNA-binding protein prediction : http://edamontology.org/operation_3901\n",
      "RNA binding site prediction : http://edamontology.org/operation_3902\n",
      "DNA binding site prediction : http://edamontology.org/operation_3903\n",
      "Protein disorder prediction : http://edamontology.org/operation_3904\n",
      "Information extraction : http://edamontology.org/operation_3907\n",
      "Information retrieval : http://edamontology.org/operation_3908\n",
      "Methylation calling : http://edamontology.org/operation_3919\n",
      "DNA testing : http://edamontology.org/operation_3920\n",
      "Sequence read processing : http://edamontology.org/operation_3921\n",
      "Pathway visualisation : http://edamontology.org/operation_3926\n",
      "Network analysis : http://edamontology.org/operation_3927\n",
      "Pathway analysis : http://edamontology.org/operation_3928\n",
      "Metabolic pathway prediction : http://edamontology.org/operation_3929\n",
      "Demultiplexing : http://edamontology.org/operation_3933\n",
      "Dimensionality reduction : http://edamontology.org/operation_3935\n",
      "Feature selection : http://edamontology.org/operation_3936\n",
      "Feature extraction : http://edamontology.org/operation_3937\n",
      "Virtual screening : http://edamontology.org/operation_3938\n",
      "Tree dating : http://edamontology.org/operation_3942\n",
      "Ecological modelling : http://edamontology.org/operation_3946\n",
      "Phylogenetic tree reconciliation : http://edamontology.org/operation_3947\n",
      "Selection detection : http://edamontology.org/operation_3950\n",
      "Principal component analysis : http://edamontology.org/operation_3960\n",
      "Copy number variation detection : http://edamontology.org/operation_3961\n",
      "Deletion detection : http://edamontology.org/operation_3962\n",
      "Duplication detection : http://edamontology.org/operation_3963\n",
      "Complex CNV detection : http://edamontology.org/operation_3964\n",
      "Amplification detection : http://edamontology.org/operation_3965\n",
      "Protein design : http://edamontology.org/operation_4008\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "PREFIX edam: <http://purl.obolibrary.org/obo/edam#> \n",
    "\n",
    "SELECT ?x ?label WHERE {\n",
    "    ?x oboInOwl:inSubset edam:operations . \n",
    "    ?x rdfs:label ?label .\n",
    "}\n",
    "ORDER BY ASC(str(?x))\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"{r['label']} : {r['x']}\") "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Citation : http://edamontology.org/citation\n",
      "None : http://edamontology.org/comment_handle\n",
      "Created in : http://edamontology.org/created_in\n",
      "deprecation_comment : http://edamontology.org/deprecation_comment\n",
      "Documentation : http://edamontology.org/documentation\n",
      "Example : http://edamontology.org/example\n",
      "File extension : http://edamontology.org/file_extension\n",
      "hasHumanReadableId : http://edamontology.org/hasHumanReadableId\n",
      "Information standard : http://edamontology.org/information_standard\n",
      "deprecation_candidate : http://edamontology.org/is_deprecation_candidate\n",
      "refactor_candidate : http://edamontology.org/is_refactor_candidate\n",
      "isdebtag : http://edamontology.org/isdebtag\n",
      "Media type : http://edamontology.org/media_type\n",
      "None : http://edamontology.org/next_id\n",
      "notRecommendedForAnnotation : http://edamontology.org/notRecommendedForAnnotation\n",
      "Obsolete since : http://edamontology.org/obsolete_since\n",
      "Old parent : http://edamontology.org/oldParent\n",
      "Old related : http://edamontology.org/oldRelated\n",
      "Ontology used : http://edamontology.org/ontology_used\n",
      "Organisation : http://edamontology.org/organisation\n",
      "refactor_comment : http://edamontology.org/refactor_comment\n",
      "Regular expression : http://edamontology.org/regex\n",
      "Repository : http://edamontology.org/repository\n",
      "thematic_editor : http://edamontology.org/thematic_editor\n",
      "None : http://purl.obolibrary.org/obo/date\n",
      "None : http://purl.obolibrary.org/obo/edam#data\n",
      "None : http://purl.obolibrary.org/obo/edam#edam\n",
      "None : http://purl.obolibrary.org/obo/edam#events\n",
      "None : http://purl.obolibrary.org/obo/edam#formats\n",
      "None : http://purl.obolibrary.org/obo/edam#identifiers\n",
      "None : http://purl.obolibrary.org/obo/edam#obsolete\n",
      "None : http://purl.obolibrary.org/obo/edam#operations\n",
      "None : http://purl.obolibrary.org/obo/edam#placeholder\n",
      "None : http://purl.obolibrary.org/obo/edam#topics\n",
      "None : http://purl.obolibrary.org/obo/idspace\n",
      "None : http://purl.obolibrary.org/obo/is_anti_symmetric\n",
      "None : http://purl.obolibrary.org/obo/is_metadata_tag\n",
      "None : http://purl.obolibrary.org/obo/is_reflexive\n",
      "None : http://purl.obolibrary.org/obo/is_symmetric\n",
      "None : http://purl.obolibrary.org/obo/namespace\n",
      "None : http://purl.obolibrary.org/obo/remark\n",
      "None : http://purl.obolibrary.org/obo/transitive_over\n",
      "None : http://purl.org/dc/elements/1.1/contributor\n",
      "None : http://purl.org/dc/elements/1.1/creator\n",
      "None : http://purl.org/dc/elements/1.1/format\n",
      "None : http://purl.org/dc/elements/1.1/title\n",
      "None : http://usefulinc.com/ns/doap#Version\n",
      "None : http://www.geneontology.org/formats/oboInOwl#SubsetProperty\n",
      "None : http://www.geneontology.org/formats/oboInOwl#comment\n",
      "None : http://www.geneontology.org/formats/oboInOwl#consider\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasAlternativeId\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasDbXRef\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasDbXref\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasDefinition\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasExactSynonym\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasHumanReadableId\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym\n",
      "None : http://www.geneontology.org/formats/oboInOwl#hasSubset\n",
      "None : http://www.geneontology.org/formats/oboInOwl#inSubset\n",
      "None : http://www.geneontology.org/formats/oboInOwl#isCyclic\n",
      "None : http://www.geneontology.org/formats/oboInOwl#replacedBy\n",
      "None : http://www.geneontology.org/formats/oboInOwl#savedBy\n",
      "None : http://xmlns.com/foaf/0.1/page\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "PREFIX edam: <http://purl.obolibrary.org/obo/edam#> \n",
    "\n",
    "SELECT ?x ?label WHERE {\n",
    "    ?x rdf:type owl:AnnotationProperty .\n",
    "    OPTIONAL {?x rdfs:label ?label .}\n",
    "}\n",
    "ORDER BY ASC(str(?x))\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "annots = []\n",
    "for r in results :\n",
    "    print(f\"{r['label']} : {r['x']}\") \n",
    "    annots.append(str(r['x']))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['http://edamontology.org/citation',\n",
       " 'http://edamontology.org/comment_handle',\n",
       " 'http://edamontology.org/created_in',\n",
       " 'http://edamontology.org/deprecation_comment',\n",
       " 'http://edamontology.org/documentation',\n",
       " 'http://edamontology.org/example',\n",
       " 'http://edamontology.org/file_extension',\n",
       " 'http://edamontology.org/hasHumanReadableId',\n",
       " 'http://edamontology.org/information_standard',\n",
       " 'http://edamontology.org/is_deprecation_candidate',\n",
       " 'http://edamontology.org/is_refactor_candidate',\n",
       " 'http://edamontology.org/isdebtag',\n",
       " 'http://edamontology.org/media_type',\n",
       " 'http://edamontology.org/next_id',\n",
       " 'http://edamontology.org/notRecommendedForAnnotation',\n",
       " 'http://edamontology.org/obsolete_since',\n",
       " 'http://edamontology.org/oldParent',\n",
       " 'http://edamontology.org/oldRelated',\n",
       " 'http://edamontology.org/ontology_used',\n",
       " 'http://edamontology.org/organisation',\n",
       " 'http://edamontology.org/refactor_comment',\n",
       " 'http://edamontology.org/regex',\n",
       " 'http://edamontology.org/repository',\n",
       " 'http://edamontology.org/thematic_editor',\n",
       " 'http://purl.obolibrary.org/obo/date',\n",
       " 'http://purl.obolibrary.org/obo/edam#data',\n",
       " 'http://purl.obolibrary.org/obo/edam#edam',\n",
       " 'http://purl.obolibrary.org/obo/edam#events',\n",
       " 'http://purl.obolibrary.org/obo/edam#formats',\n",
       " 'http://purl.obolibrary.org/obo/edam#identifiers',\n",
       " 'http://purl.obolibrary.org/obo/edam#obsolete',\n",
       " 'http://purl.obolibrary.org/obo/edam#operations',\n",
       " 'http://purl.obolibrary.org/obo/edam#placeholder',\n",
       " 'http://purl.obolibrary.org/obo/edam#topics',\n",
       " 'http://purl.obolibrary.org/obo/idspace',\n",
       " 'http://purl.obolibrary.org/obo/is_anti_symmetric',\n",
       " 'http://purl.obolibrary.org/obo/is_metadata_tag',\n",
       " 'http://purl.obolibrary.org/obo/is_reflexive',\n",
       " 'http://purl.obolibrary.org/obo/is_symmetric',\n",
       " 'http://purl.obolibrary.org/obo/namespace',\n",
       " 'http://purl.obolibrary.org/obo/remark',\n",
       " 'http://purl.obolibrary.org/obo/transitive_over',\n",
       " 'http://purl.org/dc/elements/1.1/contributor',\n",
       " 'http://purl.org/dc/elements/1.1/creator',\n",
       " 'http://purl.org/dc/elements/1.1/format',\n",
       " 'http://purl.org/dc/elements/1.1/title',\n",
       " 'http://usefulinc.com/ns/doap#Version',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#SubsetProperty',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#comment',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#consider',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasAlternativeId',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasDbXRef',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasDbXref',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasDefinition',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasExactSynonym',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasHumanReadableId',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#hasSubset',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#inSubset',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#isCyclic',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#replacedBy',\n",
       " 'http://www.geneontology.org/formats/oboInOwl#savedBy',\n",
       " 'http://xmlns.com/foaf/0.1/page']"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "annots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "ns3:citation a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Citation\" ;\n",
      "    ns3:created_in \"1.13\" ;\n",
      "    ns1:is_metadata_tag \"true\" ;\n",
      "    ns2:hasBroadSynonym \"Publication reference\" ;\n",
      "    ns2:hasDefinition \"'Citation' concept property ('citation' metadata tag) contains a dereferenceable URI, preferrably including a DOI, pointing to a citeable publication of the given data format.\" ;\n",
      "    ns2:hasRelatedSynonym \"Publication\" ;\n",
      "    ns2:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/comment_handle> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/created_in> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Created in\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"Version in which a concept was created.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/deprecation_comment> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"deprecation_comment\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"A comment explaining why the comment should be or was deprecated, including name of person commenting (jison, mkalas etc.)\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/documentation> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Documentation\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"'Documentation' trailing modifier (qualifier, 'documentation') of 'xref' links of 'Format' concepts. When 'true', the link is pointing to a page with explanation, description, documentation, or specification of the given data format.\" ;\n",
      "    ns1:hasRelatedSynonym \"Specification\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/example> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Example\" ;\n",
      "    ns1:is_metadata_tag \"true\" ;\n",
      "    ns2:hasDefinition \"'Example' concept property ('example' metadata tag) lists examples of valid values of types of identifiers (accessions). Applicable to some other types of data, too.\" ;\n",
      "    ns2:inSubset \"concept_properties\" ;\n",
      "    rdfs:comment \"Separated by bar ('|'). For more complex data and data formats, it can be a link to a website with examples, instead.\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/file_extension> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"File extension\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"'File extension' concept property ('file_extension' metadata tag) lists examples of usual file extensions of formats.\" ;\n",
      "    ns1:inSubset \"concept_properties\" ;\n",
      "    rdfs:comment \"N.B.: File extensions that are not correspondigly defined at http://filext.com are recorded in EDAM only if not in conflict with http://filext.com, and/or unique and usual within life-science computing.\",\n",
      "        \"Separated by bar ('|'), without a dot ('.') prefix, preferrably not all capital characters.\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/hasHumanReadableId> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"hasHumanReadableId\" ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#hasAlternativeId> .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/information_standard> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Information standard\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:comment \"\\\"Supported by the given data format\\\" here means, that the given format enables representation of data that satisfies the information standard.\" ;\n",
      "    ns1:hasDefinition \"'Information standard' trailing modifier (qualifier, 'information_standard') of 'xref' links of 'Format' concepts. When 'true', the link is pointing to an information standard supported by the given data format.\" ;\n",
      "    ns1:hasNarrowSynonym \"Minimum information checklist\",\n",
      "        \"Minimum information standard\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/is_deprecation_candidate> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"deprecation_candidate\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"When 'true', the concept has been proposed to be deprecated.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/is_refactor_candidate> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"refactor_candidate\" ;\n",
      "    ns1:is_metadata_tag \"true\" ;\n",
      "    ns2:hasDefinition \"When 'true', the concept has been proposed to be refactored.\" ;\n",
      "    ns2:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/isdebtag> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"isdebtag\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"When 'true', the concept has been proposed or is supported within Debian as a tag.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/media_type> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Media type\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"'Media type' trailing modifier (qualifier, 'media_type') of 'xref' links of 'Format' concepts. When 'true', the link is pointing to a page specifying a media type of the given data format.\" ;\n",
      "    ns1:hasNarrowSynonym \"MIME type\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/next_id> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/notRecommendedForAnnotation> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"notRecommendedForAnnotation\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"Whether terms associated with this concept are recommended for use in annotation.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/obsolete_since> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Obsolete since\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"Version in which a concept was made obsolete.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/oldParent> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Old parent\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"EDAM concept URI of the erstwhile \\\"parent\\\" of a now deprecated concept.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/oldRelated> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Old related\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"EDAM concept URI of an erstwhile related concept (by has_input, has_output, has_topic, is_format_of, etc.) of a now deprecated concept.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/ontology_used> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Ontology used\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"'Ontology used' concept property ('ontology_used' metadata tag) of format concepts links to a domain ontology that is used inside the given data format, or contains a note about ontology use within the format.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/organisation> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Organisation\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"'Organisation' trailing modifier (qualifier, 'organisation') of 'xref' links of 'Format' concepts. When 'true', the link is pointing to an organisation that developed, standardised, and maintains the given data format.\" ;\n",
      "    ns1:hasExactSynonym \"Organization\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/refactor_comment> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"refactor_comment\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"A comment explaining the proposed refactoring, including name of person commenting (jison, mkalas etc.)\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/regex> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Regular expression\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"'Regular expression' concept property ('regex' metadata tag) specifies the allowed values of types of identifiers (accessions). Applicable to some other types of data, too.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/repository> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"Repository\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"'Repository' trailing modifier (qualifier, 'repository') of 'xref' links of 'Format' concepts. When 'true', the link is pointing to the public source-code repository where the given data format is developed or maintained.\" ;\n",
      "    ns1:hasExactSynonym \"Public repository\",\n",
      "        \"Source-code repository\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://edamontology.org/thematic_editor> a owl:AnnotationProperty ;\n",
      "    rdfs:label \"thematic_editor\" ;\n",
      "    ns2:is_metadata_tag \"true\" ;\n",
      "    ns1:hasDefinition \"Name of thematic editor (http://biotools.readthedocs.io/en/latest/governance.html#registry-editors) responsible for this concept and its children.\" ;\n",
      "    ns1:inSubset \"concept_properties\" .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/date> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#data> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#edam> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#events> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#formats> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#identifiers> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#obsolete> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#operations> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#placeholder> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/edam#topics> a owl:AnnotationProperty ;\n",
      "    rdfs:subPropertyOf <http://www.geneontology.org/formats/oboInOwl#SubsetProperty> .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/idspace> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/is_anti_symmetric> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/is_metadata_tag> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/is_reflexive> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/is_symmetric> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/namespace> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/remark> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://purl.obolibrary.org/obo/transitive_over> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "dc:contributor a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "dc:creator a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "dc:format a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "dc:title a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "doap:Version a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#SubsetProperty> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#comment> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#consider> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasAlternativeId> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasDbXRef> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasDbXref> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasDefinition> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasHumanReadableId> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#hasSubset> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#inSubset> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#isCyclic> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#replacedBy> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "<http://www.geneontology.org/formats/oboInOwl#savedBy> a owl:AnnotationProperty .\n",
      "\n",
      "\n",
      "\n",
      "foaf:page a owl:AnnotationProperty .\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from rdflib import Namespace\n",
    "\n",
    "OBOOTHER = Namespace(\"http://purl.obolibrary.org/obo/\")\n",
    "kg.bind(\"oboOther\", OBOOTHER, override=True)\n",
    "\n",
    "classes = [\"http://edamontology.org/citation\", \"http://edamontology.org/created_in\"] \n",
    "\n",
    "for s in annots:\n",
    "    q = \"\"\"\n",
    "CONSTRUCT {\n",
    "    <\"\"\"+s+\"\"\"> ?p ?o \n",
    "} WHERE {\n",
    "    <\"\"\"+s+\"\"\"> ?p ?o \n",
    "}\n",
    "\"\"\"\n",
    "#    print(q)\n",
    "    kg_res = kg.query(q)\n",
    "    for l in kg_res.serialize(format=\"turtle\").decode().split(\"\\n\"):\n",
    "        if not \"@prefix\" in l:\n",
    "            print(l)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
