{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "abf0b4b2-9f5e-411f-8723-2b55acbf9d0b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading graph ...done!\n"
     ]
    }
   ],
   "source": [
    "from rdflib import ConjunctiveGraph, Namespace\n",
    "import json\n",
    "import os\n",
    "\n",
    "\n",
    "# Constants for script error reporting as per https://github.com/edamontology/edamverify.\n",
    "NOERR = \"NOERR\"\n",
    "INFO  = \"INFO\"\n",
    "WARN  = \"WARN\"\n",
    "ERROR = \"ERROR\"\n",
    "\n",
    "#Load EDAM_dev.owl from GitHub into an RDF graph.\n",
    "print(\"Loading graph ...\", end=\"\")\n",
    "g = ConjunctiveGraph()\n",
    "g.load(os.environ.get('EDAM_PATH', '../edamontology/EDAM_dev.owl'), format='xml')\n",
    "#g.load('https://raw.githubusercontent.com/edamontology/edamontology/master/EDAM_dev.owl', format='xml')\n",
    "# g.load('EDAM_dev.owl', format='xml')\n",
    "g.bind('edam', Namespace('http://edamontology.org#'))\n",
    "print(\"done!\")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "db5a02aa-ed1c-41dd-a7c1-47ed24229671",
   "metadata": {},
   "outputs": [],
   "source": [
    "sample_triple= \"\"\"\n",
    "\n",
    "<owl:Class rdf:about=\"http://edamontology.org/operation_0387\">\n",
    "        <rdfs:subClassOf rdf:resource=\"http://edamontology.org/operation_3351\"/>\n",
    "        <created_in>beta12orEarlier</created_in>\n",
    "        <oboInOwl:hasDbXref>WHATIF:AtomAccessibilityMolecular</oboInOwl:hasDbXref>\n",
    "        <oboInOwl:hasDbXref>WHATIF:AtomAccessibilityMolecularPlus</oboInOwl:hasDbXref>\n",
    "        <oboInOwl:hasDbXref>WHATIF:ResidueAccessibilityMolecular</oboInOwl:hasDbXref>\n",
    "        <oboInOwl:hasDbXref>WHATIF:ResidueAccessibilitySolvent</oboInOwl:hasDbXref>\n",
    "        <oboInOwl:hasDbXref>WHATIF:ResidueAccessibilityVacuum</oboInOwl:hasDbXref>\n",
    "        <oboInOwl:hasDbXref>WHATIF:ResidueAccessibilityVacuumMolecular</oboInOwl:hasDbXref>\n",
    "        <oboInOwl:hasDbXref>WHATIF:TotAccessibilityMolecular</oboInOwl:hasDbXref>\n",
    "        <oboInOwl:hasDbXref>WHATIF:TotAccessibilitySolvent</oboInOwl:hasDbXref>\n",
    "        <oboInOwl:hasDefinition>Calculate the molecular surface area in proteins and other macromolecules.</oboInOwl:hasDefinition>\n",
    "        <oboInOwl:hasNarrowSynonym>Protein atom surface calculation</oboInOwl:hasNarrowSynonym>\n",
    "        <oboInOwl:hasNarrowSynonym>Protein residue surface calculation</oboInOwl:hasNarrowSynonym>\n",
    "        <oboInOwl:hasNarrowSynonym>Protein surface and interior calculation</oboInOwl:hasNarrowSynonym>\n",
    "        <oboInOwl:hasNarrowSynonym>Protein surface calculation</oboInOwl:hasNarrowSynonym>\n",
    "        <oboInOwl:inSubset rdf:resource=\"http://purl.obolibrary.org/obo/edam#edam\"/>\n",
    "        <oboInOwl:inSubset rdf:resource=\"http://purl.obolibrary.org/obo/edam#operations\"/>\n",
    "        <rdfs:label>Molecular surface calculation</rdfs:label>\n",
    "    </owl:Class>.\n",
    "\"\"\"\n",
    "q = \"\"\"\n",
    "PREFIX obo: <http://purl.obolibrary.org/obo/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT DISTINCT ?entity ?property ?value WHERE { ?entity a owl:Class . ?entity ?property ?value . FILTER isLiteral(?value)}\n",
    "ORDER BY ?entity\n",
    "LIMIT 25\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "de4d387b-34e1-4925-9b96-9cb8c37d2d0a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25 \n",
      "http://edamontology.org/data_0005 , 'http://edamontology.org/created_in', beta12orEarlier \n",
      "http://edamontology.org/data_0005 , 'http://edamontology.org/obsolete_since', beta12orEarlier \n",
      "http://edamontology.org/data_0005 , 'http://www.geneontology.org/formats/oboInOwl#hasDefinition', A type of computational resource used in bioinformatics. \n",
      "http://edamontology.org/data_0005 , 'http://www.w3.org/2000/01/rdf-schema#label', Resource type \n",
      "http://edamontology.org/data_0005 , 'http://www.w3.org/2002/07/owl#deprecated', true \n",
      "http://edamontology.org/data_0006 , 'http://edamontology.org/created_in', beta12orEarlier \n",
      "http://edamontology.org/data_0006 , 'http://edamontology.org/notRecommendedForAnnotation', true \n",
      "http://edamontology.org/data_0006 , 'http://www.geneontology.org/formats/oboInOwl#hasDefinition', Information, represented in an information artefact (data record) that is 'understandable' by dedicated computational tools that can use the data as input or produce it as output. \n",
      "http://edamontology.org/data_0006 , 'http://www.geneontology.org/formats/oboInOwl#hasExactSynonym', Data record \n",
      "http://edamontology.org/data_0006 , 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym', Data set \n",
      "http://edamontology.org/data_0006 , 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym', Datum \n",
      "http://edamontology.org/data_0006 , 'http://www.w3.org/2000/01/rdf-schema#label', Data \n",
      "http://edamontology.org/data_0007 , 'http://edamontology.org/created_in', beta12orEarlier \n",
      "http://edamontology.org/data_0007 , 'http://edamontology.org/obsolete_since', beta12orEarlier \n",
      "http://edamontology.org/data_0007 , 'http://www.geneontology.org/formats/oboInOwl#hasDefinition', A bioinformatics package or tool, e.g. a standalone application or web service. \n",
      "http://edamontology.org/data_0007 , 'http://www.w3.org/2000/01/rdf-schema#label', Tool \n",
      "http://edamontology.org/data_0007 , 'http://www.w3.org/2002/07/owl#deprecated', true \n",
      "http://edamontology.org/data_0581 , 'http://edamontology.org/created_in', beta12orEarlier \n",
      "http://edamontology.org/data_0581 , 'http://edamontology.org/obsolete_since', beta12orEarlier \n",
      "http://edamontology.org/data_0581 , 'http://www.geneontology.org/formats/oboInOwl#hasDefinition', A digital data archive typically based around a relational model but sometimes using an object-oriented, tree or graph-based model. \n",
      "http://edamontology.org/data_0581 , 'http://www.w3.org/2000/01/rdf-schema#label', Database \n",
      "http://edamontology.org/data_0581 , 'http://www.w3.org/2002/07/owl#deprecated', true \n",
      "http://edamontology.org/data_0582 , 'http://edamontology.org/created_in', beta12orEarlier \n",
      "http://edamontology.org/data_0582 , 'http://www.geneontology.org/formats/oboInOwl#hasDefinition', An ontology of biological or bioinformatics concepts and relations, a controlled vocabulary, structured glossary etc. \n",
      "http://edamontology.org/data_0582 , 'http://www.w3.org/2000/01/rdf-schema#label', Ontology \n"
     ]
    }
   ],
   "source": [
    "\n",
    "    results = g.query(q)\n",
    "    print(f\"{len(results)} \")\n",
    "    for r in results:\n",
    "        print(f\"{r['entity']} , '{r['property']}', {r['value']} \")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "ccc052d2-835e-49da-b8bd-a0cf7dfffcea",
   "metadata": {},
   "outputs": [
    {
     "ename": "SAXParseException",
     "evalue": "<unknown>:3:0: unbound prefix",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mExpatError\u001b[0m                                Traceback (most recent call last)",
      "File \u001b[0;32m/usr/lib/python3.9/xml/sax/expatreader.py:217\u001b[0m, in \u001b[0;36mExpatParser.feed\u001b[0;34m(self, data, isFinal)\u001b[0m\n\u001b[1;32m    212\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    213\u001b[0m     \u001b[38;5;66;03m# The isFinal parameter is internal to the expat reader.\u001b[39;00m\n\u001b[1;32m    214\u001b[0m     \u001b[38;5;66;03m# If it is set to true, expat will check validity of the entire\u001b[39;00m\n\u001b[1;32m    215\u001b[0m     \u001b[38;5;66;03m# document. When feeding chunks, they are not normally final -\u001b[39;00m\n\u001b[1;32m    216\u001b[0m     \u001b[38;5;66;03m# except when invoked from close.\u001b[39;00m\n\u001b[0;32m--> 217\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_parser\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mParse\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdata\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43misFinal\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    218\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m expat\u001b[38;5;241m.\u001b[39merror \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "\u001b[0;31mExpatError\u001b[0m: unbound prefix: line 3, column 0",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mSAXParseException\u001b[0m                         Traceback (most recent call last)",
      "Input \u001b[0;32mIn [6]\u001b[0m, in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      1\u001b[0m g2 \u001b[38;5;241m=\u001b[39m ConjunctiveGraph()\n\u001b[0;32m----> 2\u001b[0m \u001b[43mg2\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mparse\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdata\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43msample_triple\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mformat\u001b[39;49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mxml\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[1;32m      3\u001b[0m \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;28mlen\u001b[39m(g2))\n",
      "File \u001b[0;32m~/.local/lib/python3.9/site-packages/rdflib/graph.py:1749\u001b[0m, in \u001b[0;36mConjunctiveGraph.parse\u001b[0;34m(self, source, publicID, format, location, file, data, **args)\u001b[0m\n\u001b[1;32m   1747\u001b[0m context \u001b[38;5;241m=\u001b[39m Graph(store\u001b[38;5;241m=\u001b[39m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mstore, identifier\u001b[38;5;241m=\u001b[39mg_id)\n\u001b[1;32m   1748\u001b[0m context\u001b[38;5;241m.\u001b[39mremove((\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;28;01mNone\u001b[39;00m))  \u001b[38;5;66;03m# hmm ?\u001b[39;00m\n\u001b[0;32m-> 1749\u001b[0m \u001b[43mcontext\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mparse\u001b[49m\u001b[43m(\u001b[49m\u001b[43msource\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mpublicID\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mpublicID\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mformat\u001b[39;49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mformat\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1750\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m context\n",
      "File \u001b[0;32m~/.local/lib/python3.9/site-packages/rdflib/graph.py:1252\u001b[0m, in \u001b[0;36mGraph.parse\u001b[0;34m(self, source, publicID, format, location, file, data, **args)\u001b[0m\n\u001b[1;32m   1250\u001b[0m parser \u001b[38;5;241m=\u001b[39m plugin\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;28mformat\u001b[39m, Parser)()\n\u001b[1;32m   1251\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1252\u001b[0m     \u001b[43mparser\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mparse\u001b[49m\u001b[43m(\u001b[49m\u001b[43msource\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43margs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1253\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mSyntaxError\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m se:\n\u001b[1;32m   1254\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m could_not_guess_format:\n",
      "File \u001b[0;32m~/.local/lib/python3.9/site-packages/rdflib/plugins/parsers/rdfxml.py:608\u001b[0m, in \u001b[0;36mRDFXMLParser.parse\u001b[0;34m(self, source, sink, **args)\u001b[0m\n\u001b[1;32m    604\u001b[0m     content_handler\u001b[38;5;241m.\u001b[39mpreserve_bnode_ids \u001b[38;5;241m=\u001b[39m preserve_bnode_ids\n\u001b[1;32m    605\u001b[0m \u001b[38;5;66;03m# # We're only using it once now\u001b[39;00m\n\u001b[1;32m    606\u001b[0m \u001b[38;5;66;03m# content_handler.reset()\u001b[39;00m\n\u001b[1;32m    607\u001b[0m \u001b[38;5;66;03m# self._parser.reset()\u001b[39;00m\n\u001b[0;32m--> 608\u001b[0m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_parser\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mparse\u001b[49m\u001b[43m(\u001b[49m\u001b[43msource\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/usr/lib/python3.9/xml/sax/expatreader.py:111\u001b[0m, in \u001b[0;36mExpatParser.parse\u001b[0;34m(self, source)\u001b[0m\n\u001b[1;32m    109\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mreset()\n\u001b[1;32m    110\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_cont_handler\u001b[38;5;241m.\u001b[39msetDocumentLocator(ExpatLocator(\u001b[38;5;28mself\u001b[39m))\n\u001b[0;32m--> 111\u001b[0m     \u001b[43mxmlreader\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mIncrementalParser\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mparse\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43msource\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    112\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m:\n\u001b[1;32m    113\u001b[0m     \u001b[38;5;66;03m# bpo-30264: Close the source on error to not leak resources:\u001b[39;00m\n\u001b[1;32m    114\u001b[0m     \u001b[38;5;66;03m# xml.sax.parse() doesn't give access to the underlying parser\u001b[39;00m\n\u001b[1;32m    115\u001b[0m     \u001b[38;5;66;03m# to the caller\u001b[39;00m\n\u001b[1;32m    116\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_close_source()\n",
      "File \u001b[0;32m/usr/lib/python3.9/xml/sax/xmlreader.py:125\u001b[0m, in \u001b[0;36mIncrementalParser.parse\u001b[0;34m(self, source)\u001b[0m\n\u001b[1;32m    123\u001b[0m buffer \u001b[38;5;241m=\u001b[39m file\u001b[38;5;241m.\u001b[39mread(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_bufsize)\n\u001b[1;32m    124\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m buffer:\n\u001b[0;32m--> 125\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfeed\u001b[49m\u001b[43m(\u001b[49m\u001b[43mbuffer\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    126\u001b[0m     buffer \u001b[38;5;241m=\u001b[39m file\u001b[38;5;241m.\u001b[39mread(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_bufsize)\n\u001b[1;32m    127\u001b[0m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mclose()\n",
      "File \u001b[0;32m/usr/lib/python3.9/xml/sax/expatreader.py:221\u001b[0m, in \u001b[0;36mExpatParser.feed\u001b[0;34m(self, data, isFinal)\u001b[0m\n\u001b[1;32m    219\u001b[0m exc \u001b[38;5;241m=\u001b[39m SAXParseException(expat\u001b[38;5;241m.\u001b[39mErrorString(e\u001b[38;5;241m.\u001b[39mcode), e, \u001b[38;5;28mself\u001b[39m)\n\u001b[1;32m    220\u001b[0m \u001b[38;5;66;03m# FIXME: when to invoke error()?\u001b[39;00m\n\u001b[0;32m--> 221\u001b[0m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_err_handler\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfatalError\u001b[49m\u001b[43m(\u001b[49m\u001b[43mexc\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/usr/lib/python3.9/xml/sax/handler.py:38\u001b[0m, in \u001b[0;36mErrorHandler.fatalError\u001b[0;34m(self, exception)\u001b[0m\n\u001b[1;32m     36\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mfatalError\u001b[39m(\u001b[38;5;28mself\u001b[39m, exception):\n\u001b[1;32m     37\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mHandle a non-recoverable error.\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[0;32m---> 38\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m exception\n",
      "\u001b[0;31mSAXParseException\u001b[0m: <unknown>:3:0: unbound prefix"
     ]
    }
   ],
   "source": [
    "g2 = ConjunctiveGraph()\n",
    "g2.parse(data=sample_triple, format=\"xml\")\n",
    "print(len(g2))\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c2c939c6-15e8-4729-b0ec-75cfdd19f78d",
   "metadata": {},
   "outputs": [],
   "source": [
    "res = g2.query(q)\n",
    "print(len(res))\n",
    "for r in res: \n",
    "    print(r) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "acacd029-ab1c-4f78-a443-5d4b103e73a9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "206 \n",
      "http://edamontology.org/data_0006 Data  \n",
      "http://edamontology.org/data_0842 Identifier  \n",
      "http://edamontology.org/data_0976 Identifier (by type of data)  \n",
      "http://edamontology.org/data_0977 Tool identifier  \n",
      "http://edamontology.org/data_0982 Molecule identifier  \n",
      "http://edamontology.org/data_0983 Atom ID  \n",
      "http://edamontology.org/data_0984 Molecule name  \n",
      "http://edamontology.org/data_0988 Peptide identifier  \n",
      "http://edamontology.org/data_0989 Protein identifier  \n",
      "http://edamontology.org/data_0993 Drug identifier  \n",
      "http://edamontology.org/data_0994 Amino acid identifier  \n",
      "http://edamontology.org/data_0995 Nucleotide identifier  \n",
      "http://edamontology.org/data_0996 Monosaccharide identifier  \n",
      "http://edamontology.org/data_1025 Gene identifier  \n",
      "http://edamontology.org/data_1038 Protein domain ID  \n",
      "http://edamontology.org/data_1047 URI  \n",
      "http://edamontology.org/data_1048 Database ID  \n",
      "http://edamontology.org/data_1063 Sequence identifier  \n",
      "http://edamontology.org/data_1064 Sequence set ID  \n",
      "http://edamontology.org/data_1066 Sequence alignment ID  \n",
      "http://edamontology.org/data_1068 Phylogenetic tree ID  \n",
      "http://edamontology.org/data_1069 Comparison matrix identifier  \n",
      "http://edamontology.org/data_1070 Structure ID  \n",
      "http://edamontology.org/data_1071 Structural (3D) profile ID  \n",
      "http://edamontology.org/data_1072 Structure alignment ID  \n",
      "http://edamontology.org/data_1073 Amino acid index ID  \n",
      "http://edamontology.org/data_1074 Protein interaction ID  \n",
      "http://edamontology.org/data_1075 Protein family identifier  \n",
      "http://edamontology.org/data_1077 Transcription factor identifier  \n",
      "http://edamontology.org/data_1078 Experiment annotation ID  \n",
      "http://edamontology.org/data_1079 Electron microscopy model ID  \n",
      "http://edamontology.org/data_1080 Gene expression report ID  \n",
      "http://edamontology.org/data_1081 Genotype and phenotype annotation ID  \n",
      "http://edamontology.org/data_1082 Pathway or network identifier  \n",
      "http://edamontology.org/data_1083 Workflow ID  \n",
      "http://edamontology.org/data_1084 Data resource definition ID  \n",
      "http://edamontology.org/data_1085 Biological model ID  \n",
      "http://edamontology.org/data_1086 Compound identifier  \n",
      "http://edamontology.org/data_1088 Article ID  \n",
      "http://edamontology.org/data_1093 Sequence accession  \n",
      "http://edamontology.org/data_1096 Sequence accession (protein)  \n",
      "http://edamontology.org/data_1097 Sequence accession (nucleic acid)  \n",
      "http://edamontology.org/data_1112 Sequence cluster ID  \n",
      "http://edamontology.org/data_1114 Sequence motif identifier  \n",
      "http://edamontology.org/data_1115 Sequence profile ID  \n",
      "http://edamontology.org/data_1150 Disease ID  \n",
      "http://edamontology.org/data_1855 Clone ID  \n",
      "http://edamontology.org/data_1869 Organism identifier  \n",
      "http://edamontology.org/data_1893 Locus ID  \n",
      "http://edamontology.org/data_1916 Alignment  \n",
      "http://edamontology.org/data_2091 Accession  \n",
      "http://edamontology.org/data_2098 Job identifier  \n",
      "http://edamontology.org/data_2099 Name  \n",
      "http://edamontology.org/data_2108 Reaction ID  \n",
      "http://edamontology.org/data_2109 Identifier (hybrid)  \n",
      "http://edamontology.org/data_2110 Molecular property identifier  \n",
      "http://edamontology.org/data_2111 Codon usage table ID  \n",
      "http://edamontology.org/data_2117 Map identifier  \n",
      "http://edamontology.org/data_2118 Person identifier  \n",
      "http://edamontology.org/data_2119 Nucleic acid identifier  \n",
      "http://edamontology.org/data_2127 Genetic code identifier  \n",
      "http://edamontology.org/data_2208 Plasmid identifier  \n",
      "http://edamontology.org/data_2209 Mutation ID  \n",
      "http://edamontology.org/data_2294 Sequence variation ID  \n",
      "http://edamontology.org/data_2295 Gene ID  \n",
      "http://edamontology.org/data_2321 Enzyme ID  \n",
      "http://edamontology.org/data_2355 RNA family identifier  \n",
      "http://edamontology.org/data_2362 Sequence accession (hybrid)  \n",
      "http://edamontology.org/data_2365 Pathway or network accession  \n",
      "http://edamontology.org/data_2373 Spot ID  \n",
      "http://edamontology.org/data_2379 Strain identifier  \n",
      "http://edamontology.org/data_2382 Genotype experiment ID  \n",
      "http://edamontology.org/data_2538 Mutation identifier  \n",
      "http://edamontology.org/data_2576 Toxin identifier  \n",
      "http://edamontology.org/data_2587 Blot ID  \n",
      "http://edamontology.org/data_2596 Catalogue ID  \n",
      "http://edamontology.org/data_2618 Protein modification ID  \n",
      "http://edamontology.org/data_2630 Mobile genetic element ID  \n",
      "http://edamontology.org/data_2633 Book ID  \n",
      "http://edamontology.org/data_2638 PubChem bioassay ID  \n",
      "http://edamontology.org/data_2655 Cell type identifier  \n",
      "http://edamontology.org/data_2663 Carbohydrate identifier  \n",
      "http://edamontology.org/data_2718 Oligonucleotide ID  \n",
      "http://edamontology.org/data_2727 Promoter ID  \n",
      "http://edamontology.org/data_2749 Genome identifier  \n",
      "http://edamontology.org/data_2769 Transcript ID  \n",
      "http://edamontology.org/data_2779 Stock number  \n",
      "http://edamontology.org/data_2785 Virus ID  \n",
      "http://edamontology.org/data_2790 Gel ID  \n",
      "http://edamontology.org/data_2794 ORF ID  \n",
      "http://edamontology.org/data_2795 ORF identifier  \n",
      "http://edamontology.org/data_2812 Lipid identifier  \n",
      "http://edamontology.org/data_2891 Biological model accession  \n",
      "http://edamontology.org/data_2893 Cell type accession  \n",
      "http://edamontology.org/data_2894 Compound accession  \n",
      "http://edamontology.org/data_2895 Drug accession  \n",
      "http://edamontology.org/data_2897 Toxin accession  \n",
      "http://edamontology.org/data_2898 Monosaccharide accession  \n",
      "http://edamontology.org/data_2900 Carbohydrate accession  \n",
      "http://edamontology.org/data_2901 Molecule accession  \n",
      "http://edamontology.org/data_2902 Data resource definition accession  \n",
      "http://edamontology.org/data_2903 Genome accession  \n",
      "http://edamontology.org/data_2904 Map accession  \n",
      "http://edamontology.org/data_2905 Lipid accession  \n",
      "http://edamontology.org/data_2906 Peptide ID  \n",
      "http://edamontology.org/data_2907 Protein accession  \n",
      "http://edamontology.org/data_2908 Organism accession  \n",
      "http://edamontology.org/data_2910 Protein family accession  \n",
      "http://edamontology.org/data_2911 Transcription factor accession  \n",
      "http://edamontology.org/data_2912 Strain accession  \n",
      "http://edamontology.org/data_2913 Virus identifier  \n",
      "http://edamontology.org/data_3034 Sequence feature identifier  \n",
      "http://edamontology.org/data_3035 Structure identifier  \n",
      "http://edamontology.org/data_3036 Matrix identifier  \n",
      "http://edamontology.org/data_3266 Sequence assembly ID  \n",
      "http://edamontology.org/data_3273 Sample ID  \n",
      "http://edamontology.org/data_3358 Format identifier  \n",
      "http://edamontology.org/data_3667 Disease identifier  \n",
      "http://edamontology.org/format_1915 Format  \n",
      "http://edamontology.org/format_1919 Sequence record format  \n",
      "http://edamontology.org/format_1920 Sequence feature annotation format  \n",
      "http://edamontology.org/format_1921 Alignment format  \n",
      "http://edamontology.org/format_2006 Phylogenetic tree format  \n",
      "http://edamontology.org/format_2013 Biological pathway or network format  \n",
      "http://edamontology.org/format_2014 Sequence-profile alignment format  \n",
      "http://edamontology.org/format_2020 Article format  \n",
      "http://edamontology.org/format_2021 Text mining report format  \n",
      "http://edamontology.org/format_2027 Enzyme kinetics report format  \n",
      "http://edamontology.org/format_2030 Chemical data format  \n",
      "http://edamontology.org/format_2031 Gene annotation format  \n",
      "http://edamontology.org/format_2032 Workflow format  \n",
      "http://edamontology.org/format_2033 Tertiary structure format  \n",
      "http://edamontology.org/format_2035 Chemical formula format  \n",
      "http://edamontology.org/format_2036 Phylogenetic character data format  \n",
      "http://edamontology.org/format_2049 Phylogenetic tree report (tree distances) format  \n",
      "http://edamontology.org/format_2052 Protein family report format  \n",
      "http://edamontology.org/format_2054 Protein interaction format  \n",
      "http://edamontology.org/format_2055 Sequence assembly format  \n",
      "http://edamontology.org/format_2058 Gene expression report format  \n",
      "http://edamontology.org/format_2060 Map format  \n",
      "http://edamontology.org/format_2061 Nucleic acid features (primers) format  \n",
      "http://edamontology.org/format_2062 Protein report format  \n",
      "http://edamontology.org/format_2065 Protein structure report (quality evaluation) format  \n",
      "http://edamontology.org/format_2066 Database hits (sequence) format  \n",
      "http://edamontology.org/format_2067 Sequence distance matrix format  \n",
      "http://edamontology.org/format_2068 Sequence motif format  \n",
      "http://edamontology.org/format_2069 Sequence profile format  \n",
      "http://edamontology.org/format_2074 Dirichlet distribution format  \n",
      "http://edamontology.org/format_2075 HMM emission and transition counts format  \n",
      "http://edamontology.org/format_2076 RNA secondary structure format  \n",
      "http://edamontology.org/format_2077 Protein secondary structure format  \n",
      "http://edamontology.org/format_2078 Sequence range format  \n",
      "http://edamontology.org/format_2155 Sequence features (repeats) format  \n",
      "http://edamontology.org/format_2158 Nucleic acid features (restriction sites) format  \n",
      "http://edamontology.org/format_2170 Sequence cluster format  \n",
      "http://edamontology.org/format_2195 Ontology format  \n",
      "http://edamontology.org/format_2333 Binary format  \n",
      "http://edamontology.org/format_2350 Format (by type of data)  \n",
      "http://edamontology.org/format_2571 Raw sequence format  \n",
      "http://edamontology.org/format_2848 Bibliographic reference format  \n",
      "http://edamontology.org/format_2921 Sequence variation annotation format  \n",
      "http://edamontology.org/format_3033 Matrix format  \n",
      "http://edamontology.org/format_3097 Protein domain classification format  \n",
      "http://edamontology.org/format_3166 Biological pathway or network report format  \n",
      "http://edamontology.org/format_3167 Experiment annotation format  \n",
      "http://edamontology.org/format_3245 Mass spectrometry data format  \n",
      "http://edamontology.org/format_3287 Individual genetic data format  \n",
      "http://edamontology.org/format_3326 Data index format  \n",
      "http://edamontology.org/format_3507 Document format  \n",
      "http://edamontology.org/format_3547 Image format  \n",
      "http://edamontology.org/format_3606 Sequence quality report format (text)  \n",
      "http://edamontology.org/format_3617 Graph format  \n",
      "http://edamontology.org/format_3706 Biodiversity data format  \n",
      "http://edamontology.org/format_3748 Linked data format  \n",
      "http://edamontology.org/format_3780 Annotated text format  \n",
      "http://edamontology.org/format_3787 Query language  \n",
      "http://edamontology.org/format_3824 NMR data format  \n",
      "http://edamontology.org/format_3828 Raw microarray data format  \n",
      "http://edamontology.org/format_3866 Trajectory format  \n",
      "http://edamontology.org/format_3867 Trajectory format (binary)  \n",
      "http://edamontology.org/format_3868 Trajectory format (text)  \n",
      "http://edamontology.org/format_3879 Topology format  \n",
      "http://edamontology.org/operation_0004 Operation  \n",
      "http://edamontology.org/operation_0226 Annotation  \n",
      "http://edamontology.org/operation_0227 Indexing  \n",
      "http://edamontology.org/operation_0337 Visualisation  \n",
      "http://edamontology.org/operation_0538 Phylogenetic inference (data centric)  \n",
      "http://edamontology.org/operation_0539 Phylogenetic inference (method centric)  \n",
      "http://edamontology.org/operation_2409 Data handling  \n",
      "http://edamontology.org/operation_2423 Prediction and recognition  \n",
      "http://edamontology.org/operation_2424 Comparison  \n",
      "http://edamontology.org/operation_2425 Optimisation and refinement  \n",
      "http://edamontology.org/operation_2426 Modelling and simulation  \n",
      "http://edamontology.org/operation_2428 Validation  \n",
      "http://edamontology.org/operation_2429 Mapping  \n",
      "http://edamontology.org/operation_2430 Design  \n",
      "http://edamontology.org/operation_2928 Alignment  \n",
      "http://edamontology.org/operation_2945 Analysis  \n",
      "http://edamontology.org/operation_2990 Classification  \n",
      "http://edamontology.org/operation_3429 Generation  \n",
      "http://edamontology.org/operation_3432 Clustering  \n",
      "http://edamontology.org/operation_3434 Conversion  \n",
      "http://edamontology.org/operation_3438 Calculation  \n",
      "http://edamontology.org/operation_3465 Correlation  \n",
      "http://edamontology.org/operation_3799 Quantification  \n",
      "http://edamontology.org/topic_0003 Topic  \n"
     ]
    }
   ],
   "source": [
    "    results = g.query(q)\n",
    "    print(f\"{len(results)} \")\n",
    "    for r in results:\n",
    "        print(f\"{r['entity']} {r['label']}  \")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6d2fb495-193d-427b-9ba7-9a28222fb8be",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
