{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Loading EDAM into an RDFlib graph"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2024-02-29T13:59:09.647266Z",
     "start_time": "2024-02-29T13:59:09.596489Z"
    }
   },
   "outputs": [],
   "source": [
    "from rdflib import ConjunctiveGraph"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Here we initialize the graph. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2024-02-29T13:59:09.649352Z",
     "start_time": "2024-02-29T13:59:09.644271Z"
    }
   },
   "outputs": [],
   "source": [
    "kg = ConjunctiveGraph()\n",
    "\n",
    "def print_size():\n",
    "    print(f\"The knowledge graph has {len(kg)} triples\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Here we load the EDAM ontology into the graph. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2024-02-29T13:59:12.865741Z",
     "start_time": "2024-02-29T13:59:09.648952Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "38279 triples in the EDAM triple store\n"
     ]
    }
   ],
   "source": [
    "#kg.parse('http://edamontology.org/EDAM.owl', format='xml')\n",
    "#print_size()\n",
    "edam_version = 'https://raw.githubusercontent.com/edamontology/edamontology/master/EDAM_dev.owl'\n",
    "kg = ConjunctiveGraph()\n",
    "kg.parse(edam_version, format='xml')\n",
    "#kg.bind('edam', Namespace('http://edamontology.org/'))\n",
    "print(str(len(kg)) + ' triples in the EDAM triple store')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2024-02-29T13:59:14.362831Z",
     "start_time": "2024-02-29T13:59:12.866174Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": "<Graph identifier=N561854ad78394981a29188bdf9e484fb (<class 'rdflib.graph.ConjunctiveGraph'>)>"
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "kg.serialize(\"edam.json\", format=\"json-ld\")\n",
    "kg.serialize(\"edam.ttl\", format=\"turtle\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2024-02-29T13:59:17.313108Z",
     "start_time": "2024-02-29T13:59:14.359360Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "38279\n"
     ]
    }
   ],
   "source": [
    "# a single function to load EDAM and get the graph object as a result\n",
    "def load_EDAM():\n",
    "    g = ConjunctiveGraph()\n",
    "    g.parse(edam_version, format='xml')\n",
    "    return g\n",
    "\n",
    "G = load_EDAM()\n",
    "print(len(G))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Listing the 100 first triples  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2024-02-29T13:59:20.850223Z",
     "start_time": "2024-02-29T13:59:20.827220Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(http://edamontology.org/data_0874, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/data)\n",
      "(http://edamontology.org/format_1980, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/topic_3415, http://edamontology.org/isdebtag, true)\n",
      "(http://edamontology.org/operation_0004, http://www.w3.org/2002/07/owl#disjointWith, http://edamontology.org/topic_0003)\n",
      "(http://edamontology.org/topic_2828, http://www.w3.org/2000/01/rdf-schema#label, X-ray diffraction)\n",
      "(http://edamontology.org/operation_3431, http://www.geneontology.org/formats/oboInOwl#hasExactSynonym, Database submission)\n",
      "(http://edamontology.org/format_2182, http://www.geneontology.org/formats/oboInOwl#hasDefinition, A text format resembling FASTQ short read format.)\n",
      "(http://edamontology.org/data_1656, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_2198, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/topic_4020, http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym, Biogeochemical cycle)\n",
      "(N93e90f33ebb749a486b5895d3f0446a8, http://www.w3.org/2002/07/owl#onProperty, http://edamontology.org/is_identifier_of)\n",
      "(N2f3376d119664710a081491fdde147ac, http://www.w3.org/2002/07/owl#someValuesFrom, http://edamontology.org/data_0962)\n",
      "(http://edamontology.org/topic_3541, http://edamontology.org/obsolete_since, 1.13)\n",
      "(http://edamontology.org/data_2966, http://edamontology.org/created_in, beta12orEarlier)\n",
      "(http://edamontology.org/operation_0402, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/operation_0337)\n",
      "(N5a05d2a1e169418b82b9b5d52cd72f75, http://www.w3.org/2002/07/owl#someValuesFrom, http://edamontology.org/data_2048)\n",
      "(http://edamontology.org/data_2365, http://www.geneontology.org/formats/oboInOwl#hasDefinition, A persistent, unique identifier of a biological pathway or network (typically a database entry).)\n",
      "(http://edamontology.org/data_3111, http://edamontology.org/created_in, beta13)\n",
      "(http://edamontology.org/data_2959, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://www.w3.org/2002/07/owl#DeprecatedClass)\n",
      "(http://edamontology.org/operation_0317, http://edamontology.org/obsolete_since, beta12orEarlier)\n",
      "(http://edamontology.org/format_1610, http://www.w3.org/2000/01/rdf-schema#label, KEGG GENES gene report format)\n",
      "(http://edamontology.org/data_2220, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/identifiers)\n",
      "(N4865fa91aa4a4909b84c55fbb68eaf6a, http://www.w3.org/2002/07/owl#onProperty, http://edamontology.org/has_input)\n",
      "(http://edamontology.org/data_1560, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_2019, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/operation_3927, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/bio)\n",
      "(http://edamontology.org/data_2385, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/identifiers)\n",
      "(http://edamontology.org/operation_0312, http://www.w3.org/2000/01/rdf-schema#label, Sequencing-based expression profile data processing)\n",
      "(http://edamontology.org/topic_0078, http://www.geneontology.org/formats/oboInOwl#hasHumanReadableId, Proteins)\n",
      "(http://edamontology.org/data_2090, http://www.w3.org/2002/07/owl#deprecated, true)\n",
      "(http://edamontology.org/operation_3896, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_0845, http://www.w3.org/2000/01/rdf-schema#label, Molecular charge)\n",
      "(http://edamontology.org/format_1551, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Format of output of the Pcons Model Quality Assessment Program (MQAP).)\n",
      "(http://edamontology.org/data_1234, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/bio)\n",
      "(http://edamontology.org/data_2902, http://edamontology.org/created_in, beta12orEarlier)\n",
      "(http://edamontology.org/data_1007, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/data)\n",
      "(http://edamontology.org/data_1567, http://edamontology.org/obsolete_since, 1.8)\n",
      "(http://edamontology.org/data_2085, http://www.geneontology.org/formats/oboInOwl#hasExactSynonym, Structure-derived report)\n",
      "(http://edamontology.org/format_4035, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/format_1475)\n",
      "(http://edamontology.org/operation_0579, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/operation_0573)\n",
      "(http://edamontology.org/data_2103, http://edamontology.org/obsolete_since, 1.3)\n",
      "(http://edamontology.org/operation_2407, http://www.w3.org/2002/07/owl#deprecated, true)\n",
      "(http://edamontology.org/data_1469, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_1368, http://www.geneontology.org/formats/oboInOwl#consider, http://edamontology.org/data_1354)\n",
      "(http://edamontology.org/topic_2811, http://edamontology.org/obsolete_since, 1.3)\n",
      "(http://edamontology.org/format_4039, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/format_2330)\n",
      "(http://edamontology.org/format_3970, http://www.w3.org/2000/01/rdf-schema#label, Vega-lite)\n",
      "(N2466636111704f33969af63f49b373fe, http://www.w3.org/2002/07/owl#someValuesFrom, http://edamontology.org/topic_0123)\n",
      "(http://edamontology.org/operation_0525, http://www.w3.org/2000/01/rdf-schema#label, Genome assembly)\n",
      "(http://edamontology.org/format_3825, http://edamontology.org/created_in, 1.20)\n",
      "(http://edamontology.org/data_1597, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/bio)\n",
      "(http://edamontology.org/format_3009, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/formats)\n",
      "(http://edamontology.org/format_1197, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Chemical structure specified in IUPAC International Chemical Identifier (InChI) line notation.)\n",
      "(http://edamontology.org/format_3818, http://edamontology.org/documentation, http://ccg.vital-it.ch/chipseq/elandformat.php)\n",
      "(http://edamontology.org/format_1606, http://edamontology.org/oldParent, http://www.w3.org/2002/07/owl#Thing)\n",
      "(http://edamontology.org/data_2335, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/format_3556, http://www.w3.org/2000/01/rdf-schema#label, MHTML)\n",
      "(N9ee8dc747e754712b3b1e4595251638d, http://www.w3.org/2002/07/owl#onProperty, http://edamontology.org/is_format_of)\n",
      "(http://edamontology.org/data_2294, http://edamontology.org/created_in, beta12orEarlier)\n",
      "(http://edamontology.org/data_1035, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/data_2091)\n",
      "(http://edamontology.org/format_2919, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/formats)\n",
      "(http://edamontology.org/operation_0253, http://www.geneontology.org/formats/oboInOwl#hasExactSynonym, Sequence feature prediction)\n",
      "(http://edamontology.org/operation_0357, http://edamontology.org/oldParent, http://www.w3.org/2002/07/owl#Thing)\n",
      "(http://edamontology.org/format_1969, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/bio)\n",
      "(http://edamontology.org/operation_1818, http://www.geneontology.org/formats/oboInOwl#replacedBy, http://edamontology.org/operation_0387)\n",
      "(http://edamontology.org/topic_0779, http://www.geneontology.org/formats/oboInOwl#hasDefinition, Mitochondria, typically of mitochondrial genes and proteins.)\n",
      "(http://edamontology.org/operation_0342, http://edamontology.org/oldParent, http://www.w3.org/2002/07/owl#Thing)\n",
      "(http://edamontology.org/operation_0295, http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym, 3D profile alignment)\n",
      "(http://edamontology.org/operation_2507, http://edamontology.org/oldParent, http://edamontology.org/operation_2501)\n",
      "(Nf2c8f98522d54c3cbe51cbf5c273c597, http://www.w3.org/2002/07/owl#annotatedTarget, http://www.w3.org/2000/01/rdf-schema#label)\n",
      "(http://edamontology.org/operation_3088, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(N87dc94b8ec5e4282ad583cf4d85dc4d5, http://www.w3.org/2002/07/owl#someValuesFrom, http://edamontology.org/data_1461)\n",
      "(http://edamontology.org/data_0884, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://www.w3.org/2002/07/owl#DeprecatedClass)\n",
      "(http://edamontology.org/format_3327, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/format_2333)\n",
      "(http://edamontology.org/format_3879, http://www.geneontology.org/formats/oboInOwl#hasExactSynonym, CG topology format)\n",
      "(http://edamontology.org/operation_2404, http://www.w3.org/2000/01/rdf-schema#label, Sequence motif analysis)\n",
      "(http://edamontology.org/data_0963, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/topic_3343, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/topics)\n",
      "(http://edamontology.org/format_1651, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/operation_4033, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/operation_3664)\n",
      "(http://edamontology.org/data_1592, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/operation_3929, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/operation_2423)\n",
      "(http://edamontology.org/data_1069, http://edamontology.org/notRecommendedForAnnotation, true)\n",
      "(http://edamontology.org/operation_2502, http://edamontology.org/oldParent, http://edamontology.org/operation_2945)\n",
      "(http://edamontology.org/data_2354, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/data)\n",
      "(http://edamontology.org/data_2615, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/data)\n",
      "(http://edamontology.org/operation_3279, http://edamontology.org/created_in, 1.3)\n",
      "(http://edamontology.org/format_3549, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/format_3547)\n",
      "(http://edamontology.org/topic_0157, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/topics)\n",
      "(http://edamontology.org/data_2318, http://www.w3.org/2000/01/rdf-schema#subClassOf, http://edamontology.org/data_2316)\n",
      "(Nce0b14eb8c574a78a8ddeb7abc74ff0a, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Restriction)\n",
      "(http://edamontology.org/data_1767, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/data_1726, http://www.w3.org/2002/07/owl#deprecated, true)\n",
      "(http://edamontology.org/data_2674, http://www.w3.org/2000/01/rdf-schema#label, Ensembl ID ('Cavia porcellus'))\n",
      "(http://edamontology.org/format_1950, http://www.w3.org/2000/01/rdf-schema#label, pdbatom)\n",
      "(Nbb7120a9d7524db1b42250501a73b78e, http://www.w3.org/2002/07/owl#someValuesFrom, http://edamontology.org/data_1636)\n",
      "(http://edamontology.org/data_1157, http://www.w3.org/1999/02/22-rdf-syntax-ns#type, http://www.w3.org/2002/07/owl#Class)\n",
      "(http://edamontology.org/format_1614, http://www.w3.org/2002/07/owl#deprecated, true)\n",
      "(http://edamontology.org/data_1143, http://www.geneontology.org/formats/oboInOwl#inSubset, http://edamontology.org/identifiers)\n",
      "(http://edamontology.org/format_3248, http://www.geneontology.org/formats/oboInOwl#hasDefinition, mzQuantML is the format for quantitation values associated with peptides, proteins and small molecules from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of quantitation software for proteomics.)\n"
     ]
    }
   ],
   "source": [
    "i = 0\n",
    "\n",
    "for subject,predicate,obj in kg:\n",
    "    print(f'({subject}, {predicate}, {obj})')\n",
    "    i+=1\n",
    "    \n",
    "    if i > 99:\n",
    "        break\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2024-02-29T13:59:23.067118Z",
     "start_time": "2024-02-29T13:59:23.059459Z"
    }
   },
   "outputs": [],
   "source": [
    "from rdflib.namespace import RDF, RDFS, OWL \n",
    "\n",
    "i = 0\n",
    "\n",
    "\n",
    "for s in kg.triples((None, RDF.type, OWL.Class)):\n",
    "    for label in kg.triples((s, RDFS.label, None)):\n",
    "        print(kg.value(s, RDFS.label))\n",
    "        i +=1\n",
    "    \n",
    "    if i > 99:\n",
    "        break\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Evaluating SPARQL queries for dashboard\n",
    "Aim: getting topics that have no wikipedia url in their properties."
   ]
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 14 topics without a URL (seeAlso property).\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "\n",
    "SELECT (count(?term) as ?nb_no_wikipedia) WHERE {\n",
    "    ?c rdfs:subClassOf+ edam:topic_0003 ;\n",
    "                rdfs:label ?term .\n",
    "        FILTER NOT EXISTS {\n",
    "        ?c rdfs:seeAlso ?seealso .\n",
    "        FILTER (regex(str(?seealso), \"wikipedia.org\", \"i\"))\n",
    "       } .\n",
    "}\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"There are {r['nb_no_wikipedia']} topics without a URL (seeAlso property).\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-29T13:59:26.200878Z",
     "start_time": "2024-02-29T13:59:26.082715Z"
    }
   },
   "execution_count": 8
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Topic 'Literature and language' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Data submission, annotation, and curation' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Data identity and mapping' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Genome resequencing' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Simulation experiment' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Nucleic acid sites, features and motifs' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Protein properties' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Protein sites, features and motifs' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Sequence composition, complexity and repeats' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Probes and primers' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Sequence sites, features and motifs' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Biomolecular simulation' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Biotherapeutics' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Quality affairs' has no wikipedia link ('seeAlso' property).\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "\n",
    "\n",
    "SELECT ?term WHERE {\n",
    "    ?c rdfs:subClassOf+ edam:topic_0003 ;\n",
    "                rdfs:label ?term .\n",
    "        FILTER NOT EXISTS {\n",
    "        ?c rdfs:seeAlso ?seealso .\n",
    "        FILTER (regex(str(?seealso), \"wikipedia.org\", \"i\"))\n",
    "       } .\n",
    "}\n",
    "\"\"\"\n",
    "results = kg.query(query)\n",
    "\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Topic '{r['term']}' has no wikipedia link ('seeAlso' property).\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-29T14:07:01.337516Z",
     "start_time": "2024-02-29T14:07:01.331591Z"
    }
   },
   "execution_count": 10
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "14\n",
      "Topic 'Literature and language' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Data submission, annotation, and curation' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Data identity and mapping' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Genome resequencing' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Simulation experiment' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Nucleic acid sites, features and motifs' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Protein properties' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Protein sites, features and motifs' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Sequence composition, complexity and repeats' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Probes and primers' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Sequence sites, features and motifs' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Biomolecular simulation' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Biotherapeutics' has no wikipedia link ('seeAlso' property).\n",
      "Topic 'Quality affairs' has no wikipedia link ('seeAlso' property).\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "\n",
    "query = os.path.dirname(\".\")  + \"../queries/no_wikipedia_link_topic.rq\"\n",
    "\n",
    "with open(query, \"r\") as f:\n",
    "    query = f.read()\n",
    "    \n",
    "    results = kg.query(query)\n",
    "    nb_err = len(results)\n",
    "f.close()\n",
    "\n",
    "print(nb_err)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Topic '{r['term']}' has no wikipedia link ('seeAlso' property).\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-29T14:18:57.077411Z",
     "start_time": "2024-02-29T14:18:57.012981Z"
    }
   },
   "execution_count": 15
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query = os.path.dirname(\".\")  + \"../queries/no_wikipedia_link_topic.rq\"\n",
    "\n",
    "with open(query, \"r\") as f:\n",
    "    query = f.read()\n",
    "    \n",
    "    results = kg.query(query)\n",
    "    nb_err = len(results)\n",
    "f.close()\n",
    "\n",
    "print(nb_err)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Topic '{r['term']}' has no wikipedia link ('seeAlso' property).\") "
   ],
   "metadata": {
    "collapsed": false
   }
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Topic 'Informatics' has no broad synonym\n",
      "Topic 'Ontology and terminology' has no broad synonym\n",
      "Topic 'Bioinformatics' has no broad synonym\n",
      "Topic 'Laboratory information management' has no broad synonym\n",
      "Topic 'Cheminformatics' has no broad synonym\n",
      "Topic 'Chemometrics' has no broad synonym\n",
      "Topic 'Medical informatics' has no broad synonym\n",
      "Topic 'Immunoinformatics' has no broad synonym\n",
      "Topic 'Literature and language' has no broad synonym\n",
      "Topic 'Natural language processing' has no broad synonym\n",
      "Topic 'Data management' has no broad synonym\n",
      "Topic 'Data submission, annotation, and curation' has no broad synonym\n",
      "Topic 'Workflows' has no broad synonym\n",
      "Topic 'Data acquisition' has no broad synonym\n",
      "Topic 'Data security' has no broad synonym\n",
      "Topic 'Data identity and mapping' has no broad synonym\n",
      "Topic 'Data architecture, analysis and design' has no broad synonym\n",
      "Topic 'Data integration and warehousing' has no broad synonym\n",
      "Topic 'Data governance' has no broad synonym\n",
      "Topic 'Data quality management' has no broad synonym\n",
      "Topic 'Data rescue' has no broad synonym\n",
      "Topic 'FAIR data' has no broad synonym\n",
      "Topic 'Biochemistry' has no broad synonym\n",
      "Topic 'Microfluidics' has no broad synonym\n",
      "Topic 'Computational chemistry' has no broad synonym\n",
      "Topic 'Drug discovery' has no broad synonym\n",
      "Topic 'Medicinal chemistry' has no broad synonym\n",
      "Topic 'Compound libraries and screening' has no broad synonym\n",
      "Topic 'Analytical chemistry' has no broad synonym\n",
      "Topic 'Synthetic chemistry' has no broad synonym\n",
      "Topic 'Chemical biology' has no broad synonym\n",
      "Topic 'Mathematics' has no broad synonym\n",
      "Topic 'Statistics and probability' has no broad synonym\n",
      "Topic 'Applied mathematics' has no broad synonym\n",
      "Topic 'Pure mathematics' has no broad synonym\n",
      "Topic 'Computer science' has no broad synonym\n",
      "Topic 'Software engineering' has no broad synonym\n",
      "Topic 'Physics' has no broad synonym\n",
      "Topic 'Biophysics' has no broad synonym\n",
      "Topic 'Acoustics' has no broad synonym\n",
      "Topic 'Laboratory techniques' has no broad synonym\n",
      "Topic 'Sequencing' has no broad synonym\n",
      "Topic 'ChIP-seq' has no broad synonym\n",
      "Topic 'RNA-Seq' has no broad synonym\n",
      "Topic 'Whole genome sequencing' has no broad synonym\n",
      "Topic 'Exome sequencing' has no broad synonym\n",
      "Topic 'Metagenomic sequencing' has no broad synonym\n",
      "Topic 'Single-Cell Sequencing' has no broad synonym\n",
      "Topic 'Imaging' has no broad synonym\n",
      "Topic 'Cryogenic electron microscopy' has no broad synonym\n",
      "Topic 'X-ray diffraction' has no broad synonym\n",
      "Topic 'Bioimaging' has no broad synonym\n",
      "Topic 'Medical imaging' has no broad synonym\n",
      "Topic 'Light microscopy' has no broad synonym\n",
      "Topic 'MRI' has no broad synonym\n",
      "Topic 'Neutron diffraction' has no broad synonym\n",
      "Topic 'Tomography' has no broad synonym\n",
      "Topic 'Echography' has no broad synonym\n",
      "Topic 'Electroencephalography' has no broad synonym\n",
      "Topic 'Electrocardiography' has no broad synonym\n",
      "Topic 'Genotyping experiment' has no broad synonym\n",
      "Topic 'Microarray experiment' has no broad synonym\n",
      "Topic 'PCR experiment' has no broad synonym\n",
      "Topic 'Proteomics experiment' has no broad synonym\n",
      "Topic 'RNAi experiment' has no broad synonym\n",
      "Topic 'Simulation experiment' has no broad synonym\n",
      "Topic 'Immunoprecipitation experiment' has no broad synonym\n",
      "Topic 'ChIP-on-chip' has no broad synonym\n",
      "Topic 'Methylated DNA immunoprecipitation' has no broad synonym\n",
      "Topic 'RNA immunoprecipitation' has no broad synonym\n",
      "Topic 'Cytometry' has no broad synonym\n",
      "Topic 'Chromosome conformation capture' has no broad synonym\n",
      "Topic 'Protein interaction experiment' has no broad synonym\n",
      "Topic 'Experimental design and studies' has no broad synonym\n",
      "Topic 'Preclinical and clinical studies' has no broad synonym\n",
      "Topic 'GWAS study' has no broad synonym\n",
      "Topic 'Animal study' has no broad synonym\n",
      "Topic 'Environmental sciences' has no broad synonym\n",
      "Topic 'Ecology' has no broad synonym\n",
      "Topic 'Biodiversity' has no broad synonym\n",
      "Topic 'Microbial ecology' has no broad synonym\n",
      "Topic 'Metabarcoding' has no broad synonym\n",
      "Topic 'Open science' has no broad synonym\n",
      "Topic 'Biosciences' has no broad synonym\n",
      "Topic 'Biology' has no broad synonym\n",
      "Topic 'Model organisms' has no broad synonym\n",
      "Topic 'Plant biology' has no broad synonym\n",
      "Topic 'Virology' has no broad synonym\n",
      "Topic 'Structural biology' has no broad synonym\n",
      "Topic 'Structural genomics' has no broad synonym\n",
      "Topic 'Cell biology' has no broad synonym\n",
      "Topic 'Systems biology' has no broad synonym\n",
      "Topic 'Human biology' has no broad synonym\n",
      "Topic 'Molecular biology' has no broad synonym\n",
      "Topic 'Genetics' has no broad synonym\n",
      "Topic 'Genotype and phenotype' has no broad synonym\n",
      "Topic 'Quantitative genetics' has no broad synonym\n",
      "Topic 'Phenomics' has no broad synonym\n",
      "Topic 'Molecular evolution' has no broad synonym\n",
      "Topic 'Epistasis' has no broad synonym\n",
      "Topic 'Population genetics' has no broad synonym\n",
      "Topic 'Epigenetics' has no broad synonym\n",
      "Topic 'Epigenomics' has no broad synonym\n",
      "Topic 'Genomic imprinting' has no broad synonym\n",
      "Topic 'Molecular genetics' has no broad synonym\n",
      "Topic 'Gene structure' has no broad synonym\n",
      "Topic 'Functional, regulatory and non-coding RNA' has no broad synonym\n",
      "Topic 'Mobile genetic elements' has no broad synonym\n",
      "Topic 'Gene transcripts' has no broad synonym\n",
      "Topic 'Genetic variation' has no broad synonym\n",
      "Topic 'DNA mutation' has no broad synonym\n",
      "Topic 'DNA polymorphism' has no broad synonym\n",
      "Topic 'Structural variation' has no broad synonym\n",
      "Topic 'Copy number variation' has no broad synonym\n",
      "Topic 'Gene expression' has no broad synonym\n",
      "Topic 'Gene regulation' has no broad synonym\n",
      "Topic 'Transcription factors and regulatory sites' has no broad synonym\n",
      "Topic 'Transcriptomics' has no broad synonym\n",
      "Topic 'Metatranscriptomics' has no broad synonym\n",
      "Topic 'Ribosome Profiling' has no broad synonym\n",
      "Topic 'RNA splicing' has no broad synonym\n",
      "Topic 'Gene and protein families' has no broad synonym\n",
      "Topic 'Immunoproteins and antigens' has no broad synonym\n",
      "Topic 'Cytogenetics' has no broad synonym\n",
      "Topic 'Human genetics' has no broad synonym\n",
      "Topic 'Genetic engineering' has no broad synonym\n",
      "Topic 'Immunogenetics' has no broad synonym\n",
      "Topic 'Developmental biology' has no broad synonym\n",
      "Topic 'Embryology' has no broad synonym\n",
      "Topic 'Biotechnology' has no broad synonym\n",
      "Topic 'Biomaterials' has no broad synonym\n",
      "Topic 'Bioengineering' has no broad synonym\n",
      "Topic 'Medical biotechnology' has no broad synonym\n",
      "Topic 'Synthetic biology' has no broad synonym\n",
      "Topic 'Metabolic engineering' has no broad synonym\n",
      "Topic 'Evolutionary biology' has no broad synonym\n",
      "Topic 'Phylogeny' has no broad synonym\n",
      "Topic 'Phylogenetics' has no broad synonym\n",
      "Topic 'Cladistics' has no broad synonym\n",
      "Topic 'Taxonomy' has no broad synonym\n",
      "Topic 'Microbiology' has no broad synonym\n",
      "Topic 'Antimicrobial Resistance' has no broad synonym\n",
      "Topic 'Biomarkers' has no broad synonym\n",
      "Topic 'Marine biology' has no broad synonym\n",
      "Topic 'Zoology' has no broad synonym\n",
      "Topic 'Medicine' has no broad synonym\n",
      "Topic 'Pathology' has no broad synonym\n",
      "Topic 'Infectious disease' has no broad synonym\n",
      "Topic 'Rare diseases' has no broad synonym\n",
      "Topic 'Oncology' has no broad synonym\n",
      "Topic 'Toxicology' has no broad synonym\n",
      "Topic 'Physiology' has no broad synonym\n",
      "Topic 'Public health and epidemiology' has no broad synonym\n",
      "Topic 'Respiratory medicine' has no broad synonym\n",
      "Topic 'Neurology' has no broad synonym\n",
      "Topic 'Cardiology' has no broad synonym\n",
      "Topic 'Translational medicine' has no broad synonym\n",
      "Topic 'Molecular medicine' has no broad synonym\n",
      "Topic 'Systems medicine' has no broad synonym\n",
      "Topic 'Veterinary medicine' has no broad synonym\n",
      "Topic 'Allergy, clinical immunology and immunotherapeutics' has no broad synonym\n",
      "Topic 'Pain medicine' has no broad synonym\n",
      "Topic 'Anaesthesiology' has no broad synonym\n",
      "Topic 'Critical care medicine' has no broad synonym\n",
      "Topic 'Dermatology' has no broad synonym\n",
      "Topic 'Dentistry' has no broad synonym\n",
      "Topic 'Ear, nose and throat medicine' has no broad synonym\n",
      "Topic 'Endocrinology and metabolism' has no broad synonym\n",
      "Topic 'Haematology' has no broad synonym\n",
      "Topic 'Gastroenterology' has no broad synonym\n",
      "Topic 'Gender medicine' has no broad synonym\n",
      "Topic 'Gynaecology and obstetrics' has no broad synonym\n",
      "Topic 'Hepatic and biliary medicine' has no broad synonym\n",
      "Topic 'Trauma medicine' has no broad synonym\n",
      "Topic 'Medical toxicology' has no broad synonym\n",
      "Topic 'Musculoskeletal medicine' has no broad synonym\n",
      "Topic 'Paediatrics' has no broad synonym\n",
      "Topic 'Reproductive health' has no broad synonym\n",
      "Topic 'Surgery' has no broad synonym\n",
      "Topic 'Urology and nephrology' has no broad synonym\n",
      "Topic 'Tropical medicine' has no broad synonym\n",
      "Topic 'Personalised medicine' has no broad synonym\n",
      "Topic 'Computational biology' has no broad synonym\n",
      "Topic 'Nucleic acids' has no broad synonym\n",
      "Topic 'Nucleic acid structure analysis' has no broad synonym\n",
      "Topic 'RNA' has no broad synonym\n",
      "Topic 'DNA' has no broad synonym\n",
      "Topic 'DNA binding sites' has no broad synonym\n",
      "Topic 'DNA replication and recombination' has no broad synonym\n",
      "Topic 'DNA packaging' has no broad synonym\n",
      "Topic 'Nucleic acid sites, features and motifs' has no broad synonym\n",
      "Topic 'Proteins' has no broad synonym\n",
      "Topic 'Protein expression' has no broad synonym\n",
      "Topic 'Protein targeting and localisation' has no broad synonym\n",
      "Topic 'Protein modifications' has no broad synonym\n",
      "Topic 'Protein variants' has no broad synonym\n",
      "Topic 'Protein properties' has no broad synonym\n",
      "Topic 'Protein interactions' has no broad synonym\n",
      "Topic 'Membrane and lipoproteins' has no broad synonym\n",
      "Topic 'Enzymes' has no broad synonym\n",
      "Topic 'Protein structure analysis' has no broad synonym\n",
      "Topic 'Protein folding, stability and design' has no broad synonym\n",
      "Topic 'Protein structural motifs and surfaces' has no broad synonym\n",
      "Topic 'Protein folds and structural domains' has no broad synonym\n",
      "Topic 'Protein disordered structure' has no broad synonym\n",
      "Topic 'Protein secondary structure' has no broad synonym\n",
      "Topic 'Protein sites, features and motifs' has no broad synonym\n",
      "Topic 'Protein binding sites' has no broad synonym\n",
      "Topic 'Sequence analysis' has no broad synonym\n",
      "Topic 'Mapping' has no broad synonym\n",
      "Topic 'Sequence composition, complexity and repeats' has no broad synonym\n",
      "Topic 'Phylogenomics' has no broad synonym\n",
      "Topic 'Sequence assembly' has no broad synonym\n",
      "Topic 'Probes and primers' has no broad synonym\n",
      "Topic 'Structure analysis' has no broad synonym\n",
      "Topic 'Structure prediction' has no broad synonym\n",
      "Topic 'Molecular modelling' has no broad synonym\n",
      "Topic 'Carbohydrates' has no broad synonym\n",
      "Topic 'Lipids' has no broad synonym\n",
      "Topic 'Small molecules' has no broad synonym\n",
      "Topic 'Sequence sites, features and motifs' has no broad synonym\n",
      "Topic 'Molecular interactions, pathways and networks' has no broad synonym\n",
      "Topic 'Function analysis' has no broad synonym\n",
      "Topic 'Functional genomics' has no broad synonym\n",
      "Topic 'Biomolecular simulation' has no broad synonym\n",
      "Topic 'Biomedical science' has no broad synonym\n",
      "Topic 'Pharmacology' has no broad synonym\n",
      "Topic 'Pharmacogenomics' has no broad synonym\n",
      "Topic 'Immunology' has no broad synonym\n",
      "Topic 'Anatomy' has no broad synonym\n",
      "Topic 'Sample collections' has no broad synonym\n",
      "Topic 'Biobank' has no broad synonym\n",
      "Topic 'Mouse clinic' has no broad synonym\n",
      "Topic 'Microbial collection' has no broad synonym\n",
      "Topic 'Cell culture collection' has no broad synonym\n",
      "Topic 'Clone library' has no broad synonym\n",
      "Topic 'Parasitology' has no broad synonym\n",
      "Topic 'Drug development' has no broad synonym\n",
      "Topic 'Drug metabolism' has no broad synonym\n",
      "Topic 'Safety sciences' has no broad synonym\n",
      "Topic 'Pharmacovigilance' has no broad synonym\n",
      "Topic 'Quality affairs' has no broad synonym\n",
      "Topic 'Regulatory affairs' has no broad synonym\n",
      "Topic 'Vaccinology' has no broad synonym\n",
      "Topic 'Laboratory animal science' has no broad synonym\n",
      "Topic 'Nutritional science' has no broad synonym\n",
      "Topic 'Regenerative medicine' has no broad synonym\n",
      "Topic 'Omics' has no broad synonym\n",
      "Topic 'Proteomics' has no broad synonym\n",
      "Topic 'Genomics' has no broad synonym\n",
      "Topic 'Comparative genomics' has no broad synonym\n",
      "Topic 'Population genomics' has no broad synonym\n",
      "Topic 'Proteogenomics' has no broad synonym\n",
      "Topic 'Paleogenomics' has no broad synonym\n",
      "Topic 'Metabolomics' has no broad synonym\n",
      "Topic 'Fluxomics' has no broad synonym\n",
      "Topic 'Immunomics' has no broad synonym\n",
      "Topic 'Multiomics' has no broad synonym\n"
     ]
    }
   ],
   "source": [
    "## Query for synonyms\n",
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "\n",
    "\n",
    "SELECT ?term ?c WHERE {\n",
    "    ?c rdfs:subClassOf+ edam:topic_0003 ;\n",
    "                rdfs:label ?term .\n",
    "        FILTER NOT EXISTS {\n",
    "        ?c oboInOwl:hasBroadSynonym  ?hasBroadSynonym .\n",
    "       } .\n",
    "}\n",
    "\"\"\"\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Topic '{r['term']}' has no broad synonym\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-29T15:17:02.172242Z",
     "start_time": "2024-02-29T15:17:02.118429Z"
    }
   },
   "execution_count": 18
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "## Query for topics with no definition (ERROR level)\n",
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "\n",
    "SELECT ?term ?concept WHERE {\n",
    "    ?concept rdfs:subClassOf+ edam:topic_0003 ;\n",
    "                rdfs:label ?term .\n",
    "                \n",
    "    FILTER NOT EXISTS {\n",
    "        ?concept oboInOwl:hasDefinition ?def \n",
    "    } .\n",
    "}\n",
    "\"\"\"\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Topic '{r['term']}' and '{r['concept']}' has no def \") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-29T15:38:16.752372Z",
     "start_time": "2024-02-29T15:38:16.700550Z"
    }
   },
   "execution_count": 38
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Data term  'Textual format' has  def 'Textual format.' \n",
      "Data term  'SMILES' has  def 'Chemical structure specified in Simplified Molecular Input Line Entry System (SMILES) line notation.' \n",
      "Data term  'smarts' has  def 'SMILES ARbitrary Target Specification (SMARTS) format for chemical structure specification, which is a subset of the SMILES line notation.' \n",
      "Data term  'InChI' has  def 'Chemical structure specified in IUPAC International Chemical Identifier (InChI) line notation.' \n",
      "Data term  'mf' has  def 'Chemical structure specified by Molecular Formula (MF), including a count of each element in a compound.' \n",
      "Data term  'InChIKey' has  def 'The InChIKey (hashed InChI) is a fixed length (25 character) condensed digital representation of an InChI chemical structure specification. It uniquely identifies a chemical compound.' \n",
      "Data term  'nucleotide' has  def 'Alphabet for a nucleotide sequence with possible ambiguity, unknown positions and non-sequence characters.' \n",
      "Data term  'pure nucleotide' has  def 'Alphabet for a nucleotide sequence with possible ambiguity and unknown positions but without non-sequence characters.' \n",
      "Data term  'pure dna' has  def 'Alphabet for a DNA sequence with possible ambiguity and unknown positions but without non-sequence characters.' \n",
      "Data term  'pure rna' has  def 'Alphabet for an RNA sequence with possible ambiguity and unknown positions but without non-sequence characters.' \n",
      "Data term  'unambiguous pure nucleotide' has  def 'Alphabet for a nucleotide sequence (characters ACGTU only) with possible unknown positions but without ambiguity or non-sequence characters .' \n",
      "Data term  'dna' has  def 'Alphabet for a DNA sequence with possible ambiguity, unknown positions and non-sequence characters.' \n",
      "Data term  'unambiguous pure dna' has  def 'Alphabet for a DNA sequence (characters ACGT only) with possible unknown positions but without ambiguity or non-sequence characters.' \n",
      "Data term  'completely unambiguous pure dna' has  def 'Alphabet for a DNA sequence (characters ACGT only) without unknown positions, ambiguity or non-sequence characters.' \n",
      "Data term  'rna' has  def 'Alphabet for an RNA sequence with possible ambiguity, unknown positions and non-sequence characters.' \n",
      "Data term  'unambiguous pure rna sequence' has  def 'Alphabet for an RNA sequence (characters ACGU only) with possible unknown positions but without ambiguity or non-sequence characters.' \n",
      "Data term  'completely unambiguous pure rna sequence' has  def 'Alphabet for an RNA sequence (characters ACGU only) without unknown positions, ambiguity or non-sequence characters.' \n",
      "Data term  'completely unambiguous pure nucleotide' has  def 'Alphabet for a nucleotide sequence (characters ACGTU only) without unknown positions, ambiguity or non-sequence characters .' \n",
      "Data term  'protein' has  def 'Alphabet for a protein sequence with possible ambiguity, unknown positions and non-sequence characters.' \n",
      "Data term  'unambiguous pure protein' has  def 'Alphabet for any protein sequence with possible unknown positions but without ambiguity or non-sequence characters.' \n",
      "Data term  'pure protein' has  def 'Alphabet for any protein sequence with possible ambiguity and unknown positions but without non-sequence characters.' \n",
      "Data term  'completely unambiguous pure protein' has  def 'Alphabet for any protein sequence without unknown positions, ambiguity or non-sequence characters.' \n",
      "Data term  'EMBL feature location' has  def 'Format for sequence positions (feature location) as used in DDBJ/EMBL/GenBank database.' \n",
      "Data term  'quicktandem' has  def 'Report format for tandem repeats in a nucleotide sequence (format generated by the Sanger Centre quicktandem program).' \n",
      "Data term  'Sanger inverted repeats' has  def 'Report format for inverted repeats in a nucleotide sequence (format generated by the Sanger Centre inverted program).' \n",
      "Data term  'EMBOSS repeat' has  def 'Report format for tandem repeats in a sequence (an EMBOSS report format).' \n",
      "Data term  'est2genome format' has  def 'Format of a report on exon-intron structure generated by EMBOSS est2genome.' \n",
      "Data term  'restrict format' has  def 'Report format for restriction enzyme recognition sites used by EMBOSS restrict program.' \n",
      "Data term  'restover format' has  def 'Report format for restriction enzyme recognition sites used by EMBOSS restover program.' \n",
      "Data term  'REBASE restriction sites' has  def 'Report format for restriction enzyme recognition sites used by REBASE database.' \n",
      "Data term  'FASTA search results format' has  def 'Format of results of a sequence database search using FASTA.' \n",
      "Data term  'BLAST results' has  def 'Format of results of a sequence database search using some variant of BLAST.' \n",
      "Data term  'BLAST XML results format' has  def 'XML format as produced by the NCBI Blast package.' \n",
      "Data term  'BLAST XML v2 results format' has  def 'XML format as produced by the NCBI Blast package v2.' \n",
      "Data term  'mspcrunch' has  def 'Format of results of a sequence database search using some variant of MSPCrunch.' \n",
      "Data term  'Smith-Waterman format' has  def 'Format of results of a sequence database search using some variant of Smith Waterman.' \n",
      "Data term  'dhf' has  def 'Format of EMBASSY domain hits file (DHF) of hits (sequences) with domain classification information.' \n",
      "Data term  'lhf' has  def 'Format of EMBASSY ligand hits file (LHF) of database hits (sequences) with ligand classification information.' \n",
      "Data term  'InterPro hits format' has  def 'Results format for searches of the InterPro database.' \n",
      "Data term  'InterPro protein view report format' has  def 'Format of results of a search of the InterPro database showing matches of query protein sequence(s) to InterPro entries.' \n",
      "Data term  'InterPro match table format' has  def 'Format of results of a search of the InterPro database showing matches between protein sequence(s) and signatures for an InterPro entry.' \n",
      "Data term  'HMMER Dirichlet prior' has  def 'Dirichlet distribution HMMER format.' \n",
      "Data term  'MEME Dirichlet prior' has  def 'Dirichlet distribution MEME format.' \n",
      "Data term  'HMMER emission and transition' has  def 'Format of a report from the HMMER package on the emission and transition counts of a hidden Markov model.' \n",
      "Data term  'prosite-pattern' has  def 'Format of a regular expression pattern from the Prosite database.' \n",
      "Data term  'EMBOSS sequence pattern' has  def 'Format of an EMBOSS sequence pattern.' \n",
      "Data term  'meme-motif' has  def 'A motif in the format generated by the MEME program.' \n",
      "Data term  'prosite-profile' has  def 'Sequence profile (sequence classifier) format used in the PROSITE database.' \n",
      "Data term  'JASPAR format' has  def 'A profile (sequence classifier) in the format used in the JASPAR database.' \n",
      "Data term  'MEME background Markov model' has  def 'Format of the model of random sequences used by MEME.' \n",
      "Data term  'HMMER format' has  def 'Format of a hidden Markov model representation used by the HMMER package.' \n",
      "Data term  'HMMER2' has  def 'HMMER profile HMM file for HMMER versions 2.x.' \n",
      "Data term  'HMMER3' has  def 'HMMER profile HMM file for HMMER versions 3.x.' \n",
      "Data term  'HMMER-aln' has  def 'FASTA-style format for multiple sequences aligned by HMMER package to an HMM.' \n",
      "Data term  'DIALIGN format' has  def 'Format of multiple sequences aligned by DIALIGN package.' \n",
      "Data term  'daf' has  def 'EMBASSY 'domain alignment file' (DAF) format, containing a sequence alignment of protein domains belonging to the same SCOP or CATH family.' \n",
      "Data term  'Sequence-MEME profile alignment' has  def 'Format for alignment of molecular sequences to MEME profiles (position-dependent scoring matrices) as generated by the MAST tool from the MEME package.' \n",
      "Data term  'HMMER profile alignment (sequences versus HMMs)' has  def 'Format used by the HMMER package for an alignment of a sequence against a hidden Markov model database.' \n",
      "Data term  'HMMER profile alignment (HMM versus sequences)' has  def 'Format used by the HMMER package for of an alignment of a hidden Markov model against a sequence database.' \n",
      "Data term  'Phylip distance matrix' has  def 'Format of PHYLIP phylogenetic distance matrix data.' \n",
      "Data term  'ClustalW dendrogram' has  def 'Dendrogram (tree file) format generated by ClustalW.' \n",
      "Data term  'Phylip tree raw' has  def 'Raw data file format used by Phylip from which a phylogenetic tree is directly generated or plotted.' \n",
      "Data term  'Phylip continuous quantitative characters' has  def 'PHYLIP file format for continuous quantitative character data.' \n",
      "Data term  'Phylip character frequencies format' has  def 'PHYLIP file format for phylogenetics character frequency data.' \n",
      "Data term  'Phylip discrete states format' has  def 'Format of PHYLIP discrete states data.' \n",
      "Data term  'Phylip cliques format' has  def 'Format of PHYLIP cliques data.' \n",
      "Data term  'Phylip tree format' has  def 'Phylogenetic tree data format used by the PHYLIP program.' \n",
      "Data term  'TreeBASE format' has  def 'The format of an entry from the TreeBASE database of phylogenetic data.' \n",
      "Data term  'TreeFam format' has  def 'The format of an entry from the TreeFam database of phylogenetic data.' \n",
      "Data term  'Phylip tree distance format' has  def 'Format for distances, such as Branch Score distance, between two or more phylogenetic trees as used by the Phylip package.' \n",
      "Data term  'dssp' has  def 'Format of an entry from the DSSP database (Dictionary of Secondary Structure in Proteins).' \n",
      "Data term  'hssp' has  def 'Entry format of the HSSP database (Homology-derived Secondary Structure in Proteins).' \n",
      "Data term  'Dot-bracket format' has  def 'Format of RNA secondary structure in dot-bracket notation, originally generated by the Vienna RNA package/server.' \n",
      "Data term  'Vienna local RNA secondary structure format' has  def 'Format of local RNA secondary structure components with free energy values, generated by the Vienna RNA package/server.' \n",
      "Data term  'PDB' has  def 'Entry format of PDB database in PDB format.' \n",
      "Data term  'mmCIF' has  def 'Entry format of PDB database in mmCIF format.' \n",
      "Data term  'aaindex' has  def 'Amino acid index format used by the AAindex database.' \n",
      "Data term  'Pcons report format' has  def 'Format of output of the Pcons Model Quality Assessment Program (MQAP).' \n",
      "Data term  'ProQ report format' has  def 'Format of output of the ProQ protein model quality predictor.' \n",
      "Data term  'findkm' has  def 'A report format for the kinetics of enzyme-catalysed reaction(s) in a format generated by EMBOSS findkm. This includes Michaelis Menten plot, Hanes Woolf plot, Michaelis Menten constant (Km) and maximum velocity (Vmax).' \n",
      "Data term  'Primer3 primer' has  def 'Report format on PCR primers and hybridisation oligos as generated by Whitehead primer3 program.' \n",
      "Data term  'mira' has  def 'Format of MIRA sequence trace information file.' \n",
      "Data term  'CAF' has  def 'Common Assembly Format (CAF). A sequence assembly format including contigs, base-call qualities, and other metadata.' \n",
      "Data term  'EXP' has  def 'Sequence assembly project file EXP format.' \n",
      "Data term  'PHD' has  def 'PHD sequence trace format to store serialised chromatogram data (reads).' \n",
      "Data term  'dat' has  def 'Format of Affymetrix data file of raw image data.' \n",
      "Data term  'cel' has  def 'Format of Affymetrix data file of information about (raw) expression levels of the individual probes.' \n",
      "Data term  'affymetrix' has  def 'Format of affymetrix gene cluster files (hc-genes.txt, hc-chips.txt) from hierarchical clustering.' \n",
      "Data term  'affymetrix-exp' has  def 'Affymetrix data file format for information about experimental conditions and protocols.' \n",
      "Data term  'CHP' has  def 'Format of Affymetrix data file of information about (normalised) expression levels of the individual probes.' \n",
      "Data term  'HET group dictionary entry format' has  def 'The format of an entry from the HET group dictionary (HET groups from PDB files).' \n",
      "Data term  'PubMed citation' has  def 'Format of bibliographic reference as used by the PubMed database.' \n",
      "Data term  'Medline Display Format' has  def 'Format for abstracts of scientific articles from the Medline database.' \n",
      "Data term  'CiteXplore-core' has  def 'CiteXplore 'core' citation format including title, journal, authors and abstract.' \n",
      "Data term  'CiteXplore-all' has  def 'CiteXplore 'all' citation format includes all known details such as Mesh terms and cross-references.' \n",
      "Data term  'pmc' has  def 'Article format of the PubMed Central database.' \n",
      "Data term  'OSCAR format' has  def 'OSCAR format of annotated chemical text.' \n",
      "Data term  'PlasMapper TextMap' has  def 'Map of a plasmid (circular DNA) in PlasMapper TextMap format.' \n",
      "Data term  'newick' has  def 'Phylogenetic tree Newick (text) format.' \n",
      "Data term  'TreeCon format' has  def 'Phylogenetic tree TreeCon (text) format.' \n",
      "Data term  'Nexus format' has  def 'Phylogenetic tree Nexus (text) format.' \n",
      "Data term  'acedb' has  def 'ACEDB sequence format.' \n",
      "Data term  'codata' has  def 'Codata entry format.' \n",
      "Data term  'Staden experiment format' has  def 'Staden experiment file format.' \n",
      "Data term  'fitch program' has  def 'Fitch program format.' \n",
      "Data term  'GCG' has  def 'GCG sequence file format.' \n",
      "Data term  'hennig86' has  def 'Hennig86 output sequence format.' \n",
      "Data term  'ig' has  def 'Intelligenetics sequence format.' \n",
      "Data term  'igstrict' has  def 'Intelligenetics sequence format (strict version).' \n",
      "Data term  'jackknifer' has  def 'Jackknifer interleaved and non-interleaved sequence format.' \n",
      "Data term  'mase format' has  def 'Mase program sequence format.' \n",
      "Data term  'mega-seq' has  def 'Mega interleaved and non-interleaved sequence format.' \n",
      "Data term  'nbrf/pir' has  def 'NBRF/PIR entry sequence format.' \n",
      "Data term  'nexus-seq' has  def 'Nexus/paup interleaved sequence format.' \n",
      "Data term  'pdbatom' has  def 'PDB sequence format (ATOM lines).' \n",
      "Data term  'pdbatomnuc' has  def 'PDB nucleotide sequence format (ATOM lines).' \n",
      "Data term  'pdbseqresnuc' has  def 'PDB nucleotide sequence format (SEQRES lines).' \n",
      "Data term  'pdbseqres' has  def 'PDB sequence format (SEQRES lines).' \n",
      "Data term  'raw' has  def 'Raw sequence format with no non-sequence characters.' \n",
      "Data term  'refseqp' has  def 'Refseq protein entry sequence format.' \n",
      "Data term  'Staden format' has  def 'Staden suite sequence format.' \n",
      "Data term  'Stockholm format' has  def 'Stockholm multiple sequence alignment format (used by Pfam and Rfam).' \n",
      "Data term  'strider format' has  def 'DNA strider output sequence format.' \n",
      "Data term  'plain text format (unformatted)' has  def 'Plain text sequence format (essentially unformatted).' \n",
      "Data term  'ASN.1 sequence format' has  def 'NCBI ASN.1-based sequence format.' \n",
      "Data term  'debug-seq' has  def 'EMBOSS debugging trace sequence format of full internal data content.' \n",
      "Data term  'jackknifernon' has  def 'Jackknifer output sequence non-interleaved format.' \n",
      "Data term  'nexusnon' has  def 'Nexus/paup non-interleaved sequence format.' \n",
      "Data term  'debug-feat' has  def 'EMBOSS debugging trace feature format of full internal data content.' \n",
      "Data term  'ClustalW format' has  def 'ClustalW format for (aligned) sequences.' \n",
      "Data term  'debug' has  def 'EMBOSS alignment format for debugging trace of full internal data content.' \n",
      "Data term  'match' has  def 'Alignment format for start and end of matches between sequence pairs.' \n",
      "Data term  'scores format' has  def 'Alignment format for score values for pairs of sequences.' \n",
      "Data term  'selex' has  def 'SELEX format for (aligned) sequences.' \n",
      "Data term  'EMBOSS simple format' has  def 'EMBOSS simple multiple alignment format.' \n",
      "Data term  'srs format' has  def 'Simple multiple sequence (alignment) format for SRS.' \n",
      "Data term  'srspair' has  def 'Simple sequence pair (alignment) format for SRS.' \n",
      "Data term  'T-Coffee format' has  def 'T-Coffee program alignment format.' \n",
      "Data term  'TreeCon-seq' has  def 'Treecon format for (aligned) sequences.' \n",
      "Data term  'pure' has  def 'Alphabet for molecular sequence with possible unknown positions but without non-sequence characters.' \n",
      "Data term  'unambiguous pure' has  def 'Alphabet for a molecular sequence with possible unknown positions but without ambiguity or non-sequence characters.' \n",
      "Data term  'completely unambiguous pure' has  def 'Alphabet for a molecular sequence without unknown positions, ambiguity or non-sequence characters.' \n",
      "Data term  'unpure' has  def 'Alphabet for a molecular sequence with possible unknown positions but possibly with non-sequence characters.' \n",
      "Data term  'consensus' has  def 'Alphabet for the consensus of two or more molecular sequences.' \n",
      "Data term  'unambiguous sequence' has  def 'Alphabet for a molecular sequence with possible unknown positions but without ambiguity characters.' \n",
      "Data term  'ambiguous' has  def 'Alphabet for a molecular sequence with possible unknown positions and possible ambiguity characters.' \n",
      "Data term  'EMBL-like (text)' has  def 'A text format resembling EMBL entry format.' \n",
      "Data term  'EMBL format' has  def 'EMBL entry format.' \n",
      "Data term  'geneseq' has  def 'Geneseq sequence format.' \n",
      "Data term  'FASTQ-like format (text)' has  def 'A text format resembling FASTQ short read format.' \n",
      "Data term  'FASTQ' has  def 'FASTQ short read format ignoring quality scores.' \n",
      "Data term  'FASTQ-illumina' has  def 'FASTQ Illumina 1.3 short read format.' \n",
      "Data term  'qualillumina' has  def 'FASTQ format subset for Phred sequencing quality score data only (no sequences) from Illumina 1.5 and before Illumina 1.8.' \n",
      "Data term  'FASTQ-sanger' has  def 'FASTQ short read format with phred quality.' \n",
      "Data term  'FASTQ-solexa' has  def 'FASTQ Solexa/Illumina 1.0 short read format.' \n",
      "Data term  'qualsolexa' has  def 'FASTQ format subset for Phred sequencing quality score data only (no sequences) for Solexa/Illumina 1.0 format.' \n",
      "Data term  'qual' has  def 'FASTQ format subset for Phred sequencing quality score data only (no sequences).' \n",
      "Data term  'qualsolid' has  def 'FASTQ format subset for Phred sequencing quality score data only (no sequences) for SOLiD data.' \n",
      "Data term  'qual454' has  def 'FASTQ format subset for Phred sequencing quality score data only (no sequences) from 454 sequencers.' \n",
      "Data term  'UniProt-like (text)' has  def 'A text sequence format resembling uniprotkb entry format.' \n",
      "Data term  'UniProtKB format' has  def 'UniProtKB entry sequence format.' \n",
      "Data term  'medline' has  def 'Abstract format used by MedLine database.' \n",
      "Data term  'FASTA-like (text)' has  def 'A text format resembling FASTA format.' \n",
      "Data term  'dbid' has  def 'Fasta format variant with database name before ID.' \n",
      "Data term  'FASTA' has  def 'FASTA format including NCBI-style IDs.' \n",
      "Data term  'giFASTA format' has  def 'FASTA sequence format including NCBI-style GIs.' \n",
      "Data term  'Pearson format' has  def 'Plain old FASTA sequence format (unspecified format for IDs).' \n",
      "Data term  'NCBI format' has  def 'NCBI FASTA sequence format with NCBI-style IDs.' \n",
      "Data term  'FASTA-aln' has  def 'Fasta format for (aligned) sequences.' \n",
      "Data term  'A2M' has  def 'The A2M format is used as the primary format for multiple alignments of protein or nucleic-acid sequences in the SAM suite of tools. It is a small modification of FASTA format for sequences and is compatible with most tools that read FASTA.' \n",
      "Data term  'csfasta' has  def 'Color space FASTA format sequence variant.' \n",
      "Data term  'XMFA' has  def 'XMFA format stands for eXtended Multi-FastA format and is used to store collinear sub-alignments that constitute a single genome alignment.' \n",
      "Data term  'GenBank-like format (text)' has  def 'A text format resembling GenBank entry (plain text) format.' \n",
      "Data term  'GenBank format' has  def 'Genbank entry format.' \n",
      "Data term  'genpept' has  def 'Genpept protein entry format.' \n",
      "Data term  'GFF' has  def 'GFF feature format (of indeterminate version).' \n",
      "Data term  'GFF2' has  def 'General Feature Format (GFF) of sequence features.' \n",
      "Data term  'GFF2-seq' has  def 'GFF feature file format with sequence in the header.' \n",
      "Data term  'GFF3' has  def 'Generic Feature Format version 3 (GFF3) of sequence features.' \n",
      "Data term  'GFF3-seq' has  def 'GFF3 feature file format with sequence.' \n",
      "Data term  'GVF' has  def 'Genome Variation Format (GVF). A GFF3-compatible format with defined header and attribute tags for sequence variation.' \n",
      "Data term  'mirGFF3' has  def 'mirGFF3 is a common format for microRNA data resulting from small-RNA RNA-Seq workflows.' \n",
      "Data term  'GTF' has  def 'Gene Transfer Format (GTF), a restricted version of GFF.' \n",
      "Data term  'OBO' has  def 'OBO ontology text format.' \n",
      "Data term  'completely unambiguous' has  def 'Alphabet for a molecular sequence without any unknown positions or ambiguity characters.' \n",
      "Data term  'SAM' has  def 'Sequence Alignment/Map (SAM) format for alignment of nucleotide sequences (e.g. sequencing reads) to (a) reference sequence(s). May contain base-call and alignment qualities and other data.' \n",
      "Data term  'markx0 variant' has  def 'Some variant of Pearson MARKX alignment format.' \n",
      "Data term  'markx0' has  def 'Pearson MARKX0 alignment format.' \n",
      "Data term  'markx1' has  def 'Pearson MARKX1 alignment format.' \n",
      "Data term  'markx10' has  def 'Pearson MARKX10 alignment format.' \n",
      "Data term  'markx2' has  def 'Pearson MARKX2 alignment format.' \n",
      "Data term  'markx3' has  def 'Pearson MARKX3 alignment format.' \n",
      "Data term  'mega variant' has  def 'Some variant of Mega format for (typically aligned) sequences.' \n",
      "Data term  'mega' has  def 'Mega format for (typically aligned) sequences.' \n",
      "Data term  'meganon' has  def 'Mega non-interleaved format for (typically aligned) sequences.' \n",
      "Data term  'Phylip format variant' has  def 'Some variant of Phylip format for (aligned) sequences.' \n",
      "Data term  'PHYLIP format' has  def 'Phylip format for (aligned) sequences.' \n",
      "Data term  'PHYLIP sequential' has  def 'Phylip non-interleaved format for (aligned) sequences.' \n",
      "Data term  'Relaxed PHYLIP Interleaved' has  def 'Phylip multiple alignment sequence format, less stringent than PHYLIP format.' \n",
      "Data term  'Relaxed PHYLIP Sequential' has  def 'Phylip multiple alignment sequence format, less stringent than PHYLIP sequential format (format_1998).' \n",
      "Data term  'ACE' has  def 'ACE sequence assembly format including contigs, base-call qualities, and other metadata (version Aug 1998 and onwards).' \n",
      "Data term  'BED' has  def 'Browser Extensible Data (BED) format of sequence annotation track, typically to be displayed in a genome browser.' \n",
      "Data term  'bedstrict' has  def 'Browser Extensible Data (BED) format of sequence annotation track that strictly does not contain non-standard fields beyond the first 3 columns.' \n",
      "Data term  'bed6' has  def 'BED file format where each feature is described by chromosome, start, end, name, score, and strand.' \n",
      "Data term  'ENCODE peak format' has  def 'Human ENCODE peak format.' \n",
      "Data term  'ENCODE narrow peak format' has  def 'Human ENCODE narrow peak format.' \n",
      "Data term  'ENCODE broad peak format' has  def 'Human ENCODE broad peak format.' \n",
      "Data term  'bed12' has  def 'A BED file where each feature is described by all twelve columns.' \n",
      "Data term  'chrominfo' has  def 'Tabular format of chromosome names and sizes used by Galaxy.' \n",
      "Data term  'WIG' has  def 'Wiggle format (WIG) of a sequence annotation track that consists of a value for each sequence position. Typically to be displayed in a genome browser.' \n",
      "Data term  'PSL' has  def 'PSL format of alignments, typically generated by BLAT or psLayout. Can be displayed in a genome browser like a sequence annotation track.' \n",
      "Data term  'MAF' has  def 'Multiple Alignment Format (MAF) supporting alignments of whole genomes with rearrangements, directions, multiple pieces to the alignment, and so forth.' \n",
      "Data term  'genePred' has  def 'genePred table format for gene prediction tracks.' \n",
      "Data term  'pgSnp' has  def 'Personal Genome SNP (pgSnp) format for sequence variation tracks (indels and polymorphisms), supported by the UCSC Genome Browser.' \n",
      "Data term  'axt' has  def 'axt format of alignments, typically produced from BLASTZ.' \n",
      "Data term  'LAV' has  def 'LAV format of alignments generated by BLASTZ and LASTZ.' \n",
      "Data term  'Pileup' has  def 'Pileup format of alignment of sequences (e.g. sequencing reads) to (a) reference sequence(s). Contains aligned bases per base of the reference sequence(s).' \n",
      "Data term  'VCF' has  def 'Variant Call Format (VCF) is tabular format for storing genomic sequence variations.' \n",
      "Data term  'gVCF' has  def 'Genomic Variant Call Format (gVCF) is a version of VCF that includes not only the positions that are variant when compared to a reference genome, but also the non-variant positions as ranges, including metrics of confidence that the positions in the range are actually non-variant e.g. minimum read-depth and genotype quality.' \n",
      "Data term  'GTrack' has  def 'GTrack is a generic and optimised tabular format for genome or sequence feature tracks. GTrack unifies the power of other track formats (e.g. GFF3, BED, WIG), and while optimised in size, adds more flexibility, customisation, and automation (\"machine understandability\").' \n",
      "Data term  'Cytoband format' has  def 'Cytoband format for chromosome cytobands.' \n",
      "Data term  'PSI MI TAB (MITAB)' has  def 'Tabular Molecular Interaction format (MITAB), standardised by HUPO PSI MI.' \n",
      "Data term  'OWL Functional Syntax' has  def 'A human-readable encoding for the Web Ontology Language (OWL).' \n",
      "Data term  'Manchester OWL Syntax' has  def 'A syntax for writing OWL class expressions.' \n",
      "Data term  'KRSS2 Syntax' has  def 'A superset of the \"Description-Logic Knowledge Representation System Specification from the KRSS Group of the ARPA Knowledge Sharing Effort\".' \n",
      "Data term  'Turtle' has  def 'The Terse RDF Triple Language (Turtle) is a human-friendly serialisation format for RDF (Resource Description Framework) graphs.' \n",
      "Data term  'N-Triples' has  def 'A plain text serialisation format for RDF (Resource Description Framework) graphs, and a subset of the Turtle (Terse RDF Triple Language) format.' \n",
      "Data term  'Notation3' has  def 'A shorthand non-XML serialisation of Resource Description Framework model, designed with human-readability in mind.' \n",
      "Data term  'PED/MAP' has  def 'The PED/MAP file describes data used by the Plink package.' \n",
      "Data term  'MAP' has  def 'The MAP file describes SNPs and is used by the Plink package.' \n",
      "Data term  'PED' has  def 'The PED file describes individuals and genetic data and is used by the Plink package.' \n",
      "Data term  'CT' has  def 'File format of a CT (Connectivity Table) file from the RNAstructure package.' \n",
      "Data term  'SS' has  def 'XRNA old input style format.' \n",
      "Data term  'GDE' has  def 'Format for the Genetic Data Environment (GDE).' \n",
      "Data term  'Cytoscape input file format' has  def 'Format of the cytoscape input file of gene expression ratios or values are specified over one or more experiments.' \n",
      "Data term  'GCG format variant' has  def 'Some format based on the GCG format.' \n",
      "Data term  'GCG MSF' has  def 'GCG MSF (multiple sequence file) file format.' \n",
      "Data term  'RSF' has  def 'Rich sequence format.' \n",
      "Data term  'Ensembl variation file format' has  def 'Ensembl standard format for variation data.' \n",
      "Data term  'mhd' has  def 'Text-based tagged file format for medical images generated using the MetaImage software package.' \n",
      "Data term  'nrrd' has  def 'Nearly Raw Rasta Data format designed to support scientific visualisation and image processing involving N-dimensional raster data.' \n",
      "Data term  'R file format' has  def 'File format used for scripts written in the R programming language for execution within the R software environment, typically for statistical computation and graphics.' \n",
      "Data term  'SPSS' has  def 'File format used for scripts for the Statistical Package for the Social Sciences.' \n",
      "Data term  'rcc' has  def 'Reporter Code Count-A data file (.csv) output by the Nanostring nCounter Digital Analyzer, which contains gene sample information, probe information and probe counts.' \n",
      "Data term  'arff' has  def 'ARFF (Attribute-Relation File Format) is an ASCII text file format that describes a list of instances sharing a set of attributes.' \n",
      "Data term  'afg' has  def 'AFG is a single text-based file assembly format that holds read and consensus information together.' \n",
      "Data term  'bedgraph' has  def 'The bedGraph format allows display of continuous-valued data in track format. This display type is useful for probability scores and transcriptome data.' \n",
      "Data term  'customtrack' has  def 'Custom Sequence annotation track format used by Galaxy.' \n",
      "Data term  'mzTab' has  def 'mzTab is a tab-delimited format for mass spectrometry-based proteomics and metabolomics results.' \n",
      "Data term  'ISA-TAB' has  def 'The Investigation / Study / Assay (ISA) tab-delimited (TAB) format incorporates metadata from experiments employing a combination of technologies.' \n",
      "Data term  'SBtab' has  def 'SBtab is a tabular format for biochemical network models.' \n",
      "Data term  'BEL' has  def 'Biological Expression Language (BEL) is a textual format for representing scientific findings in life sciences in a computable form.' \n",
      "Data term  'AGP' has  def 'AGP is a tabular format for a sequence assembly (a contig, a scaffold/supercontig, or a chromosome).' \n",
      "Data term  'PS' has  def 'PostScript format.' \n",
      "Data term  'EPS' has  def 'Encapsulated PostScript format.' \n",
      "Data term  'GCT/Res format' has  def 'Tab-delimited text files of GenePattern that contain a column for each sample, a row for each gene, and an expression value for each gene in each sample.' \n",
      "Data term  'Mascot .dat file' has  def '\"Raw\" result file from Mascot database search.' \n",
      "Data term  'MaxQuant APL peaklist format' has  def 'Format of peak list files from Andromeda search engine (MaxQuant) that consist of arbitrarily many spectra.' \n",
      "Data term  'LocARNA PP' has  def 'The LocARNA PP format combines sequence or alignment information and (respectively, single or consensus) ensemble probabilities into an PP 2.0 record.' \n",
      "Data term  'dbGaP format' has  def 'Input format used by the Database of Genotypes and Phenotypes (dbGaP).' \n",
      "Data term  'BIOM format' has  def 'The BIological Observation Matrix (BIOM) is a format for representing biological sample by observation contingency tables in broad areas of comparative omics. The primary use of this format is to represent OTU tables and metagenome tables.' \n",
      "Data term  'DSV' has  def 'Tabular data represented as values in a text file delimited by some character.' \n",
      "Data term  'TSV' has  def 'Tabular data represented as tab-separated values in a text file.' \n",
      "Data term  'MAGE-TAB' has  def 'MAGE-TAB textual format for microarray expression data, standardised by MGED (now FGED).' \n",
      "Data term  'WEGO' has  def 'WEGO native format used by the Web Gene Ontology Annotation Plot application.   Tab-delimited format with gene names and others GO IDs (columns) with one annotation record per line.' \n",
      "Data term  'RPKM' has  def 'Tab-delimited format for gene expression levels table, calculated as Reads Per Kilobase per Million (RPKM) mapped reads.' \n",
      "Data term  'PEtab' has  def 'A data format for specifying parameter estimation problems in systems biology.' \n",
      "Data term  'CSV' has  def 'Tabular data represented as comma-separated values in a text file.' \n",
      "Data term  'SEQUEST .out file' has  def '\"Raw\" result file from SEQUEST database search.' \n",
      "Data term  'GSuite' has  def 'GSuite is a tabular format for collections of genome or sequence feature tracks, suitable for integrative multi-track analysis. GSuite contains links to genome/sequence tracks, with additional metadata.' \n",
      "Data term  'MCPD' has  def 'The FAO/Bioversity/IPGRI Multi-Crop Passport Descriptors (MCPD) is an international standard format for exchange of germplasm information.' \n",
      "Data term  'PubTator format' has  def 'Native textual export format of annotated scientific text from PubTator.' \n",
      "Data term  'BioNLP Shared Task format' has  def 'A family of similar formats of text annotation, used by BRAT and other tools, known as BioNLP Shared Task format (BioNLP 2009 Shared Task on Event Extraction, BioNLP Shared Task 2011, BioNLP Shared Task 2013), BRAT format, BRAT standoff format, and similar.' \n",
      "Data term  'SQL' has  def 'SQL (Structured Query Language) is the de-facto standard query language (format of queries) for querying and manipulating data in relational databases.' \n",
      "Data term  'XQuery' has  def 'XQuery (XML Query) is a query language (format of queries) for querying and manipulating structured and unstructured data, usually in the form of XML, text, and with vendor-specific extensions for other data formats (JSON, binary, etc.).' \n",
      "Data term  'SPARQL' has  def 'SPARQL (SPARQL Protocol and RDF Query Language) is a semantic query language for querying and manipulating data stored in Resource Description Framework (RDF) format.' \n",
      "Data term  'GEN' has  def 'The GEN file format contains genetic data and describes SNPs.' \n",
      "Data term  'SAMPLE file format' has  def 'The SAMPLE file format contains information about each individual i.e. individual IDs, covariates, phenotypes and missing data proportions, from a GWAS study.' \n",
      "Data term  'SDF' has  def 'SDF is one of a family of chemical-data file formats developed by MDL Information Systems; it is intended especially for structural information.' \n",
      "Data term  'Molfile' has  def 'An MDL Molfile is a file format for holding information about the atoms, bonds, connectivity and coordinates of a molecule.' \n",
      "Data term  'Mol2' has  def 'Complete, portable representation of a SYBYL molecule. ASCII file which contains all the information needed to reconstruct a SYBYL molecule.' \n",
      "Data term  'latex' has  def 'format for the LaTeX document preparation system.' \n",
      "Data term  'ELAND format' has  def 'Tab-delimited text file format used by Eland - the read-mapping program distributed by Illumina with its sequencing analysis pipeline - which maps short Solexa sequence reads to the human reference genome.' \n",
      "Data term  'GML' has  def 'GML (Graph Modeling Language) is a text file format supporting network data with a very easy syntax. It is used by Graphlet, Pajek, yEd, LEDA and NetworkX.' \n",
      "Data term  'FASTG' has  def 'FASTG is a format for faithfully representing genome assemblies in the face of allelic polymorphism and assembly uncertainty.' \n",
      "Data term  'proBED' has  def '. proBED is an adaptation of BED (format_3003), which was extended to meet specific requirements entailed by proteomics data.' \n",
      "Data term  'GPR' has  def 'GenePix Results (GPR) text file format developed by Axon Instruments that is used to save GenePix Results data.' \n",
      "Data term  'JCAMP-DX' has  def 'A standardized file format for data exchange in mass spectrometry, initially developed for infrared spectrometry.' \n",
      "Data term  'XYZ' has  def 'The XYZ chemical file format is widely supported by many programs, although many slightly different XYZ file formats coexist (Tinker XYZ, UniChem XYZ, etc.). Basic information stored for each atom in the system are x, y and z coordinates and atom element/atomic number.' \n",
      "Data term  'mdcrd' has  def 'AMBER trajectory (also called mdcrd), with 10 coordinates per line and format F8.3 (fixed point notation with field width 8 and 3 decimal places).' \n",
      "Data term  'GROMACS top' has  def 'GROMACS MD package top textual files define an entire structure system topology, either directly, or by including itp files.' \n",
      "Data term  'AMBER top' has  def 'AMBER Prmtop file (version 7) is a structure topology text file divided in several sections designed to be parsed easily using simple Fortran code. Each section contains particular topology information, such as atom name, charge, mass, angles, dihedrals, etc.' \n",
      "Data term  'PSF' has  def 'X-Plor Protein Structure Files (PSF) are structure topology files used by NAMD and CHARMM molecular simulations programs. PSF files contain six main sections of interest: atoms, bonds, angles, dihedrals, improper dihedrals (force terms used to maintain planarity) and cross-terms.' \n",
      "Data term  'GROMACS itp' has  def 'GROMACS itp files (include topology) contain structure topology information, and are typically included in GROMACS topology files (GROMACS top). Itp files are used to define individual (or multiple) components of a topology as a separate file. This is particularly useful if there is a molecule that is used frequently, and also reduces the size of the system topology file, splitting it in different parts.' \n",
      "Data term  'RST' has  def 'AMBER coordinate/restart file with 6 coordinates per line and decimal format F12.7 (fixed point notation with field width 12 and 7 decimal places).' \n",
      "Data term  'CHARMM rtf' has  def 'Format of CHARMM Residue Topology Files (RTF), which define groups by including the atoms, the properties of the group, and bond and charge information.' \n",
      "Data term  'AMBER frcmod' has  def 'AMBER frcmod (Force field Modification) is a file format to store any modification to the standard force field needed for a particular molecule to be properly represented in the simulation.' \n",
      "Data term  'AMBER off' has  def 'AMBER Object File Format library files (OFF library files) store residue libraries (forcefield residue parameters).' \n",
      "Data term  'NMReDATA' has  def 'MReData is a text based data standard for processed NMR data. It is relying on SDF molecule data and allows to store assignments of NMR peaks to molecule features. The NMR-extracted data (or \"NMReDATA\") includes: Chemical shift,scalar coupling, 2D correlation, assignment, etc.' \n",
      "Data term  'BpForms' has  def 'BpForms is a string format for concretely representing the primary structures of biopolymers, including DNA, RNA, and proteins that include non-canonical nucleic and amino acids. See https://www.bpforms.org for more information.' \n",
      "Data term  'trr' has  def 'Format of trr files that contain the trajectory of a simulation experiment used by GROMACS.' \n",
      "Data term  'MTX' has  def 'The Matrix Market matrix (MTX) format stores numerical or pattern matrices in a dense (array format) or sparse (coordinate format) representation.' \n",
      "Data term  'BcForms' has  def 'BcForms is a format for abstractly describing the molecular structure (atoms and bonds) of macromolecular complexes as a collection of subunits and crosslinks. Each subunit can be described with BpForms (http://edamontology.org/format_3909) or SMILES (http://edamontology.org/data_2301). BcForms uses an ontology of crosslinks to abstract the chemical details of crosslinks from the descriptions of complexes (see https://bpforms.org/crosslink.html).' \n",
      "Data term  'N-Quads' has  def 'N-Quads is a line-based, plain text format for encoding an RDF dataset. It includes information about the graph each triple belongs to.' \n",
      "Data term  'BNGL' has  def 'BioNetGen is a format for the specification and simulation of rule-based models of biochemical systems, including signal transduction, metabolic, and genetic regulatory networks.' \n",
      "Data term  'GFA 1' has  def 'Graphical Fragment Assembly captures sequence graphs as the product of an assembly, a representation of variation in genomes, splice graphs in genes, or even overlap between reads from long-read sequencing technology.' \n",
      "Data term  'GFA 2' has  def 'Graphical Fragment Assembly captures sequence graphs as the product of an assembly, a representation of variation in genomes, splice graphs in genes, or even overlap between reads from long-read sequencing technology. GFA2 is an update of GFA1 which is not compatible with GFA1.' \n",
      "Data term  'CONTIG' has  def 'The CONTIG format used for output of the SOAPdenovo alignment program. It contains contig sequences generated without using mate pair information.' \n",
      "Data term  'CHAIN' has  def 'The CHAIN format describes a pairwise alignment that allow gaps in both sequences simultaneously and is used by the UCSC Genome Browser.' \n",
      "Data term  'NET' has  def 'The NET file format is used to describe the data that underlie the net alignment annotations in the UCSC Genome Browser.' \n",
      "Data term  'QMAP' has  def 'Format of QMAP files generated for methylation data from an internal BGI pipeline.' \n",
      "Data term  'gxformat2' has  def 'An emerging format for high-level Galaxy workflow description.' \n",
      "Data term  'CIGAR format' has  def 'Compact Idiosyncratic Gapped Alignment Report format is a compressed (run-length encoded) pairwise alignment format. It is useful for representing long (e.g. genomic) pairwise alignments.' \n",
      "Data term  'Python script' has  def 'Format for scripts writtenin Python - a widely used high-level programming language for general-purpose programming.' \n",
      "Data term  'Perl script' has  def 'Format for scripts written in Perl - a family of high-level, general-purpose, interpreted, dynamic programming languages.' \n",
      "Data term  'R script' has  def 'Format for scripts written in the R language - an open source programming language and software environment for statistical computing and graphics that is supported by the R Foundation for Statistical Computing.' \n",
      "Data term  'R markdown' has  def 'A file format for making dynamic documents (R Markdown scripts) with the R language.' \n",
      "Data term  'Configuration file format' has  def 'A configuration file used by various programs to store settings that are specific to their respective software.' \n",
      "Data term  'MATLAB script' has  def 'The file format for MATLAB scripts or functions.' \n",
      "Data term  'BioSimulators standard for command-line interfaces for biosimulation tools' has  def 'Outlines the syntax and semantics of the input and output arguments for command-line interfaces for biosimulation tools.' \n",
      "Data term  'PQR' has  def 'Data format derived from the standard PDB format, which enables user to incorporate parameters for charge and radius to the existing PDB data file.' \n",
      "Data term  'PDBQT' has  def 'Data format used in AutoDock 4 for storing atomic coordinates, partial atomic charges and AutoDock atom types for both receptors and ligands.' \n",
      "Data term  'MSP' has  def 'MSP is a data format for mass spectrometry data.' \n",
      "Data term  'HTML' has  def 'HTML format.' \n",
      "Data term  'iHOP format' has  def 'The format of iHOP (Information Hyperlinked over Proteins) text-mining result.' \n",
      "Data term  'FASTA-HTML' has  def 'FASTA format wrapped in HTML elements.' \n",
      "Data term  'EMBL-HTML' has  def 'EMBL entry format wrapped in HTML elements.' \n",
      "Data term  'GenBank-HTML' has  def 'Genbank entry format wrapped in HTML elements.' \n",
      "Data term  'MHTML' has  def 'MIME HTML format for Web pages, which can include external resources, including images, Flash animations and so on.' \n",
      "Data term  'XML' has  def 'eXtensible Markup Language (XML) format.' \n",
      "Data term  'PDBML' has  def 'Entry format of PDB database in PDBML (XML) format.' \n",
      "Data term  'Taverna workflow format' has  def 'Format of Taverna workflows.' \n",
      "Data term  'DAS format' has  def 'DAS sequence (XML) format (any type).' \n",
      "Data term  'dasdna' has  def 'DAS sequence (XML) format (nucleotide-only).' \n",
      "Data term  'DASGFF' has  def 'DAS GFF (XML) feature format.' \n",
      "Data term  'STRING entry format (XML)' has  def 'Entry format (XML) for the STRING database of protein interaction.' \n",
      "Data term  'BioXSD (XML)' has  def 'BioXSD-schema-based XML format of sequence-based data and some other common data - sequence records, alignments, feature records, references to resources, and more - optimised for integrative bioinformatics, Web services, and object-oriented programming.' \n",
      "Data term  'OBO-XML' has  def 'OBO ontology XML format.' \n",
      "Data term  'EMBL-like (XML)' has  def 'An XML format resembling EMBL entry format.' \n",
      "Data term  'EMBL format (XML)' has  def 'An XML format for EMBL entries.' \n",
      "Data term  'EMBLXML' has  def 'XML format for EMBL entries.' \n",
      "Data term  'cdsxml' has  def 'Specific XML format for EMBL entries (only uses certain sections).' \n",
      "Data term  'INSDSeq' has  def 'INSDSeq provides the elements of a sequence as presented in the GenBank/EMBL/DDBJ-style flatfile formats, with a small amount of additional structure.' \n",
      "Data term  'SBML' has  def 'Systems Biology Markup Language (SBML), the standard XML format for models of biological processes such as for example metabolism, cell signaling, and gene regulation.' \n",
      "Data term  'SBRML' has  def 'Systems Biology Result Markup Language (SBRML), the standard XML format for simulated or calculated results (e.g. trajectories) of systems biology models.' \n",
      "Data term  'EBI Application Result XML' has  def 'EBI Application Result XML is a format returned by sequence similarity search Web services at EBI.' \n",
      "Data term  'PSI MI XML (MIF)' has  def 'XML Molecular Interaction Format (MIF), standardised by HUPO PSI MI.' \n",
      "Data term  'PSI-PAR' has  def 'Protein affinity format (PSI-PAR), standardised by HUPO PSI MI. It is compatible with PSI MI XML (MIF) and uses the same XML Schema.' \n",
      "Data term  'phyloXML' has  def 'phyloXML is a standardised XML format for phylogenetic trees, networks, and associated data.' \n",
      "Data term  'NeXML' has  def 'NeXML is a standardised XML format for rich phyloinformatic data.' \n",
      "Data term  'MAGE-ML' has  def 'MAGE-ML XML format for microarray expression data, standardised by MGED (now FGED).' \n",
      "Data term  'GCDML' has  def 'GCDML XML format for genome and metagenome metadata according to MIGS/MIMS/MIMARKS information standards, standardised by the Genomic Standards Consortium (GSC).' \n",
      "Data term  'CopasiML' has  def 'CopasiML, the native format of COPASI.' \n",
      "Data term  'CellML' has  def 'CellML, the format for mathematical models of biological and other networks.' \n",
      "Data term  'mzML' has  def 'mzML format for raw spectrometer output data, standardised by HUPO PSI MSS.' \n",
      "Data term  'TraML' has  def 'TraML (Transition Markup Language) is the format for mass spectrometry transitions, standardised by HUPO PSI MSS.' \n",
      "Data term  'mzIdentML' has  def 'mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.' \n",
      "Data term  'mzQuantML' has  def 'mzQuantML is the format for quantitation values associated with peptides, proteins and small molecules from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of quantitation software for proteomics.' \n",
      "Data term  'GelML' has  def 'GelML is the format for describing the process of gel electrophoresis, standardised by HUPO PSI PS.' \n",
      "Data term  'spML' has  def 'spML is the format for describing proteomics sample processing, other than using gels, prior to mass spectrometric protein identification, standardised by HUPO PSI PS. It may also be applicable for metabolomics.' \n",
      "Data term  'RDF/XML' has  def 'Resource Description Framework (RDF) XML format.' \n",
      "Data term  'OWL/XML' has  def 'OWL ontology XML serialisation format.' \n",
      "Data term  'RNAML' has  def 'RNA Markup Language.' \n",
      "Data term  'xls' has  def 'Microsoft Excel spreadsheet format.' \n",
      "Data term  'BSML' has  def 'Bioinformatics Sequence Markup Language format.' \n",
      "Data term  'docx' has  def 'Microsoft Word format.' \n",
      "Data term  'SVG' has  def 'Scalable Vector Graphics (SVG) is an XML-based vector image format for two-dimensional graphics with support for interactivity and animation.' \n",
      "Data term  'pepXML' has  def 'Open data format for the storage, exchange, and processing of peptide sequence assignments of MS/MS scans, intended to provide a common data output format for many different MS/MS search engines and subsequent peptide-level analyses.' \n",
      "Data term  'GPML' has  def 'Graphical Pathway Markup Language (GPML) is an XML format used for exchanging biological pathways.' \n",
      "Data term  'imzML metadata file' has  def 'imzML metadata is a data format for mass spectrometry imaging metadata.' \n",
      "Data term  'qcML' has  def 'qcML is an XML format for quality-related data of mass spectrometry and other high-throughput measurements.' \n",
      "Data term  'PRIDE XML' has  def 'PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.' \n",
      "Data term  'SED-ML' has  def 'Simulation Experiment Description Markup Language (SED-ML) is an XML format for encoding simulation setups, according to the MIASE (Minimum Information About a Simulation Experiment) requirements.' \n",
      "Data term  'BCML' has  def 'Biological Connection Markup Language (BCML) is an XML format for biological pathways.' \n",
      "Data term  'BDML' has  def 'Biological Dynamics Markup Language (BDML) is an XML format for quantitative data describing biological dynamics.' \n",
      "Data term  'SBGN-ML' has  def 'SBGN-ML is an XML format for Systems Biology Graphical Notation (SBGN) diagrams of biological pathways or networks.' \n",
      "Data term  'X!Tandem XML' has  def 'Output format used by X! series search engines that is based on the XML language BIOML.' \n",
      "Data term  'SBOL' has  def 'Synthetic Biology Open Language (SBOL) is an XML format for the specification and exchange of biological design information in synthetic biology.' \n",
      "Data term  'PMML' has  def 'PMML uses XML to represent mining models. The structure of the models is described by an XML Schema.' \n",
      "Data term  'protXML' has  def 'A format for storage, exchange, and processing of protein identifications created from ms/ms-derived peptide sequence data.' \n",
      "Data term  'idXML' has  def 'XML file format for files containing information about peptide identifications from mass spectrometry data analysis carried out with OpenMS.' \n",
      "Data term  'UniProtKB XML' has  def 'UniProtKB XML sequence features format is an XML format available for downloading UniProt entries.' \n",
      "Data term  'BioC' has  def 'BioC is a standardised XML format for sharing and integrating text data and annotations.' \n",
      "Data term  'xsd' has  def 'XML format for XML Schema.' \n",
      "Data term  'VisML' has  def 'Default XML format of VisANT, containing all the network information.' \n",
      "Data term  'nmrML' has  def 'nmrML is an MSI supported XML-based open access format for metabolomics NMR raw and processed spectral data. It is accompanies by an nmrCV (controlled vocabulary) to allow ontology-based annotations.' \n",
      "Data term  'consensusXML' has  def 'OpenMS format for grouping features in one map or across several maps.' \n",
      "Data term  'featureXML' has  def 'OpenMS format for quantitation results (LC/MS features).' \n",
      "Data term  'mzData' has  def 'Now deprecated data format of the HUPO Proteomics Standards Initiative. Replaced by mzML (format_3244).' \n",
      "Data term  'TIDE TXT' has  def 'Format supported by the Tide tool for identifying peptides from tandem mass spectra.' \n",
      "Data term  'pptx' has  def 'Microsoft Powerpoint format.' \n",
      "Data term  'BEAST' has  def 'XML input file format for BEAST Software (Bayesian Evolutionary Analysis Sampling Trees).' \n",
      "Data term  'Chado-XML' has  def 'Chado-XML format is a direct mapping of the Chado relational schema into XML.' \n",
      "Data term  'HSAML' has  def 'An alignment format generated by PRANK/PRANKSTER consisting of four elements: newick, nodes, selection and model.' \n",
      "Data term  'InterProScan XML' has  def 'Output xml file from the InterProScan sequence analysis application.' \n",
      "Data term  'KGML' has  def 'The KEGG Markup Language (KGML) is an exchange format of the KEGG pathway maps, which is converted from internally used KGML+ (KGML+SVG) format.' \n",
      "Data term  'PubMed XML' has  def 'XML format for collected entries from bibliographic databases MEDLINE and PubMed.' \n",
      "Data term  'OrthoXML' has  def 'OrthoXML is designed broadly to allow the storage and comparison of orthology data from any ortholog database. It establishes a structure for describing orthology relationships while still allowing flexibility for database-specific information to be encapsulated in the same format.' \n",
      "Data term  'PSDML' has  def 'Tree structure of Protein Sequence Database Markup Language generated using Matra software.' \n",
      "Data term  'SeqXML' has  def 'SeqXML is an XML Schema to describe biological sequences, developed by the Stockholm Bioinformatics Centre.' \n",
      "Data term  'UniParc XML' has  def 'XML format for the UniParc database.' \n",
      "Data term  'UniRef XML' has  def 'XML format for the UniRef reference clusters.' \n",
      "Data term  'NeuroML' has  def 'A model description language for computational neuroscience.' \n",
      "Data term  'cml' has  def 'Chemical Markup Language (CML) is an XML-based format for encoding detailed information about a wide range of chemical concepts.' \n",
      "Data term  'Binary format' has  def 'Binary format.' \n",
      "Data term  'ABI' has  def 'A format of raw sequence read data from an Applied Biosystems sequencing machine.' \n",
      "Data term  'SCF' has  def 'Staden Chromatogram Files format (SCF) of base-called sequence reads, qualities, and other metadata.' \n",
      "Data term  'BAM' has  def 'BAM format, the binary, BGZF-formatted compressed version of SAM format for alignment of nucleotide sequences (e.g. sequencing reads) to (a) reference sequence(s). May contain base-call and alignment qualities and other data.' \n",
      "Data term  'AB1' has  def 'AB1 binary format of raw DNA sequence reads (output of Applied Biosystems' sequencing analysis software). Contains an electropherogram and the DNA base sequence.' \n",
      "Data term  'bigBed' has  def 'bigBed format for large sequence annotation tracks, similar to textual BED format.' \n",
      "Data term  'bigWig' has  def 'bigWig format for large sequence annotation tracks that consist of a value for each sequence position. Similar to textual WIG format.' \n",
      "Data term  '2bit' has  def '2bit binary format of nucleotide sequences using 2 bits per nucleotide. In addition encodes unknown nucleotides and lower-case 'masking'.' \n",
      "Data term  '.nib' has  def '.nib (nibble) binary format of a nucleotide sequence using 4 bits per nucleotide (including unknown) and its lower-case 'masking'.' \n",
      "Data term  'SRF' has  def 'Sequence Read Format (SRF) of sequence trace data. Supports submission to the NCBI Short Read Archive.' \n",
      "Data term  'ZTR' has  def 'ZTR format for storing chromatogram data from DNA sequencing instruments.' \n",
      "Data term  'BCF' has  def 'BCF is the binary version of Variant Call Format (VCF) for sequence variation (indels, polymorphisms, structural variation).' \n",
      "Data term  'SFF' has  def 'Standard flowgram format (SFF) is a binary file format used to encode results of pyrosequencing from the 454 Life Sciences platform for high-throughput sequencing.' \n",
      "Data term  'BAI' has  def 'BAM indexing format.' \n",
      "Data term  'CRAM' has  def 'Reference-based compression of alignment format.' \n",
      "Data term  'GIF' has  def 'Graphics Interchange Format.' \n",
      "Data term  'ebwt' has  def 'Bowtie format for indexed reference genome for \"small\" genomes.' \n",
      "Data term  'ebwtl' has  def 'Bowtie format for indexed reference genome for \"large\" genomes.' \n",
      "Data term  'PDF' has  def 'Portable Document Format.' \n",
      "Data term  'DICOM format' has  def 'Medical image format corresponding to the Digital Imaging and Communications in Medicine (DICOM) standard.' \n",
      "Data term  'nii' has  def 'An open file format from the Neuroimaging Informatics Technology Initiative (NIfTI) commonly used to store brain imaging data obtained using Magnetic Resonance Imaging (MRI) methods.' \n",
      "Data term  'IDAT' has  def 'Proprietary file format for (raw) BeadArray data used by genomewide profiling platforms from Illumina Inc. This format is output directly from the scanner and stores summary intensities for each probe-type on an array.' \n",
      "Data term  'JPG' has  def 'Joint Picture Group file format for lossy graphics file.' \n",
      "Data term  'HDF5' has  def 'HDF5 is a data model, library, and file format for storing and managing data, based on Hierarchical Data Format (HDF).' \n",
      "Data term  'Loom' has  def 'The Loom file format is based on HDF5, a standard for storing large numerical datasets. The Loom format is designed to efficiently hold large omics datasets. Typically, such data takes the form of a large matrix of numbers, along with metadata for the rows and columns.' \n",
      "Data term  'TIFF' has  def 'A versatile bitmap format.' \n",
      "Data term  'BMP' has  def 'Standard bitmap storage format in the Microsoft Windows environment.' \n",
      "Data term  'im' has  def 'IM is a format used by LabEye and other applications based on the IFUNC image processing library.' \n",
      "Data term  'pcd' has  def 'Photo CD format, which is the highest resolution format for images on a CD.' \n",
      "Data term  'pcx' has  def 'PCX is an image file format that uses a simple form of run-length encoding. It is lossless.' \n",
      "Data term  'ppm' has  def 'The PPM format is a lowest common denominator color image file format.' \n",
      "Data term  'psd' has  def 'PSD (Photoshop Document) is a proprietary file that allows the user to work with the images' individual layers even after the file has been saved.' \n",
      "Data term  'xbm' has  def 'X BitMap is a plain text binary image format used by the X Window System used for storing cursor and icon bitmaps used in the X GUI.' \n",
      "Data term  'xpm' has  def 'X PixMap (XPM) is an image file format used by the X Window System, it is intended primarily for creating icon pixmaps, and supports transparent pixels.' \n",
      "Data term  'rgb' has  def 'RGB file format is the native raster graphics file format for Silicon Graphics workstations.' \n",
      "Data term  'pbm' has  def 'The PBM format is a lowest common denominator monochrome file format. It serves as the common language of a large family of bitmap image conversion filters.' \n",
      "Data term  'pgm' has  def 'The PGM format is a lowest common denominator grayscale file format.' \n",
      "Data term  'PNG' has  def 'PNG is a file format for image compression.' \n",
      "Data term  'rast' has  def 'Sun Raster is a raster graphics file format used on SunOS by Sun Microsystems.' \n",
      "Data term  'bgzip' has  def 'Blocked GNU Zip format.' \n",
      "Data term  'tabix' has  def 'TAB-delimited genome position file index format.' \n",
      "Data term  'xlsx' has  def 'MS Excel spreadsheet format consisting of a set of XML documents stored in a ZIP-compressed file.' \n",
      "Data term  'ObjTables' has  def 'ObjTables is a toolkit for creating re-usable datasets that are both human and machine-readable, combining the ease of spreadsheets (e.g., Excel workbooks) with the rigor of schemas (classes, their attributes, the type of each attribute, and the possible relationships between instances of classes). ObjTables consists of a format for describing schemas for spreadsheets, numerous data types for science, a syntax for indicating the class and attribute represented by each table and column in a workbook, and software for using schemas to rigorously validate, merge, split, compare, and revision datasets.' \n",
      "Data term  'SQLite format' has  def 'Data format used by the SQLite database.' \n",
      "Data term  'Gemini SQLite format' has  def 'Data format used by the SQLite database conformant to the Gemini schema.' \n",
      "Data term  'snpeffdb' has  def 'An index of a genome database, indexed for use by the snpeff tool.' \n",
      "Data term  'netCDF' has  def 'Format used by netCDF software library for writing and reading chromatography-MS data files. Also used to store trajectory atom coordinates information, such as the ones obtained by Molecular Dynamics simulations.' \n",
      "Data term  'K-mer countgraph' has  def 'A list of k-mers and their occurrences in a dataset. Can also be used as an implicit De Bruijn graph.' \n",
      "Data term  'COMBINE OMEX' has  def 'Open Modeling EXchange format (OMEX) is a ZIPped format for encapsulating all information necessary for a modeling and simulation project in systems biology.' \n",
      "Data term  'SRA format' has  def 'SRA archive format (SRA) is the archive format used for input to the NCBI Sequence Read Archive.' \n",
      "Data term  'VDB' has  def 'VDB ('vertical database') is the native format used for export from the NCBI Sequence Read Archive.' \n",
      "Data term  'Tabix index file format' has  def 'Index file format used by the samtools package to index TAB-delimited genome position files.' \n",
      "Data term  'WIFF format' has  def 'Mass spectrum file format from QSTAR and QTRAP instruments (ABI/Sciex).' \n",
      "Data term  'Thermo RAW' has  def 'Proprietary file format for mass spectrometry data from Thermo Scientific.' \n",
      "Data term  'OME-TIFF' has  def 'Image file format used by the Open Microscopy Environment (OME).' \n",
      "Data term  'BTrack' has  def 'BTrack is an HDF5-based binary format for genome or sequence feature tracks and their collections, suitable for integrative multi-track analysis. BTrack is a binary, compressed alternative to the GTrack and GSuite formats.' \n",
      "Data term  'proBAM' has  def '. proBAM is an adaptation of BAM (format_2572), which was extended to meet specific requirements entailed by proteomics data.' \n",
      "Data term  'ARB' has  def 'Binary format used by the ARB software suite.' \n",
      "Data term  'ibd' has  def 'ibd is a data format for mass spectrometry imaging data.' \n",
      "Data term  'MSAML' has  def 'A set of XML compliant markup components for describing multiple sequence alignments.' \n",
      "Data term  'Waters RAW' has  def 'Proprietary file format for mass spectrometry data from Waters.' \n",
      "Data term  'HDF' has  def 'HDF is the name of a set of file formats and libraries designed to store and organize large amounts of numerical data, originally developed at the National Center for Supercomputing Applications at the University of Illinois.' \n",
      "Data term  'PCAzip' has  def 'PCAZip format is a binary compressed file to store atom coordinates based on Essential Dynamics (ED) and Principal Component Analysis (PCA).' \n",
      "Data term  'XTC' has  def 'Portable binary format for trajectories produced by GROMACS package.' \n",
      "Data term  'TNG' has  def 'Trajectory Next Generation (TNG) is a format for storage of molecular simulation data. It is designed and implemented by the GROMACS development group, and it is called to be the substitute of the XTC format.' \n",
      "Data term  'BinPos' has  def 'Scripps Research Institute BinPos format is a binary formatted file to store atom coordinates.' \n",
      "Data term  'msh' has  def 'Mash sketch is a format for sequence / sequence checksum information. To make a sketch, each k-mer in a sequence is hashed, which creates a pseudo-random identifier. By sorting these hashes, a small subset from the top of the sorted list can represent the entire sequence.' \n",
      "Data term  'Zarr' has  def 'The Zarr format is an implementation of chunked, compressed, N-dimensional arrays for storing data.' \n",
      "Data term  'Docker image' has  def 'A Docker image is a file, comprised of multiple layers, that is used to execute code in a Docker container. An image is essentially built from the instructions for a complete and executable version of an application, which relies on the host OS kernel.' \n",
      "Data term  'TAR format' has  def 'TAR archive file format generated by the Unix-based utility tar.' \n",
      "Data term  'WMV' has  def 'The proprietary native video format of various Microsoft programs such as Windows Media Player.' \n",
      "Data term  'ZIP format' has  def 'ZIP is an archive file format that supports lossless data compression.' \n",
      "Data term  'LSM' has  def 'Zeiss' proprietary image format based on TIFF.' \n",
      "Data term  'GZIP format' has  def 'GNU zip compressed file format common to Unix-based operating systems.' \n",
      "Data term  'AVI' has  def 'Audio Video Interleaved (AVI) format is a multimedia container format for AVI files, that allows synchronous audio-with-video playback.' \n",
      "Data term  'TrackDB' has  def 'A declaration file format for UCSC browsers track dataset display charateristics.' \n",
      "Data term  'Stereolithography format' has  def 'STL is a file format native to the stereolithography CAD software created by 3D Systems. The format is used to save and share surface-rendered 3D images and also for 3D printing.' \n",
      "Data term  'U3D' has  def 'U3D (Universal 3D) is a compressed file format and data structure for 3D computer graphics. It contains 3D model information such as triangle meshes, lighting, shading, motion data, lines and points with color and structure.' \n",
      "Data term  'Texture file format' has  def 'Bitmap image format used for storing textures.' \n",
      "Data term  'MPEG-4' has  def 'A digital multimedia container format most commonly used to store video and audio.' \n",
      "Data term  'pickle' has  def 'Format used by Python pickle module for serializing and de-serializing a Python object structure.' \n",
      "Data term  'NumPy format' has  def 'The standard binary file format used by NumPy - a fundamental package for scientific computing with Python - for persisting a single arbitrary NumPy array on disk. The format stores all of the shape and dtype information necessary to reconstruct the array correctly.' \n",
      "Data term  'SimTools repertoire file format' has  def 'Format of repertoire (archive) files that can be read by SimToolbox (a MATLAB toolbox for structured illumination fluorescence microscopy) or alternatively extracted with zip file archiver software.' \n",
      "Data term  'Zstandard format' has  def 'Format used by the Zstandard real-time compression algorithm.' \n",
      "Data term  'Format (by type of data)' has  def 'A placeholder concept for visual navigation by dividing data formats by the content of the data that is represented.' \n",
      "Data term  'Sequence record format' has  def 'Data format for a molecular sequence record.' \n",
      "Data term  'Sequence trace format' has  def 'Format for sequence trace data (i.e. including base call information).' \n",
      "Data term  'FASTQ-like format' has  def 'A format resembling FASTQ short read format.' \n",
      "Data term  'EMBL-like format' has  def 'A format resembling EMBL entry (plain text) format.' \n",
      "Data term  'FASTA-like' has  def 'A format resembling FASTA format.' \n",
      "Data term  'uniprotkb-like format' has  def 'A sequence format resembling uniprotkb entry format.' \n",
      "Data term  'UniProtKB RDF' has  def 'UniProtKB RDF sequence features format is an RDF format available for downloading UniProt entries (in RDF/XML).' \n",
      "Data term  'Sequence record format (text)' has  def 'Data format for a molecular sequence record (text).' \n",
      "Data term  'Sequence record format (XML)' has  def 'Data format for a molecular sequence record (XML).' \n",
      "Data term  'GenBank-like format' has  def 'A format resembling GenBank entry (plain text) format.' \n",
      "Data term  'BioJSON (BioXSD)' has  def 'BioJSON is a BioXSD-schema-based JSON format of sequence-based data and some other common data - sequence records, alignments, feature records, references to resources, and more - optimised for integrative bioinformatics, web applications and APIs, and object-oriented programming.' \n",
      "Data term  'BioYAML' has  def 'BioYAML is a BioXSD-schema-based YAML format of sequence-based data and some other common data - sequence records, alignments, feature records, references to resources, and more - optimised for integrative bioinformatics, web APIs, human readability and editing, and object-oriented programming.' \n",
      "Data term  'Sequence feature annotation format' has  def 'Data format for molecular sequence feature information.' \n",
      "Data term  'Sequence feature table format' has  def 'Format for a sequence feature table.' \n",
      "Data term  'Sequence feature table format (text)' has  def 'Text format for a sequence feature table.' \n",
      "Data term  'Sequin format' has  def 'A five-column, tab-delimited table of feature locations and qualifiers for importing annotation into an existing Sequin submission (an NCBI tool for submitting and updating GenBank entries).' \n",
      "Data term  'Sequence feature table format (XML)' has  def 'XML format for a sequence feature table.' \n",
      "Data term  'Sequence annotation track format' has  def 'Format of a sequence annotation track.' \n",
      "Data term  'BioJSON (Jalview)' has  def 'BioJSON is a JSON format of single multiple sequence alignments, with their annotations, features, and custom visualisation and application settings for the Jalview workbench.' \n",
      "Data term  'Alignment format' has  def 'Data format for molecular sequence alignment information.' \n",
      "Data term  'Alignment format (text)' has  def 'Text format for molecular sequence alignment information.' \n",
      "Data term  'pair' has  def 'EMBOSS simple sequence pairwise alignment format.' \n",
      "Data term  'BLC' has  def 'A multiple alignment in vertical format, as used in the AMPS (Alignment of Multiple Protein Sequences) package.' \n",
      "Data term  'PO' has  def 'PO is the output format of Partial Order Alignment program (POA) performing Multiple Sequence Alignment (MSA).' \n",
      "Data term  'Alignment format (XML)' has  def 'XML format for molecular sequence alignment information.' \n",
      "Data term  'Alignment format (pair only)' has  def 'Data format for molecular sequence alignment information that can hold sequence alignment(s) of only 2 sequences.' \n",
      "Data term  'Phylogenetic tree format' has  def 'Data format for a phylogenetic tree.' \n",
      "Data term  'Phylogenetic tree format (text)' has  def 'Text format for a phylogenetic tree.' \n",
      "Data term  'Phylogenetic tree format (XML)' has  def 'XML format for a phylogenetic tree.' \n",
      "Data term  'Biological pathway or network format' has  def 'Data format for a biological pathway or network.' \n",
      "Data term  'BioPAX' has  def 'BioPAX is an exchange format for pathway data, with its data model defined in OWL.' \n",
      "Data term  'sif' has  def 'SIF (simple interaction file) Format - a network/pathway format used for instance in cytoscape.' \n",
      "Data term  'Sequence-profile alignment format' has  def 'Data format for a sequence-profile alignment.' \n",
      "Data term  'Article format' has  def 'Data format for a full-text scientific article.' \n",
      "Data term  'Text mining report format' has  def 'Data format of a report from text mining.' \n",
      "Data term  'Enzyme kinetics report format' has  def 'Data format for reports on enzyme kinetics.' \n",
      "Data term  'Chemical data format' has  def 'Format of a report on a chemical compound.' \n",
      "Data term  'cif' has  def 'Crystallographic Information File (CIF) is a data exchange standard file format for Crystallographic Information and related Structural Science data.' \n",
      "Data term  'Gene annotation format' has  def 'Format of a report on a particular locus, gene, gene system or groups of genes.' \n",
      "Data term  'Workflow format' has  def 'Format of a workflow.' \n",
      "Data term  'KNIME datatable format' has  def 'Data table formatted such that it can be passed/streamed within the KNIME platform.' \n",
      "Data term  'CWL' has  def 'Common Workflow Language (CWL) format for description of command-line tools and workflows.' \n",
      "Data term  'Tertiary structure format' has  def 'Data format for a molecular tertiary structure.' \n",
      "Data term  'PDB database entry format' has  def 'Format of an entry (or part of an entry) from the PDB database.' \n",
      "Data term  'Chemical formula format' has  def 'Text format of a chemical formula.' \n",
      "Data term  'Phylogenetic character data format' has  def 'Format of raw (unplotted) phylogenetic data.' \n",
      "Data term  'Phylogenetic continuous quantitative character format' has  def 'Format of phylogenetic continuous quantitative character data.' \n",
      "Data term  'Phylogenetic discrete states format' has  def 'Format of phylogenetic discrete states data.' \n",
      "Data term  'Phylogenetic tree report (cliques) format' has  def 'Format of phylogenetic cliques data.' \n",
      "Data term  'Phylogenetic tree report (invariants) format' has  def 'Format of phylogenetic invariants data.' \n",
      "Data term  'Phylogenetic tree report (tree distances) format' has  def 'Format for phylogenetic tree distance data.' \n",
      "Data term  'Protein family report format' has  def 'Format for reports on a protein family.' \n",
      "Data term  'Protein interaction format' has  def 'Format for molecular interaction data.' \n",
      "Data term  'Sequence assembly format' has  def 'Format for sequence assembly data.' \n",
      "Data term  'Sequence assembly format (text)' has  def 'Text format for sequence assembly data.' \n",
      "Data term  'Gene expression report format' has  def 'Format of a file of gene expression data, e.g. a gene expression matrix or profile.' \n",
      "Data term  'Map format' has  def 'Format of a map of (typically one) molecular sequence annotated with features.' \n",
      "Data term  'Nucleic acid features (primers) format' has  def 'Format of a report on PCR primers or hybridisation oligos in a nucleic acid sequence.' \n",
      "Data term  'Protein report format' has  def 'Format of a report of general information about a specific protein.' \n",
      "Data term  'Protein structure report (quality evaluation) format' has  def 'Format of a report on the quality of a protein three-dimensional model.' \n",
      "Data term  'Database hits (sequence) format' has  def 'Format of a report on sequence hits and associated data from searching a sequence database.' \n",
      "Data term  'Sequence distance matrix format' has  def 'Format of a matrix of genetic distances between molecular sequences.' \n",
      "Data term  'Sequence motif format' has  def 'Format of a sequence motif.' \n",
      "Data term  'Sequence profile format' has  def 'Format of a sequence profile.' \n",
      "Data term  'Hidden Markov model format' has  def 'Format of a hidden Markov model.' \n",
      "Data term  'Dirichlet distribution format' has  def 'Data format of a dirichlet distribution.' \n",
      "Data term  'HMM emission and transition counts format' has  def 'Data format for the emission and transition counts of a hidden Markov model.' \n",
      "Data term  'RNA secondary structure format' has  def 'Format for secondary structure (predicted or real) of an RNA molecule.' \n",
      "Data term  'Protein secondary structure format' has  def 'Format for secondary structure (predicted or real) of a protein molecule.' \n",
      "Data term  'Sequence range format' has  def 'Format used to specify range(s) of sequence positions.' \n",
      "Data term  'Sequence features (repeats) format' has  def 'Format used for map of repeats in molecular (typically nucleotide) sequences.' \n",
      "Data term  'Nucleic acid features (restriction sites) format' has  def 'Format used for report on restriction enzyme recognition sites in nucleotide sequences.' \n",
      "Data term  'Sequence cluster format' has  def 'Format used for clusters of molecular sequences.' \n",
      "Data term  'Sequence cluster format (protein)' has  def 'Format used for clusters of protein sequences.' \n",
      "Data term  'Sequence cluster format (nucleic acid)' has  def 'Format used for clusters of nucleotide sequences.' \n",
      "Data term  'Ontology format' has  def 'Format used for ontologies.' \n",
      "Data term  'OBO format' has  def 'A serialisation format conforming to the Open Biomedical Ontologies (OBO) model.' \n",
      "Data term  'OWL format' has  def 'A serialisation format conforming to the Web Ontology Language (OWL) model.' \n",
      "Data term  'RDF format' has  def 'A serialisation format conforming to the Resource Description Framework (RDF) model.' \n",
      "Data term  'JSON-LD' has  def 'JSON-LD, or JavaScript Object Notation for Linked Data, is a method of encoding Linked Data using JSON.' \n",
      "Data term  'Open Annotation format' has  def 'A format of text annotation using the linked-data Open Annotation Data Model, serialised typically in RDF or JSON-LD.' \n",
      "Data term  'Raw sequence format' has  def 'Format of a raw molecular sequence (i.e. the alphabet used).' \n",
      "Data term  'Bibliographic reference format' has  def 'Format of a bibliographic reference.' \n",
      "Data term  'Sequence variation annotation format' has  def 'Format of sequence variation annotation.' \n",
      "Data term  'Matrix format' has  def 'Format of a matrix (array) of numerical values.' \n",
      "Data term  'Amino acid index format' has  def 'Data format for an amino acid index.' \n",
      "Data term  '3D-1D scoring matrix format' has  def 'Format of a matrix of 3D-1D scores (amino acid environment probabilities).' \n",
      "Data term  'MAT' has  def 'Binary format used by MATLAB files to store workspace variables.' \n",
      "Data term  'Protein domain classification format' has  def 'Format of data concerning the classification of the sequences and/or structures of protein structural domain(s).' \n",
      "Data term  'Raw SCOP domain classification format' has  def 'Format of raw SCOP domain classification data files.' \n",
      "Data term  'Raw CATH domain classification format' has  def 'Format of raw CATH domain classification data files.' \n",
      "Data term  'CATH domain report format' has  def 'Format of summary of domain classification information for a CATH domain.' \n",
      "Data term  'Biological pathway or network report format' has  def 'Data format for a report of information derived from a biological pathway or network.' \n",
      "Data term  'Experiment annotation format' has  def 'Data format for annotation on a laboratory experiment.' \n",
      "Data term  'Microarray experiment data format' has  def 'Format for information about a microarray experimental per se (not the data generated from that experiment).' \n",
      "Data term  'Mass spectrometry data format' has  def 'Format for mass pectra and derived data, include peptide sequences etc.' \n",
      "Data term  'MGF' has  def 'Mascot Generic Format. Encodes multiple MS/MS spectra in a single file.' \n",
      "Data term  'dta' has  def 'Spectral data format file where each spectrum is written to a separate file.' \n",
      "Data term  'pkl' has  def 'Spectral data file similar to dta.' \n",
      "Data term  'mzXML' has  def 'Common file format for proteomics mass spectrometric data developed at the Seattle Proteome Center/Institute for Systems Biology.' \n",
      "Data term  'MSF' has  def 'Proprietary mass-spectrometry format of Thermo Scientific's ProteomeDiscoverer software.' \n",
      "Data term  'Individual genetic data format' has  def 'Data format for a metadata on an individual and their genetic data.' \n",
      "Data term  'Data index format' has  def 'Format of a data index of some type.' \n",
      "Data term  'Document format' has  def 'Format of documents including word processor, spreadsheet and presentation.' \n",
      "Data term  'Image format' has  def 'Format used for images and image metadata.' \n",
      "Data term  'Vega' has  def 'Vega is a visualization grammar, a declarative language for creating, saving, and sharing interactive visualization designs. With Vega, you can describe the visual appearance and interactive behavior of a visualization in a JSON format, and generate web-based views using Canvas or SVG.' \n",
      "Data term  'Vega-lite' has  def 'Vega-Lite is a high-level grammar of interactive graphics. It provides a concise JSON syntax for rapidly generating visualizations to support analysis. Vega-Lite specifications can be compiled to Vega specifications.' \n",
      "Data term  'Sequence quality report format (text)' has  def 'Textual report format for sequence quality for reports from sequencing machines.' \n",
      "Data term  'Graph format' has  def 'Data format for graph data.' \n",
      "Data term  'xgmml' has  def 'XML-based format used to store graph descriptions within Galaxy.' \n",
      "Data term  'Biodiversity data format' has  def 'Data format for biodiversity data.' \n",
      "Data term  'ABCD format' has  def 'Exchange format of the Access to Biological Collections Data (ABCD) Schema; a standard for the access to and exchange of data about specimens and observations (primary biodiversity data).' \n",
      "Data term  'Linked data format' has  def 'A linked data format enables publishing structured data as linked data (Linked Data), so that the data can be interlinked and become more useful through semantic queries.' \n",
      "Data term  'Annotated text format' has  def 'Data format of an annotated text, e.g. with recognised entities, concepts, and relations.' \n",
      "Data term  'PubAnnotation format' has  def 'JSON format of annotated scientific text used by PubAnnotations and other tools.' \n",
      "Data term  'Query language' has  def 'A query language (format) for structured database queries.' \n",
      "Data term  'NMR data format' has  def 'Data format for raw data from a nuclear magnetic resonance (NMR) spectroscopy experiment.' \n",
      "Data term  'Raw microarray data format' has  def 'Data format for raw microarray data.' \n",
      "Data term  'NLP format' has  def 'Data format used in Natural Language Processing.' \n",
      "Data term  'NLP annotation format' has  def 'An NLP format used for annotated textual documents.' \n",
      "Data term  'NLP corpus format' has  def 'NLP format used by a specific type of corpus (collection of texts).' \n",
      "Data term  'RNA annotation format' has  def 'A \"placeholder\" concept for formats of annotated RNA data, including e.g. microRNA and RNA-Seq data.' \n",
      "Data term  'Trajectory format' has  def 'File format to store trajectory information for a 3D structure .' \n",
      "Data term  'Trajectory format (binary)' has  def 'Binary file format to store trajectory information for a 3D structure .' \n",
      "Data term  'Trajectory format (text)' has  def 'Textual file format to store trajectory information for a 3D structure .' \n",
      "Data term  'Topology format' has  def 'Format of topology files; containing the static information of a structure molecular system that is needed for a molecular simulation.' \n",
      "Data term  'FF parameter format' has  def 'Format of force field parameter files, which store the set of parameters (charges, masses, radii, bond lengths, bond dihedrals, etc.) that are essential for the proper description and simulation of a molecular system.' \n",
      "Data term  'JSON' has  def 'JavaScript Object Notation format; a lightweight, text-based format to represent tree-structured data using key-value pairs.' \n",
      "Data term  'BioSimulators format for the specifications of biosimulation tools' has  def 'Format for describing the capabilities of a biosimulation tool including the modeling frameworks, simulation algorithms, and modeling formats that it supports, as well as metadata such as a list of the interfaces, programming languages, and operating systems supported by the tool; a link to download the tool; a list of the authors of the tool; and the license to the tool.' \n",
      "Data term  'YAML' has  def 'YAML (YAML Ain't Markup Language) is a human-readable tree-structured data serialisation language.' \n"
     ]
    }
   ],
   "source": [
    "## Query for format with no definition (ERROR level)\n",
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "\n",
    "SELECT ?term ?concept ?def WHERE {\n",
    "    ?concept rdfs:subClassOf+ edam:format_1915 ;\n",
    "                rdfs:label ?term ;\n",
    "                \n",
    "         oboInOwl:hasDefinition ?def .\n",
    "\n",
    "}\n",
    "\"\"\"\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Format term  '{r['term']}' has  def '{r['def']}' \") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-29T15:42:02.030054Z",
     "start_time": "2024-02-29T15:42:01.964824Z"
    }
   },
   "execution_count": 42
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ID hybrid  'http://edamontology.org/data_1036'  'TIGR identifier' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2112'  'FlyBase primary identifier' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2174'  'FlyBase secondary identifier' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_1154'  'KEGG object identifier' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2102'  'KEGG organism code' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_1891'  'iHOP symbol' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2104'  'BioCyc ID' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_1157'  'Pathway ID (BioCyc)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2105'  'Compound ID (BioCyc)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2106'  'Reaction ID (BioCyc)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2107'  'Enzyme ID (BioCyc)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2113'  'WormBase identifier' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_1091'  'WormBase name' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_1092'  'WormBase class' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2367'  'ASTD ID' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2368'  'ASTD ID (exon)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2369'  'ASTD ID (intron)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2370'  'ASTD ID (polya)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2371'  'ASTD ID (tss)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2380'  'CABRI accession' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2387'  'TAIR accession' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2388'  'TAIR accession (At gene)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_1033'  'Ensembl gene ID' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2398'  'Ensembl protein ID' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2725'  'Ensembl transcript ID' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_3270'  'Ensembl gene tree ID' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_1901'  'Locus ID (SGD)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2639'  'PubChem ID' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2917'  'ConsensusPathDB identifier' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2345'  'Pathway ID (ConsensusPathDB)' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2776'  'ConsensusPathDB entity ID' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_2777'  'ConsensusPathDB entity name' has  regex 'None' \n",
      "ID hybrid  'http://edamontology.org/data_3274'  'MGI accession' has  regex 'None' \n"
     ]
    }
   ],
   "source": [
    "## Query for hybrid ID with no regex (WARNING level) (does this apply to all sorts of IDs?)\n",
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "\n",
    "SELECT ?term ?concept ?regex WHERE {\n",
    "    ?concept rdfs:subClassOf+ edam:data_2109 ;\n",
    "            rdfs:label ?term .\n",
    "                \n",
    "    FILTER NOT EXISTS {               \n",
    "        ?concept edam:regex ?regex \n",
    "        } .\n",
    "\n",
    "}\n",
    "\"\"\"\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"ID hybrid  '{r['concept']}'  '{r['term']}' has  regex '{r['regex']}' \") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-29T16:39:48.208557Z",
     "start_time": "2024-02-29T16:39:48.157318Z"
    }
   },
   "execution_count": 57
  },
  {
   "cell_type": "markdown",
   "source": [
    "## Queries from Caseologue"
   ],
   "metadata": {
    "collapsed": false
   }
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "prop 'http://www.w3.org/2000/01/rdf-schema#seeAlso' has value: 'https://en.wikipedia.org/wiki/Genomics'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym' has value: 'Whole genomes'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym' has value: 'Genome annotation'.\n",
      "prop 'http://www.w3.org/2000/01/rdf-schema#subClassOf' has value: 'http://edamontology.org/topic_3391'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#inSubset' has value: 'http://edamontology.org/topics'.\n",
      "prop 'http://edamontology.org/created_in' has value: 'beta12orEarlier'.\n",
      "prop 'http://www.w3.org/2000/01/rdf-schema#seeAlso' has value: 'http://purl.bioontology.org/ontology/MSH/D023281'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#inSubset' has value: 'http://edamontology.org/bio'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym' has value: 'Exomes'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasDefinition' has value: 'Whole genomes of one or more organisms, or genomes in general, such as meta-information on genomes, genome projects, gene names etc.'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym' has value: 'Synthetic genomics'.\n",
      "prop 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type' has value: 'http://www.w3.org/2002/07/owl#Class'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym' has value: 'Personal genomics'.\n",
      "prop 'http://www.w3.org/2000/01/rdf-schema#label' has value: 'Genomics'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym' has value: 'Genomes'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym' has value: 'Viral genomics'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#hasHumanReadableId' has value: 'Genomics'.\n",
      "prop 'http://www.geneontology.org/formats/oboInOwl#inSubset' has value: 'http://edamontology.org/events'.\n",
      "prop 'http://edamontology.org/isdebtag' has value: 'true'.\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "\n",
    "SELECT * WHERE {\n",
    "    ?x ?property ?value .\n",
    "    VALUES ?x {edam:topic_0622}\n",
    "}\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"prop '{r['property']}' has value: '{r['value']}'.\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-01-30T16:27:50.432853Z",
     "start_time": "2024-01-30T16:27:49.431765Z"
    }
   },
   "execution_count": 10
  },
  {
   "cell_type": "markdown",
   "source": [],
   "metadata": {
    "collapsed": false
   }
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "\n",
    "\n",
    "SELECT ?term ?def WHERE {\n",
    "    ?c rdfs:subClassOf+ edam:topic_0003 ;\n",
    "       rdfs:label ?term ;\n",
    "    OPTIONAL {?c oboInOwl:hasDefinition ?def} .\n",
    "\n",
    "    FILTER NOT EXISTS {\n",
    "        #?c rdfs:seeAlso [] .\n",
    "        ?c oboInOwl:hasDefinition [] .\n",
    "    } .\n",
    "}\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Topic '{r['term']}' has  def  '{r['def']}'.\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-01-30T16:27:50.482206Z",
     "start_time": "2024-01-30T16:27:50.434865Z"
    }
   },
   "execution_count": 11
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Topic 'Ontology and terminology' has  no exactSynonym.\n",
      "Topic 'Bioinformatics' has  no exactSynonym.\n",
      "Topic 'Laboratory information management' has  no exactSynonym.\n",
      "Topic 'Chemometrics' has  no exactSynonym.\n",
      "Topic 'Database management' has  no exactSynonym.\n",
      "Topic 'Data management' has  no exactSynonym.\n",
      "Topic 'Data submission, annotation, and curation' has  no exactSynonym.\n",
      "Topic 'Data identity and mapping' has  no exactSynonym.\n",
      "Topic 'Data architecture, analysis and design' has  no exactSynonym.\n",
      "Topic 'Data integration and warehousing' has  no exactSynonym.\n",
      "Topic 'Data governance' has  no exactSynonym.\n",
      "Topic 'Data quality management' has  no exactSynonym.\n",
      "Topic 'Data rescue' has  no exactSynonym.\n",
      "Topic 'Chemistry' has  no exactSynonym.\n",
      "Topic 'Microfluidics' has  no exactSynonym.\n",
      "Topic 'Computational chemistry' has  no exactSynonym.\n",
      "Topic 'Drug discovery' has  no exactSynonym.\n",
      "Topic 'Compound libraries and screening' has  no exactSynonym.\n",
      "Topic 'Analytical chemistry' has  no exactSynonym.\n",
      "Topic 'Synthetic chemistry' has  no exactSynonym.\n",
      "Topic 'Chemical biology' has  no exactSynonym.\n",
      "Topic 'Carbon cycle' has  no exactSynonym.\n",
      "Topic 'Statistics and probability' has  no exactSynonym.\n",
      "Topic 'Applied mathematics' has  no exactSynonym.\n",
      "Topic 'Pure mathematics' has  no exactSynonym.\n",
      "Topic 'Computer science' has  no exactSynonym.\n",
      "Topic 'Data mining' has  no exactSynonym.\n",
      "Topic 'Machine learning' has  no exactSynonym.\n",
      "Topic 'Physics' has  no exactSynonym.\n",
      "Topic 'Biophysics' has  no exactSynonym.\n",
      "Topic 'Acoustics' has  no exactSynonym.\n",
      "Topic 'Genome resequencing' has  no exactSynonym.\n",
      "Topic 'Single-Cell Sequencing' has  no exactSynonym.\n",
      "Topic 'Imaging' has  no exactSynonym.\n",
      "Topic 'Electron microscopy' has  no exactSynonym.\n",
      "Topic 'Medical imaging' has  no exactSynonym.\n",
      "Topic 'Light microscopy' has  no exactSynonym.\n",
      "Topic 'Genotyping experiment' has  no exactSynonym.\n",
      "Topic 'Proteomics experiment' has  no exactSynonym.\n",
      "Topic 'RNAi experiment' has  no exactSynonym.\n",
      "Topic 'Simulation experiment' has  no exactSynonym.\n",
      "Topic 'Cytometry' has  no exactSynonym.\n",
      "Topic 'Protein interaction experiment' has  no exactSynonym.\n",
      "Topic 'Preclinical and clinical studies' has  no exactSynonym.\n",
      "Topic 'Animal study' has  no exactSynonym.\n",
      "Topic 'Environmental sciences' has  no exactSynonym.\n",
      "Topic 'Ecology' has  no exactSynonym.\n",
      "Topic 'Biodiversity' has  no exactSynonym.\n",
      "Topic 'Metagenomics' has  no exactSynonym.\n",
      "Topic 'Metabarcoding' has  no exactSynonym.\n",
      "Topic 'Open science' has  no exactSynonym.\n",
      "Topic 'Virology' has  no exactSynonym.\n",
      "Topic 'Structural biology' has  no exactSynonym.\n",
      "Topic 'Structural genomics' has  no exactSynonym.\n",
      "Topic 'Cell biology' has  no exactSynonym.\n",
      "Topic 'Systems biology' has  no exactSynonym.\n",
      "Topic 'Molecular biology' has  no exactSynonym.\n",
      "Topic 'Genetics' has  no exactSynonym.\n",
      "Topic 'Quantitative genetics' has  no exactSynonym.\n",
      "Topic 'Phenomics' has  no exactSynonym.\n",
      "Topic 'Molecular evolution' has  no exactSynonym.\n",
      "Topic 'Population genetics' has  no exactSynonym.\n",
      "Topic 'Epigenetics' has  no exactSynonym.\n",
      "Topic 'Epigenomics' has  no exactSynonym.\n",
      "Topic 'Molecular genetics' has  no exactSynonym.\n",
      "Topic 'Functional, regulatory and non-coding RNA' has  no exactSynonym.\n",
      "Topic 'Mobile genetic elements' has  no exactSynonym.\n",
      "Topic 'Gene transcripts' has  no exactSynonym.\n",
      "Topic 'DNA mutation' has  no exactSynonym.\n",
      "Topic 'DNA polymorphism' has  no exactSynonym.\n",
      "Topic 'Copy number variation' has  no exactSynonym.\n",
      "Topic 'Gene regulation' has  no exactSynonym.\n",
      "Topic 'Transcription factors and regulatory sites' has  no exactSynonym.\n",
      "Topic 'Transcriptomics' has  no exactSynonym.\n",
      "Topic 'Metatranscriptomics' has  no exactSynonym.\n",
      "Topic 'Immunoproteins and antigens' has  no exactSynonym.\n",
      "Topic 'Cytogenetics' has  no exactSynonym.\n",
      "Topic 'Human genetics' has  no exactSynonym.\n",
      "Topic 'Developmental biology' has  no exactSynonym.\n",
      "Topic 'Embryology' has  no exactSynonym.\n",
      "Topic 'Biotechnology' has  no exactSynonym.\n",
      "Topic 'Biomaterials' has  no exactSynonym.\n",
      "Topic 'Medical biotechnology' has  no exactSynonym.\n",
      "Topic 'Synthetic biology' has  no exactSynonym.\n",
      "Topic 'Metabolic engineering' has  no exactSynonym.\n",
      "Topic 'Phylogeny' has  no exactSynonym.\n",
      "Topic 'Phylogenetics' has  no exactSynonym.\n",
      "Topic 'Cladistics' has  no exactSynonym.\n",
      "Topic 'Taxonomy' has  no exactSynonym.\n",
      "Topic 'Microbiology' has  no exactSynonym.\n",
      "Topic 'Marine biology' has  no exactSynonym.\n",
      "Topic 'Freshwater biology' has  no exactSynonym.\n",
      "Topic 'Agricultural science' has  no exactSynonym.\n",
      "Topic 'Rare diseases' has  no exactSynonym.\n",
      "Topic 'Toxicology' has  no exactSynonym.\n",
      "Topic 'Physiology' has  no exactSynonym.\n",
      "Topic 'Public health and epidemiology' has  no exactSynonym.\n",
      "Topic 'Neurology' has  no exactSynonym.\n",
      "Topic 'Translational medicine' has  no exactSynonym.\n",
      "Topic 'Molecular medicine' has  no exactSynonym.\n",
      "Topic 'Systems medicine' has  no exactSynonym.\n",
      "Topic 'Veterinary medicine' has  no exactSynonym.\n",
      "Topic 'Allergy, clinical immunology and immunotherapeutics' has  no exactSynonym.\n",
      "Topic 'Dermatology' has  no exactSynonym.\n",
      "Topic 'Dentistry' has  no exactSynonym.\n",
      "Topic 'Endocrinology and metabolism' has  no exactSynonym.\n",
      "Topic 'Haematology' has  no exactSynonym.\n",
      "Topic 'Gastroenterology' has  no exactSynonym.\n",
      "Topic 'Gender medicine' has  no exactSynonym.\n",
      "Topic 'Gynaecology and obstetrics' has  no exactSynonym.\n",
      "Topic 'Medical toxicology' has  no exactSynonym.\n",
      "Topic 'Musculoskeletal medicine' has  no exactSynonym.\n",
      "Topic 'Ophthalmology' has  no exactSynonym.\n",
      "Topic 'Psychiatry' has  no exactSynonym.\n",
      "Topic 'Reproductive health' has  no exactSynonym.\n",
      "Topic 'Surgery' has  no exactSynonym.\n",
      "Topic 'Urology and nephrology' has  no exactSynonym.\n",
      "Topic 'Complementary medicine' has  no exactSynonym.\n",
      "Topic 'Tropical medicine' has  no exactSynonym.\n",
      "Topic 'Computational biology' has  no exactSynonym.\n",
      "Topic 'RNA' has  no exactSynonym.\n",
      "Topic 'DNA binding sites' has  no exactSynonym.\n",
      "Topic 'DNA replication and recombination' has  no exactSynonym.\n",
      "Topic 'DNA packaging' has  no exactSynonym.\n",
      "Topic 'Nucleic acid sites, features and motifs' has  no exactSynonym.\n",
      "Topic 'Protein expression' has  no exactSynonym.\n",
      "Topic 'Protein targeting and localisation' has  no exactSynonym.\n",
      "Topic 'Protein variants' has  no exactSynonym.\n",
      "Topic 'Protein interactions' has  no exactSynonym.\n",
      "Topic 'Membrane and lipoproteins' has  no exactSynonym.\n",
      "Topic 'Protein folding, stability and design' has  no exactSynonym.\n",
      "Topic 'Protein folds and structural domains' has  no exactSynonym.\n",
      "Topic 'Protein sites, features and motifs' has  no exactSynonym.\n",
      "Topic 'Protein binding sites' has  no exactSynonym.\n",
      "Topic 'Sequence analysis' has  no exactSynonym.\n",
      "Topic 'Mapping' has  no exactSynonym.\n",
      "Topic 'Sequence composition, complexity and repeats' has  no exactSynonym.\n",
      "Topic 'Phylogenomics' has  no exactSynonym.\n",
      "Topic 'Sequence assembly' has  no exactSynonym.\n",
      "Topic 'Probes and primers' has  no exactSynonym.\n",
      "Topic 'Structure prediction' has  no exactSynonym.\n",
      "Topic 'Molecular dynamics' has  no exactSynonym.\n",
      "Topic 'Molecular modelling' has  no exactSynonym.\n",
      "Topic 'Carbohydrates' has  no exactSynonym.\n",
      "Topic 'Small molecules' has  no exactSynonym.\n",
      "Topic 'Sequence sites, features and motifs' has  no exactSynonym.\n",
      "Topic 'Molecular interactions, pathways and networks' has  no exactSynonym.\n",
      "Topic 'Functional genomics' has  no exactSynonym.\n",
      "Topic 'Biomolecular simulation' has  no exactSynonym.\n",
      "Topic 'Pharmacology' has  no exactSynonym.\n",
      "Topic 'Pharmacogenomics' has  no exactSynonym.\n",
      "Topic 'Immunology' has  no exactSynonym.\n",
      "Topic 'Anatomy' has  no exactSynonym.\n",
      "Topic 'Microbial collection' has  no exactSynonym.\n",
      "Topic 'Cell culture collection' has  no exactSynonym.\n",
      "Topic 'Clone library' has  no exactSynonym.\n",
      "Topic 'Parasitology' has  no exactSynonym.\n",
      "Topic 'Neurobiology' has  no exactSynonym.\n",
      "Topic 'Biotherapeutics' has  no exactSynonym.\n",
      "Topic 'Drug metabolism' has  no exactSynonym.\n",
      "Topic 'Pharmacovigilance' has  no exactSynonym.\n",
      "Topic 'Vaccinology' has  no exactSynonym.\n",
      "Topic 'Omics' has  no exactSynonym.\n",
      "Topic 'Proteomics' has  no exactSynonym.\n",
      "Topic 'Genomics' has  no exactSynonym.\n",
      "Topic 'Comparative genomics' has  no exactSynonym.\n",
      "Topic 'Population genomics' has  no exactSynonym.\n",
      "Topic 'Proteogenomics' has  no exactSynonym.\n",
      "Topic 'Paleogenomics' has  no exactSynonym.\n",
      "Topic 'Metabolomics' has  no exactSynonym.\n",
      "Topic 'Fluxomics' has  no exactSynonym.\n",
      "Topic 'Immunomics' has  no exactSynonym.\n"
     ]
    }
   ],
   "source": [
    "query = \"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "\n",
    "\n",
    "SELECT ?term ?syno WHERE {\n",
    "    ?c rdfs:subClassOf+ edam:topic_0003 ;\n",
    "       rdfs:label ?term ;\n",
    "    OPTIONAL {?c oboInOwl:hasExactSynonym ?syno} .\n",
    "\n",
    "    FILTER NOT EXISTS {\n",
    "        ?c oboInOwl:hasExactSynonym [] .\n",
    "    } .\n",
    "}\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "#for r in results :\n",
    "    #print(f\"Topic '{r['term']}' has  no exactSynonym.\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-01-30T16:27:50.544521Z",
     "start_time": "2024-01-30T16:27:50.539560Z"
    }
   },
   "execution_count": 12
  },
  {
   "cell_type": "markdown",
   "source": [
    "Queries from Caseologue"
   ],
   "metadata": {
    "collapsed": false
   }
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "PREFIX obo: <http://purl.obolibrary.org/obo/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT DISTINCT ?entity ?label WHERE {\n",
    "    \n",
    "    ?entity a owl:Class .\n",
    "    ?entity rdfs:label ?label .\n",
    "    FILTER NOT EXISTS { ?entity owl:deprecated true }\n",
    "                 \n",
    "    FILTER NOT EXISTS {\n",
    "        FILTER REGEX(str(?entity), \"^http://edamontology.org/(data|topic|operation|format)_[0-9]{4}$\")\n",
    "    }\n",
    "    FILTER ( ?entity != <http://www.geneontology.org/formats/oboInOwl#ObsoleteClass>)\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T16:49:01.045260Z",
     "start_time": "2024-02-13T16:49:00.721194Z"
    }
   },
   "execution_count": 11
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'http://edamontology.org/topic_3068' - Label 'Literature and language'\n",
      "Entity 'http://edamontology.org/topic_0219' - Label 'Data submission, annotation, and curation'\n",
      "Entity 'http://edamontology.org/topic_3345' - Label 'Data identity and mapping'\n",
      "Entity 'http://edamontology.org/topic_3923' - Label 'Genome resequencing'\n",
      "Entity 'http://edamontology.org/topic_3524' - Label 'Simulation experiment'\n",
      "Entity 'http://edamontology.org/topic_3511' - Label 'Nucleic acid sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_0123' - Label 'Protein properties'\n",
      "Entity 'http://edamontology.org/topic_3510' - Label 'Protein sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_0157' - Label 'Sequence composition, complexity and repeats'\n",
      "Entity 'http://edamontology.org/topic_0632' - Label 'Probes and primers'\n",
      "Entity 'http://edamontology.org/topic_0160' - Label 'Sequence sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3892' - Label 'Biomolecular simulation'\n",
      "Entity 'http://edamontology.org/topic_3374' - Label 'Biotherapeutics'\n",
      "Entity 'http://edamontology.org/topic_3393' - Label 'Quality affairs'\n"
     ]
    }
   ],
   "source": [
    "#Query aim = “check for missing wikipedia link in topic ”\n",
    "\n",
    "query=\"\"\"\n",
    "PREFIX obo: <http://purl.obolibrary.org/obo/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT DISTINCT ?entity ?label ?seealso WHERE {\n",
    "    ?entity rdfs:subClassOf+ <http://edamontology.org/topic_0003> .\n",
    "    ?entity rdfs:label ?label .\n",
    "    \n",
    "    FILTER NOT EXISTS {\n",
    "        ?entity rdfs:seeAlso ?seealso .\n",
    "        FILTER (regex(str(?seealso), \"wikipedia\", \"i\")) .\n",
    "    }\n",
    "}\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T16:50:37.407410Z",
     "start_time": "2024-02-13T16:50:37.369446Z"
    }
   },
   "execution_count": 12
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'http://edamontology.org/data_0005' - Label 'Resource type'\n",
      "Entity 'http://edamontology.org/data_0007' - Label 'Tool'\n",
      "Entity 'http://edamontology.org/data_0581' - Label 'Database'\n",
      "Entity 'http://edamontology.org/data_0583' - Label 'Directory metadata'\n",
      "Entity 'http://edamontology.org/data_0831' - Label 'MeSH vocabulary'\n",
      "Entity 'http://edamontology.org/data_0832' - Label 'HGNC vocabulary'\n",
      "Entity 'http://edamontology.org/data_0835' - Label 'UMLS vocabulary'\n",
      "Entity 'http://edamontology.org/data_0843' - Label 'Database entry'\n",
      "Entity 'http://edamontology.org/data_0848' - Label 'Raw sequence'\n",
      "Entity 'http://edamontology.org/data_0851' - Label 'Sequence mask character'\n",
      "Entity 'http://edamontology.org/data_0852' - Label 'Sequence mask type'\n",
      "Entity 'http://edamontology.org/data_0853' - Label 'DNA sense specification'\n",
      "Entity 'http://edamontology.org/data_0854' - Label 'Sequence length specification'\n",
      "Entity 'http://edamontology.org/data_0855' - Label 'Sequence metadata'\n",
      "Entity 'http://edamontology.org/data_0859' - Label 'Sequence signature model'\n",
      "Entity 'http://edamontology.org/data_0861' - Label 'Sequence alignment (words)'\n",
      "Entity 'http://edamontology.org/data_0864' - Label 'Sequence alignment parameter'\n",
      "Entity 'http://edamontology.org/data_0866' - Label 'Sequence alignment metadata'\n",
      "Entity 'http://edamontology.org/data_0868' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0869' - Label 'Sequence-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0875' - Label 'Protein topology'\n",
      "Entity 'http://edamontology.org/data_0876' - Label 'Protein features report (secondary structure)'\n",
      "Entity 'http://edamontology.org/data_0877' - Label 'Protein features report (super-secondary)'\n",
      "Entity 'http://edamontology.org/data_0879' - Label 'Secondary structure alignment metadata (protein)'\n",
      "Entity 'http://edamontology.org/data_0882' - Label 'Secondary structure alignment metadata (RNA)'\n",
      "Entity 'http://edamontology.org/data_0884' - Label 'Tertiary structure record'\n",
      "Entity 'http://edamontology.org/data_0885' - Label 'Structure database search results'\n",
      "Entity 'http://edamontology.org/data_0891' - Label 'Sequence-3D profile alignment'\n",
      "Entity 'http://edamontology.org/data_0894' - Label 'Amino acid annotation'\n",
      "Entity 'http://edamontology.org/data_0895' - Label 'Peptide annotation'\n",
      "Entity 'http://edamontology.org/data_0899' - Label 'Protein structural motifs and surfaces'\n",
      "Entity 'http://edamontology.org/data_0900' - Label 'Protein domain classification'\n",
      "Entity 'http://edamontology.org/data_0901' - Label 'Protein features report (domains)'\n",
      "Entity 'http://edamontology.org/data_0902' - Label 'Protein architecture report'\n",
      "Entity 'http://edamontology.org/data_0903' - Label 'Protein folding report'\n",
      "Entity 'http://edamontology.org/data_0904' - Label 'Protein features (mutation)'\n",
      "Entity 'http://edamontology.org/data_0911' - Label 'Nucleotide base annotation'\n",
      "Entity 'http://edamontology.org/data_0917' - Label 'Gene classification'\n",
      "Entity 'http://edamontology.org/data_0918' - Label 'DNA variation'\n",
      "Entity 'http://edamontology.org/data_0923' - Label 'PCR experiment report'\n",
      "Entity 'http://edamontology.org/data_0931' - Label 'Microarray experiment report'\n",
      "Entity 'http://edamontology.org/data_0932' - Label 'Oligonucleotide probe data'\n",
      "Entity 'http://edamontology.org/data_0933' - Label 'SAGE experimental data'\n",
      "Entity 'http://edamontology.org/data_0934' - Label 'MPSS experimental data'\n",
      "Entity 'http://edamontology.org/data_0935' - Label 'SBS experimental data'\n",
      "Entity 'http://edamontology.org/data_0936' - Label 'Sequence tag profile (with gene assignment)'\n",
      "Entity 'http://edamontology.org/data_0941' - Label 'Electron microscopy model'\n",
      "Entity 'http://edamontology.org/data_0946' - Label 'Pathway or network annotation'\n",
      "Entity 'http://edamontology.org/data_0947' - Label 'Biological pathway map'\n",
      "Entity 'http://edamontology.org/data_0948' - Label 'Data resource definition'\n",
      "Entity 'http://edamontology.org/data_0952' - Label 'EMBOSS database resource definition'\n",
      "Entity 'http://edamontology.org/data_0953' - Label 'Version information'\n",
      "Entity 'http://edamontology.org/data_0959' - Label 'Job metadata'\n",
      "Entity 'http://edamontology.org/data_0964' - Label 'Scent annotation'\n",
      "Entity 'http://edamontology.org/data_0974' - Label 'Entity identifier'\n",
      "Entity 'http://edamontology.org/data_0975' - Label 'Data resource identifier'\n",
      "Entity 'http://edamontology.org/data_0978' - Label 'Discrete entity identifier'\n",
      "Entity 'http://edamontology.org/data_0979' - Label 'Entity feature identifier'\n",
      "Entity 'http://edamontology.org/data_0980' - Label 'Entity collection identifier'\n",
      "Entity 'http://edamontology.org/data_0981' - Label 'Phenomenon identifier'\n",
      "Entity 'http://edamontology.org/data_0985' - Label 'Molecule type'\n",
      "Entity 'http://edamontology.org/data_0986' - Label 'Chemical identifier'\n",
      "Entity 'http://edamontology.org/data_0992' - Label 'Ligand identifier'\n",
      "Entity 'http://edamontology.org/data_1014' - Label 'Sequence position specification'\n",
      "Entity 'http://edamontology.org/data_1018' - Label 'Nucleic acid feature identifier'\n",
      "Entity 'http://edamontology.org/data_1019' - Label 'Protein feature identifier'\n",
      "Entity 'http://edamontology.org/data_1024' - Label 'Codon name'\n",
      "Entity 'http://edamontology.org/data_1028' - Label 'Gene identifier (NCBI RefSeq)'\n",
      "Entity 'http://edamontology.org/data_1029' - Label 'Gene identifier (NCBI UniGene)'\n",
      "Entity 'http://edamontology.org/data_1030' - Label 'Gene identifier (Entrez)'\n",
      "Entity 'http://edamontology.org/data_1057' - Label 'Sequence database name'\n",
      "Entity 'http://edamontology.org/data_1062' - Label 'Database entry identifier'\n",
      "Entity 'http://edamontology.org/data_1065' - Label 'Sequence signature identifier'\n",
      "Entity 'http://edamontology.org/data_1067' - Label 'Phylogenetic distance matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1094' - Label 'Sequence type'\n",
      "Entity 'http://edamontology.org/data_1099' - Label 'UniProt accession (extended)'\n",
      "Entity 'http://edamontology.org/data_1101' - Label 'TREMBL accession'\n",
      "Entity 'http://edamontology.org/data_1110' - Label 'EMBOSS sequence type'\n",
      "Entity 'http://edamontology.org/data_1111' - Label 'EMBOSS listfile'\n",
      "Entity 'http://edamontology.org/data_1120' - Label 'Sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1121' - Label 'BLAST sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1122' - Label 'Phylogenetic tree type'\n",
      "Entity 'http://edamontology.org/data_1125' - Label 'Comparison matrix type'\n",
      "Entity 'http://edamontology.org/data_1152' - Label 'HIVDB identifier'\n",
      "Entity 'http://edamontology.org/data_1156' - Label 'Pathway ID (aMAZE)'\n",
      "Entity 'http://edamontology.org/data_1236' - Label 'Psiblast checkpoint file'\n",
      "Entity 'http://edamontology.org/data_1237' - Label 'HMMER synthetic sequences set'\n",
      "Entity 'http://edamontology.org/data_1241' - Label 'vectorstrip cloning vector definition file'\n",
      "Entity 'http://edamontology.org/data_1242' - Label 'Primer3 internal oligo mishybridizing library'\n",
      "Entity 'http://edamontology.org/data_1243' - Label 'Primer3 mispriming library file'\n",
      "Entity 'http://edamontology.org/data_1244' - Label 'primersearch primer pairs sequence record'\n",
      "Entity 'http://edamontology.org/data_1250' - Label 'Word size'\n",
      "Entity 'http://edamontology.org/data_1251' - Label 'Window size'\n",
      "Entity 'http://edamontology.org/data_1252' - Label 'Sequence length range'\n",
      "Entity 'http://edamontology.org/data_1253' - Label 'Sequence information report'\n",
      "Entity 'http://edamontology.org/data_1256' - Label 'Sequence features (comparative)'\n",
      "Entity 'http://edamontology.org/data_1257' - Label 'Sequence property (protein)'\n",
      "Entity 'http://edamontology.org/data_1258' - Label 'Sequence property (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1264' - Label 'Sequence composition table'\n",
      "Entity 'http://edamontology.org/data_1269' - Label 'DAS sequence feature annotation'\n",
      "Entity 'http://edamontology.org/data_1281' - Label 'Sequence signature map'\n",
      "Entity 'http://edamontology.org/data_1290' - Label 'InterPro compact match image'\n",
      "Entity 'http://edamontology.org/data_1291' - Label 'InterPro detailed match image'\n",
      "Entity 'http://edamontology.org/data_1292' - Label 'InterPro architecture image'\n",
      "Entity 'http://edamontology.org/data_1293' - Label 'SMART protein schematic'\n",
      "Entity 'http://edamontology.org/data_1294' - Label 'GlobPlot domain image'\n",
      "Entity 'http://edamontology.org/data_1298' - Label 'Sequence motif matches'\n",
      "Entity 'http://edamontology.org/data_1299' - Label 'Sequence features (repeats)'\n",
      "Entity 'http://edamontology.org/data_1300' - Label 'Gene and transcript structure (report)'\n",
      "Entity 'http://edamontology.org/data_1301' - Label 'Mobile genetic elements'\n",
      "Entity 'http://edamontology.org/data_1303' - Label 'Nucleic acid features (quadruplexes)'\n",
      "Entity 'http://edamontology.org/data_1306' - Label 'Nucleosome exclusion sequences'\n",
      "Entity 'http://edamontology.org/data_1309' - Label 'Gene features (exonic splicing enhancer)'\n",
      "Entity 'http://edamontology.org/data_1310' - Label 'Nucleic acid features (microRNA)'\n",
      "Entity 'http://edamontology.org/data_1313' - Label 'Coding region'\n",
      "Entity 'http://edamontology.org/data_1314' - Label 'Gene features (SECIS element)'\n",
      "Entity 'http://edamontology.org/data_1315' - Label 'Transcription factor binding sites'\n",
      "Entity 'http://edamontology.org/data_1321' - Label 'Protein features (sites)'\n",
      "Entity 'http://edamontology.org/data_1322' - Label 'Protein features report (signal peptides)'\n",
      "Entity 'http://edamontology.org/data_1323' - Label 'Protein features report (cleavage sites)'\n",
      "Entity 'http://edamontology.org/data_1324' - Label 'Protein features (post-translation modifications)'\n",
      "Entity 'http://edamontology.org/data_1325' - Label 'Protein features report (active sites)'\n",
      "Entity 'http://edamontology.org/data_1326' - Label 'Protein features report (binding sites)'\n",
      "Entity 'http://edamontology.org/data_1327' - Label 'Protein features (epitopes)'\n",
      "Entity 'http://edamontology.org/data_1328' - Label 'Protein features report (nucleic acid binding sites)'\n",
      "Entity 'http://edamontology.org/data_1329' - Label 'MHC Class I epitopes report'\n",
      "Entity 'http://edamontology.org/data_1330' - Label 'MHC Class II epitopes report'\n",
      "Entity 'http://edamontology.org/data_1331' - Label 'Protein features (PEST sites)'\n",
      "Entity 'http://edamontology.org/data_1338' - Label 'Sequence database hits scores list'\n",
      "Entity 'http://edamontology.org/data_1339' - Label 'Sequence database hits alignments list'\n",
      "Entity 'http://edamontology.org/data_1340' - Label 'Sequence database hits evaluation data'\n",
      "Entity 'http://edamontology.org/data_1344' - Label 'MEME motif alphabet'\n",
      "Entity 'http://edamontology.org/data_1345' - Label 'MEME background frequencies file'\n",
      "Entity 'http://edamontology.org/data_1346' - Label 'MEME motifs directive file'\n",
      "Entity 'http://edamontology.org/data_1348' - Label 'HMM emission and transition counts'\n",
      "Entity 'http://edamontology.org/data_1358' - Label 'Prosite nucleotide pattern'\n",
      "Entity 'http://edamontology.org/data_1359' - Label 'Prosite protein pattern'\n",
      "Entity 'http://edamontology.org/data_1368' - Label 'Domainatrix signature'\n",
      "Entity 'http://edamontology.org/data_1371' - Label 'HMMER NULL hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_1372' - Label 'Protein family signature'\n",
      "Entity 'http://edamontology.org/data_1373' - Label 'Protein domain signature'\n",
      "Entity 'http://edamontology.org/data_1374' - Label 'Protein region signature'\n",
      "Entity 'http://edamontology.org/data_1375' - Label 'Protein repeat signature'\n",
      "Entity 'http://edamontology.org/data_1376' - Label 'Protein site signature'\n",
      "Entity 'http://edamontology.org/data_1377' - Label 'Protein conserved site signature'\n",
      "Entity 'http://edamontology.org/data_1378' - Label 'Protein active site signature'\n",
      "Entity 'http://edamontology.org/data_1379' - Label 'Protein binding site signature'\n",
      "Entity 'http://edamontology.org/data_1380' - Label 'Protein post-translational modification signature'\n",
      "Entity 'http://edamontology.org/data_1382' - Label 'Sequence alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1386' - Label 'Sequence alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1387' - Label 'Sequence alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1388' - Label 'Hybrid sequence alignment (pair)'\n",
      "Entity 'http://edamontology.org/data_1389' - Label 'Multiple nucleotide sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1390' - Label 'Multiple protein sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1395' - Label 'Score end gaps control'\n",
      "Entity 'http://edamontology.org/data_1396' - Label 'Aligned sequence order'\n",
      "Entity 'http://edamontology.org/data_1400' - Label 'Terminal gap penalty'\n",
      "Entity 'http://edamontology.org/data_1404' - Label 'Gap opening penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1405' - Label 'Gap opening penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1406' - Label 'Gap extension penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1407' - Label 'Gap extension penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1408' - Label 'Gap separation penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1409' - Label 'Gap separation penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1414' - Label 'Sequence alignment metadata (quality report)'\n",
      "Entity 'http://edamontology.org/data_1415' - Label 'Sequence alignment report (site conservation)'\n",
      "Entity 'http://edamontology.org/data_1416' - Label 'Sequence alignment report (site correlation)'\n",
      "Entity 'http://edamontology.org/data_1417' - Label 'Sequence-profile alignment (Domainatrix signature)'\n",
      "Entity 'http://edamontology.org/data_1418' - Label 'Sequence-profile alignment (HMM)'\n",
      "Entity 'http://edamontology.org/data_1420' - Label 'Sequence-profile alignment (fingerprint)'\n",
      "Entity 'http://edamontology.org/data_1438' - Label 'Phylogenetic report'\n",
      "Entity 'http://edamontology.org/data_1440' - Label 'Phylogenetic tree report (tree shape)'\n",
      "Entity 'http://edamontology.org/data_1441' - Label 'Phylogenetic tree report (tree evaluation)'\n",
      "Entity 'http://edamontology.org/data_1443' - Label 'Phylogenetic tree report (tree stratigraphic)'\n",
      "Entity 'http://edamontology.org/data_1446' - Label 'Comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1447' - Label 'Comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1450' - Label 'Nucleotide comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1451' - Label 'Nucleotide comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1452' - Label 'Amino acid comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1453' - Label 'Amino acid comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1469' - Label 'Protein structure (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1471' - Label 'Protein chain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1472' - Label 'Protein chain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1473' - Label 'Protein domain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1474' - Label 'Protein domain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1480' - Label 'Structure alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1483' - Label 'Structure alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1484' - Label 'Multiple protein tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1485' - Label 'Structure alignment (protein all atoms)'\n",
      "Entity 'http://edamontology.org/data_1486' - Label 'Structure alignment (protein C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1487' - Label 'Pairwise protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1488' - Label 'Pairwise protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1489' - Label 'Multiple protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1490' - Label 'Multiple protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1491' - Label 'Structure alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1492' - Label 'Multiple nucleic acid tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1495' - Label 'DaliLite hit table'\n",
      "Entity 'http://edamontology.org/data_1496' - Label 'Molecular similarity score'\n",
      "Entity 'http://edamontology.org/data_1509' - Label 'Enzyme report'\n",
      "Entity 'http://edamontology.org/data_1517' - Label 'Restriction enzyme report'\n",
      "Entity 'http://edamontology.org/data_1533' - Label 'Protein subcellular localisation'\n",
      "Entity 'http://edamontology.org/data_1536' - Label 'MHC peptide immunogenicity report'\n",
      "Entity 'http://edamontology.org/data_1540' - Label 'Protein non-covalent interactions report'\n",
      "Entity 'http://edamontology.org/data_1541' - Label 'Protein flexibility or motion report'\n",
      "Entity 'http://edamontology.org/data_1543' - Label 'Protein surface report'\n",
      "Entity 'http://edamontology.org/data_1550' - Label 'Protein non-canonical interactions'\n",
      "Entity 'http://edamontology.org/data_1553' - Label 'CATH node'\n",
      "Entity 'http://edamontology.org/data_1554' - Label 'SCOP node'\n",
      "Entity 'http://edamontology.org/data_1555' - Label 'EMBASSY domain classification'\n",
      "Entity 'http://edamontology.org/data_1556' - Label 'CATH class'\n",
      "Entity 'http://edamontology.org/data_1557' - Label 'CATH architecture'\n",
      "Entity 'http://edamontology.org/data_1558' - Label 'CATH topology'\n",
      "Entity 'http://edamontology.org/data_1559' - Label 'CATH homologous superfamily'\n",
      "Entity 'http://edamontology.org/data_1560' - Label 'CATH structurally similar group'\n",
      "Entity 'http://edamontology.org/data_1561' - Label 'CATH functional category'\n",
      "Entity 'http://edamontology.org/data_1564' - Label 'Protein fold recognition report'\n",
      "Entity 'http://edamontology.org/data_1565' - Label 'Protein-protein interaction report'\n",
      "Entity 'http://edamontology.org/data_1567' - Label 'Protein-nucleic acid interactions report'\n",
      "Entity 'http://edamontology.org/data_1586' - Label 'Nucleic acid melting temperature'\n",
      "Entity 'http://edamontology.org/data_1587' - Label 'Nucleic acid stitch profile'\n",
      "Entity 'http://edamontology.org/data_1591' - Label 'Vienna RNA parameters'\n",
      "Entity 'http://edamontology.org/data_1592' - Label 'Vienna RNA structure constraints'\n",
      "Entity 'http://edamontology.org/data_1593' - Label 'Vienna RNA concentration data'\n",
      "Entity 'http://edamontology.org/data_1594' - Label 'Vienna RNA calculated energy'\n",
      "Entity 'http://edamontology.org/data_1599' - Label 'Codon adaptation index'\n",
      "Entity 'http://edamontology.org/data_1601' - Label 'Nc statistic'\n",
      "Entity 'http://edamontology.org/data_1634' - Label 'Linkage disequilibrium (report)'\n",
      "Entity 'http://edamontology.org/data_1642' - Label 'Affymetrix probe sets library file'\n",
      "Entity 'http://edamontology.org/data_1643' - Label 'Affymetrix probe sets information library file'\n",
      "Entity 'http://edamontology.org/data_1646' - Label 'Molecular weights standard fingerprint'\n",
      "Entity 'http://edamontology.org/data_1656' - Label 'Metabolic pathway report'\n",
      "Entity 'http://edamontology.org/data_1657' - Label 'Genetic information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1658' - Label 'Environmental information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1659' - Label 'Signal transduction pathway report'\n",
      "Entity 'http://edamontology.org/data_1660' - Label 'Cellular process pathways report'\n",
      "Entity 'http://edamontology.org/data_1661' - Label 'Disease pathway or network report'\n",
      "Entity 'http://edamontology.org/data_1662' - Label 'Drug structure relationship map'\n",
      "Entity 'http://edamontology.org/data_1663' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/data_1664' - Label 'MIRIAM datatype'\n",
      "Entity 'http://edamontology.org/data_1670' - Label 'Database version information'\n",
      "Entity 'http://edamontology.org/data_1671' - Label 'Tool version information'\n",
      "Entity 'http://edamontology.org/data_1672' - Label 'CATH version information'\n",
      "Entity 'http://edamontology.org/data_1673' - Label 'Swiss-Prot to PDB mapping'\n",
      "Entity 'http://edamontology.org/data_1674' - Label 'Sequence database cross-references'\n",
      "Entity 'http://edamontology.org/data_1675' - Label 'Job status'\n",
      "Entity 'http://edamontology.org/data_1676' - Label 'Job ID'\n",
      "Entity 'http://edamontology.org/data_1677' - Label 'Job type'\n",
      "Entity 'http://edamontology.org/data_1678' - Label 'Tool log'\n",
      "Entity 'http://edamontology.org/data_1679' - Label 'DaliLite log file'\n",
      "Entity 'http://edamontology.org/data_1680' - Label 'STRIDE log file'\n",
      "Entity 'http://edamontology.org/data_1681' - Label 'NACCESS log file'\n",
      "Entity 'http://edamontology.org/data_1682' - Label 'EMBOSS wordfinder log file'\n",
      "Entity 'http://edamontology.org/data_1683' - Label 'EMBOSS domainatrix log file'\n",
      "Entity 'http://edamontology.org/data_1684' - Label 'EMBOSS sites log file'\n",
      "Entity 'http://edamontology.org/data_1685' - Label 'EMBOSS supermatcher error file'\n",
      "Entity 'http://edamontology.org/data_1686' - Label 'EMBOSS megamerger log file'\n",
      "Entity 'http://edamontology.org/data_1687' - Label 'EMBOSS whichdb log file'\n",
      "Entity 'http://edamontology.org/data_1688' - Label 'EMBOSS vectorstrip log file'\n",
      "Entity 'http://edamontology.org/data_1693' - Label 'Number of iterations'\n",
      "Entity 'http://edamontology.org/data_1694' - Label 'Number of output entities'\n",
      "Entity 'http://edamontology.org/data_1695' - Label 'Hit sort order'\n",
      "Entity 'http://edamontology.org/data_1715' - Label 'BioPax term'\n",
      "Entity 'http://edamontology.org/data_1716' - Label 'GO'\n",
      "Entity 'http://edamontology.org/data_1717' - Label 'MeSH'\n",
      "Entity 'http://edamontology.org/data_1718' - Label 'HGNC'\n",
      "Entity 'http://edamontology.org/data_1719' - Label 'NCBI taxonomy vocabulary'\n",
      "Entity 'http://edamontology.org/data_1720' - Label 'Plant ontology term'\n",
      "Entity 'http://edamontology.org/data_1721' - Label 'UMLS'\n",
      "Entity 'http://edamontology.org/data_1722' - Label 'FMA'\n",
      "Entity 'http://edamontology.org/data_1723' - Label 'EMAP'\n",
      "Entity 'http://edamontology.org/data_1724' - Label 'ChEBI'\n",
      "Entity 'http://edamontology.org/data_1725' - Label 'MGED'\n",
      "Entity 'http://edamontology.org/data_1726' - Label 'myGrid'\n",
      "Entity 'http://edamontology.org/data_1727' - Label 'GO (biological process)'\n",
      "Entity 'http://edamontology.org/data_1728' - Label 'GO (molecular function)'\n",
      "Entity 'http://edamontology.org/data_1729' - Label 'GO (cellular component)'\n",
      "Entity 'http://edamontology.org/data_1730' - Label 'Ontology relation type'\n",
      "Entity 'http://edamontology.org/data_1732' - Label 'Ontology concept comment'\n",
      "Entity 'http://edamontology.org/data_1733' - Label 'Ontology concept reference'\n",
      "Entity 'http://edamontology.org/data_1738' - Label 'doc2loc document information'\n",
      "Entity 'http://edamontology.org/data_1744' - Label 'Atomic x coordinate'\n",
      "Entity 'http://edamontology.org/data_1745' - Label 'Atomic y coordinate'\n",
      "Entity 'http://edamontology.org/data_1746' - Label 'Atomic z coordinate'\n",
      "Entity 'http://edamontology.org/data_1762' - Label 'CATH domain report'\n",
      "Entity 'http://edamontology.org/data_1764' - Label 'CATH representative domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1765' - Label 'CATH representative domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1766' - Label 'CATH domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1767' - Label 'CATH domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1776' - Label 'Protein report (function)'\n",
      "Entity 'http://edamontology.org/data_1783' - Label 'Gene name (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1784' - Label 'Gene name (CGD)'\n",
      "Entity 'http://edamontology.org/data_1785' - Label 'Gene name (dictyBase)'\n",
      "Entity 'http://edamontology.org/data_1786' - Label 'Gene name (EcoGene primary)'\n",
      "Entity 'http://edamontology.org/data_1787' - Label 'Gene name (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1788' - Label 'Gene name (SGD)'\n",
      "Entity 'http://edamontology.org/data_1789' - Label 'Gene name (TGD)'\n",
      "Entity 'http://edamontology.org/data_1790' - Label 'Gene name (CGSC)'\n",
      "Entity 'http://edamontology.org/data_1791' - Label 'Gene name (HGNC)'\n",
      "Entity 'http://edamontology.org/data_1792' - Label 'Gene name (MGD)'\n",
      "Entity 'http://edamontology.org/data_1793' - Label 'Gene name (Bacillus subtilis)'\n",
      "Entity 'http://edamontology.org/data_1797' - Label 'Gene ID (GeneDB Glossina morsitans)'\n",
      "Entity 'http://edamontology.org/data_1798' - Label 'Gene ID (GeneDB Leishmania major)'\n",
      "Entity 'http://edamontology.org/data_1799' - Label 'Gene ID (GeneDB Plasmodium falciparum)'\n",
      "Entity 'http://edamontology.org/data_1800' - Label 'Gene ID (GeneDB Schizosaccharomyces pombe)'\n",
      "Entity 'http://edamontology.org/data_1801' - Label 'Gene ID (GeneDB Trypanosoma brucei)'\n",
      "Entity 'http://edamontology.org/data_1806' - Label 'Gene synonym'\n",
      "Entity 'http://edamontology.org/data_1852' - Label 'Sequence assembly component'\n",
      "Entity 'http://edamontology.org/data_1853' - Label 'Chromosome annotation (aberration)'\n",
      "Entity 'http://edamontology.org/data_1864' - Label 'Map set data'\n",
      "Entity 'http://edamontology.org/data_1865' - Label 'Map feature'\n",
      "Entity 'http://edamontology.org/data_1866' - Label 'Map type'\n",
      "Entity 'http://edamontology.org/data_1877' - Label 'Synonym'\n",
      "Entity 'http://edamontology.org/data_1878' - Label 'Misspelling'\n",
      "Entity 'http://edamontology.org/data_1879' - Label 'Acronym'\n",
      "Entity 'http://edamontology.org/data_1880' - Label 'Misnomer'\n",
      "Entity 'http://edamontology.org/data_1884' - Label 'UniProt keywords'\n",
      "Entity 'http://edamontology.org/data_1887' - Label 'Gene ID (MIPS Maize)'\n",
      "Entity 'http://edamontology.org/data_1888' - Label 'Gene ID (MIPS Medicago)'\n",
      "Entity 'http://edamontology.org/data_1889' - Label 'Gene name (DragonDB)'\n",
      "Entity 'http://edamontology.org/data_1890' - Label 'Gene name (Arabidopsis)'\n",
      "Entity 'http://edamontology.org/data_1892' - Label 'Gene name (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1906' - Label 'Quantitative trait locus'\n",
      "Entity 'http://edamontology.org/data_2009' - Label 'Ordered locus name'\n",
      "Entity 'http://edamontology.org/data_2018' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/data_2022' - Label 'Vienna RNA structural data'\n",
      "Entity 'http://edamontology.org/data_2023' - Label 'Sequence mask parameter'\n",
      "Entity 'http://edamontology.org/data_2028' - Label 'Experimental data'\n",
      "Entity 'http://edamontology.org/data_2041' - Label 'Genome version information'\n",
      "Entity 'http://edamontology.org/data_2043' - Label 'Sequence record lite'\n",
      "Entity 'http://edamontology.org/data_2046' - Label 'Nucleic acid sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2047' - Label 'Protein sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2053' - Label 'Structural data'\n",
      "Entity 'http://edamontology.org/data_2079' - Label 'Search parameter'\n",
      "Entity 'http://edamontology.org/data_2081' - Label 'Secondary structure'\n",
      "Entity 'http://edamontology.org/data_2083' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2086' - Label 'Nucleic acid structure data'\n",
      "Entity 'http://edamontology.org/data_2090' - Label 'Database entry version information'\n",
      "Entity 'http://edamontology.org/data_2092' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/data_2100' - Label 'Type'\n",
      "Entity 'http://edamontology.org/data_2103' - Label 'Gene name (KEGG GENES)'\n",
      "Entity 'http://edamontology.org/data_2116' - Label 'Nucleic acid features (codon)'\n",
      "Entity 'http://edamontology.org/data_2126' - Label 'Translation frame specification'\n",
      "Entity 'http://edamontology.org/data_2130' - Label 'Sequence profile type'\n",
      "Entity 'http://edamontology.org/data_2132' - Label 'Mutation type'\n",
      "Entity 'http://edamontology.org/data_2134' - Label 'Results sort order'\n",
      "Entity 'http://edamontology.org/data_2135' - Label 'Toggle'\n",
      "Entity 'http://edamontology.org/data_2136' - Label 'Sequence width'\n",
      "Entity 'http://edamontology.org/data_2141' - Label 'Window step size'\n",
      "Entity 'http://edamontology.org/data_2142' - Label 'EMBOSS graph'\n",
      "Entity 'http://edamontology.org/data_2143' - Label 'EMBOSS report'\n",
      "Entity 'http://edamontology.org/data_2145' - Label 'Sequence offset'\n",
      "Entity 'http://edamontology.org/data_2146' - Label 'Threshold'\n",
      "Entity 'http://edamontology.org/data_2147' - Label 'Protein report (transcription factor)'\n",
      "Entity 'http://edamontology.org/data_2149' - Label 'Database category name'\n",
      "Entity 'http://edamontology.org/data_2150' - Label 'Sequence profile name'\n",
      "Entity 'http://edamontology.org/data_2151' - Label 'Color'\n",
      "Entity 'http://edamontology.org/data_2152' - Label 'Rendering parameter'\n",
      "Entity 'http://edamontology.org/data_2156' - Label 'Date'\n",
      "Entity 'http://edamontology.org/data_2157' - Label 'Word composition'\n",
      "Entity 'http://edamontology.org/data_2164' - Label 'Protein sequence properties plot'\n",
      "Entity 'http://edamontology.org/data_2169' - Label 'Nucleic acid features (siRNA)'\n",
      "Entity 'http://edamontology.org/data_2173' - Label 'Sequence set (stream)'\n",
      "Entity 'http://edamontology.org/data_2176' - Label 'Cardinality'\n",
      "Entity 'http://edamontology.org/data_2177' - Label 'Exactly 1'\n",
      "Entity 'http://edamontology.org/data_2178' - Label '1 or more'\n",
      "Entity 'http://edamontology.org/data_2179' - Label 'Exactly 2'\n",
      "Entity 'http://edamontology.org/data_2180' - Label '2 or more'\n",
      "Entity 'http://edamontology.org/data_2191' - Label 'Protein features report (chemical modifications)'\n",
      "Entity 'http://edamontology.org/data_2192' - Label 'Error'\n",
      "Entity 'http://edamontology.org/data_2198' - Label 'Gene cluster'\n",
      "Entity 'http://edamontology.org/data_2201' - Label 'Sequence record full'\n",
      "Entity 'http://edamontology.org/data_2212' - Label 'Mutation annotation (basic)'\n",
      "Entity 'http://edamontology.org/data_2213' - Label 'Mutation annotation (prevalence)'\n",
      "Entity 'http://edamontology.org/data_2214' - Label 'Mutation annotation (prognostic)'\n",
      "Entity 'http://edamontology.org/data_2215' - Label 'Mutation annotation (functional)'\n",
      "Entity 'http://edamontology.org/data_2217' - Label 'Tumor annotation'\n",
      "Entity 'http://edamontology.org/data_2218' - Label 'Server metadata'\n",
      "Entity 'http://edamontology.org/data_2235' - Label 'Raw SCOP domain classification'\n",
      "Entity 'http://edamontology.org/data_2236' - Label 'Raw CATH domain classification'\n",
      "Entity 'http://edamontology.org/data_2240' - Label 'Heterogen annotation'\n",
      "Entity 'http://edamontology.org/data_2242' - Label 'Phylogenetic property values'\n",
      "Entity 'http://edamontology.org/data_2245' - Label 'Sequence set (bootstrapped)'\n",
      "Entity 'http://edamontology.org/data_2247' - Label 'Phylogenetic consensus tree'\n",
      "Entity 'http://edamontology.org/data_2248' - Label 'Schema'\n",
      "Entity 'http://edamontology.org/data_2249' - Label 'DTD'\n",
      "Entity 'http://edamontology.org/data_2250' - Label 'XML Schema'\n",
      "Entity 'http://edamontology.org/data_2251' - Label 'Relax-NG schema'\n",
      "Entity 'http://edamontology.org/data_2252' - Label 'XSLT stylesheet'\n",
      "Entity 'http://edamontology.org/data_2288' - Label 'Sequence identifier (protein)'\n",
      "Entity 'http://edamontology.org/data_2289' - Label 'Sequence identifier (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_2296' - Label 'Gene name (AceView)'\n",
      "Entity 'http://edamontology.org/data_2300' - Label 'Gene name (NCBI)'\n",
      "Entity 'http://edamontology.org/data_2307' - Label 'Virus annotation'\n",
      "Entity 'http://edamontology.org/data_2308' - Label 'Virus annotation (taxonomy)'\n",
      "Entity 'http://edamontology.org/data_2336' - Label 'Translation phase specification'\n",
      "Entity 'http://edamontology.org/data_2357' - Label 'Protein signature type'\n",
      "Entity 'http://edamontology.org/data_2358' - Label 'Domain-nucleic acid interaction report'\n",
      "Entity 'http://edamontology.org/data_2359' - Label 'Domain-domain interactions'\n",
      "Entity 'http://edamontology.org/data_2360' - Label 'Domain-domain interaction (indirect)'\n",
      "Entity 'http://edamontology.org/data_2363' - Label '2D PAGE data'\n",
      "Entity 'http://edamontology.org/data_2364' - Label '2D PAGE report'\n",
      "Entity 'http://edamontology.org/data_2372' - Label '2D PAGE spot report'\n",
      "Entity 'http://edamontology.org/data_2378' - Label 'Protein-motif interaction'\n",
      "Entity 'http://edamontology.org/data_2381' - Label 'Experiment report (genotyping)'\n",
      "Entity 'http://edamontology.org/data_2395' - Label 'Fungi annotation'\n",
      "Entity 'http://edamontology.org/data_2396' - Label 'Fungi annotation (anamorph)'\n",
      "Entity 'http://edamontology.org/data_2400' - Label 'Toxin annotation'\n",
      "Entity 'http://edamontology.org/data_2401' - Label 'Protein report (membrane protein)'\n",
      "Entity 'http://edamontology.org/data_2402' - Label 'Protein-drug interaction report'\n",
      "Entity 'http://edamontology.org/data_2522' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2524' - Label 'Protein data'\n",
      "Entity 'http://edamontology.org/data_2525' - Label 'Nucleic acid data'\n",
      "Entity 'http://edamontology.org/data_2527' - Label 'Parameter'\n",
      "Entity 'http://edamontology.org/data_2528' - Label 'Molecular data'\n",
      "Entity 'http://edamontology.org/data_2529' - Label 'Molecule report'\n",
      "Entity 'http://edamontology.org/data_2539' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2540' - Label 'Data index data'\n",
      "Entity 'http://edamontology.org/data_2579' - Label 'Expressed gene list'\n",
      "Entity 'http://edamontology.org/data_2581' - Label 'GO concept name'\n",
      "Entity 'http://edamontology.org/data_2584' - Label 'GO concept name (cellular component)'\n",
      "Entity 'http://edamontology.org/data_2590' - Label 'Hierarchy identifier'\n",
      "Entity 'http://edamontology.org/data_2592' - Label 'Cancer type'\n",
      "Entity 'http://edamontology.org/data_2598' - Label 'Secondary structure alignment metadata'\n",
      "Entity 'http://edamontology.org/data_2599' - Label 'Molecule interaction report'\n",
      "Entity 'http://edamontology.org/data_2601' - Label 'Small molecule data'\n",
      "Entity 'http://edamontology.org/data_2602' - Label 'Genotype and phenotype data'\n",
      "Entity 'http://edamontology.org/data_2627' - Label 'Molecular interaction ID'\n",
      "Entity 'http://edamontology.org/data_2671' - Label 'Ensembl ID (Homo sapiens)'\n",
      "Entity 'http://edamontology.org/data_2672' - Label 'Ensembl ID ('Bos taurus')'\n",
      "Entity 'http://edamontology.org/data_2673' - Label 'Ensembl ID ('Canis familiaris')'\n",
      "Entity 'http://edamontology.org/data_2674' - Label 'Ensembl ID ('Cavia porcellus')'\n",
      "Entity 'http://edamontology.org/data_2675' - Label 'Ensembl ID ('Ciona intestinalis')'\n",
      "Entity 'http://edamontology.org/data_2676' - Label 'Ensembl ID ('Ciona savignyi')'\n",
      "Entity 'http://edamontology.org/data_2677' - Label 'Ensembl ID ('Danio rerio')'\n",
      "Entity 'http://edamontology.org/data_2678' - Label 'Ensembl ID ('Dasypus novemcinctus')'\n",
      "Entity 'http://edamontology.org/data_2679' - Label 'Ensembl ID ('Echinops telfairi')'\n",
      "Entity 'http://edamontology.org/data_2680' - Label 'Ensembl ID ('Erinaceus europaeus')'\n",
      "Entity 'http://edamontology.org/data_2681' - Label 'Ensembl ID ('Felis catus')'\n",
      "Entity 'http://edamontology.org/data_2682' - Label 'Ensembl ID ('Gallus gallus')'\n",
      "Entity 'http://edamontology.org/data_2683' - Label 'Ensembl ID ('Gasterosteus aculeatus')'\n",
      "Entity 'http://edamontology.org/data_2684' - Label 'Ensembl ID ('Homo sapiens')'\n",
      "Entity 'http://edamontology.org/data_2685' - Label 'Ensembl ID ('Loxodonta africana')'\n",
      "Entity 'http://edamontology.org/data_2686' - Label 'Ensembl ID ('Macaca mulatta')'\n",
      "Entity 'http://edamontology.org/data_2687' - Label 'Ensembl ID ('Monodelphis domestica')'\n",
      "Entity 'http://edamontology.org/data_2688' - Label 'Ensembl ID ('Mus musculus')'\n",
      "Entity 'http://edamontology.org/data_2689' - Label 'Ensembl ID ('Myotis lucifugus')'\n",
      "Entity 'http://edamontology.org/data_2690' - Label 'Ensembl ID (\"Ornithorhynchus anatinus\")'\n",
      "Entity 'http://edamontology.org/data_2691' - Label 'Ensembl ID ('Oryctolagus cuniculus')'\n",
      "Entity 'http://edamontology.org/data_2692' - Label 'Ensembl ID ('Oryzias latipes')'\n",
      "Entity 'http://edamontology.org/data_2693' - Label 'Ensembl ID ('Otolemur garnettii')'\n",
      "Entity 'http://edamontology.org/data_2694' - Label 'Ensembl ID ('Pan troglodytes')'\n",
      "Entity 'http://edamontology.org/data_2695' - Label 'Ensembl ID ('Rattus norvegicus')'\n",
      "Entity 'http://edamontology.org/data_2696' - Label 'Ensembl ID ('Spermophilus tridecemlineatus')'\n",
      "Entity 'http://edamontology.org/data_2697' - Label 'Ensembl ID ('Takifugu rubripes')'\n",
      "Entity 'http://edamontology.org/data_2698' - Label 'Ensembl ID ('Tupaia belangeri')'\n",
      "Entity 'http://edamontology.org/data_2699' - Label 'Ensembl ID ('Xenopus tropicalis')'\n",
      "Entity 'http://edamontology.org/data_2722' - Label 'Protein features report (disordered structure)'\n",
      "Entity 'http://edamontology.org/data_2724' - Label 'Embryo report'\n",
      "Entity 'http://edamontology.org/data_2726' - Label 'Inhibitor annotation'\n",
      "Entity 'http://edamontology.org/data_2733' - Label 'Genus name (virus)'\n",
      "Entity 'http://edamontology.org/data_2734' - Label 'Family name (virus)'\n",
      "Entity 'http://edamontology.org/data_2735' - Label 'Database name (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2740' - Label 'Gene name (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2743' - Label 'Gene name (HUGO)'\n",
      "Entity 'http://edamontology.org/data_2747' - Label 'Database name (CMD)'\n",
      "Entity 'http://edamontology.org/data_2748' - Label 'Database name (Osteogenesis)'\n",
      "Entity 'http://edamontology.org/data_2751' - Label 'GenomeReviews ID'\n",
      "Entity 'http://edamontology.org/data_2763' - Label 'Locus annotation'\n",
      "Entity 'http://edamontology.org/data_2765' - Label 'Term ID list'\n",
      "Entity 'http://edamontology.org/data_2767' - Label 'Identifier with metadata'\n",
      "Entity 'http://edamontology.org/data_2768' - Label 'Gene symbol annotation'\n",
      "Entity 'http://edamontology.org/data_2788' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2831' - Label 'Databank'\n",
      "Entity 'http://edamontology.org/data_2832' - Label 'Web portal'\n",
      "Entity 'http://edamontology.org/data_2838' - Label 'Experimental data (proteomics)'\n",
      "Entity 'http://edamontology.org/data_2857' - Label 'Article metadata'\n",
      "Entity 'http://edamontology.org/data_2866' - Label 'Northern blot report'\n",
      "Entity 'http://edamontology.org/data_2874' - Label 'Sequence set (polymorphic)'\n",
      "Entity 'http://edamontology.org/data_2875' - Label 'DRCAT resource'\n",
      "Entity 'http://edamontology.org/data_2880' - Label 'Secondary structure image'\n",
      "Entity 'http://edamontology.org/data_2881' - Label 'Secondary structure report'\n",
      "Entity 'http://edamontology.org/data_2882' - Label 'DNA features'\n",
      "Entity 'http://edamontology.org/data_2883' - Label 'RNA features report'\n",
      "Entity 'http://edamontology.org/data_2888' - Label 'Protein sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2889' - Label 'Nucleic acid sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2913' - Label 'Virus identifier'\n",
      "Entity 'http://edamontology.org/data_2925' - Label 'Sequence data'\n",
      "Entity 'http://edamontology.org/data_2927' - Label 'Codon usage'\n",
      "Entity 'http://edamontology.org/data_2954' - Label 'Article report'\n",
      "Entity 'http://edamontology.org/data_2958' - Label 'Nucleic acid melting curve'\n",
      "Entity 'http://edamontology.org/data_2959' - Label 'Nucleic acid probability profile'\n",
      "Entity 'http://edamontology.org/data_2960' - Label 'Nucleic acid temperature profile'\n",
      "Entity 'http://edamontology.org/data_2961' - Label 'Gene regulatory network report'\n",
      "Entity 'http://edamontology.org/data_2965' - Label '2D PAGE gel report'\n",
      "Entity 'http://edamontology.org/data_2966' - Label 'Oligonucleotide probe sets annotation'\n",
      "Entity 'http://edamontology.org/data_2967' - Label 'Microarray image'\n",
      "Entity 'http://edamontology.org/data_2971' - Label 'Workflow data'\n",
      "Entity 'http://edamontology.org/data_2972' - Label 'Workflow'\n",
      "Entity 'http://edamontology.org/data_2973' - Label 'Secondary structure data'\n",
      "Entity 'http://edamontology.org/data_2974' - Label 'Protein sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2975' - Label 'Nucleic acid sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2980' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/data_2981' - Label 'Sequence motif data'\n",
      "Entity 'http://edamontology.org/data_2982' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2983' - Label 'Pathway or network data'\n",
      "Entity 'http://edamontology.org/data_2986' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/data_2987' - Label 'Classification report'\n",
      "Entity 'http://edamontology.org/data_2989' - Label 'Protein features report (key folding sites)'\n",
      "Entity 'http://edamontology.org/data_3026' - Label 'GO concept name (biological process)'\n",
      "Entity 'http://edamontology.org/data_3027' - Label 'GO concept name (molecular function)'\n",
      "Entity 'http://edamontology.org/data_3031' - Label 'Core data'\n",
      "Entity 'http://edamontology.org/data_3085' - Label 'Protein sequence composition'\n",
      "Entity 'http://edamontology.org/data_3086' - Label 'Nucleic acid sequence composition (report)'\n",
      "Entity 'http://edamontology.org/data_3101' - Label 'Protein domain classification node'\n",
      "Entity 'http://edamontology.org/data_3102' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_3105' - Label 'Geotemporal metadata'\n",
      "Entity 'http://edamontology.org/data_3107' - Label 'Sequence feature name'\n",
      "Entity 'http://edamontology.org/data_3116' - Label 'Microarray protocol annotation'\n",
      "Entity 'http://edamontology.org/data_3119' - Label 'Sequence features (compositionally-biased regions)'\n",
      "Entity 'http://edamontology.org/data_3122' - Label 'Nucleic acid features (difference and change)'\n",
      "Entity 'http://edamontology.org/data_3129' - Label 'Protein features report (repeats)'\n",
      "Entity 'http://edamontology.org/data_3130' - Label 'Sequence motif matches (protein)'\n",
      "Entity 'http://edamontology.org/data_3131' - Label 'Sequence motif matches (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_3132' - Label 'Nucleic acid features (d-loop)'\n",
      "Entity 'http://edamontology.org/data_3133' - Label 'Nucleic acid features (stem loop)'\n",
      "Entity 'http://edamontology.org/data_3137' - Label 'Non-coding RNA'\n",
      "Entity 'http://edamontology.org/data_3138' - Label 'Transcriptional features (report)'\n",
      "Entity 'http://edamontology.org/data_3140' - Label 'Nucleic acid features (immunoglobulin gene structure)'\n",
      "Entity 'http://edamontology.org/data_3141' - Label 'SCOP class'\n",
      "Entity 'http://edamontology.org/data_3142' - Label 'SCOP fold'\n",
      "Entity 'http://edamontology.org/data_3143' - Label 'SCOP superfamily'\n",
      "Entity 'http://edamontology.org/data_3144' - Label 'SCOP family'\n",
      "Entity 'http://edamontology.org/data_3145' - Label 'SCOP protein'\n",
      "Entity 'http://edamontology.org/data_3146' - Label 'SCOP species'\n",
      "Entity 'http://edamontology.org/data_3147' - Label 'Mass spectrometry experiment'\n",
      "Entity 'http://edamontology.org/data_3154' - Label 'Protein alignment'\n",
      "Entity 'http://edamontology.org/data_3165' - Label 'NGS experiment'\n",
      "Entity 'http://edamontology.org/data_3231' - Label 'GWAS report'\n",
      "Entity 'http://edamontology.org/data_3268' - Label 'Sequence feature type'\n",
      "Entity 'http://edamontology.org/data_3269' - Label 'Gene homology (report)'\n",
      "Entity 'http://edamontology.org/data_3356' - Label 'Hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_3426' - Label 'Proteomics experiment report'\n",
      "Entity 'http://edamontology.org/data_3427' - Label 'RNAi report'\n",
      "Entity 'http://edamontology.org/data_3428' - Label 'Simulation experiment report'\n",
      "Entity 'http://edamontology.org/data_3490' - Label 'Chemical structure sketch'\n",
      "Entity 'http://edamontology.org/data_3496' - Label 'RNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3497' - Label 'DNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/format_1228' - Label 'UniGene entry format'\n",
      "Entity 'http://edamontology.org/format_1247' - Label 'COG sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_1431' - Label 'Phylogenetic property values format'\n",
      "Entity 'http://edamontology.org/format_1500' - Label 'Domainatrix 3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_1511' - Label 'IntEnz enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1512' - Label 'BRENDA enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1513' - Label 'KEGG REACTION enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1514' - Label 'KEGG ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1515' - Label 'REBASE proto enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1516' - Label 'REBASE withrefm enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1563' - Label 'SMART domain assignment report format'\n",
      "Entity 'http://edamontology.org/format_1568' - Label 'BIND entry format'\n",
      "Entity 'http://edamontology.org/format_1569' - Label 'IntAct entry format'\n",
      "Entity 'http://edamontology.org/format_1570' - Label 'InterPro entry format'\n",
      "Entity 'http://edamontology.org/format_1571' - Label 'InterPro entry abstract format'\n",
      "Entity 'http://edamontology.org/format_1572' - Label 'Gene3D entry format'\n",
      "Entity 'http://edamontology.org/format_1573' - Label 'PIRSF entry format'\n",
      "Entity 'http://edamontology.org/format_1574' - Label 'PRINTS entry format'\n",
      "Entity 'http://edamontology.org/format_1575' - Label 'Panther Families and HMMs entry format'\n",
      "Entity 'http://edamontology.org/format_1576' - Label 'Pfam entry format'\n",
      "Entity 'http://edamontology.org/format_1577' - Label 'SMART entry format'\n",
      "Entity 'http://edamontology.org/format_1578' - Label 'Superfamily entry format'\n",
      "Entity 'http://edamontology.org/format_1579' - Label 'TIGRFam entry format'\n",
      "Entity 'http://edamontology.org/format_1580' - Label 'ProDom entry format'\n",
      "Entity 'http://edamontology.org/format_1581' - Label 'FSSP entry format'\n",
      "Entity 'http://edamontology.org/format_1603' - Label 'Ensembl gene report format'\n",
      "Entity 'http://edamontology.org/format_1604' - Label 'DictyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1605' - Label 'CGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1606' - Label 'DragonDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1607' - Label 'EcoCyc gene report format'\n",
      "Entity 'http://edamontology.org/format_1608' - Label 'FlyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1609' - Label 'Gramene gene report format'\n",
      "Entity 'http://edamontology.org/format_1610' - Label 'KEGG GENES gene report format'\n",
      "Entity 'http://edamontology.org/format_1611' - Label 'MaizeGDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1612' - Label 'MGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1613' - Label 'RGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1614' - Label 'SGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1615' - Label 'GeneDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1616' - Label 'TAIR gene report format'\n",
      "Entity 'http://edamontology.org/format_1617' - Label 'WormBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1618' - Label 'ZFIN gene report format'\n",
      "Entity 'http://edamontology.org/format_1619' - Label 'TIGR gene report format'\n",
      "Entity 'http://edamontology.org/format_1620' - Label 'dbSNP polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_1623' - Label 'OMIM entry format'\n",
      "Entity 'http://edamontology.org/format_1624' - Label 'HGVbase entry format'\n",
      "Entity 'http://edamontology.org/format_1625' - Label 'HIVDB entry format'\n",
      "Entity 'http://edamontology.org/format_1626' - Label 'KEGG DISEASE entry format'\n",
      "Entity 'http://edamontology.org/format_1640' - Label 'ArrayExpress entry format'\n",
      "Entity 'http://edamontology.org/format_1645' - Label 'EMDB entry format'\n",
      "Entity 'http://edamontology.org/format_1647' - Label 'KEGG PATHWAY entry format'\n",
      "Entity 'http://edamontology.org/format_1648' - Label 'MetaCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1649' - Label 'HumanCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1650' - Label 'INOH entry format'\n",
      "Entity 'http://edamontology.org/format_1651' - Label 'PATIKA entry format'\n",
      "Entity 'http://edamontology.org/format_1652' - Label 'Reactome entry format'\n",
      "Entity 'http://edamontology.org/format_1653' - Label 'aMAZE entry format'\n",
      "Entity 'http://edamontology.org/format_1654' - Label 'CPDB entry format'\n",
      "Entity 'http://edamontology.org/format_1655' - Label 'Panther Pathways entry format'\n",
      "Entity 'http://edamontology.org/format_1666' - Label 'BioModel mathematical model format'\n",
      "Entity 'http://edamontology.org/format_1697' - Label 'KEGG LIGAND entry format'\n",
      "Entity 'http://edamontology.org/format_1698' - Label 'KEGG COMPOUND entry format'\n",
      "Entity 'http://edamontology.org/format_1699' - Label 'KEGG PLANT entry format'\n",
      "Entity 'http://edamontology.org/format_1700' - Label 'KEGG GLYCAN entry format'\n",
      "Entity 'http://edamontology.org/format_1701' - Label 'PubChem entry format'\n",
      "Entity 'http://edamontology.org/format_1702' - Label 'ChemSpider entry format'\n",
      "Entity 'http://edamontology.org/format_1703' - Label 'ChEBI entry format'\n",
      "Entity 'http://edamontology.org/format_1704' - Label 'MSDchem ligand dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1706' - Label 'KEGG DRUG entry format'\n",
      "Entity 'http://edamontology.org/format_1747' - Label 'PDB atom record format'\n",
      "Entity 'http://edamontology.org/format_1760' - Label 'CATH chain report format'\n",
      "Entity 'http://edamontology.org/format_1761' - Label 'CATH PDB report format'\n",
      "Entity 'http://edamontology.org/format_1782' - Label 'NCBI gene report format'\n",
      "Entity 'http://edamontology.org/format_1808' - Label 'GeneIlluminator gene report format'\n",
      "Entity 'http://edamontology.org/format_1809' - Label 'BacMap gene card format'\n",
      "Entity 'http://edamontology.org/format_1810' - Label 'ColiCard report format'\n",
      "Entity 'http://edamontology.org/format_1918' - Label 'Atomic data format'\n",
      "Entity 'http://edamontology.org/format_1924' - Label 'clustal sequence format'\n",
      "Entity 'http://edamontology.org/format_1955' - Label 'phylip sequence format'\n",
      "Entity 'http://edamontology.org/format_1956' - Label 'phylipnon sequence format'\n",
      "Entity 'http://edamontology.org/format_1959' - Label 'selex sequence format'\n",
      "Entity 'http://edamontology.org/format_1965' - Label 'treecon sequence format'\n",
      "Entity 'http://edamontology.org/format_1971' - Label 'meganon sequence format'\n",
      "Entity 'http://edamontology.org/format_1976' - Label 'pir'\n",
      "Entity 'http://edamontology.org/format_1977' - Label 'swiss feature'\n",
      "Entity 'http://edamontology.org/format_1980' - Label 'EMBL feature'\n",
      "Entity 'http://edamontology.org/format_1981' - Label 'GenBank feature'\n",
      "Entity 'http://edamontology.org/format_1993' - Label 'msf alignment format'\n",
      "Entity 'http://edamontology.org/format_1994' - Label 'nexus alignment format'\n",
      "Entity 'http://edamontology.org/format_1995' - Label 'nexusnon alignment format'\n",
      "Entity 'http://edamontology.org/format_2015' - Label 'Sequence-profile alignment (HMM) format'\n",
      "Entity 'http://edamontology.org/format_2034' - Label 'Biological model format'\n",
      "Entity 'http://edamontology.org/format_2045' - Label 'Electron microscopy model format'\n",
      "Entity 'http://edamontology.org/format_2051' - Label 'Polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_2059' - Label 'Genotype and phenotype annotation format'\n",
      "Entity 'http://edamontology.org/format_2063' - Label 'Protein report (enzyme) format'\n",
      "Entity 'http://edamontology.org/format_2159' - Label 'Gene features (coding region) format'\n",
      "Entity 'http://edamontology.org/format_2175' - Label 'Gene cluster format'\n",
      "Entity 'http://edamontology.org/format_2188' - Label 'UniProt format'\n",
      "Entity 'http://edamontology.org/format_2189' - Label 'ipi'\n",
      "Entity 'http://edamontology.org/format_2202' - Label 'Sequence record full format'\n",
      "Entity 'http://edamontology.org/format_2203' - Label 'Sequence record lite format'\n",
      "Entity 'http://edamontology.org/format_2210' - Label 'Strain data format'\n",
      "Entity 'http://edamontology.org/format_2211' - Label 'CIP strain data format'\n",
      "Entity 'http://edamontology.org/format_2243' - Label 'phylip property values'\n",
      "Entity 'http://edamontology.org/format_2303' - Label 'STRING entry format (HTML)'\n",
      "Entity 'http://edamontology.org/format_2322' - Label 'BioCyc enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2323' - Label 'ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2328' - Label 'PseudoCAP gene report format'\n",
      "Entity 'http://edamontology.org/format_2329' - Label 'GeneCards gene report format'\n",
      "Entity 'http://edamontology.org/format_2334' - Label 'URI format'\n",
      "Entity 'http://edamontology.org/format_2341' - Label 'NCI-Nature pathway entry format'\n",
      "Entity 'http://edamontology.org/format_2542' - Label 'Protein features (domains) format'\n",
      "Entity 'http://edamontology.org/format_2560' - Label 'STRING entry format'\n",
      "Entity 'http://edamontology.org/format_2562' - Label 'Amino acid identifier format'\n",
      "Entity 'http://edamontology.org/format_3476' - Label 'Gene expression data format'\n",
      "Entity 'http://edamontology.org/format_3623' - Label 'Index format'\n",
      "Entity 'http://edamontology.org/format_4001' - Label 'NIFTI format'\n",
      "Entity 'http://edamontology.org/operation_0225' - Label 'Data retrieval (database cross-reference)'\n",
      "Entity 'http://edamontology.org/operation_0228' - Label 'Data index analysis'\n",
      "Entity 'http://edamontology.org/operation_0229' - Label 'Annotation retrieval (sequence)'\n",
      "Entity 'http://edamontology.org/operation_0241' - Label 'Transcription regulatory sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0242' - Label 'Conserved transcription regulatory sequence identification'\n",
      "Entity 'http://edamontology.org/operation_0243' - Label 'Protein property calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0254' - Label 'Data retrieval (feature table)'\n",
      "Entity 'http://edamontology.org/operation_0255' - Label 'Feature table query'\n",
      "Entity 'http://edamontology.org/operation_0257' - Label 'Data retrieval (sequence alignment)'\n",
      "Entity 'http://edamontology.org/operation_0261' - Label 'Nucleic acid property processing'\n",
      "Entity 'http://edamontology.org/operation_0271' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0273' - Label 'Protein interaction raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_0274' - Label 'Protein-protein interaction prediction (from protein sequence)'\n",
      "Entity 'http://edamontology.org/operation_0275' - Label 'Protein-protein interaction prediction (from protein structure)'\n",
      "Entity 'http://edamontology.org/operation_0277' - Label 'Pathway or network comparison'\n",
      "Entity 'http://edamontology.org/operation_0280' - Label 'Data retrieval (restriction enzyme annotation)'\n",
      "Entity 'http://edamontology.org/operation_0281' - Label 'Genetic marker identification'\n",
      "Entity 'http://edamontology.org/operation_0293' - Label 'Hybrid sequence alignment construction'\n",
      "Entity 'http://edamontology.org/operation_0298' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0299' - Label '3D profile-to-3D profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0301' - Label 'Sequence-to-3D-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0304' - Label 'Metadata retrieval'\n",
      "Entity 'http://edamontology.org/operation_0311' - Label 'Microarray data standardisation and normalisation'\n",
      "Entity 'http://edamontology.org/operation_0312' - Label 'Sequencing-based expression profile data processing'\n",
      "Entity 'http://edamontology.org/operation_0316' - Label 'Functional profiling'\n",
      "Entity 'http://edamontology.org/operation_0317' - Label 'EST and cDNA sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0318' - Label 'Structural genomics target selection'\n",
      "Entity 'http://edamontology.org/operation_0328' - Label 'Protein folding simulation'\n",
      "Entity 'http://edamontology.org/operation_0329' - Label 'Protein folding pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_0330' - Label 'Protein SNP mapping'\n",
      "Entity 'http://edamontology.org/operation_0332' - Label 'Immunogen design'\n",
      "Entity 'http://edamontology.org/operation_0333' - Label 'Zinc finger prediction'\n",
      "Entity 'http://edamontology.org/operation_0340' - Label 'Protein secondary database search'\n",
      "Entity 'http://edamontology.org/operation_0341' - Label 'Motif database search'\n",
      "Entity 'http://edamontology.org/operation_0342' - Label 'Sequence profile database search'\n",
      "Entity 'http://edamontology.org/operation_0343' - Label 'Transmembrane protein database search'\n",
      "Entity 'http://edamontology.org/operation_0344' - Label 'Sequence retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0345' - Label 'Sequence retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0347' - Label 'Sequence database search (by motif or pattern)'\n",
      "Entity 'http://edamontology.org/operation_0348' - Label 'Sequence database search (by amino acid composition)'\n",
      "Entity 'http://edamontology.org/operation_0350' - Label 'Sequence database search (by sequence using word-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0351' - Label 'Sequence database search (by sequence using profile-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0352' - Label 'Sequence database search (by sequence using local alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0353' - Label 'Sequence database search (by sequence using global alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0354' - Label 'Sequence database search (by sequence for primer sequences)'\n",
      "Entity 'http://edamontology.org/operation_0355' - Label 'Sequence database search (by molecular weight)'\n",
      "Entity 'http://edamontology.org/operation_0356' - Label 'Sequence database search (by isoelectric point)'\n",
      "Entity 'http://edamontology.org/operation_0357' - Label 'Structure retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0358' - Label 'Structure retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0359' - Label 'Structure database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_0377' - Label 'Sequence composition calculation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_0378' - Label 'Sequence composition calculation (protein)'\n",
      "Entity 'http://edamontology.org/operation_0383' - Label 'Protein hydropathy calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0385' - Label 'Protein hydropathy cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0388' - Label 'Protein binding site prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0395' - Label 'Residue non-canonical interaction detection'\n",
      "Entity 'http://edamontology.org/operation_0397' - Label 'Ramachandran plot validation'\n",
      "Entity 'http://edamontology.org/operation_0401' - Label 'Protein hydropathy calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0411' - Label 'Protein signal peptide detection (eukaryotes)'\n",
      "Entity 'http://edamontology.org/operation_0412' - Label 'Protein signal peptide detection (bacteria)'\n",
      "Entity 'http://edamontology.org/operation_0413' - Label 'MHC peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0414' - Label 'Protein feature prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0419' - Label 'Protein binding site prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0421' - Label 'Protein folding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0423' - Label 'Epitope mapping (MHC Class I)'\n",
      "Entity 'http://edamontology.org/operation_0424' - Label 'Epitope mapping (MHC Class II)'\n",
      "Entity 'http://edamontology.org/operation_0425' - Label 'Whole gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0426' - Label 'Gene component prediction'\n",
      "Entity 'http://edamontology.org/operation_0434' - Label 'Integrated gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0442' - Label 'Transcriptional regulatory element prediction (RNA-cis)'\n",
      "Entity 'http://edamontology.org/operation_0453' - Label 'Nucleosome formation potential prediction'\n",
      "Entity 'http://edamontology.org/operation_0467' - Label 'Protein secondary structure prediction (integrated)'\n",
      "Entity 'http://edamontology.org/operation_0472' - Label 'GPCR prediction'\n",
      "Entity 'http://edamontology.org/operation_0473' - Label 'GPCR analysis'\n",
      "Entity 'http://edamontology.org/operation_0486' - Label 'Functional mapping'\n",
      "Entity 'http://edamontology.org/operation_0493' - Label 'Pairwise sequence alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0494' - Label 'Pairwise sequence alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0497' - Label 'Constrained sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0498' - Label 'Consensus-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0500' - Label 'Secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0501' - Label 'Protein secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0505' - Label 'Structure alignment (protein)'\n",
      "Entity 'http://edamontology.org/operation_0506' - Label 'Structure alignment (RNA)'\n",
      "Entity 'http://edamontology.org/operation_0507' - Label 'Pairwise structure alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0508' - Label 'Pairwise structure alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0511' - Label 'Profile-profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0512' - Label 'Sequence alignment generation (multiple profile)'\n",
      "Entity 'http://edamontology.org/operation_0513' - Label '3D profile-to-3D profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0514' - Label 'Structural profile alignment generation (multiple)'\n",
      "Entity 'http://edamontology.org/operation_0515' - Label 'Data retrieval (tool metadata)'\n",
      "Entity 'http://edamontology.org/operation_0516' - Label 'Data retrieval (database metadata)'\n",
      "Entity 'http://edamontology.org/operation_0517' - Label 'PCR primer design (for large scale sequencing)'\n",
      "Entity 'http://edamontology.org/operation_0518' - Label 'PCR primer design (for genotyping polymorphisms)'\n",
      "Entity 'http://edamontology.org/operation_0519' - Label 'PCR primer design (for gene transcription profiling)'\n",
      "Entity 'http://edamontology.org/operation_0520' - Label 'PCR primer design (for conserved primers)'\n",
      "Entity 'http://edamontology.org/operation_0521' - Label 'PCR primer design (based on gene structure)'\n",
      "Entity 'http://edamontology.org/operation_0522' - Label 'PCR primer design (for methylation PCRs)'\n",
      "Entity 'http://edamontology.org/operation_0528' - Label 'SAGE data processing'\n",
      "Entity 'http://edamontology.org/operation_0529' - Label 'MPSS data processing'\n",
      "Entity 'http://edamontology.org/operation_0530' - Label 'SBS data processing'\n",
      "Entity 'http://edamontology.org/operation_0532' - Label 'Gene expression profile analysis'\n",
      "Entity 'http://edamontology.org/operation_0534' - Label 'Protein secondary structure assignment (from coordinate data)'\n",
      "Entity 'http://edamontology.org/operation_0535' - Label 'Protein secondary structure assignment (from CD data)'\n",
      "Entity 'http://edamontology.org/operation_0536' - Label 'Protein structure assignment (from X-ray crystallographic data)'\n",
      "Entity 'http://edamontology.org/operation_0537' - Label 'Protein structure assignment (from NMR data)'\n",
      "Entity 'http://edamontology.org/operation_0559' - Label 'Immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0561' - Label 'Sequence formatting'\n",
      "Entity 'http://edamontology.org/operation_0562' - Label 'Sequence alignment formatting'\n",
      "Entity 'http://edamontology.org/operation_0563' - Label 'Codon usage table formatting'\n",
      "Entity 'http://edamontology.org/operation_0565' - Label 'Sequence alignment visualisation'\n",
      "Entity 'http://edamontology.org/operation_0568' - Label 'RNA secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0569' - Label 'Protein secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0572' - Label 'Protein interaction network visualisation'\n",
      "Entity 'http://edamontology.org/operation_0574' - Label 'Sequence motif rendering'\n",
      "Entity 'http://edamontology.org/operation_0577' - Label 'DNA linear map rendering'\n",
      "Entity 'http://edamontology.org/operation_1768' - Label 'Nucleic acid folding family identification'\n",
      "Entity 'http://edamontology.org/operation_1769' - Label 'Nucleic acid folding energy calculation'\n",
      "Entity 'http://edamontology.org/operation_1774' - Label 'Annotation retrieval'\n",
      "Entity 'http://edamontology.org/operation_1780' - Label 'Sequence submission'\n",
      "Entity 'http://edamontology.org/operation_1813' - Label 'Sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1814' - Label 'Structure retrieval'\n",
      "Entity 'http://edamontology.org/operation_1817' - Label 'Protein atom surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1818' - Label 'Protein atom surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1819' - Label 'Protein residue surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1820' - Label 'Protein residue surface calculation (vacuum accessible)'\n",
      "Entity 'http://edamontology.org/operation_1821' - Label 'Protein residue surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1822' - Label 'Protein residue surface calculation (vacuum molecular)'\n",
      "Entity 'http://edamontology.org/operation_1823' - Label 'Protein surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1824' - Label 'Protein surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1825' - Label 'Backbone torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1826' - Label 'Full torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1827' - Label 'Cysteine torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1828' - Label 'Tau angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1832' - Label 'Residue contact calculation (residue-nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_1835' - Label 'Residue contact calculation (residue-negative ion)'\n",
      "Entity 'http://edamontology.org/operation_1837' - Label 'Residue symmetry contact calculation'\n",
      "Entity 'http://edamontology.org/operation_1838' - Label 'Residue contact calculation (residue-ligand)'\n",
      "Entity 'http://edamontology.org/operation_1841' - Label 'Rotamer likelihood prediction'\n",
      "Entity 'http://edamontology.org/operation_1842' - Label 'Proline mutation value calculation'\n",
      "Entity 'http://edamontology.org/operation_1845' - Label 'PDB file sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1846' - Label 'HET group detection'\n",
      "Entity 'http://edamontology.org/operation_1847' - Label 'DSSP secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_1848' - Label 'Structure formatting'\n",
      "Entity 'http://edamontology.org/operation_1913' - Label 'Residue validation'\n",
      "Entity 'http://edamontology.org/operation_1914' - Label 'Structure retrieval (water)'\n",
      "Entity 'http://edamontology.org/operation_2120' - Label 'Listfile processing'\n",
      "Entity 'http://edamontology.org/operation_2122' - Label 'Sequence alignment file processing'\n",
      "Entity 'http://edamontology.org/operation_2123' - Label 'Small molecule data processing'\n",
      "Entity 'http://edamontology.org/operation_2222' - Label 'Data retrieval (ontology annotation)'\n",
      "Entity 'http://edamontology.org/operation_2224' - Label 'Data retrieval (ontology concept)'\n",
      "Entity 'http://edamontology.org/operation_2234' - Label 'Structure file processing'\n",
      "Entity 'http://edamontology.org/operation_2237' - Label 'Data retrieval (sequence profile)'\n",
      "Entity 'http://edamontology.org/operation_2246' - Label 'Demonstration'\n",
      "Entity 'http://edamontology.org/operation_2264' - Label 'Data retrieval (pathway or network)'\n",
      "Entity 'http://edamontology.org/operation_2265' - Label 'Data retrieval (identifier)'\n",
      "Entity 'http://edamontology.org/operation_2405' - Label 'Protein interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2407' - Label 'Annotation processing'\n",
      "Entity 'http://edamontology.org/operation_2408' - Label 'Sequence feature analysis'\n",
      "Entity 'http://edamontology.org/operation_2410' - Label 'Gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2411' - Label 'Structural profile processing'\n",
      "Entity 'http://edamontology.org/operation_2412' - Label 'Data index processing'\n",
      "Entity 'http://edamontology.org/operation_2413' - Label 'Sequence profile processing'\n",
      "Entity 'http://edamontology.org/operation_2414' - Label 'Protein function analysis'\n",
      "Entity 'http://edamontology.org/operation_2417' - Label 'Physicochemical property data processing'\n",
      "Entity 'http://edamontology.org/operation_2420' - Label 'Operation (typed)'\n",
      "Entity 'http://edamontology.org/operation_2427' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2432' - Label 'Microarray data processing'\n",
      "Entity 'http://edamontology.org/operation_2433' - Label 'Codon usage table processing'\n",
      "Entity 'http://edamontology.org/operation_2434' - Label 'Data retrieval (codon usage table)'\n",
      "Entity 'http://edamontology.org/operation_2435' - Label 'Gene expression profile processing'\n",
      "Entity 'http://edamontology.org/operation_2438' - Label 'Pathway or network processing'\n",
      "Entity 'http://edamontology.org/operation_2440' - Label 'Structure processing (RNA)'\n",
      "Entity 'http://edamontology.org/operation_2443' - Label 'Phylogenetic tree processing'\n",
      "Entity 'http://edamontology.org/operation_2444' - Label 'Protein secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2445' - Label 'Protein interaction network processing'\n",
      "Entity 'http://edamontology.org/operation_2446' - Label 'Sequence processing'\n",
      "Entity 'http://edamontology.org/operation_2447' - Label 'Sequence processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2448' - Label 'Sequence processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2452' - Label 'Sequence cluster processing'\n",
      "Entity 'http://edamontology.org/operation_2453' - Label 'Feature table processing'\n",
      "Entity 'http://edamontology.org/operation_2456' - Label 'GPCR classification'\n",
      "Entity 'http://edamontology.org/operation_2457' - Label 'GPCR coupling selectivity prediction'\n",
      "Entity 'http://edamontology.org/operation_2459' - Label 'Structure processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2460' - Label 'Protein atom surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2461' - Label 'Protein residue surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2462' - Label 'Protein surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2463' - Label 'Sequence alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2465' - Label 'Structure processing'\n",
      "Entity 'http://edamontology.org/operation_2466' - Label 'Map annotation'\n",
      "Entity 'http://edamontology.org/operation_2467' - Label 'Data retrieval (protein annotation)'\n",
      "Entity 'http://edamontology.org/operation_2468' - Label 'Data retrieval (phylogenetic tree)'\n",
      "Entity 'http://edamontology.org/operation_2469' - Label 'Data retrieval (protein interaction annotation)'\n",
      "Entity 'http://edamontology.org/operation_2470' - Label 'Data retrieval (protein family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2471' - Label 'Data retrieval (RNA family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2472' - Label 'Data retrieval (gene annotation)'\n",
      "Entity 'http://edamontology.org/operation_2473' - Label 'Data retrieval (genotype and phenotype annotation)'\n",
      "Entity 'http://edamontology.org/operation_2482' - Label 'Secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2490' - Label 'Residue contact calculation (residue-residue)'\n",
      "Entity 'http://edamontology.org/operation_2491' - Label 'Hydrogen bond calculation (inter-residue)'\n",
      "Entity 'http://edamontology.org/operation_2493' - Label 'Codon usage data processing'\n",
      "Entity 'http://edamontology.org/operation_2496' - Label 'Gene regulatory network processing'\n",
      "Entity 'http://edamontology.org/operation_2497' - Label 'Pathway or network analysis'\n",
      "Entity 'http://edamontology.org/operation_2498' - Label 'Sequencing-based expression profile data analysis'\n",
      "Entity 'http://edamontology.org/operation_2500' - Label 'Microarray raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_2501' - Label 'Nucleic acid analysis'\n",
      "Entity 'http://edamontology.org/operation_2502' - Label 'Protein analysis'\n",
      "Entity 'http://edamontology.org/operation_2503' - Label 'Sequence data processing'\n",
      "Entity 'http://edamontology.org/operation_2504' - Label 'Structural data processing'\n",
      "Entity 'http://edamontology.org/operation_2505' - Label 'Text processing'\n",
      "Entity 'http://edamontology.org/operation_2506' - Label 'Protein sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2507' - Label 'Nucleic acid sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2508' - Label 'Nucleic acid sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2509' - Label 'Protein sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2511' - Label 'Sequence editing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2512' - Label 'Sequence editing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2513' - Label 'Sequence generation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2514' - Label 'Sequence generation (protein)'\n",
      "Entity 'http://edamontology.org/operation_2515' - Label 'Nucleic acid sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2516' - Label 'Protein sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2519' - Label 'Structure processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2521' - Label 'Map data processing'\n",
      "Entity 'http://edamontology.org/operation_2931' - Label 'Secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2932' - Label 'Hopp and Woods plotting'\n",
      "Entity 'http://edamontology.org/operation_2934' - Label 'Cluster textual view generation'\n",
      "Entity 'http://edamontology.org/operation_2936' - Label 'Dendrograph plotting'\n",
      "Entity 'http://edamontology.org/operation_2941' - Label 'Whole microarray graph plotting'\n",
      "Entity 'http://edamontology.org/operation_2946' - Label 'Alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2947' - Label 'Article analysis'\n",
      "Entity 'http://edamontology.org/operation_2948' - Label 'Molecular interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2951' - Label 'Alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2952' - Label 'Structure alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2963' - Label 'Codon usage bias plotting'\n",
      "Entity 'http://edamontology.org/operation_2993' - Label 'Molecular interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_3023' - Label 'Prediction and recognition (protein)'\n",
      "Entity 'http://edamontology.org/operation_3024' - Label 'Prediction and recognition (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_3083' - Label 'Pathway or network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3084' - Label 'Protein function prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3087' - Label 'Protein sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3088' - Label 'Protein property calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3090' - Label 'Protein feature prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_3093' - Label 'Database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_3189' - Label 'Trim ends'\n",
      "Entity 'http://edamontology.org/operation_3190' - Label 'Trim vector'\n",
      "Entity 'http://edamontology.org/operation_3191' - Label 'Trim to reference'\n",
      "Entity 'http://edamontology.org/operation_3201' - Label 'SNP calling'\n",
      "Entity 'http://edamontology.org/operation_3202' - Label 'Polymorphism detection'\n",
      "Entity 'http://edamontology.org/operation_3205' - Label 'Methylation calling'\n",
      "Entity 'http://edamontology.org/operation_3212' - Label 'Genome indexing (Burrows-Wheeler)'\n",
      "Entity 'http://edamontology.org/operation_3213' - Label 'Genome indexing (suffix arrays)'\n",
      "Entity 'http://edamontology.org/operation_3224' - Label 'Gene set testing'\n",
      "Entity 'http://edamontology.org/operation_3259' - Label 'Transcriptome assembly (de novo)'\n",
      "Entity 'http://edamontology.org/operation_3260' - Label 'Transcriptome assembly (mapping)'\n",
      "Entity 'http://edamontology.org/operation_3289' - Label 'ID retrieval'\n",
      "Entity 'http://edamontology.org/operation_3353' - Label 'Ontology comparison'\n",
      "Entity 'http://edamontology.org/operation_3430' - Label 'Nucleic acid sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3433' - Label 'Assembly'\n",
      "Entity 'http://edamontology.org/operation_3439' - Label 'Pathway or network prediction'\n",
      "Entity 'http://edamontology.org/operation_3440' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_3441' - Label 'Plotting'\n",
      "Entity 'http://edamontology.org/operation_3470' - Label 'RNA secondary structure prediction (shape-based)'\n",
      "Entity 'http://edamontology.org/operation_3471' - Label 'Nucleic acid folding prediction (alignment-based)'\n",
      "Entity 'http://edamontology.org/operation_3545' - Label 'Mathematical modelling'\n",
      "Entity 'http://edamontology.org/operation_3562' - Label 'Network simulation'\n",
      "Entity 'http://edamontology.org/operation_3648' - Label 'Validation of peptide-spectrum matches'\n",
      "Entity 'http://edamontology.org/operation_3742' - Label 'Differential gene expression analysis'\n",
      "Entity 'http://edamontology.org/topic_0079' - Label 'Metabolites'\n",
      "Entity 'http://edamontology.org/topic_0083' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/topic_0090' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/topic_0094' - Label 'Nucleic acid thermodynamics'\n",
      "Entity 'http://edamontology.org/topic_0100' - Label 'Nucleic acid restriction'\n",
      "Entity 'http://edamontology.org/topic_0107' - Label 'Genetic codes and codon usage'\n",
      "Entity 'http://edamontology.org/topic_0109' - Label 'Gene finding'\n",
      "Entity 'http://edamontology.org/topic_0110' - Label 'Transcription'\n",
      "Entity 'http://edamontology.org/topic_0111' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_0112' - Label 'Nucleic acid folding'\n",
      "Entity 'http://edamontology.org/topic_0133' - Label 'Two-dimensional gel electrophoresis'\n",
      "Entity 'http://edamontology.org/topic_0134' - Label 'Mass spectrometry'\n",
      "Entity 'http://edamontology.org/topic_0135' - Label 'Protein microarrays'\n",
      "Entity 'http://edamontology.org/topic_0137' - Label 'Protein hydropathy'\n",
      "Entity 'http://edamontology.org/topic_0141' - Label 'Protein cleavage sites and proteolysis'\n",
      "Entity 'http://edamontology.org/topic_0143' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/topic_0144' - Label 'Protein residue interactions'\n",
      "Entity 'http://edamontology.org/topic_0147' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0148' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_0149' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_0150' - Label 'Protein design'\n",
      "Entity 'http://edamontology.org/topic_0151' - Label 'G protein-coupled receptors (GPCR)'\n",
      "Entity 'http://edamontology.org/topic_0156' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/topic_0158' - Label 'Sequence motifs'\n",
      "Entity 'http://edamontology.org/topic_0159' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/topic_0163' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/topic_0164' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/topic_0167' - Label 'Structural (3D) profiles'\n",
      "Entity 'http://edamontology.org/topic_0172' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0173' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0174' - Label 'Ab initio structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0175' - Label 'Homology modelling'\n",
      "Entity 'http://edamontology.org/topic_0177' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/topic_0178' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0179' - Label 'Protein tertiary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0180' - Label 'Protein fold recognition'\n",
      "Entity 'http://edamontology.org/topic_0182' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0183' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/topic_0184' - Label 'Threading'\n",
      "Entity 'http://edamontology.org/topic_0188' - Label 'Sequence profiles and HMMs'\n",
      "Entity 'http://edamontology.org/topic_0191' - Label 'Phylogeny reconstruction'\n",
      "Entity 'http://edamontology.org/topic_0195' - Label 'Virtual PCR'\n",
      "Entity 'http://edamontology.org/topic_0200' - Label 'Microarrays'\n",
      "Entity 'http://edamontology.org/topic_0210' - Label 'Fish'\n",
      "Entity 'http://edamontology.org/topic_0211' - Label 'Flies'\n",
      "Entity 'http://edamontology.org/topic_0213' - Label 'Mice or rats'\n",
      "Entity 'http://edamontology.org/topic_0215' - Label 'Worms'\n",
      "Entity 'http://edamontology.org/topic_0217' - Label 'Literature analysis'\n",
      "Entity 'http://edamontology.org/topic_0220' - Label 'Document, record and content management'\n",
      "Entity 'http://edamontology.org/topic_0221' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/topic_0222' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/topic_0594' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/topic_0595' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/topic_0598' - Label 'Sequence motif or profile'\n",
      "Entity 'http://edamontology.org/topic_0606' - Label 'Literature data resources'\n",
      "Entity 'http://edamontology.org/topic_0608' - Label 'Cell and tissue culture'\n",
      "Entity 'http://edamontology.org/topic_0612' - Label 'Cell cycle'\n",
      "Entity 'http://edamontology.org/topic_0613' - Label 'Peptides and amino acids'\n",
      "Entity 'http://edamontology.org/topic_0616' - Label 'Organelles'\n",
      "Entity 'http://edamontology.org/topic_0617' - Label 'Ribosomes'\n",
      "Entity 'http://edamontology.org/topic_0618' - Label 'Scents'\n",
      "Entity 'http://edamontology.org/topic_0620' - Label 'Drugs and target structures'\n",
      "Entity 'http://edamontology.org/topic_0624' - Label 'Chromosomes'\n",
      "Entity 'http://edamontology.org/topic_0629' - Label 'Gene expression and microarray'\n",
      "Entity 'http://edamontology.org/topic_0635' - Label 'Specific protein resources'\n",
      "Entity 'http://edamontology.org/topic_0639' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0640' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0641' - Label 'Repeat sequences'\n",
      "Entity 'http://edamontology.org/topic_0642' - Label 'Low complexity sequences'\n",
      "Entity 'http://edamontology.org/topic_0644' - Label 'Proteome'\n",
      "Entity 'http://edamontology.org/topic_0655' - Label 'Coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0660' - Label 'rRNA'\n",
      "Entity 'http://edamontology.org/topic_0663' - Label 'tRNA'\n",
      "Entity 'http://edamontology.org/topic_0694' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/topic_0697' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/topic_0698' - Label 'Protein tertiary structure'\n",
      "Entity 'http://edamontology.org/topic_0722' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/topic_0724' - Label 'Protein families'\n",
      "Entity 'http://edamontology.org/topic_0740' - Label 'Nucleic acid sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0741' - Label 'Protein sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0747' - Label 'Nucleic acid sites and features'\n",
      "Entity 'http://edamontology.org/topic_0748' - Label 'Protein sites and features'\n",
      "Entity 'http://edamontology.org/topic_0751' - Label 'Phosphorylation sites'\n",
      "Entity 'http://edamontology.org/topic_0753' - Label 'Metabolic pathways'\n",
      "Entity 'http://edamontology.org/topic_0754' - Label 'Signaling pathways'\n",
      "Entity 'http://edamontology.org/topic_0767' - Label 'Protein and peptide identification'\n",
      "Entity 'http://edamontology.org/topic_0770' - Label 'Data types and objects'\n",
      "Entity 'http://edamontology.org/topic_0771' - Label 'Theoretical biology'\n",
      "Entity 'http://edamontology.org/topic_0779' - Label 'Mitochondria'\n",
      "Entity 'http://edamontology.org/topic_0782' - Label 'Fungi'\n",
      "Entity 'http://edamontology.org/topic_0783' - Label 'Pathogens'\n",
      "Entity 'http://edamontology.org/topic_0786' - Label 'Arabidopsis'\n",
      "Entity 'http://edamontology.org/topic_0787' - Label 'Rice'\n",
      "Entity 'http://edamontology.org/topic_0796' - Label 'Genetic mapping and linkage'\n",
      "Entity 'http://edamontology.org/topic_0803' - Label 'Human disease'\n",
      "Entity 'http://edamontology.org/topic_0922' - Label 'Primers'\n",
      "Entity 'http://edamontology.org/topic_1302' - Label 'PolyA signal or sites'\n",
      "Entity 'http://edamontology.org/topic_1304' - Label 'CpG island and isochores'\n",
      "Entity 'http://edamontology.org/topic_1305' - Label 'Restriction sites'\n",
      "Entity 'http://edamontology.org/topic_1307' - Label 'Splice sites'\n",
      "Entity 'http://edamontology.org/topic_1308' - Label 'Matrix/scaffold attachment sites'\n",
      "Entity 'http://edamontology.org/topic_1311' - Label 'Operon'\n",
      "Entity 'http://edamontology.org/topic_1312' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_1456' - Label 'Protein membrane regions'\n",
      "Entity 'http://edamontology.org/topic_1770' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/topic_1811' - Label 'Prokaryotes and Archaea'\n",
      "Entity 'http://edamontology.org/topic_2225' - Label 'Protein databases'\n",
      "Entity 'http://edamontology.org/topic_2226' - Label 'Structure determination'\n",
      "Entity 'http://edamontology.org/topic_2230' - Label 'Classification'\n",
      "Entity 'http://edamontology.org/topic_2232' - Label 'Lipoproteins'\n",
      "Entity 'http://edamontology.org/topic_2257' - Label 'Phylogeny visualisation'\n",
      "Entity 'http://edamontology.org/topic_2271' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/topic_2276' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/topic_2277' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/topic_2278' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/topic_2280' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/topic_2397' - Label 'Exons'\n",
      "Entity 'http://edamontology.org/topic_2399' - Label 'Gene transcription'\n",
      "Entity 'http://edamontology.org/topic_2661' - Label 'Toxins and targets'\n",
      "Entity 'http://edamontology.org/topic_2754' - Label 'Introns'\n",
      "Entity 'http://edamontology.org/topic_2807' - Label 'Tool topic'\n",
      "Entity 'http://edamontology.org/topic_2809' - Label 'Study topic'\n",
      "Entity 'http://edamontology.org/topic_2811' - Label 'Nomenclature'\n",
      "Entity 'http://edamontology.org/topic_2813' - Label 'Disease genes and proteins'\n",
      "Entity 'http://edamontology.org/topic_2816' - Label 'Gene resources'\n",
      "Entity 'http://edamontology.org/topic_2817' - Label 'Yeast'\n",
      "Entity 'http://edamontology.org/topic_2818' - Label 'Eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2819' - Label 'Invertebrates'\n",
      "Entity 'http://edamontology.org/topic_2820' - Label 'Vertebrates'\n",
      "Entity 'http://edamontology.org/topic_2821' - Label 'Unicellular eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2826' - Label 'Protein structure alignment'\n",
      "Entity 'http://edamontology.org/topic_2829' - Label 'Ontologies, nomenclature and classification'\n",
      "Entity 'http://edamontology.org/topic_2839' - Label 'Molecules'\n",
      "Entity 'http://edamontology.org/topic_2842' - Label 'High-throughput sequencing'\n",
      "Entity 'http://edamontology.org/topic_2846' - Label 'Gene regulatory networks'\n",
      "Entity 'http://edamontology.org/topic_2847' - Label 'Disease (specific)'\n",
      "Entity 'http://edamontology.org/topic_2867' - Label 'VNTR'\n",
      "Entity 'http://edamontology.org/topic_2868' - Label 'Microsatellites'\n",
      "Entity 'http://edamontology.org/topic_2869' - Label 'RFLP'\n",
      "Entity 'http://edamontology.org/topic_2953' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/topic_3032' - Label 'Primer or probe design'\n",
      "Entity 'http://edamontology.org/topic_3038' - Label 'Structure databases'\n",
      "Entity 'http://edamontology.org/topic_3039' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/topic_3041' - Label 'Sequence databases'\n",
      "Entity 'http://edamontology.org/topic_3042' - Label 'Nucleic acid sequences'\n",
      "Entity 'http://edamontology.org/topic_3043' - Label 'Protein sequences'\n",
      "Entity 'http://edamontology.org/topic_3044' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/topic_3048' - Label 'Mammals'\n",
      "Entity 'http://edamontology.org/topic_3052' - Label 'Sequence clusters and classification'\n",
      "Entity 'http://edamontology.org/topic_3060' - Label 'Regulatory RNA'\n",
      "Entity 'http://edamontology.org/topic_3061' - Label 'Documentation and help'\n",
      "Entity 'http://edamontology.org/topic_3062' - Label 'Genetic organisation'\n",
      "Entity 'http://edamontology.org/topic_3072' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/topic_3073' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/topic_3074' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/topic_3075' - Label 'Biological system modelling'\n",
      "Entity 'http://edamontology.org/topic_3078' - Label 'Genes and proteins resources'\n",
      "Entity 'http://edamontology.org/topic_3118' - Label 'Protein topological domains'\n",
      "Entity 'http://edamontology.org/topic_3123' - Label 'Expression signals'\n",
      "Entity 'http://edamontology.org/topic_3126' - Label 'Nucleic acid repeats'\n",
      "Entity 'http://edamontology.org/topic_3135' - Label 'Signal or transit peptide'\n",
      "Entity 'http://edamontology.org/topic_3139' - Label 'Sequence tagged sites'\n",
      "Entity 'http://edamontology.org/topic_3171' - Label 'DNA methylation'\n",
      "Entity 'http://edamontology.org/topic_3177' - Label 'DNA-Seq'\n",
      "Entity 'http://edamontology.org/topic_3178' - Label 'RNA-Seq alignment'\n",
      "Entity 'http://edamontology.org/topic_3323' - Label 'Metabolic disease'\n",
      "Entity 'http://edamontology.org/topic_3346' - Label 'Sequence search'\n",
      "Entity 'http://edamontology.org/topic_3413' - Label 'Infectious tropical disease'\n",
      "Entity 'http://edamontology.org/topic_3514' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_3515' - Label 'Protein-drug interactions'\n",
      "Entity 'http://edamontology.org/topic_3521' - Label '2D PAGE experiment'\n",
      "Entity 'http://edamontology.org/topic_3522' - Label 'Northern blot experiment'\n",
      "Entity 'http://edamontology.org/topic_3525' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_3526' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_3527' - Label 'Cellular process pathways'\n",
      "Entity 'http://edamontology.org/topic_3528' - Label 'Disease pathways'\n",
      "Entity 'http://edamontology.org/topic_3529' - Label 'Environmental information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3530' - Label 'Genetic information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3531' - Label 'Protein super-secondary structure'\n",
      "Entity 'http://edamontology.org/topic_3533' - Label 'Protein active sites'\n",
      "Entity 'http://edamontology.org/topic_3535' - Label 'Protein-nucleic acid binding sites'\n",
      "Entity 'http://edamontology.org/topic_3536' - Label 'Protein cleavage sites'\n",
      "Entity 'http://edamontology.org/topic_3537' - Label 'Protein chemical modifications'\n",
      "Entity 'http://edamontology.org/topic_3539' - Label 'Protein domains'\n",
      "Entity 'http://edamontology.org/topic_3540' - Label 'Protein key folding sites'\n",
      "Entity 'http://edamontology.org/topic_3541' - Label 'Protein post-translational modifications'\n",
      "Entity 'http://edamontology.org/topic_3543' - Label 'Protein sequence repeats'\n",
      "Entity 'http://edamontology.org/topic_3544' - Label 'Protein signal peptides'\n",
      "Entity 'http://www.geneontology.org/formats/oboInOwl#ObsoleteClass' - Label 'Obsolete concept (EDAM)'\n"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?label WHERE {\n",
    "\n",
    "    ?entity a owl:Class;\n",
    "            owl:deprecated true .\n",
    "    FILTER NOT EXISTS { ?entity oboInOwl:replacedBy|oboInOwl:consider ?repl1 .} \n",
    "    ?entity rdfs:label ?label .\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T16:53:27.979393Z",
     "start_time": "2024-02-13T16:53:27.963669Z"
    }
   },
   "execution_count": 14
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'http://edamontology.org/data_2539' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/topic_0747' - Label 'Nucleic acid sites and features'\n",
      "Entity 'http://edamontology.org/topic_0748' - Label 'Protein sites and features'\n",
      "Entity 'http://edamontology.org/topic_0751' - Label 'Phosphorylation sites'\n",
      "Entity 'http://edamontology.org/topic_2280' - Label 'Nucleic acid structure comparison'\n"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?label ?property ?replacement WHERE {\n",
    "    VALUES ?property { oboInOwl:replacedBy\n",
    "                       oboInOwl:consider      }\n",
    "    ?entity  ?property ?replacement .\n",
    "    ?entity rdfs:label ?label .\n",
    "    ?replacement owl:deprecated true .\n",
    "    \n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T16:54:56.393383Z",
     "start_time": "2024-02-13T16:54:56.354587Z"
    }
   },
   "execution_count": 15
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception ignored in: <bound method IPythonKernel._clean_thread_parent_frames of <ipykernel.ipkernel.IPythonKernel object at 0x105d86b60>>\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/rioualen/mambaforge/lib/python3.10/site-packages/ipykernel/ipkernel.py\", line 770, in _clean_thread_parent_frames\n",
      "    def _clean_thread_parent_frames(\n",
      "KeyboardInterrupt: \n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
      "\u001B[0;31mKeyboardInterrupt\u001B[0m                         Traceback (most recent call last)",
      "Cell \u001B[0;32mIn[16], line 40\u001B[0m\n\u001B[1;32m      1\u001B[0m query\u001B[38;5;241m=\u001B[39m\u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[1;32m      2\u001B[0m \u001B[38;5;124mSELECT DISTINCT ?entity ?label ?property ?entity2 ?label2 ?property2 ?value WHERE \u001B[39m\u001B[38;5;124m{\u001B[39m\u001B[38;5;124m \u001B[39m\n\u001B[1;32m      3\u001B[0m \u001B[38;5;124m    VALUES ?property \u001B[39m\u001B[38;5;124m{\u001B[39m\n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m     35\u001B[0m \u001B[38;5;124mORDER BY ?value\u001B[39m\n\u001B[1;32m     36\u001B[0m \u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[1;32m     38\u001B[0m results \u001B[38;5;241m=\u001B[39m kg\u001B[38;5;241m.\u001B[39mquery(query)\n\u001B[0;32m---> 40\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m r \u001B[38;5;129;01min\u001B[39;00m results :\n\u001B[1;32m     41\u001B[0m     \u001B[38;5;28mprint\u001B[39m(\u001B[38;5;124mf\u001B[39m\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mEntity \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mentity\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m - Label \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mlabel\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m\"\u001B[39m) \n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/query.py:373\u001B[0m, in \u001B[0;36mResult.__iter__\u001B[0;34m(self)\u001B[0m\n\u001B[1;32m    369\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mtype \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mSELECT\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    370\u001B[0m     \u001B[38;5;66;03m# this iterates over ResultRows of variable bindings\u001B[39;00m\n\u001B[1;32m    372\u001B[0m     \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_genbindings:\n\u001B[0;32m--> 373\u001B[0m         \u001B[38;5;28;01mfor\u001B[39;00m b \u001B[38;5;129;01min\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_genbindings:\n\u001B[1;32m    374\u001B[0m             \u001B[38;5;28;01mif\u001B[39;00m b:  \u001B[38;5;66;03m# don't add a result row in case of empty binding {}\u001B[39;00m\n\u001B[1;32m    375\u001B[0m                 \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_bindings\u001B[38;5;241m.\u001B[39mappend(b)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:551\u001B[0m, in \u001B[0;36mevalDistinct\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    548\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mevalDistinct\u001B[39m(\n\u001B[1;32m    549\u001B[0m     ctx: QueryContext, part: CompValue\n\u001B[1;32m    550\u001B[0m ) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m Generator[FrozenBindings, \u001B[38;5;28;01mNone\u001B[39;00m, \u001B[38;5;28;01mNone\u001B[39;00m]:\n\u001B[0;32m--> 551\u001B[0m     res \u001B[38;5;241m=\u001B[39m \u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mp\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    553\u001B[0m     done \u001B[38;5;241m=\u001B[39m \u001B[38;5;28mset\u001B[39m()\n\u001B[1;32m    554\u001B[0m     \u001B[38;5;28;01mfor\u001B[39;00m x \u001B[38;5;129;01min\u001B[39;00m res:\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:303\u001B[0m, in \u001B[0;36mevalPart\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    300\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalMinus(ctx, part)\n\u001B[1;32m    302\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mProject\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[0;32m--> 303\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mevalProject\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    304\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mSlice\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    305\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalSlice(ctx, part)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:561\u001B[0m, in \u001B[0;36mevalProject\u001B[0;34m(ctx, project)\u001B[0m\n\u001B[1;32m    560\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mevalProject\u001B[39m(ctx: QueryContext, project: CompValue):\n\u001B[0;32m--> 561\u001B[0m     res \u001B[38;5;241m=\u001B[39m \u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mproject\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mp\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    562\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m (row\u001B[38;5;241m.\u001B[39mproject(project\u001B[38;5;241m.\u001B[39mPV) \u001B[38;5;28;01mfor\u001B[39;00m row \u001B[38;5;129;01min\u001B[39;00m res)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:312\u001B[0m, in \u001B[0;36mevalPart\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    309\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalReduced(ctx, part)\n\u001B[1;32m    311\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mOrderBy\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[0;32m--> 312\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mevalOrderBy\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    313\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mGroup\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    314\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalGroup(ctx, part)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:491\u001B[0m, in \u001B[0;36mevalOrderBy\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    489\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m e \u001B[38;5;129;01min\u001B[39;00m \u001B[38;5;28mreversed\u001B[39m(part\u001B[38;5;241m.\u001B[39mexpr):\n\u001B[1;32m    490\u001B[0m     reverse \u001B[38;5;241m=\u001B[39m \u001B[38;5;28mbool\u001B[39m(e\u001B[38;5;241m.\u001B[39morder \u001B[38;5;129;01mand\u001B[39;00m e\u001B[38;5;241m.\u001B[39morder \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mDESC\u001B[39m\u001B[38;5;124m\"\u001B[39m)\n\u001B[0;32m--> 491\u001B[0m     res \u001B[38;5;241m=\u001B[39m \u001B[38;5;28;43msorted\u001B[39;49m\u001B[43m(\u001B[49m\n\u001B[1;32m    492\u001B[0m \u001B[43m        \u001B[49m\u001B[43mres\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mkey\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[38;5;28;43;01mlambda\u001B[39;49;00m\u001B[43m \u001B[49m\u001B[43mx\u001B[49m\u001B[43m:\u001B[49m\u001B[43m \u001B[49m\u001B[43m_val\u001B[49m\u001B[43m(\u001B[49m\u001B[43mvalue\u001B[49m\u001B[43m(\u001B[49m\u001B[43mx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43me\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mexpr\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mvariables\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[38;5;28;43;01mTrue\u001B[39;49;00m\u001B[43m)\u001B[49m\u001B[43m)\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mreverse\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[43mreverse\u001B[49m\n\u001B[1;32m    493\u001B[0m \u001B[43m    \u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    495\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m res\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:204\u001B[0m, in \u001B[0;36mevalFilter\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    200\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mevalFilter\u001B[39m(\n\u001B[1;32m    201\u001B[0m     ctx: QueryContext, part: CompValue\n\u001B[1;32m    202\u001B[0m ) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m Generator[FrozenBindings, \u001B[38;5;28;01mNone\u001B[39;00m, \u001B[38;5;28;01mNone\u001B[39;00m]:\n\u001B[1;32m    203\u001B[0m     \u001B[38;5;66;03m# TODO: Deal with dict returned from evalPart!\u001B[39;00m\n\u001B[0;32m--> 204\u001B[0m     \u001B[38;5;28;01mfor\u001B[39;00m c \u001B[38;5;129;01min\u001B[39;00m \u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mp\u001B[49m\u001B[43m)\u001B[49m:\n\u001B[1;32m    205\u001B[0m         \u001B[38;5;28;01mif\u001B[39;00m _ebv(\n\u001B[1;32m    206\u001B[0m             part\u001B[38;5;241m.\u001B[39mexpr,\n\u001B[1;32m    207\u001B[0m             c\u001B[38;5;241m.\u001B[39mforget(ctx, _except\u001B[38;5;241m=\u001B[39mpart\u001B[38;5;241m.\u001B[39m_vars) \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;129;01mnot\u001B[39;00m part\u001B[38;5;241m.\u001B[39mno_isolated_scope \u001B[38;5;28;01melse\u001B[39;00m c,\n\u001B[1;32m    208\u001B[0m         ):\n\u001B[1;32m    209\u001B[0m             \u001B[38;5;28;01myield\u001B[39;00m c\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:288\u001B[0m, in \u001B[0;36mevalPart\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    286\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalFilter(ctx, part)\n\u001B[1;32m    287\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mJoin\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[0;32m--> 288\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mevalJoin\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    289\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mLeftJoin\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    290\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalLeftJoin(ctx, part)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:156\u001B[0m, in \u001B[0;36mevalJoin\u001B[0;34m(ctx, join)\u001B[0m\n\u001B[1;32m    154\u001B[0m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m    155\u001B[0m     a \u001B[38;5;241m=\u001B[39m evalPart(ctx, join\u001B[38;5;241m.\u001B[39mp1)\n\u001B[0;32m--> 156\u001B[0m     b \u001B[38;5;241m=\u001B[39m \u001B[38;5;28;43mset\u001B[39;49m\u001B[43m(\u001B[49m\u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mjoin\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mp2\u001B[49m\u001B[43m)\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    157\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m _join(a, b)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:112\u001B[0m, in \u001B[0;36mevalBGP\u001B[0;34m(ctx, bgp)\u001B[0m\n\u001B[1;32m    109\u001B[0m \u001B[38;5;28;01mexcept\u001B[39;00m AlreadyBound:\n\u001B[1;32m    110\u001B[0m     \u001B[38;5;28;01mcontinue\u001B[39;00m\n\u001B[0;32m--> 112\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m x \u001B[38;5;129;01min\u001B[39;00m evalBGP(c, bgp[\u001B[38;5;241m1\u001B[39m:]):\n\u001B[1;32m    113\u001B[0m     \u001B[38;5;28;01myield\u001B[39;00m x\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:112\u001B[0m, in \u001B[0;36mevalBGP\u001B[0;34m(ctx, bgp)\u001B[0m\n\u001B[1;32m    109\u001B[0m \u001B[38;5;28;01mexcept\u001B[39;00m AlreadyBound:\n\u001B[1;32m    110\u001B[0m     \u001B[38;5;28;01mcontinue\u001B[39;00m\n\u001B[0;32m--> 112\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m x \u001B[38;5;129;01min\u001B[39;00m evalBGP(c, bgp[\u001B[38;5;241m1\u001B[39m:]):\n\u001B[1;32m    113\u001B[0m     \u001B[38;5;28;01myield\u001B[39;00m x\n",
      "    \u001B[0;31m[... skipping similar frames: evalBGP at line 112 (2 times)]\u001B[0m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:112\u001B[0m, in \u001B[0;36mevalBGP\u001B[0;34m(ctx, bgp)\u001B[0m\n\u001B[1;32m    109\u001B[0m \u001B[38;5;28;01mexcept\u001B[39;00m AlreadyBound:\n\u001B[1;32m    110\u001B[0m     \u001B[38;5;28;01mcontinue\u001B[39;00m\n\u001B[0;32m--> 112\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m x \u001B[38;5;129;01min\u001B[39;00m evalBGP(c, bgp[\u001B[38;5;241m1\u001B[39m:]):\n\u001B[1;32m    113\u001B[0m     \u001B[38;5;28;01myield\u001B[39;00m x\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:84\u001B[0m, in \u001B[0;36mevalBGP\u001B[0;34m(ctx, bgp)\u001B[0m\n\u001B[1;32m     82\u001B[0m _s \u001B[38;5;241m=\u001B[39m ctx[s]\n\u001B[1;32m     83\u001B[0m _p \u001B[38;5;241m=\u001B[39m ctx[p]\n\u001B[0;32m---> 84\u001B[0m _o \u001B[38;5;241m=\u001B[39m \u001B[43mctx\u001B[49m\u001B[43m[\u001B[49m\u001B[43mo\u001B[49m\u001B[43m]\u001B[49m\n\u001B[1;32m     86\u001B[0m \u001B[38;5;66;03m# type error: Item \"None\" of \"Optional[Graph]\" has no attribute \"triples\"\u001B[39;00m\n\u001B[1;32m     87\u001B[0m \u001B[38;5;66;03m# type Argument 1 to \"triples\" of \"Graph\" has incompatible type \"Tuple[Union[str, Path, None], Union[str, Path, None], Union[str, Path, None]]\"; expected \"Tuple[Optional[Node], Optional[Node], Optional[Node]]\"\u001B[39;00m\n\u001B[1;32m     88\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m ss, sp, so \u001B[38;5;129;01min\u001B[39;00m ctx\u001B[38;5;241m.\u001B[39mgraph\u001B[38;5;241m.\u001B[39mtriples((_s, _p, _o)):  \u001B[38;5;66;03m# type: ignore[union-attr, arg-type]\u001B[39;00m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/sparql.py:363\u001B[0m, in \u001B[0;36mQueryContext.__getitem__\u001B[0;34m(self, key)\u001B[0m\n\u001B[1;32m    361\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m key\n\u001B[1;32m    362\u001B[0m \u001B[38;5;28;01mtry\u001B[39;00m:\n\u001B[0;32m--> 363\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;43mself\u001B[39;49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mbindings\u001B[49m\u001B[43m[\u001B[49m\u001B[43mkey\u001B[49m\u001B[43m]\u001B[49m\n\u001B[1;32m    364\u001B[0m \u001B[38;5;28;01mexcept\u001B[39;00m \u001B[38;5;167;01mKeyError\u001B[39;00m:\n\u001B[1;32m    365\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;01mNone\u001B[39;00m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/sparql.py:76\u001B[0m, in \u001B[0;36mBindings.__getitem__\u001B[0;34m(self, key)\u001B[0m\n\u001B[1;32m     74\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21m__getitem__\u001B[39m(\u001B[38;5;28mself\u001B[39m, key: \u001B[38;5;28mstr\u001B[39m) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m \u001B[38;5;28mstr\u001B[39m:\n\u001B[1;32m     75\u001B[0m     \u001B[38;5;28;01mif\u001B[39;00m key \u001B[38;5;129;01min\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_d:\n\u001B[0;32m---> 76\u001B[0m         \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;43mself\u001B[39;49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43m_d\u001B[49m\u001B[43m[\u001B[49m\u001B[43mkey\u001B[49m\u001B[43m]\u001B[49m\n\u001B[1;32m     78\u001B[0m     \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;129;01mnot\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mouter:\n\u001B[1;32m     79\u001B[0m         \u001B[38;5;28;01mraise\u001B[39;00m \u001B[38;5;167;01mKeyError\u001B[39;00m()\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/term.py:184\u001B[0m, in \u001B[0;36mIdentifier.__eq__\u001B[0;34m(self, other)\u001B[0m\n\u001B[1;32m    164\u001B[0m \u001B[38;5;250m\u001B[39m\u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m    165\u001B[0m \u001B[38;5;124;03mEquality for Nodes.\u001B[39;00m\n\u001B[1;32m    166\u001B[0m \n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m    180\u001B[0m \u001B[38;5;124;03mFalse\u001B[39;00m\n\u001B[1;32m    181\u001B[0m \u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m    183\u001B[0m \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28mtype\u001B[39m(\u001B[38;5;28mself\u001B[39m) \u001B[38;5;241m==\u001B[39m \u001B[38;5;28mtype\u001B[39m(other):\n\u001B[0;32m--> 184\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;43mstr\u001B[39;49m\u001B[43m(\u001B[49m\u001B[38;5;28;43mself\u001B[39;49m\u001B[43m)\u001B[49m \u001B[38;5;241m==\u001B[39m \u001B[38;5;28mstr\u001B[39m(other)\n\u001B[1;32m    185\u001B[0m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m    186\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;01mFalse\u001B[39;00m\n",
      "\u001B[0;31mKeyboardInterrupt\u001B[0m: "
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?label ?property ?entity2 ?label2 ?property2 ?value WHERE { \n",
    "    VALUES ?property {\n",
    "                      oboInOwl:hasDefinition\n",
    "                      oboInOwl:hasExactSynonym\n",
    "                      oboInOwl:hasNarrowSynonym\n",
    "                      #oboInOwl:hasBroadSynonym\n",
    "                      #rdfs:comment\n",
    "                      rdfs:label\n",
    "                        }\n",
    "    VALUES ?property2 {\n",
    "                      oboInOwl:hasDefinition\n",
    "                      oboInOwl:hasExactSynonym\n",
    "                      oboInOwl:hasNarrowSynonym\n",
    "                      #oboInOwl:hasBroadSynonym\n",
    "                      #rdfs:comment\n",
    "                      #rdfs:label\n",
    "                        }\n",
    "        \n",
    "    \n",
    "    ?entity a owl:Class . \n",
    "    ?entity2 a owl:Class . \n",
    "    ?entity ?property ?value .\n",
    "    \n",
    "    ?entity2 ?property2 ?value .\n",
    "\n",
    "    #?entity2 ?property2 ?value2 .    \n",
    "    #FILTER ( UCASE(str(?value)) = UCASE(str(?value2)) )      \n",
    "    \n",
    "    FILTER (?entity != ?entity2)\n",
    "    ?entity rdfs:label ?label .\n",
    "    ?entity2 rdfs:label ?label2 .\n",
    "    \n",
    "}\n",
    "ORDER BY ?value\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:14:02.239656Z",
     "start_time": "2024-02-13T17:10:00.845188Z"
    }
   },
   "execution_count": 16
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "\n",
    "SELECT DISTINCT ?entity ?property ?property2 ?value ?label WHERE {\n",
    "    VALUES ?property {oboInOwl:hasDefinition\n",
    "                      oboInOwl:hasExactSynonym\n",
    "                      oboInOwl:hasNarrowSynonym\n",
    "                      oboInOwl:hasBroadSynonym\n",
    "                      rdfs:comment\n",
    "                      rdfs:label\n",
    "                      edam:documentation\n",
    "                        }\n",
    "    VALUES ?property2 {oboInOwl:hasDefinition\n",
    "                      oboInOwl:hasExactSynonym\n",
    "                      oboInOwl:hasNarrowSynonym\n",
    "                      oboInOwl:hasBroadSynonym\n",
    "                      rdfs:comment\n",
    "                      rdfs:label\n",
    "                      edam:documentation\n",
    "                        }\n",
    "\n",
    "    ?entity a owl:Class . \n",
    "    ?entity ?property ?value .\n",
    "    ?entity ?property2 ?value2 .\n",
    "#    FILTER (?value = ?value2)\n",
    "    FILTER ( UCASE(str(?value)) = UCASE(str(?value2)) )\n",
    "\n",
    "    FILTER (?property != ?property2)    \n",
    "    ?entity rdfs:label ?label .\n",
    "  \n",
    "}\n",
    "ORDER BY ?value\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:15:39.485823Z",
     "start_time": "2024-02-13T17:15:00.501498Z"
    }
   },
   "execution_count": 18
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?property ?value ?label WHERE {\n",
    "?entity a owl:Class .\n",
    "?entity ?property ?value .    \n",
    "    FILTER REGEX (?value, \"^(.{0})$\").\n",
    "?entity rdfs:label ?label\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:20:02.693754Z",
     "start_time": "2024-02-13T17:20:01.684744Z"
    }
   },
   "execution_count": 22
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?property ?def ?label WHERE {\n",
    "   VALUES ?property {oboInOwl:hasDefinition}   \n",
    "   ?entity ?property ?def .\n",
    "   ?entity rdfs:label ?label .\n",
    "\n",
    "   FILTER NOT EXISTS {\n",
    "       FILTER REGEX(str(?def), \"['.']+$\")\n",
    "   }\n",
    "   FILTER (!isBlank(?def))\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:18:34.478418Z",
     "start_time": "2024-02-13T17:18:34.209768Z"
    }
   },
   "execution_count": 20
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?property ?value ?label WHERE {\n",
    "   VALUES ?property {rdfs:label}   \n",
    "   ?entity ?property ?value .\n",
    "   ?entity rdfs:label ?label .\n",
    "\n",
    "   FILTER REGEX(str(?value), \"['.']+$\")\n",
    "   FILTER (!isBlank(?entity))\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:19:17.268386Z",
     "start_time": "2024-02-13T17:19:17.046386Z"
    }
   },
   "execution_count": 21
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "ename": "ParseException",
     "evalue": "Expected SelectQuery, found ']'  (at char 155), (line:7, col:1)",
     "output_type": "error",
     "traceback": [
      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
      "\u001B[0;31mParseException\u001B[0m                            Traceback (most recent call last)",
      "Cell \u001B[0;32mIn[26], line 13\u001B[0m\n\u001B[1;32m      1\u001B[0m query\u001B[38;5;241m=\u001B[39m\u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[1;32m      2\u001B[0m \u001B[38;5;124mSELECT DISTINCT ?entity ?property ?value ?label WHERE \u001B[39m\u001B[38;5;124m{\u001B[39m\n\u001B[1;32m      3\u001B[0m \u001B[38;5;124m   ?entity ?property ?value .\u001B[39m\n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m     10\u001B[0m \u001B[38;5;124mORDER BY ?entity\u001B[39m\n\u001B[1;32m     11\u001B[0m \u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[0;32m---> 13\u001B[0m results \u001B[38;5;241m=\u001B[39m \u001B[43mkg\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mquery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mquery\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m     15\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m r \u001B[38;5;129;01min\u001B[39;00m results :\n\u001B[1;32m     16\u001B[0m     \u001B[38;5;28mprint\u001B[39m(\u001B[38;5;124mf\u001B[39m\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mEntity \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mentity\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m - Label \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mlabel\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m\"\u001B[39m) \n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/graph.py:1565\u001B[0m, in \u001B[0;36mGraph.query\u001B[0;34m(self, query_object, processor, result, initNs, initBindings, use_store_provided, **kwargs)\u001B[0m\n\u001B[1;32m   1562\u001B[0m     processor \u001B[38;5;241m=\u001B[39m plugin\u001B[38;5;241m.\u001B[39mget(processor, query\u001B[38;5;241m.\u001B[39mProcessor)(\u001B[38;5;28mself\u001B[39m)\n\u001B[1;32m   1564\u001B[0m \u001B[38;5;66;03m# type error: Argument 1 to \"Result\" has incompatible type \"Mapping[str, Any]\"; expected \"str\"\u001B[39;00m\n\u001B[0;32m-> 1565\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m result(\u001B[43mprocessor\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mquery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mquery_object\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43minitBindings\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43minitNs\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43mkwargs\u001B[49m\u001B[43m)\u001B[49m)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/processor.py:144\u001B[0m, in \u001B[0;36mSPARQLProcessor.query\u001B[0;34m(self, strOrQuery, initBindings, initNs, base, DEBUG)\u001B[0m\n\u001B[1;32m    124\u001B[0m \u001B[38;5;250m\u001B[39m\u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m    125\u001B[0m \u001B[38;5;124;03mEvaluate a query with the given initial bindings, and initial\u001B[39;00m\n\u001B[1;32m    126\u001B[0m \u001B[38;5;124;03mnamespaces. The given base is used to resolve relative URIs in\u001B[39;00m\n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m    140\u001B[0m \u001B[38;5;124;03m   documentation.\u001B[39;00m\n\u001B[1;32m    141\u001B[0m \u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m    143\u001B[0m \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(strOrQuery, \u001B[38;5;28mstr\u001B[39m):\n\u001B[0;32m--> 144\u001B[0m     strOrQuery \u001B[38;5;241m=\u001B[39m translateQuery(\u001B[43mparseQuery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mstrOrQuery\u001B[49m\u001B[43m)\u001B[49m, base, initNs)\n\u001B[1;32m    146\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m evalQuery(\u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mgraph, strOrQuery, initBindings, base)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parser.py:1542\u001B[0m, in \u001B[0;36mparseQuery\u001B[0;34m(q)\u001B[0m\n\u001B[1;32m   1539\u001B[0m     q \u001B[38;5;241m=\u001B[39m q\u001B[38;5;241m.\u001B[39mdecode(\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mutf-8\u001B[39m\u001B[38;5;124m\"\u001B[39m)\n\u001B[1;32m   1541\u001B[0m q \u001B[38;5;241m=\u001B[39m expandUnicodeEscapes(q)\n\u001B[0;32m-> 1542\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mQuery\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mparseString\u001B[49m\u001B[43m(\u001B[49m\u001B[43mq\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mparseAll\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[38;5;28;43;01mTrue\u001B[39;49;00m\u001B[43m)\u001B[49m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/pyparsing/util.py:256\u001B[0m, in \u001B[0;36m_make_synonym_function.<locals>._inner\u001B[0;34m(self, *args, **kwargs)\u001B[0m\n\u001B[1;32m    251\u001B[0m \u001B[38;5;129m@wraps\u001B[39m(fn)\n\u001B[1;32m    252\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21m_inner\u001B[39m(\u001B[38;5;28mself\u001B[39m, \u001B[38;5;241m*\u001B[39margs, \u001B[38;5;241m*\u001B[39m\u001B[38;5;241m*\u001B[39mkwargs):\n\u001B[1;32m    253\u001B[0m     \u001B[38;5;66;03m# warnings.warn(\u001B[39;00m\n\u001B[1;32m    254\u001B[0m     \u001B[38;5;66;03m#     f\"Deprecated - use {fn.__name__}\", DeprecationWarning, stacklevel=3\u001B[39;00m\n\u001B[1;32m    255\u001B[0m     \u001B[38;5;66;03m# )\u001B[39;00m\n\u001B[0;32m--> 256\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mfn\u001B[49m\u001B[43m(\u001B[49m\u001B[38;5;28;43mself\u001B[39;49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43margs\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43mkwargs\u001B[49m\u001B[43m)\u001B[49m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/pyparsing/core.py:1197\u001B[0m, in \u001B[0;36mParserElement.parse_string\u001B[0;34m(self, instring, parse_all, parseAll)\u001B[0m\n\u001B[1;32m   1194\u001B[0m         \u001B[38;5;28;01mraise\u001B[39;00m\n\u001B[1;32m   1195\u001B[0m     \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m   1196\u001B[0m         \u001B[38;5;66;03m# catch and re-raise exception from here, clearing out pyparsing internal stack trace\u001B[39;00m\n\u001B[0;32m-> 1197\u001B[0m         \u001B[38;5;28;01mraise\u001B[39;00m exc\u001B[38;5;241m.\u001B[39mwith_traceback(\u001B[38;5;28;01mNone\u001B[39;00m)\n\u001B[1;32m   1198\u001B[0m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m   1199\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m tokens\n",
      "\u001B[0;31mParseException\u001B[0m: Expected SelectQuery, found ']'  (at char 155), (line:7, col:1)"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?property ?value ?label WHERE {\n",
    "   ?entity ?property ?value .\n",
    "   ?entity rdfs:label ?label .\n",
    "\n",
    "   #FILTER REGEX(str(?value), \"[\\\\s\\r\\n]+$\")\n",
    "   #FILTER (!isBlank(?entity))\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:22:06.994601Z",
     "start_time": "2024-02-13T17:22:06.949433Z"
    }
   },
   "execution_count": 26
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "ename": "ParseException",
     "evalue": "Expected SelectQuery, found 'FILTER'  (at char 126), (line:6, col:4)",
     "output_type": "error",
     "traceback": [
      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
      "\u001B[0;31mParseException\u001B[0m                            Traceback (most recent call last)",
      "Cell \u001B[0;32mIn[28], line 13\u001B[0m\n\u001B[1;32m      1\u001B[0m query\u001B[38;5;241m=\u001B[39m\u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[1;32m      2\u001B[0m \u001B[38;5;124mSELECT DISTINCT ?entity ?property ?value ?label \u001B[39m\n\u001B[1;32m      3\u001B[0m \u001B[38;5;124mWHERE \u001B[39m\u001B[38;5;124m{\u001B[39m\n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m     10\u001B[0m \u001B[38;5;124mORDER BY ?entity\u001B[39m\n\u001B[1;32m     11\u001B[0m \u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[0;32m---> 13\u001B[0m results \u001B[38;5;241m=\u001B[39m \u001B[43mkg\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mquery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mquery\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m     15\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m r \u001B[38;5;129;01min\u001B[39;00m results :\n\u001B[1;32m     16\u001B[0m     \u001B[38;5;28mprint\u001B[39m(\u001B[38;5;124mf\u001B[39m\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mEntity \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mentity\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m - Label \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mlabel\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m\"\u001B[39m) \n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/graph.py:1565\u001B[0m, in \u001B[0;36mGraph.query\u001B[0;34m(self, query_object, processor, result, initNs, initBindings, use_store_provided, **kwargs)\u001B[0m\n\u001B[1;32m   1562\u001B[0m     processor \u001B[38;5;241m=\u001B[39m plugin\u001B[38;5;241m.\u001B[39mget(processor, query\u001B[38;5;241m.\u001B[39mProcessor)(\u001B[38;5;28mself\u001B[39m)\n\u001B[1;32m   1564\u001B[0m \u001B[38;5;66;03m# type error: Argument 1 to \"Result\" has incompatible type \"Mapping[str, Any]\"; expected \"str\"\u001B[39;00m\n\u001B[0;32m-> 1565\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m result(\u001B[43mprocessor\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mquery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mquery_object\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43minitBindings\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43minitNs\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43mkwargs\u001B[49m\u001B[43m)\u001B[49m)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/processor.py:144\u001B[0m, in \u001B[0;36mSPARQLProcessor.query\u001B[0;34m(self, strOrQuery, initBindings, initNs, base, DEBUG)\u001B[0m\n\u001B[1;32m    124\u001B[0m \u001B[38;5;250m\u001B[39m\u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m    125\u001B[0m \u001B[38;5;124;03mEvaluate a query with the given initial bindings, and initial\u001B[39;00m\n\u001B[1;32m    126\u001B[0m \u001B[38;5;124;03mnamespaces. The given base is used to resolve relative URIs in\u001B[39;00m\n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m    140\u001B[0m \u001B[38;5;124;03m   documentation.\u001B[39;00m\n\u001B[1;32m    141\u001B[0m \u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m    143\u001B[0m \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(strOrQuery, \u001B[38;5;28mstr\u001B[39m):\n\u001B[0;32m--> 144\u001B[0m     strOrQuery \u001B[38;5;241m=\u001B[39m translateQuery(\u001B[43mparseQuery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mstrOrQuery\u001B[49m\u001B[43m)\u001B[49m, base, initNs)\n\u001B[1;32m    146\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m evalQuery(\u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mgraph, strOrQuery, initBindings, base)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parser.py:1542\u001B[0m, in \u001B[0;36mparseQuery\u001B[0;34m(q)\u001B[0m\n\u001B[1;32m   1539\u001B[0m     q \u001B[38;5;241m=\u001B[39m q\u001B[38;5;241m.\u001B[39mdecode(\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mutf-8\u001B[39m\u001B[38;5;124m\"\u001B[39m)\n\u001B[1;32m   1541\u001B[0m q \u001B[38;5;241m=\u001B[39m expandUnicodeEscapes(q)\n\u001B[0;32m-> 1542\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mQuery\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mparseString\u001B[49m\u001B[43m(\u001B[49m\u001B[43mq\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mparseAll\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[38;5;28;43;01mTrue\u001B[39;49;00m\u001B[43m)\u001B[49m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/pyparsing/util.py:256\u001B[0m, in \u001B[0;36m_make_synonym_function.<locals>._inner\u001B[0;34m(self, *args, **kwargs)\u001B[0m\n\u001B[1;32m    251\u001B[0m \u001B[38;5;129m@wraps\u001B[39m(fn)\n\u001B[1;32m    252\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21m_inner\u001B[39m(\u001B[38;5;28mself\u001B[39m, \u001B[38;5;241m*\u001B[39margs, \u001B[38;5;241m*\u001B[39m\u001B[38;5;241m*\u001B[39mkwargs):\n\u001B[1;32m    253\u001B[0m     \u001B[38;5;66;03m# warnings.warn(\u001B[39;00m\n\u001B[1;32m    254\u001B[0m     \u001B[38;5;66;03m#     f\"Deprecated - use {fn.__name__}\", DeprecationWarning, stacklevel=3\u001B[39;00m\n\u001B[1;32m    255\u001B[0m     \u001B[38;5;66;03m# )\u001B[39;00m\n\u001B[0;32m--> 256\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mfn\u001B[49m\u001B[43m(\u001B[49m\u001B[38;5;28;43mself\u001B[39;49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43margs\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43mkwargs\u001B[49m\u001B[43m)\u001B[49m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/pyparsing/core.py:1197\u001B[0m, in \u001B[0;36mParserElement.parse_string\u001B[0;34m(self, instring, parse_all, parseAll)\u001B[0m\n\u001B[1;32m   1194\u001B[0m         \u001B[38;5;28;01mraise\u001B[39;00m\n\u001B[1;32m   1195\u001B[0m     \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m   1196\u001B[0m         \u001B[38;5;66;03m# catch and re-raise exception from here, clearing out pyparsing internal stack trace\u001B[39;00m\n\u001B[0;32m-> 1197\u001B[0m         \u001B[38;5;28;01mraise\u001B[39;00m exc\u001B[38;5;241m.\u001B[39mwith_traceback(\u001B[38;5;28;01mNone\u001B[39;00m)\n\u001B[1;32m   1198\u001B[0m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m   1199\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m tokens\n",
      "\u001B[0;31mParseException\u001B[0m: Expected SelectQuery, found 'FILTER'  (at char 126), (line:6, col:4)"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?property ?value ?label \n",
    "WHERE {\n",
    "   ?entity ?property ?value .\n",
    "   ?entity rdfs:label ?label .    \n",
    "   FILTER regex(?value, \"\\n\")\n",
    "   FILTER (!isBlank(?entity))\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:25:17.173205Z",
     "start_time": "2024-02-13T17:25:17.140331Z"
    }
   },
   "execution_count": 28
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'http://edamontology.org/format_1196' - Label 'SMILES'\n",
      "Entity 'http://edamontology.org/format_1197' - Label 'InChI'\n",
      "Entity 'http://edamontology.org/format_1197' - Label 'InChI'\n",
      "Entity 'http://edamontology.org/format_1198' - Label 'mf'\n",
      "Entity 'http://edamontology.org/format_1198' - Label 'mf'\n",
      "Entity 'http://edamontology.org/format_1199' - Label 'InChIKey'\n",
      "Entity 'http://edamontology.org/format_1199' - Label 'InChIKey'\n",
      "Entity 'http://edamontology.org/format_1200' - Label 'smarts'\n",
      "Entity 'http://edamontology.org/format_1200' - Label 'smarts'\n",
      "Entity 'http://edamontology.org/format_1206' - Label 'unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_1206' - Label 'unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_1207' - Label 'nucleotide'\n",
      "Entity 'http://edamontology.org/format_1207' - Label 'nucleotide'\n",
      "Entity 'http://edamontology.org/format_1208' - Label 'protein'\n",
      "Entity 'http://edamontology.org/format_1208' - Label 'protein'\n",
      "Entity 'http://edamontology.org/format_1209' - Label 'consensus'\n",
      "Entity 'http://edamontology.org/format_1209' - Label 'consensus'\n",
      "Entity 'http://edamontology.org/format_1210' - Label 'pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1210' - Label 'pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1211' - Label 'unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1211' - Label 'unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1212' - Label 'dna'\n",
      "Entity 'http://edamontology.org/format_1212' - Label 'dna'\n",
      "Entity 'http://edamontology.org/format_1213' - Label 'rna'\n",
      "Entity 'http://edamontology.org/format_1213' - Label 'rna'\n",
      "Entity 'http://edamontology.org/format_1214' - Label 'unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_1214' - Label 'unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_1215' - Label 'pure dna'\n",
      "Entity 'http://edamontology.org/format_1215' - Label 'pure dna'\n",
      "Entity 'http://edamontology.org/format_1216' - Label 'unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_1216' - Label 'unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_1217' - Label 'pure rna'\n",
      "Entity 'http://edamontology.org/format_1217' - Label 'pure rna'\n",
      "Entity 'http://edamontology.org/format_1218' - Label 'unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_1218' - Label 'unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_1219' - Label 'pure protein'\n",
      "Entity 'http://edamontology.org/format_1219' - Label 'pure protein'\n",
      "Entity 'http://edamontology.org/format_1248' - Label 'EMBL feature location'\n",
      "Entity 'http://edamontology.org/format_1248' - Label 'EMBL feature location'\n",
      "Entity 'http://edamontology.org/format_1295' - Label 'quicktandem'\n",
      "Entity 'http://edamontology.org/format_1295' - Label 'quicktandem'\n",
      "Entity 'http://edamontology.org/format_1296' - Label 'Sanger inverted repeats'\n",
      "Entity 'http://edamontology.org/format_1296' - Label 'Sanger inverted repeats'\n",
      "Entity 'http://edamontology.org/format_1297' - Label 'EMBOSS repeat'\n",
      "Entity 'http://edamontology.org/format_1297' - Label 'EMBOSS repeat'\n",
      "Entity 'http://edamontology.org/format_1316' - Label 'est2genome format'\n",
      "Entity 'http://edamontology.org/format_1316' - Label 'est2genome format'\n",
      "Entity 'http://edamontology.org/format_1318' - Label 'restrict format'\n",
      "Entity 'http://edamontology.org/format_1318' - Label 'restrict format'\n",
      "Entity 'http://edamontology.org/format_1319' - Label 'restover format'\n",
      "Entity 'http://edamontology.org/format_1319' - Label 'restover format'\n",
      "Entity 'http://edamontology.org/format_1320' - Label 'REBASE restriction sites'\n",
      "Entity 'http://edamontology.org/format_1320' - Label 'REBASE restriction sites'\n",
      "Entity 'http://edamontology.org/format_1332' - Label 'FASTA search results format'\n",
      "Entity 'http://edamontology.org/format_1332' - Label 'FASTA search results format'\n",
      "Entity 'http://edamontology.org/format_1333' - Label 'BLAST results'\n",
      "Entity 'http://edamontology.org/format_1333' - Label 'BLAST results'\n",
      "Entity 'http://edamontology.org/format_1334' - Label 'mspcrunch'\n",
      "Entity 'http://edamontology.org/format_1334' - Label 'mspcrunch'\n",
      "Entity 'http://edamontology.org/format_1335' - Label 'Smith-Waterman format'\n",
      "Entity 'http://edamontology.org/format_1335' - Label 'Smith-Waterman format'\n",
      "Entity 'http://edamontology.org/format_1336' - Label 'dhf'\n",
      "Entity 'http://edamontology.org/format_1336' - Label 'dhf'\n",
      "Entity 'http://edamontology.org/format_1337' - Label 'lhf'\n",
      "Entity 'http://edamontology.org/format_1337' - Label 'lhf'\n",
      "Entity 'http://edamontology.org/format_1341' - Label 'InterPro hits format'\n",
      "Entity 'http://edamontology.org/format_1341' - Label 'InterPro hits format'\n",
      "Entity 'http://edamontology.org/format_1342' - Label 'InterPro protein view report format'\n",
      "Entity 'http://edamontology.org/format_1342' - Label 'InterPro protein view report format'\n",
      "Entity 'http://edamontology.org/format_1343' - Label 'InterPro match table format'\n",
      "Entity 'http://edamontology.org/format_1343' - Label 'InterPro match table format'\n",
      "Entity 'http://edamontology.org/format_1349' - Label 'HMMER Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1349' - Label 'HMMER Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1350' - Label 'MEME Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1350' - Label 'MEME Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1351' - Label 'HMMER emission and transition'\n",
      "Entity 'http://edamontology.org/format_1351' - Label 'HMMER emission and transition'\n",
      "Entity 'http://edamontology.org/format_1356' - Label 'prosite-pattern'\n",
      "Entity 'http://edamontology.org/format_1356' - Label 'prosite-pattern'\n",
      "Entity 'http://edamontology.org/format_1357' - Label 'EMBOSS sequence pattern'\n",
      "Entity 'http://edamontology.org/format_1357' - Label 'EMBOSS sequence pattern'\n",
      "Entity 'http://edamontology.org/format_1360' - Label 'meme-motif'\n",
      "Entity 'http://edamontology.org/format_1360' - Label 'meme-motif'\n",
      "Entity 'http://edamontology.org/format_1366' - Label 'prosite-profile'\n",
      "Entity 'http://edamontology.org/format_1366' - Label 'prosite-profile'\n",
      "Entity 'http://edamontology.org/format_1367' - Label 'JASPAR format'\n",
      "Entity 'http://edamontology.org/format_1367' - Label 'JASPAR format'\n",
      "Entity 'http://edamontology.org/format_1369' - Label 'MEME background Markov model'\n",
      "Entity 'http://edamontology.org/format_1369' - Label 'MEME background Markov model'\n",
      "Entity 'http://edamontology.org/format_1370' - Label 'HMMER format'\n",
      "Entity 'http://edamontology.org/format_1370' - Label 'HMMER format'\n",
      "Entity 'http://edamontology.org/format_1391' - Label 'HMMER-aln'\n",
      "Entity 'http://edamontology.org/format_1391' - Label 'HMMER-aln'\n",
      "Entity 'http://edamontology.org/format_1392' - Label 'DIALIGN format'\n",
      "Entity 'http://edamontology.org/format_1392' - Label 'DIALIGN format'\n",
      "Entity 'http://edamontology.org/format_1393' - Label 'daf'\n",
      "Entity 'http://edamontology.org/format_1393' - Label 'daf'\n",
      "Entity 'http://edamontology.org/format_1419' - Label 'Sequence-MEME profile alignment'\n",
      "Entity 'http://edamontology.org/format_1419' - Label 'Sequence-MEME profile alignment'\n",
      "Entity 'http://edamontology.org/format_1421' - Label 'HMMER profile alignment (sequences versus HMMs)'\n",
      "Entity 'http://edamontology.org/format_1421' - Label 'HMMER profile alignment (sequences versus HMMs)'\n",
      "Entity 'http://edamontology.org/format_1422' - Label 'HMMER profile alignment (HMM versus sequences)'\n",
      "Entity 'http://edamontology.org/format_1422' - Label 'HMMER profile alignment (HMM versus sequences)'\n",
      "Entity 'http://edamontology.org/format_1423' - Label 'Phylip distance matrix'\n",
      "Entity 'http://edamontology.org/format_1423' - Label 'Phylip distance matrix'\n",
      "Entity 'http://edamontology.org/format_1424' - Label 'ClustalW dendrogram'\n",
      "Entity 'http://edamontology.org/format_1424' - Label 'ClustalW dendrogram'\n",
      "Entity 'http://edamontology.org/format_1425' - Label 'Phylip tree raw'\n",
      "Entity 'http://edamontology.org/format_1425' - Label 'Phylip tree raw'\n",
      "Entity 'http://edamontology.org/format_1430' - Label 'Phylip continuous quantitative characters'\n",
      "Entity 'http://edamontology.org/format_1430' - Label 'Phylip continuous quantitative characters'\n",
      "Entity 'http://edamontology.org/format_1432' - Label 'Phylip character frequencies format'\n",
      "Entity 'http://edamontology.org/format_1432' - Label 'Phylip character frequencies format'\n",
      "Entity 'http://edamontology.org/format_1433' - Label 'Phylip discrete states format'\n",
      "Entity 'http://edamontology.org/format_1433' - Label 'Phylip discrete states format'\n",
      "Entity 'http://edamontology.org/format_1434' - Label 'Phylip cliques format'\n",
      "Entity 'http://edamontology.org/format_1434' - Label 'Phylip cliques format'\n",
      "Entity 'http://edamontology.org/format_1435' - Label 'Phylip tree format'\n",
      "Entity 'http://edamontology.org/format_1435' - Label 'Phylip tree format'\n",
      "Entity 'http://edamontology.org/format_1436' - Label 'TreeBASE format'\n",
      "Entity 'http://edamontology.org/format_1436' - Label 'TreeBASE format'\n",
      "Entity 'http://edamontology.org/format_1437' - Label 'TreeFam format'\n",
      "Entity 'http://edamontology.org/format_1437' - Label 'TreeFam format'\n",
      "Entity 'http://edamontology.org/format_1445' - Label 'Phylip tree distance format'\n",
      "Entity 'http://edamontology.org/format_1445' - Label 'Phylip tree distance format'\n",
      "Entity 'http://edamontology.org/format_1454' - Label 'dssp'\n",
      "Entity 'http://edamontology.org/format_1454' - Label 'dssp'\n",
      "Entity 'http://edamontology.org/format_1455' - Label 'hssp'\n",
      "Entity 'http://edamontology.org/format_1455' - Label 'hssp'\n",
      "Entity 'http://edamontology.org/format_1457' - Label 'Dot-bracket format'\n",
      "Entity 'http://edamontology.org/format_1457' - Label 'Dot-bracket format'\n",
      "Entity 'http://edamontology.org/format_1458' - Label 'Vienna local RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_1458' - Label 'Vienna local RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_1475' - Label 'PDB database entry format'\n",
      "Entity 'http://edamontology.org/format_1476' - Label 'PDB'\n",
      "Entity 'http://edamontology.org/format_1476' - Label 'PDB'\n",
      "Entity 'http://edamontology.org/format_1477' - Label 'mmCIF'\n",
      "Entity 'http://edamontology.org/format_1477' - Label 'mmCIF'\n",
      "Entity 'http://edamontology.org/format_1478' - Label 'PDBML'\n",
      "Entity 'http://edamontology.org/format_1478' - Label 'PDBML'\n",
      "Entity 'http://edamontology.org/format_1504' - Label 'aaindex'\n",
      "Entity 'http://edamontology.org/format_1504' - Label 'aaindex'\n",
      "Entity 'http://edamontology.org/format_1551' - Label 'Pcons report format'\n",
      "Entity 'http://edamontology.org/format_1551' - Label 'Pcons report format'\n",
      "Entity 'http://edamontology.org/format_1552' - Label 'ProQ report format'\n",
      "Entity 'http://edamontology.org/format_1552' - Label 'ProQ report format'\n",
      "Entity 'http://edamontology.org/format_1582' - Label 'findkm'\n",
      "Entity 'http://edamontology.org/format_1582' - Label 'findkm'\n",
      "Entity 'http://edamontology.org/format_1627' - Label 'Primer3 primer'\n",
      "Entity 'http://edamontology.org/format_1627' - Label 'Primer3 primer'\n",
      "Entity 'http://edamontology.org/format_1628' - Label 'ABI'\n",
      "Entity 'http://edamontology.org/format_1628' - Label 'ABI'\n",
      "Entity 'http://edamontology.org/format_1629' - Label 'mira'\n",
      "Entity 'http://edamontology.org/format_1629' - Label 'mira'\n",
      "Entity 'http://edamontology.org/format_1630' - Label 'CAF'\n",
      "Entity 'http://edamontology.org/format_1631' - Label 'EXP'\n",
      "Entity 'http://edamontology.org/format_1632' - Label 'SCF'\n",
      "Entity 'http://edamontology.org/format_1633' - Label 'PHD'\n",
      "Entity 'http://edamontology.org/format_1637' - Label 'dat'\n",
      "Entity 'http://edamontology.org/format_1638' - Label 'cel'\n",
      "Entity 'http://edamontology.org/format_1639' - Label 'affymetrix'\n",
      "Entity 'http://edamontology.org/format_1639' - Label 'affymetrix'\n",
      "Entity 'http://edamontology.org/format_1641' - Label 'affymetrix-exp'\n",
      "Entity 'http://edamontology.org/format_1641' - Label 'affymetrix-exp'\n",
      "Entity 'http://edamontology.org/format_1665' - Label 'Taverna workflow format'\n",
      "Entity 'http://edamontology.org/format_1665' - Label 'Taverna workflow format'\n",
      "Entity 'http://edamontology.org/format_1705' - Label 'HET group dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1705' - Label 'HET group dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1734' - Label 'PubMed citation'\n",
      "Entity 'http://edamontology.org/format_1734' - Label 'PubMed citation'\n",
      "Entity 'http://edamontology.org/format_1735' - Label 'Medline Display Format'\n",
      "Entity 'http://edamontology.org/format_1735' - Label 'Medline Display Format'\n",
      "Entity 'http://edamontology.org/format_1736' - Label 'CiteXplore-core'\n",
      "Entity 'http://edamontology.org/format_1736' - Label 'CiteXplore-core'\n",
      "Entity 'http://edamontology.org/format_1737' - Label 'CiteXplore-all'\n",
      "Entity 'http://edamontology.org/format_1737' - Label 'CiteXplore-all'\n",
      "Entity 'http://edamontology.org/format_1739' - Label 'pmc'\n",
      "Entity 'http://edamontology.org/format_1739' - Label 'pmc'\n",
      "Entity 'http://edamontology.org/format_1740' - Label 'iHOP format'\n",
      "Entity 'http://edamontology.org/format_1740' - Label 'iHOP format'\n",
      "Entity 'http://edamontology.org/format_1741' - Label 'OSCAR format'\n",
      "Entity 'http://edamontology.org/format_1741' - Label 'OSCAR format'\n",
      "Entity 'http://edamontology.org/format_1861' - Label 'PlasMapper TextMap'\n",
      "Entity 'http://edamontology.org/format_1861' - Label 'PlasMapper TextMap'\n",
      "Entity 'http://edamontology.org/format_1910' - Label 'newick'\n",
      "Entity 'http://edamontology.org/format_1910' - Label 'newick'\n",
      "Entity 'http://edamontology.org/format_1911' - Label 'TreeCon format'\n",
      "Entity 'http://edamontology.org/format_1911' - Label 'TreeCon format'\n",
      "Entity 'http://edamontology.org/format_1912' - Label 'Nexus format'\n",
      "Entity 'http://edamontology.org/format_1912' - Label 'Nexus format'\n",
      "Entity 'http://edamontology.org/format_1919' - Label 'Sequence record format'\n",
      "Entity 'http://edamontology.org/format_1920' - Label 'Sequence feature annotation format'\n",
      "Entity 'http://edamontology.org/format_1921' - Label 'Alignment format'\n",
      "Entity 'http://edamontology.org/format_1923' - Label 'acedb'\n",
      "Entity 'http://edamontology.org/format_1923' - Label 'acedb'\n",
      "Entity 'http://edamontology.org/format_1925' - Label 'codata'\n",
      "Entity 'http://edamontology.org/format_1925' - Label 'codata'\n",
      "Entity 'http://edamontology.org/format_1926' - Label 'dbid'\n",
      "Entity 'http://edamontology.org/format_1926' - Label 'dbid'\n",
      "Entity 'http://edamontology.org/format_1927' - Label 'EMBL format'\n",
      "Entity 'http://edamontology.org/format_1927' - Label 'EMBL format'\n",
      "Entity 'http://edamontology.org/format_1928' - Label 'Staden experiment format'\n",
      "Entity 'http://edamontology.org/format_1928' - Label 'Staden experiment format'\n",
      "Entity 'http://edamontology.org/format_1929' - Label 'FASTA'\n",
      "Entity 'http://edamontology.org/format_1929' - Label 'FASTA'\n",
      "Entity 'http://edamontology.org/format_1930' - Label 'FASTQ'\n",
      "Entity 'http://edamontology.org/format_1930' - Label 'FASTQ'\n",
      "Entity 'http://edamontology.org/format_1931' - Label 'FASTQ-illumina'\n",
      "Entity 'http://edamontology.org/format_1931' - Label 'FASTQ-illumina'\n",
      "Entity 'http://edamontology.org/format_1932' - Label 'FASTQ-sanger'\n",
      "Entity 'http://edamontology.org/format_1932' - Label 'FASTQ-sanger'\n",
      "Entity 'http://edamontology.org/format_1933' - Label 'FASTQ-solexa'\n",
      "Entity 'http://edamontology.org/format_1933' - Label 'FASTQ-solexa'\n",
      "Entity 'http://edamontology.org/format_1934' - Label 'fitch program'\n",
      "Entity 'http://edamontology.org/format_1934' - Label 'fitch program'\n",
      "Entity 'http://edamontology.org/format_1935' - Label 'GCG'\n",
      "Entity 'http://edamontology.org/format_1935' - Label 'GCG'\n",
      "Entity 'http://edamontology.org/format_1936' - Label 'GenBank format'\n",
      "Entity 'http://edamontology.org/format_1936' - Label 'GenBank format'\n",
      "Entity 'http://edamontology.org/format_1937' - Label 'genpept'\n",
      "Entity 'http://edamontology.org/format_1937' - Label 'genpept'\n",
      "Entity 'http://edamontology.org/format_1938' - Label 'GFF2-seq'\n",
      "Entity 'http://edamontology.org/format_1938' - Label 'GFF2-seq'\n",
      "Entity 'http://edamontology.org/format_1939' - Label 'GFF3-seq'\n",
      "Entity 'http://edamontology.org/format_1939' - Label 'GFF3-seq'\n",
      "Entity 'http://edamontology.org/format_1940' - Label 'giFASTA format'\n",
      "Entity 'http://edamontology.org/format_1940' - Label 'giFASTA format'\n",
      "Entity 'http://edamontology.org/format_1941' - Label 'hennig86'\n",
      "Entity 'http://edamontology.org/format_1941' - Label 'hennig86'\n",
      "Entity 'http://edamontology.org/format_1942' - Label 'ig'\n",
      "Entity 'http://edamontology.org/format_1942' - Label 'ig'\n",
      "Entity 'http://edamontology.org/format_1943' - Label 'igstrict'\n",
      "Entity 'http://edamontology.org/format_1943' - Label 'igstrict'\n",
      "Entity 'http://edamontology.org/format_1944' - Label 'jackknifer'\n",
      "Entity 'http://edamontology.org/format_1944' - Label 'jackknifer'\n",
      "Entity 'http://edamontology.org/format_1945' - Label 'mase format'\n",
      "Entity 'http://edamontology.org/format_1945' - Label 'mase format'\n",
      "Entity 'http://edamontology.org/format_1946' - Label 'mega-seq'\n",
      "Entity 'http://edamontology.org/format_1946' - Label 'mega-seq'\n",
      "Entity 'http://edamontology.org/format_1947' - Label 'GCG MSF'\n",
      "Entity 'http://edamontology.org/format_1947' - Label 'GCG MSF'\n",
      "Entity 'http://edamontology.org/format_1948' - Label 'nbrf/pir'\n",
      "Entity 'http://edamontology.org/format_1948' - Label 'nbrf/pir'\n",
      "Entity 'http://edamontology.org/format_1949' - Label 'nexus-seq'\n",
      "Entity 'http://edamontology.org/format_1949' - Label 'nexus-seq'\n",
      "Entity 'http://edamontology.org/format_1950' - Label 'pdbatom'\n",
      "Entity 'http://edamontology.org/format_1950' - Label 'pdbatom'\n",
      "Entity 'http://edamontology.org/format_1951' - Label 'pdbatomnuc'\n",
      "Entity 'http://edamontology.org/format_1951' - Label 'pdbatomnuc'\n",
      "Entity 'http://edamontology.org/format_1952' - Label 'pdbseqresnuc'\n",
      "Entity 'http://edamontology.org/format_1952' - Label 'pdbseqresnuc'\n",
      "Entity 'http://edamontology.org/format_1953' - Label 'pdbseqres'\n",
      "Entity 'http://edamontology.org/format_1953' - Label 'pdbseqres'\n",
      "Entity 'http://edamontology.org/format_1954' - Label 'Pearson format'\n",
      "Entity 'http://edamontology.org/format_1954' - Label 'Pearson format'\n",
      "Entity 'http://edamontology.org/format_1957' - Label 'raw'\n",
      "Entity 'http://edamontology.org/format_1957' - Label 'raw'\n",
      "Entity 'http://edamontology.org/format_1958' - Label 'refseqp'\n",
      "Entity 'http://edamontology.org/format_1958' - Label 'refseqp'\n",
      "Entity 'http://edamontology.org/format_1960' - Label 'Staden format'\n",
      "Entity 'http://edamontology.org/format_1961' - Label 'Stockholm format'\n",
      "Entity 'http://edamontology.org/format_1962' - Label 'strider format'\n",
      "Entity 'http://edamontology.org/format_1962' - Label 'strider format'\n",
      "Entity 'http://edamontology.org/format_1963' - Label 'UniProtKB format'\n",
      "Entity 'http://edamontology.org/format_1963' - Label 'UniProtKB format'\n",
      "Entity 'http://edamontology.org/format_1964' - Label 'plain text format (unformatted)'\n",
      "Entity 'http://edamontology.org/format_1964' - Label 'plain text format (unformatted)'\n",
      "Entity 'http://edamontology.org/format_1966' - Label 'ASN.1 sequence format'\n",
      "Entity 'http://edamontology.org/format_1966' - Label 'ASN.1 sequence format'\n",
      "Entity 'http://edamontology.org/format_1967' - Label 'DAS format'\n",
      "Entity 'http://edamontology.org/format_1967' - Label 'DAS format'\n",
      "Entity 'http://edamontology.org/format_1968' - Label 'dasdna'\n",
      "Entity 'http://edamontology.org/format_1968' - Label 'dasdna'\n",
      "Entity 'http://edamontology.org/format_1969' - Label 'debug-seq'\n",
      "Entity 'http://edamontology.org/format_1969' - Label 'debug-seq'\n",
      "Entity 'http://edamontology.org/format_1970' - Label 'jackknifernon'\n",
      "Entity 'http://edamontology.org/format_1970' - Label 'jackknifernon'\n",
      "Entity 'http://edamontology.org/format_1972' - Label 'NCBI format'\n",
      "Entity 'http://edamontology.org/format_1972' - Label 'NCBI format'\n",
      "Entity 'http://edamontology.org/format_1973' - Label 'nexusnon'\n",
      "Entity 'http://edamontology.org/format_1973' - Label 'nexusnon'\n",
      "Entity 'http://edamontology.org/format_1974' - Label 'GFF2'\n",
      "Entity 'http://edamontology.org/format_1975' - Label 'GFF3'\n",
      "Entity 'http://edamontology.org/format_1978' - Label 'DASGFF'\n",
      "Entity 'http://edamontology.org/format_1978' - Label 'DASGFF'\n",
      "Entity 'http://edamontology.org/format_1979' - Label 'debug-feat'\n",
      "Entity 'http://edamontology.org/format_1979' - Label 'debug-feat'\n",
      "Entity 'http://edamontology.org/format_1982' - Label 'ClustalW format'\n",
      "Entity 'http://edamontology.org/format_1982' - Label 'ClustalW format'\n",
      "Entity 'http://edamontology.org/format_1983' - Label 'debug'\n",
      "Entity 'http://edamontology.org/format_1983' - Label 'debug'\n",
      "Entity 'http://edamontology.org/format_1984' - Label 'FASTA-aln'\n",
      "Entity 'http://edamontology.org/format_1984' - Label 'FASTA-aln'\n",
      "Entity 'http://edamontology.org/format_1985' - Label 'markx0'\n",
      "Entity 'http://edamontology.org/format_1985' - Label 'markx0'\n",
      "Entity 'http://edamontology.org/format_1986' - Label 'markx1'\n",
      "Entity 'http://edamontology.org/format_1986' - Label 'markx1'\n",
      "Entity 'http://edamontology.org/format_1987' - Label 'markx10'\n",
      "Entity 'http://edamontology.org/format_1987' - Label 'markx10'\n",
      "Entity 'http://edamontology.org/format_1988' - Label 'markx2'\n",
      "Entity 'http://edamontology.org/format_1988' - Label 'markx2'\n",
      "Entity 'http://edamontology.org/format_1989' - Label 'markx3'\n",
      "Entity 'http://edamontology.org/format_1989' - Label 'markx3'\n",
      "Entity 'http://edamontology.org/format_1990' - Label 'match'\n",
      "Entity 'http://edamontology.org/format_1990' - Label 'match'\n",
      "Entity 'http://edamontology.org/format_1991' - Label 'mega'\n",
      "Entity 'http://edamontology.org/format_1991' - Label 'mega'\n",
      "Entity 'http://edamontology.org/format_1992' - Label 'meganon'\n",
      "Entity 'http://edamontology.org/format_1992' - Label 'meganon'\n",
      "Entity 'http://edamontology.org/format_1996' - Label 'pair'\n",
      "Entity 'http://edamontology.org/format_1996' - Label 'pair'\n",
      "Entity 'http://edamontology.org/format_1997' - Label 'PHYLIP format'\n",
      "Entity 'http://edamontology.org/format_1998' - Label 'PHYLIP sequential'\n",
      "Entity 'http://edamontology.org/format_1999' - Label 'scores format'\n",
      "Entity 'http://edamontology.org/format_1999' - Label 'scores format'\n",
      "Entity 'http://edamontology.org/format_2000' - Label 'selex'\n",
      "Entity 'http://edamontology.org/format_2000' - Label 'selex'\n",
      "Entity 'http://edamontology.org/format_2001' - Label 'EMBOSS simple format'\n",
      "Entity 'http://edamontology.org/format_2001' - Label 'EMBOSS simple format'\n",
      "Entity 'http://edamontology.org/format_2002' - Label 'srs format'\n",
      "Entity 'http://edamontology.org/format_2002' - Label 'srs format'\n",
      "Entity 'http://edamontology.org/format_2003' - Label 'srspair'\n",
      "Entity 'http://edamontology.org/format_2003' - Label 'srspair'\n",
      "Entity 'http://edamontology.org/format_2004' - Label 'T-Coffee format'\n",
      "Entity 'http://edamontology.org/format_2004' - Label 'T-Coffee format'\n",
      "Entity 'http://edamontology.org/format_2005' - Label 'TreeCon-seq'\n",
      "Entity 'http://edamontology.org/format_2005' - Label 'TreeCon-seq'\n",
      "Entity 'http://edamontology.org/format_2006' - Label 'Phylogenetic tree format'\n",
      "Entity 'http://edamontology.org/format_2013' - Label 'Biological pathway or network format'\n",
      "Entity 'http://edamontology.org/format_2014' - Label 'Sequence-profile alignment format'\n",
      "Entity 'http://edamontology.org/format_2017' - Label 'Amino acid index format'\n",
      "Entity 'http://edamontology.org/format_2020' - Label 'Article format'\n",
      "Entity 'http://edamontology.org/format_2021' - Label 'Text mining report format'\n",
      "Entity 'http://edamontology.org/format_2027' - Label 'Enzyme kinetics report format'\n",
      "Entity 'http://edamontology.org/format_2030' - Label 'Chemical data format'\n",
      "Entity 'http://edamontology.org/format_2031' - Label 'Gene annotation format'\n",
      "Entity 'http://edamontology.org/format_2032' - Label 'Workflow format'\n",
      "Entity 'http://edamontology.org/format_2032' - Label 'Workflow format'\n",
      "Entity 'http://edamontology.org/format_2033' - Label 'Tertiary structure format'\n",
      "Entity 'http://edamontology.org/format_2033' - Label 'Tertiary structure format'\n",
      "Entity 'http://edamontology.org/format_2035' - Label 'Chemical formula format'\n",
      "Entity 'http://edamontology.org/format_2036' - Label 'Phylogenetic character data format'\n",
      "Entity 'http://edamontology.org/format_2037' - Label 'Phylogenetic continuous quantitative character format'\n",
      "Entity 'http://edamontology.org/format_2038' - Label 'Phylogenetic discrete states format'\n",
      "Entity 'http://edamontology.org/format_2039' - Label 'Phylogenetic tree report (cliques) format'\n",
      "Entity 'http://edamontology.org/format_2040' - Label 'Phylogenetic tree report (invariants) format'\n",
      "Entity 'http://edamontology.org/format_2049' - Label 'Phylogenetic tree report (tree distances) format'\n",
      "Entity 'http://edamontology.org/format_2052' - Label 'Protein family report format'\n",
      "Entity 'http://edamontology.org/format_2054' - Label 'Protein interaction format'\n",
      "Entity 'http://edamontology.org/format_2055' - Label 'Sequence assembly format'\n",
      "Entity 'http://edamontology.org/format_2056' - Label 'Microarray experiment data format'\n",
      "Entity 'http://edamontology.org/format_2056' - Label 'Microarray experiment data format'\n",
      "Entity 'http://edamontology.org/format_2057' - Label 'Sequence trace format'\n",
      "Entity 'http://edamontology.org/format_2058' - Label 'Gene expression report format'\n",
      "Entity 'http://edamontology.org/format_2060' - Label 'Map format'\n",
      "Entity 'http://edamontology.org/format_2061' - Label 'Nucleic acid features (primers) format'\n",
      "Entity 'http://edamontology.org/format_2061' - Label 'Nucleic acid features (primers) format'\n",
      "Entity 'http://edamontology.org/format_2062' - Label 'Protein report format'\n",
      "Entity 'http://edamontology.org/format_2064' - Label '3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_2065' - Label 'Protein structure report (quality evaluation) format'\n",
      "Entity 'http://edamontology.org/format_2066' - Label 'Database hits (sequence) format'\n",
      "Entity 'http://edamontology.org/format_2067' - Label 'Sequence distance matrix format'\n",
      "Entity 'http://edamontology.org/format_2068' - Label 'Sequence motif format'\n",
      "Entity 'http://edamontology.org/format_2069' - Label 'Sequence profile format'\n",
      "Entity 'http://edamontology.org/format_2072' - Label 'Hidden Markov model format'\n",
      "Entity 'http://edamontology.org/format_2074' - Label 'Dirichlet distribution format'\n",
      "Entity 'http://edamontology.org/format_2075' - Label 'HMM emission and transition counts format'\n",
      "Entity 'http://edamontology.org/format_2076' - Label 'RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2077' - Label 'Protein secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2077' - Label 'Protein secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2078' - Label 'Sequence range format'\n",
      "Entity 'http://edamontology.org/format_2094' - Label 'pure'\n",
      "Entity 'http://edamontology.org/format_2094' - Label 'pure'\n",
      "Entity 'http://edamontology.org/format_2095' - Label 'unpure'\n",
      "Entity 'http://edamontology.org/format_2095' - Label 'unpure'\n",
      "Entity 'http://edamontology.org/format_2096' - Label 'unambiguous sequence'\n",
      "Entity 'http://edamontology.org/format_2096' - Label 'unambiguous sequence'\n",
      "Entity 'http://edamontology.org/format_2097' - Label 'ambiguous'\n",
      "Entity 'http://edamontology.org/format_2097' - Label 'ambiguous'\n",
      "Entity 'http://edamontology.org/format_2155' - Label 'Sequence features (repeats) format'\n",
      "Entity 'http://edamontology.org/format_2155' - Label 'Sequence features (repeats) format'\n",
      "Entity 'http://edamontology.org/format_2158' - Label 'Nucleic acid features (restriction sites) format'\n",
      "Entity 'http://edamontology.org/format_2158' - Label 'Nucleic acid features (restriction sites) format'\n",
      "Entity 'http://edamontology.org/format_2170' - Label 'Sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_2171' - Label 'Sequence cluster format (protein)'\n",
      "Entity 'http://edamontology.org/format_2171' - Label 'Sequence cluster format (protein)'\n",
      "Entity 'http://edamontology.org/format_2172' - Label 'Sequence cluster format (nucleic acid)'\n",
      "Entity 'http://edamontology.org/format_2172' - Label 'Sequence cluster format (nucleic acid)'\n",
      "Entity 'http://edamontology.org/format_2181' - Label 'EMBL-like (text)'\n",
      "Entity 'http://edamontology.org/format_2181' - Label 'EMBL-like (text)'\n",
      "Entity 'http://edamontology.org/format_2182' - Label 'FASTQ-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2182' - Label 'FASTQ-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2183' - Label 'EMBLXML'\n",
      "Entity 'http://edamontology.org/format_2184' - Label 'cdsxml'\n",
      "Entity 'http://edamontology.org/format_2185' - Label 'INSDSeq'\n",
      "Entity 'http://edamontology.org/format_2186' - Label 'geneseq'\n",
      "Entity 'http://edamontology.org/format_2186' - Label 'geneseq'\n",
      "Entity 'http://edamontology.org/format_2187' - Label 'UniProt-like (text)'\n",
      "Entity 'http://edamontology.org/format_2187' - Label 'UniProt-like (text)'\n",
      "Entity 'http://edamontology.org/format_2194' - Label 'medline'\n",
      "Entity 'http://edamontology.org/format_2194' - Label 'medline'\n",
      "Entity 'http://edamontology.org/format_2195' - Label 'Ontology format'\n",
      "Entity 'http://edamontology.org/format_2196' - Label 'OBO format'\n",
      "Entity 'http://edamontology.org/format_2196' - Label 'OBO format'\n",
      "Entity 'http://edamontology.org/format_2197' - Label 'OWL format'\n",
      "Entity 'http://edamontology.org/format_2197' - Label 'OWL format'\n",
      "Entity 'http://edamontology.org/format_2200' - Label 'FASTA-like (text)'\n",
      "Entity 'http://edamontology.org/format_2200' - Label 'FASTA-like (text)'\n",
      "Entity 'http://edamontology.org/format_2204' - Label 'EMBL format (XML)'\n",
      "Entity 'http://edamontology.org/format_2204' - Label 'EMBL format (XML)'\n",
      "Entity 'http://edamontology.org/format_2205' - Label 'GenBank-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2205' - Label 'GenBank-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2206' - Label 'Sequence feature table format (text)'\n",
      "Entity 'http://edamontology.org/format_2206' - Label 'Sequence feature table format (text)'\n",
      "Entity 'http://edamontology.org/format_2304' - Label 'STRING entry format (XML)'\n",
      "Entity 'http://edamontology.org/format_2304' - Label 'STRING entry format (XML)'\n",
      "Entity 'http://edamontology.org/format_2305' - Label 'GFF'\n",
      "Entity 'http://edamontology.org/format_2305' - Label 'GFF'\n",
      "Entity 'http://edamontology.org/format_2306' - Label 'GTF'\n",
      "Entity 'http://edamontology.org/format_2310' - Label 'FASTA-HTML'\n",
      "Entity 'http://edamontology.org/format_2310' - Label 'FASTA-HTML'\n",
      "Entity 'http://edamontology.org/format_2311' - Label 'EMBL-HTML'\n",
      "Entity 'http://edamontology.org/format_2311' - Label 'EMBL-HTML'\n",
      "Entity 'http://edamontology.org/format_2330' - Label 'Textual format'\n",
      "Entity 'http://edamontology.org/format_2330' - Label 'Textual format'\n",
      "Entity 'http://edamontology.org/format_2331' - Label 'HTML'\n",
      "Entity 'http://edamontology.org/format_2332' - Label 'XML'\n",
      "Entity 'http://edamontology.org/format_2332' - Label 'XML'\n",
      "Entity 'http://edamontology.org/format_2333' - Label 'Binary format'\n",
      "Entity 'http://edamontology.org/format_2333' - Label 'Binary format'\n",
      "Entity 'http://edamontology.org/format_2350' - Label 'Format (by type of data)'\n",
      "Entity 'http://edamontology.org/format_2350' - Label 'Format (by type of data)'\n",
      "Entity 'http://edamontology.org/format_2376' - Label 'RDF format'\n",
      "Entity 'http://edamontology.org/format_2376' - Label 'RDF format'\n",
      "Entity 'http://edamontology.org/format_2532' - Label 'GenBank-HTML'\n",
      "Entity 'http://edamontology.org/format_2532' - Label 'GenBank-HTML'\n",
      "Entity 'http://edamontology.org/format_2543' - Label 'EMBL-like format'\n",
      "Entity 'http://edamontology.org/format_2543' - Label 'EMBL-like format'\n",
      "Entity 'http://edamontology.org/format_2545' - Label 'FASTQ-like format'\n",
      "Entity 'http://edamontology.org/format_2545' - Label 'FASTQ-like format'\n",
      "Entity 'http://edamontology.org/format_2546' - Label 'FASTA-like'\n",
      "Entity 'http://edamontology.org/format_2546' - Label 'FASTA-like'\n",
      "Entity 'http://edamontology.org/format_2547' - Label 'uniprotkb-like format'\n",
      "Entity 'http://edamontology.org/format_2547' - Label 'uniprotkb-like format'\n",
      "Entity 'http://edamontology.org/format_2548' - Label 'Sequence feature table format'\n",
      "Entity 'http://edamontology.org/format_2549' - Label 'OBO'\n",
      "Entity 'http://edamontology.org/format_2549' - Label 'OBO'\n",
      "Entity 'http://edamontology.org/format_2550' - Label 'OBO-XML'\n",
      "Entity 'http://edamontology.org/format_2550' - Label 'OBO-XML'\n",
      "Entity 'http://edamontology.org/format_2551' - Label 'Sequence record format (text)'\n",
      "Entity 'http://edamontology.org/format_2551' - Label 'Sequence record format (text)'\n",
      "Entity 'http://edamontology.org/format_2552' - Label 'Sequence record format (XML)'\n",
      "Entity 'http://edamontology.org/format_2552' - Label 'Sequence record format (XML)'\n",
      "Entity 'http://edamontology.org/format_2553' - Label 'Sequence feature table format (XML)'\n",
      "Entity 'http://edamontology.org/format_2553' - Label 'Sequence feature table format (XML)'\n",
      "Entity 'http://edamontology.org/format_2554' - Label 'Alignment format (text)'\n",
      "Entity 'http://edamontology.org/format_2554' - Label 'Alignment format (text)'\n",
      "Entity 'http://edamontology.org/format_2555' - Label 'Alignment format (XML)'\n",
      "Entity 'http://edamontology.org/format_2555' - Label 'Alignment format (XML)'\n",
      "Entity 'http://edamontology.org/format_2556' - Label 'Phylogenetic tree format (text)'\n",
      "Entity 'http://edamontology.org/format_2556' - Label 'Phylogenetic tree format (text)'\n",
      "Entity 'http://edamontology.org/format_2557' - Label 'Phylogenetic tree format (XML)'\n",
      "Entity 'http://edamontology.org/format_2557' - Label 'Phylogenetic tree format (XML)'\n",
      "Entity 'http://edamontology.org/format_2558' - Label 'EMBL-like (XML)'\n",
      "Entity 'http://edamontology.org/format_2558' - Label 'EMBL-like (XML)'\n",
      "Entity 'http://edamontology.org/format_2559' - Label 'GenBank-like format'\n",
      "Entity 'http://edamontology.org/format_2559' - Label 'GenBank-like format'\n",
      "Entity 'http://edamontology.org/format_2561' - Label 'Sequence assembly format (text)'\n",
      "Entity 'http://edamontology.org/format_2561' - Label 'Sequence assembly format (text)'\n",
      "Entity 'http://edamontology.org/format_2566' - Label 'completely unambiguous'\n",
      "Entity 'http://edamontology.org/format_2566' - Label 'completely unambiguous'\n",
      "Entity 'http://edamontology.org/format_2567' - Label 'completely unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_2567' - Label 'completely unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_2568' - Label 'completely unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_2568' - Label 'completely unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_2569' - Label 'completely unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_2569' - Label 'completely unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_2570' - Label 'completely unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_2570' - Label 'completely unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_2571' - Label 'Raw sequence format'\n",
      "Entity 'http://edamontology.org/format_2572' - Label 'BAM'\n",
      "Entity 'http://edamontology.org/format_2573' - Label 'SAM'\n",
      "Entity 'http://edamontology.org/format_2585' - Label 'SBML'\n",
      "Entity 'http://edamontology.org/format_2607' - Label 'completely unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_2607' - Label 'completely unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_2848' - Label 'Bibliographic reference format'\n",
      "Entity 'http://edamontology.org/format_2919' - Label 'Sequence annotation track format'\n",
      "Entity 'http://edamontology.org/format_2920' - Label 'Alignment format (pair only)'\n",
      "Entity 'http://edamontology.org/format_2921' - Label 'Sequence variation annotation format'\n",
      "Entity 'http://edamontology.org/format_2922' - Label 'markx0 variant'\n",
      "Entity 'http://edamontology.org/format_2922' - Label 'markx0 variant'\n",
      "Entity 'http://edamontology.org/format_2923' - Label 'mega variant'\n",
      "Entity 'http://edamontology.org/format_2923' - Label 'mega variant'\n",
      "Entity 'http://edamontology.org/format_2924' - Label 'Phylip format variant'\n",
      "Entity 'http://edamontology.org/format_2924' - Label 'Phylip format variant'\n",
      "Entity 'http://edamontology.org/format_3000' - Label 'AB1'\n",
      "Entity 'http://edamontology.org/format_3000' - Label 'AB1'\n",
      "Entity 'http://edamontology.org/format_3001' - Label 'ACE'\n",
      "Entity 'http://edamontology.org/format_3003' - Label 'BED'\n",
      "Entity 'http://edamontology.org/format_3004' - Label 'bigBed'\n",
      "Entity 'http://edamontology.org/format_3005' - Label 'WIG'\n",
      "Entity 'http://edamontology.org/format_3006' - Label 'bigWig'\n",
      "Entity 'http://edamontology.org/format_3007' - Label 'PSL'\n",
      "Entity 'http://edamontology.org/format_3008' - Label 'MAF'\n",
      "Entity 'http://edamontology.org/format_3009' - Label '2bit'\n",
      "Entity 'http://edamontology.org/format_3010' - Label '.nib'\n",
      "Entity 'http://edamontology.org/format_3011' - Label 'genePred'\n",
      "Entity 'http://edamontology.org/format_3012' - Label 'pgSnp'\n",
      "Entity 'http://edamontology.org/format_3013' - Label 'axt'\n",
      "Entity 'http://edamontology.org/format_3014' - Label 'LAV'\n",
      "Entity 'http://edamontology.org/format_3015' - Label 'Pileup'\n",
      "Entity 'http://edamontology.org/format_3016' - Label 'VCF'\n",
      "Entity 'http://edamontology.org/format_3017' - Label 'SRF'\n",
      "Entity 'http://edamontology.org/format_3018' - Label 'ZTR'\n",
      "Entity 'http://edamontology.org/format_3019' - Label 'GVF'\n",
      "Entity 'http://edamontology.org/format_3020' - Label 'BCF'\n",
      "Entity 'http://edamontology.org/format_3033' - Label 'Matrix format'\n",
      "Entity 'http://edamontology.org/format_3097' - Label 'Protein domain classification format'\n",
      "Entity 'http://edamontology.org/format_3098' - Label 'Raw SCOP domain classification format'\n",
      "Entity 'http://edamontology.org/format_3098' - Label 'Raw SCOP domain classification format'\n",
      "Entity 'http://edamontology.org/format_3099' - Label 'Raw CATH domain classification format'\n",
      "Entity 'http://edamontology.org/format_3099' - Label 'Raw CATH domain classification format'\n",
      "Entity 'http://edamontology.org/format_3100' - Label 'CATH domain report format'\n",
      "Entity 'http://edamontology.org/format_3100' - Label 'CATH domain report format'\n",
      "Entity 'http://edamontology.org/format_3155' - Label 'SBRML'\n",
      "Entity 'http://edamontology.org/format_3156' - Label 'BioPAX'\n",
      "Entity 'http://edamontology.org/format_3157' - Label 'EBI Application Result XML'\n",
      "Entity 'http://edamontology.org/format_3158' - Label 'PSI MI XML (MIF)'\n",
      "Entity 'http://edamontology.org/format_3159' - Label 'phyloXML'\n",
      "Entity 'http://edamontology.org/format_3160' - Label 'NeXML'\n",
      "Entity 'http://edamontology.org/format_3163' - Label 'GCDML'\n",
      "Entity 'http://edamontology.org/format_3164' - Label 'GTrack'\n",
      "Entity 'http://edamontology.org/format_3166' - Label 'Biological pathway or network report format'\n",
      "Entity 'http://edamontology.org/format_3167' - Label 'Experiment annotation format'\n",
      "Entity 'http://edamontology.org/format_3239' - Label 'CopasiML'\n",
      "Entity 'http://edamontology.org/format_3240' - Label 'CellML'\n",
      "Entity 'http://edamontology.org/format_3242' - Label 'PSI MI TAB (MITAB)'\n",
      "Entity 'http://edamontology.org/format_3243' - Label 'PSI-PAR'\n",
      "Entity 'http://edamontology.org/format_3244' - Label 'mzML'\n",
      "Entity 'http://edamontology.org/format_3245' - Label 'Mass spectrometry data format'\n",
      "Entity 'http://edamontology.org/format_3246' - Label 'TraML'\n",
      "Entity 'http://edamontology.org/format_3247' - Label 'mzIdentML'\n",
      "Entity 'http://edamontology.org/format_3248' - Label 'mzQuantML'\n",
      "Entity 'http://edamontology.org/format_3249' - Label 'GelML'\n",
      "Entity 'http://edamontology.org/format_3250' - Label 'spML'\n",
      "Entity 'http://edamontology.org/format_3252' - Label 'OWL Functional Syntax'\n",
      "Entity 'http://edamontology.org/format_3252' - Label 'OWL Functional Syntax'\n",
      "Entity 'http://edamontology.org/format_3253' - Label 'Manchester OWL Syntax'\n",
      "Entity 'http://edamontology.org/format_3253' - Label 'Manchester OWL Syntax'\n",
      "Entity 'http://edamontology.org/format_3254' - Label 'KRSS2 Syntax'\n",
      "Entity 'http://edamontology.org/format_3254' - Label 'KRSS2 Syntax'\n",
      "Entity 'http://edamontology.org/format_3255' - Label 'Turtle'\n",
      "Entity 'http://edamontology.org/format_3255' - Label 'Turtle'\n",
      "Entity 'http://edamontology.org/format_3256' - Label 'N-Triples'\n",
      "Entity 'http://edamontology.org/format_3256' - Label 'N-Triples'\n",
      "Entity 'http://edamontology.org/format_3257' - Label 'Notation3'\n",
      "Entity 'http://edamontology.org/format_3257' - Label 'Notation3'\n",
      "Entity 'http://edamontology.org/format_3261' - Label 'RDF/XML'\n",
      "Entity 'http://edamontology.org/format_3261' - Label 'RDF/XML'\n",
      "Entity 'http://edamontology.org/format_3262' - Label 'OWL/XML'\n",
      "Entity 'http://edamontology.org/format_3262' - Label 'OWL/XML'\n",
      "Entity 'http://edamontology.org/format_3281' - Label 'A2M'\n",
      "Entity 'http://edamontology.org/format_3284' - Label 'SFF'\n",
      "Entity 'http://edamontology.org/format_3285' - Label 'MAP'\n",
      "Entity 'http://edamontology.org/format_3286' - Label 'PED'\n",
      "Entity 'http://edamontology.org/format_3287' - Label 'Individual genetic data format'\n",
      "Entity 'http://edamontology.org/format_3287' - Label 'Individual genetic data format'\n",
      "Entity 'http://edamontology.org/format_3288' - Label 'PED/MAP'\n",
      "Entity 'http://edamontology.org/format_3309' - Label 'CT'\n",
      "Entity 'http://edamontology.org/format_3310' - Label 'SS'\n",
      "Entity 'http://edamontology.org/format_3311' - Label 'RNAML'\n",
      "Entity 'http://edamontology.org/format_3312' - Label 'GDE'\n",
      "Entity 'http://edamontology.org/format_3313' - Label 'BLC'\n",
      "Entity 'http://edamontology.org/format_3326' - Label 'Data index format'\n",
      "Entity 'http://edamontology.org/format_3328' - Label 'HMMER2'\n",
      "Entity 'http://edamontology.org/format_3329' - Label 'HMMER3'\n",
      "Entity 'http://edamontology.org/format_3330' - Label 'PO'\n",
      "Entity 'http://edamontology.org/format_3331' - Label 'BLAST XML results format'\n",
      "Entity 'http://edamontology.org/format_3331' - Label 'BLAST XML results format'\n",
      "Entity 'http://edamontology.org/format_3462' - Label 'CRAM'\n",
      "Entity 'http://edamontology.org/format_3464' - Label 'JSON'\n",
      "Entity 'http://edamontology.org/format_3464' - Label 'JSON'\n",
      "Entity 'http://edamontology.org/format_3466' - Label 'EPS'\n",
      "Entity 'http://edamontology.org/format_3466' - Label 'EPS'\n",
      "Entity 'http://edamontology.org/format_3467' - Label 'GIF'\n",
      "Entity 'http://edamontology.org/format_3467' - Label 'GIF'\n",
      "Entity 'http://edamontology.org/format_3468' - Label 'xls'\n",
      "Entity 'http://edamontology.org/format_3468' - Label 'xls'\n",
      "Entity 'http://edamontology.org/format_3475' - Label 'TSV'\n",
      "Entity 'http://edamontology.org/format_3475' - Label 'TSV'\n",
      "Entity 'http://edamontology.org/format_3477' - Label 'Cytoscape input file format'\n",
      "Entity 'http://edamontology.org/format_3477' - Label 'Cytoscape input file format'\n",
      "Entity 'http://edamontology.org/format_3485' - Label 'RSF'\n",
      "Entity 'http://edamontology.org/format_3486' - Label 'GCG format variant'\n",
      "Entity 'http://edamontology.org/format_3486' - Label 'GCG format variant'\n",
      "Entity 'http://edamontology.org/format_3487' - Label 'BSML'\n",
      "Entity 'http://edamontology.org/format_3499' - Label 'Ensembl variation file format'\n",
      "Entity 'http://edamontology.org/format_3499' - Label 'Ensembl variation file format'\n",
      "Entity 'http://edamontology.org/format_3506' - Label 'docx'\n",
      "Entity 'http://edamontology.org/format_3506' - Label 'docx'\n",
      "Entity 'http://edamontology.org/format_3507' - Label 'Document format'\n",
      "Entity 'http://edamontology.org/format_3507' - Label 'Document format'\n",
      "Entity 'http://edamontology.org/format_3508' - Label 'PDF'\n",
      "Entity 'http://edamontology.org/format_3508' - Label 'PDF'\n",
      "Entity 'http://edamontology.org/format_3547' - Label 'Image format'\n",
      "Entity 'http://edamontology.org/format_3548' - Label 'DICOM format'\n",
      "Entity 'http://edamontology.org/format_3549' - Label 'nii'\n",
      "Entity 'http://edamontology.org/format_3550' - Label 'mhd'\n",
      "Entity 'http://edamontology.org/format_3551' - Label 'nrrd'\n",
      "Entity 'http://edamontology.org/format_3554' - Label 'R file format'\n",
      "Entity 'http://edamontology.org/format_3554' - Label 'R file format'\n",
      "Entity 'http://edamontology.org/format_3555' - Label 'SPSS'\n",
      "Entity 'http://edamontology.org/format_3555' - Label 'SPSS'\n",
      "Entity 'http://edamontology.org/format_3556' - Label 'MHTML'\n",
      "Entity 'http://edamontology.org/format_3578' - Label 'IDAT'\n",
      "Entity 'http://edamontology.org/format_3579' - Label 'JPG'\n",
      "Entity 'http://edamontology.org/format_3580' - Label 'rcc'\n",
      "Entity 'http://edamontology.org/format_3580' - Label 'rcc'\n",
      "Entity 'http://edamontology.org/format_3581' - Label 'arff'\n",
      "Entity 'http://edamontology.org/format_3582' - Label 'afg'\n",
      "Entity 'http://edamontology.org/format_3583' - Label 'bedgraph'\n",
      "Entity 'http://edamontology.org/format_3584' - Label 'bedstrict'\n",
      "Entity 'http://edamontology.org/format_3585' - Label 'bed6'\n",
      "Entity 'http://edamontology.org/format_3586' - Label 'bed12'\n",
      "Entity 'http://edamontology.org/format_3587' - Label 'chrominfo'\n",
      "Entity 'http://edamontology.org/format_3588' - Label 'customtrack'\n",
      "Entity 'http://edamontology.org/format_3589' - Label 'csfasta'\n",
      "Entity 'http://edamontology.org/format_3590' - Label 'HDF5'\n",
      "Entity 'http://edamontology.org/format_3591' - Label 'TIFF'\n",
      "Entity 'http://edamontology.org/format_3592' - Label 'BMP'\n",
      "Entity 'http://edamontology.org/format_3593' - Label 'im'\n",
      "Entity 'http://edamontology.org/format_3594' - Label 'pcd'\n",
      "Entity 'http://edamontology.org/format_3595' - Label 'pcx'\n",
      "Entity 'http://edamontology.org/format_3596' - Label 'ppm'\n",
      "Entity 'http://edamontology.org/format_3597' - Label 'psd'\n",
      "Entity 'http://edamontology.org/format_3598' - Label 'xbm'\n",
      "Entity 'http://edamontology.org/format_3599' - Label 'xpm'\n",
      "Entity 'http://edamontology.org/format_3600' - Label 'rgb'\n",
      "Entity 'http://edamontology.org/format_3601' - Label 'pbm'\n",
      "Entity 'http://edamontology.org/format_3602' - Label 'pgm'\n",
      "Entity 'http://edamontology.org/format_3603' - Label 'PNG'\n",
      "Entity 'http://edamontology.org/format_3604' - Label 'SVG'\n",
      "Entity 'http://edamontology.org/format_3605' - Label 'rast'\n",
      "Entity 'http://edamontology.org/format_3606' - Label 'Sequence quality report format (text)'\n",
      "Entity 'http://edamontology.org/format_3607' - Label 'qual'\n",
      "Entity 'http://edamontology.org/format_3608' - Label 'qualsolexa'\n",
      "Entity 'http://edamontology.org/format_3608' - Label 'qualsolexa'\n",
      "Entity 'http://edamontology.org/format_3609' - Label 'qualillumina'\n",
      "Entity 'http://edamontology.org/format_3610' - Label 'qualsolid'\n",
      "Entity 'http://edamontology.org/format_3611' - Label 'qual454'\n",
      "Entity 'http://edamontology.org/format_3612' - Label 'ENCODE peak format'\n",
      "Entity 'http://edamontology.org/format_3613' - Label 'ENCODE narrow peak format'\n",
      "Entity 'http://edamontology.org/format_3614' - Label 'ENCODE broad peak format'\n",
      "Entity 'http://edamontology.org/format_3615' - Label 'bgzip'\n",
      "Entity 'http://edamontology.org/format_3616' - Label 'tabix'\n",
      "Entity 'http://edamontology.org/format_3617' - Label 'Graph format'\n",
      "Entity 'http://edamontology.org/format_3617' - Label 'Graph format'\n",
      "Entity 'http://edamontology.org/format_3618' - Label 'xgmml'\n",
      "Entity 'http://edamontology.org/format_3619' - Label 'sif'\n",
      "Entity 'http://edamontology.org/format_3620' - Label 'xlsx'\n",
      "Entity 'http://edamontology.org/format_3620' - Label 'xlsx'\n",
      "Entity 'http://edamontology.org/format_3621' - Label 'SQLite format'\n",
      "Entity 'http://edamontology.org/format_3622' - Label 'Gemini SQLite format'\n",
      "Entity 'http://edamontology.org/format_3624' - Label 'snpeffdb'\n",
      "Entity 'http://edamontology.org/format_3624' - Label 'snpeffdb'\n",
      "Entity 'http://edamontology.org/format_3650' - Label 'netCDF'\n",
      "Entity 'http://edamontology.org/format_3651' - Label 'MGF'\n",
      "Entity 'http://edamontology.org/format_3651' - Label 'MGF'\n",
      "Entity 'http://edamontology.org/format_3652' - Label 'dta'\n",
      "Entity 'http://edamontology.org/format_3652' - Label 'dta'\n",
      "Entity 'http://edamontology.org/format_3653' - Label 'pkl'\n",
      "Entity 'http://edamontology.org/format_3653' - Label 'pkl'\n",
      "Entity 'http://edamontology.org/format_3654' - Label 'mzXML'\n",
      "Entity 'http://edamontology.org/format_3655' - Label 'pepXML'\n",
      "Entity 'http://edamontology.org/format_3657' - Label 'GPML'\n",
      "Entity 'http://edamontology.org/format_3665' - Label 'K-mer countgraph'\n",
      "Entity 'http://edamontology.org/format_3681' - Label 'mzTab'\n",
      "Entity 'http://edamontology.org/format_3682' - Label 'imzML metadata file'\n",
      "Entity 'http://edamontology.org/format_3683' - Label 'qcML'\n",
      "Entity 'http://edamontology.org/format_3684' - Label 'PRIDE XML'\n",
      "Entity 'http://edamontology.org/format_3685' - Label 'SED-ML'\n",
      "Entity 'http://edamontology.org/format_3686' - Label 'COMBINE OMEX'\n",
      "Entity 'http://edamontology.org/format_3687' - Label 'ISA-TAB'\n",
      "Entity 'http://edamontology.org/format_3688' - Label 'SBtab'\n",
      "Entity 'http://edamontology.org/format_3689' - Label 'BCML'\n",
      "Entity 'http://edamontology.org/format_3690' - Label 'BDML'\n",
      "Entity 'http://edamontology.org/format_3691' - Label 'BEL'\n",
      "Entity 'http://edamontology.org/format_3692' - Label 'SBGN-ML'\n",
      "Entity 'http://edamontology.org/format_3693' - Label 'AGP'\n",
      "Entity 'http://edamontology.org/format_3696' - Label 'PS'\n",
      "Entity 'http://edamontology.org/format_3696' - Label 'PS'\n",
      "Entity 'http://edamontology.org/format_3698' - Label 'SRA format'\n",
      "Entity 'http://edamontology.org/format_3699' - Label 'VDB'\n",
      "Entity 'http://edamontology.org/format_3701' - Label 'Sequin format'\n",
      "Entity 'http://edamontology.org/format_3701' - Label 'Sequin format'\n",
      "Entity 'http://edamontology.org/format_3702' - Label 'MSF'\n",
      "Entity 'http://edamontology.org/format_3702' - Label 'MSF'\n",
      "Entity 'http://edamontology.org/format_3706' - Label 'Biodiversity data format'\n",
      "Entity 'http://edamontology.org/format_3709' - Label 'GCT/Res format'\n",
      "Entity 'http://edamontology.org/format_3709' - Label 'GCT/Res format'\n",
      "Entity 'http://edamontology.org/format_3710' - Label 'WIFF format'\n",
      "Entity 'http://edamontology.org/format_3710' - Label 'WIFF format'\n",
      "Entity 'http://edamontology.org/format_3711' - Label 'X!Tandem XML'\n",
      "Entity 'http://edamontology.org/format_3712' - Label 'Thermo RAW'\n",
      "Entity 'http://edamontology.org/format_3712' - Label 'Thermo RAW'\n",
      "Entity 'http://edamontology.org/format_3713' - Label 'Mascot .dat file'\n",
      "Entity 'http://edamontology.org/format_3714' - Label 'MaxQuant APL peaklist format'\n",
      "Entity 'http://edamontology.org/format_3725' - Label 'SBOL'\n",
      "Entity 'http://edamontology.org/format_3726' - Label 'PMML'\n",
      "Entity 'http://edamontology.org/format_3727' - Label 'OME-TIFF'\n",
      "Entity 'http://edamontology.org/format_3728' - Label 'LocARNA PP'\n",
      "Entity 'http://edamontology.org/format_3729' - Label 'dbGaP format'\n",
      "Entity 'http://edamontology.org/format_3746' - Label 'BIOM format'\n",
      "Entity 'http://edamontology.org/format_3747' - Label 'protXML'\n",
      "Entity 'http://edamontology.org/format_3748' - Label 'Linked data format'\n",
      "Entity 'http://edamontology.org/format_3748' - Label 'Linked data format'\n",
      "Entity 'http://edamontology.org/format_3749' - Label 'JSON-LD'\n",
      "Entity 'http://edamontology.org/format_3750' - Label 'YAML'\n",
      "Entity 'http://edamontology.org/format_3751' - Label 'DSV'\n",
      "Entity 'http://edamontology.org/format_3751' - Label 'DSV'\n",
      "Entity 'http://edamontology.org/format_3752' - Label 'CSV'\n",
      "Entity 'http://edamontology.org/format_3752' - Label 'CSV'\n",
      "Entity 'http://edamontology.org/format_3758' - Label 'SEQUEST .out file'\n",
      "Entity 'http://edamontology.org/format_3758' - Label 'SEQUEST .out file'\n",
      "Entity 'http://edamontology.org/format_3764' - Label 'idXML'\n",
      "Entity 'http://edamontology.org/format_3765' - Label 'KNIME datatable format'\n",
      "Entity 'http://edamontology.org/format_3765' - Label 'KNIME datatable format'\n",
      "Entity 'http://edamontology.org/format_3770' - Label 'UniProtKB XML'\n",
      "Entity 'http://edamontology.org/format_3771' - Label 'UniProtKB RDF'\n",
      "Entity 'http://edamontology.org/format_3775' - Label 'GSuite'\n",
      "Entity 'http://edamontology.org/format_3776' - Label 'BTrack'\n",
      "Entity 'http://edamontology.org/format_3776' - Label 'BTrack'\n",
      "Entity 'http://edamontology.org/format_3780' - Label 'Annotated text format'\n",
      "Entity 'http://edamontology.org/format_3781' - Label 'PubAnnotation format'\n",
      "Entity 'http://edamontology.org/format_3782' - Label 'BioC'\n",
      "Entity 'http://edamontology.org/format_3783' - Label 'PubTator format'\n",
      "Entity 'http://edamontology.org/format_3784' - Label 'Open Annotation format'\n",
      "Entity 'http://edamontology.org/format_3785' - Label 'BioNLP Shared Task format'\n",
      "Entity 'http://edamontology.org/format_3787' - Label 'Query language'\n",
      "Entity 'http://edamontology.org/format_3788' - Label 'SQL'\n",
      "Entity 'http://edamontology.org/format_3788' - Label 'SQL'\n",
      "Entity 'http://edamontology.org/format_3789' - Label 'XQuery'\n",
      "Entity 'http://edamontology.org/format_3790' - Label 'SPARQL'\n",
      "Entity 'http://edamontology.org/format_3804' - Label 'xsd'\n",
      "Entity 'http://edamontology.org/format_3804' - Label 'xsd'\n",
      "Entity 'http://edamontology.org/format_3811' - Label 'XMFA'\n",
      "Entity 'http://edamontology.org/format_3812' - Label 'GEN'\n",
      "Entity 'http://edamontology.org/format_3813' - Label 'SAMPLE file format'\n",
      "Entity 'http://edamontology.org/format_3814' - Label 'SDF'\n",
      "Entity 'http://edamontology.org/format_3815' - Label 'Molfile'\n",
      "Entity 'http://edamontology.org/format_3816' - Label 'Mol2'\n",
      "Entity 'http://edamontology.org/format_3817' - Label 'latex'\n",
      "Entity 'http://edamontology.org/format_3818' - Label 'ELAND format'\n",
      "Entity 'http://edamontology.org/format_3819' - Label 'Relaxed PHYLIP Interleaved'\n",
      "Entity 'http://edamontology.org/format_3820' - Label 'Relaxed PHYLIP Sequential'\n",
      "Entity 'http://edamontology.org/format_3821' - Label 'VisML'\n",
      "Entity 'http://edamontology.org/format_3822' - Label 'GML'\n",
      "Entity 'http://edamontology.org/format_3823' - Label 'FASTG'\n",
      "Entity 'http://edamontology.org/format_3824' - Label 'NMR data format'\n",
      "Entity 'http://edamontology.org/format_3825' - Label 'nmrML'\n",
      "Entity 'http://edamontology.org/format_3826' - Label 'proBAM'\n",
      "Entity 'http://edamontology.org/format_3827' - Label 'proBED'\n",
      "Entity 'http://edamontology.org/format_3828' - Label 'Raw microarray data format'\n",
      "Entity 'http://edamontology.org/format_3829' - Label 'GPR'\n",
      "Entity 'http://edamontology.org/format_3830' - Label 'ARB'\n",
      "Entity 'http://edamontology.org/format_3830' - Label 'ARB'\n",
      "Entity 'http://edamontology.org/format_3832' - Label 'consensusXML'\n",
      "Entity 'http://edamontology.org/format_3833' - Label 'featureXML'\n",
      "Entity 'http://edamontology.org/format_3834' - Label 'mzData'\n",
      "Entity 'http://edamontology.org/format_3835' - Label 'TIDE TXT'\n",
      "Entity 'http://edamontology.org/format_3836' - Label 'BLAST XML v2 results format'\n",
      "Entity 'http://edamontology.org/format_3838' - Label 'pptx'\n",
      "Entity 'http://edamontology.org/format_3839' - Label 'ibd'\n",
      "Entity 'http://edamontology.org/format_3841' - Label 'NLP format'\n",
      "Entity 'http://edamontology.org/format_3841' - Label 'NLP format'\n",
      "Entity 'http://edamontology.org/format_3843' - Label 'BEAST'\n",
      "Entity 'http://edamontology.org/format_3844' - Label 'Chado-XML'\n",
      "Entity 'http://edamontology.org/format_3845' - Label 'HSAML'\n",
      "Entity 'http://edamontology.org/format_3846' - Label 'InterProScan XML'\n",
      "Entity 'http://edamontology.org/format_3847' - Label 'KGML'\n",
      "Entity 'http://edamontology.org/format_3847' - Label 'KGML'\n",
      "Entity 'http://edamontology.org/format_3848' - Label 'PubMed XML'\n",
      "Entity 'http://edamontology.org/format_3848' - Label 'PubMed XML'\n",
      "Entity 'http://edamontology.org/format_3849' - Label 'MSAML'\n",
      "Entity 'http://edamontology.org/format_3850' - Label 'OrthoXML'\n",
      "Entity 'http://edamontology.org/format_3851' - Label 'PSDML'\n",
      "Entity 'http://edamontology.org/format_3852' - Label 'SeqXML'\n",
      "Entity 'http://edamontology.org/format_3853' - Label 'UniParc XML'\n",
      "Entity 'http://edamontology.org/format_3854' - Label 'UniRef XML'\n",
      "Entity 'http://edamontology.org/format_3857' - Label 'CWL'\n",
      "Entity 'http://edamontology.org/format_3858' - Label 'Waters RAW'\n",
      "Entity 'http://edamontology.org/format_3858' - Label 'Waters RAW'\n",
      "Entity 'http://edamontology.org/format_3859' - Label 'JCAMP-DX'\n",
      "Entity 'http://edamontology.org/format_3862' - Label 'NLP annotation format'\n",
      "Entity 'http://edamontology.org/format_3862' - Label 'NLP annotation format'\n",
      "Entity 'http://edamontology.org/format_3863' - Label 'NLP corpus format'\n",
      "Entity 'http://edamontology.org/format_3863' - Label 'NLP corpus format'\n",
      "Entity 'http://edamontology.org/format_3864' - Label 'mirGFF3'\n",
      "Entity 'http://edamontology.org/format_3865' - Label 'RNA annotation format'\n",
      "Entity 'http://edamontology.org/format_3865' - Label 'RNA annotation format'\n",
      "Entity 'http://edamontology.org/format_3866' - Label 'Trajectory format'\n",
      "Entity 'http://edamontology.org/format_3867' - Label 'Trajectory format (binary)'\n",
      "Entity 'http://edamontology.org/format_3867' - Label 'Trajectory format (binary)'\n",
      "Entity 'http://edamontology.org/format_3868' - Label 'Trajectory format (text)'\n",
      "Entity 'http://edamontology.org/format_3868' - Label 'Trajectory format (text)'\n",
      "Entity 'http://edamontology.org/format_3873' - Label 'HDF'\n",
      "Entity 'http://edamontology.org/format_3873' - Label 'HDF'\n",
      "Entity 'http://edamontology.org/format_3874' - Label 'PCAzip'\n",
      "Entity 'http://edamontology.org/format_3874' - Label 'PCAzip'\n",
      "Entity 'http://edamontology.org/format_3875' - Label 'XTC'\n",
      "Entity 'http://edamontology.org/format_3875' - Label 'XTC'\n",
      "Entity 'http://edamontology.org/format_3876' - Label 'TNG'\n",
      "Entity 'http://edamontology.org/format_3876' - Label 'TNG'\n",
      "Entity 'http://edamontology.org/format_3877' - Label 'XYZ'\n",
      "Entity 'http://edamontology.org/format_3877' - Label 'XYZ'\n",
      "Entity 'http://edamontology.org/format_3878' - Label 'mdcrd'\n",
      "Entity 'http://edamontology.org/format_3879' - Label 'Topology format'\n",
      "Entity 'http://edamontology.org/format_3880' - Label 'GROMACS top'\n",
      "Entity 'http://edamontology.org/format_3881' - Label 'AMBER top'\n",
      "Entity 'http://edamontology.org/format_3882' - Label 'PSF'\n",
      "Entity 'http://edamontology.org/format_3883' - Label 'GROMACS itp'\n",
      "Entity 'http://edamontology.org/format_3884' - Label 'FF parameter format'\n",
      "Entity 'http://edamontology.org/format_3885' - Label 'BinPos'\n",
      "Entity 'http://edamontology.org/format_3885' - Label 'BinPos'\n",
      "Entity 'http://edamontology.org/format_3886' - Label 'RST'\n",
      "Entity 'http://edamontology.org/format_3887' - Label 'CHARMM rtf'\n",
      "Entity 'http://edamontology.org/format_3887' - Label 'CHARMM rtf'\n",
      "Entity 'http://edamontology.org/format_3888' - Label 'AMBER frcmod'\n",
      "Entity 'http://edamontology.org/format_3889' - Label 'AMBER off'\n",
      "Entity 'http://edamontology.org/format_3906' - Label 'NMReDATA'\n",
      "Entity 'http://edamontology.org/format_3906' - Label 'NMReDATA'\n",
      "Entity 'http://edamontology.org/format_3910' - Label 'trr'\n",
      "Entity 'http://edamontology.org/format_3951' - Label 'BcForms'\n",
      "Entity 'http://edamontology.org/format_3956' - Label 'N-Quads'\n",
      "Entity 'http://edamontology.org/format_3969' - Label 'Vega'\n",
      "Entity 'http://edamontology.org/format_3970' - Label 'Vega-lite'\n",
      "Entity 'http://edamontology.org/format_3973' - Label 'Docker image'\n",
      "Entity 'http://edamontology.org/format_3975' - Label 'GFA 1'\n",
      "Entity 'http://edamontology.org/format_3976' - Label 'GFA 2'\n",
      "Entity 'http://edamontology.org/format_3977' - Label 'ObjTables'\n",
      "Entity 'http://edamontology.org/format_3978' - Label 'CONTIG'\n",
      "Entity 'http://edamontology.org/format_3978' - Label 'CONTIG'\n",
      "Entity 'http://edamontology.org/format_3979' - Label 'WEGO'\n",
      "Entity 'http://edamontology.org/format_3979' - Label 'WEGO'\n",
      "Entity 'http://edamontology.org/format_3980' - Label 'RPKM'\n",
      "Entity 'http://edamontology.org/format_3980' - Label 'RPKM'\n",
      "Entity 'http://edamontology.org/format_3981' - Label 'TAR format'\n",
      "Entity 'http://edamontology.org/format_3981' - Label 'TAR format'\n",
      "Entity 'http://edamontology.org/format_3982' - Label 'CHAIN'\n",
      "Entity 'http://edamontology.org/format_3982' - Label 'CHAIN'\n",
      "Entity 'http://edamontology.org/format_3983' - Label 'NET'\n",
      "Entity 'http://edamontology.org/format_3983' - Label 'NET'\n",
      "Entity 'http://edamontology.org/format_3984' - Label 'QMAP'\n",
      "Entity 'http://edamontology.org/format_3984' - Label 'QMAP'\n",
      "Entity 'http://edamontology.org/format_3985' - Label 'gxformat2'\n",
      "Entity 'http://edamontology.org/format_3985' - Label 'gxformat2'\n",
      "Entity 'http://edamontology.org/format_3986' - Label 'WMV'\n",
      "Entity 'http://edamontology.org/format_3986' - Label 'WMV'\n",
      "Entity 'http://edamontology.org/format_3987' - Label 'ZIP format'\n",
      "Entity 'http://edamontology.org/format_3987' - Label 'ZIP format'\n",
      "Entity 'http://edamontology.org/format_3988' - Label 'LSM'\n",
      "Entity 'http://edamontology.org/format_3988' - Label 'LSM'\n",
      "Entity 'http://edamontology.org/format_3989' - Label 'GZIP format'\n",
      "Entity 'http://edamontology.org/format_3989' - Label 'GZIP format'\n",
      "Entity 'http://edamontology.org/format_3990' - Label 'AVI'\n",
      "Entity 'http://edamontology.org/format_3990' - Label 'AVI'\n",
      "Entity 'http://edamontology.org/format_3991' - Label 'TrackDB'\n",
      "Entity 'http://edamontology.org/format_3991' - Label 'TrackDB'\n",
      "Entity 'http://edamontology.org/format_3992' - Label 'CIGAR format'\n",
      "Entity 'http://edamontology.org/format_3992' - Label 'CIGAR format'\n",
      "Entity 'http://edamontology.org/format_3993' - Label 'Stereolithography format'\n",
      "Entity 'http://edamontology.org/format_3993' - Label 'Stereolithography format'\n",
      "Entity 'http://edamontology.org/format_3994' - Label 'U3D'\n",
      "Entity 'http://edamontology.org/format_3994' - Label 'U3D'\n",
      "Entity 'http://edamontology.org/format_3995' - Label 'Texture file format'\n",
      "Entity 'http://edamontology.org/format_3995' - Label 'Texture file format'\n",
      "Entity 'http://edamontology.org/format_3996' - Label 'Python script'\n",
      "Entity 'http://edamontology.org/format_3996' - Label 'Python script'\n",
      "Entity 'http://edamontology.org/format_3997' - Label 'MPEG-4'\n",
      "Entity 'http://edamontology.org/format_3997' - Label 'MPEG-4'\n",
      "Entity 'http://edamontology.org/format_3998' - Label 'Perl script'\n",
      "Entity 'http://edamontology.org/format_3998' - Label 'Perl script'\n",
      "Entity 'http://edamontology.org/format_3999' - Label 'R script'\n",
      "Entity 'http://edamontology.org/format_3999' - Label 'R script'\n",
      "Entity 'http://edamontology.org/format_4000' - Label 'R markdown'\n",
      "Entity 'http://edamontology.org/format_4000' - Label 'R markdown'\n",
      "Entity 'http://edamontology.org/format_4002' - Label 'pickle'\n",
      "Entity 'http://edamontology.org/format_4002' - Label 'pickle'\n",
      "Entity 'http://edamontology.org/format_4003' - Label 'NumPy format'\n",
      "Entity 'http://edamontology.org/format_4003' - Label 'NumPy format'\n",
      "Entity 'http://edamontology.org/format_4004' - Label 'SimTools repertoire file format'\n",
      "Entity 'http://edamontology.org/format_4004' - Label 'SimTools repertoire file format'\n",
      "Entity 'http://edamontology.org/format_4005' - Label 'Configuration file format'\n",
      "Entity 'http://edamontology.org/format_4005' - Label 'Configuration file format'\n",
      "Entity 'http://edamontology.org/format_4006' - Label 'Zstandard format'\n",
      "Entity 'http://edamontology.org/format_4006' - Label 'Zstandard format'\n",
      "Entity 'http://edamontology.org/format_4007' - Label 'MATLAB script'\n",
      "Entity 'http://edamontology.org/format_4007' - Label 'MATLAB script'\n",
      "Entity 'http://edamontology.org/format_4018' - Label 'gVCF'\n",
      "Entity 'http://edamontology.org/format_4023' - Label 'cml'\n",
      "Entity 'http://edamontology.org/format_4025' - Label 'BioSimulators format for the specifications of biosimulation tools'\n",
      "Entity 'http://edamontology.org/format_4026' - Label 'BioSimulators standard for command-line interfaces for biosimulation tools'\n",
      "Entity 'http://edamontology.org/format_4035' - Label 'PQR'\n",
      "Entity 'http://edamontology.org/format_4035' - Label 'PQR'\n",
      "Entity 'http://edamontology.org/format_4036' - Label 'PDBQT'\n",
      "Entity 'http://edamontology.org/format_4036' - Label 'PDBQT'\n",
      "Entity 'http://edamontology.org/format_4039' - Label 'MSP'\n"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "\n",
    "SELECT ?entity ?label ?property WHERE\n",
    "{\n",
    "  \n",
    "    ?entity rdfs:subClassOf+ edam:format_1915 .\n",
    "    ?entity rdfs:label ?label .\n",
    "    {\n",
    "        VALUES ?property { edam:documentation }\n",
    "        FILTER NOT EXISTS {?entity ?property ?value .}\n",
    "    }\n",
    "    UNION\n",
    "    {\n",
    "        VALUES ?property { edam:is_format_of }\n",
    "        FILTER NOT EXISTS {    \n",
    "            ?entity rdfs:subClassOf ?restriction . \n",
    "            ?restriction rdf:type owl:Restriction ; \n",
    "                owl:onProperty ?property  ; \n",
    "                owl:someValuesFrom ?data.}\n",
    "    }\n",
    "}ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:27:04.998163Z",
     "start_time": "2024-02-13T17:27:02.325188Z"
    }
   },
   "execution_count": 30
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'http://edamontology.org/data_0005' - Label 'Resource type'\n",
      "Entity 'http://edamontology.org/data_0006' - Label 'Data'\n",
      "Entity 'http://edamontology.org/data_0007' - Label 'Tool'\n",
      "Entity 'http://edamontology.org/data_0581' - Label 'Database'\n",
      "Entity 'http://edamontology.org/data_0582' - Label 'Ontology'\n",
      "Entity 'http://edamontology.org/data_0583' - Label 'Directory metadata'\n",
      "Entity 'http://edamontology.org/data_0831' - Label 'MeSH vocabulary'\n",
      "Entity 'http://edamontology.org/data_0832' - Label 'HGNC vocabulary'\n",
      "Entity 'http://edamontology.org/data_0835' - Label 'UMLS vocabulary'\n",
      "Entity 'http://edamontology.org/data_0842' - Label 'Identifier'\n",
      "Entity 'http://edamontology.org/data_0843' - Label 'Database entry'\n",
      "Entity 'http://edamontology.org/data_0844' - Label 'Molecular mass'\n",
      "Entity 'http://edamontology.org/data_0845' - Label 'Molecular charge'\n",
      "Entity 'http://edamontology.org/data_0846' - Label 'Chemical formula'\n",
      "Entity 'http://edamontology.org/data_0847' - Label 'QSAR descriptor'\n",
      "Entity 'http://edamontology.org/data_0848' - Label 'Raw sequence'\n",
      "Entity 'http://edamontology.org/data_0849' - Label 'Sequence record'\n",
      "Entity 'http://edamontology.org/data_0850' - Label 'Sequence set'\n",
      "Entity 'http://edamontology.org/data_0851' - Label 'Sequence mask character'\n",
      "Entity 'http://edamontology.org/data_0852' - Label 'Sequence mask type'\n",
      "Entity 'http://edamontology.org/data_0853' - Label 'DNA sense specification'\n",
      "Entity 'http://edamontology.org/data_0854' - Label 'Sequence length specification'\n",
      "Entity 'http://edamontology.org/data_0855' - Label 'Sequence metadata'\n",
      "Entity 'http://edamontology.org/data_0856' - Label 'Sequence feature source'\n",
      "Entity 'http://edamontology.org/data_0857' - Label 'Sequence search results'\n",
      "Entity 'http://edamontology.org/data_0858' - Label 'Sequence signature matches'\n",
      "Entity 'http://edamontology.org/data_0859' - Label 'Sequence signature model'\n",
      "Entity 'http://edamontology.org/data_0860' - Label 'Sequence signature data'\n",
      "Entity 'http://edamontology.org/data_0861' - Label 'Sequence alignment (words)'\n",
      "Entity 'http://edamontology.org/data_0862' - Label 'Dotplot'\n",
      "Entity 'http://edamontology.org/data_0863' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/data_0864' - Label 'Sequence alignment parameter'\n",
      "Entity 'http://edamontology.org/data_0865' - Label 'Sequence similarity score'\n",
      "Entity 'http://edamontology.org/data_0866' - Label 'Sequence alignment metadata'\n",
      "Entity 'http://edamontology.org/data_0867' - Label 'Sequence alignment report'\n",
      "Entity 'http://edamontology.org/data_0868' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0869' - Label 'Sequence-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0870' - Label 'Sequence distance matrix'\n",
      "Entity 'http://edamontology.org/data_0871' - Label 'Phylogenetic character data'\n",
      "Entity 'http://edamontology.org/data_0872' - Label 'Phylogenetic tree'\n",
      "Entity 'http://edamontology.org/data_0874' - Label 'Comparison matrix'\n",
      "Entity 'http://edamontology.org/data_0875' - Label 'Protein topology'\n",
      "Entity 'http://edamontology.org/data_0876' - Label 'Protein features report (secondary structure)'\n",
      "Entity 'http://edamontology.org/data_0877' - Label 'Protein features report (super-secondary)'\n",
      "Entity 'http://edamontology.org/data_0878' - Label 'Protein secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_0879' - Label 'Secondary structure alignment metadata (protein)'\n",
      "Entity 'http://edamontology.org/data_0880' - Label 'RNA secondary structure'\n",
      "Entity 'http://edamontology.org/data_0881' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_0882' - Label 'Secondary structure alignment metadata (RNA)'\n",
      "Entity 'http://edamontology.org/data_0883' - Label 'Structure'\n",
      "Entity 'http://edamontology.org/data_0884' - Label 'Tertiary structure record'\n",
      "Entity 'http://edamontology.org/data_0885' - Label 'Structure database search results'\n",
      "Entity 'http://edamontology.org/data_0886' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/data_0887' - Label 'Structure alignment report'\n",
      "Entity 'http://edamontology.org/data_0888' - Label 'Structure similarity score'\n",
      "Entity 'http://edamontology.org/data_0889' - Label 'Structural profile'\n",
      "Entity 'http://edamontology.org/data_0890' - Label 'Structural (3D) profile alignment'\n",
      "Entity 'http://edamontology.org/data_0891' - Label 'Sequence-3D profile alignment'\n",
      "Entity 'http://edamontology.org/data_0892' - Label 'Protein sequence-structure scoring matrix'\n",
      "Entity 'http://edamontology.org/data_0893' - Label 'Sequence-structure alignment'\n",
      "Entity 'http://edamontology.org/data_0894' - Label 'Amino acid annotation'\n",
      "Entity 'http://edamontology.org/data_0895' - Label 'Peptide annotation'\n",
      "Entity 'http://edamontology.org/data_0896' - Label 'Protein report'\n",
      "Entity 'http://edamontology.org/data_0897' - Label 'Protein property'\n",
      "Entity 'http://edamontology.org/data_0899' - Label 'Protein structural motifs and surfaces'\n",
      "Entity 'http://edamontology.org/data_0900' - Label 'Protein domain classification'\n",
      "Entity 'http://edamontology.org/data_0901' - Label 'Protein features report (domains)'\n",
      "Entity 'http://edamontology.org/data_0902' - Label 'Protein architecture report'\n",
      "Entity 'http://edamontology.org/data_0903' - Label 'Protein folding report'\n",
      "Entity 'http://edamontology.org/data_0904' - Label 'Protein features (mutation)'\n",
      "Entity 'http://edamontology.org/data_0905' - Label 'Protein interaction raw data'\n",
      "Entity 'http://edamontology.org/data_0906' - Label 'Protein interaction data'\n",
      "Entity 'http://edamontology.org/data_0907' - Label 'Protein family report'\n",
      "Entity 'http://edamontology.org/data_0909' - Label 'Vmax'\n",
      "Entity 'http://edamontology.org/data_0910' - Label 'Km'\n",
      "Entity 'http://edamontology.org/data_0911' - Label 'Nucleotide base annotation'\n",
      "Entity 'http://edamontology.org/data_0912' - Label 'Nucleic acid property'\n",
      "Entity 'http://edamontology.org/data_0914' - Label 'Codon usage data'\n",
      "Entity 'http://edamontology.org/data_0916' - Label 'Gene report'\n",
      "Entity 'http://edamontology.org/data_0917' - Label 'Gene classification'\n",
      "Entity 'http://edamontology.org/data_0918' - Label 'DNA variation'\n",
      "Entity 'http://edamontology.org/data_0919' - Label 'Chromosome report'\n",
      "Entity 'http://edamontology.org/data_0920' - Label 'Genotype/phenotype report'\n",
      "Entity 'http://edamontology.org/data_0923' - Label 'PCR experiment report'\n",
      "Entity 'http://edamontology.org/data_0924' - Label 'Sequence trace'\n",
      "Entity 'http://edamontology.org/data_0925' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/data_0926' - Label 'RH scores'\n",
      "Entity 'http://edamontology.org/data_0927' - Label 'Genetic linkage report'\n",
      "Entity 'http://edamontology.org/data_0928' - Label 'Gene expression profile'\n",
      "Entity 'http://edamontology.org/data_0931' - Label 'Microarray experiment report'\n",
      "Entity 'http://edamontology.org/data_0932' - Label 'Oligonucleotide probe data'\n",
      "Entity 'http://edamontology.org/data_0933' - Label 'SAGE experimental data'\n",
      "Entity 'http://edamontology.org/data_0934' - Label 'MPSS experimental data'\n",
      "Entity 'http://edamontology.org/data_0935' - Label 'SBS experimental data'\n",
      "Entity 'http://edamontology.org/data_0936' - Label 'Sequence tag profile (with gene assignment)'\n",
      "Entity 'http://edamontology.org/data_0937' - Label 'Electron density map'\n",
      "Entity 'http://edamontology.org/data_0938' - Label 'Raw NMR data'\n",
      "Entity 'http://edamontology.org/data_0939' - Label 'CD spectra'\n",
      "Entity 'http://edamontology.org/data_0940' - Label 'Volume map'\n",
      "Entity 'http://edamontology.org/data_0941' - Label 'Electron microscopy model'\n",
      "Entity 'http://edamontology.org/data_0942' - Label '2D PAGE image'\n",
      "Entity 'http://edamontology.org/data_0943' - Label 'Mass spectrum'\n",
      "Entity 'http://edamontology.org/data_0944' - Label 'Peptide mass fingerprint'\n",
      "Entity 'http://edamontology.org/data_0945' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/data_0946' - Label 'Pathway or network annotation'\n",
      "Entity 'http://edamontology.org/data_0947' - Label 'Biological pathway map'\n",
      "Entity 'http://edamontology.org/data_0948' - Label 'Data resource definition'\n",
      "Entity 'http://edamontology.org/data_0949' - Label 'Workflow metadata'\n",
      "Entity 'http://edamontology.org/data_0950' - Label 'Mathematical model'\n",
      "Entity 'http://edamontology.org/data_0951' - Label 'Statistical estimate score'\n",
      "Entity 'http://edamontology.org/data_0952' - Label 'EMBOSS database resource definition'\n",
      "Entity 'http://edamontology.org/data_0953' - Label 'Version information'\n",
      "Entity 'http://edamontology.org/data_0954' - Label 'Database cross-mapping'\n",
      "Entity 'http://edamontology.org/data_0955' - Label 'Data index'\n",
      "Entity 'http://edamontology.org/data_0956' - Label 'Data index report'\n",
      "Entity 'http://edamontology.org/data_0957' - Label 'Database metadata'\n",
      "Entity 'http://edamontology.org/data_0958' - Label 'Tool metadata'\n",
      "Entity 'http://edamontology.org/data_0959' - Label 'Job metadata'\n",
      "Entity 'http://edamontology.org/data_0960' - Label 'User metadata'\n",
      "Entity 'http://edamontology.org/data_0962' - Label 'Small molecule report'\n",
      "Entity 'http://edamontology.org/data_0963' - Label 'Cell line report'\n",
      "Entity 'http://edamontology.org/data_0964' - Label 'Scent annotation'\n",
      "Entity 'http://edamontology.org/data_0966' - Label 'Ontology term'\n",
      "Entity 'http://edamontology.org/data_0967' - Label 'Ontology concept data'\n",
      "Entity 'http://edamontology.org/data_0968' - Label 'Keyword'\n",
      "Entity 'http://edamontology.org/data_0970' - Label 'Citation'\n",
      "Entity 'http://edamontology.org/data_0971' - Label 'Article'\n",
      "Entity 'http://edamontology.org/data_0972' - Label 'Text mining report'\n",
      "Entity 'http://edamontology.org/data_0974' - Label 'Entity identifier'\n",
      "Entity 'http://edamontology.org/data_0975' - Label 'Data resource identifier'\n",
      "Entity 'http://edamontology.org/data_0976' - Label 'Identifier (by type of entity)'\n",
      "Entity 'http://edamontology.org/data_0977' - Label 'Tool identifier'\n",
      "Entity 'http://edamontology.org/data_0978' - Label 'Discrete entity identifier'\n",
      "Entity 'http://edamontology.org/data_0979' - Label 'Entity feature identifier'\n",
      "Entity 'http://edamontology.org/data_0980' - Label 'Entity collection identifier'\n",
      "Entity 'http://edamontology.org/data_0981' - Label 'Phenomenon identifier'\n",
      "Entity 'http://edamontology.org/data_0982' - Label 'Molecule identifier'\n",
      "Entity 'http://edamontology.org/data_0983' - Label 'Atom ID'\n",
      "Entity 'http://edamontology.org/data_0984' - Label 'Molecule name'\n",
      "Entity 'http://edamontology.org/data_0985' - Label 'Molecule type'\n",
      "Entity 'http://edamontology.org/data_0986' - Label 'Chemical identifier'\n",
      "Entity 'http://edamontology.org/data_0987' - Label 'Chromosome name'\n",
      "Entity 'http://edamontology.org/data_0988' - Label 'Peptide identifier'\n",
      "Entity 'http://edamontology.org/data_0989' - Label 'Protein identifier'\n",
      "Entity 'http://edamontology.org/data_0990' - Label 'Compound name'\n",
      "Entity 'http://edamontology.org/data_0991' - Label 'Chemical registry number'\n",
      "Entity 'http://edamontology.org/data_0992' - Label 'Ligand identifier'\n",
      "Entity 'http://edamontology.org/data_0993' - Label 'Drug identifier'\n",
      "Entity 'http://edamontology.org/data_0994' - Label 'Amino acid identifier'\n",
      "Entity 'http://edamontology.org/data_0995' - Label 'Nucleotide identifier'\n",
      "Entity 'http://edamontology.org/data_0996' - Label 'Monosaccharide identifier'\n",
      "Entity 'http://edamontology.org/data_0997' - Label 'Chemical name (ChEBI)'\n",
      "Entity 'http://edamontology.org/data_0998' - Label 'Chemical name (IUPAC)'\n",
      "Entity 'http://edamontology.org/data_0999' - Label 'Chemical name (INN)'\n",
      "Entity 'http://edamontology.org/data_1000' - Label 'Chemical name (brand)'\n",
      "Entity 'http://edamontology.org/data_1001' - Label 'Chemical name (synonymous)'\n",
      "Entity 'http://edamontology.org/data_1002' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_1003' - Label 'Chemical registry number (Beilstein)'\n",
      "Entity 'http://edamontology.org/data_1004' - Label 'Chemical registry number (Gmelin)'\n",
      "Entity 'http://edamontology.org/data_1005' - Label 'HET group name'\n",
      "Entity 'http://edamontology.org/data_1006' - Label 'Amino acid name'\n",
      "Entity 'http://edamontology.org/data_1007' - Label 'Nucleotide code'\n",
      "Entity 'http://edamontology.org/data_1008' - Label 'Polypeptide chain ID'\n",
      "Entity 'http://edamontology.org/data_1009' - Label 'Protein name'\n",
      "Entity 'http://edamontology.org/data_1010' - Label 'Enzyme identifier'\n",
      "Entity 'http://edamontology.org/data_1011' - Label 'EC number'\n",
      "Entity 'http://edamontology.org/data_1012' - Label 'Enzyme name'\n",
      "Entity 'http://edamontology.org/data_1013' - Label 'Restriction enzyme name'\n",
      "Entity 'http://edamontology.org/data_1014' - Label 'Sequence position specification'\n",
      "Entity 'http://edamontology.org/data_1015' - Label 'Sequence feature ID'\n",
      "Entity 'http://edamontology.org/data_1016' - Label 'Sequence position'\n",
      "Entity 'http://edamontology.org/data_1017' - Label 'Sequence range'\n",
      "Entity 'http://edamontology.org/data_1018' - Label 'Nucleic acid feature identifier'\n",
      "Entity 'http://edamontology.org/data_1019' - Label 'Protein feature identifier'\n",
      "Entity 'http://edamontology.org/data_1020' - Label 'Sequence feature key'\n",
      "Entity 'http://edamontology.org/data_1021' - Label 'Sequence feature qualifier'\n",
      "Entity 'http://edamontology.org/data_1022' - Label 'Sequence feature label'\n",
      "Entity 'http://edamontology.org/data_1023' - Label 'EMBOSS Uniform Feature Object'\n",
      "Entity 'http://edamontology.org/data_1024' - Label 'Codon name'\n",
      "Entity 'http://edamontology.org/data_1025' - Label 'Gene identifier'\n",
      "Entity 'http://edamontology.org/data_1026' - Label 'Gene symbol'\n",
      "Entity 'http://edamontology.org/data_1027' - Label 'Gene ID (NCBI)'\n",
      "Entity 'http://edamontology.org/data_1028' - Label 'Gene identifier (NCBI RefSeq)'\n",
      "Entity 'http://edamontology.org/data_1029' - Label 'Gene identifier (NCBI UniGene)'\n",
      "Entity 'http://edamontology.org/data_1030' - Label 'Gene identifier (Entrez)'\n",
      "Entity 'http://edamontology.org/data_1031' - Label 'Gene ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1032' - Label 'Gene ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1033' - Label 'Ensembl gene ID'\n",
      "Entity 'http://edamontology.org/data_1034' - Label 'Gene ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1035' - Label 'Gene ID (GeneDB)'\n",
      "Entity 'http://edamontology.org/data_1036' - Label 'TIGR identifier'\n",
      "Entity 'http://edamontology.org/data_1037' - Label 'TAIR accession (gene)'\n",
      "Entity 'http://edamontology.org/data_1038' - Label 'Protein domain ID'\n",
      "Entity 'http://edamontology.org/data_1039' - Label 'SCOP domain identifier'\n",
      "Entity 'http://edamontology.org/data_1040' - Label 'CATH domain ID'\n",
      "Entity 'http://edamontology.org/data_1041' - Label 'SCOP concise classification string (sccs)'\n",
      "Entity 'http://edamontology.org/data_1042' - Label 'SCOP sunid'\n",
      "Entity 'http://edamontology.org/data_1043' - Label 'CATH node ID'\n",
      "Entity 'http://edamontology.org/data_1044' - Label 'Kingdom name'\n",
      "Entity 'http://edamontology.org/data_1045' - Label 'Species name'\n",
      "Entity 'http://edamontology.org/data_1046' - Label 'Strain name'\n",
      "Entity 'http://edamontology.org/data_1047' - Label 'URI'\n",
      "Entity 'http://edamontology.org/data_1048' - Label 'Database ID'\n",
      "Entity 'http://edamontology.org/data_1049' - Label 'Directory name'\n",
      "Entity 'http://edamontology.org/data_1050' - Label 'File name'\n",
      "Entity 'http://edamontology.org/data_1051' - Label 'Ontology name'\n",
      "Entity 'http://edamontology.org/data_1052' - Label 'URL'\n",
      "Entity 'http://edamontology.org/data_1053' - Label 'URN'\n",
      "Entity 'http://edamontology.org/data_1055' - Label 'LSID'\n",
      "Entity 'http://edamontology.org/data_1056' - Label 'Database name'\n",
      "Entity 'http://edamontology.org/data_1057' - Label 'Sequence database name'\n",
      "Entity 'http://edamontology.org/data_1058' - Label 'Enumerated file name'\n",
      "Entity 'http://edamontology.org/data_1059' - Label 'File name extension'\n",
      "Entity 'http://edamontology.org/data_1060' - Label 'File base name'\n",
      "Entity 'http://edamontology.org/data_1061' - Label 'QSAR descriptor name'\n",
      "Entity 'http://edamontology.org/data_1062' - Label 'Database entry identifier'\n",
      "Entity 'http://edamontology.org/data_1063' - Label 'Sequence identifier'\n",
      "Entity 'http://edamontology.org/data_1064' - Label 'Sequence set ID'\n",
      "Entity 'http://edamontology.org/data_1065' - Label 'Sequence signature identifier'\n",
      "Entity 'http://edamontology.org/data_1066' - Label 'Sequence alignment ID'\n",
      "Entity 'http://edamontology.org/data_1067' - Label 'Phylogenetic distance matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1068' - Label 'Phylogenetic tree ID'\n",
      "Entity 'http://edamontology.org/data_1069' - Label 'Comparison matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1070' - Label 'Structure ID'\n",
      "Entity 'http://edamontology.org/data_1071' - Label 'Structural (3D) profile ID'\n",
      "Entity 'http://edamontology.org/data_1072' - Label 'Structure alignment ID'\n",
      "Entity 'http://edamontology.org/data_1073' - Label 'Amino acid index ID'\n",
      "Entity 'http://edamontology.org/data_1074' - Label 'Protein interaction ID'\n",
      "Entity 'http://edamontology.org/data_1075' - Label 'Protein family identifier'\n",
      "Entity 'http://edamontology.org/data_1076' - Label 'Codon usage table name'\n",
      "Entity 'http://edamontology.org/data_1077' - Label 'Transcription factor identifier'\n",
      "Entity 'http://edamontology.org/data_1078' - Label 'Experiment annotation ID'\n",
      "Entity 'http://edamontology.org/data_1079' - Label 'Electron microscopy model ID'\n",
      "Entity 'http://edamontology.org/data_1080' - Label 'Gene expression report ID'\n",
      "Entity 'http://edamontology.org/data_1081' - Label 'Genotype and phenotype annotation ID'\n",
      "Entity 'http://edamontology.org/data_1082' - Label 'Pathway or network identifier'\n",
      "Entity 'http://edamontology.org/data_1083' - Label 'Workflow ID'\n",
      "Entity 'http://edamontology.org/data_1084' - Label 'Data resource definition ID'\n",
      "Entity 'http://edamontology.org/data_1085' - Label 'Biological model ID'\n",
      "Entity 'http://edamontology.org/data_1086' - Label 'Compound identifier'\n",
      "Entity 'http://edamontology.org/data_1087' - Label 'Ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_1088' - Label 'Article ID'\n",
      "Entity 'http://edamontology.org/data_1089' - Label 'FlyBase ID'\n",
      "Entity 'http://edamontology.org/data_1091' - Label 'WormBase name'\n",
      "Entity 'http://edamontology.org/data_1092' - Label 'WormBase class'\n",
      "Entity 'http://edamontology.org/data_1093' - Label 'Sequence accession'\n",
      "Entity 'http://edamontology.org/data_1094' - Label 'Sequence type'\n",
      "Entity 'http://edamontology.org/data_1095' - Label 'EMBOSS Uniform Sequence Address'\n",
      "Entity 'http://edamontology.org/data_1096' - Label 'Sequence accession (protein)'\n",
      "Entity 'http://edamontology.org/data_1097' - Label 'Sequence accession (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1098' - Label 'RefSeq accession'\n",
      "Entity 'http://edamontology.org/data_1099' - Label 'UniProt accession (extended)'\n",
      "Entity 'http://edamontology.org/data_1100' - Label 'PIR identifier'\n",
      "Entity 'http://edamontology.org/data_1101' - Label 'TREMBL accession'\n",
      "Entity 'http://edamontology.org/data_1102' - Label 'Gramene primary identifier'\n",
      "Entity 'http://edamontology.org/data_1103' - Label 'EMBL/GenBank/DDBJ ID'\n",
      "Entity 'http://edamontology.org/data_1104' - Label 'Sequence cluster ID (UniGene)'\n",
      "Entity 'http://edamontology.org/data_1105' - Label 'dbEST accession'\n",
      "Entity 'http://edamontology.org/data_1106' - Label 'dbSNP ID'\n",
      "Entity 'http://edamontology.org/data_1110' - Label 'EMBOSS sequence type'\n",
      "Entity 'http://edamontology.org/data_1111' - Label 'EMBOSS listfile'\n",
      "Entity 'http://edamontology.org/data_1112' - Label 'Sequence cluster ID'\n",
      "Entity 'http://edamontology.org/data_1113' - Label 'Sequence cluster ID (COG)'\n",
      "Entity 'http://edamontology.org/data_1114' - Label 'Sequence motif identifier'\n",
      "Entity 'http://edamontology.org/data_1115' - Label 'Sequence profile ID'\n",
      "Entity 'http://edamontology.org/data_1116' - Label 'ELM ID'\n",
      "Entity 'http://edamontology.org/data_1117' - Label 'Prosite accession number'\n",
      "Entity 'http://edamontology.org/data_1118' - Label 'HMMER hidden Markov model ID'\n",
      "Entity 'http://edamontology.org/data_1119' - Label 'JASPAR profile ID'\n",
      "Entity 'http://edamontology.org/data_1120' - Label 'Sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1121' - Label 'BLAST sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1122' - Label 'Phylogenetic tree type'\n",
      "Entity 'http://edamontology.org/data_1123' - Label 'TreeBASE study accession number'\n",
      "Entity 'http://edamontology.org/data_1124' - Label 'TreeFam accession number'\n",
      "Entity 'http://edamontology.org/data_1125' - Label 'Comparison matrix type'\n",
      "Entity 'http://edamontology.org/data_1126' - Label 'Comparison matrix name'\n",
      "Entity 'http://edamontology.org/data_1127' - Label 'PDB ID'\n",
      "Entity 'http://edamontology.org/data_1128' - Label 'AAindex ID'\n",
      "Entity 'http://edamontology.org/data_1129' - Label 'BIND accession number'\n",
      "Entity 'http://edamontology.org/data_1130' - Label 'IntAct accession number'\n",
      "Entity 'http://edamontology.org/data_1131' - Label 'Protein family name'\n",
      "Entity 'http://edamontology.org/data_1132' - Label 'InterPro entry name'\n",
      "Entity 'http://edamontology.org/data_1133' - Label 'InterPro accession'\n",
      "Entity 'http://edamontology.org/data_1134' - Label 'InterPro secondary accession'\n",
      "Entity 'http://edamontology.org/data_1135' - Label 'Gene3D ID'\n",
      "Entity 'http://edamontology.org/data_1136' - Label 'PIRSF ID'\n",
      "Entity 'http://edamontology.org/data_1137' - Label 'PRINTS code'\n",
      "Entity 'http://edamontology.org/data_1138' - Label 'Pfam accession number'\n",
      "Entity 'http://edamontology.org/data_1139' - Label 'SMART accession number'\n",
      "Entity 'http://edamontology.org/data_1140' - Label 'Superfamily hidden Markov model number'\n",
      "Entity 'http://edamontology.org/data_1141' - Label 'TIGRFam ID'\n",
      "Entity 'http://edamontology.org/data_1142' - Label 'ProDom accession number'\n",
      "Entity 'http://edamontology.org/data_1143' - Label 'TRANSFAC accession number'\n",
      "Entity 'http://edamontology.org/data_1144' - Label 'ArrayExpress accession number'\n",
      "Entity 'http://edamontology.org/data_1145' - Label 'PRIDE experiment accession number'\n",
      "Entity 'http://edamontology.org/data_1146' - Label 'EMDB ID'\n",
      "Entity 'http://edamontology.org/data_1147' - Label 'GEO accession number'\n",
      "Entity 'http://edamontology.org/data_1148' - Label 'GermOnline ID'\n",
      "Entity 'http://edamontology.org/data_1149' - Label 'EMAGE ID'\n",
      "Entity 'http://edamontology.org/data_1150' - Label 'Disease ID'\n",
      "Entity 'http://edamontology.org/data_1151' - Label 'HGVbase ID'\n",
      "Entity 'http://edamontology.org/data_1152' - Label 'HIVDB identifier'\n",
      "Entity 'http://edamontology.org/data_1153' - Label 'OMIM ID'\n",
      "Entity 'http://edamontology.org/data_1154' - Label 'KEGG object identifier'\n",
      "Entity 'http://edamontology.org/data_1155' - Label 'Pathway ID (reactome)'\n",
      "Entity 'http://edamontology.org/data_1156' - Label 'Pathway ID (aMAZE)'\n",
      "Entity 'http://edamontology.org/data_1157' - Label 'Pathway ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_1158' - Label 'Pathway ID (INOH)'\n",
      "Entity 'http://edamontology.org/data_1159' - Label 'Pathway ID (PATIKA)'\n",
      "Entity 'http://edamontology.org/data_1160' - Label 'Pathway ID (CPDB)'\n",
      "Entity 'http://edamontology.org/data_1161' - Label 'Pathway ID (Panther)'\n",
      "Entity 'http://edamontology.org/data_1162' - Label 'MIRIAM identifier'\n",
      "Entity 'http://edamontology.org/data_1163' - Label 'MIRIAM data type name'\n",
      "Entity 'http://edamontology.org/data_1164' - Label 'MIRIAM URI'\n",
      "Entity 'http://edamontology.org/data_1165' - Label 'MIRIAM data type primary name'\n",
      "Entity 'http://edamontology.org/data_1166' - Label 'MIRIAM data type synonymous name'\n",
      "Entity 'http://edamontology.org/data_1167' - Label 'Taverna workflow ID'\n",
      "Entity 'http://edamontology.org/data_1170' - Label 'Biological model name'\n",
      "Entity 'http://edamontology.org/data_1171' - Label 'BioModel ID'\n",
      "Entity 'http://edamontology.org/data_1172' - Label 'PubChem CID'\n",
      "Entity 'http://edamontology.org/data_1173' - Label 'ChemSpider ID'\n",
      "Entity 'http://edamontology.org/data_1174' - Label 'ChEBI ID'\n",
      "Entity 'http://edamontology.org/data_1175' - Label 'BioPax concept ID'\n",
      "Entity 'http://edamontology.org/data_1176' - Label 'GO concept ID'\n",
      "Entity 'http://edamontology.org/data_1177' - Label 'MeSH concept ID'\n",
      "Entity 'http://edamontology.org/data_1178' - Label 'HGNC concept ID'\n",
      "Entity 'http://edamontology.org/data_1179' - Label 'NCBI taxonomy ID'\n",
      "Entity 'http://edamontology.org/data_1180' - Label 'Plant Ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_1181' - Label 'UMLS concept ID'\n",
      "Entity 'http://edamontology.org/data_1182' - Label 'FMA concept ID'\n",
      "Entity 'http://edamontology.org/data_1183' - Label 'EMAP concept ID'\n",
      "Entity 'http://edamontology.org/data_1184' - Label 'ChEBI concept ID'\n",
      "Entity 'http://edamontology.org/data_1185' - Label 'MGED concept ID'\n",
      "Entity 'http://edamontology.org/data_1186' - Label 'myGrid concept ID'\n",
      "Entity 'http://edamontology.org/data_1187' - Label 'PubMed ID'\n",
      "Entity 'http://edamontology.org/data_1188' - Label 'DOI'\n",
      "Entity 'http://edamontology.org/data_1189' - Label 'Medline UI'\n",
      "Entity 'http://edamontology.org/data_1190' - Label 'Tool name'\n",
      "Entity 'http://edamontology.org/data_1191' - Label 'Tool name (signature)'\n",
      "Entity 'http://edamontology.org/data_1192' - Label 'Tool name (BLAST)'\n",
      "Entity 'http://edamontology.org/data_1193' - Label 'Tool name (FASTA)'\n",
      "Entity 'http://edamontology.org/data_1194' - Label 'Tool name (EMBOSS)'\n",
      "Entity 'http://edamontology.org/data_1195' - Label 'Tool name (EMBASSY package)'\n",
      "Entity 'http://edamontology.org/data_1201' - Label 'QSAR descriptor (constitutional)'\n",
      "Entity 'http://edamontology.org/data_1202' - Label 'QSAR descriptor (electronic)'\n",
      "Entity 'http://edamontology.org/data_1203' - Label 'QSAR descriptor (geometrical)'\n",
      "Entity 'http://edamontology.org/data_1204' - Label 'QSAR descriptor (topological)'\n",
      "Entity 'http://edamontology.org/data_1205' - Label 'QSAR descriptor (molecular)'\n",
      "Entity 'http://edamontology.org/data_1233' - Label 'Sequence set (protein)'\n",
      "Entity 'http://edamontology.org/data_1234' - Label 'Sequence set (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1235' - Label 'Sequence cluster'\n",
      "Entity 'http://edamontology.org/data_1236' - Label 'Psiblast checkpoint file'\n",
      "Entity 'http://edamontology.org/data_1237' - Label 'HMMER synthetic sequences set'\n",
      "Entity 'http://edamontology.org/data_1238' - Label 'Proteolytic digest'\n",
      "Entity 'http://edamontology.org/data_1239' - Label 'Restriction digest'\n",
      "Entity 'http://edamontology.org/data_1240' - Label 'PCR primers'\n",
      "Entity 'http://edamontology.org/data_1241' - Label 'vectorstrip cloning vector definition file'\n",
      "Entity 'http://edamontology.org/data_1242' - Label 'Primer3 internal oligo mishybridizing library'\n",
      "Entity 'http://edamontology.org/data_1243' - Label 'Primer3 mispriming library file'\n",
      "Entity 'http://edamontology.org/data_1244' - Label 'primersearch primer pairs sequence record'\n",
      "Entity 'http://edamontology.org/data_1245' - Label 'Sequence cluster (protein)'\n",
      "Entity 'http://edamontology.org/data_1246' - Label 'Sequence cluster (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1249' - Label 'Sequence length'\n",
      "Entity 'http://edamontology.org/data_1250' - Label 'Word size'\n",
      "Entity 'http://edamontology.org/data_1251' - Label 'Window size'\n",
      "Entity 'http://edamontology.org/data_1252' - Label 'Sequence length range'\n",
      "Entity 'http://edamontology.org/data_1253' - Label 'Sequence information report'\n",
      "Entity 'http://edamontology.org/data_1254' - Label 'Sequence property'\n",
      "Entity 'http://edamontology.org/data_1255' - Label 'Sequence features'\n",
      "Entity 'http://edamontology.org/data_1256' - Label 'Sequence features (comparative)'\n",
      "Entity 'http://edamontology.org/data_1257' - Label 'Sequence property (protein)'\n",
      "Entity 'http://edamontology.org/data_1258' - Label 'Sequence property (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1259' - Label 'Sequence complexity report'\n",
      "Entity 'http://edamontology.org/data_1260' - Label 'Sequence ambiguity report'\n",
      "Entity 'http://edamontology.org/data_1261' - Label 'Sequence composition report'\n",
      "Entity 'http://edamontology.org/data_1262' - Label 'Peptide molecular weight hits'\n",
      "Entity 'http://edamontology.org/data_1263' - Label 'Base position variability plot'\n",
      "Entity 'http://edamontology.org/data_1264' - Label 'Sequence composition table'\n",
      "Entity 'http://edamontology.org/data_1265' - Label 'Base frequencies table'\n",
      "Entity 'http://edamontology.org/data_1266' - Label 'Base word frequencies table'\n",
      "Entity 'http://edamontology.org/data_1267' - Label 'Amino acid frequencies table'\n",
      "Entity 'http://edamontology.org/data_1268' - Label 'Amino acid word frequencies table'\n",
      "Entity 'http://edamontology.org/data_1269' - Label 'DAS sequence feature annotation'\n",
      "Entity 'http://edamontology.org/data_1270' - Label 'Feature table'\n",
      "Entity 'http://edamontology.org/data_1274' - Label 'Map'\n",
      "Entity 'http://edamontology.org/data_1276' - Label 'Nucleic acid features'\n",
      "Entity 'http://edamontology.org/data_1277' - Label 'Protein features'\n",
      "Entity 'http://edamontology.org/data_1278' - Label 'Genetic map'\n",
      "Entity 'http://edamontology.org/data_1279' - Label 'Sequence map'\n",
      "Entity 'http://edamontology.org/data_1280' - Label 'Physical map'\n",
      "Entity 'http://edamontology.org/data_1281' - Label 'Sequence signature map'\n",
      "Entity 'http://edamontology.org/data_1283' - Label 'Cytogenetic map'\n",
      "Entity 'http://edamontology.org/data_1284' - Label 'DNA transduction map'\n",
      "Entity 'http://edamontology.org/data_1285' - Label 'Gene map'\n",
      "Entity 'http://edamontology.org/data_1286' - Label 'Plasmid map'\n",
      "Entity 'http://edamontology.org/data_1288' - Label 'Genome map'\n",
      "Entity 'http://edamontology.org/data_1289' - Label 'Restriction map'\n",
      "Entity 'http://edamontology.org/data_1290' - Label 'InterPro compact match image'\n",
      "Entity 'http://edamontology.org/data_1291' - Label 'InterPro detailed match image'\n",
      "Entity 'http://edamontology.org/data_1292' - Label 'InterPro architecture image'\n",
      "Entity 'http://edamontology.org/data_1293' - Label 'SMART protein schematic'\n",
      "Entity 'http://edamontology.org/data_1294' - Label 'GlobPlot domain image'\n",
      "Entity 'http://edamontology.org/data_1298' - Label 'Sequence motif matches'\n",
      "Entity 'http://edamontology.org/data_1299' - Label 'Sequence features (repeats)'\n",
      "Entity 'http://edamontology.org/data_1300' - Label 'Gene and transcript structure (report)'\n",
      "Entity 'http://edamontology.org/data_1301' - Label 'Mobile genetic elements'\n",
      "Entity 'http://edamontology.org/data_1303' - Label 'Nucleic acid features (quadruplexes)'\n",
      "Entity 'http://edamontology.org/data_1306' - Label 'Nucleosome exclusion sequences'\n",
      "Entity 'http://edamontology.org/data_1309' - Label 'Gene features (exonic splicing enhancer)'\n",
      "Entity 'http://edamontology.org/data_1310' - Label 'Nucleic acid features (microRNA)'\n",
      "Entity 'http://edamontology.org/data_1313' - Label 'Coding region'\n",
      "Entity 'http://edamontology.org/data_1314' - Label 'Gene features (SECIS element)'\n",
      "Entity 'http://edamontology.org/data_1315' - Label 'Transcription factor binding sites'\n",
      "Entity 'http://edamontology.org/data_1321' - Label 'Protein features (sites)'\n",
      "Entity 'http://edamontology.org/data_1322' - Label 'Protein features report (signal peptides)'\n",
      "Entity 'http://edamontology.org/data_1323' - Label 'Protein features report (cleavage sites)'\n",
      "Entity 'http://edamontology.org/data_1324' - Label 'Protein features (post-translation modifications)'\n",
      "Entity 'http://edamontology.org/data_1325' - Label 'Protein features report (active sites)'\n",
      "Entity 'http://edamontology.org/data_1326' - Label 'Protein features report (binding sites)'\n",
      "Entity 'http://edamontology.org/data_1327' - Label 'Protein features (epitopes)'\n",
      "Entity 'http://edamontology.org/data_1328' - Label 'Protein features report (nucleic acid binding sites)'\n",
      "Entity 'http://edamontology.org/data_1329' - Label 'MHC Class I epitopes report'\n",
      "Entity 'http://edamontology.org/data_1330' - Label 'MHC Class II epitopes report'\n",
      "Entity 'http://edamontology.org/data_1331' - Label 'Protein features (PEST sites)'\n",
      "Entity 'http://edamontology.org/data_1338' - Label 'Sequence database hits scores list'\n",
      "Entity 'http://edamontology.org/data_1339' - Label 'Sequence database hits alignments list'\n",
      "Entity 'http://edamontology.org/data_1340' - Label 'Sequence database hits evaluation data'\n",
      "Entity 'http://edamontology.org/data_1344' - Label 'MEME motif alphabet'\n",
      "Entity 'http://edamontology.org/data_1345' - Label 'MEME background frequencies file'\n",
      "Entity 'http://edamontology.org/data_1346' - Label 'MEME motifs directive file'\n",
      "Entity 'http://edamontology.org/data_1347' - Label 'Dirichlet distribution'\n",
      "Entity 'http://edamontology.org/data_1348' - Label 'HMM emission and transition counts'\n",
      "Entity 'http://edamontology.org/data_1352' - Label 'Regular expression'\n",
      "Entity 'http://edamontology.org/data_1353' - Label 'Sequence motif'\n",
      "Entity 'http://edamontology.org/data_1354' - Label 'Sequence profile'\n",
      "Entity 'http://edamontology.org/data_1355' - Label 'Protein signature'\n",
      "Entity 'http://edamontology.org/data_1358' - Label 'Prosite nucleotide pattern'\n",
      "Entity 'http://edamontology.org/data_1359' - Label 'Prosite protein pattern'\n",
      "Entity 'http://edamontology.org/data_1361' - Label 'Position frequency matrix'\n",
      "Entity 'http://edamontology.org/data_1362' - Label 'Position weight matrix'\n",
      "Entity 'http://edamontology.org/data_1363' - Label 'Information content matrix'\n",
      "Entity 'http://edamontology.org/data_1364' - Label 'Hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_1365' - Label 'Fingerprint'\n",
      "Entity 'http://edamontology.org/data_1368' - Label 'Domainatrix signature'\n",
      "Entity 'http://edamontology.org/data_1371' - Label 'HMMER NULL hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_1372' - Label 'Protein family signature'\n",
      "Entity 'http://edamontology.org/data_1373' - Label 'Protein domain signature'\n",
      "Entity 'http://edamontology.org/data_1374' - Label 'Protein region signature'\n",
      "Entity 'http://edamontology.org/data_1375' - Label 'Protein repeat signature'\n",
      "Entity 'http://edamontology.org/data_1376' - Label 'Protein site signature'\n",
      "Entity 'http://edamontology.org/data_1377' - Label 'Protein conserved site signature'\n",
      "Entity 'http://edamontology.org/data_1378' - Label 'Protein active site signature'\n",
      "Entity 'http://edamontology.org/data_1379' - Label 'Protein binding site signature'\n",
      "Entity 'http://edamontology.org/data_1380' - Label 'Protein post-translational modification signature'\n",
      "Entity 'http://edamontology.org/data_1381' - Label 'Pair sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1382' - Label 'Sequence alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1383' - Label 'Nucleic acid sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1384' - Label 'Protein sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1385' - Label 'Hybrid sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1386' - Label 'Sequence alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1387' - Label 'Sequence alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1388' - Label 'Hybrid sequence alignment (pair)'\n",
      "Entity 'http://edamontology.org/data_1389' - Label 'Multiple nucleotide sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1390' - Label 'Multiple protein sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1394' - Label 'Alignment score or penalty'\n",
      "Entity 'http://edamontology.org/data_1395' - Label 'Score end gaps control'\n",
      "Entity 'http://edamontology.org/data_1396' - Label 'Aligned sequence order'\n",
      "Entity 'http://edamontology.org/data_1397' - Label 'Gap opening penalty'\n",
      "Entity 'http://edamontology.org/data_1398' - Label 'Gap extension penalty'\n",
      "Entity 'http://edamontology.org/data_1399' - Label 'Gap separation penalty'\n",
      "Entity 'http://edamontology.org/data_1400' - Label 'Terminal gap penalty'\n",
      "Entity 'http://edamontology.org/data_1401' - Label 'Match reward score'\n",
      "Entity 'http://edamontology.org/data_1402' - Label 'Mismatch penalty score'\n",
      "Entity 'http://edamontology.org/data_1403' - Label 'Drop off score'\n",
      "Entity 'http://edamontology.org/data_1404' - Label 'Gap opening penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1405' - Label 'Gap opening penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1406' - Label 'Gap extension penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1407' - Label 'Gap extension penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1408' - Label 'Gap separation penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1409' - Label 'Gap separation penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1410' - Label 'Terminal gap opening penalty'\n",
      "Entity 'http://edamontology.org/data_1411' - Label 'Terminal gap extension penalty'\n",
      "Entity 'http://edamontology.org/data_1412' - Label 'Sequence identity'\n",
      "Entity 'http://edamontology.org/data_1413' - Label 'Sequence similarity'\n",
      "Entity 'http://edamontology.org/data_1414' - Label 'Sequence alignment metadata (quality report)'\n",
      "Entity 'http://edamontology.org/data_1415' - Label 'Sequence alignment report (site conservation)'\n",
      "Entity 'http://edamontology.org/data_1416' - Label 'Sequence alignment report (site correlation)'\n",
      "Entity 'http://edamontology.org/data_1417' - Label 'Sequence-profile alignment (Domainatrix signature)'\n",
      "Entity 'http://edamontology.org/data_1418' - Label 'Sequence-profile alignment (HMM)'\n",
      "Entity 'http://edamontology.org/data_1420' - Label 'Sequence-profile alignment (fingerprint)'\n",
      "Entity 'http://edamontology.org/data_1426' - Label 'Phylogenetic continuous quantitative data'\n",
      "Entity 'http://edamontology.org/data_1427' - Label 'Phylogenetic discrete data'\n",
      "Entity 'http://edamontology.org/data_1428' - Label 'Phylogenetic character cliques'\n",
      "Entity 'http://edamontology.org/data_1429' - Label 'Phylogenetic invariants'\n",
      "Entity 'http://edamontology.org/data_1438' - Label 'Phylogenetic report'\n",
      "Entity 'http://edamontology.org/data_1439' - Label 'DNA substitution model'\n",
      "Entity 'http://edamontology.org/data_1440' - Label 'Phylogenetic tree report (tree shape)'\n",
      "Entity 'http://edamontology.org/data_1441' - Label 'Phylogenetic tree report (tree evaluation)'\n",
      "Entity 'http://edamontology.org/data_1442' - Label 'Phylogenetic tree distances'\n",
      "Entity 'http://edamontology.org/data_1443' - Label 'Phylogenetic tree report (tree stratigraphic)'\n",
      "Entity 'http://edamontology.org/data_1444' - Label 'Phylogenetic character contrasts'\n",
      "Entity 'http://edamontology.org/data_1446' - Label 'Comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1447' - Label 'Comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1448' - Label 'Comparison matrix (nucleotide)'\n",
      "Entity 'http://edamontology.org/data_1449' - Label 'Comparison matrix (amino acid)'\n",
      "Entity 'http://edamontology.org/data_1450' - Label 'Nucleotide comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1451' - Label 'Nucleotide comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1452' - Label 'Amino acid comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1453' - Label 'Amino acid comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1459' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/data_1460' - Label 'Protein structure'\n",
      "Entity 'http://edamontology.org/data_1461' - Label 'Protein-ligand complex'\n",
      "Entity 'http://edamontology.org/data_1462' - Label 'Carbohydrate structure'\n",
      "Entity 'http://edamontology.org/data_1463' - Label 'Small molecule structure'\n",
      "Entity 'http://edamontology.org/data_1464' - Label 'DNA structure'\n",
      "Entity 'http://edamontology.org/data_1465' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/data_1466' - Label 'tRNA structure'\n",
      "Entity 'http://edamontology.org/data_1467' - Label 'Protein chain'\n",
      "Entity 'http://edamontology.org/data_1468' - Label 'Protein domain'\n",
      "Entity 'http://edamontology.org/data_1469' - Label 'Protein structure (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1470' - Label 'C-alpha trace'\n",
      "Entity 'http://edamontology.org/data_1471' - Label 'Protein chain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1472' - Label 'Protein chain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1473' - Label 'Protein domain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1474' - Label 'Protein domain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1479' - Label 'Structure alignment (pair)'\n",
      "Entity 'http://edamontology.org/data_1480' - Label 'Structure alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1481' - Label 'Protein structure alignment'\n",
      "Entity 'http://edamontology.org/data_1482' - Label 'Nucleic acid structure alignment'\n",
      "Entity 'http://edamontology.org/data_1483' - Label 'Structure alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1484' - Label 'Multiple protein tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1485' - Label 'Structure alignment (protein all atoms)'\n",
      "Entity 'http://edamontology.org/data_1486' - Label 'Structure alignment (protein C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1487' - Label 'Pairwise protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1488' - Label 'Pairwise protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1489' - Label 'Multiple protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1490' - Label 'Multiple protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1491' - Label 'Structure alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1492' - Label 'Multiple nucleic acid tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1493' - Label 'RNA structure alignment'\n",
      "Entity 'http://edamontology.org/data_1494' - Label 'Structural transformation matrix'\n",
      "Entity 'http://edamontology.org/data_1495' - Label 'DaliLite hit table'\n",
      "Entity 'http://edamontology.org/data_1496' - Label 'Molecular similarity score'\n",
      "Entity 'http://edamontology.org/data_1497' - Label 'Root-mean-square deviation'\n",
      "Entity 'http://edamontology.org/data_1498' - Label 'Tanimoto similarity score'\n",
      "Entity 'http://edamontology.org/data_1499' - Label '3D-1D scoring matrix'\n",
      "Entity 'http://edamontology.org/data_1501' - Label 'Amino acid index'\n",
      "Entity 'http://edamontology.org/data_1502' - Label 'Amino acid index (chemical classes)'\n",
      "Entity 'http://edamontology.org/data_1503' - Label 'Amino acid pair-wise contact potentials'\n",
      "Entity 'http://edamontology.org/data_1505' - Label 'Amino acid index (molecular weight)'\n",
      "Entity 'http://edamontology.org/data_1506' - Label 'Amino acid index (hydropathy)'\n",
      "Entity 'http://edamontology.org/data_1507' - Label 'Amino acid index (White-Wimley data)'\n",
      "Entity 'http://edamontology.org/data_1508' - Label 'Amino acid index (van der Waals radii)'\n",
      "Entity 'http://edamontology.org/data_1509' - Label 'Enzyme report'\n",
      "Entity 'http://edamontology.org/data_1517' - Label 'Restriction enzyme report'\n",
      "Entity 'http://edamontology.org/data_1519' - Label 'Peptide molecular weights'\n",
      "Entity 'http://edamontology.org/data_1520' - Label 'Peptide hydrophobic moment'\n",
      "Entity 'http://edamontology.org/data_1521' - Label 'Protein aliphatic index'\n",
      "Entity 'http://edamontology.org/data_1522' - Label 'Protein sequence hydropathy plot'\n",
      "Entity 'http://edamontology.org/data_1523' - Label 'Protein charge plot'\n",
      "Entity 'http://edamontology.org/data_1524' - Label 'Protein solubility'\n",
      "Entity 'http://edamontology.org/data_1525' - Label 'Protein crystallizability'\n",
      "Entity 'http://edamontology.org/data_1526' - Label 'Protein globularity'\n",
      "Entity 'http://edamontology.org/data_1527' - Label 'Protein titration curve'\n",
      "Entity 'http://edamontology.org/data_1528' - Label 'Protein isoelectric point'\n",
      "Entity 'http://edamontology.org/data_1529' - Label 'Protein pKa value'\n",
      "Entity 'http://edamontology.org/data_1530' - Label 'Protein hydrogen exchange rate'\n",
      "Entity 'http://edamontology.org/data_1531' - Label 'Protein extinction coefficient'\n",
      "Entity 'http://edamontology.org/data_1532' - Label 'Protein optical density'\n",
      "Entity 'http://edamontology.org/data_1533' - Label 'Protein subcellular localisation'\n",
      "Entity 'http://edamontology.org/data_1534' - Label 'Peptide immunogenicity data'\n",
      "Entity 'http://edamontology.org/data_1536' - Label 'MHC peptide immunogenicity report'\n",
      "Entity 'http://edamontology.org/data_1537' - Label 'Protein structure report'\n",
      "Entity 'http://edamontology.org/data_1539' - Label 'Protein structural quality report'\n",
      "Entity 'http://edamontology.org/data_1540' - Label 'Protein non-covalent interactions report'\n",
      "Entity 'http://edamontology.org/data_1541' - Label 'Protein flexibility or motion report'\n",
      "Entity 'http://edamontology.org/data_1542' - Label 'Protein solvent accessibility'\n",
      "Entity 'http://edamontology.org/data_1543' - Label 'Protein surface report'\n",
      "Entity 'http://edamontology.org/data_1544' - Label 'Ramachandran plot'\n",
      "Entity 'http://edamontology.org/data_1545' - Label 'Protein dipole moment'\n",
      "Entity 'http://edamontology.org/data_1546' - Label 'Protein distance matrix'\n",
      "Entity 'http://edamontology.org/data_1547' - Label 'Protein contact map'\n",
      "Entity 'http://edamontology.org/data_1548' - Label 'Protein residue 3D cluster'\n",
      "Entity 'http://edamontology.org/data_1549' - Label 'Protein hydrogen bonds'\n",
      "Entity 'http://edamontology.org/data_1550' - Label 'Protein non-canonical interactions'\n",
      "Entity 'http://edamontology.org/data_1553' - Label 'CATH node'\n",
      "Entity 'http://edamontology.org/data_1554' - Label 'SCOP node'\n",
      "Entity 'http://edamontology.org/data_1555' - Label 'EMBASSY domain classification'\n",
      "Entity 'http://edamontology.org/data_1556' - Label 'CATH class'\n",
      "Entity 'http://edamontology.org/data_1557' - Label 'CATH architecture'\n",
      "Entity 'http://edamontology.org/data_1558' - Label 'CATH topology'\n",
      "Entity 'http://edamontology.org/data_1559' - Label 'CATH homologous superfamily'\n",
      "Entity 'http://edamontology.org/data_1560' - Label 'CATH structurally similar group'\n",
      "Entity 'http://edamontology.org/data_1561' - Label 'CATH functional category'\n",
      "Entity 'http://edamontology.org/data_1564' - Label 'Protein fold recognition report'\n",
      "Entity 'http://edamontology.org/data_1565' - Label 'Protein-protein interaction report'\n",
      "Entity 'http://edamontology.org/data_1566' - Label 'Protein-ligand interaction report'\n",
      "Entity 'http://edamontology.org/data_1567' - Label 'Protein-nucleic acid interactions report'\n",
      "Entity 'http://edamontology.org/data_1583' - Label 'Nucleic acid melting profile'\n",
      "Entity 'http://edamontology.org/data_1584' - Label 'Nucleic acid enthalpy'\n",
      "Entity 'http://edamontology.org/data_1585' - Label 'Nucleic acid entropy'\n",
      "Entity 'http://edamontology.org/data_1586' - Label 'Nucleic acid melting temperature'\n",
      "Entity 'http://edamontology.org/data_1587' - Label 'Nucleic acid stitch profile'\n",
      "Entity 'http://edamontology.org/data_1588' - Label 'DNA base pair stacking energies data'\n",
      "Entity 'http://edamontology.org/data_1589' - Label 'DNA base pair twist angle data'\n",
      "Entity 'http://edamontology.org/data_1590' - Label 'DNA base trimer roll angles data'\n",
      "Entity 'http://edamontology.org/data_1591' - Label 'Vienna RNA parameters'\n",
      "Entity 'http://edamontology.org/data_1592' - Label 'Vienna RNA structure constraints'\n",
      "Entity 'http://edamontology.org/data_1593' - Label 'Vienna RNA concentration data'\n",
      "Entity 'http://edamontology.org/data_1594' - Label 'Vienna RNA calculated energy'\n",
      "Entity 'http://edamontology.org/data_1595' - Label 'Base pairing probability matrix dotplot'\n",
      "Entity 'http://edamontology.org/data_1596' - Label 'Nucleic acid folding report'\n",
      "Entity 'http://edamontology.org/data_1597' - Label 'Codon usage table'\n",
      "Entity 'http://edamontology.org/data_1598' - Label 'Genetic code'\n",
      "Entity 'http://edamontology.org/data_1599' - Label 'Codon adaptation index'\n",
      "Entity 'http://edamontology.org/data_1600' - Label 'Codon usage bias plot'\n",
      "Entity 'http://edamontology.org/data_1601' - Label 'Nc statistic'\n",
      "Entity 'http://edamontology.org/data_1602' - Label 'Codon usage fraction difference'\n",
      "Entity 'http://edamontology.org/data_1621' - Label 'Pharmacogenomic test report'\n",
      "Entity 'http://edamontology.org/data_1622' - Label 'Disease report'\n",
      "Entity 'http://edamontology.org/data_1634' - Label 'Linkage disequilibrium (report)'\n",
      "Entity 'http://edamontology.org/data_1636' - Label 'Heat map'\n",
      "Entity 'http://edamontology.org/data_1642' - Label 'Affymetrix probe sets library file'\n",
      "Entity 'http://edamontology.org/data_1643' - Label 'Affymetrix probe sets information library file'\n",
      "Entity 'http://edamontology.org/data_1646' - Label 'Molecular weights standard fingerprint'\n",
      "Entity 'http://edamontology.org/data_1656' - Label 'Metabolic pathway report'\n",
      "Entity 'http://edamontology.org/data_1657' - Label 'Genetic information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1658' - Label 'Environmental information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1659' - Label 'Signal transduction pathway report'\n",
      "Entity 'http://edamontology.org/data_1660' - Label 'Cellular process pathways report'\n",
      "Entity 'http://edamontology.org/data_1661' - Label 'Disease pathway or network report'\n",
      "Entity 'http://edamontology.org/data_1662' - Label 'Drug structure relationship map'\n",
      "Entity 'http://edamontology.org/data_1663' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/data_1664' - Label 'MIRIAM datatype'\n",
      "Entity 'http://edamontology.org/data_1667' - Label 'E-value'\n",
      "Entity 'http://edamontology.org/data_1668' - Label 'Z-value'\n",
      "Entity 'http://edamontology.org/data_1669' - Label 'P-value'\n",
      "Entity 'http://edamontology.org/data_1670' - Label 'Database version information'\n",
      "Entity 'http://edamontology.org/data_1671' - Label 'Tool version information'\n",
      "Entity 'http://edamontology.org/data_1672' - Label 'CATH version information'\n",
      "Entity 'http://edamontology.org/data_1673' - Label 'Swiss-Prot to PDB mapping'\n",
      "Entity 'http://edamontology.org/data_1674' - Label 'Sequence database cross-references'\n",
      "Entity 'http://edamontology.org/data_1675' - Label 'Job status'\n",
      "Entity 'http://edamontology.org/data_1676' - Label 'Job ID'\n",
      "Entity 'http://edamontology.org/data_1677' - Label 'Job type'\n",
      "Entity 'http://edamontology.org/data_1678' - Label 'Tool log'\n",
      "Entity 'http://edamontology.org/data_1679' - Label 'DaliLite log file'\n",
      "Entity 'http://edamontology.org/data_1680' - Label 'STRIDE log file'\n",
      "Entity 'http://edamontology.org/data_1681' - Label 'NACCESS log file'\n",
      "Entity 'http://edamontology.org/data_1682' - Label 'EMBOSS wordfinder log file'\n",
      "Entity 'http://edamontology.org/data_1683' - Label 'EMBOSS domainatrix log file'\n",
      "Entity 'http://edamontology.org/data_1684' - Label 'EMBOSS sites log file'\n",
      "Entity 'http://edamontology.org/data_1685' - Label 'EMBOSS supermatcher error file'\n",
      "Entity 'http://edamontology.org/data_1686' - Label 'EMBOSS megamerger log file'\n",
      "Entity 'http://edamontology.org/data_1687' - Label 'EMBOSS whichdb log file'\n",
      "Entity 'http://edamontology.org/data_1688' - Label 'EMBOSS vectorstrip log file'\n",
      "Entity 'http://edamontology.org/data_1689' - Label 'Username'\n",
      "Entity 'http://edamontology.org/data_1690' - Label 'Password'\n",
      "Entity 'http://edamontology.org/data_1691' - Label 'Email address'\n",
      "Entity 'http://edamontology.org/data_1692' - Label 'Person name'\n",
      "Entity 'http://edamontology.org/data_1693' - Label 'Number of iterations'\n",
      "Entity 'http://edamontology.org/data_1694' - Label 'Number of output entities'\n",
      "Entity 'http://edamontology.org/data_1695' - Label 'Hit sort order'\n",
      "Entity 'http://edamontology.org/data_1696' - Label 'Drug report'\n",
      "Entity 'http://edamontology.org/data_1707' - Label 'Phylogenetic tree image'\n",
      "Entity 'http://edamontology.org/data_1708' - Label 'RNA secondary structure image'\n",
      "Entity 'http://edamontology.org/data_1709' - Label 'Protein secondary structure image'\n",
      "Entity 'http://edamontology.org/data_1710' - Label 'Structure image'\n",
      "Entity 'http://edamontology.org/data_1711' - Label 'Sequence alignment image'\n",
      "Entity 'http://edamontology.org/data_1712' - Label 'Chemical structure image'\n",
      "Entity 'http://edamontology.org/data_1713' - Label 'Fate map'\n",
      "Entity 'http://edamontology.org/data_1714' - Label 'Microarray spots image'\n",
      "Entity 'http://edamontology.org/data_1715' - Label 'BioPax term'\n",
      "Entity 'http://edamontology.org/data_1716' - Label 'GO'\n",
      "Entity 'http://edamontology.org/data_1717' - Label 'MeSH'\n",
      "Entity 'http://edamontology.org/data_1718' - Label 'HGNC'\n",
      "Entity 'http://edamontology.org/data_1719' - Label 'NCBI taxonomy vocabulary'\n",
      "Entity 'http://edamontology.org/data_1720' - Label 'Plant ontology term'\n",
      "Entity 'http://edamontology.org/data_1721' - Label 'UMLS'\n",
      "Entity 'http://edamontology.org/data_1722' - Label 'FMA'\n",
      "Entity 'http://edamontology.org/data_1723' - Label 'EMAP'\n",
      "Entity 'http://edamontology.org/data_1724' - Label 'ChEBI'\n",
      "Entity 'http://edamontology.org/data_1725' - Label 'MGED'\n",
      "Entity 'http://edamontology.org/data_1726' - Label 'myGrid'\n",
      "Entity 'http://edamontology.org/data_1727' - Label 'GO (biological process)'\n",
      "Entity 'http://edamontology.org/data_1728' - Label 'GO (molecular function)'\n",
      "Entity 'http://edamontology.org/data_1729' - Label 'GO (cellular component)'\n",
      "Entity 'http://edamontology.org/data_1730' - Label 'Ontology relation type'\n",
      "Entity 'http://edamontology.org/data_1731' - Label 'Ontology concept definition'\n",
      "Entity 'http://edamontology.org/data_1732' - Label 'Ontology concept comment'\n",
      "Entity 'http://edamontology.org/data_1733' - Label 'Ontology concept reference'\n",
      "Entity 'http://edamontology.org/data_1738' - Label 'doc2loc document information'\n",
      "Entity 'http://edamontology.org/data_1742' - Label 'PDB residue number'\n",
      "Entity 'http://edamontology.org/data_1743' - Label 'Atomic coordinate'\n",
      "Entity 'http://edamontology.org/data_1744' - Label 'Atomic x coordinate'\n",
      "Entity 'http://edamontology.org/data_1745' - Label 'Atomic y coordinate'\n",
      "Entity 'http://edamontology.org/data_1746' - Label 'Atomic z coordinate'\n",
      "Entity 'http://edamontology.org/data_1748' - Label 'PDB atom name'\n",
      "Entity 'http://edamontology.org/data_1755' - Label 'Protein atom'\n",
      "Entity 'http://edamontology.org/data_1756' - Label 'Protein residue'\n",
      "Entity 'http://edamontology.org/data_1757' - Label 'Atom name'\n",
      "Entity 'http://edamontology.org/data_1758' - Label 'PDB residue name'\n",
      "Entity 'http://edamontology.org/data_1759' - Label 'PDB model number'\n",
      "Entity 'http://edamontology.org/data_1762' - Label 'CATH domain report'\n",
      "Entity 'http://edamontology.org/data_1764' - Label 'CATH representative domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1765' - Label 'CATH representative domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1766' - Label 'CATH domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1767' - Label 'CATH domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1771' - Label 'Sequence version'\n",
      "Entity 'http://edamontology.org/data_1772' - Label 'Score'\n",
      "Entity 'http://edamontology.org/data_1776' - Label 'Protein report (function)'\n",
      "Entity 'http://edamontology.org/data_1783' - Label 'Gene name (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1784' - Label 'Gene name (CGD)'\n",
      "Entity 'http://edamontology.org/data_1785' - Label 'Gene name (dictyBase)'\n",
      "Entity 'http://edamontology.org/data_1786' - Label 'Gene name (EcoGene primary)'\n",
      "Entity 'http://edamontology.org/data_1787' - Label 'Gene name (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1788' - Label 'Gene name (SGD)'\n",
      "Entity 'http://edamontology.org/data_1789' - Label 'Gene name (TGD)'\n",
      "Entity 'http://edamontology.org/data_1790' - Label 'Gene name (CGSC)'\n",
      "Entity 'http://edamontology.org/data_1791' - Label 'Gene name (HGNC)'\n",
      "Entity 'http://edamontology.org/data_1792' - Label 'Gene name (MGD)'\n",
      "Entity 'http://edamontology.org/data_1793' - Label 'Gene name (Bacillus subtilis)'\n",
      "Entity 'http://edamontology.org/data_1794' - Label 'Gene ID (PlasmoDB)'\n",
      "Entity 'http://edamontology.org/data_1795' - Label 'Gene ID (EcoGene)'\n",
      "Entity 'http://edamontology.org/data_1796' - Label 'Gene ID (FlyBase)'\n",
      "Entity 'http://edamontology.org/data_1797' - Label 'Gene ID (GeneDB Glossina morsitans)'\n",
      "Entity 'http://edamontology.org/data_1798' - Label 'Gene ID (GeneDB Leishmania major)'\n",
      "Entity 'http://edamontology.org/data_1799' - Label 'Gene ID (GeneDB Plasmodium falciparum)'\n",
      "Entity 'http://edamontology.org/data_1800' - Label 'Gene ID (GeneDB Schizosaccharomyces pombe)'\n",
      "Entity 'http://edamontology.org/data_1801' - Label 'Gene ID (GeneDB Trypanosoma brucei)'\n",
      "Entity 'http://edamontology.org/data_1802' - Label 'Gene ID (Gramene)'\n",
      "Entity 'http://edamontology.org/data_1803' - Label 'Gene ID (Virginia microbial)'\n",
      "Entity 'http://edamontology.org/data_1804' - Label 'Gene ID (SGN)'\n",
      "Entity 'http://edamontology.org/data_1805' - Label 'Gene ID (WormBase)'\n",
      "Entity 'http://edamontology.org/data_1806' - Label 'Gene synonym'\n",
      "Entity 'http://edamontology.org/data_1807' - Label 'ORF name'\n",
      "Entity 'http://edamontology.org/data_1852' - Label 'Sequence assembly component'\n",
      "Entity 'http://edamontology.org/data_1853' - Label 'Chromosome annotation (aberration)'\n",
      "Entity 'http://edamontology.org/data_1855' - Label 'Clone ID'\n",
      "Entity 'http://edamontology.org/data_1856' - Label 'PDB insertion code'\n",
      "Entity 'http://edamontology.org/data_1857' - Label 'Atomic occupancy'\n",
      "Entity 'http://edamontology.org/data_1858' - Label 'Isotropic B factor'\n",
      "Entity 'http://edamontology.org/data_1859' - Label 'Deletion map'\n",
      "Entity 'http://edamontology.org/data_1860' - Label 'QTL map'\n",
      "Entity 'http://edamontology.org/data_1863' - Label 'Haplotype map'\n",
      "Entity 'http://edamontology.org/data_1864' - Label 'Map set data'\n",
      "Entity 'http://edamontology.org/data_1865' - Label 'Map feature'\n",
      "Entity 'http://edamontology.org/data_1866' - Label 'Map type'\n",
      "Entity 'http://edamontology.org/data_1867' - Label 'Protein fold name'\n",
      "Entity 'http://edamontology.org/data_1868' - Label 'Taxon'\n",
      "Entity 'http://edamontology.org/data_1869' - Label 'Organism identifier'\n",
      "Entity 'http://edamontology.org/data_1870' - Label 'Genus name'\n",
      "Entity 'http://edamontology.org/data_1872' - Label 'Taxonomic classification'\n",
      "Entity 'http://edamontology.org/data_1873' - Label 'iHOP organism ID'\n",
      "Entity 'http://edamontology.org/data_1874' - Label 'Genbank common name'\n",
      "Entity 'http://edamontology.org/data_1875' - Label 'NCBI taxon'\n",
      "Entity 'http://edamontology.org/data_1877' - Label 'Synonym'\n",
      "Entity 'http://edamontology.org/data_1878' - Label 'Misspelling'\n",
      "Entity 'http://edamontology.org/data_1879' - Label 'Acronym'\n",
      "Entity 'http://edamontology.org/data_1880' - Label 'Misnomer'\n",
      "Entity 'http://edamontology.org/data_1881' - Label 'Author ID'\n",
      "Entity 'http://edamontology.org/data_1882' - Label 'DragonDB author identifier'\n",
      "Entity 'http://edamontology.org/data_1883' - Label 'Annotated URI'\n",
      "Entity 'http://edamontology.org/data_1884' - Label 'UniProt keywords'\n",
      "Entity 'http://edamontology.org/data_1885' - Label 'Gene ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1886' - Label 'Blattner number'\n",
      "Entity 'http://edamontology.org/data_1887' - Label 'Gene ID (MIPS Maize)'\n",
      "Entity 'http://edamontology.org/data_1888' - Label 'Gene ID (MIPS Medicago)'\n",
      "Entity 'http://edamontology.org/data_1889' - Label 'Gene name (DragonDB)'\n",
      "Entity 'http://edamontology.org/data_1890' - Label 'Gene name (Arabidopsis)'\n",
      "Entity 'http://edamontology.org/data_1891' - Label 'iHOP symbol'\n",
      "Entity 'http://edamontology.org/data_1892' - Label 'Gene name (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1893' - Label 'Locus ID'\n",
      "Entity 'http://edamontology.org/data_1895' - Label 'Locus ID (AGI)'\n",
      "Entity 'http://edamontology.org/data_1896' - Label 'Locus ID (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1897' - Label 'Locus ID (MGG)'\n",
      "Entity 'http://edamontology.org/data_1898' - Label 'Locus ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1899' - Label 'Locus ID (CMR)'\n",
      "Entity 'http://edamontology.org/data_1900' - Label 'NCBI locus tag'\n",
      "Entity 'http://edamontology.org/data_1901' - Label 'Locus ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1902' - Label 'Locus ID (MMP)'\n",
      "Entity 'http://edamontology.org/data_1903' - Label 'Locus ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1904' - Label 'Locus ID (EntrezGene)'\n",
      "Entity 'http://edamontology.org/data_1905' - Label 'Locus ID (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1906' - Label 'Quantitative trait locus'\n",
      "Entity 'http://edamontology.org/data_1907' - Label 'Gene ID (KOME)'\n",
      "Entity 'http://edamontology.org/data_1908' - Label 'Locus ID (Tropgene)'\n",
      "Entity 'http://edamontology.org/data_1916' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/data_1917' - Label 'Atomic property'\n",
      "Entity 'http://edamontology.org/data_2007' - Label 'UniProt keyword'\n",
      "Entity 'http://edamontology.org/data_2009' - Label 'Ordered locus name'\n",
      "Entity 'http://edamontology.org/data_2012' - Label 'Sequence coordinates'\n",
      "Entity 'http://edamontology.org/data_2016' - Label 'Amino acid property'\n",
      "Entity 'http://edamontology.org/data_2018' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/data_2019' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2022' - Label 'Vienna RNA structural data'\n",
      "Entity 'http://edamontology.org/data_2023' - Label 'Sequence mask parameter'\n",
      "Entity 'http://edamontology.org/data_2024' - Label 'Enzyme kinetics data'\n",
      "Entity 'http://edamontology.org/data_2025' - Label 'Michaelis Menten plot'\n",
      "Entity 'http://edamontology.org/data_2026' - Label 'Hanes Woolf plot'\n",
      "Entity 'http://edamontology.org/data_2028' - Label 'Experimental data'\n",
      "Entity 'http://edamontology.org/data_2041' - Label 'Genome version information'\n",
      "Entity 'http://edamontology.org/data_2042' - Label 'Evidence'\n",
      "Entity 'http://edamontology.org/data_2043' - Label 'Sequence record lite'\n",
      "Entity 'http://edamontology.org/data_2044' - Label 'Sequence'\n",
      "Entity 'http://edamontology.org/data_2046' - Label 'Nucleic acid sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2047' - Label 'Protein sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2048' - Label 'Report'\n",
      "Entity 'http://edamontology.org/data_2050' - Label 'Molecular property (general)'\n",
      "Entity 'http://edamontology.org/data_2053' - Label 'Structural data'\n",
      "Entity 'http://edamontology.org/data_2070' - Label 'Sequence motif (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_2071' - Label 'Sequence motif (protein)'\n",
      "Entity 'http://edamontology.org/data_2079' - Label 'Search parameter'\n",
      "Entity 'http://edamontology.org/data_2080' - Label 'Database search results'\n",
      "Entity 'http://edamontology.org/data_2081' - Label 'Secondary structure'\n",
      "Entity 'http://edamontology.org/data_2082' - Label 'Matrix'\n",
      "Entity 'http://edamontology.org/data_2083' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2084' - Label 'Nucleic acid report'\n",
      "Entity 'http://edamontology.org/data_2085' - Label 'Structure report'\n",
      "Entity 'http://edamontology.org/data_2086' - Label 'Nucleic acid structure data'\n",
      "Entity 'http://edamontology.org/data_2087' - Label 'Molecular property'\n",
      "Entity 'http://edamontology.org/data_2088' - Label 'DNA base structural data'\n",
      "Entity 'http://edamontology.org/data_2090' - Label 'Database entry version information'\n",
      "Entity 'http://edamontology.org/data_2091' - Label 'Accession'\n",
      "Entity 'http://edamontology.org/data_2092' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/data_2093' - Label 'Data reference'\n",
      "Entity 'http://edamontology.org/data_2098' - Label 'Job identifier'\n",
      "Entity 'http://edamontology.org/data_2099' - Label 'Name'\n",
      "Entity 'http://edamontology.org/data_2100' - Label 'Type'\n",
      "Entity 'http://edamontology.org/data_2101' - Label 'Account authentication'\n",
      "Entity 'http://edamontology.org/data_2102' - Label 'KEGG organism code'\n",
      "Entity 'http://edamontology.org/data_2103' - Label 'Gene name (KEGG GENES)'\n",
      "Entity 'http://edamontology.org/data_2104' - Label 'BioCyc ID'\n",
      "Entity 'http://edamontology.org/data_2105' - Label 'Compound ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2106' - Label 'Reaction ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2107' - Label 'Enzyme ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2108' - Label 'Reaction ID'\n",
      "Entity 'http://edamontology.org/data_2109' - Label 'Identifier (hybrid)'\n",
      "Entity 'http://edamontology.org/data_2110' - Label 'Molecular property identifier'\n",
      "Entity 'http://edamontology.org/data_2111' - Label 'Codon usage table ID'\n",
      "Entity 'http://edamontology.org/data_2112' - Label 'FlyBase primary identifier'\n",
      "Entity 'http://edamontology.org/data_2113' - Label 'WormBase identifier'\n",
      "Entity 'http://edamontology.org/data_2114' - Label 'WormBase wormpep ID'\n",
      "Entity 'http://edamontology.org/data_2116' - Label 'Nucleic acid features (codon)'\n",
      "Entity 'http://edamontology.org/data_2117' - Label 'Map identifier'\n",
      "Entity 'http://edamontology.org/data_2118' - Label 'Person identifier'\n",
      "Entity 'http://edamontology.org/data_2119' - Label 'Nucleic acid identifier'\n",
      "Entity 'http://edamontology.org/data_2126' - Label 'Translation frame specification'\n",
      "Entity 'http://edamontology.org/data_2127' - Label 'Genetic code identifier'\n",
      "Entity 'http://edamontology.org/data_2128' - Label 'Genetic code name'\n",
      "Entity 'http://edamontology.org/data_2129' - Label 'File format name'\n",
      "Entity 'http://edamontology.org/data_2130' - Label 'Sequence profile type'\n",
      "Entity 'http://edamontology.org/data_2131' - Label 'Operating system name'\n",
      "Entity 'http://edamontology.org/data_2132' - Label 'Mutation type'\n",
      "Entity 'http://edamontology.org/data_2133' - Label 'Logical operator'\n",
      "Entity 'http://edamontology.org/data_2134' - Label 'Results sort order'\n",
      "Entity 'http://edamontology.org/data_2135' - Label 'Toggle'\n",
      "Entity 'http://edamontology.org/data_2136' - Label 'Sequence width'\n",
      "Entity 'http://edamontology.org/data_2137' - Label 'Gap penalty'\n",
      "Entity 'http://edamontology.org/data_2139' - Label 'Nucleic acid melting temperature'\n",
      "Entity 'http://edamontology.org/data_2140' - Label 'Concentration'\n",
      "Entity 'http://edamontology.org/data_2141' - Label 'Window step size'\n",
      "Entity 'http://edamontology.org/data_2142' - Label 'EMBOSS graph'\n",
      "Entity 'http://edamontology.org/data_2143' - Label 'EMBOSS report'\n",
      "Entity 'http://edamontology.org/data_2145' - Label 'Sequence offset'\n",
      "Entity 'http://edamontology.org/data_2146' - Label 'Threshold'\n",
      "Entity 'http://edamontology.org/data_2147' - Label 'Protein report (transcription factor)'\n",
      "Entity 'http://edamontology.org/data_2149' - Label 'Database category name'\n",
      "Entity 'http://edamontology.org/data_2150' - Label 'Sequence profile name'\n",
      "Entity 'http://edamontology.org/data_2151' - Label 'Color'\n",
      "Entity 'http://edamontology.org/data_2152' - Label 'Rendering parameter'\n",
      "Entity 'http://edamontology.org/data_2154' - Label 'Sequence name'\n",
      "Entity 'http://edamontology.org/data_2156' - Label 'Date'\n",
      "Entity 'http://edamontology.org/data_2157' - Label 'Word composition'\n",
      "Entity 'http://edamontology.org/data_2160' - Label 'Fickett testcode plot'\n",
      "Entity 'http://edamontology.org/data_2161' - Label 'Sequence similarity plot'\n",
      "Entity 'http://edamontology.org/data_2162' - Label 'Helical wheel'\n",
      "Entity 'http://edamontology.org/data_2163' - Label 'Helical net'\n",
      "Entity 'http://edamontology.org/data_2164' - Label 'Protein sequence properties plot'\n",
      "Entity 'http://edamontology.org/data_2165' - Label 'Protein ionisation curve'\n",
      "Entity 'http://edamontology.org/data_2166' - Label 'Sequence composition plot'\n",
      "Entity 'http://edamontology.org/data_2167' - Label 'Nucleic acid density plot'\n",
      "Entity 'http://edamontology.org/data_2168' - Label 'Sequence trace image'\n",
      "Entity 'http://edamontology.org/data_2169' - Label 'Nucleic acid features (siRNA)'\n",
      "Entity 'http://edamontology.org/data_2173' - Label 'Sequence set (stream)'\n",
      "Entity 'http://edamontology.org/data_2174' - Label 'FlyBase secondary identifier'\n",
      "Entity 'http://edamontology.org/data_2176' - Label 'Cardinality'\n",
      "Entity 'http://edamontology.org/data_2177' - Label 'Exactly 1'\n",
      "Entity 'http://edamontology.org/data_2178' - Label '1 or more'\n",
      "Entity 'http://edamontology.org/data_2179' - Label 'Exactly 2'\n",
      "Entity 'http://edamontology.org/data_2180' - Label '2 or more'\n",
      "Entity 'http://edamontology.org/data_2190' - Label 'Sequence checksum'\n",
      "Entity 'http://edamontology.org/data_2191' - Label 'Protein features report (chemical modifications)'\n",
      "Entity 'http://edamontology.org/data_2192' - Label 'Error'\n",
      "Entity 'http://edamontology.org/data_2193' - Label 'Database entry metadata'\n",
      "Entity 'http://edamontology.org/data_2198' - Label 'Gene cluster'\n",
      "Entity 'http://edamontology.org/data_2201' - Label 'Sequence record full'\n",
      "Entity 'http://edamontology.org/data_2208' - Label 'Plasmid identifier'\n",
      "Entity 'http://edamontology.org/data_2209' - Label 'Mutation ID'\n",
      "Entity 'http://edamontology.org/data_2212' - Label 'Mutation annotation (basic)'\n",
      "Entity 'http://edamontology.org/data_2213' - Label 'Mutation annotation (prevalence)'\n",
      "Entity 'http://edamontology.org/data_2214' - Label 'Mutation annotation (prognostic)'\n",
      "Entity 'http://edamontology.org/data_2215' - Label 'Mutation annotation (functional)'\n",
      "Entity 'http://edamontology.org/data_2216' - Label 'Codon number'\n",
      "Entity 'http://edamontology.org/data_2217' - Label 'Tumor annotation'\n",
      "Entity 'http://edamontology.org/data_2218' - Label 'Server metadata'\n",
      "Entity 'http://edamontology.org/data_2219' - Label 'Database field name'\n",
      "Entity 'http://edamontology.org/data_2220' - Label 'Sequence cluster ID (SYSTERS)'\n",
      "Entity 'http://edamontology.org/data_2223' - Label 'Ontology metadata'\n",
      "Entity 'http://edamontology.org/data_2235' - Label 'Raw SCOP domain classification'\n",
      "Entity 'http://edamontology.org/data_2236' - Label 'Raw CATH domain classification'\n",
      "Entity 'http://edamontology.org/data_2240' - Label 'Heterogen annotation'\n",
      "Entity 'http://edamontology.org/data_2242' - Label 'Phylogenetic property values'\n",
      "Entity 'http://edamontology.org/data_2245' - Label 'Sequence set (bootstrapped)'\n",
      "Entity 'http://edamontology.org/data_2247' - Label 'Phylogenetic consensus tree'\n",
      "Entity 'http://edamontology.org/data_2248' - Label 'Schema'\n",
      "Entity 'http://edamontology.org/data_2249' - Label 'DTD'\n",
      "Entity 'http://edamontology.org/data_2250' - Label 'XML Schema'\n",
      "Entity 'http://edamontology.org/data_2251' - Label 'Relax-NG schema'\n",
      "Entity 'http://edamontology.org/data_2252' - Label 'XSLT stylesheet'\n",
      "Entity 'http://edamontology.org/data_2253' - Label 'Data resource definition name'\n",
      "Entity 'http://edamontology.org/data_2254' - Label 'OBO file format name'\n",
      "Entity 'http://edamontology.org/data_2285' - Label 'Gene ID (MIPS)'\n",
      "Entity 'http://edamontology.org/data_2288' - Label 'Sequence identifier (protein)'\n",
      "Entity 'http://edamontology.org/data_2289' - Label 'Sequence identifier (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_2290' - Label 'EMBL accession'\n",
      "Entity 'http://edamontology.org/data_2291' - Label 'UniProt ID'\n",
      "Entity 'http://edamontology.org/data_2292' - Label 'GenBank accession'\n",
      "Entity 'http://edamontology.org/data_2293' - Label 'Gramene secondary identifier'\n",
      "Entity 'http://edamontology.org/data_2294' - Label 'Sequence variation ID'\n",
      "Entity 'http://edamontology.org/data_2295' - Label 'Gene ID'\n",
      "Entity 'http://edamontology.org/data_2296' - Label 'Gene name (AceView)'\n",
      "Entity 'http://edamontology.org/data_2297' - Label 'Gene ID (ECK)'\n",
      "Entity 'http://edamontology.org/data_2298' - Label 'Gene ID (HGNC)'\n",
      "Entity 'http://edamontology.org/data_2299' - Label 'Gene name'\n",
      "Entity 'http://edamontology.org/data_2300' - Label 'Gene name (NCBI)'\n",
      "Entity 'http://edamontology.org/data_2301' - Label 'SMILES string'\n",
      "Entity 'http://edamontology.org/data_2302' - Label 'STRING ID'\n",
      "Entity 'http://edamontology.org/data_2307' - Label 'Virus annotation'\n",
      "Entity 'http://edamontology.org/data_2308' - Label 'Virus annotation (taxonomy)'\n",
      "Entity 'http://edamontology.org/data_2309' - Label 'Reaction ID (SABIO-RK)'\n",
      "Entity 'http://edamontology.org/data_2313' - Label 'Carbohydrate report'\n",
      "Entity 'http://edamontology.org/data_2314' - Label 'GI number'\n",
      "Entity 'http://edamontology.org/data_2315' - Label 'NCBI version'\n",
      "Entity 'http://edamontology.org/data_2316' - Label 'Cell line name'\n",
      "Entity 'http://edamontology.org/data_2317' - Label 'Cell line name (exact)'\n",
      "Entity 'http://edamontology.org/data_2318' - Label 'Cell line name (truncated)'\n",
      "Entity 'http://edamontology.org/data_2319' - Label 'Cell line name (no punctuation)'\n",
      "Entity 'http://edamontology.org/data_2320' - Label 'Cell line name (assonant)'\n",
      "Entity 'http://edamontology.org/data_2321' - Label 'Enzyme ID'\n",
      "Entity 'http://edamontology.org/data_2325' - Label 'REBASE enzyme number'\n",
      "Entity 'http://edamontology.org/data_2326' - Label 'DrugBank ID'\n",
      "Entity 'http://edamontology.org/data_2327' - Label 'GI number (protein)'\n",
      "Entity 'http://edamontology.org/data_2335' - Label 'Bit score'\n",
      "Entity 'http://edamontology.org/data_2336' - Label 'Translation phase specification'\n",
      "Entity 'http://edamontology.org/data_2337' - Label 'Resource metadata'\n",
      "Entity 'http://edamontology.org/data_2338' - Label 'Ontology identifier'\n",
      "Entity 'http://edamontology.org/data_2339' - Label 'Ontology concept name'\n",
      "Entity 'http://edamontology.org/data_2340' - Label 'Genome build identifier'\n",
      "Entity 'http://edamontology.org/data_2342' - Label 'Pathway or network name'\n",
      "Entity 'http://edamontology.org/data_2343' - Label 'Pathway ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2344' - Label 'Pathway ID (NCI-Nature)'\n",
      "Entity 'http://edamontology.org/data_2345' - Label 'Pathway ID (ConsensusPathDB)'\n",
      "Entity 'http://edamontology.org/data_2346' - Label 'Sequence cluster ID (UniRef)'\n",
      "Entity 'http://edamontology.org/data_2347' - Label 'Sequence cluster ID (UniRef100)'\n",
      "Entity 'http://edamontology.org/data_2348' - Label 'Sequence cluster ID (UniRef90)'\n",
      "Entity 'http://edamontology.org/data_2349' - Label 'Sequence cluster ID (UniRef50)'\n",
      "Entity 'http://edamontology.org/data_2353' - Label 'Ontology data'\n",
      "Entity 'http://edamontology.org/data_2354' - Label 'RNA family report'\n",
      "Entity 'http://edamontology.org/data_2355' - Label 'RNA family identifier'\n",
      "Entity 'http://edamontology.org/data_2356' - Label 'RFAM accession'\n",
      "Entity 'http://edamontology.org/data_2357' - Label 'Protein signature type'\n",
      "Entity 'http://edamontology.org/data_2358' - Label 'Domain-nucleic acid interaction report'\n",
      "Entity 'http://edamontology.org/data_2359' - Label 'Domain-domain interactions'\n",
      "Entity 'http://edamontology.org/data_2360' - Label 'Domain-domain interaction (indirect)'\n",
      "Entity 'http://edamontology.org/data_2362' - Label 'Sequence accession (hybrid)'\n",
      "Entity 'http://edamontology.org/data_2363' - Label '2D PAGE data'\n",
      "Entity 'http://edamontology.org/data_2364' - Label '2D PAGE report'\n",
      "Entity 'http://edamontology.org/data_2365' - Label 'Pathway or network accession'\n",
      "Entity 'http://edamontology.org/data_2366' - Label 'Secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_2367' - Label 'ASTD ID'\n",
      "Entity 'http://edamontology.org/data_2368' - Label 'ASTD ID (exon)'\n",
      "Entity 'http://edamontology.org/data_2369' - Label 'ASTD ID (intron)'\n",
      "Entity 'http://edamontology.org/data_2370' - Label 'ASTD ID (polya)'\n",
      "Entity 'http://edamontology.org/data_2371' - Label 'ASTD ID (tss)'\n",
      "Entity 'http://edamontology.org/data_2372' - Label '2D PAGE spot report'\n",
      "Entity 'http://edamontology.org/data_2373' - Label 'Spot ID'\n",
      "Entity 'http://edamontology.org/data_2374' - Label 'Spot serial number'\n",
      "Entity 'http://edamontology.org/data_2375' - Label 'Spot ID (HSC-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2378' - Label 'Protein-motif interaction'\n",
      "Entity 'http://edamontology.org/data_2379' - Label 'Strain identifier'\n",
      "Entity 'http://edamontology.org/data_2380' - Label 'CABRI accession'\n",
      "Entity 'http://edamontology.org/data_2381' - Label 'Experiment report (genotyping)'\n",
      "Entity 'http://edamontology.org/data_2382' - Label 'Genotype experiment ID'\n",
      "Entity 'http://edamontology.org/data_2383' - Label 'EGA accession'\n",
      "Entity 'http://edamontology.org/data_2384' - Label 'IPI protein ID'\n",
      "Entity 'http://edamontology.org/data_2385' - Label 'RefSeq accession (protein)'\n",
      "Entity 'http://edamontology.org/data_2386' - Label 'EPD ID'\n",
      "Entity 'http://edamontology.org/data_2387' - Label 'TAIR accession'\n",
      "Entity 'http://edamontology.org/data_2388' - Label 'TAIR accession (At gene)'\n",
      "Entity 'http://edamontology.org/data_2389' - Label 'UniSTS accession'\n",
      "Entity 'http://edamontology.org/data_2390' - Label 'UNITE accession'\n",
      "Entity 'http://edamontology.org/data_2391' - Label 'UTR accession'\n",
      "Entity 'http://edamontology.org/data_2392' - Label 'UniParc accession'\n",
      "Entity 'http://edamontology.org/data_2393' - Label 'mFLJ/mKIAA number'\n",
      "Entity 'http://edamontology.org/data_2395' - Label 'Fungi annotation'\n",
      "Entity 'http://edamontology.org/data_2396' - Label 'Fungi annotation (anamorph)'\n",
      "Entity 'http://edamontology.org/data_2398' - Label 'Ensembl protein ID'\n",
      "Entity 'http://edamontology.org/data_2400' - Label 'Toxin annotation'\n",
      "Entity 'http://edamontology.org/data_2401' - Label 'Protein report (membrane protein)'\n",
      "Entity 'http://edamontology.org/data_2402' - Label 'Protein-drug interaction report'\n",
      "Entity 'http://edamontology.org/data_2522' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2523' - Label 'Phylogenetic data'\n",
      "Entity 'http://edamontology.org/data_2524' - Label 'Protein data'\n",
      "Entity 'http://edamontology.org/data_2525' - Label 'Nucleic acid data'\n",
      "Entity 'http://edamontology.org/data_2526' - Label 'Text data'\n",
      "Entity 'http://edamontology.org/data_2527' - Label 'Parameter'\n",
      "Entity 'http://edamontology.org/data_2528' - Label 'Molecular data'\n",
      "Entity 'http://edamontology.org/data_2529' - Label 'Molecule report'\n",
      "Entity 'http://edamontology.org/data_2530' - Label 'Organism report'\n",
      "Entity 'http://edamontology.org/data_2531' - Label 'Protocol'\n",
      "Entity 'http://edamontology.org/data_2534' - Label 'Sequence attribute'\n",
      "Entity 'http://edamontology.org/data_2535' - Label 'Sequence tag profile'\n",
      "Entity 'http://edamontology.org/data_2536' - Label 'Mass spectrometry data'\n",
      "Entity 'http://edamontology.org/data_2537' - Label 'Protein structure raw data'\n",
      "Entity 'http://edamontology.org/data_2538' - Label 'Mutation identifier'\n",
      "Entity 'http://edamontology.org/data_2539' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2540' - Label 'Data index data'\n",
      "Entity 'http://edamontology.org/data_2563' - Label 'Amino acid name (single letter)'\n",
      "Entity 'http://edamontology.org/data_2564' - Label 'Amino acid name (three letter)'\n",
      "Entity 'http://edamontology.org/data_2565' - Label 'Amino acid name (full name)'\n",
      "Entity 'http://edamontology.org/data_2576' - Label 'Toxin identifier'\n",
      "Entity 'http://edamontology.org/data_2578' - Label 'ArachnoServer ID'\n",
      "Entity 'http://edamontology.org/data_2579' - Label 'Expressed gene list'\n",
      "Entity 'http://edamontology.org/data_2580' - Label 'BindingDB Monomer ID'\n",
      "Entity 'http://edamontology.org/data_2581' - Label 'GO concept name'\n",
      "Entity 'http://edamontology.org/data_2582' - Label 'GO concept ID (biological process)'\n",
      "Entity 'http://edamontology.org/data_2583' - Label 'GO concept ID (molecular function)'\n",
      "Entity 'http://edamontology.org/data_2584' - Label 'GO concept name (cellular component)'\n",
      "Entity 'http://edamontology.org/data_2586' - Label 'Northern blot image'\n",
      "Entity 'http://edamontology.org/data_2587' - Label 'Blot ID'\n",
      "Entity 'http://edamontology.org/data_2588' - Label 'BlotBase blot ID'\n",
      "Entity 'http://edamontology.org/data_2589' - Label 'Hierarchy'\n",
      "Entity 'http://edamontology.org/data_2590' - Label 'Hierarchy identifier'\n",
      "Entity 'http://edamontology.org/data_2591' - Label 'Brite hierarchy ID'\n",
      "Entity 'http://edamontology.org/data_2592' - Label 'Cancer type'\n",
      "Entity 'http://edamontology.org/data_2593' - Label 'BRENDA organism ID'\n",
      "Entity 'http://edamontology.org/data_2594' - Label 'UniGene taxon'\n",
      "Entity 'http://edamontology.org/data_2595' - Label 'UTRdb taxon'\n",
      "Entity 'http://edamontology.org/data_2596' - Label 'Catalogue ID'\n",
      "Entity 'http://edamontology.org/data_2597' - Label 'CABRI catalogue name'\n",
      "Entity 'http://edamontology.org/data_2598' - Label 'Secondary structure alignment metadata'\n",
      "Entity 'http://edamontology.org/data_2599' - Label 'Molecule interaction report'\n",
      "Entity 'http://edamontology.org/data_2600' - Label 'Pathway or network'\n",
      "Entity 'http://edamontology.org/data_2601' - Label 'Small molecule data'\n",
      "Entity 'http://edamontology.org/data_2602' - Label 'Genotype and phenotype data'\n",
      "Entity 'http://edamontology.org/data_2603' - Label 'Expression data'\n",
      "Entity 'http://edamontology.org/data_2605' - Label 'Compound ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2606' - Label 'RFAM name'\n",
      "Entity 'http://edamontology.org/data_2608' - Label 'Reaction ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2609' - Label 'Drug ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2610' - Label 'Ensembl ID'\n",
      "Entity 'http://edamontology.org/data_2611' - Label 'ICD identifier'\n",
      "Entity 'http://edamontology.org/data_2612' - Label 'Sequence cluster ID (CluSTr)'\n",
      "Entity 'http://edamontology.org/data_2613' - Label 'KEGG Glycan ID'\n",
      "Entity 'http://edamontology.org/data_2614' - Label 'TCDB ID'\n",
      "Entity 'http://edamontology.org/data_2615' - Label 'MINT ID'\n",
      "Entity 'http://edamontology.org/data_2616' - Label 'DIP ID'\n",
      "Entity 'http://edamontology.org/data_2617' - Label 'Signaling Gateway protein ID'\n",
      "Entity 'http://edamontology.org/data_2618' - Label 'Protein modification ID'\n",
      "Entity 'http://edamontology.org/data_2619' - Label 'RESID ID'\n",
      "Entity 'http://edamontology.org/data_2620' - Label 'RGD ID'\n",
      "Entity 'http://edamontology.org/data_2621' - Label 'TAIR accession (protein)'\n",
      "Entity 'http://edamontology.org/data_2622' - Label 'Compound ID (HMDB)'\n",
      "Entity 'http://edamontology.org/data_2625' - Label 'LIPID MAPS ID'\n",
      "Entity 'http://edamontology.org/data_2626' - Label 'PeptideAtlas ID'\n",
      "Entity 'http://edamontology.org/data_2627' - Label 'Molecular interaction ID'\n",
      "Entity 'http://edamontology.org/data_2628' - Label 'BioGRID interaction ID'\n",
      "Entity 'http://edamontology.org/data_2629' - Label 'Enzyme ID (MEROPS)'\n",
      "Entity 'http://edamontology.org/data_2630' - Label 'Mobile genetic element ID'\n",
      "Entity 'http://edamontology.org/data_2631' - Label 'ACLAME ID'\n",
      "Entity 'http://edamontology.org/data_2632' - Label 'SGD ID'\n",
      "Entity 'http://edamontology.org/data_2633' - Label 'Book ID'\n",
      "Entity 'http://edamontology.org/data_2634' - Label 'ISBN'\n",
      "Entity 'http://edamontology.org/data_2635' - Label 'Compound ID (3DMET)'\n",
      "Entity 'http://edamontology.org/data_2636' - Label 'MatrixDB interaction ID'\n",
      "Entity 'http://edamontology.org/data_2637' - Label 'cPath ID'\n",
      "Entity 'http://edamontology.org/data_2638' - Label 'PubChem bioassay ID'\n",
      "Entity 'http://edamontology.org/data_2639' - Label 'PubChem ID'\n",
      "Entity 'http://edamontology.org/data_2641' - Label 'Reaction ID (MACie)'\n",
      "Entity 'http://edamontology.org/data_2642' - Label 'Gene ID (miRBase)'\n",
      "Entity 'http://edamontology.org/data_2643' - Label 'Gene ID (ZFIN)'\n",
      "Entity 'http://edamontology.org/data_2644' - Label 'Reaction ID (Rhea)'\n",
      "Entity 'http://edamontology.org/data_2645' - Label 'Pathway ID (Unipathway)'\n",
      "Entity 'http://edamontology.org/data_2646' - Label 'Compound ID (ChEMBL)'\n",
      "Entity 'http://edamontology.org/data_2647' - Label 'LGICdb identifier'\n",
      "Entity 'http://edamontology.org/data_2648' - Label 'Reaction kinetics ID (SABIO-RK)'\n",
      "Entity 'http://edamontology.org/data_2649' - Label 'PharmGKB ID'\n",
      "Entity 'http://edamontology.org/data_2650' - Label 'Pathway ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2651' - Label 'Disease ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2652' - Label 'Drug ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2653' - Label 'Drug ID (TTD)'\n",
      "Entity 'http://edamontology.org/data_2654' - Label 'Target ID (TTD)'\n",
      "Entity 'http://edamontology.org/data_2655' - Label 'Cell type identifier'\n",
      "Entity 'http://edamontology.org/data_2656' - Label 'NeuronDB ID'\n",
      "Entity 'http://edamontology.org/data_2657' - Label 'NeuroMorpho ID'\n",
      "Entity 'http://edamontology.org/data_2658' - Label 'Compound ID (ChemIDplus)'\n",
      "Entity 'http://edamontology.org/data_2659' - Label 'Pathway ID (SMPDB)'\n",
      "Entity 'http://edamontology.org/data_2660' - Label 'BioNumbers ID'\n",
      "Entity 'http://edamontology.org/data_2662' - Label 'T3DB ID'\n",
      "Entity 'http://edamontology.org/data_2663' - Label 'Carbohydrate identifier'\n",
      "Entity 'http://edamontology.org/data_2664' - Label 'GlycomeDB ID'\n",
      "Entity 'http://edamontology.org/data_2665' - Label 'LipidBank ID'\n",
      "Entity 'http://edamontology.org/data_2666' - Label 'CDD ID'\n",
      "Entity 'http://edamontology.org/data_2667' - Label 'MMDB ID'\n",
      "Entity 'http://edamontology.org/data_2668' - Label 'iRefIndex ID'\n",
      "Entity 'http://edamontology.org/data_2669' - Label 'ModelDB ID'\n",
      "Entity 'http://edamontology.org/data_2670' - Label 'Pathway ID (DQCS)'\n",
      "Entity 'http://edamontology.org/data_2671' - Label 'Ensembl ID (Homo sapiens)'\n",
      "Entity 'http://edamontology.org/data_2672' - Label 'Ensembl ID ('Bos taurus')'\n",
      "Entity 'http://edamontology.org/data_2673' - Label 'Ensembl ID ('Canis familiaris')'\n",
      "Entity 'http://edamontology.org/data_2674' - Label 'Ensembl ID ('Cavia porcellus')'\n",
      "Entity 'http://edamontology.org/data_2675' - Label 'Ensembl ID ('Ciona intestinalis')'\n",
      "Entity 'http://edamontology.org/data_2676' - Label 'Ensembl ID ('Ciona savignyi')'\n",
      "Entity 'http://edamontology.org/data_2677' - Label 'Ensembl ID ('Danio rerio')'\n",
      "Entity 'http://edamontology.org/data_2678' - Label 'Ensembl ID ('Dasypus novemcinctus')'\n",
      "Entity 'http://edamontology.org/data_2679' - Label 'Ensembl ID ('Echinops telfairi')'\n",
      "Entity 'http://edamontology.org/data_2680' - Label 'Ensembl ID ('Erinaceus europaeus')'\n",
      "Entity 'http://edamontology.org/data_2681' - Label 'Ensembl ID ('Felis catus')'\n",
      "Entity 'http://edamontology.org/data_2682' - Label 'Ensembl ID ('Gallus gallus')'\n",
      "Entity 'http://edamontology.org/data_2683' - Label 'Ensembl ID ('Gasterosteus aculeatus')'\n",
      "Entity 'http://edamontology.org/data_2684' - Label 'Ensembl ID ('Homo sapiens')'\n",
      "Entity 'http://edamontology.org/data_2685' - Label 'Ensembl ID ('Loxodonta africana')'\n",
      "Entity 'http://edamontology.org/data_2686' - Label 'Ensembl ID ('Macaca mulatta')'\n",
      "Entity 'http://edamontology.org/data_2687' - Label 'Ensembl ID ('Monodelphis domestica')'\n",
      "Entity 'http://edamontology.org/data_2688' - Label 'Ensembl ID ('Mus musculus')'\n",
      "Entity 'http://edamontology.org/data_2689' - Label 'Ensembl ID ('Myotis lucifugus')'\n",
      "Entity 'http://edamontology.org/data_2690' - Label 'Ensembl ID (\"Ornithorhynchus anatinus\")'\n",
      "Entity 'http://edamontology.org/data_2691' - Label 'Ensembl ID ('Oryctolagus cuniculus')'\n",
      "Entity 'http://edamontology.org/data_2692' - Label 'Ensembl ID ('Oryzias latipes')'\n",
      "Entity 'http://edamontology.org/data_2693' - Label 'Ensembl ID ('Otolemur garnettii')'\n",
      "Entity 'http://edamontology.org/data_2694' - Label 'Ensembl ID ('Pan troglodytes')'\n",
      "Entity 'http://edamontology.org/data_2695' - Label 'Ensembl ID ('Rattus norvegicus')'\n",
      "Entity 'http://edamontology.org/data_2696' - Label 'Ensembl ID ('Spermophilus tridecemlineatus')'\n",
      "Entity 'http://edamontology.org/data_2697' - Label 'Ensembl ID ('Takifugu rubripes')'\n",
      "Entity 'http://edamontology.org/data_2698' - Label 'Ensembl ID ('Tupaia belangeri')'\n",
      "Entity 'http://edamontology.org/data_2699' - Label 'Ensembl ID ('Xenopus tropicalis')'\n",
      "Entity 'http://edamontology.org/data_2700' - Label 'CATH identifier'\n",
      "Entity 'http://edamontology.org/data_2701' - Label 'CATH node ID (family)'\n",
      "Entity 'http://edamontology.org/data_2702' - Label 'Enzyme ID (CAZy)'\n",
      "Entity 'http://edamontology.org/data_2704' - Label 'Clone ID (IMAGE)'\n",
      "Entity 'http://edamontology.org/data_2705' - Label 'GO concept ID (cellular component)'\n",
      "Entity 'http://edamontology.org/data_2706' - Label 'Chromosome name (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2709' - Label 'CleanEx entry name'\n",
      "Entity 'http://edamontology.org/data_2710' - Label 'CleanEx dataset code'\n",
      "Entity 'http://edamontology.org/data_2711' - Label 'Genome report'\n",
      "Entity 'http://edamontology.org/data_2713' - Label 'Protein ID (CORUM)'\n",
      "Entity 'http://edamontology.org/data_2714' - Label 'CDD PSSM-ID'\n",
      "Entity 'http://edamontology.org/data_2715' - Label 'Protein ID (CuticleDB)'\n",
      "Entity 'http://edamontology.org/data_2716' - Label 'DBD ID'\n",
      "Entity 'http://edamontology.org/data_2717' - Label 'Oligonucleotide probe annotation'\n",
      "Entity 'http://edamontology.org/data_2718' - Label 'Oligonucleotide ID'\n",
      "Entity 'http://edamontology.org/data_2719' - Label 'dbProbe ID'\n",
      "Entity 'http://edamontology.org/data_2720' - Label 'Dinucleotide property'\n",
      "Entity 'http://edamontology.org/data_2721' - Label 'DiProDB ID'\n",
      "Entity 'http://edamontology.org/data_2722' - Label 'Protein features report (disordered structure)'\n",
      "Entity 'http://edamontology.org/data_2723' - Label 'Protein ID (DisProt)'\n",
      "Entity 'http://edamontology.org/data_2724' - Label 'Embryo report'\n",
      "Entity 'http://edamontology.org/data_2725' - Label 'Ensembl transcript ID'\n",
      "Entity 'http://edamontology.org/data_2726' - Label 'Inhibitor annotation'\n",
      "Entity 'http://edamontology.org/data_2727' - Label 'Promoter ID'\n",
      "Entity 'http://edamontology.org/data_2728' - Label 'EST accession'\n",
      "Entity 'http://edamontology.org/data_2729' - Label 'COGEME EST ID'\n",
      "Entity 'http://edamontology.org/data_2730' - Label 'COGEME unisequence ID'\n",
      "Entity 'http://edamontology.org/data_2731' - Label 'Protein family ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_2732' - Label 'Family name'\n",
      "Entity 'http://edamontology.org/data_2733' - Label 'Genus name (virus)'\n",
      "Entity 'http://edamontology.org/data_2734' - Label 'Family name (virus)'\n",
      "Entity 'http://edamontology.org/data_2735' - Label 'Database name (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2736' - Label 'Sequence feature ID (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2737' - Label 'FIG ID'\n",
      "Entity 'http://edamontology.org/data_2738' - Label 'Gene ID (Xenbase)'\n",
      "Entity 'http://edamontology.org/data_2739' - Label 'Gene ID (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2740' - Label 'Gene name (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2741' - Label 'ABS ID'\n",
      "Entity 'http://edamontology.org/data_2742' - Label 'AraC-XylS ID'\n",
      "Entity 'http://edamontology.org/data_2743' - Label 'Gene name (HUGO)'\n",
      "Entity 'http://edamontology.org/data_2744' - Label 'Locus ID (PseudoCAP)'\n",
      "Entity 'http://edamontology.org/data_2745' - Label 'Locus ID (UTR)'\n",
      "Entity 'http://edamontology.org/data_2746' - Label 'MonosaccharideDB ID'\n",
      "Entity 'http://edamontology.org/data_2747' - Label 'Database name (CMD)'\n",
      "Entity 'http://edamontology.org/data_2748' - Label 'Database name (Osteogenesis)'\n",
      "Entity 'http://edamontology.org/data_2749' - Label 'Genome identifier'\n",
      "Entity 'http://edamontology.org/data_2751' - Label 'GenomeReviews ID'\n",
      "Entity 'http://edamontology.org/data_2752' - Label 'GlycoMap ID'\n",
      "Entity 'http://edamontology.org/data_2753' - Label 'Carbohydrate conformational map'\n",
      "Entity 'http://edamontology.org/data_2755' - Label 'Transcription factor name'\n",
      "Entity 'http://edamontology.org/data_2756' - Label 'TCID'\n",
      "Entity 'http://edamontology.org/data_2757' - Label 'Pfam domain name'\n",
      "Entity 'http://edamontology.org/data_2758' - Label 'Pfam clan ID'\n",
      "Entity 'http://edamontology.org/data_2759' - Label 'Gene ID (VectorBase)'\n",
      "Entity 'http://edamontology.org/data_2761' - Label 'UTRSite ID'\n",
      "Entity 'http://edamontology.org/data_2762' - Label 'Sequence signature report'\n",
      "Entity 'http://edamontology.org/data_2763' - Label 'Locus annotation'\n",
      "Entity 'http://edamontology.org/data_2764' - Label 'Protein name (UniProt)'\n",
      "Entity 'http://edamontology.org/data_2765' - Label 'Term ID list'\n",
      "Entity 'http://edamontology.org/data_2766' - Label 'HAMAP ID'\n",
      "Entity 'http://edamontology.org/data_2767' - Label 'Identifier with metadata'\n",
      "Entity 'http://edamontology.org/data_2768' - Label 'Gene symbol annotation'\n",
      "Entity 'http://edamontology.org/data_2769' - Label 'Transcript ID'\n",
      "Entity 'http://edamontology.org/data_2770' - Label 'HIT ID'\n",
      "Entity 'http://edamontology.org/data_2771' - Label 'HIX ID'\n",
      "Entity 'http://edamontology.org/data_2772' - Label 'HPA antibody id'\n",
      "Entity 'http://edamontology.org/data_2773' - Label 'IMGT/HLA ID'\n",
      "Entity 'http://edamontology.org/data_2774' - Label 'Gene ID (JCVI)'\n",
      "Entity 'http://edamontology.org/data_2775' - Label 'Kinase name'\n",
      "Entity 'http://edamontology.org/data_2776' - Label 'ConsensusPathDB entity ID'\n",
      "Entity 'http://edamontology.org/data_2777' - Label 'ConsensusPathDB entity name'\n",
      "Entity 'http://edamontology.org/data_2778' - Label 'CCAP strain number'\n",
      "Entity 'http://edamontology.org/data_2779' - Label 'Stock number'\n",
      "Entity 'http://edamontology.org/data_2780' - Label 'Stock number (TAIR)'\n",
      "Entity 'http://edamontology.org/data_2781' - Label 'REDIdb ID'\n",
      "Entity 'http://edamontology.org/data_2782' - Label 'SMART domain name'\n",
      "Entity 'http://edamontology.org/data_2783' - Label 'Protein family ID (PANTHER)'\n",
      "Entity 'http://edamontology.org/data_2784' - Label 'RNAVirusDB ID'\n",
      "Entity 'http://edamontology.org/data_2785' - Label 'Virus identifier'\n",
      "Entity 'http://edamontology.org/data_2786' - Label 'NCBI Genome Project ID'\n",
      "Entity 'http://edamontology.org/data_2787' - Label 'NCBI genome accession'\n",
      "Entity 'http://edamontology.org/data_2788' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2789' - Label 'Protein ID (TopDB)'\n",
      "Entity 'http://edamontology.org/data_2790' - Label 'Gel ID'\n",
      "Entity 'http://edamontology.org/data_2791' - Label 'Reference map name (SWISS-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2792' - Label 'Protein ID (PeroxiBase)'\n",
      "Entity 'http://edamontology.org/data_2793' - Label 'SISYPHUS ID'\n",
      "Entity 'http://edamontology.org/data_2794' - Label 'ORF ID'\n",
      "Entity 'http://edamontology.org/data_2795' - Label 'ORF identifier'\n",
      "Entity 'http://edamontology.org/data_2796' - Label 'LINUCS ID'\n",
      "Entity 'http://edamontology.org/data_2797' - Label 'Protein ID (LGICdb)'\n",
      "Entity 'http://edamontology.org/data_2798' - Label 'MaizeDB ID'\n",
      "Entity 'http://edamontology.org/data_2799' - Label 'Gene ID (MfunGD)'\n",
      "Entity 'http://edamontology.org/data_2800' - Label 'Orpha number'\n",
      "Entity 'http://edamontology.org/data_2802' - Label 'Protein ID (EcID)'\n",
      "Entity 'http://edamontology.org/data_2803' - Label 'Clone ID (RefSeq)'\n",
      "Entity 'http://edamontology.org/data_2804' - Label 'Protein ID (ConoServer)'\n",
      "Entity 'http://edamontology.org/data_2805' - Label 'GeneSNP ID'\n",
      "Entity 'http://edamontology.org/data_2812' - Label 'Lipid identifier'\n",
      "Entity 'http://edamontology.org/data_2831' - Label 'Databank'\n",
      "Entity 'http://edamontology.org/data_2832' - Label 'Web portal'\n",
      "Entity 'http://edamontology.org/data_2835' - Label 'Gene ID (VBASE2)'\n",
      "Entity 'http://edamontology.org/data_2836' - Label 'DPVweb ID'\n",
      "Entity 'http://edamontology.org/data_2837' - Label 'Pathway ID (BioSystems)'\n",
      "Entity 'http://edamontology.org/data_2838' - Label 'Experimental data (proteomics)'\n",
      "Entity 'http://edamontology.org/data_2849' - Label 'Abstract'\n",
      "Entity 'http://edamontology.org/data_2850' - Label 'Lipid structure'\n",
      "Entity 'http://edamontology.org/data_2851' - Label 'Drug structure'\n",
      "Entity 'http://edamontology.org/data_2852' - Label 'Toxin structure'\n",
      "Entity 'http://edamontology.org/data_2854' - Label 'Position-specific scoring matrix'\n",
      "Entity 'http://edamontology.org/data_2855' - Label 'Distance matrix'\n",
      "Entity 'http://edamontology.org/data_2856' - Label 'Structural distance matrix'\n",
      "Entity 'http://edamontology.org/data_2857' - Label 'Article metadata'\n",
      "Entity 'http://edamontology.org/data_2858' - Label 'Ontology concept'\n",
      "Entity 'http://edamontology.org/data_2865' - Label 'Codon usage bias'\n",
      "Entity 'http://edamontology.org/data_2866' - Label 'Northern blot report'\n",
      "Entity 'http://edamontology.org/data_2870' - Label 'Radiation hybrid map'\n",
      "Entity 'http://edamontology.org/data_2872' - Label 'ID list'\n",
      "Entity 'http://edamontology.org/data_2873' - Label 'Phylogenetic gene frequencies data'\n",
      "Entity 'http://edamontology.org/data_2874' - Label 'Sequence set (polymorphic)'\n",
      "Entity 'http://edamontology.org/data_2875' - Label 'DRCAT resource'\n",
      "Entity 'http://edamontology.org/data_2877' - Label 'Protein complex'\n",
      "Entity 'http://edamontology.org/data_2878' - Label 'Protein structural motif'\n",
      "Entity 'http://edamontology.org/data_2879' - Label 'Lipid report'\n",
      "Entity 'http://edamontology.org/data_2880' - Label 'Secondary structure image'\n",
      "Entity 'http://edamontology.org/data_2881' - Label 'Secondary structure report'\n",
      "Entity 'http://edamontology.org/data_2882' - Label 'DNA features'\n",
      "Entity 'http://edamontology.org/data_2883' - Label 'RNA features report'\n",
      "Entity 'http://edamontology.org/data_2884' - Label 'Plot'\n",
      "Entity 'http://edamontology.org/data_2886' - Label 'Protein sequence record'\n",
      "Entity 'http://edamontology.org/data_2887' - Label 'Nucleic acid sequence record'\n",
      "Entity 'http://edamontology.org/data_2888' - Label 'Protein sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2889' - Label 'Nucleic acid sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2891' - Label 'Biological model accession'\n",
      "Entity 'http://edamontology.org/data_2892' - Label 'Cell type name'\n",
      "Entity 'http://edamontology.org/data_2893' - Label 'Cell type accession'\n",
      "Entity 'http://edamontology.org/data_2894' - Label 'Compound accession'\n",
      "Entity 'http://edamontology.org/data_2895' - Label 'Drug accession'\n",
      "Entity 'http://edamontology.org/data_2896' - Label 'Toxin name'\n",
      "Entity 'http://edamontology.org/data_2897' - Label 'Toxin accession'\n",
      "Entity 'http://edamontology.org/data_2898' - Label 'Monosaccharide accession'\n",
      "Entity 'http://edamontology.org/data_2899' - Label 'Drug name'\n",
      "Entity 'http://edamontology.org/data_2900' - Label 'Carbohydrate accession'\n",
      "Entity 'http://edamontology.org/data_2901' - Label 'Molecule accession'\n",
      "Entity 'http://edamontology.org/data_2902' - Label 'Data resource definition accession'\n",
      "Entity 'http://edamontology.org/data_2903' - Label 'Genome accession'\n",
      "Entity 'http://edamontology.org/data_2904' - Label 'Map accession'\n",
      "Entity 'http://edamontology.org/data_2905' - Label 'Lipid accession'\n",
      "Entity 'http://edamontology.org/data_2906' - Label 'Peptide ID'\n",
      "Entity 'http://edamontology.org/data_2907' - Label 'Protein accession'\n",
      "Entity 'http://edamontology.org/data_2908' - Label 'Organism accession'\n",
      "Entity 'http://edamontology.org/data_2909' - Label 'Organism name'\n",
      "Entity 'http://edamontology.org/data_2910' - Label 'Protein family accession'\n",
      "Entity 'http://edamontology.org/data_2911' - Label 'Transcription factor accession'\n",
      "Entity 'http://edamontology.org/data_2912' - Label 'Strain accession'\n",
      "Entity 'http://edamontology.org/data_2913' - Label 'Virus identifier'\n",
      "Entity 'http://edamontology.org/data_2914' - Label 'Sequence features metadata'\n",
      "Entity 'http://edamontology.org/data_2915' - Label 'Gramene identifier'\n",
      "Entity 'http://edamontology.org/data_2916' - Label 'DDBJ accession'\n",
      "Entity 'http://edamontology.org/data_2917' - Label 'ConsensusPathDB identifier'\n",
      "Entity 'http://edamontology.org/data_2925' - Label 'Sequence data'\n",
      "Entity 'http://edamontology.org/data_2927' - Label 'Codon usage'\n",
      "Entity 'http://edamontology.org/data_2954' - Label 'Article report'\n",
      "Entity 'http://edamontology.org/data_2955' - Label 'Sequence report'\n",
      "Entity 'http://edamontology.org/data_2956' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/data_2957' - Label 'Hopp and Woods plot'\n",
      "Entity 'http://edamontology.org/data_2958' - Label 'Nucleic acid melting curve'\n",
      "Entity 'http://edamontology.org/data_2959' - Label 'Nucleic acid probability profile'\n",
      "Entity 'http://edamontology.org/data_2960' - Label 'Nucleic acid temperature profile'\n",
      "Entity 'http://edamontology.org/data_2961' - Label 'Gene regulatory network report'\n",
      "Entity 'http://edamontology.org/data_2965' - Label '2D PAGE gel report'\n",
      "Entity 'http://edamontology.org/data_2966' - Label 'Oligonucleotide probe sets annotation'\n",
      "Entity 'http://edamontology.org/data_2967' - Label 'Microarray image'\n",
      "Entity 'http://edamontology.org/data_2968' - Label 'Image'\n",
      "Entity 'http://edamontology.org/data_2969' - Label 'Sequence image'\n",
      "Entity 'http://edamontology.org/data_2970' - Label 'Protein hydropathy data'\n",
      "Entity 'http://edamontology.org/data_2971' - Label 'Workflow data'\n",
      "Entity 'http://edamontology.org/data_2972' - Label 'Workflow'\n",
      "Entity 'http://edamontology.org/data_2973' - Label 'Secondary structure data'\n",
      "Entity 'http://edamontology.org/data_2974' - Label 'Protein sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2975' - Label 'Nucleic acid sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2976' - Label 'Protein sequence'\n",
      "Entity 'http://edamontology.org/data_2977' - Label 'Nucleic acid sequence'\n",
      "Entity 'http://edamontology.org/data_2978' - Label 'Reaction data'\n",
      "Entity 'http://edamontology.org/data_2979' - Label 'Peptide property'\n",
      "Entity 'http://edamontology.org/data_2980' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/data_2981' - Label 'Sequence motif data'\n",
      "Entity 'http://edamontology.org/data_2982' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2983' - Label 'Pathway or network data'\n",
      "Entity 'http://edamontology.org/data_2984' - Label 'Pathway or network report'\n",
      "Entity 'http://edamontology.org/data_2985' - Label 'Nucleic acid thermodynamic data'\n",
      "Entity 'http://edamontology.org/data_2986' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/data_2987' - Label 'Classification report'\n",
      "Entity 'http://edamontology.org/data_2989' - Label 'Protein features report (key folding sites)'\n",
      "Entity 'http://edamontology.org/data_2991' - Label 'Protein geometry data'\n",
      "Entity 'http://edamontology.org/data_2992' - Label 'Protein structure image'\n",
      "Entity 'http://edamontology.org/data_2994' - Label 'Phylogenetic character weights'\n",
      "Entity 'http://edamontology.org/data_3002' - Label 'Annotation track'\n",
      "Entity 'http://edamontology.org/data_3021' - Label 'UniProt accession'\n",
      "Entity 'http://edamontology.org/data_3022' - Label 'NCBI genetic code ID'\n",
      "Entity 'http://edamontology.org/data_3025' - Label 'Ontology concept identifier'\n",
      "Entity 'http://edamontology.org/data_3026' - Label 'GO concept name (biological process)'\n",
      "Entity 'http://edamontology.org/data_3027' - Label 'GO concept name (molecular function)'\n",
      "Entity 'http://edamontology.org/data_3028' - Label 'Taxonomy'\n",
      "Entity 'http://edamontology.org/data_3029' - Label 'Protein ID (EMBL/GenBank/DDBJ)'\n",
      "Entity 'http://edamontology.org/data_3031' - Label 'Core data'\n",
      "Entity 'http://edamontology.org/data_3034' - Label 'Sequence feature identifier'\n",
      "Entity 'http://edamontology.org/data_3035' - Label 'Structure identifier'\n",
      "Entity 'http://edamontology.org/data_3036' - Label 'Matrix identifier'\n",
      "Entity 'http://edamontology.org/data_3085' - Label 'Protein sequence composition'\n",
      "Entity 'http://edamontology.org/data_3086' - Label 'Nucleic acid sequence composition (report)'\n",
      "Entity 'http://edamontology.org/data_3101' - Label 'Protein domain classification node'\n",
      "Entity 'http://edamontology.org/data_3102' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_3103' - Label 'ATC code'\n",
      "Entity 'http://edamontology.org/data_3104' - Label 'UNII'\n",
      "Entity 'http://edamontology.org/data_3105' - Label 'Geotemporal metadata'\n",
      "Entity 'http://edamontology.org/data_3106' - Label 'System metadata'\n",
      "Entity 'http://edamontology.org/data_3107' - Label 'Sequence feature name'\n",
      "Entity 'http://edamontology.org/data_3108' - Label 'Experimental measurement'\n",
      "Entity 'http://edamontology.org/data_3110' - Label 'Raw microarray data'\n",
      "Entity 'http://edamontology.org/data_3111' - Label 'Processed microarray data'\n",
      "Entity 'http://edamontology.org/data_3112' - Label 'Gene expression matrix'\n",
      "Entity 'http://edamontology.org/data_3113' - Label 'Sample annotation'\n",
      "Entity 'http://edamontology.org/data_3115' - Label 'Microarray metadata'\n",
      "Entity 'http://edamontology.org/data_3116' - Label 'Microarray protocol annotation'\n",
      "Entity 'http://edamontology.org/data_3117' - Label 'Microarray hybridisation data'\n",
      "Entity 'http://edamontology.org/data_3119' - Label 'Sequence features (compositionally-biased regions)'\n",
      "Entity 'http://edamontology.org/data_3122' - Label 'Nucleic acid features (difference and change)'\n",
      "Entity 'http://edamontology.org/data_3128' - Label 'Nucleic acid structure report'\n",
      "Entity 'http://edamontology.org/data_3129' - Label 'Protein features report (repeats)'\n",
      "Entity 'http://edamontology.org/data_3130' - Label 'Sequence motif matches (protein)'\n",
      "Entity 'http://edamontology.org/data_3131' - Label 'Sequence motif matches (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_3132' - Label 'Nucleic acid features (d-loop)'\n",
      "Entity 'http://edamontology.org/data_3133' - Label 'Nucleic acid features (stem loop)'\n",
      "Entity 'http://edamontology.org/data_3134' - Label 'Gene transcript report'\n",
      "Entity 'http://edamontology.org/data_3137' - Label 'Non-coding RNA'\n",
      "Entity 'http://edamontology.org/data_3138' - Label 'Transcriptional features (report)'\n",
      "Entity 'http://edamontology.org/data_3140' - Label 'Nucleic acid features (immunoglobulin gene structure)'\n",
      "Entity 'http://edamontology.org/data_3141' - Label 'SCOP class'\n",
      "Entity 'http://edamontology.org/data_3142' - Label 'SCOP fold'\n",
      "Entity 'http://edamontology.org/data_3143' - Label 'SCOP superfamily'\n",
      "Entity 'http://edamontology.org/data_3144' - Label 'SCOP family'\n",
      "Entity 'http://edamontology.org/data_3145' - Label 'SCOP protein'\n",
      "Entity 'http://edamontology.org/data_3146' - Label 'SCOP species'\n",
      "Entity 'http://edamontology.org/data_3147' - Label 'Mass spectrometry experiment'\n",
      "Entity 'http://edamontology.org/data_3148' - Label 'Gene family report'\n",
      "Entity 'http://edamontology.org/data_3153' - Label 'Protein image'\n",
      "Entity 'http://edamontology.org/data_3154' - Label 'Protein alignment'\n",
      "Entity 'http://edamontology.org/data_3165' - Label 'NGS experiment'\n",
      "Entity 'http://edamontology.org/data_3181' - Label 'Sequence assembly report'\n",
      "Entity 'http://edamontology.org/data_3210' - Label 'Genome index'\n",
      "Entity 'http://edamontology.org/data_3231' - Label 'GWAS report'\n",
      "Entity 'http://edamontology.org/data_3236' - Label 'Cytoband position'\n",
      "Entity 'http://edamontology.org/data_3238' - Label 'Cell type ontology ID'\n",
      "Entity 'http://edamontology.org/data_3241' - Label 'Kinetic model'\n",
      "Entity 'http://edamontology.org/data_3264' - Label 'COSMIC ID'\n",
      "Entity 'http://edamontology.org/data_3265' - Label 'HGMD ID'\n",
      "Entity 'http://edamontology.org/data_3266' - Label 'Sequence assembly ID'\n",
      "Entity 'http://edamontology.org/data_3268' - Label 'Sequence feature type'\n",
      "Entity 'http://edamontology.org/data_3269' - Label 'Gene homology (report)'\n",
      "Entity 'http://edamontology.org/data_3270' - Label 'Ensembl gene tree ID'\n",
      "Entity 'http://edamontology.org/data_3271' - Label 'Gene tree'\n",
      "Entity 'http://edamontology.org/data_3272' - Label 'Species tree'\n",
      "Entity 'http://edamontology.org/data_3273' - Label 'Sample ID'\n",
      "Entity 'http://edamontology.org/data_3274' - Label 'MGI accession'\n",
      "Entity 'http://edamontology.org/data_3275' - Label 'Phenotype name'\n",
      "Entity 'http://edamontology.org/data_3354' - Label 'Transition matrix'\n",
      "Entity 'http://edamontology.org/data_3355' - Label 'Emission matrix'\n",
      "Entity 'http://edamontology.org/data_3356' - Label 'Hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_3358' - Label 'Format identifier'\n",
      "Entity 'http://edamontology.org/data_3424' - Label 'Raw image'\n",
      "Entity 'http://edamontology.org/data_3425' - Label 'Carbohydrate property'\n",
      "Entity 'http://edamontology.org/data_3426' - Label 'Proteomics experiment report'\n",
      "Entity 'http://edamontology.org/data_3427' - Label 'RNAi report'\n",
      "Entity 'http://edamontology.org/data_3428' - Label 'Simulation experiment report'\n",
      "Entity 'http://edamontology.org/data_3442' - Label 'MRI image'\n",
      "Entity 'http://edamontology.org/data_3449' - Label 'Cell migration track image'\n",
      "Entity 'http://edamontology.org/data_3451' - Label 'Rate of association'\n",
      "Entity 'http://edamontology.org/data_3479' - Label 'Gene order'\n",
      "Entity 'http://edamontology.org/data_3483' - Label 'Spectrum'\n",
      "Entity 'http://edamontology.org/data_3488' - Label 'NMR spectrum'\n",
      "Entity 'http://edamontology.org/data_3490' - Label 'Chemical structure sketch'\n",
      "Entity 'http://edamontology.org/data_3492' - Label 'Nucleic acid signature'\n",
      "Entity 'http://edamontology.org/data_3494' - Label 'DNA sequence'\n",
      "Entity 'http://edamontology.org/data_3495' - Label 'RNA sequence'\n",
      "Entity 'http://edamontology.org/data_3496' - Label 'RNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3497' - Label 'DNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3498' - Label 'Sequence variations'\n",
      "Entity 'http://edamontology.org/data_3505' - Label 'Bibliography'\n",
      "Entity 'http://edamontology.org/data_3509' - Label 'Ontology mapping'\n",
      "Entity 'http://edamontology.org/data_3546' - Label 'Image metadata'\n",
      "Entity 'http://edamontology.org/data_3558' - Label 'Clinical trial report'\n",
      "Entity 'http://edamontology.org/data_3567' - Label 'Reference sample report'\n",
      "Entity 'http://edamontology.org/data_3568' - Label 'Gene Expression Atlas Experiment ID'\n",
      "Entity 'http://edamontology.org/data_3667' - Label 'Disease identifier'\n",
      "Entity 'http://edamontology.org/data_3668' - Label 'Disease name'\n",
      "Entity 'http://edamontology.org/data_3669' - Label 'Training material'\n",
      "Entity 'http://edamontology.org/data_3670' - Label 'Online course'\n",
      "Entity 'http://edamontology.org/data_3671' - Label 'Text'\n",
      "Entity 'http://edamontology.org/data_3707' - Label 'Biodiversity data'\n",
      "Entity 'http://edamontology.org/data_3716' - Label 'Biosafety report'\n",
      "Entity 'http://edamontology.org/data_3717' - Label 'Isolation report'\n",
      "Entity 'http://edamontology.org/data_3718' - Label 'Pathogenicity report'\n",
      "Entity 'http://edamontology.org/data_3719' - Label 'Biosafety classification'\n",
      "Entity 'http://edamontology.org/data_3720' - Label 'Geographic location'\n",
      "Entity 'http://edamontology.org/data_3721' - Label 'Isolation source'\n",
      "Entity 'http://edamontology.org/data_3722' - Label 'Physiology parameter'\n",
      "Entity 'http://edamontology.org/data_3723' - Label 'Morphology parameter'\n",
      "Entity 'http://edamontology.org/data_3724' - Label 'Cultivation parameter'\n",
      "Entity 'http://edamontology.org/data_3732' - Label 'Sequencing metadata name'\n",
      "Entity 'http://edamontology.org/data_3733' - Label 'Flow cell identifier'\n",
      "Entity 'http://edamontology.org/data_3734' - Label 'Lane identifier'\n",
      "Entity 'http://edamontology.org/data_3735' - Label 'Run number'\n",
      "Entity 'http://edamontology.org/data_3736' - Label 'Ecological data'\n",
      "Entity 'http://edamontology.org/data_3737' - Label 'Alpha diversity data'\n",
      "Entity 'http://edamontology.org/data_3738' - Label 'Beta diversity data'\n",
      "Entity 'http://edamontology.org/data_3739' - Label 'Gamma diversity data'\n",
      "Entity 'http://edamontology.org/data_3743' - Label 'Ordination plot'\n",
      "Entity 'http://edamontology.org/data_3753' - Label 'Over-representation data'\n",
      "Entity 'http://edamontology.org/data_3754' - Label 'GO-term enrichment data'\n",
      "Entity 'http://edamontology.org/data_3756' - Label 'Localisation score'\n",
      "Entity 'http://edamontology.org/data_3757' - Label 'Unimod ID'\n",
      "Entity 'http://edamontology.org/data_3759' - Label 'ProteomeXchange ID'\n",
      "Entity 'http://edamontology.org/data_3768' - Label 'Clustered expression profiles'\n",
      "Entity 'http://edamontology.org/data_3769' - Label 'BRENDA ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_3779' - Label 'Annotated text'\n",
      "Entity 'http://edamontology.org/data_3786' - Label 'Query script'\n",
      "Entity 'http://edamontology.org/data_3805' - Label '3D EM Map'\n",
      "Entity 'http://edamontology.org/data_3806' - Label '3D EM Mask'\n",
      "Entity 'http://edamontology.org/data_3807' - Label 'EM Movie'\n",
      "Entity 'http://edamontology.org/data_3808' - Label 'EM Micrograph'\n",
      "Entity 'http://edamontology.org/data_3842' - Label 'Molecular simulation data'\n",
      "Entity 'http://edamontology.org/data_3856' - Label 'RNA central ID'\n",
      "Entity 'http://edamontology.org/data_3861' - Label 'Electronic health record'\n",
      "Entity 'http://edamontology.org/data_3869' - Label 'Simulation'\n",
      "Entity 'http://edamontology.org/data_3870' - Label 'Trajectory data'\n",
      "Entity 'http://edamontology.org/data_3871' - Label 'Forcefield parameters'\n",
      "Entity 'http://edamontology.org/data_3872' - Label 'Topology data'\n",
      "Entity 'http://edamontology.org/data_3905' - Label 'Histogram'\n",
      "Entity 'http://edamontology.org/data_3914' - Label 'Quality control report'\n",
      "Entity 'http://edamontology.org/data_3917' - Label 'Count matrix'\n",
      "Entity 'http://edamontology.org/data_3924' - Label 'DNA structure alignment'\n",
      "Entity 'http://edamontology.org/data_3932' - Label 'Q-value'\n",
      "Entity 'http://edamontology.org/data_3949' - Label 'Profile HMM'\n",
      "Entity 'http://edamontology.org/data_3952' - Label 'Pathway ID (WikiPathways)'\n",
      "Entity 'http://edamontology.org/data_3953' - Label 'Pathway overrepresentation data'\n",
      "Entity 'http://edamontology.org/data_4022' - Label 'ORCID Identifier'\n",
      "Entity 'http://edamontology.org/format_1196' - Label 'SMILES'\n",
      "Entity 'http://edamontology.org/format_1197' - Label 'InChI'\n",
      "Entity 'http://edamontology.org/format_1198' - Label 'mf'\n",
      "Entity 'http://edamontology.org/format_1199' - Label 'InChIKey'\n",
      "Entity 'http://edamontology.org/format_1200' - Label 'smarts'\n",
      "Entity 'http://edamontology.org/format_1206' - Label 'unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_1207' - Label 'nucleotide'\n",
      "Entity 'http://edamontology.org/format_1208' - Label 'protein'\n",
      "Entity 'http://edamontology.org/format_1209' - Label 'consensus'\n",
      "Entity 'http://edamontology.org/format_1210' - Label 'pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1211' - Label 'unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1212' - Label 'dna'\n",
      "Entity 'http://edamontology.org/format_1213' - Label 'rna'\n",
      "Entity 'http://edamontology.org/format_1214' - Label 'unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_1215' - Label 'pure dna'\n",
      "Entity 'http://edamontology.org/format_1216' - Label 'unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_1217' - Label 'pure rna'\n",
      "Entity 'http://edamontology.org/format_1218' - Label 'unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_1219' - Label 'pure protein'\n",
      "Entity 'http://edamontology.org/format_1228' - Label 'UniGene entry format'\n",
      "Entity 'http://edamontology.org/format_1247' - Label 'COG sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_1248' - Label 'EMBL feature location'\n",
      "Entity 'http://edamontology.org/format_1295' - Label 'quicktandem'\n",
      "Entity 'http://edamontology.org/format_1296' - Label 'Sanger inverted repeats'\n",
      "Entity 'http://edamontology.org/format_1297' - Label 'EMBOSS repeat'\n",
      "Entity 'http://edamontology.org/format_1316' - Label 'est2genome format'\n",
      "Entity 'http://edamontology.org/format_1318' - Label 'restrict format'\n",
      "Entity 'http://edamontology.org/format_1319' - Label 'restover format'\n",
      "Entity 'http://edamontology.org/format_1320' - Label 'REBASE restriction sites'\n",
      "Entity 'http://edamontology.org/format_1332' - Label 'FASTA search results format'\n",
      "Entity 'http://edamontology.org/format_1333' - Label 'BLAST results'\n",
      "Entity 'http://edamontology.org/format_1334' - Label 'mspcrunch'\n",
      "Entity 'http://edamontology.org/format_1335' - Label 'Smith-Waterman format'\n",
      "Entity 'http://edamontology.org/format_1336' - Label 'dhf'\n",
      "Entity 'http://edamontology.org/format_1337' - Label 'lhf'\n",
      "Entity 'http://edamontology.org/format_1341' - Label 'InterPro hits format'\n",
      "Entity 'http://edamontology.org/format_1342' - Label 'InterPro protein view report format'\n",
      "Entity 'http://edamontology.org/format_1343' - Label 'InterPro match table format'\n",
      "Entity 'http://edamontology.org/format_1349' - Label 'HMMER Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1350' - Label 'MEME Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1351' - Label 'HMMER emission and transition'\n",
      "Entity 'http://edamontology.org/format_1356' - Label 'prosite-pattern'\n",
      "Entity 'http://edamontology.org/format_1357' - Label 'EMBOSS sequence pattern'\n",
      "Entity 'http://edamontology.org/format_1360' - Label 'meme-motif'\n",
      "Entity 'http://edamontology.org/format_1366' - Label 'prosite-profile'\n",
      "Entity 'http://edamontology.org/format_1367' - Label 'JASPAR format'\n",
      "Entity 'http://edamontology.org/format_1369' - Label 'MEME background Markov model'\n",
      "Entity 'http://edamontology.org/format_1370' - Label 'HMMER format'\n",
      "Entity 'http://edamontology.org/format_1391' - Label 'HMMER-aln'\n",
      "Entity 'http://edamontology.org/format_1392' - Label 'DIALIGN format'\n",
      "Entity 'http://edamontology.org/format_1393' - Label 'daf'\n",
      "Entity 'http://edamontology.org/format_1419' - Label 'Sequence-MEME profile alignment'\n",
      "Entity 'http://edamontology.org/format_1421' - Label 'HMMER profile alignment (sequences versus HMMs)'\n",
      "Entity 'http://edamontology.org/format_1422' - Label 'HMMER profile alignment (HMM versus sequences)'\n",
      "Entity 'http://edamontology.org/format_1423' - Label 'Phylip distance matrix'\n",
      "Entity 'http://edamontology.org/format_1424' - Label 'ClustalW dendrogram'\n",
      "Entity 'http://edamontology.org/format_1425' - Label 'Phylip tree raw'\n",
      "Entity 'http://edamontology.org/format_1430' - Label 'Phylip continuous quantitative characters'\n",
      "Entity 'http://edamontology.org/format_1431' - Label 'Phylogenetic property values format'\n",
      "Entity 'http://edamontology.org/format_1432' - Label 'Phylip character frequencies format'\n",
      "Entity 'http://edamontology.org/format_1433' - Label 'Phylip discrete states format'\n",
      "Entity 'http://edamontology.org/format_1434' - Label 'Phylip cliques format'\n",
      "Entity 'http://edamontology.org/format_1435' - Label 'Phylip tree format'\n",
      "Entity 'http://edamontology.org/format_1436' - Label 'TreeBASE format'\n",
      "Entity 'http://edamontology.org/format_1437' - Label 'TreeFam format'\n",
      "Entity 'http://edamontology.org/format_1445' - Label 'Phylip tree distance format'\n",
      "Entity 'http://edamontology.org/format_1454' - Label 'dssp'\n",
      "Entity 'http://edamontology.org/format_1455' - Label 'hssp'\n",
      "Entity 'http://edamontology.org/format_1457' - Label 'Dot-bracket format'\n",
      "Entity 'http://edamontology.org/format_1458' - Label 'Vienna local RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_1475' - Label 'PDB database entry format'\n",
      "Entity 'http://edamontology.org/format_1476' - Label 'PDB'\n",
      "Entity 'http://edamontology.org/format_1477' - Label 'mmCIF'\n",
      "Entity 'http://edamontology.org/format_1478' - Label 'PDBML'\n",
      "Entity 'http://edamontology.org/format_1500' - Label 'Domainatrix 3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_1504' - Label 'aaindex'\n",
      "Entity 'http://edamontology.org/format_1511' - Label 'IntEnz enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1512' - Label 'BRENDA enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1513' - Label 'KEGG REACTION enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1514' - Label 'KEGG ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1515' - Label 'REBASE proto enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1516' - Label 'REBASE withrefm enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1551' - Label 'Pcons report format'\n",
      "Entity 'http://edamontology.org/format_1552' - Label 'ProQ report format'\n",
      "Entity 'http://edamontology.org/format_1563' - Label 'SMART domain assignment report format'\n",
      "Entity 'http://edamontology.org/format_1568' - Label 'BIND entry format'\n",
      "Entity 'http://edamontology.org/format_1569' - Label 'IntAct entry format'\n",
      "Entity 'http://edamontology.org/format_1570' - Label 'InterPro entry format'\n",
      "Entity 'http://edamontology.org/format_1571' - Label 'InterPro entry abstract format'\n",
      "Entity 'http://edamontology.org/format_1572' - Label 'Gene3D entry format'\n",
      "Entity 'http://edamontology.org/format_1573' - Label 'PIRSF entry format'\n",
      "Entity 'http://edamontology.org/format_1574' - Label 'PRINTS entry format'\n",
      "Entity 'http://edamontology.org/format_1575' - Label 'Panther Families and HMMs entry format'\n",
      "Entity 'http://edamontology.org/format_1576' - Label 'Pfam entry format'\n",
      "Entity 'http://edamontology.org/format_1577' - Label 'SMART entry format'\n",
      "Entity 'http://edamontology.org/format_1578' - Label 'Superfamily entry format'\n",
      "Entity 'http://edamontology.org/format_1579' - Label 'TIGRFam entry format'\n",
      "Entity 'http://edamontology.org/format_1580' - Label 'ProDom entry format'\n",
      "Entity 'http://edamontology.org/format_1581' - Label 'FSSP entry format'\n",
      "Entity 'http://edamontology.org/format_1582' - Label 'findkm'\n",
      "Entity 'http://edamontology.org/format_1603' - Label 'Ensembl gene report format'\n",
      "Entity 'http://edamontology.org/format_1604' - Label 'DictyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1605' - Label 'CGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1606' - Label 'DragonDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1607' - Label 'EcoCyc gene report format'\n",
      "Entity 'http://edamontology.org/format_1608' - Label 'FlyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1609' - Label 'Gramene gene report format'\n",
      "Entity 'http://edamontology.org/format_1610' - Label 'KEGG GENES gene report format'\n",
      "Entity 'http://edamontology.org/format_1611' - Label 'MaizeGDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1612' - Label 'MGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1613' - Label 'RGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1614' - Label 'SGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1615' - Label 'GeneDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1616' - Label 'TAIR gene report format'\n",
      "Entity 'http://edamontology.org/format_1617' - Label 'WormBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1618' - Label 'ZFIN gene report format'\n",
      "Entity 'http://edamontology.org/format_1619' - Label 'TIGR gene report format'\n",
      "Entity 'http://edamontology.org/format_1620' - Label 'dbSNP polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_1623' - Label 'OMIM entry format'\n",
      "Entity 'http://edamontology.org/format_1624' - Label 'HGVbase entry format'\n",
      "Entity 'http://edamontology.org/format_1625' - Label 'HIVDB entry format'\n",
      "Entity 'http://edamontology.org/format_1626' - Label 'KEGG DISEASE entry format'\n",
      "Entity 'http://edamontology.org/format_1627' - Label 'Primer3 primer'\n",
      "Entity 'http://edamontology.org/format_1628' - Label 'ABI'\n",
      "Entity 'http://edamontology.org/format_1629' - Label 'mira'\n",
      "Entity 'http://edamontology.org/format_1630' - Label 'CAF'\n",
      "Entity 'http://edamontology.org/format_1631' - Label 'EXP'\n",
      "Entity 'http://edamontology.org/format_1632' - Label 'SCF'\n",
      "Entity 'http://edamontology.org/format_1633' - Label 'PHD'\n",
      "Entity 'http://edamontology.org/format_1637' - Label 'dat'\n",
      "Entity 'http://edamontology.org/format_1638' - Label 'cel'\n",
      "Entity 'http://edamontology.org/format_1639' - Label 'affymetrix'\n",
      "Entity 'http://edamontology.org/format_1640' - Label 'ArrayExpress entry format'\n",
      "Entity 'http://edamontology.org/format_1641' - Label 'affymetrix-exp'\n",
      "Entity 'http://edamontology.org/format_1644' - Label 'CHP'\n",
      "Entity 'http://edamontology.org/format_1645' - Label 'EMDB entry format'\n",
      "Entity 'http://edamontology.org/format_1647' - Label 'KEGG PATHWAY entry format'\n",
      "Entity 'http://edamontology.org/format_1648' - Label 'MetaCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1649' - Label 'HumanCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1650' - Label 'INOH entry format'\n",
      "Entity 'http://edamontology.org/format_1651' - Label 'PATIKA entry format'\n",
      "Entity 'http://edamontology.org/format_1652' - Label 'Reactome entry format'\n",
      "Entity 'http://edamontology.org/format_1653' - Label 'aMAZE entry format'\n",
      "Entity 'http://edamontology.org/format_1654' - Label 'CPDB entry format'\n",
      "Entity 'http://edamontology.org/format_1655' - Label 'Panther Pathways entry format'\n",
      "Entity 'http://edamontology.org/format_1665' - Label 'Taverna workflow format'\n",
      "Entity 'http://edamontology.org/format_1666' - Label 'BioModel mathematical model format'\n",
      "Entity 'http://edamontology.org/format_1697' - Label 'KEGG LIGAND entry format'\n",
      "Entity 'http://edamontology.org/format_1698' - Label 'KEGG COMPOUND entry format'\n",
      "Entity 'http://edamontology.org/format_1699' - Label 'KEGG PLANT entry format'\n",
      "Entity 'http://edamontology.org/format_1700' - Label 'KEGG GLYCAN entry format'\n",
      "Entity 'http://edamontology.org/format_1701' - Label 'PubChem entry format'\n",
      "Entity 'http://edamontology.org/format_1702' - Label 'ChemSpider entry format'\n",
      "Entity 'http://edamontology.org/format_1703' - Label 'ChEBI entry format'\n",
      "Entity 'http://edamontology.org/format_1704' - Label 'MSDchem ligand dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1705' - Label 'HET group dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1706' - Label 'KEGG DRUG entry format'\n",
      "Entity 'http://edamontology.org/format_1734' - Label 'PubMed citation'\n",
      "Entity 'http://edamontology.org/format_1735' - Label 'Medline Display Format'\n",
      "Entity 'http://edamontology.org/format_1736' - Label 'CiteXplore-core'\n",
      "Entity 'http://edamontology.org/format_1737' - Label 'CiteXplore-all'\n",
      "Entity 'http://edamontology.org/format_1739' - Label 'pmc'\n",
      "Entity 'http://edamontology.org/format_1740' - Label 'iHOP format'\n",
      "Entity 'http://edamontology.org/format_1741' - Label 'OSCAR format'\n",
      "Entity 'http://edamontology.org/format_1747' - Label 'PDB atom record format'\n",
      "Entity 'http://edamontology.org/format_1760' - Label 'CATH chain report format'\n",
      "Entity 'http://edamontology.org/format_1761' - Label 'CATH PDB report format'\n",
      "Entity 'http://edamontology.org/format_1782' - Label 'NCBI gene report format'\n",
      "Entity 'http://edamontology.org/format_1808' - Label 'GeneIlluminator gene report format'\n",
      "Entity 'http://edamontology.org/format_1809' - Label 'BacMap gene card format'\n",
      "Entity 'http://edamontology.org/format_1810' - Label 'ColiCard report format'\n",
      "Entity 'http://edamontology.org/format_1861' - Label 'PlasMapper TextMap'\n",
      "Entity 'http://edamontology.org/format_1910' - Label 'newick'\n",
      "Entity 'http://edamontology.org/format_1911' - Label 'TreeCon format'\n",
      "Entity 'http://edamontology.org/format_1912' - Label 'Nexus format'\n",
      "Entity 'http://edamontology.org/format_1915' - Label 'Format'\n",
      "Entity 'http://edamontology.org/format_1918' - Label 'Atomic data format'\n",
      "Entity 'http://edamontology.org/format_1919' - Label 'Sequence record format'\n",
      "Entity 'http://edamontology.org/format_1920' - Label 'Sequence feature annotation format'\n",
      "Entity 'http://edamontology.org/format_1921' - Label 'Alignment format'\n",
      "Entity 'http://edamontology.org/format_1923' - Label 'acedb'\n",
      "Entity 'http://edamontology.org/format_1924' - Label 'clustal sequence format'\n",
      "Entity 'http://edamontology.org/format_1925' - Label 'codata'\n",
      "Entity 'http://edamontology.org/format_1926' - Label 'dbid'\n",
      "Entity 'http://edamontology.org/format_1927' - Label 'EMBL format'\n",
      "Entity 'http://edamontology.org/format_1928' - Label 'Staden experiment format'\n",
      "Entity 'http://edamontology.org/format_1929' - Label 'FASTA'\n",
      "Entity 'http://edamontology.org/format_1930' - Label 'FASTQ'\n",
      "Entity 'http://edamontology.org/format_1931' - Label 'FASTQ-illumina'\n",
      "Entity 'http://edamontology.org/format_1932' - Label 'FASTQ-sanger'\n",
      "Entity 'http://edamontology.org/format_1933' - Label 'FASTQ-solexa'\n",
      "Entity 'http://edamontology.org/format_1934' - Label 'fitch program'\n",
      "Entity 'http://edamontology.org/format_1935' - Label 'GCG'\n",
      "Entity 'http://edamontology.org/format_1936' - Label 'GenBank format'\n",
      "Entity 'http://edamontology.org/format_1937' - Label 'genpept'\n",
      "Entity 'http://edamontology.org/format_1938' - Label 'GFF2-seq'\n",
      "Entity 'http://edamontology.org/format_1939' - Label 'GFF3-seq'\n",
      "Entity 'http://edamontology.org/format_1940' - Label 'giFASTA format'\n",
      "Entity 'http://edamontology.org/format_1941' - Label 'hennig86'\n",
      "Entity 'http://edamontology.org/format_1942' - Label 'ig'\n",
      "Entity 'http://edamontology.org/format_1943' - Label 'igstrict'\n",
      "Entity 'http://edamontology.org/format_1944' - Label 'jackknifer'\n",
      "Entity 'http://edamontology.org/format_1945' - Label 'mase format'\n",
      "Entity 'http://edamontology.org/format_1946' - Label 'mega-seq'\n",
      "Entity 'http://edamontology.org/format_1947' - Label 'GCG MSF'\n",
      "Entity 'http://edamontology.org/format_1948' - Label 'nbrf/pir'\n",
      "Entity 'http://edamontology.org/format_1949' - Label 'nexus-seq'\n",
      "Entity 'http://edamontology.org/format_1950' - Label 'pdbatom'\n",
      "Entity 'http://edamontology.org/format_1951' - Label 'pdbatomnuc'\n",
      "Entity 'http://edamontology.org/format_1952' - Label 'pdbseqresnuc'\n",
      "Entity 'http://edamontology.org/format_1953' - Label 'pdbseqres'\n",
      "Entity 'http://edamontology.org/format_1954' - Label 'Pearson format'\n",
      "Entity 'http://edamontology.org/format_1955' - Label 'phylip sequence format'\n",
      "Entity 'http://edamontology.org/format_1956' - Label 'phylipnon sequence format'\n",
      "Entity 'http://edamontology.org/format_1957' - Label 'raw'\n",
      "Entity 'http://edamontology.org/format_1958' - Label 'refseqp'\n",
      "Entity 'http://edamontology.org/format_1959' - Label 'selex sequence format'\n",
      "Entity 'http://edamontology.org/format_1960' - Label 'Staden format'\n",
      "Entity 'http://edamontology.org/format_1961' - Label 'Stockholm format'\n",
      "Entity 'http://edamontology.org/format_1962' - Label 'strider format'\n",
      "Entity 'http://edamontology.org/format_1963' - Label 'UniProtKB format'\n",
      "Entity 'http://edamontology.org/format_1964' - Label 'plain text format (unformatted)'\n",
      "Entity 'http://edamontology.org/format_1965' - Label 'treecon sequence format'\n",
      "Entity 'http://edamontology.org/format_1966' - Label 'ASN.1 sequence format'\n",
      "Entity 'http://edamontology.org/format_1967' - Label 'DAS format'\n",
      "Entity 'http://edamontology.org/format_1968' - Label 'dasdna'\n",
      "Entity 'http://edamontology.org/format_1969' - Label 'debug-seq'\n",
      "Entity 'http://edamontology.org/format_1970' - Label 'jackknifernon'\n",
      "Entity 'http://edamontology.org/format_1971' - Label 'meganon sequence format'\n",
      "Entity 'http://edamontology.org/format_1972' - Label 'NCBI format'\n",
      "Entity 'http://edamontology.org/format_1973' - Label 'nexusnon'\n",
      "Entity 'http://edamontology.org/format_1974' - Label 'GFF2'\n",
      "Entity 'http://edamontology.org/format_1975' - Label 'GFF3'\n",
      "Entity 'http://edamontology.org/format_1976' - Label 'pir'\n",
      "Entity 'http://edamontology.org/format_1977' - Label 'swiss feature'\n",
      "Entity 'http://edamontology.org/format_1978' - Label 'DASGFF'\n",
      "Entity 'http://edamontology.org/format_1979' - Label 'debug-feat'\n",
      "Entity 'http://edamontology.org/format_1980' - Label 'EMBL feature'\n",
      "Entity 'http://edamontology.org/format_1981' - Label 'GenBank feature'\n",
      "Entity 'http://edamontology.org/format_1982' - Label 'ClustalW format'\n",
      "Entity 'http://edamontology.org/format_1983' - Label 'debug'\n",
      "Entity 'http://edamontology.org/format_1984' - Label 'FASTA-aln'\n",
      "Entity 'http://edamontology.org/format_1985' - Label 'markx0'\n",
      "Entity 'http://edamontology.org/format_1986' - Label 'markx1'\n",
      "Entity 'http://edamontology.org/format_1987' - Label 'markx10'\n",
      "Entity 'http://edamontology.org/format_1988' - Label 'markx2'\n",
      "Entity 'http://edamontology.org/format_1989' - Label 'markx3'\n",
      "Entity 'http://edamontology.org/format_1990' - Label 'match'\n",
      "Entity 'http://edamontology.org/format_1991' - Label 'mega'\n",
      "Entity 'http://edamontology.org/format_1992' - Label 'meganon'\n",
      "Entity 'http://edamontology.org/format_1993' - Label 'msf alignment format'\n",
      "Entity 'http://edamontology.org/format_1994' - Label 'nexus alignment format'\n",
      "Entity 'http://edamontology.org/format_1995' - Label 'nexusnon alignment format'\n",
      "Entity 'http://edamontology.org/format_1996' - Label 'pair'\n",
      "Entity 'http://edamontology.org/format_1997' - Label 'PHYLIP format'\n",
      "Entity 'http://edamontology.org/format_1998' - Label 'PHYLIP sequential'\n",
      "Entity 'http://edamontology.org/format_1999' - Label 'scores format'\n",
      "Entity 'http://edamontology.org/format_2000' - Label 'selex'\n",
      "Entity 'http://edamontology.org/format_2001' - Label 'EMBOSS simple format'\n",
      "Entity 'http://edamontology.org/format_2002' - Label 'srs format'\n",
      "Entity 'http://edamontology.org/format_2003' - Label 'srspair'\n",
      "Entity 'http://edamontology.org/format_2004' - Label 'T-Coffee format'\n",
      "Entity 'http://edamontology.org/format_2005' - Label 'TreeCon-seq'\n",
      "Entity 'http://edamontology.org/format_2006' - Label 'Phylogenetic tree format'\n",
      "Entity 'http://edamontology.org/format_2013' - Label 'Biological pathway or network format'\n",
      "Entity 'http://edamontology.org/format_2014' - Label 'Sequence-profile alignment format'\n",
      "Entity 'http://edamontology.org/format_2015' - Label 'Sequence-profile alignment (HMM) format'\n",
      "Entity 'http://edamontology.org/format_2017' - Label 'Amino acid index format'\n",
      "Entity 'http://edamontology.org/format_2020' - Label 'Article format'\n",
      "Entity 'http://edamontology.org/format_2021' - Label 'Text mining report format'\n",
      "Entity 'http://edamontology.org/format_2027' - Label 'Enzyme kinetics report format'\n",
      "Entity 'http://edamontology.org/format_2030' - Label 'Chemical data format'\n",
      "Entity 'http://edamontology.org/format_2031' - Label 'Gene annotation format'\n",
      "Entity 'http://edamontology.org/format_2032' - Label 'Workflow format'\n",
      "Entity 'http://edamontology.org/format_2033' - Label 'Tertiary structure format'\n",
      "Entity 'http://edamontology.org/format_2034' - Label 'Biological model format'\n",
      "Entity 'http://edamontology.org/format_2035' - Label 'Chemical formula format'\n",
      "Entity 'http://edamontology.org/format_2036' - Label 'Phylogenetic character data format'\n",
      "Entity 'http://edamontology.org/format_2037' - Label 'Phylogenetic continuous quantitative character format'\n",
      "Entity 'http://edamontology.org/format_2038' - Label 'Phylogenetic discrete states format'\n",
      "Entity 'http://edamontology.org/format_2039' - Label 'Phylogenetic tree report (cliques) format'\n",
      "Entity 'http://edamontology.org/format_2040' - Label 'Phylogenetic tree report (invariants) format'\n",
      "Entity 'http://edamontology.org/format_2045' - Label 'Electron microscopy model format'\n",
      "Entity 'http://edamontology.org/format_2049' - Label 'Phylogenetic tree report (tree distances) format'\n",
      "Entity 'http://edamontology.org/format_2051' - Label 'Polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_2052' - Label 'Protein family report format'\n",
      "Entity 'http://edamontology.org/format_2054' - Label 'Protein interaction format'\n",
      "Entity 'http://edamontology.org/format_2055' - Label 'Sequence assembly format'\n",
      "Entity 'http://edamontology.org/format_2056' - Label 'Microarray experiment data format'\n",
      "Entity 'http://edamontology.org/format_2057' - Label 'Sequence trace format'\n",
      "Entity 'http://edamontology.org/format_2058' - Label 'Gene expression report format'\n",
      "Entity 'http://edamontology.org/format_2059' - Label 'Genotype and phenotype annotation format'\n",
      "Entity 'http://edamontology.org/format_2060' - Label 'Map format'\n",
      "Entity 'http://edamontology.org/format_2061' - Label 'Nucleic acid features (primers) format'\n",
      "Entity 'http://edamontology.org/format_2062' - Label 'Protein report format'\n",
      "Entity 'http://edamontology.org/format_2063' - Label 'Protein report (enzyme) format'\n",
      "Entity 'http://edamontology.org/format_2064' - Label '3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_2065' - Label 'Protein structure report (quality evaluation) format'\n",
      "Entity 'http://edamontology.org/format_2066' - Label 'Database hits (sequence) format'\n",
      "Entity 'http://edamontology.org/format_2067' - Label 'Sequence distance matrix format'\n",
      "Entity 'http://edamontology.org/format_2068' - Label 'Sequence motif format'\n",
      "Entity 'http://edamontology.org/format_2069' - Label 'Sequence profile format'\n",
      "Entity 'http://edamontology.org/format_2072' - Label 'Hidden Markov model format'\n",
      "Entity 'http://edamontology.org/format_2074' - Label 'Dirichlet distribution format'\n",
      "Entity 'http://edamontology.org/format_2075' - Label 'HMM emission and transition counts format'\n",
      "Entity 'http://edamontology.org/format_2076' - Label 'RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2077' - Label 'Protein secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2078' - Label 'Sequence range format'\n",
      "Entity 'http://edamontology.org/format_2094' - Label 'pure'\n",
      "Entity 'http://edamontology.org/format_2095' - Label 'unpure'\n",
      "Entity 'http://edamontology.org/format_2096' - Label 'unambiguous sequence'\n",
      "Entity 'http://edamontology.org/format_2097' - Label 'ambiguous'\n",
      "Entity 'http://edamontology.org/format_2155' - Label 'Sequence features (repeats) format'\n",
      "Entity 'http://edamontology.org/format_2158' - Label 'Nucleic acid features (restriction sites) format'\n",
      "Entity 'http://edamontology.org/format_2159' - Label 'Gene features (coding region) format'\n",
      "Entity 'http://edamontology.org/format_2170' - Label 'Sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_2171' - Label 'Sequence cluster format (protein)'\n",
      "Entity 'http://edamontology.org/format_2172' - Label 'Sequence cluster format (nucleic acid)'\n",
      "Entity 'http://edamontology.org/format_2175' - Label 'Gene cluster format'\n",
      "Entity 'http://edamontology.org/format_2181' - Label 'EMBL-like (text)'\n",
      "Entity 'http://edamontology.org/format_2182' - Label 'FASTQ-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2183' - Label 'EMBLXML'\n",
      "Entity 'http://edamontology.org/format_2184' - Label 'cdsxml'\n",
      "Entity 'http://edamontology.org/format_2185' - Label 'INSDSeq'\n",
      "Entity 'http://edamontology.org/format_2186' - Label 'geneseq'\n",
      "Entity 'http://edamontology.org/format_2187' - Label 'UniProt-like (text)'\n",
      "Entity 'http://edamontology.org/format_2188' - Label 'UniProt format'\n",
      "Entity 'http://edamontology.org/format_2189' - Label 'ipi'\n",
      "Entity 'http://edamontology.org/format_2194' - Label 'medline'\n",
      "Entity 'http://edamontology.org/format_2195' - Label 'Ontology format'\n",
      "Entity 'http://edamontology.org/format_2196' - Label 'OBO format'\n",
      "Entity 'http://edamontology.org/format_2197' - Label 'OWL format'\n",
      "Entity 'http://edamontology.org/format_2200' - Label 'FASTA-like (text)'\n",
      "Entity 'http://edamontology.org/format_2202' - Label 'Sequence record full format'\n",
      "Entity 'http://edamontology.org/format_2203' - Label 'Sequence record lite format'\n",
      "Entity 'http://edamontology.org/format_2204' - Label 'EMBL format (XML)'\n",
      "Entity 'http://edamontology.org/format_2205' - Label 'GenBank-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2206' - Label 'Sequence feature table format (text)'\n",
      "Entity 'http://edamontology.org/format_2210' - Label 'Strain data format'\n",
      "Entity 'http://edamontology.org/format_2211' - Label 'CIP strain data format'\n",
      "Entity 'http://edamontology.org/format_2243' - Label 'phylip property values'\n",
      "Entity 'http://edamontology.org/format_2303' - Label 'STRING entry format (HTML)'\n",
      "Entity 'http://edamontology.org/format_2304' - Label 'STRING entry format (XML)'\n",
      "Entity 'http://edamontology.org/format_2305' - Label 'GFF'\n",
      "Entity 'http://edamontology.org/format_2306' - Label 'GTF'\n",
      "Entity 'http://edamontology.org/format_2310' - Label 'FASTA-HTML'\n",
      "Entity 'http://edamontology.org/format_2311' - Label 'EMBL-HTML'\n",
      "Entity 'http://edamontology.org/format_2322' - Label 'BioCyc enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2323' - Label 'ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2328' - Label 'PseudoCAP gene report format'\n",
      "Entity 'http://edamontology.org/format_2329' - Label 'GeneCards gene report format'\n",
      "Entity 'http://edamontology.org/format_2330' - Label 'Textual format'\n",
      "Entity 'http://edamontology.org/format_2331' - Label 'HTML'\n",
      "Entity 'http://edamontology.org/format_2332' - Label 'XML'\n",
      "Entity 'http://edamontology.org/format_2333' - Label 'Binary format'\n",
      "Entity 'http://edamontology.org/format_2334' - Label 'URI format'\n",
      "Entity 'http://edamontology.org/format_2341' - Label 'NCI-Nature pathway entry format'\n",
      "Entity 'http://edamontology.org/format_2350' - Label 'Format (by type of data)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2376' - Label 'RDF format'\n",
      "Entity 'http://edamontology.org/format_2532' - Label 'GenBank-HTML'\n",
      "Entity 'http://edamontology.org/format_2542' - Label 'Protein features (domains) format'\n",
      "Entity 'http://edamontology.org/format_2543' - Label 'EMBL-like format'\n",
      "Entity 'http://edamontology.org/format_2545' - Label 'FASTQ-like format'\n",
      "Entity 'http://edamontology.org/format_2546' - Label 'FASTA-like'\n",
      "Entity 'http://edamontology.org/format_2547' - Label 'uniprotkb-like format'\n",
      "Entity 'http://edamontology.org/format_2548' - Label 'Sequence feature table format'\n",
      "Entity 'http://edamontology.org/format_2549' - Label 'OBO'\n",
      "Entity 'http://edamontology.org/format_2550' - Label 'OBO-XML'\n",
      "Entity 'http://edamontology.org/format_2551' - Label 'Sequence record format (text)'\n",
      "Entity 'http://edamontology.org/format_2552' - Label 'Sequence record format (XML)'\n",
      "Entity 'http://edamontology.org/format_2553' - Label 'Sequence feature table format (XML)'\n",
      "Entity 'http://edamontology.org/format_2554' - Label 'Alignment format (text)'\n",
      "Entity 'http://edamontology.org/format_2555' - Label 'Alignment format (XML)'\n",
      "Entity 'http://edamontology.org/format_2556' - Label 'Phylogenetic tree format (text)'\n",
      "Entity 'http://edamontology.org/format_2557' - Label 'Phylogenetic tree format (XML)'\n",
      "Entity 'http://edamontology.org/format_2558' - Label 'EMBL-like (XML)'\n",
      "Entity 'http://edamontology.org/format_2559' - Label 'GenBank-like format'\n",
      "Entity 'http://edamontology.org/format_2560' - Label 'STRING entry format'\n",
      "Entity 'http://edamontology.org/format_2561' - Label 'Sequence assembly format (text)'\n",
      "Entity 'http://edamontology.org/format_2562' - Label 'Amino acid identifier format'\n",
      "Entity 'http://edamontology.org/format_2566' - Label 'completely unambiguous'\n",
      "Entity 'http://edamontology.org/format_2567' - Label 'completely unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_2568' - Label 'completely unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_2569' - Label 'completely unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_2570' - Label 'completely unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_2571' - Label 'Raw sequence format'\n",
      "Entity 'http://edamontology.org/format_2572' - Label 'BAM'\n",
      "Entity 'http://edamontology.org/format_2573' - Label 'SAM'\n",
      "Entity 'http://edamontology.org/format_2585' - Label 'SBML'\n",
      "Entity 'http://edamontology.org/format_2607' - Label 'completely unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_2848' - Label 'Bibliographic reference format'\n",
      "Entity 'http://edamontology.org/format_2919' - Label 'Sequence annotation track format'\n",
      "Entity 'http://edamontology.org/format_2920' - Label 'Alignment format (pair only)'\n",
      "Entity 'http://edamontology.org/format_2921' - Label 'Sequence variation annotation format'\n",
      "Entity 'http://edamontology.org/format_2922' - Label 'markx0 variant'\n",
      "Entity 'http://edamontology.org/format_2923' - Label 'mega variant'\n",
      "Entity 'http://edamontology.org/format_2924' - Label 'Phylip format variant'\n",
      "Entity 'http://edamontology.org/format_3000' - Label 'AB1'\n",
      "Entity 'http://edamontology.org/format_3001' - Label 'ACE'\n",
      "Entity 'http://edamontology.org/format_3003' - Label 'BED'\n",
      "Entity 'http://edamontology.org/format_3004' - Label 'bigBed'\n",
      "Entity 'http://edamontology.org/format_3005' - Label 'WIG'\n",
      "Entity 'http://edamontology.org/format_3006' - Label 'bigWig'\n",
      "Entity 'http://edamontology.org/format_3007' - Label 'PSL'\n",
      "Entity 'http://edamontology.org/format_3008' - Label 'MAF'\n",
      "Entity 'http://edamontology.org/format_3009' - Label '2bit'\n",
      "Entity 'http://edamontology.org/format_3010' - Label '.nib'\n",
      "Entity 'http://edamontology.org/format_3011' - Label 'genePred'\n",
      "Entity 'http://edamontology.org/format_3012' - Label 'pgSnp'\n",
      "Entity 'http://edamontology.org/format_3013' - Label 'axt'\n",
      "Entity 'http://edamontology.org/format_3014' - Label 'LAV'\n",
      "Entity 'http://edamontology.org/format_3015' - Label 'Pileup'\n",
      "Entity 'http://edamontology.org/format_3016' - Label 'VCF'\n",
      "Entity 'http://edamontology.org/format_3017' - Label 'SRF'\n",
      "Entity 'http://edamontology.org/format_3018' - Label 'ZTR'\n",
      "Entity 'http://edamontology.org/format_3019' - Label 'GVF'\n",
      "Entity 'http://edamontology.org/format_3020' - Label 'BCF'\n",
      "Entity 'http://edamontology.org/format_3033' - Label 'Matrix format'\n",
      "Entity 'http://edamontology.org/format_3097' - Label 'Protein domain classification format'\n",
      "Entity 'http://edamontology.org/format_3098' - Label 'Raw SCOP domain classification format'\n",
      "Entity 'http://edamontology.org/format_3099' - Label 'Raw CATH domain classification format'\n",
      "Entity 'http://edamontology.org/format_3100' - Label 'CATH domain report format'\n",
      "Entity 'http://edamontology.org/format_3155' - Label 'SBRML'\n",
      "Entity 'http://edamontology.org/format_3156' - Label 'BioPAX'\n",
      "Entity 'http://edamontology.org/format_3157' - Label 'EBI Application Result XML'\n",
      "Entity 'http://edamontology.org/format_3158' - Label 'PSI MI XML (MIF)'\n",
      "Entity 'http://edamontology.org/format_3159' - Label 'phyloXML'\n",
      "Entity 'http://edamontology.org/format_3160' - Label 'NeXML'\n",
      "Entity 'http://edamontology.org/format_3161' - Label 'MAGE-ML'\n",
      "Entity 'http://edamontology.org/format_3162' - Label 'MAGE-TAB'\n",
      "Entity 'http://edamontology.org/format_3163' - Label 'GCDML'\n",
      "Entity 'http://edamontology.org/format_3164' - Label 'GTrack'\n",
      "Entity 'http://edamontology.org/format_3166' - Label 'Biological pathway or network report format'\n",
      "Entity 'http://edamontology.org/format_3167' - Label 'Experiment annotation format'\n",
      "Entity 'http://edamontology.org/format_3235' - Label 'Cytoband format'\n",
      "Entity 'http://edamontology.org/format_3239' - Label 'CopasiML'\n",
      "Entity 'http://edamontology.org/format_3240' - Label 'CellML'\n",
      "Entity 'http://edamontology.org/format_3242' - Label 'PSI MI TAB (MITAB)'\n",
      "Entity 'http://edamontology.org/format_3243' - Label 'PSI-PAR'\n",
      "Entity 'http://edamontology.org/format_3244' - Label 'mzML'\n",
      "Entity 'http://edamontology.org/format_3245' - Label 'Mass spectrometry data format'\n",
      "Entity 'http://edamontology.org/format_3246' - Label 'TraML'\n",
      "Entity 'http://edamontology.org/format_3247' - Label 'mzIdentML'\n",
      "Entity 'http://edamontology.org/format_3248' - Label 'mzQuantML'\n",
      "Entity 'http://edamontology.org/format_3249' - Label 'GelML'\n",
      "Entity 'http://edamontology.org/format_3250' - Label 'spML'\n",
      "Entity 'http://edamontology.org/format_3252' - Label 'OWL Functional Syntax'\n",
      "Entity 'http://edamontology.org/format_3253' - Label 'Manchester OWL Syntax'\n",
      "Entity 'http://edamontology.org/format_3254' - Label 'KRSS2 Syntax'\n",
      "Entity 'http://edamontology.org/format_3255' - Label 'Turtle'\n",
      "Entity 'http://edamontology.org/format_3256' - Label 'N-Triples'\n",
      "Entity 'http://edamontology.org/format_3257' - Label 'Notation3'\n",
      "Entity 'http://edamontology.org/format_3261' - Label 'RDF/XML'\n",
      "Entity 'http://edamontology.org/format_3262' - Label 'OWL/XML'\n",
      "Entity 'http://edamontology.org/format_3281' - Label 'A2M'\n",
      "Entity 'http://edamontology.org/format_3284' - Label 'SFF'\n",
      "Entity 'http://edamontology.org/format_3285' - Label 'MAP'\n",
      "Entity 'http://edamontology.org/format_3286' - Label 'PED'\n",
      "Entity 'http://edamontology.org/format_3287' - Label 'Individual genetic data format'\n",
      "Entity 'http://edamontology.org/format_3288' - Label 'PED/MAP'\n",
      "Entity 'http://edamontology.org/format_3309' - Label 'CT'\n",
      "Entity 'http://edamontology.org/format_3310' - Label 'SS'\n",
      "Entity 'http://edamontology.org/format_3311' - Label 'RNAML'\n",
      "Entity 'http://edamontology.org/format_3312' - Label 'GDE'\n",
      "Entity 'http://edamontology.org/format_3313' - Label 'BLC'\n",
      "Entity 'http://edamontology.org/format_3326' - Label 'Data index format'\n",
      "Entity 'http://edamontology.org/format_3327' - Label 'BAI'\n",
      "Entity 'http://edamontology.org/format_3328' - Label 'HMMER2'\n",
      "Entity 'http://edamontology.org/format_3329' - Label 'HMMER3'\n",
      "Entity 'http://edamontology.org/format_3330' - Label 'PO'\n",
      "Entity 'http://edamontology.org/format_3331' - Label 'BLAST XML results format'\n",
      "Entity 'http://edamontology.org/format_3462' - Label 'CRAM'\n",
      "Entity 'http://edamontology.org/format_3464' - Label 'JSON'\n",
      "Entity 'http://edamontology.org/format_3466' - Label 'EPS'\n",
      "Entity 'http://edamontology.org/format_3467' - Label 'GIF'\n",
      "Entity 'http://edamontology.org/format_3468' - Label 'xls'\n",
      "Entity 'http://edamontology.org/format_3475' - Label 'TSV'\n",
      "Entity 'http://edamontology.org/format_3476' - Label 'Gene expression data format'\n",
      "Entity 'http://edamontology.org/format_3477' - Label 'Cytoscape input file format'\n",
      "Entity 'http://edamontology.org/format_3484' - Label 'ebwt'\n",
      "Entity 'http://edamontology.org/format_3485' - Label 'RSF'\n",
      "Entity 'http://edamontology.org/format_3486' - Label 'GCG format variant'\n",
      "Entity 'http://edamontology.org/format_3487' - Label 'BSML'\n",
      "Entity 'http://edamontology.org/format_3491' - Label 'ebwtl'\n",
      "Entity 'http://edamontology.org/format_3499' - Label 'Ensembl variation file format'\n",
      "Entity 'http://edamontology.org/format_3506' - Label 'docx'\n",
      "Entity 'http://edamontology.org/format_3507' - Label 'Document format'\n",
      "Entity 'http://edamontology.org/format_3508' - Label 'PDF'\n",
      "Entity 'http://edamontology.org/format_3547' - Label 'Image format'\n",
      "Entity 'http://edamontology.org/format_3548' - Label 'DICOM format'\n",
      "Entity 'http://edamontology.org/format_3549' - Label 'nii'\n",
      "Entity 'http://edamontology.org/format_3550' - Label 'mhd'\n",
      "Entity 'http://edamontology.org/format_3551' - Label 'nrrd'\n",
      "Entity 'http://edamontology.org/format_3554' - Label 'R file format'\n",
      "Entity 'http://edamontology.org/format_3555' - Label 'SPSS'\n",
      "Entity 'http://edamontology.org/format_3556' - Label 'MHTML'\n",
      "Entity 'http://edamontology.org/format_3578' - Label 'IDAT'\n",
      "Entity 'http://edamontology.org/format_3579' - Label 'JPG'\n",
      "Entity 'http://edamontology.org/format_3580' - Label 'rcc'\n",
      "Entity 'http://edamontology.org/format_3581' - Label 'arff'\n",
      "Entity 'http://edamontology.org/format_3582' - Label 'afg'\n",
      "Entity 'http://edamontology.org/format_3583' - Label 'bedgraph'\n",
      "Entity 'http://edamontology.org/format_3584' - Label 'bedstrict'\n",
      "Entity 'http://edamontology.org/format_3585' - Label 'bed6'\n",
      "Entity 'http://edamontology.org/format_3586' - Label 'bed12'\n",
      "Entity 'http://edamontology.org/format_3587' - Label 'chrominfo'\n",
      "Entity 'http://edamontology.org/format_3588' - Label 'customtrack'\n",
      "Entity 'http://edamontology.org/format_3589' - Label 'csfasta'\n",
      "Entity 'http://edamontology.org/format_3590' - Label 'HDF5'\n",
      "Entity 'http://edamontology.org/format_3591' - Label 'TIFF'\n",
      "Entity 'http://edamontology.org/format_3592' - Label 'BMP'\n",
      "Entity 'http://edamontology.org/format_3593' - Label 'im'\n",
      "Entity 'http://edamontology.org/format_3594' - Label 'pcd'\n",
      "Entity 'http://edamontology.org/format_3595' - Label 'pcx'\n",
      "Entity 'http://edamontology.org/format_3596' - Label 'ppm'\n",
      "Entity 'http://edamontology.org/format_3597' - Label 'psd'\n",
      "Entity 'http://edamontology.org/format_3598' - Label 'xbm'\n",
      "Entity 'http://edamontology.org/format_3599' - Label 'xpm'\n",
      "Entity 'http://edamontology.org/format_3600' - Label 'rgb'\n",
      "Entity 'http://edamontology.org/format_3601' - Label 'pbm'\n",
      "Entity 'http://edamontology.org/format_3602' - Label 'pgm'\n",
      "Entity 'http://edamontology.org/format_3603' - Label 'PNG'\n",
      "Entity 'http://edamontology.org/format_3604' - Label 'SVG'\n",
      "Entity 'http://edamontology.org/format_3605' - Label 'rast'\n",
      "Entity 'http://edamontology.org/format_3606' - Label 'Sequence quality report format (text)'\n",
      "Entity 'http://edamontology.org/format_3607' - Label 'qual'\n",
      "Entity 'http://edamontology.org/format_3608' - Label 'qualsolexa'\n",
      "Entity 'http://edamontology.org/format_3609' - Label 'qualillumina'\n",
      "Entity 'http://edamontology.org/format_3610' - Label 'qualsolid'\n",
      "Entity 'http://edamontology.org/format_3611' - Label 'qual454'\n",
      "Entity 'http://edamontology.org/format_3612' - Label 'ENCODE peak format'\n",
      "Entity 'http://edamontology.org/format_3613' - Label 'ENCODE narrow peak format'\n",
      "Entity 'http://edamontology.org/format_3614' - Label 'ENCODE broad peak format'\n",
      "Entity 'http://edamontology.org/format_3615' - Label 'bgzip'\n",
      "Entity 'http://edamontology.org/format_3616' - Label 'tabix'\n",
      "Entity 'http://edamontology.org/format_3617' - Label 'Graph format'\n",
      "Entity 'http://edamontology.org/format_3618' - Label 'xgmml'\n",
      "Entity 'http://edamontology.org/format_3619' - Label 'sif'\n",
      "Entity 'http://edamontology.org/format_3620' - Label 'xlsx'\n",
      "Entity 'http://edamontology.org/format_3621' - Label 'SQLite format'\n",
      "Entity 'http://edamontology.org/format_3622' - Label 'Gemini SQLite format'\n",
      "Entity 'http://edamontology.org/format_3623' - Label 'Index format'\n",
      "Entity 'http://edamontology.org/format_3624' - Label 'snpeffdb'\n",
      "Entity 'http://edamontology.org/format_3626' - Label 'MAT'\n",
      "Entity 'http://edamontology.org/format_3650' - Label 'netCDF'\n",
      "Entity 'http://edamontology.org/format_3651' - Label 'MGF'\n",
      "Entity 'http://edamontology.org/format_3652' - Label 'dta'\n",
      "Entity 'http://edamontology.org/format_3653' - Label 'pkl'\n",
      "Entity 'http://edamontology.org/format_3654' - Label 'mzXML'\n",
      "Entity 'http://edamontology.org/format_3655' - Label 'pepXML'\n",
      "Entity 'http://edamontology.org/format_3657' - Label 'GPML'\n",
      "Entity 'http://edamontology.org/format_3665' - Label 'K-mer countgraph'\n",
      "Entity 'http://edamontology.org/format_3681' - Label 'mzTab'\n",
      "Entity 'http://edamontology.org/format_3682' - Label 'imzML metadata file'\n",
      "Entity 'http://edamontology.org/format_3683' - Label 'qcML'\n",
      "Entity 'http://edamontology.org/format_3684' - Label 'PRIDE XML'\n",
      "Entity 'http://edamontology.org/format_3685' - Label 'SED-ML'\n",
      "Entity 'http://edamontology.org/format_3686' - Label 'COMBINE OMEX'\n",
      "Entity 'http://edamontology.org/format_3687' - Label 'ISA-TAB'\n",
      "Entity 'http://edamontology.org/format_3688' - Label 'SBtab'\n",
      "Entity 'http://edamontology.org/format_3689' - Label 'BCML'\n",
      "Entity 'http://edamontology.org/format_3690' - Label 'BDML'\n",
      "Entity 'http://edamontology.org/format_3691' - Label 'BEL'\n",
      "Entity 'http://edamontology.org/format_3692' - Label 'SBGN-ML'\n",
      "Entity 'http://edamontology.org/format_3693' - Label 'AGP'\n",
      "Entity 'http://edamontology.org/format_3696' - Label 'PS'\n",
      "Entity 'http://edamontology.org/format_3698' - Label 'SRA format'\n",
      "Entity 'http://edamontology.org/format_3699' - Label 'VDB'\n",
      "Entity 'http://edamontology.org/format_3700' - Label 'Tabix index file format'\n",
      "Entity 'http://edamontology.org/format_3701' - Label 'Sequin format'\n",
      "Entity 'http://edamontology.org/format_3702' - Label 'MSF'\n",
      "Entity 'http://edamontology.org/format_3706' - Label 'Biodiversity data format'\n",
      "Entity 'http://edamontology.org/format_3708' - Label 'ABCD format'\n",
      "Entity 'http://edamontology.org/format_3709' - Label 'GCT/Res format'\n",
      "Entity 'http://edamontology.org/format_3710' - Label 'WIFF format'\n",
      "Entity 'http://edamontology.org/format_3711' - Label 'X!Tandem XML'\n",
      "Entity 'http://edamontology.org/format_3712' - Label 'Thermo RAW'\n",
      "Entity 'http://edamontology.org/format_3713' - Label 'Mascot .dat file'\n",
      "Entity 'http://edamontology.org/format_3714' - Label 'MaxQuant APL peaklist format'\n",
      "Entity 'http://edamontology.org/format_3725' - Label 'SBOL'\n",
      "Entity 'http://edamontology.org/format_3726' - Label 'PMML'\n",
      "Entity 'http://edamontology.org/format_3727' - Label 'OME-TIFF'\n",
      "Entity 'http://edamontology.org/format_3728' - Label 'LocARNA PP'\n",
      "Entity 'http://edamontology.org/format_3729' - Label 'dbGaP format'\n",
      "Entity 'http://edamontology.org/format_3746' - Label 'BIOM format'\n",
      "Entity 'http://edamontology.org/format_3747' - Label 'protXML'\n",
      "Entity 'http://edamontology.org/format_3748' - Label 'Linked data format'\n",
      "Entity 'http://edamontology.org/format_3749' - Label 'JSON-LD'\n",
      "Entity 'http://edamontology.org/format_3750' - Label 'YAML'\n",
      "Entity 'http://edamontology.org/format_3751' - Label 'DSV'\n",
      "Entity 'http://edamontology.org/format_3752' - Label 'CSV'\n",
      "Entity 'http://edamontology.org/format_3758' - Label 'SEQUEST .out file'\n",
      "Entity 'http://edamontology.org/format_3764' - Label 'idXML'\n",
      "Entity 'http://edamontology.org/format_3765' - Label 'KNIME datatable format'\n",
      "Entity 'http://edamontology.org/format_3770' - Label 'UniProtKB XML'\n",
      "Entity 'http://edamontology.org/format_3771' - Label 'UniProtKB RDF'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3774' - Label 'BioJSON (Jalview)'\n",
      "Entity 'http://edamontology.org/format_3775' - Label 'GSuite'\n",
      "Entity 'http://edamontology.org/format_3776' - Label 'BTrack'\n",
      "Entity 'http://edamontology.org/format_3777' - Label 'MCPD'\n",
      "Entity 'http://edamontology.org/format_3780' - Label 'Annotated text format'\n",
      "Entity 'http://edamontology.org/format_3781' - Label 'PubAnnotation format'\n",
      "Entity 'http://edamontology.org/format_3782' - Label 'BioC'\n",
      "Entity 'http://edamontology.org/format_3783' - Label 'PubTator format'\n",
      "Entity 'http://edamontology.org/format_3784' - Label 'Open Annotation format'\n",
      "Entity 'http://edamontology.org/format_3785' - Label 'BioNLP Shared Task format'\n",
      "Entity 'http://edamontology.org/format_3787' - Label 'Query language'\n",
      "Entity 'http://edamontology.org/format_3788' - Label 'SQL'\n",
      "Entity 'http://edamontology.org/format_3789' - Label 'XQuery'\n",
      "Entity 'http://edamontology.org/format_3790' - Label 'SPARQL'\n",
      "Entity 'http://edamontology.org/format_3804' - Label 'xsd'\n",
      "Entity 'http://edamontology.org/format_3811' - Label 'XMFA'\n",
      "Entity 'http://edamontology.org/format_3812' - Label 'GEN'\n",
      "Entity 'http://edamontology.org/format_3813' - Label 'SAMPLE file format'\n",
      "Entity 'http://edamontology.org/format_3814' - Label 'SDF'\n",
      "Entity 'http://edamontology.org/format_3815' - Label 'Molfile'\n",
      "Entity 'http://edamontology.org/format_3816' - Label 'Mol2'\n",
      "Entity 'http://edamontology.org/format_3817' - Label 'latex'\n",
      "Entity 'http://edamontology.org/format_3818' - Label 'ELAND format'\n",
      "Entity 'http://edamontology.org/format_3819' - Label 'Relaxed PHYLIP Interleaved'\n",
      "Entity 'http://edamontology.org/format_3820' - Label 'Relaxed PHYLIP Sequential'\n",
      "Entity 'http://edamontology.org/format_3821' - Label 'VisML'\n",
      "Entity 'http://edamontology.org/format_3822' - Label 'GML'\n",
      "Entity 'http://edamontology.org/format_3823' - Label 'FASTG'\n",
      "Entity 'http://edamontology.org/format_3824' - Label 'NMR data format'\n",
      "Entity 'http://edamontology.org/format_3825' - Label 'nmrML'\n",
      "Entity 'http://edamontology.org/format_3826' - Label 'proBAM'\n",
      "Entity 'http://edamontology.org/format_3827' - Label 'proBED'\n",
      "Entity 'http://edamontology.org/format_3828' - Label 'Raw microarray data format'\n",
      "Entity 'http://edamontology.org/format_3829' - Label 'GPR'\n",
      "Entity 'http://edamontology.org/format_3830' - Label 'ARB'\n",
      "Entity 'http://edamontology.org/format_3832' - Label 'consensusXML'\n",
      "Entity 'http://edamontology.org/format_3833' - Label 'featureXML'\n",
      "Entity 'http://edamontology.org/format_3834' - Label 'mzData'\n",
      "Entity 'http://edamontology.org/format_3835' - Label 'TIDE TXT'\n",
      "Entity 'http://edamontology.org/format_3836' - Label 'BLAST XML v2 results format'\n",
      "Entity 'http://edamontology.org/format_3838' - Label 'pptx'\n",
      "Entity 'http://edamontology.org/format_3839' - Label 'ibd'\n",
      "Entity 'http://edamontology.org/format_3841' - Label 'NLP format'\n",
      "Entity 'http://edamontology.org/format_3843' - Label 'BEAST'\n",
      "Entity 'http://edamontology.org/format_3844' - Label 'Chado-XML'\n",
      "Entity 'http://edamontology.org/format_3845' - Label 'HSAML'\n",
      "Entity 'http://edamontology.org/format_3846' - Label 'InterProScan XML'\n",
      "Entity 'http://edamontology.org/format_3847' - Label 'KGML'\n",
      "Entity 'http://edamontology.org/format_3848' - Label 'PubMed XML'\n",
      "Entity 'http://edamontology.org/format_3849' - Label 'MSAML'\n",
      "Entity 'http://edamontology.org/format_3850' - Label 'OrthoXML'\n",
      "Entity 'http://edamontology.org/format_3851' - Label 'PSDML'\n",
      "Entity 'http://edamontology.org/format_3852' - Label 'SeqXML'\n",
      "Entity 'http://edamontology.org/format_3853' - Label 'UniParc XML'\n",
      "Entity 'http://edamontology.org/format_3854' - Label 'UniRef XML'\n",
      "Entity 'http://edamontology.org/format_3857' - Label 'CWL'\n",
      "Entity 'http://edamontology.org/format_3858' - Label 'Waters RAW'\n",
      "Entity 'http://edamontology.org/format_3859' - Label 'JCAMP-DX'\n",
      "Entity 'http://edamontology.org/format_3862' - Label 'NLP annotation format'\n",
      "Entity 'http://edamontology.org/format_3863' - Label 'NLP corpus format'\n",
      "Entity 'http://edamontology.org/format_3864' - Label 'mirGFF3'\n",
      "Entity 'http://edamontology.org/format_3865' - Label 'RNA annotation format'\n",
      "Entity 'http://edamontology.org/format_3866' - Label 'Trajectory format'\n",
      "Entity 'http://edamontology.org/format_3867' - Label 'Trajectory format (binary)'\n",
      "Entity 'http://edamontology.org/format_3868' - Label 'Trajectory format (text)'\n",
      "Entity 'http://edamontology.org/format_3873' - Label 'HDF'\n",
      "Entity 'http://edamontology.org/format_3874' - Label 'PCAzip'\n",
      "Entity 'http://edamontology.org/format_3875' - Label 'XTC'\n",
      "Entity 'http://edamontology.org/format_3876' - Label 'TNG'\n",
      "Entity 'http://edamontology.org/format_3877' - Label 'XYZ'\n",
      "Entity 'http://edamontology.org/format_3878' - Label 'mdcrd'\n",
      "Entity 'http://edamontology.org/format_3879' - Label 'Topology format'\n",
      "Entity 'http://edamontology.org/format_3880' - Label 'GROMACS top'\n",
      "Entity 'http://edamontology.org/format_3881' - Label 'AMBER top'\n",
      "Entity 'http://edamontology.org/format_3882' - Label 'PSF'\n",
      "Entity 'http://edamontology.org/format_3883' - Label 'GROMACS itp'\n",
      "Entity 'http://edamontology.org/format_3884' - Label 'FF parameter format'\n",
      "Entity 'http://edamontology.org/format_3885' - Label 'BinPos'\n",
      "Entity 'http://edamontology.org/format_3886' - Label 'RST'\n",
      "Entity 'http://edamontology.org/format_3887' - Label 'CHARMM rtf'\n",
      "Entity 'http://edamontology.org/format_3888' - Label 'AMBER frcmod'\n",
      "Entity 'http://edamontology.org/format_3889' - Label 'AMBER off'\n",
      "Entity 'http://edamontology.org/format_3906' - Label 'NMReDATA'\n",
      "Entity 'http://edamontology.org/format_3909' - Label 'BpForms'\n",
      "Entity 'http://edamontology.org/format_3910' - Label 'trr'\n",
      "Entity 'http://edamontology.org/format_3911' - Label 'msh'\n",
      "Entity 'http://edamontology.org/format_3913' - Label 'Loom'\n",
      "Entity 'http://edamontology.org/format_3915' - Label 'Zarr'\n",
      "Entity 'http://edamontology.org/format_3916' - Label 'MTX'\n",
      "Entity 'http://edamontology.org/format_3951' - Label 'BcForms'\n",
      "Entity 'http://edamontology.org/format_3956' - Label 'N-Quads'\n",
      "Entity 'http://edamontology.org/format_3969' - Label 'Vega'\n",
      "Entity 'http://edamontology.org/format_3970' - Label 'Vega-lite'\n",
      "Entity 'http://edamontology.org/format_3971' - Label 'NeuroML'\n",
      "Entity 'http://edamontology.org/format_3972' - Label 'BNGL'\n",
      "Entity 'http://edamontology.org/format_3973' - Label 'Docker image'\n",
      "Entity 'http://edamontology.org/format_3975' - Label 'GFA 1'\n",
      "Entity 'http://edamontology.org/format_3976' - Label 'GFA 2'\n",
      "Entity 'http://edamontology.org/format_3977' - Label 'ObjTables'\n",
      "Entity 'http://edamontology.org/format_3978' - Label 'CONTIG'\n",
      "Entity 'http://edamontology.org/format_3979' - Label 'WEGO'\n",
      "Entity 'http://edamontology.org/format_3980' - Label 'RPKM'\n",
      "Entity 'http://edamontology.org/format_3981' - Label 'TAR format'\n",
      "Entity 'http://edamontology.org/format_3982' - Label 'CHAIN'\n",
      "Entity 'http://edamontology.org/format_3983' - Label 'NET'\n",
      "Entity 'http://edamontology.org/format_3984' - Label 'QMAP'\n",
      "Entity 'http://edamontology.org/format_3985' - Label 'gxformat2'\n",
      "Entity 'http://edamontology.org/format_3986' - Label 'WMV'\n",
      "Entity 'http://edamontology.org/format_3987' - Label 'ZIP format'\n",
      "Entity 'http://edamontology.org/format_3988' - Label 'LSM'\n",
      "Entity 'http://edamontology.org/format_3989' - Label 'GZIP format'\n",
      "Entity 'http://edamontology.org/format_3990' - Label 'AVI'\n",
      "Entity 'http://edamontology.org/format_3991' - Label 'TrackDB'\n",
      "Entity 'http://edamontology.org/format_3992' - Label 'CIGAR format'\n",
      "Entity 'http://edamontology.org/format_3993' - Label 'Stereolithography format'\n",
      "Entity 'http://edamontology.org/format_3994' - Label 'U3D'\n",
      "Entity 'http://edamontology.org/format_3995' - Label 'Texture file format'\n",
      "Entity 'http://edamontology.org/format_3996' - Label 'Python script'\n",
      "Entity 'http://edamontology.org/format_3997' - Label 'MPEG-4'\n",
      "Entity 'http://edamontology.org/format_3998' - Label 'Perl script'\n",
      "Entity 'http://edamontology.org/format_3999' - Label 'R script'\n",
      "Entity 'http://edamontology.org/format_4000' - Label 'R markdown'\n",
      "Entity 'http://edamontology.org/format_4001' - Label 'NIFTI format'\n",
      "Entity 'http://edamontology.org/format_4002' - Label 'pickle'\n",
      "Entity 'http://edamontology.org/format_4003' - Label 'NumPy format'\n",
      "Entity 'http://edamontology.org/format_4004' - Label 'SimTools repertoire file format'\n",
      "Entity 'http://edamontology.org/format_4005' - Label 'Configuration file format'\n",
      "Entity 'http://edamontology.org/format_4006' - Label 'Zstandard format'\n",
      "Entity 'http://edamontology.org/format_4007' - Label 'MATLAB script'\n",
      "Entity 'http://edamontology.org/format_4015' - Label 'PEtab'\n",
      "Entity 'http://edamontology.org/format_4018' - Label 'gVCF'\n",
      "Entity 'http://edamontology.org/format_4023' - Label 'cml'\n",
      "Entity 'http://edamontology.org/format_4024' - Label 'cif'\n",
      "Entity 'http://edamontology.org/format_4025' - Label 'BioSimulators format for the specifications of biosimulation tools'\n",
      "Entity 'http://edamontology.org/format_4026' - Label 'BioSimulators standard for command-line interfaces for biosimulation tools'\n",
      "Entity 'http://edamontology.org/format_4035' - Label 'PQR'\n",
      "Entity 'http://edamontology.org/format_4036' - Label 'PDBQT'\n",
      "Entity 'http://edamontology.org/format_4039' - Label 'MSP'\n",
      "Entity 'http://edamontology.org/operation_0004' - Label 'Operation'\n",
      "Entity 'http://edamontology.org/operation_0224' - Label 'Query and retrieval'\n",
      "Entity 'http://edamontology.org/operation_0225' - Label 'Data retrieval (database cross-reference)'\n",
      "Entity 'http://edamontology.org/operation_0226' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/operation_0227' - Label 'Indexing'\n",
      "Entity 'http://edamontology.org/operation_0228' - Label 'Data index analysis'\n",
      "Entity 'http://edamontology.org/operation_0229' - Label 'Annotation retrieval (sequence)'\n",
      "Entity 'http://edamontology.org/operation_0230' - Label 'Sequence generation'\n",
      "Entity 'http://edamontology.org/operation_0231' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/operation_0232' - Label 'Sequence merging'\n",
      "Entity 'http://edamontology.org/operation_0233' - Label 'Sequence conversion'\n",
      "Entity 'http://edamontology.org/operation_0234' - Label 'Sequence complexity calculation'\n",
      "Entity 'http://edamontology.org/operation_0235' - Label 'Sequence ambiguity calculation'\n",
      "Entity 'http://edamontology.org/operation_0236' - Label 'Sequence composition calculation'\n",
      "Entity 'http://edamontology.org/operation_0237' - Label 'Repeat sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0238' - Label 'Sequence motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0239' - Label 'Sequence motif recognition'\n",
      "Entity 'http://edamontology.org/operation_0240' - Label 'Sequence motif comparison'\n",
      "Entity 'http://edamontology.org/operation_0241' - Label 'Transcription regulatory sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0242' - Label 'Conserved transcription regulatory sequence identification'\n",
      "Entity 'http://edamontology.org/operation_0243' - Label 'Protein property calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0244' - Label 'Simulation analysis'\n",
      "Entity 'http://edamontology.org/operation_0245' - Label 'Structural motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0246' - Label 'Protein domain recognition'\n",
      "Entity 'http://edamontology.org/operation_0247' - Label 'Protein architecture analysis'\n",
      "Entity 'http://edamontology.org/operation_0248' - Label 'Residue interaction calculation'\n",
      "Entity 'http://edamontology.org/operation_0249' - Label 'Protein geometry calculation'\n",
      "Entity 'http://edamontology.org/operation_0250' - Label 'Protein property calculation'\n",
      "Entity 'http://edamontology.org/operation_0252' - Label 'Peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0253' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_0254' - Label 'Data retrieval (feature table)'\n",
      "Entity 'http://edamontology.org/operation_0255' - Label 'Feature table query'\n",
      "Entity 'http://edamontology.org/operation_0256' - Label 'Sequence feature comparison'\n",
      "Entity 'http://edamontology.org/operation_0257' - Label 'Data retrieval (sequence alignment)'\n",
      "Entity 'http://edamontology.org/operation_0258' - Label 'Sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_0259' - Label 'Sequence alignment comparison'\n",
      "Entity 'http://edamontology.org/operation_0260' - Label 'Sequence alignment conversion'\n",
      "Entity 'http://edamontology.org/operation_0261' - Label 'Nucleic acid property processing'\n",
      "Entity 'http://edamontology.org/operation_0262' - Label 'Nucleic acid property calculation'\n",
      "Entity 'http://edamontology.org/operation_0264' - Label 'Alternative splicing prediction'\n",
      "Entity 'http://edamontology.org/operation_0265' - Label 'Frameshift detection'\n",
      "Entity 'http://edamontology.org/operation_0266' - Label 'Vector sequence detection'\n",
      "Entity 'http://edamontology.org/operation_0267' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0268' - Label 'Protein super-secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0269' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/operation_0270' - Label 'Transmembrane protein analysis'\n",
      "Entity 'http://edamontology.org/operation_0271' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0272' - Label 'Residue contact prediction'\n",
      "Entity 'http://edamontology.org/operation_0273' - Label 'Protein interaction raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_0274' - Label 'Protein-protein interaction prediction (from protein sequence)'\n",
      "Entity 'http://edamontology.org/operation_0275' - Label 'Protein-protein interaction prediction (from protein structure)'\n",
      "Entity 'http://edamontology.org/operation_0276' - Label 'Protein interaction network analysis'\n",
      "Entity 'http://edamontology.org/operation_0277' - Label 'Pathway or network comparison'\n",
      "Entity 'http://edamontology.org/operation_0278' - Label 'RNA secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0279' - Label 'Nucleic acid folding analysis'\n",
      "Entity 'http://edamontology.org/operation_0280' - Label 'Data retrieval (restriction enzyme annotation)'\n",
      "Entity 'http://edamontology.org/operation_0281' - Label 'Genetic marker identification'\n",
      "Entity 'http://edamontology.org/operation_0282' - Label 'Genetic mapping'\n",
      "Entity 'http://edamontology.org/operation_0283' - Label 'Linkage analysis'\n",
      "Entity 'http://edamontology.org/operation_0284' - Label 'Codon usage table generation'\n",
      "Entity 'http://edamontology.org/operation_0285' - Label 'Codon usage table comparison'\n",
      "Entity 'http://edamontology.org/operation_0286' - Label 'Codon usage analysis'\n",
      "Entity 'http://edamontology.org/operation_0287' - Label 'Base position variability plotting'\n",
      "Entity 'http://edamontology.org/operation_0288' - Label 'Sequence word comparison'\n",
      "Entity 'http://edamontology.org/operation_0289' - Label 'Sequence distance matrix generation'\n",
      "Entity 'http://edamontology.org/operation_0290' - Label 'Sequence redundancy removal'\n",
      "Entity 'http://edamontology.org/operation_0291' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/operation_0292' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0293' - Label 'Hybrid sequence alignment construction'\n",
      "Entity 'http://edamontology.org/operation_0294' - Label 'Structure-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0295' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0296' - Label 'Sequence profile generation'\n",
      "Entity 'http://edamontology.org/operation_0297' - Label '3D profile generation'\n",
      "Entity 'http://edamontology.org/operation_0298' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0299' - Label '3D profile-to-3D profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0300' - Label 'Sequence profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0301' - Label 'Sequence-to-3D-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0302' - Label 'Protein threading'\n",
      "Entity 'http://edamontology.org/operation_0303' - Label 'Fold recognition'\n",
      "Entity 'http://edamontology.org/operation_0304' - Label 'Metadata retrieval'\n",
      "Entity 'http://edamontology.org/operation_0305' - Label 'Literature search'\n",
      "Entity 'http://edamontology.org/operation_0306' - Label 'Text mining'\n",
      "Entity 'http://edamontology.org/operation_0307' - Label 'Virtual PCR'\n",
      "Entity 'http://edamontology.org/operation_0308' - Label 'PCR primer design'\n",
      "Entity 'http://edamontology.org/operation_0309' - Label 'Microarray probe design'\n",
      "Entity 'http://edamontology.org/operation_0310' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/operation_0311' - Label 'Microarray data standardisation and normalisation'\n",
      "Entity 'http://edamontology.org/operation_0312' - Label 'Sequencing-based expression profile data processing'\n",
      "Entity 'http://edamontology.org/operation_0313' - Label 'Expression profile clustering'\n",
      "Entity 'http://edamontology.org/operation_0314' - Label 'Gene expression profiling'\n",
      "Entity 'http://edamontology.org/operation_0315' - Label 'Expression profile comparison'\n",
      "Entity 'http://edamontology.org/operation_0316' - Label 'Functional profiling'\n",
      "Entity 'http://edamontology.org/operation_0317' - Label 'EST and cDNA sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0318' - Label 'Structural genomics target selection'\n",
      "Entity 'http://edamontology.org/operation_0319' - Label 'Protein secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0320' - Label 'Protein structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0321' - Label 'Protein structure validation'\n",
      "Entity 'http://edamontology.org/operation_0322' - Label 'Molecular model refinement'\n",
      "Entity 'http://edamontology.org/operation_0323' - Label 'Phylogenetic inference'\n",
      "Entity 'http://edamontology.org/operation_0324' - Label 'Phylogenetic analysis'\n",
      "Entity 'http://edamontology.org/operation_0325' - Label 'Phylogenetic tree comparison'\n",
      "Entity 'http://edamontology.org/operation_0326' - Label 'Phylogenetic tree editing'\n",
      "Entity 'http://edamontology.org/operation_0327' - Label 'Phylogenetic footprinting'\n",
      "Entity 'http://edamontology.org/operation_0328' - Label 'Protein folding simulation'\n",
      "Entity 'http://edamontology.org/operation_0329' - Label 'Protein folding pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_0330' - Label 'Protein SNP mapping'\n",
      "Entity 'http://edamontology.org/operation_0331' - Label 'Variant effect prediction'\n",
      "Entity 'http://edamontology.org/operation_0332' - Label 'Immunogen design'\n",
      "Entity 'http://edamontology.org/operation_0333' - Label 'Zinc finger prediction'\n",
      "Entity 'http://edamontology.org/operation_0334' - Label 'Enzyme kinetics calculation'\n",
      "Entity 'http://edamontology.org/operation_0335' - Label 'Formatting'\n",
      "Entity 'http://edamontology.org/operation_0336' - Label 'Format validation'\n",
      "Entity 'http://edamontology.org/operation_0337' - Label 'Visualisation'\n",
      "Entity 'http://edamontology.org/operation_0338' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/operation_0339' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/operation_0340' - Label 'Protein secondary database search'\n",
      "Entity 'http://edamontology.org/operation_0341' - Label 'Motif database search'\n",
      "Entity 'http://edamontology.org/operation_0342' - Label 'Sequence profile database search'\n",
      "Entity 'http://edamontology.org/operation_0343' - Label 'Transmembrane protein database search'\n",
      "Entity 'http://edamontology.org/operation_0344' - Label 'Sequence retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0345' - Label 'Sequence retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0346' - Label 'Sequence similarity search'\n",
      "Entity 'http://edamontology.org/operation_0347' - Label 'Sequence database search (by motif or pattern)'\n",
      "Entity 'http://edamontology.org/operation_0348' - Label 'Sequence database search (by amino acid composition)'\n",
      "Entity 'http://edamontology.org/operation_0349' - Label 'Sequence database search (by property)'\n",
      "Entity 'http://edamontology.org/operation_0350' - Label 'Sequence database search (by sequence using word-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0351' - Label 'Sequence database search (by sequence using profile-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0352' - Label 'Sequence database search (by sequence using local alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0353' - Label 'Sequence database search (by sequence using global alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0354' - Label 'Sequence database search (by sequence for primer sequences)'\n",
      "Entity 'http://edamontology.org/operation_0355' - Label 'Sequence database search (by molecular weight)'\n",
      "Entity 'http://edamontology.org/operation_0356' - Label 'Sequence database search (by isoelectric point)'\n",
      "Entity 'http://edamontology.org/operation_0357' - Label 'Structure retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0358' - Label 'Structure retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0359' - Label 'Structure database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_0360' - Label 'Structural similarity search'\n",
      "Entity 'http://edamontology.org/operation_0361' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/operation_0362' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/operation_0363' - Label 'Reverse complement'\n",
      "Entity 'http://edamontology.org/operation_0364' - Label 'Random sequence generation'\n",
      "Entity 'http://edamontology.org/operation_0365' - Label 'Restriction digest'\n",
      "Entity 'http://edamontology.org/operation_0366' - Label 'Protein sequence cleavage'\n",
      "Entity 'http://edamontology.org/operation_0367' - Label 'Sequence mutation and randomisation'\n",
      "Entity 'http://edamontology.org/operation_0368' - Label 'Sequence masking'\n",
      "Entity 'http://edamontology.org/operation_0369' - Label 'Sequence cutting'\n",
      "Entity 'http://edamontology.org/operation_0370' - Label 'Restriction site creation'\n",
      "Entity 'http://edamontology.org/operation_0371' - Label 'DNA translation'\n",
      "Entity 'http://edamontology.org/operation_0372' - Label 'DNA transcription'\n",
      "Entity 'http://edamontology.org/operation_0377' - Label 'Sequence composition calculation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_0378' - Label 'Sequence composition calculation (protein)'\n",
      "Entity 'http://edamontology.org/operation_0379' - Label 'Repeat sequence detection'\n",
      "Entity 'http://edamontology.org/operation_0380' - Label 'Repeat sequence organisation analysis'\n",
      "Entity 'http://edamontology.org/operation_0383' - Label 'Protein hydropathy calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0384' - Label 'Accessible surface calculation'\n",
      "Entity 'http://edamontology.org/operation_0385' - Label 'Protein hydropathy cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0386' - Label 'Protein dipole moment calculation'\n",
      "Entity 'http://edamontology.org/operation_0387' - Label 'Molecular surface calculation'\n",
      "Entity 'http://edamontology.org/operation_0388' - Label 'Protein binding site prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0389' - Label 'Protein-nucleic acid interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_0390' - Label 'Protein peeling'\n",
      "Entity 'http://edamontology.org/operation_0391' - Label 'Protein distance matrix calculation'\n",
      "Entity 'http://edamontology.org/operation_0392' - Label 'Contact map calculation'\n",
      "Entity 'http://edamontology.org/operation_0393' - Label 'Residue cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0394' - Label 'Hydrogen bond calculation'\n",
      "Entity 'http://edamontology.org/operation_0395' - Label 'Residue non-canonical interaction detection'\n",
      "Entity 'http://edamontology.org/operation_0396' - Label 'Ramachandran plot calculation'\n",
      "Entity 'http://edamontology.org/operation_0397' - Label 'Ramachandran plot validation'\n",
      "Entity 'http://edamontology.org/operation_0398' - Label 'Protein molecular weight calculation'\n",
      "Entity 'http://edamontology.org/operation_0399' - Label 'Protein extinction coefficient calculation'\n",
      "Entity 'http://edamontology.org/operation_0400' - Label 'Protein pKa calculation'\n",
      "Entity 'http://edamontology.org/operation_0401' - Label 'Protein hydropathy calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0402' - Label 'Protein titration curve plotting'\n",
      "Entity 'http://edamontology.org/operation_0403' - Label 'Protein isoelectric point calculation'\n",
      "Entity 'http://edamontology.org/operation_0404' - Label 'Protein hydrogen exchange rate calculation'\n",
      "Entity 'http://edamontology.org/operation_0405' - Label 'Protein hydrophobic region calculation'\n",
      "Entity 'http://edamontology.org/operation_0406' - Label 'Protein aliphatic index calculation'\n",
      "Entity 'http://edamontology.org/operation_0407' - Label 'Protein hydrophobic moment plotting'\n",
      "Entity 'http://edamontology.org/operation_0408' - Label 'Protein globularity prediction'\n",
      "Entity 'http://edamontology.org/operation_0409' - Label 'Protein solubility prediction'\n",
      "Entity 'http://edamontology.org/operation_0410' - Label 'Protein crystallizability prediction'\n",
      "Entity 'http://edamontology.org/operation_0411' - Label 'Protein signal peptide detection (eukaryotes)'\n",
      "Entity 'http://edamontology.org/operation_0412' - Label 'Protein signal peptide detection (bacteria)'\n",
      "Entity 'http://edamontology.org/operation_0413' - Label 'MHC peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0414' - Label 'Protein feature prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0415' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/operation_0416' - Label 'Epitope mapping'\n",
      "Entity 'http://edamontology.org/operation_0417' - Label 'Post-translational modification site prediction'\n",
      "Entity 'http://edamontology.org/operation_0418' - Label 'Protein signal peptide detection'\n",
      "Entity 'http://edamontology.org/operation_0419' - Label 'Protein binding site prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0420' - Label 'Nucleic acids-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0421' - Label 'Protein folding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0422' - Label 'Protein cleavage site prediction'\n",
      "Entity 'http://edamontology.org/operation_0423' - Label 'Epitope mapping (MHC Class I)'\n",
      "Entity 'http://edamontology.org/operation_0424' - Label 'Epitope mapping (MHC Class II)'\n",
      "Entity 'http://edamontology.org/operation_0425' - Label 'Whole gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0426' - Label 'Gene component prediction'\n",
      "Entity 'http://edamontology.org/operation_0427' - Label 'Transposon prediction'\n",
      "Entity 'http://edamontology.org/operation_0428' - Label 'PolyA signal detection'\n",
      "Entity 'http://edamontology.org/operation_0429' - Label 'Quadruplex formation site detection'\n",
      "Entity 'http://edamontology.org/operation_0430' - Label 'CpG island and isochore detection'\n",
      "Entity 'http://edamontology.org/operation_0431' - Label 'Restriction site recognition'\n",
      "Entity 'http://edamontology.org/operation_0432' - Label 'Nucleosome position prediction'\n",
      "Entity 'http://edamontology.org/operation_0433' - Label 'Splice site prediction'\n",
      "Entity 'http://edamontology.org/operation_0434' - Label 'Integrated gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0435' - Label 'Operon prediction'\n",
      "Entity 'http://edamontology.org/operation_0436' - Label 'Coding region prediction'\n",
      "Entity 'http://edamontology.org/operation_0437' - Label 'SECIS element prediction'\n",
      "Entity 'http://edamontology.org/operation_0438' - Label 'Transcriptional regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0439' - Label 'Translation initiation site prediction'\n",
      "Entity 'http://edamontology.org/operation_0440' - Label 'Promoter prediction'\n",
      "Entity 'http://edamontology.org/operation_0441' - Label 'cis-regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0442' - Label 'Transcriptional regulatory element prediction (RNA-cis)'\n",
      "Entity 'http://edamontology.org/operation_0443' - Label 'trans-regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0444' - Label 'S/MAR prediction'\n",
      "Entity 'http://edamontology.org/operation_0445' - Label 'Transcription factor binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0446' - Label 'Exonic splicing enhancer prediction'\n",
      "Entity 'http://edamontology.org/operation_0447' - Label 'Sequence alignment validation'\n",
      "Entity 'http://edamontology.org/operation_0448' - Label 'Sequence alignment analysis (conservation)'\n",
      "Entity 'http://edamontology.org/operation_0449' - Label 'Sequence alignment analysis (site correlation)'\n",
      "Entity 'http://edamontology.org/operation_0450' - Label 'Chimera detection'\n",
      "Entity 'http://edamontology.org/operation_0451' - Label 'Recombination detection'\n",
      "Entity 'http://edamontology.org/operation_0452' - Label 'Indel detection'\n",
      "Entity 'http://edamontology.org/operation_0453' - Label 'Nucleosome formation potential prediction'\n",
      "Entity 'http://edamontology.org/operation_0455' - Label 'Nucleic acid thermodynamic property calculation'\n",
      "Entity 'http://edamontology.org/operation_0456' - Label 'Nucleic acid melting profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0457' - Label 'Nucleic acid stitch profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0458' - Label 'Nucleic acid melting curve plotting'\n",
      "Entity 'http://edamontology.org/operation_0459' - Label 'Nucleic acid probability profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0460' - Label 'Nucleic acid temperature profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0461' - Label 'Nucleic acid curvature calculation'\n",
      "Entity 'http://edamontology.org/operation_0463' - Label 'miRNA target prediction'\n",
      "Entity 'http://edamontology.org/operation_0464' - Label 'tRNA gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0465' - Label 'siRNA binding specificity prediction'\n",
      "Entity 'http://edamontology.org/operation_0467' - Label 'Protein secondary structure prediction (integrated)'\n",
      "Entity 'http://edamontology.org/operation_0468' - Label 'Protein secondary structure prediction (helices)'\n",
      "Entity 'http://edamontology.org/operation_0469' - Label 'Protein secondary structure prediction (turns)'\n",
      "Entity 'http://edamontology.org/operation_0470' - Label 'Protein secondary structure prediction (coils)'\n",
      "Entity 'http://edamontology.org/operation_0471' - Label 'Disulfide bond prediction'\n",
      "Entity 'http://edamontology.org/operation_0472' - Label 'GPCR prediction'\n",
      "Entity 'http://edamontology.org/operation_0473' - Label 'GPCR analysis'\n",
      "Entity 'http://edamontology.org/operation_0474' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0475' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0476' - Label 'Ab initio structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0477' - Label 'Protein modelling'\n",
      "Entity 'http://edamontology.org/operation_0478' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/operation_0479' - Label 'Backbone modelling'\n",
      "Entity 'http://edamontology.org/operation_0480' - Label 'Side chain modelling'\n",
      "Entity 'http://edamontology.org/operation_0481' - Label 'Loop modelling'\n",
      "Entity 'http://edamontology.org/operation_0482' - Label 'Protein-ligand docking'\n",
      "Entity 'http://edamontology.org/operation_0483' - Label 'RNA inverse folding'\n",
      "Entity 'http://edamontology.org/operation_0484' - Label 'SNP detection'\n",
      "Entity 'http://edamontology.org/operation_0485' - Label 'Radiation Hybrid Mapping'\n",
      "Entity 'http://edamontology.org/operation_0486' - Label 'Functional mapping'\n",
      "Entity 'http://edamontology.org/operation_0487' - Label 'Haplotype mapping'\n",
      "Entity 'http://edamontology.org/operation_0488' - Label 'Linkage disequilibrium calculation'\n",
      "Entity 'http://edamontology.org/operation_0489' - Label 'Genetic code prediction'\n",
      "Entity 'http://edamontology.org/operation_0490' - Label 'Dot plot plotting'\n",
      "Entity 'http://edamontology.org/operation_0491' - Label 'Pairwise sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0492' - Label 'Multiple sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0493' - Label 'Pairwise sequence alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0494' - Label 'Pairwise sequence alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0495' - Label 'Local alignment'\n",
      "Entity 'http://edamontology.org/operation_0496' - Label 'Global alignment'\n",
      "Entity 'http://edamontology.org/operation_0497' - Label 'Constrained sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0498' - Label 'Consensus-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0499' - Label 'Tree-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0500' - Label 'Secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0501' - Label 'Protein secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0502' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0503' - Label 'Pairwise structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0504' - Label 'Multiple structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0505' - Label 'Structure alignment (protein)'\n",
      "Entity 'http://edamontology.org/operation_0506' - Label 'Structure alignment (RNA)'\n",
      "Entity 'http://edamontology.org/operation_0507' - Label 'Pairwise structure alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0508' - Label 'Pairwise structure alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0509' - Label 'Local structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0510' - Label 'Global structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0511' - Label 'Profile-profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0512' - Label 'Sequence alignment generation (multiple profile)'\n",
      "Entity 'http://edamontology.org/operation_0513' - Label '3D profile-to-3D profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0514' - Label 'Structural profile alignment generation (multiple)'\n",
      "Entity 'http://edamontology.org/operation_0515' - Label 'Data retrieval (tool metadata)'\n",
      "Entity 'http://edamontology.org/operation_0516' - Label 'Data retrieval (database metadata)'\n",
      "Entity 'http://edamontology.org/operation_0517' - Label 'PCR primer design (for large scale sequencing)'\n",
      "Entity 'http://edamontology.org/operation_0518' - Label 'PCR primer design (for genotyping polymorphisms)'\n",
      "Entity 'http://edamontology.org/operation_0519' - Label 'PCR primer design (for gene transcription profiling)'\n",
      "Entity 'http://edamontology.org/operation_0520' - Label 'PCR primer design (for conserved primers)'\n",
      "Entity 'http://edamontology.org/operation_0521' - Label 'PCR primer design (based on gene structure)'\n",
      "Entity 'http://edamontology.org/operation_0522' - Label 'PCR primer design (for methylation PCRs)'\n",
      "Entity 'http://edamontology.org/operation_0523' - Label 'Mapping assembly'\n",
      "Entity 'http://edamontology.org/operation_0524' - Label 'De-novo assembly'\n",
      "Entity 'http://edamontology.org/operation_0525' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_0526' - Label 'EST assembly'\n",
      "Entity 'http://edamontology.org/operation_0527' - Label 'Sequence tag mapping'\n",
      "Entity 'http://edamontology.org/operation_0528' - Label 'SAGE data processing'\n",
      "Entity 'http://edamontology.org/operation_0529' - Label 'MPSS data processing'\n",
      "Entity 'http://edamontology.org/operation_0530' - Label 'SBS data processing'\n",
      "Entity 'http://edamontology.org/operation_0531' - Label 'Heat map generation'\n",
      "Entity 'http://edamontology.org/operation_0532' - Label 'Gene expression profile analysis'\n",
      "Entity 'http://edamontology.org/operation_0533' - Label 'Expression profile pathway mapping'\n",
      "Entity 'http://edamontology.org/operation_0534' - Label 'Protein secondary structure assignment (from coordinate data)'\n",
      "Entity 'http://edamontology.org/operation_0535' - Label 'Protein secondary structure assignment (from CD data)'\n",
      "Entity 'http://edamontology.org/operation_0536' - Label 'Protein structure assignment (from X-ray crystallographic data)'\n",
      "Entity 'http://edamontology.org/operation_0537' - Label 'Protein structure assignment (from NMR data)'\n",
      "Entity 'http://edamontology.org/operation_0538' - Label 'Phylogenetic inference (data centric)'\n",
      "Entity 'http://edamontology.org/operation_0539' - Label 'Phylogenetic inference (method centric)'\n",
      "Entity 'http://edamontology.org/operation_0540' - Label 'Phylogenetic inference (from molecular sequences)'\n",
      "Entity 'http://edamontology.org/operation_0541' - Label 'Phylogenetic inference (from continuous quantitative characters)'\n",
      "Entity 'http://edamontology.org/operation_0542' - Label 'Phylogenetic inference (from gene frequencies)'\n",
      "Entity 'http://edamontology.org/operation_0543' - Label 'Phylogenetic inference (from polymorphism data)'\n",
      "Entity 'http://edamontology.org/operation_0544' - Label 'Species tree construction'\n",
      "Entity 'http://edamontology.org/operation_0545' - Label 'Phylogenetic inference (parsimony methods)'\n",
      "Entity 'http://edamontology.org/operation_0546' - Label 'Phylogenetic inference (minimum distance methods)'\n",
      "Entity 'http://edamontology.org/operation_0547' - Label 'Phylogenetic inference (maximum likelihood and Bayesian methods)'\n",
      "Entity 'http://edamontology.org/operation_0548' - Label 'Phylogenetic inference (quartet methods)'\n",
      "Entity 'http://edamontology.org/operation_0549' - Label 'Phylogenetic inference (AI methods)'\n",
      "Entity 'http://edamontology.org/operation_0550' - Label 'DNA substitution modelling'\n",
      "Entity 'http://edamontology.org/operation_0551' - Label 'Phylogenetic tree topology analysis'\n",
      "Entity 'http://edamontology.org/operation_0552' - Label 'Phylogenetic tree bootstrapping'\n",
      "Entity 'http://edamontology.org/operation_0553' - Label 'Gene tree construction'\n",
      "Entity 'http://edamontology.org/operation_0554' - Label 'Allele frequency distribution analysis'\n",
      "Entity 'http://edamontology.org/operation_0555' - Label 'Consensus tree construction'\n",
      "Entity 'http://edamontology.org/operation_0556' - Label 'Phylogenetic sub/super tree construction'\n",
      "Entity 'http://edamontology.org/operation_0557' - Label 'Phylogenetic tree distances calculation'\n",
      "Entity 'http://edamontology.org/operation_0558' - Label 'Phylogenetic tree annotation'\n",
      "Entity 'http://edamontology.org/operation_0559' - Label 'Immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0560' - Label 'DNA vaccine design'\n",
      "Entity 'http://edamontology.org/operation_0561' - Label 'Sequence formatting'\n",
      "Entity 'http://edamontology.org/operation_0562' - Label 'Sequence alignment formatting'\n",
      "Entity 'http://edamontology.org/operation_0563' - Label 'Codon usage table formatting'\n",
      "Entity 'http://edamontology.org/operation_0564' - Label 'Sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_0565' - Label 'Sequence alignment visualisation'\n",
      "Entity 'http://edamontology.org/operation_0566' - Label 'Sequence cluster visualisation'\n",
      "Entity 'http://edamontology.org/operation_0567' - Label 'Phylogenetic tree visualisation'\n",
      "Entity 'http://edamontology.org/operation_0568' - Label 'RNA secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0569' - Label 'Protein secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0570' - Label 'Structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0571' - Label 'Expression data visualisation'\n",
      "Entity 'http://edamontology.org/operation_0572' - Label 'Protein interaction network visualisation'\n",
      "Entity 'http://edamontology.org/operation_0573' - Label 'Map drawing'\n",
      "Entity 'http://edamontology.org/operation_0574' - Label 'Sequence motif rendering'\n",
      "Entity 'http://edamontology.org/operation_0575' - Label 'Restriction map drawing'\n",
      "Entity 'http://edamontology.org/operation_0577' - Label 'DNA linear map rendering'\n",
      "Entity 'http://edamontology.org/operation_0578' - Label 'Plasmid map drawing'\n",
      "Entity 'http://edamontology.org/operation_0579' - Label 'Operon drawing'\n",
      "Entity 'http://edamontology.org/operation_1768' - Label 'Nucleic acid folding family identification'\n",
      "Entity 'http://edamontology.org/operation_1769' - Label 'Nucleic acid folding energy calculation'\n",
      "Entity 'http://edamontology.org/operation_1774' - Label 'Annotation retrieval'\n",
      "Entity 'http://edamontology.org/operation_1777' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/operation_1778' - Label 'Protein function comparison'\n",
      "Entity 'http://edamontology.org/operation_1780' - Label 'Sequence submission'\n",
      "Entity 'http://edamontology.org/operation_1781' - Label 'Gene regulatory network analysis'\n",
      "Entity 'http://edamontology.org/operation_1812' - Label 'Parsing'\n",
      "Entity 'http://edamontology.org/operation_1813' - Label 'Sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1814' - Label 'Structure retrieval'\n",
      "Entity 'http://edamontology.org/operation_1816' - Label 'Surface rendering'\n",
      "Entity 'http://edamontology.org/operation_1817' - Label 'Protein atom surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1818' - Label 'Protein atom surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1819' - Label 'Protein residue surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1820' - Label 'Protein residue surface calculation (vacuum accessible)'\n",
      "Entity 'http://edamontology.org/operation_1821' - Label 'Protein residue surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1822' - Label 'Protein residue surface calculation (vacuum molecular)'\n",
      "Entity 'http://edamontology.org/operation_1823' - Label 'Protein surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1824' - Label 'Protein surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1825' - Label 'Backbone torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1826' - Label 'Full torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1827' - Label 'Cysteine torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1828' - Label 'Tau angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1829' - Label 'Cysteine bridge detection'\n",
      "Entity 'http://edamontology.org/operation_1830' - Label 'Free cysteine detection'\n",
      "Entity 'http://edamontology.org/operation_1831' - Label 'Metal-bound cysteine detection'\n",
      "Entity 'http://edamontology.org/operation_1832' - Label 'Residue contact calculation (residue-nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_1834' - Label 'Protein-metal contact calculation'\n",
      "Entity 'http://edamontology.org/operation_1835' - Label 'Residue contact calculation (residue-negative ion)'\n",
      "Entity 'http://edamontology.org/operation_1836' - Label 'Residue bump detection'\n",
      "Entity 'http://edamontology.org/operation_1837' - Label 'Residue symmetry contact calculation'\n",
      "Entity 'http://edamontology.org/operation_1838' - Label 'Residue contact calculation (residue-ligand)'\n",
      "Entity 'http://edamontology.org/operation_1839' - Label 'Salt bridge calculation'\n",
      "Entity 'http://edamontology.org/operation_1841' - Label 'Rotamer likelihood prediction'\n",
      "Entity 'http://edamontology.org/operation_1842' - Label 'Proline mutation value calculation'\n",
      "Entity 'http://edamontology.org/operation_1843' - Label 'Residue packing validation'\n",
      "Entity 'http://edamontology.org/operation_1844' - Label 'Protein geometry validation'\n",
      "Entity 'http://edamontology.org/operation_1845' - Label 'PDB file sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1846' - Label 'HET group detection'\n",
      "Entity 'http://edamontology.org/operation_1847' - Label 'DSSP secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_1848' - Label 'Structure formatting'\n",
      "Entity 'http://edamontology.org/operation_1850' - Label 'Protein cysteine and disulfide bond assignment'\n",
      "Entity 'http://edamontology.org/operation_1913' - Label 'Residue validation'\n",
      "Entity 'http://edamontology.org/operation_1914' - Label 'Structure retrieval (water)'\n",
      "Entity 'http://edamontology.org/operation_2008' - Label 'siRNA duplex prediction'\n",
      "Entity 'http://edamontology.org/operation_2089' - Label 'Sequence alignment refinement'\n",
      "Entity 'http://edamontology.org/operation_2120' - Label 'Listfile processing'\n",
      "Entity 'http://edamontology.org/operation_2121' - Label 'Sequence file editing'\n",
      "Entity 'http://edamontology.org/operation_2122' - Label 'Sequence alignment file processing'\n",
      "Entity 'http://edamontology.org/operation_2123' - Label 'Small molecule data processing'\n",
      "Entity 'http://edamontology.org/operation_2222' - Label 'Data retrieval (ontology annotation)'\n",
      "Entity 'http://edamontology.org/operation_2224' - Label 'Data retrieval (ontology concept)'\n",
      "Entity 'http://edamontology.org/operation_2233' - Label 'Representative sequence identification'\n",
      "Entity 'http://edamontology.org/operation_2234' - Label 'Structure file processing'\n",
      "Entity 'http://edamontology.org/operation_2237' - Label 'Data retrieval (sequence profile)'\n",
      "Entity 'http://edamontology.org/operation_2238' - Label 'Statistical calculation'\n",
      "Entity 'http://edamontology.org/operation_2239' - Label '3D-1D scoring matrix generation'\n",
      "Entity 'http://edamontology.org/operation_2241' - Label 'Transmembrane protein visualisation'\n",
      "Entity 'http://edamontology.org/operation_2246' - Label 'Demonstration'\n",
      "Entity 'http://edamontology.org/operation_2264' - Label 'Data retrieval (pathway or network)'\n",
      "Entity 'http://edamontology.org/operation_2265' - Label 'Data retrieval (identifier)'\n",
      "Entity 'http://edamontology.org/operation_2284' - Label 'Nucleic acid density plotting'\n",
      "Entity 'http://edamontology.org/operation_2403' - Label 'Sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2404' - Label 'Sequence motif analysis'\n",
      "Entity 'http://edamontology.org/operation_2405' - Label 'Protein interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2406' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2407' - Label 'Annotation processing'\n",
      "Entity 'http://edamontology.org/operation_2408' - Label 'Sequence feature analysis'\n",
      "Entity 'http://edamontology.org/operation_2409' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2410' - Label 'Gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2411' - Label 'Structural profile processing'\n",
      "Entity 'http://edamontology.org/operation_2412' - Label 'Data index processing'\n",
      "Entity 'http://edamontology.org/operation_2413' - Label 'Sequence profile processing'\n",
      "Entity 'http://edamontology.org/operation_2414' - Label 'Protein function analysis'\n",
      "Entity 'http://edamontology.org/operation_2415' - Label 'Protein folding analysis'\n",
      "Entity 'http://edamontology.org/operation_2416' - Label 'Protein secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2417' - Label 'Physicochemical property data processing'\n",
      "Entity 'http://edamontology.org/operation_2419' - Label 'Primer and probe design'\n",
      "Entity 'http://edamontology.org/operation_2420' - Label 'Operation (typed)'\n",
      "Entity 'http://edamontology.org/operation_2421' - Label 'Database search'\n",
      "Entity 'http://edamontology.org/operation_2422' - Label 'Data retrieval'\n",
      "Entity 'http://edamontology.org/operation_2423' - Label 'Prediction and recognition'\n",
      "Entity 'http://edamontology.org/operation_2424' - Label 'Comparison'\n",
      "Entity 'http://edamontology.org/operation_2425' - Label 'Optimisation and refinement'\n",
      "Entity 'http://edamontology.org/operation_2426' - Label 'Modelling and simulation'\n",
      "Entity 'http://edamontology.org/operation_2427' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2428' - Label 'Validation'\n",
      "Entity 'http://edamontology.org/operation_2429' - Label 'Mapping'\n",
      "Entity 'http://edamontology.org/operation_2430' - Label 'Design'\n",
      "Entity 'http://edamontology.org/operation_2432' - Label 'Microarray data processing'\n",
      "Entity 'http://edamontology.org/operation_2433' - Label 'Codon usage table processing'\n",
      "Entity 'http://edamontology.org/operation_2434' - Label 'Data retrieval (codon usage table)'\n",
      "Entity 'http://edamontology.org/operation_2435' - Label 'Gene expression profile processing'\n",
      "Entity 'http://edamontology.org/operation_2436' - Label 'Gene-set enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_2437' - Label 'Gene regulatory network prediction'\n",
      "Entity 'http://edamontology.org/operation_2438' - Label 'Pathway or network processing'\n",
      "Entity 'http://edamontology.org/operation_2439' - Label 'RNA secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2440' - Label 'Structure processing (RNA)'\n",
      "Entity 'http://edamontology.org/operation_2441' - Label 'RNA structure prediction'\n",
      "Entity 'http://edamontology.org/operation_2442' - Label 'DNA structure prediction'\n",
      "Entity 'http://edamontology.org/operation_2443' - Label 'Phylogenetic tree processing'\n",
      "Entity 'http://edamontology.org/operation_2444' - Label 'Protein secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2445' - Label 'Protein interaction network processing'\n",
      "Entity 'http://edamontology.org/operation_2446' - Label 'Sequence processing'\n",
      "Entity 'http://edamontology.org/operation_2447' - Label 'Sequence processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2448' - Label 'Sequence processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2451' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2452' - Label 'Sequence cluster processing'\n",
      "Entity 'http://edamontology.org/operation_2453' - Label 'Feature table processing'\n",
      "Entity 'http://edamontology.org/operation_2454' - Label 'Gene prediction'\n",
      "Entity 'http://edamontology.org/operation_2456' - Label 'GPCR classification'\n",
      "Entity 'http://edamontology.org/operation_2457' - Label 'GPCR coupling selectivity prediction'\n",
      "Entity 'http://edamontology.org/operation_2459' - Label 'Structure processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2460' - Label 'Protein atom surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2461' - Label 'Protein residue surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2462' - Label 'Protein surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2463' - Label 'Sequence alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2464' - Label 'Protein-protein binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2465' - Label 'Structure processing'\n",
      "Entity 'http://edamontology.org/operation_2466' - Label 'Map annotation'\n",
      "Entity 'http://edamontology.org/operation_2467' - Label 'Data retrieval (protein annotation)'\n",
      "Entity 'http://edamontology.org/operation_2468' - Label 'Data retrieval (phylogenetic tree)'\n",
      "Entity 'http://edamontology.org/operation_2469' - Label 'Data retrieval (protein interaction annotation)'\n",
      "Entity 'http://edamontology.org/operation_2470' - Label 'Data retrieval (protein family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2471' - Label 'Data retrieval (RNA family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2472' - Label 'Data retrieval (gene annotation)'\n",
      "Entity 'http://edamontology.org/operation_2473' - Label 'Data retrieval (genotype and phenotype annotation)'\n",
      "Entity 'http://edamontology.org/operation_2474' - Label 'Protein architecture comparison'\n",
      "Entity 'http://edamontology.org/operation_2475' - Label 'Protein architecture recognition'\n",
      "Entity 'http://edamontology.org/operation_2476' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/operation_2478' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2479' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2480' - Label 'Structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2481' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2482' - Label 'Secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2483' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2485' - Label 'Helical wheel drawing'\n",
      "Entity 'http://edamontology.org/operation_2486' - Label 'Topology diagram drawing'\n",
      "Entity 'http://edamontology.org/operation_2487' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2488' - Label 'Protein secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2489' - Label 'Subcellular localisation prediction'\n",
      "Entity 'http://edamontology.org/operation_2490' - Label 'Residue contact calculation (residue-residue)'\n",
      "Entity 'http://edamontology.org/operation_2491' - Label 'Hydrogen bond calculation (inter-residue)'\n",
      "Entity 'http://edamontology.org/operation_2492' - Label 'Protein interaction prediction'\n",
      "Entity 'http://edamontology.org/operation_2493' - Label 'Codon usage data processing'\n",
      "Entity 'http://edamontology.org/operation_2495' - Label 'Expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2496' - Label 'Gene regulatory network processing'\n",
      "Entity 'http://edamontology.org/operation_2497' - Label 'Pathway or network analysis'\n",
      "Entity 'http://edamontology.org/operation_2498' - Label 'Sequencing-based expression profile data analysis'\n",
      "Entity 'http://edamontology.org/operation_2499' - Label 'Splicing analysis'\n",
      "Entity 'http://edamontology.org/operation_2500' - Label 'Microarray raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_2501' - Label 'Nucleic acid analysis'\n",
      "Entity 'http://edamontology.org/operation_2502' - Label 'Protein analysis'\n",
      "Entity 'http://edamontology.org/operation_2503' - Label 'Sequence data processing'\n",
      "Entity 'http://edamontology.org/operation_2504' - Label 'Structural data processing'\n",
      "Entity 'http://edamontology.org/operation_2505' - Label 'Text processing'\n",
      "Entity 'http://edamontology.org/operation_2506' - Label 'Protein sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2507' - Label 'Nucleic acid sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2508' - Label 'Nucleic acid sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2509' - Label 'Protein sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2510' - Label 'DNA back-translation'\n",
      "Entity 'http://edamontology.org/operation_2511' - Label 'Sequence editing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2512' - Label 'Sequence editing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2513' - Label 'Sequence generation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2514' - Label 'Sequence generation (protein)'\n",
      "Entity 'http://edamontology.org/operation_2515' - Label 'Nucleic acid sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2516' - Label 'Protein sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2518' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2519' - Label 'Structure processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2520' - Label 'DNA mapping'\n",
      "Entity 'http://edamontology.org/operation_2521' - Label 'Map data processing'\n",
      "Entity 'http://edamontology.org/operation_2574' - Label 'Protein hydropathy calculation'\n",
      "Entity 'http://edamontology.org/operation_2575' - Label 'Binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2844' - Label 'Structure clustering'\n",
      "Entity 'http://edamontology.org/operation_2871' - Label 'Sequence tagged site (STS) mapping'\n",
      "Entity 'http://edamontology.org/operation_2928' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/operation_2929' - Label 'Protein fragment weight comparison'\n",
      "Entity 'http://edamontology.org/operation_2930' - Label 'Protein property comparison'\n",
      "Entity 'http://edamontology.org/operation_2931' - Label 'Secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2932' - Label 'Hopp and Woods plotting'\n",
      "Entity 'http://edamontology.org/operation_2934' - Label 'Cluster textual view generation'\n",
      "Entity 'http://edamontology.org/operation_2935' - Label 'Clustering profile plotting'\n",
      "Entity 'http://edamontology.org/operation_2936' - Label 'Dendrograph plotting'\n",
      "Entity 'http://edamontology.org/operation_2937' - Label 'Proximity map plotting'\n",
      "Entity 'http://edamontology.org/operation_2938' - Label 'Dendrogram visualisation'\n",
      "Entity 'http://edamontology.org/operation_2939' - Label 'Principal component visualisation'\n",
      "Entity 'http://edamontology.org/operation_2940' - Label 'Scatter plot plotting'\n",
      "Entity 'http://edamontology.org/operation_2941' - Label 'Whole microarray graph plotting'\n",
      "Entity 'http://edamontology.org/operation_2942' - Label 'Treemap visualisation'\n",
      "Entity 'http://edamontology.org/operation_2943' - Label 'Box-Whisker plot plotting'\n",
      "Entity 'http://edamontology.org/operation_2944' - Label 'Physical mapping'\n",
      "Entity 'http://edamontology.org/operation_2945' - Label 'Analysis'\n",
      "Entity 'http://edamontology.org/operation_2946' - Label 'Alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2947' - Label 'Article analysis'\n",
      "Entity 'http://edamontology.org/operation_2948' - Label 'Molecular interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2949' - Label 'Protein-protein interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2950' - Label 'Residue distance calculation'\n",
      "Entity 'http://edamontology.org/operation_2951' - Label 'Alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2952' - Label 'Structure alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2962' - Label 'Codon usage bias calculation'\n",
      "Entity 'http://edamontology.org/operation_2963' - Label 'Codon usage bias plotting'\n",
      "Entity 'http://edamontology.org/operation_2964' - Label 'Codon usage fraction calculation'\n",
      "Entity 'http://edamontology.org/operation_2990' - Label 'Classification'\n",
      "Entity 'http://edamontology.org/operation_2993' - Label 'Molecular interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2995' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/operation_2996' - Label 'Structure classification'\n",
      "Entity 'http://edamontology.org/operation_2997' - Label 'Protein comparison'\n",
      "Entity 'http://edamontology.org/operation_2998' - Label 'Nucleic acid comparison'\n",
      "Entity 'http://edamontology.org/operation_3023' - Label 'Prediction and recognition (protein)'\n",
      "Entity 'http://edamontology.org/operation_3024' - Label 'Prediction and recognition (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_3080' - Label 'Structure editing'\n",
      "Entity 'http://edamontology.org/operation_3081' - Label 'Sequence alignment editing'\n",
      "Entity 'http://edamontology.org/operation_3083' - Label 'Pathway or network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3084' - Label 'Protein function prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3087' - Label 'Protein sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3088' - Label 'Protein property calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3090' - Label 'Protein feature prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_3092' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/operation_3093' - Label 'Database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_3094' - Label 'Protein interaction network prediction'\n",
      "Entity 'http://edamontology.org/operation_3095' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/operation_3096' - Label 'Editing'\n",
      "Entity 'http://edamontology.org/operation_3180' - Label 'Sequence assembly validation'\n",
      "Entity 'http://edamontology.org/operation_3182' - Label 'Genome alignment'\n",
      "Entity 'http://edamontology.org/operation_3183' - Label 'Localised reassembly'\n",
      "Entity 'http://edamontology.org/operation_3184' - Label 'Sequence assembly visualisation'\n",
      "Entity 'http://edamontology.org/operation_3185' - Label 'Base-calling'\n",
      "Entity 'http://edamontology.org/operation_3186' - Label 'Bisulfite mapping'\n",
      "Entity 'http://edamontology.org/operation_3187' - Label 'Sequence contamination filtering'\n",
      "Entity 'http://edamontology.org/operation_3189' - Label 'Trim ends'\n",
      "Entity 'http://edamontology.org/operation_3190' - Label 'Trim vector'\n",
      "Entity 'http://edamontology.org/operation_3191' - Label 'Trim to reference'\n",
      "Entity 'http://edamontology.org/operation_3192' - Label 'Sequence trimming'\n",
      "Entity 'http://edamontology.org/operation_3194' - Label 'Genome feature comparison'\n",
      "Entity 'http://edamontology.org/operation_3195' - Label 'Sequencing error detection'\n",
      "Entity 'http://edamontology.org/operation_3196' - Label 'Genotyping'\n",
      "Entity 'http://edamontology.org/operation_3197' - Label 'Genetic variation analysis'\n",
      "Entity 'http://edamontology.org/operation_3198' - Label 'Read mapping'\n",
      "Entity 'http://edamontology.org/operation_3199' - Label 'Split read mapping'\n",
      "Entity 'http://edamontology.org/operation_3200' - Label 'DNA barcoding'\n",
      "Entity 'http://edamontology.org/operation_3201' - Label 'SNP calling'\n",
      "Entity 'http://edamontology.org/operation_3202' - Label 'Polymorphism detection'\n",
      "Entity 'http://edamontology.org/operation_3203' - Label 'Chromatogram visualisation'\n",
      "Entity 'http://edamontology.org/operation_3204' - Label 'Methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3205' - Label 'Methylation calling'\n",
      "Entity 'http://edamontology.org/operation_3206' - Label 'Whole genome methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3207' - Label 'Gene methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3208' - Label 'Genome visualisation'\n",
      "Entity 'http://edamontology.org/operation_3209' - Label 'Genome comparison'\n",
      "Entity 'http://edamontology.org/operation_3211' - Label 'Genome indexing'\n",
      "Entity 'http://edamontology.org/operation_3212' - Label 'Genome indexing (Burrows-Wheeler)'\n",
      "Entity 'http://edamontology.org/operation_3213' - Label 'Genome indexing (suffix arrays)'\n",
      "Entity 'http://edamontology.org/operation_3214' - Label 'Spectral analysis'\n",
      "Entity 'http://edamontology.org/operation_3215' - Label 'Peak detection'\n",
      "Entity 'http://edamontology.org/operation_3216' - Label 'Scaffolding'\n",
      "Entity 'http://edamontology.org/operation_3217' - Label 'Scaffold gap completion'\n",
      "Entity 'http://edamontology.org/operation_3218' - Label 'Sequencing quality control'\n",
      "Entity 'http://edamontology.org/operation_3219' - Label 'Read pre-processing'\n",
      "Entity 'http://edamontology.org/operation_3221' - Label 'Species frequency estimation'\n",
      "Entity 'http://edamontology.org/operation_3222' - Label 'Peak calling'\n",
      "Entity 'http://edamontology.org/operation_3223' - Label 'Differential gene expression profiling'\n",
      "Entity 'http://edamontology.org/operation_3224' - Label 'Gene set testing'\n",
      "Entity 'http://edamontology.org/operation_3225' - Label 'Variant classification'\n",
      "Entity 'http://edamontology.org/operation_3226' - Label 'Variant prioritisation'\n",
      "Entity 'http://edamontology.org/operation_3227' - Label 'Variant calling'\n",
      "Entity 'http://edamontology.org/operation_3228' - Label 'Structural variation detection'\n",
      "Entity 'http://edamontology.org/operation_3229' - Label 'Exome assembly'\n",
      "Entity 'http://edamontology.org/operation_3230' - Label 'Read depth analysis'\n",
      "Entity 'http://edamontology.org/operation_3232' - Label 'Gene expression QTL analysis'\n",
      "Entity 'http://edamontology.org/operation_3233' - Label 'Copy number estimation'\n",
      "Entity 'http://edamontology.org/operation_3237' - Label 'Primer removal'\n",
      "Entity 'http://edamontology.org/operation_3258' - Label 'Transcriptome assembly'\n",
      "Entity 'http://edamontology.org/operation_3259' - Label 'Transcriptome assembly (de novo)'\n",
      "Entity 'http://edamontology.org/operation_3260' - Label 'Transcriptome assembly (mapping)'\n",
      "Entity 'http://edamontology.org/operation_3267' - Label 'Sequence coordinate conversion'\n",
      "Entity 'http://edamontology.org/operation_3278' - Label 'Document similarity calculation'\n",
      "Entity 'http://edamontology.org/operation_3279' - Label 'Document clustering'\n",
      "Entity 'http://edamontology.org/operation_3280' - Label 'Named-entity and concept recognition'\n",
      "Entity 'http://edamontology.org/operation_3282' - Label 'ID mapping'\n",
      "Entity 'http://edamontology.org/operation_3283' - Label 'Anonymisation'\n",
      "Entity 'http://edamontology.org/operation_3289' - Label 'ID retrieval'\n",
      "Entity 'http://edamontology.org/operation_3348' - Label 'Sequence checksum generation'\n",
      "Entity 'http://edamontology.org/operation_3349' - Label 'Bibliography generation'\n",
      "Entity 'http://edamontology.org/operation_3350' - Label 'Protein quaternary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_3351' - Label 'Molecular surface analysis'\n",
      "Entity 'http://edamontology.org/operation_3352' - Label 'Ontology comparison'\n",
      "Entity 'http://edamontology.org/operation_3353' - Label 'Ontology comparison'\n",
      "Entity 'http://edamontology.org/operation_3357' - Label 'Format detection'\n",
      "Entity 'http://edamontology.org/operation_3359' - Label 'Splitting'\n",
      "Entity 'http://edamontology.org/operation_3429' - Label 'Generation'\n",
      "Entity 'http://edamontology.org/operation_3430' - Label 'Nucleic acid sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3431' - Label 'Deposition'\n",
      "Entity 'http://edamontology.org/operation_3432' - Label 'Clustering'\n",
      "Entity 'http://edamontology.org/operation_3433' - Label 'Assembly'\n",
      "Entity 'http://edamontology.org/operation_3434' - Label 'Conversion'\n",
      "Entity 'http://edamontology.org/operation_3435' - Label 'Standardisation and normalisation'\n",
      "Entity 'http://edamontology.org/operation_3436' - Label 'Aggregation'\n",
      "Entity 'http://edamontology.org/operation_3437' - Label 'Article comparison'\n",
      "Entity 'http://edamontology.org/operation_3438' - Label 'Calculation'\n",
      "Entity 'http://edamontology.org/operation_3439' - Label 'Pathway or network prediction'\n",
      "Entity 'http://edamontology.org/operation_3440' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_3441' - Label 'Plotting'\n",
      "Entity 'http://edamontology.org/operation_3443' - Label 'Image analysis'\n",
      "Entity 'http://edamontology.org/operation_3445' - Label 'Diffraction data analysis'\n",
      "Entity 'http://edamontology.org/operation_3446' - Label 'Cell migration analysis'\n",
      "Entity 'http://edamontology.org/operation_3447' - Label 'Diffraction data reduction'\n",
      "Entity 'http://edamontology.org/operation_3450' - Label 'Neurite measurement'\n",
      "Entity 'http://edamontology.org/operation_3453' - Label 'Diffraction data integration'\n",
      "Entity 'http://edamontology.org/operation_3454' - Label 'Phasing'\n",
      "Entity 'http://edamontology.org/operation_3455' - Label 'Molecular replacement'\n",
      "Entity 'http://edamontology.org/operation_3456' - Label 'Rigid body refinement'\n",
      "Entity 'http://edamontology.org/operation_3457' - Label 'Single particle analysis'\n",
      "Entity 'http://edamontology.org/operation_3458' - Label 'Single particle alignment and classification'\n",
      "Entity 'http://edamontology.org/operation_3459' - Label 'Functional clustering'\n",
      "Entity 'http://edamontology.org/operation_3460' - Label 'Taxonomic classification'\n",
      "Entity 'http://edamontology.org/operation_3461' - Label 'Virulence prediction'\n",
      "Entity 'http://edamontology.org/operation_3463' - Label 'Expression correlation analysis'\n",
      "Entity 'http://edamontology.org/operation_3465' - Label 'Correlation'\n",
      "Entity 'http://edamontology.org/operation_3469' - Label 'RNA structure covariance model generation'\n",
      "Entity 'http://edamontology.org/operation_3470' - Label 'RNA secondary structure prediction (shape-based)'\n",
      "Entity 'http://edamontology.org/operation_3471' - Label 'Nucleic acid folding prediction (alignment-based)'\n",
      "Entity 'http://edamontology.org/operation_3472' - Label 'k-mer counting'\n",
      "Entity 'http://edamontology.org/operation_3478' - Label 'Phylogenetic reconstruction'\n",
      "Entity 'http://edamontology.org/operation_3480' - Label 'Probabilistic data generation'\n",
      "Entity 'http://edamontology.org/operation_3481' - Label 'Probabilistic sequence generation'\n",
      "Entity 'http://edamontology.org/operation_3482' - Label 'Antimicrobial resistance prediction'\n",
      "Entity 'http://edamontology.org/operation_3501' - Label 'Enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_3502' - Label 'Chemical similarity enrichment'\n",
      "Entity 'http://edamontology.org/operation_3503' - Label 'Incident curve plotting'\n",
      "Entity 'http://edamontology.org/operation_3504' - Label 'Variant pattern analysis'\n",
      "Entity 'http://edamontology.org/operation_3545' - Label 'Mathematical modelling'\n",
      "Entity 'http://edamontology.org/operation_3552' - Label 'Microscope image visualisation'\n",
      "Entity 'http://edamontology.org/operation_3553' - Label 'Image annotation'\n",
      "Entity 'http://edamontology.org/operation_3557' - Label 'Imputation'\n",
      "Entity 'http://edamontology.org/operation_3559' - Label 'Ontology visualisation'\n",
      "Entity 'http://edamontology.org/operation_3560' - Label 'Maximum occurrence analysis'\n",
      "Entity 'http://edamontology.org/operation_3561' - Label 'Database comparison'\n",
      "Entity 'http://edamontology.org/operation_3562' - Label 'Network simulation'\n",
      "Entity 'http://edamontology.org/operation_3563' - Label 'RNA-seq read count analysis'\n",
      "Entity 'http://edamontology.org/operation_3564' - Label 'Chemical redundancy removal'\n",
      "Entity 'http://edamontology.org/operation_3565' - Label 'RNA-seq time series data analysis'\n",
      "Entity 'http://edamontology.org/operation_3566' - Label 'Simulated gene expression data generation'\n",
      "Entity 'http://edamontology.org/operation_3625' - Label 'Relation extraction'\n",
      "Entity 'http://edamontology.org/operation_3627' - Label 'Mass spectra calibration'\n",
      "Entity 'http://edamontology.org/operation_3628' - Label 'Chromatographic alignment'\n",
      "Entity 'http://edamontology.org/operation_3629' - Label 'Deisotoping'\n",
      "Entity 'http://edamontology.org/operation_3630' - Label 'Protein quantification'\n",
      "Entity 'http://edamontology.org/operation_3631' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/operation_3632' - Label 'Isotopic distributions calculation'\n",
      "Entity 'http://edamontology.org/operation_3633' - Label 'Retention time prediction'\n",
      "Entity 'http://edamontology.org/operation_3634' - Label 'Label-free quantification'\n",
      "Entity 'http://edamontology.org/operation_3635' - Label 'Labeled quantification'\n",
      "Entity 'http://edamontology.org/operation_3636' - Label 'MRM/SRM'\n",
      "Entity 'http://edamontology.org/operation_3637' - Label 'Spectral counting'\n",
      "Entity 'http://edamontology.org/operation_3638' - Label 'SILAC'\n",
      "Entity 'http://edamontology.org/operation_3639' - Label 'iTRAQ'\n",
      "Entity 'http://edamontology.org/operation_3640' - Label '18O labeling'\n",
      "Entity 'http://edamontology.org/operation_3641' - Label 'TMT-tag'\n",
      "Entity 'http://edamontology.org/operation_3642' - Label 'Stable isotope dimethyl labelling'\n",
      "Entity 'http://edamontology.org/operation_3643' - Label 'Tag-based peptide identification'\n",
      "Entity 'http://edamontology.org/operation_3644' - Label 'de Novo sequencing'\n",
      "Entity 'http://edamontology.org/operation_3645' - Label 'PTM identification'\n",
      "Entity 'http://edamontology.org/operation_3646' - Label 'Peptide database search'\n",
      "Entity 'http://edamontology.org/operation_3647' - Label 'Blind peptide database search'\n",
      "Entity 'http://edamontology.org/operation_3648' - Label 'Validation of peptide-spectrum matches'\n",
      "Entity 'http://edamontology.org/operation_3649' - Label 'Target-Decoy'\n",
      "Entity 'http://edamontology.org/operation_3658' - Label 'Statistical inference'\n",
      "Entity 'http://edamontology.org/operation_3659' - Label 'Regression analysis'\n",
      "Entity 'http://edamontology.org/operation_3660' - Label 'Metabolic network modelling'\n",
      "Entity 'http://edamontology.org/operation_3661' - Label 'SNP annotation'\n",
      "Entity 'http://edamontology.org/operation_3662' - Label 'Ab-initio gene prediction'\n",
      "Entity 'http://edamontology.org/operation_3663' - Label 'Homology-based gene prediction'\n",
      "Entity 'http://edamontology.org/operation_3664' - Label 'Statistical modelling'\n",
      "Entity 'http://edamontology.org/operation_3666' - Label 'Molecular surface comparison'\n",
      "Entity 'http://edamontology.org/operation_3672' - Label 'Gene functional annotation'\n",
      "Entity 'http://edamontology.org/operation_3675' - Label 'Variant filtering'\n",
      "Entity 'http://edamontology.org/operation_3677' - Label 'Differential binding analysis'\n",
      "Entity 'http://edamontology.org/operation_3680' - Label 'RNA-Seq analysis'\n",
      "Entity 'http://edamontology.org/operation_3694' - Label 'Mass spectrum visualisation'\n",
      "Entity 'http://edamontology.org/operation_3695' - Label 'Filtering'\n",
      "Entity 'http://edamontology.org/operation_3703' - Label 'Reference identification'\n",
      "Entity 'http://edamontology.org/operation_3704' - Label 'Ion counting'\n",
      "Entity 'http://edamontology.org/operation_3705' - Label 'Isotope-coded protein label'\n",
      "Entity 'http://edamontology.org/operation_3715' - Label 'Metabolic labeling'\n",
      "Entity 'http://edamontology.org/operation_3730' - Label 'Cross-assembly'\n",
      "Entity 'http://edamontology.org/operation_3731' - Label 'Sample comparison'\n",
      "Entity 'http://edamontology.org/operation_3741' - Label 'Differential protein expression profiling'\n",
      "Entity 'http://edamontology.org/operation_3742' - Label 'Differential gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_3744' - Label 'Multiple sample visualisation'\n",
      "Entity 'http://edamontology.org/operation_3745' - Label 'Ancestral reconstruction'\n",
      "Entity 'http://edamontology.org/operation_3755' - Label 'PTM localisation'\n",
      "Entity 'http://edamontology.org/operation_3760' - Label 'Service management'\n",
      "Entity 'http://edamontology.org/operation_3761' - Label 'Service discovery'\n",
      "Entity 'http://edamontology.org/operation_3762' - Label 'Service composition'\n",
      "Entity 'http://edamontology.org/operation_3763' - Label 'Service invocation'\n",
      "Entity 'http://edamontology.org/operation_3766' - Label 'Weighted correlation network analysis'\n",
      "Entity 'http://edamontology.org/operation_3767' - Label 'Protein identification'\n",
      "Entity 'http://edamontology.org/operation_3778' - Label 'Text annotation'\n",
      "Entity 'http://edamontology.org/operation_3791' - Label 'Collapsing methods'\n",
      "Entity 'http://edamontology.org/operation_3792' - Label 'miRNA expression analysis'\n",
      "Entity 'http://edamontology.org/operation_3793' - Label 'Read summarisation'\n",
      "Entity 'http://edamontology.org/operation_3795' - Label 'In vitro selection'\n",
      "Entity 'http://edamontology.org/operation_3797' - Label 'Rarefaction'\n",
      "Entity 'http://edamontology.org/operation_3798' - Label 'Read binning'\n",
      "Entity 'http://edamontology.org/operation_3799' - Label 'Quantification'\n",
      "Entity 'http://edamontology.org/operation_3800' - Label 'RNA-Seq quantification'\n",
      "Entity 'http://edamontology.org/operation_3801' - Label 'Spectral library search'\n",
      "Entity 'http://edamontology.org/operation_3802' - Label 'Sorting'\n",
      "Entity 'http://edamontology.org/operation_3803' - Label 'Natural product identification'\n",
      "Entity 'http://edamontology.org/operation_3809' - Label 'DMR identification'\n",
      "Entity 'http://edamontology.org/operation_3840' - Label 'Multilocus sequence typing'\n",
      "Entity 'http://edamontology.org/operation_3860' - Label 'Spectrum calculation'\n",
      "Entity 'http://edamontology.org/operation_3890' - Label 'Trajectory visualization'\n",
      "Entity 'http://edamontology.org/operation_3891' - Label 'Essential dynamics'\n",
      "Entity 'http://edamontology.org/operation_3893' - Label 'Forcefield parameterisation'\n",
      "Entity 'http://edamontology.org/operation_3894' - Label 'DNA profiling'\n",
      "Entity 'http://edamontology.org/operation_3896' - Label 'Active site prediction'\n",
      "Entity 'http://edamontology.org/operation_3897' - Label 'Ligand-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3898' - Label 'Metal-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3899' - Label 'Protein-protein docking'\n",
      "Entity 'http://edamontology.org/operation_3900' - Label 'DNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3901' - Label 'RNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3902' - Label 'RNA binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3903' - Label 'DNA binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3904' - Label 'Protein disorder prediction'\n",
      "Entity 'http://edamontology.org/operation_3907' - Label 'Information extraction'\n",
      "Entity 'http://edamontology.org/operation_3908' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/operation_3918' - Label 'Genome analysis'\n",
      "Entity 'http://edamontology.org/operation_3919' - Label 'Methylation calling'\n",
      "Entity 'http://edamontology.org/operation_3920' - Label 'DNA testing'\n",
      "Entity 'http://edamontology.org/operation_3921' - Label 'Sequence read processing'\n",
      "Entity 'http://edamontology.org/operation_3925' - Label 'Network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3926' - Label 'Pathway visualisation'\n",
      "Entity 'http://edamontology.org/operation_3927' - Label 'Network analysis'\n",
      "Entity 'http://edamontology.org/operation_3928' - Label 'Pathway analysis'\n",
      "Entity 'http://edamontology.org/operation_3929' - Label 'Metabolic pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_3933' - Label 'Demultiplexing'\n",
      "Entity 'http://edamontology.org/operation_3935' - Label 'Dimensionality reduction'\n",
      "Entity 'http://edamontology.org/operation_3936' - Label 'Feature selection'\n",
      "Entity 'http://edamontology.org/operation_3937' - Label 'Feature extraction'\n",
      "Entity 'http://edamontology.org/operation_3938' - Label 'Virtual screening'\n",
      "Entity 'http://edamontology.org/operation_3942' - Label 'Tree dating'\n",
      "Entity 'http://edamontology.org/operation_3946' - Label 'Ecological modelling'\n",
      "Entity 'http://edamontology.org/operation_3947' - Label 'Phylogenetic tree reconciliation'\n",
      "Entity 'http://edamontology.org/operation_3950' - Label 'Selection detection'\n",
      "Entity 'http://edamontology.org/operation_3960' - Label 'Principal component analysis'\n",
      "Entity 'http://edamontology.org/operation_3961' - Label 'Copy number variation detection'\n",
      "Entity 'http://edamontology.org/operation_3962' - Label 'Deletion detection'\n",
      "Entity 'http://edamontology.org/operation_3963' - Label 'Duplication detection'\n",
      "Entity 'http://edamontology.org/operation_3964' - Label 'Complex CNV detection'\n",
      "Entity 'http://edamontology.org/operation_3965' - Label 'Amplification detection'\n",
      "Entity 'http://edamontology.org/operation_3968' - Label 'Adhesin prediction'\n",
      "Entity 'http://edamontology.org/operation_4008' - Label 'Protein design'\n",
      "Entity 'http://edamontology.org/operation_4009' - Label 'Small molecule design'\n",
      "Entity 'http://edamontology.org/operation_4031' - Label 'Power test'\n",
      "Entity 'http://edamontology.org/operation_4032' - Label 'DNA modification prediction'\n",
      "Entity 'http://edamontology.org/operation_4033' - Label 'Disease transmission analysis'\n",
      "Entity 'http://edamontology.org/operation_4034' - Label 'Multiple testing correction'\n",
      "Entity 'http://edamontology.org/topic_0003' - Label 'Topic'\n",
      "Entity 'http://edamontology.org/topic_0077' - Label 'Nucleic acids'\n",
      "Entity 'http://edamontology.org/topic_0078' - Label 'Proteins'\n",
      "Entity 'http://edamontology.org/topic_0079' - Label 'Metabolites'\n",
      "Entity 'http://edamontology.org/topic_0080' - Label 'Sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0081' - Label 'Structure analysis'\n",
      "Entity 'http://edamontology.org/topic_0082' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0083' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/topic_0084' - Label 'Phylogeny'\n",
      "Entity 'http://edamontology.org/topic_0085' - Label 'Functional genomics'\n",
      "Entity 'http://edamontology.org/topic_0089' - Label 'Ontology and terminology'\n",
      "Entity 'http://edamontology.org/topic_0090' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/topic_0091' - Label 'Bioinformatics'\n",
      "Entity 'http://edamontology.org/topic_0092' - Label 'Data visualisation'\n",
      "Entity 'http://edamontology.org/topic_0094' - Label 'Nucleic acid thermodynamics'\n",
      "Entity 'http://edamontology.org/topic_0097' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/topic_0099' - Label 'RNA'\n",
      "Entity 'http://edamontology.org/topic_0100' - Label 'Nucleic acid restriction'\n",
      "Entity 'http://edamontology.org/topic_0102' - Label 'Mapping'\n",
      "Entity 'http://edamontology.org/topic_0107' - Label 'Genetic codes and codon usage'\n",
      "Entity 'http://edamontology.org/topic_0108' - Label 'Protein expression'\n",
      "Entity 'http://edamontology.org/topic_0109' - Label 'Gene finding'\n",
      "Entity 'http://edamontology.org/topic_0110' - Label 'Transcription'\n",
      "Entity 'http://edamontology.org/topic_0111' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_0112' - Label 'Nucleic acid folding'\n",
      "Entity 'http://edamontology.org/topic_0114' - Label 'Gene structure'\n",
      "Entity 'http://edamontology.org/topic_0121' - Label 'Proteomics'\n",
      "Entity 'http://edamontology.org/topic_0122' - Label 'Structural genomics'\n",
      "Entity 'http://edamontology.org/topic_0123' - Label 'Protein properties'\n",
      "Entity 'http://edamontology.org/topic_0128' - Label 'Protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0130' - Label 'Protein folding, stability and design'\n",
      "Entity 'http://edamontology.org/topic_0133' - Label 'Two-dimensional gel electrophoresis'\n",
      "Entity 'http://edamontology.org/topic_0134' - Label 'Mass spectrometry'\n",
      "Entity 'http://edamontology.org/topic_0135' - Label 'Protein microarrays'\n",
      "Entity 'http://edamontology.org/topic_0137' - Label 'Protein hydropathy'\n",
      "Entity 'http://edamontology.org/topic_0140' - Label 'Protein targeting and localisation'\n",
      "Entity 'http://edamontology.org/topic_0141' - Label 'Protein cleavage sites and proteolysis'\n",
      "Entity 'http://edamontology.org/topic_0143' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/topic_0144' - Label 'Protein residue interactions'\n",
      "Entity 'http://edamontology.org/topic_0147' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0148' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_0149' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_0150' - Label 'Protein design'\n",
      "Entity 'http://edamontology.org/topic_0151' - Label 'G protein-coupled receptors (GPCR)'\n",
      "Entity 'http://edamontology.org/topic_0152' - Label 'Carbohydrates'\n",
      "Entity 'http://edamontology.org/topic_0153' - Label 'Lipids'\n",
      "Entity 'http://edamontology.org/topic_0154' - Label 'Small molecules'\n",
      "Entity 'http://edamontology.org/topic_0156' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/topic_0157' - Label 'Sequence composition, complexity and repeats'\n",
      "Entity 'http://edamontology.org/topic_0158' - Label 'Sequence motifs'\n",
      "Entity 'http://edamontology.org/topic_0159' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/topic_0160' - Label 'Sequence sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_0163' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/topic_0164' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/topic_0166' - Label 'Protein structural motifs and surfaces'\n",
      "Entity 'http://edamontology.org/topic_0167' - Label 'Structural (3D) profiles'\n",
      "Entity 'http://edamontology.org/topic_0172' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0173' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0174' - Label 'Ab initio structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0175' - Label 'Homology modelling'\n",
      "Entity 'http://edamontology.org/topic_0176' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/topic_0177' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/topic_0178' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0179' - Label 'Protein tertiary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0180' - Label 'Protein fold recognition'\n",
      "Entity 'http://edamontology.org/topic_0182' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0183' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/topic_0184' - Label 'Threading'\n",
      "Entity 'http://edamontology.org/topic_0188' - Label 'Sequence profiles and HMMs'\n",
      "Entity 'http://edamontology.org/topic_0191' - Label 'Phylogeny reconstruction'\n",
      "Entity 'http://edamontology.org/topic_0194' - Label 'Phylogenomics'\n",
      "Entity 'http://edamontology.org/topic_0195' - Label 'Virtual PCR'\n",
      "Entity 'http://edamontology.org/topic_0196' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/topic_0199' - Label 'Genetic variation'\n",
      "Entity 'http://edamontology.org/topic_0200' - Label 'Microarrays'\n",
      "Entity 'http://edamontology.org/topic_0202' - Label 'Pharmacology'\n",
      "Entity 'http://edamontology.org/topic_0203' - Label 'Gene expression'\n",
      "Entity 'http://edamontology.org/topic_0204' - Label 'Gene regulation'\n",
      "Entity 'http://edamontology.org/topic_0208' - Label 'Pharmacogenomics'\n",
      "Entity 'http://edamontology.org/topic_0209' - Label 'Medicinal chemistry'\n",
      "Entity 'http://edamontology.org/topic_0210' - Label 'Fish'\n",
      "Entity 'http://edamontology.org/topic_0211' - Label 'Flies'\n",
      "Entity 'http://edamontology.org/topic_0213' - Label 'Mice or rats'\n",
      "Entity 'http://edamontology.org/topic_0215' - Label 'Worms'\n",
      "Entity 'http://edamontology.org/topic_0217' - Label 'Literature analysis'\n",
      "Entity 'http://edamontology.org/topic_0218' - Label 'Natural language processing'\n",
      "Entity 'http://edamontology.org/topic_0219' - Label 'Data submission, annotation, and curation'\n",
      "Entity 'http://edamontology.org/topic_0220' - Label 'Document, record and content management'\n",
      "Entity 'http://edamontology.org/topic_0221' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/topic_0222' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/topic_0593' - Label 'NMR'\n",
      "Entity 'http://edamontology.org/topic_0594' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/topic_0595' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/topic_0598' - Label 'Sequence motif or profile'\n",
      "Entity 'http://edamontology.org/topic_0601' - Label 'Protein modifications'\n",
      "Entity 'http://edamontology.org/topic_0602' - Label 'Molecular interactions, pathways and networks'\n",
      "Entity 'http://edamontology.org/topic_0605' - Label 'Informatics'\n",
      "Entity 'http://edamontology.org/topic_0606' - Label 'Literature data resources'\n",
      "Entity 'http://edamontology.org/topic_0607' - Label 'Laboratory information management'\n",
      "Entity 'http://edamontology.org/topic_0608' - Label 'Cell and tissue culture'\n",
      "Entity 'http://edamontology.org/topic_0610' - Label 'Ecology'\n",
      "Entity 'http://edamontology.org/topic_0611' - Label 'Electron microscopy'\n",
      "Entity 'http://edamontology.org/topic_0612' - Label 'Cell cycle'\n",
      "Entity 'http://edamontology.org/topic_0613' - Label 'Peptides and amino acids'\n",
      "Entity 'http://edamontology.org/topic_0616' - Label 'Organelles'\n",
      "Entity 'http://edamontology.org/topic_0617' - Label 'Ribosomes'\n",
      "Entity 'http://edamontology.org/topic_0618' - Label 'Scents'\n",
      "Entity 'http://edamontology.org/topic_0620' - Label 'Drugs and target structures'\n",
      "Entity 'http://edamontology.org/topic_0621' - Label 'Model organisms'\n",
      "Entity 'http://edamontology.org/topic_0622' - Label 'Genomics'\n",
      "Entity 'http://edamontology.org/topic_0623' - Label 'Gene and protein families'\n",
      "Entity 'http://edamontology.org/topic_0624' - Label 'Chromosomes'\n",
      "Entity 'http://edamontology.org/topic_0625' - Label 'Genotype and phenotype'\n",
      "Entity 'http://edamontology.org/topic_0629' - Label 'Gene expression and microarray'\n",
      "Entity 'http://edamontology.org/topic_0632' - Label 'Probes and primers'\n",
      "Entity 'http://edamontology.org/topic_0634' - Label 'Pathology'\n",
      "Entity 'http://edamontology.org/topic_0635' - Label 'Specific protein resources'\n",
      "Entity 'http://edamontology.org/topic_0637' - Label 'Taxonomy'\n",
      "Entity 'http://edamontology.org/topic_0639' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0640' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0641' - Label 'Repeat sequences'\n",
      "Entity 'http://edamontology.org/topic_0642' - Label 'Low complexity sequences'\n",
      "Entity 'http://edamontology.org/topic_0644' - Label 'Proteome'\n",
      "Entity 'http://edamontology.org/topic_0654' - Label 'DNA'\n",
      "Entity 'http://edamontology.org/topic_0655' - Label 'Coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0659' - Label 'Functional, regulatory and non-coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0660' - Label 'rRNA'\n",
      "Entity 'http://edamontology.org/topic_0663' - Label 'tRNA'\n",
      "Entity 'http://edamontology.org/topic_0694' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/topic_0697' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/topic_0698' - Label 'Protein tertiary structure'\n",
      "Entity 'http://edamontology.org/topic_0722' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/topic_0724' - Label 'Protein families'\n",
      "Entity 'http://edamontology.org/topic_0736' - Label 'Protein folds and structural domains'\n",
      "Entity 'http://edamontology.org/topic_0740' - Label 'Nucleic acid sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0741' - Label 'Protein sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0747' - Label 'Nucleic acid sites and features'\n",
      "Entity 'http://edamontology.org/topic_0748' - Label 'Protein sites and features'\n",
      "Entity 'http://edamontology.org/topic_0749' - Label 'Transcription factors and regulatory sites'\n",
      "Entity 'http://edamontology.org/topic_0751' - Label 'Phosphorylation sites'\n",
      "Entity 'http://edamontology.org/topic_0753' - Label 'Metabolic pathways'\n",
      "Entity 'http://edamontology.org/topic_0754' - Label 'Signaling pathways'\n",
      "Entity 'http://edamontology.org/topic_0767' - Label 'Protein and peptide identification'\n",
      "Entity 'http://edamontology.org/topic_0769' - Label 'Workflows'\n",
      "Entity 'http://edamontology.org/topic_0770' - Label 'Data types and objects'\n",
      "Entity 'http://edamontology.org/topic_0771' - Label 'Theoretical biology'\n",
      "Entity 'http://edamontology.org/topic_0779' - Label 'Mitochondria'\n",
      "Entity 'http://edamontology.org/topic_0780' - Label 'Plant biology'\n",
      "Entity 'http://edamontology.org/topic_0781' - Label 'Virology'\n",
      "Entity 'http://edamontology.org/topic_0782' - Label 'Fungi'\n",
      "Entity 'http://edamontology.org/topic_0783' - Label 'Pathogens'\n",
      "Entity 'http://edamontology.org/topic_0786' - Label 'Arabidopsis'\n",
      "Entity 'http://edamontology.org/topic_0787' - Label 'Rice'\n",
      "Entity 'http://edamontology.org/topic_0796' - Label 'Genetic mapping and linkage'\n",
      "Entity 'http://edamontology.org/topic_0797' - Label 'Comparative genomics'\n",
      "Entity 'http://edamontology.org/topic_0798' - Label 'Mobile genetic elements'\n",
      "Entity 'http://edamontology.org/topic_0803' - Label 'Human disease'\n",
      "Entity 'http://edamontology.org/topic_0804' - Label 'Immunology'\n",
      "Entity 'http://edamontology.org/topic_0820' - Label 'Membrane and lipoproteins'\n",
      "Entity 'http://edamontology.org/topic_0821' - Label 'Enzymes'\n",
      "Entity 'http://edamontology.org/topic_0922' - Label 'Primers'\n",
      "Entity 'http://edamontology.org/topic_1302' - Label 'PolyA signal or sites'\n",
      "Entity 'http://edamontology.org/topic_1304' - Label 'CpG island and isochores'\n",
      "Entity 'http://edamontology.org/topic_1305' - Label 'Restriction sites'\n",
      "Entity 'http://edamontology.org/topic_1307' - Label 'Splice sites'\n",
      "Entity 'http://edamontology.org/topic_1308' - Label 'Matrix/scaffold attachment sites'\n",
      "Entity 'http://edamontology.org/topic_1311' - Label 'Operon'\n",
      "Entity 'http://edamontology.org/topic_1312' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_1317' - Label 'Structural biology'\n",
      "Entity 'http://edamontology.org/topic_1456' - Label 'Protein membrane regions'\n",
      "Entity 'http://edamontology.org/topic_1770' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/topic_1775' - Label 'Function analysis'\n",
      "Entity 'http://edamontology.org/topic_1811' - Label 'Prokaryotes and Archaea'\n",
      "Entity 'http://edamontology.org/topic_2225' - Label 'Protein databases'\n",
      "Entity 'http://edamontology.org/topic_2226' - Label 'Structure determination'\n",
      "Entity 'http://edamontology.org/topic_2229' - Label 'Cell biology'\n",
      "Entity 'http://edamontology.org/topic_2230' - Label 'Classification'\n",
      "Entity 'http://edamontology.org/topic_2232' - Label 'Lipoproteins'\n",
      "Entity 'http://edamontology.org/topic_2257' - Label 'Phylogeny visualisation'\n",
      "Entity 'http://edamontology.org/topic_2258' - Label 'Cheminformatics'\n",
      "Entity 'http://edamontology.org/topic_2259' - Label 'Systems biology'\n",
      "Entity 'http://edamontology.org/topic_2269' - Label 'Statistics and probability'\n",
      "Entity 'http://edamontology.org/topic_2271' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/topic_2275' - Label 'Molecular modelling'\n",
      "Entity 'http://edamontology.org/topic_2276' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/topic_2277' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/topic_2278' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/topic_2280' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/topic_2397' - Label 'Exons'\n",
      "Entity 'http://edamontology.org/topic_2399' - Label 'Gene transcription'\n",
      "Entity 'http://edamontology.org/topic_2533' - Label 'DNA mutation'\n",
      "Entity 'http://edamontology.org/topic_2640' - Label 'Oncology'\n",
      "Entity 'http://edamontology.org/topic_2661' - Label 'Toxins and targets'\n",
      "Entity 'http://edamontology.org/topic_2754' - Label 'Introns'\n",
      "Entity 'http://edamontology.org/topic_2807' - Label 'Tool topic'\n",
      "Entity 'http://edamontology.org/topic_2809' - Label 'Study topic'\n",
      "Entity 'http://edamontology.org/topic_2811' - Label 'Nomenclature'\n",
      "Entity 'http://edamontology.org/topic_2813' - Label 'Disease genes and proteins'\n",
      "Entity 'http://edamontology.org/topic_2814' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/topic_2815' - Label 'Human biology'\n",
      "Entity 'http://edamontology.org/topic_2816' - Label 'Gene resources'\n",
      "Entity 'http://edamontology.org/topic_2817' - Label 'Yeast'\n",
      "Entity 'http://edamontology.org/topic_2818' - Label 'Eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2819' - Label 'Invertebrates'\n",
      "Entity 'http://edamontology.org/topic_2820' - Label 'Vertebrates'\n",
      "Entity 'http://edamontology.org/topic_2821' - Label 'Unicellular eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2826' - Label 'Protein structure alignment'\n",
      "Entity 'http://edamontology.org/topic_2828' - Label 'X-ray diffraction'\n",
      "Entity 'http://edamontology.org/topic_2829' - Label 'Ontologies, nomenclature and classification'\n",
      "Entity 'http://edamontology.org/topic_2830' - Label 'Immunoproteins and antigens'\n",
      "Entity 'http://edamontology.org/topic_2839' - Label 'Molecules'\n",
      "Entity 'http://edamontology.org/topic_2840' - Label 'Toxicology'\n",
      "Entity 'http://edamontology.org/topic_2842' - Label 'High-throughput sequencing'\n",
      "Entity 'http://edamontology.org/topic_2846' - Label 'Gene regulatory networks'\n",
      "Entity 'http://edamontology.org/topic_2847' - Label 'Disease (specific)'\n",
      "Entity 'http://edamontology.org/topic_2867' - Label 'VNTR'\n",
      "Entity 'http://edamontology.org/topic_2868' - Label 'Microsatellites'\n",
      "Entity 'http://edamontology.org/topic_2869' - Label 'RFLP'\n",
      "Entity 'http://edamontology.org/topic_2885' - Label 'DNA polymorphism'\n",
      "Entity 'http://edamontology.org/topic_2953' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/topic_3032' - Label 'Primer or probe design'\n",
      "Entity 'http://edamontology.org/topic_3038' - Label 'Structure databases'\n",
      "Entity 'http://edamontology.org/topic_3039' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/topic_3041' - Label 'Sequence databases'\n",
      "Entity 'http://edamontology.org/topic_3042' - Label 'Nucleic acid sequences'\n",
      "Entity 'http://edamontology.org/topic_3043' - Label 'Protein sequences'\n",
      "Entity 'http://edamontology.org/topic_3044' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/topic_3047' - Label 'Molecular biology'\n",
      "Entity 'http://edamontology.org/topic_3048' - Label 'Mammals'\n",
      "Entity 'http://edamontology.org/topic_3050' - Label 'Biodiversity'\n",
      "Entity 'http://edamontology.org/topic_3052' - Label 'Sequence clusters and classification'\n",
      "Entity 'http://edamontology.org/topic_3053' - Label 'Genetics'\n",
      "Entity 'http://edamontology.org/topic_3055' - Label 'Quantitative genetics'\n",
      "Entity 'http://edamontology.org/topic_3056' - Label 'Population genetics'\n",
      "Entity 'http://edamontology.org/topic_3060' - Label 'Regulatory RNA'\n",
      "Entity 'http://edamontology.org/topic_3061' - Label 'Documentation and help'\n",
      "Entity 'http://edamontology.org/topic_3062' - Label 'Genetic organisation'\n",
      "Entity 'http://edamontology.org/topic_3063' - Label 'Medical informatics'\n",
      "Entity 'http://edamontology.org/topic_3064' - Label 'Developmental biology'\n",
      "Entity 'http://edamontology.org/topic_3065' - Label 'Embryology'\n",
      "Entity 'http://edamontology.org/topic_3067' - Label 'Anatomy'\n",
      "Entity 'http://edamontology.org/topic_3068' - Label 'Literature and language'\n",
      "Entity 'http://edamontology.org/topic_3070' - Label 'Biology'\n",
      "Entity 'http://edamontology.org/topic_3071' - Label 'Data management'\n",
      "Entity 'http://edamontology.org/topic_3072' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/topic_3073' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/topic_3074' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/topic_3075' - Label 'Biological system modelling'\n",
      "Entity 'http://edamontology.org/topic_3077' - Label 'Data acquisition'\n",
      "Entity 'http://edamontology.org/topic_3078' - Label 'Genes and proteins resources'\n",
      "Entity 'http://edamontology.org/topic_3118' - Label 'Protein topological domains'\n",
      "Entity 'http://edamontology.org/topic_3120' - Label 'Protein variants'\n",
      "Entity 'http://edamontology.org/topic_3123' - Label 'Expression signals'\n",
      "Entity 'http://edamontology.org/topic_3125' - Label 'DNA binding sites'\n",
      "Entity 'http://edamontology.org/topic_3126' - Label 'Nucleic acid repeats'\n",
      "Entity 'http://edamontology.org/topic_3127' - Label 'DNA replication and recombination'\n",
      "Entity 'http://edamontology.org/topic_3135' - Label 'Signal or transit peptide'\n",
      "Entity 'http://edamontology.org/topic_3139' - Label 'Sequence tagged sites'\n",
      "Entity 'http://edamontology.org/topic_3168' - Label 'Sequencing'\n",
      "Entity 'http://edamontology.org/topic_3169' - Label 'ChIP-seq'\n",
      "Entity 'http://edamontology.org/topic_3170' - Label 'RNA-Seq'\n",
      "Entity 'http://edamontology.org/topic_3171' - Label 'DNA methylation'\n",
      "Entity 'http://edamontology.org/topic_3172' - Label 'Metabolomics'\n",
      "Entity 'http://edamontology.org/topic_3173' - Label 'Epigenomics'\n",
      "Entity 'http://edamontology.org/topic_3174' - Label 'Metagenomics'\n",
      "Entity 'http://edamontology.org/topic_3175' - Label 'Structural variation'\n",
      "Entity 'http://edamontology.org/topic_3176' - Label 'DNA packaging'\n",
      "Entity 'http://edamontology.org/topic_3177' - Label 'DNA-Seq'\n",
      "Entity 'http://edamontology.org/topic_3178' - Label 'RNA-Seq alignment'\n",
      "Entity 'http://edamontology.org/topic_3179' - Label 'ChIP-on-chip'\n",
      "Entity 'http://edamontology.org/topic_3263' - Label 'Data security'\n",
      "Entity 'http://edamontology.org/topic_3277' - Label 'Sample collections'\n",
      "Entity 'http://edamontology.org/topic_3292' - Label 'Biochemistry'\n",
      "Entity 'http://edamontology.org/topic_3293' - Label 'Phylogenetics'\n",
      "Entity 'http://edamontology.org/topic_3295' - Label 'Epigenetics'\n",
      "Entity 'http://edamontology.org/topic_3297' - Label 'Biotechnology'\n",
      "Entity 'http://edamontology.org/topic_3298' - Label 'Phenomics'\n",
      "Entity 'http://edamontology.org/topic_3299' - Label 'Evolutionary biology'\n",
      "Entity 'http://edamontology.org/topic_3300' - Label 'Physiology'\n",
      "Entity 'http://edamontology.org/topic_3301' - Label 'Microbiology'\n",
      "Entity 'http://edamontology.org/topic_3302' - Label 'Parasitology'\n",
      "Entity 'http://edamontology.org/topic_3303' - Label 'Medicine'\n",
      "Entity 'http://edamontology.org/topic_3304' - Label 'Neurobiology'\n",
      "Entity 'http://edamontology.org/topic_3305' - Label 'Public health and epidemiology'\n",
      "Entity 'http://edamontology.org/topic_3306' - Label 'Biophysics'\n",
      "Entity 'http://edamontology.org/topic_3307' - Label 'Computational biology'\n",
      "Entity 'http://edamontology.org/topic_3308' - Label 'Transcriptomics'\n",
      "Entity 'http://edamontology.org/topic_3314' - Label 'Chemistry'\n",
      "Entity 'http://edamontology.org/topic_3315' - Label 'Mathematics'\n",
      "Entity 'http://edamontology.org/topic_3316' - Label 'Computer science'\n",
      "Entity 'http://edamontology.org/topic_3318' - Label 'Physics'\n",
      "Entity 'http://edamontology.org/topic_3320' - Label 'RNA splicing'\n",
      "Entity 'http://edamontology.org/topic_3321' - Label 'Molecular genetics'\n",
      "Entity 'http://edamontology.org/topic_3322' - Label 'Respiratory medicine'\n",
      "Entity 'http://edamontology.org/topic_3323' - Label 'Metabolic disease'\n",
      "Entity 'http://edamontology.org/topic_3324' - Label 'Infectious disease'\n",
      "Entity 'http://edamontology.org/topic_3325' - Label 'Rare diseases'\n",
      "Entity 'http://edamontology.org/topic_3332' - Label 'Computational chemistry'\n",
      "Entity 'http://edamontology.org/topic_3334' - Label 'Neurology'\n",
      "Entity 'http://edamontology.org/topic_3335' - Label 'Cardiology'\n",
      "Entity 'http://edamontology.org/topic_3336' - Label 'Drug discovery'\n",
      "Entity 'http://edamontology.org/topic_3337' - Label 'Biobank'\n",
      "Entity 'http://edamontology.org/topic_3338' - Label 'Mouse clinic'\n",
      "Entity 'http://edamontology.org/topic_3339' - Label 'Microbial collection'\n",
      "Entity 'http://edamontology.org/topic_3340' - Label 'Cell culture collection'\n",
      "Entity 'http://edamontology.org/topic_3341' - Label 'Clone library'\n",
      "Entity 'http://edamontology.org/topic_3342' - Label 'Translational medicine'\n",
      "Entity 'http://edamontology.org/topic_3343' - Label 'Compound libraries and screening'\n",
      "Entity 'http://edamontology.org/topic_3344' - Label 'Biomedical science'\n",
      "Entity 'http://edamontology.org/topic_3345' - Label 'Data identity and mapping'\n",
      "Entity 'http://edamontology.org/topic_3346' - Label 'Sequence search'\n",
      "Entity 'http://edamontology.org/topic_3360' - Label 'Biomarkers'\n",
      "Entity 'http://edamontology.org/topic_3361' - Label 'Laboratory techniques'\n",
      "Entity 'http://edamontology.org/topic_3365' - Label 'Data architecture, analysis and design'\n",
      "Entity 'http://edamontology.org/topic_3366' - Label 'Data integration and warehousing'\n",
      "Entity 'http://edamontology.org/topic_3368' - Label 'Biomaterials'\n",
      "Entity 'http://edamontology.org/topic_3369' - Label 'Chemical biology'\n",
      "Entity 'http://edamontology.org/topic_3370' - Label 'Analytical chemistry'\n",
      "Entity 'http://edamontology.org/topic_3371' - Label 'Synthetic chemistry'\n",
      "Entity 'http://edamontology.org/topic_3372' - Label 'Software engineering'\n",
      "Entity 'http://edamontology.org/topic_3373' - Label 'Drug development'\n",
      "Entity 'http://edamontology.org/topic_3374' - Label 'Biotherapeutics'\n",
      "Entity 'http://edamontology.org/topic_3375' - Label 'Drug metabolism'\n",
      "Entity 'http://edamontology.org/topic_3376' - Label 'Medicines research and development'\n",
      "Entity 'http://edamontology.org/topic_3377' - Label 'Safety sciences'\n",
      "Entity 'http://edamontology.org/topic_3378' - Label 'Pharmacovigilance'\n",
      "Entity 'http://edamontology.org/topic_3379' - Label 'Preclinical and clinical studies'\n",
      "Entity 'http://edamontology.org/topic_3382' - Label 'Imaging'\n",
      "Entity 'http://edamontology.org/topic_3383' - Label 'Bioimaging'\n",
      "Entity 'http://edamontology.org/topic_3384' - Label 'Medical imaging'\n",
      "Entity 'http://edamontology.org/topic_3385' - Label 'Light microscopy'\n",
      "Entity 'http://edamontology.org/topic_3386' - Label 'Laboratory animal science'\n",
      "Entity 'http://edamontology.org/topic_3387' - Label 'Marine biology'\n",
      "Entity 'http://edamontology.org/topic_3388' - Label 'Molecular medicine'\n",
      "Entity 'http://edamontology.org/topic_3390' - Label 'Nutritional science'\n",
      "Entity 'http://edamontology.org/topic_3391' - Label 'Omics'\n",
      "Entity 'http://edamontology.org/topic_3393' - Label 'Quality affairs'\n",
      "Entity 'http://edamontology.org/topic_3394' - Label 'Regulatory affairs'\n",
      "Entity 'http://edamontology.org/topic_3395' - Label 'Regenerative medicine'\n",
      "Entity 'http://edamontology.org/topic_3396' - Label 'Systems medicine'\n",
      "Entity 'http://edamontology.org/topic_3397' - Label 'Veterinary medicine'\n",
      "Entity 'http://edamontology.org/topic_3398' - Label 'Bioengineering'\n",
      "Entity 'http://edamontology.org/topic_3399' - Label 'Geriatric medicine'\n",
      "Entity 'http://edamontology.org/topic_3400' - Label 'Allergy, clinical immunology and immunotherapeutics'\n",
      "Entity 'http://edamontology.org/topic_3401' - Label 'Pain medicine'\n",
      "Entity 'http://edamontology.org/topic_3402' - Label 'Anaesthesiology'\n",
      "Entity 'http://edamontology.org/topic_3403' - Label 'Critical care medicine'\n",
      "Entity 'http://edamontology.org/topic_3404' - Label 'Dermatology'\n",
      "Entity 'http://edamontology.org/topic_3405' - Label 'Dentistry'\n",
      "Entity 'http://edamontology.org/topic_3406' - Label 'Ear, nose and throat medicine'\n",
      "Entity 'http://edamontology.org/topic_3407' - Label 'Endocrinology and metabolism'\n",
      "Entity 'http://edamontology.org/topic_3408' - Label 'Haematology'\n",
      "Entity 'http://edamontology.org/topic_3409' - Label 'Gastroenterology'\n",
      "Entity 'http://edamontology.org/topic_3410' - Label 'Gender medicine'\n",
      "Entity 'http://edamontology.org/topic_3411' - Label 'Gynaecology and obstetrics'\n",
      "Entity 'http://edamontology.org/topic_3412' - Label 'Hepatic and biliary medicine'\n",
      "Entity 'http://edamontology.org/topic_3413' - Label 'Infectious tropical disease'\n",
      "Entity 'http://edamontology.org/topic_3414' - Label 'Trauma medicine'\n",
      "Entity 'http://edamontology.org/topic_3415' - Label 'Medical toxicology'\n",
      "Entity 'http://edamontology.org/topic_3416' - Label 'Musculoskeletal medicine'\n",
      "Entity 'http://edamontology.org/topic_3417' - Label 'Ophthalmology'\n",
      "Entity 'http://edamontology.org/topic_3418' - Label 'Paediatrics'\n",
      "Entity 'http://edamontology.org/topic_3419' - Label 'Psychiatry'\n",
      "Entity 'http://edamontology.org/topic_3420' - Label 'Reproductive health'\n",
      "Entity 'http://edamontology.org/topic_3421' - Label 'Surgery'\n",
      "Entity 'http://edamontology.org/topic_3422' - Label 'Urology and nephrology'\n",
      "Entity 'http://edamontology.org/topic_3423' - Label 'Complementary medicine'\n",
      "Entity 'http://edamontology.org/topic_3444' - Label 'MRI'\n",
      "Entity 'http://edamontology.org/topic_3448' - Label 'Neutron diffraction'\n",
      "Entity 'http://edamontology.org/topic_3452' - Label 'Tomography'\n",
      "Entity 'http://edamontology.org/topic_3473' - Label 'Data mining'\n",
      "Entity 'http://edamontology.org/topic_3474' - Label 'Machine learning'\n",
      "Entity 'http://edamontology.org/topic_3489' - Label 'Database management'\n",
      "Entity 'http://edamontology.org/topic_3500' - Label 'Zoology'\n",
      "Entity 'http://edamontology.org/topic_3510' - Label 'Protein sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3511' - Label 'Nucleic acid sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3512' - Label 'Gene transcripts'\n",
      "Entity 'http://edamontology.org/topic_3514' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_3515' - Label 'Protein-drug interactions'\n",
      "Entity 'http://edamontology.org/topic_3516' - Label 'Genotyping experiment'\n",
      "Entity 'http://edamontology.org/topic_3517' - Label 'GWAS study'\n",
      "Entity 'http://edamontology.org/topic_3518' - Label 'Microarray experiment'\n",
      "Entity 'http://edamontology.org/topic_3519' - Label 'PCR experiment'\n",
      "Entity 'http://edamontology.org/topic_3520' - Label 'Proteomics experiment'\n",
      "Entity 'http://edamontology.org/topic_3521' - Label '2D PAGE experiment'\n",
      "Entity 'http://edamontology.org/topic_3522' - Label 'Northern blot experiment'\n",
      "Entity 'http://edamontology.org/topic_3523' - Label 'RNAi experiment'\n",
      "Entity 'http://edamontology.org/topic_3524' - Label 'Simulation experiment'\n",
      "Entity 'http://edamontology.org/topic_3525' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_3526' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_3527' - Label 'Cellular process pathways'\n",
      "Entity 'http://edamontology.org/topic_3528' - Label 'Disease pathways'\n",
      "Entity 'http://edamontology.org/topic_3529' - Label 'Environmental information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3530' - Label 'Genetic information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3531' - Label 'Protein super-secondary structure'\n",
      "Entity 'http://edamontology.org/topic_3533' - Label 'Protein active sites'\n",
      "Entity 'http://edamontology.org/topic_3534' - Label 'Protein binding sites'\n",
      "Entity 'http://edamontology.org/topic_3535' - Label 'Protein-nucleic acid binding sites'\n",
      "Entity 'http://edamontology.org/topic_3536' - Label 'Protein cleavage sites'\n",
      "Entity 'http://edamontology.org/topic_3537' - Label 'Protein chemical modifications'\n",
      "Entity 'http://edamontology.org/topic_3538' - Label 'Protein disordered structure'\n",
      "Entity 'http://edamontology.org/topic_3539' - Label 'Protein domains'\n",
      "Entity 'http://edamontology.org/topic_3540' - Label 'Protein key folding sites'\n",
      "Entity 'http://edamontology.org/topic_3541' - Label 'Protein post-translational modifications'\n",
      "Entity 'http://edamontology.org/topic_3542' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/topic_3543' - Label 'Protein sequence repeats'\n",
      "Entity 'http://edamontology.org/topic_3544' - Label 'Protein signal peptides'\n",
      "Entity 'http://edamontology.org/topic_3569' - Label 'Applied mathematics'\n",
      "Entity 'http://edamontology.org/topic_3570' - Label 'Pure mathematics'\n",
      "Entity 'http://edamontology.org/topic_3571' - Label 'Data governance'\n",
      "Entity 'http://edamontology.org/topic_3572' - Label 'Data quality management'\n",
      "Entity 'http://edamontology.org/topic_3573' - Label 'Freshwater biology'\n",
      "Entity 'http://edamontology.org/topic_3574' - Label 'Human genetics'\n",
      "Entity 'http://edamontology.org/topic_3575' - Label 'Tropical medicine'\n",
      "Entity 'http://edamontology.org/topic_3576' - Label 'Medical biotechnology'\n",
      "Entity 'http://edamontology.org/topic_3577' - Label 'Personalised medicine'\n",
      "Entity 'http://edamontology.org/topic_3656' - Label 'Immunoprecipitation experiment'\n",
      "Entity 'http://edamontology.org/topic_3673' - Label 'Whole genome sequencing'\n",
      "Entity 'http://edamontology.org/topic_3674' - Label 'Methylated DNA immunoprecipitation'\n",
      "Entity 'http://edamontology.org/topic_3676' - Label 'Exome sequencing'\n",
      "Entity 'http://edamontology.org/topic_3678' - Label 'Experimental design and studies'\n",
      "Entity 'http://edamontology.org/topic_3679' - Label 'Animal study'\n",
      "Entity 'http://edamontology.org/topic_3697' - Label 'Microbial ecology'\n",
      "Entity 'http://edamontology.org/topic_3794' - Label 'RNA immunoprecipitation'\n",
      "Entity 'http://edamontology.org/topic_3796' - Label 'Population genomics'\n",
      "Entity 'http://edamontology.org/topic_3810' - Label 'Agricultural science'\n",
      "Entity 'http://edamontology.org/topic_3837' - Label 'Metagenomic sequencing'\n",
      "Entity 'http://edamontology.org/topic_3855' - Label 'Environmental sciences'\n",
      "Entity 'http://edamontology.org/topic_3892' - Label 'Biomolecular simulation'\n",
      "Entity 'http://edamontology.org/topic_3895' - Label 'Synthetic biology'\n",
      "Entity 'http://edamontology.org/topic_3912' - Label 'Genetic engineering'\n",
      "Entity 'http://edamontology.org/topic_3922' - Label 'Proteogenomics'\n",
      "Entity 'http://edamontology.org/topic_3923' - Label 'Genome resequencing'\n",
      "Entity 'http://edamontology.org/topic_3930' - Label 'Immunogenetics'\n",
      "Entity 'http://edamontology.org/topic_3931' - Label 'Chemometrics'\n",
      "Entity 'http://edamontology.org/topic_3934' - Label 'Cytometry'\n",
      "Entity 'http://edamontology.org/topic_3939' - Label 'Metabolic engineering'\n",
      "Entity 'http://edamontology.org/topic_3940' - Label 'Chromosome conformation capture'\n",
      "Entity 'http://edamontology.org/topic_3941' - Label 'Metatranscriptomics'\n",
      "Entity 'http://edamontology.org/topic_3943' - Label 'Paleogenomics'\n",
      "Entity 'http://edamontology.org/topic_3944' - Label 'Cladistics'\n",
      "Entity 'http://edamontology.org/topic_3945' - Label 'Molecular evolution'\n",
      "Entity 'http://edamontology.org/topic_3948' - Label 'Immunoinformatics'\n",
      "Entity 'http://edamontology.org/topic_3954' - Label 'Echography'\n",
      "Entity 'http://edamontology.org/topic_3955' - Label 'Fluxomics'\n",
      "Entity 'http://edamontology.org/topic_3957' - Label 'Protein interaction experiment'\n",
      "Entity 'http://edamontology.org/topic_3958' - Label 'Copy number variation'\n",
      "Entity 'http://edamontology.org/topic_3959' - Label 'Cytogenetics'\n",
      "Entity 'http://edamontology.org/topic_3966' - Label 'Vaccinology'\n",
      "Entity 'http://edamontology.org/topic_3967' - Label 'Immunomics'\n",
      "Entity 'http://edamontology.org/topic_3974' - Label 'Epistasis'\n",
      "Entity 'http://edamontology.org/topic_4010' - Label 'Open science'\n",
      "Entity 'http://edamontology.org/topic_4011' - Label 'Data rescue'\n",
      "Entity 'http://edamontology.org/topic_4012' - Label 'FAIR data'\n",
      "Entity 'http://edamontology.org/topic_4013' - Label 'Antimicrobial Resistance'\n",
      "Entity 'http://edamontology.org/topic_4014' - Label 'Electroencephalography'\n",
      "Entity 'http://edamontology.org/topic_4016' - Label 'Electrocardiography'\n",
      "Entity 'http://edamontology.org/topic_4017' - Label 'Cryogenic electron microscopy'\n",
      "Entity 'http://edamontology.org/topic_4019' - Label 'Biosciences'\n",
      "Entity 'http://edamontology.org/topic_4020' - Label 'Carbon cycle'\n",
      "Entity 'http://edamontology.org/topic_4021' - Label 'Multiomics'\n",
      "Entity 'http://edamontology.org/topic_4027' - Label 'Ribosome Profiling'\n",
      "Entity 'http://edamontology.org/topic_4028' - Label 'Single-Cell Sequencing'\n",
      "Entity 'http://edamontology.org/topic_4029' - Label 'Acoustics'\n",
      "Entity 'http://edamontology.org/topic_4030' - Label 'Microfluidics'\n",
      "Entity 'http://edamontology.org/topic_4037' - Label 'Genomic imprinting'\n",
      "Entity 'http://edamontology.org/topic_4038' - Label 'Metabarcoding'\n"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "\n",
    "SELECT ?entity ?property ?value ?label ?id WHERE {\n",
    "    \n",
    "    VALUES ?property { edam:next_id }\n",
    "    #retrieve the last ID\n",
    "    <http://edamontology.org> ?property ?value .\n",
    "    \n",
    "    #match all IDs\n",
    "    ?entity rdf:type owl:Class .\n",
    "    FILTER ( ?entity != <http://www.geneontology.org/formats/oboInOwl#ObsoleteClass>) .\n",
    "\n",
    "    ?entity rdfs:label ?label .    \n",
    "    BIND(strafter(str(?entity), \"_\") AS ?id) .\n",
    "} \n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:28:05.062069Z",
     "start_time": "2024-02-13T17:28:04.747090Z"
    }
   },
   "execution_count": 32
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'N135c4b5b1ace416c880ccdef24bdefe6' - Label 'None'\n",
      "Entity 'Nc0796c735a0249f3a51f0e3cb8920cb4' - Label 'None'\n",
      "Entity 'http://edamontology.org/data_0005' - Label 'Resource type'\n",
      "Entity 'http://edamontology.org/data_0006' - Label 'Data'\n",
      "Entity 'http://edamontology.org/data_0007' - Label 'Tool'\n",
      "Entity 'http://edamontology.org/data_0581' - Label 'Database'\n",
      "Entity 'http://edamontology.org/data_0582' - Label 'Ontology'\n",
      "Entity 'http://edamontology.org/data_0583' - Label 'Directory metadata'\n",
      "Entity 'http://edamontology.org/data_0831' - Label 'MeSH vocabulary'\n",
      "Entity 'http://edamontology.org/data_0832' - Label 'HGNC vocabulary'\n",
      "Entity 'http://edamontology.org/data_0835' - Label 'UMLS vocabulary'\n",
      "Entity 'http://edamontology.org/data_0842' - Label 'Identifier'\n",
      "Entity 'http://edamontology.org/data_0843' - Label 'Database entry'\n",
      "Entity 'http://edamontology.org/data_0844' - Label 'Molecular mass'\n",
      "Entity 'http://edamontology.org/data_0845' - Label 'Molecular charge'\n",
      "Entity 'http://edamontology.org/data_0846' - Label 'Chemical formula'\n",
      "Entity 'http://edamontology.org/data_0847' - Label 'QSAR descriptor'\n",
      "Entity 'http://edamontology.org/data_0848' - Label 'Raw sequence'\n",
      "Entity 'http://edamontology.org/data_0849' - Label 'Sequence record'\n",
      "Entity 'http://edamontology.org/data_0850' - Label 'Sequence set'\n",
      "Entity 'http://edamontology.org/data_0851' - Label 'Sequence mask character'\n",
      "Entity 'http://edamontology.org/data_0852' - Label 'Sequence mask type'\n",
      "Entity 'http://edamontology.org/data_0853' - Label 'DNA sense specification'\n",
      "Entity 'http://edamontology.org/data_0854' - Label 'Sequence length specification'\n",
      "Entity 'http://edamontology.org/data_0855' - Label 'Sequence metadata'\n",
      "Entity 'http://edamontology.org/data_0856' - Label 'Sequence feature source'\n",
      "Entity 'http://edamontology.org/data_0857' - Label 'Sequence search results'\n",
      "Entity 'http://edamontology.org/data_0858' - Label 'Sequence signature matches'\n",
      "Entity 'http://edamontology.org/data_0859' - Label 'Sequence signature model'\n",
      "Entity 'http://edamontology.org/data_0860' - Label 'Sequence signature data'\n",
      "Entity 'http://edamontology.org/data_0861' - Label 'Sequence alignment (words)'\n",
      "Entity 'http://edamontology.org/data_0862' - Label 'Dotplot'\n",
      "Entity 'http://edamontology.org/data_0863' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/data_0864' - Label 'Sequence alignment parameter'\n",
      "Entity 'http://edamontology.org/data_0865' - Label 'Sequence similarity score'\n",
      "Entity 'http://edamontology.org/data_0866' - Label 'Sequence alignment metadata'\n",
      "Entity 'http://edamontology.org/data_0867' - Label 'Sequence alignment report'\n",
      "Entity 'http://edamontology.org/data_0868' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0869' - Label 'Sequence-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0870' - Label 'Sequence distance matrix'\n",
      "Entity 'http://edamontology.org/data_0871' - Label 'Phylogenetic character data'\n",
      "Entity 'http://edamontology.org/data_0872' - Label 'Phylogenetic tree'\n",
      "Entity 'http://edamontology.org/data_0874' - Label 'Comparison matrix'\n",
      "Entity 'http://edamontology.org/data_0875' - Label 'Protein topology'\n",
      "Entity 'http://edamontology.org/data_0876' - Label 'Protein features report (secondary structure)'\n",
      "Entity 'http://edamontology.org/data_0877' - Label 'Protein features report (super-secondary)'\n",
      "Entity 'http://edamontology.org/data_0878' - Label 'Protein secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_0879' - Label 'Secondary structure alignment metadata (protein)'\n",
      "Entity 'http://edamontology.org/data_0880' - Label 'RNA secondary structure'\n",
      "Entity 'http://edamontology.org/data_0881' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_0882' - Label 'Secondary structure alignment metadata (RNA)'\n",
      "Entity 'http://edamontology.org/data_0883' - Label 'Structure'\n",
      "Entity 'http://edamontology.org/data_0884' - Label 'Tertiary structure record'\n",
      "Entity 'http://edamontology.org/data_0885' - Label 'Structure database search results'\n",
      "Entity 'http://edamontology.org/data_0886' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/data_0887' - Label 'Structure alignment report'\n",
      "Entity 'http://edamontology.org/data_0888' - Label 'Structure similarity score'\n",
      "Entity 'http://edamontology.org/data_0889' - Label 'Structural profile'\n",
      "Entity 'http://edamontology.org/data_0890' - Label 'Structural (3D) profile alignment'\n",
      "Entity 'http://edamontology.org/data_0891' - Label 'Sequence-3D profile alignment'\n",
      "Entity 'http://edamontology.org/data_0892' - Label 'Protein sequence-structure scoring matrix'\n",
      "Entity 'http://edamontology.org/data_0893' - Label 'Sequence-structure alignment'\n",
      "Entity 'http://edamontology.org/data_0894' - Label 'Amino acid annotation'\n",
      "Entity 'http://edamontology.org/data_0895' - Label 'Peptide annotation'\n",
      "Entity 'http://edamontology.org/data_0896' - Label 'Protein report'\n",
      "Entity 'http://edamontology.org/data_0897' - Label 'Protein property'\n",
      "Entity 'http://edamontology.org/data_0899' - Label 'Protein structural motifs and surfaces'\n",
      "Entity 'http://edamontology.org/data_0900' - Label 'Protein domain classification'\n",
      "Entity 'http://edamontology.org/data_0901' - Label 'Protein features report (domains)'\n",
      "Entity 'http://edamontology.org/data_0902' - Label 'Protein architecture report'\n",
      "Entity 'http://edamontology.org/data_0903' - Label 'Protein folding report'\n",
      "Entity 'http://edamontology.org/data_0904' - Label 'Protein features (mutation)'\n",
      "Entity 'http://edamontology.org/data_0905' - Label 'Protein interaction raw data'\n",
      "Entity 'http://edamontology.org/data_0906' - Label 'Protein interaction data'\n",
      "Entity 'http://edamontology.org/data_0907' - Label 'Protein family report'\n",
      "Entity 'http://edamontology.org/data_0909' - Label 'Vmax'\n",
      "Entity 'http://edamontology.org/data_0910' - Label 'Km'\n",
      "Entity 'http://edamontology.org/data_0911' - Label 'Nucleotide base annotation'\n",
      "Entity 'http://edamontology.org/data_0912' - Label 'Nucleic acid property'\n",
      "Entity 'http://edamontology.org/data_0914' - Label 'Codon usage data'\n",
      "Entity 'http://edamontology.org/data_0916' - Label 'Gene report'\n",
      "Entity 'http://edamontology.org/data_0917' - Label 'Gene classification'\n",
      "Entity 'http://edamontology.org/data_0918' - Label 'DNA variation'\n",
      "Entity 'http://edamontology.org/data_0919' - Label 'Chromosome report'\n",
      "Entity 'http://edamontology.org/data_0920' - Label 'Genotype/phenotype report'\n",
      "Entity 'http://edamontology.org/data_0923' - Label 'PCR experiment report'\n",
      "Entity 'http://edamontology.org/data_0924' - Label 'Sequence trace'\n",
      "Entity 'http://edamontology.org/data_0925' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/data_0926' - Label 'RH scores'\n",
      "Entity 'http://edamontology.org/data_0927' - Label 'Genetic linkage report'\n",
      "Entity 'http://edamontology.org/data_0928' - Label 'Gene expression profile'\n",
      "Entity 'http://edamontology.org/data_0931' - Label 'Microarray experiment report'\n",
      "Entity 'http://edamontology.org/data_0932' - Label 'Oligonucleotide probe data'\n",
      "Entity 'http://edamontology.org/data_0933' - Label 'SAGE experimental data'\n",
      "Entity 'http://edamontology.org/data_0934' - Label 'MPSS experimental data'\n",
      "Entity 'http://edamontology.org/data_0935' - Label 'SBS experimental data'\n",
      "Entity 'http://edamontology.org/data_0936' - Label 'Sequence tag profile (with gene assignment)'\n",
      "Entity 'http://edamontology.org/data_0937' - Label 'Electron density map'\n",
      "Entity 'http://edamontology.org/data_0938' - Label 'Raw NMR data'\n",
      "Entity 'http://edamontology.org/data_0939' - Label 'CD spectra'\n",
      "Entity 'http://edamontology.org/data_0940' - Label 'Volume map'\n",
      "Entity 'http://edamontology.org/data_0941' - Label 'Electron microscopy model'\n",
      "Entity 'http://edamontology.org/data_0942' - Label '2D PAGE image'\n",
      "Entity 'http://edamontology.org/data_0943' - Label 'Mass spectrum'\n",
      "Entity 'http://edamontology.org/data_0944' - Label 'Peptide mass fingerprint'\n",
      "Entity 'http://edamontology.org/data_0945' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/data_0946' - Label 'Pathway or network annotation'\n",
      "Entity 'http://edamontology.org/data_0947' - Label 'Biological pathway map'\n",
      "Entity 'http://edamontology.org/data_0948' - Label 'Data resource definition'\n",
      "Entity 'http://edamontology.org/data_0949' - Label 'Workflow metadata'\n",
      "Entity 'http://edamontology.org/data_0950' - Label 'Mathematical model'\n",
      "Entity 'http://edamontology.org/data_0951' - Label 'Statistical estimate score'\n",
      "Entity 'http://edamontology.org/data_0952' - Label 'EMBOSS database resource definition'\n",
      "Entity 'http://edamontology.org/data_0953' - Label 'Version information'\n",
      "Entity 'http://edamontology.org/data_0954' - Label 'Database cross-mapping'\n",
      "Entity 'http://edamontology.org/data_0955' - Label 'Data index'\n",
      "Entity 'http://edamontology.org/data_0956' - Label 'Data index report'\n",
      "Entity 'http://edamontology.org/data_0957' - Label 'Database metadata'\n",
      "Entity 'http://edamontology.org/data_0958' - Label 'Tool metadata'\n",
      "Entity 'http://edamontology.org/data_0959' - Label 'Job metadata'\n",
      "Entity 'http://edamontology.org/data_0960' - Label 'User metadata'\n",
      "Entity 'http://edamontology.org/data_0962' - Label 'Small molecule report'\n",
      "Entity 'http://edamontology.org/data_0963' - Label 'Cell line report'\n",
      "Entity 'http://edamontology.org/data_0964' - Label 'Scent annotation'\n",
      "Entity 'http://edamontology.org/data_0966' - Label 'Ontology term'\n",
      "Entity 'http://edamontology.org/data_0967' - Label 'Ontology concept data'\n",
      "Entity 'http://edamontology.org/data_0968' - Label 'Keyword'\n",
      "Entity 'http://edamontology.org/data_0970' - Label 'Citation'\n",
      "Entity 'http://edamontology.org/data_0971' - Label 'Article'\n",
      "Entity 'http://edamontology.org/data_0972' - Label 'Text mining report'\n",
      "Entity 'http://edamontology.org/data_0974' - Label 'Entity identifier'\n",
      "Entity 'http://edamontology.org/data_0975' - Label 'Data resource identifier'\n",
      "Entity 'http://edamontology.org/data_0976' - Label 'Identifier (by type of entity)'\n",
      "Entity 'http://edamontology.org/data_0977' - Label 'Tool identifier'\n",
      "Entity 'http://edamontology.org/data_0978' - Label 'Discrete entity identifier'\n",
      "Entity 'http://edamontology.org/data_0979' - Label 'Entity feature identifier'\n",
      "Entity 'http://edamontology.org/data_0980' - Label 'Entity collection identifier'\n",
      "Entity 'http://edamontology.org/data_0981' - Label 'Phenomenon identifier'\n",
      "Entity 'http://edamontology.org/data_0982' - Label 'Molecule identifier'\n",
      "Entity 'http://edamontology.org/data_0983' - Label 'Atom ID'\n",
      "Entity 'http://edamontology.org/data_0984' - Label 'Molecule name'\n",
      "Entity 'http://edamontology.org/data_0985' - Label 'Molecule type'\n",
      "Entity 'http://edamontology.org/data_0986' - Label 'Chemical identifier'\n",
      "Entity 'http://edamontology.org/data_0987' - Label 'Chromosome name'\n",
      "Entity 'http://edamontology.org/data_0988' - Label 'Peptide identifier'\n",
      "Entity 'http://edamontology.org/data_0989' - Label 'Protein identifier'\n",
      "Entity 'http://edamontology.org/data_0990' - Label 'Compound name'\n",
      "Entity 'http://edamontology.org/data_0991' - Label 'Chemical registry number'\n",
      "Entity 'http://edamontology.org/data_0992' - Label 'Ligand identifier'\n",
      "Entity 'http://edamontology.org/data_0993' - Label 'Drug identifier'\n",
      "Entity 'http://edamontology.org/data_0994' - Label 'Amino acid identifier'\n",
      "Entity 'http://edamontology.org/data_0995' - Label 'Nucleotide identifier'\n",
      "Entity 'http://edamontology.org/data_0996' - Label 'Monosaccharide identifier'\n",
      "Entity 'http://edamontology.org/data_0997' - Label 'Chemical name (ChEBI)'\n",
      "Entity 'http://edamontology.org/data_0998' - Label 'Chemical name (IUPAC)'\n",
      "Entity 'http://edamontology.org/data_0999' - Label 'Chemical name (INN)'\n",
      "Entity 'http://edamontology.org/data_1000' - Label 'Chemical name (brand)'\n",
      "Entity 'http://edamontology.org/data_1001' - Label 'Chemical name (synonymous)'\n",
      "Entity 'http://edamontology.org/data_1002' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_1003' - Label 'Chemical registry number (Beilstein)'\n",
      "Entity 'http://edamontology.org/data_1004' - Label 'Chemical registry number (Gmelin)'\n",
      "Entity 'http://edamontology.org/data_1005' - Label 'HET group name'\n",
      "Entity 'http://edamontology.org/data_1006' - Label 'Amino acid name'\n",
      "Entity 'http://edamontology.org/data_1007' - Label 'Nucleotide code'\n",
      "Entity 'http://edamontology.org/data_1008' - Label 'Polypeptide chain ID'\n",
      "Entity 'http://edamontology.org/data_1009' - Label 'Protein name'\n",
      "Entity 'http://edamontology.org/data_1010' - Label 'Enzyme identifier'\n",
      "Entity 'http://edamontology.org/data_1011' - Label 'EC number'\n",
      "Entity 'http://edamontology.org/data_1012' - Label 'Enzyme name'\n",
      "Entity 'http://edamontology.org/data_1013' - Label 'Restriction enzyme name'\n",
      "Entity 'http://edamontology.org/data_1014' - Label 'Sequence position specification'\n",
      "Entity 'http://edamontology.org/data_1015' - Label 'Sequence feature ID'\n",
      "Entity 'http://edamontology.org/data_1016' - Label 'Sequence position'\n",
      "Entity 'http://edamontology.org/data_1017' - Label 'Sequence range'\n",
      "Entity 'http://edamontology.org/data_1018' - Label 'Nucleic acid feature identifier'\n",
      "Entity 'http://edamontology.org/data_1019' - Label 'Protein feature identifier'\n",
      "Entity 'http://edamontology.org/data_1020' - Label 'Sequence feature key'\n",
      "Entity 'http://edamontology.org/data_1021' - Label 'Sequence feature qualifier'\n",
      "Entity 'http://edamontology.org/data_1022' - Label 'Sequence feature label'\n",
      "Entity 'http://edamontology.org/data_1023' - Label 'EMBOSS Uniform Feature Object'\n",
      "Entity 'http://edamontology.org/data_1024' - Label 'Codon name'\n",
      "Entity 'http://edamontology.org/data_1025' - Label 'Gene identifier'\n",
      "Entity 'http://edamontology.org/data_1026' - Label 'Gene symbol'\n",
      "Entity 'http://edamontology.org/data_1027' - Label 'Gene ID (NCBI)'\n",
      "Entity 'http://edamontology.org/data_1028' - Label 'Gene identifier (NCBI RefSeq)'\n",
      "Entity 'http://edamontology.org/data_1029' - Label 'Gene identifier (NCBI UniGene)'\n",
      "Entity 'http://edamontology.org/data_1030' - Label 'Gene identifier (Entrez)'\n",
      "Entity 'http://edamontology.org/data_1031' - Label 'Gene ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1032' - Label 'Gene ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1033' - Label 'Ensembl gene ID'\n",
      "Entity 'http://edamontology.org/data_1034' - Label 'Gene ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1035' - Label 'Gene ID (GeneDB)'\n",
      "Entity 'http://edamontology.org/data_1036' - Label 'TIGR identifier'\n",
      "Entity 'http://edamontology.org/data_1037' - Label 'TAIR accession (gene)'\n",
      "Entity 'http://edamontology.org/data_1038' - Label 'Protein domain ID'\n",
      "Entity 'http://edamontology.org/data_1039' - Label 'SCOP domain identifier'\n",
      "Entity 'http://edamontology.org/data_1040' - Label 'CATH domain ID'\n",
      "Entity 'http://edamontology.org/data_1041' - Label 'SCOP concise classification string (sccs)'\n",
      "Entity 'http://edamontology.org/data_1042' - Label 'SCOP sunid'\n",
      "Entity 'http://edamontology.org/data_1043' - Label 'CATH node ID'\n",
      "Entity 'http://edamontology.org/data_1044' - Label 'Kingdom name'\n",
      "Entity 'http://edamontology.org/data_1045' - Label 'Species name'\n",
      "Entity 'http://edamontology.org/data_1046' - Label 'Strain name'\n",
      "Entity 'http://edamontology.org/data_1047' - Label 'URI'\n",
      "Entity 'http://edamontology.org/data_1048' - Label 'Database ID'\n",
      "Entity 'http://edamontology.org/data_1049' - Label 'Directory name'\n",
      "Entity 'http://edamontology.org/data_1050' - Label 'File name'\n",
      "Entity 'http://edamontology.org/data_1051' - Label 'Ontology name'\n",
      "Entity 'http://edamontology.org/data_1052' - Label 'URL'\n",
      "Entity 'http://edamontology.org/data_1053' - Label 'URN'\n",
      "Entity 'http://edamontology.org/data_1055' - Label 'LSID'\n",
      "Entity 'http://edamontology.org/data_1056' - Label 'Database name'\n",
      "Entity 'http://edamontology.org/data_1057' - Label 'Sequence database name'\n",
      "Entity 'http://edamontology.org/data_1058' - Label 'Enumerated file name'\n",
      "Entity 'http://edamontology.org/data_1059' - Label 'File name extension'\n",
      "Entity 'http://edamontology.org/data_1060' - Label 'File base name'\n",
      "Entity 'http://edamontology.org/data_1061' - Label 'QSAR descriptor name'\n",
      "Entity 'http://edamontology.org/data_1062' - Label 'Database entry identifier'\n",
      "Entity 'http://edamontology.org/data_1063' - Label 'Sequence identifier'\n",
      "Entity 'http://edamontology.org/data_1064' - Label 'Sequence set ID'\n",
      "Entity 'http://edamontology.org/data_1065' - Label 'Sequence signature identifier'\n",
      "Entity 'http://edamontology.org/data_1066' - Label 'Sequence alignment ID'\n",
      "Entity 'http://edamontology.org/data_1067' - Label 'Phylogenetic distance matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1068' - Label 'Phylogenetic tree ID'\n",
      "Entity 'http://edamontology.org/data_1069' - Label 'Comparison matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1070' - Label 'Structure ID'\n",
      "Entity 'http://edamontology.org/data_1071' - Label 'Structural (3D) profile ID'\n",
      "Entity 'http://edamontology.org/data_1072' - Label 'Structure alignment ID'\n",
      "Entity 'http://edamontology.org/data_1073' - Label 'Amino acid index ID'\n",
      "Entity 'http://edamontology.org/data_1074' - Label 'Protein interaction ID'\n",
      "Entity 'http://edamontology.org/data_1075' - Label 'Protein family identifier'\n",
      "Entity 'http://edamontology.org/data_1076' - Label 'Codon usage table name'\n",
      "Entity 'http://edamontology.org/data_1077' - Label 'Transcription factor identifier'\n",
      "Entity 'http://edamontology.org/data_1078' - Label 'Experiment annotation ID'\n",
      "Entity 'http://edamontology.org/data_1079' - Label 'Electron microscopy model ID'\n",
      "Entity 'http://edamontology.org/data_1080' - Label 'Gene expression report ID'\n",
      "Entity 'http://edamontology.org/data_1081' - Label 'Genotype and phenotype annotation ID'\n",
      "Entity 'http://edamontology.org/data_1082' - Label 'Pathway or network identifier'\n",
      "Entity 'http://edamontology.org/data_1083' - Label 'Workflow ID'\n",
      "Entity 'http://edamontology.org/data_1084' - Label 'Data resource definition ID'\n",
      "Entity 'http://edamontology.org/data_1085' - Label 'Biological model ID'\n",
      "Entity 'http://edamontology.org/data_1086' - Label 'Compound identifier'\n",
      "Entity 'http://edamontology.org/data_1087' - Label 'Ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_1088' - Label 'Article ID'\n",
      "Entity 'http://edamontology.org/data_1089' - Label 'FlyBase ID'\n",
      "Entity 'http://edamontology.org/data_1091' - Label 'WormBase name'\n",
      "Entity 'http://edamontology.org/data_1092' - Label 'WormBase class'\n",
      "Entity 'http://edamontology.org/data_1093' - Label 'Sequence accession'\n",
      "Entity 'http://edamontology.org/data_1094' - Label 'Sequence type'\n",
      "Entity 'http://edamontology.org/data_1095' - Label 'EMBOSS Uniform Sequence Address'\n",
      "Entity 'http://edamontology.org/data_1096' - Label 'Sequence accession (protein)'\n",
      "Entity 'http://edamontology.org/data_1097' - Label 'Sequence accession (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1098' - Label 'RefSeq accession'\n",
      "Entity 'http://edamontology.org/data_1099' - Label 'UniProt accession (extended)'\n",
      "Entity 'http://edamontology.org/data_1100' - Label 'PIR identifier'\n",
      "Entity 'http://edamontology.org/data_1101' - Label 'TREMBL accession'\n",
      "Entity 'http://edamontology.org/data_1102' - Label 'Gramene primary identifier'\n",
      "Entity 'http://edamontology.org/data_1103' - Label 'EMBL/GenBank/DDBJ ID'\n",
      "Entity 'http://edamontology.org/data_1104' - Label 'Sequence cluster ID (UniGene)'\n",
      "Entity 'http://edamontology.org/data_1105' - Label 'dbEST accession'\n",
      "Entity 'http://edamontology.org/data_1106' - Label 'dbSNP ID'\n",
      "Entity 'http://edamontology.org/data_1110' - Label 'EMBOSS sequence type'\n",
      "Entity 'http://edamontology.org/data_1111' - Label 'EMBOSS listfile'\n",
      "Entity 'http://edamontology.org/data_1112' - Label 'Sequence cluster ID'\n",
      "Entity 'http://edamontology.org/data_1113' - Label 'Sequence cluster ID (COG)'\n",
      "Entity 'http://edamontology.org/data_1114' - Label 'Sequence motif identifier'\n",
      "Entity 'http://edamontology.org/data_1115' - Label 'Sequence profile ID'\n",
      "Entity 'http://edamontology.org/data_1116' - Label 'ELM ID'\n",
      "Entity 'http://edamontology.org/data_1117' - Label 'Prosite accession number'\n",
      "Entity 'http://edamontology.org/data_1118' - Label 'HMMER hidden Markov model ID'\n",
      "Entity 'http://edamontology.org/data_1119' - Label 'JASPAR profile ID'\n",
      "Entity 'http://edamontology.org/data_1120' - Label 'Sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1121' - Label 'BLAST sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1122' - Label 'Phylogenetic tree type'\n",
      "Entity 'http://edamontology.org/data_1123' - Label 'TreeBASE study accession number'\n",
      "Entity 'http://edamontology.org/data_1124' - Label 'TreeFam accession number'\n",
      "Entity 'http://edamontology.org/data_1125' - Label 'Comparison matrix type'\n",
      "Entity 'http://edamontology.org/data_1126' - Label 'Comparison matrix name'\n",
      "Entity 'http://edamontology.org/data_1127' - Label 'PDB ID'\n",
      "Entity 'http://edamontology.org/data_1128' - Label 'AAindex ID'\n",
      "Entity 'http://edamontology.org/data_1129' - Label 'BIND accession number'\n",
      "Entity 'http://edamontology.org/data_1130' - Label 'IntAct accession number'\n",
      "Entity 'http://edamontology.org/data_1131' - Label 'Protein family name'\n",
      "Entity 'http://edamontology.org/data_1132' - Label 'InterPro entry name'\n",
      "Entity 'http://edamontology.org/data_1133' - Label 'InterPro accession'\n",
      "Entity 'http://edamontology.org/data_1134' - Label 'InterPro secondary accession'\n",
      "Entity 'http://edamontology.org/data_1135' - Label 'Gene3D ID'\n",
      "Entity 'http://edamontology.org/data_1136' - Label 'PIRSF ID'\n",
      "Entity 'http://edamontology.org/data_1137' - Label 'PRINTS code'\n",
      "Entity 'http://edamontology.org/data_1138' - Label 'Pfam accession number'\n",
      "Entity 'http://edamontology.org/data_1139' - Label 'SMART accession number'\n",
      "Entity 'http://edamontology.org/data_1140' - Label 'Superfamily hidden Markov model number'\n",
      "Entity 'http://edamontology.org/data_1141' - Label 'TIGRFam ID'\n",
      "Entity 'http://edamontology.org/data_1142' - Label 'ProDom accession number'\n",
      "Entity 'http://edamontology.org/data_1143' - Label 'TRANSFAC accession number'\n",
      "Entity 'http://edamontology.org/data_1144' - Label 'ArrayExpress accession number'\n",
      "Entity 'http://edamontology.org/data_1145' - Label 'PRIDE experiment accession number'\n",
      "Entity 'http://edamontology.org/data_1146' - Label 'EMDB ID'\n",
      "Entity 'http://edamontology.org/data_1147' - Label 'GEO accession number'\n",
      "Entity 'http://edamontology.org/data_1148' - Label 'GermOnline ID'\n",
      "Entity 'http://edamontology.org/data_1149' - Label 'EMAGE ID'\n",
      "Entity 'http://edamontology.org/data_1150' - Label 'Disease ID'\n",
      "Entity 'http://edamontology.org/data_1151' - Label 'HGVbase ID'\n",
      "Entity 'http://edamontology.org/data_1152' - Label 'HIVDB identifier'\n",
      "Entity 'http://edamontology.org/data_1153' - Label 'OMIM ID'\n",
      "Entity 'http://edamontology.org/data_1154' - Label 'KEGG object identifier'\n",
      "Entity 'http://edamontology.org/data_1155' - Label 'Pathway ID (reactome)'\n",
      "Entity 'http://edamontology.org/data_1156' - Label 'Pathway ID (aMAZE)'\n",
      "Entity 'http://edamontology.org/data_1157' - Label 'Pathway ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_1158' - Label 'Pathway ID (INOH)'\n",
      "Entity 'http://edamontology.org/data_1159' - Label 'Pathway ID (PATIKA)'\n",
      "Entity 'http://edamontology.org/data_1160' - Label 'Pathway ID (CPDB)'\n",
      "Entity 'http://edamontology.org/data_1161' - Label 'Pathway ID (Panther)'\n",
      "Entity 'http://edamontology.org/data_1162' - Label 'MIRIAM identifier'\n",
      "Entity 'http://edamontology.org/data_1163' - Label 'MIRIAM data type name'\n",
      "Entity 'http://edamontology.org/data_1164' - Label 'MIRIAM URI'\n",
      "Entity 'http://edamontology.org/data_1165' - Label 'MIRIAM data type primary name'\n",
      "Entity 'http://edamontology.org/data_1166' - Label 'MIRIAM data type synonymous name'\n",
      "Entity 'http://edamontology.org/data_1167' - Label 'Taverna workflow ID'\n",
      "Entity 'http://edamontology.org/data_1170' - Label 'Biological model name'\n",
      "Entity 'http://edamontology.org/data_1171' - Label 'BioModel ID'\n",
      "Entity 'http://edamontology.org/data_1172' - Label 'PubChem CID'\n",
      "Entity 'http://edamontology.org/data_1173' - Label 'ChemSpider ID'\n",
      "Entity 'http://edamontology.org/data_1174' - Label 'ChEBI ID'\n",
      "Entity 'http://edamontology.org/data_1175' - Label 'BioPax concept ID'\n",
      "Entity 'http://edamontology.org/data_1176' - Label 'GO concept ID'\n",
      "Entity 'http://edamontology.org/data_1177' - Label 'MeSH concept ID'\n",
      "Entity 'http://edamontology.org/data_1178' - Label 'HGNC concept ID'\n",
      "Entity 'http://edamontology.org/data_1179' - Label 'NCBI taxonomy ID'\n",
      "Entity 'http://edamontology.org/data_1180' - Label 'Plant Ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_1181' - Label 'UMLS concept ID'\n",
      "Entity 'http://edamontology.org/data_1182' - Label 'FMA concept ID'\n",
      "Entity 'http://edamontology.org/data_1183' - Label 'EMAP concept ID'\n",
      "Entity 'http://edamontology.org/data_1184' - Label 'ChEBI concept ID'\n",
      "Entity 'http://edamontology.org/data_1185' - Label 'MGED concept ID'\n",
      "Entity 'http://edamontology.org/data_1186' - Label 'myGrid concept ID'\n",
      "Entity 'http://edamontology.org/data_1187' - Label 'PubMed ID'\n",
      "Entity 'http://edamontology.org/data_1188' - Label 'DOI'\n",
      "Entity 'http://edamontology.org/data_1189' - Label 'Medline UI'\n",
      "Entity 'http://edamontology.org/data_1190' - Label 'Tool name'\n",
      "Entity 'http://edamontology.org/data_1191' - Label 'Tool name (signature)'\n",
      "Entity 'http://edamontology.org/data_1192' - Label 'Tool name (BLAST)'\n",
      "Entity 'http://edamontology.org/data_1193' - Label 'Tool name (FASTA)'\n",
      "Entity 'http://edamontology.org/data_1194' - Label 'Tool name (EMBOSS)'\n",
      "Entity 'http://edamontology.org/data_1195' - Label 'Tool name (EMBASSY package)'\n",
      "Entity 'http://edamontology.org/data_1201' - Label 'QSAR descriptor (constitutional)'\n",
      "Entity 'http://edamontology.org/data_1202' - Label 'QSAR descriptor (electronic)'\n",
      "Entity 'http://edamontology.org/data_1203' - Label 'QSAR descriptor (geometrical)'\n",
      "Entity 'http://edamontology.org/data_1204' - Label 'QSAR descriptor (topological)'\n",
      "Entity 'http://edamontology.org/data_1205' - Label 'QSAR descriptor (molecular)'\n",
      "Entity 'http://edamontology.org/data_1233' - Label 'Sequence set (protein)'\n",
      "Entity 'http://edamontology.org/data_1234' - Label 'Sequence set (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1235' - Label 'Sequence cluster'\n",
      "Entity 'http://edamontology.org/data_1236' - Label 'Psiblast checkpoint file'\n",
      "Entity 'http://edamontology.org/data_1237' - Label 'HMMER synthetic sequences set'\n",
      "Entity 'http://edamontology.org/data_1238' - Label 'Proteolytic digest'\n",
      "Entity 'http://edamontology.org/data_1239' - Label 'Restriction digest'\n",
      "Entity 'http://edamontology.org/data_1240' - Label 'PCR primers'\n",
      "Entity 'http://edamontology.org/data_1241' - Label 'vectorstrip cloning vector definition file'\n",
      "Entity 'http://edamontology.org/data_1242' - Label 'Primer3 internal oligo mishybridizing library'\n",
      "Entity 'http://edamontology.org/data_1243' - Label 'Primer3 mispriming library file'\n",
      "Entity 'http://edamontology.org/data_1244' - Label 'primersearch primer pairs sequence record'\n",
      "Entity 'http://edamontology.org/data_1245' - Label 'Sequence cluster (protein)'\n",
      "Entity 'http://edamontology.org/data_1246' - Label 'Sequence cluster (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1249' - Label 'Sequence length'\n",
      "Entity 'http://edamontology.org/data_1250' - Label 'Word size'\n",
      "Entity 'http://edamontology.org/data_1251' - Label 'Window size'\n",
      "Entity 'http://edamontology.org/data_1252' - Label 'Sequence length range'\n",
      "Entity 'http://edamontology.org/data_1253' - Label 'Sequence information report'\n",
      "Entity 'http://edamontology.org/data_1254' - Label 'Sequence property'\n",
      "Entity 'http://edamontology.org/data_1255' - Label 'Sequence features'\n",
      "Entity 'http://edamontology.org/data_1256' - Label 'Sequence features (comparative)'\n",
      "Entity 'http://edamontology.org/data_1257' - Label 'Sequence property (protein)'\n",
      "Entity 'http://edamontology.org/data_1258' - Label 'Sequence property (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1259' - Label 'Sequence complexity report'\n",
      "Entity 'http://edamontology.org/data_1260' - Label 'Sequence ambiguity report'\n",
      "Entity 'http://edamontology.org/data_1261' - Label 'Sequence composition report'\n",
      "Entity 'http://edamontology.org/data_1262' - Label 'Peptide molecular weight hits'\n",
      "Entity 'http://edamontology.org/data_1263' - Label 'Base position variability plot'\n",
      "Entity 'http://edamontology.org/data_1264' - Label 'Sequence composition table'\n",
      "Entity 'http://edamontology.org/data_1265' - Label 'Base frequencies table'\n",
      "Entity 'http://edamontology.org/data_1266' - Label 'Base word frequencies table'\n",
      "Entity 'http://edamontology.org/data_1267' - Label 'Amino acid frequencies table'\n",
      "Entity 'http://edamontology.org/data_1268' - Label 'Amino acid word frequencies table'\n",
      "Entity 'http://edamontology.org/data_1269' - Label 'DAS sequence feature annotation'\n",
      "Entity 'http://edamontology.org/data_1270' - Label 'Feature table'\n",
      "Entity 'http://edamontology.org/data_1274' - Label 'Map'\n",
      "Entity 'http://edamontology.org/data_1276' - Label 'Nucleic acid features'\n",
      "Entity 'http://edamontology.org/data_1277' - Label 'Protein features'\n",
      "Entity 'http://edamontology.org/data_1278' - Label 'Genetic map'\n",
      "Entity 'http://edamontology.org/data_1279' - Label 'Sequence map'\n",
      "Entity 'http://edamontology.org/data_1280' - Label 'Physical map'\n",
      "Entity 'http://edamontology.org/data_1281' - Label 'Sequence signature map'\n",
      "Entity 'http://edamontology.org/data_1283' - Label 'Cytogenetic map'\n",
      "Entity 'http://edamontology.org/data_1284' - Label 'DNA transduction map'\n",
      "Entity 'http://edamontology.org/data_1285' - Label 'Gene map'\n",
      "Entity 'http://edamontology.org/data_1286' - Label 'Plasmid map'\n",
      "Entity 'http://edamontology.org/data_1288' - Label 'Genome map'\n",
      "Entity 'http://edamontology.org/data_1289' - Label 'Restriction map'\n",
      "Entity 'http://edamontology.org/data_1290' - Label 'InterPro compact match image'\n",
      "Entity 'http://edamontology.org/data_1291' - Label 'InterPro detailed match image'\n",
      "Entity 'http://edamontology.org/data_1292' - Label 'InterPro architecture image'\n",
      "Entity 'http://edamontology.org/data_1293' - Label 'SMART protein schematic'\n",
      "Entity 'http://edamontology.org/data_1294' - Label 'GlobPlot domain image'\n",
      "Entity 'http://edamontology.org/data_1298' - Label 'Sequence motif matches'\n",
      "Entity 'http://edamontology.org/data_1299' - Label 'Sequence features (repeats)'\n",
      "Entity 'http://edamontology.org/data_1300' - Label 'Gene and transcript structure (report)'\n",
      "Entity 'http://edamontology.org/data_1301' - Label 'Mobile genetic elements'\n",
      "Entity 'http://edamontology.org/data_1303' - Label 'Nucleic acid features (quadruplexes)'\n",
      "Entity 'http://edamontology.org/data_1306' - Label 'Nucleosome exclusion sequences'\n",
      "Entity 'http://edamontology.org/data_1309' - Label 'Gene features (exonic splicing enhancer)'\n",
      "Entity 'http://edamontology.org/data_1310' - Label 'Nucleic acid features (microRNA)'\n",
      "Entity 'http://edamontology.org/data_1313' - Label 'Coding region'\n",
      "Entity 'http://edamontology.org/data_1314' - Label 'Gene features (SECIS element)'\n",
      "Entity 'http://edamontology.org/data_1315' - Label 'Transcription factor binding sites'\n",
      "Entity 'http://edamontology.org/data_1321' - Label 'Protein features (sites)'\n",
      "Entity 'http://edamontology.org/data_1322' - Label 'Protein features report (signal peptides)'\n",
      "Entity 'http://edamontology.org/data_1323' - Label 'Protein features report (cleavage sites)'\n",
      "Entity 'http://edamontology.org/data_1324' - Label 'Protein features (post-translation modifications)'\n",
      "Entity 'http://edamontology.org/data_1325' - Label 'Protein features report (active sites)'\n",
      "Entity 'http://edamontology.org/data_1326' - Label 'Protein features report (binding sites)'\n",
      "Entity 'http://edamontology.org/data_1327' - Label 'Protein features (epitopes)'\n",
      "Entity 'http://edamontology.org/data_1328' - Label 'Protein features report (nucleic acid binding sites)'\n",
      "Entity 'http://edamontology.org/data_1329' - Label 'MHC Class I epitopes report'\n",
      "Entity 'http://edamontology.org/data_1330' - Label 'MHC Class II epitopes report'\n",
      "Entity 'http://edamontology.org/data_1331' - Label 'Protein features (PEST sites)'\n",
      "Entity 'http://edamontology.org/data_1338' - Label 'Sequence database hits scores list'\n",
      "Entity 'http://edamontology.org/data_1339' - Label 'Sequence database hits alignments list'\n",
      "Entity 'http://edamontology.org/data_1340' - Label 'Sequence database hits evaluation data'\n",
      "Entity 'http://edamontology.org/data_1344' - Label 'MEME motif alphabet'\n",
      "Entity 'http://edamontology.org/data_1345' - Label 'MEME background frequencies file'\n",
      "Entity 'http://edamontology.org/data_1346' - Label 'MEME motifs directive file'\n",
      "Entity 'http://edamontology.org/data_1347' - Label 'Dirichlet distribution'\n",
      "Entity 'http://edamontology.org/data_1348' - Label 'HMM emission and transition counts'\n",
      "Entity 'http://edamontology.org/data_1352' - Label 'Regular expression'\n",
      "Entity 'http://edamontology.org/data_1353' - Label 'Sequence motif'\n",
      "Entity 'http://edamontology.org/data_1354' - Label 'Sequence profile'\n",
      "Entity 'http://edamontology.org/data_1355' - Label 'Protein signature'\n",
      "Entity 'http://edamontology.org/data_1358' - Label 'Prosite nucleotide pattern'\n",
      "Entity 'http://edamontology.org/data_1359' - Label 'Prosite protein pattern'\n",
      "Entity 'http://edamontology.org/data_1361' - Label 'Position frequency matrix'\n",
      "Entity 'http://edamontology.org/data_1362' - Label 'Position weight matrix'\n",
      "Entity 'http://edamontology.org/data_1363' - Label 'Information content matrix'\n",
      "Entity 'http://edamontology.org/data_1364' - Label 'Hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_1365' - Label 'Fingerprint'\n",
      "Entity 'http://edamontology.org/data_1368' - Label 'Domainatrix signature'\n",
      "Entity 'http://edamontology.org/data_1371' - Label 'HMMER NULL hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_1372' - Label 'Protein family signature'\n",
      "Entity 'http://edamontology.org/data_1373' - Label 'Protein domain signature'\n",
      "Entity 'http://edamontology.org/data_1374' - Label 'Protein region signature'\n",
      "Entity 'http://edamontology.org/data_1375' - Label 'Protein repeat signature'\n",
      "Entity 'http://edamontology.org/data_1376' - Label 'Protein site signature'\n",
      "Entity 'http://edamontology.org/data_1377' - Label 'Protein conserved site signature'\n",
      "Entity 'http://edamontology.org/data_1378' - Label 'Protein active site signature'\n",
      "Entity 'http://edamontology.org/data_1379' - Label 'Protein binding site signature'\n",
      "Entity 'http://edamontology.org/data_1380' - Label 'Protein post-translational modification signature'\n",
      "Entity 'http://edamontology.org/data_1381' - Label 'Pair sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1382' - Label 'Sequence alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1383' - Label 'Nucleic acid sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1384' - Label 'Protein sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1385' - Label 'Hybrid sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1386' - Label 'Sequence alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1387' - Label 'Sequence alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1388' - Label 'Hybrid sequence alignment (pair)'\n",
      "Entity 'http://edamontology.org/data_1389' - Label 'Multiple nucleotide sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1390' - Label 'Multiple protein sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1394' - Label 'Alignment score or penalty'\n",
      "Entity 'http://edamontology.org/data_1395' - Label 'Score end gaps control'\n",
      "Entity 'http://edamontology.org/data_1396' - Label 'Aligned sequence order'\n",
      "Entity 'http://edamontology.org/data_1397' - Label 'Gap opening penalty'\n",
      "Entity 'http://edamontology.org/data_1398' - Label 'Gap extension penalty'\n",
      "Entity 'http://edamontology.org/data_1399' - Label 'Gap separation penalty'\n",
      "Entity 'http://edamontology.org/data_1400' - Label 'Terminal gap penalty'\n",
      "Entity 'http://edamontology.org/data_1401' - Label 'Match reward score'\n",
      "Entity 'http://edamontology.org/data_1402' - Label 'Mismatch penalty score'\n",
      "Entity 'http://edamontology.org/data_1403' - Label 'Drop off score'\n",
      "Entity 'http://edamontology.org/data_1404' - Label 'Gap opening penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1405' - Label 'Gap opening penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1406' - Label 'Gap extension penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1407' - Label 'Gap extension penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1408' - Label 'Gap separation penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1409' - Label 'Gap separation penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1410' - Label 'Terminal gap opening penalty'\n",
      "Entity 'http://edamontology.org/data_1411' - Label 'Terminal gap extension penalty'\n",
      "Entity 'http://edamontology.org/data_1412' - Label 'Sequence identity'\n",
      "Entity 'http://edamontology.org/data_1413' - Label 'Sequence similarity'\n",
      "Entity 'http://edamontology.org/data_1414' - Label 'Sequence alignment metadata (quality report)'\n",
      "Entity 'http://edamontology.org/data_1415' - Label 'Sequence alignment report (site conservation)'\n",
      "Entity 'http://edamontology.org/data_1416' - Label 'Sequence alignment report (site correlation)'\n",
      "Entity 'http://edamontology.org/data_1417' - Label 'Sequence-profile alignment (Domainatrix signature)'\n",
      "Entity 'http://edamontology.org/data_1418' - Label 'Sequence-profile alignment (HMM)'\n",
      "Entity 'http://edamontology.org/data_1420' - Label 'Sequence-profile alignment (fingerprint)'\n",
      "Entity 'http://edamontology.org/data_1426' - Label 'Phylogenetic continuous quantitative data'\n",
      "Entity 'http://edamontology.org/data_1427' - Label 'Phylogenetic discrete data'\n",
      "Entity 'http://edamontology.org/data_1428' - Label 'Phylogenetic character cliques'\n",
      "Entity 'http://edamontology.org/data_1429' - Label 'Phylogenetic invariants'\n",
      "Entity 'http://edamontology.org/data_1438' - Label 'Phylogenetic report'\n",
      "Entity 'http://edamontology.org/data_1439' - Label 'DNA substitution model'\n",
      "Entity 'http://edamontology.org/data_1440' - Label 'Phylogenetic tree report (tree shape)'\n",
      "Entity 'http://edamontology.org/data_1441' - Label 'Phylogenetic tree report (tree evaluation)'\n",
      "Entity 'http://edamontology.org/data_1442' - Label 'Phylogenetic tree distances'\n",
      "Entity 'http://edamontology.org/data_1443' - Label 'Phylogenetic tree report (tree stratigraphic)'\n",
      "Entity 'http://edamontology.org/data_1444' - Label 'Phylogenetic character contrasts'\n",
      "Entity 'http://edamontology.org/data_1446' - Label 'Comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1447' - Label 'Comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1448' - Label 'Comparison matrix (nucleotide)'\n",
      "Entity 'http://edamontology.org/data_1449' - Label 'Comparison matrix (amino acid)'\n",
      "Entity 'http://edamontology.org/data_1450' - Label 'Nucleotide comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1451' - Label 'Nucleotide comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1452' - Label 'Amino acid comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1453' - Label 'Amino acid comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1459' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/data_1460' - Label 'Protein structure'\n",
      "Entity 'http://edamontology.org/data_1461' - Label 'Protein-ligand complex'\n",
      "Entity 'http://edamontology.org/data_1462' - Label 'Carbohydrate structure'\n",
      "Entity 'http://edamontology.org/data_1463' - Label 'Small molecule structure'\n",
      "Entity 'http://edamontology.org/data_1464' - Label 'DNA structure'\n",
      "Entity 'http://edamontology.org/data_1465' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/data_1466' - Label 'tRNA structure'\n",
      "Entity 'http://edamontology.org/data_1467' - Label 'Protein chain'\n",
      "Entity 'http://edamontology.org/data_1468' - Label 'Protein domain'\n",
      "Entity 'http://edamontology.org/data_1469' - Label 'Protein structure (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1470' - Label 'C-alpha trace'\n",
      "Entity 'http://edamontology.org/data_1471' - Label 'Protein chain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1472' - Label 'Protein chain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1473' - Label 'Protein domain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1474' - Label 'Protein domain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1479' - Label 'Structure alignment (pair)'\n",
      "Entity 'http://edamontology.org/data_1480' - Label 'Structure alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1481' - Label 'Protein structure alignment'\n",
      "Entity 'http://edamontology.org/data_1482' - Label 'Nucleic acid structure alignment'\n",
      "Entity 'http://edamontology.org/data_1483' - Label 'Structure alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1484' - Label 'Multiple protein tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1485' - Label 'Structure alignment (protein all atoms)'\n",
      "Entity 'http://edamontology.org/data_1486' - Label 'Structure alignment (protein C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1487' - Label 'Pairwise protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1488' - Label 'Pairwise protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1489' - Label 'Multiple protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1490' - Label 'Multiple protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1491' - Label 'Structure alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1492' - Label 'Multiple nucleic acid tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1493' - Label 'RNA structure alignment'\n",
      "Entity 'http://edamontology.org/data_1494' - Label 'Structural transformation matrix'\n",
      "Entity 'http://edamontology.org/data_1495' - Label 'DaliLite hit table'\n",
      "Entity 'http://edamontology.org/data_1496' - Label 'Molecular similarity score'\n",
      "Entity 'http://edamontology.org/data_1497' - Label 'Root-mean-square deviation'\n",
      "Entity 'http://edamontology.org/data_1498' - Label 'Tanimoto similarity score'\n",
      "Entity 'http://edamontology.org/data_1499' - Label '3D-1D scoring matrix'\n",
      "Entity 'http://edamontology.org/data_1501' - Label 'Amino acid index'\n",
      "Entity 'http://edamontology.org/data_1502' - Label 'Amino acid index (chemical classes)'\n",
      "Entity 'http://edamontology.org/data_1503' - Label 'Amino acid pair-wise contact potentials'\n",
      "Entity 'http://edamontology.org/data_1505' - Label 'Amino acid index (molecular weight)'\n",
      "Entity 'http://edamontology.org/data_1506' - Label 'Amino acid index (hydropathy)'\n",
      "Entity 'http://edamontology.org/data_1507' - Label 'Amino acid index (White-Wimley data)'\n",
      "Entity 'http://edamontology.org/data_1508' - Label 'Amino acid index (van der Waals radii)'\n",
      "Entity 'http://edamontology.org/data_1509' - Label 'Enzyme report'\n",
      "Entity 'http://edamontology.org/data_1517' - Label 'Restriction enzyme report'\n",
      "Entity 'http://edamontology.org/data_1519' - Label 'Peptide molecular weights'\n",
      "Entity 'http://edamontology.org/data_1520' - Label 'Peptide hydrophobic moment'\n",
      "Entity 'http://edamontology.org/data_1521' - Label 'Protein aliphatic index'\n",
      "Entity 'http://edamontology.org/data_1522' - Label 'Protein sequence hydropathy plot'\n",
      "Entity 'http://edamontology.org/data_1523' - Label 'Protein charge plot'\n",
      "Entity 'http://edamontology.org/data_1524' - Label 'Protein solubility'\n",
      "Entity 'http://edamontology.org/data_1525' - Label 'Protein crystallizability'\n",
      "Entity 'http://edamontology.org/data_1526' - Label 'Protein globularity'\n",
      "Entity 'http://edamontology.org/data_1527' - Label 'Protein titration curve'\n",
      "Entity 'http://edamontology.org/data_1528' - Label 'Protein isoelectric point'\n",
      "Entity 'http://edamontology.org/data_1529' - Label 'Protein pKa value'\n",
      "Entity 'http://edamontology.org/data_1530' - Label 'Protein hydrogen exchange rate'\n",
      "Entity 'http://edamontology.org/data_1531' - Label 'Protein extinction coefficient'\n",
      "Entity 'http://edamontology.org/data_1532' - Label 'Protein optical density'\n",
      "Entity 'http://edamontology.org/data_1533' - Label 'Protein subcellular localisation'\n",
      "Entity 'http://edamontology.org/data_1534' - Label 'Peptide immunogenicity data'\n",
      "Entity 'http://edamontology.org/data_1536' - Label 'MHC peptide immunogenicity report'\n",
      "Entity 'http://edamontology.org/data_1537' - Label 'Protein structure report'\n",
      "Entity 'http://edamontology.org/data_1539' - Label 'Protein structural quality report'\n",
      "Entity 'http://edamontology.org/data_1540' - Label 'Protein non-covalent interactions report'\n",
      "Entity 'http://edamontology.org/data_1541' - Label 'Protein flexibility or motion report'\n",
      "Entity 'http://edamontology.org/data_1542' - Label 'Protein solvent accessibility'\n",
      "Entity 'http://edamontology.org/data_1543' - Label 'Protein surface report'\n",
      "Entity 'http://edamontology.org/data_1544' - Label 'Ramachandran plot'\n",
      "Entity 'http://edamontology.org/data_1545' - Label 'Protein dipole moment'\n",
      "Entity 'http://edamontology.org/data_1546' - Label 'Protein distance matrix'\n",
      "Entity 'http://edamontology.org/data_1547' - Label 'Protein contact map'\n",
      "Entity 'http://edamontology.org/data_1548' - Label 'Protein residue 3D cluster'\n",
      "Entity 'http://edamontology.org/data_1549' - Label 'Protein hydrogen bonds'\n",
      "Entity 'http://edamontology.org/data_1550' - Label 'Protein non-canonical interactions'\n",
      "Entity 'http://edamontology.org/data_1553' - Label 'CATH node'\n",
      "Entity 'http://edamontology.org/data_1554' - Label 'SCOP node'\n",
      "Entity 'http://edamontology.org/data_1555' - Label 'EMBASSY domain classification'\n",
      "Entity 'http://edamontology.org/data_1556' - Label 'CATH class'\n",
      "Entity 'http://edamontology.org/data_1557' - Label 'CATH architecture'\n",
      "Entity 'http://edamontology.org/data_1558' - Label 'CATH topology'\n",
      "Entity 'http://edamontology.org/data_1559' - Label 'CATH homologous superfamily'\n",
      "Entity 'http://edamontology.org/data_1560' - Label 'CATH structurally similar group'\n",
      "Entity 'http://edamontology.org/data_1561' - Label 'CATH functional category'\n",
      "Entity 'http://edamontology.org/data_1564' - Label 'Protein fold recognition report'\n",
      "Entity 'http://edamontology.org/data_1565' - Label 'Protein-protein interaction report'\n",
      "Entity 'http://edamontology.org/data_1566' - Label 'Protein-ligand interaction report'\n",
      "Entity 'http://edamontology.org/data_1567' - Label 'Protein-nucleic acid interactions report'\n",
      "Entity 'http://edamontology.org/data_1583' - Label 'Nucleic acid melting profile'\n",
      "Entity 'http://edamontology.org/data_1584' - Label 'Nucleic acid enthalpy'\n",
      "Entity 'http://edamontology.org/data_1585' - Label 'Nucleic acid entropy'\n",
      "Entity 'http://edamontology.org/data_1586' - Label 'Nucleic acid melting temperature'\n",
      "Entity 'http://edamontology.org/data_1587' - Label 'Nucleic acid stitch profile'\n",
      "Entity 'http://edamontology.org/data_1588' - Label 'DNA base pair stacking energies data'\n",
      "Entity 'http://edamontology.org/data_1589' - Label 'DNA base pair twist angle data'\n",
      "Entity 'http://edamontology.org/data_1590' - Label 'DNA base trimer roll angles data'\n",
      "Entity 'http://edamontology.org/data_1591' - Label 'Vienna RNA parameters'\n",
      "Entity 'http://edamontology.org/data_1592' - Label 'Vienna RNA structure constraints'\n",
      "Entity 'http://edamontology.org/data_1593' - Label 'Vienna RNA concentration data'\n",
      "Entity 'http://edamontology.org/data_1594' - Label 'Vienna RNA calculated energy'\n",
      "Entity 'http://edamontology.org/data_1595' - Label 'Base pairing probability matrix dotplot'\n",
      "Entity 'http://edamontology.org/data_1596' - Label 'Nucleic acid folding report'\n",
      "Entity 'http://edamontology.org/data_1597' - Label 'Codon usage table'\n",
      "Entity 'http://edamontology.org/data_1598' - Label 'Genetic code'\n",
      "Entity 'http://edamontology.org/data_1599' - Label 'Codon adaptation index'\n",
      "Entity 'http://edamontology.org/data_1600' - Label 'Codon usage bias plot'\n",
      "Entity 'http://edamontology.org/data_1601' - Label 'Nc statistic'\n",
      "Entity 'http://edamontology.org/data_1602' - Label 'Codon usage fraction difference'\n",
      "Entity 'http://edamontology.org/data_1621' - Label 'Pharmacogenomic test report'\n",
      "Entity 'http://edamontology.org/data_1622' - Label 'Disease report'\n",
      "Entity 'http://edamontology.org/data_1634' - Label 'Linkage disequilibrium (report)'\n",
      "Entity 'http://edamontology.org/data_1636' - Label 'Heat map'\n",
      "Entity 'http://edamontology.org/data_1642' - Label 'Affymetrix probe sets library file'\n",
      "Entity 'http://edamontology.org/data_1643' - Label 'Affymetrix probe sets information library file'\n",
      "Entity 'http://edamontology.org/data_1646' - Label 'Molecular weights standard fingerprint'\n",
      "Entity 'http://edamontology.org/data_1656' - Label 'Metabolic pathway report'\n",
      "Entity 'http://edamontology.org/data_1657' - Label 'Genetic information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1658' - Label 'Environmental information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1659' - Label 'Signal transduction pathway report'\n",
      "Entity 'http://edamontology.org/data_1660' - Label 'Cellular process pathways report'\n",
      "Entity 'http://edamontology.org/data_1661' - Label 'Disease pathway or network report'\n",
      "Entity 'http://edamontology.org/data_1662' - Label 'Drug structure relationship map'\n",
      "Entity 'http://edamontology.org/data_1663' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/data_1664' - Label 'MIRIAM datatype'\n",
      "Entity 'http://edamontology.org/data_1667' - Label 'E-value'\n",
      "Entity 'http://edamontology.org/data_1668' - Label 'Z-value'\n",
      "Entity 'http://edamontology.org/data_1669' - Label 'P-value'\n",
      "Entity 'http://edamontology.org/data_1670' - Label 'Database version information'\n",
      "Entity 'http://edamontology.org/data_1671' - Label 'Tool version information'\n",
      "Entity 'http://edamontology.org/data_1672' - Label 'CATH version information'\n",
      "Entity 'http://edamontology.org/data_1673' - Label 'Swiss-Prot to PDB mapping'\n",
      "Entity 'http://edamontology.org/data_1674' - Label 'Sequence database cross-references'\n",
      "Entity 'http://edamontology.org/data_1675' - Label 'Job status'\n",
      "Entity 'http://edamontology.org/data_1676' - Label 'Job ID'\n",
      "Entity 'http://edamontology.org/data_1677' - Label 'Job type'\n",
      "Entity 'http://edamontology.org/data_1678' - Label 'Tool log'\n",
      "Entity 'http://edamontology.org/data_1679' - Label 'DaliLite log file'\n",
      "Entity 'http://edamontology.org/data_1680' - Label 'STRIDE log file'\n",
      "Entity 'http://edamontology.org/data_1681' - Label 'NACCESS log file'\n",
      "Entity 'http://edamontology.org/data_1682' - Label 'EMBOSS wordfinder log file'\n",
      "Entity 'http://edamontology.org/data_1683' - Label 'EMBOSS domainatrix log file'\n",
      "Entity 'http://edamontology.org/data_1684' - Label 'EMBOSS sites log file'\n",
      "Entity 'http://edamontology.org/data_1685' - Label 'EMBOSS supermatcher error file'\n",
      "Entity 'http://edamontology.org/data_1686' - Label 'EMBOSS megamerger log file'\n",
      "Entity 'http://edamontology.org/data_1687' - Label 'EMBOSS whichdb log file'\n",
      "Entity 'http://edamontology.org/data_1688' - Label 'EMBOSS vectorstrip log file'\n",
      "Entity 'http://edamontology.org/data_1689' - Label 'Username'\n",
      "Entity 'http://edamontology.org/data_1690' - Label 'Password'\n",
      "Entity 'http://edamontology.org/data_1691' - Label 'Email address'\n",
      "Entity 'http://edamontology.org/data_1692' - Label 'Person name'\n",
      "Entity 'http://edamontology.org/data_1693' - Label 'Number of iterations'\n",
      "Entity 'http://edamontology.org/data_1694' - Label 'Number of output entities'\n",
      "Entity 'http://edamontology.org/data_1695' - Label 'Hit sort order'\n",
      "Entity 'http://edamontology.org/data_1696' - Label 'Drug report'\n",
      "Entity 'http://edamontology.org/data_1707' - Label 'Phylogenetic tree image'\n",
      "Entity 'http://edamontology.org/data_1708' - Label 'RNA secondary structure image'\n",
      "Entity 'http://edamontology.org/data_1709' - Label 'Protein secondary structure image'\n",
      "Entity 'http://edamontology.org/data_1710' - Label 'Structure image'\n",
      "Entity 'http://edamontology.org/data_1711' - Label 'Sequence alignment image'\n",
      "Entity 'http://edamontology.org/data_1712' - Label 'Chemical structure image'\n",
      "Entity 'http://edamontology.org/data_1713' - Label 'Fate map'\n",
      "Entity 'http://edamontology.org/data_1714' - Label 'Microarray spots image'\n",
      "Entity 'http://edamontology.org/data_1715' - Label 'BioPax term'\n",
      "Entity 'http://edamontology.org/data_1716' - Label 'GO'\n",
      "Entity 'http://edamontology.org/data_1717' - Label 'MeSH'\n",
      "Entity 'http://edamontology.org/data_1718' - Label 'HGNC'\n",
      "Entity 'http://edamontology.org/data_1719' - Label 'NCBI taxonomy vocabulary'\n",
      "Entity 'http://edamontology.org/data_1720' - Label 'Plant ontology term'\n",
      "Entity 'http://edamontology.org/data_1721' - Label 'UMLS'\n",
      "Entity 'http://edamontology.org/data_1722' - Label 'FMA'\n",
      "Entity 'http://edamontology.org/data_1723' - Label 'EMAP'\n",
      "Entity 'http://edamontology.org/data_1724' - Label 'ChEBI'\n",
      "Entity 'http://edamontology.org/data_1725' - Label 'MGED'\n",
      "Entity 'http://edamontology.org/data_1726' - Label 'myGrid'\n",
      "Entity 'http://edamontology.org/data_1727' - Label 'GO (biological process)'\n",
      "Entity 'http://edamontology.org/data_1728' - Label 'GO (molecular function)'\n",
      "Entity 'http://edamontology.org/data_1729' - Label 'GO (cellular component)'\n",
      "Entity 'http://edamontology.org/data_1730' - Label 'Ontology relation type'\n",
      "Entity 'http://edamontology.org/data_1731' - Label 'Ontology concept definition'\n",
      "Entity 'http://edamontology.org/data_1732' - Label 'Ontology concept comment'\n",
      "Entity 'http://edamontology.org/data_1733' - Label 'Ontology concept reference'\n",
      "Entity 'http://edamontology.org/data_1738' - Label 'doc2loc document information'\n",
      "Entity 'http://edamontology.org/data_1742' - Label 'PDB residue number'\n",
      "Entity 'http://edamontology.org/data_1743' - Label 'Atomic coordinate'\n",
      "Entity 'http://edamontology.org/data_1744' - Label 'Atomic x coordinate'\n",
      "Entity 'http://edamontology.org/data_1745' - Label 'Atomic y coordinate'\n",
      "Entity 'http://edamontology.org/data_1746' - Label 'Atomic z coordinate'\n",
      "Entity 'http://edamontology.org/data_1748' - Label 'PDB atom name'\n",
      "Entity 'http://edamontology.org/data_1755' - Label 'Protein atom'\n",
      "Entity 'http://edamontology.org/data_1756' - Label 'Protein residue'\n",
      "Entity 'http://edamontology.org/data_1757' - Label 'Atom name'\n",
      "Entity 'http://edamontology.org/data_1758' - Label 'PDB residue name'\n",
      "Entity 'http://edamontology.org/data_1759' - Label 'PDB model number'\n",
      "Entity 'http://edamontology.org/data_1762' - Label 'CATH domain report'\n",
      "Entity 'http://edamontology.org/data_1764' - Label 'CATH representative domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1765' - Label 'CATH representative domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1766' - Label 'CATH domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1767' - Label 'CATH domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1771' - Label 'Sequence version'\n",
      "Entity 'http://edamontology.org/data_1772' - Label 'Score'\n",
      "Entity 'http://edamontology.org/data_1776' - Label 'Protein report (function)'\n",
      "Entity 'http://edamontology.org/data_1783' - Label 'Gene name (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1784' - Label 'Gene name (CGD)'\n",
      "Entity 'http://edamontology.org/data_1785' - Label 'Gene name (dictyBase)'\n",
      "Entity 'http://edamontology.org/data_1786' - Label 'Gene name (EcoGene primary)'\n",
      "Entity 'http://edamontology.org/data_1787' - Label 'Gene name (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1788' - Label 'Gene name (SGD)'\n",
      "Entity 'http://edamontology.org/data_1789' - Label 'Gene name (TGD)'\n",
      "Entity 'http://edamontology.org/data_1790' - Label 'Gene name (CGSC)'\n",
      "Entity 'http://edamontology.org/data_1791' - Label 'Gene name (HGNC)'\n",
      "Entity 'http://edamontology.org/data_1792' - Label 'Gene name (MGD)'\n",
      "Entity 'http://edamontology.org/data_1793' - Label 'Gene name (Bacillus subtilis)'\n",
      "Entity 'http://edamontology.org/data_1794' - Label 'Gene ID (PlasmoDB)'\n",
      "Entity 'http://edamontology.org/data_1795' - Label 'Gene ID (EcoGene)'\n",
      "Entity 'http://edamontology.org/data_1796' - Label 'Gene ID (FlyBase)'\n",
      "Entity 'http://edamontology.org/data_1797' - Label 'Gene ID (GeneDB Glossina morsitans)'\n",
      "Entity 'http://edamontology.org/data_1798' - Label 'Gene ID (GeneDB Leishmania major)'\n",
      "Entity 'http://edamontology.org/data_1799' - Label 'Gene ID (GeneDB Plasmodium falciparum)'\n",
      "Entity 'http://edamontology.org/data_1800' - Label 'Gene ID (GeneDB Schizosaccharomyces pombe)'\n",
      "Entity 'http://edamontology.org/data_1801' - Label 'Gene ID (GeneDB Trypanosoma brucei)'\n",
      "Entity 'http://edamontology.org/data_1802' - Label 'Gene ID (Gramene)'\n",
      "Entity 'http://edamontology.org/data_1803' - Label 'Gene ID (Virginia microbial)'\n",
      "Entity 'http://edamontology.org/data_1804' - Label 'Gene ID (SGN)'\n",
      "Entity 'http://edamontology.org/data_1805' - Label 'Gene ID (WormBase)'\n",
      "Entity 'http://edamontology.org/data_1806' - Label 'Gene synonym'\n",
      "Entity 'http://edamontology.org/data_1807' - Label 'ORF name'\n",
      "Entity 'http://edamontology.org/data_1852' - Label 'Sequence assembly component'\n",
      "Entity 'http://edamontology.org/data_1853' - Label 'Chromosome annotation (aberration)'\n",
      "Entity 'http://edamontology.org/data_1855' - Label 'Clone ID'\n",
      "Entity 'http://edamontology.org/data_1856' - Label 'PDB insertion code'\n",
      "Entity 'http://edamontology.org/data_1857' - Label 'Atomic occupancy'\n",
      "Entity 'http://edamontology.org/data_1858' - Label 'Isotropic B factor'\n",
      "Entity 'http://edamontology.org/data_1859' - Label 'Deletion map'\n",
      "Entity 'http://edamontology.org/data_1860' - Label 'QTL map'\n",
      "Entity 'http://edamontology.org/data_1863' - Label 'Haplotype map'\n",
      "Entity 'http://edamontology.org/data_1864' - Label 'Map set data'\n",
      "Entity 'http://edamontology.org/data_1865' - Label 'Map feature'\n",
      "Entity 'http://edamontology.org/data_1866' - Label 'Map type'\n",
      "Entity 'http://edamontology.org/data_1867' - Label 'Protein fold name'\n",
      "Entity 'http://edamontology.org/data_1868' - Label 'Taxon'\n",
      "Entity 'http://edamontology.org/data_1869' - Label 'Organism identifier'\n",
      "Entity 'http://edamontology.org/data_1870' - Label 'Genus name'\n",
      "Entity 'http://edamontology.org/data_1872' - Label 'Taxonomic classification'\n",
      "Entity 'http://edamontology.org/data_1873' - Label 'iHOP organism ID'\n",
      "Entity 'http://edamontology.org/data_1874' - Label 'Genbank common name'\n",
      "Entity 'http://edamontology.org/data_1875' - Label 'NCBI taxon'\n",
      "Entity 'http://edamontology.org/data_1877' - Label 'Synonym'\n",
      "Entity 'http://edamontology.org/data_1878' - Label 'Misspelling'\n",
      "Entity 'http://edamontology.org/data_1879' - Label 'Acronym'\n",
      "Entity 'http://edamontology.org/data_1880' - Label 'Misnomer'\n",
      "Entity 'http://edamontology.org/data_1881' - Label 'Author ID'\n",
      "Entity 'http://edamontology.org/data_1882' - Label 'DragonDB author identifier'\n",
      "Entity 'http://edamontology.org/data_1883' - Label 'Annotated URI'\n",
      "Entity 'http://edamontology.org/data_1884' - Label 'UniProt keywords'\n",
      "Entity 'http://edamontology.org/data_1885' - Label 'Gene ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1886' - Label 'Blattner number'\n",
      "Entity 'http://edamontology.org/data_1887' - Label 'Gene ID (MIPS Maize)'\n",
      "Entity 'http://edamontology.org/data_1888' - Label 'Gene ID (MIPS Medicago)'\n",
      "Entity 'http://edamontology.org/data_1889' - Label 'Gene name (DragonDB)'\n",
      "Entity 'http://edamontology.org/data_1890' - Label 'Gene name (Arabidopsis)'\n",
      "Entity 'http://edamontology.org/data_1891' - Label 'iHOP symbol'\n",
      "Entity 'http://edamontology.org/data_1892' - Label 'Gene name (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1893' - Label 'Locus ID'\n",
      "Entity 'http://edamontology.org/data_1895' - Label 'Locus ID (AGI)'\n",
      "Entity 'http://edamontology.org/data_1896' - Label 'Locus ID (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1897' - Label 'Locus ID (MGG)'\n",
      "Entity 'http://edamontology.org/data_1898' - Label 'Locus ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1899' - Label 'Locus ID (CMR)'\n",
      "Entity 'http://edamontology.org/data_1900' - Label 'NCBI locus tag'\n",
      "Entity 'http://edamontology.org/data_1901' - Label 'Locus ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1902' - Label 'Locus ID (MMP)'\n",
      "Entity 'http://edamontology.org/data_1903' - Label 'Locus ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1904' - Label 'Locus ID (EntrezGene)'\n",
      "Entity 'http://edamontology.org/data_1905' - Label 'Locus ID (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1906' - Label 'Quantitative trait locus'\n",
      "Entity 'http://edamontology.org/data_1907' - Label 'Gene ID (KOME)'\n",
      "Entity 'http://edamontology.org/data_1908' - Label 'Locus ID (Tropgene)'\n",
      "Entity 'http://edamontology.org/data_1916' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/data_1917' - Label 'Atomic property'\n",
      "Entity 'http://edamontology.org/data_2007' - Label 'UniProt keyword'\n",
      "Entity 'http://edamontology.org/data_2009' - Label 'Ordered locus name'\n",
      "Entity 'http://edamontology.org/data_2012' - Label 'Sequence coordinates'\n",
      "Entity 'http://edamontology.org/data_2016' - Label 'Amino acid property'\n",
      "Entity 'http://edamontology.org/data_2018' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/data_2019' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2022' - Label 'Vienna RNA structural data'\n",
      "Entity 'http://edamontology.org/data_2023' - Label 'Sequence mask parameter'\n",
      "Entity 'http://edamontology.org/data_2024' - Label 'Enzyme kinetics data'\n",
      "Entity 'http://edamontology.org/data_2025' - Label 'Michaelis Menten plot'\n",
      "Entity 'http://edamontology.org/data_2026' - Label 'Hanes Woolf plot'\n",
      "Entity 'http://edamontology.org/data_2028' - Label 'Experimental data'\n",
      "Entity 'http://edamontology.org/data_2041' - Label 'Genome version information'\n",
      "Entity 'http://edamontology.org/data_2042' - Label 'Evidence'\n",
      "Entity 'http://edamontology.org/data_2043' - Label 'Sequence record lite'\n",
      "Entity 'http://edamontology.org/data_2044' - Label 'Sequence'\n",
      "Entity 'http://edamontology.org/data_2046' - Label 'Nucleic acid sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2047' - Label 'Protein sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2048' - Label 'Report'\n",
      "Entity 'http://edamontology.org/data_2050' - Label 'Molecular property (general)'\n",
      "Entity 'http://edamontology.org/data_2053' - Label 'Structural data'\n",
      "Entity 'http://edamontology.org/data_2070' - Label 'Sequence motif (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_2071' - Label 'Sequence motif (protein)'\n",
      "Entity 'http://edamontology.org/data_2079' - Label 'Search parameter'\n",
      "Entity 'http://edamontology.org/data_2080' - Label 'Database search results'\n",
      "Entity 'http://edamontology.org/data_2081' - Label 'Secondary structure'\n",
      "Entity 'http://edamontology.org/data_2082' - Label 'Matrix'\n",
      "Entity 'http://edamontology.org/data_2083' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2084' - Label 'Nucleic acid report'\n",
      "Entity 'http://edamontology.org/data_2085' - Label 'Structure report'\n",
      "Entity 'http://edamontology.org/data_2086' - Label 'Nucleic acid structure data'\n",
      "Entity 'http://edamontology.org/data_2087' - Label 'Molecular property'\n",
      "Entity 'http://edamontology.org/data_2088' - Label 'DNA base structural data'\n",
      "Entity 'http://edamontology.org/data_2090' - Label 'Database entry version information'\n",
      "Entity 'http://edamontology.org/data_2091' - Label 'Accession'\n",
      "Entity 'http://edamontology.org/data_2092' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/data_2093' - Label 'Data reference'\n",
      "Entity 'http://edamontology.org/data_2098' - Label 'Job identifier'\n",
      "Entity 'http://edamontology.org/data_2099' - Label 'Name'\n",
      "Entity 'http://edamontology.org/data_2100' - Label 'Type'\n",
      "Entity 'http://edamontology.org/data_2101' - Label 'Account authentication'\n",
      "Entity 'http://edamontology.org/data_2102' - Label 'KEGG organism code'\n",
      "Entity 'http://edamontology.org/data_2103' - Label 'Gene name (KEGG GENES)'\n",
      "Entity 'http://edamontology.org/data_2104' - Label 'BioCyc ID'\n",
      "Entity 'http://edamontology.org/data_2105' - Label 'Compound ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2106' - Label 'Reaction ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2107' - Label 'Enzyme ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2108' - Label 'Reaction ID'\n",
      "Entity 'http://edamontology.org/data_2109' - Label 'Identifier (hybrid)'\n",
      "Entity 'http://edamontology.org/data_2110' - Label 'Molecular property identifier'\n",
      "Entity 'http://edamontology.org/data_2111' - Label 'Codon usage table ID'\n",
      "Entity 'http://edamontology.org/data_2112' - Label 'FlyBase primary identifier'\n",
      "Entity 'http://edamontology.org/data_2113' - Label 'WormBase identifier'\n",
      "Entity 'http://edamontology.org/data_2114' - Label 'WormBase wormpep ID'\n",
      "Entity 'http://edamontology.org/data_2116' - Label 'Nucleic acid features (codon)'\n",
      "Entity 'http://edamontology.org/data_2117' - Label 'Map identifier'\n",
      "Entity 'http://edamontology.org/data_2118' - Label 'Person identifier'\n",
      "Entity 'http://edamontology.org/data_2119' - Label 'Nucleic acid identifier'\n",
      "Entity 'http://edamontology.org/data_2126' - Label 'Translation frame specification'\n",
      "Entity 'http://edamontology.org/data_2127' - Label 'Genetic code identifier'\n",
      "Entity 'http://edamontology.org/data_2128' - Label 'Genetic code name'\n",
      "Entity 'http://edamontology.org/data_2129' - Label 'File format name'\n",
      "Entity 'http://edamontology.org/data_2130' - Label 'Sequence profile type'\n",
      "Entity 'http://edamontology.org/data_2131' - Label 'Operating system name'\n",
      "Entity 'http://edamontology.org/data_2132' - Label 'Mutation type'\n",
      "Entity 'http://edamontology.org/data_2133' - Label 'Logical operator'\n",
      "Entity 'http://edamontology.org/data_2134' - Label 'Results sort order'\n",
      "Entity 'http://edamontology.org/data_2135' - Label 'Toggle'\n",
      "Entity 'http://edamontology.org/data_2136' - Label 'Sequence width'\n",
      "Entity 'http://edamontology.org/data_2137' - Label 'Gap penalty'\n",
      "Entity 'http://edamontology.org/data_2139' - Label 'Nucleic acid melting temperature'\n",
      "Entity 'http://edamontology.org/data_2140' - Label 'Concentration'\n",
      "Entity 'http://edamontology.org/data_2141' - Label 'Window step size'\n",
      "Entity 'http://edamontology.org/data_2142' - Label 'EMBOSS graph'\n",
      "Entity 'http://edamontology.org/data_2143' - Label 'EMBOSS report'\n",
      "Entity 'http://edamontology.org/data_2145' - Label 'Sequence offset'\n",
      "Entity 'http://edamontology.org/data_2146' - Label 'Threshold'\n",
      "Entity 'http://edamontology.org/data_2147' - Label 'Protein report (transcription factor)'\n",
      "Entity 'http://edamontology.org/data_2149' - Label 'Database category name'\n",
      "Entity 'http://edamontology.org/data_2150' - Label 'Sequence profile name'\n",
      "Entity 'http://edamontology.org/data_2151' - Label 'Color'\n",
      "Entity 'http://edamontology.org/data_2152' - Label 'Rendering parameter'\n",
      "Entity 'http://edamontology.org/data_2154' - Label 'Sequence name'\n",
      "Entity 'http://edamontology.org/data_2156' - Label 'Date'\n",
      "Entity 'http://edamontology.org/data_2157' - Label 'Word composition'\n",
      "Entity 'http://edamontology.org/data_2160' - Label 'Fickett testcode plot'\n",
      "Entity 'http://edamontology.org/data_2161' - Label 'Sequence similarity plot'\n",
      "Entity 'http://edamontology.org/data_2162' - Label 'Helical wheel'\n",
      "Entity 'http://edamontology.org/data_2163' - Label 'Helical net'\n",
      "Entity 'http://edamontology.org/data_2164' - Label 'Protein sequence properties plot'\n",
      "Entity 'http://edamontology.org/data_2165' - Label 'Protein ionisation curve'\n",
      "Entity 'http://edamontology.org/data_2166' - Label 'Sequence composition plot'\n",
      "Entity 'http://edamontology.org/data_2167' - Label 'Nucleic acid density plot'\n",
      "Entity 'http://edamontology.org/data_2168' - Label 'Sequence trace image'\n",
      "Entity 'http://edamontology.org/data_2169' - Label 'Nucleic acid features (siRNA)'\n",
      "Entity 'http://edamontology.org/data_2173' - Label 'Sequence set (stream)'\n",
      "Entity 'http://edamontology.org/data_2174' - Label 'FlyBase secondary identifier'\n",
      "Entity 'http://edamontology.org/data_2176' - Label 'Cardinality'\n",
      "Entity 'http://edamontology.org/data_2177' - Label 'Exactly 1'\n",
      "Entity 'http://edamontology.org/data_2178' - Label '1 or more'\n",
      "Entity 'http://edamontology.org/data_2179' - Label 'Exactly 2'\n",
      "Entity 'http://edamontology.org/data_2180' - Label '2 or more'\n",
      "Entity 'http://edamontology.org/data_2190' - Label 'Sequence checksum'\n",
      "Entity 'http://edamontology.org/data_2191' - Label 'Protein features report (chemical modifications)'\n",
      "Entity 'http://edamontology.org/data_2192' - Label 'Error'\n",
      "Entity 'http://edamontology.org/data_2193' - Label 'Database entry metadata'\n",
      "Entity 'http://edamontology.org/data_2198' - Label 'Gene cluster'\n",
      "Entity 'http://edamontology.org/data_2201' - Label 'Sequence record full'\n",
      "Entity 'http://edamontology.org/data_2208' - Label 'Plasmid identifier'\n",
      "Entity 'http://edamontology.org/data_2209' - Label 'Mutation ID'\n",
      "Entity 'http://edamontology.org/data_2212' - Label 'Mutation annotation (basic)'\n",
      "Entity 'http://edamontology.org/data_2213' - Label 'Mutation annotation (prevalence)'\n",
      "Entity 'http://edamontology.org/data_2214' - Label 'Mutation annotation (prognostic)'\n",
      "Entity 'http://edamontology.org/data_2215' - Label 'Mutation annotation (functional)'\n",
      "Entity 'http://edamontology.org/data_2216' - Label 'Codon number'\n",
      "Entity 'http://edamontology.org/data_2217' - Label 'Tumor annotation'\n",
      "Entity 'http://edamontology.org/data_2218' - Label 'Server metadata'\n",
      "Entity 'http://edamontology.org/data_2219' - Label 'Database field name'\n",
      "Entity 'http://edamontology.org/data_2220' - Label 'Sequence cluster ID (SYSTERS)'\n",
      "Entity 'http://edamontology.org/data_2223' - Label 'Ontology metadata'\n",
      "Entity 'http://edamontology.org/data_2235' - Label 'Raw SCOP domain classification'\n",
      "Entity 'http://edamontology.org/data_2236' - Label 'Raw CATH domain classification'\n",
      "Entity 'http://edamontology.org/data_2240' - Label 'Heterogen annotation'\n",
      "Entity 'http://edamontology.org/data_2242' - Label 'Phylogenetic property values'\n",
      "Entity 'http://edamontology.org/data_2245' - Label 'Sequence set (bootstrapped)'\n",
      "Entity 'http://edamontology.org/data_2247' - Label 'Phylogenetic consensus tree'\n",
      "Entity 'http://edamontology.org/data_2248' - Label 'Schema'\n",
      "Entity 'http://edamontology.org/data_2249' - Label 'DTD'\n",
      "Entity 'http://edamontology.org/data_2250' - Label 'XML Schema'\n",
      "Entity 'http://edamontology.org/data_2251' - Label 'Relax-NG schema'\n",
      "Entity 'http://edamontology.org/data_2252' - Label 'XSLT stylesheet'\n",
      "Entity 'http://edamontology.org/data_2253' - Label 'Data resource definition name'\n",
      "Entity 'http://edamontology.org/data_2254' - Label 'OBO file format name'\n",
      "Entity 'http://edamontology.org/data_2285' - Label 'Gene ID (MIPS)'\n",
      "Entity 'http://edamontology.org/data_2288' - Label 'Sequence identifier (protein)'\n",
      "Entity 'http://edamontology.org/data_2289' - Label 'Sequence identifier (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_2290' - Label 'EMBL accession'\n",
      "Entity 'http://edamontology.org/data_2291' - Label 'UniProt ID'\n",
      "Entity 'http://edamontology.org/data_2292' - Label 'GenBank accession'\n",
      "Entity 'http://edamontology.org/data_2293' - Label 'Gramene secondary identifier'\n",
      "Entity 'http://edamontology.org/data_2294' - Label 'Sequence variation ID'\n",
      "Entity 'http://edamontology.org/data_2295' - Label 'Gene ID'\n",
      "Entity 'http://edamontology.org/data_2296' - Label 'Gene name (AceView)'\n",
      "Entity 'http://edamontology.org/data_2297' - Label 'Gene ID (ECK)'\n",
      "Entity 'http://edamontology.org/data_2298' - Label 'Gene ID (HGNC)'\n",
      "Entity 'http://edamontology.org/data_2299' - Label 'Gene name'\n",
      "Entity 'http://edamontology.org/data_2300' - Label 'Gene name (NCBI)'\n",
      "Entity 'http://edamontology.org/data_2301' - Label 'SMILES string'\n",
      "Entity 'http://edamontology.org/data_2302' - Label 'STRING ID'\n",
      "Entity 'http://edamontology.org/data_2307' - Label 'Virus annotation'\n",
      "Entity 'http://edamontology.org/data_2308' - Label 'Virus annotation (taxonomy)'\n",
      "Entity 'http://edamontology.org/data_2309' - Label 'Reaction ID (SABIO-RK)'\n",
      "Entity 'http://edamontology.org/data_2313' - Label 'Carbohydrate report'\n",
      "Entity 'http://edamontology.org/data_2314' - Label 'GI number'\n",
      "Entity 'http://edamontology.org/data_2315' - Label 'NCBI version'\n",
      "Entity 'http://edamontology.org/data_2316' - Label 'Cell line name'\n",
      "Entity 'http://edamontology.org/data_2317' - Label 'Cell line name (exact)'\n",
      "Entity 'http://edamontology.org/data_2318' - Label 'Cell line name (truncated)'\n",
      "Entity 'http://edamontology.org/data_2319' - Label 'Cell line name (no punctuation)'\n",
      "Entity 'http://edamontology.org/data_2320' - Label 'Cell line name (assonant)'\n",
      "Entity 'http://edamontology.org/data_2321' - Label 'Enzyme ID'\n",
      "Entity 'http://edamontology.org/data_2325' - Label 'REBASE enzyme number'\n",
      "Entity 'http://edamontology.org/data_2326' - Label 'DrugBank ID'\n",
      "Entity 'http://edamontology.org/data_2327' - Label 'GI number (protein)'\n",
      "Entity 'http://edamontology.org/data_2335' - Label 'Bit score'\n",
      "Entity 'http://edamontology.org/data_2336' - Label 'Translation phase specification'\n",
      "Entity 'http://edamontology.org/data_2337' - Label 'Resource metadata'\n",
      "Entity 'http://edamontology.org/data_2338' - Label 'Ontology identifier'\n",
      "Entity 'http://edamontology.org/data_2339' - Label 'Ontology concept name'\n",
      "Entity 'http://edamontology.org/data_2340' - Label 'Genome build identifier'\n",
      "Entity 'http://edamontology.org/data_2342' - Label 'Pathway or network name'\n",
      "Entity 'http://edamontology.org/data_2343' - Label 'Pathway ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2344' - Label 'Pathway ID (NCI-Nature)'\n",
      "Entity 'http://edamontology.org/data_2345' - Label 'Pathway ID (ConsensusPathDB)'\n",
      "Entity 'http://edamontology.org/data_2346' - Label 'Sequence cluster ID (UniRef)'\n",
      "Entity 'http://edamontology.org/data_2347' - Label 'Sequence cluster ID (UniRef100)'\n",
      "Entity 'http://edamontology.org/data_2348' - Label 'Sequence cluster ID (UniRef90)'\n",
      "Entity 'http://edamontology.org/data_2349' - Label 'Sequence cluster ID (UniRef50)'\n",
      "Entity 'http://edamontology.org/data_2353' - Label 'Ontology data'\n",
      "Entity 'http://edamontology.org/data_2354' - Label 'RNA family report'\n",
      "Entity 'http://edamontology.org/data_2355' - Label 'RNA family identifier'\n",
      "Entity 'http://edamontology.org/data_2356' - Label 'RFAM accession'\n",
      "Entity 'http://edamontology.org/data_2357' - Label 'Protein signature type'\n",
      "Entity 'http://edamontology.org/data_2358' - Label 'Domain-nucleic acid interaction report'\n",
      "Entity 'http://edamontology.org/data_2359' - Label 'Domain-domain interactions'\n",
      "Entity 'http://edamontology.org/data_2360' - Label 'Domain-domain interaction (indirect)'\n",
      "Entity 'http://edamontology.org/data_2362' - Label 'Sequence accession (hybrid)'\n",
      "Entity 'http://edamontology.org/data_2363' - Label '2D PAGE data'\n",
      "Entity 'http://edamontology.org/data_2364' - Label '2D PAGE report'\n",
      "Entity 'http://edamontology.org/data_2365' - Label 'Pathway or network accession'\n",
      "Entity 'http://edamontology.org/data_2366' - Label 'Secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_2367' - Label 'ASTD ID'\n",
      "Entity 'http://edamontology.org/data_2368' - Label 'ASTD ID (exon)'\n",
      "Entity 'http://edamontology.org/data_2369' - Label 'ASTD ID (intron)'\n",
      "Entity 'http://edamontology.org/data_2370' - Label 'ASTD ID (polya)'\n",
      "Entity 'http://edamontology.org/data_2371' - Label 'ASTD ID (tss)'\n",
      "Entity 'http://edamontology.org/data_2372' - Label '2D PAGE spot report'\n",
      "Entity 'http://edamontology.org/data_2373' - Label 'Spot ID'\n",
      "Entity 'http://edamontology.org/data_2374' - Label 'Spot serial number'\n",
      "Entity 'http://edamontology.org/data_2375' - Label 'Spot ID (HSC-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2378' - Label 'Protein-motif interaction'\n",
      "Entity 'http://edamontology.org/data_2379' - Label 'Strain identifier'\n",
      "Entity 'http://edamontology.org/data_2380' - Label 'CABRI accession'\n",
      "Entity 'http://edamontology.org/data_2381' - Label 'Experiment report (genotyping)'\n",
      "Entity 'http://edamontology.org/data_2382' - Label 'Genotype experiment ID'\n",
      "Entity 'http://edamontology.org/data_2383' - Label 'EGA accession'\n",
      "Entity 'http://edamontology.org/data_2384' - Label 'IPI protein ID'\n",
      "Entity 'http://edamontology.org/data_2385' - Label 'RefSeq accession (protein)'\n",
      "Entity 'http://edamontology.org/data_2386' - Label 'EPD ID'\n",
      "Entity 'http://edamontology.org/data_2387' - Label 'TAIR accession'\n",
      "Entity 'http://edamontology.org/data_2388' - Label 'TAIR accession (At gene)'\n",
      "Entity 'http://edamontology.org/data_2389' - Label 'UniSTS accession'\n",
      "Entity 'http://edamontology.org/data_2390' - Label 'UNITE accession'\n",
      "Entity 'http://edamontology.org/data_2391' - Label 'UTR accession'\n",
      "Entity 'http://edamontology.org/data_2392' - Label 'UniParc accession'\n",
      "Entity 'http://edamontology.org/data_2393' - Label 'mFLJ/mKIAA number'\n",
      "Entity 'http://edamontology.org/data_2395' - Label 'Fungi annotation'\n",
      "Entity 'http://edamontology.org/data_2396' - Label 'Fungi annotation (anamorph)'\n",
      "Entity 'http://edamontology.org/data_2398' - Label 'Ensembl protein ID'\n",
      "Entity 'http://edamontology.org/data_2400' - Label 'Toxin annotation'\n",
      "Entity 'http://edamontology.org/data_2401' - Label 'Protein report (membrane protein)'\n",
      "Entity 'http://edamontology.org/data_2402' - Label 'Protein-drug interaction report'\n",
      "Entity 'http://edamontology.org/data_2522' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2523' - Label 'Phylogenetic data'\n",
      "Entity 'http://edamontology.org/data_2524' - Label 'Protein data'\n",
      "Entity 'http://edamontology.org/data_2525' - Label 'Nucleic acid data'\n",
      "Entity 'http://edamontology.org/data_2526' - Label 'Text data'\n",
      "Entity 'http://edamontology.org/data_2527' - Label 'Parameter'\n",
      "Entity 'http://edamontology.org/data_2528' - Label 'Molecular data'\n",
      "Entity 'http://edamontology.org/data_2529' - Label 'Molecule report'\n",
      "Entity 'http://edamontology.org/data_2530' - Label 'Organism report'\n",
      "Entity 'http://edamontology.org/data_2531' - Label 'Protocol'\n",
      "Entity 'http://edamontology.org/data_2534' - Label 'Sequence attribute'\n",
      "Entity 'http://edamontology.org/data_2535' - Label 'Sequence tag profile'\n",
      "Entity 'http://edamontology.org/data_2536' - Label 'Mass spectrometry data'\n",
      "Entity 'http://edamontology.org/data_2537' - Label 'Protein structure raw data'\n",
      "Entity 'http://edamontology.org/data_2538' - Label 'Mutation identifier'\n",
      "Entity 'http://edamontology.org/data_2539' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2540' - Label 'Data index data'\n",
      "Entity 'http://edamontology.org/data_2563' - Label 'Amino acid name (single letter)'\n",
      "Entity 'http://edamontology.org/data_2564' - Label 'Amino acid name (three letter)'\n",
      "Entity 'http://edamontology.org/data_2565' - Label 'Amino acid name (full name)'\n",
      "Entity 'http://edamontology.org/data_2576' - Label 'Toxin identifier'\n",
      "Entity 'http://edamontology.org/data_2578' - Label 'ArachnoServer ID'\n",
      "Entity 'http://edamontology.org/data_2579' - Label 'Expressed gene list'\n",
      "Entity 'http://edamontology.org/data_2580' - Label 'BindingDB Monomer ID'\n",
      "Entity 'http://edamontology.org/data_2581' - Label 'GO concept name'\n",
      "Entity 'http://edamontology.org/data_2582' - Label 'GO concept ID (biological process)'\n",
      "Entity 'http://edamontology.org/data_2583' - Label 'GO concept ID (molecular function)'\n",
      "Entity 'http://edamontology.org/data_2584' - Label 'GO concept name (cellular component)'\n",
      "Entity 'http://edamontology.org/data_2586' - Label 'Northern blot image'\n",
      "Entity 'http://edamontology.org/data_2587' - Label 'Blot ID'\n",
      "Entity 'http://edamontology.org/data_2588' - Label 'BlotBase blot ID'\n",
      "Entity 'http://edamontology.org/data_2589' - Label 'Hierarchy'\n",
      "Entity 'http://edamontology.org/data_2590' - Label 'Hierarchy identifier'\n",
      "Entity 'http://edamontology.org/data_2591' - Label 'Brite hierarchy ID'\n",
      "Entity 'http://edamontology.org/data_2592' - Label 'Cancer type'\n",
      "Entity 'http://edamontology.org/data_2593' - Label 'BRENDA organism ID'\n",
      "Entity 'http://edamontology.org/data_2594' - Label 'UniGene taxon'\n",
      "Entity 'http://edamontology.org/data_2595' - Label 'UTRdb taxon'\n",
      "Entity 'http://edamontology.org/data_2596' - Label 'Catalogue ID'\n",
      "Entity 'http://edamontology.org/data_2597' - Label 'CABRI catalogue name'\n",
      "Entity 'http://edamontology.org/data_2598' - Label 'Secondary structure alignment metadata'\n",
      "Entity 'http://edamontology.org/data_2599' - Label 'Molecule interaction report'\n",
      "Entity 'http://edamontology.org/data_2600' - Label 'Pathway or network'\n",
      "Entity 'http://edamontology.org/data_2601' - Label 'Small molecule data'\n",
      "Entity 'http://edamontology.org/data_2602' - Label 'Genotype and phenotype data'\n",
      "Entity 'http://edamontology.org/data_2603' - Label 'Expression data'\n",
      "Entity 'http://edamontology.org/data_2605' - Label 'Compound ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2606' - Label 'RFAM name'\n",
      "Entity 'http://edamontology.org/data_2608' - Label 'Reaction ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2609' - Label 'Drug ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2610' - Label 'Ensembl ID'\n",
      "Entity 'http://edamontology.org/data_2611' - Label 'ICD identifier'\n",
      "Entity 'http://edamontology.org/data_2612' - Label 'Sequence cluster ID (CluSTr)'\n",
      "Entity 'http://edamontology.org/data_2613' - Label 'KEGG Glycan ID'\n",
      "Entity 'http://edamontology.org/data_2614' - Label 'TCDB ID'\n",
      "Entity 'http://edamontology.org/data_2615' - Label 'MINT ID'\n",
      "Entity 'http://edamontology.org/data_2616' - Label 'DIP ID'\n",
      "Entity 'http://edamontology.org/data_2617' - Label 'Signaling Gateway protein ID'\n",
      "Entity 'http://edamontology.org/data_2618' - Label 'Protein modification ID'\n",
      "Entity 'http://edamontology.org/data_2619' - Label 'RESID ID'\n",
      "Entity 'http://edamontology.org/data_2620' - Label 'RGD ID'\n",
      "Entity 'http://edamontology.org/data_2621' - Label 'TAIR accession (protein)'\n",
      "Entity 'http://edamontology.org/data_2622' - Label 'Compound ID (HMDB)'\n",
      "Entity 'http://edamontology.org/data_2625' - Label 'LIPID MAPS ID'\n",
      "Entity 'http://edamontology.org/data_2626' - Label 'PeptideAtlas ID'\n",
      "Entity 'http://edamontology.org/data_2627' - Label 'Molecular interaction ID'\n",
      "Entity 'http://edamontology.org/data_2628' - Label 'BioGRID interaction ID'\n",
      "Entity 'http://edamontology.org/data_2629' - Label 'Enzyme ID (MEROPS)'\n",
      "Entity 'http://edamontology.org/data_2630' - Label 'Mobile genetic element ID'\n",
      "Entity 'http://edamontology.org/data_2631' - Label 'ACLAME ID'\n",
      "Entity 'http://edamontology.org/data_2632' - Label 'SGD ID'\n",
      "Entity 'http://edamontology.org/data_2633' - Label 'Book ID'\n",
      "Entity 'http://edamontology.org/data_2634' - Label 'ISBN'\n",
      "Entity 'http://edamontology.org/data_2635' - Label 'Compound ID (3DMET)'\n",
      "Entity 'http://edamontology.org/data_2636' - Label 'MatrixDB interaction ID'\n",
      "Entity 'http://edamontology.org/data_2637' - Label 'cPath ID'\n",
      "Entity 'http://edamontology.org/data_2638' - Label 'PubChem bioassay ID'\n",
      "Entity 'http://edamontology.org/data_2639' - Label 'PubChem ID'\n",
      "Entity 'http://edamontology.org/data_2641' - Label 'Reaction ID (MACie)'\n",
      "Entity 'http://edamontology.org/data_2642' - Label 'Gene ID (miRBase)'\n",
      "Entity 'http://edamontology.org/data_2643' - Label 'Gene ID (ZFIN)'\n",
      "Entity 'http://edamontology.org/data_2644' - Label 'Reaction ID (Rhea)'\n",
      "Entity 'http://edamontology.org/data_2645' - Label 'Pathway ID (Unipathway)'\n",
      "Entity 'http://edamontology.org/data_2646' - Label 'Compound ID (ChEMBL)'\n",
      "Entity 'http://edamontology.org/data_2647' - Label 'LGICdb identifier'\n",
      "Entity 'http://edamontology.org/data_2648' - Label 'Reaction kinetics ID (SABIO-RK)'\n",
      "Entity 'http://edamontology.org/data_2649' - Label 'PharmGKB ID'\n",
      "Entity 'http://edamontology.org/data_2650' - Label 'Pathway ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2651' - Label 'Disease ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2652' - Label 'Drug ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2653' - Label 'Drug ID (TTD)'\n",
      "Entity 'http://edamontology.org/data_2654' - Label 'Target ID (TTD)'\n",
      "Entity 'http://edamontology.org/data_2655' - Label 'Cell type identifier'\n",
      "Entity 'http://edamontology.org/data_2656' - Label 'NeuronDB ID'\n",
      "Entity 'http://edamontology.org/data_2657' - Label 'NeuroMorpho ID'\n",
      "Entity 'http://edamontology.org/data_2658' - Label 'Compound ID (ChemIDplus)'\n",
      "Entity 'http://edamontology.org/data_2659' - Label 'Pathway ID (SMPDB)'\n",
      "Entity 'http://edamontology.org/data_2660' - Label 'BioNumbers ID'\n",
      "Entity 'http://edamontology.org/data_2662' - Label 'T3DB ID'\n",
      "Entity 'http://edamontology.org/data_2663' - Label 'Carbohydrate identifier'\n",
      "Entity 'http://edamontology.org/data_2664' - Label 'GlycomeDB ID'\n",
      "Entity 'http://edamontology.org/data_2665' - Label 'LipidBank ID'\n",
      "Entity 'http://edamontology.org/data_2666' - Label 'CDD ID'\n",
      "Entity 'http://edamontology.org/data_2667' - Label 'MMDB ID'\n",
      "Entity 'http://edamontology.org/data_2668' - Label 'iRefIndex ID'\n",
      "Entity 'http://edamontology.org/data_2669' - Label 'ModelDB ID'\n",
      "Entity 'http://edamontology.org/data_2670' - Label 'Pathway ID (DQCS)'\n",
      "Entity 'http://edamontology.org/data_2671' - Label 'Ensembl ID (Homo sapiens)'\n",
      "Entity 'http://edamontology.org/data_2672' - Label 'Ensembl ID ('Bos taurus')'\n",
      "Entity 'http://edamontology.org/data_2673' - Label 'Ensembl ID ('Canis familiaris')'\n",
      "Entity 'http://edamontology.org/data_2674' - Label 'Ensembl ID ('Cavia porcellus')'\n",
      "Entity 'http://edamontology.org/data_2675' - Label 'Ensembl ID ('Ciona intestinalis')'\n",
      "Entity 'http://edamontology.org/data_2676' - Label 'Ensembl ID ('Ciona savignyi')'\n",
      "Entity 'http://edamontology.org/data_2677' - Label 'Ensembl ID ('Danio rerio')'\n",
      "Entity 'http://edamontology.org/data_2678' - Label 'Ensembl ID ('Dasypus novemcinctus')'\n",
      "Entity 'http://edamontology.org/data_2679' - Label 'Ensembl ID ('Echinops telfairi')'\n",
      "Entity 'http://edamontology.org/data_2680' - Label 'Ensembl ID ('Erinaceus europaeus')'\n",
      "Entity 'http://edamontology.org/data_2681' - Label 'Ensembl ID ('Felis catus')'\n",
      "Entity 'http://edamontology.org/data_2682' - Label 'Ensembl ID ('Gallus gallus')'\n",
      "Entity 'http://edamontology.org/data_2683' - Label 'Ensembl ID ('Gasterosteus aculeatus')'\n",
      "Entity 'http://edamontology.org/data_2684' - Label 'Ensembl ID ('Homo sapiens')'\n",
      "Entity 'http://edamontology.org/data_2685' - Label 'Ensembl ID ('Loxodonta africana')'\n",
      "Entity 'http://edamontology.org/data_2686' - Label 'Ensembl ID ('Macaca mulatta')'\n",
      "Entity 'http://edamontology.org/data_2687' - Label 'Ensembl ID ('Monodelphis domestica')'\n",
      "Entity 'http://edamontology.org/data_2688' - Label 'Ensembl ID ('Mus musculus')'\n",
      "Entity 'http://edamontology.org/data_2689' - Label 'Ensembl ID ('Myotis lucifugus')'\n",
      "Entity 'http://edamontology.org/data_2690' - Label 'Ensembl ID (\"Ornithorhynchus anatinus\")'\n",
      "Entity 'http://edamontology.org/data_2691' - Label 'Ensembl ID ('Oryctolagus cuniculus')'\n",
      "Entity 'http://edamontology.org/data_2692' - Label 'Ensembl ID ('Oryzias latipes')'\n",
      "Entity 'http://edamontology.org/data_2693' - Label 'Ensembl ID ('Otolemur garnettii')'\n",
      "Entity 'http://edamontology.org/data_2694' - Label 'Ensembl ID ('Pan troglodytes')'\n",
      "Entity 'http://edamontology.org/data_2695' - Label 'Ensembl ID ('Rattus norvegicus')'\n",
      "Entity 'http://edamontology.org/data_2696' - Label 'Ensembl ID ('Spermophilus tridecemlineatus')'\n",
      "Entity 'http://edamontology.org/data_2697' - Label 'Ensembl ID ('Takifugu rubripes')'\n",
      "Entity 'http://edamontology.org/data_2698' - Label 'Ensembl ID ('Tupaia belangeri')'\n",
      "Entity 'http://edamontology.org/data_2699' - Label 'Ensembl ID ('Xenopus tropicalis')'\n",
      "Entity 'http://edamontology.org/data_2700' - Label 'CATH identifier'\n",
      "Entity 'http://edamontology.org/data_2701' - Label 'CATH node ID (family)'\n",
      "Entity 'http://edamontology.org/data_2702' - Label 'Enzyme ID (CAZy)'\n",
      "Entity 'http://edamontology.org/data_2704' - Label 'Clone ID (IMAGE)'\n",
      "Entity 'http://edamontology.org/data_2705' - Label 'GO concept ID (cellular component)'\n",
      "Entity 'http://edamontology.org/data_2706' - Label 'Chromosome name (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2709' - Label 'CleanEx entry name'\n",
      "Entity 'http://edamontology.org/data_2710' - Label 'CleanEx dataset code'\n",
      "Entity 'http://edamontology.org/data_2711' - Label 'Genome report'\n",
      "Entity 'http://edamontology.org/data_2713' - Label 'Protein ID (CORUM)'\n",
      "Entity 'http://edamontology.org/data_2714' - Label 'CDD PSSM-ID'\n",
      "Entity 'http://edamontology.org/data_2715' - Label 'Protein ID (CuticleDB)'\n",
      "Entity 'http://edamontology.org/data_2716' - Label 'DBD ID'\n",
      "Entity 'http://edamontology.org/data_2717' - Label 'Oligonucleotide probe annotation'\n",
      "Entity 'http://edamontology.org/data_2718' - Label 'Oligonucleotide ID'\n",
      "Entity 'http://edamontology.org/data_2719' - Label 'dbProbe ID'\n",
      "Entity 'http://edamontology.org/data_2720' - Label 'Dinucleotide property'\n",
      "Entity 'http://edamontology.org/data_2721' - Label 'DiProDB ID'\n",
      "Entity 'http://edamontology.org/data_2722' - Label 'Protein features report (disordered structure)'\n",
      "Entity 'http://edamontology.org/data_2723' - Label 'Protein ID (DisProt)'\n",
      "Entity 'http://edamontology.org/data_2724' - Label 'Embryo report'\n",
      "Entity 'http://edamontology.org/data_2725' - Label 'Ensembl transcript ID'\n",
      "Entity 'http://edamontology.org/data_2726' - Label 'Inhibitor annotation'\n",
      "Entity 'http://edamontology.org/data_2727' - Label 'Promoter ID'\n",
      "Entity 'http://edamontology.org/data_2728' - Label 'EST accession'\n",
      "Entity 'http://edamontology.org/data_2729' - Label 'COGEME EST ID'\n",
      "Entity 'http://edamontology.org/data_2730' - Label 'COGEME unisequence ID'\n",
      "Entity 'http://edamontology.org/data_2731' - Label 'Protein family ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_2732' - Label 'Family name'\n",
      "Entity 'http://edamontology.org/data_2733' - Label 'Genus name (virus)'\n",
      "Entity 'http://edamontology.org/data_2734' - Label 'Family name (virus)'\n",
      "Entity 'http://edamontology.org/data_2735' - Label 'Database name (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2736' - Label 'Sequence feature ID (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2737' - Label 'FIG ID'\n",
      "Entity 'http://edamontology.org/data_2738' - Label 'Gene ID (Xenbase)'\n",
      "Entity 'http://edamontology.org/data_2739' - Label 'Gene ID (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2740' - Label 'Gene name (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2741' - Label 'ABS ID'\n",
      "Entity 'http://edamontology.org/data_2742' - Label 'AraC-XylS ID'\n",
      "Entity 'http://edamontology.org/data_2743' - Label 'Gene name (HUGO)'\n",
      "Entity 'http://edamontology.org/data_2744' - Label 'Locus ID (PseudoCAP)'\n",
      "Entity 'http://edamontology.org/data_2745' - Label 'Locus ID (UTR)'\n",
      "Entity 'http://edamontology.org/data_2746' - Label 'MonosaccharideDB ID'\n",
      "Entity 'http://edamontology.org/data_2747' - Label 'Database name (CMD)'\n",
      "Entity 'http://edamontology.org/data_2748' - Label 'Database name (Osteogenesis)'\n",
      "Entity 'http://edamontology.org/data_2749' - Label 'Genome identifier'\n",
      "Entity 'http://edamontology.org/data_2751' - Label 'GenomeReviews ID'\n",
      "Entity 'http://edamontology.org/data_2752' - Label 'GlycoMap ID'\n",
      "Entity 'http://edamontology.org/data_2753' - Label 'Carbohydrate conformational map'\n",
      "Entity 'http://edamontology.org/data_2755' - Label 'Transcription factor name'\n",
      "Entity 'http://edamontology.org/data_2756' - Label 'TCID'\n",
      "Entity 'http://edamontology.org/data_2757' - Label 'Pfam domain name'\n",
      "Entity 'http://edamontology.org/data_2758' - Label 'Pfam clan ID'\n",
      "Entity 'http://edamontology.org/data_2759' - Label 'Gene ID (VectorBase)'\n",
      "Entity 'http://edamontology.org/data_2761' - Label 'UTRSite ID'\n",
      "Entity 'http://edamontology.org/data_2762' - Label 'Sequence signature report'\n",
      "Entity 'http://edamontology.org/data_2763' - Label 'Locus annotation'\n",
      "Entity 'http://edamontology.org/data_2764' - Label 'Protein name (UniProt)'\n",
      "Entity 'http://edamontology.org/data_2765' - Label 'Term ID list'\n",
      "Entity 'http://edamontology.org/data_2766' - Label 'HAMAP ID'\n",
      "Entity 'http://edamontology.org/data_2767' - Label 'Identifier with metadata'\n",
      "Entity 'http://edamontology.org/data_2768' - Label 'Gene symbol annotation'\n",
      "Entity 'http://edamontology.org/data_2769' - Label 'Transcript ID'\n",
      "Entity 'http://edamontology.org/data_2770' - Label 'HIT ID'\n",
      "Entity 'http://edamontology.org/data_2771' - Label 'HIX ID'\n",
      "Entity 'http://edamontology.org/data_2772' - Label 'HPA antibody id'\n",
      "Entity 'http://edamontology.org/data_2773' - Label 'IMGT/HLA ID'\n",
      "Entity 'http://edamontology.org/data_2774' - Label 'Gene ID (JCVI)'\n",
      "Entity 'http://edamontology.org/data_2775' - Label 'Kinase name'\n",
      "Entity 'http://edamontology.org/data_2776' - Label 'ConsensusPathDB entity ID'\n",
      "Entity 'http://edamontology.org/data_2777' - Label 'ConsensusPathDB entity name'\n",
      "Entity 'http://edamontology.org/data_2778' - Label 'CCAP strain number'\n",
      "Entity 'http://edamontology.org/data_2779' - Label 'Stock number'\n",
      "Entity 'http://edamontology.org/data_2780' - Label 'Stock number (TAIR)'\n",
      "Entity 'http://edamontology.org/data_2781' - Label 'REDIdb ID'\n",
      "Entity 'http://edamontology.org/data_2782' - Label 'SMART domain name'\n",
      "Entity 'http://edamontology.org/data_2783' - Label 'Protein family ID (PANTHER)'\n",
      "Entity 'http://edamontology.org/data_2784' - Label 'RNAVirusDB ID'\n",
      "Entity 'http://edamontology.org/data_2785' - Label 'Virus identifier'\n",
      "Entity 'http://edamontology.org/data_2786' - Label 'NCBI Genome Project ID'\n",
      "Entity 'http://edamontology.org/data_2787' - Label 'NCBI genome accession'\n",
      "Entity 'http://edamontology.org/data_2788' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2789' - Label 'Protein ID (TopDB)'\n",
      "Entity 'http://edamontology.org/data_2790' - Label 'Gel ID'\n",
      "Entity 'http://edamontology.org/data_2791' - Label 'Reference map name (SWISS-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2792' - Label 'Protein ID (PeroxiBase)'\n",
      "Entity 'http://edamontology.org/data_2793' - Label 'SISYPHUS ID'\n",
      "Entity 'http://edamontology.org/data_2794' - Label 'ORF ID'\n",
      "Entity 'http://edamontology.org/data_2795' - Label 'ORF identifier'\n",
      "Entity 'http://edamontology.org/data_2796' - Label 'LINUCS ID'\n",
      "Entity 'http://edamontology.org/data_2797' - Label 'Protein ID (LGICdb)'\n",
      "Entity 'http://edamontology.org/data_2798' - Label 'MaizeDB ID'\n",
      "Entity 'http://edamontology.org/data_2799' - Label 'Gene ID (MfunGD)'\n",
      "Entity 'http://edamontology.org/data_2800' - Label 'Orpha number'\n",
      "Entity 'http://edamontology.org/data_2802' - Label 'Protein ID (EcID)'\n",
      "Entity 'http://edamontology.org/data_2803' - Label 'Clone ID (RefSeq)'\n",
      "Entity 'http://edamontology.org/data_2804' - Label 'Protein ID (ConoServer)'\n",
      "Entity 'http://edamontology.org/data_2805' - Label 'GeneSNP ID'\n",
      "Entity 'http://edamontology.org/data_2812' - Label 'Lipid identifier'\n",
      "Entity 'http://edamontology.org/data_2831' - Label 'Databank'\n",
      "Entity 'http://edamontology.org/data_2832' - Label 'Web portal'\n",
      "Entity 'http://edamontology.org/data_2835' - Label 'Gene ID (VBASE2)'\n",
      "Entity 'http://edamontology.org/data_2836' - Label 'DPVweb ID'\n",
      "Entity 'http://edamontology.org/data_2837' - Label 'Pathway ID (BioSystems)'\n",
      "Entity 'http://edamontology.org/data_2838' - Label 'Experimental data (proteomics)'\n",
      "Entity 'http://edamontology.org/data_2849' - Label 'Abstract'\n",
      "Entity 'http://edamontology.org/data_2850' - Label 'Lipid structure'\n",
      "Entity 'http://edamontology.org/data_2851' - Label 'Drug structure'\n",
      "Entity 'http://edamontology.org/data_2852' - Label 'Toxin structure'\n",
      "Entity 'http://edamontology.org/data_2854' - Label 'Position-specific scoring matrix'\n",
      "Entity 'http://edamontology.org/data_2855' - Label 'Distance matrix'\n",
      "Entity 'http://edamontology.org/data_2856' - Label 'Structural distance matrix'\n",
      "Entity 'http://edamontology.org/data_2857' - Label 'Article metadata'\n",
      "Entity 'http://edamontology.org/data_2858' - Label 'Ontology concept'\n",
      "Entity 'http://edamontology.org/data_2865' - Label 'Codon usage bias'\n",
      "Entity 'http://edamontology.org/data_2866' - Label 'Northern blot report'\n",
      "Entity 'http://edamontology.org/data_2870' - Label 'Radiation hybrid map'\n",
      "Entity 'http://edamontology.org/data_2872' - Label 'ID list'\n",
      "Entity 'http://edamontology.org/data_2873' - Label 'Phylogenetic gene frequencies data'\n",
      "Entity 'http://edamontology.org/data_2874' - Label 'Sequence set (polymorphic)'\n",
      "Entity 'http://edamontology.org/data_2875' - Label 'DRCAT resource'\n",
      "Entity 'http://edamontology.org/data_2877' - Label 'Protein complex'\n",
      "Entity 'http://edamontology.org/data_2878' - Label 'Protein structural motif'\n",
      "Entity 'http://edamontology.org/data_2879' - Label 'Lipid report'\n",
      "Entity 'http://edamontology.org/data_2880' - Label 'Secondary structure image'\n",
      "Entity 'http://edamontology.org/data_2881' - Label 'Secondary structure report'\n",
      "Entity 'http://edamontology.org/data_2882' - Label 'DNA features'\n",
      "Entity 'http://edamontology.org/data_2883' - Label 'RNA features report'\n",
      "Entity 'http://edamontology.org/data_2884' - Label 'Plot'\n",
      "Entity 'http://edamontology.org/data_2886' - Label 'Protein sequence record'\n",
      "Entity 'http://edamontology.org/data_2887' - Label 'Nucleic acid sequence record'\n",
      "Entity 'http://edamontology.org/data_2888' - Label 'Protein sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2889' - Label 'Nucleic acid sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2891' - Label 'Biological model accession'\n",
      "Entity 'http://edamontology.org/data_2892' - Label 'Cell type name'\n",
      "Entity 'http://edamontology.org/data_2893' - Label 'Cell type accession'\n",
      "Entity 'http://edamontology.org/data_2894' - Label 'Compound accession'\n",
      "Entity 'http://edamontology.org/data_2895' - Label 'Drug accession'\n",
      "Entity 'http://edamontology.org/data_2896' - Label 'Toxin name'\n",
      "Entity 'http://edamontology.org/data_2897' - Label 'Toxin accession'\n",
      "Entity 'http://edamontology.org/data_2898' - Label 'Monosaccharide accession'\n",
      "Entity 'http://edamontology.org/data_2899' - Label 'Drug name'\n",
      "Entity 'http://edamontology.org/data_2900' - Label 'Carbohydrate accession'\n",
      "Entity 'http://edamontology.org/data_2901' - Label 'Molecule accession'\n",
      "Entity 'http://edamontology.org/data_2902' - Label 'Data resource definition accession'\n",
      "Entity 'http://edamontology.org/data_2903' - Label 'Genome accession'\n",
      "Entity 'http://edamontology.org/data_2904' - Label 'Map accession'\n",
      "Entity 'http://edamontology.org/data_2905' - Label 'Lipid accession'\n",
      "Entity 'http://edamontology.org/data_2906' - Label 'Peptide ID'\n",
      "Entity 'http://edamontology.org/data_2907' - Label 'Protein accession'\n",
      "Entity 'http://edamontology.org/data_2908' - Label 'Organism accession'\n",
      "Entity 'http://edamontology.org/data_2909' - Label 'Organism name'\n",
      "Entity 'http://edamontology.org/data_2910' - Label 'Protein family accession'\n",
      "Entity 'http://edamontology.org/data_2911' - Label 'Transcription factor accession'\n",
      "Entity 'http://edamontology.org/data_2912' - Label 'Strain accession'\n",
      "Entity 'http://edamontology.org/data_2913' - Label 'Virus identifier'\n",
      "Entity 'http://edamontology.org/data_2914' - Label 'Sequence features metadata'\n",
      "Entity 'http://edamontology.org/data_2915' - Label 'Gramene identifier'\n",
      "Entity 'http://edamontology.org/data_2916' - Label 'DDBJ accession'\n",
      "Entity 'http://edamontology.org/data_2917' - Label 'ConsensusPathDB identifier'\n",
      "Entity 'http://edamontology.org/data_2925' - Label 'Sequence data'\n",
      "Entity 'http://edamontology.org/data_2927' - Label 'Codon usage'\n",
      "Entity 'http://edamontology.org/data_2954' - Label 'Article report'\n",
      "Entity 'http://edamontology.org/data_2955' - Label 'Sequence report'\n",
      "Entity 'http://edamontology.org/data_2956' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/data_2957' - Label 'Hopp and Woods plot'\n",
      "Entity 'http://edamontology.org/data_2958' - Label 'Nucleic acid melting curve'\n",
      "Entity 'http://edamontology.org/data_2959' - Label 'Nucleic acid probability profile'\n",
      "Entity 'http://edamontology.org/data_2960' - Label 'Nucleic acid temperature profile'\n",
      "Entity 'http://edamontology.org/data_2961' - Label 'Gene regulatory network report'\n",
      "Entity 'http://edamontology.org/data_2965' - Label '2D PAGE gel report'\n",
      "Entity 'http://edamontology.org/data_2966' - Label 'Oligonucleotide probe sets annotation'\n",
      "Entity 'http://edamontology.org/data_2967' - Label 'Microarray image'\n",
      "Entity 'http://edamontology.org/data_2968' - Label 'Image'\n",
      "Entity 'http://edamontology.org/data_2969' - Label 'Sequence image'\n",
      "Entity 'http://edamontology.org/data_2970' - Label 'Protein hydropathy data'\n",
      "Entity 'http://edamontology.org/data_2971' - Label 'Workflow data'\n",
      "Entity 'http://edamontology.org/data_2972' - Label 'Workflow'\n",
      "Entity 'http://edamontology.org/data_2973' - Label 'Secondary structure data'\n",
      "Entity 'http://edamontology.org/data_2974' - Label 'Protein sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2975' - Label 'Nucleic acid sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2976' - Label 'Protein sequence'\n",
      "Entity 'http://edamontology.org/data_2977' - Label 'Nucleic acid sequence'\n",
      "Entity 'http://edamontology.org/data_2978' - Label 'Reaction data'\n",
      "Entity 'http://edamontology.org/data_2979' - Label 'Peptide property'\n",
      "Entity 'http://edamontology.org/data_2980' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/data_2981' - Label 'Sequence motif data'\n",
      "Entity 'http://edamontology.org/data_2982' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2983' - Label 'Pathway or network data'\n",
      "Entity 'http://edamontology.org/data_2984' - Label 'Pathway or network report'\n",
      "Entity 'http://edamontology.org/data_2985' - Label 'Nucleic acid thermodynamic data'\n",
      "Entity 'http://edamontology.org/data_2986' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/data_2987' - Label 'Classification report'\n",
      "Entity 'http://edamontology.org/data_2989' - Label 'Protein features report (key folding sites)'\n",
      "Entity 'http://edamontology.org/data_2991' - Label 'Protein geometry data'\n",
      "Entity 'http://edamontology.org/data_2992' - Label 'Protein structure image'\n",
      "Entity 'http://edamontology.org/data_2994' - Label 'Phylogenetic character weights'\n",
      "Entity 'http://edamontology.org/data_3002' - Label 'Annotation track'\n",
      "Entity 'http://edamontology.org/data_3021' - Label 'UniProt accession'\n",
      "Entity 'http://edamontology.org/data_3022' - Label 'NCBI genetic code ID'\n",
      "Entity 'http://edamontology.org/data_3025' - Label 'Ontology concept identifier'\n",
      "Entity 'http://edamontology.org/data_3026' - Label 'GO concept name (biological process)'\n",
      "Entity 'http://edamontology.org/data_3027' - Label 'GO concept name (molecular function)'\n",
      "Entity 'http://edamontology.org/data_3028' - Label 'Taxonomy'\n",
      "Entity 'http://edamontology.org/data_3029' - Label 'Protein ID (EMBL/GenBank/DDBJ)'\n",
      "Entity 'http://edamontology.org/data_3031' - Label 'Core data'\n",
      "Entity 'http://edamontology.org/data_3034' - Label 'Sequence feature identifier'\n",
      "Entity 'http://edamontology.org/data_3035' - Label 'Structure identifier'\n",
      "Entity 'http://edamontology.org/data_3036' - Label 'Matrix identifier'\n",
      "Entity 'http://edamontology.org/data_3085' - Label 'Protein sequence composition'\n",
      "Entity 'http://edamontology.org/data_3086' - Label 'Nucleic acid sequence composition (report)'\n",
      "Entity 'http://edamontology.org/data_3101' - Label 'Protein domain classification node'\n",
      "Entity 'http://edamontology.org/data_3102' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_3103' - Label 'ATC code'\n",
      "Entity 'http://edamontology.org/data_3104' - Label 'UNII'\n",
      "Entity 'http://edamontology.org/data_3105' - Label 'Geotemporal metadata'\n",
      "Entity 'http://edamontology.org/data_3106' - Label 'System metadata'\n",
      "Entity 'http://edamontology.org/data_3107' - Label 'Sequence feature name'\n",
      "Entity 'http://edamontology.org/data_3108' - Label 'Experimental measurement'\n",
      "Entity 'http://edamontology.org/data_3110' - Label 'Raw microarray data'\n",
      "Entity 'http://edamontology.org/data_3111' - Label 'Processed microarray data'\n",
      "Entity 'http://edamontology.org/data_3112' - Label 'Gene expression matrix'\n",
      "Entity 'http://edamontology.org/data_3113' - Label 'Sample annotation'\n",
      "Entity 'http://edamontology.org/data_3115' - Label 'Microarray metadata'\n",
      "Entity 'http://edamontology.org/data_3116' - Label 'Microarray protocol annotation'\n",
      "Entity 'http://edamontology.org/data_3117' - Label 'Microarray hybridisation data'\n",
      "Entity 'http://edamontology.org/data_3119' - Label 'Sequence features (compositionally-biased regions)'\n",
      "Entity 'http://edamontology.org/data_3122' - Label 'Nucleic acid features (difference and change)'\n",
      "Entity 'http://edamontology.org/data_3128' - Label 'Nucleic acid structure report'\n",
      "Entity 'http://edamontology.org/data_3129' - Label 'Protein features report (repeats)'\n",
      "Entity 'http://edamontology.org/data_3130' - Label 'Sequence motif matches (protein)'\n",
      "Entity 'http://edamontology.org/data_3131' - Label 'Sequence motif matches (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_3132' - Label 'Nucleic acid features (d-loop)'\n",
      "Entity 'http://edamontology.org/data_3133' - Label 'Nucleic acid features (stem loop)'\n",
      "Entity 'http://edamontology.org/data_3134' - Label 'Gene transcript report'\n",
      "Entity 'http://edamontology.org/data_3137' - Label 'Non-coding RNA'\n",
      "Entity 'http://edamontology.org/data_3138' - Label 'Transcriptional features (report)'\n",
      "Entity 'http://edamontology.org/data_3140' - Label 'Nucleic acid features (immunoglobulin gene structure)'\n",
      "Entity 'http://edamontology.org/data_3141' - Label 'SCOP class'\n",
      "Entity 'http://edamontology.org/data_3142' - Label 'SCOP fold'\n",
      "Entity 'http://edamontology.org/data_3143' - Label 'SCOP superfamily'\n",
      "Entity 'http://edamontology.org/data_3144' - Label 'SCOP family'\n",
      "Entity 'http://edamontology.org/data_3145' - Label 'SCOP protein'\n",
      "Entity 'http://edamontology.org/data_3146' - Label 'SCOP species'\n",
      "Entity 'http://edamontology.org/data_3147' - Label 'Mass spectrometry experiment'\n",
      "Entity 'http://edamontology.org/data_3148' - Label 'Gene family report'\n",
      "Entity 'http://edamontology.org/data_3153' - Label 'Protein image'\n",
      "Entity 'http://edamontology.org/data_3154' - Label 'Protein alignment'\n",
      "Entity 'http://edamontology.org/data_3165' - Label 'NGS experiment'\n",
      "Entity 'http://edamontology.org/data_3181' - Label 'Sequence assembly report'\n",
      "Entity 'http://edamontology.org/data_3210' - Label 'Genome index'\n",
      "Entity 'http://edamontology.org/data_3231' - Label 'GWAS report'\n",
      "Entity 'http://edamontology.org/data_3236' - Label 'Cytoband position'\n",
      "Entity 'http://edamontology.org/data_3238' - Label 'Cell type ontology ID'\n",
      "Entity 'http://edamontology.org/data_3241' - Label 'Kinetic model'\n",
      "Entity 'http://edamontology.org/data_3264' - Label 'COSMIC ID'\n",
      "Entity 'http://edamontology.org/data_3265' - Label 'HGMD ID'\n",
      "Entity 'http://edamontology.org/data_3266' - Label 'Sequence assembly ID'\n",
      "Entity 'http://edamontology.org/data_3268' - Label 'Sequence feature type'\n",
      "Entity 'http://edamontology.org/data_3269' - Label 'Gene homology (report)'\n",
      "Entity 'http://edamontology.org/data_3270' - Label 'Ensembl gene tree ID'\n",
      "Entity 'http://edamontology.org/data_3271' - Label 'Gene tree'\n",
      "Entity 'http://edamontology.org/data_3272' - Label 'Species tree'\n",
      "Entity 'http://edamontology.org/data_3273' - Label 'Sample ID'\n",
      "Entity 'http://edamontology.org/data_3274' - Label 'MGI accession'\n",
      "Entity 'http://edamontology.org/data_3275' - Label 'Phenotype name'\n",
      "Entity 'http://edamontology.org/data_3354' - Label 'Transition matrix'\n",
      "Entity 'http://edamontology.org/data_3355' - Label 'Emission matrix'\n",
      "Entity 'http://edamontology.org/data_3356' - Label 'Hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_3358' - Label 'Format identifier'\n",
      "Entity 'http://edamontology.org/data_3424' - Label 'Raw image'\n",
      "Entity 'http://edamontology.org/data_3425' - Label 'Carbohydrate property'\n",
      "Entity 'http://edamontology.org/data_3426' - Label 'Proteomics experiment report'\n",
      "Entity 'http://edamontology.org/data_3427' - Label 'RNAi report'\n",
      "Entity 'http://edamontology.org/data_3428' - Label 'Simulation experiment report'\n",
      "Entity 'http://edamontology.org/data_3442' - Label 'MRI image'\n",
      "Entity 'http://edamontology.org/data_3449' - Label 'Cell migration track image'\n",
      "Entity 'http://edamontology.org/data_3451' - Label 'Rate of association'\n",
      "Entity 'http://edamontology.org/data_3479' - Label 'Gene order'\n",
      "Entity 'http://edamontology.org/data_3483' - Label 'Spectrum'\n",
      "Entity 'http://edamontology.org/data_3488' - Label 'NMR spectrum'\n",
      "Entity 'http://edamontology.org/data_3490' - Label 'Chemical structure sketch'\n",
      "Entity 'http://edamontology.org/data_3492' - Label 'Nucleic acid signature'\n",
      "Entity 'http://edamontology.org/data_3494' - Label 'DNA sequence'\n",
      "Entity 'http://edamontology.org/data_3495' - Label 'RNA sequence'\n",
      "Entity 'http://edamontology.org/data_3496' - Label 'RNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3497' - Label 'DNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3498' - Label 'Sequence variations'\n",
      "Entity 'http://edamontology.org/data_3505' - Label 'Bibliography'\n",
      "Entity 'http://edamontology.org/data_3509' - Label 'Ontology mapping'\n",
      "Entity 'http://edamontology.org/data_3546' - Label 'Image metadata'\n",
      "Entity 'http://edamontology.org/data_3558' - Label 'Clinical trial report'\n",
      "Entity 'http://edamontology.org/data_3567' - Label 'Reference sample report'\n",
      "Entity 'http://edamontology.org/data_3568' - Label 'Gene Expression Atlas Experiment ID'\n",
      "Entity 'http://edamontology.org/data_3667' - Label 'Disease identifier'\n",
      "Entity 'http://edamontology.org/data_3668' - Label 'Disease name'\n",
      "Entity 'http://edamontology.org/data_3669' - Label 'Training material'\n",
      "Entity 'http://edamontology.org/data_3670' - Label 'Online course'\n",
      "Entity 'http://edamontology.org/data_3671' - Label 'Text'\n",
      "Entity 'http://edamontology.org/data_3707' - Label 'Biodiversity data'\n",
      "Entity 'http://edamontology.org/data_3716' - Label 'Biosafety report'\n",
      "Entity 'http://edamontology.org/data_3717' - Label 'Isolation report'\n",
      "Entity 'http://edamontology.org/data_3718' - Label 'Pathogenicity report'\n",
      "Entity 'http://edamontology.org/data_3719' - Label 'Biosafety classification'\n",
      "Entity 'http://edamontology.org/data_3720' - Label 'Geographic location'\n",
      "Entity 'http://edamontology.org/data_3721' - Label 'Isolation source'\n",
      "Entity 'http://edamontology.org/data_3722' - Label 'Physiology parameter'\n",
      "Entity 'http://edamontology.org/data_3723' - Label 'Morphology parameter'\n",
      "Entity 'http://edamontology.org/data_3724' - Label 'Cultivation parameter'\n",
      "Entity 'http://edamontology.org/data_3732' - Label 'Sequencing metadata name'\n",
      "Entity 'http://edamontology.org/data_3733' - Label 'Flow cell identifier'\n",
      "Entity 'http://edamontology.org/data_3734' - Label 'Lane identifier'\n",
      "Entity 'http://edamontology.org/data_3735' - Label 'Run number'\n",
      "Entity 'http://edamontology.org/data_3736' - Label 'Ecological data'\n",
      "Entity 'http://edamontology.org/data_3737' - Label 'Alpha diversity data'\n",
      "Entity 'http://edamontology.org/data_3738' - Label 'Beta diversity data'\n",
      "Entity 'http://edamontology.org/data_3739' - Label 'Gamma diversity data'\n",
      "Entity 'http://edamontology.org/data_3743' - Label 'Ordination plot'\n",
      "Entity 'http://edamontology.org/data_3753' - Label 'Over-representation data'\n",
      "Entity 'http://edamontology.org/data_3754' - Label 'GO-term enrichment data'\n",
      "Entity 'http://edamontology.org/data_3756' - Label 'Localisation score'\n",
      "Entity 'http://edamontology.org/data_3757' - Label 'Unimod ID'\n",
      "Entity 'http://edamontology.org/data_3759' - Label 'ProteomeXchange ID'\n",
      "Entity 'http://edamontology.org/data_3768' - Label 'Clustered expression profiles'\n",
      "Entity 'http://edamontology.org/data_3769' - Label 'BRENDA ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_3779' - Label 'Annotated text'\n",
      "Entity 'http://edamontology.org/data_3786' - Label 'Query script'\n",
      "Entity 'http://edamontology.org/data_3805' - Label '3D EM Map'\n",
      "Entity 'http://edamontology.org/data_3806' - Label '3D EM Mask'\n",
      "Entity 'http://edamontology.org/data_3807' - Label 'EM Movie'\n",
      "Entity 'http://edamontology.org/data_3808' - Label 'EM Micrograph'\n",
      "Entity 'http://edamontology.org/data_3842' - Label 'Molecular simulation data'\n",
      "Entity 'http://edamontology.org/data_3856' - Label 'RNA central ID'\n",
      "Entity 'http://edamontology.org/data_3861' - Label 'Electronic health record'\n",
      "Entity 'http://edamontology.org/data_3869' - Label 'Simulation'\n",
      "Entity 'http://edamontology.org/data_3870' - Label 'Trajectory data'\n",
      "Entity 'http://edamontology.org/data_3871' - Label 'Forcefield parameters'\n",
      "Entity 'http://edamontology.org/data_3872' - Label 'Topology data'\n",
      "Entity 'http://edamontology.org/data_3905' - Label 'Histogram'\n",
      "Entity 'http://edamontology.org/data_3914' - Label 'Quality control report'\n",
      "Entity 'http://edamontology.org/data_3917' - Label 'Count matrix'\n",
      "Entity 'http://edamontology.org/data_3924' - Label 'DNA structure alignment'\n",
      "Entity 'http://edamontology.org/data_3932' - Label 'Q-value'\n",
      "Entity 'http://edamontology.org/data_3949' - Label 'Profile HMM'\n",
      "Entity 'http://edamontology.org/data_3952' - Label 'Pathway ID (WikiPathways)'\n",
      "Entity 'http://edamontology.org/data_3953' - Label 'Pathway overrepresentation data'\n",
      "Entity 'http://edamontology.org/data_4022' - Label 'ORCID Identifier'\n",
      "Entity 'http://edamontology.org/format_1196' - Label 'SMILES'\n",
      "Entity 'http://edamontology.org/format_1197' - Label 'InChI'\n",
      "Entity 'http://edamontology.org/format_1198' - Label 'mf'\n",
      "Entity 'http://edamontology.org/format_1199' - Label 'InChIKey'\n",
      "Entity 'http://edamontology.org/format_1200' - Label 'smarts'\n",
      "Entity 'http://edamontology.org/format_1206' - Label 'unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_1207' - Label 'nucleotide'\n",
      "Entity 'http://edamontology.org/format_1208' - Label 'protein'\n",
      "Entity 'http://edamontology.org/format_1209' - Label 'consensus'\n",
      "Entity 'http://edamontology.org/format_1210' - Label 'pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1211' - Label 'unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1212' - Label 'dna'\n",
      "Entity 'http://edamontology.org/format_1213' - Label 'rna'\n",
      "Entity 'http://edamontology.org/format_1214' - Label 'unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_1215' - Label 'pure dna'\n",
      "Entity 'http://edamontology.org/format_1216' - Label 'unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_1217' - Label 'pure rna'\n",
      "Entity 'http://edamontology.org/format_1218' - Label 'unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_1219' - Label 'pure protein'\n",
      "Entity 'http://edamontology.org/format_1228' - Label 'UniGene entry format'\n",
      "Entity 'http://edamontology.org/format_1247' - Label 'COG sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_1248' - Label 'EMBL feature location'\n",
      "Entity 'http://edamontology.org/format_1295' - Label 'quicktandem'\n",
      "Entity 'http://edamontology.org/format_1296' - Label 'Sanger inverted repeats'\n",
      "Entity 'http://edamontology.org/format_1297' - Label 'EMBOSS repeat'\n",
      "Entity 'http://edamontology.org/format_1316' - Label 'est2genome format'\n",
      "Entity 'http://edamontology.org/format_1318' - Label 'restrict format'\n",
      "Entity 'http://edamontology.org/format_1319' - Label 'restover format'\n",
      "Entity 'http://edamontology.org/format_1320' - Label 'REBASE restriction sites'\n",
      "Entity 'http://edamontology.org/format_1332' - Label 'FASTA search results format'\n",
      "Entity 'http://edamontology.org/format_1333' - Label 'BLAST results'\n",
      "Entity 'http://edamontology.org/format_1334' - Label 'mspcrunch'\n",
      "Entity 'http://edamontology.org/format_1335' - Label 'Smith-Waterman format'\n",
      "Entity 'http://edamontology.org/format_1336' - Label 'dhf'\n",
      "Entity 'http://edamontology.org/format_1337' - Label 'lhf'\n",
      "Entity 'http://edamontology.org/format_1341' - Label 'InterPro hits format'\n",
      "Entity 'http://edamontology.org/format_1342' - Label 'InterPro protein view report format'\n",
      "Entity 'http://edamontology.org/format_1343' - Label 'InterPro match table format'\n",
      "Entity 'http://edamontology.org/format_1349' - Label 'HMMER Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1350' - Label 'MEME Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1351' - Label 'HMMER emission and transition'\n",
      "Entity 'http://edamontology.org/format_1356' - Label 'prosite-pattern'\n",
      "Entity 'http://edamontology.org/format_1357' - Label 'EMBOSS sequence pattern'\n",
      "Entity 'http://edamontology.org/format_1360' - Label 'meme-motif'\n",
      "Entity 'http://edamontology.org/format_1366' - Label 'prosite-profile'\n",
      "Entity 'http://edamontology.org/format_1367' - Label 'JASPAR format'\n",
      "Entity 'http://edamontology.org/format_1369' - Label 'MEME background Markov model'\n",
      "Entity 'http://edamontology.org/format_1370' - Label 'HMMER format'\n",
      "Entity 'http://edamontology.org/format_1391' - Label 'HMMER-aln'\n",
      "Entity 'http://edamontology.org/format_1392' - Label 'DIALIGN format'\n",
      "Entity 'http://edamontology.org/format_1393' - Label 'daf'\n",
      "Entity 'http://edamontology.org/format_1419' - Label 'Sequence-MEME profile alignment'\n",
      "Entity 'http://edamontology.org/format_1421' - Label 'HMMER profile alignment (sequences versus HMMs)'\n",
      "Entity 'http://edamontology.org/format_1422' - Label 'HMMER profile alignment (HMM versus sequences)'\n",
      "Entity 'http://edamontology.org/format_1423' - Label 'Phylip distance matrix'\n",
      "Entity 'http://edamontology.org/format_1424' - Label 'ClustalW dendrogram'\n",
      "Entity 'http://edamontology.org/format_1425' - Label 'Phylip tree raw'\n",
      "Entity 'http://edamontology.org/format_1430' - Label 'Phylip continuous quantitative characters'\n",
      "Entity 'http://edamontology.org/format_1431' - Label 'Phylogenetic property values format'\n",
      "Entity 'http://edamontology.org/format_1432' - Label 'Phylip character frequencies format'\n",
      "Entity 'http://edamontology.org/format_1433' - Label 'Phylip discrete states format'\n",
      "Entity 'http://edamontology.org/format_1434' - Label 'Phylip cliques format'\n",
      "Entity 'http://edamontology.org/format_1435' - Label 'Phylip tree format'\n",
      "Entity 'http://edamontology.org/format_1436' - Label 'TreeBASE format'\n",
      "Entity 'http://edamontology.org/format_1437' - Label 'TreeFam format'\n",
      "Entity 'http://edamontology.org/format_1445' - Label 'Phylip tree distance format'\n",
      "Entity 'http://edamontology.org/format_1454' - Label 'dssp'\n",
      "Entity 'http://edamontology.org/format_1455' - Label 'hssp'\n",
      "Entity 'http://edamontology.org/format_1457' - Label 'Dot-bracket format'\n",
      "Entity 'http://edamontology.org/format_1458' - Label 'Vienna local RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_1475' - Label 'PDB database entry format'\n",
      "Entity 'http://edamontology.org/format_1476' - Label 'PDB'\n",
      "Entity 'http://edamontology.org/format_1477' - Label 'mmCIF'\n",
      "Entity 'http://edamontology.org/format_1478' - Label 'PDBML'\n",
      "Entity 'http://edamontology.org/format_1500' - Label 'Domainatrix 3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_1504' - Label 'aaindex'\n",
      "Entity 'http://edamontology.org/format_1511' - Label 'IntEnz enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1512' - Label 'BRENDA enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1513' - Label 'KEGG REACTION enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1514' - Label 'KEGG ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1515' - Label 'REBASE proto enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1516' - Label 'REBASE withrefm enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1551' - Label 'Pcons report format'\n",
      "Entity 'http://edamontology.org/format_1552' - Label 'ProQ report format'\n",
      "Entity 'http://edamontology.org/format_1563' - Label 'SMART domain assignment report format'\n",
      "Entity 'http://edamontology.org/format_1568' - Label 'BIND entry format'\n",
      "Entity 'http://edamontology.org/format_1569' - Label 'IntAct entry format'\n",
      "Entity 'http://edamontology.org/format_1570' - Label 'InterPro entry format'\n",
      "Entity 'http://edamontology.org/format_1571' - Label 'InterPro entry abstract format'\n",
      "Entity 'http://edamontology.org/format_1572' - Label 'Gene3D entry format'\n",
      "Entity 'http://edamontology.org/format_1573' - Label 'PIRSF entry format'\n",
      "Entity 'http://edamontology.org/format_1574' - Label 'PRINTS entry format'\n",
      "Entity 'http://edamontology.org/format_1575' - Label 'Panther Families and HMMs entry format'\n",
      "Entity 'http://edamontology.org/format_1576' - Label 'Pfam entry format'\n",
      "Entity 'http://edamontology.org/format_1577' - Label 'SMART entry format'\n",
      "Entity 'http://edamontology.org/format_1578' - Label 'Superfamily entry format'\n",
      "Entity 'http://edamontology.org/format_1579' - Label 'TIGRFam entry format'\n",
      "Entity 'http://edamontology.org/format_1580' - Label 'ProDom entry format'\n",
      "Entity 'http://edamontology.org/format_1581' - Label 'FSSP entry format'\n",
      "Entity 'http://edamontology.org/format_1582' - Label 'findkm'\n",
      "Entity 'http://edamontology.org/format_1603' - Label 'Ensembl gene report format'\n",
      "Entity 'http://edamontology.org/format_1604' - Label 'DictyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1605' - Label 'CGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1606' - Label 'DragonDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1607' - Label 'EcoCyc gene report format'\n",
      "Entity 'http://edamontology.org/format_1608' - Label 'FlyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1609' - Label 'Gramene gene report format'\n",
      "Entity 'http://edamontology.org/format_1610' - Label 'KEGG GENES gene report format'\n",
      "Entity 'http://edamontology.org/format_1611' - Label 'MaizeGDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1612' - Label 'MGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1613' - Label 'RGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1614' - Label 'SGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1615' - Label 'GeneDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1616' - Label 'TAIR gene report format'\n",
      "Entity 'http://edamontology.org/format_1617' - Label 'WormBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1618' - Label 'ZFIN gene report format'\n",
      "Entity 'http://edamontology.org/format_1619' - Label 'TIGR gene report format'\n",
      "Entity 'http://edamontology.org/format_1620' - Label 'dbSNP polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_1623' - Label 'OMIM entry format'\n",
      "Entity 'http://edamontology.org/format_1624' - Label 'HGVbase entry format'\n",
      "Entity 'http://edamontology.org/format_1625' - Label 'HIVDB entry format'\n",
      "Entity 'http://edamontology.org/format_1626' - Label 'KEGG DISEASE entry format'\n",
      "Entity 'http://edamontology.org/format_1627' - Label 'Primer3 primer'\n",
      "Entity 'http://edamontology.org/format_1628' - Label 'ABI'\n",
      "Entity 'http://edamontology.org/format_1629' - Label 'mira'\n",
      "Entity 'http://edamontology.org/format_1630' - Label 'CAF'\n",
      "Entity 'http://edamontology.org/format_1631' - Label 'EXP'\n",
      "Entity 'http://edamontology.org/format_1632' - Label 'SCF'\n",
      "Entity 'http://edamontology.org/format_1633' - Label 'PHD'\n",
      "Entity 'http://edamontology.org/format_1637' - Label 'dat'\n",
      "Entity 'http://edamontology.org/format_1638' - Label 'cel'\n",
      "Entity 'http://edamontology.org/format_1639' - Label 'affymetrix'\n",
      "Entity 'http://edamontology.org/format_1640' - Label 'ArrayExpress entry format'\n",
      "Entity 'http://edamontology.org/format_1641' - Label 'affymetrix-exp'\n",
      "Entity 'http://edamontology.org/format_1644' - Label 'CHP'\n",
      "Entity 'http://edamontology.org/format_1645' - Label 'EMDB entry format'\n",
      "Entity 'http://edamontology.org/format_1647' - Label 'KEGG PATHWAY entry format'\n",
      "Entity 'http://edamontology.org/format_1648' - Label 'MetaCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1649' - Label 'HumanCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1650' - Label 'INOH entry format'\n",
      "Entity 'http://edamontology.org/format_1651' - Label 'PATIKA entry format'\n",
      "Entity 'http://edamontology.org/format_1652' - Label 'Reactome entry format'\n",
      "Entity 'http://edamontology.org/format_1653' - Label 'aMAZE entry format'\n",
      "Entity 'http://edamontology.org/format_1654' - Label 'CPDB entry format'\n",
      "Entity 'http://edamontology.org/format_1655' - Label 'Panther Pathways entry format'\n",
      "Entity 'http://edamontology.org/format_1665' - Label 'Taverna workflow format'\n",
      "Entity 'http://edamontology.org/format_1666' - Label 'BioModel mathematical model format'\n",
      "Entity 'http://edamontology.org/format_1697' - Label 'KEGG LIGAND entry format'\n",
      "Entity 'http://edamontology.org/format_1698' - Label 'KEGG COMPOUND entry format'\n",
      "Entity 'http://edamontology.org/format_1699' - Label 'KEGG PLANT entry format'\n",
      "Entity 'http://edamontology.org/format_1700' - Label 'KEGG GLYCAN entry format'\n",
      "Entity 'http://edamontology.org/format_1701' - Label 'PubChem entry format'\n",
      "Entity 'http://edamontology.org/format_1702' - Label 'ChemSpider entry format'\n",
      "Entity 'http://edamontology.org/format_1703' - Label 'ChEBI entry format'\n",
      "Entity 'http://edamontology.org/format_1704' - Label 'MSDchem ligand dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1705' - Label 'HET group dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1706' - Label 'KEGG DRUG entry format'\n",
      "Entity 'http://edamontology.org/format_1734' - Label 'PubMed citation'\n",
      "Entity 'http://edamontology.org/format_1735' - Label 'Medline Display Format'\n",
      "Entity 'http://edamontology.org/format_1736' - Label 'CiteXplore-core'\n",
      "Entity 'http://edamontology.org/format_1737' - Label 'CiteXplore-all'\n",
      "Entity 'http://edamontology.org/format_1739' - Label 'pmc'\n",
      "Entity 'http://edamontology.org/format_1740' - Label 'iHOP format'\n",
      "Entity 'http://edamontology.org/format_1741' - Label 'OSCAR format'\n",
      "Entity 'http://edamontology.org/format_1747' - Label 'PDB atom record format'\n",
      "Entity 'http://edamontology.org/format_1760' - Label 'CATH chain report format'\n",
      "Entity 'http://edamontology.org/format_1761' - Label 'CATH PDB report format'\n",
      "Entity 'http://edamontology.org/format_1782' - Label 'NCBI gene report format'\n",
      "Entity 'http://edamontology.org/format_1808' - Label 'GeneIlluminator gene report format'\n",
      "Entity 'http://edamontology.org/format_1809' - Label 'BacMap gene card format'\n",
      "Entity 'http://edamontology.org/format_1810' - Label 'ColiCard report format'\n",
      "Entity 'http://edamontology.org/format_1861' - Label 'PlasMapper TextMap'\n",
      "Entity 'http://edamontology.org/format_1910' - Label 'newick'\n",
      "Entity 'http://edamontology.org/format_1911' - Label 'TreeCon format'\n",
      "Entity 'http://edamontology.org/format_1912' - Label 'Nexus format'\n",
      "Entity 'http://edamontology.org/format_1915' - Label 'Format'\n",
      "Entity 'http://edamontology.org/format_1918' - Label 'Atomic data format'\n",
      "Entity 'http://edamontology.org/format_1919' - Label 'Sequence record format'\n",
      "Entity 'http://edamontology.org/format_1920' - Label 'Sequence feature annotation format'\n",
      "Entity 'http://edamontology.org/format_1921' - Label 'Alignment format'\n",
      "Entity 'http://edamontology.org/format_1923' - Label 'acedb'\n",
      "Entity 'http://edamontology.org/format_1924' - Label 'clustal sequence format'\n",
      "Entity 'http://edamontology.org/format_1925' - Label 'codata'\n",
      "Entity 'http://edamontology.org/format_1926' - Label 'dbid'\n",
      "Entity 'http://edamontology.org/format_1927' - Label 'EMBL format'\n",
      "Entity 'http://edamontology.org/format_1928' - Label 'Staden experiment format'\n",
      "Entity 'http://edamontology.org/format_1929' - Label 'FASTA'\n",
      "Entity 'http://edamontology.org/format_1930' - Label 'FASTQ'\n",
      "Entity 'http://edamontology.org/format_1931' - Label 'FASTQ-illumina'\n",
      "Entity 'http://edamontology.org/format_1932' - Label 'FASTQ-sanger'\n",
      "Entity 'http://edamontology.org/format_1933' - Label 'FASTQ-solexa'\n",
      "Entity 'http://edamontology.org/format_1934' - Label 'fitch program'\n",
      "Entity 'http://edamontology.org/format_1935' - Label 'GCG'\n",
      "Entity 'http://edamontology.org/format_1936' - Label 'GenBank format'\n",
      "Entity 'http://edamontology.org/format_1937' - Label 'genpept'\n",
      "Entity 'http://edamontology.org/format_1938' - Label 'GFF2-seq'\n",
      "Entity 'http://edamontology.org/format_1939' - Label 'GFF3-seq'\n",
      "Entity 'http://edamontology.org/format_1940' - Label 'giFASTA format'\n",
      "Entity 'http://edamontology.org/format_1941' - Label 'hennig86'\n",
      "Entity 'http://edamontology.org/format_1942' - Label 'ig'\n",
      "Entity 'http://edamontology.org/format_1943' - Label 'igstrict'\n",
      "Entity 'http://edamontology.org/format_1944' - Label 'jackknifer'\n",
      "Entity 'http://edamontology.org/format_1945' - Label 'mase format'\n",
      "Entity 'http://edamontology.org/format_1946' - Label 'mega-seq'\n",
      "Entity 'http://edamontology.org/format_1947' - Label 'GCG MSF'\n",
      "Entity 'http://edamontology.org/format_1948' - Label 'nbrf/pir'\n",
      "Entity 'http://edamontology.org/format_1949' - Label 'nexus-seq'\n",
      "Entity 'http://edamontology.org/format_1950' - Label 'pdbatom'\n",
      "Entity 'http://edamontology.org/format_1951' - Label 'pdbatomnuc'\n",
      "Entity 'http://edamontology.org/format_1952' - Label 'pdbseqresnuc'\n",
      "Entity 'http://edamontology.org/format_1953' - Label 'pdbseqres'\n",
      "Entity 'http://edamontology.org/format_1954' - Label 'Pearson format'\n",
      "Entity 'http://edamontology.org/format_1955' - Label 'phylip sequence format'\n",
      "Entity 'http://edamontology.org/format_1956' - Label 'phylipnon sequence format'\n",
      "Entity 'http://edamontology.org/format_1957' - Label 'raw'\n",
      "Entity 'http://edamontology.org/format_1958' - Label 'refseqp'\n",
      "Entity 'http://edamontology.org/format_1959' - Label 'selex sequence format'\n",
      "Entity 'http://edamontology.org/format_1960' - Label 'Staden format'\n",
      "Entity 'http://edamontology.org/format_1961' - Label 'Stockholm format'\n",
      "Entity 'http://edamontology.org/format_1962' - Label 'strider format'\n",
      "Entity 'http://edamontology.org/format_1963' - Label 'UniProtKB format'\n",
      "Entity 'http://edamontology.org/format_1964' - Label 'plain text format (unformatted)'\n",
      "Entity 'http://edamontology.org/format_1965' - Label 'treecon sequence format'\n",
      "Entity 'http://edamontology.org/format_1966' - Label 'ASN.1 sequence format'\n",
      "Entity 'http://edamontology.org/format_1967' - Label 'DAS format'\n",
      "Entity 'http://edamontology.org/format_1968' - Label 'dasdna'\n",
      "Entity 'http://edamontology.org/format_1969' - Label 'debug-seq'\n",
      "Entity 'http://edamontology.org/format_1970' - Label 'jackknifernon'\n",
      "Entity 'http://edamontology.org/format_1971' - Label 'meganon sequence format'\n",
      "Entity 'http://edamontology.org/format_1972' - Label 'NCBI format'\n",
      "Entity 'http://edamontology.org/format_1973' - Label 'nexusnon'\n",
      "Entity 'http://edamontology.org/format_1974' - Label 'GFF2'\n",
      "Entity 'http://edamontology.org/format_1975' - Label 'GFF3'\n",
      "Entity 'http://edamontology.org/format_1976' - Label 'pir'\n",
      "Entity 'http://edamontology.org/format_1977' - Label 'swiss feature'\n",
      "Entity 'http://edamontology.org/format_1978' - Label 'DASGFF'\n",
      "Entity 'http://edamontology.org/format_1979' - Label 'debug-feat'\n",
      "Entity 'http://edamontology.org/format_1980' - Label 'EMBL feature'\n",
      "Entity 'http://edamontology.org/format_1981' - Label 'GenBank feature'\n",
      "Entity 'http://edamontology.org/format_1982' - Label 'ClustalW format'\n",
      "Entity 'http://edamontology.org/format_1983' - Label 'debug'\n",
      "Entity 'http://edamontology.org/format_1984' - Label 'FASTA-aln'\n",
      "Entity 'http://edamontology.org/format_1985' - Label 'markx0'\n",
      "Entity 'http://edamontology.org/format_1986' - Label 'markx1'\n",
      "Entity 'http://edamontology.org/format_1987' - Label 'markx10'\n",
      "Entity 'http://edamontology.org/format_1988' - Label 'markx2'\n",
      "Entity 'http://edamontology.org/format_1989' - Label 'markx3'\n",
      "Entity 'http://edamontology.org/format_1990' - Label 'match'\n",
      "Entity 'http://edamontology.org/format_1991' - Label 'mega'\n",
      "Entity 'http://edamontology.org/format_1992' - Label 'meganon'\n",
      "Entity 'http://edamontology.org/format_1993' - Label 'msf alignment format'\n",
      "Entity 'http://edamontology.org/format_1994' - Label 'nexus alignment format'\n",
      "Entity 'http://edamontology.org/format_1995' - Label 'nexusnon alignment format'\n",
      "Entity 'http://edamontology.org/format_1996' - Label 'pair'\n",
      "Entity 'http://edamontology.org/format_1997' - Label 'PHYLIP format'\n",
      "Entity 'http://edamontology.org/format_1998' - Label 'PHYLIP sequential'\n",
      "Entity 'http://edamontology.org/format_1999' - Label 'scores format'\n",
      "Entity 'http://edamontology.org/format_2000' - Label 'selex'\n",
      "Entity 'http://edamontology.org/format_2001' - Label 'EMBOSS simple format'\n",
      "Entity 'http://edamontology.org/format_2002' - Label 'srs format'\n",
      "Entity 'http://edamontology.org/format_2003' - Label 'srspair'\n",
      "Entity 'http://edamontology.org/format_2004' - Label 'T-Coffee format'\n",
      "Entity 'http://edamontology.org/format_2005' - Label 'TreeCon-seq'\n",
      "Entity 'http://edamontology.org/format_2006' - Label 'Phylogenetic tree format'\n",
      "Entity 'http://edamontology.org/format_2013' - Label 'Biological pathway or network format'\n",
      "Entity 'http://edamontology.org/format_2014' - Label 'Sequence-profile alignment format'\n",
      "Entity 'http://edamontology.org/format_2015' - Label 'Sequence-profile alignment (HMM) format'\n",
      "Entity 'http://edamontology.org/format_2017' - Label 'Amino acid index format'\n",
      "Entity 'http://edamontology.org/format_2020' - Label 'Article format'\n",
      "Entity 'http://edamontology.org/format_2021' - Label 'Text mining report format'\n",
      "Entity 'http://edamontology.org/format_2027' - Label 'Enzyme kinetics report format'\n",
      "Entity 'http://edamontology.org/format_2030' - Label 'Chemical data format'\n",
      "Entity 'http://edamontology.org/format_2031' - Label 'Gene annotation format'\n",
      "Entity 'http://edamontology.org/format_2032' - Label 'Workflow format'\n",
      "Entity 'http://edamontology.org/format_2033' - Label 'Tertiary structure format'\n",
      "Entity 'http://edamontology.org/format_2034' - Label 'Biological model format'\n",
      "Entity 'http://edamontology.org/format_2035' - Label 'Chemical formula format'\n",
      "Entity 'http://edamontology.org/format_2036' - Label 'Phylogenetic character data format'\n",
      "Entity 'http://edamontology.org/format_2037' - Label 'Phylogenetic continuous quantitative character format'\n",
      "Entity 'http://edamontology.org/format_2038' - Label 'Phylogenetic discrete states format'\n",
      "Entity 'http://edamontology.org/format_2039' - Label 'Phylogenetic tree report (cliques) format'\n",
      "Entity 'http://edamontology.org/format_2040' - Label 'Phylogenetic tree report (invariants) format'\n",
      "Entity 'http://edamontology.org/format_2045' - Label 'Electron microscopy model format'\n",
      "Entity 'http://edamontology.org/format_2049' - Label 'Phylogenetic tree report (tree distances) format'\n",
      "Entity 'http://edamontology.org/format_2051' - Label 'Polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_2052' - Label 'Protein family report format'\n",
      "Entity 'http://edamontology.org/format_2054' - Label 'Protein interaction format'\n",
      "Entity 'http://edamontology.org/format_2055' - Label 'Sequence assembly format'\n",
      "Entity 'http://edamontology.org/format_2056' - Label 'Microarray experiment data format'\n",
      "Entity 'http://edamontology.org/format_2057' - Label 'Sequence trace format'\n",
      "Entity 'http://edamontology.org/format_2058' - Label 'Gene expression report format'\n",
      "Entity 'http://edamontology.org/format_2059' - Label 'Genotype and phenotype annotation format'\n",
      "Entity 'http://edamontology.org/format_2060' - Label 'Map format'\n",
      "Entity 'http://edamontology.org/format_2061' - Label 'Nucleic acid features (primers) format'\n",
      "Entity 'http://edamontology.org/format_2062' - Label 'Protein report format'\n",
      "Entity 'http://edamontology.org/format_2063' - Label 'Protein report (enzyme) format'\n",
      "Entity 'http://edamontology.org/format_2064' - Label '3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_2065' - Label 'Protein structure report (quality evaluation) format'\n",
      "Entity 'http://edamontology.org/format_2066' - Label 'Database hits (sequence) format'\n",
      "Entity 'http://edamontology.org/format_2067' - Label 'Sequence distance matrix format'\n",
      "Entity 'http://edamontology.org/format_2068' - Label 'Sequence motif format'\n",
      "Entity 'http://edamontology.org/format_2069' - Label 'Sequence profile format'\n",
      "Entity 'http://edamontology.org/format_2072' - Label 'Hidden Markov model format'\n",
      "Entity 'http://edamontology.org/format_2074' - Label 'Dirichlet distribution format'\n",
      "Entity 'http://edamontology.org/format_2075' - Label 'HMM emission and transition counts format'\n",
      "Entity 'http://edamontology.org/format_2076' - Label 'RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2077' - Label 'Protein secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2078' - Label 'Sequence range format'\n",
      "Entity 'http://edamontology.org/format_2094' - Label 'pure'\n",
      "Entity 'http://edamontology.org/format_2095' - Label 'unpure'\n",
      "Entity 'http://edamontology.org/format_2096' - Label 'unambiguous sequence'\n",
      "Entity 'http://edamontology.org/format_2097' - Label 'ambiguous'\n",
      "Entity 'http://edamontology.org/format_2155' - Label 'Sequence features (repeats) format'\n",
      "Entity 'http://edamontology.org/format_2158' - Label 'Nucleic acid features (restriction sites) format'\n",
      "Entity 'http://edamontology.org/format_2159' - Label 'Gene features (coding region) format'\n",
      "Entity 'http://edamontology.org/format_2170' - Label 'Sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_2171' - Label 'Sequence cluster format (protein)'\n",
      "Entity 'http://edamontology.org/format_2172' - Label 'Sequence cluster format (nucleic acid)'\n",
      "Entity 'http://edamontology.org/format_2175' - Label 'Gene cluster format'\n",
      "Entity 'http://edamontology.org/format_2181' - Label 'EMBL-like (text)'\n",
      "Entity 'http://edamontology.org/format_2182' - Label 'FASTQ-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2183' - Label 'EMBLXML'\n",
      "Entity 'http://edamontology.org/format_2184' - Label 'cdsxml'\n",
      "Entity 'http://edamontology.org/format_2185' - Label 'INSDSeq'\n",
      "Entity 'http://edamontology.org/format_2186' - Label 'geneseq'\n",
      "Entity 'http://edamontology.org/format_2187' - Label 'UniProt-like (text)'\n",
      "Entity 'http://edamontology.org/format_2188' - Label 'UniProt format'\n",
      "Entity 'http://edamontology.org/format_2189' - Label 'ipi'\n",
      "Entity 'http://edamontology.org/format_2194' - Label 'medline'\n",
      "Entity 'http://edamontology.org/format_2195' - Label 'Ontology format'\n",
      "Entity 'http://edamontology.org/format_2196' - Label 'OBO format'\n",
      "Entity 'http://edamontology.org/format_2197' - Label 'OWL format'\n",
      "Entity 'http://edamontology.org/format_2200' - Label 'FASTA-like (text)'\n",
      "Entity 'http://edamontology.org/format_2202' - Label 'Sequence record full format'\n",
      "Entity 'http://edamontology.org/format_2203' - Label 'Sequence record lite format'\n",
      "Entity 'http://edamontology.org/format_2204' - Label 'EMBL format (XML)'\n",
      "Entity 'http://edamontology.org/format_2205' - Label 'GenBank-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2206' - Label 'Sequence feature table format (text)'\n",
      "Entity 'http://edamontology.org/format_2210' - Label 'Strain data format'\n",
      "Entity 'http://edamontology.org/format_2211' - Label 'CIP strain data format'\n",
      "Entity 'http://edamontology.org/format_2243' - Label 'phylip property values'\n",
      "Entity 'http://edamontology.org/format_2303' - Label 'STRING entry format (HTML)'\n",
      "Entity 'http://edamontology.org/format_2304' - Label 'STRING entry format (XML)'\n",
      "Entity 'http://edamontology.org/format_2305' - Label 'GFF'\n",
      "Entity 'http://edamontology.org/format_2306' - Label 'GTF'\n",
      "Entity 'http://edamontology.org/format_2310' - Label 'FASTA-HTML'\n",
      "Entity 'http://edamontology.org/format_2311' - Label 'EMBL-HTML'\n",
      "Entity 'http://edamontology.org/format_2322' - Label 'BioCyc enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2323' - Label 'ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2328' - Label 'PseudoCAP gene report format'\n",
      "Entity 'http://edamontology.org/format_2329' - Label 'GeneCards gene report format'\n",
      "Entity 'http://edamontology.org/format_2330' - Label 'Textual format'\n",
      "Entity 'http://edamontology.org/format_2331' - Label 'HTML'\n",
      "Entity 'http://edamontology.org/format_2332' - Label 'XML'\n",
      "Entity 'http://edamontology.org/format_2333' - Label 'Binary format'\n",
      "Entity 'http://edamontology.org/format_2334' - Label 'URI format'\n",
      "Entity 'http://edamontology.org/format_2341' - Label 'NCI-Nature pathway entry format'\n",
      "Entity 'http://edamontology.org/format_2350' - Label 'Format (by type of data)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2376' - Label 'RDF format'\n",
      "Entity 'http://edamontology.org/format_2532' - Label 'GenBank-HTML'\n",
      "Entity 'http://edamontology.org/format_2542' - Label 'Protein features (domains) format'\n",
      "Entity 'http://edamontology.org/format_2543' - Label 'EMBL-like format'\n",
      "Entity 'http://edamontology.org/format_2545' - Label 'FASTQ-like format'\n",
      "Entity 'http://edamontology.org/format_2546' - Label 'FASTA-like'\n",
      "Entity 'http://edamontology.org/format_2547' - Label 'uniprotkb-like format'\n",
      "Entity 'http://edamontology.org/format_2548' - Label 'Sequence feature table format'\n",
      "Entity 'http://edamontology.org/format_2549' - Label 'OBO'\n",
      "Entity 'http://edamontology.org/format_2550' - Label 'OBO-XML'\n",
      "Entity 'http://edamontology.org/format_2551' - Label 'Sequence record format (text)'\n",
      "Entity 'http://edamontology.org/format_2552' - Label 'Sequence record format (XML)'\n",
      "Entity 'http://edamontology.org/format_2553' - Label 'Sequence feature table format (XML)'\n",
      "Entity 'http://edamontology.org/format_2554' - Label 'Alignment format (text)'\n",
      "Entity 'http://edamontology.org/format_2555' - Label 'Alignment format (XML)'\n",
      "Entity 'http://edamontology.org/format_2556' - Label 'Phylogenetic tree format (text)'\n",
      "Entity 'http://edamontology.org/format_2557' - Label 'Phylogenetic tree format (XML)'\n",
      "Entity 'http://edamontology.org/format_2558' - Label 'EMBL-like (XML)'\n",
      "Entity 'http://edamontology.org/format_2559' - Label 'GenBank-like format'\n",
      "Entity 'http://edamontology.org/format_2560' - Label 'STRING entry format'\n",
      "Entity 'http://edamontology.org/format_2561' - Label 'Sequence assembly format (text)'\n",
      "Entity 'http://edamontology.org/format_2562' - Label 'Amino acid identifier format'\n",
      "Entity 'http://edamontology.org/format_2566' - Label 'completely unambiguous'\n",
      "Entity 'http://edamontology.org/format_2567' - Label 'completely unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_2568' - Label 'completely unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_2569' - Label 'completely unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_2570' - Label 'completely unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_2571' - Label 'Raw sequence format'\n",
      "Entity 'http://edamontology.org/format_2572' - Label 'BAM'\n",
      "Entity 'http://edamontology.org/format_2573' - Label 'SAM'\n",
      "Entity 'http://edamontology.org/format_2585' - Label 'SBML'\n",
      "Entity 'http://edamontology.org/format_2607' - Label 'completely unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_2848' - Label 'Bibliographic reference format'\n",
      "Entity 'http://edamontology.org/format_2919' - Label 'Sequence annotation track format'\n",
      "Entity 'http://edamontology.org/format_2920' - Label 'Alignment format (pair only)'\n",
      "Entity 'http://edamontology.org/format_2921' - Label 'Sequence variation annotation format'\n",
      "Entity 'http://edamontology.org/format_2922' - Label 'markx0 variant'\n",
      "Entity 'http://edamontology.org/format_2923' - Label 'mega variant'\n",
      "Entity 'http://edamontology.org/format_2924' - Label 'Phylip format variant'\n",
      "Entity 'http://edamontology.org/format_3000' - Label 'AB1'\n",
      "Entity 'http://edamontology.org/format_3001' - Label 'ACE'\n",
      "Entity 'http://edamontology.org/format_3003' - Label 'BED'\n",
      "Entity 'http://edamontology.org/format_3004' - Label 'bigBed'\n",
      "Entity 'http://edamontology.org/format_3005' - Label 'WIG'\n",
      "Entity 'http://edamontology.org/format_3006' - Label 'bigWig'\n",
      "Entity 'http://edamontology.org/format_3007' - Label 'PSL'\n",
      "Entity 'http://edamontology.org/format_3008' - Label 'MAF'\n",
      "Entity 'http://edamontology.org/format_3009' - Label '2bit'\n",
      "Entity 'http://edamontology.org/format_3010' - Label '.nib'\n",
      "Entity 'http://edamontology.org/format_3011' - Label 'genePred'\n",
      "Entity 'http://edamontology.org/format_3012' - Label 'pgSnp'\n",
      "Entity 'http://edamontology.org/format_3013' - Label 'axt'\n",
      "Entity 'http://edamontology.org/format_3014' - Label 'LAV'\n",
      "Entity 'http://edamontology.org/format_3015' - Label 'Pileup'\n",
      "Entity 'http://edamontology.org/format_3016' - Label 'VCF'\n",
      "Entity 'http://edamontology.org/format_3017' - Label 'SRF'\n",
      "Entity 'http://edamontology.org/format_3018' - Label 'ZTR'\n",
      "Entity 'http://edamontology.org/format_3019' - Label 'GVF'\n",
      "Entity 'http://edamontology.org/format_3020' - Label 'BCF'\n",
      "Entity 'http://edamontology.org/format_3033' - Label 'Matrix format'\n",
      "Entity 'http://edamontology.org/format_3097' - Label 'Protein domain classification format'\n",
      "Entity 'http://edamontology.org/format_3098' - Label 'Raw SCOP domain classification format'\n",
      "Entity 'http://edamontology.org/format_3099' - Label 'Raw CATH domain classification format'\n",
      "Entity 'http://edamontology.org/format_3100' - Label 'CATH domain report format'\n",
      "Entity 'http://edamontology.org/format_3155' - Label 'SBRML'\n",
      "Entity 'http://edamontology.org/format_3156' - Label 'BioPAX'\n",
      "Entity 'http://edamontology.org/format_3157' - Label 'EBI Application Result XML'\n",
      "Entity 'http://edamontology.org/format_3158' - Label 'PSI MI XML (MIF)'\n",
      "Entity 'http://edamontology.org/format_3159' - Label 'phyloXML'\n",
      "Entity 'http://edamontology.org/format_3160' - Label 'NeXML'\n",
      "Entity 'http://edamontology.org/format_3161' - Label 'MAGE-ML'\n",
      "Entity 'http://edamontology.org/format_3162' - Label 'MAGE-TAB'\n",
      "Entity 'http://edamontology.org/format_3163' - Label 'GCDML'\n",
      "Entity 'http://edamontology.org/format_3164' - Label 'GTrack'\n",
      "Entity 'http://edamontology.org/format_3166' - Label 'Biological pathway or network report format'\n",
      "Entity 'http://edamontology.org/format_3167' - Label 'Experiment annotation format'\n",
      "Entity 'http://edamontology.org/format_3235' - Label 'Cytoband format'\n",
      "Entity 'http://edamontology.org/format_3239' - Label 'CopasiML'\n",
      "Entity 'http://edamontology.org/format_3240' - Label 'CellML'\n",
      "Entity 'http://edamontology.org/format_3242' - Label 'PSI MI TAB (MITAB)'\n",
      "Entity 'http://edamontology.org/format_3243' - Label 'PSI-PAR'\n",
      "Entity 'http://edamontology.org/format_3244' - Label 'mzML'\n",
      "Entity 'http://edamontology.org/format_3245' - Label 'Mass spectrometry data format'\n",
      "Entity 'http://edamontology.org/format_3246' - Label 'TraML'\n",
      "Entity 'http://edamontology.org/format_3247' - Label 'mzIdentML'\n",
      "Entity 'http://edamontology.org/format_3248' - Label 'mzQuantML'\n",
      "Entity 'http://edamontology.org/format_3249' - Label 'GelML'\n",
      "Entity 'http://edamontology.org/format_3250' - Label 'spML'\n",
      "Entity 'http://edamontology.org/format_3252' - Label 'OWL Functional Syntax'\n",
      "Entity 'http://edamontology.org/format_3253' - Label 'Manchester OWL Syntax'\n",
      "Entity 'http://edamontology.org/format_3254' - Label 'KRSS2 Syntax'\n",
      "Entity 'http://edamontology.org/format_3255' - Label 'Turtle'\n",
      "Entity 'http://edamontology.org/format_3256' - Label 'N-Triples'\n",
      "Entity 'http://edamontology.org/format_3257' - Label 'Notation3'\n",
      "Entity 'http://edamontology.org/format_3261' - Label 'RDF/XML'\n",
      "Entity 'http://edamontology.org/format_3262' - Label 'OWL/XML'\n",
      "Entity 'http://edamontology.org/format_3281' - Label 'A2M'\n",
      "Entity 'http://edamontology.org/format_3284' - Label 'SFF'\n",
      "Entity 'http://edamontology.org/format_3285' - Label 'MAP'\n",
      "Entity 'http://edamontology.org/format_3286' - Label 'PED'\n",
      "Entity 'http://edamontology.org/format_3287' - Label 'Individual genetic data format'\n",
      "Entity 'http://edamontology.org/format_3288' - Label 'PED/MAP'\n",
      "Entity 'http://edamontology.org/format_3309' - Label 'CT'\n",
      "Entity 'http://edamontology.org/format_3310' - Label 'SS'\n",
      "Entity 'http://edamontology.org/format_3311' - Label 'RNAML'\n",
      "Entity 'http://edamontology.org/format_3312' - Label 'GDE'\n",
      "Entity 'http://edamontology.org/format_3313' - Label 'BLC'\n",
      "Entity 'http://edamontology.org/format_3326' - Label 'Data index format'\n",
      "Entity 'http://edamontology.org/format_3327' - Label 'BAI'\n",
      "Entity 'http://edamontology.org/format_3328' - Label 'HMMER2'\n",
      "Entity 'http://edamontology.org/format_3329' - Label 'HMMER3'\n",
      "Entity 'http://edamontology.org/format_3330' - Label 'PO'\n",
      "Entity 'http://edamontology.org/format_3331' - Label 'BLAST XML results format'\n",
      "Entity 'http://edamontology.org/format_3462' - Label 'CRAM'\n",
      "Entity 'http://edamontology.org/format_3464' - Label 'JSON'\n",
      "Entity 'http://edamontology.org/format_3466' - Label 'EPS'\n",
      "Entity 'http://edamontology.org/format_3467' - Label 'GIF'\n",
      "Entity 'http://edamontology.org/format_3468' - Label 'xls'\n",
      "Entity 'http://edamontology.org/format_3475' - Label 'TSV'\n",
      "Entity 'http://edamontology.org/format_3476' - Label 'Gene expression data format'\n",
      "Entity 'http://edamontology.org/format_3477' - Label 'Cytoscape input file format'\n",
      "Entity 'http://edamontology.org/format_3484' - Label 'ebwt'\n",
      "Entity 'http://edamontology.org/format_3485' - Label 'RSF'\n",
      "Entity 'http://edamontology.org/format_3486' - Label 'GCG format variant'\n",
      "Entity 'http://edamontology.org/format_3487' - Label 'BSML'\n",
      "Entity 'http://edamontology.org/format_3491' - Label 'ebwtl'\n",
      "Entity 'http://edamontology.org/format_3499' - Label 'Ensembl variation file format'\n",
      "Entity 'http://edamontology.org/format_3506' - Label 'docx'\n",
      "Entity 'http://edamontology.org/format_3507' - Label 'Document format'\n",
      "Entity 'http://edamontology.org/format_3508' - Label 'PDF'\n",
      "Entity 'http://edamontology.org/format_3547' - Label 'Image format'\n",
      "Entity 'http://edamontology.org/format_3548' - Label 'DICOM format'\n",
      "Entity 'http://edamontology.org/format_3549' - Label 'nii'\n",
      "Entity 'http://edamontology.org/format_3550' - Label 'mhd'\n",
      "Entity 'http://edamontology.org/format_3551' - Label 'nrrd'\n",
      "Entity 'http://edamontology.org/format_3554' - Label 'R file format'\n",
      "Entity 'http://edamontology.org/format_3555' - Label 'SPSS'\n",
      "Entity 'http://edamontology.org/format_3556' - Label 'MHTML'\n",
      "Entity 'http://edamontology.org/format_3578' - Label 'IDAT'\n",
      "Entity 'http://edamontology.org/format_3579' - Label 'JPG'\n",
      "Entity 'http://edamontology.org/format_3580' - Label 'rcc'\n",
      "Entity 'http://edamontology.org/format_3581' - Label 'arff'\n",
      "Entity 'http://edamontology.org/format_3582' - Label 'afg'\n",
      "Entity 'http://edamontology.org/format_3583' - Label 'bedgraph'\n",
      "Entity 'http://edamontology.org/format_3584' - Label 'bedstrict'\n",
      "Entity 'http://edamontology.org/format_3585' - Label 'bed6'\n",
      "Entity 'http://edamontology.org/format_3586' - Label 'bed12'\n",
      "Entity 'http://edamontology.org/format_3587' - Label 'chrominfo'\n",
      "Entity 'http://edamontology.org/format_3588' - Label 'customtrack'\n",
      "Entity 'http://edamontology.org/format_3589' - Label 'csfasta'\n",
      "Entity 'http://edamontology.org/format_3590' - Label 'HDF5'\n",
      "Entity 'http://edamontology.org/format_3591' - Label 'TIFF'\n",
      "Entity 'http://edamontology.org/format_3592' - Label 'BMP'\n",
      "Entity 'http://edamontology.org/format_3593' - Label 'im'\n",
      "Entity 'http://edamontology.org/format_3594' - Label 'pcd'\n",
      "Entity 'http://edamontology.org/format_3595' - Label 'pcx'\n",
      "Entity 'http://edamontology.org/format_3596' - Label 'ppm'\n",
      "Entity 'http://edamontology.org/format_3597' - Label 'psd'\n",
      "Entity 'http://edamontology.org/format_3598' - Label 'xbm'\n",
      "Entity 'http://edamontology.org/format_3599' - Label 'xpm'\n",
      "Entity 'http://edamontology.org/format_3600' - Label 'rgb'\n",
      "Entity 'http://edamontology.org/format_3601' - Label 'pbm'\n",
      "Entity 'http://edamontology.org/format_3602' - Label 'pgm'\n",
      "Entity 'http://edamontology.org/format_3603' - Label 'PNG'\n",
      "Entity 'http://edamontology.org/format_3604' - Label 'SVG'\n",
      "Entity 'http://edamontology.org/format_3605' - Label 'rast'\n",
      "Entity 'http://edamontology.org/format_3606' - Label 'Sequence quality report format (text)'\n",
      "Entity 'http://edamontology.org/format_3607' - Label 'qual'\n",
      "Entity 'http://edamontology.org/format_3608' - Label 'qualsolexa'\n",
      "Entity 'http://edamontology.org/format_3609' - Label 'qualillumina'\n",
      "Entity 'http://edamontology.org/format_3610' - Label 'qualsolid'\n",
      "Entity 'http://edamontology.org/format_3611' - Label 'qual454'\n",
      "Entity 'http://edamontology.org/format_3612' - Label 'ENCODE peak format'\n",
      "Entity 'http://edamontology.org/format_3613' - Label 'ENCODE narrow peak format'\n",
      "Entity 'http://edamontology.org/format_3614' - Label 'ENCODE broad peak format'\n",
      "Entity 'http://edamontology.org/format_3615' - Label 'bgzip'\n",
      "Entity 'http://edamontology.org/format_3616' - Label 'tabix'\n",
      "Entity 'http://edamontology.org/format_3617' - Label 'Graph format'\n",
      "Entity 'http://edamontology.org/format_3618' - Label 'xgmml'\n",
      "Entity 'http://edamontology.org/format_3619' - Label 'sif'\n",
      "Entity 'http://edamontology.org/format_3620' - Label 'xlsx'\n",
      "Entity 'http://edamontology.org/format_3621' - Label 'SQLite format'\n",
      "Entity 'http://edamontology.org/format_3622' - Label 'Gemini SQLite format'\n",
      "Entity 'http://edamontology.org/format_3623' - Label 'Index format'\n",
      "Entity 'http://edamontology.org/format_3624' - Label 'snpeffdb'\n",
      "Entity 'http://edamontology.org/format_3626' - Label 'MAT'\n",
      "Entity 'http://edamontology.org/format_3650' - Label 'netCDF'\n",
      "Entity 'http://edamontology.org/format_3651' - Label 'MGF'\n",
      "Entity 'http://edamontology.org/format_3652' - Label 'dta'\n",
      "Entity 'http://edamontology.org/format_3653' - Label 'pkl'\n",
      "Entity 'http://edamontology.org/format_3654' - Label 'mzXML'\n",
      "Entity 'http://edamontology.org/format_3655' - Label 'pepXML'\n",
      "Entity 'http://edamontology.org/format_3657' - Label 'GPML'\n",
      "Entity 'http://edamontology.org/format_3665' - Label 'K-mer countgraph'\n",
      "Entity 'http://edamontology.org/format_3681' - Label 'mzTab'\n",
      "Entity 'http://edamontology.org/format_3682' - Label 'imzML metadata file'\n",
      "Entity 'http://edamontology.org/format_3683' - Label 'qcML'\n",
      "Entity 'http://edamontology.org/format_3684' - Label 'PRIDE XML'\n",
      "Entity 'http://edamontology.org/format_3685' - Label 'SED-ML'\n",
      "Entity 'http://edamontology.org/format_3686' - Label 'COMBINE OMEX'\n",
      "Entity 'http://edamontology.org/format_3687' - Label 'ISA-TAB'\n",
      "Entity 'http://edamontology.org/format_3688' - Label 'SBtab'\n",
      "Entity 'http://edamontology.org/format_3689' - Label 'BCML'\n",
      "Entity 'http://edamontology.org/format_3690' - Label 'BDML'\n",
      "Entity 'http://edamontology.org/format_3691' - Label 'BEL'\n",
      "Entity 'http://edamontology.org/format_3692' - Label 'SBGN-ML'\n",
      "Entity 'http://edamontology.org/format_3693' - Label 'AGP'\n",
      "Entity 'http://edamontology.org/format_3696' - Label 'PS'\n",
      "Entity 'http://edamontology.org/format_3698' - Label 'SRA format'\n",
      "Entity 'http://edamontology.org/format_3699' - Label 'VDB'\n",
      "Entity 'http://edamontology.org/format_3700' - Label 'Tabix index file format'\n",
      "Entity 'http://edamontology.org/format_3701' - Label 'Sequin format'\n",
      "Entity 'http://edamontology.org/format_3702' - Label 'MSF'\n",
      "Entity 'http://edamontology.org/format_3706' - Label 'Biodiversity data format'\n",
      "Entity 'http://edamontology.org/format_3708' - Label 'ABCD format'\n",
      "Entity 'http://edamontology.org/format_3709' - Label 'GCT/Res format'\n",
      "Entity 'http://edamontology.org/format_3710' - Label 'WIFF format'\n",
      "Entity 'http://edamontology.org/format_3711' - Label 'X!Tandem XML'\n",
      "Entity 'http://edamontology.org/format_3712' - Label 'Thermo RAW'\n",
      "Entity 'http://edamontology.org/format_3713' - Label 'Mascot .dat file'\n",
      "Entity 'http://edamontology.org/format_3714' - Label 'MaxQuant APL peaklist format'\n",
      "Entity 'http://edamontology.org/format_3725' - Label 'SBOL'\n",
      "Entity 'http://edamontology.org/format_3726' - Label 'PMML'\n",
      "Entity 'http://edamontology.org/format_3727' - Label 'OME-TIFF'\n",
      "Entity 'http://edamontology.org/format_3728' - Label 'LocARNA PP'\n",
      "Entity 'http://edamontology.org/format_3729' - Label 'dbGaP format'\n",
      "Entity 'http://edamontology.org/format_3746' - Label 'BIOM format'\n",
      "Entity 'http://edamontology.org/format_3747' - Label 'protXML'\n",
      "Entity 'http://edamontology.org/format_3748' - Label 'Linked data format'\n",
      "Entity 'http://edamontology.org/format_3749' - Label 'JSON-LD'\n",
      "Entity 'http://edamontology.org/format_3750' - Label 'YAML'\n",
      "Entity 'http://edamontology.org/format_3751' - Label 'DSV'\n",
      "Entity 'http://edamontology.org/format_3752' - Label 'CSV'\n",
      "Entity 'http://edamontology.org/format_3758' - Label 'SEQUEST .out file'\n",
      "Entity 'http://edamontology.org/format_3764' - Label 'idXML'\n",
      "Entity 'http://edamontology.org/format_3765' - Label 'KNIME datatable format'\n",
      "Entity 'http://edamontology.org/format_3770' - Label 'UniProtKB XML'\n",
      "Entity 'http://edamontology.org/format_3771' - Label 'UniProtKB RDF'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3774' - Label 'BioJSON (Jalview)'\n",
      "Entity 'http://edamontology.org/format_3775' - Label 'GSuite'\n",
      "Entity 'http://edamontology.org/format_3776' - Label 'BTrack'\n",
      "Entity 'http://edamontology.org/format_3777' - Label 'MCPD'\n",
      "Entity 'http://edamontology.org/format_3780' - Label 'Annotated text format'\n",
      "Entity 'http://edamontology.org/format_3781' - Label 'PubAnnotation format'\n",
      "Entity 'http://edamontology.org/format_3782' - Label 'BioC'\n",
      "Entity 'http://edamontology.org/format_3783' - Label 'PubTator format'\n",
      "Entity 'http://edamontology.org/format_3784' - Label 'Open Annotation format'\n",
      "Entity 'http://edamontology.org/format_3785' - Label 'BioNLP Shared Task format'\n",
      "Entity 'http://edamontology.org/format_3787' - Label 'Query language'\n",
      "Entity 'http://edamontology.org/format_3788' - Label 'SQL'\n",
      "Entity 'http://edamontology.org/format_3789' - Label 'XQuery'\n",
      "Entity 'http://edamontology.org/format_3790' - Label 'SPARQL'\n",
      "Entity 'http://edamontology.org/format_3804' - Label 'xsd'\n",
      "Entity 'http://edamontology.org/format_3811' - Label 'XMFA'\n",
      "Entity 'http://edamontology.org/format_3812' - Label 'GEN'\n",
      "Entity 'http://edamontology.org/format_3813' - Label 'SAMPLE file format'\n",
      "Entity 'http://edamontology.org/format_3814' - Label 'SDF'\n",
      "Entity 'http://edamontology.org/format_3815' - Label 'Molfile'\n",
      "Entity 'http://edamontology.org/format_3816' - Label 'Mol2'\n",
      "Entity 'http://edamontology.org/format_3817' - Label 'latex'\n",
      "Entity 'http://edamontology.org/format_3818' - Label 'ELAND format'\n",
      "Entity 'http://edamontology.org/format_3819' - Label 'Relaxed PHYLIP Interleaved'\n",
      "Entity 'http://edamontology.org/format_3820' - Label 'Relaxed PHYLIP Sequential'\n",
      "Entity 'http://edamontology.org/format_3821' - Label 'VisML'\n",
      "Entity 'http://edamontology.org/format_3822' - Label 'GML'\n",
      "Entity 'http://edamontology.org/format_3823' - Label 'FASTG'\n",
      "Entity 'http://edamontology.org/format_3824' - Label 'NMR data format'\n",
      "Entity 'http://edamontology.org/format_3825' - Label 'nmrML'\n",
      "Entity 'http://edamontology.org/format_3826' - Label 'proBAM'\n",
      "Entity 'http://edamontology.org/format_3827' - Label 'proBED'\n",
      "Entity 'http://edamontology.org/format_3828' - Label 'Raw microarray data format'\n",
      "Entity 'http://edamontology.org/format_3829' - Label 'GPR'\n",
      "Entity 'http://edamontology.org/format_3830' - Label 'ARB'\n",
      "Entity 'http://edamontology.org/format_3832' - Label 'consensusXML'\n",
      "Entity 'http://edamontology.org/format_3833' - Label 'featureXML'\n",
      "Entity 'http://edamontology.org/format_3834' - Label 'mzData'\n",
      "Entity 'http://edamontology.org/format_3835' - Label 'TIDE TXT'\n",
      "Entity 'http://edamontology.org/format_3836' - Label 'BLAST XML v2 results format'\n",
      "Entity 'http://edamontology.org/format_3838' - Label 'pptx'\n",
      "Entity 'http://edamontology.org/format_3839' - Label 'ibd'\n",
      "Entity 'http://edamontology.org/format_3841' - Label 'NLP format'\n",
      "Entity 'http://edamontology.org/format_3843' - Label 'BEAST'\n",
      "Entity 'http://edamontology.org/format_3844' - Label 'Chado-XML'\n",
      "Entity 'http://edamontology.org/format_3845' - Label 'HSAML'\n",
      "Entity 'http://edamontology.org/format_3846' - Label 'InterProScan XML'\n",
      "Entity 'http://edamontology.org/format_3847' - Label 'KGML'\n",
      "Entity 'http://edamontology.org/format_3848' - Label 'PubMed XML'\n",
      "Entity 'http://edamontology.org/format_3849' - Label 'MSAML'\n",
      "Entity 'http://edamontology.org/format_3850' - Label 'OrthoXML'\n",
      "Entity 'http://edamontology.org/format_3851' - Label 'PSDML'\n",
      "Entity 'http://edamontology.org/format_3852' - Label 'SeqXML'\n",
      "Entity 'http://edamontology.org/format_3853' - Label 'UniParc XML'\n",
      "Entity 'http://edamontology.org/format_3854' - Label 'UniRef XML'\n",
      "Entity 'http://edamontology.org/format_3857' - Label 'CWL'\n",
      "Entity 'http://edamontology.org/format_3858' - Label 'Waters RAW'\n",
      "Entity 'http://edamontology.org/format_3859' - Label 'JCAMP-DX'\n",
      "Entity 'http://edamontology.org/format_3862' - Label 'NLP annotation format'\n",
      "Entity 'http://edamontology.org/format_3863' - Label 'NLP corpus format'\n",
      "Entity 'http://edamontology.org/format_3864' - Label 'mirGFF3'\n",
      "Entity 'http://edamontology.org/format_3865' - Label 'RNA annotation format'\n",
      "Entity 'http://edamontology.org/format_3866' - Label 'Trajectory format'\n",
      "Entity 'http://edamontology.org/format_3867' - Label 'Trajectory format (binary)'\n",
      "Entity 'http://edamontology.org/format_3868' - Label 'Trajectory format (text)'\n",
      "Entity 'http://edamontology.org/format_3873' - Label 'HDF'\n",
      "Entity 'http://edamontology.org/format_3874' - Label 'PCAzip'\n",
      "Entity 'http://edamontology.org/format_3875' - Label 'XTC'\n",
      "Entity 'http://edamontology.org/format_3876' - Label 'TNG'\n",
      "Entity 'http://edamontology.org/format_3877' - Label 'XYZ'\n",
      "Entity 'http://edamontology.org/format_3878' - Label 'mdcrd'\n",
      "Entity 'http://edamontology.org/format_3879' - Label 'Topology format'\n",
      "Entity 'http://edamontology.org/format_3880' - Label 'GROMACS top'\n",
      "Entity 'http://edamontology.org/format_3881' - Label 'AMBER top'\n",
      "Entity 'http://edamontology.org/format_3882' - Label 'PSF'\n",
      "Entity 'http://edamontology.org/format_3883' - Label 'GROMACS itp'\n",
      "Entity 'http://edamontology.org/format_3884' - Label 'FF parameter format'\n",
      "Entity 'http://edamontology.org/format_3885' - Label 'BinPos'\n",
      "Entity 'http://edamontology.org/format_3886' - Label 'RST'\n",
      "Entity 'http://edamontology.org/format_3887' - Label 'CHARMM rtf'\n",
      "Entity 'http://edamontology.org/format_3888' - Label 'AMBER frcmod'\n",
      "Entity 'http://edamontology.org/format_3889' - Label 'AMBER off'\n",
      "Entity 'http://edamontology.org/format_3906' - Label 'NMReDATA'\n",
      "Entity 'http://edamontology.org/format_3909' - Label 'BpForms'\n",
      "Entity 'http://edamontology.org/format_3910' - Label 'trr'\n",
      "Entity 'http://edamontology.org/format_3911' - Label 'msh'\n",
      "Entity 'http://edamontology.org/format_3913' - Label 'Loom'\n",
      "Entity 'http://edamontology.org/format_3915' - Label 'Zarr'\n",
      "Entity 'http://edamontology.org/format_3916' - Label 'MTX'\n",
      "Entity 'http://edamontology.org/format_3951' - Label 'BcForms'\n",
      "Entity 'http://edamontology.org/format_3956' - Label 'N-Quads'\n",
      "Entity 'http://edamontology.org/format_3969' - Label 'Vega'\n",
      "Entity 'http://edamontology.org/format_3970' - Label 'Vega-lite'\n",
      "Entity 'http://edamontology.org/format_3971' - Label 'NeuroML'\n",
      "Entity 'http://edamontology.org/format_3972' - Label 'BNGL'\n",
      "Entity 'http://edamontology.org/format_3973' - Label 'Docker image'\n",
      "Entity 'http://edamontology.org/format_3975' - Label 'GFA 1'\n",
      "Entity 'http://edamontology.org/format_3976' - Label 'GFA 2'\n",
      "Entity 'http://edamontology.org/format_3977' - Label 'ObjTables'\n",
      "Entity 'http://edamontology.org/format_3978' - Label 'CONTIG'\n",
      "Entity 'http://edamontology.org/format_3979' - Label 'WEGO'\n",
      "Entity 'http://edamontology.org/format_3980' - Label 'RPKM'\n",
      "Entity 'http://edamontology.org/format_3981' - Label 'TAR format'\n",
      "Entity 'http://edamontology.org/format_3982' - Label 'CHAIN'\n",
      "Entity 'http://edamontology.org/format_3983' - Label 'NET'\n",
      "Entity 'http://edamontology.org/format_3984' - Label 'QMAP'\n",
      "Entity 'http://edamontology.org/format_3985' - Label 'gxformat2'\n",
      "Entity 'http://edamontology.org/format_3986' - Label 'WMV'\n",
      "Entity 'http://edamontology.org/format_3987' - Label 'ZIP format'\n",
      "Entity 'http://edamontology.org/format_3988' - Label 'LSM'\n",
      "Entity 'http://edamontology.org/format_3989' - Label 'GZIP format'\n",
      "Entity 'http://edamontology.org/format_3990' - Label 'AVI'\n",
      "Entity 'http://edamontology.org/format_3991' - Label 'TrackDB'\n",
      "Entity 'http://edamontology.org/format_3992' - Label 'CIGAR format'\n",
      "Entity 'http://edamontology.org/format_3993' - Label 'Stereolithography format'\n",
      "Entity 'http://edamontology.org/format_3994' - Label 'U3D'\n",
      "Entity 'http://edamontology.org/format_3995' - Label 'Texture file format'\n",
      "Entity 'http://edamontology.org/format_3996' - Label 'Python script'\n",
      "Entity 'http://edamontology.org/format_3997' - Label 'MPEG-4'\n",
      "Entity 'http://edamontology.org/format_3998' - Label 'Perl script'\n",
      "Entity 'http://edamontology.org/format_3999' - Label 'R script'\n",
      "Entity 'http://edamontology.org/format_4000' - Label 'R markdown'\n",
      "Entity 'http://edamontology.org/format_4001' - Label 'NIFTI format'\n",
      "Entity 'http://edamontology.org/format_4002' - Label 'pickle'\n",
      "Entity 'http://edamontology.org/format_4003' - Label 'NumPy format'\n",
      "Entity 'http://edamontology.org/format_4004' - Label 'SimTools repertoire file format'\n",
      "Entity 'http://edamontology.org/format_4005' - Label 'Configuration file format'\n",
      "Entity 'http://edamontology.org/format_4006' - Label 'Zstandard format'\n",
      "Entity 'http://edamontology.org/format_4007' - Label 'MATLAB script'\n",
      "Entity 'http://edamontology.org/format_4015' - Label 'PEtab'\n",
      "Entity 'http://edamontology.org/format_4018' - Label 'gVCF'\n",
      "Entity 'http://edamontology.org/format_4023' - Label 'cml'\n",
      "Entity 'http://edamontology.org/format_4024' - Label 'cif'\n",
      "Entity 'http://edamontology.org/format_4025' - Label 'BioSimulators format for the specifications of biosimulation tools'\n",
      "Entity 'http://edamontology.org/format_4026' - Label 'BioSimulators standard for command-line interfaces for biosimulation tools'\n",
      "Entity 'http://edamontology.org/format_4035' - Label 'PQR'\n",
      "Entity 'http://edamontology.org/format_4036' - Label 'PDBQT'\n",
      "Entity 'http://edamontology.org/format_4039' - Label 'MSP'\n",
      "Entity 'http://edamontology.org/operation_0004' - Label 'Operation'\n",
      "Entity 'http://edamontology.org/operation_0224' - Label 'Query and retrieval'\n",
      "Entity 'http://edamontology.org/operation_0225' - Label 'Data retrieval (database cross-reference)'\n",
      "Entity 'http://edamontology.org/operation_0226' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/operation_0227' - Label 'Indexing'\n",
      "Entity 'http://edamontology.org/operation_0228' - Label 'Data index analysis'\n",
      "Entity 'http://edamontology.org/operation_0229' - Label 'Annotation retrieval (sequence)'\n",
      "Entity 'http://edamontology.org/operation_0230' - Label 'Sequence generation'\n",
      "Entity 'http://edamontology.org/operation_0231' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/operation_0232' - Label 'Sequence merging'\n",
      "Entity 'http://edamontology.org/operation_0233' - Label 'Sequence conversion'\n",
      "Entity 'http://edamontology.org/operation_0234' - Label 'Sequence complexity calculation'\n",
      "Entity 'http://edamontology.org/operation_0235' - Label 'Sequence ambiguity calculation'\n",
      "Entity 'http://edamontology.org/operation_0236' - Label 'Sequence composition calculation'\n",
      "Entity 'http://edamontology.org/operation_0237' - Label 'Repeat sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0238' - Label 'Sequence motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0239' - Label 'Sequence motif recognition'\n",
      "Entity 'http://edamontology.org/operation_0240' - Label 'Sequence motif comparison'\n",
      "Entity 'http://edamontology.org/operation_0241' - Label 'Transcription regulatory sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0242' - Label 'Conserved transcription regulatory sequence identification'\n",
      "Entity 'http://edamontology.org/operation_0243' - Label 'Protein property calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0244' - Label 'Simulation analysis'\n",
      "Entity 'http://edamontology.org/operation_0245' - Label 'Structural motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0246' - Label 'Protein domain recognition'\n",
      "Entity 'http://edamontology.org/operation_0247' - Label 'Protein architecture analysis'\n",
      "Entity 'http://edamontology.org/operation_0248' - Label 'Residue interaction calculation'\n",
      "Entity 'http://edamontology.org/operation_0249' - Label 'Protein geometry calculation'\n",
      "Entity 'http://edamontology.org/operation_0250' - Label 'Protein property calculation'\n",
      "Entity 'http://edamontology.org/operation_0252' - Label 'Peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0253' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_0254' - Label 'Data retrieval (feature table)'\n",
      "Entity 'http://edamontology.org/operation_0255' - Label 'Feature table query'\n",
      "Entity 'http://edamontology.org/operation_0256' - Label 'Sequence feature comparison'\n",
      "Entity 'http://edamontology.org/operation_0257' - Label 'Data retrieval (sequence alignment)'\n",
      "Entity 'http://edamontology.org/operation_0258' - Label 'Sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_0259' - Label 'Sequence alignment comparison'\n",
      "Entity 'http://edamontology.org/operation_0260' - Label 'Sequence alignment conversion'\n",
      "Entity 'http://edamontology.org/operation_0261' - Label 'Nucleic acid property processing'\n",
      "Entity 'http://edamontology.org/operation_0262' - Label 'Nucleic acid property calculation'\n",
      "Entity 'http://edamontology.org/operation_0264' - Label 'Alternative splicing prediction'\n",
      "Entity 'http://edamontology.org/operation_0265' - Label 'Frameshift detection'\n",
      "Entity 'http://edamontology.org/operation_0266' - Label 'Vector sequence detection'\n",
      "Entity 'http://edamontology.org/operation_0267' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0268' - Label 'Protein super-secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0269' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/operation_0270' - Label 'Transmembrane protein analysis'\n",
      "Entity 'http://edamontology.org/operation_0271' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0272' - Label 'Residue contact prediction'\n",
      "Entity 'http://edamontology.org/operation_0273' - Label 'Protein interaction raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_0274' - Label 'Protein-protein interaction prediction (from protein sequence)'\n",
      "Entity 'http://edamontology.org/operation_0275' - Label 'Protein-protein interaction prediction (from protein structure)'\n",
      "Entity 'http://edamontology.org/operation_0276' - Label 'Protein interaction network analysis'\n",
      "Entity 'http://edamontology.org/operation_0277' - Label 'Pathway or network comparison'\n",
      "Entity 'http://edamontology.org/operation_0278' - Label 'RNA secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0279' - Label 'Nucleic acid folding analysis'\n",
      "Entity 'http://edamontology.org/operation_0280' - Label 'Data retrieval (restriction enzyme annotation)'\n",
      "Entity 'http://edamontology.org/operation_0281' - Label 'Genetic marker identification'\n",
      "Entity 'http://edamontology.org/operation_0282' - Label 'Genetic mapping'\n",
      "Entity 'http://edamontology.org/operation_0283' - Label 'Linkage analysis'\n",
      "Entity 'http://edamontology.org/operation_0284' - Label 'Codon usage table generation'\n",
      "Entity 'http://edamontology.org/operation_0285' - Label 'Codon usage table comparison'\n",
      "Entity 'http://edamontology.org/operation_0286' - Label 'Codon usage analysis'\n",
      "Entity 'http://edamontology.org/operation_0287' - Label 'Base position variability plotting'\n",
      "Entity 'http://edamontology.org/operation_0288' - Label 'Sequence word comparison'\n",
      "Entity 'http://edamontology.org/operation_0289' - Label 'Sequence distance matrix generation'\n",
      "Entity 'http://edamontology.org/operation_0290' - Label 'Sequence redundancy removal'\n",
      "Entity 'http://edamontology.org/operation_0291' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/operation_0292' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0293' - Label 'Hybrid sequence alignment construction'\n",
      "Entity 'http://edamontology.org/operation_0294' - Label 'Structure-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0295' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0296' - Label 'Sequence profile generation'\n",
      "Entity 'http://edamontology.org/operation_0297' - Label '3D profile generation'\n",
      "Entity 'http://edamontology.org/operation_0298' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0299' - Label '3D profile-to-3D profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0300' - Label 'Sequence profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0301' - Label 'Sequence-to-3D-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0302' - Label 'Protein threading'\n",
      "Entity 'http://edamontology.org/operation_0303' - Label 'Fold recognition'\n",
      "Entity 'http://edamontology.org/operation_0304' - Label 'Metadata retrieval'\n",
      "Entity 'http://edamontology.org/operation_0305' - Label 'Literature search'\n",
      "Entity 'http://edamontology.org/operation_0306' - Label 'Text mining'\n",
      "Entity 'http://edamontology.org/operation_0307' - Label 'Virtual PCR'\n",
      "Entity 'http://edamontology.org/operation_0308' - Label 'PCR primer design'\n",
      "Entity 'http://edamontology.org/operation_0309' - Label 'Microarray probe design'\n",
      "Entity 'http://edamontology.org/operation_0310' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/operation_0311' - Label 'Microarray data standardisation and normalisation'\n",
      "Entity 'http://edamontology.org/operation_0312' - Label 'Sequencing-based expression profile data processing'\n",
      "Entity 'http://edamontology.org/operation_0313' - Label 'Expression profile clustering'\n",
      "Entity 'http://edamontology.org/operation_0314' - Label 'Gene expression profiling'\n",
      "Entity 'http://edamontology.org/operation_0315' - Label 'Expression profile comparison'\n",
      "Entity 'http://edamontology.org/operation_0316' - Label 'Functional profiling'\n",
      "Entity 'http://edamontology.org/operation_0317' - Label 'EST and cDNA sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0318' - Label 'Structural genomics target selection'\n",
      "Entity 'http://edamontology.org/operation_0319' - Label 'Protein secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0320' - Label 'Protein structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0321' - Label 'Protein structure validation'\n",
      "Entity 'http://edamontology.org/operation_0322' - Label 'Molecular model refinement'\n",
      "Entity 'http://edamontology.org/operation_0323' - Label 'Phylogenetic inference'\n",
      "Entity 'http://edamontology.org/operation_0324' - Label 'Phylogenetic analysis'\n",
      "Entity 'http://edamontology.org/operation_0325' - Label 'Phylogenetic tree comparison'\n",
      "Entity 'http://edamontology.org/operation_0326' - Label 'Phylogenetic tree editing'\n",
      "Entity 'http://edamontology.org/operation_0327' - Label 'Phylogenetic footprinting'\n",
      "Entity 'http://edamontology.org/operation_0328' - Label 'Protein folding simulation'\n",
      "Entity 'http://edamontology.org/operation_0329' - Label 'Protein folding pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_0330' - Label 'Protein SNP mapping'\n",
      "Entity 'http://edamontology.org/operation_0331' - Label 'Variant effect prediction'\n",
      "Entity 'http://edamontology.org/operation_0332' - Label 'Immunogen design'\n",
      "Entity 'http://edamontology.org/operation_0333' - Label 'Zinc finger prediction'\n",
      "Entity 'http://edamontology.org/operation_0334' - Label 'Enzyme kinetics calculation'\n",
      "Entity 'http://edamontology.org/operation_0335' - Label 'Formatting'\n",
      "Entity 'http://edamontology.org/operation_0336' - Label 'Format validation'\n",
      "Entity 'http://edamontology.org/operation_0337' - Label 'Visualisation'\n",
      "Entity 'http://edamontology.org/operation_0338' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/operation_0339' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/operation_0340' - Label 'Protein secondary database search'\n",
      "Entity 'http://edamontology.org/operation_0341' - Label 'Motif database search'\n",
      "Entity 'http://edamontology.org/operation_0342' - Label 'Sequence profile database search'\n",
      "Entity 'http://edamontology.org/operation_0343' - Label 'Transmembrane protein database search'\n",
      "Entity 'http://edamontology.org/operation_0344' - Label 'Sequence retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0345' - Label 'Sequence retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0346' - Label 'Sequence similarity search'\n",
      "Entity 'http://edamontology.org/operation_0347' - Label 'Sequence database search (by motif or pattern)'\n",
      "Entity 'http://edamontology.org/operation_0348' - Label 'Sequence database search (by amino acid composition)'\n",
      "Entity 'http://edamontology.org/operation_0349' - Label 'Sequence database search (by property)'\n",
      "Entity 'http://edamontology.org/operation_0350' - Label 'Sequence database search (by sequence using word-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0351' - Label 'Sequence database search (by sequence using profile-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0352' - Label 'Sequence database search (by sequence using local alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0353' - Label 'Sequence database search (by sequence using global alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0354' - Label 'Sequence database search (by sequence for primer sequences)'\n",
      "Entity 'http://edamontology.org/operation_0355' - Label 'Sequence database search (by molecular weight)'\n",
      "Entity 'http://edamontology.org/operation_0356' - Label 'Sequence database search (by isoelectric point)'\n",
      "Entity 'http://edamontology.org/operation_0357' - Label 'Structure retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0358' - Label 'Structure retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0359' - Label 'Structure database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_0360' - Label 'Structural similarity search'\n",
      "Entity 'http://edamontology.org/operation_0361' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/operation_0362' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/operation_0363' - Label 'Reverse complement'\n",
      "Entity 'http://edamontology.org/operation_0364' - Label 'Random sequence generation'\n",
      "Entity 'http://edamontology.org/operation_0365' - Label 'Restriction digest'\n",
      "Entity 'http://edamontology.org/operation_0366' - Label 'Protein sequence cleavage'\n",
      "Entity 'http://edamontology.org/operation_0367' - Label 'Sequence mutation and randomisation'\n",
      "Entity 'http://edamontology.org/operation_0368' - Label 'Sequence masking'\n",
      "Entity 'http://edamontology.org/operation_0369' - Label 'Sequence cutting'\n",
      "Entity 'http://edamontology.org/operation_0370' - Label 'Restriction site creation'\n",
      "Entity 'http://edamontology.org/operation_0371' - Label 'DNA translation'\n",
      "Entity 'http://edamontology.org/operation_0372' - Label 'DNA transcription'\n",
      "Entity 'http://edamontology.org/operation_0377' - Label 'Sequence composition calculation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_0378' - Label 'Sequence composition calculation (protein)'\n",
      "Entity 'http://edamontology.org/operation_0379' - Label 'Repeat sequence detection'\n",
      "Entity 'http://edamontology.org/operation_0380' - Label 'Repeat sequence organisation analysis'\n",
      "Entity 'http://edamontology.org/operation_0383' - Label 'Protein hydropathy calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0384' - Label 'Accessible surface calculation'\n",
      "Entity 'http://edamontology.org/operation_0385' - Label 'Protein hydropathy cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0386' - Label 'Protein dipole moment calculation'\n",
      "Entity 'http://edamontology.org/operation_0387' - Label 'Molecular surface calculation'\n",
      "Entity 'http://edamontology.org/operation_0388' - Label 'Protein binding site prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0389' - Label 'Protein-nucleic acid interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_0390' - Label 'Protein peeling'\n",
      "Entity 'http://edamontology.org/operation_0391' - Label 'Protein distance matrix calculation'\n",
      "Entity 'http://edamontology.org/operation_0392' - Label 'Contact map calculation'\n",
      "Entity 'http://edamontology.org/operation_0393' - Label 'Residue cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0394' - Label 'Hydrogen bond calculation'\n",
      "Entity 'http://edamontology.org/operation_0395' - Label 'Residue non-canonical interaction detection'\n",
      "Entity 'http://edamontology.org/operation_0396' - Label 'Ramachandran plot calculation'\n",
      "Entity 'http://edamontology.org/operation_0397' - Label 'Ramachandran plot validation'\n",
      "Entity 'http://edamontology.org/operation_0398' - Label 'Protein molecular weight calculation'\n",
      "Entity 'http://edamontology.org/operation_0399' - Label 'Protein extinction coefficient calculation'\n",
      "Entity 'http://edamontology.org/operation_0400' - Label 'Protein pKa calculation'\n",
      "Entity 'http://edamontology.org/operation_0401' - Label 'Protein hydropathy calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0402' - Label 'Protein titration curve plotting'\n",
      "Entity 'http://edamontology.org/operation_0403' - Label 'Protein isoelectric point calculation'\n",
      "Entity 'http://edamontology.org/operation_0404' - Label 'Protein hydrogen exchange rate calculation'\n",
      "Entity 'http://edamontology.org/operation_0405' - Label 'Protein hydrophobic region calculation'\n",
      "Entity 'http://edamontology.org/operation_0406' - Label 'Protein aliphatic index calculation'\n",
      "Entity 'http://edamontology.org/operation_0407' - Label 'Protein hydrophobic moment plotting'\n",
      "Entity 'http://edamontology.org/operation_0408' - Label 'Protein globularity prediction'\n",
      "Entity 'http://edamontology.org/operation_0409' - Label 'Protein solubility prediction'\n",
      "Entity 'http://edamontology.org/operation_0410' - Label 'Protein crystallizability prediction'\n",
      "Entity 'http://edamontology.org/operation_0411' - Label 'Protein signal peptide detection (eukaryotes)'\n",
      "Entity 'http://edamontology.org/operation_0412' - Label 'Protein signal peptide detection (bacteria)'\n",
      "Entity 'http://edamontology.org/operation_0413' - Label 'MHC peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0414' - Label 'Protein feature prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0415' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/operation_0416' - Label 'Epitope mapping'\n",
      "Entity 'http://edamontology.org/operation_0417' - Label 'Post-translational modification site prediction'\n",
      "Entity 'http://edamontology.org/operation_0418' - Label 'Protein signal peptide detection'\n",
      "Entity 'http://edamontology.org/operation_0419' - Label 'Protein binding site prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0420' - Label 'Nucleic acids-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0421' - Label 'Protein folding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0422' - Label 'Protein cleavage site prediction'\n",
      "Entity 'http://edamontology.org/operation_0423' - Label 'Epitope mapping (MHC Class I)'\n",
      "Entity 'http://edamontology.org/operation_0424' - Label 'Epitope mapping (MHC Class II)'\n",
      "Entity 'http://edamontology.org/operation_0425' - Label 'Whole gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0426' - Label 'Gene component prediction'\n",
      "Entity 'http://edamontology.org/operation_0427' - Label 'Transposon prediction'\n",
      "Entity 'http://edamontology.org/operation_0428' - Label 'PolyA signal detection'\n",
      "Entity 'http://edamontology.org/operation_0429' - Label 'Quadruplex formation site detection'\n",
      "Entity 'http://edamontology.org/operation_0430' - Label 'CpG island and isochore detection'\n",
      "Entity 'http://edamontology.org/operation_0431' - Label 'Restriction site recognition'\n",
      "Entity 'http://edamontology.org/operation_0432' - Label 'Nucleosome position prediction'\n",
      "Entity 'http://edamontology.org/operation_0433' - Label 'Splice site prediction'\n",
      "Entity 'http://edamontology.org/operation_0434' - Label 'Integrated gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0435' - Label 'Operon prediction'\n",
      "Entity 'http://edamontology.org/operation_0436' - Label 'Coding region prediction'\n",
      "Entity 'http://edamontology.org/operation_0437' - Label 'SECIS element prediction'\n",
      "Entity 'http://edamontology.org/operation_0438' - Label 'Transcriptional regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0439' - Label 'Translation initiation site prediction'\n",
      "Entity 'http://edamontology.org/operation_0440' - Label 'Promoter prediction'\n",
      "Entity 'http://edamontology.org/operation_0441' - Label 'cis-regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0442' - Label 'Transcriptional regulatory element prediction (RNA-cis)'\n",
      "Entity 'http://edamontology.org/operation_0443' - Label 'trans-regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0444' - Label 'S/MAR prediction'\n",
      "Entity 'http://edamontology.org/operation_0445' - Label 'Transcription factor binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0446' - Label 'Exonic splicing enhancer prediction'\n",
      "Entity 'http://edamontology.org/operation_0447' - Label 'Sequence alignment validation'\n",
      "Entity 'http://edamontology.org/operation_0448' - Label 'Sequence alignment analysis (conservation)'\n",
      "Entity 'http://edamontology.org/operation_0449' - Label 'Sequence alignment analysis (site correlation)'\n",
      "Entity 'http://edamontology.org/operation_0450' - Label 'Chimera detection'\n",
      "Entity 'http://edamontology.org/operation_0451' - Label 'Recombination detection'\n",
      "Entity 'http://edamontology.org/operation_0452' - Label 'Indel detection'\n",
      "Entity 'http://edamontology.org/operation_0453' - Label 'Nucleosome formation potential prediction'\n",
      "Entity 'http://edamontology.org/operation_0455' - Label 'Nucleic acid thermodynamic property calculation'\n",
      "Entity 'http://edamontology.org/operation_0456' - Label 'Nucleic acid melting profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0457' - Label 'Nucleic acid stitch profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0458' - Label 'Nucleic acid melting curve plotting'\n",
      "Entity 'http://edamontology.org/operation_0459' - Label 'Nucleic acid probability profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0460' - Label 'Nucleic acid temperature profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0461' - Label 'Nucleic acid curvature calculation'\n",
      "Entity 'http://edamontology.org/operation_0463' - Label 'miRNA target prediction'\n",
      "Entity 'http://edamontology.org/operation_0464' - Label 'tRNA gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0465' - Label 'siRNA binding specificity prediction'\n",
      "Entity 'http://edamontology.org/operation_0467' - Label 'Protein secondary structure prediction (integrated)'\n",
      "Entity 'http://edamontology.org/operation_0468' - Label 'Protein secondary structure prediction (helices)'\n",
      "Entity 'http://edamontology.org/operation_0469' - Label 'Protein secondary structure prediction (turns)'\n",
      "Entity 'http://edamontology.org/operation_0470' - Label 'Protein secondary structure prediction (coils)'\n",
      "Entity 'http://edamontology.org/operation_0471' - Label 'Disulfide bond prediction'\n",
      "Entity 'http://edamontology.org/operation_0472' - Label 'GPCR prediction'\n",
      "Entity 'http://edamontology.org/operation_0473' - Label 'GPCR analysis'\n",
      "Entity 'http://edamontology.org/operation_0474' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0475' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0476' - Label 'Ab initio structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0477' - Label 'Protein modelling'\n",
      "Entity 'http://edamontology.org/operation_0478' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/operation_0479' - Label 'Backbone modelling'\n",
      "Entity 'http://edamontology.org/operation_0480' - Label 'Side chain modelling'\n",
      "Entity 'http://edamontology.org/operation_0481' - Label 'Loop modelling'\n",
      "Entity 'http://edamontology.org/operation_0482' - Label 'Protein-ligand docking'\n",
      "Entity 'http://edamontology.org/operation_0483' - Label 'RNA inverse folding'\n",
      "Entity 'http://edamontology.org/operation_0484' - Label 'SNP detection'\n",
      "Entity 'http://edamontology.org/operation_0485' - Label 'Radiation Hybrid Mapping'\n",
      "Entity 'http://edamontology.org/operation_0486' - Label 'Functional mapping'\n",
      "Entity 'http://edamontology.org/operation_0487' - Label 'Haplotype mapping'\n",
      "Entity 'http://edamontology.org/operation_0488' - Label 'Linkage disequilibrium calculation'\n",
      "Entity 'http://edamontology.org/operation_0489' - Label 'Genetic code prediction'\n",
      "Entity 'http://edamontology.org/operation_0490' - Label 'Dot plot plotting'\n",
      "Entity 'http://edamontology.org/operation_0491' - Label 'Pairwise sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0492' - Label 'Multiple sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0493' - Label 'Pairwise sequence alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0494' - Label 'Pairwise sequence alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0495' - Label 'Local alignment'\n",
      "Entity 'http://edamontology.org/operation_0496' - Label 'Global alignment'\n",
      "Entity 'http://edamontology.org/operation_0497' - Label 'Constrained sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0498' - Label 'Consensus-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0499' - Label 'Tree-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0500' - Label 'Secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0501' - Label 'Protein secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0502' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0503' - Label 'Pairwise structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0504' - Label 'Multiple structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0505' - Label 'Structure alignment (protein)'\n",
      "Entity 'http://edamontology.org/operation_0506' - Label 'Structure alignment (RNA)'\n",
      "Entity 'http://edamontology.org/operation_0507' - Label 'Pairwise structure alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0508' - Label 'Pairwise structure alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0509' - Label 'Local structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0510' - Label 'Global structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0511' - Label 'Profile-profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0512' - Label 'Sequence alignment generation (multiple profile)'\n",
      "Entity 'http://edamontology.org/operation_0513' - Label '3D profile-to-3D profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0514' - Label 'Structural profile alignment generation (multiple)'\n",
      "Entity 'http://edamontology.org/operation_0515' - Label 'Data retrieval (tool metadata)'\n",
      "Entity 'http://edamontology.org/operation_0516' - Label 'Data retrieval (database metadata)'\n",
      "Entity 'http://edamontology.org/operation_0517' - Label 'PCR primer design (for large scale sequencing)'\n",
      "Entity 'http://edamontology.org/operation_0518' - Label 'PCR primer design (for genotyping polymorphisms)'\n",
      "Entity 'http://edamontology.org/operation_0519' - Label 'PCR primer design (for gene transcription profiling)'\n",
      "Entity 'http://edamontology.org/operation_0520' - Label 'PCR primer design (for conserved primers)'\n",
      "Entity 'http://edamontology.org/operation_0521' - Label 'PCR primer design (based on gene structure)'\n",
      "Entity 'http://edamontology.org/operation_0522' - Label 'PCR primer design (for methylation PCRs)'\n",
      "Entity 'http://edamontology.org/operation_0523' - Label 'Mapping assembly'\n",
      "Entity 'http://edamontology.org/operation_0524' - Label 'De-novo assembly'\n",
      "Entity 'http://edamontology.org/operation_0525' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_0526' - Label 'EST assembly'\n",
      "Entity 'http://edamontology.org/operation_0527' - Label 'Sequence tag mapping'\n",
      "Entity 'http://edamontology.org/operation_0528' - Label 'SAGE data processing'\n",
      "Entity 'http://edamontology.org/operation_0529' - Label 'MPSS data processing'\n",
      "Entity 'http://edamontology.org/operation_0530' - Label 'SBS data processing'\n",
      "Entity 'http://edamontology.org/operation_0531' - Label 'Heat map generation'\n",
      "Entity 'http://edamontology.org/operation_0532' - Label 'Gene expression profile analysis'\n",
      "Entity 'http://edamontology.org/operation_0533' - Label 'Expression profile pathway mapping'\n",
      "Entity 'http://edamontology.org/operation_0534' - Label 'Protein secondary structure assignment (from coordinate data)'\n",
      "Entity 'http://edamontology.org/operation_0535' - Label 'Protein secondary structure assignment (from CD data)'\n",
      "Entity 'http://edamontology.org/operation_0536' - Label 'Protein structure assignment (from X-ray crystallographic data)'\n",
      "Entity 'http://edamontology.org/operation_0537' - Label 'Protein structure assignment (from NMR data)'\n",
      "Entity 'http://edamontology.org/operation_0538' - Label 'Phylogenetic inference (data centric)'\n",
      "Entity 'http://edamontology.org/operation_0539' - Label 'Phylogenetic inference (method centric)'\n",
      "Entity 'http://edamontology.org/operation_0540' - Label 'Phylogenetic inference (from molecular sequences)'\n",
      "Entity 'http://edamontology.org/operation_0541' - Label 'Phylogenetic inference (from continuous quantitative characters)'\n",
      "Entity 'http://edamontology.org/operation_0542' - Label 'Phylogenetic inference (from gene frequencies)'\n",
      "Entity 'http://edamontology.org/operation_0543' - Label 'Phylogenetic inference (from polymorphism data)'\n",
      "Entity 'http://edamontology.org/operation_0544' - Label 'Species tree construction'\n",
      "Entity 'http://edamontology.org/operation_0545' - Label 'Phylogenetic inference (parsimony methods)'\n",
      "Entity 'http://edamontology.org/operation_0546' - Label 'Phylogenetic inference (minimum distance methods)'\n",
      "Entity 'http://edamontology.org/operation_0547' - Label 'Phylogenetic inference (maximum likelihood and Bayesian methods)'\n",
      "Entity 'http://edamontology.org/operation_0548' - Label 'Phylogenetic inference (quartet methods)'\n",
      "Entity 'http://edamontology.org/operation_0549' - Label 'Phylogenetic inference (AI methods)'\n",
      "Entity 'http://edamontology.org/operation_0550' - Label 'DNA substitution modelling'\n",
      "Entity 'http://edamontology.org/operation_0551' - Label 'Phylogenetic tree topology analysis'\n",
      "Entity 'http://edamontology.org/operation_0552' - Label 'Phylogenetic tree bootstrapping'\n",
      "Entity 'http://edamontology.org/operation_0553' - Label 'Gene tree construction'\n",
      "Entity 'http://edamontology.org/operation_0554' - Label 'Allele frequency distribution analysis'\n",
      "Entity 'http://edamontology.org/operation_0555' - Label 'Consensus tree construction'\n",
      "Entity 'http://edamontology.org/operation_0556' - Label 'Phylogenetic sub/super tree construction'\n",
      "Entity 'http://edamontology.org/operation_0557' - Label 'Phylogenetic tree distances calculation'\n",
      "Entity 'http://edamontology.org/operation_0558' - Label 'Phylogenetic tree annotation'\n",
      "Entity 'http://edamontology.org/operation_0559' - Label 'Immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0560' - Label 'DNA vaccine design'\n",
      "Entity 'http://edamontology.org/operation_0561' - Label 'Sequence formatting'\n",
      "Entity 'http://edamontology.org/operation_0562' - Label 'Sequence alignment formatting'\n",
      "Entity 'http://edamontology.org/operation_0563' - Label 'Codon usage table formatting'\n",
      "Entity 'http://edamontology.org/operation_0564' - Label 'Sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_0565' - Label 'Sequence alignment visualisation'\n",
      "Entity 'http://edamontology.org/operation_0566' - Label 'Sequence cluster visualisation'\n",
      "Entity 'http://edamontology.org/operation_0567' - Label 'Phylogenetic tree visualisation'\n",
      "Entity 'http://edamontology.org/operation_0568' - Label 'RNA secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0569' - Label 'Protein secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0570' - Label 'Structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0571' - Label 'Expression data visualisation'\n",
      "Entity 'http://edamontology.org/operation_0572' - Label 'Protein interaction network visualisation'\n",
      "Entity 'http://edamontology.org/operation_0573' - Label 'Map drawing'\n",
      "Entity 'http://edamontology.org/operation_0574' - Label 'Sequence motif rendering'\n",
      "Entity 'http://edamontology.org/operation_0575' - Label 'Restriction map drawing'\n",
      "Entity 'http://edamontology.org/operation_0577' - Label 'DNA linear map rendering'\n",
      "Entity 'http://edamontology.org/operation_0578' - Label 'Plasmid map drawing'\n",
      "Entity 'http://edamontology.org/operation_0579' - Label 'Operon drawing'\n",
      "Entity 'http://edamontology.org/operation_1768' - Label 'Nucleic acid folding family identification'\n",
      "Entity 'http://edamontology.org/operation_1769' - Label 'Nucleic acid folding energy calculation'\n",
      "Entity 'http://edamontology.org/operation_1774' - Label 'Annotation retrieval'\n",
      "Entity 'http://edamontology.org/operation_1777' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/operation_1778' - Label 'Protein function comparison'\n",
      "Entity 'http://edamontology.org/operation_1780' - Label 'Sequence submission'\n",
      "Entity 'http://edamontology.org/operation_1781' - Label 'Gene regulatory network analysis'\n",
      "Entity 'http://edamontology.org/operation_1812' - Label 'Parsing'\n",
      "Entity 'http://edamontology.org/operation_1813' - Label 'Sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1814' - Label 'Structure retrieval'\n",
      "Entity 'http://edamontology.org/operation_1816' - Label 'Surface rendering'\n",
      "Entity 'http://edamontology.org/operation_1817' - Label 'Protein atom surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1818' - Label 'Protein atom surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1819' - Label 'Protein residue surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1820' - Label 'Protein residue surface calculation (vacuum accessible)'\n",
      "Entity 'http://edamontology.org/operation_1821' - Label 'Protein residue surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1822' - Label 'Protein residue surface calculation (vacuum molecular)'\n",
      "Entity 'http://edamontology.org/operation_1823' - Label 'Protein surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1824' - Label 'Protein surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1825' - Label 'Backbone torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1826' - Label 'Full torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1827' - Label 'Cysteine torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1828' - Label 'Tau angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1829' - Label 'Cysteine bridge detection'\n",
      "Entity 'http://edamontology.org/operation_1830' - Label 'Free cysteine detection'\n",
      "Entity 'http://edamontology.org/operation_1831' - Label 'Metal-bound cysteine detection'\n",
      "Entity 'http://edamontology.org/operation_1832' - Label 'Residue contact calculation (residue-nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_1834' - Label 'Protein-metal contact calculation'\n",
      "Entity 'http://edamontology.org/operation_1835' - Label 'Residue contact calculation (residue-negative ion)'\n",
      "Entity 'http://edamontology.org/operation_1836' - Label 'Residue bump detection'\n",
      "Entity 'http://edamontology.org/operation_1837' - Label 'Residue symmetry contact calculation'\n",
      "Entity 'http://edamontology.org/operation_1838' - Label 'Residue contact calculation (residue-ligand)'\n",
      "Entity 'http://edamontology.org/operation_1839' - Label 'Salt bridge calculation'\n",
      "Entity 'http://edamontology.org/operation_1841' - Label 'Rotamer likelihood prediction'\n",
      "Entity 'http://edamontology.org/operation_1842' - Label 'Proline mutation value calculation'\n",
      "Entity 'http://edamontology.org/operation_1843' - Label 'Residue packing validation'\n",
      "Entity 'http://edamontology.org/operation_1844' - Label 'Protein geometry validation'\n",
      "Entity 'http://edamontology.org/operation_1845' - Label 'PDB file sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1846' - Label 'HET group detection'\n",
      "Entity 'http://edamontology.org/operation_1847' - Label 'DSSP secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_1848' - Label 'Structure formatting'\n",
      "Entity 'http://edamontology.org/operation_1850' - Label 'Protein cysteine and disulfide bond assignment'\n",
      "Entity 'http://edamontology.org/operation_1913' - Label 'Residue validation'\n",
      "Entity 'http://edamontology.org/operation_1914' - Label 'Structure retrieval (water)'\n",
      "Entity 'http://edamontology.org/operation_2008' - Label 'siRNA duplex prediction'\n",
      "Entity 'http://edamontology.org/operation_2089' - Label 'Sequence alignment refinement'\n",
      "Entity 'http://edamontology.org/operation_2120' - Label 'Listfile processing'\n",
      "Entity 'http://edamontology.org/operation_2121' - Label 'Sequence file editing'\n",
      "Entity 'http://edamontology.org/operation_2122' - Label 'Sequence alignment file processing'\n",
      "Entity 'http://edamontology.org/operation_2123' - Label 'Small molecule data processing'\n",
      "Entity 'http://edamontology.org/operation_2222' - Label 'Data retrieval (ontology annotation)'\n",
      "Entity 'http://edamontology.org/operation_2224' - Label 'Data retrieval (ontology concept)'\n",
      "Entity 'http://edamontology.org/operation_2233' - Label 'Representative sequence identification'\n",
      "Entity 'http://edamontology.org/operation_2234' - Label 'Structure file processing'\n",
      "Entity 'http://edamontology.org/operation_2237' - Label 'Data retrieval (sequence profile)'\n",
      "Entity 'http://edamontology.org/operation_2238' - Label 'Statistical calculation'\n",
      "Entity 'http://edamontology.org/operation_2239' - Label '3D-1D scoring matrix generation'\n",
      "Entity 'http://edamontology.org/operation_2241' - Label 'Transmembrane protein visualisation'\n",
      "Entity 'http://edamontology.org/operation_2246' - Label 'Demonstration'\n",
      "Entity 'http://edamontology.org/operation_2264' - Label 'Data retrieval (pathway or network)'\n",
      "Entity 'http://edamontology.org/operation_2265' - Label 'Data retrieval (identifier)'\n",
      "Entity 'http://edamontology.org/operation_2284' - Label 'Nucleic acid density plotting'\n",
      "Entity 'http://edamontology.org/operation_2403' - Label 'Sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2404' - Label 'Sequence motif analysis'\n",
      "Entity 'http://edamontology.org/operation_2405' - Label 'Protein interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2406' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2407' - Label 'Annotation processing'\n",
      "Entity 'http://edamontology.org/operation_2408' - Label 'Sequence feature analysis'\n",
      "Entity 'http://edamontology.org/operation_2409' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2410' - Label 'Gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2411' - Label 'Structural profile processing'\n",
      "Entity 'http://edamontology.org/operation_2412' - Label 'Data index processing'\n",
      "Entity 'http://edamontology.org/operation_2413' - Label 'Sequence profile processing'\n",
      "Entity 'http://edamontology.org/operation_2414' - Label 'Protein function analysis'\n",
      "Entity 'http://edamontology.org/operation_2415' - Label 'Protein folding analysis'\n",
      "Entity 'http://edamontology.org/operation_2416' - Label 'Protein secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2417' - Label 'Physicochemical property data processing'\n",
      "Entity 'http://edamontology.org/operation_2419' - Label 'Primer and probe design'\n",
      "Entity 'http://edamontology.org/operation_2420' - Label 'Operation (typed)'\n",
      "Entity 'http://edamontology.org/operation_2421' - Label 'Database search'\n",
      "Entity 'http://edamontology.org/operation_2422' - Label 'Data retrieval'\n",
      "Entity 'http://edamontology.org/operation_2423' - Label 'Prediction and recognition'\n",
      "Entity 'http://edamontology.org/operation_2424' - Label 'Comparison'\n",
      "Entity 'http://edamontology.org/operation_2425' - Label 'Optimisation and refinement'\n",
      "Entity 'http://edamontology.org/operation_2426' - Label 'Modelling and simulation'\n",
      "Entity 'http://edamontology.org/operation_2427' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2428' - Label 'Validation'\n",
      "Entity 'http://edamontology.org/operation_2429' - Label 'Mapping'\n",
      "Entity 'http://edamontology.org/operation_2430' - Label 'Design'\n",
      "Entity 'http://edamontology.org/operation_2432' - Label 'Microarray data processing'\n",
      "Entity 'http://edamontology.org/operation_2433' - Label 'Codon usage table processing'\n",
      "Entity 'http://edamontology.org/operation_2434' - Label 'Data retrieval (codon usage table)'\n",
      "Entity 'http://edamontology.org/operation_2435' - Label 'Gene expression profile processing'\n",
      "Entity 'http://edamontology.org/operation_2436' - Label 'Gene-set enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_2437' - Label 'Gene regulatory network prediction'\n",
      "Entity 'http://edamontology.org/operation_2438' - Label 'Pathway or network processing'\n",
      "Entity 'http://edamontology.org/operation_2439' - Label 'RNA secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2440' - Label 'Structure processing (RNA)'\n",
      "Entity 'http://edamontology.org/operation_2441' - Label 'RNA structure prediction'\n",
      "Entity 'http://edamontology.org/operation_2442' - Label 'DNA structure prediction'\n",
      "Entity 'http://edamontology.org/operation_2443' - Label 'Phylogenetic tree processing'\n",
      "Entity 'http://edamontology.org/operation_2444' - Label 'Protein secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2445' - Label 'Protein interaction network processing'\n",
      "Entity 'http://edamontology.org/operation_2446' - Label 'Sequence processing'\n",
      "Entity 'http://edamontology.org/operation_2447' - Label 'Sequence processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2448' - Label 'Sequence processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2451' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2452' - Label 'Sequence cluster processing'\n",
      "Entity 'http://edamontology.org/operation_2453' - Label 'Feature table processing'\n",
      "Entity 'http://edamontology.org/operation_2454' - Label 'Gene prediction'\n",
      "Entity 'http://edamontology.org/operation_2456' - Label 'GPCR classification'\n",
      "Entity 'http://edamontology.org/operation_2457' - Label 'GPCR coupling selectivity prediction'\n",
      "Entity 'http://edamontology.org/operation_2459' - Label 'Structure processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2460' - Label 'Protein atom surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2461' - Label 'Protein residue surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2462' - Label 'Protein surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2463' - Label 'Sequence alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2464' - Label 'Protein-protein binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2465' - Label 'Structure processing'\n",
      "Entity 'http://edamontology.org/operation_2466' - Label 'Map annotation'\n",
      "Entity 'http://edamontology.org/operation_2467' - Label 'Data retrieval (protein annotation)'\n",
      "Entity 'http://edamontology.org/operation_2468' - Label 'Data retrieval (phylogenetic tree)'\n",
      "Entity 'http://edamontology.org/operation_2469' - Label 'Data retrieval (protein interaction annotation)'\n",
      "Entity 'http://edamontology.org/operation_2470' - Label 'Data retrieval (protein family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2471' - Label 'Data retrieval (RNA family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2472' - Label 'Data retrieval (gene annotation)'\n",
      "Entity 'http://edamontology.org/operation_2473' - Label 'Data retrieval (genotype and phenotype annotation)'\n",
      "Entity 'http://edamontology.org/operation_2474' - Label 'Protein architecture comparison'\n",
      "Entity 'http://edamontology.org/operation_2475' - Label 'Protein architecture recognition'\n",
      "Entity 'http://edamontology.org/operation_2476' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/operation_2478' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2479' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2480' - Label 'Structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2481' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2482' - Label 'Secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2483' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2485' - Label 'Helical wheel drawing'\n",
      "Entity 'http://edamontology.org/operation_2486' - Label 'Topology diagram drawing'\n",
      "Entity 'http://edamontology.org/operation_2487' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2488' - Label 'Protein secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2489' - Label 'Subcellular localisation prediction'\n",
      "Entity 'http://edamontology.org/operation_2490' - Label 'Residue contact calculation (residue-residue)'\n",
      "Entity 'http://edamontology.org/operation_2491' - Label 'Hydrogen bond calculation (inter-residue)'\n",
      "Entity 'http://edamontology.org/operation_2492' - Label 'Protein interaction prediction'\n",
      "Entity 'http://edamontology.org/operation_2493' - Label 'Codon usage data processing'\n",
      "Entity 'http://edamontology.org/operation_2495' - Label 'Expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2496' - Label 'Gene regulatory network processing'\n",
      "Entity 'http://edamontology.org/operation_2497' - Label 'Pathway or network analysis'\n",
      "Entity 'http://edamontology.org/operation_2498' - Label 'Sequencing-based expression profile data analysis'\n",
      "Entity 'http://edamontology.org/operation_2499' - Label 'Splicing analysis'\n",
      "Entity 'http://edamontology.org/operation_2500' - Label 'Microarray raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_2501' - Label 'Nucleic acid analysis'\n",
      "Entity 'http://edamontology.org/operation_2502' - Label 'Protein analysis'\n",
      "Entity 'http://edamontology.org/operation_2503' - Label 'Sequence data processing'\n",
      "Entity 'http://edamontology.org/operation_2504' - Label 'Structural data processing'\n",
      "Entity 'http://edamontology.org/operation_2505' - Label 'Text processing'\n",
      "Entity 'http://edamontology.org/operation_2506' - Label 'Protein sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2507' - Label 'Nucleic acid sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2508' - Label 'Nucleic acid sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2509' - Label 'Protein sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2510' - Label 'DNA back-translation'\n",
      "Entity 'http://edamontology.org/operation_2511' - Label 'Sequence editing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2512' - Label 'Sequence editing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2513' - Label 'Sequence generation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2514' - Label 'Sequence generation (protein)'\n",
      "Entity 'http://edamontology.org/operation_2515' - Label 'Nucleic acid sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2516' - Label 'Protein sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2518' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2519' - Label 'Structure processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2520' - Label 'DNA mapping'\n",
      "Entity 'http://edamontology.org/operation_2521' - Label 'Map data processing'\n",
      "Entity 'http://edamontology.org/operation_2574' - Label 'Protein hydropathy calculation'\n",
      "Entity 'http://edamontology.org/operation_2575' - Label 'Binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2844' - Label 'Structure clustering'\n",
      "Entity 'http://edamontology.org/operation_2871' - Label 'Sequence tagged site (STS) mapping'\n",
      "Entity 'http://edamontology.org/operation_2928' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/operation_2929' - Label 'Protein fragment weight comparison'\n",
      "Entity 'http://edamontology.org/operation_2930' - Label 'Protein property comparison'\n",
      "Entity 'http://edamontology.org/operation_2931' - Label 'Secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2932' - Label 'Hopp and Woods plotting'\n",
      "Entity 'http://edamontology.org/operation_2934' - Label 'Cluster textual view generation'\n",
      "Entity 'http://edamontology.org/operation_2935' - Label 'Clustering profile plotting'\n",
      "Entity 'http://edamontology.org/operation_2936' - Label 'Dendrograph plotting'\n",
      "Entity 'http://edamontology.org/operation_2937' - Label 'Proximity map plotting'\n",
      "Entity 'http://edamontology.org/operation_2938' - Label 'Dendrogram visualisation'\n",
      "Entity 'http://edamontology.org/operation_2939' - Label 'Principal component visualisation'\n",
      "Entity 'http://edamontology.org/operation_2940' - Label 'Scatter plot plotting'\n",
      "Entity 'http://edamontology.org/operation_2941' - Label 'Whole microarray graph plotting'\n",
      "Entity 'http://edamontology.org/operation_2942' - Label 'Treemap visualisation'\n",
      "Entity 'http://edamontology.org/operation_2943' - Label 'Box-Whisker plot plotting'\n",
      "Entity 'http://edamontology.org/operation_2944' - Label 'Physical mapping'\n",
      "Entity 'http://edamontology.org/operation_2945' - Label 'Analysis'\n",
      "Entity 'http://edamontology.org/operation_2946' - Label 'Alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2947' - Label 'Article analysis'\n",
      "Entity 'http://edamontology.org/operation_2948' - Label 'Molecular interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2949' - Label 'Protein-protein interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2950' - Label 'Residue distance calculation'\n",
      "Entity 'http://edamontology.org/operation_2951' - Label 'Alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2952' - Label 'Structure alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2962' - Label 'Codon usage bias calculation'\n",
      "Entity 'http://edamontology.org/operation_2963' - Label 'Codon usage bias plotting'\n",
      "Entity 'http://edamontology.org/operation_2964' - Label 'Codon usage fraction calculation'\n",
      "Entity 'http://edamontology.org/operation_2990' - Label 'Classification'\n",
      "Entity 'http://edamontology.org/operation_2993' - Label 'Molecular interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2995' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/operation_2996' - Label 'Structure classification'\n",
      "Entity 'http://edamontology.org/operation_2997' - Label 'Protein comparison'\n",
      "Entity 'http://edamontology.org/operation_2998' - Label 'Nucleic acid comparison'\n",
      "Entity 'http://edamontology.org/operation_3023' - Label 'Prediction and recognition (protein)'\n",
      "Entity 'http://edamontology.org/operation_3024' - Label 'Prediction and recognition (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_3080' - Label 'Structure editing'\n",
      "Entity 'http://edamontology.org/operation_3081' - Label 'Sequence alignment editing'\n",
      "Entity 'http://edamontology.org/operation_3083' - Label 'Pathway or network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3084' - Label 'Protein function prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3087' - Label 'Protein sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3088' - Label 'Protein property calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3090' - Label 'Protein feature prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_3092' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/operation_3093' - Label 'Database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_3094' - Label 'Protein interaction network prediction'\n",
      "Entity 'http://edamontology.org/operation_3095' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/operation_3096' - Label 'Editing'\n",
      "Entity 'http://edamontology.org/operation_3180' - Label 'Sequence assembly validation'\n",
      "Entity 'http://edamontology.org/operation_3182' - Label 'Genome alignment'\n",
      "Entity 'http://edamontology.org/operation_3183' - Label 'Localised reassembly'\n",
      "Entity 'http://edamontology.org/operation_3184' - Label 'Sequence assembly visualisation'\n",
      "Entity 'http://edamontology.org/operation_3185' - Label 'Base-calling'\n",
      "Entity 'http://edamontology.org/operation_3186' - Label 'Bisulfite mapping'\n",
      "Entity 'http://edamontology.org/operation_3187' - Label 'Sequence contamination filtering'\n",
      "Entity 'http://edamontology.org/operation_3189' - Label 'Trim ends'\n",
      "Entity 'http://edamontology.org/operation_3190' - Label 'Trim vector'\n",
      "Entity 'http://edamontology.org/operation_3191' - Label 'Trim to reference'\n",
      "Entity 'http://edamontology.org/operation_3192' - Label 'Sequence trimming'\n",
      "Entity 'http://edamontology.org/operation_3194' - Label 'Genome feature comparison'\n",
      "Entity 'http://edamontology.org/operation_3195' - Label 'Sequencing error detection'\n",
      "Entity 'http://edamontology.org/operation_3196' - Label 'Genotyping'\n",
      "Entity 'http://edamontology.org/operation_3197' - Label 'Genetic variation analysis'\n",
      "Entity 'http://edamontology.org/operation_3198' - Label 'Read mapping'\n",
      "Entity 'http://edamontology.org/operation_3199' - Label 'Split read mapping'\n",
      "Entity 'http://edamontology.org/operation_3200' - Label 'DNA barcoding'\n",
      "Entity 'http://edamontology.org/operation_3201' - Label 'SNP calling'\n",
      "Entity 'http://edamontology.org/operation_3202' - Label 'Polymorphism detection'\n",
      "Entity 'http://edamontology.org/operation_3203' - Label 'Chromatogram visualisation'\n",
      "Entity 'http://edamontology.org/operation_3204' - Label 'Methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3205' - Label 'Methylation calling'\n",
      "Entity 'http://edamontology.org/operation_3206' - Label 'Whole genome methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3207' - Label 'Gene methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3208' - Label 'Genome visualisation'\n",
      "Entity 'http://edamontology.org/operation_3209' - Label 'Genome comparison'\n",
      "Entity 'http://edamontology.org/operation_3211' - Label 'Genome indexing'\n",
      "Entity 'http://edamontology.org/operation_3212' - Label 'Genome indexing (Burrows-Wheeler)'\n",
      "Entity 'http://edamontology.org/operation_3213' - Label 'Genome indexing (suffix arrays)'\n",
      "Entity 'http://edamontology.org/operation_3214' - Label 'Spectral analysis'\n",
      "Entity 'http://edamontology.org/operation_3215' - Label 'Peak detection'\n",
      "Entity 'http://edamontology.org/operation_3216' - Label 'Scaffolding'\n",
      "Entity 'http://edamontology.org/operation_3217' - Label 'Scaffold gap completion'\n",
      "Entity 'http://edamontology.org/operation_3218' - Label 'Sequencing quality control'\n",
      "Entity 'http://edamontology.org/operation_3219' - Label 'Read pre-processing'\n",
      "Entity 'http://edamontology.org/operation_3221' - Label 'Species frequency estimation'\n",
      "Entity 'http://edamontology.org/operation_3222' - Label 'Peak calling'\n",
      "Entity 'http://edamontology.org/operation_3223' - Label 'Differential gene expression profiling'\n",
      "Entity 'http://edamontology.org/operation_3224' - Label 'Gene set testing'\n",
      "Entity 'http://edamontology.org/operation_3225' - Label 'Variant classification'\n",
      "Entity 'http://edamontology.org/operation_3226' - Label 'Variant prioritisation'\n",
      "Entity 'http://edamontology.org/operation_3227' - Label 'Variant calling'\n",
      "Entity 'http://edamontology.org/operation_3228' - Label 'Structural variation detection'\n",
      "Entity 'http://edamontology.org/operation_3229' - Label 'Exome assembly'\n",
      "Entity 'http://edamontology.org/operation_3230' - Label 'Read depth analysis'\n",
      "Entity 'http://edamontology.org/operation_3232' - Label 'Gene expression QTL analysis'\n",
      "Entity 'http://edamontology.org/operation_3233' - Label 'Copy number estimation'\n",
      "Entity 'http://edamontology.org/operation_3237' - Label 'Primer removal'\n",
      "Entity 'http://edamontology.org/operation_3258' - Label 'Transcriptome assembly'\n",
      "Entity 'http://edamontology.org/operation_3259' - Label 'Transcriptome assembly (de novo)'\n",
      "Entity 'http://edamontology.org/operation_3260' - Label 'Transcriptome assembly (mapping)'\n",
      "Entity 'http://edamontology.org/operation_3267' - Label 'Sequence coordinate conversion'\n",
      "Entity 'http://edamontology.org/operation_3278' - Label 'Document similarity calculation'\n",
      "Entity 'http://edamontology.org/operation_3279' - Label 'Document clustering'\n",
      "Entity 'http://edamontology.org/operation_3280' - Label 'Named-entity and concept recognition'\n",
      "Entity 'http://edamontology.org/operation_3282' - Label 'ID mapping'\n",
      "Entity 'http://edamontology.org/operation_3283' - Label 'Anonymisation'\n",
      "Entity 'http://edamontology.org/operation_3289' - Label 'ID retrieval'\n",
      "Entity 'http://edamontology.org/operation_3348' - Label 'Sequence checksum generation'\n",
      "Entity 'http://edamontology.org/operation_3349' - Label 'Bibliography generation'\n",
      "Entity 'http://edamontology.org/operation_3350' - Label 'Protein quaternary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_3351' - Label 'Molecular surface analysis'\n",
      "Entity 'http://edamontology.org/operation_3352' - Label 'Ontology comparison'\n",
      "Entity 'http://edamontology.org/operation_3353' - Label 'Ontology comparison'\n",
      "Entity 'http://edamontology.org/operation_3357' - Label 'Format detection'\n",
      "Entity 'http://edamontology.org/operation_3359' - Label 'Splitting'\n",
      "Entity 'http://edamontology.org/operation_3429' - Label 'Generation'\n",
      "Entity 'http://edamontology.org/operation_3430' - Label 'Nucleic acid sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3431' - Label 'Deposition'\n",
      "Entity 'http://edamontology.org/operation_3432' - Label 'Clustering'\n",
      "Entity 'http://edamontology.org/operation_3433' - Label 'Assembly'\n",
      "Entity 'http://edamontology.org/operation_3434' - Label 'Conversion'\n",
      "Entity 'http://edamontology.org/operation_3435' - Label 'Standardisation and normalisation'\n",
      "Entity 'http://edamontology.org/operation_3436' - Label 'Aggregation'\n",
      "Entity 'http://edamontology.org/operation_3437' - Label 'Article comparison'\n",
      "Entity 'http://edamontology.org/operation_3438' - Label 'Calculation'\n",
      "Entity 'http://edamontology.org/operation_3439' - Label 'Pathway or network prediction'\n",
      "Entity 'http://edamontology.org/operation_3440' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_3441' - Label 'Plotting'\n",
      "Entity 'http://edamontology.org/operation_3443' - Label 'Image analysis'\n",
      "Entity 'http://edamontology.org/operation_3445' - Label 'Diffraction data analysis'\n",
      "Entity 'http://edamontology.org/operation_3446' - Label 'Cell migration analysis'\n",
      "Entity 'http://edamontology.org/operation_3447' - Label 'Diffraction data reduction'\n",
      "Entity 'http://edamontology.org/operation_3450' - Label 'Neurite measurement'\n",
      "Entity 'http://edamontology.org/operation_3453' - Label 'Diffraction data integration'\n",
      "Entity 'http://edamontology.org/operation_3454' - Label 'Phasing'\n",
      "Entity 'http://edamontology.org/operation_3455' - Label 'Molecular replacement'\n",
      "Entity 'http://edamontology.org/operation_3456' - Label 'Rigid body refinement'\n",
      "Entity 'http://edamontology.org/operation_3457' - Label 'Single particle analysis'\n",
      "Entity 'http://edamontology.org/operation_3458' - Label 'Single particle alignment and classification'\n",
      "Entity 'http://edamontology.org/operation_3459' - Label 'Functional clustering'\n",
      "Entity 'http://edamontology.org/operation_3460' - Label 'Taxonomic classification'\n",
      "Entity 'http://edamontology.org/operation_3461' - Label 'Virulence prediction'\n",
      "Entity 'http://edamontology.org/operation_3463' - Label 'Expression correlation analysis'\n",
      "Entity 'http://edamontology.org/operation_3465' - Label 'Correlation'\n",
      "Entity 'http://edamontology.org/operation_3469' - Label 'RNA structure covariance model generation'\n",
      "Entity 'http://edamontology.org/operation_3470' - Label 'RNA secondary structure prediction (shape-based)'\n",
      "Entity 'http://edamontology.org/operation_3471' - Label 'Nucleic acid folding prediction (alignment-based)'\n",
      "Entity 'http://edamontology.org/operation_3472' - Label 'k-mer counting'\n",
      "Entity 'http://edamontology.org/operation_3478' - Label 'Phylogenetic reconstruction'\n",
      "Entity 'http://edamontology.org/operation_3480' - Label 'Probabilistic data generation'\n",
      "Entity 'http://edamontology.org/operation_3481' - Label 'Probabilistic sequence generation'\n",
      "Entity 'http://edamontology.org/operation_3482' - Label 'Antimicrobial resistance prediction'\n",
      "Entity 'http://edamontology.org/operation_3501' - Label 'Enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_3502' - Label 'Chemical similarity enrichment'\n",
      "Entity 'http://edamontology.org/operation_3503' - Label 'Incident curve plotting'\n",
      "Entity 'http://edamontology.org/operation_3504' - Label 'Variant pattern analysis'\n",
      "Entity 'http://edamontology.org/operation_3545' - Label 'Mathematical modelling'\n",
      "Entity 'http://edamontology.org/operation_3552' - Label 'Microscope image visualisation'\n",
      "Entity 'http://edamontology.org/operation_3553' - Label 'Image annotation'\n",
      "Entity 'http://edamontology.org/operation_3557' - Label 'Imputation'\n",
      "Entity 'http://edamontology.org/operation_3559' - Label 'Ontology visualisation'\n",
      "Entity 'http://edamontology.org/operation_3560' - Label 'Maximum occurrence analysis'\n",
      "Entity 'http://edamontology.org/operation_3561' - Label 'Database comparison'\n",
      "Entity 'http://edamontology.org/operation_3562' - Label 'Network simulation'\n",
      "Entity 'http://edamontology.org/operation_3563' - Label 'RNA-seq read count analysis'\n",
      "Entity 'http://edamontology.org/operation_3564' - Label 'Chemical redundancy removal'\n",
      "Entity 'http://edamontology.org/operation_3565' - Label 'RNA-seq time series data analysis'\n",
      "Entity 'http://edamontology.org/operation_3566' - Label 'Simulated gene expression data generation'\n",
      "Entity 'http://edamontology.org/operation_3625' - Label 'Relation extraction'\n",
      "Entity 'http://edamontology.org/operation_3627' - Label 'Mass spectra calibration'\n",
      "Entity 'http://edamontology.org/operation_3628' - Label 'Chromatographic alignment'\n",
      "Entity 'http://edamontology.org/operation_3629' - Label 'Deisotoping'\n",
      "Entity 'http://edamontology.org/operation_3630' - Label 'Protein quantification'\n",
      "Entity 'http://edamontology.org/operation_3631' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/operation_3632' - Label 'Isotopic distributions calculation'\n",
      "Entity 'http://edamontology.org/operation_3633' - Label 'Retention time prediction'\n",
      "Entity 'http://edamontology.org/operation_3634' - Label 'Label-free quantification'\n",
      "Entity 'http://edamontology.org/operation_3635' - Label 'Labeled quantification'\n",
      "Entity 'http://edamontology.org/operation_3636' - Label 'MRM/SRM'\n",
      "Entity 'http://edamontology.org/operation_3637' - Label 'Spectral counting'\n",
      "Entity 'http://edamontology.org/operation_3638' - Label 'SILAC'\n",
      "Entity 'http://edamontology.org/operation_3639' - Label 'iTRAQ'\n",
      "Entity 'http://edamontology.org/operation_3640' - Label '18O labeling'\n",
      "Entity 'http://edamontology.org/operation_3641' - Label 'TMT-tag'\n",
      "Entity 'http://edamontology.org/operation_3642' - Label 'Stable isotope dimethyl labelling'\n",
      "Entity 'http://edamontology.org/operation_3643' - Label 'Tag-based peptide identification'\n",
      "Entity 'http://edamontology.org/operation_3644' - Label 'de Novo sequencing'\n",
      "Entity 'http://edamontology.org/operation_3645' - Label 'PTM identification'\n",
      "Entity 'http://edamontology.org/operation_3646' - Label 'Peptide database search'\n",
      "Entity 'http://edamontology.org/operation_3647' - Label 'Blind peptide database search'\n",
      "Entity 'http://edamontology.org/operation_3648' - Label 'Validation of peptide-spectrum matches'\n",
      "Entity 'http://edamontology.org/operation_3649' - Label 'Target-Decoy'\n",
      "Entity 'http://edamontology.org/operation_3658' - Label 'Statistical inference'\n",
      "Entity 'http://edamontology.org/operation_3659' - Label 'Regression analysis'\n",
      "Entity 'http://edamontology.org/operation_3660' - Label 'Metabolic network modelling'\n",
      "Entity 'http://edamontology.org/operation_3661' - Label 'SNP annotation'\n",
      "Entity 'http://edamontology.org/operation_3662' - Label 'Ab-initio gene prediction'\n",
      "Entity 'http://edamontology.org/operation_3663' - Label 'Homology-based gene prediction'\n",
      "Entity 'http://edamontology.org/operation_3664' - Label 'Statistical modelling'\n",
      "Entity 'http://edamontology.org/operation_3666' - Label 'Molecular surface comparison'\n",
      "Entity 'http://edamontology.org/operation_3672' - Label 'Gene functional annotation'\n",
      "Entity 'http://edamontology.org/operation_3675' - Label 'Variant filtering'\n",
      "Entity 'http://edamontology.org/operation_3677' - Label 'Differential binding analysis'\n",
      "Entity 'http://edamontology.org/operation_3680' - Label 'RNA-Seq analysis'\n",
      "Entity 'http://edamontology.org/operation_3694' - Label 'Mass spectrum visualisation'\n",
      "Entity 'http://edamontology.org/operation_3695' - Label 'Filtering'\n",
      "Entity 'http://edamontology.org/operation_3703' - Label 'Reference identification'\n",
      "Entity 'http://edamontology.org/operation_3704' - Label 'Ion counting'\n",
      "Entity 'http://edamontology.org/operation_3705' - Label 'Isotope-coded protein label'\n",
      "Entity 'http://edamontology.org/operation_3715' - Label 'Metabolic labeling'\n",
      "Entity 'http://edamontology.org/operation_3730' - Label 'Cross-assembly'\n",
      "Entity 'http://edamontology.org/operation_3731' - Label 'Sample comparison'\n",
      "Entity 'http://edamontology.org/operation_3741' - Label 'Differential protein expression profiling'\n",
      "Entity 'http://edamontology.org/operation_3742' - Label 'Differential gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_3744' - Label 'Multiple sample visualisation'\n",
      "Entity 'http://edamontology.org/operation_3745' - Label 'Ancestral reconstruction'\n",
      "Entity 'http://edamontology.org/operation_3755' - Label 'PTM localisation'\n",
      "Entity 'http://edamontology.org/operation_3760' - Label 'Service management'\n",
      "Entity 'http://edamontology.org/operation_3761' - Label 'Service discovery'\n",
      "Entity 'http://edamontology.org/operation_3762' - Label 'Service composition'\n",
      "Entity 'http://edamontology.org/operation_3763' - Label 'Service invocation'\n",
      "Entity 'http://edamontology.org/operation_3766' - Label 'Weighted correlation network analysis'\n",
      "Entity 'http://edamontology.org/operation_3767' - Label 'Protein identification'\n",
      "Entity 'http://edamontology.org/operation_3778' - Label 'Text annotation'\n",
      "Entity 'http://edamontology.org/operation_3791' - Label 'Collapsing methods'\n",
      "Entity 'http://edamontology.org/operation_3792' - Label 'miRNA expression analysis'\n",
      "Entity 'http://edamontology.org/operation_3793' - Label 'Read summarisation'\n",
      "Entity 'http://edamontology.org/operation_3795' - Label 'In vitro selection'\n",
      "Entity 'http://edamontology.org/operation_3797' - Label 'Rarefaction'\n",
      "Entity 'http://edamontology.org/operation_3798' - Label 'Read binning'\n",
      "Entity 'http://edamontology.org/operation_3799' - Label 'Quantification'\n",
      "Entity 'http://edamontology.org/operation_3800' - Label 'RNA-Seq quantification'\n",
      "Entity 'http://edamontology.org/operation_3801' - Label 'Spectral library search'\n",
      "Entity 'http://edamontology.org/operation_3802' - Label 'Sorting'\n",
      "Entity 'http://edamontology.org/operation_3803' - Label 'Natural product identification'\n",
      "Entity 'http://edamontology.org/operation_3809' - Label 'DMR identification'\n",
      "Entity 'http://edamontology.org/operation_3840' - Label 'Multilocus sequence typing'\n",
      "Entity 'http://edamontology.org/operation_3860' - Label 'Spectrum calculation'\n",
      "Entity 'http://edamontology.org/operation_3890' - Label 'Trajectory visualization'\n",
      "Entity 'http://edamontology.org/operation_3891' - Label 'Essential dynamics'\n",
      "Entity 'http://edamontology.org/operation_3893' - Label 'Forcefield parameterisation'\n",
      "Entity 'http://edamontology.org/operation_3894' - Label 'DNA profiling'\n",
      "Entity 'http://edamontology.org/operation_3896' - Label 'Active site prediction'\n",
      "Entity 'http://edamontology.org/operation_3897' - Label 'Ligand-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3898' - Label 'Metal-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3899' - Label 'Protein-protein docking'\n",
      "Entity 'http://edamontology.org/operation_3900' - Label 'DNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3901' - Label 'RNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3902' - Label 'RNA binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3903' - Label 'DNA binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3904' - Label 'Protein disorder prediction'\n",
      "Entity 'http://edamontology.org/operation_3907' - Label 'Information extraction'\n",
      "Entity 'http://edamontology.org/operation_3908' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/operation_3918' - Label 'Genome analysis'\n",
      "Entity 'http://edamontology.org/operation_3919' - Label 'Methylation calling'\n",
      "Entity 'http://edamontology.org/operation_3920' - Label 'DNA testing'\n",
      "Entity 'http://edamontology.org/operation_3921' - Label 'Sequence read processing'\n",
      "Entity 'http://edamontology.org/operation_3925' - Label 'Network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3926' - Label 'Pathway visualisation'\n",
      "Entity 'http://edamontology.org/operation_3927' - Label 'Network analysis'\n",
      "Entity 'http://edamontology.org/operation_3928' - Label 'Pathway analysis'\n",
      "Entity 'http://edamontology.org/operation_3929' - Label 'Metabolic pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_3933' - Label 'Demultiplexing'\n",
      "Entity 'http://edamontology.org/operation_3935' - Label 'Dimensionality reduction'\n",
      "Entity 'http://edamontology.org/operation_3936' - Label 'Feature selection'\n",
      "Entity 'http://edamontology.org/operation_3937' - Label 'Feature extraction'\n",
      "Entity 'http://edamontology.org/operation_3938' - Label 'Virtual screening'\n",
      "Entity 'http://edamontology.org/operation_3942' - Label 'Tree dating'\n",
      "Entity 'http://edamontology.org/operation_3946' - Label 'Ecological modelling'\n",
      "Entity 'http://edamontology.org/operation_3947' - Label 'Phylogenetic tree reconciliation'\n",
      "Entity 'http://edamontology.org/operation_3950' - Label 'Selection detection'\n",
      "Entity 'http://edamontology.org/operation_3960' - Label 'Principal component analysis'\n",
      "Entity 'http://edamontology.org/operation_3961' - Label 'Copy number variation detection'\n",
      "Entity 'http://edamontology.org/operation_3962' - Label 'Deletion detection'\n",
      "Entity 'http://edamontology.org/operation_3963' - Label 'Duplication detection'\n",
      "Entity 'http://edamontology.org/operation_3964' - Label 'Complex CNV detection'\n",
      "Entity 'http://edamontology.org/operation_3965' - Label 'Amplification detection'\n",
      "Entity 'http://edamontology.org/operation_3968' - Label 'Adhesin prediction'\n",
      "Entity 'http://edamontology.org/operation_4008' - Label 'Protein design'\n",
      "Entity 'http://edamontology.org/operation_4009' - Label 'Small molecule design'\n",
      "Entity 'http://edamontology.org/operation_4031' - Label 'Power test'\n",
      "Entity 'http://edamontology.org/operation_4032' - Label 'DNA modification prediction'\n",
      "Entity 'http://edamontology.org/operation_4033' - Label 'Disease transmission analysis'\n",
      "Entity 'http://edamontology.org/operation_4034' - Label 'Multiple testing correction'\n",
      "Entity 'http://edamontology.org/topic_0003' - Label 'Topic'\n",
      "Entity 'http://edamontology.org/topic_0077' - Label 'Nucleic acids'\n",
      "Entity 'http://edamontology.org/topic_0078' - Label 'Proteins'\n",
      "Entity 'http://edamontology.org/topic_0079' - Label 'Metabolites'\n",
      "Entity 'http://edamontology.org/topic_0080' - Label 'Sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0081' - Label 'Structure analysis'\n",
      "Entity 'http://edamontology.org/topic_0082' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0083' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/topic_0084' - Label 'Phylogeny'\n",
      "Entity 'http://edamontology.org/topic_0085' - Label 'Functional genomics'\n",
      "Entity 'http://edamontology.org/topic_0089' - Label 'Ontology and terminology'\n",
      "Entity 'http://edamontology.org/topic_0090' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/topic_0091' - Label 'Bioinformatics'\n",
      "Entity 'http://edamontology.org/topic_0092' - Label 'Data visualisation'\n",
      "Entity 'http://edamontology.org/topic_0094' - Label 'Nucleic acid thermodynamics'\n",
      "Entity 'http://edamontology.org/topic_0097' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/topic_0099' - Label 'RNA'\n",
      "Entity 'http://edamontology.org/topic_0100' - Label 'Nucleic acid restriction'\n",
      "Entity 'http://edamontology.org/topic_0102' - Label 'Mapping'\n",
      "Entity 'http://edamontology.org/topic_0107' - Label 'Genetic codes and codon usage'\n",
      "Entity 'http://edamontology.org/topic_0108' - Label 'Protein expression'\n",
      "Entity 'http://edamontology.org/topic_0109' - Label 'Gene finding'\n",
      "Entity 'http://edamontology.org/topic_0110' - Label 'Transcription'\n",
      "Entity 'http://edamontology.org/topic_0111' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_0112' - Label 'Nucleic acid folding'\n",
      "Entity 'http://edamontology.org/topic_0114' - Label 'Gene structure'\n",
      "Entity 'http://edamontology.org/topic_0121' - Label 'Proteomics'\n",
      "Entity 'http://edamontology.org/topic_0122' - Label 'Structural genomics'\n",
      "Entity 'http://edamontology.org/topic_0123' - Label 'Protein properties'\n",
      "Entity 'http://edamontology.org/topic_0128' - Label 'Protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0130' - Label 'Protein folding, stability and design'\n",
      "Entity 'http://edamontology.org/topic_0133' - Label 'Two-dimensional gel electrophoresis'\n",
      "Entity 'http://edamontology.org/topic_0134' - Label 'Mass spectrometry'\n",
      "Entity 'http://edamontology.org/topic_0135' - Label 'Protein microarrays'\n",
      "Entity 'http://edamontology.org/topic_0137' - Label 'Protein hydropathy'\n",
      "Entity 'http://edamontology.org/topic_0140' - Label 'Protein targeting and localisation'\n",
      "Entity 'http://edamontology.org/topic_0141' - Label 'Protein cleavage sites and proteolysis'\n",
      "Entity 'http://edamontology.org/topic_0143' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/topic_0144' - Label 'Protein residue interactions'\n",
      "Entity 'http://edamontology.org/topic_0147' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0148' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_0149' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_0150' - Label 'Protein design'\n",
      "Entity 'http://edamontology.org/topic_0151' - Label 'G protein-coupled receptors (GPCR)'\n",
      "Entity 'http://edamontology.org/topic_0152' - Label 'Carbohydrates'\n",
      "Entity 'http://edamontology.org/topic_0153' - Label 'Lipids'\n",
      "Entity 'http://edamontology.org/topic_0154' - Label 'Small molecules'\n",
      "Entity 'http://edamontology.org/topic_0156' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/topic_0157' - Label 'Sequence composition, complexity and repeats'\n",
      "Entity 'http://edamontology.org/topic_0158' - Label 'Sequence motifs'\n",
      "Entity 'http://edamontology.org/topic_0159' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/topic_0160' - Label 'Sequence sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_0163' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/topic_0164' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/topic_0166' - Label 'Protein structural motifs and surfaces'\n",
      "Entity 'http://edamontology.org/topic_0167' - Label 'Structural (3D) profiles'\n",
      "Entity 'http://edamontology.org/topic_0172' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0173' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0174' - Label 'Ab initio structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0175' - Label 'Homology modelling'\n",
      "Entity 'http://edamontology.org/topic_0176' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/topic_0177' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/topic_0178' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0179' - Label 'Protein tertiary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0180' - Label 'Protein fold recognition'\n",
      "Entity 'http://edamontology.org/topic_0182' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0183' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/topic_0184' - Label 'Threading'\n",
      "Entity 'http://edamontology.org/topic_0188' - Label 'Sequence profiles and HMMs'\n",
      "Entity 'http://edamontology.org/topic_0191' - Label 'Phylogeny reconstruction'\n",
      "Entity 'http://edamontology.org/topic_0194' - Label 'Phylogenomics'\n",
      "Entity 'http://edamontology.org/topic_0195' - Label 'Virtual PCR'\n",
      "Entity 'http://edamontology.org/topic_0196' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/topic_0199' - Label 'Genetic variation'\n",
      "Entity 'http://edamontology.org/topic_0200' - Label 'Microarrays'\n",
      "Entity 'http://edamontology.org/topic_0202' - Label 'Pharmacology'\n",
      "Entity 'http://edamontology.org/topic_0203' - Label 'Gene expression'\n",
      "Entity 'http://edamontology.org/topic_0204' - Label 'Gene regulation'\n",
      "Entity 'http://edamontology.org/topic_0208' - Label 'Pharmacogenomics'\n",
      "Entity 'http://edamontology.org/topic_0209' - Label 'Medicinal chemistry'\n",
      "Entity 'http://edamontology.org/topic_0210' - Label 'Fish'\n",
      "Entity 'http://edamontology.org/topic_0211' - Label 'Flies'\n",
      "Entity 'http://edamontology.org/topic_0213' - Label 'Mice or rats'\n",
      "Entity 'http://edamontology.org/topic_0215' - Label 'Worms'\n",
      "Entity 'http://edamontology.org/topic_0217' - Label 'Literature analysis'\n",
      "Entity 'http://edamontology.org/topic_0218' - Label 'Natural language processing'\n",
      "Entity 'http://edamontology.org/topic_0219' - Label 'Data submission, annotation, and curation'\n",
      "Entity 'http://edamontology.org/topic_0220' - Label 'Document, record and content management'\n",
      "Entity 'http://edamontology.org/topic_0221' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/topic_0222' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/topic_0593' - Label 'NMR'\n",
      "Entity 'http://edamontology.org/topic_0594' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/topic_0595' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/topic_0598' - Label 'Sequence motif or profile'\n",
      "Entity 'http://edamontology.org/topic_0601' - Label 'Protein modifications'\n",
      "Entity 'http://edamontology.org/topic_0602' - Label 'Molecular interactions, pathways and networks'\n",
      "Entity 'http://edamontology.org/topic_0605' - Label 'Informatics'\n",
      "Entity 'http://edamontology.org/topic_0606' - Label 'Literature data resources'\n",
      "Entity 'http://edamontology.org/topic_0607' - Label 'Laboratory information management'\n",
      "Entity 'http://edamontology.org/topic_0608' - Label 'Cell and tissue culture'\n",
      "Entity 'http://edamontology.org/topic_0610' - Label 'Ecology'\n",
      "Entity 'http://edamontology.org/topic_0611' - Label 'Electron microscopy'\n",
      "Entity 'http://edamontology.org/topic_0612' - Label 'Cell cycle'\n",
      "Entity 'http://edamontology.org/topic_0613' - Label 'Peptides and amino acids'\n",
      "Entity 'http://edamontology.org/topic_0616' - Label 'Organelles'\n",
      "Entity 'http://edamontology.org/topic_0617' - Label 'Ribosomes'\n",
      "Entity 'http://edamontology.org/topic_0618' - Label 'Scents'\n",
      "Entity 'http://edamontology.org/topic_0620' - Label 'Drugs and target structures'\n",
      "Entity 'http://edamontology.org/topic_0621' - Label 'Model organisms'\n",
      "Entity 'http://edamontology.org/topic_0622' - Label 'Genomics'\n",
      "Entity 'http://edamontology.org/topic_0623' - Label 'Gene and protein families'\n",
      "Entity 'http://edamontology.org/topic_0624' - Label 'Chromosomes'\n",
      "Entity 'http://edamontology.org/topic_0625' - Label 'Genotype and phenotype'\n",
      "Entity 'http://edamontology.org/topic_0629' - Label 'Gene expression and microarray'\n",
      "Entity 'http://edamontology.org/topic_0632' - Label 'Probes and primers'\n",
      "Entity 'http://edamontology.org/topic_0634' - Label 'Pathology'\n",
      "Entity 'http://edamontology.org/topic_0635' - Label 'Specific protein resources'\n",
      "Entity 'http://edamontology.org/topic_0637' - Label 'Taxonomy'\n",
      "Entity 'http://edamontology.org/topic_0639' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0640' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0641' - Label 'Repeat sequences'\n",
      "Entity 'http://edamontology.org/topic_0642' - Label 'Low complexity sequences'\n",
      "Entity 'http://edamontology.org/topic_0644' - Label 'Proteome'\n",
      "Entity 'http://edamontology.org/topic_0654' - Label 'DNA'\n",
      "Entity 'http://edamontology.org/topic_0655' - Label 'Coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0659' - Label 'Functional, regulatory and non-coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0660' - Label 'rRNA'\n",
      "Entity 'http://edamontology.org/topic_0663' - Label 'tRNA'\n",
      "Entity 'http://edamontology.org/topic_0694' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/topic_0697' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/topic_0698' - Label 'Protein tertiary structure'\n",
      "Entity 'http://edamontology.org/topic_0722' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/topic_0724' - Label 'Protein families'\n",
      "Entity 'http://edamontology.org/topic_0736' - Label 'Protein folds and structural domains'\n",
      "Entity 'http://edamontology.org/topic_0740' - Label 'Nucleic acid sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0741' - Label 'Protein sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0747' - Label 'Nucleic acid sites and features'\n",
      "Entity 'http://edamontology.org/topic_0748' - Label 'Protein sites and features'\n",
      "Entity 'http://edamontology.org/topic_0749' - Label 'Transcription factors and regulatory sites'\n",
      "Entity 'http://edamontology.org/topic_0751' - Label 'Phosphorylation sites'\n",
      "Entity 'http://edamontology.org/topic_0753' - Label 'Metabolic pathways'\n",
      "Entity 'http://edamontology.org/topic_0754' - Label 'Signaling pathways'\n",
      "Entity 'http://edamontology.org/topic_0767' - Label 'Protein and peptide identification'\n",
      "Entity 'http://edamontology.org/topic_0769' - Label 'Workflows'\n",
      "Entity 'http://edamontology.org/topic_0770' - Label 'Data types and objects'\n",
      "Entity 'http://edamontology.org/topic_0771' - Label 'Theoretical biology'\n",
      "Entity 'http://edamontology.org/topic_0779' - Label 'Mitochondria'\n",
      "Entity 'http://edamontology.org/topic_0780' - Label 'Plant biology'\n",
      "Entity 'http://edamontology.org/topic_0781' - Label 'Virology'\n",
      "Entity 'http://edamontology.org/topic_0782' - Label 'Fungi'\n",
      "Entity 'http://edamontology.org/topic_0783' - Label 'Pathogens'\n",
      "Entity 'http://edamontology.org/topic_0786' - Label 'Arabidopsis'\n",
      "Entity 'http://edamontology.org/topic_0787' - Label 'Rice'\n",
      "Entity 'http://edamontology.org/topic_0796' - Label 'Genetic mapping and linkage'\n",
      "Entity 'http://edamontology.org/topic_0797' - Label 'Comparative genomics'\n",
      "Entity 'http://edamontology.org/topic_0798' - Label 'Mobile genetic elements'\n",
      "Entity 'http://edamontology.org/topic_0803' - Label 'Human disease'\n",
      "Entity 'http://edamontology.org/topic_0804' - Label 'Immunology'\n",
      "Entity 'http://edamontology.org/topic_0820' - Label 'Membrane and lipoproteins'\n",
      "Entity 'http://edamontology.org/topic_0821' - Label 'Enzymes'\n",
      "Entity 'http://edamontology.org/topic_0922' - Label 'Primers'\n",
      "Entity 'http://edamontology.org/topic_1302' - Label 'PolyA signal or sites'\n",
      "Entity 'http://edamontology.org/topic_1304' - Label 'CpG island and isochores'\n",
      "Entity 'http://edamontology.org/topic_1305' - Label 'Restriction sites'\n",
      "Entity 'http://edamontology.org/topic_1307' - Label 'Splice sites'\n",
      "Entity 'http://edamontology.org/topic_1308' - Label 'Matrix/scaffold attachment sites'\n",
      "Entity 'http://edamontology.org/topic_1311' - Label 'Operon'\n",
      "Entity 'http://edamontology.org/topic_1312' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_1317' - Label 'Structural biology'\n",
      "Entity 'http://edamontology.org/topic_1456' - Label 'Protein membrane regions'\n",
      "Entity 'http://edamontology.org/topic_1770' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/topic_1775' - Label 'Function analysis'\n",
      "Entity 'http://edamontology.org/topic_1811' - Label 'Prokaryotes and Archaea'\n",
      "Entity 'http://edamontology.org/topic_2225' - Label 'Protein databases'\n",
      "Entity 'http://edamontology.org/topic_2226' - Label 'Structure determination'\n",
      "Entity 'http://edamontology.org/topic_2229' - Label 'Cell biology'\n",
      "Entity 'http://edamontology.org/topic_2230' - Label 'Classification'\n",
      "Entity 'http://edamontology.org/topic_2232' - Label 'Lipoproteins'\n",
      "Entity 'http://edamontology.org/topic_2257' - Label 'Phylogeny visualisation'\n",
      "Entity 'http://edamontology.org/topic_2258' - Label 'Cheminformatics'\n",
      "Entity 'http://edamontology.org/topic_2259' - Label 'Systems biology'\n",
      "Entity 'http://edamontology.org/topic_2269' - Label 'Statistics and probability'\n",
      "Entity 'http://edamontology.org/topic_2271' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/topic_2275' - Label 'Molecular modelling'\n",
      "Entity 'http://edamontology.org/topic_2276' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/topic_2277' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/topic_2278' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/topic_2280' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/topic_2397' - Label 'Exons'\n",
      "Entity 'http://edamontology.org/topic_2399' - Label 'Gene transcription'\n",
      "Entity 'http://edamontology.org/topic_2533' - Label 'DNA mutation'\n",
      "Entity 'http://edamontology.org/topic_2640' - Label 'Oncology'\n",
      "Entity 'http://edamontology.org/topic_2661' - Label 'Toxins and targets'\n",
      "Entity 'http://edamontology.org/topic_2754' - Label 'Introns'\n",
      "Entity 'http://edamontology.org/topic_2807' - Label 'Tool topic'\n",
      "Entity 'http://edamontology.org/topic_2809' - Label 'Study topic'\n",
      "Entity 'http://edamontology.org/topic_2811' - Label 'Nomenclature'\n",
      "Entity 'http://edamontology.org/topic_2813' - Label 'Disease genes and proteins'\n",
      "Entity 'http://edamontology.org/topic_2814' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/topic_2815' - Label 'Human biology'\n",
      "Entity 'http://edamontology.org/topic_2816' - Label 'Gene resources'\n",
      "Entity 'http://edamontology.org/topic_2817' - Label 'Yeast'\n",
      "Entity 'http://edamontology.org/topic_2818' - Label 'Eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2819' - Label 'Invertebrates'\n",
      "Entity 'http://edamontology.org/topic_2820' - Label 'Vertebrates'\n",
      "Entity 'http://edamontology.org/topic_2821' - Label 'Unicellular eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2826' - Label 'Protein structure alignment'\n",
      "Entity 'http://edamontology.org/topic_2828' - Label 'X-ray diffraction'\n",
      "Entity 'http://edamontology.org/topic_2829' - Label 'Ontologies, nomenclature and classification'\n",
      "Entity 'http://edamontology.org/topic_2830' - Label 'Immunoproteins and antigens'\n",
      "Entity 'http://edamontology.org/topic_2839' - Label 'Molecules'\n",
      "Entity 'http://edamontology.org/topic_2840' - Label 'Toxicology'\n",
      "Entity 'http://edamontology.org/topic_2842' - Label 'High-throughput sequencing'\n",
      "Entity 'http://edamontology.org/topic_2846' - Label 'Gene regulatory networks'\n",
      "Entity 'http://edamontology.org/topic_2847' - Label 'Disease (specific)'\n",
      "Entity 'http://edamontology.org/topic_2867' - Label 'VNTR'\n",
      "Entity 'http://edamontology.org/topic_2868' - Label 'Microsatellites'\n",
      "Entity 'http://edamontology.org/topic_2869' - Label 'RFLP'\n",
      "Entity 'http://edamontology.org/topic_2885' - Label 'DNA polymorphism'\n",
      "Entity 'http://edamontology.org/topic_2953' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/topic_3032' - Label 'Primer or probe design'\n",
      "Entity 'http://edamontology.org/topic_3038' - Label 'Structure databases'\n",
      "Entity 'http://edamontology.org/topic_3039' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/topic_3041' - Label 'Sequence databases'\n",
      "Entity 'http://edamontology.org/topic_3042' - Label 'Nucleic acid sequences'\n",
      "Entity 'http://edamontology.org/topic_3043' - Label 'Protein sequences'\n",
      "Entity 'http://edamontology.org/topic_3044' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/topic_3047' - Label 'Molecular biology'\n",
      "Entity 'http://edamontology.org/topic_3048' - Label 'Mammals'\n",
      "Entity 'http://edamontology.org/topic_3050' - Label 'Biodiversity'\n",
      "Entity 'http://edamontology.org/topic_3052' - Label 'Sequence clusters and classification'\n",
      "Entity 'http://edamontology.org/topic_3053' - Label 'Genetics'\n",
      "Entity 'http://edamontology.org/topic_3055' - Label 'Quantitative genetics'\n",
      "Entity 'http://edamontology.org/topic_3056' - Label 'Population genetics'\n",
      "Entity 'http://edamontology.org/topic_3060' - Label 'Regulatory RNA'\n",
      "Entity 'http://edamontology.org/topic_3061' - Label 'Documentation and help'\n",
      "Entity 'http://edamontology.org/topic_3062' - Label 'Genetic organisation'\n",
      "Entity 'http://edamontology.org/topic_3063' - Label 'Medical informatics'\n",
      "Entity 'http://edamontology.org/topic_3064' - Label 'Developmental biology'\n",
      "Entity 'http://edamontology.org/topic_3065' - Label 'Embryology'\n",
      "Entity 'http://edamontology.org/topic_3067' - Label 'Anatomy'\n",
      "Entity 'http://edamontology.org/topic_3068' - Label 'Literature and language'\n",
      "Entity 'http://edamontology.org/topic_3070' - Label 'Biology'\n",
      "Entity 'http://edamontology.org/topic_3071' - Label 'Data management'\n",
      "Entity 'http://edamontology.org/topic_3072' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/topic_3073' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/topic_3074' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/topic_3075' - Label 'Biological system modelling'\n",
      "Entity 'http://edamontology.org/topic_3077' - Label 'Data acquisition'\n",
      "Entity 'http://edamontology.org/topic_3078' - Label 'Genes and proteins resources'\n",
      "Entity 'http://edamontology.org/topic_3118' - Label 'Protein topological domains'\n",
      "Entity 'http://edamontology.org/topic_3120' - Label 'Protein variants'\n",
      "Entity 'http://edamontology.org/topic_3123' - Label 'Expression signals'\n",
      "Entity 'http://edamontology.org/topic_3125' - Label 'DNA binding sites'\n",
      "Entity 'http://edamontology.org/topic_3126' - Label 'Nucleic acid repeats'\n",
      "Entity 'http://edamontology.org/topic_3127' - Label 'DNA replication and recombination'\n",
      "Entity 'http://edamontology.org/topic_3135' - Label 'Signal or transit peptide'\n",
      "Entity 'http://edamontology.org/topic_3139' - Label 'Sequence tagged sites'\n",
      "Entity 'http://edamontology.org/topic_3168' - Label 'Sequencing'\n",
      "Entity 'http://edamontology.org/topic_3169' - Label 'ChIP-seq'\n",
      "Entity 'http://edamontology.org/topic_3170' - Label 'RNA-Seq'\n",
      "Entity 'http://edamontology.org/topic_3171' - Label 'DNA methylation'\n",
      "Entity 'http://edamontology.org/topic_3172' - Label 'Metabolomics'\n",
      "Entity 'http://edamontology.org/topic_3173' - Label 'Epigenomics'\n",
      "Entity 'http://edamontology.org/topic_3174' - Label 'Metagenomics'\n",
      "Entity 'http://edamontology.org/topic_3175' - Label 'Structural variation'\n",
      "Entity 'http://edamontology.org/topic_3176' - Label 'DNA packaging'\n",
      "Entity 'http://edamontology.org/topic_3177' - Label 'DNA-Seq'\n",
      "Entity 'http://edamontology.org/topic_3178' - Label 'RNA-Seq alignment'\n",
      "Entity 'http://edamontology.org/topic_3179' - Label 'ChIP-on-chip'\n",
      "Entity 'http://edamontology.org/topic_3263' - Label 'Data security'\n",
      "Entity 'http://edamontology.org/topic_3277' - Label 'Sample collections'\n",
      "Entity 'http://edamontology.org/topic_3292' - Label 'Biochemistry'\n",
      "Entity 'http://edamontology.org/topic_3293' - Label 'Phylogenetics'\n",
      "Entity 'http://edamontology.org/topic_3295' - Label 'Epigenetics'\n",
      "Entity 'http://edamontology.org/topic_3297' - Label 'Biotechnology'\n",
      "Entity 'http://edamontology.org/topic_3298' - Label 'Phenomics'\n",
      "Entity 'http://edamontology.org/topic_3299' - Label 'Evolutionary biology'\n",
      "Entity 'http://edamontology.org/topic_3300' - Label 'Physiology'\n",
      "Entity 'http://edamontology.org/topic_3301' - Label 'Microbiology'\n",
      "Entity 'http://edamontology.org/topic_3302' - Label 'Parasitology'\n",
      "Entity 'http://edamontology.org/topic_3303' - Label 'Medicine'\n",
      "Entity 'http://edamontology.org/topic_3304' - Label 'Neurobiology'\n",
      "Entity 'http://edamontology.org/topic_3305' - Label 'Public health and epidemiology'\n",
      "Entity 'http://edamontology.org/topic_3306' - Label 'Biophysics'\n",
      "Entity 'http://edamontology.org/topic_3307' - Label 'Computational biology'\n",
      "Entity 'http://edamontology.org/topic_3308' - Label 'Transcriptomics'\n",
      "Entity 'http://edamontology.org/topic_3314' - Label 'Chemistry'\n",
      "Entity 'http://edamontology.org/topic_3315' - Label 'Mathematics'\n",
      "Entity 'http://edamontology.org/topic_3316' - Label 'Computer science'\n",
      "Entity 'http://edamontology.org/topic_3318' - Label 'Physics'\n",
      "Entity 'http://edamontology.org/topic_3320' - Label 'RNA splicing'\n",
      "Entity 'http://edamontology.org/topic_3321' - Label 'Molecular genetics'\n",
      "Entity 'http://edamontology.org/topic_3322' - Label 'Respiratory medicine'\n",
      "Entity 'http://edamontology.org/topic_3323' - Label 'Metabolic disease'\n",
      "Entity 'http://edamontology.org/topic_3324' - Label 'Infectious disease'\n",
      "Entity 'http://edamontology.org/topic_3325' - Label 'Rare diseases'\n",
      "Entity 'http://edamontology.org/topic_3332' - Label 'Computational chemistry'\n",
      "Entity 'http://edamontology.org/topic_3334' - Label 'Neurology'\n",
      "Entity 'http://edamontology.org/topic_3335' - Label 'Cardiology'\n",
      "Entity 'http://edamontology.org/topic_3336' - Label 'Drug discovery'\n",
      "Entity 'http://edamontology.org/topic_3337' - Label 'Biobank'\n",
      "Entity 'http://edamontology.org/topic_3338' - Label 'Mouse clinic'\n",
      "Entity 'http://edamontology.org/topic_3339' - Label 'Microbial collection'\n",
      "Entity 'http://edamontology.org/topic_3340' - Label 'Cell culture collection'\n",
      "Entity 'http://edamontology.org/topic_3341' - Label 'Clone library'\n",
      "Entity 'http://edamontology.org/topic_3342' - Label 'Translational medicine'\n",
      "Entity 'http://edamontology.org/topic_3343' - Label 'Compound libraries and screening'\n",
      "Entity 'http://edamontology.org/topic_3344' - Label 'Biomedical science'\n",
      "Entity 'http://edamontology.org/topic_3345' - Label 'Data identity and mapping'\n",
      "Entity 'http://edamontology.org/topic_3346' - Label 'Sequence search'\n",
      "Entity 'http://edamontology.org/topic_3360' - Label 'Biomarkers'\n",
      "Entity 'http://edamontology.org/topic_3361' - Label 'Laboratory techniques'\n",
      "Entity 'http://edamontology.org/topic_3365' - Label 'Data architecture, analysis and design'\n",
      "Entity 'http://edamontology.org/topic_3366' - Label 'Data integration and warehousing'\n",
      "Entity 'http://edamontology.org/topic_3368' - Label 'Biomaterials'\n",
      "Entity 'http://edamontology.org/topic_3369' - Label 'Chemical biology'\n",
      "Entity 'http://edamontology.org/topic_3370' - Label 'Analytical chemistry'\n",
      "Entity 'http://edamontology.org/topic_3371' - Label 'Synthetic chemistry'\n",
      "Entity 'http://edamontology.org/topic_3372' - Label 'Software engineering'\n",
      "Entity 'http://edamontology.org/topic_3373' - Label 'Drug development'\n",
      "Entity 'http://edamontology.org/topic_3374' - Label 'Biotherapeutics'\n",
      "Entity 'http://edamontology.org/topic_3375' - Label 'Drug metabolism'\n",
      "Entity 'http://edamontology.org/topic_3376' - Label 'Medicines research and development'\n",
      "Entity 'http://edamontology.org/topic_3377' - Label 'Safety sciences'\n",
      "Entity 'http://edamontology.org/topic_3378' - Label 'Pharmacovigilance'\n",
      "Entity 'http://edamontology.org/topic_3379' - Label 'Preclinical and clinical studies'\n",
      "Entity 'http://edamontology.org/topic_3382' - Label 'Imaging'\n",
      "Entity 'http://edamontology.org/topic_3383' - Label 'Bioimaging'\n",
      "Entity 'http://edamontology.org/topic_3384' - Label 'Medical imaging'\n",
      "Entity 'http://edamontology.org/topic_3385' - Label 'Light microscopy'\n",
      "Entity 'http://edamontology.org/topic_3386' - Label 'Laboratory animal science'\n",
      "Entity 'http://edamontology.org/topic_3387' - Label 'Marine biology'\n",
      "Entity 'http://edamontology.org/topic_3388' - Label 'Molecular medicine'\n",
      "Entity 'http://edamontology.org/topic_3390' - Label 'Nutritional science'\n",
      "Entity 'http://edamontology.org/topic_3391' - Label 'Omics'\n",
      "Entity 'http://edamontology.org/topic_3393' - Label 'Quality affairs'\n",
      "Entity 'http://edamontology.org/topic_3394' - Label 'Regulatory affairs'\n",
      "Entity 'http://edamontology.org/topic_3395' - Label 'Regenerative medicine'\n",
      "Entity 'http://edamontology.org/topic_3396' - Label 'Systems medicine'\n",
      "Entity 'http://edamontology.org/topic_3397' - Label 'Veterinary medicine'\n",
      "Entity 'http://edamontology.org/topic_3398' - Label 'Bioengineering'\n",
      "Entity 'http://edamontology.org/topic_3399' - Label 'Geriatric medicine'\n",
      "Entity 'http://edamontology.org/topic_3400' - Label 'Allergy, clinical immunology and immunotherapeutics'\n",
      "Entity 'http://edamontology.org/topic_3401' - Label 'Pain medicine'\n",
      "Entity 'http://edamontology.org/topic_3402' - Label 'Anaesthesiology'\n",
      "Entity 'http://edamontology.org/topic_3403' - Label 'Critical care medicine'\n",
      "Entity 'http://edamontology.org/topic_3404' - Label 'Dermatology'\n",
      "Entity 'http://edamontology.org/topic_3405' - Label 'Dentistry'\n",
      "Entity 'http://edamontology.org/topic_3406' - Label 'Ear, nose and throat medicine'\n",
      "Entity 'http://edamontology.org/topic_3407' - Label 'Endocrinology and metabolism'\n",
      "Entity 'http://edamontology.org/topic_3408' - Label 'Haematology'\n",
      "Entity 'http://edamontology.org/topic_3409' - Label 'Gastroenterology'\n",
      "Entity 'http://edamontology.org/topic_3410' - Label 'Gender medicine'\n",
      "Entity 'http://edamontology.org/topic_3411' - Label 'Gynaecology and obstetrics'\n",
      "Entity 'http://edamontology.org/topic_3412' - Label 'Hepatic and biliary medicine'\n",
      "Entity 'http://edamontology.org/topic_3413' - Label 'Infectious tropical disease'\n",
      "Entity 'http://edamontology.org/topic_3414' - Label 'Trauma medicine'\n",
      "Entity 'http://edamontology.org/topic_3415' - Label 'Medical toxicology'\n",
      "Entity 'http://edamontology.org/topic_3416' - Label 'Musculoskeletal medicine'\n",
      "Entity 'http://edamontology.org/topic_3417' - Label 'Ophthalmology'\n",
      "Entity 'http://edamontology.org/topic_3418' - Label 'Paediatrics'\n",
      "Entity 'http://edamontology.org/topic_3419' - Label 'Psychiatry'\n",
      "Entity 'http://edamontology.org/topic_3420' - Label 'Reproductive health'\n",
      "Entity 'http://edamontology.org/topic_3421' - Label 'Surgery'\n",
      "Entity 'http://edamontology.org/topic_3422' - Label 'Urology and nephrology'\n",
      "Entity 'http://edamontology.org/topic_3423' - Label 'Complementary medicine'\n",
      "Entity 'http://edamontology.org/topic_3444' - Label 'MRI'\n",
      "Entity 'http://edamontology.org/topic_3448' - Label 'Neutron diffraction'\n",
      "Entity 'http://edamontology.org/topic_3452' - Label 'Tomography'\n",
      "Entity 'http://edamontology.org/topic_3473' - Label 'Data mining'\n",
      "Entity 'http://edamontology.org/topic_3474' - Label 'Machine learning'\n",
      "Entity 'http://edamontology.org/topic_3489' - Label 'Database management'\n",
      "Entity 'http://edamontology.org/topic_3500' - Label 'Zoology'\n",
      "Entity 'http://edamontology.org/topic_3510' - Label 'Protein sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3511' - Label 'Nucleic acid sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3512' - Label 'Gene transcripts'\n",
      "Entity 'http://edamontology.org/topic_3514' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_3515' - Label 'Protein-drug interactions'\n",
      "Entity 'http://edamontology.org/topic_3516' - Label 'Genotyping experiment'\n",
      "Entity 'http://edamontology.org/topic_3517' - Label 'GWAS study'\n",
      "Entity 'http://edamontology.org/topic_3518' - Label 'Microarray experiment'\n",
      "Entity 'http://edamontology.org/topic_3519' - Label 'PCR experiment'\n",
      "Entity 'http://edamontology.org/topic_3520' - Label 'Proteomics experiment'\n",
      "Entity 'http://edamontology.org/topic_3521' - Label '2D PAGE experiment'\n",
      "Entity 'http://edamontology.org/topic_3522' - Label 'Northern blot experiment'\n",
      "Entity 'http://edamontology.org/topic_3523' - Label 'RNAi experiment'\n",
      "Entity 'http://edamontology.org/topic_3524' - Label 'Simulation experiment'\n",
      "Entity 'http://edamontology.org/topic_3525' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_3526' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_3527' - Label 'Cellular process pathways'\n",
      "Entity 'http://edamontology.org/topic_3528' - Label 'Disease pathways'\n",
      "Entity 'http://edamontology.org/topic_3529' - Label 'Environmental information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3530' - Label 'Genetic information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3531' - Label 'Protein super-secondary structure'\n",
      "Entity 'http://edamontology.org/topic_3533' - Label 'Protein active sites'\n",
      "Entity 'http://edamontology.org/topic_3534' - Label 'Protein binding sites'\n",
      "Entity 'http://edamontology.org/topic_3535' - Label 'Protein-nucleic acid binding sites'\n",
      "Entity 'http://edamontology.org/topic_3536' - Label 'Protein cleavage sites'\n",
      "Entity 'http://edamontology.org/topic_3537' - Label 'Protein chemical modifications'\n",
      "Entity 'http://edamontology.org/topic_3538' - Label 'Protein disordered structure'\n",
      "Entity 'http://edamontology.org/topic_3539' - Label 'Protein domains'\n",
      "Entity 'http://edamontology.org/topic_3540' - Label 'Protein key folding sites'\n",
      "Entity 'http://edamontology.org/topic_3541' - Label 'Protein post-translational modifications'\n",
      "Entity 'http://edamontology.org/topic_3542' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/topic_3543' - Label 'Protein sequence repeats'\n",
      "Entity 'http://edamontology.org/topic_3544' - Label 'Protein signal peptides'\n",
      "Entity 'http://edamontology.org/topic_3569' - Label 'Applied mathematics'\n",
      "Entity 'http://edamontology.org/topic_3570' - Label 'Pure mathematics'\n",
      "Entity 'http://edamontology.org/topic_3571' - Label 'Data governance'\n",
      "Entity 'http://edamontology.org/topic_3572' - Label 'Data quality management'\n",
      "Entity 'http://edamontology.org/topic_3573' - Label 'Freshwater biology'\n",
      "Entity 'http://edamontology.org/topic_3574' - Label 'Human genetics'\n",
      "Entity 'http://edamontology.org/topic_3575' - Label 'Tropical medicine'\n",
      "Entity 'http://edamontology.org/topic_3576' - Label 'Medical biotechnology'\n",
      "Entity 'http://edamontology.org/topic_3577' - Label 'Personalised medicine'\n",
      "Entity 'http://edamontology.org/topic_3656' - Label 'Immunoprecipitation experiment'\n",
      "Entity 'http://edamontology.org/topic_3673' - Label 'Whole genome sequencing'\n",
      "Entity 'http://edamontology.org/topic_3674' - Label 'Methylated DNA immunoprecipitation'\n",
      "Entity 'http://edamontology.org/topic_3676' - Label 'Exome sequencing'\n",
      "Entity 'http://edamontology.org/topic_3678' - Label 'Experimental design and studies'\n",
      "Entity 'http://edamontology.org/topic_3679' - Label 'Animal study'\n",
      "Entity 'http://edamontology.org/topic_3697' - Label 'Microbial ecology'\n",
      "Entity 'http://edamontology.org/topic_3794' - Label 'RNA immunoprecipitation'\n",
      "Entity 'http://edamontology.org/topic_3796' - Label 'Population genomics'\n",
      "Entity 'http://edamontology.org/topic_3810' - Label 'Agricultural science'\n",
      "Entity 'http://edamontology.org/topic_3837' - Label 'Metagenomic sequencing'\n",
      "Entity 'http://edamontology.org/topic_3855' - Label 'Environmental sciences'\n",
      "Entity 'http://edamontology.org/topic_3892' - Label 'Biomolecular simulation'\n",
      "Entity 'http://edamontology.org/topic_3895' - Label 'Synthetic biology'\n",
      "Entity 'http://edamontology.org/topic_3912' - Label 'Genetic engineering'\n",
      "Entity 'http://edamontology.org/topic_3922' - Label 'Proteogenomics'\n",
      "Entity 'http://edamontology.org/topic_3923' - Label 'Genome resequencing'\n",
      "Entity 'http://edamontology.org/topic_3930' - Label 'Immunogenetics'\n",
      "Entity 'http://edamontology.org/topic_3931' - Label 'Chemometrics'\n",
      "Entity 'http://edamontology.org/topic_3934' - Label 'Cytometry'\n",
      "Entity 'http://edamontology.org/topic_3939' - Label 'Metabolic engineering'\n",
      "Entity 'http://edamontology.org/topic_3940' - Label 'Chromosome conformation capture'\n",
      "Entity 'http://edamontology.org/topic_3941' - Label 'Metatranscriptomics'\n",
      "Entity 'http://edamontology.org/topic_3943' - Label 'Paleogenomics'\n",
      "Entity 'http://edamontology.org/topic_3944' - Label 'Cladistics'\n",
      "Entity 'http://edamontology.org/topic_3945' - Label 'Molecular evolution'\n",
      "Entity 'http://edamontology.org/topic_3948' - Label 'Immunoinformatics'\n",
      "Entity 'http://edamontology.org/topic_3954' - Label 'Echography'\n",
      "Entity 'http://edamontology.org/topic_3955' - Label 'Fluxomics'\n",
      "Entity 'http://edamontology.org/topic_3957' - Label 'Protein interaction experiment'\n",
      "Entity 'http://edamontology.org/topic_3958' - Label 'Copy number variation'\n",
      "Entity 'http://edamontology.org/topic_3959' - Label 'Cytogenetics'\n",
      "Entity 'http://edamontology.org/topic_3966' - Label 'Vaccinology'\n",
      "Entity 'http://edamontology.org/topic_3967' - Label 'Immunomics'\n",
      "Entity 'http://edamontology.org/topic_3974' - Label 'Epistasis'\n",
      "Entity 'http://edamontology.org/topic_4010' - Label 'Open science'\n",
      "Entity 'http://edamontology.org/topic_4011' - Label 'Data rescue'\n",
      "Entity 'http://edamontology.org/topic_4012' - Label 'FAIR data'\n",
      "Entity 'http://edamontology.org/topic_4013' - Label 'Antimicrobial Resistance'\n",
      "Entity 'http://edamontology.org/topic_4014' - Label 'Electroencephalography'\n",
      "Entity 'http://edamontology.org/topic_4016' - Label 'Electrocardiography'\n",
      "Entity 'http://edamontology.org/topic_4017' - Label 'Cryogenic electron microscopy'\n",
      "Entity 'http://edamontology.org/topic_4019' - Label 'Biosciences'\n",
      "Entity 'http://edamontology.org/topic_4020' - Label 'Carbon cycle'\n",
      "Entity 'http://edamontology.org/topic_4021' - Label 'Multiomics'\n",
      "Entity 'http://edamontology.org/topic_4027' - Label 'Ribosome Profiling'\n",
      "Entity 'http://edamontology.org/topic_4028' - Label 'Single-Cell Sequencing'\n",
      "Entity 'http://edamontology.org/topic_4029' - Label 'Acoustics'\n",
      "Entity 'http://edamontology.org/topic_4030' - Label 'Microfluidics'\n",
      "Entity 'http://edamontology.org/topic_4037' - Label 'Genomic imprinting'\n",
      "Entity 'http://edamontology.org/topic_4038' - Label 'Metabarcoding'\n",
      "Entity 'http://www.geneontology.org/formats/oboInOwl#ObsoleteClass' - Label 'Obsolete concept (EDAM)'\n",
      "Entity 'http://www.w3.org/2002/07/owl#DeprecatedClass' - Label 'None'\n"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?label WHERE {\n",
    "    \n",
    "    ?entity a owl:Class .\n",
    "    OPTIONAL  {?entity rdfs:label ?label .}\n",
    "    \n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:28:58.299173Z",
     "start_time": "2024-02-13T17:28:58.179473Z"
    }
   },
   "execution_count": 33
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'http://edamontology.org/data_1002' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_1003' - Label 'Chemical registry number (Beilstein)'\n",
      "Entity 'http://edamontology.org/data_1004' - Label 'Chemical registry number (Gmelin)'\n",
      "Entity 'http://edamontology.org/data_1027' - Label 'Gene ID (NCBI)'\n",
      "Entity 'http://edamontology.org/data_1031' - Label 'Gene ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1032' - Label 'Gene ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1033' - Label 'Ensembl gene ID'\n",
      "Entity 'http://edamontology.org/data_1036' - Label 'TIGR identifier'\n",
      "Entity 'http://edamontology.org/data_1039' - Label 'SCOP domain identifier'\n",
      "Entity 'http://edamontology.org/data_1040' - Label 'CATH domain ID'\n",
      "Entity 'http://edamontology.org/data_1041' - Label 'SCOP concise classification string (sccs)'\n",
      "Entity 'http://edamontology.org/data_1042' - Label 'SCOP sunid'\n",
      "Entity 'http://edamontology.org/data_1043' - Label 'CATH node ID'\n",
      "Entity 'http://edamontology.org/data_1066' - Label 'Sequence alignment ID'\n",
      "Entity 'http://edamontology.org/data_1100' - Label 'PIR identifier'\n",
      "Entity 'http://edamontology.org/data_1102' - Label 'Gramene primary identifier'\n",
      "Entity 'http://edamontology.org/data_1103' - Label 'EMBL/GenBank/DDBJ ID'\n",
      "Entity 'http://edamontology.org/data_1104' - Label 'Sequence cluster ID (UniGene)'\n",
      "Entity 'http://edamontology.org/data_1105' - Label 'dbEST accession'\n",
      "Entity 'http://edamontology.org/data_1106' - Label 'dbSNP ID'\n",
      "Entity 'http://edamontology.org/data_1113' - Label 'Sequence cluster ID (COG)'\n",
      "Entity 'http://edamontology.org/data_1118' - Label 'HMMER hidden Markov model ID'\n",
      "Entity 'http://edamontology.org/data_1119' - Label 'JASPAR profile ID'\n",
      "Entity 'http://edamontology.org/data_1123' - Label 'TreeBASE study accession number'\n",
      "Entity 'http://edamontology.org/data_1124' - Label 'TreeFam accession number'\n",
      "Entity 'http://edamontology.org/data_1128' - Label 'AAindex ID'\n",
      "Entity 'http://edamontology.org/data_1129' - Label 'BIND accession number'\n",
      "Entity 'http://edamontology.org/data_1134' - Label 'InterPro secondary accession'\n",
      "Entity 'http://edamontology.org/data_1135' - Label 'Gene3D ID'\n",
      "Entity 'http://edamontology.org/data_1140' - Label 'Superfamily hidden Markov model number'\n",
      "Entity 'http://edamontology.org/data_1141' - Label 'TIGRFam ID'\n",
      "Entity 'http://edamontology.org/data_1143' - Label 'TRANSFAC accession number'\n",
      "Entity 'http://edamontology.org/data_1146' - Label 'EMDB ID'\n",
      "Entity 'http://edamontology.org/data_1148' - Label 'GermOnline ID'\n",
      "Entity 'http://edamontology.org/data_1149' - Label 'EMAGE ID'\n",
      "Entity 'http://edamontology.org/data_1151' - Label 'HGVbase ID'\n",
      "Entity 'http://edamontology.org/data_1154' - Label 'KEGG object identifier'\n",
      "Entity 'http://edamontology.org/data_1157' - Label 'Pathway ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_1158' - Label 'Pathway ID (INOH)'\n",
      "Entity 'http://edamontology.org/data_1159' - Label 'Pathway ID (PATIKA)'\n",
      "Entity 'http://edamontology.org/data_1160' - Label 'Pathway ID (CPDB)'\n",
      "Entity 'http://edamontology.org/data_1164' - Label 'MIRIAM URI'\n",
      "Entity 'http://edamontology.org/data_1167' - Label 'Taverna workflow ID'\n",
      "Entity 'http://edamontology.org/data_1175' - Label 'BioPax concept ID'\n",
      "Entity 'http://edamontology.org/data_1177' - Label 'MeSH concept ID'\n",
      "Entity 'http://edamontology.org/data_1178' - Label 'HGNC concept ID'\n",
      "Entity 'http://edamontology.org/data_1180' - Label 'Plant Ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_1181' - Label 'UMLS concept ID'\n",
      "Entity 'http://edamontology.org/data_1183' - Label 'EMAP concept ID'\n",
      "Entity 'http://edamontology.org/data_1184' - Label 'ChEBI concept ID'\n",
      "Entity 'http://edamontology.org/data_1185' - Label 'MGED concept ID'\n",
      "Entity 'http://edamontology.org/data_1186' - Label 'myGrid concept ID'\n",
      "Entity 'http://edamontology.org/data_1189' - Label 'Medline UI'\n",
      "Entity 'http://edamontology.org/data_1794' - Label 'Gene ID (PlasmoDB)'\n",
      "Entity 'http://edamontology.org/data_1795' - Label 'Gene ID (EcoGene)'\n",
      "Entity 'http://edamontology.org/data_1796' - Label 'Gene ID (FlyBase)'\n",
      "Entity 'http://edamontology.org/data_1802' - Label 'Gene ID (Gramene)'\n",
      "Entity 'http://edamontology.org/data_1803' - Label 'Gene ID (Virginia microbial)'\n",
      "Entity 'http://edamontology.org/data_1804' - Label 'Gene ID (SGN)'\n",
      "Entity 'http://edamontology.org/data_1873' - Label 'iHOP organism ID'\n",
      "Entity 'http://edamontology.org/data_1885' - Label 'Gene ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1886' - Label 'Blattner number'\n",
      "Entity 'http://edamontology.org/data_1891' - Label 'iHOP symbol'\n",
      "Entity 'http://edamontology.org/data_1896' - Label 'Locus ID (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1897' - Label 'Locus ID (MGG)'\n",
      "Entity 'http://edamontology.org/data_1898' - Label 'Locus ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1899' - Label 'Locus ID (CMR)'\n",
      "Entity 'http://edamontology.org/data_1900' - Label 'NCBI locus tag'\n",
      "Entity 'http://edamontology.org/data_1901' - Label 'Locus ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1902' - Label 'Locus ID (MMP)'\n",
      "Entity 'http://edamontology.org/data_1903' - Label 'Locus ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1904' - Label 'Locus ID (EntrezGene)'\n",
      "Entity 'http://edamontology.org/data_1905' - Label 'Locus ID (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1907' - Label 'Gene ID (KOME)'\n",
      "Entity 'http://edamontology.org/data_1908' - Label 'Locus ID (Tropgene)'\n",
      "Entity 'http://edamontology.org/data_2102' - Label 'KEGG organism code'\n",
      "Entity 'http://edamontology.org/data_2104' - Label 'BioCyc ID'\n",
      "Entity 'http://edamontology.org/data_2105' - Label 'Compound ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2106' - Label 'Reaction ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2107' - Label 'Enzyme ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2112' - Label 'FlyBase primary identifier'\n",
      "Entity 'http://edamontology.org/data_2174' - Label 'FlyBase secondary identifier'\n",
      "Entity 'http://edamontology.org/data_2209' - Label 'Mutation ID'\n",
      "Entity 'http://edamontology.org/data_2220' - Label 'Sequence cluster ID (SYSTERS)'\n",
      "Entity 'http://edamontology.org/data_2285' - Label 'Gene ID (MIPS)'\n",
      "Entity 'http://edamontology.org/data_2290' - Label 'EMBL accession'\n",
      "Entity 'http://edamontology.org/data_2292' - Label 'GenBank accession'\n",
      "Entity 'http://edamontology.org/data_2293' - Label 'Gramene secondary identifier'\n",
      "Entity 'http://edamontology.org/data_2297' - Label 'Gene ID (ECK)'\n",
      "Entity 'http://edamontology.org/data_2298' - Label 'Gene ID (HGNC)'\n",
      "Entity 'http://edamontology.org/data_2302' - Label 'STRING ID'\n",
      "Entity 'http://edamontology.org/data_2314' - Label 'GI number'\n",
      "Entity 'http://edamontology.org/data_2315' - Label 'NCBI version'\n",
      "Entity 'http://edamontology.org/data_2325' - Label 'REBASE enzyme number'\n",
      "Entity 'http://edamontology.org/data_2327' - Label 'GI number (protein)'\n",
      "Entity 'http://edamontology.org/data_2345' - Label 'Pathway ID (ConsensusPathDB)'\n",
      "Entity 'http://edamontology.org/data_2346' - Label 'Sequence cluster ID (UniRef)'\n",
      "Entity 'http://edamontology.org/data_2347' - Label 'Sequence cluster ID (UniRef100)'\n",
      "Entity 'http://edamontology.org/data_2348' - Label 'Sequence cluster ID (UniRef90)'\n",
      "Entity 'http://edamontology.org/data_2349' - Label 'Sequence cluster ID (UniRef50)'\n",
      "Entity 'http://edamontology.org/data_2356' - Label 'RFAM accession'\n",
      "Entity 'http://edamontology.org/data_2367' - Label 'ASTD ID'\n",
      "Entity 'http://edamontology.org/data_2368' - Label 'ASTD ID (exon)'\n",
      "Entity 'http://edamontology.org/data_2369' - Label 'ASTD ID (intron)'\n",
      "Entity 'http://edamontology.org/data_2370' - Label 'ASTD ID (polya)'\n",
      "Entity 'http://edamontology.org/data_2371' - Label 'ASTD ID (tss)'\n",
      "Entity 'http://edamontology.org/data_2374' - Label 'Spot serial number'\n",
      "Entity 'http://edamontology.org/data_2375' - Label 'Spot ID (HSC-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2380' - Label 'CABRI accession'\n",
      "Entity 'http://edamontology.org/data_2383' - Label 'EGA accession'\n",
      "Entity 'http://edamontology.org/data_2385' - Label 'RefSeq accession (protein)'\n",
      "Entity 'http://edamontology.org/data_2386' - Label 'EPD ID'\n",
      "Entity 'http://edamontology.org/data_2387' - Label 'TAIR accession'\n",
      "Entity 'http://edamontology.org/data_2388' - Label 'TAIR accession (At gene)'\n",
      "Entity 'http://edamontology.org/data_2389' - Label 'UniSTS accession'\n",
      "Entity 'http://edamontology.org/data_2390' - Label 'UNITE accession'\n",
      "Entity 'http://edamontology.org/data_2391' - Label 'UTR accession'\n",
      "Entity 'http://edamontology.org/data_2393' - Label 'mFLJ/mKIAA number'\n",
      "Entity 'http://edamontology.org/data_2398' - Label 'Ensembl protein ID'\n",
      "Entity 'http://edamontology.org/data_2578' - Label 'ArachnoServer ID'\n",
      "Entity 'http://edamontology.org/data_2580' - Label 'BindingDB Monomer ID'\n",
      "Entity 'http://edamontology.org/data_2588' - Label 'BlotBase blot ID'\n",
      "Entity 'http://edamontology.org/data_2591' - Label 'Brite hierarchy ID'\n",
      "Entity 'http://edamontology.org/data_2593' - Label 'BRENDA organism ID'\n",
      "Entity 'http://edamontology.org/data_2639' - Label 'PubChem ID'\n",
      "Entity 'http://edamontology.org/data_2700' - Label 'CATH identifier'\n",
      "Entity 'http://edamontology.org/data_2701' - Label 'CATH node ID (family)'\n",
      "Entity 'http://edamontology.org/data_2702' - Label 'Enzyme ID (CAZy)'\n",
      "Entity 'http://edamontology.org/data_2704' - Label 'Clone ID (IMAGE)'\n",
      "Entity 'http://edamontology.org/data_2709' - Label 'CleanEx entry name'\n",
      "Entity 'http://edamontology.org/data_2713' - Label 'Protein ID (CORUM)'\n",
      "Entity 'http://edamontology.org/data_2714' - Label 'CDD PSSM-ID'\n",
      "Entity 'http://edamontology.org/data_2715' - Label 'Protein ID (CuticleDB)'\n",
      "Entity 'http://edamontology.org/data_2716' - Label 'DBD ID'\n",
      "Entity 'http://edamontology.org/data_2719' - Label 'dbProbe ID'\n",
      "Entity 'http://edamontology.org/data_2721' - Label 'DiProDB ID'\n",
      "Entity 'http://edamontology.org/data_2723' - Label 'Protein ID (DisProt)'\n",
      "Entity 'http://edamontology.org/data_2725' - Label 'Ensembl transcript ID'\n",
      "Entity 'http://edamontology.org/data_2729' - Label 'COGEME EST ID'\n",
      "Entity 'http://edamontology.org/data_2730' - Label 'COGEME unisequence ID'\n",
      "Entity 'http://edamontology.org/data_2731' - Label 'Protein family ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_2736' - Label 'Sequence feature ID (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2737' - Label 'FIG ID'\n",
      "Entity 'http://edamontology.org/data_2738' - Label 'Gene ID (Xenbase)'\n",
      "Entity 'http://edamontology.org/data_2739' - Label 'Gene ID (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2741' - Label 'ABS ID'\n",
      "Entity 'http://edamontology.org/data_2742' - Label 'AraC-XylS ID'\n",
      "Entity 'http://edamontology.org/data_2744' - Label 'Locus ID (PseudoCAP)'\n",
      "Entity 'http://edamontology.org/data_2745' - Label 'Locus ID (UTR)'\n",
      "Entity 'http://edamontology.org/data_2746' - Label 'MonosaccharideDB ID'\n",
      "Entity 'http://edamontology.org/data_2756' - Label 'TCID'\n",
      "Entity 'http://edamontology.org/data_2759' - Label 'Gene ID (VectorBase)'\n",
      "Entity 'http://edamontology.org/data_2766' - Label 'HAMAP ID'\n",
      "Entity 'http://edamontology.org/data_2770' - Label 'HIT ID'\n",
      "Entity 'http://edamontology.org/data_2771' - Label 'HIX ID'\n",
      "Entity 'http://edamontology.org/data_2772' - Label 'HPA antibody id'\n",
      "Entity 'http://edamontology.org/data_2773' - Label 'IMGT/HLA ID'\n",
      "Entity 'http://edamontology.org/data_2774' - Label 'Gene ID (JCVI)'\n",
      "Entity 'http://edamontology.org/data_2776' - Label 'ConsensusPathDB entity ID'\n",
      "Entity 'http://edamontology.org/data_2778' - Label 'CCAP strain number'\n",
      "Entity 'http://edamontology.org/data_2780' - Label 'Stock number (TAIR)'\n",
      "Entity 'http://edamontology.org/data_2781' - Label 'REDIdb ID'\n",
      "Entity 'http://edamontology.org/data_2783' - Label 'Protein family ID (PANTHER)'\n",
      "Entity 'http://edamontology.org/data_2784' - Label 'RNAVirusDB ID'\n",
      "Entity 'http://edamontology.org/data_2786' - Label 'NCBI Genome Project ID'\n",
      "Entity 'http://edamontology.org/data_2787' - Label 'NCBI genome accession'\n",
      "Entity 'http://edamontology.org/data_2789' - Label 'Protein ID (TopDB)'\n",
      "Entity 'http://edamontology.org/data_2792' - Label 'Protein ID (PeroxiBase)'\n",
      "Entity 'http://edamontology.org/data_2793' - Label 'SISYPHUS ID'\n",
      "Entity 'http://edamontology.org/data_2794' - Label 'ORF ID'\n",
      "Entity 'http://edamontology.org/data_2797' - Label 'Protein ID (LGICdb)'\n",
      "Entity 'http://edamontology.org/data_2798' - Label 'MaizeDB ID'\n",
      "Entity 'http://edamontology.org/data_2799' - Label 'Gene ID (MfunGD)'\n",
      "Entity 'http://edamontology.org/data_2800' - Label 'Orpha number'\n",
      "Entity 'http://edamontology.org/data_2802' - Label 'Protein ID (EcID)'\n",
      "Entity 'http://edamontology.org/data_2803' - Label 'Clone ID (RefSeq)'\n",
      "Entity 'http://edamontology.org/data_2804' - Label 'Protein ID (ConoServer)'\n",
      "Entity 'http://edamontology.org/data_2805' - Label 'GeneSNP ID'\n",
      "Entity 'http://edamontology.org/data_2835' - Label 'Gene ID (VBASE2)'\n",
      "Entity 'http://edamontology.org/data_2836' - Label 'DPVweb ID'\n",
      "Entity 'http://edamontology.org/data_2915' - Label 'Gramene identifier'\n",
      "Entity 'http://edamontology.org/data_2916' - Label 'DDBJ accession'\n",
      "Entity 'http://edamontology.org/data_3029' - Label 'Protein ID (EMBL/GenBank/DDBJ)'\n",
      "Entity 'http://edamontology.org/data_3103' - Label 'ATC code'\n",
      "Entity 'http://edamontology.org/data_3264' - Label 'COSMIC ID'\n",
      "Entity 'http://edamontology.org/data_3265' - Label 'HGMD ID'\n",
      "Entity 'http://edamontology.org/data_3270' - Label 'Ensembl gene tree ID'\n",
      "Entity 'http://edamontology.org/data_3274' - Label 'MGI accession'\n",
      "Entity 'http://edamontology.org/data_3757' - Label 'Unimod ID'\n",
      "Entity 'http://edamontology.org/data_3769' - Label 'BRENDA ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_3856' - Label 'RNA central ID'\n"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "\n",
    "SELECT ?entity ?label ?property WHERE\n",
    "{\n",
    "    ?entity rdfs:subClassOf+ edam:data_2091 .\n",
    "    ?entity rdfs:label ?label .\n",
    "        VALUES ?property { edam:regex               \n",
    "                        }\n",
    "    FILTER NOT EXISTS {?entity ?property ?value .}\n",
    "    \n",
    "}ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:30:45.202465Z",
     "start_time": "2024-02-13T17:30:45.093914Z"
    }
   },
   "execution_count": 35
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "\n",
    "CONSTRUCT { \n",
    "    ?children_format rdfs:subClassOf ?restriction . \n",
    "    ?restriction rdf:type owl:Restriction ; \n",
    "            owl:onProperty edam:is_format_of  ; \n",
    "            owl:someValuesFrom ?data. \n",
    "}\n",
    "WHERE {\n",
    "    ?parent_format rdfs:subClassOf ?restriction . \n",
    "    ?restriction rdf:type owl:Restriction ; \n",
    "            owl:onProperty edam:is_format_of  ; \n",
    "            owl:someValuesFrom ?data.\n",
    "    ?children_format rdfs:subClassOf+ ?parent_format .\n",
    "    }\n",
    "\"\"\"\n",
    "\n",
    "#results = kg.query(query)\n",
    "\n",
    "#for r in results :\n",
    "#    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:31:55.773119Z",
     "start_time": "2024-02-13T17:31:55.728822Z"
    }
   },
   "execution_count": 39
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'http://edamontology.org/data_0849' - Label 'Sequence record'\n",
      "Entity 'http://edamontology.org/data_0863' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/data_0863' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/data_0871' - Label 'Phylogenetic character data'\n",
      "Entity 'http://edamontology.org/data_0872' - Label 'Phylogenetic tree'\n",
      "Entity 'http://edamontology.org/data_0872' - Label 'Phylogenetic tree'\n",
      "Entity 'http://edamontology.org/data_0883' - Label 'Structure'\n",
      "Entity 'http://edamontology.org/data_1255' - Label 'Sequence features'\n",
      "Entity 'http://edamontology.org/data_1354' - Label 'Sequence profile'\n",
      "Entity 'http://edamontology.org/data_1381' - Label 'Pair sequence alignment'\n",
      "Entity 'http://edamontology.org/data_2044' - Label 'Sequence'\n",
      "Entity 'http://edamontology.org/data_2044' - Label 'Sequence'\n",
      "Entity 'http://edamontology.org/data_2048' - Label 'Report'\n",
      "Entity 'http://edamontology.org/data_2091' - Label 'Accession'\n",
      "Entity 'http://edamontology.org/data_2091' - Label 'Accession'\n",
      "Entity 'http://edamontology.org/data_2098' - Label 'Job identifier'\n",
      "Entity 'http://edamontology.org/data_2099' - Label 'Name'\n",
      "Entity 'http://edamontology.org/data_2099' - Label 'Name'\n",
      "Entity 'http://edamontology.org/data_2099' - Label 'Name'\n",
      "Entity 'http://edamontology.org/data_2100' - Label 'Type'\n",
      "Entity 'http://edamontology.org/data_2968' - Label 'Image'\n",
      "Entity 'http://edamontology.org/data_2968' - Label 'Image'\n",
      "Entity 'http://edamontology.org/data_2976' - Label 'Protein sequence'\n",
      "Entity 'http://edamontology.org/data_2977' - Label 'Nucleic acid sequence'\n",
      "Entity 'http://edamontology.org/data_3424' - Label 'Raw image'\n",
      "Entity 'http://edamontology.org/format_1207' - Label 'nucleotide'\n",
      "Entity 'http://edamontology.org/format_1208' - Label 'protein'\n",
      "Entity 'http://edamontology.org/format_1212' - Label 'dna'\n",
      "Entity 'http://edamontology.org/format_1213' - Label 'rna'\n",
      "Entity 'http://edamontology.org/format_1997' - Label 'PHYLIP format'\n",
      "Entity 'http://edamontology.org/format_1998' - Label 'PHYLIP sequential'\n",
      "Entity 'http://edamontology.org/format_2183' - Label 'EMBLXML'\n",
      "Entity 'http://edamontology.org/format_2184' - Label 'cdsxml'\n",
      "Entity 'http://edamontology.org/format_2200' - Label 'FASTA-like (text)'\n",
      "Entity 'http://edamontology.org/format_2330' - Label 'Textual format'\n",
      "Entity 'http://edamontology.org/format_2330' - Label 'Textual format'\n",
      "Entity 'http://edamontology.org/format_2330' - Label 'Textual format'\n",
      "Entity 'http://edamontology.org/format_2331' - Label 'HTML'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2571' - Label 'Raw sequence format'\n",
      "Entity 'http://edamontology.org/format_3261' - Label 'RDF/XML'\n",
      "Entity 'http://edamontology.org/format_3462' - Label 'CRAM'\n",
      "Entity 'http://edamontology.org/format_3484' - Label 'ebwt'\n",
      "Entity 'http://edamontology.org/format_3485' - Label 'RSF'\n",
      "Entity 'http://edamontology.org/format_3487' - Label 'BSML'\n",
      "Entity 'http://edamontology.org/format_3491' - Label 'ebwtl'\n",
      "Entity 'http://edamontology.org/format_3607' - Label 'qual'\n",
      "Entity 'http://edamontology.org/format_3609' - Label 'qualillumina'\n",
      "Entity 'http://edamontology.org/format_3610' - Label 'qualsolid'\n",
      "Entity 'http://edamontology.org/format_3611' - Label 'qual454'\n",
      "Entity 'http://edamontology.org/format_3654' - Label 'mzXML'\n",
      "Entity 'http://edamontology.org/format_3655' - Label 'pepXML'\n",
      "Entity 'http://edamontology.org/format_3747' - Label 'protXML'\n",
      "Entity 'http://edamontology.org/format_3747' - Label 'protXML'\n",
      "Entity 'http://edamontology.org/format_3764' - Label 'idXML'\n",
      "Entity 'http://edamontology.org/format_3764' - Label 'idXML'\n",
      "Entity 'http://edamontology.org/format_3832' - Label 'consensusXML'\n",
      "Entity 'http://edamontology.org/format_3833' - Label 'featureXML'\n",
      "Entity 'http://edamontology.org/format_3834' - Label 'mzData'\n",
      "Entity 'http://edamontology.org/format_3835' - Label 'TIDE TXT'\n",
      "Entity 'http://edamontology.org/format_3836' - Label 'BLAST XML v2 results format'\n",
      "Entity 'http://edamontology.org/format_3836' - Label 'BLAST XML v2 results format'\n",
      "Entity 'http://edamontology.org/format_3836' - Label 'BLAST XML v2 results format'\n",
      "Entity 'http://edamontology.org/format_3982' - Label 'CHAIN'\n",
      "Entity 'http://edamontology.org/format_3983' - Label 'NET'\n",
      "Entity 'http://edamontology.org/format_3985' - Label 'gxformat2'\n",
      "Entity 'http://edamontology.org/format_3992' - Label 'CIGAR format'\n",
      "Entity 'http://edamontology.org/format_4000' - Label 'R markdown'\n",
      "Entity 'http://edamontology.org/format_4002' - Label 'pickle'\n",
      "Entity 'http://edamontology.org/format_4004' - Label 'SimTools repertoire file format'\n",
      "Entity 'http://edamontology.org/format_4006' - Label 'Zstandard format'\n",
      "Entity 'http://edamontology.org/operation_0558' - Label 'Phylogenetic tree annotation'\n",
      "Entity 'http://edamontology.org/topic_0077' - Label 'Nucleic acids'\n",
      "Entity 'http://edamontology.org/topic_0077' - Label 'Nucleic acids'\n",
      "Entity 'http://edamontology.org/topic_0078' - Label 'Proteins'\n",
      "Entity 'http://edamontology.org/topic_0080' - Label 'Sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0081' - Label 'Structure analysis'\n",
      "Entity 'http://edamontology.org/topic_0084' - Label 'Phylogeny'\n",
      "Entity 'http://edamontology.org/topic_0089' - Label 'Ontology and terminology'\n",
      "Entity 'http://edamontology.org/topic_0091' - Label 'Bioinformatics'\n",
      "Entity 'http://edamontology.org/topic_0121' - Label 'Proteomics'\n",
      "Entity 'http://edamontology.org/topic_0199' - Label 'Genetic variation'\n",
      "Entity 'http://edamontology.org/topic_0200' - Label 'Microarrays'\n",
      "Entity 'http://edamontology.org/topic_0203' - Label 'Gene expression'\n",
      "Entity 'http://edamontology.org/topic_0610' - Label 'Ecology'\n",
      "Entity 'http://edamontology.org/topic_0622' - Label 'Genomics'\n",
      "Entity 'http://edamontology.org/topic_0632' - Label 'Probes and primers'\n",
      "Entity 'http://edamontology.org/topic_0804' - Label 'Immunology'\n",
      "Entity 'http://edamontology.org/topic_0804' - Label 'Immunology'\n",
      "Entity 'http://edamontology.org/topic_2259' - Label 'Systems biology'\n",
      "Entity 'http://edamontology.org/topic_2269' - Label 'Statistics and probability'\n",
      "Entity 'http://edamontology.org/topic_2829' - Label 'Ontologies, nomenclature and classification'\n",
      "Entity 'http://edamontology.org/topic_3050' - Label 'Biodiversity'\n",
      "Entity 'http://edamontology.org/topic_3053' - Label 'Genetics'\n",
      "Entity 'http://edamontology.org/topic_3068' - Label 'Literature and language'\n",
      "Entity 'http://edamontology.org/topic_3071' - Label 'Data management'\n",
      "Entity 'http://edamontology.org/topic_3072' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/topic_3168' - Label 'Sequencing'\n",
      "Entity 'http://edamontology.org/topic_3172' - Label 'Metabolomics'\n",
      "Entity 'http://edamontology.org/topic_3173' - Label 'Epigenomics'\n",
      "Entity 'http://edamontology.org/topic_3176' - Label 'DNA packaging'\n",
      "Entity 'http://edamontology.org/topic_3293' - Label 'Phylogenetics'\n",
      "Entity 'http://edamontology.org/topic_3295' - Label 'Epigenetics'\n",
      "Entity 'http://edamontology.org/topic_3412' - Label 'Hepatic and biliary medicine'\n",
      "Entity 'http://edamontology.org/topic_3571' - Label 'Data governance'\n",
      "Entity 'http://edamontology.org/topic_3974' - Label 'Epistasis'\n"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?property ?label ?value WHERE {\n",
    "    VALUES ?property {\n",
    "                    <http://edamontology.org/citation>\n",
    "                    <http://edamontology.org/documentation>\n",
    "                    <http://edamontology.org/information_standard>\n",
    "                    <http://edamontology.org/oldParent>\n",
    "                    <http://edamontology.org/ontology_used>\n",
    "                    <http://edamontology.org/organisation>\n",
    "                    <http://www.geneontology.org/formats/oboInOwl#consider>\n",
    "                    <http://www.geneontology.org/formats/oboInOwl#inSubset>\n",
    "                    <http://www.geneontology.org/formats/oboInOwl#replacedBy>\n",
    "                    <http://www.w3.org/2000/01/rdf-schema#seeAlso>\n",
    "                    <http://www.w3.org/2000/01/rdf-schema#subClassOf>\n",
    "                    <http://www.w3.org/2000/01/rdf-schema#subPropertyOf>\n",
    "                    <http://www.w3.org/2002/07/owl#annotatedProperty>\n",
    "                    <http://www.w3.org/2002/07/owl#inverseOf>\n",
    "                    <http://www.w3.org/2002/07/owl#disjointWith>\n",
    "                    <http://www.w3.org/2000/01/rdf-schema#domain>\n",
    "                    <http://www.w3.org/2000/01/rdf-schema#range>\n",
    "\n",
    "\n",
    "}\n",
    "\n",
    "?entity ?property ?value .\n",
    "\n",
    "FILTER isLiteral(?value) \n",
    "?entity rdfs:label ?label .\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:32:43.406278Z",
     "start_time": "2024-02-13T17:32:42.503590Z"
    }
   },
   "execution_count": 40
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
      "\u001B[0;31mKeyboardInterrupt\u001B[0m                         Traceback (most recent call last)",
      "Cell \u001B[0;32mIn[42], line 37\u001B[0m\n\u001B[1;32m      1\u001B[0m query\u001B[38;5;241m=\u001B[39m\u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[1;32m      2\u001B[0m \u001B[38;5;124mPREFIX edam:<http://edamontology.org/>\u001B[39m\n\u001B[1;32m      3\u001B[0m \n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m     32\u001B[0m \u001B[38;5;124mORDER BY ?entity\u001B[39m\n\u001B[1;32m     33\u001B[0m \u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[1;32m     35\u001B[0m results \u001B[38;5;241m=\u001B[39m kg\u001B[38;5;241m.\u001B[39mquery(query)\n\u001B[0;32m---> 37\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m r \u001B[38;5;129;01min\u001B[39;00m results :\n\u001B[1;32m     38\u001B[0m     \u001B[38;5;28mprint\u001B[39m(\u001B[38;5;124mf\u001B[39m\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mEntity \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mentity\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m - Label \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mlabel\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m\"\u001B[39m) \n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/query.py:373\u001B[0m, in \u001B[0;36mResult.__iter__\u001B[0;34m(self)\u001B[0m\n\u001B[1;32m    369\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mtype \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mSELECT\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    370\u001B[0m     \u001B[38;5;66;03m# this iterates over ResultRows of variable bindings\u001B[39;00m\n\u001B[1;32m    372\u001B[0m     \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_genbindings:\n\u001B[0;32m--> 373\u001B[0m         \u001B[38;5;28;01mfor\u001B[39;00m b \u001B[38;5;129;01min\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_genbindings:\n\u001B[1;32m    374\u001B[0m             \u001B[38;5;28;01mif\u001B[39;00m b:  \u001B[38;5;66;03m# don't add a result row in case of empty binding {}\u001B[39;00m\n\u001B[1;32m    375\u001B[0m                 \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_bindings\u001B[38;5;241m.\u001B[39mappend(b)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:551\u001B[0m, in \u001B[0;36mevalDistinct\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    548\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mevalDistinct\u001B[39m(\n\u001B[1;32m    549\u001B[0m     ctx: QueryContext, part: CompValue\n\u001B[1;32m    550\u001B[0m ) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m Generator[FrozenBindings, \u001B[38;5;28;01mNone\u001B[39;00m, \u001B[38;5;28;01mNone\u001B[39;00m]:\n\u001B[0;32m--> 551\u001B[0m     res \u001B[38;5;241m=\u001B[39m \u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mp\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    553\u001B[0m     done \u001B[38;5;241m=\u001B[39m \u001B[38;5;28mset\u001B[39m()\n\u001B[1;32m    554\u001B[0m     \u001B[38;5;28;01mfor\u001B[39;00m x \u001B[38;5;129;01min\u001B[39;00m res:\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:303\u001B[0m, in \u001B[0;36mevalPart\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    300\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalMinus(ctx, part)\n\u001B[1;32m    302\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mProject\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[0;32m--> 303\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mevalProject\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    304\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mSlice\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    305\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalSlice(ctx, part)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:561\u001B[0m, in \u001B[0;36mevalProject\u001B[0;34m(ctx, project)\u001B[0m\n\u001B[1;32m    560\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mevalProject\u001B[39m(ctx: QueryContext, project: CompValue):\n\u001B[0;32m--> 561\u001B[0m     res \u001B[38;5;241m=\u001B[39m \u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mproject\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mp\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    562\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m (row\u001B[38;5;241m.\u001B[39mproject(project\u001B[38;5;241m.\u001B[39mPV) \u001B[38;5;28;01mfor\u001B[39;00m row \u001B[38;5;129;01min\u001B[39;00m res)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:312\u001B[0m, in \u001B[0;36mevalPart\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    309\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalReduced(ctx, part)\n\u001B[1;32m    311\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mOrderBy\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[0;32m--> 312\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mevalOrderBy\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    313\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mGroup\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    314\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalGroup(ctx, part)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:491\u001B[0m, in \u001B[0;36mevalOrderBy\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    489\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m e \u001B[38;5;129;01min\u001B[39;00m \u001B[38;5;28mreversed\u001B[39m(part\u001B[38;5;241m.\u001B[39mexpr):\n\u001B[1;32m    490\u001B[0m     reverse \u001B[38;5;241m=\u001B[39m \u001B[38;5;28mbool\u001B[39m(e\u001B[38;5;241m.\u001B[39morder \u001B[38;5;129;01mand\u001B[39;00m e\u001B[38;5;241m.\u001B[39morder \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mDESC\u001B[39m\u001B[38;5;124m\"\u001B[39m)\n\u001B[0;32m--> 491\u001B[0m     res \u001B[38;5;241m=\u001B[39m \u001B[38;5;28;43msorted\u001B[39;49m\u001B[43m(\u001B[49m\n\u001B[1;32m    492\u001B[0m \u001B[43m        \u001B[49m\u001B[43mres\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mkey\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[38;5;28;43;01mlambda\u001B[39;49;00m\u001B[43m \u001B[49m\u001B[43mx\u001B[49m\u001B[43m:\u001B[49m\u001B[43m \u001B[49m\u001B[43m_val\u001B[49m\u001B[43m(\u001B[49m\u001B[43mvalue\u001B[49m\u001B[43m(\u001B[49m\u001B[43mx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43me\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mexpr\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mvariables\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[38;5;28;43;01mTrue\u001B[39;49;00m\u001B[43m)\u001B[49m\u001B[43m)\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mreverse\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[43mreverse\u001B[49m\n\u001B[1;32m    493\u001B[0m \u001B[43m    \u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    495\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m res\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:205\u001B[0m, in \u001B[0;36mevalFilter\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    200\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mevalFilter\u001B[39m(\n\u001B[1;32m    201\u001B[0m     ctx: QueryContext, part: CompValue\n\u001B[1;32m    202\u001B[0m ) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m Generator[FrozenBindings, \u001B[38;5;28;01mNone\u001B[39;00m, \u001B[38;5;28;01mNone\u001B[39;00m]:\n\u001B[1;32m    203\u001B[0m     \u001B[38;5;66;03m# TODO: Deal with dict returned from evalPart!\u001B[39;00m\n\u001B[1;32m    204\u001B[0m     \u001B[38;5;28;01mfor\u001B[39;00m c \u001B[38;5;129;01min\u001B[39;00m evalPart(ctx, part\u001B[38;5;241m.\u001B[39mp):\n\u001B[0;32m--> 205\u001B[0m         \u001B[38;5;28;01mif\u001B[39;00m \u001B[43m_ebv\u001B[49m\u001B[43m(\u001B[49m\n\u001B[1;32m    206\u001B[0m \u001B[43m            \u001B[49m\u001B[43mpart\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mexpr\u001B[49m\u001B[43m,\u001B[49m\n\u001B[1;32m    207\u001B[0m \u001B[43m            \u001B[49m\u001B[43mc\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mforget\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43m_except\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[43mpart\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43m_vars\u001B[49m\u001B[43m)\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;28;43;01mif\u001B[39;49;00m\u001B[43m \u001B[49m\u001B[38;5;129;43;01mnot\u001B[39;49;00m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mno_isolated_scope\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;28;43;01melse\u001B[39;49;00m\u001B[43m \u001B[49m\u001B[43mc\u001B[49m\u001B[43m,\u001B[49m\n\u001B[1;32m    208\u001B[0m \u001B[43m        \u001B[49m\u001B[43m)\u001B[49m:\n\u001B[1;32m    209\u001B[0m             \u001B[38;5;28;01myield\u001B[39;00m c\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evalutils.py:90\u001B[0m, in \u001B[0;36m_ebv\u001B[0;34m(expr, ctx)\u001B[0m\n\u001B[1;32m     88\u001B[0m \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(expr, Expr):\n\u001B[1;32m     89\u001B[0m     \u001B[38;5;28;01mtry\u001B[39;00m:\n\u001B[0;32m---> 90\u001B[0m         \u001B[38;5;28;01mreturn\u001B[39;00m EBV(\u001B[43mexpr\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43meval\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m)\u001B[49m)\n\u001B[1;32m     91\u001B[0m     \u001B[38;5;28;01mexcept\u001B[39;00m SPARQLError:\n\u001B[1;32m     92\u001B[0m         \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;01mFalse\u001B[39;00m  \u001B[38;5;66;03m# filter error == False\u001B[39;00m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parserutils.py:228\u001B[0m, in \u001B[0;36mExpr.eval\u001B[0;34m(self, ctx)\u001B[0m\n\u001B[1;32m    226\u001B[0m     \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mctx: Optional[Union[Mapping, FrozenBindings]] \u001B[38;5;241m=\u001B[39m ctx\n\u001B[1;32m    227\u001B[0m     \u001B[38;5;66;03m# type error: \"None\" not callable\u001B[39;00m\n\u001B[0;32m--> 228\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;43mself\u001B[39;49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43m_evalfn\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m)\u001B[49m  \u001B[38;5;66;03m# type: ignore[misc]\u001B[39;00m\n\u001B[1;32m    229\u001B[0m \u001B[38;5;28;01mexcept\u001B[39;00m SPARQLError \u001B[38;5;28;01mas\u001B[39;00m e:\n\u001B[1;32m    230\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m e\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/operators.py:929\u001B[0m, in \u001B[0;36mConditionalAndExpression\u001B[0;34m(e, ctx)\u001B[0m\n\u001B[1;32m    923\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mConditionalAndExpression\u001B[39m(\n\u001B[1;32m    924\u001B[0m     e: Expr, ctx: Union[QueryContext, FrozenBindings]\n\u001B[1;32m    925\u001B[0m ) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m Literal:\n\u001B[1;32m    926\u001B[0m     \u001B[38;5;66;03m# TODO: handle returned errors\u001B[39;00m\n\u001B[1;32m    928\u001B[0m     expr \u001B[38;5;241m=\u001B[39m e\u001B[38;5;241m.\u001B[39mexpr\n\u001B[0;32m--> 929\u001B[0m     other \u001B[38;5;241m=\u001B[39m \u001B[43me\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mother\u001B[49m\n\u001B[1;32m    931\u001B[0m     \u001B[38;5;66;03m# because of the way the add-expr production handled operator precedence\u001B[39;00m\n\u001B[1;32m    932\u001B[0m     \u001B[38;5;66;03m# we sometimes have nothing to do\u001B[39;00m\n\u001B[1;32m    933\u001B[0m     \u001B[38;5;28;01mif\u001B[39;00m other \u001B[38;5;129;01mis\u001B[39;00m \u001B[38;5;28;01mNone\u001B[39;00m:\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parserutils.py:196\u001B[0m, in \u001B[0;36mCompValue.__getattr__\u001B[0;34m(self, a)\u001B[0m\n\u001B[1;32m    194\u001B[0m     \u001B[38;5;28;01mraise\u001B[39;00m \u001B[38;5;167;01mAttributeError\u001B[39;00m()\n\u001B[1;32m    195\u001B[0m \u001B[38;5;28;01mtry\u001B[39;00m:\n\u001B[0;32m--> 196\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;43mself\u001B[39;49m\u001B[43m[\u001B[49m\u001B[43ma\u001B[49m\u001B[43m]\u001B[49m\n\u001B[1;32m    197\u001B[0m \u001B[38;5;28;01mexcept\u001B[39;00m \u001B[38;5;167;01mKeyError\u001B[39;00m:\n\u001B[1;32m    198\u001B[0m     \u001B[38;5;66;03m# raise AttributeError('no such attribute '+a)\u001B[39;00m\n\u001B[1;32m    199\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;01mNone\u001B[39;00m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parserutils.py:184\u001B[0m, in \u001B[0;36mCompValue.__getitem__\u001B[0;34m(self, a)\u001B[0m\n\u001B[1;32m    183\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21m__getitem__\u001B[39m(\u001B[38;5;28mself\u001B[39m, a):\n\u001B[0;32m--> 184\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;43mself\u001B[39;49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43m_value\u001B[49m\u001B[43m(\u001B[49m\u001B[43mOrderedDict\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[38;5;21;43m__getitem__\u001B[39;49m\u001B[43m(\u001B[49m\u001B[38;5;28;43mself\u001B[39;49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43ma\u001B[49m\u001B[43m)\u001B[49m\u001B[43m)\u001B[49m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parserutils.py:179\u001B[0m, in \u001B[0;36mCompValue._value\u001B[0;34m(self, val, variables, errors)\u001B[0m\n\u001B[1;32m    175\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21m_value\u001B[39m(\n\u001B[1;32m    176\u001B[0m     \u001B[38;5;28mself\u001B[39m, val: _ValT, variables: \u001B[38;5;28mbool\u001B[39m \u001B[38;5;241m=\u001B[39m \u001B[38;5;28;01mFalse\u001B[39;00m, errors: \u001B[38;5;28mbool\u001B[39m \u001B[38;5;241m=\u001B[39m \u001B[38;5;28;01mFalse\u001B[39;00m\n\u001B[1;32m    177\u001B[0m ) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m Union[_ValT, Any]:\n\u001B[1;32m    178\u001B[0m     \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mctx \u001B[38;5;129;01mis\u001B[39;00m \u001B[38;5;129;01mnot\u001B[39;00m \u001B[38;5;28;01mNone\u001B[39;00m:\n\u001B[0;32m--> 179\u001B[0m         \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mvalue\u001B[49m\u001B[43m(\u001B[49m\u001B[38;5;28;43mself\u001B[39;49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mval\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mvariables\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    180\u001B[0m     \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m    181\u001B[0m         \u001B[38;5;28;01mreturn\u001B[39;00m val\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parserutils.py:81\u001B[0m, in \u001B[0;36mvalue\u001B[0;34m(ctx, val, variables, errors)\u001B[0m\n\u001B[1;32m     78\u001B[0m     \u001B[38;5;28;01mraise\u001B[39;00m \u001B[38;5;167;01mException\u001B[39;00m(\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mWhat do I do with this CompValue? \u001B[39m\u001B[38;5;132;01m%s\u001B[39;00m\u001B[38;5;124m\"\u001B[39m \u001B[38;5;241m%\u001B[39m val)\n\u001B[1;32m     80\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(val, \u001B[38;5;28mlist\u001B[39m):\n\u001B[0;32m---> 81\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m [value(ctx, x, variables, errors) \u001B[38;5;28;01mfor\u001B[39;00m x \u001B[38;5;129;01min\u001B[39;00m val]\n\u001B[1;32m     83\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(val, (BNode, Variable)):\n\u001B[1;32m     84\u001B[0m     r \u001B[38;5;241m=\u001B[39m ctx\u001B[38;5;241m.\u001B[39mget(val)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parserutils.py:81\u001B[0m, in \u001B[0;36m<listcomp>\u001B[0;34m(.0)\u001B[0m\n\u001B[1;32m     78\u001B[0m     \u001B[38;5;28;01mraise\u001B[39;00m \u001B[38;5;167;01mException\u001B[39;00m(\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mWhat do I do with this CompValue? \u001B[39m\u001B[38;5;132;01m%s\u001B[39;00m\u001B[38;5;124m\"\u001B[39m \u001B[38;5;241m%\u001B[39m val)\n\u001B[1;32m     80\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(val, \u001B[38;5;28mlist\u001B[39m):\n\u001B[0;32m---> 81\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m [\u001B[43mvalue\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mvariables\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43merrors\u001B[49m\u001B[43m)\u001B[49m \u001B[38;5;28;01mfor\u001B[39;00m x \u001B[38;5;129;01min\u001B[39;00m val]\n\u001B[1;32m     83\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(val, (BNode, Variable)):\n\u001B[1;32m     84\u001B[0m     r \u001B[38;5;241m=\u001B[39m ctx\u001B[38;5;241m.\u001B[39mget(val)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parserutils.py:76\u001B[0m, in \u001B[0;36mvalue\u001B[0;34m(ctx, val, variables, errors)\u001B[0m\n\u001B[1;32m     63\u001B[0m \u001B[38;5;250m\u001B[39m\u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m     64\u001B[0m \u001B[38;5;124;03mutility function for evaluating something...\u001B[39;00m\n\u001B[1;32m     65\u001B[0m \n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m     72\u001B[0m \n\u001B[1;32m     73\u001B[0m \u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m     75\u001B[0m \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(val, Expr):\n\u001B[0;32m---> 76\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mval\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43meval\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m)\u001B[49m  \u001B[38;5;66;03m# recurse?\u001B[39;00m\n\u001B[1;32m     77\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(val, CompValue):\n\u001B[1;32m     78\u001B[0m     \u001B[38;5;28;01mraise\u001B[39;00m \u001B[38;5;167;01mException\u001B[39;00m(\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mWhat do I do with this CompValue? \u001B[39m\u001B[38;5;132;01m%s\u001B[39;00m\u001B[38;5;124m\"\u001B[39m \u001B[38;5;241m%\u001B[39m val)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parserutils.py:228\u001B[0m, in \u001B[0;36mExpr.eval\u001B[0;34m(self, ctx)\u001B[0m\n\u001B[1;32m    226\u001B[0m     \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mctx: Optional[Union[Mapping, FrozenBindings]] \u001B[38;5;241m=\u001B[39m ctx\n\u001B[1;32m    227\u001B[0m     \u001B[38;5;66;03m# type error: \"None\" not callable\u001B[39;00m\n\u001B[0;32m--> 228\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28;43mself\u001B[39;49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43m_evalfn\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m)\u001B[49m  \u001B[38;5;66;03m# type: ignore[misc]\u001B[39;00m\n\u001B[1;32m    229\u001B[0m \u001B[38;5;28;01mexcept\u001B[39;00m SPARQLError \u001B[38;5;28;01mas\u001B[39;00m e:\n\u001B[1;32m    230\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m e\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/operators.py:583\u001B[0m, in \u001B[0;36mBuiltin_EXISTS\u001B[0;34m(e, ctx)\u001B[0m\n\u001B[1;32m    581\u001B[0m ctx \u001B[38;5;241m=\u001B[39m ctx\u001B[38;5;241m.\u001B[39mctx\u001B[38;5;241m.\u001B[39mthaw(ctx)  \u001B[38;5;66;03m# type: ignore[assignment] # hmm\u001B[39;00m\n\u001B[1;32m    582\u001B[0m \u001B[38;5;66;03m# type error: Argument 1 to \"evalPart\" has incompatible type \"FrozenBindings\"; expected \"QueryContext\"\u001B[39;00m\n\u001B[0;32m--> 583\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m x \u001B[38;5;129;01min\u001B[39;00m \u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43me\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mgraph\u001B[49m\u001B[43m)\u001B[49m:  \u001B[38;5;66;03m# type: ignore[arg-type]\u001B[39;00m\n\u001B[1;32m    584\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m Literal(exists)\n\u001B[1;32m    585\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m Literal(\u001B[38;5;129;01mnot\u001B[39;00m exists)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:300\u001B[0m, in \u001B[0;36mevalPart\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    298\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalExtend(ctx, part)\n\u001B[1;32m    299\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mMinus\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[0;32m--> 300\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mevalMinus\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    302\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mProject\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    303\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalProject(ctx, part)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:171\u001B[0m, in \u001B[0;36mevalMinus\u001B[0;34m(ctx, minus)\u001B[0m\n\u001B[1;32m    169\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mevalMinus\u001B[39m(ctx: QueryContext, minus: CompValue) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m Generator[FrozenDict, \u001B[38;5;28;01mNone\u001B[39;00m, \u001B[38;5;28;01mNone\u001B[39;00m]:\n\u001B[1;32m    170\u001B[0m     a \u001B[38;5;241m=\u001B[39m evalPart(ctx, minus\u001B[38;5;241m.\u001B[39mp1)\n\u001B[0;32m--> 171\u001B[0m     b \u001B[38;5;241m=\u001B[39m \u001B[38;5;28mset\u001B[39m(\u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mminus\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mp2\u001B[49m\u001B[43m)\u001B[49m)\n\u001B[1;32m    172\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m _minus(a, b)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:288\u001B[0m, in \u001B[0;36mevalPart\u001B[0;34m(ctx, part)\u001B[0m\n\u001B[1;32m    286\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalFilter(ctx, part)\n\u001B[1;32m    287\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mJoin\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[0;32m--> 288\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mevalJoin\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mpart\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    289\u001B[0m \u001B[38;5;28;01melif\u001B[39;00m part\u001B[38;5;241m.\u001B[39mname \u001B[38;5;241m==\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mLeftJoin\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    290\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m evalLeftJoin(ctx, part)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:156\u001B[0m, in \u001B[0;36mevalJoin\u001B[0;34m(ctx, join)\u001B[0m\n\u001B[1;32m    154\u001B[0m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m    155\u001B[0m     a \u001B[38;5;241m=\u001B[39m evalPart(ctx, join\u001B[38;5;241m.\u001B[39mp1)\n\u001B[0;32m--> 156\u001B[0m     b \u001B[38;5;241m=\u001B[39m \u001B[38;5;28;43mset\u001B[39;49m\u001B[43m(\u001B[49m\u001B[43mevalPart\u001B[49m\u001B[43m(\u001B[49m\u001B[43mctx\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mjoin\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mp2\u001B[49m\u001B[43m)\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    157\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m _join(a, b)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/evaluate.py:90\u001B[0m, in \u001B[0;36mevalBGP\u001B[0;34m(ctx, bgp)\u001B[0m\n\u001B[1;32m     88\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m ss, sp, so \u001B[38;5;129;01min\u001B[39;00m ctx\u001B[38;5;241m.\u001B[39mgraph\u001B[38;5;241m.\u001B[39mtriples((_s, _p, _o)):  \u001B[38;5;66;03m# type: ignore[union-attr, arg-type]\u001B[39;00m\n\u001B[1;32m     89\u001B[0m     \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28;01mNone\u001B[39;00m \u001B[38;5;129;01min\u001B[39;00m (_s, _p, _o):\n\u001B[0;32m---> 90\u001B[0m         c \u001B[38;5;241m=\u001B[39m \u001B[43mctx\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mpush\u001B[49m\u001B[43m(\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m     91\u001B[0m     \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m     92\u001B[0m         c \u001B[38;5;241m=\u001B[39m ctx\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/sparql.py:396\u001B[0m, in \u001B[0;36mQueryContext.push\u001B[0;34m(self)\u001B[0m\n\u001B[1;32m    395\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mpush\u001B[39m(\u001B[38;5;28mself\u001B[39m) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mQueryContext\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[0;32m--> 396\u001B[0m     r \u001B[38;5;241m=\u001B[39m \u001B[38;5;28;43mself\u001B[39;49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mclone\u001B[49m\u001B[43m(\u001B[49m\u001B[43mBindings\u001B[49m\u001B[43m(\u001B[49m\u001B[38;5;28;43mself\u001B[39;49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mbindings\u001B[49m\u001B[43m)\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m    397\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m r\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/sparql.py:295\u001B[0m, in \u001B[0;36mQueryContext.clone\u001B[0;34m(self, bindings)\u001B[0m\n\u001B[1;32m    290\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21mclone\u001B[39m(\n\u001B[1;32m    291\u001B[0m     \u001B[38;5;28mself\u001B[39m, bindings: Optional[Union[FrozenBindings, Bindings, List[Any]]] \u001B[38;5;241m=\u001B[39m \u001B[38;5;28;01mNone\u001B[39;00m\n\u001B[1;32m    292\u001B[0m ) \u001B[38;5;241m-\u001B[39m\u001B[38;5;241m>\u001B[39m \u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mQueryContext\u001B[39m\u001B[38;5;124m\"\u001B[39m:\n\u001B[1;32m    293\u001B[0m     r \u001B[38;5;241m=\u001B[39m QueryContext(\n\u001B[1;32m    294\u001B[0m         \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_dataset \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39m_dataset \u001B[38;5;129;01mis\u001B[39;00m \u001B[38;5;129;01mnot\u001B[39;00m \u001B[38;5;28;01mNone\u001B[39;00m \u001B[38;5;28;01melse\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mgraph,\n\u001B[0;32m--> 295\u001B[0m         bindings \u001B[38;5;129;01mor\u001B[39;00m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mbindings,\n\u001B[1;32m    296\u001B[0m         initBindings\u001B[38;5;241m=\u001B[39m\u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39minitBindings,\n\u001B[1;32m    297\u001B[0m     )\n\u001B[1;32m    298\u001B[0m     r\u001B[38;5;241m.\u001B[39mprologue \u001B[38;5;241m=\u001B[39m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mprologue\n\u001B[1;32m    299\u001B[0m     r\u001B[38;5;241m.\u001B[39mgraph \u001B[38;5;241m=\u001B[39m \u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mgraph\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/sparql.py:98\u001B[0m, in \u001B[0;36mBindings.__len__\u001B[0;34m(self)\u001B[0m\n\u001B[1;32m     96\u001B[0m i \u001B[38;5;241m=\u001B[39m \u001B[38;5;241m0\u001B[39m\n\u001B[1;32m     97\u001B[0m d: Optional[Bindings] \u001B[38;5;241m=\u001B[39m \u001B[38;5;28mself\u001B[39m\n\u001B[0;32m---> 98\u001B[0m \u001B[38;5;28;01mwhile\u001B[39;00m d \u001B[38;5;129;01mis\u001B[39;00m \u001B[38;5;129;01mnot\u001B[39;00m \u001B[38;5;28;01mNone\u001B[39;00m:\n\u001B[1;32m     99\u001B[0m     i \u001B[38;5;241m+\u001B[39m\u001B[38;5;241m=\u001B[39m \u001B[38;5;28mlen\u001B[39m(d\u001B[38;5;241m.\u001B[39m_d)\n\u001B[1;32m    100\u001B[0m     d \u001B[38;5;241m=\u001B[39m d\u001B[38;5;241m.\u001B[39mouter\n",
      "\u001B[0;31mKeyboardInterrupt\u001B[0m: "
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "\n",
    "SELECT DISTINCT ?entity ?property ?label ?property_subs_edam WHERE {\n",
    "\n",
    "    VALUES ?property {oboInOwl:hasDefinition\n",
    "                       edam:created_in\n",
    "                       #oboInOwl:inSubset\n",
    "                       rdfs:label\n",
    "                       rdfs:subClassOf }\n",
    "    ?entity a owl:Class .\n",
    "\n",
    "    FILTER NOT EXISTS {?entity owl:deprecated true .}\n",
    "    OPTIONAL {?entity rdfs:label ?label .}\n",
    "    FILTER ( ?entity != <http://edamontology.org/data_0006>)\n",
    "    FILTER ( ?entity != <http://edamontology.org/topic_0003>)\n",
    "    FILTER ( ?entity != <http://www.w3.org/2002/07/owl#DeprecatedClass>)\n",
    "    FILTER ( ?entity != <http://edamontology.org/operation_0004>)\n",
    "    FILTER ( ?entity != <http://edamontology.org/format_1915>)\n",
    "\n",
    "    FILTER NOT EXISTS {?entity ?property ?value .\n",
    "                       MINUS { ?value rdf:type owl:Restriction .} #to prevent concept with rdfs:subClassOf property being only restriction (e.g. has_topic)\n",
    "                       }\n",
    "    FILTER (!isBlank(?entity))\n",
    "    # UNION\n",
    "    # {\n",
    "    # VALUES ?property { oboInOwl:inSubset\n",
    "    #                     }\n",
    "    # FILTER NOT EXISTS {?entity ?property <http://purl.obolibrary.org/obo/edam#edam> .}\n",
    "    # }\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:34:51.916015Z",
     "start_time": "2024-02-13T17:33:28.687288Z"
    }
   },
   "execution_count": 42
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "\n",
    "SELECT DISTINCT ?entity ?property ?label WHERE {\n",
    "    \n",
    "    ?entity owl:deprecated true .\n",
    "    ?entity rdfs:label ?label .\n",
    "    FILTER ( ?entity != <http://www.geneontology.org/formats/oboInOwl#ObsoleteClass>)\n",
    "    \n",
    "    {VALUES ?property { edam:obsolete_since\n",
    "                       edam:oldParent                \n",
    "                        }\n",
    "        FILTER NOT EXISTS {?entity ?property ?value .}\n",
    "    }\n",
    "    # UNION\n",
    "    # {VALUES ?property { oboInOwl:inSubset            \n",
    "    #                     }\n",
    "    #     FILTER NOT EXISTS {?entity  ?property <http://purl.obolibrary.org/obo/edam#obsolete> .}\n",
    "    # }\n",
    "    UNION\n",
    "    {VALUES ?property { rdfs:subClassOf             \n",
    "                        }\n",
    "        FILTER NOT EXISTS {?entity ?property <http://www.w3.org/2002/07/owl#DeprecatedClass>.}}\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:35:44.305467Z",
     "start_time": "2024-02-13T17:35:44.131381Z"
    }
   },
   "execution_count": 44
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT DISTINCT ?entity ?target ?label ?property\n",
    "WHERE {\n",
    "    VALUES ?property { edam:has_topic\n",
    "                       edam:has_input\n",
    "                       edam:has_output\n",
    "                       edam:is_format_of\n",
    "                       edam:is_identifier_of\n",
    "                       \n",
    "                        }\n",
    "    ?entity ?p ?value .\n",
    "    #{?entity rdfs:subClassOf+ edam:operation_0004 .}\n",
    "    #UNION\n",
    "    #{\n",
    "    #?entity rdfs:subClassOf+ edam:data_0006 .\n",
    "    #}\n",
    "    ?entity rdfs:label ?label .\n",
    "    ?entity rdfs:subClassOf ?restriction . \n",
    "    ?restriction rdf:type owl:Restriction ; \n",
    "            owl:onProperty ?property  ; \n",
    "            owl:someValuesFrom ?target.\n",
    "    ?target owl:deprecated ?deprecated . \n",
    "    FILTER (?deprecated = true)\n",
    "}\n",
    "ORDER BY ?entity \n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:36:22.365946Z",
     "start_time": "2024-02-13T17:36:22.359500Z"
    }
   },
   "execution_count": 45
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT DISTINCT ?entity ?label ?value ?property WHERE {\n",
    "    \n",
    "    ?entity rdfs:subClassOf ?restriction . \n",
    "    ?restriction rdf:type owl:Restriction ;\n",
    "        owl:onProperty ?property  ; \n",
    "        owl:someValuesFrom ?value .\n",
    "    ?value edam:notRecommendedForAnnotation true .\n",
    "    \n",
    "    FILTER ( ?entity != <http://edamontology.org/data_0842>)\n",
    "    FILTER ( ?entity != <http://edamontology.org/operation_3357>)\n",
    "    FILTER ( ?entity != <http://edamontology.org/operation_2422>)\n",
    "    ?entity rdfs:label ?label .\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:36:58.570387Z",
     "start_time": "2024-02-13T17:36:57.590318Z"
    }
   },
   "execution_count": 46
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "ename": "ParseException",
     "evalue": "Expected SelectQuery, found 'FILTER'  (at char 169), (line:7, col:4)",
     "output_type": "error",
     "traceback": [
      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
      "\u001B[0;31mParseException\u001B[0m                            Traceback (most recent call last)",
      "Cell \u001B[0;32mIn[48], line 14\u001B[0m\n\u001B[1;32m      1\u001B[0m query\u001B[38;5;241m=\u001B[39m\u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[1;32m      2\u001B[0m \u001B[38;5;124mPREFIX edam:<http://edamontology.org/>\u001B[39m\n\u001B[1;32m      3\u001B[0m \n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m     11\u001B[0m \u001B[38;5;124mORDER BY ?entity\u001B[39m\n\u001B[1;32m     12\u001B[0m \u001B[38;5;124m\"\"\"\u001B[39m\n\u001B[0;32m---> 14\u001B[0m results \u001B[38;5;241m=\u001B[39m \u001B[43mkg\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mquery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mquery\u001B[49m\u001B[43m)\u001B[49m\n\u001B[1;32m     16\u001B[0m \u001B[38;5;28;01mfor\u001B[39;00m r \u001B[38;5;129;01min\u001B[39;00m results :\n\u001B[1;32m     17\u001B[0m     \u001B[38;5;28mprint\u001B[39m(\u001B[38;5;124mf\u001B[39m\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mEntity \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mentity\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m - Label \u001B[39m\u001B[38;5;124m'\u001B[39m\u001B[38;5;132;01m{\u001B[39;00mr[\u001B[38;5;124m'\u001B[39m\u001B[38;5;124mlabel\u001B[39m\u001B[38;5;124m'\u001B[39m]\u001B[38;5;132;01m}\u001B[39;00m\u001B[38;5;124m'\u001B[39m\u001B[38;5;124m\"\u001B[39m) \n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/graph.py:1565\u001B[0m, in \u001B[0;36mGraph.query\u001B[0;34m(self, query_object, processor, result, initNs, initBindings, use_store_provided, **kwargs)\u001B[0m\n\u001B[1;32m   1562\u001B[0m     processor \u001B[38;5;241m=\u001B[39m plugin\u001B[38;5;241m.\u001B[39mget(processor, query\u001B[38;5;241m.\u001B[39mProcessor)(\u001B[38;5;28mself\u001B[39m)\n\u001B[1;32m   1564\u001B[0m \u001B[38;5;66;03m# type error: Argument 1 to \"Result\" has incompatible type \"Mapping[str, Any]\"; expected \"str\"\u001B[39;00m\n\u001B[0;32m-> 1565\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m result(\u001B[43mprocessor\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mquery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mquery_object\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43minitBindings\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43minitNs\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43mkwargs\u001B[49m\u001B[43m)\u001B[49m)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/processor.py:144\u001B[0m, in \u001B[0;36mSPARQLProcessor.query\u001B[0;34m(self, strOrQuery, initBindings, initNs, base, DEBUG)\u001B[0m\n\u001B[1;32m    124\u001B[0m \u001B[38;5;250m\u001B[39m\u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m    125\u001B[0m \u001B[38;5;124;03mEvaluate a query with the given initial bindings, and initial\u001B[39;00m\n\u001B[1;32m    126\u001B[0m \u001B[38;5;124;03mnamespaces. The given base is used to resolve relative URIs in\u001B[39;00m\n\u001B[0;32m   (...)\u001B[0m\n\u001B[1;32m    140\u001B[0m \u001B[38;5;124;03m   documentation.\u001B[39;00m\n\u001B[1;32m    141\u001B[0m \u001B[38;5;124;03m\"\"\"\u001B[39;00m\n\u001B[1;32m    143\u001B[0m \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(strOrQuery, \u001B[38;5;28mstr\u001B[39m):\n\u001B[0;32m--> 144\u001B[0m     strOrQuery \u001B[38;5;241m=\u001B[39m translateQuery(\u001B[43mparseQuery\u001B[49m\u001B[43m(\u001B[49m\u001B[43mstrOrQuery\u001B[49m\u001B[43m)\u001B[49m, base, initNs)\n\u001B[1;32m    146\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m evalQuery(\u001B[38;5;28mself\u001B[39m\u001B[38;5;241m.\u001B[39mgraph, strOrQuery, initBindings, base)\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/rdflib/plugins/sparql/parser.py:1542\u001B[0m, in \u001B[0;36mparseQuery\u001B[0;34m(q)\u001B[0m\n\u001B[1;32m   1539\u001B[0m     q \u001B[38;5;241m=\u001B[39m q\u001B[38;5;241m.\u001B[39mdecode(\u001B[38;5;124m\"\u001B[39m\u001B[38;5;124mutf-8\u001B[39m\u001B[38;5;124m\"\u001B[39m)\n\u001B[1;32m   1541\u001B[0m q \u001B[38;5;241m=\u001B[39m expandUnicodeEscapes(q)\n\u001B[0;32m-> 1542\u001B[0m \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mQuery\u001B[49m\u001B[38;5;241;43m.\u001B[39;49m\u001B[43mparseString\u001B[49m\u001B[43m(\u001B[49m\u001B[43mq\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[43mparseAll\u001B[49m\u001B[38;5;241;43m=\u001B[39;49m\u001B[38;5;28;43;01mTrue\u001B[39;49;00m\u001B[43m)\u001B[49m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/pyparsing/util.py:256\u001B[0m, in \u001B[0;36m_make_synonym_function.<locals>._inner\u001B[0;34m(self, *args, **kwargs)\u001B[0m\n\u001B[1;32m    251\u001B[0m \u001B[38;5;129m@wraps\u001B[39m(fn)\n\u001B[1;32m    252\u001B[0m \u001B[38;5;28;01mdef\u001B[39;00m \u001B[38;5;21m_inner\u001B[39m(\u001B[38;5;28mself\u001B[39m, \u001B[38;5;241m*\u001B[39margs, \u001B[38;5;241m*\u001B[39m\u001B[38;5;241m*\u001B[39mkwargs):\n\u001B[1;32m    253\u001B[0m     \u001B[38;5;66;03m# warnings.warn(\u001B[39;00m\n\u001B[1;32m    254\u001B[0m     \u001B[38;5;66;03m#     f\"Deprecated - use {fn.__name__}\", DeprecationWarning, stacklevel=3\u001B[39;00m\n\u001B[1;32m    255\u001B[0m     \u001B[38;5;66;03m# )\u001B[39;00m\n\u001B[0;32m--> 256\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[43mfn\u001B[49m\u001B[43m(\u001B[49m\u001B[38;5;28;43mself\u001B[39;49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43margs\u001B[49m\u001B[43m,\u001B[49m\u001B[43m \u001B[49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[38;5;241;43m*\u001B[39;49m\u001B[43mkwargs\u001B[49m\u001B[43m)\u001B[49m\n",
      "File \u001B[0;32m~/mambaforge/lib/python3.10/site-packages/pyparsing/core.py:1197\u001B[0m, in \u001B[0;36mParserElement.parse_string\u001B[0;34m(self, instring, parse_all, parseAll)\u001B[0m\n\u001B[1;32m   1194\u001B[0m         \u001B[38;5;28;01mraise\u001B[39;00m\n\u001B[1;32m   1195\u001B[0m     \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m   1196\u001B[0m         \u001B[38;5;66;03m# catch and re-raise exception from here, clearing out pyparsing internal stack trace\u001B[39;00m\n\u001B[0;32m-> 1197\u001B[0m         \u001B[38;5;28;01mraise\u001B[39;00m exc\u001B[38;5;241m.\u001B[39mwith_traceback(\u001B[38;5;28;01mNone\u001B[39;00m)\n\u001B[1;32m   1198\u001B[0m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[1;32m   1199\u001B[0m     \u001B[38;5;28;01mreturn\u001B[39;00m tokens\n",
      "\u001B[0;31mParseException\u001B[0m: Expected SelectQuery, found 'FILTER'  (at char 169), (line:7, col:4)"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "\n",
    "SELECT DISTINCT ?entity ?property ?value ?label WHERE {\n",
    "   ?entity ?property ?value .\n",
    "   ?entity rdfs:label ?label .        \n",
    "   FILTER REGEX(str(?value), \"^[\\\\s\\r\\n]+\")\n",
    "   FILTER (!isBlank(?entity))\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:37:43.255350Z",
     "start_time": "2024-02-13T17:37:43.213404Z"
    }
   },
   "execution_count": 48
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT ?entity ?label ?id ?subset ?superclass ?label_sc ?id_sc ?subset_sc WHERE {\n",
    "\n",
    "    ?entity a owl:Class .\n",
    "    ?entity rdfs:label ?label .    \n",
    "    BIND(strafter(str(?entity), \"org/\") AS ?id) .\n",
    "    BIND(strbefore(str(?id), \"_\") AS ?subset) .\n",
    "    \n",
    "    ?entity rdfs:subClassOf ?superclass .\n",
    "    ?superclass rdfs:label ?label_sc .    \n",
    "    BIND(strafter(str(?superclass), \"org/\") AS ?id_sc) .\n",
    "    BIND(strbefore(str(?id_sc), \"_\") AS ?subset_sc) .\n",
    "    \n",
    "    FILTER ( ?subset_sc != ?subset) .\n",
    "} \n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:38:31.933779Z",
     "start_time": "2024-02-13T17:38:31.263856Z"
    }
   },
   "execution_count": 49
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?label ?property ?value WHERE {\n",
    "        VALUES ?property { rdfs:subClassOf }\n",
    "        ?entity ?property ?value\n",
    "        FILTER (?entity = ?value)\n",
    "\t    ?entity rdfs:label ?label \n",
    "\t \n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:39:00.957289Z",
     "start_time": "2024-02-13T17:39:00.769363Z"
    }
   },
   "execution_count": 50
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "SELECT DISTINCT ?entity ?property ?value ?label WHERE {\n",
    "   ?entity ?property ?value .\n",
    "   ?entity rdfs:label ?label .        \n",
    "    \n",
    "   FILTER regex(?value, \"\\t\")\n",
    "   FILTER (!isBlank(?entity))\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:39:44.609247Z",
     "start_time": "2024-02-13T17:39:42.950288Z"
    }
   },
   "execution_count": 51
  },
  {
   "cell_type": "code",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity 'http://edamontology.org/data_0005' - Label 'Resource type'\n",
      "Entity 'http://edamontology.org/data_0005' - Label 'Resource type'\n",
      "Entity 'http://edamontology.org/data_0007' - Label 'Tool'\n",
      "Entity 'http://edamontology.org/data_0007' - Label 'Tool'\n",
      "Entity 'http://edamontology.org/data_0581' - Label 'Database'\n",
      "Entity 'http://edamontology.org/data_0581' - Label 'Database'\n",
      "Entity 'http://edamontology.org/data_0582' - Label 'Ontology'\n",
      "Entity 'http://edamontology.org/data_0582' - Label 'Ontology'\n",
      "Entity 'http://edamontology.org/data_0583' - Label 'Directory metadata'\n",
      "Entity 'http://edamontology.org/data_0583' - Label 'Directory metadata'\n",
      "Entity 'http://edamontology.org/data_0831' - Label 'MeSH vocabulary'\n",
      "Entity 'http://edamontology.org/data_0831' - Label 'MeSH vocabulary'\n",
      "Entity 'http://edamontology.org/data_0832' - Label 'HGNC vocabulary'\n",
      "Entity 'http://edamontology.org/data_0832' - Label 'HGNC vocabulary'\n",
      "Entity 'http://edamontology.org/data_0835' - Label 'UMLS vocabulary'\n",
      "Entity 'http://edamontology.org/data_0835' - Label 'UMLS vocabulary'\n",
      "Entity 'http://edamontology.org/data_0842' - Label 'Identifier'\n",
      "Entity 'http://edamontology.org/data_0842' - Label 'Identifier'\n",
      "Entity 'http://edamontology.org/data_0843' - Label 'Database entry'\n",
      "Entity 'http://edamontology.org/data_0843' - Label 'Database entry'\n",
      "Entity 'http://edamontology.org/data_0844' - Label 'Molecular mass'\n",
      "Entity 'http://edamontology.org/data_0845' - Label 'Molecular charge'\n",
      "Entity 'http://edamontology.org/data_0846' - Label 'Chemical formula'\n",
      "Entity 'http://edamontology.org/data_0847' - Label 'QSAR descriptor'\n",
      "Entity 'http://edamontology.org/data_0848' - Label 'Raw sequence'\n",
      "Entity 'http://edamontology.org/data_0848' - Label 'Raw sequence'\n",
      "Entity 'http://edamontology.org/data_0849' - Label 'Sequence record'\n",
      "Entity 'http://edamontology.org/data_0850' - Label 'Sequence set'\n",
      "Entity 'http://edamontology.org/data_0851' - Label 'Sequence mask character'\n",
      "Entity 'http://edamontology.org/data_0851' - Label 'Sequence mask character'\n",
      "Entity 'http://edamontology.org/data_0852' - Label 'Sequence mask type'\n",
      "Entity 'http://edamontology.org/data_0852' - Label 'Sequence mask type'\n",
      "Entity 'http://edamontology.org/data_0853' - Label 'DNA sense specification'\n",
      "Entity 'http://edamontology.org/data_0853' - Label 'DNA sense specification'\n",
      "Entity 'http://edamontology.org/data_0854' - Label 'Sequence length specification'\n",
      "Entity 'http://edamontology.org/data_0854' - Label 'Sequence length specification'\n",
      "Entity 'http://edamontology.org/data_0855' - Label 'Sequence metadata'\n",
      "Entity 'http://edamontology.org/data_0855' - Label 'Sequence metadata'\n",
      "Entity 'http://edamontology.org/data_0856' - Label 'Sequence feature source'\n",
      "Entity 'http://edamontology.org/data_0857' - Label 'Sequence search results'\n",
      "Entity 'http://edamontology.org/data_0858' - Label 'Sequence signature matches'\n",
      "Entity 'http://edamontology.org/data_0858' - Label 'Sequence signature matches'\n",
      "Entity 'http://edamontology.org/data_0859' - Label 'Sequence signature model'\n",
      "Entity 'http://edamontology.org/data_0859' - Label 'Sequence signature model'\n",
      "Entity 'http://edamontology.org/data_0860' - Label 'Sequence signature data'\n",
      "Entity 'http://edamontology.org/data_0860' - Label 'Sequence signature data'\n",
      "Entity 'http://edamontology.org/data_0861' - Label 'Sequence alignment (words)'\n",
      "Entity 'http://edamontology.org/data_0861' - Label 'Sequence alignment (words)'\n",
      "Entity 'http://edamontology.org/data_0862' - Label 'Dotplot'\n",
      "Entity 'http://edamontology.org/data_0863' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/data_0863' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/data_0864' - Label 'Sequence alignment parameter'\n",
      "Entity 'http://edamontology.org/data_0864' - Label 'Sequence alignment parameter'\n",
      "Entity 'http://edamontology.org/data_0865' - Label 'Sequence similarity score'\n",
      "Entity 'http://edamontology.org/data_0866' - Label 'Sequence alignment metadata'\n",
      "Entity 'http://edamontology.org/data_0866' - Label 'Sequence alignment metadata'\n",
      "Entity 'http://edamontology.org/data_0867' - Label 'Sequence alignment report'\n",
      "Entity 'http://edamontology.org/data_0868' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0868' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0869' - Label 'Sequence-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0869' - Label 'Sequence-profile alignment'\n",
      "Entity 'http://edamontology.org/data_0870' - Label 'Sequence distance matrix'\n",
      "Entity 'http://edamontology.org/data_0871' - Label 'Phylogenetic character data'\n",
      "Entity 'http://edamontology.org/data_0872' - Label 'Phylogenetic tree'\n",
      "Entity 'http://edamontology.org/data_0872' - Label 'Phylogenetic tree'\n",
      "Entity 'http://edamontology.org/data_0874' - Label 'Comparison matrix'\n",
      "Entity 'http://edamontology.org/data_0875' - Label 'Protein topology'\n",
      "Entity 'http://edamontology.org/data_0875' - Label 'Protein topology'\n",
      "Entity 'http://edamontology.org/data_0876' - Label 'Protein features report (secondary structure)'\n",
      "Entity 'http://edamontology.org/data_0876' - Label 'Protein features report (secondary structure)'\n",
      "Entity 'http://edamontology.org/data_0877' - Label 'Protein features report (super-secondary)'\n",
      "Entity 'http://edamontology.org/data_0877' - Label 'Protein features report (super-secondary)'\n",
      "Entity 'http://edamontology.org/data_0878' - Label 'Protein secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_0879' - Label 'Secondary structure alignment metadata (protein)'\n",
      "Entity 'http://edamontology.org/data_0879' - Label 'Secondary structure alignment metadata (protein)'\n",
      "Entity 'http://edamontology.org/data_0880' - Label 'RNA secondary structure'\n",
      "Entity 'http://edamontology.org/data_0880' - Label 'RNA secondary structure'\n",
      "Entity 'http://edamontology.org/data_0881' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_0882' - Label 'Secondary structure alignment metadata (RNA)'\n",
      "Entity 'http://edamontology.org/data_0882' - Label 'Secondary structure alignment metadata (RNA)'\n",
      "Entity 'http://edamontology.org/data_0883' - Label 'Structure'\n",
      "Entity 'http://edamontology.org/data_0883' - Label 'Structure'\n",
      "Entity 'http://edamontology.org/data_0884' - Label 'Tertiary structure record'\n",
      "Entity 'http://edamontology.org/data_0884' - Label 'Tertiary structure record'\n",
      "Entity 'http://edamontology.org/data_0885' - Label 'Structure database search results'\n",
      "Entity 'http://edamontology.org/data_0885' - Label 'Structure database search results'\n",
      "Entity 'http://edamontology.org/data_0886' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/data_0886' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/data_0887' - Label 'Structure alignment report'\n",
      "Entity 'http://edamontology.org/data_0888' - Label 'Structure similarity score'\n",
      "Entity 'http://edamontology.org/data_0889' - Label 'Structural profile'\n",
      "Entity 'http://edamontology.org/data_0889' - Label 'Structural profile'\n",
      "Entity 'http://edamontology.org/data_0890' - Label 'Structural (3D) profile alignment'\n",
      "Entity 'http://edamontology.org/data_0891' - Label 'Sequence-3D profile alignment'\n",
      "Entity 'http://edamontology.org/data_0891' - Label 'Sequence-3D profile alignment'\n",
      "Entity 'http://edamontology.org/data_0892' - Label 'Protein sequence-structure scoring matrix'\n",
      "Entity 'http://edamontology.org/data_0893' - Label 'Sequence-structure alignment'\n",
      "Entity 'http://edamontology.org/data_0894' - Label 'Amino acid annotation'\n",
      "Entity 'http://edamontology.org/data_0894' - Label 'Amino acid annotation'\n",
      "Entity 'http://edamontology.org/data_0895' - Label 'Peptide annotation'\n",
      "Entity 'http://edamontology.org/data_0895' - Label 'Peptide annotation'\n",
      "Entity 'http://edamontology.org/data_0896' - Label 'Protein report'\n",
      "Entity 'http://edamontology.org/data_0897' - Label 'Protein property'\n",
      "Entity 'http://edamontology.org/data_0899' - Label 'Protein structural motifs and surfaces'\n",
      "Entity 'http://edamontology.org/data_0899' - Label 'Protein structural motifs and surfaces'\n",
      "Entity 'http://edamontology.org/data_0900' - Label 'Protein domain classification'\n",
      "Entity 'http://edamontology.org/data_0900' - Label 'Protein domain classification'\n",
      "Entity 'http://edamontology.org/data_0901' - Label 'Protein features report (domains)'\n",
      "Entity 'http://edamontology.org/data_0901' - Label 'Protein features report (domains)'\n",
      "Entity 'http://edamontology.org/data_0902' - Label 'Protein architecture report'\n",
      "Entity 'http://edamontology.org/data_0902' - Label 'Protein architecture report'\n",
      "Entity 'http://edamontology.org/data_0903' - Label 'Protein folding report'\n",
      "Entity 'http://edamontology.org/data_0903' - Label 'Protein folding report'\n",
      "Entity 'http://edamontology.org/data_0904' - Label 'Protein features (mutation)'\n",
      "Entity 'http://edamontology.org/data_0904' - Label 'Protein features (mutation)'\n",
      "Entity 'http://edamontology.org/data_0905' - Label 'Protein interaction raw data'\n",
      "Entity 'http://edamontology.org/data_0906' - Label 'Protein interaction data'\n",
      "Entity 'http://edamontology.org/data_0906' - Label 'Protein interaction data'\n",
      "Entity 'http://edamontology.org/data_0907' - Label 'Protein family report'\n",
      "Entity 'http://edamontology.org/data_0907' - Label 'Protein family report'\n",
      "Entity 'http://edamontology.org/data_0909' - Label 'Vmax'\n",
      "Entity 'http://edamontology.org/data_0910' - Label 'Km'\n",
      "Entity 'http://edamontology.org/data_0911' - Label 'Nucleotide base annotation'\n",
      "Entity 'http://edamontology.org/data_0911' - Label 'Nucleotide base annotation'\n",
      "Entity 'http://edamontology.org/data_0912' - Label 'Nucleic acid property'\n",
      "Entity 'http://edamontology.org/data_0914' - Label 'Codon usage data'\n",
      "Entity 'http://edamontology.org/data_0914' - Label 'Codon usage data'\n",
      "Entity 'http://edamontology.org/data_0916' - Label 'Gene report'\n",
      "Entity 'http://edamontology.org/data_0917' - Label 'Gene classification'\n",
      "Entity 'http://edamontology.org/data_0917' - Label 'Gene classification'\n",
      "Entity 'http://edamontology.org/data_0918' - Label 'DNA variation'\n",
      "Entity 'http://edamontology.org/data_0918' - Label 'DNA variation'\n",
      "Entity 'http://edamontology.org/data_0919' - Label 'Chromosome report'\n",
      "Entity 'http://edamontology.org/data_0920' - Label 'Genotype/phenotype report'\n",
      "Entity 'http://edamontology.org/data_0923' - Label 'PCR experiment report'\n",
      "Entity 'http://edamontology.org/data_0923' - Label 'PCR experiment report'\n",
      "Entity 'http://edamontology.org/data_0924' - Label 'Sequence trace'\n",
      "Entity 'http://edamontology.org/data_0924' - Label 'Sequence trace'\n",
      "Entity 'http://edamontology.org/data_0925' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/data_0926' - Label 'RH scores'\n",
      "Entity 'http://edamontology.org/data_0927' - Label 'Genetic linkage report'\n",
      "Entity 'http://edamontology.org/data_0928' - Label 'Gene expression profile'\n",
      "Entity 'http://edamontology.org/data_0931' - Label 'Microarray experiment report'\n",
      "Entity 'http://edamontology.org/data_0931' - Label 'Microarray experiment report'\n",
      "Entity 'http://edamontology.org/data_0932' - Label 'Oligonucleotide probe data'\n",
      "Entity 'http://edamontology.org/data_0932' - Label 'Oligonucleotide probe data'\n",
      "Entity 'http://edamontology.org/data_0933' - Label 'SAGE experimental data'\n",
      "Entity 'http://edamontology.org/data_0933' - Label 'SAGE experimental data'\n",
      "Entity 'http://edamontology.org/data_0934' - Label 'MPSS experimental data'\n",
      "Entity 'http://edamontology.org/data_0934' - Label 'MPSS experimental data'\n",
      "Entity 'http://edamontology.org/data_0935' - Label 'SBS experimental data'\n",
      "Entity 'http://edamontology.org/data_0935' - Label 'SBS experimental data'\n",
      "Entity 'http://edamontology.org/data_0936' - Label 'Sequence tag profile (with gene assignment)'\n",
      "Entity 'http://edamontology.org/data_0936' - Label 'Sequence tag profile (with gene assignment)'\n",
      "Entity 'http://edamontology.org/data_0937' - Label 'Electron density map'\n",
      "Entity 'http://edamontology.org/data_0938' - Label 'Raw NMR data'\n",
      "Entity 'http://edamontology.org/data_0939' - Label 'CD spectra'\n",
      "Entity 'http://edamontology.org/data_0940' - Label 'Volume map'\n",
      "Entity 'http://edamontology.org/data_0940' - Label 'Volume map'\n",
      "Entity 'http://edamontology.org/data_0941' - Label 'Electron microscopy model'\n",
      "Entity 'http://edamontology.org/data_0941' - Label 'Electron microscopy model'\n",
      "Entity 'http://edamontology.org/data_0942' - Label '2D PAGE image'\n",
      "Entity 'http://edamontology.org/data_0942' - Label '2D PAGE image'\n",
      "Entity 'http://edamontology.org/data_0943' - Label 'Mass spectrum'\n",
      "Entity 'http://edamontology.org/data_0943' - Label 'Mass spectrum'\n",
      "Entity 'http://edamontology.org/data_0944' - Label 'Peptide mass fingerprint'\n",
      "Entity 'http://edamontology.org/data_0944' - Label 'Peptide mass fingerprint'\n",
      "Entity 'http://edamontology.org/data_0944' - Label 'Peptide mass fingerprint'\n",
      "Entity 'http://edamontology.org/data_0945' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/data_0945' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/data_0945' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/data_0946' - Label 'Pathway or network annotation'\n",
      "Entity 'http://edamontology.org/data_0946' - Label 'Pathway or network annotation'\n",
      "Entity 'http://edamontology.org/data_0947' - Label 'Biological pathway map'\n",
      "Entity 'http://edamontology.org/data_0947' - Label 'Biological pathway map'\n",
      "Entity 'http://edamontology.org/data_0948' - Label 'Data resource definition'\n",
      "Entity 'http://edamontology.org/data_0948' - Label 'Data resource definition'\n",
      "Entity 'http://edamontology.org/data_0949' - Label 'Workflow metadata'\n",
      "Entity 'http://edamontology.org/data_0950' - Label 'Mathematical model'\n",
      "Entity 'http://edamontology.org/data_0950' - Label 'Mathematical model'\n",
      "Entity 'http://edamontology.org/data_0951' - Label 'Statistical estimate score'\n",
      "Entity 'http://edamontology.org/data_0952' - Label 'EMBOSS database resource definition'\n",
      "Entity 'http://edamontology.org/data_0952' - Label 'EMBOSS database resource definition'\n",
      "Entity 'http://edamontology.org/data_0953' - Label 'Version information'\n",
      "Entity 'http://edamontology.org/data_0953' - Label 'Version information'\n",
      "Entity 'http://edamontology.org/data_0954' - Label 'Database cross-mapping'\n",
      "Entity 'http://edamontology.org/data_0955' - Label 'Data index'\n",
      "Entity 'http://edamontology.org/data_0955' - Label 'Data index'\n",
      "Entity 'http://edamontology.org/data_0956' - Label 'Data index report'\n",
      "Entity 'http://edamontology.org/data_0956' - Label 'Data index report'\n",
      "Entity 'http://edamontology.org/data_0957' - Label 'Database metadata'\n",
      "Entity 'http://edamontology.org/data_0958' - Label 'Tool metadata'\n",
      "Entity 'http://edamontology.org/data_0959' - Label 'Job metadata'\n",
      "Entity 'http://edamontology.org/data_0959' - Label 'Job metadata'\n",
      "Entity 'http://edamontology.org/data_0960' - Label 'User metadata'\n",
      "Entity 'http://edamontology.org/data_0962' - Label 'Small molecule report'\n",
      "Entity 'http://edamontology.org/data_0962' - Label 'Small molecule report'\n",
      "Entity 'http://edamontology.org/data_0963' - Label 'Cell line report'\n",
      "Entity 'http://edamontology.org/data_0964' - Label 'Scent annotation'\n",
      "Entity 'http://edamontology.org/data_0964' - Label 'Scent annotation'\n",
      "Entity 'http://edamontology.org/data_0966' - Label 'Ontology term'\n",
      "Entity 'http://edamontology.org/data_0967' - Label 'Ontology concept data'\n",
      "Entity 'http://edamontology.org/data_0968' - Label 'Keyword'\n",
      "Entity 'http://edamontology.org/data_0970' - Label 'Citation'\n",
      "Entity 'http://edamontology.org/data_0971' - Label 'Article'\n",
      "Entity 'http://edamontology.org/data_0971' - Label 'Article'\n",
      "Entity 'http://edamontology.org/data_0972' - Label 'Text mining report'\n",
      "Entity 'http://edamontology.org/data_0972' - Label 'Text mining report'\n",
      "Entity 'http://edamontology.org/data_0974' - Label 'Entity identifier'\n",
      "Entity 'http://edamontology.org/data_0974' - Label 'Entity identifier'\n",
      "Entity 'http://edamontology.org/data_0975' - Label 'Data resource identifier'\n",
      "Entity 'http://edamontology.org/data_0975' - Label 'Data resource identifier'\n",
      "Entity 'http://edamontology.org/data_0976' - Label 'Identifier (by type of entity)'\n",
      "Entity 'http://edamontology.org/data_0977' - Label 'Tool identifier'\n",
      "Entity 'http://edamontology.org/data_0978' - Label 'Discrete entity identifier'\n",
      "Entity 'http://edamontology.org/data_0978' - Label 'Discrete entity identifier'\n",
      "Entity 'http://edamontology.org/data_0979' - Label 'Entity feature identifier'\n",
      "Entity 'http://edamontology.org/data_0979' - Label 'Entity feature identifier'\n",
      "Entity 'http://edamontology.org/data_0980' - Label 'Entity collection identifier'\n",
      "Entity 'http://edamontology.org/data_0980' - Label 'Entity collection identifier'\n",
      "Entity 'http://edamontology.org/data_0981' - Label 'Phenomenon identifier'\n",
      "Entity 'http://edamontology.org/data_0981' - Label 'Phenomenon identifier'\n",
      "Entity 'http://edamontology.org/data_0982' - Label 'Molecule identifier'\n",
      "Entity 'http://edamontology.org/data_0983' - Label 'Atom ID'\n",
      "Entity 'http://edamontology.org/data_0984' - Label 'Molecule name'\n",
      "Entity 'http://edamontology.org/data_0984' - Label 'Molecule name'\n",
      "Entity 'http://edamontology.org/data_0985' - Label 'Molecule type'\n",
      "Entity 'http://edamontology.org/data_0985' - Label 'Molecule type'\n",
      "Entity 'http://edamontology.org/data_0986' - Label 'Chemical identifier'\n",
      "Entity 'http://edamontology.org/data_0986' - Label 'Chemical identifier'\n",
      "Entity 'http://edamontology.org/data_0987' - Label 'Chromosome name'\n",
      "Entity 'http://edamontology.org/data_0987' - Label 'Chromosome name'\n",
      "Entity 'http://edamontology.org/data_0987' - Label 'Chromosome name'\n",
      "Entity 'http://edamontology.org/data_0988' - Label 'Peptide identifier'\n",
      "Entity 'http://edamontology.org/data_0989' - Label 'Protein identifier'\n",
      "Entity 'http://edamontology.org/data_0989' - Label 'Protein identifier'\n",
      "Entity 'http://edamontology.org/data_0990' - Label 'Compound name'\n",
      "Entity 'http://edamontology.org/data_0990' - Label 'Compound name'\n",
      "Entity 'http://edamontology.org/data_0991' - Label 'Chemical registry number'\n",
      "Entity 'http://edamontology.org/data_0992' - Label 'Ligand identifier'\n",
      "Entity 'http://edamontology.org/data_0992' - Label 'Ligand identifier'\n",
      "Entity 'http://edamontology.org/data_0993' - Label 'Drug identifier'\n",
      "Entity 'http://edamontology.org/data_0993' - Label 'Drug identifier'\n",
      "Entity 'http://edamontology.org/data_0994' - Label 'Amino acid identifier'\n",
      "Entity 'http://edamontology.org/data_0994' - Label 'Amino acid identifier'\n",
      "Entity 'http://edamontology.org/data_0995' - Label 'Nucleotide identifier'\n",
      "Entity 'http://edamontology.org/data_0996' - Label 'Monosaccharide identifier'\n",
      "Entity 'http://edamontology.org/data_0997' - Label 'Chemical name (ChEBI)'\n",
      "Entity 'http://edamontology.org/data_0998' - Label 'Chemical name (IUPAC)'\n",
      "Entity 'http://edamontology.org/data_0999' - Label 'Chemical name (INN)'\n",
      "Entity 'http://edamontology.org/data_1000' - Label 'Chemical name (brand)'\n",
      "Entity 'http://edamontology.org/data_1001' - Label 'Chemical name (synonymous)'\n",
      "Entity 'http://edamontology.org/data_1002' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_1002' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_1002' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_1003' - Label 'Chemical registry number (Beilstein)'\n",
      "Entity 'http://edamontology.org/data_1003' - Label 'Chemical registry number (Beilstein)'\n",
      "Entity 'http://edamontology.org/data_1004' - Label 'Chemical registry number (Gmelin)'\n",
      "Entity 'http://edamontology.org/data_1004' - Label 'Chemical registry number (Gmelin)'\n",
      "Entity 'http://edamontology.org/data_1005' - Label 'HET group name'\n",
      "Entity 'http://edamontology.org/data_1006' - Label 'Amino acid name'\n",
      "Entity 'http://edamontology.org/data_1007' - Label 'Nucleotide code'\n",
      "Entity 'http://edamontology.org/data_1007' - Label 'Nucleotide code'\n",
      "Entity 'http://edamontology.org/data_1008' - Label 'Polypeptide chain ID'\n",
      "Entity 'http://edamontology.org/data_1008' - Label 'Polypeptide chain ID'\n",
      "Entity 'http://edamontology.org/data_1009' - Label 'Protein name'\n",
      "Entity 'http://edamontology.org/data_1009' - Label 'Protein name'\n",
      "Entity 'http://edamontology.org/data_1010' - Label 'Enzyme identifier'\n",
      "Entity 'http://edamontology.org/data_1011' - Label 'EC number'\n",
      "Entity 'http://edamontology.org/data_1011' - Label 'EC number'\n",
      "Entity 'http://edamontology.org/data_1012' - Label 'Enzyme name'\n",
      "Entity 'http://edamontology.org/data_1012' - Label 'Enzyme name'\n",
      "Entity 'http://edamontology.org/data_1013' - Label 'Restriction enzyme name'\n",
      "Entity 'http://edamontology.org/data_1014' - Label 'Sequence position specification'\n",
      "Entity 'http://edamontology.org/data_1014' - Label 'Sequence position specification'\n",
      "Entity 'http://edamontology.org/data_1015' - Label 'Sequence feature ID'\n",
      "Entity 'http://edamontology.org/data_1016' - Label 'Sequence position'\n",
      "Entity 'http://edamontology.org/data_1017' - Label 'Sequence range'\n",
      "Entity 'http://edamontology.org/data_1018' - Label 'Nucleic acid feature identifier'\n",
      "Entity 'http://edamontology.org/data_1018' - Label 'Nucleic acid feature identifier'\n",
      "Entity 'http://edamontology.org/data_1019' - Label 'Protein feature identifier'\n",
      "Entity 'http://edamontology.org/data_1019' - Label 'Protein feature identifier'\n",
      "Entity 'http://edamontology.org/data_1020' - Label 'Sequence feature key'\n",
      "Entity 'http://edamontology.org/data_1021' - Label 'Sequence feature qualifier'\n",
      "Entity 'http://edamontology.org/data_1022' - Label 'Sequence feature label'\n",
      "Entity 'http://edamontology.org/data_1022' - Label 'Sequence feature label'\n",
      "Entity 'http://edamontology.org/data_1022' - Label 'Sequence feature label'\n",
      "Entity 'http://edamontology.org/data_1023' - Label 'EMBOSS Uniform Feature Object'\n",
      "Entity 'http://edamontology.org/data_1023' - Label 'EMBOSS Uniform Feature Object'\n",
      "Entity 'http://edamontology.org/data_1024' - Label 'Codon name'\n",
      "Entity 'http://edamontology.org/data_1024' - Label 'Codon name'\n",
      "Entity 'http://edamontology.org/data_1025' - Label 'Gene identifier'\n",
      "Entity 'http://edamontology.org/data_1025' - Label 'Gene identifier'\n",
      "Entity 'http://edamontology.org/data_1026' - Label 'Gene symbol'\n",
      "Entity 'http://edamontology.org/data_1027' - Label 'Gene ID (NCBI)'\n",
      "Entity 'http://edamontology.org/data_1027' - Label 'Gene ID (NCBI)'\n",
      "Entity 'http://edamontology.org/data_1027' - Label 'Gene ID (NCBI)'\n",
      "Entity 'http://edamontology.org/data_1028' - Label 'Gene identifier (NCBI RefSeq)'\n",
      "Entity 'http://edamontology.org/data_1028' - Label 'Gene identifier (NCBI RefSeq)'\n",
      "Entity 'http://edamontology.org/data_1029' - Label 'Gene identifier (NCBI UniGene)'\n",
      "Entity 'http://edamontology.org/data_1029' - Label 'Gene identifier (NCBI UniGene)'\n",
      "Entity 'http://edamontology.org/data_1030' - Label 'Gene identifier (Entrez)'\n",
      "Entity 'http://edamontology.org/data_1030' - Label 'Gene identifier (Entrez)'\n",
      "Entity 'http://edamontology.org/data_1031' - Label 'Gene ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1031' - Label 'Gene ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1032' - Label 'Gene ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1032' - Label 'Gene ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1033' - Label 'Ensembl gene ID'\n",
      "Entity 'http://edamontology.org/data_1033' - Label 'Ensembl gene ID'\n",
      "Entity 'http://edamontology.org/data_1033' - Label 'Ensembl gene ID'\n",
      "Entity 'http://edamontology.org/data_1034' - Label 'Gene ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1034' - Label 'Gene ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1034' - Label 'Gene ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1035' - Label 'Gene ID (GeneDB)'\n",
      "Entity 'http://edamontology.org/data_1035' - Label 'Gene ID (GeneDB)'\n",
      "Entity 'http://edamontology.org/data_1036' - Label 'TIGR identifier'\n",
      "Entity 'http://edamontology.org/data_1036' - Label 'TIGR identifier'\n",
      "Entity 'http://edamontology.org/data_1036' - Label 'TIGR identifier'\n",
      "Entity 'http://edamontology.org/data_1037' - Label 'TAIR accession (gene)'\n",
      "Entity 'http://edamontology.org/data_1037' - Label 'TAIR accession (gene)'\n",
      "Entity 'http://edamontology.org/data_1038' - Label 'Protein domain ID'\n",
      "Entity 'http://edamontology.org/data_1038' - Label 'Protein domain ID'\n",
      "Entity 'http://edamontology.org/data_1039' - Label 'SCOP domain identifier'\n",
      "Entity 'http://edamontology.org/data_1039' - Label 'SCOP domain identifier'\n",
      "Entity 'http://edamontology.org/data_1040' - Label 'CATH domain ID'\n",
      "Entity 'http://edamontology.org/data_1041' - Label 'SCOP concise classification string (sccs)'\n",
      "Entity 'http://edamontology.org/data_1042' - Label 'SCOP sunid'\n",
      "Entity 'http://edamontology.org/data_1043' - Label 'CATH node ID'\n",
      "Entity 'http://edamontology.org/data_1044' - Label 'Kingdom name'\n",
      "Entity 'http://edamontology.org/data_1045' - Label 'Species name'\n",
      "Entity 'http://edamontology.org/data_1046' - Label 'Strain name'\n",
      "Entity 'http://edamontology.org/data_1046' - Label 'Strain name'\n",
      "Entity 'http://edamontology.org/data_1047' - Label 'URI'\n",
      "Entity 'http://edamontology.org/data_1048' - Label 'Database ID'\n",
      "Entity 'http://edamontology.org/data_1048' - Label 'Database ID'\n",
      "Entity 'http://edamontology.org/data_1049' - Label 'Directory name'\n",
      "Entity 'http://edamontology.org/data_1050' - Label 'File name'\n",
      "Entity 'http://edamontology.org/data_1051' - Label 'Ontology name'\n",
      "Entity 'http://edamontology.org/data_1051' - Label 'Ontology name'\n",
      "Entity 'http://edamontology.org/data_1051' - Label 'Ontology name'\n",
      "Entity 'http://edamontology.org/data_1052' - Label 'URL'\n",
      "Entity 'http://edamontology.org/data_1053' - Label 'URN'\n",
      "Entity 'http://edamontology.org/data_1055' - Label 'LSID'\n",
      "Entity 'http://edamontology.org/data_1056' - Label 'Database name'\n",
      "Entity 'http://edamontology.org/data_1056' - Label 'Database name'\n",
      "Entity 'http://edamontology.org/data_1057' - Label 'Sequence database name'\n",
      "Entity 'http://edamontology.org/data_1057' - Label 'Sequence database name'\n",
      "Entity 'http://edamontology.org/data_1058' - Label 'Enumerated file name'\n",
      "Entity 'http://edamontology.org/data_1059' - Label 'File name extension'\n",
      "Entity 'http://edamontology.org/data_1060' - Label 'File base name'\n",
      "Entity 'http://edamontology.org/data_1061' - Label 'QSAR descriptor name'\n",
      "Entity 'http://edamontology.org/data_1061' - Label 'QSAR descriptor name'\n",
      "Entity 'http://edamontology.org/data_1061' - Label 'QSAR descriptor name'\n",
      "Entity 'http://edamontology.org/data_1062' - Label 'Database entry identifier'\n",
      "Entity 'http://edamontology.org/data_1062' - Label 'Database entry identifier'\n",
      "Entity 'http://edamontology.org/data_1063' - Label 'Sequence identifier'\n",
      "Entity 'http://edamontology.org/data_1063' - Label 'Sequence identifier'\n",
      "Entity 'http://edamontology.org/data_1064' - Label 'Sequence set ID'\n",
      "Entity 'http://edamontology.org/data_1064' - Label 'Sequence set ID'\n",
      "Entity 'http://edamontology.org/data_1065' - Label 'Sequence signature identifier'\n",
      "Entity 'http://edamontology.org/data_1065' - Label 'Sequence signature identifier'\n",
      "Entity 'http://edamontology.org/data_1065' - Label 'Sequence signature identifier'\n",
      "Entity 'http://edamontology.org/data_1066' - Label 'Sequence alignment ID'\n",
      "Entity 'http://edamontology.org/data_1066' - Label 'Sequence alignment ID'\n",
      "Entity 'http://edamontology.org/data_1066' - Label 'Sequence alignment ID'\n",
      "Entity 'http://edamontology.org/data_1067' - Label 'Phylogenetic distance matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1067' - Label 'Phylogenetic distance matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1068' - Label 'Phylogenetic tree ID'\n",
      "Entity 'http://edamontology.org/data_1068' - Label 'Phylogenetic tree ID'\n",
      "Entity 'http://edamontology.org/data_1069' - Label 'Comparison matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1069' - Label 'Comparison matrix identifier'\n",
      "Entity 'http://edamontology.org/data_1070' - Label 'Structure ID'\n",
      "Entity 'http://edamontology.org/data_1071' - Label 'Structural (3D) profile ID'\n",
      "Entity 'http://edamontology.org/data_1071' - Label 'Structural (3D) profile ID'\n",
      "Entity 'http://edamontology.org/data_1072' - Label 'Structure alignment ID'\n",
      "Entity 'http://edamontology.org/data_1072' - Label 'Structure alignment ID'\n",
      "Entity 'http://edamontology.org/data_1073' - Label 'Amino acid index ID'\n",
      "Entity 'http://edamontology.org/data_1073' - Label 'Amino acid index ID'\n",
      "Entity 'http://edamontology.org/data_1073' - Label 'Amino acid index ID'\n",
      "Entity 'http://edamontology.org/data_1074' - Label 'Protein interaction ID'\n",
      "Entity 'http://edamontology.org/data_1074' - Label 'Protein interaction ID'\n",
      "Entity 'http://edamontology.org/data_1075' - Label 'Protein family identifier'\n",
      "Entity 'http://edamontology.org/data_1075' - Label 'Protein family identifier'\n",
      "Entity 'http://edamontology.org/data_1076' - Label 'Codon usage table name'\n",
      "Entity 'http://edamontology.org/data_1076' - Label 'Codon usage table name'\n",
      "Entity 'http://edamontology.org/data_1076' - Label 'Codon usage table name'\n",
      "Entity 'http://edamontology.org/data_1076' - Label 'Codon usage table name'\n",
      "Entity 'http://edamontology.org/data_1077' - Label 'Transcription factor identifier'\n",
      "Entity 'http://edamontology.org/data_1077' - Label 'Transcription factor identifier'\n",
      "Entity 'http://edamontology.org/data_1078' - Label 'Experiment annotation ID'\n",
      "Entity 'http://edamontology.org/data_1078' - Label 'Experiment annotation ID'\n",
      "Entity 'http://edamontology.org/data_1079' - Label 'Electron microscopy model ID'\n",
      "Entity 'http://edamontology.org/data_1079' - Label 'Electron microscopy model ID'\n",
      "Entity 'http://edamontology.org/data_1080' - Label 'Gene expression report ID'\n",
      "Entity 'http://edamontology.org/data_1080' - Label 'Gene expression report ID'\n",
      "Entity 'http://edamontology.org/data_1081' - Label 'Genotype and phenotype annotation ID'\n",
      "Entity 'http://edamontology.org/data_1081' - Label 'Genotype and phenotype annotation ID'\n",
      "Entity 'http://edamontology.org/data_1082' - Label 'Pathway or network identifier'\n",
      "Entity 'http://edamontology.org/data_1082' - Label 'Pathway or network identifier'\n",
      "Entity 'http://edamontology.org/data_1083' - Label 'Workflow ID'\n",
      "Entity 'http://edamontology.org/data_1084' - Label 'Data resource definition ID'\n",
      "Entity 'http://edamontology.org/data_1085' - Label 'Biological model ID'\n",
      "Entity 'http://edamontology.org/data_1085' - Label 'Biological model ID'\n",
      "Entity 'http://edamontology.org/data_1086' - Label 'Compound identifier'\n",
      "Entity 'http://edamontology.org/data_1086' - Label 'Compound identifier'\n",
      "Entity 'http://edamontology.org/data_1087' - Label 'Ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_1088' - Label 'Article ID'\n",
      "Entity 'http://edamontology.org/data_1088' - Label 'Article ID'\n",
      "Entity 'http://edamontology.org/data_1089' - Label 'FlyBase ID'\n",
      "Entity 'http://edamontology.org/data_1089' - Label 'FlyBase ID'\n",
      "Entity 'http://edamontology.org/data_1091' - Label 'WormBase name'\n",
      "Entity 'http://edamontology.org/data_1091' - Label 'WormBase name'\n",
      "Entity 'http://edamontology.org/data_1092' - Label 'WormBase class'\n",
      "Entity 'http://edamontology.org/data_1093' - Label 'Sequence accession'\n",
      "Entity 'http://edamontology.org/data_1094' - Label 'Sequence type'\n",
      "Entity 'http://edamontology.org/data_1094' - Label 'Sequence type'\n",
      "Entity 'http://edamontology.org/data_1095' - Label 'EMBOSS Uniform Sequence Address'\n",
      "Entity 'http://edamontology.org/data_1095' - Label 'EMBOSS Uniform Sequence Address'\n",
      "Entity 'http://edamontology.org/data_1096' - Label 'Sequence accession (protein)'\n",
      "Entity 'http://edamontology.org/data_1096' - Label 'Sequence accession (protein)'\n",
      "Entity 'http://edamontology.org/data_1097' - Label 'Sequence accession (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1097' - Label 'Sequence accession (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1098' - Label 'RefSeq accession'\n",
      "Entity 'http://edamontology.org/data_1098' - Label 'RefSeq accession'\n",
      "Entity 'http://edamontology.org/data_1099' - Label 'UniProt accession (extended)'\n",
      "Entity 'http://edamontology.org/data_1099' - Label 'UniProt accession (extended)'\n",
      "Entity 'http://edamontology.org/data_1100' - Label 'PIR identifier'\n",
      "Entity 'http://edamontology.org/data_1100' - Label 'PIR identifier'\n",
      "Entity 'http://edamontology.org/data_1101' - Label 'TREMBL accession'\n",
      "Entity 'http://edamontology.org/data_1101' - Label 'TREMBL accession'\n",
      "Entity 'http://edamontology.org/data_1102' - Label 'Gramene primary identifier'\n",
      "Entity 'http://edamontology.org/data_1103' - Label 'EMBL/GenBank/DDBJ ID'\n",
      "Entity 'http://edamontology.org/data_1103' - Label 'EMBL/GenBank/DDBJ ID'\n",
      "Entity 'http://edamontology.org/data_1104' - Label 'Sequence cluster ID (UniGene)'\n",
      "Entity 'http://edamontology.org/data_1104' - Label 'Sequence cluster ID (UniGene)'\n",
      "Entity 'http://edamontology.org/data_1105' - Label 'dbEST accession'\n",
      "Entity 'http://edamontology.org/data_1105' - Label 'dbEST accession'\n",
      "Entity 'http://edamontology.org/data_1105' - Label 'dbEST accession'\n",
      "Entity 'http://edamontology.org/data_1106' - Label 'dbSNP ID'\n",
      "Entity 'http://edamontology.org/data_1106' - Label 'dbSNP ID'\n",
      "Entity 'http://edamontology.org/data_1110' - Label 'EMBOSS sequence type'\n",
      "Entity 'http://edamontology.org/data_1110' - Label 'EMBOSS sequence type'\n",
      "Entity 'http://edamontology.org/data_1111' - Label 'EMBOSS listfile'\n",
      "Entity 'http://edamontology.org/data_1111' - Label 'EMBOSS listfile'\n",
      "Entity 'http://edamontology.org/data_1112' - Label 'Sequence cluster ID'\n",
      "Entity 'http://edamontology.org/data_1112' - Label 'Sequence cluster ID'\n",
      "Entity 'http://edamontology.org/data_1113' - Label 'Sequence cluster ID (COG)'\n",
      "Entity 'http://edamontology.org/data_1113' - Label 'Sequence cluster ID (COG)'\n",
      "Entity 'http://edamontology.org/data_1114' - Label 'Sequence motif identifier'\n",
      "Entity 'http://edamontology.org/data_1114' - Label 'Sequence motif identifier'\n",
      "Entity 'http://edamontology.org/data_1115' - Label 'Sequence profile ID'\n",
      "Entity 'http://edamontology.org/data_1115' - Label 'Sequence profile ID'\n",
      "Entity 'http://edamontology.org/data_1116' - Label 'ELM ID'\n",
      "Entity 'http://edamontology.org/data_1117' - Label 'Prosite accession number'\n",
      "Entity 'http://edamontology.org/data_1118' - Label 'HMMER hidden Markov model ID'\n",
      "Entity 'http://edamontology.org/data_1118' - Label 'HMMER hidden Markov model ID'\n",
      "Entity 'http://edamontology.org/data_1118' - Label 'HMMER hidden Markov model ID'\n",
      "Entity 'http://edamontology.org/data_1119' - Label 'JASPAR profile ID'\n",
      "Entity 'http://edamontology.org/data_1119' - Label 'JASPAR profile ID'\n",
      "Entity 'http://edamontology.org/data_1120' - Label 'Sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1120' - Label 'Sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1121' - Label 'BLAST sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1121' - Label 'BLAST sequence alignment type'\n",
      "Entity 'http://edamontology.org/data_1122' - Label 'Phylogenetic tree type'\n",
      "Entity 'http://edamontology.org/data_1122' - Label 'Phylogenetic tree type'\n",
      "Entity 'http://edamontology.org/data_1123' - Label 'TreeBASE study accession number'\n",
      "Entity 'http://edamontology.org/data_1123' - Label 'TreeBASE study accession number'\n",
      "Entity 'http://edamontology.org/data_1124' - Label 'TreeFam accession number'\n",
      "Entity 'http://edamontology.org/data_1124' - Label 'TreeFam accession number'\n",
      "Entity 'http://edamontology.org/data_1125' - Label 'Comparison matrix type'\n",
      "Entity 'http://edamontology.org/data_1125' - Label 'Comparison matrix type'\n",
      "Entity 'http://edamontology.org/data_1126' - Label 'Comparison matrix name'\n",
      "Entity 'http://edamontology.org/data_1126' - Label 'Comparison matrix name'\n",
      "Entity 'http://edamontology.org/data_1126' - Label 'Comparison matrix name'\n",
      "Entity 'http://edamontology.org/data_1127' - Label 'PDB ID'\n",
      "Entity 'http://edamontology.org/data_1127' - Label 'PDB ID'\n",
      "Entity 'http://edamontology.org/data_1128' - Label 'AAindex ID'\n",
      "Entity 'http://edamontology.org/data_1128' - Label 'AAindex ID'\n",
      "Entity 'http://edamontology.org/data_1129' - Label 'BIND accession number'\n",
      "Entity 'http://edamontology.org/data_1129' - Label 'BIND accession number'\n",
      "Entity 'http://edamontology.org/data_1130' - Label 'IntAct accession number'\n",
      "Entity 'http://edamontology.org/data_1130' - Label 'IntAct accession number'\n",
      "Entity 'http://edamontology.org/data_1131' - Label 'Protein family name'\n",
      "Entity 'http://edamontology.org/data_1131' - Label 'Protein family name'\n",
      "Entity 'http://edamontology.org/data_1132' - Label 'InterPro entry name'\n",
      "Entity 'http://edamontology.org/data_1132' - Label 'InterPro entry name'\n",
      "Entity 'http://edamontology.org/data_1133' - Label 'InterPro accession'\n",
      "Entity 'http://edamontology.org/data_1133' - Label 'InterPro accession'\n",
      "Entity 'http://edamontology.org/data_1133' - Label 'InterPro accession'\n",
      "Entity 'http://edamontology.org/data_1134' - Label 'InterPro secondary accession'\n",
      "Entity 'http://edamontology.org/data_1134' - Label 'InterPro secondary accession'\n",
      "Entity 'http://edamontology.org/data_1135' - Label 'Gene3D ID'\n",
      "Entity 'http://edamontology.org/data_1135' - Label 'Gene3D ID'\n",
      "Entity 'http://edamontology.org/data_1136' - Label 'PIRSF ID'\n",
      "Entity 'http://edamontology.org/data_1136' - Label 'PIRSF ID'\n",
      "Entity 'http://edamontology.org/data_1137' - Label 'PRINTS code'\n",
      "Entity 'http://edamontology.org/data_1137' - Label 'PRINTS code'\n",
      "Entity 'http://edamontology.org/data_1138' - Label 'Pfam accession number'\n",
      "Entity 'http://edamontology.org/data_1138' - Label 'Pfam accession number'\n",
      "Entity 'http://edamontology.org/data_1139' - Label 'SMART accession number'\n",
      "Entity 'http://edamontology.org/data_1139' - Label 'SMART accession number'\n",
      "Entity 'http://edamontology.org/data_1140' - Label 'Superfamily hidden Markov model number'\n",
      "Entity 'http://edamontology.org/data_1140' - Label 'Superfamily hidden Markov model number'\n",
      "Entity 'http://edamontology.org/data_1141' - Label 'TIGRFam ID'\n",
      "Entity 'http://edamontology.org/data_1141' - Label 'TIGRFam ID'\n",
      "Entity 'http://edamontology.org/data_1142' - Label 'ProDom accession number'\n",
      "Entity 'http://edamontology.org/data_1142' - Label 'ProDom accession number'\n",
      "Entity 'http://edamontology.org/data_1143' - Label 'TRANSFAC accession number'\n",
      "Entity 'http://edamontology.org/data_1143' - Label 'TRANSFAC accession number'\n",
      "Entity 'http://edamontology.org/data_1144' - Label 'ArrayExpress accession number'\n",
      "Entity 'http://edamontology.org/data_1145' - Label 'PRIDE experiment accession number'\n",
      "Entity 'http://edamontology.org/data_1146' - Label 'EMDB ID'\n",
      "Entity 'http://edamontology.org/data_1146' - Label 'EMDB ID'\n",
      "Entity 'http://edamontology.org/data_1147' - Label 'GEO accession number'\n",
      "Entity 'http://edamontology.org/data_1147' - Label 'GEO accession number'\n",
      "Entity 'http://edamontology.org/data_1148' - Label 'GermOnline ID'\n",
      "Entity 'http://edamontology.org/data_1148' - Label 'GermOnline ID'\n",
      "Entity 'http://edamontology.org/data_1149' - Label 'EMAGE ID'\n",
      "Entity 'http://edamontology.org/data_1149' - Label 'EMAGE ID'\n",
      "Entity 'http://edamontology.org/data_1150' - Label 'Disease ID'\n",
      "Entity 'http://edamontology.org/data_1151' - Label 'HGVbase ID'\n",
      "Entity 'http://edamontology.org/data_1151' - Label 'HGVbase ID'\n",
      "Entity 'http://edamontology.org/data_1152' - Label 'HIVDB identifier'\n",
      "Entity 'http://edamontology.org/data_1152' - Label 'HIVDB identifier'\n",
      "Entity 'http://edamontology.org/data_1153' - Label 'OMIM ID'\n",
      "Entity 'http://edamontology.org/data_1153' - Label 'OMIM ID'\n",
      "Entity 'http://edamontology.org/data_1154' - Label 'KEGG object identifier'\n",
      "Entity 'http://edamontology.org/data_1154' - Label 'KEGG object identifier'\n",
      "Entity 'http://edamontology.org/data_1155' - Label 'Pathway ID (reactome)'\n",
      "Entity 'http://edamontology.org/data_1155' - Label 'Pathway ID (reactome)'\n",
      "Entity 'http://edamontology.org/data_1156' - Label 'Pathway ID (aMAZE)'\n",
      "Entity 'http://edamontology.org/data_1156' - Label 'Pathway ID (aMAZE)'\n",
      "Entity 'http://edamontology.org/data_1157' - Label 'Pathway ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_1157' - Label 'Pathway ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_1157' - Label 'Pathway ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_1158' - Label 'Pathway ID (INOH)'\n",
      "Entity 'http://edamontology.org/data_1158' - Label 'Pathway ID (INOH)'\n",
      "Entity 'http://edamontology.org/data_1159' - Label 'Pathway ID (PATIKA)'\n",
      "Entity 'http://edamontology.org/data_1159' - Label 'Pathway ID (PATIKA)'\n",
      "Entity 'http://edamontology.org/data_1160' - Label 'Pathway ID (CPDB)'\n",
      "Entity 'http://edamontology.org/data_1160' - Label 'Pathway ID (CPDB)'\n",
      "Entity 'http://edamontology.org/data_1161' - Label 'Pathway ID (Panther)'\n",
      "Entity 'http://edamontology.org/data_1161' - Label 'Pathway ID (Panther)'\n",
      "Entity 'http://edamontology.org/data_1162' - Label 'MIRIAM identifier'\n",
      "Entity 'http://edamontology.org/data_1162' - Label 'MIRIAM identifier'\n",
      "Entity 'http://edamontology.org/data_1162' - Label 'MIRIAM identifier'\n",
      "Entity 'http://edamontology.org/data_1163' - Label 'MIRIAM data type name'\n",
      "Entity 'http://edamontology.org/data_1163' - Label 'MIRIAM data type name'\n",
      "Entity 'http://edamontology.org/data_1164' - Label 'MIRIAM URI'\n",
      "Entity 'http://edamontology.org/data_1164' - Label 'MIRIAM URI'\n",
      "Entity 'http://edamontology.org/data_1164' - Label 'MIRIAM URI'\n",
      "Entity 'http://edamontology.org/data_1164' - Label 'MIRIAM URI'\n",
      "Entity 'http://edamontology.org/data_1165' - Label 'MIRIAM data type primary name'\n",
      "Entity 'http://edamontology.org/data_1166' - Label 'MIRIAM data type synonymous name'\n",
      "Entity 'http://edamontology.org/data_1167' - Label 'Taverna workflow ID'\n",
      "Entity 'http://edamontology.org/data_1167' - Label 'Taverna workflow ID'\n",
      "Entity 'http://edamontology.org/data_1170' - Label 'Biological model name'\n",
      "Entity 'http://edamontology.org/data_1170' - Label 'Biological model name'\n",
      "Entity 'http://edamontology.org/data_1171' - Label 'BioModel ID'\n",
      "Entity 'http://edamontology.org/data_1171' - Label 'BioModel ID'\n",
      "Entity 'http://edamontology.org/data_1172' - Label 'PubChem CID'\n",
      "Entity 'http://edamontology.org/data_1172' - Label 'PubChem CID'\n",
      "Entity 'http://edamontology.org/data_1173' - Label 'ChemSpider ID'\n",
      "Entity 'http://edamontology.org/data_1173' - Label 'ChemSpider ID'\n",
      "Entity 'http://edamontology.org/data_1174' - Label 'ChEBI ID'\n",
      "Entity 'http://edamontology.org/data_1174' - Label 'ChEBI ID'\n",
      "Entity 'http://edamontology.org/data_1175' - Label 'BioPax concept ID'\n",
      "Entity 'http://edamontology.org/data_1175' - Label 'BioPax concept ID'\n",
      "Entity 'http://edamontology.org/data_1176' - Label 'GO concept ID'\n",
      "Entity 'http://edamontology.org/data_1176' - Label 'GO concept ID'\n",
      "Entity 'http://edamontology.org/data_1177' - Label 'MeSH concept ID'\n",
      "Entity 'http://edamontology.org/data_1177' - Label 'MeSH concept ID'\n",
      "Entity 'http://edamontology.org/data_1178' - Label 'HGNC concept ID'\n",
      "Entity 'http://edamontology.org/data_1178' - Label 'HGNC concept ID'\n",
      "Entity 'http://edamontology.org/data_1179' - Label 'NCBI taxonomy ID'\n",
      "Entity 'http://edamontology.org/data_1179' - Label 'NCBI taxonomy ID'\n",
      "Entity 'http://edamontology.org/data_1179' - Label 'NCBI taxonomy ID'\n",
      "Entity 'http://edamontology.org/data_1180' - Label 'Plant Ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_1180' - Label 'Plant Ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_1181' - Label 'UMLS concept ID'\n",
      "Entity 'http://edamontology.org/data_1181' - Label 'UMLS concept ID'\n",
      "Entity 'http://edamontology.org/data_1182' - Label 'FMA concept ID'\n",
      "Entity 'http://edamontology.org/data_1182' - Label 'FMA concept ID'\n",
      "Entity 'http://edamontology.org/data_1183' - Label 'EMAP concept ID'\n",
      "Entity 'http://edamontology.org/data_1183' - Label 'EMAP concept ID'\n",
      "Entity 'http://edamontology.org/data_1184' - Label 'ChEBI concept ID'\n",
      "Entity 'http://edamontology.org/data_1184' - Label 'ChEBI concept ID'\n",
      "Entity 'http://edamontology.org/data_1185' - Label 'MGED concept ID'\n",
      "Entity 'http://edamontology.org/data_1185' - Label 'MGED concept ID'\n",
      "Entity 'http://edamontology.org/data_1186' - Label 'myGrid concept ID'\n",
      "Entity 'http://edamontology.org/data_1186' - Label 'myGrid concept ID'\n",
      "Entity 'http://edamontology.org/data_1187' - Label 'PubMed ID'\n",
      "Entity 'http://edamontology.org/data_1187' - Label 'PubMed ID'\n",
      "Entity 'http://edamontology.org/data_1188' - Label 'DOI'\n",
      "Entity 'http://edamontology.org/data_1188' - Label 'DOI'\n",
      "Entity 'http://edamontology.org/data_1189' - Label 'Medline UI'\n",
      "Entity 'http://edamontology.org/data_1189' - Label 'Medline UI'\n",
      "Entity 'http://edamontology.org/data_1190' - Label 'Tool name'\n",
      "Entity 'http://edamontology.org/data_1191' - Label 'Tool name (signature)'\n",
      "Entity 'http://edamontology.org/data_1192' - Label 'Tool name (BLAST)'\n",
      "Entity 'http://edamontology.org/data_1193' - Label 'Tool name (FASTA)'\n",
      "Entity 'http://edamontology.org/data_1194' - Label 'Tool name (EMBOSS)'\n",
      "Entity 'http://edamontology.org/data_1195' - Label 'Tool name (EMBASSY package)'\n",
      "Entity 'http://edamontology.org/data_1201' - Label 'QSAR descriptor (constitutional)'\n",
      "Entity 'http://edamontology.org/data_1202' - Label 'QSAR descriptor (electronic)'\n",
      "Entity 'http://edamontology.org/data_1203' - Label 'QSAR descriptor (geometrical)'\n",
      "Entity 'http://edamontology.org/data_1204' - Label 'QSAR descriptor (topological)'\n",
      "Entity 'http://edamontology.org/data_1205' - Label 'QSAR descriptor (molecular)'\n",
      "Entity 'http://edamontology.org/data_1233' - Label 'Sequence set (protein)'\n",
      "Entity 'http://edamontology.org/data_1234' - Label 'Sequence set (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1235' - Label 'Sequence cluster'\n",
      "Entity 'http://edamontology.org/data_1235' - Label 'Sequence cluster'\n",
      "Entity 'http://edamontology.org/data_1236' - Label 'Psiblast checkpoint file'\n",
      "Entity 'http://edamontology.org/data_1236' - Label 'Psiblast checkpoint file'\n",
      "Entity 'http://edamontology.org/data_1237' - Label 'HMMER synthetic sequences set'\n",
      "Entity 'http://edamontology.org/data_1237' - Label 'HMMER synthetic sequences set'\n",
      "Entity 'http://edamontology.org/data_1238' - Label 'Proteolytic digest'\n",
      "Entity 'http://edamontology.org/data_1238' - Label 'Proteolytic digest'\n",
      "Entity 'http://edamontology.org/data_1239' - Label 'Restriction digest'\n",
      "Entity 'http://edamontology.org/data_1240' - Label 'PCR primers'\n",
      "Entity 'http://edamontology.org/data_1241' - Label 'vectorstrip cloning vector definition file'\n",
      "Entity 'http://edamontology.org/data_1241' - Label 'vectorstrip cloning vector definition file'\n",
      "Entity 'http://edamontology.org/data_1242' - Label 'Primer3 internal oligo mishybridizing library'\n",
      "Entity 'http://edamontology.org/data_1242' - Label 'Primer3 internal oligo mishybridizing library'\n",
      "Entity 'http://edamontology.org/data_1243' - Label 'Primer3 mispriming library file'\n",
      "Entity 'http://edamontology.org/data_1243' - Label 'Primer3 mispriming library file'\n",
      "Entity 'http://edamontology.org/data_1244' - Label 'primersearch primer pairs sequence record'\n",
      "Entity 'http://edamontology.org/data_1244' - Label 'primersearch primer pairs sequence record'\n",
      "Entity 'http://edamontology.org/data_1245' - Label 'Sequence cluster (protein)'\n",
      "Entity 'http://edamontology.org/data_1245' - Label 'Sequence cluster (protein)'\n",
      "Entity 'http://edamontology.org/data_1246' - Label 'Sequence cluster (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1246' - Label 'Sequence cluster (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1249' - Label 'Sequence length'\n",
      "Entity 'http://edamontology.org/data_1250' - Label 'Word size'\n",
      "Entity 'http://edamontology.org/data_1250' - Label 'Word size'\n",
      "Entity 'http://edamontology.org/data_1251' - Label 'Window size'\n",
      "Entity 'http://edamontology.org/data_1251' - Label 'Window size'\n",
      "Entity 'http://edamontology.org/data_1252' - Label 'Sequence length range'\n",
      "Entity 'http://edamontology.org/data_1252' - Label 'Sequence length range'\n",
      "Entity 'http://edamontology.org/data_1253' - Label 'Sequence information report'\n",
      "Entity 'http://edamontology.org/data_1253' - Label 'Sequence information report'\n",
      "Entity 'http://edamontology.org/data_1254' - Label 'Sequence property'\n",
      "Entity 'http://edamontology.org/data_1255' - Label 'Sequence features'\n",
      "Entity 'http://edamontology.org/data_1256' - Label 'Sequence features (comparative)'\n",
      "Entity 'http://edamontology.org/data_1256' - Label 'Sequence features (comparative)'\n",
      "Entity 'http://edamontology.org/data_1257' - Label 'Sequence property (protein)'\n",
      "Entity 'http://edamontology.org/data_1257' - Label 'Sequence property (protein)'\n",
      "Entity 'http://edamontology.org/data_1258' - Label 'Sequence property (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1258' - Label 'Sequence property (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_1259' - Label 'Sequence complexity report'\n",
      "Entity 'http://edamontology.org/data_1260' - Label 'Sequence ambiguity report'\n",
      "Entity 'http://edamontology.org/data_1261' - Label 'Sequence composition report'\n",
      "Entity 'http://edamontology.org/data_1262' - Label 'Peptide molecular weight hits'\n",
      "Entity 'http://edamontology.org/data_1263' - Label 'Base position variability plot'\n",
      "Entity 'http://edamontology.org/data_1264' - Label 'Sequence composition table'\n",
      "Entity 'http://edamontology.org/data_1264' - Label 'Sequence composition table'\n",
      "Entity 'http://edamontology.org/data_1265' - Label 'Base frequencies table'\n",
      "Entity 'http://edamontology.org/data_1265' - Label 'Base frequencies table'\n",
      "Entity 'http://edamontology.org/data_1266' - Label 'Base word frequencies table'\n",
      "Entity 'http://edamontology.org/data_1266' - Label 'Base word frequencies table'\n",
      "Entity 'http://edamontology.org/data_1267' - Label 'Amino acid frequencies table'\n",
      "Entity 'http://edamontology.org/data_1267' - Label 'Amino acid frequencies table'\n",
      "Entity 'http://edamontology.org/data_1268' - Label 'Amino acid word frequencies table'\n",
      "Entity 'http://edamontology.org/data_1268' - Label 'Amino acid word frequencies table'\n",
      "Entity 'http://edamontology.org/data_1269' - Label 'DAS sequence feature annotation'\n",
      "Entity 'http://edamontology.org/data_1269' - Label 'DAS sequence feature annotation'\n",
      "Entity 'http://edamontology.org/data_1270' - Label 'Feature table'\n",
      "Entity 'http://edamontology.org/data_1274' - Label 'Map'\n",
      "Entity 'http://edamontology.org/data_1274' - Label 'Map'\n",
      "Entity 'http://edamontology.org/data_1276' - Label 'Nucleic acid features'\n",
      "Entity 'http://edamontology.org/data_1277' - Label 'Protein features'\n",
      "Entity 'http://edamontology.org/data_1277' - Label 'Protein features'\n",
      "Entity 'http://edamontology.org/data_1278' - Label 'Genetic map'\n",
      "Entity 'http://edamontology.org/data_1279' - Label 'Sequence map'\n",
      "Entity 'http://edamontology.org/data_1280' - Label 'Physical map'\n",
      "Entity 'http://edamontology.org/data_1281' - Label 'Sequence signature map'\n",
      "Entity 'http://edamontology.org/data_1281' - Label 'Sequence signature map'\n",
      "Entity 'http://edamontology.org/data_1283' - Label 'Cytogenetic map'\n",
      "Entity 'http://edamontology.org/data_1284' - Label 'DNA transduction map'\n",
      "Entity 'http://edamontology.org/data_1285' - Label 'Gene map'\n",
      "Entity 'http://edamontology.org/data_1286' - Label 'Plasmid map'\n",
      "Entity 'http://edamontology.org/data_1288' - Label 'Genome map'\n",
      "Entity 'http://edamontology.org/data_1289' - Label 'Restriction map'\n",
      "Entity 'http://edamontology.org/data_1289' - Label 'Restriction map'\n",
      "Entity 'http://edamontology.org/data_1290' - Label 'InterPro compact match image'\n",
      "Entity 'http://edamontology.org/data_1290' - Label 'InterPro compact match image'\n",
      "Entity 'http://edamontology.org/data_1291' - Label 'InterPro detailed match image'\n",
      "Entity 'http://edamontology.org/data_1291' - Label 'InterPro detailed match image'\n",
      "Entity 'http://edamontology.org/data_1292' - Label 'InterPro architecture image'\n",
      "Entity 'http://edamontology.org/data_1292' - Label 'InterPro architecture image'\n",
      "Entity 'http://edamontology.org/data_1293' - Label 'SMART protein schematic'\n",
      "Entity 'http://edamontology.org/data_1293' - Label 'SMART protein schematic'\n",
      "Entity 'http://edamontology.org/data_1294' - Label 'GlobPlot domain image'\n",
      "Entity 'http://edamontology.org/data_1294' - Label 'GlobPlot domain image'\n",
      "Entity 'http://edamontology.org/data_1298' - Label 'Sequence motif matches'\n",
      "Entity 'http://edamontology.org/data_1298' - Label 'Sequence motif matches'\n",
      "Entity 'http://edamontology.org/data_1299' - Label 'Sequence features (repeats)'\n",
      "Entity 'http://edamontology.org/data_1299' - Label 'Sequence features (repeats)'\n",
      "Entity 'http://edamontology.org/data_1300' - Label 'Gene and transcript structure (report)'\n",
      "Entity 'http://edamontology.org/data_1300' - Label 'Gene and transcript structure (report)'\n",
      "Entity 'http://edamontology.org/data_1301' - Label 'Mobile genetic elements'\n",
      "Entity 'http://edamontology.org/data_1301' - Label 'Mobile genetic elements'\n",
      "Entity 'http://edamontology.org/data_1303' - Label 'Nucleic acid features (quadruplexes)'\n",
      "Entity 'http://edamontology.org/data_1303' - Label 'Nucleic acid features (quadruplexes)'\n",
      "Entity 'http://edamontology.org/data_1306' - Label 'Nucleosome exclusion sequences'\n",
      "Entity 'http://edamontology.org/data_1306' - Label 'Nucleosome exclusion sequences'\n",
      "Entity 'http://edamontology.org/data_1309' - Label 'Gene features (exonic splicing enhancer)'\n",
      "Entity 'http://edamontology.org/data_1309' - Label 'Gene features (exonic splicing enhancer)'\n",
      "Entity 'http://edamontology.org/data_1310' - Label 'Nucleic acid features (microRNA)'\n",
      "Entity 'http://edamontology.org/data_1310' - Label 'Nucleic acid features (microRNA)'\n",
      "Entity 'http://edamontology.org/data_1313' - Label 'Coding region'\n",
      "Entity 'http://edamontology.org/data_1313' - Label 'Coding region'\n",
      "Entity 'http://edamontology.org/data_1314' - Label 'Gene features (SECIS element)'\n",
      "Entity 'http://edamontology.org/data_1314' - Label 'Gene features (SECIS element)'\n",
      "Entity 'http://edamontology.org/data_1315' - Label 'Transcription factor binding sites'\n",
      "Entity 'http://edamontology.org/data_1315' - Label 'Transcription factor binding sites'\n",
      "Entity 'http://edamontology.org/data_1321' - Label 'Protein features (sites)'\n",
      "Entity 'http://edamontology.org/data_1321' - Label 'Protein features (sites)'\n",
      "Entity 'http://edamontology.org/data_1322' - Label 'Protein features report (signal peptides)'\n",
      "Entity 'http://edamontology.org/data_1322' - Label 'Protein features report (signal peptides)'\n",
      "Entity 'http://edamontology.org/data_1323' - Label 'Protein features report (cleavage sites)'\n",
      "Entity 'http://edamontology.org/data_1323' - Label 'Protein features report (cleavage sites)'\n",
      "Entity 'http://edamontology.org/data_1324' - Label 'Protein features (post-translation modifications)'\n",
      "Entity 'http://edamontology.org/data_1324' - Label 'Protein features (post-translation modifications)'\n",
      "Entity 'http://edamontology.org/data_1325' - Label 'Protein features report (active sites)'\n",
      "Entity 'http://edamontology.org/data_1325' - Label 'Protein features report (active sites)'\n",
      "Entity 'http://edamontology.org/data_1326' - Label 'Protein features report (binding sites)'\n",
      "Entity 'http://edamontology.org/data_1326' - Label 'Protein features report (binding sites)'\n",
      "Entity 'http://edamontology.org/data_1327' - Label 'Protein features (epitopes)'\n",
      "Entity 'http://edamontology.org/data_1327' - Label 'Protein features (epitopes)'\n",
      "Entity 'http://edamontology.org/data_1328' - Label 'Protein features report (nucleic acid binding sites)'\n",
      "Entity 'http://edamontology.org/data_1328' - Label 'Protein features report (nucleic acid binding sites)'\n",
      "Entity 'http://edamontology.org/data_1329' - Label 'MHC Class I epitopes report'\n",
      "Entity 'http://edamontology.org/data_1329' - Label 'MHC Class I epitopes report'\n",
      "Entity 'http://edamontology.org/data_1330' - Label 'MHC Class II epitopes report'\n",
      "Entity 'http://edamontology.org/data_1330' - Label 'MHC Class II epitopes report'\n",
      "Entity 'http://edamontology.org/data_1331' - Label 'Protein features (PEST sites)'\n",
      "Entity 'http://edamontology.org/data_1331' - Label 'Protein features (PEST sites)'\n",
      "Entity 'http://edamontology.org/data_1338' - Label 'Sequence database hits scores list'\n",
      "Entity 'http://edamontology.org/data_1338' - Label 'Sequence database hits scores list'\n",
      "Entity 'http://edamontology.org/data_1339' - Label 'Sequence database hits alignments list'\n",
      "Entity 'http://edamontology.org/data_1339' - Label 'Sequence database hits alignments list'\n",
      "Entity 'http://edamontology.org/data_1340' - Label 'Sequence database hits evaluation data'\n",
      "Entity 'http://edamontology.org/data_1340' - Label 'Sequence database hits evaluation data'\n",
      "Entity 'http://edamontology.org/data_1344' - Label 'MEME motif alphabet'\n",
      "Entity 'http://edamontology.org/data_1344' - Label 'MEME motif alphabet'\n",
      "Entity 'http://edamontology.org/data_1345' - Label 'MEME background frequencies file'\n",
      "Entity 'http://edamontology.org/data_1345' - Label 'MEME background frequencies file'\n",
      "Entity 'http://edamontology.org/data_1346' - Label 'MEME motifs directive file'\n",
      "Entity 'http://edamontology.org/data_1346' - Label 'MEME motifs directive file'\n",
      "Entity 'http://edamontology.org/data_1347' - Label 'Dirichlet distribution'\n",
      "Entity 'http://edamontology.org/data_1348' - Label 'HMM emission and transition counts'\n",
      "Entity 'http://edamontology.org/data_1348' - Label 'HMM emission and transition counts'\n",
      "Entity 'http://edamontology.org/data_1348' - Label 'HMM emission and transition counts'\n",
      "Entity 'http://edamontology.org/data_1352' - Label 'Regular expression'\n",
      "Entity 'http://edamontology.org/data_1353' - Label 'Sequence motif'\n",
      "Entity 'http://edamontology.org/data_1353' - Label 'Sequence motif'\n",
      "Entity 'http://edamontology.org/data_1354' - Label 'Sequence profile'\n",
      "Entity 'http://edamontology.org/data_1354' - Label 'Sequence profile'\n",
      "Entity 'http://edamontology.org/data_1355' - Label 'Protein signature'\n",
      "Entity 'http://edamontology.org/data_1358' - Label 'Prosite nucleotide pattern'\n",
      "Entity 'http://edamontology.org/data_1358' - Label 'Prosite nucleotide pattern'\n",
      "Entity 'http://edamontology.org/data_1359' - Label 'Prosite protein pattern'\n",
      "Entity 'http://edamontology.org/data_1359' - Label 'Prosite protein pattern'\n",
      "Entity 'http://edamontology.org/data_1361' - Label 'Position frequency matrix'\n",
      "Entity 'http://edamontology.org/data_1362' - Label 'Position weight matrix'\n",
      "Entity 'http://edamontology.org/data_1363' - Label 'Information content matrix'\n",
      "Entity 'http://edamontology.org/data_1364' - Label 'Hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_1365' - Label 'Fingerprint'\n",
      "Entity 'http://edamontology.org/data_1368' - Label 'Domainatrix signature'\n",
      "Entity 'http://edamontology.org/data_1368' - Label 'Domainatrix signature'\n",
      "Entity 'http://edamontology.org/data_1371' - Label 'HMMER NULL hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_1371' - Label 'HMMER NULL hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_1372' - Label 'Protein family signature'\n",
      "Entity 'http://edamontology.org/data_1372' - Label 'Protein family signature'\n",
      "Entity 'http://edamontology.org/data_1373' - Label 'Protein domain signature'\n",
      "Entity 'http://edamontology.org/data_1373' - Label 'Protein domain signature'\n",
      "Entity 'http://edamontology.org/data_1374' - Label 'Protein region signature'\n",
      "Entity 'http://edamontology.org/data_1374' - Label 'Protein region signature'\n",
      "Entity 'http://edamontology.org/data_1375' - Label 'Protein repeat signature'\n",
      "Entity 'http://edamontology.org/data_1375' - Label 'Protein repeat signature'\n",
      "Entity 'http://edamontology.org/data_1376' - Label 'Protein site signature'\n",
      "Entity 'http://edamontology.org/data_1376' - Label 'Protein site signature'\n",
      "Entity 'http://edamontology.org/data_1377' - Label 'Protein conserved site signature'\n",
      "Entity 'http://edamontology.org/data_1377' - Label 'Protein conserved site signature'\n",
      "Entity 'http://edamontology.org/data_1378' - Label 'Protein active site signature'\n",
      "Entity 'http://edamontology.org/data_1378' - Label 'Protein active site signature'\n",
      "Entity 'http://edamontology.org/data_1379' - Label 'Protein binding site signature'\n",
      "Entity 'http://edamontology.org/data_1379' - Label 'Protein binding site signature'\n",
      "Entity 'http://edamontology.org/data_1380' - Label 'Protein post-translational modification signature'\n",
      "Entity 'http://edamontology.org/data_1380' - Label 'Protein post-translational modification signature'\n",
      "Entity 'http://edamontology.org/data_1381' - Label 'Pair sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1382' - Label 'Sequence alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1382' - Label 'Sequence alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1383' - Label 'Nucleic acid sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1384' - Label 'Protein sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1385' - Label 'Hybrid sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1386' - Label 'Sequence alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1386' - Label 'Sequence alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1386' - Label 'Sequence alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1387' - Label 'Sequence alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1387' - Label 'Sequence alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1387' - Label 'Sequence alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1388' - Label 'Hybrid sequence alignment (pair)'\n",
      "Entity 'http://edamontology.org/data_1388' - Label 'Hybrid sequence alignment (pair)'\n",
      "Entity 'http://edamontology.org/data_1389' - Label 'Multiple nucleotide sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1389' - Label 'Multiple nucleotide sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1390' - Label 'Multiple protein sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1390' - Label 'Multiple protein sequence alignment'\n",
      "Entity 'http://edamontology.org/data_1394' - Label 'Alignment score or penalty'\n",
      "Entity 'http://edamontology.org/data_1395' - Label 'Score end gaps control'\n",
      "Entity 'http://edamontology.org/data_1395' - Label 'Score end gaps control'\n",
      "Entity 'http://edamontology.org/data_1396' - Label 'Aligned sequence order'\n",
      "Entity 'http://edamontology.org/data_1396' - Label 'Aligned sequence order'\n",
      "Entity 'http://edamontology.org/data_1397' - Label 'Gap opening penalty'\n",
      "Entity 'http://edamontology.org/data_1398' - Label 'Gap extension penalty'\n",
      "Entity 'http://edamontology.org/data_1399' - Label 'Gap separation penalty'\n",
      "Entity 'http://edamontology.org/data_1400' - Label 'Terminal gap penalty'\n",
      "Entity 'http://edamontology.org/data_1400' - Label 'Terminal gap penalty'\n",
      "Entity 'http://edamontology.org/data_1400' - Label 'Terminal gap penalty'\n",
      "Entity 'http://edamontology.org/data_1401' - Label 'Match reward score'\n",
      "Entity 'http://edamontology.org/data_1402' - Label 'Mismatch penalty score'\n",
      "Entity 'http://edamontology.org/data_1403' - Label 'Drop off score'\n",
      "Entity 'http://edamontology.org/data_1404' - Label 'Gap opening penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1404' - Label 'Gap opening penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1405' - Label 'Gap opening penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1405' - Label 'Gap opening penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1406' - Label 'Gap extension penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1406' - Label 'Gap extension penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1407' - Label 'Gap extension penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1407' - Label 'Gap extension penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1408' - Label 'Gap separation penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1408' - Label 'Gap separation penalty (integer)'\n",
      "Entity 'http://edamontology.org/data_1409' - Label 'Gap separation penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1409' - Label 'Gap separation penalty (float)'\n",
      "Entity 'http://edamontology.org/data_1410' - Label 'Terminal gap opening penalty'\n",
      "Entity 'http://edamontology.org/data_1411' - Label 'Terminal gap extension penalty'\n",
      "Entity 'http://edamontology.org/data_1412' - Label 'Sequence identity'\n",
      "Entity 'http://edamontology.org/data_1413' - Label 'Sequence similarity'\n",
      "Entity 'http://edamontology.org/data_1414' - Label 'Sequence alignment metadata (quality report)'\n",
      "Entity 'http://edamontology.org/data_1414' - Label 'Sequence alignment metadata (quality report)'\n",
      "Entity 'http://edamontology.org/data_1415' - Label 'Sequence alignment report (site conservation)'\n",
      "Entity 'http://edamontology.org/data_1415' - Label 'Sequence alignment report (site conservation)'\n",
      "Entity 'http://edamontology.org/data_1416' - Label 'Sequence alignment report (site correlation)'\n",
      "Entity 'http://edamontology.org/data_1416' - Label 'Sequence alignment report (site correlation)'\n",
      "Entity 'http://edamontology.org/data_1417' - Label 'Sequence-profile alignment (Domainatrix signature)'\n",
      "Entity 'http://edamontology.org/data_1417' - Label 'Sequence-profile alignment (Domainatrix signature)'\n",
      "Entity 'http://edamontology.org/data_1418' - Label 'Sequence-profile alignment (HMM)'\n",
      "Entity 'http://edamontology.org/data_1418' - Label 'Sequence-profile alignment (HMM)'\n",
      "Entity 'http://edamontology.org/data_1420' - Label 'Sequence-profile alignment (fingerprint)'\n",
      "Entity 'http://edamontology.org/data_1420' - Label 'Sequence-profile alignment (fingerprint)'\n",
      "Entity 'http://edamontology.org/data_1426' - Label 'Phylogenetic continuous quantitative data'\n",
      "Entity 'http://edamontology.org/data_1427' - Label 'Phylogenetic discrete data'\n",
      "Entity 'http://edamontology.org/data_1428' - Label 'Phylogenetic character cliques'\n",
      "Entity 'http://edamontology.org/data_1429' - Label 'Phylogenetic invariants'\n",
      "Entity 'http://edamontology.org/data_1429' - Label 'Phylogenetic invariants'\n",
      "Entity 'http://edamontology.org/data_1438' - Label 'Phylogenetic report'\n",
      "Entity 'http://edamontology.org/data_1438' - Label 'Phylogenetic report'\n",
      "Entity 'http://edamontology.org/data_1439' - Label 'DNA substitution model'\n",
      "Entity 'http://edamontology.org/data_1440' - Label 'Phylogenetic tree report (tree shape)'\n",
      "Entity 'http://edamontology.org/data_1440' - Label 'Phylogenetic tree report (tree shape)'\n",
      "Entity 'http://edamontology.org/data_1441' - Label 'Phylogenetic tree report (tree evaluation)'\n",
      "Entity 'http://edamontology.org/data_1441' - Label 'Phylogenetic tree report (tree evaluation)'\n",
      "Entity 'http://edamontology.org/data_1442' - Label 'Phylogenetic tree distances'\n",
      "Entity 'http://edamontology.org/data_1443' - Label 'Phylogenetic tree report (tree stratigraphic)'\n",
      "Entity 'http://edamontology.org/data_1443' - Label 'Phylogenetic tree report (tree stratigraphic)'\n",
      "Entity 'http://edamontology.org/data_1444' - Label 'Phylogenetic character contrasts'\n",
      "Entity 'http://edamontology.org/data_1446' - Label 'Comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1446' - Label 'Comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1447' - Label 'Comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1447' - Label 'Comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1448' - Label 'Comparison matrix (nucleotide)'\n",
      "Entity 'http://edamontology.org/data_1449' - Label 'Comparison matrix (amino acid)'\n",
      "Entity 'http://edamontology.org/data_1449' - Label 'Comparison matrix (amino acid)'\n",
      "Entity 'http://edamontology.org/data_1450' - Label 'Nucleotide comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1450' - Label 'Nucleotide comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1451' - Label 'Nucleotide comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1451' - Label 'Nucleotide comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1452' - Label 'Amino acid comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1452' - Label 'Amino acid comparison matrix (integers)'\n",
      "Entity 'http://edamontology.org/data_1453' - Label 'Amino acid comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1453' - Label 'Amino acid comparison matrix (floats)'\n",
      "Entity 'http://edamontology.org/data_1459' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/data_1459' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/data_1460' - Label 'Protein structure'\n",
      "Entity 'http://edamontology.org/data_1460' - Label 'Protein structure'\n",
      "Entity 'http://edamontology.org/data_1461' - Label 'Protein-ligand complex'\n",
      "Entity 'http://edamontology.org/data_1462' - Label 'Carbohydrate structure'\n",
      "Entity 'http://edamontology.org/data_1462' - Label 'Carbohydrate structure'\n",
      "Entity 'http://edamontology.org/data_1462' - Label 'Carbohydrate structure'\n",
      "Entity 'http://edamontology.org/data_1463' - Label 'Small molecule structure'\n",
      "Entity 'http://edamontology.org/data_1463' - Label 'Small molecule structure'\n",
      "Entity 'http://edamontology.org/data_1464' - Label 'DNA structure'\n",
      "Entity 'http://edamontology.org/data_1465' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/data_1465' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/data_1466' - Label 'tRNA structure'\n",
      "Entity 'http://edamontology.org/data_1467' - Label 'Protein chain'\n",
      "Entity 'http://edamontology.org/data_1468' - Label 'Protein domain'\n",
      "Entity 'http://edamontology.org/data_1468' - Label 'Protein domain'\n",
      "Entity 'http://edamontology.org/data_1469' - Label 'Protein structure (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1469' - Label 'Protein structure (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1470' - Label 'C-alpha trace'\n",
      "Entity 'http://edamontology.org/data_1471' - Label 'Protein chain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1471' - Label 'Protein chain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1472' - Label 'Protein chain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1472' - Label 'Protein chain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1473' - Label 'Protein domain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1473' - Label 'Protein domain (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1474' - Label 'Protein domain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1474' - Label 'Protein domain (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1479' - Label 'Structure alignment (pair)'\n",
      "Entity 'http://edamontology.org/data_1480' - Label 'Structure alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1480' - Label 'Structure alignment (multiple)'\n",
      "Entity 'http://edamontology.org/data_1481' - Label 'Protein structure alignment'\n",
      "Entity 'http://edamontology.org/data_1482' - Label 'Nucleic acid structure alignment'\n",
      "Entity 'http://edamontology.org/data_1483' - Label 'Structure alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1483' - Label 'Structure alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1483' - Label 'Structure alignment (protein pair)'\n",
      "Entity 'http://edamontology.org/data_1484' - Label 'Multiple protein tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1484' - Label 'Multiple protein tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1485' - Label 'Structure alignment (protein all atoms)'\n",
      "Entity 'http://edamontology.org/data_1485' - Label 'Structure alignment (protein all atoms)'\n",
      "Entity 'http://edamontology.org/data_1486' - Label 'Structure alignment (protein C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1486' - Label 'Structure alignment (protein C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1487' - Label 'Pairwise protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1487' - Label 'Pairwise protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1488' - Label 'Pairwise protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1488' - Label 'Pairwise protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1489' - Label 'Multiple protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1489' - Label 'Multiple protein tertiary structure alignment (all atoms)'\n",
      "Entity 'http://edamontology.org/data_1490' - Label 'Multiple protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1490' - Label 'Multiple protein tertiary structure alignment (C-alpha atoms)'\n",
      "Entity 'http://edamontology.org/data_1491' - Label 'Structure alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1491' - Label 'Structure alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1491' - Label 'Structure alignment (nucleic acid pair)'\n",
      "Entity 'http://edamontology.org/data_1492' - Label 'Multiple nucleic acid tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1492' - Label 'Multiple nucleic acid tertiary structure alignment'\n",
      "Entity 'http://edamontology.org/data_1493' - Label 'RNA structure alignment'\n",
      "Entity 'http://edamontology.org/data_1494' - Label 'Structural transformation matrix'\n",
      "Entity 'http://edamontology.org/data_1495' - Label 'DaliLite hit table'\n",
      "Entity 'http://edamontology.org/data_1495' - Label 'DaliLite hit table'\n",
      "Entity 'http://edamontology.org/data_1496' - Label 'Molecular similarity score'\n",
      "Entity 'http://edamontology.org/data_1496' - Label 'Molecular similarity score'\n",
      "Entity 'http://edamontology.org/data_1497' - Label 'Root-mean-square deviation'\n",
      "Entity 'http://edamontology.org/data_1498' - Label 'Tanimoto similarity score'\n",
      "Entity 'http://edamontology.org/data_1499' - Label '3D-1D scoring matrix'\n",
      "Entity 'http://edamontology.org/data_1501' - Label 'Amino acid index'\n",
      "Entity 'http://edamontology.org/data_1501' - Label 'Amino acid index'\n",
      "Entity 'http://edamontology.org/data_1502' - Label 'Amino acid index (chemical classes)'\n",
      "Entity 'http://edamontology.org/data_1503' - Label 'Amino acid pair-wise contact potentials'\n",
      "Entity 'http://edamontology.org/data_1505' - Label 'Amino acid index (molecular weight)'\n",
      "Entity 'http://edamontology.org/data_1506' - Label 'Amino acid index (hydropathy)'\n",
      "Entity 'http://edamontology.org/data_1507' - Label 'Amino acid index (White-Wimley data)'\n",
      "Entity 'http://edamontology.org/data_1508' - Label 'Amino acid index (van der Waals radii)'\n",
      "Entity 'http://edamontology.org/data_1509' - Label 'Enzyme report'\n",
      "Entity 'http://edamontology.org/data_1509' - Label 'Enzyme report'\n",
      "Entity 'http://edamontology.org/data_1517' - Label 'Restriction enzyme report'\n",
      "Entity 'http://edamontology.org/data_1517' - Label 'Restriction enzyme report'\n",
      "Entity 'http://edamontology.org/data_1519' - Label 'Peptide molecular weights'\n",
      "Entity 'http://edamontology.org/data_1520' - Label 'Peptide hydrophobic moment'\n",
      "Entity 'http://edamontology.org/data_1521' - Label 'Protein aliphatic index'\n",
      "Entity 'http://edamontology.org/data_1522' - Label 'Protein sequence hydropathy plot'\n",
      "Entity 'http://edamontology.org/data_1523' - Label 'Protein charge plot'\n",
      "Entity 'http://edamontology.org/data_1524' - Label 'Protein solubility'\n",
      "Entity 'http://edamontology.org/data_1525' - Label 'Protein crystallizability'\n",
      "Entity 'http://edamontology.org/data_1526' - Label 'Protein globularity'\n",
      "Entity 'http://edamontology.org/data_1527' - Label 'Protein titration curve'\n",
      "Entity 'http://edamontology.org/data_1527' - Label 'Protein titration curve'\n",
      "Entity 'http://edamontology.org/data_1528' - Label 'Protein isoelectric point'\n",
      "Entity 'http://edamontology.org/data_1529' - Label 'Protein pKa value'\n",
      "Entity 'http://edamontology.org/data_1530' - Label 'Protein hydrogen exchange rate'\n",
      "Entity 'http://edamontology.org/data_1531' - Label 'Protein extinction coefficient'\n",
      "Entity 'http://edamontology.org/data_1532' - Label 'Protein optical density'\n",
      "Entity 'http://edamontology.org/data_1533' - Label 'Protein subcellular localisation'\n",
      "Entity 'http://edamontology.org/data_1533' - Label 'Protein subcellular localisation'\n",
      "Entity 'http://edamontology.org/data_1534' - Label 'Peptide immunogenicity data'\n",
      "Entity 'http://edamontology.org/data_1536' - Label 'MHC peptide immunogenicity report'\n",
      "Entity 'http://edamontology.org/data_1536' - Label 'MHC peptide immunogenicity report'\n",
      "Entity 'http://edamontology.org/data_1537' - Label 'Protein structure report'\n",
      "Entity 'http://edamontology.org/data_1537' - Label 'Protein structure report'\n",
      "Entity 'http://edamontology.org/data_1539' - Label 'Protein structural quality report'\n",
      "Entity 'http://edamontology.org/data_1540' - Label 'Protein non-covalent interactions report'\n",
      "Entity 'http://edamontology.org/data_1540' - Label 'Protein non-covalent interactions report'\n",
      "Entity 'http://edamontology.org/data_1541' - Label 'Protein flexibility or motion report'\n",
      "Entity 'http://edamontology.org/data_1541' - Label 'Protein flexibility or motion report'\n",
      "Entity 'http://edamontology.org/data_1542' - Label 'Protein solvent accessibility'\n",
      "Entity 'http://edamontology.org/data_1543' - Label 'Protein surface report'\n",
      "Entity 'http://edamontology.org/data_1543' - Label 'Protein surface report'\n",
      "Entity 'http://edamontology.org/data_1544' - Label 'Ramachandran plot'\n",
      "Entity 'http://edamontology.org/data_1545' - Label 'Protein dipole moment'\n",
      "Entity 'http://edamontology.org/data_1546' - Label 'Protein distance matrix'\n",
      "Entity 'http://edamontology.org/data_1546' - Label 'Protein distance matrix'\n",
      "Entity 'http://edamontology.org/data_1547' - Label 'Protein contact map'\n",
      "Entity 'http://edamontology.org/data_1548' - Label 'Protein residue 3D cluster'\n",
      "Entity 'http://edamontology.org/data_1549' - Label 'Protein hydrogen bonds'\n",
      "Entity 'http://edamontology.org/data_1550' - Label 'Protein non-canonical interactions'\n",
      "Entity 'http://edamontology.org/data_1550' - Label 'Protein non-canonical interactions'\n",
      "Entity 'http://edamontology.org/data_1553' - Label 'CATH node'\n",
      "Entity 'http://edamontology.org/data_1553' - Label 'CATH node'\n",
      "Entity 'http://edamontology.org/data_1554' - Label 'SCOP node'\n",
      "Entity 'http://edamontology.org/data_1554' - Label 'SCOP node'\n",
      "Entity 'http://edamontology.org/data_1555' - Label 'EMBASSY domain classification'\n",
      "Entity 'http://edamontology.org/data_1555' - Label 'EMBASSY domain classification'\n",
      "Entity 'http://edamontology.org/data_1556' - Label 'CATH class'\n",
      "Entity 'http://edamontology.org/data_1556' - Label 'CATH class'\n",
      "Entity 'http://edamontology.org/data_1557' - Label 'CATH architecture'\n",
      "Entity 'http://edamontology.org/data_1557' - Label 'CATH architecture'\n",
      "Entity 'http://edamontology.org/data_1558' - Label 'CATH topology'\n",
      "Entity 'http://edamontology.org/data_1558' - Label 'CATH topology'\n",
      "Entity 'http://edamontology.org/data_1559' - Label 'CATH homologous superfamily'\n",
      "Entity 'http://edamontology.org/data_1559' - Label 'CATH homologous superfamily'\n",
      "Entity 'http://edamontology.org/data_1560' - Label 'CATH structurally similar group'\n",
      "Entity 'http://edamontology.org/data_1560' - Label 'CATH structurally similar group'\n",
      "Entity 'http://edamontology.org/data_1561' - Label 'CATH functional category'\n",
      "Entity 'http://edamontology.org/data_1561' - Label 'CATH functional category'\n",
      "Entity 'http://edamontology.org/data_1564' - Label 'Protein fold recognition report'\n",
      "Entity 'http://edamontology.org/data_1564' - Label 'Protein fold recognition report'\n",
      "Entity 'http://edamontology.org/data_1565' - Label 'Protein-protein interaction report'\n",
      "Entity 'http://edamontology.org/data_1565' - Label 'Protein-protein interaction report'\n",
      "Entity 'http://edamontology.org/data_1566' - Label 'Protein-ligand interaction report'\n",
      "Entity 'http://edamontology.org/data_1567' - Label 'Protein-nucleic acid interactions report'\n",
      "Entity 'http://edamontology.org/data_1567' - Label 'Protein-nucleic acid interactions report'\n",
      "Entity 'http://edamontology.org/data_1583' - Label 'Nucleic acid melting profile'\n",
      "Entity 'http://edamontology.org/data_1584' - Label 'Nucleic acid enthalpy'\n",
      "Entity 'http://edamontology.org/data_1585' - Label 'Nucleic acid entropy'\n",
      "Entity 'http://edamontology.org/data_1586' - Label 'Nucleic acid melting temperature'\n",
      "Entity 'http://edamontology.org/data_1586' - Label 'Nucleic acid melting temperature'\n",
      "Entity 'http://edamontology.org/data_1587' - Label 'Nucleic acid stitch profile'\n",
      "Entity 'http://edamontology.org/data_1587' - Label 'Nucleic acid stitch profile'\n",
      "Entity 'http://edamontology.org/data_1588' - Label 'DNA base pair stacking energies data'\n",
      "Entity 'http://edamontology.org/data_1589' - Label 'DNA base pair twist angle data'\n",
      "Entity 'http://edamontology.org/data_1590' - Label 'DNA base trimer roll angles data'\n",
      "Entity 'http://edamontology.org/data_1591' - Label 'Vienna RNA parameters'\n",
      "Entity 'http://edamontology.org/data_1591' - Label 'Vienna RNA parameters'\n",
      "Entity 'http://edamontology.org/data_1592' - Label 'Vienna RNA structure constraints'\n",
      "Entity 'http://edamontology.org/data_1592' - Label 'Vienna RNA structure constraints'\n",
      "Entity 'http://edamontology.org/data_1593' - Label 'Vienna RNA concentration data'\n",
      "Entity 'http://edamontology.org/data_1593' - Label 'Vienna RNA concentration data'\n",
      "Entity 'http://edamontology.org/data_1594' - Label 'Vienna RNA calculated energy'\n",
      "Entity 'http://edamontology.org/data_1594' - Label 'Vienna RNA calculated energy'\n",
      "Entity 'http://edamontology.org/data_1595' - Label 'Base pairing probability matrix dotplot'\n",
      "Entity 'http://edamontology.org/data_1595' - Label 'Base pairing probability matrix dotplot'\n",
      "Entity 'http://edamontology.org/data_1596' - Label 'Nucleic acid folding report'\n",
      "Entity 'http://edamontology.org/data_1597' - Label 'Codon usage table'\n",
      "Entity 'http://edamontology.org/data_1597' - Label 'Codon usage table'\n",
      "Entity 'http://edamontology.org/data_1598' - Label 'Genetic code'\n",
      "Entity 'http://edamontology.org/data_1599' - Label 'Codon adaptation index'\n",
      "Entity 'http://edamontology.org/data_1599' - Label 'Codon adaptation index'\n",
      "Entity 'http://edamontology.org/data_1600' - Label 'Codon usage bias plot'\n",
      "Entity 'http://edamontology.org/data_1601' - Label 'Nc statistic'\n",
      "Entity 'http://edamontology.org/data_1601' - Label 'Nc statistic'\n",
      "Entity 'http://edamontology.org/data_1602' - Label 'Codon usage fraction difference'\n",
      "Entity 'http://edamontology.org/data_1621' - Label 'Pharmacogenomic test report'\n",
      "Entity 'http://edamontology.org/data_1622' - Label 'Disease report'\n",
      "Entity 'http://edamontology.org/data_1622' - Label 'Disease report'\n",
      "Entity 'http://edamontology.org/data_1634' - Label 'Linkage disequilibrium (report)'\n",
      "Entity 'http://edamontology.org/data_1634' - Label 'Linkage disequilibrium (report)'\n",
      "Entity 'http://edamontology.org/data_1636' - Label 'Heat map'\n",
      "Entity 'http://edamontology.org/data_1636' - Label 'Heat map'\n",
      "Entity 'http://edamontology.org/data_1642' - Label 'Affymetrix probe sets library file'\n",
      "Entity 'http://edamontology.org/data_1642' - Label 'Affymetrix probe sets library file'\n",
      "Entity 'http://edamontology.org/data_1643' - Label 'Affymetrix probe sets information library file'\n",
      "Entity 'http://edamontology.org/data_1643' - Label 'Affymetrix probe sets information library file'\n",
      "Entity 'http://edamontology.org/data_1646' - Label 'Molecular weights standard fingerprint'\n",
      "Entity 'http://edamontology.org/data_1646' - Label 'Molecular weights standard fingerprint'\n",
      "Entity 'http://edamontology.org/data_1656' - Label 'Metabolic pathway report'\n",
      "Entity 'http://edamontology.org/data_1656' - Label 'Metabolic pathway report'\n",
      "Entity 'http://edamontology.org/data_1657' - Label 'Genetic information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1657' - Label 'Genetic information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1658' - Label 'Environmental information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1658' - Label 'Environmental information processing pathway report'\n",
      "Entity 'http://edamontology.org/data_1659' - Label 'Signal transduction pathway report'\n",
      "Entity 'http://edamontology.org/data_1659' - Label 'Signal transduction pathway report'\n",
      "Entity 'http://edamontology.org/data_1660' - Label 'Cellular process pathways report'\n",
      "Entity 'http://edamontology.org/data_1660' - Label 'Cellular process pathways report'\n",
      "Entity 'http://edamontology.org/data_1661' - Label 'Disease pathway or network report'\n",
      "Entity 'http://edamontology.org/data_1661' - Label 'Disease pathway or network report'\n",
      "Entity 'http://edamontology.org/data_1662' - Label 'Drug structure relationship map'\n",
      "Entity 'http://edamontology.org/data_1662' - Label 'Drug structure relationship map'\n",
      "Entity 'http://edamontology.org/data_1663' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/data_1663' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/data_1664' - Label 'MIRIAM datatype'\n",
      "Entity 'http://edamontology.org/data_1664' - Label 'MIRIAM datatype'\n",
      "Entity 'http://edamontology.org/data_1667' - Label 'E-value'\n",
      "Entity 'http://edamontology.org/data_1668' - Label 'Z-value'\n",
      "Entity 'http://edamontology.org/data_1669' - Label 'P-value'\n",
      "Entity 'http://edamontology.org/data_1670' - Label 'Database version information'\n",
      "Entity 'http://edamontology.org/data_1670' - Label 'Database version information'\n",
      "Entity 'http://edamontology.org/data_1671' - Label 'Tool version information'\n",
      "Entity 'http://edamontology.org/data_1671' - Label 'Tool version information'\n",
      "Entity 'http://edamontology.org/data_1672' - Label 'CATH version information'\n",
      "Entity 'http://edamontology.org/data_1672' - Label 'CATH version information'\n",
      "Entity 'http://edamontology.org/data_1673' - Label 'Swiss-Prot to PDB mapping'\n",
      "Entity 'http://edamontology.org/data_1673' - Label 'Swiss-Prot to PDB mapping'\n",
      "Entity 'http://edamontology.org/data_1674' - Label 'Sequence database cross-references'\n",
      "Entity 'http://edamontology.org/data_1674' - Label 'Sequence database cross-references'\n",
      "Entity 'http://edamontology.org/data_1675' - Label 'Job status'\n",
      "Entity 'http://edamontology.org/data_1675' - Label 'Job status'\n",
      "Entity 'http://edamontology.org/data_1676' - Label 'Job ID'\n",
      "Entity 'http://edamontology.org/data_1676' - Label 'Job ID'\n",
      "Entity 'http://edamontology.org/data_1677' - Label 'Job type'\n",
      "Entity 'http://edamontology.org/data_1677' - Label 'Job type'\n",
      "Entity 'http://edamontology.org/data_1678' - Label 'Tool log'\n",
      "Entity 'http://edamontology.org/data_1678' - Label 'Tool log'\n",
      "Entity 'http://edamontology.org/data_1679' - Label 'DaliLite log file'\n",
      "Entity 'http://edamontology.org/data_1679' - Label 'DaliLite log file'\n",
      "Entity 'http://edamontology.org/data_1680' - Label 'STRIDE log file'\n",
      "Entity 'http://edamontology.org/data_1680' - Label 'STRIDE log file'\n",
      "Entity 'http://edamontology.org/data_1681' - Label 'NACCESS log file'\n",
      "Entity 'http://edamontology.org/data_1681' - Label 'NACCESS log file'\n",
      "Entity 'http://edamontology.org/data_1682' - Label 'EMBOSS wordfinder log file'\n",
      "Entity 'http://edamontology.org/data_1682' - Label 'EMBOSS wordfinder log file'\n",
      "Entity 'http://edamontology.org/data_1683' - Label 'EMBOSS domainatrix log file'\n",
      "Entity 'http://edamontology.org/data_1683' - Label 'EMBOSS domainatrix log file'\n",
      "Entity 'http://edamontology.org/data_1684' - Label 'EMBOSS sites log file'\n",
      "Entity 'http://edamontology.org/data_1684' - Label 'EMBOSS sites log file'\n",
      "Entity 'http://edamontology.org/data_1685' - Label 'EMBOSS supermatcher error file'\n",
      "Entity 'http://edamontology.org/data_1685' - Label 'EMBOSS supermatcher error file'\n",
      "Entity 'http://edamontology.org/data_1686' - Label 'EMBOSS megamerger log file'\n",
      "Entity 'http://edamontology.org/data_1686' - Label 'EMBOSS megamerger log file'\n",
      "Entity 'http://edamontology.org/data_1687' - Label 'EMBOSS whichdb log file'\n",
      "Entity 'http://edamontology.org/data_1687' - Label 'EMBOSS whichdb log file'\n",
      "Entity 'http://edamontology.org/data_1688' - Label 'EMBOSS vectorstrip log file'\n",
      "Entity 'http://edamontology.org/data_1688' - Label 'EMBOSS vectorstrip log file'\n",
      "Entity 'http://edamontology.org/data_1689' - Label 'Username'\n",
      "Entity 'http://edamontology.org/data_1690' - Label 'Password'\n",
      "Entity 'http://edamontology.org/data_1691' - Label 'Email address'\n",
      "Entity 'http://edamontology.org/data_1692' - Label 'Person name'\n",
      "Entity 'http://edamontology.org/data_1693' - Label 'Number of iterations'\n",
      "Entity 'http://edamontology.org/data_1693' - Label 'Number of iterations'\n",
      "Entity 'http://edamontology.org/data_1694' - Label 'Number of output entities'\n",
      "Entity 'http://edamontology.org/data_1694' - Label 'Number of output entities'\n",
      "Entity 'http://edamontology.org/data_1695' - Label 'Hit sort order'\n",
      "Entity 'http://edamontology.org/data_1695' - Label 'Hit sort order'\n",
      "Entity 'http://edamontology.org/data_1696' - Label 'Drug report'\n",
      "Entity 'http://edamontology.org/data_1696' - Label 'Drug report'\n",
      "Entity 'http://edamontology.org/data_1707' - Label 'Phylogenetic tree image'\n",
      "Entity 'http://edamontology.org/data_1708' - Label 'RNA secondary structure image'\n",
      "Entity 'http://edamontology.org/data_1709' - Label 'Protein secondary structure image'\n",
      "Entity 'http://edamontology.org/data_1710' - Label 'Structure image'\n",
      "Entity 'http://edamontology.org/data_1711' - Label 'Sequence alignment image'\n",
      "Entity 'http://edamontology.org/data_1712' - Label 'Chemical structure image'\n",
      "Entity 'http://edamontology.org/data_1713' - Label 'Fate map'\n",
      "Entity 'http://edamontology.org/data_1713' - Label 'Fate map'\n",
      "Entity 'http://edamontology.org/data_1714' - Label 'Microarray spots image'\n",
      "Entity 'http://edamontology.org/data_1714' - Label 'Microarray spots image'\n",
      "Entity 'http://edamontology.org/data_1715' - Label 'BioPax term'\n",
      "Entity 'http://edamontology.org/data_1715' - Label 'BioPax term'\n",
      "Entity 'http://edamontology.org/data_1716' - Label 'GO'\n",
      "Entity 'http://edamontology.org/data_1716' - Label 'GO'\n",
      "Entity 'http://edamontology.org/data_1717' - Label 'MeSH'\n",
      "Entity 'http://edamontology.org/data_1717' - Label 'MeSH'\n",
      "Entity 'http://edamontology.org/data_1718' - Label 'HGNC'\n",
      "Entity 'http://edamontology.org/data_1718' - Label 'HGNC'\n",
      "Entity 'http://edamontology.org/data_1719' - Label 'NCBI taxonomy vocabulary'\n",
      "Entity 'http://edamontology.org/data_1719' - Label 'NCBI taxonomy vocabulary'\n",
      "Entity 'http://edamontology.org/data_1720' - Label 'Plant ontology term'\n",
      "Entity 'http://edamontology.org/data_1720' - Label 'Plant ontology term'\n",
      "Entity 'http://edamontology.org/data_1721' - Label 'UMLS'\n",
      "Entity 'http://edamontology.org/data_1721' - Label 'UMLS'\n",
      "Entity 'http://edamontology.org/data_1722' - Label 'FMA'\n",
      "Entity 'http://edamontology.org/data_1722' - Label 'FMA'\n",
      "Entity 'http://edamontology.org/data_1723' - Label 'EMAP'\n",
      "Entity 'http://edamontology.org/data_1723' - Label 'EMAP'\n",
      "Entity 'http://edamontology.org/data_1724' - Label 'ChEBI'\n",
      "Entity 'http://edamontology.org/data_1724' - Label 'ChEBI'\n",
      "Entity 'http://edamontology.org/data_1725' - Label 'MGED'\n",
      "Entity 'http://edamontology.org/data_1725' - Label 'MGED'\n",
      "Entity 'http://edamontology.org/data_1726' - Label 'myGrid'\n",
      "Entity 'http://edamontology.org/data_1726' - Label 'myGrid'\n",
      "Entity 'http://edamontology.org/data_1727' - Label 'GO (biological process)'\n",
      "Entity 'http://edamontology.org/data_1727' - Label 'GO (biological process)'\n",
      "Entity 'http://edamontology.org/data_1728' - Label 'GO (molecular function)'\n",
      "Entity 'http://edamontology.org/data_1728' - Label 'GO (molecular function)'\n",
      "Entity 'http://edamontology.org/data_1729' - Label 'GO (cellular component)'\n",
      "Entity 'http://edamontology.org/data_1729' - Label 'GO (cellular component)'\n",
      "Entity 'http://edamontology.org/data_1730' - Label 'Ontology relation type'\n",
      "Entity 'http://edamontology.org/data_1730' - Label 'Ontology relation type'\n",
      "Entity 'http://edamontology.org/data_1731' - Label 'Ontology concept definition'\n",
      "Entity 'http://edamontology.org/data_1732' - Label 'Ontology concept comment'\n",
      "Entity 'http://edamontology.org/data_1732' - Label 'Ontology concept comment'\n",
      "Entity 'http://edamontology.org/data_1733' - Label 'Ontology concept reference'\n",
      "Entity 'http://edamontology.org/data_1733' - Label 'Ontology concept reference'\n",
      "Entity 'http://edamontology.org/data_1738' - Label 'doc2loc document information'\n",
      "Entity 'http://edamontology.org/data_1738' - Label 'doc2loc document information'\n",
      "Entity 'http://edamontology.org/data_1742' - Label 'PDB residue number'\n",
      "Entity 'http://edamontology.org/data_1743' - Label 'Atomic coordinate'\n",
      "Entity 'http://edamontology.org/data_1744' - Label 'Atomic x coordinate'\n",
      "Entity 'http://edamontology.org/data_1744' - Label 'Atomic x coordinate'\n",
      "Entity 'http://edamontology.org/data_1745' - Label 'Atomic y coordinate'\n",
      "Entity 'http://edamontology.org/data_1745' - Label 'Atomic y coordinate'\n",
      "Entity 'http://edamontology.org/data_1746' - Label 'Atomic z coordinate'\n",
      "Entity 'http://edamontology.org/data_1746' - Label 'Atomic z coordinate'\n",
      "Entity 'http://edamontology.org/data_1748' - Label 'PDB atom name'\n",
      "Entity 'http://edamontology.org/data_1755' - Label 'Protein atom'\n",
      "Entity 'http://edamontology.org/data_1756' - Label 'Protein residue'\n",
      "Entity 'http://edamontology.org/data_1757' - Label 'Atom name'\n",
      "Entity 'http://edamontology.org/data_1757' - Label 'Atom name'\n",
      "Entity 'http://edamontology.org/data_1758' - Label 'PDB residue name'\n",
      "Entity 'http://edamontology.org/data_1759' - Label 'PDB model number'\n",
      "Entity 'http://edamontology.org/data_1762' - Label 'CATH domain report'\n",
      "Entity 'http://edamontology.org/data_1762' - Label 'CATH domain report'\n",
      "Entity 'http://edamontology.org/data_1764' - Label 'CATH representative domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1764' - Label 'CATH representative domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1765' - Label 'CATH representative domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1765' - Label 'CATH representative domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1766' - Label 'CATH domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1766' - Label 'CATH domain sequences (ATOM)'\n",
      "Entity 'http://edamontology.org/data_1767' - Label 'CATH domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1767' - Label 'CATH domain sequences (COMBS)'\n",
      "Entity 'http://edamontology.org/data_1771' - Label 'Sequence version'\n",
      "Entity 'http://edamontology.org/data_1772' - Label 'Score'\n",
      "Entity 'http://edamontology.org/data_1776' - Label 'Protein report (function)'\n",
      "Entity 'http://edamontology.org/data_1776' - Label 'Protein report (function)'\n",
      "Entity 'http://edamontology.org/data_1783' - Label 'Gene name (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1783' - Label 'Gene name (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1784' - Label 'Gene name (CGD)'\n",
      "Entity 'http://edamontology.org/data_1784' - Label 'Gene name (CGD)'\n",
      "Entity 'http://edamontology.org/data_1785' - Label 'Gene name (dictyBase)'\n",
      "Entity 'http://edamontology.org/data_1785' - Label 'Gene name (dictyBase)'\n",
      "Entity 'http://edamontology.org/data_1786' - Label 'Gene name (EcoGene primary)'\n",
      "Entity 'http://edamontology.org/data_1786' - Label 'Gene name (EcoGene primary)'\n",
      "Entity 'http://edamontology.org/data_1787' - Label 'Gene name (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1787' - Label 'Gene name (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1788' - Label 'Gene name (SGD)'\n",
      "Entity 'http://edamontology.org/data_1788' - Label 'Gene name (SGD)'\n",
      "Entity 'http://edamontology.org/data_1789' - Label 'Gene name (TGD)'\n",
      "Entity 'http://edamontology.org/data_1789' - Label 'Gene name (TGD)'\n",
      "Entity 'http://edamontology.org/data_1790' - Label 'Gene name (CGSC)'\n",
      "Entity 'http://edamontology.org/data_1790' - Label 'Gene name (CGSC)'\n",
      "Entity 'http://edamontology.org/data_1791' - Label 'Gene name (HGNC)'\n",
      "Entity 'http://edamontology.org/data_1791' - Label 'Gene name (HGNC)'\n",
      "Entity 'http://edamontology.org/data_1792' - Label 'Gene name (MGD)'\n",
      "Entity 'http://edamontology.org/data_1792' - Label 'Gene name (MGD)'\n",
      "Entity 'http://edamontology.org/data_1793' - Label 'Gene name (Bacillus subtilis)'\n",
      "Entity 'http://edamontology.org/data_1793' - Label 'Gene name (Bacillus subtilis)'\n",
      "Entity 'http://edamontology.org/data_1794' - Label 'Gene ID (PlasmoDB)'\n",
      "Entity 'http://edamontology.org/data_1794' - Label 'Gene ID (PlasmoDB)'\n",
      "Entity 'http://edamontology.org/data_1795' - Label 'Gene ID (EcoGene)'\n",
      "Entity 'http://edamontology.org/data_1795' - Label 'Gene ID (EcoGene)'\n",
      "Entity 'http://edamontology.org/data_1796' - Label 'Gene ID (FlyBase)'\n",
      "Entity 'http://edamontology.org/data_1796' - Label 'Gene ID (FlyBase)'\n",
      "Entity 'http://edamontology.org/data_1797' - Label 'Gene ID (GeneDB Glossina morsitans)'\n",
      "Entity 'http://edamontology.org/data_1797' - Label 'Gene ID (GeneDB Glossina morsitans)'\n",
      "Entity 'http://edamontology.org/data_1798' - Label 'Gene ID (GeneDB Leishmania major)'\n",
      "Entity 'http://edamontology.org/data_1798' - Label 'Gene ID (GeneDB Leishmania major)'\n",
      "Entity 'http://edamontology.org/data_1799' - Label 'Gene ID (GeneDB Plasmodium falciparum)'\n",
      "Entity 'http://edamontology.org/data_1799' - Label 'Gene ID (GeneDB Plasmodium falciparum)'\n",
      "Entity 'http://edamontology.org/data_1800' - Label 'Gene ID (GeneDB Schizosaccharomyces pombe)'\n",
      "Entity 'http://edamontology.org/data_1800' - Label 'Gene ID (GeneDB Schizosaccharomyces pombe)'\n",
      "Entity 'http://edamontology.org/data_1801' - Label 'Gene ID (GeneDB Trypanosoma brucei)'\n",
      "Entity 'http://edamontology.org/data_1801' - Label 'Gene ID (GeneDB Trypanosoma brucei)'\n",
      "Entity 'http://edamontology.org/data_1802' - Label 'Gene ID (Gramene)'\n",
      "Entity 'http://edamontology.org/data_1802' - Label 'Gene ID (Gramene)'\n",
      "Entity 'http://edamontology.org/data_1803' - Label 'Gene ID (Virginia microbial)'\n",
      "Entity 'http://edamontology.org/data_1803' - Label 'Gene ID (Virginia microbial)'\n",
      "Entity 'http://edamontology.org/data_1804' - Label 'Gene ID (SGN)'\n",
      "Entity 'http://edamontology.org/data_1804' - Label 'Gene ID (SGN)'\n",
      "Entity 'http://edamontology.org/data_1805' - Label 'Gene ID (WormBase)'\n",
      "Entity 'http://edamontology.org/data_1805' - Label 'Gene ID (WormBase)'\n",
      "Entity 'http://edamontology.org/data_1805' - Label 'Gene ID (WormBase)'\n",
      "Entity 'http://edamontology.org/data_1806' - Label 'Gene synonym'\n",
      "Entity 'http://edamontology.org/data_1806' - Label 'Gene synonym'\n",
      "Entity 'http://edamontology.org/data_1807' - Label 'ORF name'\n",
      "Entity 'http://edamontology.org/data_1807' - Label 'ORF name'\n",
      "Entity 'http://edamontology.org/data_1852' - Label 'Sequence assembly component'\n",
      "Entity 'http://edamontology.org/data_1852' - Label 'Sequence assembly component'\n",
      "Entity 'http://edamontology.org/data_1853' - Label 'Chromosome annotation (aberration)'\n",
      "Entity 'http://edamontology.org/data_1853' - Label 'Chromosome annotation (aberration)'\n",
      "Entity 'http://edamontology.org/data_1855' - Label 'Clone ID'\n",
      "Entity 'http://edamontology.org/data_1856' - Label 'PDB insertion code'\n",
      "Entity 'http://edamontology.org/data_1857' - Label 'Atomic occupancy'\n",
      "Entity 'http://edamontology.org/data_1858' - Label 'Isotropic B factor'\n",
      "Entity 'http://edamontology.org/data_1859' - Label 'Deletion map'\n",
      "Entity 'http://edamontology.org/data_1860' - Label 'QTL map'\n",
      "Entity 'http://edamontology.org/data_1863' - Label 'Haplotype map'\n",
      "Entity 'http://edamontology.org/data_1864' - Label 'Map set data'\n",
      "Entity 'http://edamontology.org/data_1864' - Label 'Map set data'\n",
      "Entity 'http://edamontology.org/data_1865' - Label 'Map feature'\n",
      "Entity 'http://edamontology.org/data_1865' - Label 'Map feature'\n",
      "Entity 'http://edamontology.org/data_1865' - Label 'Map feature'\n",
      "Entity 'http://edamontology.org/data_1865' - Label 'Map feature'\n",
      "Entity 'http://edamontology.org/data_1866' - Label 'Map type'\n",
      "Entity 'http://edamontology.org/data_1866' - Label 'Map type'\n",
      "Entity 'http://edamontology.org/data_1867' - Label 'Protein fold name'\n",
      "Entity 'http://edamontology.org/data_1868' - Label 'Taxon'\n",
      "Entity 'http://edamontology.org/data_1869' - Label 'Organism identifier'\n",
      "Entity 'http://edamontology.org/data_1869' - Label 'Organism identifier'\n",
      "Entity 'http://edamontology.org/data_1870' - Label 'Genus name'\n",
      "Entity 'http://edamontology.org/data_1872' - Label 'Taxonomic classification'\n",
      "Entity 'http://edamontology.org/data_1873' - Label 'iHOP organism ID'\n",
      "Entity 'http://edamontology.org/data_1873' - Label 'iHOP organism ID'\n",
      "Entity 'http://edamontology.org/data_1874' - Label 'Genbank common name'\n",
      "Entity 'http://edamontology.org/data_1875' - Label 'NCBI taxon'\n",
      "Entity 'http://edamontology.org/data_1877' - Label 'Synonym'\n",
      "Entity 'http://edamontology.org/data_1877' - Label 'Synonym'\n",
      "Entity 'http://edamontology.org/data_1878' - Label 'Misspelling'\n",
      "Entity 'http://edamontology.org/data_1878' - Label 'Misspelling'\n",
      "Entity 'http://edamontology.org/data_1879' - Label 'Acronym'\n",
      "Entity 'http://edamontology.org/data_1879' - Label 'Acronym'\n",
      "Entity 'http://edamontology.org/data_1880' - Label 'Misnomer'\n",
      "Entity 'http://edamontology.org/data_1880' - Label 'Misnomer'\n",
      "Entity 'http://edamontology.org/data_1881' - Label 'Author ID'\n",
      "Entity 'http://edamontology.org/data_1882' - Label 'DragonDB author identifier'\n",
      "Entity 'http://edamontology.org/data_1883' - Label 'Annotated URI'\n",
      "Entity 'http://edamontology.org/data_1884' - Label 'UniProt keywords'\n",
      "Entity 'http://edamontology.org/data_1884' - Label 'UniProt keywords'\n",
      "Entity 'http://edamontology.org/data_1885' - Label 'Gene ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1885' - Label 'Gene ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1886' - Label 'Blattner number'\n",
      "Entity 'http://edamontology.org/data_1886' - Label 'Blattner number'\n",
      "Entity 'http://edamontology.org/data_1887' - Label 'Gene ID (MIPS Maize)'\n",
      "Entity 'http://edamontology.org/data_1887' - Label 'Gene ID (MIPS Maize)'\n",
      "Entity 'http://edamontology.org/data_1888' - Label 'Gene ID (MIPS Medicago)'\n",
      "Entity 'http://edamontology.org/data_1888' - Label 'Gene ID (MIPS Medicago)'\n",
      "Entity 'http://edamontology.org/data_1889' - Label 'Gene name (DragonDB)'\n",
      "Entity 'http://edamontology.org/data_1889' - Label 'Gene name (DragonDB)'\n",
      "Entity 'http://edamontology.org/data_1890' - Label 'Gene name (Arabidopsis)'\n",
      "Entity 'http://edamontology.org/data_1890' - Label 'Gene name (Arabidopsis)'\n",
      "Entity 'http://edamontology.org/data_1891' - Label 'iHOP symbol'\n",
      "Entity 'http://edamontology.org/data_1891' - Label 'iHOP symbol'\n",
      "Entity 'http://edamontology.org/data_1891' - Label 'iHOP symbol'\n",
      "Entity 'http://edamontology.org/data_1891' - Label 'iHOP symbol'\n",
      "Entity 'http://edamontology.org/data_1892' - Label 'Gene name (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1892' - Label 'Gene name (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_1893' - Label 'Locus ID'\n",
      "Entity 'http://edamontology.org/data_1893' - Label 'Locus ID'\n",
      "Entity 'http://edamontology.org/data_1895' - Label 'Locus ID (AGI)'\n",
      "Entity 'http://edamontology.org/data_1895' - Label 'Locus ID (AGI)'\n",
      "Entity 'http://edamontology.org/data_1896' - Label 'Locus ID (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1896' - Label 'Locus ID (ASPGD)'\n",
      "Entity 'http://edamontology.org/data_1897' - Label 'Locus ID (MGG)'\n",
      "Entity 'http://edamontology.org/data_1897' - Label 'Locus ID (MGG)'\n",
      "Entity 'http://edamontology.org/data_1898' - Label 'Locus ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1898' - Label 'Locus ID (CGD)'\n",
      "Entity 'http://edamontology.org/data_1899' - Label 'Locus ID (CMR)'\n",
      "Entity 'http://edamontology.org/data_1899' - Label 'Locus ID (CMR)'\n",
      "Entity 'http://edamontology.org/data_1900' - Label 'NCBI locus tag'\n",
      "Entity 'http://edamontology.org/data_1900' - Label 'NCBI locus tag'\n",
      "Entity 'http://edamontology.org/data_1901' - Label 'Locus ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1901' - Label 'Locus ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1901' - Label 'Locus ID (SGD)'\n",
      "Entity 'http://edamontology.org/data_1902' - Label 'Locus ID (MMP)'\n",
      "Entity 'http://edamontology.org/data_1902' - Label 'Locus ID (MMP)'\n",
      "Entity 'http://edamontology.org/data_1903' - Label 'Locus ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1903' - Label 'Locus ID (DictyBase)'\n",
      "Entity 'http://edamontology.org/data_1904' - Label 'Locus ID (EntrezGene)'\n",
      "Entity 'http://edamontology.org/data_1904' - Label 'Locus ID (EntrezGene)'\n",
      "Entity 'http://edamontology.org/data_1905' - Label 'Locus ID (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1905' - Label 'Locus ID (MaizeGDB)'\n",
      "Entity 'http://edamontology.org/data_1906' - Label 'Quantitative trait locus'\n",
      "Entity 'http://edamontology.org/data_1906' - Label 'Quantitative trait locus'\n",
      "Entity 'http://edamontology.org/data_1907' - Label 'Gene ID (KOME)'\n",
      "Entity 'http://edamontology.org/data_1907' - Label 'Gene ID (KOME)'\n",
      "Entity 'http://edamontology.org/data_1908' - Label 'Locus ID (Tropgene)'\n",
      "Entity 'http://edamontology.org/data_1908' - Label 'Locus ID (Tropgene)'\n",
      "Entity 'http://edamontology.org/data_1916' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/data_1917' - Label 'Atomic property'\n",
      "Entity 'http://edamontology.org/data_2007' - Label 'UniProt keyword'\n",
      "Entity 'http://edamontology.org/data_2009' - Label 'Ordered locus name'\n",
      "Entity 'http://edamontology.org/data_2009' - Label 'Ordered locus name'\n",
      "Entity 'http://edamontology.org/data_2012' - Label 'Sequence coordinates'\n",
      "Entity 'http://edamontology.org/data_2012' - Label 'Sequence coordinates'\n",
      "Entity 'http://edamontology.org/data_2012' - Label 'Sequence coordinates'\n",
      "Entity 'http://edamontology.org/data_2016' - Label 'Amino acid property'\n",
      "Entity 'http://edamontology.org/data_2018' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/data_2018' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/data_2019' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2019' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2022' - Label 'Vienna RNA structural data'\n",
      "Entity 'http://edamontology.org/data_2022' - Label 'Vienna RNA structural data'\n",
      "Entity 'http://edamontology.org/data_2023' - Label 'Sequence mask parameter'\n",
      "Entity 'http://edamontology.org/data_2023' - Label 'Sequence mask parameter'\n",
      "Entity 'http://edamontology.org/data_2024' - Label 'Enzyme kinetics data'\n",
      "Entity 'http://edamontology.org/data_2024' - Label 'Enzyme kinetics data'\n",
      "Entity 'http://edamontology.org/data_2025' - Label 'Michaelis Menten plot'\n",
      "Entity 'http://edamontology.org/data_2026' - Label 'Hanes Woolf plot'\n",
      "Entity 'http://edamontology.org/data_2028' - Label 'Experimental data'\n",
      "Entity 'http://edamontology.org/data_2028' - Label 'Experimental data'\n",
      "Entity 'http://edamontology.org/data_2028' - Label 'Experimental data'\n",
      "Entity 'http://edamontology.org/data_2041' - Label 'Genome version information'\n",
      "Entity 'http://edamontology.org/data_2041' - Label 'Genome version information'\n",
      "Entity 'http://edamontology.org/data_2042' - Label 'Evidence'\n",
      "Entity 'http://edamontology.org/data_2043' - Label 'Sequence record lite'\n",
      "Entity 'http://edamontology.org/data_2043' - Label 'Sequence record lite'\n",
      "Entity 'http://edamontology.org/data_2044' - Label 'Sequence'\n",
      "Entity 'http://edamontology.org/data_2044' - Label 'Sequence'\n",
      "Entity 'http://edamontology.org/data_2046' - Label 'Nucleic acid sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2046' - Label 'Nucleic acid sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2047' - Label 'Protein sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2047' - Label 'Protein sequence record (lite)'\n",
      "Entity 'http://edamontology.org/data_2048' - Label 'Report'\n",
      "Entity 'http://edamontology.org/data_2050' - Label 'Molecular property (general)'\n",
      "Entity 'http://edamontology.org/data_2053' - Label 'Structural data'\n",
      "Entity 'http://edamontology.org/data_2053' - Label 'Structural data'\n",
      "Entity 'http://edamontology.org/data_2053' - Label 'Structural data'\n",
      "Entity 'http://edamontology.org/data_2070' - Label 'Sequence motif (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_2071' - Label 'Sequence motif (protein)'\n",
      "Entity 'http://edamontology.org/data_2079' - Label 'Search parameter'\n",
      "Entity 'http://edamontology.org/data_2079' - Label 'Search parameter'\n",
      "Entity 'http://edamontology.org/data_2080' - Label 'Database search results'\n",
      "Entity 'http://edamontology.org/data_2081' - Label 'Secondary structure'\n",
      "Entity 'http://edamontology.org/data_2081' - Label 'Secondary structure'\n",
      "Entity 'http://edamontology.org/data_2082' - Label 'Matrix'\n",
      "Entity 'http://edamontology.org/data_2083' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2083' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2084' - Label 'Nucleic acid report'\n",
      "Entity 'http://edamontology.org/data_2085' - Label 'Structure report'\n",
      "Entity 'http://edamontology.org/data_2086' - Label 'Nucleic acid structure data'\n",
      "Entity 'http://edamontology.org/data_2086' - Label 'Nucleic acid structure data'\n",
      "Entity 'http://edamontology.org/data_2086' - Label 'Nucleic acid structure data'\n",
      "Entity 'http://edamontology.org/data_2087' - Label 'Molecular property'\n",
      "Entity 'http://edamontology.org/data_2088' - Label 'DNA base structural data'\n",
      "Entity 'http://edamontology.org/data_2090' - Label 'Database entry version information'\n",
      "Entity 'http://edamontology.org/data_2090' - Label 'Database entry version information'\n",
      "Entity 'http://edamontology.org/data_2091' - Label 'Accession'\n",
      "Entity 'http://edamontology.org/data_2092' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/data_2092' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/data_2093' - Label 'Data reference'\n",
      "Entity 'http://edamontology.org/data_2098' - Label 'Job identifier'\n",
      "Entity 'http://edamontology.org/data_2099' - Label 'Name'\n",
      "Entity 'http://edamontology.org/data_2100' - Label 'Type'\n",
      "Entity 'http://edamontology.org/data_2100' - Label 'Type'\n",
      "Entity 'http://edamontology.org/data_2101' - Label 'Account authentication'\n",
      "Entity 'http://edamontology.org/data_2102' - Label 'KEGG organism code'\n",
      "Entity 'http://edamontology.org/data_2102' - Label 'KEGG organism code'\n",
      "Entity 'http://edamontology.org/data_2103' - Label 'Gene name (KEGG GENES)'\n",
      "Entity 'http://edamontology.org/data_2103' - Label 'Gene name (KEGG GENES)'\n",
      "Entity 'http://edamontology.org/data_2104' - Label 'BioCyc ID'\n",
      "Entity 'http://edamontology.org/data_2104' - Label 'BioCyc ID'\n",
      "Entity 'http://edamontology.org/data_2105' - Label 'Compound ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2105' - Label 'Compound ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2105' - Label 'Compound ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2106' - Label 'Reaction ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2106' - Label 'Reaction ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2106' - Label 'Reaction ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2107' - Label 'Enzyme ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2107' - Label 'Enzyme ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2107' - Label 'Enzyme ID (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2108' - Label 'Reaction ID'\n",
      "Entity 'http://edamontology.org/data_2108' - Label 'Reaction ID'\n",
      "Entity 'http://edamontology.org/data_2109' - Label 'Identifier (hybrid)'\n",
      "Entity 'http://edamontology.org/data_2110' - Label 'Molecular property identifier'\n",
      "Entity 'http://edamontology.org/data_2110' - Label 'Molecular property identifier'\n",
      "Entity 'http://edamontology.org/data_2111' - Label 'Codon usage table ID'\n",
      "Entity 'http://edamontology.org/data_2111' - Label 'Codon usage table ID'\n",
      "Entity 'http://edamontology.org/data_2111' - Label 'Codon usage table ID'\n",
      "Entity 'http://edamontology.org/data_2112' - Label 'FlyBase primary identifier'\n",
      "Entity 'http://edamontology.org/data_2113' - Label 'WormBase identifier'\n",
      "Entity 'http://edamontology.org/data_2114' - Label 'WormBase wormpep ID'\n",
      "Entity 'http://edamontology.org/data_2114' - Label 'WormBase wormpep ID'\n",
      "Entity 'http://edamontology.org/data_2114' - Label 'WormBase wormpep ID'\n",
      "Entity 'http://edamontology.org/data_2116' - Label 'Nucleic acid features (codon)'\n",
      "Entity 'http://edamontology.org/data_2116' - Label 'Nucleic acid features (codon)'\n",
      "Entity 'http://edamontology.org/data_2117' - Label 'Map identifier'\n",
      "Entity 'http://edamontology.org/data_2117' - Label 'Map identifier'\n",
      "Entity 'http://edamontology.org/data_2118' - Label 'Person identifier'\n",
      "Entity 'http://edamontology.org/data_2119' - Label 'Nucleic acid identifier'\n",
      "Entity 'http://edamontology.org/data_2119' - Label 'Nucleic acid identifier'\n",
      "Entity 'http://edamontology.org/data_2126' - Label 'Translation frame specification'\n",
      "Entity 'http://edamontology.org/data_2126' - Label 'Translation frame specification'\n",
      "Entity 'http://edamontology.org/data_2127' - Label 'Genetic code identifier'\n",
      "Entity 'http://edamontology.org/data_2127' - Label 'Genetic code identifier'\n",
      "Entity 'http://edamontology.org/data_2128' - Label 'Genetic code name'\n",
      "Entity 'http://edamontology.org/data_2128' - Label 'Genetic code name'\n",
      "Entity 'http://edamontology.org/data_2129' - Label 'File format name'\n",
      "Entity 'http://edamontology.org/data_2129' - Label 'File format name'\n",
      "Entity 'http://edamontology.org/data_2130' - Label 'Sequence profile type'\n",
      "Entity 'http://edamontology.org/data_2130' - Label 'Sequence profile type'\n",
      "Entity 'http://edamontology.org/data_2131' - Label 'Operating system name'\n",
      "Entity 'http://edamontology.org/data_2132' - Label 'Mutation type'\n",
      "Entity 'http://edamontology.org/data_2132' - Label 'Mutation type'\n",
      "Entity 'http://edamontology.org/data_2133' - Label 'Logical operator'\n",
      "Entity 'http://edamontology.org/data_2134' - Label 'Results sort order'\n",
      "Entity 'http://edamontology.org/data_2134' - Label 'Results sort order'\n",
      "Entity 'http://edamontology.org/data_2135' - Label 'Toggle'\n",
      "Entity 'http://edamontology.org/data_2135' - Label 'Toggle'\n",
      "Entity 'http://edamontology.org/data_2136' - Label 'Sequence width'\n",
      "Entity 'http://edamontology.org/data_2136' - Label 'Sequence width'\n",
      "Entity 'http://edamontology.org/data_2137' - Label 'Gap penalty'\n",
      "Entity 'http://edamontology.org/data_2139' - Label 'Nucleic acid melting temperature'\n",
      "Entity 'http://edamontology.org/data_2140' - Label 'Concentration'\n",
      "Entity 'http://edamontology.org/data_2141' - Label 'Window step size'\n",
      "Entity 'http://edamontology.org/data_2141' - Label 'Window step size'\n",
      "Entity 'http://edamontology.org/data_2142' - Label 'EMBOSS graph'\n",
      "Entity 'http://edamontology.org/data_2142' - Label 'EMBOSS graph'\n",
      "Entity 'http://edamontology.org/data_2143' - Label 'EMBOSS report'\n",
      "Entity 'http://edamontology.org/data_2143' - Label 'EMBOSS report'\n",
      "Entity 'http://edamontology.org/data_2145' - Label 'Sequence offset'\n",
      "Entity 'http://edamontology.org/data_2145' - Label 'Sequence offset'\n",
      "Entity 'http://edamontology.org/data_2146' - Label 'Threshold'\n",
      "Entity 'http://edamontology.org/data_2146' - Label 'Threshold'\n",
      "Entity 'http://edamontology.org/data_2147' - Label 'Protein report (transcription factor)'\n",
      "Entity 'http://edamontology.org/data_2147' - Label 'Protein report (transcription factor)'\n",
      "Entity 'http://edamontology.org/data_2149' - Label 'Database category name'\n",
      "Entity 'http://edamontology.org/data_2149' - Label 'Database category name'\n",
      "Entity 'http://edamontology.org/data_2150' - Label 'Sequence profile name'\n",
      "Entity 'http://edamontology.org/data_2150' - Label 'Sequence profile name'\n",
      "Entity 'http://edamontology.org/data_2151' - Label 'Color'\n",
      "Entity 'http://edamontology.org/data_2151' - Label 'Color'\n",
      "Entity 'http://edamontology.org/data_2152' - Label 'Rendering parameter'\n",
      "Entity 'http://edamontology.org/data_2152' - Label 'Rendering parameter'\n",
      "Entity 'http://edamontology.org/data_2154' - Label 'Sequence name'\n",
      "Entity 'http://edamontology.org/data_2154' - Label 'Sequence name'\n",
      "Entity 'http://edamontology.org/data_2156' - Label 'Date'\n",
      "Entity 'http://edamontology.org/data_2156' - Label 'Date'\n",
      "Entity 'http://edamontology.org/data_2157' - Label 'Word composition'\n",
      "Entity 'http://edamontology.org/data_2157' - Label 'Word composition'\n",
      "Entity 'http://edamontology.org/data_2157' - Label 'Word composition'\n",
      "Entity 'http://edamontology.org/data_2160' - Label 'Fickett testcode plot'\n",
      "Entity 'http://edamontology.org/data_2161' - Label 'Sequence similarity plot'\n",
      "Entity 'http://edamontology.org/data_2161' - Label 'Sequence similarity plot'\n",
      "Entity 'http://edamontology.org/data_2162' - Label 'Helical wheel'\n",
      "Entity 'http://edamontology.org/data_2163' - Label 'Helical net'\n",
      "Entity 'http://edamontology.org/data_2164' - Label 'Protein sequence properties plot'\n",
      "Entity 'http://edamontology.org/data_2164' - Label 'Protein sequence properties plot'\n",
      "Entity 'http://edamontology.org/data_2165' - Label 'Protein ionisation curve'\n",
      "Entity 'http://edamontology.org/data_2165' - Label 'Protein ionisation curve'\n",
      "Entity 'http://edamontology.org/data_2166' - Label 'Sequence composition plot'\n",
      "Entity 'http://edamontology.org/data_2166' - Label 'Sequence composition plot'\n",
      "Entity 'http://edamontology.org/data_2167' - Label 'Nucleic acid density plot'\n",
      "Entity 'http://edamontology.org/data_2167' - Label 'Nucleic acid density plot'\n",
      "Entity 'http://edamontology.org/data_2168' - Label 'Sequence trace image'\n",
      "Entity 'http://edamontology.org/data_2169' - Label 'Nucleic acid features (siRNA)'\n",
      "Entity 'http://edamontology.org/data_2169' - Label 'Nucleic acid features (siRNA)'\n",
      "Entity 'http://edamontology.org/data_2173' - Label 'Sequence set (stream)'\n",
      "Entity 'http://edamontology.org/data_2173' - Label 'Sequence set (stream)'\n",
      "Entity 'http://edamontology.org/data_2174' - Label 'FlyBase secondary identifier'\n",
      "Entity 'http://edamontology.org/data_2176' - Label 'Cardinality'\n",
      "Entity 'http://edamontology.org/data_2176' - Label 'Cardinality'\n",
      "Entity 'http://edamontology.org/data_2177' - Label 'Exactly 1'\n",
      "Entity 'http://edamontology.org/data_2177' - Label 'Exactly 1'\n",
      "Entity 'http://edamontology.org/data_2178' - Label '1 or more'\n",
      "Entity 'http://edamontology.org/data_2178' - Label '1 or more'\n",
      "Entity 'http://edamontology.org/data_2179' - Label 'Exactly 2'\n",
      "Entity 'http://edamontology.org/data_2179' - Label 'Exactly 2'\n",
      "Entity 'http://edamontology.org/data_2180' - Label '2 or more'\n",
      "Entity 'http://edamontology.org/data_2180' - Label '2 or more'\n",
      "Entity 'http://edamontology.org/data_2190' - Label 'Sequence checksum'\n",
      "Entity 'http://edamontology.org/data_2191' - Label 'Protein features report (chemical modifications)'\n",
      "Entity 'http://edamontology.org/data_2191' - Label 'Protein features report (chemical modifications)'\n",
      "Entity 'http://edamontology.org/data_2192' - Label 'Error'\n",
      "Entity 'http://edamontology.org/data_2192' - Label 'Error'\n",
      "Entity 'http://edamontology.org/data_2193' - Label 'Database entry metadata'\n",
      "Entity 'http://edamontology.org/data_2198' - Label 'Gene cluster'\n",
      "Entity 'http://edamontology.org/data_2198' - Label 'Gene cluster'\n",
      "Entity 'http://edamontology.org/data_2201' - Label 'Sequence record full'\n",
      "Entity 'http://edamontology.org/data_2201' - Label 'Sequence record full'\n",
      "Entity 'http://edamontology.org/data_2208' - Label 'Plasmid identifier'\n",
      "Entity 'http://edamontology.org/data_2209' - Label 'Mutation ID'\n",
      "Entity 'http://edamontology.org/data_2209' - Label 'Mutation ID'\n",
      "Entity 'http://edamontology.org/data_2212' - Label 'Mutation annotation (basic)'\n",
      "Entity 'http://edamontology.org/data_2212' - Label 'Mutation annotation (basic)'\n",
      "Entity 'http://edamontology.org/data_2213' - Label 'Mutation annotation (prevalence)'\n",
      "Entity 'http://edamontology.org/data_2213' - Label 'Mutation annotation (prevalence)'\n",
      "Entity 'http://edamontology.org/data_2214' - Label 'Mutation annotation (prognostic)'\n",
      "Entity 'http://edamontology.org/data_2214' - Label 'Mutation annotation (prognostic)'\n",
      "Entity 'http://edamontology.org/data_2215' - Label 'Mutation annotation (functional)'\n",
      "Entity 'http://edamontology.org/data_2215' - Label 'Mutation annotation (functional)'\n",
      "Entity 'http://edamontology.org/data_2216' - Label 'Codon number'\n",
      "Entity 'http://edamontology.org/data_2217' - Label 'Tumor annotation'\n",
      "Entity 'http://edamontology.org/data_2217' - Label 'Tumor annotation'\n",
      "Entity 'http://edamontology.org/data_2218' - Label 'Server metadata'\n",
      "Entity 'http://edamontology.org/data_2218' - Label 'Server metadata'\n",
      "Entity 'http://edamontology.org/data_2219' - Label 'Database field name'\n",
      "Entity 'http://edamontology.org/data_2220' - Label 'Sequence cluster ID (SYSTERS)'\n",
      "Entity 'http://edamontology.org/data_2220' - Label 'Sequence cluster ID (SYSTERS)'\n",
      "Entity 'http://edamontology.org/data_2223' - Label 'Ontology metadata'\n",
      "Entity 'http://edamontology.org/data_2223' - Label 'Ontology metadata'\n",
      "Entity 'http://edamontology.org/data_2235' - Label 'Raw SCOP domain classification'\n",
      "Entity 'http://edamontology.org/data_2235' - Label 'Raw SCOP domain classification'\n",
      "Entity 'http://edamontology.org/data_2236' - Label 'Raw CATH domain classification'\n",
      "Entity 'http://edamontology.org/data_2236' - Label 'Raw CATH domain classification'\n",
      "Entity 'http://edamontology.org/data_2240' - Label 'Heterogen annotation'\n",
      "Entity 'http://edamontology.org/data_2240' - Label 'Heterogen annotation'\n",
      "Entity 'http://edamontology.org/data_2242' - Label 'Phylogenetic property values'\n",
      "Entity 'http://edamontology.org/data_2242' - Label 'Phylogenetic property values'\n",
      "Entity 'http://edamontology.org/data_2245' - Label 'Sequence set (bootstrapped)'\n",
      "Entity 'http://edamontology.org/data_2245' - Label 'Sequence set (bootstrapped)'\n",
      "Entity 'http://edamontology.org/data_2247' - Label 'Phylogenetic consensus tree'\n",
      "Entity 'http://edamontology.org/data_2247' - Label 'Phylogenetic consensus tree'\n",
      "Entity 'http://edamontology.org/data_2248' - Label 'Schema'\n",
      "Entity 'http://edamontology.org/data_2248' - Label 'Schema'\n",
      "Entity 'http://edamontology.org/data_2249' - Label 'DTD'\n",
      "Entity 'http://edamontology.org/data_2249' - Label 'DTD'\n",
      "Entity 'http://edamontology.org/data_2250' - Label 'XML Schema'\n",
      "Entity 'http://edamontology.org/data_2250' - Label 'XML Schema'\n",
      "Entity 'http://edamontology.org/data_2251' - Label 'Relax-NG schema'\n",
      "Entity 'http://edamontology.org/data_2251' - Label 'Relax-NG schema'\n",
      "Entity 'http://edamontology.org/data_2252' - Label 'XSLT stylesheet'\n",
      "Entity 'http://edamontology.org/data_2252' - Label 'XSLT stylesheet'\n",
      "Entity 'http://edamontology.org/data_2253' - Label 'Data resource definition name'\n",
      "Entity 'http://edamontology.org/data_2253' - Label 'Data resource definition name'\n",
      "Entity 'http://edamontology.org/data_2254' - Label 'OBO file format name'\n",
      "Entity 'http://edamontology.org/data_2285' - Label 'Gene ID (MIPS)'\n",
      "Entity 'http://edamontology.org/data_2285' - Label 'Gene ID (MIPS)'\n",
      "Entity 'http://edamontology.org/data_2288' - Label 'Sequence identifier (protein)'\n",
      "Entity 'http://edamontology.org/data_2288' - Label 'Sequence identifier (protein)'\n",
      "Entity 'http://edamontology.org/data_2289' - Label 'Sequence identifier (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_2289' - Label 'Sequence identifier (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_2290' - Label 'EMBL accession'\n",
      "Entity 'http://edamontology.org/data_2291' - Label 'UniProt ID'\n",
      "Entity 'http://edamontology.org/data_2291' - Label 'UniProt ID'\n",
      "Entity 'http://edamontology.org/data_2292' - Label 'GenBank accession'\n",
      "Entity 'http://edamontology.org/data_2293' - Label 'Gramene secondary identifier'\n",
      "Entity 'http://edamontology.org/data_2294' - Label 'Sequence variation ID'\n",
      "Entity 'http://edamontology.org/data_2295' - Label 'Gene ID'\n",
      "Entity 'http://edamontology.org/data_2295' - Label 'Gene ID'\n",
      "Entity 'http://edamontology.org/data_2296' - Label 'Gene name (AceView)'\n",
      "Entity 'http://edamontology.org/data_2296' - Label 'Gene name (AceView)'\n",
      "Entity 'http://edamontology.org/data_2297' - Label 'Gene ID (ECK)'\n",
      "Entity 'http://edamontology.org/data_2298' - Label 'Gene ID (HGNC)'\n",
      "Entity 'http://edamontology.org/data_2298' - Label 'Gene ID (HGNC)'\n",
      "Entity 'http://edamontology.org/data_2299' - Label 'Gene name'\n",
      "Entity 'http://edamontology.org/data_2299' - Label 'Gene name'\n",
      "Entity 'http://edamontology.org/data_2300' - Label 'Gene name (NCBI)'\n",
      "Entity 'http://edamontology.org/data_2300' - Label 'Gene name (NCBI)'\n",
      "Entity 'http://edamontology.org/data_2301' - Label 'SMILES string'\n",
      "Entity 'http://edamontology.org/data_2302' - Label 'STRING ID'\n",
      "Entity 'http://edamontology.org/data_2302' - Label 'STRING ID'\n",
      "Entity 'http://edamontology.org/data_2307' - Label 'Virus annotation'\n",
      "Entity 'http://edamontology.org/data_2307' - Label 'Virus annotation'\n",
      "Entity 'http://edamontology.org/data_2308' - Label 'Virus annotation (taxonomy)'\n",
      "Entity 'http://edamontology.org/data_2308' - Label 'Virus annotation (taxonomy)'\n",
      "Entity 'http://edamontology.org/data_2309' - Label 'Reaction ID (SABIO-RK)'\n",
      "Entity 'http://edamontology.org/data_2309' - Label 'Reaction ID (SABIO-RK)'\n",
      "Entity 'http://edamontology.org/data_2313' - Label 'Carbohydrate report'\n",
      "Entity 'http://edamontology.org/data_2314' - Label 'GI number'\n",
      "Entity 'http://edamontology.org/data_2314' - Label 'GI number'\n",
      "Entity 'http://edamontology.org/data_2315' - Label 'NCBI version'\n",
      "Entity 'http://edamontology.org/data_2315' - Label 'NCBI version'\n",
      "Entity 'http://edamontology.org/data_2316' - Label 'Cell line name'\n",
      "Entity 'http://edamontology.org/data_2317' - Label 'Cell line name (exact)'\n",
      "Entity 'http://edamontology.org/data_2318' - Label 'Cell line name (truncated)'\n",
      "Entity 'http://edamontology.org/data_2319' - Label 'Cell line name (no punctuation)'\n",
      "Entity 'http://edamontology.org/data_2320' - Label 'Cell line name (assonant)'\n",
      "Entity 'http://edamontology.org/data_2321' - Label 'Enzyme ID'\n",
      "Entity 'http://edamontology.org/data_2321' - Label 'Enzyme ID'\n",
      "Entity 'http://edamontology.org/data_2325' - Label 'REBASE enzyme number'\n",
      "Entity 'http://edamontology.org/data_2325' - Label 'REBASE enzyme number'\n",
      "Entity 'http://edamontology.org/data_2326' - Label 'DrugBank ID'\n",
      "Entity 'http://edamontology.org/data_2326' - Label 'DrugBank ID'\n",
      "Entity 'http://edamontology.org/data_2327' - Label 'GI number (protein)'\n",
      "Entity 'http://edamontology.org/data_2335' - Label 'Bit score'\n",
      "Entity 'http://edamontology.org/data_2336' - Label 'Translation phase specification'\n",
      "Entity 'http://edamontology.org/data_2336' - Label 'Translation phase specification'\n",
      "Entity 'http://edamontology.org/data_2337' - Label 'Resource metadata'\n",
      "Entity 'http://edamontology.org/data_2338' - Label 'Ontology identifier'\n",
      "Entity 'http://edamontology.org/data_2338' - Label 'Ontology identifier'\n",
      "Entity 'http://edamontology.org/data_2339' - Label 'Ontology concept name'\n",
      "Entity 'http://edamontology.org/data_2339' - Label 'Ontology concept name'\n",
      "Entity 'http://edamontology.org/data_2340' - Label 'Genome build identifier'\n",
      "Entity 'http://edamontology.org/data_2342' - Label 'Pathway or network name'\n",
      "Entity 'http://edamontology.org/data_2343' - Label 'Pathway ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2343' - Label 'Pathway ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2343' - Label 'Pathway ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2344' - Label 'Pathway ID (NCI-Nature)'\n",
      "Entity 'http://edamontology.org/data_2344' - Label 'Pathway ID (NCI-Nature)'\n",
      "Entity 'http://edamontology.org/data_2345' - Label 'Pathway ID (ConsensusPathDB)'\n",
      "Entity 'http://edamontology.org/data_2345' - Label 'Pathway ID (ConsensusPathDB)'\n",
      "Entity 'http://edamontology.org/data_2345' - Label 'Pathway ID (ConsensusPathDB)'\n",
      "Entity 'http://edamontology.org/data_2346' - Label 'Sequence cluster ID (UniRef)'\n",
      "Entity 'http://edamontology.org/data_2346' - Label 'Sequence cluster ID (UniRef)'\n",
      "Entity 'http://edamontology.org/data_2347' - Label 'Sequence cluster ID (UniRef100)'\n",
      "Entity 'http://edamontology.org/data_2348' - Label 'Sequence cluster ID (UniRef90)'\n",
      "Entity 'http://edamontology.org/data_2349' - Label 'Sequence cluster ID (UniRef50)'\n",
      "Entity 'http://edamontology.org/data_2353' - Label 'Ontology data'\n",
      "Entity 'http://edamontology.org/data_2353' - Label 'Ontology data'\n",
      "Entity 'http://edamontology.org/data_2354' - Label 'RNA family report'\n",
      "Entity 'http://edamontology.org/data_2355' - Label 'RNA family identifier'\n",
      "Entity 'http://edamontology.org/data_2355' - Label 'RNA family identifier'\n",
      "Entity 'http://edamontology.org/data_2356' - Label 'RFAM accession'\n",
      "Entity 'http://edamontology.org/data_2356' - Label 'RFAM accession'\n",
      "Entity 'http://edamontology.org/data_2357' - Label 'Protein signature type'\n",
      "Entity 'http://edamontology.org/data_2357' - Label 'Protein signature type'\n",
      "Entity 'http://edamontology.org/data_2358' - Label 'Domain-nucleic acid interaction report'\n",
      "Entity 'http://edamontology.org/data_2358' - Label 'Domain-nucleic acid interaction report'\n",
      "Entity 'http://edamontology.org/data_2359' - Label 'Domain-domain interactions'\n",
      "Entity 'http://edamontology.org/data_2359' - Label 'Domain-domain interactions'\n",
      "Entity 'http://edamontology.org/data_2360' - Label 'Domain-domain interaction (indirect)'\n",
      "Entity 'http://edamontology.org/data_2360' - Label 'Domain-domain interaction (indirect)'\n",
      "Entity 'http://edamontology.org/data_2362' - Label 'Sequence accession (hybrid)'\n",
      "Entity 'http://edamontology.org/data_2362' - Label 'Sequence accession (hybrid)'\n",
      "Entity 'http://edamontology.org/data_2363' - Label '2D PAGE data'\n",
      "Entity 'http://edamontology.org/data_2363' - Label '2D PAGE data'\n",
      "Entity 'http://edamontology.org/data_2364' - Label '2D PAGE report'\n",
      "Entity 'http://edamontology.org/data_2364' - Label '2D PAGE report'\n",
      "Entity 'http://edamontology.org/data_2365' - Label 'Pathway or network accession'\n",
      "Entity 'http://edamontology.org/data_2366' - Label 'Secondary structure alignment'\n",
      "Entity 'http://edamontology.org/data_2367' - Label 'ASTD ID'\n",
      "Entity 'http://edamontology.org/data_2367' - Label 'ASTD ID'\n",
      "Entity 'http://edamontology.org/data_2367' - Label 'ASTD ID'\n",
      "Entity 'http://edamontology.org/data_2368' - Label 'ASTD ID (exon)'\n",
      "Entity 'http://edamontology.org/data_2369' - Label 'ASTD ID (intron)'\n",
      "Entity 'http://edamontology.org/data_2370' - Label 'ASTD ID (polya)'\n",
      "Entity 'http://edamontology.org/data_2371' - Label 'ASTD ID (tss)'\n",
      "Entity 'http://edamontology.org/data_2372' - Label '2D PAGE spot report'\n",
      "Entity 'http://edamontology.org/data_2372' - Label '2D PAGE spot report'\n",
      "Entity 'http://edamontology.org/data_2373' - Label 'Spot ID'\n",
      "Entity 'http://edamontology.org/data_2374' - Label 'Spot serial number'\n",
      "Entity 'http://edamontology.org/data_2374' - Label 'Spot serial number'\n",
      "Entity 'http://edamontology.org/data_2375' - Label 'Spot ID (HSC-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2375' - Label 'Spot ID (HSC-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2378' - Label 'Protein-motif interaction'\n",
      "Entity 'http://edamontology.org/data_2378' - Label 'Protein-motif interaction'\n",
      "Entity 'http://edamontology.org/data_2379' - Label 'Strain identifier'\n",
      "Entity 'http://edamontology.org/data_2380' - Label 'CABRI accession'\n",
      "Entity 'http://edamontology.org/data_2380' - Label 'CABRI accession'\n",
      "Entity 'http://edamontology.org/data_2381' - Label 'Experiment report (genotyping)'\n",
      "Entity 'http://edamontology.org/data_2381' - Label 'Experiment report (genotyping)'\n",
      "Entity 'http://edamontology.org/data_2382' - Label 'Genotype experiment ID'\n",
      "Entity 'http://edamontology.org/data_2382' - Label 'Genotype experiment ID'\n",
      "Entity 'http://edamontology.org/data_2383' - Label 'EGA accession'\n",
      "Entity 'http://edamontology.org/data_2383' - Label 'EGA accession'\n",
      "Entity 'http://edamontology.org/data_2384' - Label 'IPI protein ID'\n",
      "Entity 'http://edamontology.org/data_2384' - Label 'IPI protein ID'\n",
      "Entity 'http://edamontology.org/data_2385' - Label 'RefSeq accession (protein)'\n",
      "Entity 'http://edamontology.org/data_2386' - Label 'EPD ID'\n",
      "Entity 'http://edamontology.org/data_2386' - Label 'EPD ID'\n",
      "Entity 'http://edamontology.org/data_2387' - Label 'TAIR accession'\n",
      "Entity 'http://edamontology.org/data_2387' - Label 'TAIR accession'\n",
      "Entity 'http://edamontology.org/data_2388' - Label 'TAIR accession (At gene)'\n",
      "Entity 'http://edamontology.org/data_2389' - Label 'UniSTS accession'\n",
      "Entity 'http://edamontology.org/data_2389' - Label 'UniSTS accession'\n",
      "Entity 'http://edamontology.org/data_2390' - Label 'UNITE accession'\n",
      "Entity 'http://edamontology.org/data_2390' - Label 'UNITE accession'\n",
      "Entity 'http://edamontology.org/data_2391' - Label 'UTR accession'\n",
      "Entity 'http://edamontology.org/data_2391' - Label 'UTR accession'\n",
      "Entity 'http://edamontology.org/data_2392' - Label 'UniParc accession'\n",
      "Entity 'http://edamontology.org/data_2392' - Label 'UniParc accession'\n",
      "Entity 'http://edamontology.org/data_2393' - Label 'mFLJ/mKIAA number'\n",
      "Entity 'http://edamontology.org/data_2393' - Label 'mFLJ/mKIAA number'\n",
      "Entity 'http://edamontology.org/data_2395' - Label 'Fungi annotation'\n",
      "Entity 'http://edamontology.org/data_2395' - Label 'Fungi annotation'\n",
      "Entity 'http://edamontology.org/data_2396' - Label 'Fungi annotation (anamorph)'\n",
      "Entity 'http://edamontology.org/data_2396' - Label 'Fungi annotation (anamorph)'\n",
      "Entity 'http://edamontology.org/data_2398' - Label 'Ensembl protein ID'\n",
      "Entity 'http://edamontology.org/data_2398' - Label 'Ensembl protein ID'\n",
      "Entity 'http://edamontology.org/data_2398' - Label 'Ensembl protein ID'\n",
      "Entity 'http://edamontology.org/data_2400' - Label 'Toxin annotation'\n",
      "Entity 'http://edamontology.org/data_2400' - Label 'Toxin annotation'\n",
      "Entity 'http://edamontology.org/data_2401' - Label 'Protein report (membrane protein)'\n",
      "Entity 'http://edamontology.org/data_2401' - Label 'Protein report (membrane protein)'\n",
      "Entity 'http://edamontology.org/data_2402' - Label 'Protein-drug interaction report'\n",
      "Entity 'http://edamontology.org/data_2402' - Label 'Protein-drug interaction report'\n",
      "Entity 'http://edamontology.org/data_2522' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2522' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2522' - Label 'Map data'\n",
      "Entity 'http://edamontology.org/data_2523' - Label 'Phylogenetic data'\n",
      "Entity 'http://edamontology.org/data_2524' - Label 'Protein data'\n",
      "Entity 'http://edamontology.org/data_2524' - Label 'Protein data'\n",
      "Entity 'http://edamontology.org/data_2525' - Label 'Nucleic acid data'\n",
      "Entity 'http://edamontology.org/data_2525' - Label 'Nucleic acid data'\n",
      "Entity 'http://edamontology.org/data_2526' - Label 'Text data'\n",
      "Entity 'http://edamontology.org/data_2526' - Label 'Text data'\n",
      "Entity 'http://edamontology.org/data_2527' - Label 'Parameter'\n",
      "Entity 'http://edamontology.org/data_2527' - Label 'Parameter'\n",
      "Entity 'http://edamontology.org/data_2528' - Label 'Molecular data'\n",
      "Entity 'http://edamontology.org/data_2528' - Label 'Molecular data'\n",
      "Entity 'http://edamontology.org/data_2529' - Label 'Molecule report'\n",
      "Entity 'http://edamontology.org/data_2529' - Label 'Molecule report'\n",
      "Entity 'http://edamontology.org/data_2529' - Label 'Molecule report'\n",
      "Entity 'http://edamontology.org/data_2530' - Label 'Organism report'\n",
      "Entity 'http://edamontology.org/data_2531' - Label 'Protocol'\n",
      "Entity 'http://edamontology.org/data_2534' - Label 'Sequence attribute'\n",
      "Entity 'http://edamontology.org/data_2535' - Label 'Sequence tag profile'\n",
      "Entity 'http://edamontology.org/data_2536' - Label 'Mass spectrometry data'\n",
      "Entity 'http://edamontology.org/data_2537' - Label 'Protein structure raw data'\n",
      "Entity 'http://edamontology.org/data_2538' - Label 'Mutation identifier'\n",
      "Entity 'http://edamontology.org/data_2539' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2539' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2539' - Label 'Alignment data'\n",
      "Entity 'http://edamontology.org/data_2540' - Label 'Data index data'\n",
      "Entity 'http://edamontology.org/data_2540' - Label 'Data index data'\n",
      "Entity 'http://edamontology.org/data_2563' - Label 'Amino acid name (single letter)'\n",
      "Entity 'http://edamontology.org/data_2564' - Label 'Amino acid name (three letter)'\n",
      "Entity 'http://edamontology.org/data_2565' - Label 'Amino acid name (full name)'\n",
      "Entity 'http://edamontology.org/data_2576' - Label 'Toxin identifier'\n",
      "Entity 'http://edamontology.org/data_2576' - Label 'Toxin identifier'\n",
      "Entity 'http://edamontology.org/data_2578' - Label 'ArachnoServer ID'\n",
      "Entity 'http://edamontology.org/data_2578' - Label 'ArachnoServer ID'\n",
      "Entity 'http://edamontology.org/data_2579' - Label 'Expressed gene list'\n",
      "Entity 'http://edamontology.org/data_2579' - Label 'Expressed gene list'\n",
      "Entity 'http://edamontology.org/data_2580' - Label 'BindingDB Monomer ID'\n",
      "Entity 'http://edamontology.org/data_2580' - Label 'BindingDB Monomer ID'\n",
      "Entity 'http://edamontology.org/data_2581' - Label 'GO concept name'\n",
      "Entity 'http://edamontology.org/data_2581' - Label 'GO concept name'\n",
      "Entity 'http://edamontology.org/data_2582' - Label 'GO concept ID (biological process)'\n",
      "Entity 'http://edamontology.org/data_2583' - Label 'GO concept ID (molecular function)'\n",
      "Entity 'http://edamontology.org/data_2584' - Label 'GO concept name (cellular component)'\n",
      "Entity 'http://edamontology.org/data_2584' - Label 'GO concept name (cellular component)'\n",
      "Entity 'http://edamontology.org/data_2586' - Label 'Northern blot image'\n",
      "Entity 'http://edamontology.org/data_2587' - Label 'Blot ID'\n",
      "Entity 'http://edamontology.org/data_2588' - Label 'BlotBase blot ID'\n",
      "Entity 'http://edamontology.org/data_2588' - Label 'BlotBase blot ID'\n",
      "Entity 'http://edamontology.org/data_2589' - Label 'Hierarchy'\n",
      "Entity 'http://edamontology.org/data_2590' - Label 'Hierarchy identifier'\n",
      "Entity 'http://edamontology.org/data_2590' - Label 'Hierarchy identifier'\n",
      "Entity 'http://edamontology.org/data_2591' - Label 'Brite hierarchy ID'\n",
      "Entity 'http://edamontology.org/data_2591' - Label 'Brite hierarchy ID'\n",
      "Entity 'http://edamontology.org/data_2592' - Label 'Cancer type'\n",
      "Entity 'http://edamontology.org/data_2592' - Label 'Cancer type'\n",
      "Entity 'http://edamontology.org/data_2593' - Label 'BRENDA organism ID'\n",
      "Entity 'http://edamontology.org/data_2593' - Label 'BRENDA organism ID'\n",
      "Entity 'http://edamontology.org/data_2594' - Label 'UniGene taxon'\n",
      "Entity 'http://edamontology.org/data_2595' - Label 'UTRdb taxon'\n",
      "Entity 'http://edamontology.org/data_2596' - Label 'Catalogue ID'\n",
      "Entity 'http://edamontology.org/data_2597' - Label 'CABRI catalogue name'\n",
      "Entity 'http://edamontology.org/data_2597' - Label 'CABRI catalogue name'\n",
      "Entity 'http://edamontology.org/data_2598' - Label 'Secondary structure alignment metadata'\n",
      "Entity 'http://edamontology.org/data_2598' - Label 'Secondary structure alignment metadata'\n",
      "Entity 'http://edamontology.org/data_2599' - Label 'Molecule interaction report'\n",
      "Entity 'http://edamontology.org/data_2599' - Label 'Molecule interaction report'\n",
      "Entity 'http://edamontology.org/data_2600' - Label 'Pathway or network'\n",
      "Entity 'http://edamontology.org/data_2600' - Label 'Pathway or network'\n",
      "Entity 'http://edamontology.org/data_2601' - Label 'Small molecule data'\n",
      "Entity 'http://edamontology.org/data_2601' - Label 'Small molecule data'\n",
      "Entity 'http://edamontology.org/data_2602' - Label 'Genotype and phenotype data'\n",
      "Entity 'http://edamontology.org/data_2602' - Label 'Genotype and phenotype data'\n",
      "Entity 'http://edamontology.org/data_2603' - Label 'Expression data'\n",
      "Entity 'http://edamontology.org/data_2603' - Label 'Expression data'\n",
      "Entity 'http://edamontology.org/data_2605' - Label 'Compound ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2605' - Label 'Compound ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2606' - Label 'RFAM name'\n",
      "Entity 'http://edamontology.org/data_2606' - Label 'RFAM name'\n",
      "Entity 'http://edamontology.org/data_2608' - Label 'Reaction ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2608' - Label 'Reaction ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2609' - Label 'Drug ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2609' - Label 'Drug ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2609' - Label 'Drug ID (KEGG)'\n",
      "Entity 'http://edamontology.org/data_2610' - Label 'Ensembl ID'\n",
      "Entity 'http://edamontology.org/data_2610' - Label 'Ensembl ID'\n",
      "Entity 'http://edamontology.org/data_2611' - Label 'ICD identifier'\n",
      "Entity 'http://edamontology.org/data_2611' - Label 'ICD identifier'\n",
      "Entity 'http://edamontology.org/data_2611' - Label 'ICD identifier'\n",
      "Entity 'http://edamontology.org/data_2612' - Label 'Sequence cluster ID (CluSTr)'\n",
      "Entity 'http://edamontology.org/data_2612' - Label 'Sequence cluster ID (CluSTr)'\n",
      "Entity 'http://edamontology.org/data_2613' - Label 'KEGG Glycan ID'\n",
      "Entity 'http://edamontology.org/data_2613' - Label 'KEGG Glycan ID'\n",
      "Entity 'http://edamontology.org/data_2613' - Label 'KEGG Glycan ID'\n",
      "Entity 'http://edamontology.org/data_2614' - Label 'TCDB ID'\n",
      "Entity 'http://edamontology.org/data_2614' - Label 'TCDB ID'\n",
      "Entity 'http://edamontology.org/data_2615' - Label 'MINT ID'\n",
      "Entity 'http://edamontology.org/data_2615' - Label 'MINT ID'\n",
      "Entity 'http://edamontology.org/data_2616' - Label 'DIP ID'\n",
      "Entity 'http://edamontology.org/data_2616' - Label 'DIP ID'\n",
      "Entity 'http://edamontology.org/data_2617' - Label 'Signaling Gateway protein ID'\n",
      "Entity 'http://edamontology.org/data_2617' - Label 'Signaling Gateway protein ID'\n",
      "Entity 'http://edamontology.org/data_2618' - Label 'Protein modification ID'\n",
      "Entity 'http://edamontology.org/data_2619' - Label 'RESID ID'\n",
      "Entity 'http://edamontology.org/data_2619' - Label 'RESID ID'\n",
      "Entity 'http://edamontology.org/data_2620' - Label 'RGD ID'\n",
      "Entity 'http://edamontology.org/data_2620' - Label 'RGD ID'\n",
      "Entity 'http://edamontology.org/data_2621' - Label 'TAIR accession (protein)'\n",
      "Entity 'http://edamontology.org/data_2621' - Label 'TAIR accession (protein)'\n",
      "Entity 'http://edamontology.org/data_2622' - Label 'Compound ID (HMDB)'\n",
      "Entity 'http://edamontology.org/data_2622' - Label 'Compound ID (HMDB)'\n",
      "Entity 'http://edamontology.org/data_2625' - Label 'LIPID MAPS ID'\n",
      "Entity 'http://edamontology.org/data_2625' - Label 'LIPID MAPS ID'\n",
      "Entity 'http://edamontology.org/data_2626' - Label 'PeptideAtlas ID'\n",
      "Entity 'http://edamontology.org/data_2626' - Label 'PeptideAtlas ID'\n",
      "Entity 'http://edamontology.org/data_2627' - Label 'Molecular interaction ID'\n",
      "Entity 'http://edamontology.org/data_2627' - Label 'Molecular interaction ID'\n",
      "Entity 'http://edamontology.org/data_2628' - Label 'BioGRID interaction ID'\n",
      "Entity 'http://edamontology.org/data_2628' - Label 'BioGRID interaction ID'\n",
      "Entity 'http://edamontology.org/data_2629' - Label 'Enzyme ID (MEROPS)'\n",
      "Entity 'http://edamontology.org/data_2629' - Label 'Enzyme ID (MEROPS)'\n",
      "Entity 'http://edamontology.org/data_2630' - Label 'Mobile genetic element ID'\n",
      "Entity 'http://edamontology.org/data_2631' - Label 'ACLAME ID'\n",
      "Entity 'http://edamontology.org/data_2631' - Label 'ACLAME ID'\n",
      "Entity 'http://edamontology.org/data_2632' - Label 'SGD ID'\n",
      "Entity 'http://edamontology.org/data_2632' - Label 'SGD ID'\n",
      "Entity 'http://edamontology.org/data_2633' - Label 'Book ID'\n",
      "Entity 'http://edamontology.org/data_2634' - Label 'ISBN'\n",
      "Entity 'http://edamontology.org/data_2634' - Label 'ISBN'\n",
      "Entity 'http://edamontology.org/data_2635' - Label 'Compound ID (3DMET)'\n",
      "Entity 'http://edamontology.org/data_2635' - Label 'Compound ID (3DMET)'\n",
      "Entity 'http://edamontology.org/data_2636' - Label 'MatrixDB interaction ID'\n",
      "Entity 'http://edamontology.org/data_2636' - Label 'MatrixDB interaction ID'\n",
      "Entity 'http://edamontology.org/data_2637' - Label 'cPath ID'\n",
      "Entity 'http://edamontology.org/data_2637' - Label 'cPath ID'\n",
      "Entity 'http://edamontology.org/data_2637' - Label 'cPath ID'\n",
      "Entity 'http://edamontology.org/data_2638' - Label 'PubChem bioassay ID'\n",
      "Entity 'http://edamontology.org/data_2638' - Label 'PubChem bioassay ID'\n",
      "Entity 'http://edamontology.org/data_2639' - Label 'PubChem ID'\n",
      "Entity 'http://edamontology.org/data_2639' - Label 'PubChem ID'\n",
      "Entity 'http://edamontology.org/data_2641' - Label 'Reaction ID (MACie)'\n",
      "Entity 'http://edamontology.org/data_2641' - Label 'Reaction ID (MACie)'\n",
      "Entity 'http://edamontology.org/data_2642' - Label 'Gene ID (miRBase)'\n",
      "Entity 'http://edamontology.org/data_2642' - Label 'Gene ID (miRBase)'\n",
      "Entity 'http://edamontology.org/data_2643' - Label 'Gene ID (ZFIN)'\n",
      "Entity 'http://edamontology.org/data_2643' - Label 'Gene ID (ZFIN)'\n",
      "Entity 'http://edamontology.org/data_2644' - Label 'Reaction ID (Rhea)'\n",
      "Entity 'http://edamontology.org/data_2644' - Label 'Reaction ID (Rhea)'\n",
      "Entity 'http://edamontology.org/data_2645' - Label 'Pathway ID (Unipathway)'\n",
      "Entity 'http://edamontology.org/data_2645' - Label 'Pathway ID (Unipathway)'\n",
      "Entity 'http://edamontology.org/data_2646' - Label 'Compound ID (ChEMBL)'\n",
      "Entity 'http://edamontology.org/data_2646' - Label 'Compound ID (ChEMBL)'\n",
      "Entity 'http://edamontology.org/data_2647' - Label 'LGICdb identifier'\n",
      "Entity 'http://edamontology.org/data_2647' - Label 'LGICdb identifier'\n",
      "Entity 'http://edamontology.org/data_2648' - Label 'Reaction kinetics ID (SABIO-RK)'\n",
      "Entity 'http://edamontology.org/data_2649' - Label 'PharmGKB ID'\n",
      "Entity 'http://edamontology.org/data_2649' - Label 'PharmGKB ID'\n",
      "Entity 'http://edamontology.org/data_2650' - Label 'Pathway ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2650' - Label 'Pathway ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2650' - Label 'Pathway ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2651' - Label 'Disease ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2651' - Label 'Disease ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2651' - Label 'Disease ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2652' - Label 'Drug ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2652' - Label 'Drug ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2652' - Label 'Drug ID (PharmGKB)'\n",
      "Entity 'http://edamontology.org/data_2653' - Label 'Drug ID (TTD)'\n",
      "Entity 'http://edamontology.org/data_2653' - Label 'Drug ID (TTD)'\n",
      "Entity 'http://edamontology.org/data_2654' - Label 'Target ID (TTD)'\n",
      "Entity 'http://edamontology.org/data_2654' - Label 'Target ID (TTD)'\n",
      "Entity 'http://edamontology.org/data_2655' - Label 'Cell type identifier'\n",
      "Entity 'http://edamontology.org/data_2656' - Label 'NeuronDB ID'\n",
      "Entity 'http://edamontology.org/data_2656' - Label 'NeuronDB ID'\n",
      "Entity 'http://edamontology.org/data_2657' - Label 'NeuroMorpho ID'\n",
      "Entity 'http://edamontology.org/data_2657' - Label 'NeuroMorpho ID'\n",
      "Entity 'http://edamontology.org/data_2658' - Label 'Compound ID (ChemIDplus)'\n",
      "Entity 'http://edamontology.org/data_2658' - Label 'Compound ID (ChemIDplus)'\n",
      "Entity 'http://edamontology.org/data_2659' - Label 'Pathway ID (SMPDB)'\n",
      "Entity 'http://edamontology.org/data_2659' - Label 'Pathway ID (SMPDB)'\n",
      "Entity 'http://edamontology.org/data_2660' - Label 'BioNumbers ID'\n",
      "Entity 'http://edamontology.org/data_2662' - Label 'T3DB ID'\n",
      "Entity 'http://edamontology.org/data_2662' - Label 'T3DB ID'\n",
      "Entity 'http://edamontology.org/data_2663' - Label 'Carbohydrate identifier'\n",
      "Entity 'http://edamontology.org/data_2663' - Label 'Carbohydrate identifier'\n",
      "Entity 'http://edamontology.org/data_2663' - Label 'Carbohydrate identifier'\n",
      "Entity 'http://edamontology.org/data_2664' - Label 'GlycomeDB ID'\n",
      "Entity 'http://edamontology.org/data_2664' - Label 'GlycomeDB ID'\n",
      "Entity 'http://edamontology.org/data_2665' - Label 'LipidBank ID'\n",
      "Entity 'http://edamontology.org/data_2665' - Label 'LipidBank ID'\n",
      "Entity 'http://edamontology.org/data_2666' - Label 'CDD ID'\n",
      "Entity 'http://edamontology.org/data_2666' - Label 'CDD ID'\n",
      "Entity 'http://edamontology.org/data_2667' - Label 'MMDB ID'\n",
      "Entity 'http://edamontology.org/data_2667' - Label 'MMDB ID'\n",
      "Entity 'http://edamontology.org/data_2668' - Label 'iRefIndex ID'\n",
      "Entity 'http://edamontology.org/data_2668' - Label 'iRefIndex ID'\n",
      "Entity 'http://edamontology.org/data_2669' - Label 'ModelDB ID'\n",
      "Entity 'http://edamontology.org/data_2669' - Label 'ModelDB ID'\n",
      "Entity 'http://edamontology.org/data_2670' - Label 'Pathway ID (DQCS)'\n",
      "Entity 'http://edamontology.org/data_2670' - Label 'Pathway ID (DQCS)'\n",
      "Entity 'http://edamontology.org/data_2671' - Label 'Ensembl ID (Homo sapiens)'\n",
      "Entity 'http://edamontology.org/data_2671' - Label 'Ensembl ID (Homo sapiens)'\n",
      "Entity 'http://edamontology.org/data_2672' - Label 'Ensembl ID ('Bos taurus')'\n",
      "Entity 'http://edamontology.org/data_2672' - Label 'Ensembl ID ('Bos taurus')'\n",
      "Entity 'http://edamontology.org/data_2673' - Label 'Ensembl ID ('Canis familiaris')'\n",
      "Entity 'http://edamontology.org/data_2673' - Label 'Ensembl ID ('Canis familiaris')'\n",
      "Entity 'http://edamontology.org/data_2674' - Label 'Ensembl ID ('Cavia porcellus')'\n",
      "Entity 'http://edamontology.org/data_2674' - Label 'Ensembl ID ('Cavia porcellus')'\n",
      "Entity 'http://edamontology.org/data_2675' - Label 'Ensembl ID ('Ciona intestinalis')'\n",
      "Entity 'http://edamontology.org/data_2675' - Label 'Ensembl ID ('Ciona intestinalis')'\n",
      "Entity 'http://edamontology.org/data_2676' - Label 'Ensembl ID ('Ciona savignyi')'\n",
      "Entity 'http://edamontology.org/data_2676' - Label 'Ensembl ID ('Ciona savignyi')'\n",
      "Entity 'http://edamontology.org/data_2677' - Label 'Ensembl ID ('Danio rerio')'\n",
      "Entity 'http://edamontology.org/data_2677' - Label 'Ensembl ID ('Danio rerio')'\n",
      "Entity 'http://edamontology.org/data_2678' - Label 'Ensembl ID ('Dasypus novemcinctus')'\n",
      "Entity 'http://edamontology.org/data_2678' - Label 'Ensembl ID ('Dasypus novemcinctus')'\n",
      "Entity 'http://edamontology.org/data_2679' - Label 'Ensembl ID ('Echinops telfairi')'\n",
      "Entity 'http://edamontology.org/data_2679' - Label 'Ensembl ID ('Echinops telfairi')'\n",
      "Entity 'http://edamontology.org/data_2680' - Label 'Ensembl ID ('Erinaceus europaeus')'\n",
      "Entity 'http://edamontology.org/data_2680' - Label 'Ensembl ID ('Erinaceus europaeus')'\n",
      "Entity 'http://edamontology.org/data_2681' - Label 'Ensembl ID ('Felis catus')'\n",
      "Entity 'http://edamontology.org/data_2681' - Label 'Ensembl ID ('Felis catus')'\n",
      "Entity 'http://edamontology.org/data_2682' - Label 'Ensembl ID ('Gallus gallus')'\n",
      "Entity 'http://edamontology.org/data_2682' - Label 'Ensembl ID ('Gallus gallus')'\n",
      "Entity 'http://edamontology.org/data_2683' - Label 'Ensembl ID ('Gasterosteus aculeatus')'\n",
      "Entity 'http://edamontology.org/data_2683' - Label 'Ensembl ID ('Gasterosteus aculeatus')'\n",
      "Entity 'http://edamontology.org/data_2684' - Label 'Ensembl ID ('Homo sapiens')'\n",
      "Entity 'http://edamontology.org/data_2684' - Label 'Ensembl ID ('Homo sapiens')'\n",
      "Entity 'http://edamontology.org/data_2685' - Label 'Ensembl ID ('Loxodonta africana')'\n",
      "Entity 'http://edamontology.org/data_2685' - Label 'Ensembl ID ('Loxodonta africana')'\n",
      "Entity 'http://edamontology.org/data_2686' - Label 'Ensembl ID ('Macaca mulatta')'\n",
      "Entity 'http://edamontology.org/data_2686' - Label 'Ensembl ID ('Macaca mulatta')'\n",
      "Entity 'http://edamontology.org/data_2687' - Label 'Ensembl ID ('Monodelphis domestica')'\n",
      "Entity 'http://edamontology.org/data_2687' - Label 'Ensembl ID ('Monodelphis domestica')'\n",
      "Entity 'http://edamontology.org/data_2688' - Label 'Ensembl ID ('Mus musculus')'\n",
      "Entity 'http://edamontology.org/data_2688' - Label 'Ensembl ID ('Mus musculus')'\n",
      "Entity 'http://edamontology.org/data_2689' - Label 'Ensembl ID ('Myotis lucifugus')'\n",
      "Entity 'http://edamontology.org/data_2689' - Label 'Ensembl ID ('Myotis lucifugus')'\n",
      "Entity 'http://edamontology.org/data_2690' - Label 'Ensembl ID (\"Ornithorhynchus anatinus\")'\n",
      "Entity 'http://edamontology.org/data_2690' - Label 'Ensembl ID (\"Ornithorhynchus anatinus\")'\n",
      "Entity 'http://edamontology.org/data_2691' - Label 'Ensembl ID ('Oryctolagus cuniculus')'\n",
      "Entity 'http://edamontology.org/data_2691' - Label 'Ensembl ID ('Oryctolagus cuniculus')'\n",
      "Entity 'http://edamontology.org/data_2692' - Label 'Ensembl ID ('Oryzias latipes')'\n",
      "Entity 'http://edamontology.org/data_2692' - Label 'Ensembl ID ('Oryzias latipes')'\n",
      "Entity 'http://edamontology.org/data_2693' - Label 'Ensembl ID ('Otolemur garnettii')'\n",
      "Entity 'http://edamontology.org/data_2693' - Label 'Ensembl ID ('Otolemur garnettii')'\n",
      "Entity 'http://edamontology.org/data_2694' - Label 'Ensembl ID ('Pan troglodytes')'\n",
      "Entity 'http://edamontology.org/data_2694' - Label 'Ensembl ID ('Pan troglodytes')'\n",
      "Entity 'http://edamontology.org/data_2695' - Label 'Ensembl ID ('Rattus norvegicus')'\n",
      "Entity 'http://edamontology.org/data_2695' - Label 'Ensembl ID ('Rattus norvegicus')'\n",
      "Entity 'http://edamontology.org/data_2696' - Label 'Ensembl ID ('Spermophilus tridecemlineatus')'\n",
      "Entity 'http://edamontology.org/data_2696' - Label 'Ensembl ID ('Spermophilus tridecemlineatus')'\n",
      "Entity 'http://edamontology.org/data_2697' - Label 'Ensembl ID ('Takifugu rubripes')'\n",
      "Entity 'http://edamontology.org/data_2697' - Label 'Ensembl ID ('Takifugu rubripes')'\n",
      "Entity 'http://edamontology.org/data_2698' - Label 'Ensembl ID ('Tupaia belangeri')'\n",
      "Entity 'http://edamontology.org/data_2698' - Label 'Ensembl ID ('Tupaia belangeri')'\n",
      "Entity 'http://edamontology.org/data_2699' - Label 'Ensembl ID ('Xenopus tropicalis')'\n",
      "Entity 'http://edamontology.org/data_2699' - Label 'Ensembl ID ('Xenopus tropicalis')'\n",
      "Entity 'http://edamontology.org/data_2700' - Label 'CATH identifier'\n",
      "Entity 'http://edamontology.org/data_2700' - Label 'CATH identifier'\n",
      "Entity 'http://edamontology.org/data_2701' - Label 'CATH node ID (family)'\n",
      "Entity 'http://edamontology.org/data_2702' - Label 'Enzyme ID (CAZy)'\n",
      "Entity 'http://edamontology.org/data_2702' - Label 'Enzyme ID (CAZy)'\n",
      "Entity 'http://edamontology.org/data_2704' - Label 'Clone ID (IMAGE)'\n",
      "Entity 'http://edamontology.org/data_2704' - Label 'Clone ID (IMAGE)'\n",
      "Entity 'http://edamontology.org/data_2705' - Label 'GO concept ID (cellular component)'\n",
      "Entity 'http://edamontology.org/data_2706' - Label 'Chromosome name (BioCyc)'\n",
      "Entity 'http://edamontology.org/data_2709' - Label 'CleanEx entry name'\n",
      "Entity 'http://edamontology.org/data_2709' - Label 'CleanEx entry name'\n",
      "Entity 'http://edamontology.org/data_2710' - Label 'CleanEx dataset code'\n",
      "Entity 'http://edamontology.org/data_2711' - Label 'Genome report'\n",
      "Entity 'http://edamontology.org/data_2713' - Label 'Protein ID (CORUM)'\n",
      "Entity 'http://edamontology.org/data_2713' - Label 'Protein ID (CORUM)'\n",
      "Entity 'http://edamontology.org/data_2714' - Label 'CDD PSSM-ID'\n",
      "Entity 'http://edamontology.org/data_2714' - Label 'CDD PSSM-ID'\n",
      "Entity 'http://edamontology.org/data_2715' - Label 'Protein ID (CuticleDB)'\n",
      "Entity 'http://edamontology.org/data_2715' - Label 'Protein ID (CuticleDB)'\n",
      "Entity 'http://edamontology.org/data_2716' - Label 'DBD ID'\n",
      "Entity 'http://edamontology.org/data_2716' - Label 'DBD ID'\n",
      "Entity 'http://edamontology.org/data_2717' - Label 'Oligonucleotide probe annotation'\n",
      "Entity 'http://edamontology.org/data_2717' - Label 'Oligonucleotide probe annotation'\n",
      "Entity 'http://edamontology.org/data_2718' - Label 'Oligonucleotide ID'\n",
      "Entity 'http://edamontology.org/data_2718' - Label 'Oligonucleotide ID'\n",
      "Entity 'http://edamontology.org/data_2719' - Label 'dbProbe ID'\n",
      "Entity 'http://edamontology.org/data_2719' - Label 'dbProbe ID'\n",
      "Entity 'http://edamontology.org/data_2720' - Label 'Dinucleotide property'\n",
      "Entity 'http://edamontology.org/data_2721' - Label 'DiProDB ID'\n",
      "Entity 'http://edamontology.org/data_2721' - Label 'DiProDB ID'\n",
      "Entity 'http://edamontology.org/data_2722' - Label 'Protein features report (disordered structure)'\n",
      "Entity 'http://edamontology.org/data_2722' - Label 'Protein features report (disordered structure)'\n",
      "Entity 'http://edamontology.org/data_2723' - Label 'Protein ID (DisProt)'\n",
      "Entity 'http://edamontology.org/data_2723' - Label 'Protein ID (DisProt)'\n",
      "Entity 'http://edamontology.org/data_2724' - Label 'Embryo report'\n",
      "Entity 'http://edamontology.org/data_2724' - Label 'Embryo report'\n",
      "Entity 'http://edamontology.org/data_2725' - Label 'Ensembl transcript ID'\n",
      "Entity 'http://edamontology.org/data_2725' - Label 'Ensembl transcript ID'\n",
      "Entity 'http://edamontology.org/data_2725' - Label 'Ensembl transcript ID'\n",
      "Entity 'http://edamontology.org/data_2726' - Label 'Inhibitor annotation'\n",
      "Entity 'http://edamontology.org/data_2726' - Label 'Inhibitor annotation'\n",
      "Entity 'http://edamontology.org/data_2727' - Label 'Promoter ID'\n",
      "Entity 'http://edamontology.org/data_2728' - Label 'EST accession'\n",
      "Entity 'http://edamontology.org/data_2729' - Label 'COGEME EST ID'\n",
      "Entity 'http://edamontology.org/data_2729' - Label 'COGEME EST ID'\n",
      "Entity 'http://edamontology.org/data_2730' - Label 'COGEME unisequence ID'\n",
      "Entity 'http://edamontology.org/data_2730' - Label 'COGEME unisequence ID'\n",
      "Entity 'http://edamontology.org/data_2731' - Label 'Protein family ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_2731' - Label 'Protein family ID (GeneFarm)'\n",
      "Entity 'http://edamontology.org/data_2732' - Label 'Family name'\n",
      "Entity 'http://edamontology.org/data_2733' - Label 'Genus name (virus)'\n",
      "Entity 'http://edamontology.org/data_2733' - Label 'Genus name (virus)'\n",
      "Entity 'http://edamontology.org/data_2734' - Label 'Family name (virus)'\n",
      "Entity 'http://edamontology.org/data_2734' - Label 'Family name (virus)'\n",
      "Entity 'http://edamontology.org/data_2735' - Label 'Database name (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2735' - Label 'Database name (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2736' - Label 'Sequence feature ID (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2736' - Label 'Sequence feature ID (SwissRegulon)'\n",
      "Entity 'http://edamontology.org/data_2737' - Label 'FIG ID'\n",
      "Entity 'http://edamontology.org/data_2737' - Label 'FIG ID'\n",
      "Entity 'http://edamontology.org/data_2738' - Label 'Gene ID (Xenbase)'\n",
      "Entity 'http://edamontology.org/data_2738' - Label 'Gene ID (Xenbase)'\n",
      "Entity 'http://edamontology.org/data_2739' - Label 'Gene ID (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2739' - Label 'Gene ID (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2740' - Label 'Gene name (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2740' - Label 'Gene name (Genolist)'\n",
      "Entity 'http://edamontology.org/data_2741' - Label 'ABS ID'\n",
      "Entity 'http://edamontology.org/data_2741' - Label 'ABS ID'\n",
      "Entity 'http://edamontology.org/data_2742' - Label 'AraC-XylS ID'\n",
      "Entity 'http://edamontology.org/data_2742' - Label 'AraC-XylS ID'\n",
      "Entity 'http://edamontology.org/data_2743' - Label 'Gene name (HUGO)'\n",
      "Entity 'http://edamontology.org/data_2743' - Label 'Gene name (HUGO)'\n",
      "Entity 'http://edamontology.org/data_2744' - Label 'Locus ID (PseudoCAP)'\n",
      "Entity 'http://edamontology.org/data_2744' - Label 'Locus ID (PseudoCAP)'\n",
      "Entity 'http://edamontology.org/data_2745' - Label 'Locus ID (UTR)'\n",
      "Entity 'http://edamontology.org/data_2745' - Label 'Locus ID (UTR)'\n",
      "Entity 'http://edamontology.org/data_2746' - Label 'MonosaccharideDB ID'\n",
      "Entity 'http://edamontology.org/data_2746' - Label 'MonosaccharideDB ID'\n",
      "Entity 'http://edamontology.org/data_2747' - Label 'Database name (CMD)'\n",
      "Entity 'http://edamontology.org/data_2747' - Label 'Database name (CMD)'\n",
      "Entity 'http://edamontology.org/data_2748' - Label 'Database name (Osteogenesis)'\n",
      "Entity 'http://edamontology.org/data_2748' - Label 'Database name (Osteogenesis)'\n",
      "Entity 'http://edamontology.org/data_2749' - Label 'Genome identifier'\n",
      "Entity 'http://edamontology.org/data_2751' - Label 'GenomeReviews ID'\n",
      "Entity 'http://edamontology.org/data_2751' - Label 'GenomeReviews ID'\n",
      "Entity 'http://edamontology.org/data_2752' - Label 'GlycoMap ID'\n",
      "Entity 'http://edamontology.org/data_2752' - Label 'GlycoMap ID'\n",
      "Entity 'http://edamontology.org/data_2753' - Label 'Carbohydrate conformational map'\n",
      "Entity 'http://edamontology.org/data_2755' - Label 'Transcription factor name'\n",
      "Entity 'http://edamontology.org/data_2755' - Label 'Transcription factor name'\n",
      "Entity 'http://edamontology.org/data_2756' - Label 'TCID'\n",
      "Entity 'http://edamontology.org/data_2756' - Label 'TCID'\n",
      "Entity 'http://edamontology.org/data_2757' - Label 'Pfam domain name'\n",
      "Entity 'http://edamontology.org/data_2758' - Label 'Pfam clan ID'\n",
      "Entity 'http://edamontology.org/data_2758' - Label 'Pfam clan ID'\n",
      "Entity 'http://edamontology.org/data_2759' - Label 'Gene ID (VectorBase)'\n",
      "Entity 'http://edamontology.org/data_2759' - Label 'Gene ID (VectorBase)'\n",
      "Entity 'http://edamontology.org/data_2761' - Label 'UTRSite ID'\n",
      "Entity 'http://edamontology.org/data_2762' - Label 'Sequence signature report'\n",
      "Entity 'http://edamontology.org/data_2762' - Label 'Sequence signature report'\n",
      "Entity 'http://edamontology.org/data_2763' - Label 'Locus annotation'\n",
      "Entity 'http://edamontology.org/data_2763' - Label 'Locus annotation'\n",
      "Entity 'http://edamontology.org/data_2764' - Label 'Protein name (UniProt)'\n",
      "Entity 'http://edamontology.org/data_2765' - Label 'Term ID list'\n",
      "Entity 'http://edamontology.org/data_2765' - Label 'Term ID list'\n",
      "Entity 'http://edamontology.org/data_2766' - Label 'HAMAP ID'\n",
      "Entity 'http://edamontology.org/data_2766' - Label 'HAMAP ID'\n",
      "Entity 'http://edamontology.org/data_2767' - Label 'Identifier with metadata'\n",
      "Entity 'http://edamontology.org/data_2767' - Label 'Identifier with metadata'\n",
      "Entity 'http://edamontology.org/data_2768' - Label 'Gene symbol annotation'\n",
      "Entity 'http://edamontology.org/data_2768' - Label 'Gene symbol annotation'\n",
      "Entity 'http://edamontology.org/data_2769' - Label 'Transcript ID'\n",
      "Entity 'http://edamontology.org/data_2769' - Label 'Transcript ID'\n",
      "Entity 'http://edamontology.org/data_2769' - Label 'Transcript ID'\n",
      "Entity 'http://edamontology.org/data_2770' - Label 'HIT ID'\n",
      "Entity 'http://edamontology.org/data_2770' - Label 'HIT ID'\n",
      "Entity 'http://edamontology.org/data_2771' - Label 'HIX ID'\n",
      "Entity 'http://edamontology.org/data_2771' - Label 'HIX ID'\n",
      "Entity 'http://edamontology.org/data_2772' - Label 'HPA antibody id'\n",
      "Entity 'http://edamontology.org/data_2772' - Label 'HPA antibody id'\n",
      "Entity 'http://edamontology.org/data_2773' - Label 'IMGT/HLA ID'\n",
      "Entity 'http://edamontology.org/data_2773' - Label 'IMGT/HLA ID'\n",
      "Entity 'http://edamontology.org/data_2774' - Label 'Gene ID (JCVI)'\n",
      "Entity 'http://edamontology.org/data_2774' - Label 'Gene ID (JCVI)'\n",
      "Entity 'http://edamontology.org/data_2775' - Label 'Kinase name'\n",
      "Entity 'http://edamontology.org/data_2776' - Label 'ConsensusPathDB entity ID'\n",
      "Entity 'http://edamontology.org/data_2776' - Label 'ConsensusPathDB entity ID'\n",
      "Entity 'http://edamontology.org/data_2777' - Label 'ConsensusPathDB entity name'\n",
      "Entity 'http://edamontology.org/data_2777' - Label 'ConsensusPathDB entity name'\n",
      "Entity 'http://edamontology.org/data_2778' - Label 'CCAP strain number'\n",
      "Entity 'http://edamontology.org/data_2778' - Label 'CCAP strain number'\n",
      "Entity 'http://edamontology.org/data_2779' - Label 'Stock number'\n",
      "Entity 'http://edamontology.org/data_2780' - Label 'Stock number (TAIR)'\n",
      "Entity 'http://edamontology.org/data_2780' - Label 'Stock number (TAIR)'\n",
      "Entity 'http://edamontology.org/data_2781' - Label 'REDIdb ID'\n",
      "Entity 'http://edamontology.org/data_2781' - Label 'REDIdb ID'\n",
      "Entity 'http://edamontology.org/data_2782' - Label 'SMART domain name'\n",
      "Entity 'http://edamontology.org/data_2783' - Label 'Protein family ID (PANTHER)'\n",
      "Entity 'http://edamontology.org/data_2783' - Label 'Protein family ID (PANTHER)'\n",
      "Entity 'http://edamontology.org/data_2784' - Label 'RNAVirusDB ID'\n",
      "Entity 'http://edamontology.org/data_2784' - Label 'RNAVirusDB ID'\n",
      "Entity 'http://edamontology.org/data_2785' - Label 'Virus identifier'\n",
      "Entity 'http://edamontology.org/data_2786' - Label 'NCBI Genome Project ID'\n",
      "Entity 'http://edamontology.org/data_2786' - Label 'NCBI Genome Project ID'\n",
      "Entity 'http://edamontology.org/data_2787' - Label 'NCBI genome accession'\n",
      "Entity 'http://edamontology.org/data_2787' - Label 'NCBI genome accession'\n",
      "Entity 'http://edamontology.org/data_2788' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2788' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2789' - Label 'Protein ID (TopDB)'\n",
      "Entity 'http://edamontology.org/data_2789' - Label 'Protein ID (TopDB)'\n",
      "Entity 'http://edamontology.org/data_2790' - Label 'Gel ID'\n",
      "Entity 'http://edamontology.org/data_2791' - Label 'Reference map name (SWISS-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2791' - Label 'Reference map name (SWISS-2DPAGE)'\n",
      "Entity 'http://edamontology.org/data_2792' - Label 'Protein ID (PeroxiBase)'\n",
      "Entity 'http://edamontology.org/data_2792' - Label 'Protein ID (PeroxiBase)'\n",
      "Entity 'http://edamontology.org/data_2793' - Label 'SISYPHUS ID'\n",
      "Entity 'http://edamontology.org/data_2793' - Label 'SISYPHUS ID'\n",
      "Entity 'http://edamontology.org/data_2794' - Label 'ORF ID'\n",
      "Entity 'http://edamontology.org/data_2794' - Label 'ORF ID'\n",
      "Entity 'http://edamontology.org/data_2794' - Label 'ORF ID'\n",
      "Entity 'http://edamontology.org/data_2795' - Label 'ORF identifier'\n",
      "Entity 'http://edamontology.org/data_2796' - Label 'LINUCS ID'\n",
      "Entity 'http://edamontology.org/data_2796' - Label 'LINUCS ID'\n",
      "Entity 'http://edamontology.org/data_2797' - Label 'Protein ID (LGICdb)'\n",
      "Entity 'http://edamontology.org/data_2797' - Label 'Protein ID (LGICdb)'\n",
      "Entity 'http://edamontology.org/data_2798' - Label 'MaizeDB ID'\n",
      "Entity 'http://edamontology.org/data_2798' - Label 'MaizeDB ID'\n",
      "Entity 'http://edamontology.org/data_2799' - Label 'Gene ID (MfunGD)'\n",
      "Entity 'http://edamontology.org/data_2799' - Label 'Gene ID (MfunGD)'\n",
      "Entity 'http://edamontology.org/data_2800' - Label 'Orpha number'\n",
      "Entity 'http://edamontology.org/data_2800' - Label 'Orpha number'\n",
      "Entity 'http://edamontology.org/data_2800' - Label 'Orpha number'\n",
      "Entity 'http://edamontology.org/data_2802' - Label 'Protein ID (EcID)'\n",
      "Entity 'http://edamontology.org/data_2802' - Label 'Protein ID (EcID)'\n",
      "Entity 'http://edamontology.org/data_2803' - Label 'Clone ID (RefSeq)'\n",
      "Entity 'http://edamontology.org/data_2803' - Label 'Clone ID (RefSeq)'\n",
      "Entity 'http://edamontology.org/data_2804' - Label 'Protein ID (ConoServer)'\n",
      "Entity 'http://edamontology.org/data_2804' - Label 'Protein ID (ConoServer)'\n",
      "Entity 'http://edamontology.org/data_2805' - Label 'GeneSNP ID'\n",
      "Entity 'http://edamontology.org/data_2805' - Label 'GeneSNP ID'\n",
      "Entity 'http://edamontology.org/data_2812' - Label 'Lipid identifier'\n",
      "Entity 'http://edamontology.org/data_2812' - Label 'Lipid identifier'\n",
      "Entity 'http://edamontology.org/data_2812' - Label 'Lipid identifier'\n",
      "Entity 'http://edamontology.org/data_2831' - Label 'Databank'\n",
      "Entity 'http://edamontology.org/data_2831' - Label 'Databank'\n",
      "Entity 'http://edamontology.org/data_2832' - Label 'Web portal'\n",
      "Entity 'http://edamontology.org/data_2832' - Label 'Web portal'\n",
      "Entity 'http://edamontology.org/data_2835' - Label 'Gene ID (VBASE2)'\n",
      "Entity 'http://edamontology.org/data_2835' - Label 'Gene ID (VBASE2)'\n",
      "Entity 'http://edamontology.org/data_2836' - Label 'DPVweb ID'\n",
      "Entity 'http://edamontology.org/data_2836' - Label 'DPVweb ID'\n",
      "Entity 'http://edamontology.org/data_2837' - Label 'Pathway ID (BioSystems)'\n",
      "Entity 'http://edamontology.org/data_2837' - Label 'Pathway ID (BioSystems)'\n",
      "Entity 'http://edamontology.org/data_2838' - Label 'Experimental data (proteomics)'\n",
      "Entity 'http://edamontology.org/data_2838' - Label 'Experimental data (proteomics)'\n",
      "Entity 'http://edamontology.org/data_2849' - Label 'Abstract'\n",
      "Entity 'http://edamontology.org/data_2850' - Label 'Lipid structure'\n",
      "Entity 'http://edamontology.org/data_2851' - Label 'Drug structure'\n",
      "Entity 'http://edamontology.org/data_2852' - Label 'Toxin structure'\n",
      "Entity 'http://edamontology.org/data_2854' - Label 'Position-specific scoring matrix'\n",
      "Entity 'http://edamontology.org/data_2854' - Label 'Position-specific scoring matrix'\n",
      "Entity 'http://edamontology.org/data_2855' - Label 'Distance matrix'\n",
      "Entity 'http://edamontology.org/data_2856' - Label 'Structural distance matrix'\n",
      "Entity 'http://edamontology.org/data_2857' - Label 'Article metadata'\n",
      "Entity 'http://edamontology.org/data_2857' - Label 'Article metadata'\n",
      "Entity 'http://edamontology.org/data_2858' - Label 'Ontology concept'\n",
      "Entity 'http://edamontology.org/data_2865' - Label 'Codon usage bias'\n",
      "Entity 'http://edamontology.org/data_2866' - Label 'Northern blot report'\n",
      "Entity 'http://edamontology.org/data_2866' - Label 'Northern blot report'\n",
      "Entity 'http://edamontology.org/data_2870' - Label 'Radiation hybrid map'\n",
      "Entity 'http://edamontology.org/data_2872' - Label 'ID list'\n",
      "Entity 'http://edamontology.org/data_2873' - Label 'Phylogenetic gene frequencies data'\n",
      "Entity 'http://edamontology.org/data_2874' - Label 'Sequence set (polymorphic)'\n",
      "Entity 'http://edamontology.org/data_2874' - Label 'Sequence set (polymorphic)'\n",
      "Entity 'http://edamontology.org/data_2875' - Label 'DRCAT resource'\n",
      "Entity 'http://edamontology.org/data_2875' - Label 'DRCAT resource'\n",
      "Entity 'http://edamontology.org/data_2877' - Label 'Protein complex'\n",
      "Entity 'http://edamontology.org/data_2878' - Label 'Protein structural motif'\n",
      "Entity 'http://edamontology.org/data_2879' - Label 'Lipid report'\n",
      "Entity 'http://edamontology.org/data_2880' - Label 'Secondary structure image'\n",
      "Entity 'http://edamontology.org/data_2880' - Label 'Secondary structure image'\n",
      "Entity 'http://edamontology.org/data_2881' - Label 'Secondary structure report'\n",
      "Entity 'http://edamontology.org/data_2881' - Label 'Secondary structure report'\n",
      "Entity 'http://edamontology.org/data_2882' - Label 'DNA features'\n",
      "Entity 'http://edamontology.org/data_2882' - Label 'DNA features'\n",
      "Entity 'http://edamontology.org/data_2883' - Label 'RNA features report'\n",
      "Entity 'http://edamontology.org/data_2883' - Label 'RNA features report'\n",
      "Entity 'http://edamontology.org/data_2884' - Label 'Plot'\n",
      "Entity 'http://edamontology.org/data_2886' - Label 'Protein sequence record'\n",
      "Entity 'http://edamontology.org/data_2886' - Label 'Protein sequence record'\n",
      "Entity 'http://edamontology.org/data_2887' - Label 'Nucleic acid sequence record'\n",
      "Entity 'http://edamontology.org/data_2887' - Label 'Nucleic acid sequence record'\n",
      "Entity 'http://edamontology.org/data_2888' - Label 'Protein sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2888' - Label 'Protein sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2889' - Label 'Nucleic acid sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2889' - Label 'Nucleic acid sequence record (full)'\n",
      "Entity 'http://edamontology.org/data_2891' - Label 'Biological model accession'\n",
      "Entity 'http://edamontology.org/data_2892' - Label 'Cell type name'\n",
      "Entity 'http://edamontology.org/data_2892' - Label 'Cell type name'\n",
      "Entity 'http://edamontology.org/data_2893' - Label 'Cell type accession'\n",
      "Entity 'http://edamontology.org/data_2894' - Label 'Compound accession'\n",
      "Entity 'http://edamontology.org/data_2894' - Label 'Compound accession'\n",
      "Entity 'http://edamontology.org/data_2895' - Label 'Drug accession'\n",
      "Entity 'http://edamontology.org/data_2896' - Label 'Toxin name'\n",
      "Entity 'http://edamontology.org/data_2896' - Label 'Toxin name'\n",
      "Entity 'http://edamontology.org/data_2897' - Label 'Toxin accession'\n",
      "Entity 'http://edamontology.org/data_2898' - Label 'Monosaccharide accession'\n",
      "Entity 'http://edamontology.org/data_2899' - Label 'Drug name'\n",
      "Entity 'http://edamontology.org/data_2899' - Label 'Drug name'\n",
      "Entity 'http://edamontology.org/data_2900' - Label 'Carbohydrate accession'\n",
      "Entity 'http://edamontology.org/data_2900' - Label 'Carbohydrate accession'\n",
      "Entity 'http://edamontology.org/data_2901' - Label 'Molecule accession'\n",
      "Entity 'http://edamontology.org/data_2902' - Label 'Data resource definition accession'\n",
      "Entity 'http://edamontology.org/data_2903' - Label 'Genome accession'\n",
      "Entity 'http://edamontology.org/data_2904' - Label 'Map accession'\n",
      "Entity 'http://edamontology.org/data_2905' - Label 'Lipid accession'\n",
      "Entity 'http://edamontology.org/data_2905' - Label 'Lipid accession'\n",
      "Entity 'http://edamontology.org/data_2906' - Label 'Peptide ID'\n",
      "Entity 'http://edamontology.org/data_2906' - Label 'Peptide ID'\n",
      "Entity 'http://edamontology.org/data_2907' - Label 'Protein accession'\n",
      "Entity 'http://edamontology.org/data_2907' - Label 'Protein accession'\n",
      "Entity 'http://edamontology.org/data_2908' - Label 'Organism accession'\n",
      "Entity 'http://edamontology.org/data_2909' - Label 'Organism name'\n",
      "Entity 'http://edamontology.org/data_2909' - Label 'Organism name'\n",
      "Entity 'http://edamontology.org/data_2910' - Label 'Protein family accession'\n",
      "Entity 'http://edamontology.org/data_2911' - Label 'Transcription factor accession'\n",
      "Entity 'http://edamontology.org/data_2911' - Label 'Transcription factor accession'\n",
      "Entity 'http://edamontology.org/data_2912' - Label 'Strain accession'\n",
      "Entity 'http://edamontology.org/data_2912' - Label 'Strain accession'\n",
      "Entity 'http://edamontology.org/data_2912' - Label 'Strain accession'\n",
      "Entity 'http://edamontology.org/data_2913' - Label 'Virus identifier'\n",
      "Entity 'http://edamontology.org/data_2913' - Label 'Virus identifier'\n",
      "Entity 'http://edamontology.org/data_2914' - Label 'Sequence features metadata'\n",
      "Entity 'http://edamontology.org/data_2915' - Label 'Gramene identifier'\n",
      "Entity 'http://edamontology.org/data_2915' - Label 'Gramene identifier'\n",
      "Entity 'http://edamontology.org/data_2916' - Label 'DDBJ accession'\n",
      "Entity 'http://edamontology.org/data_2917' - Label 'ConsensusPathDB identifier'\n",
      "Entity 'http://edamontology.org/data_2925' - Label 'Sequence data'\n",
      "Entity 'http://edamontology.org/data_2925' - Label 'Sequence data'\n",
      "Entity 'http://edamontology.org/data_2927' - Label 'Codon usage'\n",
      "Entity 'http://edamontology.org/data_2927' - Label 'Codon usage'\n",
      "Entity 'http://edamontology.org/data_2954' - Label 'Article report'\n",
      "Entity 'http://edamontology.org/data_2954' - Label 'Article report'\n",
      "Entity 'http://edamontology.org/data_2954' - Label 'Article report'\n",
      "Entity 'http://edamontology.org/data_2955' - Label 'Sequence report'\n",
      "Entity 'http://edamontology.org/data_2956' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/data_2957' - Label 'Hopp and Woods plot'\n",
      "Entity 'http://edamontology.org/data_2957' - Label 'Hopp and Woods plot'\n",
      "Entity 'http://edamontology.org/data_2958' - Label 'Nucleic acid melting curve'\n",
      "Entity 'http://edamontology.org/data_2958' - Label 'Nucleic acid melting curve'\n",
      "Entity 'http://edamontology.org/data_2959' - Label 'Nucleic acid probability profile'\n",
      "Entity 'http://edamontology.org/data_2959' - Label 'Nucleic acid probability profile'\n",
      "Entity 'http://edamontology.org/data_2960' - Label 'Nucleic acid temperature profile'\n",
      "Entity 'http://edamontology.org/data_2960' - Label 'Nucleic acid temperature profile'\n",
      "Entity 'http://edamontology.org/data_2961' - Label 'Gene regulatory network report'\n",
      "Entity 'http://edamontology.org/data_2961' - Label 'Gene regulatory network report'\n",
      "Entity 'http://edamontology.org/data_2965' - Label '2D PAGE gel report'\n",
      "Entity 'http://edamontology.org/data_2965' - Label '2D PAGE gel report'\n",
      "Entity 'http://edamontology.org/data_2966' - Label 'Oligonucleotide probe sets annotation'\n",
      "Entity 'http://edamontology.org/data_2966' - Label 'Oligonucleotide probe sets annotation'\n",
      "Entity 'http://edamontology.org/data_2967' - Label 'Microarray image'\n",
      "Entity 'http://edamontology.org/data_2967' - Label 'Microarray image'\n",
      "Entity 'http://edamontology.org/data_2968' - Label 'Image'\n",
      "Entity 'http://edamontology.org/data_2969' - Label 'Sequence image'\n",
      "Entity 'http://edamontology.org/data_2969' - Label 'Sequence image'\n",
      "Entity 'http://edamontology.org/data_2970' - Label 'Protein hydropathy data'\n",
      "Entity 'http://edamontology.org/data_2971' - Label 'Workflow data'\n",
      "Entity 'http://edamontology.org/data_2971' - Label 'Workflow data'\n",
      "Entity 'http://edamontology.org/data_2972' - Label 'Workflow'\n",
      "Entity 'http://edamontology.org/data_2972' - Label 'Workflow'\n",
      "Entity 'http://edamontology.org/data_2973' - Label 'Secondary structure data'\n",
      "Entity 'http://edamontology.org/data_2973' - Label 'Secondary structure data'\n",
      "Entity 'http://edamontology.org/data_2974' - Label 'Protein sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2974' - Label 'Protein sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2975' - Label 'Nucleic acid sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2975' - Label 'Nucleic acid sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_2976' - Label 'Protein sequence'\n",
      "Entity 'http://edamontology.org/data_2977' - Label 'Nucleic acid sequence'\n",
      "Entity 'http://edamontology.org/data_2978' - Label 'Reaction data'\n",
      "Entity 'http://edamontology.org/data_2979' - Label 'Peptide property'\n",
      "Entity 'http://edamontology.org/data_2980' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/data_2980' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/data_2981' - Label 'Sequence motif data'\n",
      "Entity 'http://edamontology.org/data_2981' - Label 'Sequence motif data'\n",
      "Entity 'http://edamontology.org/data_2982' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2982' - Label 'Sequence profile data'\n",
      "Entity 'http://edamontology.org/data_2983' - Label 'Pathway or network data'\n",
      "Entity 'http://edamontology.org/data_2983' - Label 'Pathway or network data'\n",
      "Entity 'http://edamontology.org/data_2983' - Label 'Pathway or network data'\n",
      "Entity 'http://edamontology.org/data_2984' - Label 'Pathway or network report'\n",
      "Entity 'http://edamontology.org/data_2984' - Label 'Pathway or network report'\n",
      "Entity 'http://edamontology.org/data_2985' - Label 'Nucleic acid thermodynamic data'\n",
      "Entity 'http://edamontology.org/data_2986' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/data_2986' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/data_2987' - Label 'Classification report'\n",
      "Entity 'http://edamontology.org/data_2987' - Label 'Classification report'\n",
      "Entity 'http://edamontology.org/data_2989' - Label 'Protein features report (key folding sites)'\n",
      "Entity 'http://edamontology.org/data_2989' - Label 'Protein features report (key folding sites)'\n",
      "Entity 'http://edamontology.org/data_2991' - Label 'Protein geometry data'\n",
      "Entity 'http://edamontology.org/data_2992' - Label 'Protein structure image'\n",
      "Entity 'http://edamontology.org/data_2992' - Label 'Protein structure image'\n",
      "Entity 'http://edamontology.org/data_2994' - Label 'Phylogenetic character weights'\n",
      "Entity 'http://edamontology.org/data_3002' - Label 'Annotation track'\n",
      "Entity 'http://edamontology.org/data_3021' - Label 'UniProt accession'\n",
      "Entity 'http://edamontology.org/data_3021' - Label 'UniProt accession'\n",
      "Entity 'http://edamontology.org/data_3022' - Label 'NCBI genetic code ID'\n",
      "Entity 'http://edamontology.org/data_3022' - Label 'NCBI genetic code ID'\n",
      "Entity 'http://edamontology.org/data_3025' - Label 'Ontology concept identifier'\n",
      "Entity 'http://edamontology.org/data_3025' - Label 'Ontology concept identifier'\n",
      "Entity 'http://edamontology.org/data_3026' - Label 'GO concept name (biological process)'\n",
      "Entity 'http://edamontology.org/data_3026' - Label 'GO concept name (biological process)'\n",
      "Entity 'http://edamontology.org/data_3027' - Label 'GO concept name (molecular function)'\n",
      "Entity 'http://edamontology.org/data_3027' - Label 'GO concept name (molecular function)'\n",
      "Entity 'http://edamontology.org/data_3028' - Label 'Taxonomy'\n",
      "Entity 'http://edamontology.org/data_3028' - Label 'Taxonomy'\n",
      "Entity 'http://edamontology.org/data_3029' - Label 'Protein ID (EMBL/GenBank/DDBJ)'\n",
      "Entity 'http://edamontology.org/data_3029' - Label 'Protein ID (EMBL/GenBank/DDBJ)'\n",
      "Entity 'http://edamontology.org/data_3031' - Label 'Core data'\n",
      "Entity 'http://edamontology.org/data_3031' - Label 'Core data'\n",
      "Entity 'http://edamontology.org/data_3034' - Label 'Sequence feature identifier'\n",
      "Entity 'http://edamontology.org/data_3034' - Label 'Sequence feature identifier'\n",
      "Entity 'http://edamontology.org/data_3035' - Label 'Structure identifier'\n",
      "Entity 'http://edamontology.org/data_3035' - Label 'Structure identifier'\n",
      "Entity 'http://edamontology.org/data_3036' - Label 'Matrix identifier'\n",
      "Entity 'http://edamontology.org/data_3036' - Label 'Matrix identifier'\n",
      "Entity 'http://edamontology.org/data_3085' - Label 'Protein sequence composition'\n",
      "Entity 'http://edamontology.org/data_3085' - Label 'Protein sequence composition'\n",
      "Entity 'http://edamontology.org/data_3086' - Label 'Nucleic acid sequence composition (report)'\n",
      "Entity 'http://edamontology.org/data_3086' - Label 'Nucleic acid sequence composition (report)'\n",
      "Entity 'http://edamontology.org/data_3101' - Label 'Protein domain classification node'\n",
      "Entity 'http://edamontology.org/data_3101' - Label 'Protein domain classification node'\n",
      "Entity 'http://edamontology.org/data_3102' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_3102' - Label 'CAS number'\n",
      "Entity 'http://edamontology.org/data_3103' - Label 'ATC code'\n",
      "Entity 'http://edamontology.org/data_3103' - Label 'ATC code'\n",
      "Entity 'http://edamontology.org/data_3104' - Label 'UNII'\n",
      "Entity 'http://edamontology.org/data_3105' - Label 'Geotemporal metadata'\n",
      "Entity 'http://edamontology.org/data_3105' - Label 'Geotemporal metadata'\n",
      "Entity 'http://edamontology.org/data_3106' - Label 'System metadata'\n",
      "Entity 'http://edamontology.org/data_3107' - Label 'Sequence feature name'\n",
      "Entity 'http://edamontology.org/data_3107' - Label 'Sequence feature name'\n",
      "Entity 'http://edamontology.org/data_3108' - Label 'Experimental measurement'\n",
      "Entity 'http://edamontology.org/data_3110' - Label 'Raw microarray data'\n",
      "Entity 'http://edamontology.org/data_3110' - Label 'Raw microarray data'\n",
      "Entity 'http://edamontology.org/data_3111' - Label 'Processed microarray data'\n",
      "Entity 'http://edamontology.org/data_3111' - Label 'Processed microarray data'\n",
      "Entity 'http://edamontology.org/data_3112' - Label 'Gene expression matrix'\n",
      "Entity 'http://edamontology.org/data_3112' - Label 'Gene expression matrix'\n",
      "Entity 'http://edamontology.org/data_3113' - Label 'Sample annotation'\n",
      "Entity 'http://edamontology.org/data_3115' - Label 'Microarray metadata'\n",
      "Entity 'http://edamontology.org/data_3116' - Label 'Microarray protocol annotation'\n",
      "Entity 'http://edamontology.org/data_3116' - Label 'Microarray protocol annotation'\n",
      "Entity 'http://edamontology.org/data_3117' - Label 'Microarray hybridisation data'\n",
      "Entity 'http://edamontology.org/data_3119' - Label 'Sequence features (compositionally-biased regions)'\n",
      "Entity 'http://edamontology.org/data_3119' - Label 'Sequence features (compositionally-biased regions)'\n",
      "Entity 'http://edamontology.org/data_3122' - Label 'Nucleic acid features (difference and change)'\n",
      "Entity 'http://edamontology.org/data_3122' - Label 'Nucleic acid features (difference and change)'\n",
      "Entity 'http://edamontology.org/data_3128' - Label 'Nucleic acid structure report'\n",
      "Entity 'http://edamontology.org/data_3128' - Label 'Nucleic acid structure report'\n",
      "Entity 'http://edamontology.org/data_3129' - Label 'Protein features report (repeats)'\n",
      "Entity 'http://edamontology.org/data_3129' - Label 'Protein features report (repeats)'\n",
      "Entity 'http://edamontology.org/data_3130' - Label 'Sequence motif matches (protein)'\n",
      "Entity 'http://edamontology.org/data_3130' - Label 'Sequence motif matches (protein)'\n",
      "Entity 'http://edamontology.org/data_3131' - Label 'Sequence motif matches (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_3131' - Label 'Sequence motif matches (nucleic acid)'\n",
      "Entity 'http://edamontology.org/data_3132' - Label 'Nucleic acid features (d-loop)'\n",
      "Entity 'http://edamontology.org/data_3132' - Label 'Nucleic acid features (d-loop)'\n",
      "Entity 'http://edamontology.org/data_3133' - Label 'Nucleic acid features (stem loop)'\n",
      "Entity 'http://edamontology.org/data_3133' - Label 'Nucleic acid features (stem loop)'\n",
      "Entity 'http://edamontology.org/data_3134' - Label 'Gene transcript report'\n",
      "Entity 'http://edamontology.org/data_3137' - Label 'Non-coding RNA'\n",
      "Entity 'http://edamontology.org/data_3137' - Label 'Non-coding RNA'\n",
      "Entity 'http://edamontology.org/data_3138' - Label 'Transcriptional features (report)'\n",
      "Entity 'http://edamontology.org/data_3138' - Label 'Transcriptional features (report)'\n",
      "Entity 'http://edamontology.org/data_3140' - Label 'Nucleic acid features (immunoglobulin gene structure)'\n",
      "Entity 'http://edamontology.org/data_3140' - Label 'Nucleic acid features (immunoglobulin gene structure)'\n",
      "Entity 'http://edamontology.org/data_3141' - Label 'SCOP class'\n",
      "Entity 'http://edamontology.org/data_3141' - Label 'SCOP class'\n",
      "Entity 'http://edamontology.org/data_3142' - Label 'SCOP fold'\n",
      "Entity 'http://edamontology.org/data_3142' - Label 'SCOP fold'\n",
      "Entity 'http://edamontology.org/data_3143' - Label 'SCOP superfamily'\n",
      "Entity 'http://edamontology.org/data_3143' - Label 'SCOP superfamily'\n",
      "Entity 'http://edamontology.org/data_3144' - Label 'SCOP family'\n",
      "Entity 'http://edamontology.org/data_3144' - Label 'SCOP family'\n",
      "Entity 'http://edamontology.org/data_3145' - Label 'SCOP protein'\n",
      "Entity 'http://edamontology.org/data_3145' - Label 'SCOP protein'\n",
      "Entity 'http://edamontology.org/data_3146' - Label 'SCOP species'\n",
      "Entity 'http://edamontology.org/data_3146' - Label 'SCOP species'\n",
      "Entity 'http://edamontology.org/data_3147' - Label 'Mass spectrometry experiment'\n",
      "Entity 'http://edamontology.org/data_3147' - Label 'Mass spectrometry experiment'\n",
      "Entity 'http://edamontology.org/data_3148' - Label 'Gene family report'\n",
      "Entity 'http://edamontology.org/data_3153' - Label 'Protein image'\n",
      "Entity 'http://edamontology.org/data_3154' - Label 'Protein alignment'\n",
      "Entity 'http://edamontology.org/data_3154' - Label 'Protein alignment'\n",
      "Entity 'http://edamontology.org/data_3154' - Label 'Protein alignment'\n",
      "Entity 'http://edamontology.org/data_3154' - Label 'Protein alignment'\n",
      "Entity 'http://edamontology.org/data_3165' - Label 'NGS experiment'\n",
      "Entity 'http://edamontology.org/data_3165' - Label 'NGS experiment'\n",
      "Entity 'http://edamontology.org/data_3181' - Label 'Sequence assembly report'\n",
      "Entity 'http://edamontology.org/data_3210' - Label 'Genome index'\n",
      "Entity 'http://edamontology.org/data_3231' - Label 'GWAS report'\n",
      "Entity 'http://edamontology.org/data_3231' - Label 'GWAS report'\n",
      "Entity 'http://edamontology.org/data_3236' - Label 'Cytoband position'\n",
      "Entity 'http://edamontology.org/data_3238' - Label 'Cell type ontology ID'\n",
      "Entity 'http://edamontology.org/data_3238' - Label 'Cell type ontology ID'\n",
      "Entity 'http://edamontology.org/data_3238' - Label 'Cell type ontology ID'\n",
      "Entity 'http://edamontology.org/data_3241' - Label 'Kinetic model'\n",
      "Entity 'http://edamontology.org/data_3264' - Label 'COSMIC ID'\n",
      "Entity 'http://edamontology.org/data_3264' - Label 'COSMIC ID'\n",
      "Entity 'http://edamontology.org/data_3265' - Label 'HGMD ID'\n",
      "Entity 'http://edamontology.org/data_3265' - Label 'HGMD ID'\n",
      "Entity 'http://edamontology.org/data_3266' - Label 'Sequence assembly ID'\n",
      "Entity 'http://edamontology.org/data_3268' - Label 'Sequence feature type'\n",
      "Entity 'http://edamontology.org/data_3268' - Label 'Sequence feature type'\n",
      "Entity 'http://edamontology.org/data_3269' - Label 'Gene homology (report)'\n",
      "Entity 'http://edamontology.org/data_3269' - Label 'Gene homology (report)'\n",
      "Entity 'http://edamontology.org/data_3270' - Label 'Ensembl gene tree ID'\n",
      "Entity 'http://edamontology.org/data_3270' - Label 'Ensembl gene tree ID'\n",
      "Entity 'http://edamontology.org/data_3270' - Label 'Ensembl gene tree ID'\n",
      "Entity 'http://edamontology.org/data_3271' - Label 'Gene tree'\n",
      "Entity 'http://edamontology.org/data_3272' - Label 'Species tree'\n",
      "Entity 'http://edamontology.org/data_3273' - Label 'Sample ID'\n",
      "Entity 'http://edamontology.org/data_3273' - Label 'Sample ID'\n",
      "Entity 'http://edamontology.org/data_3274' - Label 'MGI accession'\n",
      "Entity 'http://edamontology.org/data_3274' - Label 'MGI accession'\n",
      "Entity 'http://edamontology.org/data_3275' - Label 'Phenotype name'\n",
      "Entity 'http://edamontology.org/data_3354' - Label 'Transition matrix'\n",
      "Entity 'http://edamontology.org/data_3355' - Label 'Emission matrix'\n",
      "Entity 'http://edamontology.org/data_3356' - Label 'Hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_3356' - Label 'Hidden Markov model'\n",
      "Entity 'http://edamontology.org/data_3358' - Label 'Format identifier'\n",
      "Entity 'http://edamontology.org/data_3424' - Label 'Raw image'\n",
      "Entity 'http://edamontology.org/data_3425' - Label 'Carbohydrate property'\n",
      "Entity 'http://edamontology.org/data_3426' - Label 'Proteomics experiment report'\n",
      "Entity 'http://edamontology.org/data_3426' - Label 'Proteomics experiment report'\n",
      "Entity 'http://edamontology.org/data_3427' - Label 'RNAi report'\n",
      "Entity 'http://edamontology.org/data_3427' - Label 'RNAi report'\n",
      "Entity 'http://edamontology.org/data_3428' - Label 'Simulation experiment report'\n",
      "Entity 'http://edamontology.org/data_3428' - Label 'Simulation experiment report'\n",
      "Entity 'http://edamontology.org/data_3442' - Label 'MRI image'\n",
      "Entity 'http://edamontology.org/data_3442' - Label 'MRI image'\n",
      "Entity 'http://edamontology.org/data_3449' - Label 'Cell migration track image'\n",
      "Entity 'http://edamontology.org/data_3449' - Label 'Cell migration track image'\n",
      "Entity 'http://edamontology.org/data_3451' - Label 'Rate of association'\n",
      "Entity 'http://edamontology.org/data_3479' - Label 'Gene order'\n",
      "Entity 'http://edamontology.org/data_3483' - Label 'Spectrum'\n",
      "Entity 'http://edamontology.org/data_3488' - Label 'NMR spectrum'\n",
      "Entity 'http://edamontology.org/data_3488' - Label 'NMR spectrum'\n",
      "Entity 'http://edamontology.org/data_3490' - Label 'Chemical structure sketch'\n",
      "Entity 'http://edamontology.org/data_3490' - Label 'Chemical structure sketch'\n",
      "Entity 'http://edamontology.org/data_3492' - Label 'Nucleic acid signature'\n",
      "Entity 'http://edamontology.org/data_3494' - Label 'DNA sequence'\n",
      "Entity 'http://edamontology.org/data_3495' - Label 'RNA sequence'\n",
      "Entity 'http://edamontology.org/data_3496' - Label 'RNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3496' - Label 'RNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3497' - Label 'DNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3497' - Label 'DNA sequence (raw)'\n",
      "Entity 'http://edamontology.org/data_3498' - Label 'Sequence variations'\n",
      "Entity 'http://edamontology.org/data_3498' - Label 'Sequence variations'\n",
      "Entity 'http://edamontology.org/data_3505' - Label 'Bibliography'\n",
      "Entity 'http://edamontology.org/data_3509' - Label 'Ontology mapping'\n",
      "Entity 'http://edamontology.org/data_3546' - Label 'Image metadata'\n",
      "Entity 'http://edamontology.org/data_3558' - Label 'Clinical trial report'\n",
      "Entity 'http://edamontology.org/data_3567' - Label 'Reference sample report'\n",
      "Entity 'http://edamontology.org/data_3568' - Label 'Gene Expression Atlas Experiment ID'\n",
      "Entity 'http://edamontology.org/data_3667' - Label 'Disease identifier'\n",
      "Entity 'http://edamontology.org/data_3667' - Label 'Disease identifier'\n",
      "Entity 'http://edamontology.org/data_3668' - Label 'Disease name'\n",
      "Entity 'http://edamontology.org/data_3668' - Label 'Disease name'\n",
      "Entity 'http://edamontology.org/data_3669' - Label 'Training material'\n",
      "Entity 'http://edamontology.org/data_3670' - Label 'Online course'\n",
      "Entity 'http://edamontology.org/data_3671' - Label 'Text'\n",
      "Entity 'http://edamontology.org/data_3707' - Label 'Biodiversity data'\n",
      "Entity 'http://edamontology.org/data_3707' - Label 'Biodiversity data'\n",
      "Entity 'http://edamontology.org/data_3716' - Label 'Biosafety report'\n",
      "Entity 'http://edamontology.org/data_3717' - Label 'Isolation report'\n",
      "Entity 'http://edamontology.org/data_3718' - Label 'Pathogenicity report'\n",
      "Entity 'http://edamontology.org/data_3719' - Label 'Biosafety classification'\n",
      "Entity 'http://edamontology.org/data_3720' - Label 'Geographic location'\n",
      "Entity 'http://edamontology.org/data_3721' - Label 'Isolation source'\n",
      "Entity 'http://edamontology.org/data_3722' - Label 'Physiology parameter'\n",
      "Entity 'http://edamontology.org/data_3723' - Label 'Morphology parameter'\n",
      "Entity 'http://edamontology.org/data_3724' - Label 'Cultivation parameter'\n",
      "Entity 'http://edamontology.org/data_3732' - Label 'Sequencing metadata name'\n",
      "Entity 'http://edamontology.org/data_3733' - Label 'Flow cell identifier'\n",
      "Entity 'http://edamontology.org/data_3734' - Label 'Lane identifier'\n",
      "Entity 'http://edamontology.org/data_3735' - Label 'Run number'\n",
      "Entity 'http://edamontology.org/data_3736' - Label 'Ecological data'\n",
      "Entity 'http://edamontology.org/data_3737' - Label 'Alpha diversity data'\n",
      "Entity 'http://edamontology.org/data_3738' - Label 'Beta diversity data'\n",
      "Entity 'http://edamontology.org/data_3739' - Label 'Gamma diversity data'\n",
      "Entity 'http://edamontology.org/data_3743' - Label 'Ordination plot'\n",
      "Entity 'http://edamontology.org/data_3743' - Label 'Ordination plot'\n",
      "Entity 'http://edamontology.org/data_3753' - Label 'Over-representation data'\n",
      "Entity 'http://edamontology.org/data_3754' - Label 'GO-term enrichment data'\n",
      "Entity 'http://edamontology.org/data_3754' - Label 'GO-term enrichment data'\n",
      "Entity 'http://edamontology.org/data_3756' - Label 'Localisation score'\n",
      "Entity 'http://edamontology.org/data_3757' - Label 'Unimod ID'\n",
      "Entity 'http://edamontology.org/data_3757' - Label 'Unimod ID'\n",
      "Entity 'http://edamontology.org/data_3759' - Label 'ProteomeXchange ID'\n",
      "Entity 'http://edamontology.org/data_3768' - Label 'Clustered expression profiles'\n",
      "Entity 'http://edamontology.org/data_3769' - Label 'BRENDA ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_3769' - Label 'BRENDA ontology concept ID'\n",
      "Entity 'http://edamontology.org/data_3779' - Label 'Annotated text'\n",
      "Entity 'http://edamontology.org/data_3779' - Label 'Annotated text'\n",
      "Entity 'http://edamontology.org/data_3786' - Label 'Query script'\n",
      "Entity 'http://edamontology.org/data_3805' - Label '3D EM Map'\n",
      "Entity 'http://edamontology.org/data_3805' - Label '3D EM Map'\n",
      "Entity 'http://edamontology.org/data_3806' - Label '3D EM Mask'\n",
      "Entity 'http://edamontology.org/data_3806' - Label '3D EM Mask'\n",
      "Entity 'http://edamontology.org/data_3807' - Label 'EM Movie'\n",
      "Entity 'http://edamontology.org/data_3807' - Label 'EM Movie'\n",
      "Entity 'http://edamontology.org/data_3808' - Label 'EM Micrograph'\n",
      "Entity 'http://edamontology.org/data_3808' - Label 'EM Micrograph'\n",
      "Entity 'http://edamontology.org/data_3842' - Label 'Molecular simulation data'\n",
      "Entity 'http://edamontology.org/data_3842' - Label 'Molecular simulation data'\n",
      "Entity 'http://edamontology.org/data_3856' - Label 'RNA central ID'\n",
      "Entity 'http://edamontology.org/data_3856' - Label 'RNA central ID'\n",
      "Entity 'http://edamontology.org/data_3861' - Label 'Electronic health record'\n",
      "Entity 'http://edamontology.org/data_3869' - Label 'Simulation'\n",
      "Entity 'http://edamontology.org/data_3870' - Label 'Trajectory data'\n",
      "Entity 'http://edamontology.org/data_3871' - Label 'Forcefield parameters'\n",
      "Entity 'http://edamontology.org/data_3872' - Label 'Topology data'\n",
      "Entity 'http://edamontology.org/data_3905' - Label 'Histogram'\n",
      "Entity 'http://edamontology.org/data_3914' - Label 'Quality control report'\n",
      "Entity 'http://edamontology.org/data_3917' - Label 'Count matrix'\n",
      "Entity 'http://edamontology.org/data_3924' - Label 'DNA structure alignment'\n",
      "Entity 'http://edamontology.org/data_3932' - Label 'Q-value'\n",
      "Entity 'http://edamontology.org/data_3949' - Label 'Profile HMM'\n",
      "Entity 'http://edamontology.org/data_3949' - Label 'Profile HMM'\n",
      "Entity 'http://edamontology.org/data_3952' - Label 'Pathway ID (WikiPathways)'\n",
      "Entity 'http://edamontology.org/data_3952' - Label 'Pathway ID (WikiPathways)'\n",
      "Entity 'http://edamontology.org/data_3952' - Label 'Pathway ID (WikiPathways)'\n",
      "Entity 'http://edamontology.org/data_3953' - Label 'Pathway overrepresentation data'\n",
      "Entity 'http://edamontology.org/data_3953' - Label 'Pathway overrepresentation data'\n",
      "Entity 'http://edamontology.org/data_4022' - Label 'ORCID Identifier'\n",
      "Entity 'http://edamontology.org/format_1196' - Label 'SMILES'\n",
      "Entity 'http://edamontology.org/format_1196' - Label 'SMILES'\n",
      "Entity 'http://edamontology.org/format_1197' - Label 'InChI'\n",
      "Entity 'http://edamontology.org/format_1197' - Label 'InChI'\n",
      "Entity 'http://edamontology.org/format_1198' - Label 'mf'\n",
      "Entity 'http://edamontology.org/format_1198' - Label 'mf'\n",
      "Entity 'http://edamontology.org/format_1199' - Label 'InChIKey'\n",
      "Entity 'http://edamontology.org/format_1199' - Label 'InChIKey'\n",
      "Entity 'http://edamontology.org/format_1200' - Label 'smarts'\n",
      "Entity 'http://edamontology.org/format_1206' - Label 'unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_1206' - Label 'unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_1207' - Label 'nucleotide'\n",
      "Entity 'http://edamontology.org/format_1207' - Label 'nucleotide'\n",
      "Entity 'http://edamontology.org/format_1208' - Label 'protein'\n",
      "Entity 'http://edamontology.org/format_1208' - Label 'protein'\n",
      "Entity 'http://edamontology.org/format_1209' - Label 'consensus'\n",
      "Entity 'http://edamontology.org/format_1209' - Label 'consensus'\n",
      "Entity 'http://edamontology.org/format_1210' - Label 'pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1210' - Label 'pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1211' - Label 'unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1211' - Label 'unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_1212' - Label 'dna'\n",
      "Entity 'http://edamontology.org/format_1213' - Label 'rna'\n",
      "Entity 'http://edamontology.org/format_1214' - Label 'unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_1214' - Label 'unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_1215' - Label 'pure dna'\n",
      "Entity 'http://edamontology.org/format_1215' - Label 'pure dna'\n",
      "Entity 'http://edamontology.org/format_1216' - Label 'unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_1216' - Label 'unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_1217' - Label 'pure rna'\n",
      "Entity 'http://edamontology.org/format_1217' - Label 'pure rna'\n",
      "Entity 'http://edamontology.org/format_1218' - Label 'unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_1218' - Label 'unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_1219' - Label 'pure protein'\n",
      "Entity 'http://edamontology.org/format_1219' - Label 'pure protein'\n",
      "Entity 'http://edamontology.org/format_1228' - Label 'UniGene entry format'\n",
      "Entity 'http://edamontology.org/format_1228' - Label 'UniGene entry format'\n",
      "Entity 'http://edamontology.org/format_1247' - Label 'COG sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_1247' - Label 'COG sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_1248' - Label 'EMBL feature location'\n",
      "Entity 'http://edamontology.org/format_1248' - Label 'EMBL feature location'\n",
      "Entity 'http://edamontology.org/format_1295' - Label 'quicktandem'\n",
      "Entity 'http://edamontology.org/format_1295' - Label 'quicktandem'\n",
      "Entity 'http://edamontology.org/format_1296' - Label 'Sanger inverted repeats'\n",
      "Entity 'http://edamontology.org/format_1296' - Label 'Sanger inverted repeats'\n",
      "Entity 'http://edamontology.org/format_1297' - Label 'EMBOSS repeat'\n",
      "Entity 'http://edamontology.org/format_1297' - Label 'EMBOSS repeat'\n",
      "Entity 'http://edamontology.org/format_1316' - Label 'est2genome format'\n",
      "Entity 'http://edamontology.org/format_1316' - Label 'est2genome format'\n",
      "Entity 'http://edamontology.org/format_1318' - Label 'restrict format'\n",
      "Entity 'http://edamontology.org/format_1318' - Label 'restrict format'\n",
      "Entity 'http://edamontology.org/format_1319' - Label 'restover format'\n",
      "Entity 'http://edamontology.org/format_1319' - Label 'restover format'\n",
      "Entity 'http://edamontology.org/format_1320' - Label 'REBASE restriction sites'\n",
      "Entity 'http://edamontology.org/format_1320' - Label 'REBASE restriction sites'\n",
      "Entity 'http://edamontology.org/format_1332' - Label 'FASTA search results format'\n",
      "Entity 'http://edamontology.org/format_1332' - Label 'FASTA search results format'\n",
      "Entity 'http://edamontology.org/format_1333' - Label 'BLAST results'\n",
      "Entity 'http://edamontology.org/format_1333' - Label 'BLAST results'\n",
      "Entity 'http://edamontology.org/format_1334' - Label 'mspcrunch'\n",
      "Entity 'http://edamontology.org/format_1334' - Label 'mspcrunch'\n",
      "Entity 'http://edamontology.org/format_1335' - Label 'Smith-Waterman format'\n",
      "Entity 'http://edamontology.org/format_1335' - Label 'Smith-Waterman format'\n",
      "Entity 'http://edamontology.org/format_1336' - Label 'dhf'\n",
      "Entity 'http://edamontology.org/format_1336' - Label 'dhf'\n",
      "Entity 'http://edamontology.org/format_1337' - Label 'lhf'\n",
      "Entity 'http://edamontology.org/format_1337' - Label 'lhf'\n",
      "Entity 'http://edamontology.org/format_1341' - Label 'InterPro hits format'\n",
      "Entity 'http://edamontology.org/format_1341' - Label 'InterPro hits format'\n",
      "Entity 'http://edamontology.org/format_1342' - Label 'InterPro protein view report format'\n",
      "Entity 'http://edamontology.org/format_1343' - Label 'InterPro match table format'\n",
      "Entity 'http://edamontology.org/format_1349' - Label 'HMMER Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1349' - Label 'HMMER Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1350' - Label 'MEME Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1350' - Label 'MEME Dirichlet prior'\n",
      "Entity 'http://edamontology.org/format_1351' - Label 'HMMER emission and transition'\n",
      "Entity 'http://edamontology.org/format_1351' - Label 'HMMER emission and transition'\n",
      "Entity 'http://edamontology.org/format_1356' - Label 'prosite-pattern'\n",
      "Entity 'http://edamontology.org/format_1356' - Label 'prosite-pattern'\n",
      "Entity 'http://edamontology.org/format_1357' - Label 'EMBOSS sequence pattern'\n",
      "Entity 'http://edamontology.org/format_1357' - Label 'EMBOSS sequence pattern'\n",
      "Entity 'http://edamontology.org/format_1360' - Label 'meme-motif'\n",
      "Entity 'http://edamontology.org/format_1360' - Label 'meme-motif'\n",
      "Entity 'http://edamontology.org/format_1366' - Label 'prosite-profile'\n",
      "Entity 'http://edamontology.org/format_1366' - Label 'prosite-profile'\n",
      "Entity 'http://edamontology.org/format_1367' - Label 'JASPAR format'\n",
      "Entity 'http://edamontology.org/format_1367' - Label 'JASPAR format'\n",
      "Entity 'http://edamontology.org/format_1369' - Label 'MEME background Markov model'\n",
      "Entity 'http://edamontology.org/format_1369' - Label 'MEME background Markov model'\n",
      "Entity 'http://edamontology.org/format_1370' - Label 'HMMER format'\n",
      "Entity 'http://edamontology.org/format_1370' - Label 'HMMER format'\n",
      "Entity 'http://edamontology.org/format_1391' - Label 'HMMER-aln'\n",
      "Entity 'http://edamontology.org/format_1391' - Label 'HMMER-aln'\n",
      "Entity 'http://edamontology.org/format_1391' - Label 'HMMER-aln'\n",
      "Entity 'http://edamontology.org/format_1392' - Label 'DIALIGN format'\n",
      "Entity 'http://edamontology.org/format_1392' - Label 'DIALIGN format'\n",
      "Entity 'http://edamontology.org/format_1393' - Label 'daf'\n",
      "Entity 'http://edamontology.org/format_1393' - Label 'daf'\n",
      "Entity 'http://edamontology.org/format_1419' - Label 'Sequence-MEME profile alignment'\n",
      "Entity 'http://edamontology.org/format_1419' - Label 'Sequence-MEME profile alignment'\n",
      "Entity 'http://edamontology.org/format_1421' - Label 'HMMER profile alignment (sequences versus HMMs)'\n",
      "Entity 'http://edamontology.org/format_1421' - Label 'HMMER profile alignment (sequences versus HMMs)'\n",
      "Entity 'http://edamontology.org/format_1422' - Label 'HMMER profile alignment (HMM versus sequences)'\n",
      "Entity 'http://edamontology.org/format_1422' - Label 'HMMER profile alignment (HMM versus sequences)'\n",
      "Entity 'http://edamontology.org/format_1423' - Label 'Phylip distance matrix'\n",
      "Entity 'http://edamontology.org/format_1423' - Label 'Phylip distance matrix'\n",
      "Entity 'http://edamontology.org/format_1424' - Label 'ClustalW dendrogram'\n",
      "Entity 'http://edamontology.org/format_1424' - Label 'ClustalW dendrogram'\n",
      "Entity 'http://edamontology.org/format_1425' - Label 'Phylip tree raw'\n",
      "Entity 'http://edamontology.org/format_1425' - Label 'Phylip tree raw'\n",
      "Entity 'http://edamontology.org/format_1430' - Label 'Phylip continuous quantitative characters'\n",
      "Entity 'http://edamontology.org/format_1430' - Label 'Phylip continuous quantitative characters'\n",
      "Entity 'http://edamontology.org/format_1431' - Label 'Phylogenetic property values format'\n",
      "Entity 'http://edamontology.org/format_1431' - Label 'Phylogenetic property values format'\n",
      "Entity 'http://edamontology.org/format_1432' - Label 'Phylip character frequencies format'\n",
      "Entity 'http://edamontology.org/format_1432' - Label 'Phylip character frequencies format'\n",
      "Entity 'http://edamontology.org/format_1433' - Label 'Phylip discrete states format'\n",
      "Entity 'http://edamontology.org/format_1433' - Label 'Phylip discrete states format'\n",
      "Entity 'http://edamontology.org/format_1434' - Label 'Phylip cliques format'\n",
      "Entity 'http://edamontology.org/format_1434' - Label 'Phylip cliques format'\n",
      "Entity 'http://edamontology.org/format_1435' - Label 'Phylip tree format'\n",
      "Entity 'http://edamontology.org/format_1435' - Label 'Phylip tree format'\n",
      "Entity 'http://edamontology.org/format_1436' - Label 'TreeBASE format'\n",
      "Entity 'http://edamontology.org/format_1436' - Label 'TreeBASE format'\n",
      "Entity 'http://edamontology.org/format_1437' - Label 'TreeFam format'\n",
      "Entity 'http://edamontology.org/format_1437' - Label 'TreeFam format'\n",
      "Entity 'http://edamontology.org/format_1445' - Label 'Phylip tree distance format'\n",
      "Entity 'http://edamontology.org/format_1445' - Label 'Phylip tree distance format'\n",
      "Entity 'http://edamontology.org/format_1454' - Label 'dssp'\n",
      "Entity 'http://edamontology.org/format_1454' - Label 'dssp'\n",
      "Entity 'http://edamontology.org/format_1455' - Label 'hssp'\n",
      "Entity 'http://edamontology.org/format_1455' - Label 'hssp'\n",
      "Entity 'http://edamontology.org/format_1457' - Label 'Dot-bracket format'\n",
      "Entity 'http://edamontology.org/format_1457' - Label 'Dot-bracket format'\n",
      "Entity 'http://edamontology.org/format_1458' - Label 'Vienna local RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_1458' - Label 'Vienna local RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_1475' - Label 'PDB database entry format'\n",
      "Entity 'http://edamontology.org/format_1475' - Label 'PDB database entry format'\n",
      "Entity 'http://edamontology.org/format_1475' - Label 'PDB database entry format'\n",
      "Entity 'http://edamontology.org/format_1476' - Label 'PDB'\n",
      "Entity 'http://edamontology.org/format_1476' - Label 'PDB'\n",
      "Entity 'http://edamontology.org/format_1477' - Label 'mmCIF'\n",
      "Entity 'http://edamontology.org/format_1477' - Label 'mmCIF'\n",
      "Entity 'http://edamontology.org/format_1478' - Label 'PDBML'\n",
      "Entity 'http://edamontology.org/format_1478' - Label 'PDBML'\n",
      "Entity 'http://edamontology.org/format_1500' - Label 'Domainatrix 3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_1500' - Label 'Domainatrix 3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_1504' - Label 'aaindex'\n",
      "Entity 'http://edamontology.org/format_1504' - Label 'aaindex'\n",
      "Entity 'http://edamontology.org/format_1511' - Label 'IntEnz enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1511' - Label 'IntEnz enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1512' - Label 'BRENDA enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1512' - Label 'BRENDA enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1513' - Label 'KEGG REACTION enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1513' - Label 'KEGG REACTION enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1514' - Label 'KEGG ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1514' - Label 'KEGG ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1515' - Label 'REBASE proto enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1515' - Label 'REBASE proto enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1516' - Label 'REBASE withrefm enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1516' - Label 'REBASE withrefm enzyme report format'\n",
      "Entity 'http://edamontology.org/format_1551' - Label 'Pcons report format'\n",
      "Entity 'http://edamontology.org/format_1551' - Label 'Pcons report format'\n",
      "Entity 'http://edamontology.org/format_1552' - Label 'ProQ report format'\n",
      "Entity 'http://edamontology.org/format_1552' - Label 'ProQ report format'\n",
      "Entity 'http://edamontology.org/format_1563' - Label 'SMART domain assignment report format'\n",
      "Entity 'http://edamontology.org/format_1563' - Label 'SMART domain assignment report format'\n",
      "Entity 'http://edamontology.org/format_1568' - Label 'BIND entry format'\n",
      "Entity 'http://edamontology.org/format_1568' - Label 'BIND entry format'\n",
      "Entity 'http://edamontology.org/format_1569' - Label 'IntAct entry format'\n",
      "Entity 'http://edamontology.org/format_1569' - Label 'IntAct entry format'\n",
      "Entity 'http://edamontology.org/format_1570' - Label 'InterPro entry format'\n",
      "Entity 'http://edamontology.org/format_1570' - Label 'InterPro entry format'\n",
      "Entity 'http://edamontology.org/format_1571' - Label 'InterPro entry abstract format'\n",
      "Entity 'http://edamontology.org/format_1571' - Label 'InterPro entry abstract format'\n",
      "Entity 'http://edamontology.org/format_1572' - Label 'Gene3D entry format'\n",
      "Entity 'http://edamontology.org/format_1572' - Label 'Gene3D entry format'\n",
      "Entity 'http://edamontology.org/format_1573' - Label 'PIRSF entry format'\n",
      "Entity 'http://edamontology.org/format_1573' - Label 'PIRSF entry format'\n",
      "Entity 'http://edamontology.org/format_1574' - Label 'PRINTS entry format'\n",
      "Entity 'http://edamontology.org/format_1574' - Label 'PRINTS entry format'\n",
      "Entity 'http://edamontology.org/format_1575' - Label 'Panther Families and HMMs entry format'\n",
      "Entity 'http://edamontology.org/format_1575' - Label 'Panther Families and HMMs entry format'\n",
      "Entity 'http://edamontology.org/format_1576' - Label 'Pfam entry format'\n",
      "Entity 'http://edamontology.org/format_1576' - Label 'Pfam entry format'\n",
      "Entity 'http://edamontology.org/format_1577' - Label 'SMART entry format'\n",
      "Entity 'http://edamontology.org/format_1577' - Label 'SMART entry format'\n",
      "Entity 'http://edamontology.org/format_1578' - Label 'Superfamily entry format'\n",
      "Entity 'http://edamontology.org/format_1578' - Label 'Superfamily entry format'\n",
      "Entity 'http://edamontology.org/format_1579' - Label 'TIGRFam entry format'\n",
      "Entity 'http://edamontology.org/format_1579' - Label 'TIGRFam entry format'\n",
      "Entity 'http://edamontology.org/format_1580' - Label 'ProDom entry format'\n",
      "Entity 'http://edamontology.org/format_1580' - Label 'ProDom entry format'\n",
      "Entity 'http://edamontology.org/format_1581' - Label 'FSSP entry format'\n",
      "Entity 'http://edamontology.org/format_1581' - Label 'FSSP entry format'\n",
      "Entity 'http://edamontology.org/format_1582' - Label 'findkm'\n",
      "Entity 'http://edamontology.org/format_1582' - Label 'findkm'\n",
      "Entity 'http://edamontology.org/format_1603' - Label 'Ensembl gene report format'\n",
      "Entity 'http://edamontology.org/format_1603' - Label 'Ensembl gene report format'\n",
      "Entity 'http://edamontology.org/format_1604' - Label 'DictyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1604' - Label 'DictyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1605' - Label 'CGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1605' - Label 'CGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1606' - Label 'DragonDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1606' - Label 'DragonDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1607' - Label 'EcoCyc gene report format'\n",
      "Entity 'http://edamontology.org/format_1607' - Label 'EcoCyc gene report format'\n",
      "Entity 'http://edamontology.org/format_1608' - Label 'FlyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1608' - Label 'FlyBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1609' - Label 'Gramene gene report format'\n",
      "Entity 'http://edamontology.org/format_1609' - Label 'Gramene gene report format'\n",
      "Entity 'http://edamontology.org/format_1610' - Label 'KEGG GENES gene report format'\n",
      "Entity 'http://edamontology.org/format_1610' - Label 'KEGG GENES gene report format'\n",
      "Entity 'http://edamontology.org/format_1611' - Label 'MaizeGDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1611' - Label 'MaizeGDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1612' - Label 'MGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1612' - Label 'MGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1613' - Label 'RGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1613' - Label 'RGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1614' - Label 'SGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1614' - Label 'SGD gene report format'\n",
      "Entity 'http://edamontology.org/format_1615' - Label 'GeneDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1615' - Label 'GeneDB gene report format'\n",
      "Entity 'http://edamontology.org/format_1616' - Label 'TAIR gene report format'\n",
      "Entity 'http://edamontology.org/format_1616' - Label 'TAIR gene report format'\n",
      "Entity 'http://edamontology.org/format_1617' - Label 'WormBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1617' - Label 'WormBase gene report format'\n",
      "Entity 'http://edamontology.org/format_1618' - Label 'ZFIN gene report format'\n",
      "Entity 'http://edamontology.org/format_1618' - Label 'ZFIN gene report format'\n",
      "Entity 'http://edamontology.org/format_1619' - Label 'TIGR gene report format'\n",
      "Entity 'http://edamontology.org/format_1619' - Label 'TIGR gene report format'\n",
      "Entity 'http://edamontology.org/format_1620' - Label 'dbSNP polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_1620' - Label 'dbSNP polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_1623' - Label 'OMIM entry format'\n",
      "Entity 'http://edamontology.org/format_1623' - Label 'OMIM entry format'\n",
      "Entity 'http://edamontology.org/format_1624' - Label 'HGVbase entry format'\n",
      "Entity 'http://edamontology.org/format_1624' - Label 'HGVbase entry format'\n",
      "Entity 'http://edamontology.org/format_1625' - Label 'HIVDB entry format'\n",
      "Entity 'http://edamontology.org/format_1625' - Label 'HIVDB entry format'\n",
      "Entity 'http://edamontology.org/format_1626' - Label 'KEGG DISEASE entry format'\n",
      "Entity 'http://edamontology.org/format_1626' - Label 'KEGG DISEASE entry format'\n",
      "Entity 'http://edamontology.org/format_1627' - Label 'Primer3 primer'\n",
      "Entity 'http://edamontology.org/format_1627' - Label 'Primer3 primer'\n",
      "Entity 'http://edamontology.org/format_1628' - Label 'ABI'\n",
      "Entity 'http://edamontology.org/format_1628' - Label 'ABI'\n",
      "Entity 'http://edamontology.org/format_1629' - Label 'mira'\n",
      "Entity 'http://edamontology.org/format_1629' - Label 'mira'\n",
      "Entity 'http://edamontology.org/format_1630' - Label 'CAF'\n",
      "Entity 'http://edamontology.org/format_1630' - Label 'CAF'\n",
      "Entity 'http://edamontology.org/format_1631' - Label 'EXP'\n",
      "Entity 'http://edamontology.org/format_1631' - Label 'EXP'\n",
      "Entity 'http://edamontology.org/format_1632' - Label 'SCF'\n",
      "Entity 'http://edamontology.org/format_1632' - Label 'SCF'\n",
      "Entity 'http://edamontology.org/format_1633' - Label 'PHD'\n",
      "Entity 'http://edamontology.org/format_1633' - Label 'PHD'\n",
      "Entity 'http://edamontology.org/format_1637' - Label 'dat'\n",
      "Entity 'http://edamontology.org/format_1637' - Label 'dat'\n",
      "Entity 'http://edamontology.org/format_1637' - Label 'dat'\n",
      "Entity 'http://edamontology.org/format_1638' - Label 'cel'\n",
      "Entity 'http://edamontology.org/format_1638' - Label 'cel'\n",
      "Entity 'http://edamontology.org/format_1638' - Label 'cel'\n",
      "Entity 'http://edamontology.org/format_1639' - Label 'affymetrix'\n",
      "Entity 'http://edamontology.org/format_1639' - Label 'affymetrix'\n",
      "Entity 'http://edamontology.org/format_1640' - Label 'ArrayExpress entry format'\n",
      "Entity 'http://edamontology.org/format_1640' - Label 'ArrayExpress entry format'\n",
      "Entity 'http://edamontology.org/format_1641' - Label 'affymetrix-exp'\n",
      "Entity 'http://edamontology.org/format_1641' - Label 'affymetrix-exp'\n",
      "Entity 'http://edamontology.org/format_1644' - Label 'CHP'\n",
      "Entity 'http://edamontology.org/format_1644' - Label 'CHP'\n",
      "Entity 'http://edamontology.org/format_1644' - Label 'CHP'\n",
      "Entity 'http://edamontology.org/format_1645' - Label 'EMDB entry format'\n",
      "Entity 'http://edamontology.org/format_1645' - Label 'EMDB entry format'\n",
      "Entity 'http://edamontology.org/format_1647' - Label 'KEGG PATHWAY entry format'\n",
      "Entity 'http://edamontology.org/format_1647' - Label 'KEGG PATHWAY entry format'\n",
      "Entity 'http://edamontology.org/format_1648' - Label 'MetaCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1648' - Label 'MetaCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1649' - Label 'HumanCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1649' - Label 'HumanCyc entry format'\n",
      "Entity 'http://edamontology.org/format_1650' - Label 'INOH entry format'\n",
      "Entity 'http://edamontology.org/format_1650' - Label 'INOH entry format'\n",
      "Entity 'http://edamontology.org/format_1651' - Label 'PATIKA entry format'\n",
      "Entity 'http://edamontology.org/format_1651' - Label 'PATIKA entry format'\n",
      "Entity 'http://edamontology.org/format_1652' - Label 'Reactome entry format'\n",
      "Entity 'http://edamontology.org/format_1652' - Label 'Reactome entry format'\n",
      "Entity 'http://edamontology.org/format_1653' - Label 'aMAZE entry format'\n",
      "Entity 'http://edamontology.org/format_1653' - Label 'aMAZE entry format'\n",
      "Entity 'http://edamontology.org/format_1654' - Label 'CPDB entry format'\n",
      "Entity 'http://edamontology.org/format_1654' - Label 'CPDB entry format'\n",
      "Entity 'http://edamontology.org/format_1655' - Label 'Panther Pathways entry format'\n",
      "Entity 'http://edamontology.org/format_1655' - Label 'Panther Pathways entry format'\n",
      "Entity 'http://edamontology.org/format_1665' - Label 'Taverna workflow format'\n",
      "Entity 'http://edamontology.org/format_1665' - Label 'Taverna workflow format'\n",
      "Entity 'http://edamontology.org/format_1666' - Label 'BioModel mathematical model format'\n",
      "Entity 'http://edamontology.org/format_1666' - Label 'BioModel mathematical model format'\n",
      "Entity 'http://edamontology.org/format_1697' - Label 'KEGG LIGAND entry format'\n",
      "Entity 'http://edamontology.org/format_1697' - Label 'KEGG LIGAND entry format'\n",
      "Entity 'http://edamontology.org/format_1698' - Label 'KEGG COMPOUND entry format'\n",
      "Entity 'http://edamontology.org/format_1698' - Label 'KEGG COMPOUND entry format'\n",
      "Entity 'http://edamontology.org/format_1699' - Label 'KEGG PLANT entry format'\n",
      "Entity 'http://edamontology.org/format_1699' - Label 'KEGG PLANT entry format'\n",
      "Entity 'http://edamontology.org/format_1700' - Label 'KEGG GLYCAN entry format'\n",
      "Entity 'http://edamontology.org/format_1700' - Label 'KEGG GLYCAN entry format'\n",
      "Entity 'http://edamontology.org/format_1701' - Label 'PubChem entry format'\n",
      "Entity 'http://edamontology.org/format_1701' - Label 'PubChem entry format'\n",
      "Entity 'http://edamontology.org/format_1702' - Label 'ChemSpider entry format'\n",
      "Entity 'http://edamontology.org/format_1702' - Label 'ChemSpider entry format'\n",
      "Entity 'http://edamontology.org/format_1703' - Label 'ChEBI entry format'\n",
      "Entity 'http://edamontology.org/format_1703' - Label 'ChEBI entry format'\n",
      "Entity 'http://edamontology.org/format_1704' - Label 'MSDchem ligand dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1704' - Label 'MSDchem ligand dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1705' - Label 'HET group dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1705' - Label 'HET group dictionary entry format'\n",
      "Entity 'http://edamontology.org/format_1706' - Label 'KEGG DRUG entry format'\n",
      "Entity 'http://edamontology.org/format_1706' - Label 'KEGG DRUG entry format'\n",
      "Entity 'http://edamontology.org/format_1734' - Label 'PubMed citation'\n",
      "Entity 'http://edamontology.org/format_1734' - Label 'PubMed citation'\n",
      "Entity 'http://edamontology.org/format_1735' - Label 'Medline Display Format'\n",
      "Entity 'http://edamontology.org/format_1735' - Label 'Medline Display Format'\n",
      "Entity 'http://edamontology.org/format_1736' - Label 'CiteXplore-core'\n",
      "Entity 'http://edamontology.org/format_1736' - Label 'CiteXplore-core'\n",
      "Entity 'http://edamontology.org/format_1737' - Label 'CiteXplore-all'\n",
      "Entity 'http://edamontology.org/format_1737' - Label 'CiteXplore-all'\n",
      "Entity 'http://edamontology.org/format_1739' - Label 'pmc'\n",
      "Entity 'http://edamontology.org/format_1739' - Label 'pmc'\n",
      "Entity 'http://edamontology.org/format_1740' - Label 'iHOP format'\n",
      "Entity 'http://edamontology.org/format_1740' - Label 'iHOP format'\n",
      "Entity 'http://edamontology.org/format_1740' - Label 'iHOP format'\n",
      "Entity 'http://edamontology.org/format_1741' - Label 'OSCAR format'\n",
      "Entity 'http://edamontology.org/format_1741' - Label 'OSCAR format'\n",
      "Entity 'http://edamontology.org/format_1741' - Label 'OSCAR format'\n",
      "Entity 'http://edamontology.org/format_1747' - Label 'PDB atom record format'\n",
      "Entity 'http://edamontology.org/format_1747' - Label 'PDB atom record format'\n",
      "Entity 'http://edamontology.org/format_1760' - Label 'CATH chain report format'\n",
      "Entity 'http://edamontology.org/format_1760' - Label 'CATH chain report format'\n",
      "Entity 'http://edamontology.org/format_1761' - Label 'CATH PDB report format'\n",
      "Entity 'http://edamontology.org/format_1761' - Label 'CATH PDB report format'\n",
      "Entity 'http://edamontology.org/format_1782' - Label 'NCBI gene report format'\n",
      "Entity 'http://edamontology.org/format_1782' - Label 'NCBI gene report format'\n",
      "Entity 'http://edamontology.org/format_1808' - Label 'GeneIlluminator gene report format'\n",
      "Entity 'http://edamontology.org/format_1808' - Label 'GeneIlluminator gene report format'\n",
      "Entity 'http://edamontology.org/format_1809' - Label 'BacMap gene card format'\n",
      "Entity 'http://edamontology.org/format_1809' - Label 'BacMap gene card format'\n",
      "Entity 'http://edamontology.org/format_1810' - Label 'ColiCard report format'\n",
      "Entity 'http://edamontology.org/format_1810' - Label 'ColiCard report format'\n",
      "Entity 'http://edamontology.org/format_1861' - Label 'PlasMapper TextMap'\n",
      "Entity 'http://edamontology.org/format_1861' - Label 'PlasMapper TextMap'\n",
      "Entity 'http://edamontology.org/format_1910' - Label 'newick'\n",
      "Entity 'http://edamontology.org/format_1910' - Label 'newick'\n",
      "Entity 'http://edamontology.org/format_1911' - Label 'TreeCon format'\n",
      "Entity 'http://edamontology.org/format_1911' - Label 'TreeCon format'\n",
      "Entity 'http://edamontology.org/format_1912' - Label 'Nexus format'\n",
      "Entity 'http://edamontology.org/format_1912' - Label 'Nexus format'\n",
      "Entity 'http://edamontology.org/format_1918' - Label 'Atomic data format'\n",
      "Entity 'http://edamontology.org/format_1918' - Label 'Atomic data format'\n",
      "Entity 'http://edamontology.org/format_1919' - Label 'Sequence record format'\n",
      "Entity 'http://edamontology.org/format_1919' - Label 'Sequence record format'\n",
      "Entity 'http://edamontology.org/format_1920' - Label 'Sequence feature annotation format'\n",
      "Entity 'http://edamontology.org/format_1920' - Label 'Sequence feature annotation format'\n",
      "Entity 'http://edamontology.org/format_1921' - Label 'Alignment format'\n",
      "Entity 'http://edamontology.org/format_1921' - Label 'Alignment format'\n",
      "Entity 'http://edamontology.org/format_1923' - Label 'acedb'\n",
      "Entity 'http://edamontology.org/format_1923' - Label 'acedb'\n",
      "Entity 'http://edamontology.org/format_1924' - Label 'clustal sequence format'\n",
      "Entity 'http://edamontology.org/format_1924' - Label 'clustal sequence format'\n",
      "Entity 'http://edamontology.org/format_1925' - Label 'codata'\n",
      "Entity 'http://edamontology.org/format_1925' - Label 'codata'\n",
      "Entity 'http://edamontology.org/format_1926' - Label 'dbid'\n",
      "Entity 'http://edamontology.org/format_1927' - Label 'EMBL format'\n",
      "Entity 'http://edamontology.org/format_1927' - Label 'EMBL format'\n",
      "Entity 'http://edamontology.org/format_1928' - Label 'Staden experiment format'\n",
      "Entity 'http://edamontology.org/format_1928' - Label 'Staden experiment format'\n",
      "Entity 'http://edamontology.org/format_1929' - Label 'FASTA'\n",
      "Entity 'http://edamontology.org/format_1929' - Label 'FASTA'\n",
      "Entity 'http://edamontology.org/format_1930' - Label 'FASTQ'\n",
      "Entity 'http://edamontology.org/format_1931' - Label 'FASTQ-illumina'\n",
      "Entity 'http://edamontology.org/format_1932' - Label 'FASTQ-sanger'\n",
      "Entity 'http://edamontology.org/format_1933' - Label 'FASTQ-solexa'\n",
      "Entity 'http://edamontology.org/format_1934' - Label 'fitch program'\n",
      "Entity 'http://edamontology.org/format_1934' - Label 'fitch program'\n",
      "Entity 'http://edamontology.org/format_1935' - Label 'GCG'\n",
      "Entity 'http://edamontology.org/format_1935' - Label 'GCG'\n",
      "Entity 'http://edamontology.org/format_1936' - Label 'GenBank format'\n",
      "Entity 'http://edamontology.org/format_1936' - Label 'GenBank format'\n",
      "Entity 'http://edamontology.org/format_1937' - Label 'genpept'\n",
      "Entity 'http://edamontology.org/format_1938' - Label 'GFF2-seq'\n",
      "Entity 'http://edamontology.org/format_1938' - Label 'GFF2-seq'\n",
      "Entity 'http://edamontology.org/format_1939' - Label 'GFF3-seq'\n",
      "Entity 'http://edamontology.org/format_1939' - Label 'GFF3-seq'\n",
      "Entity 'http://edamontology.org/format_1940' - Label 'giFASTA format'\n",
      "Entity 'http://edamontology.org/format_1941' - Label 'hennig86'\n",
      "Entity 'http://edamontology.org/format_1941' - Label 'hennig86'\n",
      "Entity 'http://edamontology.org/format_1942' - Label 'ig'\n",
      "Entity 'http://edamontology.org/format_1942' - Label 'ig'\n",
      "Entity 'http://edamontology.org/format_1943' - Label 'igstrict'\n",
      "Entity 'http://edamontology.org/format_1943' - Label 'igstrict'\n",
      "Entity 'http://edamontology.org/format_1944' - Label 'jackknifer'\n",
      "Entity 'http://edamontology.org/format_1944' - Label 'jackknifer'\n",
      "Entity 'http://edamontology.org/format_1945' - Label 'mase format'\n",
      "Entity 'http://edamontology.org/format_1945' - Label 'mase format'\n",
      "Entity 'http://edamontology.org/format_1946' - Label 'mega-seq'\n",
      "Entity 'http://edamontology.org/format_1946' - Label 'mega-seq'\n",
      "Entity 'http://edamontology.org/format_1947' - Label 'GCG MSF'\n",
      "Entity 'http://edamontology.org/format_1948' - Label 'nbrf/pir'\n",
      "Entity 'http://edamontology.org/format_1949' - Label 'nexus-seq'\n",
      "Entity 'http://edamontology.org/format_1949' - Label 'nexus-seq'\n",
      "Entity 'http://edamontology.org/format_1949' - Label 'nexus-seq'\n",
      "Entity 'http://edamontology.org/format_1950' - Label 'pdbatom'\n",
      "Entity 'http://edamontology.org/format_1950' - Label 'pdbatom'\n",
      "Entity 'http://edamontology.org/format_1950' - Label 'pdbatom'\n",
      "Entity 'http://edamontology.org/format_1951' - Label 'pdbatomnuc'\n",
      "Entity 'http://edamontology.org/format_1951' - Label 'pdbatomnuc'\n",
      "Entity 'http://edamontology.org/format_1951' - Label 'pdbatomnuc'\n",
      "Entity 'http://edamontology.org/format_1952' - Label 'pdbseqresnuc'\n",
      "Entity 'http://edamontology.org/format_1952' - Label 'pdbseqresnuc'\n",
      "Entity 'http://edamontology.org/format_1952' - Label 'pdbseqresnuc'\n",
      "Entity 'http://edamontology.org/format_1953' - Label 'pdbseqres'\n",
      "Entity 'http://edamontology.org/format_1953' - Label 'pdbseqres'\n",
      "Entity 'http://edamontology.org/format_1953' - Label 'pdbseqres'\n",
      "Entity 'http://edamontology.org/format_1954' - Label 'Pearson format'\n",
      "Entity 'http://edamontology.org/format_1955' - Label 'phylip sequence format'\n",
      "Entity 'http://edamontology.org/format_1955' - Label 'phylip sequence format'\n",
      "Entity 'http://edamontology.org/format_1956' - Label 'phylipnon sequence format'\n",
      "Entity 'http://edamontology.org/format_1956' - Label 'phylipnon sequence format'\n",
      "Entity 'http://edamontology.org/format_1957' - Label 'raw'\n",
      "Entity 'http://edamontology.org/format_1957' - Label 'raw'\n",
      "Entity 'http://edamontology.org/format_1958' - Label 'refseqp'\n",
      "Entity 'http://edamontology.org/format_1958' - Label 'refseqp'\n",
      "Entity 'http://edamontology.org/format_1959' - Label 'selex sequence format'\n",
      "Entity 'http://edamontology.org/format_1959' - Label 'selex sequence format'\n",
      "Entity 'http://edamontology.org/format_1960' - Label 'Staden format'\n",
      "Entity 'http://edamontology.org/format_1960' - Label 'Staden format'\n",
      "Entity 'http://edamontology.org/format_1961' - Label 'Stockholm format'\n",
      "Entity 'http://edamontology.org/format_1961' - Label 'Stockholm format'\n",
      "Entity 'http://edamontology.org/format_1962' - Label 'strider format'\n",
      "Entity 'http://edamontology.org/format_1962' - Label 'strider format'\n",
      "Entity 'http://edamontology.org/format_1963' - Label 'UniProtKB format'\n",
      "Entity 'http://edamontology.org/format_1964' - Label 'plain text format (unformatted)'\n",
      "Entity 'http://edamontology.org/format_1965' - Label 'treecon sequence format'\n",
      "Entity 'http://edamontology.org/format_1965' - Label 'treecon sequence format'\n",
      "Entity 'http://edamontology.org/format_1966' - Label 'ASN.1 sequence format'\n",
      "Entity 'http://edamontology.org/format_1966' - Label 'ASN.1 sequence format'\n",
      "Entity 'http://edamontology.org/format_1967' - Label 'DAS format'\n",
      "Entity 'http://edamontology.org/format_1967' - Label 'DAS format'\n",
      "Entity 'http://edamontology.org/format_1968' - Label 'dasdna'\n",
      "Entity 'http://edamontology.org/format_1968' - Label 'dasdna'\n",
      "Entity 'http://edamontology.org/format_1969' - Label 'debug-seq'\n",
      "Entity 'http://edamontology.org/format_1969' - Label 'debug-seq'\n",
      "Entity 'http://edamontology.org/format_1970' - Label 'jackknifernon'\n",
      "Entity 'http://edamontology.org/format_1970' - Label 'jackknifernon'\n",
      "Entity 'http://edamontology.org/format_1971' - Label 'meganon sequence format'\n",
      "Entity 'http://edamontology.org/format_1971' - Label 'meganon sequence format'\n",
      "Entity 'http://edamontology.org/format_1972' - Label 'NCBI format'\n",
      "Entity 'http://edamontology.org/format_1973' - Label 'nexusnon'\n",
      "Entity 'http://edamontology.org/format_1973' - Label 'nexusnon'\n",
      "Entity 'http://edamontology.org/format_1973' - Label 'nexusnon'\n",
      "Entity 'http://edamontology.org/format_1974' - Label 'GFF2'\n",
      "Entity 'http://edamontology.org/format_1975' - Label 'GFF3'\n",
      "Entity 'http://edamontology.org/format_1976' - Label 'pir'\n",
      "Entity 'http://edamontology.org/format_1976' - Label 'pir'\n",
      "Entity 'http://edamontology.org/format_1977' - Label 'swiss feature'\n",
      "Entity 'http://edamontology.org/format_1977' - Label 'swiss feature'\n",
      "Entity 'http://edamontology.org/format_1978' - Label 'DASGFF'\n",
      "Entity 'http://edamontology.org/format_1978' - Label 'DASGFF'\n",
      "Entity 'http://edamontology.org/format_1979' - Label 'debug-feat'\n",
      "Entity 'http://edamontology.org/format_1979' - Label 'debug-feat'\n",
      "Entity 'http://edamontology.org/format_1980' - Label 'EMBL feature'\n",
      "Entity 'http://edamontology.org/format_1980' - Label 'EMBL feature'\n",
      "Entity 'http://edamontology.org/format_1981' - Label 'GenBank feature'\n",
      "Entity 'http://edamontology.org/format_1981' - Label 'GenBank feature'\n",
      "Entity 'http://edamontology.org/format_1982' - Label 'ClustalW format'\n",
      "Entity 'http://edamontology.org/format_1982' - Label 'ClustalW format'\n",
      "Entity 'http://edamontology.org/format_1983' - Label 'debug'\n",
      "Entity 'http://edamontology.org/format_1983' - Label 'debug'\n",
      "Entity 'http://edamontology.org/format_1984' - Label 'FASTA-aln'\n",
      "Entity 'http://edamontology.org/format_1984' - Label 'FASTA-aln'\n",
      "Entity 'http://edamontology.org/format_1985' - Label 'markx0'\n",
      "Entity 'http://edamontology.org/format_1986' - Label 'markx1'\n",
      "Entity 'http://edamontology.org/format_1987' - Label 'markx10'\n",
      "Entity 'http://edamontology.org/format_1988' - Label 'markx2'\n",
      "Entity 'http://edamontology.org/format_1989' - Label 'markx3'\n",
      "Entity 'http://edamontology.org/format_1990' - Label 'match'\n",
      "Entity 'http://edamontology.org/format_1990' - Label 'match'\n",
      "Entity 'http://edamontology.org/format_1991' - Label 'mega'\n",
      "Entity 'http://edamontology.org/format_1992' - Label 'meganon'\n",
      "Entity 'http://edamontology.org/format_1993' - Label 'msf alignment format'\n",
      "Entity 'http://edamontology.org/format_1993' - Label 'msf alignment format'\n",
      "Entity 'http://edamontology.org/format_1994' - Label 'nexus alignment format'\n",
      "Entity 'http://edamontology.org/format_1994' - Label 'nexus alignment format'\n",
      "Entity 'http://edamontology.org/format_1995' - Label 'nexusnon alignment format'\n",
      "Entity 'http://edamontology.org/format_1995' - Label 'nexusnon alignment format'\n",
      "Entity 'http://edamontology.org/format_1996' - Label 'pair'\n",
      "Entity 'http://edamontology.org/format_1997' - Label 'PHYLIP format'\n",
      "Entity 'http://edamontology.org/format_1998' - Label 'PHYLIP sequential'\n",
      "Entity 'http://edamontology.org/format_1999' - Label 'scores format'\n",
      "Entity 'http://edamontology.org/format_1999' - Label 'scores format'\n",
      "Entity 'http://edamontology.org/format_2000' - Label 'selex'\n",
      "Entity 'http://edamontology.org/format_2000' - Label 'selex'\n",
      "Entity 'http://edamontology.org/format_2000' - Label 'selex'\n",
      "Entity 'http://edamontology.org/format_2001' - Label 'EMBOSS simple format'\n",
      "Entity 'http://edamontology.org/format_2001' - Label 'EMBOSS simple format'\n",
      "Entity 'http://edamontology.org/format_2002' - Label 'srs format'\n",
      "Entity 'http://edamontology.org/format_2002' - Label 'srs format'\n",
      "Entity 'http://edamontology.org/format_2003' - Label 'srspair'\n",
      "Entity 'http://edamontology.org/format_2003' - Label 'srspair'\n",
      "Entity 'http://edamontology.org/format_2004' - Label 'T-Coffee format'\n",
      "Entity 'http://edamontology.org/format_2004' - Label 'T-Coffee format'\n",
      "Entity 'http://edamontology.org/format_2005' - Label 'TreeCon-seq'\n",
      "Entity 'http://edamontology.org/format_2005' - Label 'TreeCon-seq'\n",
      "Entity 'http://edamontology.org/format_2005' - Label 'TreeCon-seq'\n",
      "Entity 'http://edamontology.org/format_2006' - Label 'Phylogenetic tree format'\n",
      "Entity 'http://edamontology.org/format_2006' - Label 'Phylogenetic tree format'\n",
      "Entity 'http://edamontology.org/format_2013' - Label 'Biological pathway or network format'\n",
      "Entity 'http://edamontology.org/format_2013' - Label 'Biological pathway or network format'\n",
      "Entity 'http://edamontology.org/format_2014' - Label 'Sequence-profile alignment format'\n",
      "Entity 'http://edamontology.org/format_2014' - Label 'Sequence-profile alignment format'\n",
      "Entity 'http://edamontology.org/format_2015' - Label 'Sequence-profile alignment (HMM) format'\n",
      "Entity 'http://edamontology.org/format_2015' - Label 'Sequence-profile alignment (HMM) format'\n",
      "Entity 'http://edamontology.org/format_2017' - Label 'Amino acid index format'\n",
      "Entity 'http://edamontology.org/format_2017' - Label 'Amino acid index format'\n",
      "Entity 'http://edamontology.org/format_2020' - Label 'Article format'\n",
      "Entity 'http://edamontology.org/format_2020' - Label 'Article format'\n",
      "Entity 'http://edamontology.org/format_2021' - Label 'Text mining report format'\n",
      "Entity 'http://edamontology.org/format_2021' - Label 'Text mining report format'\n",
      "Entity 'http://edamontology.org/format_2027' - Label 'Enzyme kinetics report format'\n",
      "Entity 'http://edamontology.org/format_2027' - Label 'Enzyme kinetics report format'\n",
      "Entity 'http://edamontology.org/format_2030' - Label 'Chemical data format'\n",
      "Entity 'http://edamontology.org/format_2030' - Label 'Chemical data format'\n",
      "Entity 'http://edamontology.org/format_2031' - Label 'Gene annotation format'\n",
      "Entity 'http://edamontology.org/format_2031' - Label 'Gene annotation format'\n",
      "Entity 'http://edamontology.org/format_2032' - Label 'Workflow format'\n",
      "Entity 'http://edamontology.org/format_2033' - Label 'Tertiary structure format'\n",
      "Entity 'http://edamontology.org/format_2034' - Label 'Biological model format'\n",
      "Entity 'http://edamontology.org/format_2034' - Label 'Biological model format'\n",
      "Entity 'http://edamontology.org/format_2035' - Label 'Chemical formula format'\n",
      "Entity 'http://edamontology.org/format_2035' - Label 'Chemical formula format'\n",
      "Entity 'http://edamontology.org/format_2036' - Label 'Phylogenetic character data format'\n",
      "Entity 'http://edamontology.org/format_2036' - Label 'Phylogenetic character data format'\n",
      "Entity 'http://edamontology.org/format_2037' - Label 'Phylogenetic continuous quantitative character format'\n",
      "Entity 'http://edamontology.org/format_2037' - Label 'Phylogenetic continuous quantitative character format'\n",
      "Entity 'http://edamontology.org/format_2038' - Label 'Phylogenetic discrete states format'\n",
      "Entity 'http://edamontology.org/format_2038' - Label 'Phylogenetic discrete states format'\n",
      "Entity 'http://edamontology.org/format_2039' - Label 'Phylogenetic tree report (cliques) format'\n",
      "Entity 'http://edamontology.org/format_2039' - Label 'Phylogenetic tree report (cliques) format'\n",
      "Entity 'http://edamontology.org/format_2040' - Label 'Phylogenetic tree report (invariants) format'\n",
      "Entity 'http://edamontology.org/format_2040' - Label 'Phylogenetic tree report (invariants) format'\n",
      "Entity 'http://edamontology.org/format_2045' - Label 'Electron microscopy model format'\n",
      "Entity 'http://edamontology.org/format_2045' - Label 'Electron microscopy model format'\n",
      "Entity 'http://edamontology.org/format_2049' - Label 'Phylogenetic tree report (tree distances) format'\n",
      "Entity 'http://edamontology.org/format_2049' - Label 'Phylogenetic tree report (tree distances) format'\n",
      "Entity 'http://edamontology.org/format_2051' - Label 'Polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_2051' - Label 'Polymorphism report format'\n",
      "Entity 'http://edamontology.org/format_2052' - Label 'Protein family report format'\n",
      "Entity 'http://edamontology.org/format_2052' - Label 'Protein family report format'\n",
      "Entity 'http://edamontology.org/format_2054' - Label 'Protein interaction format'\n",
      "Entity 'http://edamontology.org/format_2054' - Label 'Protein interaction format'\n",
      "Entity 'http://edamontology.org/format_2055' - Label 'Sequence assembly format'\n",
      "Entity 'http://edamontology.org/format_2055' - Label 'Sequence assembly format'\n",
      "Entity 'http://edamontology.org/format_2056' - Label 'Microarray experiment data format'\n",
      "Entity 'http://edamontology.org/format_2057' - Label 'Sequence trace format'\n",
      "Entity 'http://edamontology.org/format_2057' - Label 'Sequence trace format'\n",
      "Entity 'http://edamontology.org/format_2058' - Label 'Gene expression report format'\n",
      "Entity 'http://edamontology.org/format_2058' - Label 'Gene expression report format'\n",
      "Entity 'http://edamontology.org/format_2059' - Label 'Genotype and phenotype annotation format'\n",
      "Entity 'http://edamontology.org/format_2059' - Label 'Genotype and phenotype annotation format'\n",
      "Entity 'http://edamontology.org/format_2060' - Label 'Map format'\n",
      "Entity 'http://edamontology.org/format_2060' - Label 'Map format'\n",
      "Entity 'http://edamontology.org/format_2061' - Label 'Nucleic acid features (primers) format'\n",
      "Entity 'http://edamontology.org/format_2062' - Label 'Protein report format'\n",
      "Entity 'http://edamontology.org/format_2062' - Label 'Protein report format'\n",
      "Entity 'http://edamontology.org/format_2063' - Label 'Protein report (enzyme) format'\n",
      "Entity 'http://edamontology.org/format_2063' - Label 'Protein report (enzyme) format'\n",
      "Entity 'http://edamontology.org/format_2064' - Label '3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_2064' - Label '3D-1D scoring matrix format'\n",
      "Entity 'http://edamontology.org/format_2065' - Label 'Protein structure report (quality evaluation) format'\n",
      "Entity 'http://edamontology.org/format_2065' - Label 'Protein structure report (quality evaluation) format'\n",
      "Entity 'http://edamontology.org/format_2066' - Label 'Database hits (sequence) format'\n",
      "Entity 'http://edamontology.org/format_2066' - Label 'Database hits (sequence) format'\n",
      "Entity 'http://edamontology.org/format_2067' - Label 'Sequence distance matrix format'\n",
      "Entity 'http://edamontology.org/format_2067' - Label 'Sequence distance matrix format'\n",
      "Entity 'http://edamontology.org/format_2068' - Label 'Sequence motif format'\n",
      "Entity 'http://edamontology.org/format_2068' - Label 'Sequence motif format'\n",
      "Entity 'http://edamontology.org/format_2069' - Label 'Sequence profile format'\n",
      "Entity 'http://edamontology.org/format_2069' - Label 'Sequence profile format'\n",
      "Entity 'http://edamontology.org/format_2072' - Label 'Hidden Markov model format'\n",
      "Entity 'http://edamontology.org/format_2072' - Label 'Hidden Markov model format'\n",
      "Entity 'http://edamontology.org/format_2074' - Label 'Dirichlet distribution format'\n",
      "Entity 'http://edamontology.org/format_2074' - Label 'Dirichlet distribution format'\n",
      "Entity 'http://edamontology.org/format_2075' - Label 'HMM emission and transition counts format'\n",
      "Entity 'http://edamontology.org/format_2075' - Label 'HMM emission and transition counts format'\n",
      "Entity 'http://edamontology.org/format_2075' - Label 'HMM emission and transition counts format'\n",
      "Entity 'http://edamontology.org/format_2076' - Label 'RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2076' - Label 'RNA secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2077' - Label 'Protein secondary structure format'\n",
      "Entity 'http://edamontology.org/format_2078' - Label 'Sequence range format'\n",
      "Entity 'http://edamontology.org/format_2078' - Label 'Sequence range format'\n",
      "Entity 'http://edamontology.org/format_2094' - Label 'pure'\n",
      "Entity 'http://edamontology.org/format_2094' - Label 'pure'\n",
      "Entity 'http://edamontology.org/format_2095' - Label 'unpure'\n",
      "Entity 'http://edamontology.org/format_2095' - Label 'unpure'\n",
      "Entity 'http://edamontology.org/format_2096' - Label 'unambiguous sequence'\n",
      "Entity 'http://edamontology.org/format_2096' - Label 'unambiguous sequence'\n",
      "Entity 'http://edamontology.org/format_2097' - Label 'ambiguous'\n",
      "Entity 'http://edamontology.org/format_2097' - Label 'ambiguous'\n",
      "Entity 'http://edamontology.org/format_2155' - Label 'Sequence features (repeats) format'\n",
      "Entity 'http://edamontology.org/format_2158' - Label 'Nucleic acid features (restriction sites) format'\n",
      "Entity 'http://edamontology.org/format_2159' - Label 'Gene features (coding region) format'\n",
      "Entity 'http://edamontology.org/format_2159' - Label 'Gene features (coding region) format'\n",
      "Entity 'http://edamontology.org/format_2170' - Label 'Sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_2170' - Label 'Sequence cluster format'\n",
      "Entity 'http://edamontology.org/format_2171' - Label 'Sequence cluster format (protein)'\n",
      "Entity 'http://edamontology.org/format_2172' - Label 'Sequence cluster format (nucleic acid)'\n",
      "Entity 'http://edamontology.org/format_2175' - Label 'Gene cluster format'\n",
      "Entity 'http://edamontology.org/format_2175' - Label 'Gene cluster format'\n",
      "Entity 'http://edamontology.org/format_2181' - Label 'EMBL-like (text)'\n",
      "Entity 'http://edamontology.org/format_2181' - Label 'EMBL-like (text)'\n",
      "Entity 'http://edamontology.org/format_2182' - Label 'FASTQ-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2182' - Label 'FASTQ-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2183' - Label 'EMBLXML'\n",
      "Entity 'http://edamontology.org/format_2184' - Label 'cdsxml'\n",
      "Entity 'http://edamontology.org/format_2185' - Label 'INSDSeq'\n",
      "Entity 'http://edamontology.org/format_2186' - Label 'geneseq'\n",
      "Entity 'http://edamontology.org/format_2187' - Label 'UniProt-like (text)'\n",
      "Entity 'http://edamontology.org/format_2187' - Label 'UniProt-like (text)'\n",
      "Entity 'http://edamontology.org/format_2188' - Label 'UniProt format'\n",
      "Entity 'http://edamontology.org/format_2188' - Label 'UniProt format'\n",
      "Entity 'http://edamontology.org/format_2189' - Label 'ipi'\n",
      "Entity 'http://edamontology.org/format_2189' - Label 'ipi'\n",
      "Entity 'http://edamontology.org/format_2194' - Label 'medline'\n",
      "Entity 'http://edamontology.org/format_2194' - Label 'medline'\n",
      "Entity 'http://edamontology.org/format_2195' - Label 'Ontology format'\n",
      "Entity 'http://edamontology.org/format_2195' - Label 'Ontology format'\n",
      "Entity 'http://edamontology.org/format_2196' - Label 'OBO format'\n",
      "Entity 'http://edamontology.org/format_2197' - Label 'OWL format'\n",
      "Entity 'http://edamontology.org/format_2197' - Label 'OWL format'\n",
      "Entity 'http://edamontology.org/format_2200' - Label 'FASTA-like (text)'\n",
      "Entity 'http://edamontology.org/format_2200' - Label 'FASTA-like (text)'\n",
      "Entity 'http://edamontology.org/format_2202' - Label 'Sequence record full format'\n",
      "Entity 'http://edamontology.org/format_2202' - Label 'Sequence record full format'\n",
      "Entity 'http://edamontology.org/format_2203' - Label 'Sequence record lite format'\n",
      "Entity 'http://edamontology.org/format_2203' - Label 'Sequence record lite format'\n",
      "Entity 'http://edamontology.org/format_2204' - Label 'EMBL format (XML)'\n",
      "Entity 'http://edamontology.org/format_2205' - Label 'GenBank-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2205' - Label 'GenBank-like format (text)'\n",
      "Entity 'http://edamontology.org/format_2206' - Label 'Sequence feature table format (text)'\n",
      "Entity 'http://edamontology.org/format_2210' - Label 'Strain data format'\n",
      "Entity 'http://edamontology.org/format_2210' - Label 'Strain data format'\n",
      "Entity 'http://edamontology.org/format_2211' - Label 'CIP strain data format'\n",
      "Entity 'http://edamontology.org/format_2211' - Label 'CIP strain data format'\n",
      "Entity 'http://edamontology.org/format_2243' - Label 'phylip property values'\n",
      "Entity 'http://edamontology.org/format_2243' - Label 'phylip property values'\n",
      "Entity 'http://edamontology.org/format_2303' - Label 'STRING entry format (HTML)'\n",
      "Entity 'http://edamontology.org/format_2303' - Label 'STRING entry format (HTML)'\n",
      "Entity 'http://edamontology.org/format_2304' - Label 'STRING entry format (XML)'\n",
      "Entity 'http://edamontology.org/format_2304' - Label 'STRING entry format (XML)'\n",
      "Entity 'http://edamontology.org/format_2305' - Label 'GFF'\n",
      "Entity 'http://edamontology.org/format_2305' - Label 'GFF'\n",
      "Entity 'http://edamontology.org/format_2306' - Label 'GTF'\n",
      "Entity 'http://edamontology.org/format_2310' - Label 'FASTA-HTML'\n",
      "Entity 'http://edamontology.org/format_2310' - Label 'FASTA-HTML'\n",
      "Entity 'http://edamontology.org/format_2311' - Label 'EMBL-HTML'\n",
      "Entity 'http://edamontology.org/format_2311' - Label 'EMBL-HTML'\n",
      "Entity 'http://edamontology.org/format_2322' - Label 'BioCyc enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2322' - Label 'BioCyc enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2323' - Label 'ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2323' - Label 'ENZYME enzyme report format'\n",
      "Entity 'http://edamontology.org/format_2328' - Label 'PseudoCAP gene report format'\n",
      "Entity 'http://edamontology.org/format_2328' - Label 'PseudoCAP gene report format'\n",
      "Entity 'http://edamontology.org/format_2329' - Label 'GeneCards gene report format'\n",
      "Entity 'http://edamontology.org/format_2329' - Label 'GeneCards gene report format'\n",
      "Entity 'http://edamontology.org/format_2330' - Label 'Textual format'\n",
      "Entity 'http://edamontology.org/format_2331' - Label 'HTML'\n",
      "Entity 'http://edamontology.org/format_2331' - Label 'HTML'\n",
      "Entity 'http://edamontology.org/format_2332' - Label 'XML'\n",
      "Entity 'http://edamontology.org/format_2333' - Label 'Binary format'\n",
      "Entity 'http://edamontology.org/format_2334' - Label 'URI format'\n",
      "Entity 'http://edamontology.org/format_2334' - Label 'URI format'\n",
      "Entity 'http://edamontology.org/format_2341' - Label 'NCI-Nature pathway entry format'\n",
      "Entity 'http://edamontology.org/format_2341' - Label 'NCI-Nature pathway entry format'\n",
      "Entity 'http://edamontology.org/format_2350' - Label 'Format (by type of data)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2352' - Label 'BioXSD (XML)'\n",
      "Entity 'http://edamontology.org/format_2376' - Label 'RDF format'\n",
      "Entity 'http://edamontology.org/format_2376' - Label 'RDF format'\n",
      "Entity 'http://edamontology.org/format_2376' - Label 'RDF format'\n",
      "Entity 'http://edamontology.org/format_2532' - Label 'GenBank-HTML'\n",
      "Entity 'http://edamontology.org/format_2532' - Label 'GenBank-HTML'\n",
      "Entity 'http://edamontology.org/format_2542' - Label 'Protein features (domains) format'\n",
      "Entity 'http://edamontology.org/format_2542' - Label 'Protein features (domains) format'\n",
      "Entity 'http://edamontology.org/format_2543' - Label 'EMBL-like format'\n",
      "Entity 'http://edamontology.org/format_2545' - Label 'FASTQ-like format'\n",
      "Entity 'http://edamontology.org/format_2546' - Label 'FASTA-like'\n",
      "Entity 'http://edamontology.org/format_2547' - Label 'uniprotkb-like format'\n",
      "Entity 'http://edamontology.org/format_2547' - Label 'uniprotkb-like format'\n",
      "Entity 'http://edamontology.org/format_2548' - Label 'Sequence feature table format'\n",
      "Entity 'http://edamontology.org/format_2548' - Label 'Sequence feature table format'\n",
      "Entity 'http://edamontology.org/format_2549' - Label 'OBO'\n",
      "Entity 'http://edamontology.org/format_2549' - Label 'OBO'\n",
      "Entity 'http://edamontology.org/format_2550' - Label 'OBO-XML'\n",
      "Entity 'http://edamontology.org/format_2550' - Label 'OBO-XML'\n",
      "Entity 'http://edamontology.org/format_2551' - Label 'Sequence record format (text)'\n",
      "Entity 'http://edamontology.org/format_2552' - Label 'Sequence record format (XML)'\n",
      "Entity 'http://edamontology.org/format_2553' - Label 'Sequence feature table format (XML)'\n",
      "Entity 'http://edamontology.org/format_2554' - Label 'Alignment format (text)'\n",
      "Entity 'http://edamontology.org/format_2555' - Label 'Alignment format (XML)'\n",
      "Entity 'http://edamontology.org/format_2556' - Label 'Phylogenetic tree format (text)'\n",
      "Entity 'http://edamontology.org/format_2557' - Label 'Phylogenetic tree format (XML)'\n",
      "Entity 'http://edamontology.org/format_2558' - Label 'EMBL-like (XML)'\n",
      "Entity 'http://edamontology.org/format_2558' - Label 'EMBL-like (XML)'\n",
      "Entity 'http://edamontology.org/format_2559' - Label 'GenBank-like format'\n",
      "Entity 'http://edamontology.org/format_2560' - Label 'STRING entry format'\n",
      "Entity 'http://edamontology.org/format_2560' - Label 'STRING entry format'\n",
      "Entity 'http://edamontology.org/format_2561' - Label 'Sequence assembly format (text)'\n",
      "Entity 'http://edamontology.org/format_2562' - Label 'Amino acid identifier format'\n",
      "Entity 'http://edamontology.org/format_2562' - Label 'Amino acid identifier format'\n",
      "Entity 'http://edamontology.org/format_2566' - Label 'completely unambiguous'\n",
      "Entity 'http://edamontology.org/format_2566' - Label 'completely unambiguous'\n",
      "Entity 'http://edamontology.org/format_2567' - Label 'completely unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_2567' - Label 'completely unambiguous pure'\n",
      "Entity 'http://edamontology.org/format_2568' - Label 'completely unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_2568' - Label 'completely unambiguous pure nucleotide'\n",
      "Entity 'http://edamontology.org/format_2569' - Label 'completely unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_2569' - Label 'completely unambiguous pure dna'\n",
      "Entity 'http://edamontology.org/format_2570' - Label 'completely unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_2570' - Label 'completely unambiguous pure rna sequence'\n",
      "Entity 'http://edamontology.org/format_2571' - Label 'Raw sequence format'\n",
      "Entity 'http://edamontology.org/format_2571' - Label 'Raw sequence format'\n",
      "Entity 'http://edamontology.org/format_2572' - Label 'BAM'\n",
      "Entity 'http://edamontology.org/format_2572' - Label 'BAM'\n",
      "Entity 'http://edamontology.org/format_2572' - Label 'BAM'\n",
      "Entity 'http://edamontology.org/format_2573' - Label 'SAM'\n",
      "Entity 'http://edamontology.org/format_2573' - Label 'SAM'\n",
      "Entity 'http://edamontology.org/format_2573' - Label 'SAM'\n",
      "Entity 'http://edamontology.org/format_2585' - Label 'SBML'\n",
      "Entity 'http://edamontology.org/format_2585' - Label 'SBML'\n",
      "Entity 'http://edamontology.org/format_2607' - Label 'completely unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_2607' - Label 'completely unambiguous pure protein'\n",
      "Entity 'http://edamontology.org/format_2848' - Label 'Bibliographic reference format'\n",
      "Entity 'http://edamontology.org/format_2848' - Label 'Bibliographic reference format'\n",
      "Entity 'http://edamontology.org/format_2848' - Label 'Bibliographic reference format'\n",
      "Entity 'http://edamontology.org/format_2919' - Label 'Sequence annotation track format'\n",
      "Entity 'http://edamontology.org/format_2919' - Label 'Sequence annotation track format'\n",
      "Entity 'http://edamontology.org/format_2920' - Label 'Alignment format (pair only)'\n",
      "Entity 'http://edamontology.org/format_2920' - Label 'Alignment format (pair only)'\n",
      "Entity 'http://edamontology.org/format_2921' - Label 'Sequence variation annotation format'\n",
      "Entity 'http://edamontology.org/format_2921' - Label 'Sequence variation annotation format'\n",
      "Entity 'http://edamontology.org/format_2922' - Label 'markx0 variant'\n",
      "Entity 'http://edamontology.org/format_2922' - Label 'markx0 variant'\n",
      "Entity 'http://edamontology.org/format_2923' - Label 'mega variant'\n",
      "Entity 'http://edamontology.org/format_2923' - Label 'mega variant'\n",
      "Entity 'http://edamontology.org/format_2923' - Label 'mega variant'\n",
      "Entity 'http://edamontology.org/format_2924' - Label 'Phylip format variant'\n",
      "Entity 'http://edamontology.org/format_2924' - Label 'Phylip format variant'\n",
      "Entity 'http://edamontology.org/format_2924' - Label 'Phylip format variant'\n",
      "Entity 'http://edamontology.org/format_3000' - Label 'AB1'\n",
      "Entity 'http://edamontology.org/format_3000' - Label 'AB1'\n",
      "Entity 'http://edamontology.org/format_3001' - Label 'ACE'\n",
      "Entity 'http://edamontology.org/format_3001' - Label 'ACE'\n",
      "Entity 'http://edamontology.org/format_3003' - Label 'BED'\n",
      "Entity 'http://edamontology.org/format_3003' - Label 'BED'\n",
      "Entity 'http://edamontology.org/format_3004' - Label 'bigBed'\n",
      "Entity 'http://edamontology.org/format_3004' - Label 'bigBed'\n",
      "Entity 'http://edamontology.org/format_3005' - Label 'WIG'\n",
      "Entity 'http://edamontology.org/format_3005' - Label 'WIG'\n",
      "Entity 'http://edamontology.org/format_3006' - Label 'bigWig'\n",
      "Entity 'http://edamontology.org/format_3006' - Label 'bigWig'\n",
      "Entity 'http://edamontology.org/format_3007' - Label 'PSL'\n",
      "Entity 'http://edamontology.org/format_3007' - Label 'PSL'\n",
      "Entity 'http://edamontology.org/format_3007' - Label 'PSL'\n",
      "Entity 'http://edamontology.org/format_3008' - Label 'MAF'\n",
      "Entity 'http://edamontology.org/format_3008' - Label 'MAF'\n",
      "Entity 'http://edamontology.org/format_3008' - Label 'MAF'\n",
      "Entity 'http://edamontology.org/format_3009' - Label '2bit'\n",
      "Entity 'http://edamontology.org/format_3009' - Label '2bit'\n",
      "Entity 'http://edamontology.org/format_3010' - Label '.nib'\n",
      "Entity 'http://edamontology.org/format_3010' - Label '.nib'\n",
      "Entity 'http://edamontology.org/format_3011' - Label 'genePred'\n",
      "Entity 'http://edamontology.org/format_3011' - Label 'genePred'\n",
      "Entity 'http://edamontology.org/format_3012' - Label 'pgSnp'\n",
      "Entity 'http://edamontology.org/format_3012' - Label 'pgSnp'\n",
      "Entity 'http://edamontology.org/format_3013' - Label 'axt'\n",
      "Entity 'http://edamontology.org/format_3013' - Label 'axt'\n",
      "Entity 'http://edamontology.org/format_3014' - Label 'LAV'\n",
      "Entity 'http://edamontology.org/format_3014' - Label 'LAV'\n",
      "Entity 'http://edamontology.org/format_3015' - Label 'Pileup'\n",
      "Entity 'http://edamontology.org/format_3015' - Label 'Pileup'\n",
      "Entity 'http://edamontology.org/format_3016' - Label 'VCF'\n",
      "Entity 'http://edamontology.org/format_3016' - Label 'VCF'\n",
      "Entity 'http://edamontology.org/format_3017' - Label 'SRF'\n",
      "Entity 'http://edamontology.org/format_3017' - Label 'SRF'\n",
      "Entity 'http://edamontology.org/format_3018' - Label 'ZTR'\n",
      "Entity 'http://edamontology.org/format_3018' - Label 'ZTR'\n",
      "Entity 'http://edamontology.org/format_3019' - Label 'GVF'\n",
      "Entity 'http://edamontology.org/format_3019' - Label 'GVF'\n",
      "Entity 'http://edamontology.org/format_3020' - Label 'BCF'\n",
      "Entity 'http://edamontology.org/format_3020' - Label 'BCF'\n",
      "Entity 'http://edamontology.org/format_3033' - Label 'Matrix format'\n",
      "Entity 'http://edamontology.org/format_3033' - Label 'Matrix format'\n",
      "Entity 'http://edamontology.org/format_3097' - Label 'Protein domain classification format'\n",
      "Entity 'http://edamontology.org/format_3097' - Label 'Protein domain classification format'\n",
      "Entity 'http://edamontology.org/format_3098' - Label 'Raw SCOP domain classification format'\n",
      "Entity 'http://edamontology.org/format_3099' - Label 'Raw CATH domain classification format'\n",
      "Entity 'http://edamontology.org/format_3100' - Label 'CATH domain report format'\n",
      "Entity 'http://edamontology.org/format_3155' - Label 'SBRML'\n",
      "Entity 'http://edamontology.org/format_3155' - Label 'SBRML'\n",
      "Entity 'http://edamontology.org/format_3156' - Label 'BioPAX'\n",
      "Entity 'http://edamontology.org/format_3157' - Label 'EBI Application Result XML'\n",
      "Entity 'http://edamontology.org/format_3157' - Label 'EBI Application Result XML'\n",
      "Entity 'http://edamontology.org/format_3157' - Label 'EBI Application Result XML'\n",
      "Entity 'http://edamontology.org/format_3158' - Label 'PSI MI XML (MIF)'\n",
      "Entity 'http://edamontology.org/format_3158' - Label 'PSI MI XML (MIF)'\n",
      "Entity 'http://edamontology.org/format_3159' - Label 'phyloXML'\n",
      "Entity 'http://edamontology.org/format_3159' - Label 'phyloXML'\n",
      "Entity 'http://edamontology.org/format_3160' - Label 'NeXML'\n",
      "Entity 'http://edamontology.org/format_3160' - Label 'NeXML'\n",
      "Entity 'http://edamontology.org/format_3161' - Label 'MAGE-ML'\n",
      "Entity 'http://edamontology.org/format_3161' - Label 'MAGE-ML'\n",
      "Entity 'http://edamontology.org/format_3161' - Label 'MAGE-ML'\n",
      "Entity 'http://edamontology.org/format_3162' - Label 'MAGE-TAB'\n",
      "Entity 'http://edamontology.org/format_3162' - Label 'MAGE-TAB'\n",
      "Entity 'http://edamontology.org/format_3162' - Label 'MAGE-TAB'\n",
      "Entity 'http://edamontology.org/format_3163' - Label 'GCDML'\n",
      "Entity 'http://edamontology.org/format_3163' - Label 'GCDML'\n",
      "Entity 'http://edamontology.org/format_3164' - Label 'GTrack'\n",
      "Entity 'http://edamontology.org/format_3164' - Label 'GTrack'\n",
      "Entity 'http://edamontology.org/format_3164' - Label 'GTrack'\n",
      "Entity 'http://edamontology.org/format_3166' - Label 'Biological pathway or network report format'\n",
      "Entity 'http://edamontology.org/format_3166' - Label 'Biological pathway or network report format'\n",
      "Entity 'http://edamontology.org/format_3167' - Label 'Experiment annotation format'\n",
      "Entity 'http://edamontology.org/format_3167' - Label 'Experiment annotation format'\n",
      "Entity 'http://edamontology.org/format_3235' - Label 'Cytoband format'\n",
      "Entity 'http://edamontology.org/format_3235' - Label 'Cytoband format'\n",
      "Entity 'http://edamontology.org/format_3235' - Label 'Cytoband format'\n",
      "Entity 'http://edamontology.org/format_3239' - Label 'CopasiML'\n",
      "Entity 'http://edamontology.org/format_3239' - Label 'CopasiML'\n",
      "Entity 'http://edamontology.org/format_3239' - Label 'CopasiML'\n",
      "Entity 'http://edamontology.org/format_3240' - Label 'CellML'\n",
      "Entity 'http://edamontology.org/format_3240' - Label 'CellML'\n",
      "Entity 'http://edamontology.org/format_3242' - Label 'PSI MI TAB (MITAB)'\n",
      "Entity 'http://edamontology.org/format_3242' - Label 'PSI MI TAB (MITAB)'\n",
      "Entity 'http://edamontology.org/format_3243' - Label 'PSI-PAR'\n",
      "Entity 'http://edamontology.org/format_3244' - Label 'mzML'\n",
      "Entity 'http://edamontology.org/format_3244' - Label 'mzML'\n",
      "Entity 'http://edamontology.org/format_3245' - Label 'Mass spectrometry data format'\n",
      "Entity 'http://edamontology.org/format_3245' - Label 'Mass spectrometry data format'\n",
      "Entity 'http://edamontology.org/format_3246' - Label 'TraML'\n",
      "Entity 'http://edamontology.org/format_3246' - Label 'TraML'\n",
      "Entity 'http://edamontology.org/format_3247' - Label 'mzIdentML'\n",
      "Entity 'http://edamontology.org/format_3247' - Label 'mzIdentML'\n",
      "Entity 'http://edamontology.org/format_3248' - Label 'mzQuantML'\n",
      "Entity 'http://edamontology.org/format_3248' - Label 'mzQuantML'\n",
      "Entity 'http://edamontology.org/format_3249' - Label 'GelML'\n",
      "Entity 'http://edamontology.org/format_3249' - Label 'GelML'\n",
      "Entity 'http://edamontology.org/format_3250' - Label 'spML'\n",
      "Entity 'http://edamontology.org/format_3250' - Label 'spML'\n",
      "Entity 'http://edamontology.org/format_3252' - Label 'OWL Functional Syntax'\n",
      "Entity 'http://edamontology.org/format_3252' - Label 'OWL Functional Syntax'\n",
      "Entity 'http://edamontology.org/format_3253' - Label 'Manchester OWL Syntax'\n",
      "Entity 'http://edamontology.org/format_3253' - Label 'Manchester OWL Syntax'\n",
      "Entity 'http://edamontology.org/format_3254' - Label 'KRSS2 Syntax'\n",
      "Entity 'http://edamontology.org/format_3254' - Label 'KRSS2 Syntax'\n",
      "Entity 'http://edamontology.org/format_3255' - Label 'Turtle'\n",
      "Entity 'http://edamontology.org/format_3255' - Label 'Turtle'\n",
      "Entity 'http://edamontology.org/format_3256' - Label 'N-Triples'\n",
      "Entity 'http://edamontology.org/format_3256' - Label 'N-Triples'\n",
      "Entity 'http://edamontology.org/format_3257' - Label 'Notation3'\n",
      "Entity 'http://edamontology.org/format_3257' - Label 'Notation3'\n",
      "Entity 'http://edamontology.org/format_3261' - Label 'RDF/XML'\n",
      "Entity 'http://edamontology.org/format_3261' - Label 'RDF/XML'\n",
      "Entity 'http://edamontology.org/format_3262' - Label 'OWL/XML'\n",
      "Entity 'http://edamontology.org/format_3262' - Label 'OWL/XML'\n",
      "Entity 'http://edamontology.org/format_3281' - Label 'A2M'\n",
      "Entity 'http://edamontology.org/format_3281' - Label 'A2M'\n",
      "Entity 'http://edamontology.org/format_3284' - Label 'SFF'\n",
      "Entity 'http://edamontology.org/format_3284' - Label 'SFF'\n",
      "Entity 'http://edamontology.org/format_3285' - Label 'MAP'\n",
      "Entity 'http://edamontology.org/format_3286' - Label 'PED'\n",
      "Entity 'http://edamontology.org/format_3287' - Label 'Individual genetic data format'\n",
      "Entity 'http://edamontology.org/format_3288' - Label 'PED/MAP'\n",
      "Entity 'http://edamontology.org/format_3288' - Label 'PED/MAP'\n",
      "Entity 'http://edamontology.org/format_3309' - Label 'CT'\n",
      "Entity 'http://edamontology.org/format_3309' - Label 'CT'\n",
      "Entity 'http://edamontology.org/format_3310' - Label 'SS'\n",
      "Entity 'http://edamontology.org/format_3310' - Label 'SS'\n",
      "Entity 'http://edamontology.org/format_3311' - Label 'RNAML'\n",
      "Entity 'http://edamontology.org/format_3311' - Label 'RNAML'\n",
      "Entity 'http://edamontology.org/format_3311' - Label 'RNAML'\n",
      "Entity 'http://edamontology.org/format_3312' - Label 'GDE'\n",
      "Entity 'http://edamontology.org/format_3312' - Label 'GDE'\n",
      "Entity 'http://edamontology.org/format_3313' - Label 'BLC'\n",
      "Entity 'http://edamontology.org/format_3326' - Label 'Data index format'\n",
      "Entity 'http://edamontology.org/format_3326' - Label 'Data index format'\n",
      "Entity 'http://edamontology.org/format_3327' - Label 'BAI'\n",
      "Entity 'http://edamontology.org/format_3327' - Label 'BAI'\n",
      "Entity 'http://edamontology.org/format_3327' - Label 'BAI'\n",
      "Entity 'http://edamontology.org/format_3328' - Label 'HMMER2'\n",
      "Entity 'http://edamontology.org/format_3329' - Label 'HMMER3'\n",
      "Entity 'http://edamontology.org/format_3330' - Label 'PO'\n",
      "Entity 'http://edamontology.org/format_3331' - Label 'BLAST XML results format'\n",
      "Entity 'http://edamontology.org/format_3331' - Label 'BLAST XML results format'\n",
      "Entity 'http://edamontology.org/format_3462' - Label 'CRAM'\n",
      "Entity 'http://edamontology.org/format_3462' - Label 'CRAM'\n",
      "Entity 'http://edamontology.org/format_3464' - Label 'JSON'\n",
      "Entity 'http://edamontology.org/format_3464' - Label 'JSON'\n",
      "Entity 'http://edamontology.org/format_3466' - Label 'EPS'\n",
      "Entity 'http://edamontology.org/format_3467' - Label 'GIF'\n",
      "Entity 'http://edamontology.org/format_3468' - Label 'xls'\n",
      "Entity 'http://edamontology.org/format_3468' - Label 'xls'\n",
      "Entity 'http://edamontology.org/format_3475' - Label 'TSV'\n",
      "Entity 'http://edamontology.org/format_3476' - Label 'Gene expression data format'\n",
      "Entity 'http://edamontology.org/format_3476' - Label 'Gene expression data format'\n",
      "Entity 'http://edamontology.org/format_3477' - Label 'Cytoscape input file format'\n",
      "Entity 'http://edamontology.org/format_3477' - Label 'Cytoscape input file format'\n",
      "Entity 'http://edamontology.org/format_3484' - Label 'ebwt'\n",
      "Entity 'http://edamontology.org/format_3484' - Label 'ebwt'\n",
      "Entity 'http://edamontology.org/format_3484' - Label 'ebwt'\n",
      "Entity 'http://edamontology.org/format_3485' - Label 'RSF'\n",
      "Entity 'http://edamontology.org/format_3486' - Label 'GCG format variant'\n",
      "Entity 'http://edamontology.org/format_3486' - Label 'GCG format variant'\n",
      "Entity 'http://edamontology.org/format_3486' - Label 'GCG format variant'\n",
      "Entity 'http://edamontology.org/format_3487' - Label 'BSML'\n",
      "Entity 'http://edamontology.org/format_3487' - Label 'BSML'\n",
      "Entity 'http://edamontology.org/format_3491' - Label 'ebwtl'\n",
      "Entity 'http://edamontology.org/format_3491' - Label 'ebwtl'\n",
      "Entity 'http://edamontology.org/format_3491' - Label 'ebwtl'\n",
      "Entity 'http://edamontology.org/format_3499' - Label 'Ensembl variation file format'\n",
      "Entity 'http://edamontology.org/format_3499' - Label 'Ensembl variation file format'\n",
      "Entity 'http://edamontology.org/format_3506' - Label 'docx'\n",
      "Entity 'http://edamontology.org/format_3506' - Label 'docx'\n",
      "Entity 'http://edamontology.org/format_3507' - Label 'Document format'\n",
      "Entity 'http://edamontology.org/format_3508' - Label 'PDF'\n",
      "Entity 'http://edamontology.org/format_3508' - Label 'PDF'\n",
      "Entity 'http://edamontology.org/format_3547' - Label 'Image format'\n",
      "Entity 'http://edamontology.org/format_3547' - Label 'Image format'\n",
      "Entity 'http://edamontology.org/format_3548' - Label 'DICOM format'\n",
      "Entity 'http://edamontology.org/format_3548' - Label 'DICOM format'\n",
      "Entity 'http://edamontology.org/format_3549' - Label 'nii'\n",
      "Entity 'http://edamontology.org/format_3549' - Label 'nii'\n",
      "Entity 'http://edamontology.org/format_3550' - Label 'mhd'\n",
      "Entity 'http://edamontology.org/format_3550' - Label 'mhd'\n",
      "Entity 'http://edamontology.org/format_3551' - Label 'nrrd'\n",
      "Entity 'http://edamontology.org/format_3551' - Label 'nrrd'\n",
      "Entity 'http://edamontology.org/format_3554' - Label 'R file format'\n",
      "Entity 'http://edamontology.org/format_3555' - Label 'SPSS'\n",
      "Entity 'http://edamontology.org/format_3556' - Label 'MHTML'\n",
      "Entity 'http://edamontology.org/format_3578' - Label 'IDAT'\n",
      "Entity 'http://edamontology.org/format_3578' - Label 'IDAT'\n",
      "Entity 'http://edamontology.org/format_3578' - Label 'IDAT'\n",
      "Entity 'http://edamontology.org/format_3579' - Label 'JPG'\n",
      "Entity 'http://edamontology.org/format_3579' - Label 'JPG'\n",
      "Entity 'http://edamontology.org/format_3580' - Label 'rcc'\n",
      "Entity 'http://edamontology.org/format_3580' - Label 'rcc'\n",
      "Entity 'http://edamontology.org/format_3581' - Label 'arff'\n",
      "Entity 'http://edamontology.org/format_3582' - Label 'afg'\n",
      "Entity 'http://edamontology.org/format_3582' - Label 'afg'\n",
      "Entity 'http://edamontology.org/format_3583' - Label 'bedgraph'\n",
      "Entity 'http://edamontology.org/format_3583' - Label 'bedgraph'\n",
      "Entity 'http://edamontology.org/format_3584' - Label 'bedstrict'\n",
      "Entity 'http://edamontology.org/format_3585' - Label 'bed6'\n",
      "Entity 'http://edamontology.org/format_3586' - Label 'bed12'\n",
      "Entity 'http://edamontology.org/format_3587' - Label 'chrominfo'\n",
      "Entity 'http://edamontology.org/format_3587' - Label 'chrominfo'\n",
      "Entity 'http://edamontology.org/format_3588' - Label 'customtrack'\n",
      "Entity 'http://edamontology.org/format_3588' - Label 'customtrack'\n",
      "Entity 'http://edamontology.org/format_3589' - Label 'csfasta'\n",
      "Entity 'http://edamontology.org/format_3589' - Label 'csfasta'\n",
      "Entity 'http://edamontology.org/format_3590' - Label 'HDF5'\n",
      "Entity 'http://edamontology.org/format_3590' - Label 'HDF5'\n",
      "Entity 'http://edamontology.org/format_3591' - Label 'TIFF'\n",
      "Entity 'http://edamontology.org/format_3591' - Label 'TIFF'\n",
      "Entity 'http://edamontology.org/format_3592' - Label 'BMP'\n",
      "Entity 'http://edamontology.org/format_3592' - Label 'BMP'\n",
      "Entity 'http://edamontology.org/format_3593' - Label 'im'\n",
      "Entity 'http://edamontology.org/format_3593' - Label 'im'\n",
      "Entity 'http://edamontology.org/format_3594' - Label 'pcd'\n",
      "Entity 'http://edamontology.org/format_3594' - Label 'pcd'\n",
      "Entity 'http://edamontology.org/format_3595' - Label 'pcx'\n",
      "Entity 'http://edamontology.org/format_3595' - Label 'pcx'\n",
      "Entity 'http://edamontology.org/format_3596' - Label 'ppm'\n",
      "Entity 'http://edamontology.org/format_3596' - Label 'ppm'\n",
      "Entity 'http://edamontology.org/format_3597' - Label 'psd'\n",
      "Entity 'http://edamontology.org/format_3597' - Label 'psd'\n",
      "Entity 'http://edamontology.org/format_3598' - Label 'xbm'\n",
      "Entity 'http://edamontology.org/format_3598' - Label 'xbm'\n",
      "Entity 'http://edamontology.org/format_3599' - Label 'xpm'\n",
      "Entity 'http://edamontology.org/format_3599' - Label 'xpm'\n",
      "Entity 'http://edamontology.org/format_3600' - Label 'rgb'\n",
      "Entity 'http://edamontology.org/format_3600' - Label 'rgb'\n",
      "Entity 'http://edamontology.org/format_3601' - Label 'pbm'\n",
      "Entity 'http://edamontology.org/format_3601' - Label 'pbm'\n",
      "Entity 'http://edamontology.org/format_3602' - Label 'pgm'\n",
      "Entity 'http://edamontology.org/format_3602' - Label 'pgm'\n",
      "Entity 'http://edamontology.org/format_3603' - Label 'PNG'\n",
      "Entity 'http://edamontology.org/format_3603' - Label 'PNG'\n",
      "Entity 'http://edamontology.org/format_3604' - Label 'SVG'\n",
      "Entity 'http://edamontology.org/format_3604' - Label 'SVG'\n",
      "Entity 'http://edamontology.org/format_3605' - Label 'rast'\n",
      "Entity 'http://edamontology.org/format_3605' - Label 'rast'\n",
      "Entity 'http://edamontology.org/format_3606' - Label 'Sequence quality report format (text)'\n",
      "Entity 'http://edamontology.org/format_3606' - Label 'Sequence quality report format (text)'\n",
      "Entity 'http://edamontology.org/format_3607' - Label 'qual'\n",
      "Entity 'http://edamontology.org/format_3607' - Label 'qual'\n",
      "Entity 'http://edamontology.org/format_3607' - Label 'qual'\n",
      "Entity 'http://edamontology.org/format_3608' - Label 'qualsolexa'\n",
      "Entity 'http://edamontology.org/format_3608' - Label 'qualsolexa'\n",
      "Entity 'http://edamontology.org/format_3609' - Label 'qualillumina'\n",
      "Entity 'http://edamontology.org/format_3609' - Label 'qualillumina'\n",
      "Entity 'http://edamontology.org/format_3610' - Label 'qualsolid'\n",
      "Entity 'http://edamontology.org/format_3611' - Label 'qual454'\n",
      "Entity 'http://edamontology.org/format_3612' - Label 'ENCODE peak format'\n",
      "Entity 'http://edamontology.org/format_3613' - Label 'ENCODE narrow peak format'\n",
      "Entity 'http://edamontology.org/format_3614' - Label 'ENCODE broad peak format'\n",
      "Entity 'http://edamontology.org/format_3615' - Label 'bgzip'\n",
      "Entity 'http://edamontology.org/format_3615' - Label 'bgzip'\n",
      "Entity 'http://edamontology.org/format_3616' - Label 'tabix'\n",
      "Entity 'http://edamontology.org/format_3616' - Label 'tabix'\n",
      "Entity 'http://edamontology.org/format_3617' - Label 'Graph format'\n",
      "Entity 'http://edamontology.org/format_3618' - Label 'xgmml'\n",
      "Entity 'http://edamontology.org/format_3619' - Label 'sif'\n",
      "Entity 'http://edamontology.org/format_3620' - Label 'xlsx'\n",
      "Entity 'http://edamontology.org/format_3620' - Label 'xlsx'\n",
      "Entity 'http://edamontology.org/format_3621' - Label 'SQLite format'\n",
      "Entity 'http://edamontology.org/format_3622' - Label 'Gemini SQLite format'\n",
      "Entity 'http://edamontology.org/format_3622' - Label 'Gemini SQLite format'\n",
      "Entity 'http://edamontology.org/format_3623' - Label 'Index format'\n",
      "Entity 'http://edamontology.org/format_3623' - Label 'Index format'\n",
      "Entity 'http://edamontology.org/format_3624' - Label 'snpeffdb'\n",
      "Entity 'http://edamontology.org/format_3624' - Label 'snpeffdb'\n",
      "Entity 'http://edamontology.org/format_3626' - Label 'MAT'\n",
      "Entity 'http://edamontology.org/format_3626' - Label 'MAT'\n",
      "Entity 'http://edamontology.org/format_3650' - Label 'netCDF'\n",
      "Entity 'http://edamontology.org/format_3650' - Label 'netCDF'\n",
      "Entity 'http://edamontology.org/format_3650' - Label 'netCDF'\n",
      "Entity 'http://edamontology.org/format_3651' - Label 'MGF'\n",
      "Entity 'http://edamontology.org/format_3652' - Label 'dta'\n",
      "Entity 'http://edamontology.org/format_3653' - Label 'pkl'\n",
      "Entity 'http://edamontology.org/format_3654' - Label 'mzXML'\n",
      "Entity 'http://edamontology.org/format_3655' - Label 'pepXML'\n",
      "Entity 'http://edamontology.org/format_3655' - Label 'pepXML'\n",
      "Entity 'http://edamontology.org/format_3657' - Label 'GPML'\n",
      "Entity 'http://edamontology.org/format_3657' - Label 'GPML'\n",
      "Entity 'http://edamontology.org/format_3665' - Label 'K-mer countgraph'\n",
      "Entity 'http://edamontology.org/format_3665' - Label 'K-mer countgraph'\n",
      "Entity 'http://edamontology.org/format_3681' - Label 'mzTab'\n",
      "Entity 'http://edamontology.org/format_3681' - Label 'mzTab'\n",
      "Entity 'http://edamontology.org/format_3682' - Label 'imzML metadata file'\n",
      "Entity 'http://edamontology.org/format_3682' - Label 'imzML metadata file'\n",
      "Entity 'http://edamontology.org/format_3683' - Label 'qcML'\n",
      "Entity 'http://edamontology.org/format_3683' - Label 'qcML'\n",
      "Entity 'http://edamontology.org/format_3683' - Label 'qcML'\n",
      "Entity 'http://edamontology.org/format_3684' - Label 'PRIDE XML'\n",
      "Entity 'http://edamontology.org/format_3684' - Label 'PRIDE XML'\n",
      "Entity 'http://edamontology.org/format_3684' - Label 'PRIDE XML'\n",
      "Entity 'http://edamontology.org/format_3685' - Label 'SED-ML'\n",
      "Entity 'http://edamontology.org/format_3685' - Label 'SED-ML'\n",
      "Entity 'http://edamontology.org/format_3686' - Label 'COMBINE OMEX'\n",
      "Entity 'http://edamontology.org/format_3686' - Label 'COMBINE OMEX'\n",
      "Entity 'http://edamontology.org/format_3686' - Label 'COMBINE OMEX'\n",
      "Entity 'http://edamontology.org/format_3687' - Label 'ISA-TAB'\n",
      "Entity 'http://edamontology.org/format_3687' - Label 'ISA-TAB'\n",
      "Entity 'http://edamontology.org/format_3687' - Label 'ISA-TAB'\n",
      "Entity 'http://edamontology.org/format_3688' - Label 'SBtab'\n",
      "Entity 'http://edamontology.org/format_3688' - Label 'SBtab'\n",
      "Entity 'http://edamontology.org/format_3689' - Label 'BCML'\n",
      "Entity 'http://edamontology.org/format_3689' - Label 'BCML'\n",
      "Entity 'http://edamontology.org/format_3690' - Label 'BDML'\n",
      "Entity 'http://edamontology.org/format_3691' - Label 'BEL'\n",
      "Entity 'http://edamontology.org/format_3692' - Label 'SBGN-ML'\n",
      "Entity 'http://edamontology.org/format_3692' - Label 'SBGN-ML'\n",
      "Entity 'http://edamontology.org/format_3693' - Label 'AGP'\n",
      "Entity 'http://edamontology.org/format_3693' - Label 'AGP'\n",
      "Entity 'http://edamontology.org/format_3696' - Label 'PS'\n",
      "Entity 'http://edamontology.org/format_3698' - Label 'SRA format'\n",
      "Entity 'http://edamontology.org/format_3699' - Label 'VDB'\n",
      "Entity 'http://edamontology.org/format_3700' - Label 'Tabix index file format'\n",
      "Entity 'http://edamontology.org/format_3700' - Label 'Tabix index file format'\n",
      "Entity 'http://edamontology.org/format_3700' - Label 'Tabix index file format'\n",
      "Entity 'http://edamontology.org/format_3701' - Label 'Sequin format'\n",
      "Entity 'http://edamontology.org/format_3702' - Label 'MSF'\n",
      "Entity 'http://edamontology.org/format_3706' - Label 'Biodiversity data format'\n",
      "Entity 'http://edamontology.org/format_3706' - Label 'Biodiversity data format'\n",
      "Entity 'http://edamontology.org/format_3708' - Label 'ABCD format'\n",
      "Entity 'http://edamontology.org/format_3708' - Label 'ABCD format'\n",
      "Entity 'http://edamontology.org/format_3709' - Label 'GCT/Res format'\n",
      "Entity 'http://edamontology.org/format_3709' - Label 'GCT/Res format'\n",
      "Entity 'http://edamontology.org/format_3710' - Label 'WIFF format'\n",
      "Entity 'http://edamontology.org/format_3710' - Label 'WIFF format'\n",
      "Entity 'http://edamontology.org/format_3711' - Label 'X!Tandem XML'\n",
      "Entity 'http://edamontology.org/format_3711' - Label 'X!Tandem XML'\n",
      "Entity 'http://edamontology.org/format_3712' - Label 'Thermo RAW'\n",
      "Entity 'http://edamontology.org/format_3712' - Label 'Thermo RAW'\n",
      "Entity 'http://edamontology.org/format_3713' - Label 'Mascot .dat file'\n",
      "Entity 'http://edamontology.org/format_3713' - Label 'Mascot .dat file'\n",
      "Entity 'http://edamontology.org/format_3714' - Label 'MaxQuant APL peaklist format'\n",
      "Entity 'http://edamontology.org/format_3714' - Label 'MaxQuant APL peaklist format'\n",
      "Entity 'http://edamontology.org/format_3725' - Label 'SBOL'\n",
      "Entity 'http://edamontology.org/format_3726' - Label 'PMML'\n",
      "Entity 'http://edamontology.org/format_3727' - Label 'OME-TIFF'\n",
      "Entity 'http://edamontology.org/format_3727' - Label 'OME-TIFF'\n",
      "Entity 'http://edamontology.org/format_3728' - Label 'LocARNA PP'\n",
      "Entity 'http://edamontology.org/format_3729' - Label 'dbGaP format'\n",
      "Entity 'http://edamontology.org/format_3746' - Label 'BIOM format'\n",
      "Entity 'http://edamontology.org/format_3746' - Label 'BIOM format'\n",
      "Entity 'http://edamontology.org/format_3747' - Label 'protXML'\n",
      "Entity 'http://edamontology.org/format_3747' - Label 'protXML'\n",
      "Entity 'http://edamontology.org/format_3748' - Label 'Linked data format'\n",
      "Entity 'http://edamontology.org/format_3749' - Label 'JSON-LD'\n",
      "Entity 'http://edamontology.org/format_3749' - Label 'JSON-LD'\n",
      "Entity 'http://edamontology.org/format_3749' - Label 'JSON-LD'\n",
      "Entity 'http://edamontology.org/format_3750' - Label 'YAML'\n",
      "Entity 'http://edamontology.org/format_3751' - Label 'DSV'\n",
      "Entity 'http://edamontology.org/format_3752' - Label 'CSV'\n",
      "Entity 'http://edamontology.org/format_3758' - Label 'SEQUEST .out file'\n",
      "Entity 'http://edamontology.org/format_3758' - Label 'SEQUEST .out file'\n",
      "Entity 'http://edamontology.org/format_3764' - Label 'idXML'\n",
      "Entity 'http://edamontology.org/format_3764' - Label 'idXML'\n",
      "Entity 'http://edamontology.org/format_3765' - Label 'KNIME datatable format'\n",
      "Entity 'http://edamontology.org/format_3770' - Label 'UniProtKB XML'\n",
      "Entity 'http://edamontology.org/format_3770' - Label 'UniProtKB XML'\n",
      "Entity 'http://edamontology.org/format_3770' - Label 'UniProtKB XML'\n",
      "Entity 'http://edamontology.org/format_3771' - Label 'UniProtKB RDF'\n",
      "Entity 'http://edamontology.org/format_3771' - Label 'UniProtKB RDF'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3772' - Label 'BioJSON (BioXSD)'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3773' - Label 'BioYAML'\n",
      "Entity 'http://edamontology.org/format_3774' - Label 'BioJSON (Jalview)'\n",
      "Entity 'http://edamontology.org/format_3774' - Label 'BioJSON (Jalview)'\n",
      "Entity 'http://edamontology.org/format_3774' - Label 'BioJSON (Jalview)'\n",
      "Entity 'http://edamontology.org/format_3774' - Label 'BioJSON (Jalview)'\n",
      "Entity 'http://edamontology.org/format_3774' - Label 'BioJSON (Jalview)'\n",
      "Entity 'http://edamontology.org/format_3775' - Label 'GSuite'\n",
      "Entity 'http://edamontology.org/format_3775' - Label 'GSuite'\n",
      "Entity 'http://edamontology.org/format_3776' - Label 'BTrack'\n",
      "Entity 'http://edamontology.org/format_3776' - Label 'BTrack'\n",
      "Entity 'http://edamontology.org/format_3776' - Label 'BTrack'\n",
      "Entity 'http://edamontology.org/format_3776' - Label 'BTrack'\n",
      "Entity 'http://edamontology.org/format_3777' - Label 'MCPD'\n",
      "Entity 'http://edamontology.org/format_3777' - Label 'MCPD'\n",
      "Entity 'http://edamontology.org/format_3777' - Label 'MCPD'\n",
      "Entity 'http://edamontology.org/format_3777' - Label 'MCPD'\n",
      "Entity 'http://edamontology.org/format_3777' - Label 'MCPD'\n",
      "Entity 'http://edamontology.org/format_3780' - Label 'Annotated text format'\n",
      "Entity 'http://edamontology.org/format_3780' - Label 'Annotated text format'\n",
      "Entity 'http://edamontology.org/format_3781' - Label 'PubAnnotation format'\n",
      "Entity 'http://edamontology.org/format_3781' - Label 'PubAnnotation format'\n",
      "Entity 'http://edamontology.org/format_3782' - Label 'BioC'\n",
      "Entity 'http://edamontology.org/format_3782' - Label 'BioC'\n",
      "Entity 'http://edamontology.org/format_3783' - Label 'PubTator format'\n",
      "Entity 'http://edamontology.org/format_3783' - Label 'PubTator format'\n",
      "Entity 'http://edamontology.org/format_3784' - Label 'Open Annotation format'\n",
      "Entity 'http://edamontology.org/format_3784' - Label 'Open Annotation format'\n",
      "Entity 'http://edamontology.org/format_3784' - Label 'Open Annotation format'\n",
      "Entity 'http://edamontology.org/format_3785' - Label 'BioNLP Shared Task format'\n",
      "Entity 'http://edamontology.org/format_3785' - Label 'BioNLP Shared Task format'\n",
      "Entity 'http://edamontology.org/format_3787' - Label 'Query language'\n",
      "Entity 'http://edamontology.org/format_3787' - Label 'Query language'\n",
      "Entity 'http://edamontology.org/format_3788' - Label 'SQL'\n",
      "Entity 'http://edamontology.org/format_3789' - Label 'XQuery'\n",
      "Entity 'http://edamontology.org/format_3790' - Label 'SPARQL'\n",
      "Entity 'http://edamontology.org/format_3804' - Label 'xsd'\n",
      "Entity 'http://edamontology.org/format_3811' - Label 'XMFA'\n",
      "Entity 'http://edamontology.org/format_3811' - Label 'XMFA'\n",
      "Entity 'http://edamontology.org/format_3812' - Label 'GEN'\n",
      "Entity 'http://edamontology.org/format_3812' - Label 'GEN'\n",
      "Entity 'http://edamontology.org/format_3813' - Label 'SAMPLE file format'\n",
      "Entity 'http://edamontology.org/format_3814' - Label 'SDF'\n",
      "Entity 'http://edamontology.org/format_3814' - Label 'SDF'\n",
      "Entity 'http://edamontology.org/format_3815' - Label 'Molfile'\n",
      "Entity 'http://edamontology.org/format_3815' - Label 'Molfile'\n",
      "Entity 'http://edamontology.org/format_3816' - Label 'Mol2'\n",
      "Entity 'http://edamontology.org/format_3816' - Label 'Mol2'\n",
      "Entity 'http://edamontology.org/format_3817' - Label 'latex'\n",
      "Entity 'http://edamontology.org/format_3817' - Label 'latex'\n",
      "Entity 'http://edamontology.org/format_3818' - Label 'ELAND format'\n",
      "Entity 'http://edamontology.org/format_3818' - Label 'ELAND format'\n",
      "Entity 'http://edamontology.org/format_3819' - Label 'Relaxed PHYLIP Interleaved'\n",
      "Entity 'http://edamontology.org/format_3820' - Label 'Relaxed PHYLIP Sequential'\n",
      "Entity 'http://edamontology.org/format_3821' - Label 'VisML'\n",
      "Entity 'http://edamontology.org/format_3821' - Label 'VisML'\n",
      "Entity 'http://edamontology.org/format_3822' - Label 'GML'\n",
      "Entity 'http://edamontology.org/format_3822' - Label 'GML'\n",
      "Entity 'http://edamontology.org/format_3823' - Label 'FASTG'\n",
      "Entity 'http://edamontology.org/format_3823' - Label 'FASTG'\n",
      "Entity 'http://edamontology.org/format_3824' - Label 'NMR data format'\n",
      "Entity 'http://edamontology.org/format_3824' - Label 'NMR data format'\n",
      "Entity 'http://edamontology.org/format_3825' - Label 'nmrML'\n",
      "Entity 'http://edamontology.org/format_3825' - Label 'nmrML'\n",
      "Entity 'http://edamontology.org/format_3826' - Label 'proBAM'\n",
      "Entity 'http://edamontology.org/format_3826' - Label 'proBAM'\n",
      "Entity 'http://edamontology.org/format_3826' - Label 'proBAM'\n",
      "Entity 'http://edamontology.org/format_3827' - Label 'proBED'\n",
      "Entity 'http://edamontology.org/format_3827' - Label 'proBED'\n",
      "Entity 'http://edamontology.org/format_3828' - Label 'Raw microarray data format'\n",
      "Entity 'http://edamontology.org/format_3828' - Label 'Raw microarray data format'\n",
      "Entity 'http://edamontology.org/format_3829' - Label 'GPR'\n",
      "Entity 'http://edamontology.org/format_3829' - Label 'GPR'\n",
      "Entity 'http://edamontology.org/format_3830' - Label 'ARB'\n",
      "Entity 'http://edamontology.org/format_3830' - Label 'ARB'\n",
      "Entity 'http://edamontology.org/format_3832' - Label 'consensusXML'\n",
      "Entity 'http://edamontology.org/format_3832' - Label 'consensusXML'\n",
      "Entity 'http://edamontology.org/format_3833' - Label 'featureXML'\n",
      "Entity 'http://edamontology.org/format_3833' - Label 'featureXML'\n",
      "Entity 'http://edamontology.org/format_3834' - Label 'mzData'\n",
      "Entity 'http://edamontology.org/format_3834' - Label 'mzData'\n",
      "Entity 'http://edamontology.org/format_3835' - Label 'TIDE TXT'\n",
      "Entity 'http://edamontology.org/format_3835' - Label 'TIDE TXT'\n",
      "Entity 'http://edamontology.org/format_3836' - Label 'BLAST XML v2 results format'\n",
      "Entity 'http://edamontology.org/format_3836' - Label 'BLAST XML v2 results format'\n",
      "Entity 'http://edamontology.org/format_3838' - Label 'pptx'\n",
      "Entity 'http://edamontology.org/format_3838' - Label 'pptx'\n",
      "Entity 'http://edamontology.org/format_3839' - Label 'ibd'\n",
      "Entity 'http://edamontology.org/format_3839' - Label 'ibd'\n",
      "Entity 'http://edamontology.org/format_3841' - Label 'NLP format'\n",
      "Entity 'http://edamontology.org/format_3843' - Label 'BEAST'\n",
      "Entity 'http://edamontology.org/format_3843' - Label 'BEAST'\n",
      "Entity 'http://edamontology.org/format_3844' - Label 'Chado-XML'\n",
      "Entity 'http://edamontology.org/format_3844' - Label 'Chado-XML'\n",
      "Entity 'http://edamontology.org/format_3845' - Label 'HSAML'\n",
      "Entity 'http://edamontology.org/format_3845' - Label 'HSAML'\n",
      "Entity 'http://edamontology.org/format_3846' - Label 'InterProScan XML'\n",
      "Entity 'http://edamontology.org/format_3846' - Label 'InterProScan XML'\n",
      "Entity 'http://edamontology.org/format_3847' - Label 'KGML'\n",
      "Entity 'http://edamontology.org/format_3847' - Label 'KGML'\n",
      "Entity 'http://edamontology.org/format_3848' - Label 'PubMed XML'\n",
      "Entity 'http://edamontology.org/format_3848' - Label 'PubMed XML'\n",
      "Entity 'http://edamontology.org/format_3849' - Label 'MSAML'\n",
      "Entity 'http://edamontology.org/format_3849' - Label 'MSAML'\n",
      "Entity 'http://edamontology.org/format_3850' - Label 'OrthoXML'\n",
      "Entity 'http://edamontology.org/format_3850' - Label 'OrthoXML'\n",
      "Entity 'http://edamontology.org/format_3850' - Label 'OrthoXML'\n",
      "Entity 'http://edamontology.org/format_3851' - Label 'PSDML'\n",
      "Entity 'http://edamontology.org/format_3851' - Label 'PSDML'\n",
      "Entity 'http://edamontology.org/format_3852' - Label 'SeqXML'\n",
      "Entity 'http://edamontology.org/format_3852' - Label 'SeqXML'\n",
      "Entity 'http://edamontology.org/format_3853' - Label 'UniParc XML'\n",
      "Entity 'http://edamontology.org/format_3853' - Label 'UniParc XML'\n",
      "Entity 'http://edamontology.org/format_3854' - Label 'UniRef XML'\n",
      "Entity 'http://edamontology.org/format_3854' - Label 'UniRef XML'\n",
      "Entity 'http://edamontology.org/format_3857' - Label 'CWL'\n",
      "Entity 'http://edamontology.org/format_3857' - Label 'CWL'\n",
      "Entity 'http://edamontology.org/format_3858' - Label 'Waters RAW'\n",
      "Entity 'http://edamontology.org/format_3858' - Label 'Waters RAW'\n",
      "Entity 'http://edamontology.org/format_3859' - Label 'JCAMP-DX'\n",
      "Entity 'http://edamontology.org/format_3859' - Label 'JCAMP-DX'\n",
      "Entity 'http://edamontology.org/format_3862' - Label 'NLP annotation format'\n",
      "Entity 'http://edamontology.org/format_3863' - Label 'NLP corpus format'\n",
      "Entity 'http://edamontology.org/format_3864' - Label 'mirGFF3'\n",
      "Entity 'http://edamontology.org/format_3864' - Label 'mirGFF3'\n",
      "Entity 'http://edamontology.org/format_3865' - Label 'RNA annotation format'\n",
      "Entity 'http://edamontology.org/format_3866' - Label 'Trajectory format'\n",
      "Entity 'http://edamontology.org/format_3866' - Label 'Trajectory format'\n",
      "Entity 'http://edamontology.org/format_3867' - Label 'Trajectory format (binary)'\n",
      "Entity 'http://edamontology.org/format_3868' - Label 'Trajectory format (text)'\n",
      "Entity 'http://edamontology.org/format_3873' - Label 'HDF'\n",
      "Entity 'http://edamontology.org/format_3873' - Label 'HDF'\n",
      "Entity 'http://edamontology.org/format_3874' - Label 'PCAzip'\n",
      "Entity 'http://edamontology.org/format_3874' - Label 'PCAzip'\n",
      "Entity 'http://edamontology.org/format_3875' - Label 'XTC'\n",
      "Entity 'http://edamontology.org/format_3875' - Label 'XTC'\n",
      "Entity 'http://edamontology.org/format_3876' - Label 'TNG'\n",
      "Entity 'http://edamontology.org/format_3876' - Label 'TNG'\n",
      "Entity 'http://edamontology.org/format_3877' - Label 'XYZ'\n",
      "Entity 'http://edamontology.org/format_3877' - Label 'XYZ'\n",
      "Entity 'http://edamontology.org/format_3877' - Label 'XYZ'\n",
      "Entity 'http://edamontology.org/format_3878' - Label 'mdcrd'\n",
      "Entity 'http://edamontology.org/format_3878' - Label 'mdcrd'\n",
      "Entity 'http://edamontology.org/format_3878' - Label 'mdcrd'\n",
      "Entity 'http://edamontology.org/format_3879' - Label 'Topology format'\n",
      "Entity 'http://edamontology.org/format_3879' - Label 'Topology format'\n",
      "Entity 'http://edamontology.org/format_3880' - Label 'GROMACS top'\n",
      "Entity 'http://edamontology.org/format_3880' - Label 'GROMACS top'\n",
      "Entity 'http://edamontology.org/format_3880' - Label 'GROMACS top'\n",
      "Entity 'http://edamontology.org/format_3881' - Label 'AMBER top'\n",
      "Entity 'http://edamontology.org/format_3881' - Label 'AMBER top'\n",
      "Entity 'http://edamontology.org/format_3881' - Label 'AMBER top'\n",
      "Entity 'http://edamontology.org/format_3882' - Label 'PSF'\n",
      "Entity 'http://edamontology.org/format_3882' - Label 'PSF'\n",
      "Entity 'http://edamontology.org/format_3882' - Label 'PSF'\n",
      "Entity 'http://edamontology.org/format_3883' - Label 'GROMACS itp'\n",
      "Entity 'http://edamontology.org/format_3883' - Label 'GROMACS itp'\n",
      "Entity 'http://edamontology.org/format_3883' - Label 'GROMACS itp'\n",
      "Entity 'http://edamontology.org/format_3883' - Label 'GROMACS itp'\n",
      "Entity 'http://edamontology.org/format_3884' - Label 'FF parameter format'\n",
      "Entity 'http://edamontology.org/format_3884' - Label 'FF parameter format'\n",
      "Entity 'http://edamontology.org/format_3885' - Label 'BinPos'\n",
      "Entity 'http://edamontology.org/format_3885' - Label 'BinPos'\n",
      "Entity 'http://edamontology.org/format_3886' - Label 'RST'\n",
      "Entity 'http://edamontology.org/format_3886' - Label 'RST'\n",
      "Entity 'http://edamontology.org/format_3886' - Label 'RST'\n",
      "Entity 'http://edamontology.org/format_3887' - Label 'CHARMM rtf'\n",
      "Entity 'http://edamontology.org/format_3887' - Label 'CHARMM rtf'\n",
      "Entity 'http://edamontology.org/format_3887' - Label 'CHARMM rtf'\n",
      "Entity 'http://edamontology.org/format_3888' - Label 'AMBER frcmod'\n",
      "Entity 'http://edamontology.org/format_3888' - Label 'AMBER frcmod'\n",
      "Entity 'http://edamontology.org/format_3889' - Label 'AMBER off'\n",
      "Entity 'http://edamontology.org/format_3889' - Label 'AMBER off'\n",
      "Entity 'http://edamontology.org/format_3906' - Label 'NMReDATA'\n",
      "Entity 'http://edamontology.org/format_3906' - Label 'NMReDATA'\n",
      "Entity 'http://edamontology.org/format_3909' - Label 'BpForms'\n",
      "Entity 'http://edamontology.org/format_3909' - Label 'BpForms'\n",
      "Entity 'http://edamontology.org/format_3909' - Label 'BpForms'\n",
      "Entity 'http://edamontology.org/format_3909' - Label 'BpForms'\n",
      "Entity 'http://edamontology.org/format_3910' - Label 'trr'\n",
      "Entity 'http://edamontology.org/format_3910' - Label 'trr'\n",
      "Entity 'http://edamontology.org/format_3910' - Label 'trr'\n",
      "Entity 'http://edamontology.org/format_3911' - Label 'msh'\n",
      "Entity 'http://edamontology.org/format_3911' - Label 'msh'\n",
      "Entity 'http://edamontology.org/format_3911' - Label 'msh'\n",
      "Entity 'http://edamontology.org/format_3913' - Label 'Loom'\n",
      "Entity 'http://edamontology.org/format_3913' - Label 'Loom'\n",
      "Entity 'http://edamontology.org/format_3913' - Label 'Loom'\n",
      "Entity 'http://edamontology.org/format_3913' - Label 'Loom'\n",
      "Entity 'http://edamontology.org/format_3915' - Label 'Zarr'\n",
      "Entity 'http://edamontology.org/format_3915' - Label 'Zarr'\n",
      "Entity 'http://edamontology.org/format_3915' - Label 'Zarr'\n",
      "Entity 'http://edamontology.org/format_3915' - Label 'Zarr'\n",
      "Entity 'http://edamontology.org/format_3915' - Label 'Zarr'\n",
      "Entity 'http://edamontology.org/format_3916' - Label 'MTX'\n",
      "Entity 'http://edamontology.org/format_3916' - Label 'MTX'\n",
      "Entity 'http://edamontology.org/format_3916' - Label 'MTX'\n",
      "Entity 'http://edamontology.org/format_3916' - Label 'MTX'\n",
      "Entity 'http://edamontology.org/format_3916' - Label 'MTX'\n",
      "Entity 'http://edamontology.org/format_3951' - Label 'BcForms'\n",
      "Entity 'http://edamontology.org/format_3951' - Label 'BcForms'\n",
      "Entity 'http://edamontology.org/format_3951' - Label 'BcForms'\n",
      "Entity 'http://edamontology.org/format_3956' - Label 'N-Quads'\n",
      "Entity 'http://edamontology.org/format_3956' - Label 'N-Quads'\n",
      "Entity 'http://edamontology.org/format_3969' - Label 'Vega'\n",
      "Entity 'http://edamontology.org/format_3969' - Label 'Vega'\n",
      "Entity 'http://edamontology.org/format_3970' - Label 'Vega-lite'\n",
      "Entity 'http://edamontology.org/format_3970' - Label 'Vega-lite'\n",
      "Entity 'http://edamontology.org/format_3971' - Label 'NeuroML'\n",
      "Entity 'http://edamontology.org/format_3971' - Label 'NeuroML'\n",
      "Entity 'http://edamontology.org/format_3971' - Label 'NeuroML'\n",
      "Entity 'http://edamontology.org/format_3972' - Label 'BNGL'\n",
      "Entity 'http://edamontology.org/format_3972' - Label 'BNGL'\n",
      "Entity 'http://edamontology.org/format_3972' - Label 'BNGL'\n",
      "Entity 'http://edamontology.org/format_3973' - Label 'Docker image'\n",
      "Entity 'http://edamontology.org/format_3975' - Label 'GFA 1'\n",
      "Entity 'http://edamontology.org/format_3975' - Label 'GFA 1'\n",
      "Entity 'http://edamontology.org/format_3976' - Label 'GFA 2'\n",
      "Entity 'http://edamontology.org/format_3976' - Label 'GFA 2'\n",
      "Entity 'http://edamontology.org/format_3977' - Label 'ObjTables'\n",
      "Entity 'http://edamontology.org/format_3978' - Label 'CONTIG'\n",
      "Entity 'http://edamontology.org/format_3978' - Label 'CONTIG'\n",
      "Entity 'http://edamontology.org/format_3979' - Label 'WEGO'\n",
      "Entity 'http://edamontology.org/format_3979' - Label 'WEGO'\n",
      "Entity 'http://edamontology.org/format_3980' - Label 'RPKM'\n",
      "Entity 'http://edamontology.org/format_3980' - Label 'RPKM'\n",
      "Entity 'http://edamontology.org/format_3981' - Label 'TAR format'\n",
      "Entity 'http://edamontology.org/format_3982' - Label 'CHAIN'\n",
      "Entity 'http://edamontology.org/format_3982' - Label 'CHAIN'\n",
      "Entity 'http://edamontology.org/format_3983' - Label 'NET'\n",
      "Entity 'http://edamontology.org/format_3983' - Label 'NET'\n",
      "Entity 'http://edamontology.org/format_3984' - Label 'QMAP'\n",
      "Entity 'http://edamontology.org/format_3984' - Label 'QMAP'\n",
      "Entity 'http://edamontology.org/format_3985' - Label 'gxformat2'\n",
      "Entity 'http://edamontology.org/format_3985' - Label 'gxformat2'\n",
      "Entity 'http://edamontology.org/format_3986' - Label 'WMV'\n",
      "Entity 'http://edamontology.org/format_3986' - Label 'WMV'\n",
      "Entity 'http://edamontology.org/format_3987' - Label 'ZIP format'\n",
      "Entity 'http://edamontology.org/format_3988' - Label 'LSM'\n",
      "Entity 'http://edamontology.org/format_3988' - Label 'LSM'\n",
      "Entity 'http://edamontology.org/format_3989' - Label 'GZIP format'\n",
      "Entity 'http://edamontology.org/format_3990' - Label 'AVI'\n",
      "Entity 'http://edamontology.org/format_3990' - Label 'AVI'\n",
      "Entity 'http://edamontology.org/format_3991' - Label 'TrackDB'\n",
      "Entity 'http://edamontology.org/format_3991' - Label 'TrackDB'\n",
      "Entity 'http://edamontology.org/format_3992' - Label 'CIGAR format'\n",
      "Entity 'http://edamontology.org/format_3992' - Label 'CIGAR format'\n",
      "Entity 'http://edamontology.org/format_3993' - Label 'Stereolithography format'\n",
      "Entity 'http://edamontology.org/format_3993' - Label 'Stereolithography format'\n",
      "Entity 'http://edamontology.org/format_3994' - Label 'U3D'\n",
      "Entity 'http://edamontology.org/format_3994' - Label 'U3D'\n",
      "Entity 'http://edamontology.org/format_3995' - Label 'Texture file format'\n",
      "Entity 'http://edamontology.org/format_3995' - Label 'Texture file format'\n",
      "Entity 'http://edamontology.org/format_3996' - Label 'Python script'\n",
      "Entity 'http://edamontology.org/format_3996' - Label 'Python script'\n",
      "Entity 'http://edamontology.org/format_3997' - Label 'MPEG-4'\n",
      "Entity 'http://edamontology.org/format_3997' - Label 'MPEG-4'\n",
      "Entity 'http://edamontology.org/format_3998' - Label 'Perl script'\n",
      "Entity 'http://edamontology.org/format_3998' - Label 'Perl script'\n",
      "Entity 'http://edamontology.org/format_3999' - Label 'R script'\n",
      "Entity 'http://edamontology.org/format_3999' - Label 'R script'\n",
      "Entity 'http://edamontology.org/format_4000' - Label 'R markdown'\n",
      "Entity 'http://edamontology.org/format_4000' - Label 'R markdown'\n",
      "Entity 'http://edamontology.org/format_4001' - Label 'NIFTI format'\n",
      "Entity 'http://edamontology.org/format_4001' - Label 'NIFTI format'\n",
      "Entity 'http://edamontology.org/format_4002' - Label 'pickle'\n",
      "Entity 'http://edamontology.org/format_4003' - Label 'NumPy format'\n",
      "Entity 'http://edamontology.org/format_4004' - Label 'SimTools repertoire file format'\n",
      "Entity 'http://edamontology.org/format_4005' - Label 'Configuration file format'\n",
      "Entity 'http://edamontology.org/format_4006' - Label 'Zstandard format'\n",
      "Entity 'http://edamontology.org/format_4007' - Label 'MATLAB script'\n",
      "Entity 'http://edamontology.org/format_4007' - Label 'MATLAB script'\n",
      "Entity 'http://edamontology.org/format_4015' - Label 'PEtab'\n",
      "Entity 'http://edamontology.org/format_4015' - Label 'PEtab'\n",
      "Entity 'http://edamontology.org/format_4018' - Label 'gVCF'\n",
      "Entity 'http://edamontology.org/format_4023' - Label 'cml'\n",
      "Entity 'http://edamontology.org/format_4023' - Label 'cml'\n",
      "Entity 'http://edamontology.org/format_4024' - Label 'cif'\n",
      "Entity 'http://edamontology.org/format_4024' - Label 'cif'\n",
      "Entity 'http://edamontology.org/format_4025' - Label 'BioSimulators format for the specifications of biosimulation tools'\n",
      "Entity 'http://edamontology.org/format_4026' - Label 'BioSimulators standard for command-line interfaces for biosimulation tools'\n",
      "Entity 'http://edamontology.org/format_4035' - Label 'PQR'\n",
      "Entity 'http://edamontology.org/format_4035' - Label 'PQR'\n",
      "Entity 'http://edamontology.org/format_4036' - Label 'PDBQT'\n",
      "Entity 'http://edamontology.org/format_4036' - Label 'PDBQT'\n",
      "Entity 'http://edamontology.org/format_4039' - Label 'MSP'\n",
      "Entity 'http://edamontology.org/format_4039' - Label 'MSP'\n",
      "Entity 'http://edamontology.org/operation_0224' - Label 'Query and retrieval'\n",
      "Entity 'http://edamontology.org/operation_0225' - Label 'Data retrieval (database cross-reference)'\n",
      "Entity 'http://edamontology.org/operation_0225' - Label 'Data retrieval (database cross-reference)'\n",
      "Entity 'http://edamontology.org/operation_0226' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/operation_0226' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/operation_0226' - Label 'Annotation'\n",
      "Entity 'http://edamontology.org/operation_0227' - Label 'Indexing'\n",
      "Entity 'http://edamontology.org/operation_0227' - Label 'Indexing'\n",
      "Entity 'http://edamontology.org/operation_0228' - Label 'Data index analysis'\n",
      "Entity 'http://edamontology.org/operation_0228' - Label 'Data index analysis'\n",
      "Entity 'http://edamontology.org/operation_0229' - Label 'Annotation retrieval (sequence)'\n",
      "Entity 'http://edamontology.org/operation_0229' - Label 'Annotation retrieval (sequence)'\n",
      "Entity 'http://edamontology.org/operation_0230' - Label 'Sequence generation'\n",
      "Entity 'http://edamontology.org/operation_0230' - Label 'Sequence generation'\n",
      "Entity 'http://edamontology.org/operation_0231' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/operation_0231' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/operation_0232' - Label 'Sequence merging'\n",
      "Entity 'http://edamontology.org/operation_0233' - Label 'Sequence conversion'\n",
      "Entity 'http://edamontology.org/operation_0233' - Label 'Sequence conversion'\n",
      "Entity 'http://edamontology.org/operation_0234' - Label 'Sequence complexity calculation'\n",
      "Entity 'http://edamontology.org/operation_0234' - Label 'Sequence complexity calculation'\n",
      "Entity 'http://edamontology.org/operation_0234' - Label 'Sequence complexity calculation'\n",
      "Entity 'http://edamontology.org/operation_0235' - Label 'Sequence ambiguity calculation'\n",
      "Entity 'http://edamontology.org/operation_0235' - Label 'Sequence ambiguity calculation'\n",
      "Entity 'http://edamontology.org/operation_0235' - Label 'Sequence ambiguity calculation'\n",
      "Entity 'http://edamontology.org/operation_0236' - Label 'Sequence composition calculation'\n",
      "Entity 'http://edamontology.org/operation_0236' - Label 'Sequence composition calculation'\n",
      "Entity 'http://edamontology.org/operation_0236' - Label 'Sequence composition calculation'\n",
      "Entity 'http://edamontology.org/operation_0236' - Label 'Sequence composition calculation'\n",
      "Entity 'http://edamontology.org/operation_0237' - Label 'Repeat sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0237' - Label 'Repeat sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0238' - Label 'Sequence motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0238' - Label 'Sequence motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0238' - Label 'Sequence motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0238' - Label 'Sequence motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0239' - Label 'Sequence motif recognition'\n",
      "Entity 'http://edamontology.org/operation_0239' - Label 'Sequence motif recognition'\n",
      "Entity 'http://edamontology.org/operation_0239' - Label 'Sequence motif recognition'\n",
      "Entity 'http://edamontology.org/operation_0239' - Label 'Sequence motif recognition'\n",
      "Entity 'http://edamontology.org/operation_0240' - Label 'Sequence motif comparison'\n",
      "Entity 'http://edamontology.org/operation_0240' - Label 'Sequence motif comparison'\n",
      "Entity 'http://edamontology.org/operation_0240' - Label 'Sequence motif comparison'\n",
      "Entity 'http://edamontology.org/operation_0240' - Label 'Sequence motif comparison'\n",
      "Entity 'http://edamontology.org/operation_0241' - Label 'Transcription regulatory sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0241' - Label 'Transcription regulatory sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0242' - Label 'Conserved transcription regulatory sequence identification'\n",
      "Entity 'http://edamontology.org/operation_0242' - Label 'Conserved transcription regulatory sequence identification'\n",
      "Entity 'http://edamontology.org/operation_0243' - Label 'Protein property calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0243' - Label 'Protein property calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0244' - Label 'Simulation analysis'\n",
      "Entity 'http://edamontology.org/operation_0245' - Label 'Structural motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0245' - Label 'Structural motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0245' - Label 'Structural motif discovery'\n",
      "Entity 'http://edamontology.org/operation_0246' - Label 'Protein domain recognition'\n",
      "Entity 'http://edamontology.org/operation_0246' - Label 'Protein domain recognition'\n",
      "Entity 'http://edamontology.org/operation_0246' - Label 'Protein domain recognition'\n",
      "Entity 'http://edamontology.org/operation_0247' - Label 'Protein architecture analysis'\n",
      "Entity 'http://edamontology.org/operation_0248' - Label 'Residue interaction calculation'\n",
      "Entity 'http://edamontology.org/operation_0248' - Label 'Residue interaction calculation'\n",
      "Entity 'http://edamontology.org/operation_0249' - Label 'Protein geometry calculation'\n",
      "Entity 'http://edamontology.org/operation_0249' - Label 'Protein geometry calculation'\n",
      "Entity 'http://edamontology.org/operation_0250' - Label 'Protein property calculation'\n",
      "Entity 'http://edamontology.org/operation_0250' - Label 'Protein property calculation'\n",
      "Entity 'http://edamontology.org/operation_0252' - Label 'Peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0252' - Label 'Peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0252' - Label 'Peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0252' - Label 'Peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0253' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_0253' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_0253' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_0253' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_0254' - Label 'Data retrieval (feature table)'\n",
      "Entity 'http://edamontology.org/operation_0254' - Label 'Data retrieval (feature table)'\n",
      "Entity 'http://edamontology.org/operation_0255' - Label 'Feature table query'\n",
      "Entity 'http://edamontology.org/operation_0255' - Label 'Feature table query'\n",
      "Entity 'http://edamontology.org/operation_0256' - Label 'Sequence feature comparison'\n",
      "Entity 'http://edamontology.org/operation_0256' - Label 'Sequence feature comparison'\n",
      "Entity 'http://edamontology.org/operation_0256' - Label 'Sequence feature comparison'\n",
      "Entity 'http://edamontology.org/operation_0256' - Label 'Sequence feature comparison'\n",
      "Entity 'http://edamontology.org/operation_0257' - Label 'Data retrieval (sequence alignment)'\n",
      "Entity 'http://edamontology.org/operation_0257' - Label 'Data retrieval (sequence alignment)'\n",
      "Entity 'http://edamontology.org/operation_0258' - Label 'Sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_0258' - Label 'Sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_0259' - Label 'Sequence alignment comparison'\n",
      "Entity 'http://edamontology.org/operation_0259' - Label 'Sequence alignment comparison'\n",
      "Entity 'http://edamontology.org/operation_0260' - Label 'Sequence alignment conversion'\n",
      "Entity 'http://edamontology.org/operation_0260' - Label 'Sequence alignment conversion'\n",
      "Entity 'http://edamontology.org/operation_0261' - Label 'Nucleic acid property processing'\n",
      "Entity 'http://edamontology.org/operation_0261' - Label 'Nucleic acid property processing'\n",
      "Entity 'http://edamontology.org/operation_0262' - Label 'Nucleic acid property calculation'\n",
      "Entity 'http://edamontology.org/operation_0262' - Label 'Nucleic acid property calculation'\n",
      "Entity 'http://edamontology.org/operation_0264' - Label 'Alternative splicing prediction'\n",
      "Entity 'http://edamontology.org/operation_0264' - Label 'Alternative splicing prediction'\n",
      "Entity 'http://edamontology.org/operation_0265' - Label 'Frameshift detection'\n",
      "Entity 'http://edamontology.org/operation_0265' - Label 'Frameshift detection'\n",
      "Entity 'http://edamontology.org/operation_0265' - Label 'Frameshift detection'\n",
      "Entity 'http://edamontology.org/operation_0266' - Label 'Vector sequence detection'\n",
      "Entity 'http://edamontology.org/operation_0267' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0267' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0267' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0267' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0268' - Label 'Protein super-secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0268' - Label 'Protein super-secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0269' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/operation_0269' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/operation_0270' - Label 'Transmembrane protein analysis'\n",
      "Entity 'http://edamontology.org/operation_0270' - Label 'Transmembrane protein analysis'\n",
      "Entity 'http://edamontology.org/operation_0271' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0271' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0271' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0272' - Label 'Residue contact prediction'\n",
      "Entity 'http://edamontology.org/operation_0272' - Label 'Residue contact prediction'\n",
      "Entity 'http://edamontology.org/operation_0272' - Label 'Residue contact prediction'\n",
      "Entity 'http://edamontology.org/operation_0273' - Label 'Protein interaction raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_0273' - Label 'Protein interaction raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_0274' - Label 'Protein-protein interaction prediction (from protein sequence)'\n",
      "Entity 'http://edamontology.org/operation_0274' - Label 'Protein-protein interaction prediction (from protein sequence)'\n",
      "Entity 'http://edamontology.org/operation_0275' - Label 'Protein-protein interaction prediction (from protein structure)'\n",
      "Entity 'http://edamontology.org/operation_0275' - Label 'Protein-protein interaction prediction (from protein structure)'\n",
      "Entity 'http://edamontology.org/operation_0276' - Label 'Protein interaction network analysis'\n",
      "Entity 'http://edamontology.org/operation_0276' - Label 'Protein interaction network analysis'\n",
      "Entity 'http://edamontology.org/operation_0276' - Label 'Protein interaction network analysis'\n",
      "Entity 'http://edamontology.org/operation_0276' - Label 'Protein interaction network analysis'\n",
      "Entity 'http://edamontology.org/operation_0277' - Label 'Pathway or network comparison'\n",
      "Entity 'http://edamontology.org/operation_0277' - Label 'Pathway or network comparison'\n",
      "Entity 'http://edamontology.org/operation_0277' - Label 'Pathway or network comparison'\n",
      "Entity 'http://edamontology.org/operation_0278' - Label 'RNA secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0278' - Label 'RNA secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0278' - Label 'RNA secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0278' - Label 'RNA secondary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0279' - Label 'Nucleic acid folding analysis'\n",
      "Entity 'http://edamontology.org/operation_0279' - Label 'Nucleic acid folding analysis'\n",
      "Entity 'http://edamontology.org/operation_0279' - Label 'Nucleic acid folding analysis'\n",
      "Entity 'http://edamontology.org/operation_0279' - Label 'Nucleic acid folding analysis'\n",
      "Entity 'http://edamontology.org/operation_0280' - Label 'Data retrieval (restriction enzyme annotation)'\n",
      "Entity 'http://edamontology.org/operation_0280' - Label 'Data retrieval (restriction enzyme annotation)'\n",
      "Entity 'http://edamontology.org/operation_0281' - Label 'Genetic marker identification'\n",
      "Entity 'http://edamontology.org/operation_0281' - Label 'Genetic marker identification'\n",
      "Entity 'http://edamontology.org/operation_0282' - Label 'Genetic mapping'\n",
      "Entity 'http://edamontology.org/operation_0282' - Label 'Genetic mapping'\n",
      "Entity 'http://edamontology.org/operation_0282' - Label 'Genetic mapping'\n",
      "Entity 'http://edamontology.org/operation_0283' - Label 'Linkage analysis'\n",
      "Entity 'http://edamontology.org/operation_0283' - Label 'Linkage analysis'\n",
      "Entity 'http://edamontology.org/operation_0283' - Label 'Linkage analysis'\n",
      "Entity 'http://edamontology.org/operation_0284' - Label 'Codon usage table generation'\n",
      "Entity 'http://edamontology.org/operation_0284' - Label 'Codon usage table generation'\n",
      "Entity 'http://edamontology.org/operation_0284' - Label 'Codon usage table generation'\n",
      "Entity 'http://edamontology.org/operation_0285' - Label 'Codon usage table comparison'\n",
      "Entity 'http://edamontology.org/operation_0285' - Label 'Codon usage table comparison'\n",
      "Entity 'http://edamontology.org/operation_0286' - Label 'Codon usage analysis'\n",
      "Entity 'http://edamontology.org/operation_0286' - Label 'Codon usage analysis'\n",
      "Entity 'http://edamontology.org/operation_0286' - Label 'Codon usage analysis'\n",
      "Entity 'http://edamontology.org/operation_0286' - Label 'Codon usage analysis'\n",
      "Entity 'http://edamontology.org/operation_0286' - Label 'Codon usage analysis'\n",
      "Entity 'http://edamontology.org/operation_0286' - Label 'Codon usage analysis'\n",
      "Entity 'http://edamontology.org/operation_0287' - Label 'Base position variability plotting'\n",
      "Entity 'http://edamontology.org/operation_0287' - Label 'Base position variability plotting'\n",
      "Entity 'http://edamontology.org/operation_0287' - Label 'Base position variability plotting'\n",
      "Entity 'http://edamontology.org/operation_0287' - Label 'Base position variability plotting'\n",
      "Entity 'http://edamontology.org/operation_0288' - Label 'Sequence word comparison'\n",
      "Entity 'http://edamontology.org/operation_0289' - Label 'Sequence distance matrix generation'\n",
      "Entity 'http://edamontology.org/operation_0289' - Label 'Sequence distance matrix generation'\n",
      "Entity 'http://edamontology.org/operation_0289' - Label 'Sequence distance matrix generation'\n",
      "Entity 'http://edamontology.org/operation_0289' - Label 'Sequence distance matrix generation'\n",
      "Entity 'http://edamontology.org/operation_0290' - Label 'Sequence redundancy removal'\n",
      "Entity 'http://edamontology.org/operation_0290' - Label 'Sequence redundancy removal'\n",
      "Entity 'http://edamontology.org/operation_0291' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/operation_0291' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/operation_0291' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/operation_0291' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/operation_0292' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0292' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0292' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0292' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0293' - Label 'Hybrid sequence alignment construction'\n",
      "Entity 'http://edamontology.org/operation_0293' - Label 'Hybrid sequence alignment construction'\n",
      "Entity 'http://edamontology.org/operation_0294' - Label 'Structure-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0295' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0295' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0295' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0295' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0296' - Label 'Sequence profile generation'\n",
      "Entity 'http://edamontology.org/operation_0296' - Label 'Sequence profile generation'\n",
      "Entity 'http://edamontology.org/operation_0296' - Label 'Sequence profile generation'\n",
      "Entity 'http://edamontology.org/operation_0296' - Label 'Sequence profile generation'\n",
      "Entity 'http://edamontology.org/operation_0296' - Label 'Sequence profile generation'\n",
      "Entity 'http://edamontology.org/operation_0297' - Label '3D profile generation'\n",
      "Entity 'http://edamontology.org/operation_0297' - Label '3D profile generation'\n",
      "Entity 'http://edamontology.org/operation_0297' - Label '3D profile generation'\n",
      "Entity 'http://edamontology.org/operation_0297' - Label '3D profile generation'\n",
      "Entity 'http://edamontology.org/operation_0297' - Label '3D profile generation'\n",
      "Entity 'http://edamontology.org/operation_0298' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0298' - Label 'Profile-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0299' - Label '3D profile-to-3D profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0299' - Label '3D profile-to-3D profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0300' - Label 'Sequence profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0300' - Label 'Sequence profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0300' - Label 'Sequence profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0301' - Label 'Sequence-to-3D-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0301' - Label 'Sequence-to-3D-profile alignment'\n",
      "Entity 'http://edamontology.org/operation_0302' - Label 'Protein threading'\n",
      "Entity 'http://edamontology.org/operation_0302' - Label 'Protein threading'\n",
      "Entity 'http://edamontology.org/operation_0302' - Label 'Protein threading'\n",
      "Entity 'http://edamontology.org/operation_0303' - Label 'Fold recognition'\n",
      "Entity 'http://edamontology.org/operation_0303' - Label 'Fold recognition'\n",
      "Entity 'http://edamontology.org/operation_0303' - Label 'Fold recognition'\n",
      "Entity 'http://edamontology.org/operation_0303' - Label 'Fold recognition'\n",
      "Entity 'http://edamontology.org/operation_0303' - Label 'Fold recognition'\n",
      "Entity 'http://edamontology.org/operation_0304' - Label 'Metadata retrieval'\n",
      "Entity 'http://edamontology.org/operation_0304' - Label 'Metadata retrieval'\n",
      "Entity 'http://edamontology.org/operation_0305' - Label 'Literature search'\n",
      "Entity 'http://edamontology.org/operation_0305' - Label 'Literature search'\n",
      "Entity 'http://edamontology.org/operation_0306' - Label 'Text mining'\n",
      "Entity 'http://edamontology.org/operation_0306' - Label 'Text mining'\n",
      "Entity 'http://edamontology.org/operation_0306' - Label 'Text mining'\n",
      "Entity 'http://edamontology.org/operation_0306' - Label 'Text mining'\n",
      "Entity 'http://edamontology.org/operation_0306' - Label 'Text mining'\n",
      "Entity 'http://edamontology.org/operation_0307' - Label 'Virtual PCR'\n",
      "Entity 'http://edamontology.org/operation_0308' - Label 'PCR primer design'\n",
      "Entity 'http://edamontology.org/operation_0308' - Label 'PCR primer design'\n",
      "Entity 'http://edamontology.org/operation_0308' - Label 'PCR primer design'\n",
      "Entity 'http://edamontology.org/operation_0308' - Label 'PCR primer design'\n",
      "Entity 'http://edamontology.org/operation_0309' - Label 'Microarray probe design'\n",
      "Entity 'http://edamontology.org/operation_0309' - Label 'Microarray probe design'\n",
      "Entity 'http://edamontology.org/operation_0309' - Label 'Microarray probe design'\n",
      "Entity 'http://edamontology.org/operation_0309' - Label 'Microarray probe design'\n",
      "Entity 'http://edamontology.org/operation_0309' - Label 'Microarray probe design'\n",
      "Entity 'http://edamontology.org/operation_0309' - Label 'Microarray probe design'\n",
      "Entity 'http://edamontology.org/operation_0310' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/operation_0310' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/operation_0310' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/operation_0311' - Label 'Microarray data standardisation and normalisation'\n",
      "Entity 'http://edamontology.org/operation_0311' - Label 'Microarray data standardisation and normalisation'\n",
      "Entity 'http://edamontology.org/operation_0312' - Label 'Sequencing-based expression profile data processing'\n",
      "Entity 'http://edamontology.org/operation_0312' - Label 'Sequencing-based expression profile data processing'\n",
      "Entity 'http://edamontology.org/operation_0313' - Label 'Expression profile clustering'\n",
      "Entity 'http://edamontology.org/operation_0313' - Label 'Expression profile clustering'\n",
      "Entity 'http://edamontology.org/operation_0313' - Label 'Expression profile clustering'\n",
      "Entity 'http://edamontology.org/operation_0314' - Label 'Gene expression profiling'\n",
      "Entity 'http://edamontology.org/operation_0315' - Label 'Expression profile comparison'\n",
      "Entity 'http://edamontology.org/operation_0316' - Label 'Functional profiling'\n",
      "Entity 'http://edamontology.org/operation_0316' - Label 'Functional profiling'\n",
      "Entity 'http://edamontology.org/operation_0317' - Label 'EST and cDNA sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0317' - Label 'EST and cDNA sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_0318' - Label 'Structural genomics target selection'\n",
      "Entity 'http://edamontology.org/operation_0318' - Label 'Structural genomics target selection'\n",
      "Entity 'http://edamontology.org/operation_0319' - Label 'Protein secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0319' - Label 'Protein secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0319' - Label 'Protein secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0320' - Label 'Protein structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0320' - Label 'Protein structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0320' - Label 'Protein structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0320' - Label 'Protein structure assignment'\n",
      "Entity 'http://edamontology.org/operation_0321' - Label 'Protein structure validation'\n",
      "Entity 'http://edamontology.org/operation_0321' - Label 'Protein structure validation'\n",
      "Entity 'http://edamontology.org/operation_0321' - Label 'Protein structure validation'\n",
      "Entity 'http://edamontology.org/operation_0321' - Label 'Protein structure validation'\n",
      "Entity 'http://edamontology.org/operation_0322' - Label 'Molecular model refinement'\n",
      "Entity 'http://edamontology.org/operation_0322' - Label 'Molecular model refinement'\n",
      "Entity 'http://edamontology.org/operation_0323' - Label 'Phylogenetic inference'\n",
      "Entity 'http://edamontology.org/operation_0323' - Label 'Phylogenetic inference'\n",
      "Entity 'http://edamontology.org/operation_0323' - Label 'Phylogenetic inference'\n",
      "Entity 'http://edamontology.org/operation_0323' - Label 'Phylogenetic inference'\n",
      "Entity 'http://edamontology.org/operation_0324' - Label 'Phylogenetic analysis'\n",
      "Entity 'http://edamontology.org/operation_0324' - Label 'Phylogenetic analysis'\n",
      "Entity 'http://edamontology.org/operation_0325' - Label 'Phylogenetic tree comparison'\n",
      "Entity 'http://edamontology.org/operation_0325' - Label 'Phylogenetic tree comparison'\n",
      "Entity 'http://edamontology.org/operation_0326' - Label 'Phylogenetic tree editing'\n",
      "Entity 'http://edamontology.org/operation_0326' - Label 'Phylogenetic tree editing'\n",
      "Entity 'http://edamontology.org/operation_0326' - Label 'Phylogenetic tree editing'\n",
      "Entity 'http://edamontology.org/operation_0326' - Label 'Phylogenetic tree editing'\n",
      "Entity 'http://edamontology.org/operation_0327' - Label 'Phylogenetic footprinting'\n",
      "Entity 'http://edamontology.org/operation_0327' - Label 'Phylogenetic footprinting'\n",
      "Entity 'http://edamontology.org/operation_0328' - Label 'Protein folding simulation'\n",
      "Entity 'http://edamontology.org/operation_0328' - Label 'Protein folding simulation'\n",
      "Entity 'http://edamontology.org/operation_0329' - Label 'Protein folding pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_0329' - Label 'Protein folding pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_0330' - Label 'Protein SNP mapping'\n",
      "Entity 'http://edamontology.org/operation_0330' - Label 'Protein SNP mapping'\n",
      "Entity 'http://edamontology.org/operation_0331' - Label 'Variant effect prediction'\n",
      "Entity 'http://edamontology.org/operation_0331' - Label 'Variant effect prediction'\n",
      "Entity 'http://edamontology.org/operation_0331' - Label 'Variant effect prediction'\n",
      "Entity 'http://edamontology.org/operation_0332' - Label 'Immunogen design'\n",
      "Entity 'http://edamontology.org/operation_0332' - Label 'Immunogen design'\n",
      "Entity 'http://edamontology.org/operation_0333' - Label 'Zinc finger prediction'\n",
      "Entity 'http://edamontology.org/operation_0333' - Label 'Zinc finger prediction'\n",
      "Entity 'http://edamontology.org/operation_0334' - Label 'Enzyme kinetics calculation'\n",
      "Entity 'http://edamontology.org/operation_0334' - Label 'Enzyme kinetics calculation'\n",
      "Entity 'http://edamontology.org/operation_0334' - Label 'Enzyme kinetics calculation'\n",
      "Entity 'http://edamontology.org/operation_0335' - Label 'Formatting'\n",
      "Entity 'http://edamontology.org/operation_0336' - Label 'Format validation'\n",
      "Entity 'http://edamontology.org/operation_0337' - Label 'Visualisation'\n",
      "Entity 'http://edamontology.org/operation_0337' - Label 'Visualisation'\n",
      "Entity 'http://edamontology.org/operation_0337' - Label 'Visualisation'\n",
      "Entity 'http://edamontology.org/operation_0337' - Label 'Visualisation'\n",
      "Entity 'http://edamontology.org/operation_0338' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/operation_0338' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/operation_0338' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/operation_0339' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/operation_0339' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/operation_0340' - Label 'Protein secondary database search'\n",
      "Entity 'http://edamontology.org/operation_0340' - Label 'Protein secondary database search'\n",
      "Entity 'http://edamontology.org/operation_0341' - Label 'Motif database search'\n",
      "Entity 'http://edamontology.org/operation_0341' - Label 'Motif database search'\n",
      "Entity 'http://edamontology.org/operation_0342' - Label 'Sequence profile database search'\n",
      "Entity 'http://edamontology.org/operation_0342' - Label 'Sequence profile database search'\n",
      "Entity 'http://edamontology.org/operation_0343' - Label 'Transmembrane protein database search'\n",
      "Entity 'http://edamontology.org/operation_0343' - Label 'Transmembrane protein database search'\n",
      "Entity 'http://edamontology.org/operation_0344' - Label 'Sequence retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0344' - Label 'Sequence retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0345' - Label 'Sequence retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0345' - Label 'Sequence retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0346' - Label 'Sequence similarity search'\n",
      "Entity 'http://edamontology.org/operation_0346' - Label 'Sequence similarity search'\n",
      "Entity 'http://edamontology.org/operation_0346' - Label 'Sequence similarity search'\n",
      "Entity 'http://edamontology.org/operation_0347' - Label 'Sequence database search (by motif or pattern)'\n",
      "Entity 'http://edamontology.org/operation_0347' - Label 'Sequence database search (by motif or pattern)'\n",
      "Entity 'http://edamontology.org/operation_0348' - Label 'Sequence database search (by amino acid composition)'\n",
      "Entity 'http://edamontology.org/operation_0348' - Label 'Sequence database search (by amino acid composition)'\n",
      "Entity 'http://edamontology.org/operation_0349' - Label 'Sequence database search (by property)'\n",
      "Entity 'http://edamontology.org/operation_0350' - Label 'Sequence database search (by sequence using word-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0350' - Label 'Sequence database search (by sequence using word-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0351' - Label 'Sequence database search (by sequence using profile-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0351' - Label 'Sequence database search (by sequence using profile-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0352' - Label 'Sequence database search (by sequence using local alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0352' - Label 'Sequence database search (by sequence using local alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0353' - Label 'Sequence database search (by sequence using global alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0353' - Label 'Sequence database search (by sequence using global alignment-based methods)'\n",
      "Entity 'http://edamontology.org/operation_0354' - Label 'Sequence database search (by sequence for primer sequences)'\n",
      "Entity 'http://edamontology.org/operation_0354' - Label 'Sequence database search (by sequence for primer sequences)'\n",
      "Entity 'http://edamontology.org/operation_0355' - Label 'Sequence database search (by molecular weight)'\n",
      "Entity 'http://edamontology.org/operation_0355' - Label 'Sequence database search (by molecular weight)'\n",
      "Entity 'http://edamontology.org/operation_0356' - Label 'Sequence database search (by isoelectric point)'\n",
      "Entity 'http://edamontology.org/operation_0356' - Label 'Sequence database search (by isoelectric point)'\n",
      "Entity 'http://edamontology.org/operation_0357' - Label 'Structure retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0357' - Label 'Structure retrieval (by code)'\n",
      "Entity 'http://edamontology.org/operation_0358' - Label 'Structure retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0358' - Label 'Structure retrieval (by keyword)'\n",
      "Entity 'http://edamontology.org/operation_0359' - Label 'Structure database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_0359' - Label 'Structure database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_0360' - Label 'Structural similarity search'\n",
      "Entity 'http://edamontology.org/operation_0360' - Label 'Structural similarity search'\n",
      "Entity 'http://edamontology.org/operation_0361' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/operation_0361' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/operation_0361' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/operation_0362' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/operation_0362' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/operation_0363' - Label 'Reverse complement'\n",
      "Entity 'http://edamontology.org/operation_0364' - Label 'Random sequence generation'\n",
      "Entity 'http://edamontology.org/operation_0365' - Label 'Restriction digest'\n",
      "Entity 'http://edamontology.org/operation_0365' - Label 'Restriction digest'\n",
      "Entity 'http://edamontology.org/operation_0365' - Label 'Restriction digest'\n",
      "Entity 'http://edamontology.org/operation_0366' - Label 'Protein sequence cleavage'\n",
      "Entity 'http://edamontology.org/operation_0366' - Label 'Protein sequence cleavage'\n",
      "Entity 'http://edamontology.org/operation_0366' - Label 'Protein sequence cleavage'\n",
      "Entity 'http://edamontology.org/operation_0367' - Label 'Sequence mutation and randomisation'\n",
      "Entity 'http://edamontology.org/operation_0368' - Label 'Sequence masking'\n",
      "Entity 'http://edamontology.org/operation_0369' - Label 'Sequence cutting'\n",
      "Entity 'http://edamontology.org/operation_0370' - Label 'Restriction site creation'\n",
      "Entity 'http://edamontology.org/operation_0371' - Label 'DNA translation'\n",
      "Entity 'http://edamontology.org/operation_0371' - Label 'DNA translation'\n",
      "Entity 'http://edamontology.org/operation_0372' - Label 'DNA transcription'\n",
      "Entity 'http://edamontology.org/operation_0372' - Label 'DNA transcription'\n",
      "Entity 'http://edamontology.org/operation_0377' - Label 'Sequence composition calculation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_0377' - Label 'Sequence composition calculation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_0378' - Label 'Sequence composition calculation (protein)'\n",
      "Entity 'http://edamontology.org/operation_0378' - Label 'Sequence composition calculation (protein)'\n",
      "Entity 'http://edamontology.org/operation_0379' - Label 'Repeat sequence detection'\n",
      "Entity 'http://edamontology.org/operation_0379' - Label 'Repeat sequence detection'\n",
      "Entity 'http://edamontology.org/operation_0380' - Label 'Repeat sequence organisation analysis'\n",
      "Entity 'http://edamontology.org/operation_0380' - Label 'Repeat sequence organisation analysis'\n",
      "Entity 'http://edamontology.org/operation_0383' - Label 'Protein hydropathy calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0383' - Label 'Protein hydropathy calculation (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0384' - Label 'Accessible surface calculation'\n",
      "Entity 'http://edamontology.org/operation_0384' - Label 'Accessible surface calculation'\n",
      "Entity 'http://edamontology.org/operation_0385' - Label 'Protein hydropathy cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0385' - Label 'Protein hydropathy cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0386' - Label 'Protein dipole moment calculation'\n",
      "Entity 'http://edamontology.org/operation_0386' - Label 'Protein dipole moment calculation'\n",
      "Entity 'http://edamontology.org/operation_0387' - Label 'Molecular surface calculation'\n",
      "Entity 'http://edamontology.org/operation_0388' - Label 'Protein binding site prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0388' - Label 'Protein binding site prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_0389' - Label 'Protein-nucleic acid interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_0389' - Label 'Protein-nucleic acid interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_0390' - Label 'Protein peeling'\n",
      "Entity 'http://edamontology.org/operation_0391' - Label 'Protein distance matrix calculation'\n",
      "Entity 'http://edamontology.org/operation_0391' - Label 'Protein distance matrix calculation'\n",
      "Entity 'http://edamontology.org/operation_0392' - Label 'Contact map calculation'\n",
      "Entity 'http://edamontology.org/operation_0392' - Label 'Contact map calculation'\n",
      "Entity 'http://edamontology.org/operation_0393' - Label 'Residue cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0393' - Label 'Residue cluster calculation'\n",
      "Entity 'http://edamontology.org/operation_0394' - Label 'Hydrogen bond calculation'\n",
      "Entity 'http://edamontology.org/operation_0394' - Label 'Hydrogen bond calculation'\n",
      "Entity 'http://edamontology.org/operation_0395' - Label 'Residue non-canonical interaction detection'\n",
      "Entity 'http://edamontology.org/operation_0395' - Label 'Residue non-canonical interaction detection'\n",
      "Entity 'http://edamontology.org/operation_0396' - Label 'Ramachandran plot calculation'\n",
      "Entity 'http://edamontology.org/operation_0396' - Label 'Ramachandran plot calculation'\n",
      "Entity 'http://edamontology.org/operation_0397' - Label 'Ramachandran plot validation'\n",
      "Entity 'http://edamontology.org/operation_0397' - Label 'Ramachandran plot validation'\n",
      "Entity 'http://edamontology.org/operation_0398' - Label 'Protein molecular weight calculation'\n",
      "Entity 'http://edamontology.org/operation_0398' - Label 'Protein molecular weight calculation'\n",
      "Entity 'http://edamontology.org/operation_0398' - Label 'Protein molecular weight calculation'\n",
      "Entity 'http://edamontology.org/operation_0399' - Label 'Protein extinction coefficient calculation'\n",
      "Entity 'http://edamontology.org/operation_0399' - Label 'Protein extinction coefficient calculation'\n",
      "Entity 'http://edamontology.org/operation_0400' - Label 'Protein pKa calculation'\n",
      "Entity 'http://edamontology.org/operation_0400' - Label 'Protein pKa calculation'\n",
      "Entity 'http://edamontology.org/operation_0400' - Label 'Protein pKa calculation'\n",
      "Entity 'http://edamontology.org/operation_0401' - Label 'Protein hydropathy calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0401' - Label 'Protein hydropathy calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0402' - Label 'Protein titration curve plotting'\n",
      "Entity 'http://edamontology.org/operation_0402' - Label 'Protein titration curve plotting'\n",
      "Entity 'http://edamontology.org/operation_0402' - Label 'Protein titration curve plotting'\n",
      "Entity 'http://edamontology.org/operation_0403' - Label 'Protein isoelectric point calculation'\n",
      "Entity 'http://edamontology.org/operation_0403' - Label 'Protein isoelectric point calculation'\n",
      "Entity 'http://edamontology.org/operation_0404' - Label 'Protein hydrogen exchange rate calculation'\n",
      "Entity 'http://edamontology.org/operation_0404' - Label 'Protein hydrogen exchange rate calculation'\n",
      "Entity 'http://edamontology.org/operation_0405' - Label 'Protein hydrophobic region calculation'\n",
      "Entity 'http://edamontology.org/operation_0406' - Label 'Protein aliphatic index calculation'\n",
      "Entity 'http://edamontology.org/operation_0406' - Label 'Protein aliphatic index calculation'\n",
      "Entity 'http://edamontology.org/operation_0407' - Label 'Protein hydrophobic moment plotting'\n",
      "Entity 'http://edamontology.org/operation_0407' - Label 'Protein hydrophobic moment plotting'\n",
      "Entity 'http://edamontology.org/operation_0407' - Label 'Protein hydrophobic moment plotting'\n",
      "Entity 'http://edamontology.org/operation_0408' - Label 'Protein globularity prediction'\n",
      "Entity 'http://edamontology.org/operation_0408' - Label 'Protein globularity prediction'\n",
      "Entity 'http://edamontology.org/operation_0409' - Label 'Protein solubility prediction'\n",
      "Entity 'http://edamontology.org/operation_0409' - Label 'Protein solubility prediction'\n",
      "Entity 'http://edamontology.org/operation_0410' - Label 'Protein crystallizability prediction'\n",
      "Entity 'http://edamontology.org/operation_0410' - Label 'Protein crystallizability prediction'\n",
      "Entity 'http://edamontology.org/operation_0411' - Label 'Protein signal peptide detection (eukaryotes)'\n",
      "Entity 'http://edamontology.org/operation_0411' - Label 'Protein signal peptide detection (eukaryotes)'\n",
      "Entity 'http://edamontology.org/operation_0412' - Label 'Protein signal peptide detection (bacteria)'\n",
      "Entity 'http://edamontology.org/operation_0412' - Label 'Protein signal peptide detection (bacteria)'\n",
      "Entity 'http://edamontology.org/operation_0413' - Label 'MHC peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0413' - Label 'MHC peptide immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0414' - Label 'Protein feature prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0414' - Label 'Protein feature prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0415' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/operation_0415' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/operation_0415' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/operation_0415' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/operation_0416' - Label 'Epitope mapping'\n",
      "Entity 'http://edamontology.org/operation_0416' - Label 'Epitope mapping'\n",
      "Entity 'http://edamontology.org/operation_0416' - Label 'Epitope mapping'\n",
      "Entity 'http://edamontology.org/operation_0417' - Label 'Post-translational modification site prediction'\n",
      "Entity 'http://edamontology.org/operation_0417' - Label 'Post-translational modification site prediction'\n",
      "Entity 'http://edamontology.org/operation_0418' - Label 'Protein signal peptide detection'\n",
      "Entity 'http://edamontology.org/operation_0418' - Label 'Protein signal peptide detection'\n",
      "Entity 'http://edamontology.org/operation_0418' - Label 'Protein signal peptide detection'\n",
      "Entity 'http://edamontology.org/operation_0419' - Label 'Protein binding site prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0419' - Label 'Protein binding site prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_0420' - Label 'Nucleic acids-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0421' - Label 'Protein folding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0421' - Label 'Protein folding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0422' - Label 'Protein cleavage site prediction'\n",
      "Entity 'http://edamontology.org/operation_0422' - Label 'Protein cleavage site prediction'\n",
      "Entity 'http://edamontology.org/operation_0423' - Label 'Epitope mapping (MHC Class I)'\n",
      "Entity 'http://edamontology.org/operation_0423' - Label 'Epitope mapping (MHC Class I)'\n",
      "Entity 'http://edamontology.org/operation_0424' - Label 'Epitope mapping (MHC Class II)'\n",
      "Entity 'http://edamontology.org/operation_0424' - Label 'Epitope mapping (MHC Class II)'\n",
      "Entity 'http://edamontology.org/operation_0425' - Label 'Whole gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0425' - Label 'Whole gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0426' - Label 'Gene component prediction'\n",
      "Entity 'http://edamontology.org/operation_0426' - Label 'Gene component prediction'\n",
      "Entity 'http://edamontology.org/operation_0427' - Label 'Transposon prediction'\n",
      "Entity 'http://edamontology.org/operation_0428' - Label 'PolyA signal detection'\n",
      "Entity 'http://edamontology.org/operation_0429' - Label 'Quadruplex formation site detection'\n",
      "Entity 'http://edamontology.org/operation_0429' - Label 'Quadruplex formation site detection'\n",
      "Entity 'http://edamontology.org/operation_0430' - Label 'CpG island and isochore detection'\n",
      "Entity 'http://edamontology.org/operation_0430' - Label 'CpG island and isochore detection'\n",
      "Entity 'http://edamontology.org/operation_0431' - Label 'Restriction site recognition'\n",
      "Entity 'http://edamontology.org/operation_0431' - Label 'Restriction site recognition'\n",
      "Entity 'http://edamontology.org/operation_0432' - Label 'Nucleosome position prediction'\n",
      "Entity 'http://edamontology.org/operation_0433' - Label 'Splice site prediction'\n",
      "Entity 'http://edamontology.org/operation_0433' - Label 'Splice site prediction'\n",
      "Entity 'http://edamontology.org/operation_0434' - Label 'Integrated gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0434' - Label 'Integrated gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0435' - Label 'Operon prediction'\n",
      "Entity 'http://edamontology.org/operation_0436' - Label 'Coding region prediction'\n",
      "Entity 'http://edamontology.org/operation_0437' - Label 'SECIS element prediction'\n",
      "Entity 'http://edamontology.org/operation_0437' - Label 'SECIS element prediction'\n",
      "Entity 'http://edamontology.org/operation_0438' - Label 'Transcriptional regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0438' - Label 'Transcriptional regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0439' - Label 'Translation initiation site prediction'\n",
      "Entity 'http://edamontology.org/operation_0439' - Label 'Translation initiation site prediction'\n",
      "Entity 'http://edamontology.org/operation_0440' - Label 'Promoter prediction'\n",
      "Entity 'http://edamontology.org/operation_0441' - Label 'cis-regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0442' - Label 'Transcriptional regulatory element prediction (RNA-cis)'\n",
      "Entity 'http://edamontology.org/operation_0442' - Label 'Transcriptional regulatory element prediction (RNA-cis)'\n",
      "Entity 'http://edamontology.org/operation_0443' - Label 'trans-regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0443' - Label 'trans-regulatory element prediction'\n",
      "Entity 'http://edamontology.org/operation_0444' - Label 'S/MAR prediction'\n",
      "Entity 'http://edamontology.org/operation_0445' - Label 'Transcription factor binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_0446' - Label 'Exonic splicing enhancer prediction'\n",
      "Entity 'http://edamontology.org/operation_0446' - Label 'Exonic splicing enhancer prediction'\n",
      "Entity 'http://edamontology.org/operation_0447' - Label 'Sequence alignment validation'\n",
      "Entity 'http://edamontology.org/operation_0447' - Label 'Sequence alignment validation'\n",
      "Entity 'http://edamontology.org/operation_0448' - Label 'Sequence alignment analysis (conservation)'\n",
      "Entity 'http://edamontology.org/operation_0449' - Label 'Sequence alignment analysis (site correlation)'\n",
      "Entity 'http://edamontology.org/operation_0449' - Label 'Sequence alignment analysis (site correlation)'\n",
      "Entity 'http://edamontology.org/operation_0450' - Label 'Chimera detection'\n",
      "Entity 'http://edamontology.org/operation_0451' - Label 'Recombination detection'\n",
      "Entity 'http://edamontology.org/operation_0452' - Label 'Indel detection'\n",
      "Entity 'http://edamontology.org/operation_0453' - Label 'Nucleosome formation potential prediction'\n",
      "Entity 'http://edamontology.org/operation_0453' - Label 'Nucleosome formation potential prediction'\n",
      "Entity 'http://edamontology.org/operation_0455' - Label 'Nucleic acid thermodynamic property calculation'\n",
      "Entity 'http://edamontology.org/operation_0455' - Label 'Nucleic acid thermodynamic property calculation'\n",
      "Entity 'http://edamontology.org/operation_0456' - Label 'Nucleic acid melting profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0456' - Label 'Nucleic acid melting profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0456' - Label 'Nucleic acid melting profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0457' - Label 'Nucleic acid stitch profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0458' - Label 'Nucleic acid melting curve plotting'\n",
      "Entity 'http://edamontology.org/operation_0459' - Label 'Nucleic acid probability profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0460' - Label 'Nucleic acid temperature profile plotting'\n",
      "Entity 'http://edamontology.org/operation_0461' - Label 'Nucleic acid curvature calculation'\n",
      "Entity 'http://edamontology.org/operation_0461' - Label 'Nucleic acid curvature calculation'\n",
      "Entity 'http://edamontology.org/operation_0463' - Label 'miRNA target prediction'\n",
      "Entity 'http://edamontology.org/operation_0464' - Label 'tRNA gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0464' - Label 'tRNA gene prediction'\n",
      "Entity 'http://edamontology.org/operation_0465' - Label 'siRNA binding specificity prediction'\n",
      "Entity 'http://edamontology.org/operation_0465' - Label 'siRNA binding specificity prediction'\n",
      "Entity 'http://edamontology.org/operation_0467' - Label 'Protein secondary structure prediction (integrated)'\n",
      "Entity 'http://edamontology.org/operation_0467' - Label 'Protein secondary structure prediction (integrated)'\n",
      "Entity 'http://edamontology.org/operation_0468' - Label 'Protein secondary structure prediction (helices)'\n",
      "Entity 'http://edamontology.org/operation_0469' - Label 'Protein secondary structure prediction (turns)'\n",
      "Entity 'http://edamontology.org/operation_0470' - Label 'Protein secondary structure prediction (coils)'\n",
      "Entity 'http://edamontology.org/operation_0471' - Label 'Disulfide bond prediction'\n",
      "Entity 'http://edamontology.org/operation_0472' - Label 'GPCR prediction'\n",
      "Entity 'http://edamontology.org/operation_0472' - Label 'GPCR prediction'\n",
      "Entity 'http://edamontology.org/operation_0473' - Label 'GPCR analysis'\n",
      "Entity 'http://edamontology.org/operation_0473' - Label 'GPCR analysis'\n",
      "Entity 'http://edamontology.org/operation_0474' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0474' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0474' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0474' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0475' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0475' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0475' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0476' - Label 'Ab initio structure prediction'\n",
      "Entity 'http://edamontology.org/operation_0477' - Label 'Protein modelling'\n",
      "Entity 'http://edamontology.org/operation_0477' - Label 'Protein modelling'\n",
      "Entity 'http://edamontology.org/operation_0477' - Label 'Protein modelling'\n",
      "Entity 'http://edamontology.org/operation_0478' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/operation_0478' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/operation_0478' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/operation_0478' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/operation_0479' - Label 'Backbone modelling'\n",
      "Entity 'http://edamontology.org/operation_0480' - Label 'Side chain modelling'\n",
      "Entity 'http://edamontology.org/operation_0481' - Label 'Loop modelling'\n",
      "Entity 'http://edamontology.org/operation_0482' - Label 'Protein-ligand docking'\n",
      "Entity 'http://edamontology.org/operation_0482' - Label 'Protein-ligand docking'\n",
      "Entity 'http://edamontology.org/operation_0482' - Label 'Protein-ligand docking'\n",
      "Entity 'http://edamontology.org/operation_0483' - Label 'RNA inverse folding'\n",
      "Entity 'http://edamontology.org/operation_0483' - Label 'RNA inverse folding'\n",
      "Entity 'http://edamontology.org/operation_0484' - Label 'SNP detection'\n",
      "Entity 'http://edamontology.org/operation_0485' - Label 'Radiation Hybrid Mapping'\n",
      "Entity 'http://edamontology.org/operation_0485' - Label 'Radiation Hybrid Mapping'\n",
      "Entity 'http://edamontology.org/operation_0486' - Label 'Functional mapping'\n",
      "Entity 'http://edamontology.org/operation_0486' - Label 'Functional mapping'\n",
      "Entity 'http://edamontology.org/operation_0487' - Label 'Haplotype mapping'\n",
      "Entity 'http://edamontology.org/operation_0487' - Label 'Haplotype mapping'\n",
      "Entity 'http://edamontology.org/operation_0488' - Label 'Linkage disequilibrium calculation'\n",
      "Entity 'http://edamontology.org/operation_0488' - Label 'Linkage disequilibrium calculation'\n",
      "Entity 'http://edamontology.org/operation_0489' - Label 'Genetic code prediction'\n",
      "Entity 'http://edamontology.org/operation_0489' - Label 'Genetic code prediction'\n",
      "Entity 'http://edamontology.org/operation_0489' - Label 'Genetic code prediction'\n",
      "Entity 'http://edamontology.org/operation_0490' - Label 'Dot plot plotting'\n",
      "Entity 'http://edamontology.org/operation_0490' - Label 'Dot plot plotting'\n",
      "Entity 'http://edamontology.org/operation_0490' - Label 'Dot plot plotting'\n",
      "Entity 'http://edamontology.org/operation_0491' - Label 'Pairwise sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0491' - Label 'Pairwise sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0492' - Label 'Multiple sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0493' - Label 'Pairwise sequence alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0493' - Label 'Pairwise sequence alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0493' - Label 'Pairwise sequence alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0494' - Label 'Pairwise sequence alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0494' - Label 'Pairwise sequence alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0494' - Label 'Pairwise sequence alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0495' - Label 'Local alignment'\n",
      "Entity 'http://edamontology.org/operation_0496' - Label 'Global alignment'\n",
      "Entity 'http://edamontology.org/operation_0497' - Label 'Constrained sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0497' - Label 'Constrained sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0498' - Label 'Consensus-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0498' - Label 'Consensus-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0499' - Label 'Tree-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0499' - Label 'Tree-based sequence alignment'\n",
      "Entity 'http://edamontology.org/operation_0500' - Label 'Secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0500' - Label 'Secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0501' - Label 'Protein secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0501' - Label 'Protein secondary structure alignment generation'\n",
      "Entity 'http://edamontology.org/operation_0502' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0502' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0502' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0502' - Label 'RNA secondary structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0503' - Label 'Pairwise structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0504' - Label 'Multiple structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0505' - Label 'Structure alignment (protein)'\n",
      "Entity 'http://edamontology.org/operation_0505' - Label 'Structure alignment (protein)'\n",
      "Entity 'http://edamontology.org/operation_0506' - Label 'Structure alignment (RNA)'\n",
      "Entity 'http://edamontology.org/operation_0506' - Label 'Structure alignment (RNA)'\n",
      "Entity 'http://edamontology.org/operation_0507' - Label 'Pairwise structure alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0507' - Label 'Pairwise structure alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0507' - Label 'Pairwise structure alignment generation (local)'\n",
      "Entity 'http://edamontology.org/operation_0508' - Label 'Pairwise structure alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0508' - Label 'Pairwise structure alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0508' - Label 'Pairwise structure alignment generation (global)'\n",
      "Entity 'http://edamontology.org/operation_0509' - Label 'Local structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0510' - Label 'Global structure alignment'\n",
      "Entity 'http://edamontology.org/operation_0511' - Label 'Profile-profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0511' - Label 'Profile-profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0511' - Label 'Profile-profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0512' - Label 'Sequence alignment generation (multiple profile)'\n",
      "Entity 'http://edamontology.org/operation_0512' - Label 'Sequence alignment generation (multiple profile)'\n",
      "Entity 'http://edamontology.org/operation_0512' - Label 'Sequence alignment generation (multiple profile)'\n",
      "Entity 'http://edamontology.org/operation_0513' - Label '3D profile-to-3D profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0513' - Label '3D profile-to-3D profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0513' - Label '3D profile-to-3D profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0513' - Label '3D profile-to-3D profile alignment (pairwise)'\n",
      "Entity 'http://edamontology.org/operation_0514' - Label 'Structural profile alignment generation (multiple)'\n",
      "Entity 'http://edamontology.org/operation_0514' - Label 'Structural profile alignment generation (multiple)'\n",
      "Entity 'http://edamontology.org/operation_0514' - Label 'Structural profile alignment generation (multiple)'\n",
      "Entity 'http://edamontology.org/operation_0514' - Label 'Structural profile alignment generation (multiple)'\n",
      "Entity 'http://edamontology.org/operation_0515' - Label 'Data retrieval (tool metadata)'\n",
      "Entity 'http://edamontology.org/operation_0515' - Label 'Data retrieval (tool metadata)'\n",
      "Entity 'http://edamontology.org/operation_0516' - Label 'Data retrieval (database metadata)'\n",
      "Entity 'http://edamontology.org/operation_0516' - Label 'Data retrieval (database metadata)'\n",
      "Entity 'http://edamontology.org/operation_0517' - Label 'PCR primer design (for large scale sequencing)'\n",
      "Entity 'http://edamontology.org/operation_0517' - Label 'PCR primer design (for large scale sequencing)'\n",
      "Entity 'http://edamontology.org/operation_0518' - Label 'PCR primer design (for genotyping polymorphisms)'\n",
      "Entity 'http://edamontology.org/operation_0518' - Label 'PCR primer design (for genotyping polymorphisms)'\n",
      "Entity 'http://edamontology.org/operation_0519' - Label 'PCR primer design (for gene transcription profiling)'\n",
      "Entity 'http://edamontology.org/operation_0519' - Label 'PCR primer design (for gene transcription profiling)'\n",
      "Entity 'http://edamontology.org/operation_0520' - Label 'PCR primer design (for conserved primers)'\n",
      "Entity 'http://edamontology.org/operation_0520' - Label 'PCR primer design (for conserved primers)'\n",
      "Entity 'http://edamontology.org/operation_0521' - Label 'PCR primer design (based on gene structure)'\n",
      "Entity 'http://edamontology.org/operation_0521' - Label 'PCR primer design (based on gene structure)'\n",
      "Entity 'http://edamontology.org/operation_0522' - Label 'PCR primer design (for methylation PCRs)'\n",
      "Entity 'http://edamontology.org/operation_0522' - Label 'PCR primer design (for methylation PCRs)'\n",
      "Entity 'http://edamontology.org/operation_0523' - Label 'Mapping assembly'\n",
      "Entity 'http://edamontology.org/operation_0524' - Label 'De-novo assembly'\n",
      "Entity 'http://edamontology.org/operation_0525' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_0525' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_0526' - Label 'EST assembly'\n",
      "Entity 'http://edamontology.org/operation_0527' - Label 'Sequence tag mapping'\n",
      "Entity 'http://edamontology.org/operation_0527' - Label 'Sequence tag mapping'\n",
      "Entity 'http://edamontology.org/operation_0528' - Label 'SAGE data processing'\n",
      "Entity 'http://edamontology.org/operation_0528' - Label 'SAGE data processing'\n",
      "Entity 'http://edamontology.org/operation_0529' - Label 'MPSS data processing'\n",
      "Entity 'http://edamontology.org/operation_0529' - Label 'MPSS data processing'\n",
      "Entity 'http://edamontology.org/operation_0530' - Label 'SBS data processing'\n",
      "Entity 'http://edamontology.org/operation_0530' - Label 'SBS data processing'\n",
      "Entity 'http://edamontology.org/operation_0531' - Label 'Heat map generation'\n",
      "Entity 'http://edamontology.org/operation_0531' - Label 'Heat map generation'\n",
      "Entity 'http://edamontology.org/operation_0531' - Label 'Heat map generation'\n",
      "Entity 'http://edamontology.org/operation_0532' - Label 'Gene expression profile analysis'\n",
      "Entity 'http://edamontology.org/operation_0532' - Label 'Gene expression profile analysis'\n",
      "Entity 'http://edamontology.org/operation_0533' - Label 'Expression profile pathway mapping'\n",
      "Entity 'http://edamontology.org/operation_0533' - Label 'Expression profile pathway mapping'\n",
      "Entity 'http://edamontology.org/operation_0533' - Label 'Expression profile pathway mapping'\n",
      "Entity 'http://edamontology.org/operation_0533' - Label 'Expression profile pathway mapping'\n",
      "Entity 'http://edamontology.org/operation_0534' - Label 'Protein secondary structure assignment (from coordinate data)'\n",
      "Entity 'http://edamontology.org/operation_0534' - Label 'Protein secondary structure assignment (from coordinate data)'\n",
      "Entity 'http://edamontology.org/operation_0535' - Label 'Protein secondary structure assignment (from CD data)'\n",
      "Entity 'http://edamontology.org/operation_0535' - Label 'Protein secondary structure assignment (from CD data)'\n",
      "Entity 'http://edamontology.org/operation_0536' - Label 'Protein structure assignment (from X-ray crystallographic data)'\n",
      "Entity 'http://edamontology.org/operation_0536' - Label 'Protein structure assignment (from X-ray crystallographic data)'\n",
      "Entity 'http://edamontology.org/operation_0537' - Label 'Protein structure assignment (from NMR data)'\n",
      "Entity 'http://edamontology.org/operation_0537' - Label 'Protein structure assignment (from NMR data)'\n",
      "Entity 'http://edamontology.org/operation_0538' - Label 'Phylogenetic inference (data centric)'\n",
      "Entity 'http://edamontology.org/operation_0539' - Label 'Phylogenetic inference (method centric)'\n",
      "Entity 'http://edamontology.org/operation_0540' - Label 'Phylogenetic inference (from molecular sequences)'\n",
      "Entity 'http://edamontology.org/operation_0540' - Label 'Phylogenetic inference (from molecular sequences)'\n",
      "Entity 'http://edamontology.org/operation_0541' - Label 'Phylogenetic inference (from continuous quantitative characters)'\n",
      "Entity 'http://edamontology.org/operation_0541' - Label 'Phylogenetic inference (from continuous quantitative characters)'\n",
      "Entity 'http://edamontology.org/operation_0542' - Label 'Phylogenetic inference (from gene frequencies)'\n",
      "Entity 'http://edamontology.org/operation_0542' - Label 'Phylogenetic inference (from gene frequencies)'\n",
      "Entity 'http://edamontology.org/operation_0542' - Label 'Phylogenetic inference (from gene frequencies)'\n",
      "Entity 'http://edamontology.org/operation_0543' - Label 'Phylogenetic inference (from polymorphism data)'\n",
      "Entity 'http://edamontology.org/operation_0543' - Label 'Phylogenetic inference (from polymorphism data)'\n",
      "Entity 'http://edamontology.org/operation_0544' - Label 'Species tree construction'\n",
      "Entity 'http://edamontology.org/operation_0545' - Label 'Phylogenetic inference (parsimony methods)'\n",
      "Entity 'http://edamontology.org/operation_0546' - Label 'Phylogenetic inference (minimum distance methods)'\n",
      "Entity 'http://edamontology.org/operation_0547' - Label 'Phylogenetic inference (maximum likelihood and Bayesian methods)'\n",
      "Entity 'http://edamontology.org/operation_0548' - Label 'Phylogenetic inference (quartet methods)'\n",
      "Entity 'http://edamontology.org/operation_0549' - Label 'Phylogenetic inference (AI methods)'\n",
      "Entity 'http://edamontology.org/operation_0550' - Label 'DNA substitution modelling'\n",
      "Entity 'http://edamontology.org/operation_0550' - Label 'DNA substitution modelling'\n",
      "Entity 'http://edamontology.org/operation_0550' - Label 'DNA substitution modelling'\n",
      "Entity 'http://edamontology.org/operation_0550' - Label 'DNA substitution modelling'\n",
      "Entity 'http://edamontology.org/operation_0551' - Label 'Phylogenetic tree topology analysis'\n",
      "Entity 'http://edamontology.org/operation_0552' - Label 'Phylogenetic tree bootstrapping'\n",
      "Entity 'http://edamontology.org/operation_0552' - Label 'Phylogenetic tree bootstrapping'\n",
      "Entity 'http://edamontology.org/operation_0553' - Label 'Gene tree construction'\n",
      "Entity 'http://edamontology.org/operation_0553' - Label 'Gene tree construction'\n",
      "Entity 'http://edamontology.org/operation_0553' - Label 'Gene tree construction'\n",
      "Entity 'http://edamontology.org/operation_0554' - Label 'Allele frequency distribution analysis'\n",
      "Entity 'http://edamontology.org/operation_0555' - Label 'Consensus tree construction'\n",
      "Entity 'http://edamontology.org/operation_0555' - Label 'Consensus tree construction'\n",
      "Entity 'http://edamontology.org/operation_0556' - Label 'Phylogenetic sub/super tree construction'\n",
      "Entity 'http://edamontology.org/operation_0557' - Label 'Phylogenetic tree distances calculation'\n",
      "Entity 'http://edamontology.org/operation_0557' - Label 'Phylogenetic tree distances calculation'\n",
      "Entity 'http://edamontology.org/operation_0558' - Label 'Phylogenetic tree annotation'\n",
      "Entity 'http://edamontology.org/operation_0559' - Label 'Immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0559' - Label 'Immunogenicity prediction'\n",
      "Entity 'http://edamontology.org/operation_0560' - Label 'DNA vaccine design'\n",
      "Entity 'http://edamontology.org/operation_0560' - Label 'DNA vaccine design'\n",
      "Entity 'http://edamontology.org/operation_0561' - Label 'Sequence formatting'\n",
      "Entity 'http://edamontology.org/operation_0561' - Label 'Sequence formatting'\n",
      "Entity 'http://edamontology.org/operation_0562' - Label 'Sequence alignment formatting'\n",
      "Entity 'http://edamontology.org/operation_0562' - Label 'Sequence alignment formatting'\n",
      "Entity 'http://edamontology.org/operation_0563' - Label 'Codon usage table formatting'\n",
      "Entity 'http://edamontology.org/operation_0563' - Label 'Codon usage table formatting'\n",
      "Entity 'http://edamontology.org/operation_0564' - Label 'Sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_0564' - Label 'Sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_0564' - Label 'Sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_0564' - Label 'Sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_0565' - Label 'Sequence alignment visualisation'\n",
      "Entity 'http://edamontology.org/operation_0565' - Label 'Sequence alignment visualisation'\n",
      "Entity 'http://edamontology.org/operation_0566' - Label 'Sequence cluster visualisation'\n",
      "Entity 'http://edamontology.org/operation_0566' - Label 'Sequence cluster visualisation'\n",
      "Entity 'http://edamontology.org/operation_0567' - Label 'Phylogenetic tree visualisation'\n",
      "Entity 'http://edamontology.org/operation_0567' - Label 'Phylogenetic tree visualisation'\n",
      "Entity 'http://edamontology.org/operation_0567' - Label 'Phylogenetic tree visualisation'\n",
      "Entity 'http://edamontology.org/operation_0568' - Label 'RNA secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0568' - Label 'RNA secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0569' - Label 'Protein secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0569' - Label 'Protein secondary structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0570' - Label 'Structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0570' - Label 'Structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0570' - Label 'Structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0570' - Label 'Structure visualisation'\n",
      "Entity 'http://edamontology.org/operation_0571' - Label 'Expression data visualisation'\n",
      "Entity 'http://edamontology.org/operation_0571' - Label 'Expression data visualisation'\n",
      "Entity 'http://edamontology.org/operation_0571' - Label 'Expression data visualisation'\n",
      "Entity 'http://edamontology.org/operation_0572' - Label 'Protein interaction network visualisation'\n",
      "Entity 'http://edamontology.org/operation_0572' - Label 'Protein interaction network visualisation'\n",
      "Entity 'http://edamontology.org/operation_0573' - Label 'Map drawing'\n",
      "Entity 'http://edamontology.org/operation_0573' - Label 'Map drawing'\n",
      "Entity 'http://edamontology.org/operation_0574' - Label 'Sequence motif rendering'\n",
      "Entity 'http://edamontology.org/operation_0574' - Label 'Sequence motif rendering'\n",
      "Entity 'http://edamontology.org/operation_0575' - Label 'Restriction map drawing'\n",
      "Entity 'http://edamontology.org/operation_0575' - Label 'Restriction map drawing'\n",
      "Entity 'http://edamontology.org/operation_0577' - Label 'DNA linear map rendering'\n",
      "Entity 'http://edamontology.org/operation_0577' - Label 'DNA linear map rendering'\n",
      "Entity 'http://edamontology.org/operation_0578' - Label 'Plasmid map drawing'\n",
      "Entity 'http://edamontology.org/operation_0579' - Label 'Operon drawing'\n",
      "Entity 'http://edamontology.org/operation_0579' - Label 'Operon drawing'\n",
      "Entity 'http://edamontology.org/operation_1768' - Label 'Nucleic acid folding family identification'\n",
      "Entity 'http://edamontology.org/operation_1768' - Label 'Nucleic acid folding family identification'\n",
      "Entity 'http://edamontology.org/operation_1769' - Label 'Nucleic acid folding energy calculation'\n",
      "Entity 'http://edamontology.org/operation_1769' - Label 'Nucleic acid folding energy calculation'\n",
      "Entity 'http://edamontology.org/operation_1774' - Label 'Annotation retrieval'\n",
      "Entity 'http://edamontology.org/operation_1774' - Label 'Annotation retrieval'\n",
      "Entity 'http://edamontology.org/operation_1777' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/operation_1777' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/operation_1777' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/operation_1778' - Label 'Protein function comparison'\n",
      "Entity 'http://edamontology.org/operation_1778' - Label 'Protein function comparison'\n",
      "Entity 'http://edamontology.org/operation_1778' - Label 'Protein function comparison'\n",
      "Entity 'http://edamontology.org/operation_1780' - Label 'Sequence submission'\n",
      "Entity 'http://edamontology.org/operation_1780' - Label 'Sequence submission'\n",
      "Entity 'http://edamontology.org/operation_1781' - Label 'Gene regulatory network analysis'\n",
      "Entity 'http://edamontology.org/operation_1781' - Label 'Gene regulatory network analysis'\n",
      "Entity 'http://edamontology.org/operation_1781' - Label 'Gene regulatory network analysis'\n",
      "Entity 'http://edamontology.org/operation_1812' - Label 'Parsing'\n",
      "Entity 'http://edamontology.org/operation_1813' - Label 'Sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1813' - Label 'Sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1814' - Label 'Structure retrieval'\n",
      "Entity 'http://edamontology.org/operation_1814' - Label 'Structure retrieval'\n",
      "Entity 'http://edamontology.org/operation_1816' - Label 'Surface rendering'\n",
      "Entity 'http://edamontology.org/operation_1816' - Label 'Surface rendering'\n",
      "Entity 'http://edamontology.org/operation_1817' - Label 'Protein atom surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1817' - Label 'Protein atom surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1818' - Label 'Protein atom surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1818' - Label 'Protein atom surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1819' - Label 'Protein residue surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1819' - Label 'Protein residue surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1820' - Label 'Protein residue surface calculation (vacuum accessible)'\n",
      "Entity 'http://edamontology.org/operation_1820' - Label 'Protein residue surface calculation (vacuum accessible)'\n",
      "Entity 'http://edamontology.org/operation_1821' - Label 'Protein residue surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1821' - Label 'Protein residue surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1822' - Label 'Protein residue surface calculation (vacuum molecular)'\n",
      "Entity 'http://edamontology.org/operation_1822' - Label 'Protein residue surface calculation (vacuum molecular)'\n",
      "Entity 'http://edamontology.org/operation_1823' - Label 'Protein surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1823' - Label 'Protein surface calculation (accessible molecular)'\n",
      "Entity 'http://edamontology.org/operation_1824' - Label 'Protein surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1824' - Label 'Protein surface calculation (accessible)'\n",
      "Entity 'http://edamontology.org/operation_1825' - Label 'Backbone torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1825' - Label 'Backbone torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1826' - Label 'Full torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1826' - Label 'Full torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1827' - Label 'Cysteine torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1827' - Label 'Cysteine torsion angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1828' - Label 'Tau angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1828' - Label 'Tau angle calculation'\n",
      "Entity 'http://edamontology.org/operation_1829' - Label 'Cysteine bridge detection'\n",
      "Entity 'http://edamontology.org/operation_1830' - Label 'Free cysteine detection'\n",
      "Entity 'http://edamontology.org/operation_1831' - Label 'Metal-bound cysteine detection'\n",
      "Entity 'http://edamontology.org/operation_1831' - Label 'Metal-bound cysteine detection'\n",
      "Entity 'http://edamontology.org/operation_1832' - Label 'Residue contact calculation (residue-nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_1832' - Label 'Residue contact calculation (residue-nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_1834' - Label 'Protein-metal contact calculation'\n",
      "Entity 'http://edamontology.org/operation_1835' - Label 'Residue contact calculation (residue-negative ion)'\n",
      "Entity 'http://edamontology.org/operation_1835' - Label 'Residue contact calculation (residue-negative ion)'\n",
      "Entity 'http://edamontology.org/operation_1836' - Label 'Residue bump detection'\n",
      "Entity 'http://edamontology.org/operation_1837' - Label 'Residue symmetry contact calculation'\n",
      "Entity 'http://edamontology.org/operation_1837' - Label 'Residue symmetry contact calculation'\n",
      "Entity 'http://edamontology.org/operation_1838' - Label 'Residue contact calculation (residue-ligand)'\n",
      "Entity 'http://edamontology.org/operation_1838' - Label 'Residue contact calculation (residue-ligand)'\n",
      "Entity 'http://edamontology.org/operation_1839' - Label 'Salt bridge calculation'\n",
      "Entity 'http://edamontology.org/operation_1841' - Label 'Rotamer likelihood prediction'\n",
      "Entity 'http://edamontology.org/operation_1841' - Label 'Rotamer likelihood prediction'\n",
      "Entity 'http://edamontology.org/operation_1842' - Label 'Proline mutation value calculation'\n",
      "Entity 'http://edamontology.org/operation_1842' - Label 'Proline mutation value calculation'\n",
      "Entity 'http://edamontology.org/operation_1843' - Label 'Residue packing validation'\n",
      "Entity 'http://edamontology.org/operation_1844' - Label 'Protein geometry validation'\n",
      "Entity 'http://edamontology.org/operation_1845' - Label 'PDB file sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1845' - Label 'PDB file sequence retrieval'\n",
      "Entity 'http://edamontology.org/operation_1846' - Label 'HET group detection'\n",
      "Entity 'http://edamontology.org/operation_1846' - Label 'HET group detection'\n",
      "Entity 'http://edamontology.org/operation_1847' - Label 'DSSP secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_1847' - Label 'DSSP secondary structure assignment'\n",
      "Entity 'http://edamontology.org/operation_1848' - Label 'Structure formatting'\n",
      "Entity 'http://edamontology.org/operation_1848' - Label 'Structure formatting'\n",
      "Entity 'http://edamontology.org/operation_1850' - Label 'Protein cysteine and disulfide bond assignment'\n",
      "Entity 'http://edamontology.org/operation_1850' - Label 'Protein cysteine and disulfide bond assignment'\n",
      "Entity 'http://edamontology.org/operation_1913' - Label 'Residue validation'\n",
      "Entity 'http://edamontology.org/operation_1913' - Label 'Residue validation'\n",
      "Entity 'http://edamontology.org/operation_1914' - Label 'Structure retrieval (water)'\n",
      "Entity 'http://edamontology.org/operation_1914' - Label 'Structure retrieval (water)'\n",
      "Entity 'http://edamontology.org/operation_2008' - Label 'siRNA duplex prediction'\n",
      "Entity 'http://edamontology.org/operation_2008' - Label 'siRNA duplex prediction'\n",
      "Entity 'http://edamontology.org/operation_2089' - Label 'Sequence alignment refinement'\n",
      "Entity 'http://edamontology.org/operation_2089' - Label 'Sequence alignment refinement'\n",
      "Entity 'http://edamontology.org/operation_2120' - Label 'Listfile processing'\n",
      "Entity 'http://edamontology.org/operation_2120' - Label 'Listfile processing'\n",
      "Entity 'http://edamontology.org/operation_2121' - Label 'Sequence file editing'\n",
      "Entity 'http://edamontology.org/operation_2121' - Label 'Sequence file editing'\n",
      "Entity 'http://edamontology.org/operation_2122' - Label 'Sequence alignment file processing'\n",
      "Entity 'http://edamontology.org/operation_2122' - Label 'Sequence alignment file processing'\n",
      "Entity 'http://edamontology.org/operation_2123' - Label 'Small molecule data processing'\n",
      "Entity 'http://edamontology.org/operation_2123' - Label 'Small molecule data processing'\n",
      "Entity 'http://edamontology.org/operation_2222' - Label 'Data retrieval (ontology annotation)'\n",
      "Entity 'http://edamontology.org/operation_2222' - Label 'Data retrieval (ontology annotation)'\n",
      "Entity 'http://edamontology.org/operation_2224' - Label 'Data retrieval (ontology concept)'\n",
      "Entity 'http://edamontology.org/operation_2224' - Label 'Data retrieval (ontology concept)'\n",
      "Entity 'http://edamontology.org/operation_2233' - Label 'Representative sequence identification'\n",
      "Entity 'http://edamontology.org/operation_2234' - Label 'Structure file processing'\n",
      "Entity 'http://edamontology.org/operation_2234' - Label 'Structure file processing'\n",
      "Entity 'http://edamontology.org/operation_2237' - Label 'Data retrieval (sequence profile)'\n",
      "Entity 'http://edamontology.org/operation_2237' - Label 'Data retrieval (sequence profile)'\n",
      "Entity 'http://edamontology.org/operation_2238' - Label 'Statistical calculation'\n",
      "Entity 'http://edamontology.org/operation_2239' - Label '3D-1D scoring matrix generation'\n",
      "Entity 'http://edamontology.org/operation_2239' - Label '3D-1D scoring matrix generation'\n",
      "Entity 'http://edamontology.org/operation_2239' - Label '3D-1D scoring matrix generation'\n",
      "Entity 'http://edamontology.org/operation_2239' - Label '3D-1D scoring matrix generation'\n",
      "Entity 'http://edamontology.org/operation_2241' - Label 'Transmembrane protein visualisation'\n",
      "Entity 'http://edamontology.org/operation_2241' - Label 'Transmembrane protein visualisation'\n",
      "Entity 'http://edamontology.org/operation_2241' - Label 'Transmembrane protein visualisation'\n",
      "Entity 'http://edamontology.org/operation_2246' - Label 'Demonstration'\n",
      "Entity 'http://edamontology.org/operation_2246' - Label 'Demonstration'\n",
      "Entity 'http://edamontology.org/operation_2264' - Label 'Data retrieval (pathway or network)'\n",
      "Entity 'http://edamontology.org/operation_2264' - Label 'Data retrieval (pathway or network)'\n",
      "Entity 'http://edamontology.org/operation_2265' - Label 'Data retrieval (identifier)'\n",
      "Entity 'http://edamontology.org/operation_2265' - Label 'Data retrieval (identifier)'\n",
      "Entity 'http://edamontology.org/operation_2284' - Label 'Nucleic acid density plotting'\n",
      "Entity 'http://edamontology.org/operation_2284' - Label 'Nucleic acid density plotting'\n",
      "Entity 'http://edamontology.org/operation_2403' - Label 'Sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2403' - Label 'Sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2404' - Label 'Sequence motif analysis'\n",
      "Entity 'http://edamontology.org/operation_2405' - Label 'Protein interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2405' - Label 'Protein interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2406' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2406' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2406' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2407' - Label 'Annotation processing'\n",
      "Entity 'http://edamontology.org/operation_2407' - Label 'Annotation processing'\n",
      "Entity 'http://edamontology.org/operation_2408' - Label 'Sequence feature analysis'\n",
      "Entity 'http://edamontology.org/operation_2408' - Label 'Sequence feature analysis'\n",
      "Entity 'http://edamontology.org/operation_2409' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2409' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2410' - Label 'Gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2410' - Label 'Gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2411' - Label 'Structural profile processing'\n",
      "Entity 'http://edamontology.org/operation_2411' - Label 'Structural profile processing'\n",
      "Entity 'http://edamontology.org/operation_2412' - Label 'Data index processing'\n",
      "Entity 'http://edamontology.org/operation_2412' - Label 'Data index processing'\n",
      "Entity 'http://edamontology.org/operation_2413' - Label 'Sequence profile processing'\n",
      "Entity 'http://edamontology.org/operation_2413' - Label 'Sequence profile processing'\n",
      "Entity 'http://edamontology.org/operation_2414' - Label 'Protein function analysis'\n",
      "Entity 'http://edamontology.org/operation_2414' - Label 'Protein function analysis'\n",
      "Entity 'http://edamontology.org/operation_2415' - Label 'Protein folding analysis'\n",
      "Entity 'http://edamontology.org/operation_2415' - Label 'Protein folding analysis'\n",
      "Entity 'http://edamontology.org/operation_2415' - Label 'Protein folding analysis'\n",
      "Entity 'http://edamontology.org/operation_2415' - Label 'Protein folding analysis'\n",
      "Entity 'http://edamontology.org/operation_2416' - Label 'Protein secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2416' - Label 'Protein secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2416' - Label 'Protein secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2417' - Label 'Physicochemical property data processing'\n",
      "Entity 'http://edamontology.org/operation_2417' - Label 'Physicochemical property data processing'\n",
      "Entity 'http://edamontology.org/operation_2419' - Label 'Primer and probe design'\n",
      "Entity 'http://edamontology.org/operation_2419' - Label 'Primer and probe design'\n",
      "Entity 'http://edamontology.org/operation_2419' - Label 'Primer and probe design'\n",
      "Entity 'http://edamontology.org/operation_2420' - Label 'Operation (typed)'\n",
      "Entity 'http://edamontology.org/operation_2420' - Label 'Operation (typed)'\n",
      "Entity 'http://edamontology.org/operation_2421' - Label 'Database search'\n",
      "Entity 'http://edamontology.org/operation_2421' - Label 'Database search'\n",
      "Entity 'http://edamontology.org/operation_2422' - Label 'Data retrieval'\n",
      "Entity 'http://edamontology.org/operation_2422' - Label 'Data retrieval'\n",
      "Entity 'http://edamontology.org/operation_2422' - Label 'Data retrieval'\n",
      "Entity 'http://edamontology.org/operation_2423' - Label 'Prediction and recognition'\n",
      "Entity 'http://edamontology.org/operation_2424' - Label 'Comparison'\n",
      "Entity 'http://edamontology.org/operation_2425' - Label 'Optimisation and refinement'\n",
      "Entity 'http://edamontology.org/operation_2426' - Label 'Modelling and simulation'\n",
      "Entity 'http://edamontology.org/operation_2426' - Label 'Modelling and simulation'\n",
      "Entity 'http://edamontology.org/operation_2427' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2427' - Label 'Data handling'\n",
      "Entity 'http://edamontology.org/operation_2428' - Label 'Validation'\n",
      "Entity 'http://edamontology.org/operation_2429' - Label 'Mapping'\n",
      "Entity 'http://edamontology.org/operation_2430' - Label 'Design'\n",
      "Entity 'http://edamontology.org/operation_2432' - Label 'Microarray data processing'\n",
      "Entity 'http://edamontology.org/operation_2432' - Label 'Microarray data processing'\n",
      "Entity 'http://edamontology.org/operation_2433' - Label 'Codon usage table processing'\n",
      "Entity 'http://edamontology.org/operation_2433' - Label 'Codon usage table processing'\n",
      "Entity 'http://edamontology.org/operation_2433' - Label 'Codon usage table processing'\n",
      "Entity 'http://edamontology.org/operation_2434' - Label 'Data retrieval (codon usage table)'\n",
      "Entity 'http://edamontology.org/operation_2434' - Label 'Data retrieval (codon usage table)'\n",
      "Entity 'http://edamontology.org/operation_2435' - Label 'Gene expression profile processing'\n",
      "Entity 'http://edamontology.org/operation_2435' - Label 'Gene expression profile processing'\n",
      "Entity 'http://edamontology.org/operation_2436' - Label 'Gene-set enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_2436' - Label 'Gene-set enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_2436' - Label 'Gene-set enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_2436' - Label 'Gene-set enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_2437' - Label 'Gene regulatory network prediction'\n",
      "Entity 'http://edamontology.org/operation_2437' - Label 'Gene regulatory network prediction'\n",
      "Entity 'http://edamontology.org/operation_2437' - Label 'Gene regulatory network prediction'\n",
      "Entity 'http://edamontology.org/operation_2438' - Label 'Pathway or network processing'\n",
      "Entity 'http://edamontology.org/operation_2438' - Label 'Pathway or network processing'\n",
      "Entity 'http://edamontology.org/operation_2438' - Label 'Pathway or network processing'\n",
      "Entity 'http://edamontology.org/operation_2439' - Label 'RNA secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2439' - Label 'RNA secondary structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2440' - Label 'Structure processing (RNA)'\n",
      "Entity 'http://edamontology.org/operation_2440' - Label 'Structure processing (RNA)'\n",
      "Entity 'http://edamontology.org/operation_2441' - Label 'RNA structure prediction'\n",
      "Entity 'http://edamontology.org/operation_2441' - Label 'RNA structure prediction'\n",
      "Entity 'http://edamontology.org/operation_2442' - Label 'DNA structure prediction'\n",
      "Entity 'http://edamontology.org/operation_2442' - Label 'DNA structure prediction'\n",
      "Entity 'http://edamontology.org/operation_2443' - Label 'Phylogenetic tree processing'\n",
      "Entity 'http://edamontology.org/operation_2443' - Label 'Phylogenetic tree processing'\n",
      "Entity 'http://edamontology.org/operation_2444' - Label 'Protein secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2444' - Label 'Protein secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2445' - Label 'Protein interaction network processing'\n",
      "Entity 'http://edamontology.org/operation_2445' - Label 'Protein interaction network processing'\n",
      "Entity 'http://edamontology.org/operation_2446' - Label 'Sequence processing'\n",
      "Entity 'http://edamontology.org/operation_2446' - Label 'Sequence processing'\n",
      "Entity 'http://edamontology.org/operation_2447' - Label 'Sequence processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2447' - Label 'Sequence processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2448' - Label 'Sequence processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2448' - Label 'Sequence processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2451' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2451' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2451' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2451' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2452' - Label 'Sequence cluster processing'\n",
      "Entity 'http://edamontology.org/operation_2452' - Label 'Sequence cluster processing'\n",
      "Entity 'http://edamontology.org/operation_2453' - Label 'Feature table processing'\n",
      "Entity 'http://edamontology.org/operation_2453' - Label 'Feature table processing'\n",
      "Entity 'http://edamontology.org/operation_2454' - Label 'Gene prediction'\n",
      "Entity 'http://edamontology.org/operation_2454' - Label 'Gene prediction'\n",
      "Entity 'http://edamontology.org/operation_2454' - Label 'Gene prediction'\n",
      "Entity 'http://edamontology.org/operation_2456' - Label 'GPCR classification'\n",
      "Entity 'http://edamontology.org/operation_2456' - Label 'GPCR classification'\n",
      "Entity 'http://edamontology.org/operation_2457' - Label 'GPCR coupling selectivity prediction'\n",
      "Entity 'http://edamontology.org/operation_2457' - Label 'GPCR coupling selectivity prediction'\n",
      "Entity 'http://edamontology.org/operation_2459' - Label 'Structure processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2459' - Label 'Structure processing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2460' - Label 'Protein atom surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2460' - Label 'Protein atom surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2461' - Label 'Protein residue surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2461' - Label 'Protein residue surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2462' - Label 'Protein surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2462' - Label 'Protein surface calculation'\n",
      "Entity 'http://edamontology.org/operation_2463' - Label 'Sequence alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2463' - Label 'Sequence alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2464' - Label 'Protein-protein binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2464' - Label 'Protein-protein binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2464' - Label 'Protein-protein binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2465' - Label 'Structure processing'\n",
      "Entity 'http://edamontology.org/operation_2465' - Label 'Structure processing'\n",
      "Entity 'http://edamontology.org/operation_2466' - Label 'Map annotation'\n",
      "Entity 'http://edamontology.org/operation_2466' - Label 'Map annotation'\n",
      "Entity 'http://edamontology.org/operation_2467' - Label 'Data retrieval (protein annotation)'\n",
      "Entity 'http://edamontology.org/operation_2467' - Label 'Data retrieval (protein annotation)'\n",
      "Entity 'http://edamontology.org/operation_2468' - Label 'Data retrieval (phylogenetic tree)'\n",
      "Entity 'http://edamontology.org/operation_2468' - Label 'Data retrieval (phylogenetic tree)'\n",
      "Entity 'http://edamontology.org/operation_2469' - Label 'Data retrieval (protein interaction annotation)'\n",
      "Entity 'http://edamontology.org/operation_2469' - Label 'Data retrieval (protein interaction annotation)'\n",
      "Entity 'http://edamontology.org/operation_2470' - Label 'Data retrieval (protein family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2470' - Label 'Data retrieval (protein family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2471' - Label 'Data retrieval (RNA family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2471' - Label 'Data retrieval (RNA family annotation)'\n",
      "Entity 'http://edamontology.org/operation_2472' - Label 'Data retrieval (gene annotation)'\n",
      "Entity 'http://edamontology.org/operation_2472' - Label 'Data retrieval (gene annotation)'\n",
      "Entity 'http://edamontology.org/operation_2473' - Label 'Data retrieval (genotype and phenotype annotation)'\n",
      "Entity 'http://edamontology.org/operation_2473' - Label 'Data retrieval (genotype and phenotype annotation)'\n",
      "Entity 'http://edamontology.org/operation_2474' - Label 'Protein architecture comparison'\n",
      "Entity 'http://edamontology.org/operation_2474' - Label 'Protein architecture comparison'\n",
      "Entity 'http://edamontology.org/operation_2475' - Label 'Protein architecture recognition'\n",
      "Entity 'http://edamontology.org/operation_2475' - Label 'Protein architecture recognition'\n",
      "Entity 'http://edamontology.org/operation_2475' - Label 'Protein architecture recognition'\n",
      "Entity 'http://edamontology.org/operation_2476' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/operation_2476' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/operation_2476' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/operation_2476' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/operation_2476' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/operation_2476' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/operation_2478' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2478' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2478' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2479' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2479' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2479' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/operation_2480' - Label 'Structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2480' - Label 'Structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2481' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2481' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2481' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/operation_2482' - Label 'Secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2482' - Label 'Secondary structure processing'\n",
      "Entity 'http://edamontology.org/operation_2483' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2483' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2483' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2485' - Label 'Helical wheel drawing'\n",
      "Entity 'http://edamontology.org/operation_2485' - Label 'Helical wheel drawing'\n",
      "Entity 'http://edamontology.org/operation_2486' - Label 'Topology diagram drawing'\n",
      "Entity 'http://edamontology.org/operation_2486' - Label 'Topology diagram drawing'\n",
      "Entity 'http://edamontology.org/operation_2487' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2487' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2487' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2487' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2488' - Label 'Protein secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2488' - Label 'Protein secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2489' - Label 'Subcellular localisation prediction'\n",
      "Entity 'http://edamontology.org/operation_2489' - Label 'Subcellular localisation prediction'\n",
      "Entity 'http://edamontology.org/operation_2490' - Label 'Residue contact calculation (residue-residue)'\n",
      "Entity 'http://edamontology.org/operation_2490' - Label 'Residue contact calculation (residue-residue)'\n",
      "Entity 'http://edamontology.org/operation_2491' - Label 'Hydrogen bond calculation (inter-residue)'\n",
      "Entity 'http://edamontology.org/operation_2491' - Label 'Hydrogen bond calculation (inter-residue)'\n",
      "Entity 'http://edamontology.org/operation_2492' - Label 'Protein interaction prediction'\n",
      "Entity 'http://edamontology.org/operation_2492' - Label 'Protein interaction prediction'\n",
      "Entity 'http://edamontology.org/operation_2492' - Label 'Protein interaction prediction'\n",
      "Entity 'http://edamontology.org/operation_2493' - Label 'Codon usage data processing'\n",
      "Entity 'http://edamontology.org/operation_2493' - Label 'Codon usage data processing'\n",
      "Entity 'http://edamontology.org/operation_2495' - Label 'Expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2495' - Label 'Expression analysis'\n",
      "Entity 'http://edamontology.org/operation_2496' - Label 'Gene regulatory network processing'\n",
      "Entity 'http://edamontology.org/operation_2496' - Label 'Gene regulatory network processing'\n",
      "Entity 'http://edamontology.org/operation_2497' - Label 'Pathway or network analysis'\n",
      "Entity 'http://edamontology.org/operation_2497' - Label 'Pathway or network analysis'\n",
      "Entity 'http://edamontology.org/operation_2497' - Label 'Pathway or network analysis'\n",
      "Entity 'http://edamontology.org/operation_2498' - Label 'Sequencing-based expression profile data analysis'\n",
      "Entity 'http://edamontology.org/operation_2498' - Label 'Sequencing-based expression profile data analysis'\n",
      "Entity 'http://edamontology.org/operation_2499' - Label 'Splicing analysis'\n",
      "Entity 'http://edamontology.org/operation_2499' - Label 'Splicing analysis'\n",
      "Entity 'http://edamontology.org/operation_2499' - Label 'Splicing analysis'\n",
      "Entity 'http://edamontology.org/operation_2499' - Label 'Splicing analysis'\n",
      "Entity 'http://edamontology.org/operation_2500' - Label 'Microarray raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_2500' - Label 'Microarray raw data analysis'\n",
      "Entity 'http://edamontology.org/operation_2501' - Label 'Nucleic acid analysis'\n",
      "Entity 'http://edamontology.org/operation_2501' - Label 'Nucleic acid analysis'\n",
      "Entity 'http://edamontology.org/operation_2501' - Label 'Nucleic acid analysis'\n",
      "Entity 'http://edamontology.org/operation_2502' - Label 'Protein analysis'\n",
      "Entity 'http://edamontology.org/operation_2502' - Label 'Protein analysis'\n",
      "Entity 'http://edamontology.org/operation_2502' - Label 'Protein analysis'\n",
      "Entity 'http://edamontology.org/operation_2503' - Label 'Sequence data processing'\n",
      "Entity 'http://edamontology.org/operation_2503' - Label 'Sequence data processing'\n",
      "Entity 'http://edamontology.org/operation_2504' - Label 'Structural data processing'\n",
      "Entity 'http://edamontology.org/operation_2504' - Label 'Structural data processing'\n",
      "Entity 'http://edamontology.org/operation_2505' - Label 'Text processing'\n",
      "Entity 'http://edamontology.org/operation_2505' - Label 'Text processing'\n",
      "Entity 'http://edamontology.org/operation_2505' - Label 'Text processing'\n",
      "Entity 'http://edamontology.org/operation_2506' - Label 'Protein sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2506' - Label 'Protein sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2507' - Label 'Nucleic acid sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2507' - Label 'Nucleic acid sequence alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2508' - Label 'Nucleic acid sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2508' - Label 'Nucleic acid sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2509' - Label 'Protein sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2509' - Label 'Protein sequence comparison'\n",
      "Entity 'http://edamontology.org/operation_2510' - Label 'DNA back-translation'\n",
      "Entity 'http://edamontology.org/operation_2510' - Label 'DNA back-translation'\n",
      "Entity 'http://edamontology.org/operation_2511' - Label 'Sequence editing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2511' - Label 'Sequence editing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2512' - Label 'Sequence editing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2512' - Label 'Sequence editing (protein)'\n",
      "Entity 'http://edamontology.org/operation_2513' - Label 'Sequence generation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2513' - Label 'Sequence generation (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2514' - Label 'Sequence generation (protein)'\n",
      "Entity 'http://edamontology.org/operation_2514' - Label 'Sequence generation (protein)'\n",
      "Entity 'http://edamontology.org/operation_2515' - Label 'Nucleic acid sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2515' - Label 'Nucleic acid sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2516' - Label 'Protein sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2516' - Label 'Protein sequence visualisation'\n",
      "Entity 'http://edamontology.org/operation_2518' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2518' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2518' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2519' - Label 'Structure processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2519' - Label 'Structure processing (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_2520' - Label 'DNA mapping'\n",
      "Entity 'http://edamontology.org/operation_2520' - Label 'DNA mapping'\n",
      "Entity 'http://edamontology.org/operation_2520' - Label 'DNA mapping'\n",
      "Entity 'http://edamontology.org/operation_2520' - Label 'DNA mapping'\n",
      "Entity 'http://edamontology.org/operation_2521' - Label 'Map data processing'\n",
      "Entity 'http://edamontology.org/operation_2521' - Label 'Map data processing'\n",
      "Entity 'http://edamontology.org/operation_2574' - Label 'Protein hydropathy calculation'\n",
      "Entity 'http://edamontology.org/operation_2574' - Label 'Protein hydropathy calculation'\n",
      "Entity 'http://edamontology.org/operation_2574' - Label 'Protein hydropathy calculation'\n",
      "Entity 'http://edamontology.org/operation_2575' - Label 'Binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2575' - Label 'Binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2575' - Label 'Binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_2844' - Label 'Structure clustering'\n",
      "Entity 'http://edamontology.org/operation_2871' - Label 'Sequence tagged site (STS) mapping'\n",
      "Entity 'http://edamontology.org/operation_2871' - Label 'Sequence tagged site (STS) mapping'\n",
      "Entity 'http://edamontology.org/operation_2928' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/operation_2928' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/operation_2929' - Label 'Protein fragment weight comparison'\n",
      "Entity 'http://edamontology.org/operation_2929' - Label 'Protein fragment weight comparison'\n",
      "Entity 'http://edamontology.org/operation_2930' - Label 'Protein property comparison'\n",
      "Entity 'http://edamontology.org/operation_2930' - Label 'Protein property comparison'\n",
      "Entity 'http://edamontology.org/operation_2931' - Label 'Secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2931' - Label 'Secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2931' - Label 'Secondary structure comparison'\n",
      "Entity 'http://edamontology.org/operation_2932' - Label 'Hopp and Woods plotting'\n",
      "Entity 'http://edamontology.org/operation_2932' - Label 'Hopp and Woods plotting'\n",
      "Entity 'http://edamontology.org/operation_2934' - Label 'Cluster textual view generation'\n",
      "Entity 'http://edamontology.org/operation_2934' - Label 'Cluster textual view generation'\n",
      "Entity 'http://edamontology.org/operation_2935' - Label 'Clustering profile plotting'\n",
      "Entity 'http://edamontology.org/operation_2936' - Label 'Dendrograph plotting'\n",
      "Entity 'http://edamontology.org/operation_2936' - Label 'Dendrograph plotting'\n",
      "Entity 'http://edamontology.org/operation_2937' - Label 'Proximity map plotting'\n",
      "Entity 'http://edamontology.org/operation_2938' - Label 'Dendrogram visualisation'\n",
      "Entity 'http://edamontology.org/operation_2939' - Label 'Principal component visualisation'\n",
      "Entity 'http://edamontology.org/operation_2940' - Label 'Scatter plot plotting'\n",
      "Entity 'http://edamontology.org/operation_2941' - Label 'Whole microarray graph plotting'\n",
      "Entity 'http://edamontology.org/operation_2941' - Label 'Whole microarray graph plotting'\n",
      "Entity 'http://edamontology.org/operation_2942' - Label 'Treemap visualisation'\n",
      "Entity 'http://edamontology.org/operation_2943' - Label 'Box-Whisker plot plotting'\n",
      "Entity 'http://edamontology.org/operation_2944' - Label 'Physical mapping'\n",
      "Entity 'http://edamontology.org/operation_2944' - Label 'Physical mapping'\n",
      "Entity 'http://edamontology.org/operation_2944' - Label 'Physical mapping'\n",
      "Entity 'http://edamontology.org/operation_2945' - Label 'Analysis'\n",
      "Entity 'http://edamontology.org/operation_2946' - Label 'Alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2946' - Label 'Alignment analysis'\n",
      "Entity 'http://edamontology.org/operation_2947' - Label 'Article analysis'\n",
      "Entity 'http://edamontology.org/operation_2947' - Label 'Article analysis'\n",
      "Entity 'http://edamontology.org/operation_2947' - Label 'Article analysis'\n",
      "Entity 'http://edamontology.org/operation_2948' - Label 'Molecular interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2948' - Label 'Molecular interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2949' - Label 'Protein-protein interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2949' - Label 'Protein-protein interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2949' - Label 'Protein-protein interaction analysis'\n",
      "Entity 'http://edamontology.org/operation_2950' - Label 'Residue distance calculation'\n",
      "Entity 'http://edamontology.org/operation_2951' - Label 'Alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2951' - Label 'Alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2951' - Label 'Alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2952' - Label 'Structure alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2952' - Label 'Structure alignment processing'\n",
      "Entity 'http://edamontology.org/operation_2962' - Label 'Codon usage bias calculation'\n",
      "Entity 'http://edamontology.org/operation_2962' - Label 'Codon usage bias calculation'\n",
      "Entity 'http://edamontology.org/operation_2963' - Label 'Codon usage bias plotting'\n",
      "Entity 'http://edamontology.org/operation_2963' - Label 'Codon usage bias plotting'\n",
      "Entity 'http://edamontology.org/operation_2964' - Label 'Codon usage fraction calculation'\n",
      "Entity 'http://edamontology.org/operation_2964' - Label 'Codon usage fraction calculation'\n",
      "Entity 'http://edamontology.org/operation_2990' - Label 'Classification'\n",
      "Entity 'http://edamontology.org/operation_2993' - Label 'Molecular interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2993' - Label 'Molecular interaction data processing'\n",
      "Entity 'http://edamontology.org/operation_2995' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/operation_2995' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/operation_2996' - Label 'Structure classification'\n",
      "Entity 'http://edamontology.org/operation_2996' - Label 'Structure classification'\n",
      "Entity 'http://edamontology.org/operation_2997' - Label 'Protein comparison'\n",
      "Entity 'http://edamontology.org/operation_2998' - Label 'Nucleic acid comparison'\n",
      "Entity 'http://edamontology.org/operation_3023' - Label 'Prediction and recognition (protein)'\n",
      "Entity 'http://edamontology.org/operation_3023' - Label 'Prediction and recognition (protein)'\n",
      "Entity 'http://edamontology.org/operation_3024' - Label 'Prediction and recognition (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_3024' - Label 'Prediction and recognition (nucleic acid)'\n",
      "Entity 'http://edamontology.org/operation_3080' - Label 'Structure editing'\n",
      "Entity 'http://edamontology.org/operation_3080' - Label 'Structure editing'\n",
      "Entity 'http://edamontology.org/operation_3081' - Label 'Sequence alignment editing'\n",
      "Entity 'http://edamontology.org/operation_3083' - Label 'Pathway or network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3083' - Label 'Pathway or network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3083' - Label 'Pathway or network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3084' - Label 'Protein function prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3084' - Label 'Protein function prediction (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3087' - Label 'Protein sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3087' - Label 'Protein sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3088' - Label 'Protein property calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3088' - Label 'Protein property calculation (from sequence)'\n",
      "Entity 'http://edamontology.org/operation_3090' - Label 'Protein feature prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_3090' - Label 'Protein feature prediction (from structure)'\n",
      "Entity 'http://edamontology.org/operation_3092' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/operation_3092' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/operation_3092' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/operation_3092' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/operation_3092' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/operation_3093' - Label 'Database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_3093' - Label 'Database search (by sequence)'\n",
      "Entity 'http://edamontology.org/operation_3094' - Label 'Protein interaction network prediction'\n",
      "Entity 'http://edamontology.org/operation_3094' - Label 'Protein interaction network prediction'\n",
      "Entity 'http://edamontology.org/operation_3095' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/operation_3095' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/operation_3096' - Label 'Editing'\n",
      "Entity 'http://edamontology.org/operation_3180' - Label 'Sequence assembly validation'\n",
      "Entity 'http://edamontology.org/operation_3180' - Label 'Sequence assembly validation'\n",
      "Entity 'http://edamontology.org/operation_3180' - Label 'Sequence assembly validation'\n",
      "Entity 'http://edamontology.org/operation_3180' - Label 'Sequence assembly validation'\n",
      "Entity 'http://edamontology.org/operation_3180' - Label 'Sequence assembly validation'\n",
      "Entity 'http://edamontology.org/operation_3182' - Label 'Genome alignment'\n",
      "Entity 'http://edamontology.org/operation_3182' - Label 'Genome alignment'\n",
      "Entity 'http://edamontology.org/operation_3183' - Label 'Localised reassembly'\n",
      "Entity 'http://edamontology.org/operation_3184' - Label 'Sequence assembly visualisation'\n",
      "Entity 'http://edamontology.org/operation_3185' - Label 'Base-calling'\n",
      "Entity 'http://edamontology.org/operation_3185' - Label 'Base-calling'\n",
      "Entity 'http://edamontology.org/operation_3186' - Label 'Bisulfite mapping'\n",
      "Entity 'http://edamontology.org/operation_3186' - Label 'Bisulfite mapping'\n",
      "Entity 'http://edamontology.org/operation_3187' - Label 'Sequence contamination filtering'\n",
      "Entity 'http://edamontology.org/operation_3189' - Label 'Trim ends'\n",
      "Entity 'http://edamontology.org/operation_3189' - Label 'Trim ends'\n",
      "Entity 'http://edamontology.org/operation_3190' - Label 'Trim vector'\n",
      "Entity 'http://edamontology.org/operation_3190' - Label 'Trim vector'\n",
      "Entity 'http://edamontology.org/operation_3191' - Label 'Trim to reference'\n",
      "Entity 'http://edamontology.org/operation_3191' - Label 'Trim to reference'\n",
      "Entity 'http://edamontology.org/operation_3192' - Label 'Sequence trimming'\n",
      "Entity 'http://edamontology.org/operation_3194' - Label 'Genome feature comparison'\n",
      "Entity 'http://edamontology.org/operation_3194' - Label 'Genome feature comparison'\n",
      "Entity 'http://edamontology.org/operation_3195' - Label 'Sequencing error detection'\n",
      "Entity 'http://edamontology.org/operation_3195' - Label 'Sequencing error detection'\n",
      "Entity 'http://edamontology.org/operation_3196' - Label 'Genotyping'\n",
      "Entity 'http://edamontology.org/operation_3197' - Label 'Genetic variation analysis'\n",
      "Entity 'http://edamontology.org/operation_3198' - Label 'Read mapping'\n",
      "Entity 'http://edamontology.org/operation_3198' - Label 'Read mapping'\n",
      "Entity 'http://edamontology.org/operation_3199' - Label 'Split read mapping'\n",
      "Entity 'http://edamontology.org/operation_3200' - Label 'DNA barcoding'\n",
      "Entity 'http://edamontology.org/operation_3201' - Label 'SNP calling'\n",
      "Entity 'http://edamontology.org/operation_3201' - Label 'SNP calling'\n",
      "Entity 'http://edamontology.org/operation_3202' - Label 'Polymorphism detection'\n",
      "Entity 'http://edamontology.org/operation_3202' - Label 'Polymorphism detection'\n",
      "Entity 'http://edamontology.org/operation_3203' - Label 'Chromatogram visualisation'\n",
      "Entity 'http://edamontology.org/operation_3204' - Label 'Methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3205' - Label 'Methylation calling'\n",
      "Entity 'http://edamontology.org/operation_3205' - Label 'Methylation calling'\n",
      "Entity 'http://edamontology.org/operation_3206' - Label 'Whole genome methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3206' - Label 'Whole genome methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3207' - Label 'Gene methylation analysis'\n",
      "Entity 'http://edamontology.org/operation_3208' - Label 'Genome visualisation'\n",
      "Entity 'http://edamontology.org/operation_3208' - Label 'Genome visualisation'\n",
      "Entity 'http://edamontology.org/operation_3209' - Label 'Genome comparison'\n",
      "Entity 'http://edamontology.org/operation_3209' - Label 'Genome comparison'\n",
      "Entity 'http://edamontology.org/operation_3211' - Label 'Genome indexing'\n",
      "Entity 'http://edamontology.org/operation_3211' - Label 'Genome indexing'\n",
      "Entity 'http://edamontology.org/operation_3211' - Label 'Genome indexing'\n",
      "Entity 'http://edamontology.org/operation_3212' - Label 'Genome indexing (Burrows-Wheeler)'\n",
      "Entity 'http://edamontology.org/operation_3212' - Label 'Genome indexing (Burrows-Wheeler)'\n",
      "Entity 'http://edamontology.org/operation_3213' - Label 'Genome indexing (suffix arrays)'\n",
      "Entity 'http://edamontology.org/operation_3213' - Label 'Genome indexing (suffix arrays)'\n",
      "Entity 'http://edamontology.org/operation_3214' - Label 'Spectral analysis'\n",
      "Entity 'http://edamontology.org/operation_3214' - Label 'Spectral analysis'\n",
      "Entity 'http://edamontology.org/operation_3215' - Label 'Peak detection'\n",
      "Entity 'http://edamontology.org/operation_3215' - Label 'Peak detection'\n",
      "Entity 'http://edamontology.org/operation_3216' - Label 'Scaffolding'\n",
      "Entity 'http://edamontology.org/operation_3216' - Label 'Scaffolding'\n",
      "Entity 'http://edamontology.org/operation_3216' - Label 'Scaffolding'\n",
      "Entity 'http://edamontology.org/operation_3217' - Label 'Scaffold gap completion'\n",
      "Entity 'http://edamontology.org/operation_3218' - Label 'Sequencing quality control'\n",
      "Entity 'http://edamontology.org/operation_3218' - Label 'Sequencing quality control'\n",
      "Entity 'http://edamontology.org/operation_3219' - Label 'Read pre-processing'\n",
      "Entity 'http://edamontology.org/operation_3219' - Label 'Read pre-processing'\n",
      "Entity 'http://edamontology.org/operation_3221' - Label 'Species frequency estimation'\n",
      "Entity 'http://edamontology.org/operation_3221' - Label 'Species frequency estimation'\n",
      "Entity 'http://edamontology.org/operation_3222' - Label 'Peak calling'\n",
      "Entity 'http://edamontology.org/operation_3223' - Label 'Differential gene expression profiling'\n",
      "Entity 'http://edamontology.org/operation_3224' - Label 'Gene set testing'\n",
      "Entity 'http://edamontology.org/operation_3224' - Label 'Gene set testing'\n",
      "Entity 'http://edamontology.org/operation_3225' - Label 'Variant classification'\n",
      "Entity 'http://edamontology.org/operation_3225' - Label 'Variant classification'\n",
      "Entity 'http://edamontology.org/operation_3226' - Label 'Variant prioritisation'\n",
      "Entity 'http://edamontology.org/operation_3227' - Label 'Variant calling'\n",
      "Entity 'http://edamontology.org/operation_3227' - Label 'Variant calling'\n",
      "Entity 'http://edamontology.org/operation_3228' - Label 'Structural variation detection'\n",
      "Entity 'http://edamontology.org/operation_3229' - Label 'Exome assembly'\n",
      "Entity 'http://edamontology.org/operation_3230' - Label 'Read depth analysis'\n",
      "Entity 'http://edamontology.org/operation_3232' - Label 'Gene expression QTL analysis'\n",
      "Entity 'http://edamontology.org/operation_3233' - Label 'Copy number estimation'\n",
      "Entity 'http://edamontology.org/operation_3237' - Label 'Primer removal'\n",
      "Entity 'http://edamontology.org/operation_3258' - Label 'Transcriptome assembly'\n",
      "Entity 'http://edamontology.org/operation_3258' - Label 'Transcriptome assembly'\n",
      "Entity 'http://edamontology.org/operation_3258' - Label 'Transcriptome assembly'\n",
      "Entity 'http://edamontology.org/operation_3259' - Label 'Transcriptome assembly (de novo)'\n",
      "Entity 'http://edamontology.org/operation_3259' - Label 'Transcriptome assembly (de novo)'\n",
      "Entity 'http://edamontology.org/operation_3260' - Label 'Transcriptome assembly (mapping)'\n",
      "Entity 'http://edamontology.org/operation_3260' - Label 'Transcriptome assembly (mapping)'\n",
      "Entity 'http://edamontology.org/operation_3267' - Label 'Sequence coordinate conversion'\n",
      "Entity 'http://edamontology.org/operation_3267' - Label 'Sequence coordinate conversion'\n",
      "Entity 'http://edamontology.org/operation_3267' - Label 'Sequence coordinate conversion'\n",
      "Entity 'http://edamontology.org/operation_3278' - Label 'Document similarity calculation'\n",
      "Entity 'http://edamontology.org/operation_3279' - Label 'Document clustering'\n",
      "Entity 'http://edamontology.org/operation_3279' - Label 'Document clustering'\n",
      "Entity 'http://edamontology.org/operation_3280' - Label 'Named-entity and concept recognition'\n",
      "Entity 'http://edamontology.org/operation_3282' - Label 'ID mapping'\n",
      "Entity 'http://edamontology.org/operation_3282' - Label 'ID mapping'\n",
      "Entity 'http://edamontology.org/operation_3283' - Label 'Anonymisation'\n",
      "Entity 'http://edamontology.org/operation_3289' - Label 'ID retrieval'\n",
      "Entity 'http://edamontology.org/operation_3289' - Label 'ID retrieval'\n",
      "Entity 'http://edamontology.org/operation_3348' - Label 'Sequence checksum generation'\n",
      "Entity 'http://edamontology.org/operation_3348' - Label 'Sequence checksum generation'\n",
      "Entity 'http://edamontology.org/operation_3348' - Label 'Sequence checksum generation'\n",
      "Entity 'http://edamontology.org/operation_3349' - Label 'Bibliography generation'\n",
      "Entity 'http://edamontology.org/operation_3349' - Label 'Bibliography generation'\n",
      "Entity 'http://edamontology.org/operation_3350' - Label 'Protein quaternary structure prediction'\n",
      "Entity 'http://edamontology.org/operation_3351' - Label 'Molecular surface analysis'\n",
      "Entity 'http://edamontology.org/operation_3351' - Label 'Molecular surface analysis'\n",
      "Entity 'http://edamontology.org/operation_3351' - Label 'Molecular surface analysis'\n",
      "Entity 'http://edamontology.org/operation_3352' - Label 'Ontology comparison'\n",
      "Entity 'http://edamontology.org/operation_3353' - Label 'Ontology comparison'\n",
      "Entity 'http://edamontology.org/operation_3353' - Label 'Ontology comparison'\n",
      "Entity 'http://edamontology.org/operation_3357' - Label 'Format detection'\n",
      "Entity 'http://edamontology.org/operation_3357' - Label 'Format detection'\n",
      "Entity 'http://edamontology.org/operation_3359' - Label 'Splitting'\n",
      "Entity 'http://edamontology.org/operation_3429' - Label 'Generation'\n",
      "Entity 'http://edamontology.org/operation_3430' - Label 'Nucleic acid sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3430' - Label 'Nucleic acid sequence feature detection'\n",
      "Entity 'http://edamontology.org/operation_3431' - Label 'Deposition'\n",
      "Entity 'http://edamontology.org/operation_3432' - Label 'Clustering'\n",
      "Entity 'http://edamontology.org/operation_3433' - Label 'Assembly'\n",
      "Entity 'http://edamontology.org/operation_3433' - Label 'Assembly'\n",
      "Entity 'http://edamontology.org/operation_3434' - Label 'Conversion'\n",
      "Entity 'http://edamontology.org/operation_3435' - Label 'Standardisation and normalisation'\n",
      "Entity 'http://edamontology.org/operation_3436' - Label 'Aggregation'\n",
      "Entity 'http://edamontology.org/operation_3437' - Label 'Article comparison'\n",
      "Entity 'http://edamontology.org/operation_3437' - Label 'Article comparison'\n",
      "Entity 'http://edamontology.org/operation_3438' - Label 'Calculation'\n",
      "Entity 'http://edamontology.org/operation_3439' - Label 'Pathway or network prediction'\n",
      "Entity 'http://edamontology.org/operation_3439' - Label 'Pathway or network prediction'\n",
      "Entity 'http://edamontology.org/operation_3439' - Label 'Pathway or network prediction'\n",
      "Entity 'http://edamontology.org/operation_3439' - Label 'Pathway or network prediction'\n",
      "Entity 'http://edamontology.org/operation_3440' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_3440' - Label 'Genome assembly'\n",
      "Entity 'http://edamontology.org/operation_3441' - Label 'Plotting'\n",
      "Entity 'http://edamontology.org/operation_3441' - Label 'Plotting'\n",
      "Entity 'http://edamontology.org/operation_3443' - Label 'Image analysis'\n",
      "Entity 'http://edamontology.org/operation_3443' - Label 'Image analysis'\n",
      "Entity 'http://edamontology.org/operation_3445' - Label 'Diffraction data analysis'\n",
      "Entity 'http://edamontology.org/operation_3446' - Label 'Cell migration analysis'\n",
      "Entity 'http://edamontology.org/operation_3446' - Label 'Cell migration analysis'\n",
      "Entity 'http://edamontology.org/operation_3447' - Label 'Diffraction data reduction'\n",
      "Entity 'http://edamontology.org/operation_3450' - Label 'Neurite measurement'\n",
      "Entity 'http://edamontology.org/operation_3450' - Label 'Neurite measurement'\n",
      "Entity 'http://edamontology.org/operation_3453' - Label 'Diffraction data integration'\n",
      "Entity 'http://edamontology.org/operation_3454' - Label 'Phasing'\n",
      "Entity 'http://edamontology.org/operation_3455' - Label 'Molecular replacement'\n",
      "Entity 'http://edamontology.org/operation_3456' - Label 'Rigid body refinement'\n",
      "Entity 'http://edamontology.org/operation_3457' - Label 'Single particle analysis'\n",
      "Entity 'http://edamontology.org/operation_3457' - Label 'Single particle analysis'\n",
      "Entity 'http://edamontology.org/operation_3457' - Label 'Single particle analysis'\n",
      "Entity 'http://edamontology.org/operation_3458' - Label 'Single particle alignment and classification'\n",
      "Entity 'http://edamontology.org/operation_3458' - Label 'Single particle alignment and classification'\n",
      "Entity 'http://edamontology.org/operation_3459' - Label 'Functional clustering'\n",
      "Entity 'http://edamontology.org/operation_3459' - Label 'Functional clustering'\n",
      "Entity 'http://edamontology.org/operation_3460' - Label 'Taxonomic classification'\n",
      "Entity 'http://edamontology.org/operation_3461' - Label 'Virulence prediction'\n",
      "Entity 'http://edamontology.org/operation_3461' - Label 'Virulence prediction'\n",
      "Entity 'http://edamontology.org/operation_3461' - Label 'Virulence prediction'\n",
      "Entity 'http://edamontology.org/operation_3463' - Label 'Expression correlation analysis'\n",
      "Entity 'http://edamontology.org/operation_3463' - Label 'Expression correlation analysis'\n",
      "Entity 'http://edamontology.org/operation_3465' - Label 'Correlation'\n",
      "Entity 'http://edamontology.org/operation_3465' - Label 'Correlation'\n",
      "Entity 'http://edamontology.org/operation_3469' - Label 'RNA structure covariance model generation'\n",
      "Entity 'http://edamontology.org/operation_3469' - Label 'RNA structure covariance model generation'\n",
      "Entity 'http://edamontology.org/operation_3469' - Label 'RNA structure covariance model generation'\n",
      "Entity 'http://edamontology.org/operation_3470' - Label 'RNA secondary structure prediction (shape-based)'\n",
      "Entity 'http://edamontology.org/operation_3470' - Label 'RNA secondary structure prediction (shape-based)'\n",
      "Entity 'http://edamontology.org/operation_3471' - Label 'Nucleic acid folding prediction (alignment-based)'\n",
      "Entity 'http://edamontology.org/operation_3471' - Label 'Nucleic acid folding prediction (alignment-based)'\n",
      "Entity 'http://edamontology.org/operation_3472' - Label 'k-mer counting'\n",
      "Entity 'http://edamontology.org/operation_3478' - Label 'Phylogenetic reconstruction'\n",
      "Entity 'http://edamontology.org/operation_3478' - Label 'Phylogenetic reconstruction'\n",
      "Entity 'http://edamontology.org/operation_3480' - Label 'Probabilistic data generation'\n",
      "Entity 'http://edamontology.org/operation_3481' - Label 'Probabilistic sequence generation'\n",
      "Entity 'http://edamontology.org/operation_3481' - Label 'Probabilistic sequence generation'\n",
      "Entity 'http://edamontology.org/operation_3482' - Label 'Antimicrobial resistance prediction'\n",
      "Entity 'http://edamontology.org/operation_3482' - Label 'Antimicrobial resistance prediction'\n",
      "Entity 'http://edamontology.org/operation_3482' - Label 'Antimicrobial resistance prediction'\n",
      "Entity 'http://edamontology.org/operation_3501' - Label 'Enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_3501' - Label 'Enrichment analysis'\n",
      "Entity 'http://edamontology.org/operation_3502' - Label 'Chemical similarity enrichment'\n",
      "Entity 'http://edamontology.org/operation_3502' - Label 'Chemical similarity enrichment'\n",
      "Entity 'http://edamontology.org/operation_3503' - Label 'Incident curve plotting'\n",
      "Entity 'http://edamontology.org/operation_3504' - Label 'Variant pattern analysis'\n",
      "Entity 'http://edamontology.org/operation_3545' - Label 'Mathematical modelling'\n",
      "Entity 'http://edamontology.org/operation_3545' - Label 'Mathematical modelling'\n",
      "Entity 'http://edamontology.org/operation_3552' - Label 'Microscope image visualisation'\n",
      "Entity 'http://edamontology.org/operation_3552' - Label 'Microscope image visualisation'\n",
      "Entity 'http://edamontology.org/operation_3553' - Label 'Image annotation'\n",
      "Entity 'http://edamontology.org/operation_3557' - Label 'Imputation'\n",
      "Entity 'http://edamontology.org/operation_3559' - Label 'Ontology visualisation'\n",
      "Entity 'http://edamontology.org/operation_3560' - Label 'Maximum occurrence analysis'\n",
      "Entity 'http://edamontology.org/operation_3561' - Label 'Database comparison'\n",
      "Entity 'http://edamontology.org/operation_3561' - Label 'Database comparison'\n",
      "Entity 'http://edamontology.org/operation_3562' - Label 'Network simulation'\n",
      "Entity 'http://edamontology.org/operation_3562' - Label 'Network simulation'\n",
      "Entity 'http://edamontology.org/operation_3562' - Label 'Network simulation'\n",
      "Entity 'http://edamontology.org/operation_3563' - Label 'RNA-seq read count analysis'\n",
      "Entity 'http://edamontology.org/operation_3564' - Label 'Chemical redundancy removal'\n",
      "Entity 'http://edamontology.org/operation_3565' - Label 'RNA-seq time series data analysis'\n",
      "Entity 'http://edamontology.org/operation_3566' - Label 'Simulated gene expression data generation'\n",
      "Entity 'http://edamontology.org/operation_3625' - Label 'Relation extraction'\n",
      "Entity 'http://edamontology.org/operation_3627' - Label 'Mass spectra calibration'\n",
      "Entity 'http://edamontology.org/operation_3627' - Label 'Mass spectra calibration'\n",
      "Entity 'http://edamontology.org/operation_3628' - Label 'Chromatographic alignment'\n",
      "Entity 'http://edamontology.org/operation_3628' - Label 'Chromatographic alignment'\n",
      "Entity 'http://edamontology.org/operation_3629' - Label 'Deisotoping'\n",
      "Entity 'http://edamontology.org/operation_3629' - Label 'Deisotoping'\n",
      "Entity 'http://edamontology.org/operation_3630' - Label 'Protein quantification'\n",
      "Entity 'http://edamontology.org/operation_3630' - Label 'Protein quantification'\n",
      "Entity 'http://edamontology.org/operation_3630' - Label 'Protein quantification'\n",
      "Entity 'http://edamontology.org/operation_3631' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/operation_3631' - Label 'Peptide identification'\n",
      "Entity 'http://edamontology.org/operation_3632' - Label 'Isotopic distributions calculation'\n",
      "Entity 'http://edamontology.org/operation_3632' - Label 'Isotopic distributions calculation'\n",
      "Entity 'http://edamontology.org/operation_3632' - Label 'Isotopic distributions calculation'\n",
      "Entity 'http://edamontology.org/operation_3633' - Label 'Retention time prediction'\n",
      "Entity 'http://edamontology.org/operation_3634' - Label 'Label-free quantification'\n",
      "Entity 'http://edamontology.org/operation_3635' - Label 'Labeled quantification'\n",
      "Entity 'http://edamontology.org/operation_3636' - Label 'MRM/SRM'\n",
      "Entity 'http://edamontology.org/operation_3637' - Label 'Spectral counting'\n",
      "Entity 'http://edamontology.org/operation_3638' - Label 'SILAC'\n",
      "Entity 'http://edamontology.org/operation_3639' - Label 'iTRAQ'\n",
      "Entity 'http://edamontology.org/operation_3640' - Label '18O labeling'\n",
      "Entity 'http://edamontology.org/operation_3641' - Label 'TMT-tag'\n",
      "Entity 'http://edamontology.org/operation_3642' - Label 'Stable isotope dimethyl labelling'\n",
      "Entity 'http://edamontology.org/operation_3643' - Label 'Tag-based peptide identification'\n",
      "Entity 'http://edamontology.org/operation_3644' - Label 'de Novo sequencing'\n",
      "Entity 'http://edamontology.org/operation_3644' - Label 'de Novo sequencing'\n",
      "Entity 'http://edamontology.org/operation_3645' - Label 'PTM identification'\n",
      "Entity 'http://edamontology.org/operation_3646' - Label 'Peptide database search'\n",
      "Entity 'http://edamontology.org/operation_3646' - Label 'Peptide database search'\n",
      "Entity 'http://edamontology.org/operation_3647' - Label 'Blind peptide database search'\n",
      "Entity 'http://edamontology.org/operation_3648' - Label 'Validation of peptide-spectrum matches'\n",
      "Entity 'http://edamontology.org/operation_3648' - Label 'Validation of peptide-spectrum matches'\n",
      "Entity 'http://edamontology.org/operation_3649' - Label 'Target-Decoy'\n",
      "Entity 'http://edamontology.org/operation_3649' - Label 'Target-Decoy'\n",
      "Entity 'http://edamontology.org/operation_3658' - Label 'Statistical inference'\n",
      "Entity 'http://edamontology.org/operation_3659' - Label 'Regression analysis'\n",
      "Entity 'http://edamontology.org/operation_3660' - Label 'Metabolic network modelling'\n",
      "Entity 'http://edamontology.org/operation_3660' - Label 'Metabolic network modelling'\n",
      "Entity 'http://edamontology.org/operation_3660' - Label 'Metabolic network modelling'\n",
      "Entity 'http://edamontology.org/operation_3661' - Label 'SNP annotation'\n",
      "Entity 'http://edamontology.org/operation_3662' - Label 'Ab-initio gene prediction'\n",
      "Entity 'http://edamontology.org/operation_3663' - Label 'Homology-based gene prediction'\n",
      "Entity 'http://edamontology.org/operation_3664' - Label 'Statistical modelling'\n",
      "Entity 'http://edamontology.org/operation_3666' - Label 'Molecular surface comparison'\n",
      "Entity 'http://edamontology.org/operation_3666' - Label 'Molecular surface comparison'\n",
      "Entity 'http://edamontology.org/operation_3672' - Label 'Gene functional annotation'\n",
      "Entity 'http://edamontology.org/operation_3675' - Label 'Variant filtering'\n",
      "Entity 'http://edamontology.org/operation_3677' - Label 'Differential binding analysis'\n",
      "Entity 'http://edamontology.org/operation_3680' - Label 'RNA-Seq analysis'\n",
      "Entity 'http://edamontology.org/operation_3680' - Label 'RNA-Seq analysis'\n",
      "Entity 'http://edamontology.org/operation_3694' - Label 'Mass spectrum visualisation'\n",
      "Entity 'http://edamontology.org/operation_3695' - Label 'Filtering'\n",
      "Entity 'http://edamontology.org/operation_3703' - Label 'Reference identification'\n",
      "Entity 'http://edamontology.org/operation_3704' - Label 'Ion counting'\n",
      "Entity 'http://edamontology.org/operation_3705' - Label 'Isotope-coded protein label'\n",
      "Entity 'http://edamontology.org/operation_3715' - Label 'Metabolic labeling'\n",
      "Entity 'http://edamontology.org/operation_3730' - Label 'Cross-assembly'\n",
      "Entity 'http://edamontology.org/operation_3731' - Label 'Sample comparison'\n",
      "Entity 'http://edamontology.org/operation_3741' - Label 'Differential protein expression profiling'\n",
      "Entity 'http://edamontology.org/operation_3741' - Label 'Differential protein expression profiling'\n",
      "Entity 'http://edamontology.org/operation_3742' - Label 'Differential gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_3742' - Label 'Differential gene expression analysis'\n",
      "Entity 'http://edamontology.org/operation_3744' - Label 'Multiple sample visualisation'\n",
      "Entity 'http://edamontology.org/operation_3745' - Label 'Ancestral reconstruction'\n",
      "Entity 'http://edamontology.org/operation_3755' - Label 'PTM localisation'\n",
      "Entity 'http://edamontology.org/operation_3760' - Label 'Service management'\n",
      "Entity 'http://edamontology.org/operation_3761' - Label 'Service discovery'\n",
      "Entity 'http://edamontology.org/operation_3762' - Label 'Service composition'\n",
      "Entity 'http://edamontology.org/operation_3763' - Label 'Service invocation'\n",
      "Entity 'http://edamontology.org/operation_3766' - Label 'Weighted correlation network analysis'\n",
      "Entity 'http://edamontology.org/operation_3766' - Label 'Weighted correlation network analysis'\n",
      "Entity 'http://edamontology.org/operation_3767' - Label 'Protein identification'\n",
      "Entity 'http://edamontology.org/operation_3767' - Label 'Protein identification'\n",
      "Entity 'http://edamontology.org/operation_3767' - Label 'Protein identification'\n",
      "Entity 'http://edamontology.org/operation_3778' - Label 'Text annotation'\n",
      "Entity 'http://edamontology.org/operation_3778' - Label 'Text annotation'\n",
      "Entity 'http://edamontology.org/operation_3778' - Label 'Text annotation'\n",
      "Entity 'http://edamontology.org/operation_3778' - Label 'Text annotation'\n",
      "Entity 'http://edamontology.org/operation_3791' - Label 'Collapsing methods'\n",
      "Entity 'http://edamontology.org/operation_3792' - Label 'miRNA expression analysis'\n",
      "Entity 'http://edamontology.org/operation_3793' - Label 'Read summarisation'\n",
      "Entity 'http://edamontology.org/operation_3795' - Label 'In vitro selection'\n",
      "Entity 'http://edamontology.org/operation_3797' - Label 'Rarefaction'\n",
      "Entity 'http://edamontology.org/operation_3798' - Label 'Read binning'\n",
      "Entity 'http://edamontology.org/operation_3799' - Label 'Quantification'\n",
      "Entity 'http://edamontology.org/operation_3800' - Label 'RNA-Seq quantification'\n",
      "Entity 'http://edamontology.org/operation_3801' - Label 'Spectral library search'\n",
      "Entity 'http://edamontology.org/operation_3801' - Label 'Spectral library search'\n",
      "Entity 'http://edamontology.org/operation_3802' - Label 'Sorting'\n",
      "Entity 'http://edamontology.org/operation_3803' - Label 'Natural product identification'\n",
      "Entity 'http://edamontology.org/operation_3809' - Label 'DMR identification'\n",
      "Entity 'http://edamontology.org/operation_3840' - Label 'Multilocus sequence typing'\n",
      "Entity 'http://edamontology.org/operation_3860' - Label 'Spectrum calculation'\n",
      "Entity 'http://edamontology.org/operation_3860' - Label 'Spectrum calculation'\n",
      "Entity 'http://edamontology.org/operation_3860' - Label 'Spectrum calculation'\n",
      "Entity 'http://edamontology.org/operation_3890' - Label 'Trajectory visualization'\n",
      "Entity 'http://edamontology.org/operation_3890' - Label 'Trajectory visualization'\n",
      "Entity 'http://edamontology.org/operation_3891' - Label 'Essential dynamics'\n",
      "Entity 'http://edamontology.org/operation_3891' - Label 'Essential dynamics'\n",
      "Entity 'http://edamontology.org/operation_3893' - Label 'Forcefield parameterisation'\n",
      "Entity 'http://edamontology.org/operation_3893' - Label 'Forcefield parameterisation'\n",
      "Entity 'http://edamontology.org/operation_3893' - Label 'Forcefield parameterisation'\n",
      "Entity 'http://edamontology.org/operation_3894' - Label 'DNA profiling'\n",
      "Entity 'http://edamontology.org/operation_3896' - Label 'Active site prediction'\n",
      "Entity 'http://edamontology.org/operation_3897' - Label 'Ligand-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3898' - Label 'Metal-binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3899' - Label 'Protein-protein docking'\n",
      "Entity 'http://edamontology.org/operation_3899' - Label 'Protein-protein docking'\n",
      "Entity 'http://edamontology.org/operation_3899' - Label 'Protein-protein docking'\n",
      "Entity 'http://edamontology.org/operation_3900' - Label 'DNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3900' - Label 'DNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3900' - Label 'DNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3901' - Label 'RNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3901' - Label 'RNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3901' - Label 'RNA-binding protein prediction'\n",
      "Entity 'http://edamontology.org/operation_3902' - Label 'RNA binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3903' - Label 'DNA binding site prediction'\n",
      "Entity 'http://edamontology.org/operation_3904' - Label 'Protein disorder prediction'\n",
      "Entity 'http://edamontology.org/operation_3904' - Label 'Protein disorder prediction'\n",
      "Entity 'http://edamontology.org/operation_3904' - Label 'Protein disorder prediction'\n",
      "Entity 'http://edamontology.org/operation_3907' - Label 'Information extraction'\n",
      "Entity 'http://edamontology.org/operation_3908' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/operation_3918' - Label 'Genome analysis'\n",
      "Entity 'http://edamontology.org/operation_3918' - Label 'Genome analysis'\n",
      "Entity 'http://edamontology.org/operation_3919' - Label 'Methylation calling'\n",
      "Entity 'http://edamontology.org/operation_3920' - Label 'DNA testing'\n",
      "Entity 'http://edamontology.org/operation_3920' - Label 'DNA testing'\n",
      "Entity 'http://edamontology.org/operation_3921' - Label 'Sequence read processing'\n",
      "Entity 'http://edamontology.org/operation_3925' - Label 'Network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3925' - Label 'Network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3925' - Label 'Network visualisation'\n",
      "Entity 'http://edamontology.org/operation_3926' - Label 'Pathway visualisation'\n",
      "Entity 'http://edamontology.org/operation_3926' - Label 'Pathway visualisation'\n",
      "Entity 'http://edamontology.org/operation_3926' - Label 'Pathway visualisation'\n",
      "Entity 'http://edamontology.org/operation_3927' - Label 'Network analysis'\n",
      "Entity 'http://edamontology.org/operation_3927' - Label 'Network analysis'\n",
      "Entity 'http://edamontology.org/operation_3927' - Label 'Network analysis'\n",
      "Entity 'http://edamontology.org/operation_3928' - Label 'Pathway analysis'\n",
      "Entity 'http://edamontology.org/operation_3928' - Label 'Pathway analysis'\n",
      "Entity 'http://edamontology.org/operation_3928' - Label 'Pathway analysis'\n",
      "Entity 'http://edamontology.org/operation_3929' - Label 'Metabolic pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_3929' - Label 'Metabolic pathway prediction'\n",
      "Entity 'http://edamontology.org/operation_3933' - Label 'Demultiplexing'\n",
      "Entity 'http://edamontology.org/operation_3935' - Label 'Dimensionality reduction'\n",
      "Entity 'http://edamontology.org/operation_3935' - Label 'Dimensionality reduction'\n",
      "Entity 'http://edamontology.org/operation_3935' - Label 'Dimensionality reduction'\n",
      "Entity 'http://edamontology.org/operation_3936' - Label 'Feature selection'\n",
      "Entity 'http://edamontology.org/operation_3936' - Label 'Feature selection'\n",
      "Entity 'http://edamontology.org/operation_3936' - Label 'Feature selection'\n",
      "Entity 'http://edamontology.org/operation_3937' - Label 'Feature extraction'\n",
      "Entity 'http://edamontology.org/operation_3937' - Label 'Feature extraction'\n",
      "Entity 'http://edamontology.org/operation_3937' - Label 'Feature extraction'\n",
      "Entity 'http://edamontology.org/operation_3938' - Label 'Virtual screening'\n",
      "Entity 'http://edamontology.org/operation_3938' - Label 'Virtual screening'\n",
      "Entity 'http://edamontology.org/operation_3938' - Label 'Virtual screening'\n",
      "Entity 'http://edamontology.org/operation_3938' - Label 'Virtual screening'\n",
      "Entity 'http://edamontology.org/operation_3942' - Label 'Tree dating'\n",
      "Entity 'http://edamontology.org/operation_3946' - Label 'Ecological modelling'\n",
      "Entity 'http://edamontology.org/operation_3946' - Label 'Ecological modelling'\n",
      "Entity 'http://edamontology.org/operation_3946' - Label 'Ecological modelling'\n",
      "Entity 'http://edamontology.org/operation_3946' - Label 'Ecological modelling'\n",
      "Entity 'http://edamontology.org/operation_3947' - Label 'Phylogenetic tree reconciliation'\n",
      "Entity 'http://edamontology.org/operation_3950' - Label 'Selection detection'\n",
      "Entity 'http://edamontology.org/operation_3960' - Label 'Principal component analysis'\n",
      "Entity 'http://edamontology.org/operation_3961' - Label 'Copy number variation detection'\n",
      "Entity 'http://edamontology.org/operation_3962' - Label 'Deletion detection'\n",
      "Entity 'http://edamontology.org/operation_3963' - Label 'Duplication detection'\n",
      "Entity 'http://edamontology.org/operation_3964' - Label 'Complex CNV detection'\n",
      "Entity 'http://edamontology.org/operation_3965' - Label 'Amplification detection'\n",
      "Entity 'http://edamontology.org/operation_3968' - Label 'Adhesin prediction'\n",
      "Entity 'http://edamontology.org/operation_3968' - Label 'Adhesin prediction'\n",
      "Entity 'http://edamontology.org/operation_3968' - Label 'Adhesin prediction'\n",
      "Entity 'http://edamontology.org/operation_4008' - Label 'Protein design'\n",
      "Entity 'http://edamontology.org/operation_4009' - Label 'Small molecule design'\n",
      "Entity 'http://edamontology.org/operation_4031' - Label 'Power test'\n",
      "Entity 'http://edamontology.org/operation_4031' - Label 'Power test'\n",
      "Entity 'http://edamontology.org/operation_4031' - Label 'Power test'\n",
      "Entity 'http://edamontology.org/operation_4031' - Label 'Power test'\n",
      "Entity 'http://edamontology.org/operation_4031' - Label 'Power test'\n",
      "Entity 'http://edamontology.org/operation_4031' - Label 'Power test'\n",
      "Entity 'http://edamontology.org/operation_4032' - Label 'DNA modification prediction'\n",
      "Entity 'http://edamontology.org/operation_4032' - Label 'DNA modification prediction'\n",
      "Entity 'http://edamontology.org/operation_4032' - Label 'DNA modification prediction'\n",
      "Entity 'http://edamontology.org/operation_4032' - Label 'DNA modification prediction'\n",
      "Entity 'http://edamontology.org/operation_4032' - Label 'DNA modification prediction'\n",
      "Entity 'http://edamontology.org/operation_4032' - Label 'DNA modification prediction'\n",
      "Entity 'http://edamontology.org/operation_4033' - Label 'Disease transmission analysis'\n",
      "Entity 'http://edamontology.org/operation_4033' - Label 'Disease transmission analysis'\n",
      "Entity 'http://edamontology.org/operation_4033' - Label 'Disease transmission analysis'\n",
      "Entity 'http://edamontology.org/operation_4033' - Label 'Disease transmission analysis'\n",
      "Entity 'http://edamontology.org/operation_4033' - Label 'Disease transmission analysis'\n",
      "Entity 'http://edamontology.org/operation_4033' - Label 'Disease transmission analysis'\n",
      "Entity 'http://edamontology.org/operation_4034' - Label 'Multiple testing correction'\n",
      "Entity 'http://edamontology.org/operation_4034' - Label 'Multiple testing correction'\n",
      "Entity 'http://edamontology.org/operation_4034' - Label 'Multiple testing correction'\n",
      "Entity 'http://edamontology.org/operation_4034' - Label 'Multiple testing correction'\n",
      "Entity 'http://edamontology.org/topic_0077' - Label 'Nucleic acids'\n",
      "Entity 'http://edamontology.org/topic_0078' - Label 'Proteins'\n",
      "Entity 'http://edamontology.org/topic_0079' - Label 'Metabolites'\n",
      "Entity 'http://edamontology.org/topic_0079' - Label 'Metabolites'\n",
      "Entity 'http://edamontology.org/topic_0080' - Label 'Sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0081' - Label 'Structure analysis'\n",
      "Entity 'http://edamontology.org/topic_0082' - Label 'Structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0083' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/topic_0083' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/topic_0083' - Label 'Alignment'\n",
      "Entity 'http://edamontology.org/topic_0084' - Label 'Phylogeny'\n",
      "Entity 'http://edamontology.org/topic_0084' - Label 'Phylogeny'\n",
      "Entity 'http://edamontology.org/topic_0085' - Label 'Functional genomics'\n",
      "Entity 'http://edamontology.org/topic_0085' - Label 'Functional genomics'\n",
      "Entity 'http://edamontology.org/topic_0089' - Label 'Ontology and terminology'\n",
      "Entity 'http://edamontology.org/topic_0090' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/topic_0090' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/topic_0090' - Label 'Information retrieval'\n",
      "Entity 'http://edamontology.org/topic_0091' - Label 'Bioinformatics'\n",
      "Entity 'http://edamontology.org/topic_0092' - Label 'Data visualisation'\n",
      "Entity 'http://edamontology.org/topic_0094' - Label 'Nucleic acid thermodynamics'\n",
      "Entity 'http://edamontology.org/topic_0094' - Label 'Nucleic acid thermodynamics'\n",
      "Entity 'http://edamontology.org/topic_0097' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/topic_0097' - Label 'Nucleic acid structure analysis'\n",
      "Entity 'http://edamontology.org/topic_0099' - Label 'RNA'\n",
      "Entity 'http://edamontology.org/topic_0100' - Label 'Nucleic acid restriction'\n",
      "Entity 'http://edamontology.org/topic_0100' - Label 'Nucleic acid restriction'\n",
      "Entity 'http://edamontology.org/topic_0102' - Label 'Mapping'\n",
      "Entity 'http://edamontology.org/topic_0107' - Label 'Genetic codes and codon usage'\n",
      "Entity 'http://edamontology.org/topic_0107' - Label 'Genetic codes and codon usage'\n",
      "Entity 'http://edamontology.org/topic_0108' - Label 'Protein expression'\n",
      "Entity 'http://edamontology.org/topic_0109' - Label 'Gene finding'\n",
      "Entity 'http://edamontology.org/topic_0109' - Label 'Gene finding'\n",
      "Entity 'http://edamontology.org/topic_0110' - Label 'Transcription'\n",
      "Entity 'http://edamontology.org/topic_0110' - Label 'Transcription'\n",
      "Entity 'http://edamontology.org/topic_0111' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_0111' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_0112' - Label 'Nucleic acid folding'\n",
      "Entity 'http://edamontology.org/topic_0112' - Label 'Nucleic acid folding'\n",
      "Entity 'http://edamontology.org/topic_0114' - Label 'Gene structure'\n",
      "Entity 'http://edamontology.org/topic_0121' - Label 'Proteomics'\n",
      "Entity 'http://edamontology.org/topic_0122' - Label 'Structural genomics'\n",
      "Entity 'http://edamontology.org/topic_0122' - Label 'Structural genomics'\n",
      "Entity 'http://edamontology.org/topic_0123' - Label 'Protein properties'\n",
      "Entity 'http://edamontology.org/topic_0128' - Label 'Protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0128' - Label 'Protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0130' - Label 'Protein folding, stability and design'\n",
      "Entity 'http://edamontology.org/topic_0133' - Label 'Two-dimensional gel electrophoresis'\n",
      "Entity 'http://edamontology.org/topic_0133' - Label 'Two-dimensional gel electrophoresis'\n",
      "Entity 'http://edamontology.org/topic_0134' - Label 'Mass spectrometry'\n",
      "Entity 'http://edamontology.org/topic_0134' - Label 'Mass spectrometry'\n",
      "Entity 'http://edamontology.org/topic_0135' - Label 'Protein microarrays'\n",
      "Entity 'http://edamontology.org/topic_0135' - Label 'Protein microarrays'\n",
      "Entity 'http://edamontology.org/topic_0137' - Label 'Protein hydropathy'\n",
      "Entity 'http://edamontology.org/topic_0137' - Label 'Protein hydropathy'\n",
      "Entity 'http://edamontology.org/topic_0140' - Label 'Protein targeting and localisation'\n",
      "Entity 'http://edamontology.org/topic_0141' - Label 'Protein cleavage sites and proteolysis'\n",
      "Entity 'http://edamontology.org/topic_0141' - Label 'Protein cleavage sites and proteolysis'\n",
      "Entity 'http://edamontology.org/topic_0143' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/topic_0143' - Label 'Protein structure comparison'\n",
      "Entity 'http://edamontology.org/topic_0144' - Label 'Protein residue interactions'\n",
      "Entity 'http://edamontology.org/topic_0144' - Label 'Protein residue interactions'\n",
      "Entity 'http://edamontology.org/topic_0147' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0147' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_0148' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_0148' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_0149' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_0149' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_0150' - Label 'Protein design'\n",
      "Entity 'http://edamontology.org/topic_0150' - Label 'Protein design'\n",
      "Entity 'http://edamontology.org/topic_0151' - Label 'G protein-coupled receptors (GPCR)'\n",
      "Entity 'http://edamontology.org/topic_0151' - Label 'G protein-coupled receptors (GPCR)'\n",
      "Entity 'http://edamontology.org/topic_0152' - Label 'Carbohydrates'\n",
      "Entity 'http://edamontology.org/topic_0153' - Label 'Lipids'\n",
      "Entity 'http://edamontology.org/topic_0154' - Label 'Small molecules'\n",
      "Entity 'http://edamontology.org/topic_0156' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/topic_0156' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/topic_0156' - Label 'Sequence editing'\n",
      "Entity 'http://edamontology.org/topic_0157' - Label 'Sequence composition, complexity and repeats'\n",
      "Entity 'http://edamontology.org/topic_0158' - Label 'Sequence motifs'\n",
      "Entity 'http://edamontology.org/topic_0158' - Label 'Sequence motifs'\n",
      "Entity 'http://edamontology.org/topic_0159' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/topic_0159' - Label 'Sequence comparison'\n",
      "Entity 'http://edamontology.org/topic_0160' - Label 'Sequence sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_0163' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/topic_0163' - Label 'Sequence database search'\n",
      "Entity 'http://edamontology.org/topic_0164' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/topic_0164' - Label 'Sequence clustering'\n",
      "Entity 'http://edamontology.org/topic_0166' - Label 'Protein structural motifs and surfaces'\n",
      "Entity 'http://edamontology.org/topic_0167' - Label 'Structural (3D) profiles'\n",
      "Entity 'http://edamontology.org/topic_0167' - Label 'Structural (3D) profiles'\n",
      "Entity 'http://edamontology.org/topic_0172' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0172' - Label 'Protein structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0173' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0173' - Label 'Nucleic acid structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0174' - Label 'Ab initio structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0174' - Label 'Ab initio structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0175' - Label 'Homology modelling'\n",
      "Entity 'http://edamontology.org/topic_0175' - Label 'Homology modelling'\n",
      "Entity 'http://edamontology.org/topic_0176' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/topic_0176' - Label 'Molecular dynamics'\n",
      "Entity 'http://edamontology.org/topic_0177' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/topic_0177' - Label 'Molecular docking'\n",
      "Entity 'http://edamontology.org/topic_0178' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0178' - Label 'Protein secondary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0179' - Label 'Protein tertiary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0179' - Label 'Protein tertiary structure prediction'\n",
      "Entity 'http://edamontology.org/topic_0180' - Label 'Protein fold recognition'\n",
      "Entity 'http://edamontology.org/topic_0180' - Label 'Protein fold recognition'\n",
      "Entity 'http://edamontology.org/topic_0182' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0182' - Label 'Sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0183' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/topic_0183' - Label 'Structure alignment'\n",
      "Entity 'http://edamontology.org/topic_0184' - Label 'Threading'\n",
      "Entity 'http://edamontology.org/topic_0184' - Label 'Threading'\n",
      "Entity 'http://edamontology.org/topic_0188' - Label 'Sequence profiles and HMMs'\n",
      "Entity 'http://edamontology.org/topic_0188' - Label 'Sequence profiles and HMMs'\n",
      "Entity 'http://edamontology.org/topic_0191' - Label 'Phylogeny reconstruction'\n",
      "Entity 'http://edamontology.org/topic_0191' - Label 'Phylogeny reconstruction'\n",
      "Entity 'http://edamontology.org/topic_0194' - Label 'Phylogenomics'\n",
      "Entity 'http://edamontology.org/topic_0194' - Label 'Phylogenomics'\n",
      "Entity 'http://edamontology.org/topic_0195' - Label 'Virtual PCR'\n",
      "Entity 'http://edamontology.org/topic_0195' - Label 'Virtual PCR'\n",
      "Entity 'http://edamontology.org/topic_0196' - Label 'Sequence assembly'\n",
      "Entity 'http://edamontology.org/topic_0199' - Label 'Genetic variation'\n",
      "Entity 'http://edamontology.org/topic_0199' - Label 'Genetic variation'\n",
      "Entity 'http://edamontology.org/topic_0200' - Label 'Microarrays'\n",
      "Entity 'http://edamontology.org/topic_0200' - Label 'Microarrays'\n",
      "Entity 'http://edamontology.org/topic_0202' - Label 'Pharmacology'\n",
      "Entity 'http://edamontology.org/topic_0203' - Label 'Gene expression'\n",
      "Entity 'http://edamontology.org/topic_0204' - Label 'Gene regulation'\n",
      "Entity 'http://edamontology.org/topic_0208' - Label 'Pharmacogenomics'\n",
      "Entity 'http://edamontology.org/topic_0208' - Label 'Pharmacogenomics'\n",
      "Entity 'http://edamontology.org/topic_0209' - Label 'Medicinal chemistry'\n",
      "Entity 'http://edamontology.org/topic_0209' - Label 'Medicinal chemistry'\n",
      "Entity 'http://edamontology.org/topic_0210' - Label 'Fish'\n",
      "Entity 'http://edamontology.org/topic_0210' - Label 'Fish'\n",
      "Entity 'http://edamontology.org/topic_0211' - Label 'Flies'\n",
      "Entity 'http://edamontology.org/topic_0211' - Label 'Flies'\n",
      "Entity 'http://edamontology.org/topic_0213' - Label 'Mice or rats'\n",
      "Entity 'http://edamontology.org/topic_0213' - Label 'Mice or rats'\n",
      "Entity 'http://edamontology.org/topic_0215' - Label 'Worms'\n",
      "Entity 'http://edamontology.org/topic_0215' - Label 'Worms'\n",
      "Entity 'http://edamontology.org/topic_0217' - Label 'Literature analysis'\n",
      "Entity 'http://edamontology.org/topic_0217' - Label 'Literature analysis'\n",
      "Entity 'http://edamontology.org/topic_0218' - Label 'Natural language processing'\n",
      "Entity 'http://edamontology.org/topic_0218' - Label 'Natural language processing'\n",
      "Entity 'http://edamontology.org/topic_0219' - Label 'Data submission, annotation, and curation'\n",
      "Entity 'http://edamontology.org/topic_0220' - Label 'Document, record and content management'\n",
      "Entity 'http://edamontology.org/topic_0220' - Label 'Document, record and content management'\n",
      "Entity 'http://edamontology.org/topic_0221' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/topic_0221' - Label 'Sequence annotation'\n",
      "Entity 'http://edamontology.org/topic_0222' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/topic_0222' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/topic_0222' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/topic_0222' - Label 'Genome annotation'\n",
      "Entity 'http://edamontology.org/topic_0593' - Label 'NMR'\n",
      "Entity 'http://edamontology.org/topic_0593' - Label 'NMR'\n",
      "Entity 'http://edamontology.org/topic_0594' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/topic_0594' - Label 'Sequence classification'\n",
      "Entity 'http://edamontology.org/topic_0595' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/topic_0595' - Label 'Protein classification'\n",
      "Entity 'http://edamontology.org/topic_0598' - Label 'Sequence motif or profile'\n",
      "Entity 'http://edamontology.org/topic_0598' - Label 'Sequence motif or profile'\n",
      "Entity 'http://edamontology.org/topic_0601' - Label 'Protein modifications'\n",
      "Entity 'http://edamontology.org/topic_0602' - Label 'Molecular interactions, pathways and networks'\n",
      "Entity 'http://edamontology.org/topic_0605' - Label 'Informatics'\n",
      "Entity 'http://edamontology.org/topic_0606' - Label 'Literature data resources'\n",
      "Entity 'http://edamontology.org/topic_0606' - Label 'Literature data resources'\n",
      "Entity 'http://edamontology.org/topic_0607' - Label 'Laboratory information management'\n",
      "Entity 'http://edamontology.org/topic_0608' - Label 'Cell and tissue culture'\n",
      "Entity 'http://edamontology.org/topic_0608' - Label 'Cell and tissue culture'\n",
      "Entity 'http://edamontology.org/topic_0610' - Label 'Ecology'\n",
      "Entity 'http://edamontology.org/topic_0610' - Label 'Ecology'\n",
      "Entity 'http://edamontology.org/topic_0611' - Label 'Electron microscopy'\n",
      "Entity 'http://edamontology.org/topic_0611' - Label 'Electron microscopy'\n",
      "Entity 'http://edamontology.org/topic_0612' - Label 'Cell cycle'\n",
      "Entity 'http://edamontology.org/topic_0612' - Label 'Cell cycle'\n",
      "Entity 'http://edamontology.org/topic_0613' - Label 'Peptides and amino acids'\n",
      "Entity 'http://edamontology.org/topic_0613' - Label 'Peptides and amino acids'\n",
      "Entity 'http://edamontology.org/topic_0616' - Label 'Organelles'\n",
      "Entity 'http://edamontology.org/topic_0616' - Label 'Organelles'\n",
      "Entity 'http://edamontology.org/topic_0617' - Label 'Ribosomes'\n",
      "Entity 'http://edamontology.org/topic_0617' - Label 'Ribosomes'\n",
      "Entity 'http://edamontology.org/topic_0618' - Label 'Scents'\n",
      "Entity 'http://edamontology.org/topic_0618' - Label 'Scents'\n",
      "Entity 'http://edamontology.org/topic_0620' - Label 'Drugs and target structures'\n",
      "Entity 'http://edamontology.org/topic_0620' - Label 'Drugs and target structures'\n",
      "Entity 'http://edamontology.org/topic_0621' - Label 'Model organisms'\n",
      "Entity 'http://edamontology.org/topic_0622' - Label 'Genomics'\n",
      "Entity 'http://edamontology.org/topic_0623' - Label 'Gene and protein families'\n",
      "Entity 'http://edamontology.org/topic_0623' - Label 'Gene and protein families'\n",
      "Entity 'http://edamontology.org/topic_0624' - Label 'Chromosomes'\n",
      "Entity 'http://edamontology.org/topic_0624' - Label 'Chromosomes'\n",
      "Entity 'http://edamontology.org/topic_0625' - Label 'Genotype and phenotype'\n",
      "Entity 'http://edamontology.org/topic_0629' - Label 'Gene expression and microarray'\n",
      "Entity 'http://edamontology.org/topic_0629' - Label 'Gene expression and microarray'\n",
      "Entity 'http://edamontology.org/topic_0632' - Label 'Probes and primers'\n",
      "Entity 'http://edamontology.org/topic_0634' - Label 'Pathology'\n",
      "Entity 'http://edamontology.org/topic_0635' - Label 'Specific protein resources'\n",
      "Entity 'http://edamontology.org/topic_0635' - Label 'Specific protein resources'\n",
      "Entity 'http://edamontology.org/topic_0637' - Label 'Taxonomy'\n",
      "Entity 'http://edamontology.org/topic_0639' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0639' - Label 'Protein sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0640' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0640' - Label 'Nucleic acid sequence analysis'\n",
      "Entity 'http://edamontology.org/topic_0641' - Label 'Repeat sequences'\n",
      "Entity 'http://edamontology.org/topic_0641' - Label 'Repeat sequences'\n",
      "Entity 'http://edamontology.org/topic_0642' - Label 'Low complexity sequences'\n",
      "Entity 'http://edamontology.org/topic_0642' - Label 'Low complexity sequences'\n",
      "Entity 'http://edamontology.org/topic_0644' - Label 'Proteome'\n",
      "Entity 'http://edamontology.org/topic_0644' - Label 'Proteome'\n",
      "Entity 'http://edamontology.org/topic_0654' - Label 'DNA'\n",
      "Entity 'http://edamontology.org/topic_0655' - Label 'Coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0655' - Label 'Coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0659' - Label 'Functional, regulatory and non-coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0659' - Label 'Functional, regulatory and non-coding RNA'\n",
      "Entity 'http://edamontology.org/topic_0660' - Label 'rRNA'\n",
      "Entity 'http://edamontology.org/topic_0660' - Label 'rRNA'\n",
      "Entity 'http://edamontology.org/topic_0663' - Label 'tRNA'\n",
      "Entity 'http://edamontology.org/topic_0663' - Label 'tRNA'\n",
      "Entity 'http://edamontology.org/topic_0694' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/topic_0694' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/topic_0697' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/topic_0697' - Label 'RNA structure'\n",
      "Entity 'http://edamontology.org/topic_0698' - Label 'Protein tertiary structure'\n",
      "Entity 'http://edamontology.org/topic_0698' - Label 'Protein tertiary structure'\n",
      "Entity 'http://edamontology.org/topic_0722' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/topic_0722' - Label 'Nucleic acid classification'\n",
      "Entity 'http://edamontology.org/topic_0724' - Label 'Protein families'\n",
      "Entity 'http://edamontology.org/topic_0724' - Label 'Protein families'\n",
      "Entity 'http://edamontology.org/topic_0736' - Label 'Protein folds and structural domains'\n",
      "Entity 'http://edamontology.org/topic_0740' - Label 'Nucleic acid sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0740' - Label 'Nucleic acid sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0741' - Label 'Protein sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0741' - Label 'Protein sequence alignment'\n",
      "Entity 'http://edamontology.org/topic_0747' - Label 'Nucleic acid sites and features'\n",
      "Entity 'http://edamontology.org/topic_0747' - Label 'Nucleic acid sites and features'\n",
      "Entity 'http://edamontology.org/topic_0747' - Label 'Nucleic acid sites and features'\n",
      "Entity 'http://edamontology.org/topic_0748' - Label 'Protein sites and features'\n",
      "Entity 'http://edamontology.org/topic_0748' - Label 'Protein sites and features'\n",
      "Entity 'http://edamontology.org/topic_0748' - Label 'Protein sites and features'\n",
      "Entity 'http://edamontology.org/topic_0749' - Label 'Transcription factors and regulatory sites'\n",
      "Entity 'http://edamontology.org/topic_0749' - Label 'Transcription factors and regulatory sites'\n",
      "Entity 'http://edamontology.org/topic_0749' - Label 'Transcription factors and regulatory sites'\n",
      "Entity 'http://edamontology.org/topic_0751' - Label 'Phosphorylation sites'\n",
      "Entity 'http://edamontology.org/topic_0751' - Label 'Phosphorylation sites'\n",
      "Entity 'http://edamontology.org/topic_0751' - Label 'Phosphorylation sites'\n",
      "Entity 'http://edamontology.org/topic_0753' - Label 'Metabolic pathways'\n",
      "Entity 'http://edamontology.org/topic_0753' - Label 'Metabolic pathways'\n",
      "Entity 'http://edamontology.org/topic_0754' - Label 'Signaling pathways'\n",
      "Entity 'http://edamontology.org/topic_0754' - Label 'Signaling pathways'\n",
      "Entity 'http://edamontology.org/topic_0767' - Label 'Protein and peptide identification'\n",
      "Entity 'http://edamontology.org/topic_0767' - Label 'Protein and peptide identification'\n",
      "Entity 'http://edamontology.org/topic_0769' - Label 'Workflows'\n",
      "Entity 'http://edamontology.org/topic_0770' - Label 'Data types and objects'\n",
      "Entity 'http://edamontology.org/topic_0770' - Label 'Data types and objects'\n",
      "Entity 'http://edamontology.org/topic_0771' - Label 'Theoretical biology'\n",
      "Entity 'http://edamontology.org/topic_0771' - Label 'Theoretical biology'\n",
      "Entity 'http://edamontology.org/topic_0779' - Label 'Mitochondria'\n",
      "Entity 'http://edamontology.org/topic_0779' - Label 'Mitochondria'\n",
      "Entity 'http://edamontology.org/topic_0780' - Label 'Plant biology'\n",
      "Entity 'http://edamontology.org/topic_0781' - Label 'Virology'\n",
      "Entity 'http://edamontology.org/topic_0782' - Label 'Fungi'\n",
      "Entity 'http://edamontology.org/topic_0782' - Label 'Fungi'\n",
      "Entity 'http://edamontology.org/topic_0783' - Label 'Pathogens'\n",
      "Entity 'http://edamontology.org/topic_0783' - Label 'Pathogens'\n",
      "Entity 'http://edamontology.org/topic_0786' - Label 'Arabidopsis'\n",
      "Entity 'http://edamontology.org/topic_0786' - Label 'Arabidopsis'\n",
      "Entity 'http://edamontology.org/topic_0787' - Label 'Rice'\n",
      "Entity 'http://edamontology.org/topic_0787' - Label 'Rice'\n",
      "Entity 'http://edamontology.org/topic_0796' - Label 'Genetic mapping and linkage'\n",
      "Entity 'http://edamontology.org/topic_0796' - Label 'Genetic mapping and linkage'\n",
      "Entity 'http://edamontology.org/topic_0797' - Label 'Comparative genomics'\n",
      "Entity 'http://edamontology.org/topic_0798' - Label 'Mobile genetic elements'\n",
      "Entity 'http://edamontology.org/topic_0803' - Label 'Human disease'\n",
      "Entity 'http://edamontology.org/topic_0803' - Label 'Human disease'\n",
      "Entity 'http://edamontology.org/topic_0804' - Label 'Immunology'\n",
      "Entity 'http://edamontology.org/topic_0820' - Label 'Membrane and lipoproteins'\n",
      "Entity 'http://edamontology.org/topic_0821' - Label 'Enzymes'\n",
      "Entity 'http://edamontology.org/topic_0922' - Label 'Primers'\n",
      "Entity 'http://edamontology.org/topic_0922' - Label 'Primers'\n",
      "Entity 'http://edamontology.org/topic_1302' - Label 'PolyA signal or sites'\n",
      "Entity 'http://edamontology.org/topic_1302' - Label 'PolyA signal or sites'\n",
      "Entity 'http://edamontology.org/topic_1304' - Label 'CpG island and isochores'\n",
      "Entity 'http://edamontology.org/topic_1304' - Label 'CpG island and isochores'\n",
      "Entity 'http://edamontology.org/topic_1305' - Label 'Restriction sites'\n",
      "Entity 'http://edamontology.org/topic_1305' - Label 'Restriction sites'\n",
      "Entity 'http://edamontology.org/topic_1307' - Label 'Splice sites'\n",
      "Entity 'http://edamontology.org/topic_1307' - Label 'Splice sites'\n",
      "Entity 'http://edamontology.org/topic_1307' - Label 'Splice sites'\n",
      "Entity 'http://edamontology.org/topic_1308' - Label 'Matrix/scaffold attachment sites'\n",
      "Entity 'http://edamontology.org/topic_1308' - Label 'Matrix/scaffold attachment sites'\n",
      "Entity 'http://edamontology.org/topic_1311' - Label 'Operon'\n",
      "Entity 'http://edamontology.org/topic_1311' - Label 'Operon'\n",
      "Entity 'http://edamontology.org/topic_1312' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_1312' - Label 'Promoters'\n",
      "Entity 'http://edamontology.org/topic_1317' - Label 'Structural biology'\n",
      "Entity 'http://edamontology.org/topic_1456' - Label 'Protein membrane regions'\n",
      "Entity 'http://edamontology.org/topic_1456' - Label 'Protein membrane regions'\n",
      "Entity 'http://edamontology.org/topic_1770' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/topic_1770' - Label 'Structure comparison'\n",
      "Entity 'http://edamontology.org/topic_1775' - Label 'Function analysis'\n",
      "Entity 'http://edamontology.org/topic_1811' - Label 'Prokaryotes and Archaea'\n",
      "Entity 'http://edamontology.org/topic_1811' - Label 'Prokaryotes and Archaea'\n",
      "Entity 'http://edamontology.org/topic_2225' - Label 'Protein databases'\n",
      "Entity 'http://edamontology.org/topic_2225' - Label 'Protein databases'\n",
      "Entity 'http://edamontology.org/topic_2226' - Label 'Structure determination'\n",
      "Entity 'http://edamontology.org/topic_2226' - Label 'Structure determination'\n",
      "Entity 'http://edamontology.org/topic_2229' - Label 'Cell biology'\n",
      "Entity 'http://edamontology.org/topic_2230' - Label 'Classification'\n",
      "Entity 'http://edamontology.org/topic_2230' - Label 'Classification'\n",
      "Entity 'http://edamontology.org/topic_2232' - Label 'Lipoproteins'\n",
      "Entity 'http://edamontology.org/topic_2232' - Label 'Lipoproteins'\n",
      "Entity 'http://edamontology.org/topic_2257' - Label 'Phylogeny visualisation'\n",
      "Entity 'http://edamontology.org/topic_2257' - Label 'Phylogeny visualisation'\n",
      "Entity 'http://edamontology.org/topic_2258' - Label 'Cheminformatics'\n",
      "Entity 'http://edamontology.org/topic_2259' - Label 'Systems biology'\n",
      "Entity 'http://edamontology.org/topic_2269' - Label 'Statistics and probability'\n",
      "Entity 'http://edamontology.org/topic_2271' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/topic_2271' - Label 'Structure database search'\n",
      "Entity 'http://edamontology.org/topic_2275' - Label 'Molecular modelling'\n",
      "Entity 'http://edamontology.org/topic_2276' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/topic_2276' - Label 'Protein function prediction'\n",
      "Entity 'http://edamontology.org/topic_2277' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/topic_2277' - Label 'SNP'\n",
      "Entity 'http://edamontology.org/topic_2278' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/topic_2278' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/topic_2278' - Label 'Transmembrane protein prediction'\n",
      "Entity 'http://edamontology.org/topic_2280' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/topic_2280' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/topic_2280' - Label 'Nucleic acid structure comparison'\n",
      "Entity 'http://edamontology.org/topic_2397' - Label 'Exons'\n",
      "Entity 'http://edamontology.org/topic_2397' - Label 'Exons'\n",
      "Entity 'http://edamontology.org/topic_2399' - Label 'Gene transcription'\n",
      "Entity 'http://edamontology.org/topic_2399' - Label 'Gene transcription'\n",
      "Entity 'http://edamontology.org/topic_2533' - Label 'DNA mutation'\n",
      "Entity 'http://edamontology.org/topic_2533' - Label 'DNA mutation'\n",
      "Entity 'http://edamontology.org/topic_2640' - Label 'Oncology'\n",
      "Entity 'http://edamontology.org/topic_2661' - Label 'Toxins and targets'\n",
      "Entity 'http://edamontology.org/topic_2661' - Label 'Toxins and targets'\n",
      "Entity 'http://edamontology.org/topic_2754' - Label 'Introns'\n",
      "Entity 'http://edamontology.org/topic_2754' - Label 'Introns'\n",
      "Entity 'http://edamontology.org/topic_2807' - Label 'Tool topic'\n",
      "Entity 'http://edamontology.org/topic_2807' - Label 'Tool topic'\n",
      "Entity 'http://edamontology.org/topic_2809' - Label 'Study topic'\n",
      "Entity 'http://edamontology.org/topic_2809' - Label 'Study topic'\n",
      "Entity 'http://edamontology.org/topic_2811' - Label 'Nomenclature'\n",
      "Entity 'http://edamontology.org/topic_2811' - Label 'Nomenclature'\n",
      "Entity 'http://edamontology.org/topic_2813' - Label 'Disease genes and proteins'\n",
      "Entity 'http://edamontology.org/topic_2813' - Label 'Disease genes and proteins'\n",
      "Entity 'http://edamontology.org/topic_2814' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/topic_2814' - Label 'Protein structure analysis'\n",
      "Entity 'http://edamontology.org/topic_2815' - Label 'Human biology'\n",
      "Entity 'http://edamontology.org/topic_2816' - Label 'Gene resources'\n",
      "Entity 'http://edamontology.org/topic_2816' - Label 'Gene resources'\n",
      "Entity 'http://edamontology.org/topic_2817' - Label 'Yeast'\n",
      "Entity 'http://edamontology.org/topic_2817' - Label 'Yeast'\n",
      "Entity 'http://edamontology.org/topic_2818' - Label 'Eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2818' - Label 'Eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2819' - Label 'Invertebrates'\n",
      "Entity 'http://edamontology.org/topic_2819' - Label 'Invertebrates'\n",
      "Entity 'http://edamontology.org/topic_2820' - Label 'Vertebrates'\n",
      "Entity 'http://edamontology.org/topic_2820' - Label 'Vertebrates'\n",
      "Entity 'http://edamontology.org/topic_2821' - Label 'Unicellular eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2821' - Label 'Unicellular eukaryotes'\n",
      "Entity 'http://edamontology.org/topic_2826' - Label 'Protein structure alignment'\n",
      "Entity 'http://edamontology.org/topic_2826' - Label 'Protein structure alignment'\n",
      "Entity 'http://edamontology.org/topic_2828' - Label 'X-ray diffraction'\n",
      "Entity 'http://edamontology.org/topic_2828' - Label 'X-ray diffraction'\n",
      "Entity 'http://edamontology.org/topic_2829' - Label 'Ontologies, nomenclature and classification'\n",
      "Entity 'http://edamontology.org/topic_2829' - Label 'Ontologies, nomenclature and classification'\n",
      "Entity 'http://edamontology.org/topic_2830' - Label 'Immunoproteins and antigens'\n",
      "Entity 'http://edamontology.org/topic_2830' - Label 'Immunoproteins and antigens'\n",
      "Entity 'http://edamontology.org/topic_2839' - Label 'Molecules'\n",
      "Entity 'http://edamontology.org/topic_2839' - Label 'Molecules'\n",
      "Entity 'http://edamontology.org/topic_2840' - Label 'Toxicology'\n",
      "Entity 'http://edamontology.org/topic_2840' - Label 'Toxicology'\n",
      "Entity 'http://edamontology.org/topic_2842' - Label 'High-throughput sequencing'\n",
      "Entity 'http://edamontology.org/topic_2842' - Label 'High-throughput sequencing'\n",
      "Entity 'http://edamontology.org/topic_2846' - Label 'Gene regulatory networks'\n",
      "Entity 'http://edamontology.org/topic_2846' - Label 'Gene regulatory networks'\n",
      "Entity 'http://edamontology.org/topic_2847' - Label 'Disease (specific)'\n",
      "Entity 'http://edamontology.org/topic_2847' - Label 'Disease (specific)'\n",
      "Entity 'http://edamontology.org/topic_2867' - Label 'VNTR'\n",
      "Entity 'http://edamontology.org/topic_2867' - Label 'VNTR'\n",
      "Entity 'http://edamontology.org/topic_2868' - Label 'Microsatellites'\n",
      "Entity 'http://edamontology.org/topic_2868' - Label 'Microsatellites'\n",
      "Entity 'http://edamontology.org/topic_2869' - Label 'RFLP'\n",
      "Entity 'http://edamontology.org/topic_2869' - Label 'RFLP'\n",
      "Entity 'http://edamontology.org/topic_2885' - Label 'DNA polymorphism'\n",
      "Entity 'http://edamontology.org/topic_2885' - Label 'DNA polymorphism'\n",
      "Entity 'http://edamontology.org/topic_2953' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/topic_2953' - Label 'Nucleic acid design'\n",
      "Entity 'http://edamontology.org/topic_3032' - Label 'Primer or probe design'\n",
      "Entity 'http://edamontology.org/topic_3032' - Label 'Primer or probe design'\n",
      "Entity 'http://edamontology.org/topic_3038' - Label 'Structure databases'\n",
      "Entity 'http://edamontology.org/topic_3038' - Label 'Structure databases'\n",
      "Entity 'http://edamontology.org/topic_3039' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/topic_3039' - Label 'Nucleic acid structure'\n",
      "Entity 'http://edamontology.org/topic_3041' - Label 'Sequence databases'\n",
      "Entity 'http://edamontology.org/topic_3041' - Label 'Sequence databases'\n",
      "Entity 'http://edamontology.org/topic_3042' - Label 'Nucleic acid sequences'\n",
      "Entity 'http://edamontology.org/topic_3042' - Label 'Nucleic acid sequences'\n",
      "Entity 'http://edamontology.org/topic_3043' - Label 'Protein sequences'\n",
      "Entity 'http://edamontology.org/topic_3043' - Label 'Protein sequences'\n",
      "Entity 'http://edamontology.org/topic_3044' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/topic_3044' - Label 'Protein interaction networks'\n",
      "Entity 'http://edamontology.org/topic_3047' - Label 'Molecular biology'\n",
      "Entity 'http://edamontology.org/topic_3048' - Label 'Mammals'\n",
      "Entity 'http://edamontology.org/topic_3048' - Label 'Mammals'\n",
      "Entity 'http://edamontology.org/topic_3050' - Label 'Biodiversity'\n",
      "Entity 'http://edamontology.org/topic_3052' - Label 'Sequence clusters and classification'\n",
      "Entity 'http://edamontology.org/topic_3052' - Label 'Sequence clusters and classification'\n",
      "Entity 'http://edamontology.org/topic_3053' - Label 'Genetics'\n",
      "Entity 'http://edamontology.org/topic_3055' - Label 'Quantitative genetics'\n",
      "Entity 'http://edamontology.org/topic_3056' - Label 'Population genetics'\n",
      "Entity 'http://edamontology.org/topic_3060' - Label 'Regulatory RNA'\n",
      "Entity 'http://edamontology.org/topic_3060' - Label 'Regulatory RNA'\n",
      "Entity 'http://edamontology.org/topic_3061' - Label 'Documentation and help'\n",
      "Entity 'http://edamontology.org/topic_3061' - Label 'Documentation and help'\n",
      "Entity 'http://edamontology.org/topic_3062' - Label 'Genetic organisation'\n",
      "Entity 'http://edamontology.org/topic_3062' - Label 'Genetic organisation'\n",
      "Entity 'http://edamontology.org/topic_3063' - Label 'Medical informatics'\n",
      "Entity 'http://edamontology.org/topic_3064' - Label 'Developmental biology'\n",
      "Entity 'http://edamontology.org/topic_3065' - Label 'Embryology'\n",
      "Entity 'http://edamontology.org/topic_3067' - Label 'Anatomy'\n",
      "Entity 'http://edamontology.org/topic_3068' - Label 'Literature and language'\n",
      "Entity 'http://edamontology.org/topic_3070' - Label 'Biology'\n",
      "Entity 'http://edamontology.org/topic_3071' - Label 'Data management'\n",
      "Entity 'http://edamontology.org/topic_3072' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/topic_3072' - Label 'Sequence feature detection'\n",
      "Entity 'http://edamontology.org/topic_3073' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/topic_3073' - Label 'Nucleic acid feature detection'\n",
      "Entity 'http://edamontology.org/topic_3074' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/topic_3074' - Label 'Protein feature detection'\n",
      "Entity 'http://edamontology.org/topic_3075' - Label 'Biological system modelling'\n",
      "Entity 'http://edamontology.org/topic_3075' - Label 'Biological system modelling'\n",
      "Entity 'http://edamontology.org/topic_3077' - Label 'Data acquisition'\n",
      "Entity 'http://edamontology.org/topic_3078' - Label 'Genes and proteins resources'\n",
      "Entity 'http://edamontology.org/topic_3078' - Label 'Genes and proteins resources'\n",
      "Entity 'http://edamontology.org/topic_3118' - Label 'Protein topological domains'\n",
      "Entity 'http://edamontology.org/topic_3118' - Label 'Protein topological domains'\n",
      "Entity 'http://edamontology.org/topic_3120' - Label 'Protein variants'\n",
      "Entity 'http://edamontology.org/topic_3123' - Label 'Expression signals'\n",
      "Entity 'http://edamontology.org/topic_3123' - Label 'Expression signals'\n",
      "Entity 'http://edamontology.org/topic_3125' - Label 'DNA binding sites'\n",
      "Entity 'http://edamontology.org/topic_3125' - Label 'DNA binding sites'\n",
      "Entity 'http://edamontology.org/topic_3126' - Label 'Nucleic acid repeats'\n",
      "Entity 'http://edamontology.org/topic_3126' - Label 'Nucleic acid repeats'\n",
      "Entity 'http://edamontology.org/topic_3127' - Label 'DNA replication and recombination'\n",
      "Entity 'http://edamontology.org/topic_3135' - Label 'Signal or transit peptide'\n",
      "Entity 'http://edamontology.org/topic_3135' - Label 'Signal or transit peptide'\n",
      "Entity 'http://edamontology.org/topic_3139' - Label 'Sequence tagged sites'\n",
      "Entity 'http://edamontology.org/topic_3139' - Label 'Sequence tagged sites'\n",
      "Entity 'http://edamontology.org/topic_3168' - Label 'Sequencing'\n",
      "Entity 'http://edamontology.org/topic_3169' - Label 'ChIP-seq'\n",
      "Entity 'http://edamontology.org/topic_3169' - Label 'ChIP-seq'\n",
      "Entity 'http://edamontology.org/topic_3170' - Label 'RNA-Seq'\n",
      "Entity 'http://edamontology.org/topic_3171' - Label 'DNA methylation'\n",
      "Entity 'http://edamontology.org/topic_3171' - Label 'DNA methylation'\n",
      "Entity 'http://edamontology.org/topic_3172' - Label 'Metabolomics'\n",
      "Entity 'http://edamontology.org/topic_3173' - Label 'Epigenomics'\n",
      "Entity 'http://edamontology.org/topic_3173' - Label 'Epigenomics'\n",
      "Entity 'http://edamontology.org/topic_3174' - Label 'Metagenomics'\n",
      "Entity 'http://edamontology.org/topic_3174' - Label 'Metagenomics'\n",
      "Entity 'http://edamontology.org/topic_3175' - Label 'Structural variation'\n",
      "Entity 'http://edamontology.org/topic_3175' - Label 'Structural variation'\n",
      "Entity 'http://edamontology.org/topic_3176' - Label 'DNA packaging'\n",
      "Entity 'http://edamontology.org/topic_3177' - Label 'DNA-Seq'\n",
      "Entity 'http://edamontology.org/topic_3177' - Label 'DNA-Seq'\n",
      "Entity 'http://edamontology.org/topic_3178' - Label 'RNA-Seq alignment'\n",
      "Entity 'http://edamontology.org/topic_3178' - Label 'RNA-Seq alignment'\n",
      "Entity 'http://edamontology.org/topic_3179' - Label 'ChIP-on-chip'\n",
      "Entity 'http://edamontology.org/topic_3263' - Label 'Data security'\n",
      "Entity 'http://edamontology.org/topic_3277' - Label 'Sample collections'\n",
      "Entity 'http://edamontology.org/topic_3292' - Label 'Biochemistry'\n",
      "Entity 'http://edamontology.org/topic_3292' - Label 'Biochemistry'\n",
      "Entity 'http://edamontology.org/topic_3293' - Label 'Phylogenetics'\n",
      "Entity 'http://edamontology.org/topic_3293' - Label 'Phylogenetics'\n",
      "Entity 'http://edamontology.org/topic_3295' - Label 'Epigenetics'\n",
      "Entity 'http://edamontology.org/topic_3297' - Label 'Biotechnology'\n",
      "Entity 'http://edamontology.org/topic_3298' - Label 'Phenomics'\n",
      "Entity 'http://edamontology.org/topic_3298' - Label 'Phenomics'\n",
      "Entity 'http://edamontology.org/topic_3298' - Label 'Phenomics'\n",
      "Entity 'http://edamontology.org/topic_3299' - Label 'Evolutionary biology'\n",
      "Entity 'http://edamontology.org/topic_3300' - Label 'Physiology'\n",
      "Entity 'http://edamontology.org/topic_3301' - Label 'Microbiology'\n",
      "Entity 'http://edamontology.org/topic_3302' - Label 'Parasitology'\n",
      "Entity 'http://edamontology.org/topic_3303' - Label 'Medicine'\n",
      "Entity 'http://edamontology.org/topic_3304' - Label 'Neurobiology'\n",
      "Entity 'http://edamontology.org/topic_3305' - Label 'Public health and epidemiology'\n",
      "Entity 'http://edamontology.org/topic_3306' - Label 'Biophysics'\n",
      "Entity 'http://edamontology.org/topic_3306' - Label 'Biophysics'\n",
      "Entity 'http://edamontology.org/topic_3307' - Label 'Computational biology'\n",
      "Entity 'http://edamontology.org/topic_3308' - Label 'Transcriptomics'\n",
      "Entity 'http://edamontology.org/topic_3308' - Label 'Transcriptomics'\n",
      "Entity 'http://edamontology.org/topic_3314' - Label 'Chemistry'\n",
      "Entity 'http://edamontology.org/topic_3315' - Label 'Mathematics'\n",
      "Entity 'http://edamontology.org/topic_3316' - Label 'Computer science'\n",
      "Entity 'http://edamontology.org/topic_3318' - Label 'Physics'\n",
      "Entity 'http://edamontology.org/topic_3320' - Label 'RNA splicing'\n",
      "Entity 'http://edamontology.org/topic_3320' - Label 'RNA splicing'\n",
      "Entity 'http://edamontology.org/topic_3321' - Label 'Molecular genetics'\n",
      "Entity 'http://edamontology.org/topic_3321' - Label 'Molecular genetics'\n",
      "Entity 'http://edamontology.org/topic_3322' - Label 'Respiratory medicine'\n",
      "Entity 'http://edamontology.org/topic_3323' - Label 'Metabolic disease'\n",
      "Entity 'http://edamontology.org/topic_3323' - Label 'Metabolic disease'\n",
      "Entity 'http://edamontology.org/topic_3324' - Label 'Infectious disease'\n",
      "Entity 'http://edamontology.org/topic_3325' - Label 'Rare diseases'\n",
      "Entity 'http://edamontology.org/topic_3332' - Label 'Computational chemistry'\n",
      "Entity 'http://edamontology.org/topic_3332' - Label 'Computational chemistry'\n",
      "Entity 'http://edamontology.org/topic_3334' - Label 'Neurology'\n",
      "Entity 'http://edamontology.org/topic_3335' - Label 'Cardiology'\n",
      "Entity 'http://edamontology.org/topic_3336' - Label 'Drug discovery'\n",
      "Entity 'http://edamontology.org/topic_3336' - Label 'Drug discovery'\n",
      "Entity 'http://edamontology.org/topic_3337' - Label 'Biobank'\n",
      "Entity 'http://edamontology.org/topic_3338' - Label 'Mouse clinic'\n",
      "Entity 'http://edamontology.org/topic_3339' - Label 'Microbial collection'\n",
      "Entity 'http://edamontology.org/topic_3340' - Label 'Cell culture collection'\n",
      "Entity 'http://edamontology.org/topic_3341' - Label 'Clone library'\n",
      "Entity 'http://edamontology.org/topic_3342' - Label 'Translational medicine'\n",
      "Entity 'http://edamontology.org/topic_3343' - Label 'Compound libraries and screening'\n",
      "Entity 'http://edamontology.org/topic_3344' - Label 'Biomedical science'\n",
      "Entity 'http://edamontology.org/topic_3345' - Label 'Data identity and mapping'\n",
      "Entity 'http://edamontology.org/topic_3346' - Label 'Sequence search'\n",
      "Entity 'http://edamontology.org/topic_3346' - Label 'Sequence search'\n",
      "Entity 'http://edamontology.org/topic_3360' - Label 'Biomarkers'\n",
      "Entity 'http://edamontology.org/topic_3361' - Label 'Laboratory techniques'\n",
      "Entity 'http://edamontology.org/topic_3365' - Label 'Data architecture, analysis and design'\n",
      "Entity 'http://edamontology.org/topic_3366' - Label 'Data integration and warehousing'\n",
      "Entity 'http://edamontology.org/topic_3368' - Label 'Biomaterials'\n",
      "Entity 'http://edamontology.org/topic_3369' - Label 'Chemical biology'\n",
      "Entity 'http://edamontology.org/topic_3369' - Label 'Chemical biology'\n",
      "Entity 'http://edamontology.org/topic_3370' - Label 'Analytical chemistry'\n",
      "Entity 'http://edamontology.org/topic_3371' - Label 'Synthetic chemistry'\n",
      "Entity 'http://edamontology.org/topic_3372' - Label 'Software engineering'\n",
      "Entity 'http://edamontology.org/topic_3373' - Label 'Drug development'\n",
      "Entity 'http://edamontology.org/topic_3374' - Label 'Biotherapeutics'\n",
      "Entity 'http://edamontology.org/topic_3375' - Label 'Drug metabolism'\n",
      "Entity 'http://edamontology.org/topic_3376' - Label 'Medicines research and development'\n",
      "Entity 'http://edamontology.org/topic_3377' - Label 'Safety sciences'\n",
      "Entity 'http://edamontology.org/topic_3378' - Label 'Pharmacovigilance'\n",
      "Entity 'http://edamontology.org/topic_3379' - Label 'Preclinical and clinical studies'\n",
      "Entity 'http://edamontology.org/topic_3379' - Label 'Preclinical and clinical studies'\n",
      "Entity 'http://edamontology.org/topic_3382' - Label 'Imaging'\n",
      "Entity 'http://edamontology.org/topic_3383' - Label 'Bioimaging'\n",
      "Entity 'http://edamontology.org/topic_3384' - Label 'Medical imaging'\n",
      "Entity 'http://edamontology.org/topic_3385' - Label 'Light microscopy'\n",
      "Entity 'http://edamontology.org/topic_3386' - Label 'Laboratory animal science'\n",
      "Entity 'http://edamontology.org/topic_3387' - Label 'Marine biology'\n",
      "Entity 'http://edamontology.org/topic_3388' - Label 'Molecular medicine'\n",
      "Entity 'http://edamontology.org/topic_3390' - Label 'Nutritional science'\n",
      "Entity 'http://edamontology.org/topic_3391' - Label 'Omics'\n",
      "Entity 'http://edamontology.org/topic_3393' - Label 'Quality affairs'\n",
      "Entity 'http://edamontology.org/topic_3394' - Label 'Regulatory affairs'\n",
      "Entity 'http://edamontology.org/topic_3395' - Label 'Regenerative medicine'\n",
      "Entity 'http://edamontology.org/topic_3396' - Label 'Systems medicine'\n",
      "Entity 'http://edamontology.org/topic_3397' - Label 'Veterinary medicine'\n",
      "Entity 'http://edamontology.org/topic_3398' - Label 'Bioengineering'\n",
      "Entity 'http://edamontology.org/topic_3399' - Label 'Geriatric medicine'\n",
      "Entity 'http://edamontology.org/topic_3400' - Label 'Allergy, clinical immunology and immunotherapeutics'\n",
      "Entity 'http://edamontology.org/topic_3401' - Label 'Pain medicine'\n",
      "Entity 'http://edamontology.org/topic_3402' - Label 'Anaesthesiology'\n",
      "Entity 'http://edamontology.org/topic_3403' - Label 'Critical care medicine'\n",
      "Entity 'http://edamontology.org/topic_3404' - Label 'Dermatology'\n",
      "Entity 'http://edamontology.org/topic_3405' - Label 'Dentistry'\n",
      "Entity 'http://edamontology.org/topic_3406' - Label 'Ear, nose and throat medicine'\n",
      "Entity 'http://edamontology.org/topic_3407' - Label 'Endocrinology and metabolism'\n",
      "Entity 'http://edamontology.org/topic_3408' - Label 'Haematology'\n",
      "Entity 'http://edamontology.org/topic_3409' - Label 'Gastroenterology'\n",
      "Entity 'http://edamontology.org/topic_3410' - Label 'Gender medicine'\n",
      "Entity 'http://edamontology.org/topic_3411' - Label 'Gynaecology and obstetrics'\n",
      "Entity 'http://edamontology.org/topic_3412' - Label 'Hepatic and biliary medicine'\n",
      "Entity 'http://edamontology.org/topic_3413' - Label 'Infectious tropical disease'\n",
      "Entity 'http://edamontology.org/topic_3413' - Label 'Infectious tropical disease'\n",
      "Entity 'http://edamontology.org/topic_3414' - Label 'Trauma medicine'\n",
      "Entity 'http://edamontology.org/topic_3415' - Label 'Medical toxicology'\n",
      "Entity 'http://edamontology.org/topic_3416' - Label 'Musculoskeletal medicine'\n",
      "Entity 'http://edamontology.org/topic_3417' - Label 'Ophthalmology'\n",
      "Entity 'http://edamontology.org/topic_3418' - Label 'Paediatrics'\n",
      "Entity 'http://edamontology.org/topic_3419' - Label 'Psychiatry'\n",
      "Entity 'http://edamontology.org/topic_3420' - Label 'Reproductive health'\n",
      "Entity 'http://edamontology.org/topic_3421' - Label 'Surgery'\n",
      "Entity 'http://edamontology.org/topic_3422' - Label 'Urology and nephrology'\n",
      "Entity 'http://edamontology.org/topic_3423' - Label 'Complementary medicine'\n",
      "Entity 'http://edamontology.org/topic_3444' - Label 'MRI'\n",
      "Entity 'http://edamontology.org/topic_3448' - Label 'Neutron diffraction'\n",
      "Entity 'http://edamontology.org/topic_3448' - Label 'Neutron diffraction'\n",
      "Entity 'http://edamontology.org/topic_3452' - Label 'Tomography'\n",
      "Entity 'http://edamontology.org/topic_3473' - Label 'Data mining'\n",
      "Entity 'http://edamontology.org/topic_3474' - Label 'Machine learning'\n",
      "Entity 'http://edamontology.org/topic_3489' - Label 'Database management'\n",
      "Entity 'http://edamontology.org/topic_3500' - Label 'Zoology'\n",
      "Entity 'http://edamontology.org/topic_3510' - Label 'Protein sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3510' - Label 'Protein sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3511' - Label 'Nucleic acid sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3511' - Label 'Nucleic acid sites, features and motifs'\n",
      "Entity 'http://edamontology.org/topic_3512' - Label 'Gene transcripts'\n",
      "Entity 'http://edamontology.org/topic_3512' - Label 'Gene transcripts'\n",
      "Entity 'http://edamontology.org/topic_3514' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_3514' - Label 'Protein-ligand interactions'\n",
      "Entity 'http://edamontology.org/topic_3515' - Label 'Protein-drug interactions'\n",
      "Entity 'http://edamontology.org/topic_3515' - Label 'Protein-drug interactions'\n",
      "Entity 'http://edamontology.org/topic_3516' - Label 'Genotyping experiment'\n",
      "Entity 'http://edamontology.org/topic_3517' - Label 'GWAS study'\n",
      "Entity 'http://edamontology.org/topic_3518' - Label 'Microarray experiment'\n",
      "Entity 'http://edamontology.org/topic_3519' - Label 'PCR experiment'\n",
      "Entity 'http://edamontology.org/topic_3520' - Label 'Proteomics experiment'\n",
      "Entity 'http://edamontology.org/topic_3521' - Label '2D PAGE experiment'\n",
      "Entity 'http://edamontology.org/topic_3521' - Label '2D PAGE experiment'\n",
      "Entity 'http://edamontology.org/topic_3522' - Label 'Northern blot experiment'\n",
      "Entity 'http://edamontology.org/topic_3522' - Label 'Northern blot experiment'\n",
      "Entity 'http://edamontology.org/topic_3523' - Label 'RNAi experiment'\n",
      "Entity 'http://edamontology.org/topic_3524' - Label 'Simulation experiment'\n",
      "Entity 'http://edamontology.org/topic_3525' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_3525' - Label 'Protein-nucleic acid interactions'\n",
      "Entity 'http://edamontology.org/topic_3526' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_3526' - Label 'Protein-protein interactions'\n",
      "Entity 'http://edamontology.org/topic_3527' - Label 'Cellular process pathways'\n",
      "Entity 'http://edamontology.org/topic_3527' - Label 'Cellular process pathways'\n",
      "Entity 'http://edamontology.org/topic_3528' - Label 'Disease pathways'\n",
      "Entity 'http://edamontology.org/topic_3528' - Label 'Disease pathways'\n",
      "Entity 'http://edamontology.org/topic_3529' - Label 'Environmental information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3529' - Label 'Environmental information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3530' - Label 'Genetic information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3530' - Label 'Genetic information processing pathways'\n",
      "Entity 'http://edamontology.org/topic_3531' - Label 'Protein super-secondary structure'\n",
      "Entity 'http://edamontology.org/topic_3531' - Label 'Protein super-secondary structure'\n",
      "Entity 'http://edamontology.org/topic_3533' - Label 'Protein active sites'\n",
      "Entity 'http://edamontology.org/topic_3533' - Label 'Protein active sites'\n",
      "Entity 'http://edamontology.org/topic_3534' - Label 'Protein binding sites'\n",
      "Entity 'http://edamontology.org/topic_3535' - Label 'Protein-nucleic acid binding sites'\n",
      "Entity 'http://edamontology.org/topic_3535' - Label 'Protein-nucleic acid binding sites'\n",
      "Entity 'http://edamontology.org/topic_3536' - Label 'Protein cleavage sites'\n",
      "Entity 'http://edamontology.org/topic_3536' - Label 'Protein cleavage sites'\n",
      "Entity 'http://edamontology.org/topic_3537' - Label 'Protein chemical modifications'\n",
      "Entity 'http://edamontology.org/topic_3537' - Label 'Protein chemical modifications'\n",
      "Entity 'http://edamontology.org/topic_3538' - Label 'Protein disordered structure'\n",
      "Entity 'http://edamontology.org/topic_3539' - Label 'Protein domains'\n",
      "Entity 'http://edamontology.org/topic_3539' - Label 'Protein domains'\n",
      "Entity 'http://edamontology.org/topic_3540' - Label 'Protein key folding sites'\n",
      "Entity 'http://edamontology.org/topic_3540' - Label 'Protein key folding sites'\n",
      "Entity 'http://edamontology.org/topic_3541' - Label 'Protein post-translational modifications'\n",
      "Entity 'http://edamontology.org/topic_3541' - Label 'Protein post-translational modifications'\n",
      "Entity 'http://edamontology.org/topic_3542' - Label 'Protein secondary structure'\n",
      "Entity 'http://edamontology.org/topic_3543' - Label 'Protein sequence repeats'\n",
      "Entity 'http://edamontology.org/topic_3543' - Label 'Protein sequence repeats'\n",
      "Entity 'http://edamontology.org/topic_3544' - Label 'Protein signal peptides'\n",
      "Entity 'http://edamontology.org/topic_3544' - Label 'Protein signal peptides'\n",
      "Entity 'http://edamontology.org/topic_3569' - Label 'Applied mathematics'\n",
      "Entity 'http://edamontology.org/topic_3570' - Label 'Pure mathematics'\n",
      "Entity 'http://edamontology.org/topic_3571' - Label 'Data governance'\n",
      "Entity 'http://edamontology.org/topic_3572' - Label 'Data quality management'\n",
      "Entity 'http://edamontology.org/topic_3573' - Label 'Freshwater biology'\n",
      "Entity 'http://edamontology.org/topic_3574' - Label 'Human genetics'\n",
      "Entity 'http://edamontology.org/topic_3575' - Label 'Tropical medicine'\n",
      "Entity 'http://edamontology.org/topic_3576' - Label 'Medical biotechnology'\n",
      "Entity 'http://edamontology.org/topic_3577' - Label 'Personalised medicine'\n",
      "Entity 'http://edamontology.org/topic_3656' - Label 'Immunoprecipitation experiment'\n",
      "Entity 'http://edamontology.org/topic_3673' - Label 'Whole genome sequencing'\n",
      "Entity 'http://edamontology.org/topic_3674' - Label 'Methylated DNA immunoprecipitation'\n",
      "Entity 'http://edamontology.org/topic_3676' - Label 'Exome sequencing'\n",
      "Entity 'http://edamontology.org/topic_3678' - Label 'Experimental design and studies'\n",
      "Entity 'http://edamontology.org/topic_3679' - Label 'Animal study'\n",
      "Entity 'http://edamontology.org/topic_3679' - Label 'Animal study'\n",
      "Entity 'http://edamontology.org/topic_3697' - Label 'Microbial ecology'\n",
      "Entity 'http://edamontology.org/topic_3697' - Label 'Microbial ecology'\n",
      "Entity 'http://edamontology.org/topic_3794' - Label 'RNA immunoprecipitation'\n",
      "Entity 'http://edamontology.org/topic_3796' - Label 'Population genomics'\n",
      "Entity 'http://edamontology.org/topic_3810' - Label 'Agricultural science'\n",
      "Entity 'http://edamontology.org/topic_3837' - Label 'Metagenomic sequencing'\n",
      "Entity 'http://edamontology.org/topic_3855' - Label 'Environmental sciences'\n",
      "Entity 'http://edamontology.org/topic_3892' - Label 'Biomolecular simulation'\n",
      "Entity 'http://edamontology.org/topic_3895' - Label 'Synthetic biology'\n",
      "Entity 'http://edamontology.org/topic_3895' - Label 'Synthetic biology'\n",
      "Entity 'http://edamontology.org/topic_3912' - Label 'Genetic engineering'\n",
      "Entity 'http://edamontology.org/topic_3912' - Label 'Genetic engineering'\n",
      "Entity 'http://edamontology.org/topic_3922' - Label 'Proteogenomics'\n",
      "Entity 'http://edamontology.org/topic_3923' - Label 'Genome resequencing'\n",
      "Entity 'http://edamontology.org/topic_3930' - Label 'Immunogenetics'\n",
      "Entity 'http://edamontology.org/topic_3930' - Label 'Immunogenetics'\n",
      "Entity 'http://edamontology.org/topic_3931' - Label 'Chemometrics'\n",
      "Entity 'http://edamontology.org/topic_3934' - Label 'Cytometry'\n",
      "Entity 'http://edamontology.org/topic_3939' - Label 'Metabolic engineering'\n",
      "Entity 'http://edamontology.org/topic_3940' - Label 'Chromosome conformation capture'\n",
      "Entity 'http://edamontology.org/topic_3941' - Label 'Metatranscriptomics'\n",
      "Entity 'http://edamontology.org/topic_3941' - Label 'Metatranscriptomics'\n",
      "Entity 'http://edamontology.org/topic_3943' - Label 'Paleogenomics'\n",
      "Entity 'http://edamontology.org/topic_3944' - Label 'Cladistics'\n",
      "Entity 'http://edamontology.org/topic_3944' - Label 'Cladistics'\n",
      "Entity 'http://edamontology.org/topic_3945' - Label 'Molecular evolution'\n",
      "Entity 'http://edamontology.org/topic_3945' - Label 'Molecular evolution'\n",
      "Entity 'http://edamontology.org/topic_3945' - Label 'Molecular evolution'\n",
      "Entity 'http://edamontology.org/topic_3948' - Label 'Immunoinformatics'\n",
      "Entity 'http://edamontology.org/topic_3948' - Label 'Immunoinformatics'\n",
      "Entity 'http://edamontology.org/topic_3954' - Label 'Echography'\n",
      "Entity 'http://edamontology.org/topic_3955' - Label 'Fluxomics'\n",
      "Entity 'http://edamontology.org/topic_3957' - Label 'Protein interaction experiment'\n",
      "Entity 'http://edamontology.org/topic_3958' - Label 'Copy number variation'\n",
      "Entity 'http://edamontology.org/topic_3959' - Label 'Cytogenetics'\n",
      "Entity 'http://edamontology.org/topic_3966' - Label 'Vaccinology'\n",
      "Entity 'http://edamontology.org/topic_3967' - Label 'Immunomics'\n",
      "Entity 'http://edamontology.org/topic_3974' - Label 'Epistasis'\n",
      "Entity 'http://edamontology.org/topic_4010' - Label 'Open science'\n",
      "Entity 'http://edamontology.org/topic_4011' - Label 'Data rescue'\n",
      "Entity 'http://edamontology.org/topic_4012' - Label 'FAIR data'\n",
      "Entity 'http://edamontology.org/topic_4012' - Label 'FAIR data'\n",
      "Entity 'http://edamontology.org/topic_4013' - Label 'Antimicrobial Resistance'\n",
      "Entity 'http://edamontology.org/topic_4013' - Label 'Antimicrobial Resistance'\n",
      "Entity 'http://edamontology.org/topic_4014' - Label 'Electroencephalography'\n",
      "Entity 'http://edamontology.org/topic_4014' - Label 'Electroencephalography'\n",
      "Entity 'http://edamontology.org/topic_4016' - Label 'Electrocardiography'\n",
      "Entity 'http://edamontology.org/topic_4016' - Label 'Electrocardiography'\n",
      "Entity 'http://edamontology.org/topic_4017' - Label 'Cryogenic electron microscopy'\n",
      "Entity 'http://edamontology.org/topic_4019' - Label 'Biosciences'\n",
      "Entity 'http://edamontology.org/topic_4020' - Label 'Carbon cycle'\n",
      "Entity 'http://edamontology.org/topic_4020' - Label 'Carbon cycle'\n",
      "Entity 'http://edamontology.org/topic_4020' - Label 'Carbon cycle'\n",
      "Entity 'http://edamontology.org/topic_4020' - Label 'Carbon cycle'\n",
      "Entity 'http://edamontology.org/topic_4021' - Label 'Multiomics'\n",
      "Entity 'http://edamontology.org/topic_4027' - Label 'Ribosome Profiling'\n",
      "Entity 'http://edamontology.org/topic_4028' - Label 'Single-Cell Sequencing'\n",
      "Entity 'http://edamontology.org/topic_4029' - Label 'Acoustics'\n",
      "Entity 'http://edamontology.org/topic_4030' - Label 'Microfluidics'\n",
      "Entity 'http://edamontology.org/topic_4030' - Label 'Microfluidics'\n",
      "Entity 'http://edamontology.org/topic_4030' - Label 'Microfluidics'\n",
      "Entity 'http://edamontology.org/topic_4037' - Label 'Genomic imprinting'\n",
      "Entity 'http://edamontology.org/topic_4038' - Label 'Metabarcoding'\n",
      "Entity 'http://edamontology.org/topic_4038' - Label 'Metabarcoding'\n",
      "Entity 'http://edamontology.org/topic_4038' - Label 'Metabarcoding'\n",
      "Entity 'http://www.geneontology.org/formats/oboInOwl#ObsoleteClass' - Label 'Obsolete concept (EDAM)'\n",
      "Entity 'http://www.geneontology.org/formats/oboInOwl#ObsoleteClass' - Label 'Obsolete concept (EDAM)'\n"
     ]
    }
   ],
   "source": [
    "query=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT DISTINCT ?entity ?property ?label ?reference WHERE {\n",
    "    {\n",
    "        VALUES ?property { rdfs:subClassOf\n",
    "                           oboInOwl:replacedBy\n",
    "                           oboInOwl:consider  }\n",
    "        ?entity ?property ?reference .\n",
    "        ?entity rdfs:label ?label .\n",
    "        MINUS {?reference rdf:type owl:Restriction . }\n",
    "    }\n",
    "    UNION\n",
    "    {\n",
    "        ?entity rdfs:subClassOf ?restriction . \n",
    "            ?restriction rdf:type owl:Restriction ; \n",
    "            owl:onProperty ?property  ; \n",
    "            owl:someValuesFrom ?reference.\n",
    "        ?entity rdfs:label ?label .\n",
    "    }\n",
    "\n",
    "}\n",
    "ORDER BY ?entity\n",
    "\"\"\"\n",
    "\n",
    "results = kg.query(query)\n",
    "\n",
    "for r in results :\n",
    "    print(f\"Entity '{r['entity']}' - Label '{r['label']}'\") "
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2024-02-13T17:40:27.339868Z",
     "start_time": "2024-02-13T17:40:18.682864Z"
    }
   },
   "execution_count": 52
  },
  {
   "cell_type": "code",
   "outputs": [],
   "source": [],
   "metadata": {
    "collapsed": false
   }
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
